---
figid: PMC11280061__plants-13-01905-g006
figtitle: Auxin-regulated TFRN attenuates lignin biosynthesis
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11280061
filename: PMC11280061__plants-13-01905-g006.jpg
figlink: /pmc/articles/PMC11280061/figure/F6
number: F6
caption: 'Auxin-regulated TFRN attenuates lignin biosynthesis. Green boxes stand for
  the substrates/products of the reactions involved in lignin biosynthesis. Green
  arrows denote chemical transformations. Orange and violet filled boxes present the
  genes encoding lignin biosynthesis enzymes and their regulators, respectively. First,
  p-Coumaric acid is transformed into several p-hydroxycinnamoyl CoA thioesters (highlighted
  in pastel mint), which are interconverted by CCoAOMT and F5H. Second, p-hydroxycinnamoyl
  CoA thioesters are transformed to p-hydroxycinnamyl aldehydes (highlighted in light
  green) by CCR, which are further converted to p-hydroxycinnamyl alcohols (monolignols)
  (highlighted in aquamarine) by CAD. p-hydroxycinnamyl aldehydes and monolignols
  interconversion is provided by COMT and F5H. Finally, monolignols polymerize into
  lignin. UGT72B1 catalyzes monolignol conjugate formation, and BGLU45 and BGLU46
  promote their deconjugation. Laccases, peroxidases and ESB1 enable monolignols’
  polymerization to lignin and its deposition. TFRN is represented by its interconnected
  US–DA part supplemented with the free-standing US NAM, which is a regulator of two
  dDEGs encoding lignin biosynthesis enzymes. The numbers in blue boxes mark the steps
  in the lignin biosynthesis pathway (only the ones affected by USes/DAs are represented).
  Next to a TF, these numbers indicate that the TF regulates lignin biosynthesis genes
  at the corresponding step. A “plus” sign in a blue box denotes entering of an additional
  chain of reactions to the main one. Asterisks point out the genes activated by MYB63
  [57]. US, upregulated suppressor; DA, downregulated activator. Abbreviations for
  the lignin biosynthesis genes (or genes/enzymes if gene and enzyme names differ):
  HYDROXYCINNAMOYL:COA SHIKIMATE HYDROXYCINNAMOYL TRANSFERASE (HCT); CYTOCHROME P450,
  FAMILY 98, SUBFAMILY A, POLYPEPTIDE 3/Coumaric acid 3-hydrolase (CYP98A3/C3H); LYSOPHOSPHOLIPASE
  2/Caffeoyl shikimate esterase (LysoPL2/CSE); CAFFEOYL COA O-METHYLTRANSFERASE (CCoAOMT);
  DHFS-FPGS HOMOLOG B/Folylpolyglutamate synthetase 1 (DFB/FPGS1); METHIONINE OVER-ACCUMULATOR
  3/S-adenosylmethionine synthetase 3 (MTO3/SAMS3); FERULIC ACID 5-HYDROXYLASE 1/FERULATE
  5-HYDROXYLASE 1 (FAH1/F5H1); CYTOCHROME P450, FAMILY 84, SUBFAMILY A, POLYPEPTIDE
  4/FERULATE 5-HYDROXYLASE 2 (CYP84A4/F5H2); O-METHYLTRANSFERASE 1/Caffeic acid O-methyltransferase
  1 (OMT1/COMT1); IRREGULAR XYLEM 4/Cinnamoyl CoA reductase 1 (IRX4/CCR1); CINNAMOYL
  COA REDUCTASE 2 (CCR2); CINNAMYL ALCOHOL DEHYDROGENASE 5 (CAD5); ELICITOR-ACTIVATED
  GENE 3-1/Cinnamyl alcohol dehydrogenase 7 (ELI3-1/CAD7); BETA-GLUCOSIDASE (BGLU);
  UDP-GLYCOSYLTRANSFERASE 72B1 (UGT72B1); IRREGULAR XYLEM 12/Laccase 4 (IRX12/LAC4);
  LACCASE (LAC); PEROXIDASE 72 (PRX72/PER72); ENHANCED SUBERIN 1 (ESB1); MYB DOMAIN
  PROTEIN 63 (MYB63)'
papertitle: Computational Reconstruction of the Transcription Factor Regulatory Network
  Induced by Auxin in Arabidopsis thaliana L.
reftext: Nadya A. Omelyanchuk, et al. Plants (Basel). 2024 Jul;13(14).
year: '2024'
doi: 10.3390/plants13141905
journal_title: Plants
journal_nlm_ta: Plants (Basel)
publisher_name: MDPI
keywords: microarray | transcriptome | auxin | transcription factor | regulatory network
automl_pathway: 0.8854172
figid_alias: PMC11280061__F6
figtype: Figure
redirect_from: /figures/PMC11280061__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11280061__plants-13-01905-g006.html
  '@type': Dataset
  description: 'Auxin-regulated TFRN attenuates lignin biosynthesis. Green boxes stand
    for the substrates/products of the reactions involved in lignin biosynthesis.
    Green arrows denote chemical transformations. Orange and violet filled boxes present
    the genes encoding lignin biosynthesis enzymes and their regulators, respectively.
    First, p-Coumaric acid is transformed into several p-hydroxycinnamoyl CoA thioesters
    (highlighted in pastel mint), which are interconverted by CCoAOMT and F5H. Second,
    p-hydroxycinnamoyl CoA thioesters are transformed to p-hydroxycinnamyl aldehydes
    (highlighted in light green) by CCR, which are further converted to p-hydroxycinnamyl
    alcohols (monolignols) (highlighted in aquamarine) by CAD. p-hydroxycinnamyl aldehydes
    and monolignols interconversion is provided by COMT and F5H. Finally, monolignols
    polymerize into lignin. UGT72B1 catalyzes monolignol conjugate formation, and
    BGLU45 and BGLU46 promote their deconjugation. Laccases, peroxidases and ESB1
    enable monolignols’ polymerization to lignin and its deposition. TFRN is represented
    by its interconnected US–DA part supplemented with the free-standing US NAM, which
    is a regulator of two dDEGs encoding lignin biosynthesis enzymes. The numbers
    in blue boxes mark the steps in the lignin biosynthesis pathway (only the ones
    affected by USes/DAs are represented). Next to a TF, these numbers indicate that
    the TF regulates lignin biosynthesis genes at the corresponding step. A “plus”
    sign in a blue box denotes entering of an additional chain of reactions to the
    main one. Asterisks point out the genes activated by MYB63 [57]. US, upregulated
    suppressor; DA, downregulated activator. Abbreviations for the lignin biosynthesis
    genes (or genes/enzymes if gene and enzyme names differ): HYDROXYCINNAMOYL:COA
    SHIKIMATE HYDROXYCINNAMOYL TRANSFERASE (HCT); CYTOCHROME P450, FAMILY 98, SUBFAMILY
    A, POLYPEPTIDE 3/Coumaric acid 3-hydrolase (CYP98A3/C3H); LYSOPHOSPHOLIPASE 2/Caffeoyl
    shikimate esterase (LysoPL2/CSE); CAFFEOYL COA O-METHYLTRANSFERASE (CCoAOMT);
    DHFS-FPGS HOMOLOG B/Folylpolyglutamate synthetase 1 (DFB/FPGS1); METHIONINE OVER-ACCUMULATOR
    3/S-adenosylmethionine synthetase 3 (MTO3/SAMS3); FERULIC ACID 5-HYDROXYLASE 1/FERULATE
    5-HYDROXYLASE 1 (FAH1/F5H1); CYTOCHROME P450, FAMILY 84, SUBFAMILY A, POLYPEPTIDE
    4/FERULATE 5-HYDROXYLASE 2 (CYP84A4/F5H2); O-METHYLTRANSFERASE 1/Caffeic acid
    O-methyltransferase 1 (OMT1/COMT1); IRREGULAR XYLEM 4/Cinnamoyl CoA reductase
    1 (IRX4/CCR1); CINNAMOYL COA REDUCTASE 2 (CCR2); CINNAMYL ALCOHOL DEHYDROGENASE
    5 (CAD5); ELICITOR-ACTIVATED GENE 3-1/Cinnamyl alcohol dehydrogenase 7 (ELI3-1/CAD7);
    BETA-GLUCOSIDASE (BGLU); UDP-GLYCOSYLTRANSFERASE 72B1 (UGT72B1); IRREGULAR XYLEM
    12/Laccase 4 (IRX12/LAC4); LACCASE (LAC); PEROXIDASE 72 (PRX72/PER72); ENHANCED
    SUBERIN 1 (ESB1); MYB DOMAIN PROTEIN 63 (MYB63)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - STAC3
  - CCT6A
  - BIRC5
  - ELF3
  - DUSP8
  - KRT85
  - FA2H
  - SKP1
  - CAMTA1
  - IRX4
  - CCR2
  - p-Coumaric acid
  - Caffeic acid
  - Methionine
  - BEH2
  - aldehydes
  - Caffeyl alcohol
  - Coniferaldehyde
  - Coniferyl alcohol
  - Sinapaldehyde
  - Sinapyl alcohol
---
