---
figid: PMC11851563__biology-14-00185-g002
figtitle: Beta-Catenin and DNA repair pathways
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11851563
filename: biology-14-00185-g002.jpg
figlink: /pmc/articles/PMC11851563/figure/F2/
number: F2
caption: 'β-Catenin and DNA repair pathways. (a) DNA damage response (DDR). (a) DDR
  involves a signaling cascade from DNA damage sites, which ultimately activates the
  cell-cycle checkpoints to allow for repair or triggers cell death. ATM detects DNA
  double-strand breaks (DSBs), and ATR DNA single-strand breaks (SSBs). MRN complex
  is another important sensor of DNA damage. γH2AX amplifies the DDR signaling. The
  downstream effectors CHK1 and CHK2 phosphorylate p53, and p53 are important to activate
  the DNA repair or to initiate apoptosis. The specific participation of β-catenin
  and GSK3β in DDR is indicated in the figure. (b) Direct repair. This mechanism specializes
  in correcting the DNA damage induced by alkylating agents. The most frequent lesion
  is O6-methylguanine (O6-meG), recognized by O6-meG-DNA-methyltransferase (MGMT)
  and regulated by β-catenin. (c) Base excision repair (BER). BER is the most versatile
  DNA repair system for maintaining genome stability. The first step consists of the
  recognition step by specific DNA glycosylases. Two BER pathways are known: short
  patch-BER and long patch-BER. The DNA glycosylase excises the DNA damage, leaving
  an AP site (apurinic/apyrimidinic). AP endonuclease incises the AP site creating
  a nick, the DNA polymerase β replaces the damaged base in the short patch-BER. However,
  if the lesion consists of 2–11 nucleotides, PCNA and FEN1 endonuclease, which adds
  to the process known as long patch-BER. The specific participation of β-catenin
  in DDR is indicated in the figure, particularly with glycosylases. (d) Nucleotide
  excision repair (NER). NER is a multi-step system that recognizes a wide spectrum
  of lesions causing important distortions in DNA and involves the action of more
  than 30 different proteins. The global genome NER (GG-NER) identifies lesions throughout
  the genome. XPC-HR23B detects the lesion and additional factors such as TFIIH, XPB,
  and XPD are essential to unwinding the double helix. XPA-RPA, XPG, and XPF-ERCC1
  nucleases are needed to initiate the incision 15-24 nucleotides away. Finally, the
  gap filling is completed by Polδ/ε, RFC, PCNA, LIG1, LIG3/XRCC1. The specific participation
  of β-catenin, TCF, and GSK3β in NER are indicated in the figure. (e) Mismatch repair
  (MMR). The heterodimer MSH2-MSH6 detects base–base mismatches and insertion–deletion
  loops (IDLs). MSH2-MSH3 recognizes larger IDLs. The heterodimer MLH1-PMS2 binds
  to the recognition complex and initiates the repair with the components EXO1, PCNA,
  RFC, RPA, Polδ/ε, and LIG1. The dotted lines indicate the participation of β-catenin.
  (f) Double-strand break repair (DSB repair). DSBs can be repaired by homologous
  recombination (HR) and non-homologous-end joining (NHEJ). HR acts after replication
  and begins when the MRN complex (MRE11-NBS1-RAD50) recognizes the DSB. BRCA1 binds
  to p53, BRCA2, RAD51, and the MRN complex. Resection of the 5′ DNA on either side
  of the DSB is BRCA1-dependent, resulting in the single-stranded DNA (ssDNA) exposure.
  BRCA2 localizes RAD51 to ssDNA regions to form a nucleoprotein filament capable
  of invading and binding homologous sequences. Then, DNA polymerases δ/ε and DNA
  ligase 1 end the repair of the lesion. The NHEJ system joins the two broken ends
  without homology. The heterodimer Ku70/Ku80 protects the DNA from exonuclease digestion
  and DNA-PK (DNA-dependent protein kinase). Artemis preserves the ends. DNA ligation
  is executed by the DNA ligase 4 (LIG4)-XRCC4 and end processing by DNA polymerase
  μ (Pol μ) and DNA polymerase λ (Pol λ). The participation of β-catenin, TCF4, and
  GSK3β is indicated with dotted lines'
papertitle: Overview of Wnt/β-Catenin Pathway and DNA Damage/Repair in Cancer
reftext: Silvina B. Nadin, et al. Biology (Basel). 2025 Feb;14(2).
year: '2025'
doi: 10.3390/biology14020185
journal_title: Biology
journal_nlm_ta: Biology (Basel)
publisher_name: Multidisciplinary Digital Publishing Institute  (MDPI)
keywords: Wnt/β-catenin | DNA repair | cancer
automl_pathway: 0.8979789
figid_alias: PMC11851563__F2
figtype: Figure
redirect_from: /figures/PMC11851563__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11851563__biology-14-00185-g002.html
  '@type': Dataset
  description: 'β-Catenin and DNA repair pathways. (a) DNA damage response (DDR).
    (a) DDR involves a signaling cascade from DNA damage sites, which ultimately activates
    the cell-cycle checkpoints to allow for repair or triggers cell death. ATM detects
    DNA double-strand breaks (DSBs), and ATR DNA single-strand breaks (SSBs). MRN
    complex is another important sensor of DNA damage. γH2AX amplifies the DDR signaling.
    The downstream effectors CHK1 and CHK2 phosphorylate p53, and p53 are important
    to activate the DNA repair or to initiate apoptosis. The specific participation
    of β-catenin and GSK3β in DDR is indicated in the figure. (b) Direct repair. This
    mechanism specializes in correcting the DNA damage induced by alkylating agents.
    The most frequent lesion is O6-methylguanine (O6-meG), recognized by O6-meG-DNA-methyltransferase
    (MGMT) and regulated by β-catenin. (c) Base excision repair (BER). BER is the
    most versatile DNA repair system for maintaining genome stability. The first step
    consists of the recognition step by specific DNA glycosylases. Two BER pathways
    are known: short patch-BER and long patch-BER. The DNA glycosylase excises the
    DNA damage, leaving an AP site (apurinic/apyrimidinic). AP endonuclease incises
    the AP site creating a nick, the DNA polymerase β replaces the damaged base in
    the short patch-BER. However, if the lesion consists of 2–11 nucleotides, PCNA
    and FEN1 endonuclease, which adds to the process known as long patch-BER. The
    specific participation of β-catenin in DDR is indicated in the figure, particularly
    with glycosylases. (d) Nucleotide excision repair (NER). NER is a multi-step system
    that recognizes a wide spectrum of lesions causing important distortions in DNA
    and involves the action of more than 30 different proteins. The global genome
    NER (GG-NER) identifies lesions throughout the genome. XPC-HR23B detects the lesion
    and additional factors such as TFIIH, XPB, and XPD are essential to unwinding
    the double helix. XPA-RPA, XPG, and XPF-ERCC1 nucleases are needed to initiate
    the incision 15-24 nucleotides away. Finally, the gap filling is completed by
    Polδ/ε, RFC, PCNA, LIG1, LIG3/XRCC1. The specific participation of β-catenin,
    TCF, and GSK3β in NER are indicated in the figure. (e) Mismatch repair (MMR).
    The heterodimer MSH2-MSH6 detects base–base mismatches and insertion–deletion
    loops (IDLs). MSH2-MSH3 recognizes larger IDLs. The heterodimer MLH1-PMS2 binds
    to the recognition complex and initiates the repair with the components EXO1,
    PCNA, RFC, RPA, Polδ/ε, and LIG1. The dotted lines indicate the participation
    of β-catenin. (f) Double-strand break repair (DSB repair). DSBs can be repaired
    by homologous recombination (HR) and non-homologous-end joining (NHEJ). HR acts
    after replication and begins when the MRN complex (MRE11-NBS1-RAD50) recognizes
    the DSB. BRCA1 binds to p53, BRCA2, RAD51, and the MRN complex. Resection of the
    5′ DNA on either side of the DSB is BRCA1-dependent, resulting in the single-stranded
    DNA (ssDNA) exposure. BRCA2 localizes RAD51 to ssDNA regions to form a nucleoprotein
    filament capable of invading and binding homologous sequences. Then, DNA polymerases
    δ/ε and DNA ligase 1 end the repair of the lesion. The NHEJ system joins the two
    broken ends without homology. The heterodimer Ku70/Ku80 protects the DNA from
    exonuclease digestion and DNA-PK (DNA-dependent protein kinase). Artemis preserves
    the ends. DNA ligation is executed by the DNA ligase 4 (LIG4)-XRCC4 and end processing
    by DNA polymerase μ (Pol μ) and DNA polymerase λ (Pol λ). The participation of
    β-catenin, TCF4, and GSK3β is indicated with dotted lines'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CTNNB1
  - SIAH1
  - TCF4
  - TCF7L2
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - ATM
  - CHEK2
  - TP53
  - TP63
  - TP73
  - GSK3B
  - ATR
  - ANTXR1
  - MMAB
  - SERPINA2
  - CHEK1
  - BPIFA4P
  - TDG
  - HNF4A
  - APEX1
  - MGMT
  - DDB1
  - DDB2
  - ERCC2
  - XPA
  - HMGB1
  - RPA1
  - RPA2
  - RPA3
  - ERCC3
  - GTF2H1
  - GTF2H2
  - GTF2H3
  - GTF2H4
  - GTF2H5
  - CDX2
  - ERCC5
  - ERCC4
  - ERCC1
  - PCNA
  - ERVW-4
  - FEN1
  - XRCC1
  - LIG3
  - LRIG3
  - RFC1
  - SLC19A1
  - RFC2
  - RFC3
  - RFC4
  - RFC5
  - PARP1
  - LIG1
  - LRIG1
  - MSH2
  - MSH6
  - XRCC6
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - XRCC5
  - MSH3
  - GSK3A
  - MYBL2
  - BRCA1
  - PRMT1
  - BRCA2
  - MLH1
  - PMS2
  - RAD51
  - FZD5
  - RNU6-664P
  - EXO1
  - RUVBL1
  - PRKDC
  - PKP2
  - LIG4
  - XRCC4
  - Polo
---
