---
figid: PMC11245093__nihpp-2024.07.06.602360v2-f0007
figtitle: Reconstruction of carbohydrate utilization pathways and associated phenotypes
organisms:
- NA
organisms_ner:
- NA
pmcid: PMC11245093
filename: nihpp-2024.07.06.602360v2-f0007.jpg
figlink: /pmc/articles/PMC11245093/figure/F2/
number: F2
caption: '(1) A reference set of 263 Bifidobacterium genomes (Supplementary Table
  2) and a list of 589 curated functional roles (Supplementary Table 4) were used
  to populate 25 metabolic subsystems capturing catabolic pathways for 68 glycans.
  (2) Functional gene annotation was performed using homology-based methods and three
  genome context techniques: (i) clustering of genes on the chromosome (operons),
  (ii) co-regulation of genes by a common transcription factor, and (iii) co-occurrence
  of genes across related genomes. Carbohydrate utilization pathways were reconstructed
  based on the distribution of functional roles. (3) Each genome was assigned a detailed
  pathway variant based on the presence of signature genes (Supplementary Table 6).
  These variants were then converted to predicted binary phenotypes (1 = utilizer,
  0 = non-utilizer) to create a Binary Phenotype Matrix (BPM)'
papertitle: Integrative genomic reconstruction reveals heterogeneity in carbohydrate
  utilization across human gut bifidobacteria
reftext: Aleksandr A. Arzamasov, et al. bioRxiv. 2025 May 17;NA(NA).
year: '2025'
doi: 10.1101/2024.07.06.602360
journal_title: bioRxiv
journal_nlm_ta: bioRxiv
publisher_name: Cold Spring Harbor Laboratory Preprints
keywords: ''
automl_pathway: 0.6388833
figid_alias: PMC11245093__F2
figtype: Figure
redirect_from: /figures/PMC11245093__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11245093__nihpp-2024.07.06.602360v2-f0007.html
  '@type': Dataset
  description: '(1) A reference set of 263 Bifidobacterium genomes (Supplementary
    Table 2) and a list of 589 curated functional roles (Supplementary Table 4) were
    used to populate 25 metabolic subsystems capturing catabolic pathways for 68 glycans.
    (2) Functional gene annotation was performed using homology-based methods and
    three genome context techniques: (i) clustering of genes on the chromosome (operons),
    (ii) co-regulation of genes by a common transcription factor, and (iii) co-occurrence
    of genes across related genomes. Carbohydrate utilization pathways were reconstructed
    based on the distribution of functional roles. (3) Each genome was assigned a
    detailed pathway variant based on the presence of signature genes (Supplementary
    Table 6). These variants were then converted to predicted binary phenotypes (1
    = utilizer, 0 = non-utilizer) to create a Binary Phenotype Matrix (BPM)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Glycan
  - D-xylose
  - D-xylulose
---
