---
figid: PMC1896182__1471-2105-8-179-4
figtitle: Views of data interpretation
organisms:
- Homo sapiens
- Mus musculus
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC1896182
filename: 1471-2105-8-179-4.jpg
figlink: /pmc/articles/PMC1896182/figure/F4/
number: F4
caption: Views of data interpretation. The overlaid differential expression values
  are the result of the t-test shown in figure 3a. Node colors reflect the differential
  gene expression. The light gray nodes represent associated genes not on the array.
  a)Comparing the genes with the lowest p-value shows enriched lymphocyte activation
  in the GO Term enrichment analysis. b)Glucolysis KEGG map overlaid with differential
  expression result. c) gene association network of the glucolysis genes. d)Chromosomal
  location detail view of a chromosome part containing a differential expressed gene.
  Genes measured on the chip are marked in yellow on the chromosome, differential
  gene expression is shown above and below the chromosome.
papertitle: Genome Expression Pathway Analysis Tool â€“ Analysis and visualization of
  microarray gene expression data under genomic, proteomic and metabolic context.
reftext: Markus Weniger, et al. BMC Bioinformatics. 2007;8:179-179.
year: '2007'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5368638
figid_alias: PMC1896182__F4
figtype: Figure
redirect_from: /figures/PMC1896182__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC1896182__1471-2105-8-179-4.html
  '@type': Dataset
  description: Views of data interpretation. The overlaid differential expression
    values are the result of the t-test shown in figure 3a. Node colors reflect the
    differential gene expression. The light gray nodes represent associated genes
    not on the array. a)Comparing the genes with the lowest p-value shows enriched
    lymphocyte activation in the GO Term enrichment analysis. b)Glucolysis KEGG map
    overlaid with differential expression result. c) gene association network of the
    glucolysis genes. d)Chromosomal location detail view of a chromosome part containing
    a differential expressed gene. Genes measured on the chip are marked in yellow
    on the chromosome, differential gene expression is shown above and below the chromosome.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TW
  - BMAL1
  - PSD4
  - SARNP
  - ETV6
  - TERC
  - TECR
  - tw
  - Tmc
  - Tel
  - tei
  - ter
  - term
---
