---
figid: PMC10991915__40643_2021_426_Fig4_HTML
figtitle: Physiological and transcriptome analysis elucidates the metabolic mechanism
  of versatile Porphyridium purpureum under nitrogen deprivation for exopolysaccharides
  accumulation
organisms:
- NA
organisms_ner:
- Homo sapiens
- Arabidopsis thaliana
pmcid: PMC10991915
filename: 40643_2021_426_Fig4_HTML.jpg
figlink: /pmc/articles/PMC10991915/figure/Fig4/
number: F4
caption: Regulation of genes involved in central carbon metabolism. This image shows
  central carbon metabolic pathways, including the Calvin cycle, gluconeogenesis,
  glycolysis, pentose phosphate pathway, and the tricarboxylic acid cycle. The red
  plus sign indicates upregulation, the blue minus sign indicates downregulation,
  and the black sign indicates no significant difference. The front and back positions
  indicate the differential expression at day 5 and day 15, respectively. The regions
  in red and blue indicate the upregulation and downregulation of the overall pathway,
  respectively. Abbreviations for metabolites were shown in black. PEP, phosphoenolpyruvate;
  OAA, oxaloacetate; MAL, malate; Pyr, pyruvate; 3PG, 3-phosphoglycerate; 1,3-2PG,
  3-phospho-glycerol phosphate; G-3-P, glyceraldehyde 3-phosphate; F-1,6-2P, fructose
  1,6-bisphosphate; F-6-P, fructose 6-phosphate; G-6-P, glucose 6-phosphate; G-1-P,
  glucose 1-phosphate; UDP-Glc, UDP-glucose; 2PG, 2-phospho-glycerate; CIT, citrate;
  iCIT, isocitrate; α-KG, 2-oxo-glutarate; Suc-CoA, succinyl-CoA; SUC, succinate;
  FUM, fumarate. Abbreviations for enzymes were shown in gray. PEPC, phosphoenolpyruvate
  carboxylase; ME, malic enzyme (malate dehydrogenase); PGK, phosphoglycerate kinase;
  GAPDH, glyceraldehyde 3-phosphate dehydrogenase; FBA, fructose-bisphosphate aldolase;
  PBP, fructose-1,6-bisphosphatase; GPI, glucose-6-phosphate isomerase; PGM2, phosphoglucomutase;
  UGP, UTP-glucose-1-phosphate uridylyltransferase; TPS/TP, trehalose 6-phosphate
  synthase/phosphatase; TL, α,α-trehalase; HK, hexokinase; PFK, phosphofructokinase;
  PGM, 2,3-diphosphoglycerate-independent phosphoglycerate mutase; ENO, enolase; PK,
  pyruvate kinase; PDHC, pyruvate dehydrogenase complex; PDC, pyruvate decarboxylase;
  ALDH, aldehyde dehydrogenase; FUM, fumarate hydratase; GPD, glucose-6-phosphate
  1-dehydrogenase; PGL, 6-phosphogluconolactonase; PGD, 6-phosphogluconate dehydrogenase;
  RPE, ribulose-phosphate 3-epimerase; RPI, ribose-5-phosphate isomerase; ACC, acetyl-CoA
  carboxylase; MAT, malonyltransferase; KAS, β-ketoacyl-ACP synthase; KAR, β-ketoacyl-ACP
  reductase; HAD, β-hydroxyoctanoyl-ACP-dehydrase; EAR, enoyl-ACP reductase; LACS,
  long-chain acyl-CoA synthetase; ACX, acyl-CoA oxidase. ACAT, acetyl-CoA C-acetyltransferase
papertitle: Physiological and transcriptome analysis elucidates the metabolic mechanism
  of versatile Porphyridium purpureum under nitrogen deprivation for exopolysaccharides
  accumulation.
reftext: Liang Ji, et al. Bioresour Bioprocess. 2021 Dec;8(1):73.
year: '2021'
doi: 10.1186/s40643-021-00426-x
journal_title: Bioresources and Bioprocessing
journal_nlm_ta: Bioresour Bioprocess
publisher_name: Springer Singapore
keywords: Porphyridium purpureum | Nitrogen deprivation | Transcriptome analysis |
  Exopolysaccharides | Polyunsaturated fatty acids | Phycoerythrin
automl_pathway: 0.9651475
figid_alias: PMC10991915__F4
figtype: Figure
redirect_from: /figures/PMC10991915__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10991915__40643_2021_426_Fig4_HTML.html
  '@type': Dataset
  description: Regulation of genes involved in central carbon metabolism. This image
    shows central carbon metabolic pathways, including the Calvin cycle, gluconeogenesis,
    glycolysis, pentose phosphate pathway, and the tricarboxylic acid cycle. The red
    plus sign indicates upregulation, the blue minus sign indicates downregulation,
    and the black sign indicates no significant difference. The front and back positions
    indicate the differential expression at day 5 and day 15, respectively. The regions
    in red and blue indicate the upregulation and downregulation of the overall pathway,
    respectively. Abbreviations for metabolites were shown in black. PEP, phosphoenolpyruvate;
    OAA, oxaloacetate; MAL, malate; Pyr, pyruvate; 3PG, 3-phosphoglycerate; 1,3-2PG,
    3-phospho-glycerol phosphate; G-3-P, glyceraldehyde 3-phosphate; F-1,6-2P, fructose
    1,6-bisphosphate; F-6-P, fructose 6-phosphate; G-6-P, glucose 6-phosphate; G-1-P,
    glucose 1-phosphate; UDP-Glc, UDP-glucose; 2PG, 2-phospho-glycerate; CIT, citrate;
    iCIT, isocitrate; α-KG, 2-oxo-glutarate; Suc-CoA, succinyl-CoA; SUC, succinate;
    FUM, fumarate. Abbreviations for enzymes were shown in gray. PEPC, phosphoenolpyruvate
    carboxylase; ME, malic enzyme (malate dehydrogenase); PGK, phosphoglycerate kinase;
    GAPDH, glyceraldehyde 3-phosphate dehydrogenase; FBA, fructose-bisphosphate aldolase;
    PBP, fructose-1,6-bisphosphatase; GPI, glucose-6-phosphate isomerase; PGM2, phosphoglucomutase;
    UGP, UTP-glucose-1-phosphate uridylyltransferase; TPS/TP, trehalose 6-phosphate
    synthase/phosphatase; TL, α,α-trehalase; HK, hexokinase; PFK, phosphofructokinase;
    PGM, 2,3-diphosphoglycerate-independent phosphoglycerate mutase; ENO, enolase;
    PK, pyruvate kinase; PDHC, pyruvate dehydrogenase complex; PDC, pyruvate decarboxylase;
    ALDH, aldehyde dehydrogenase; FUM, fumarate hydratase; GPD, glucose-6-phosphate
    1-dehydrogenase; PGL, 6-phosphogluconolactonase; PGD, 6-phosphogluconate dehydrogenase;
    RPE, ribulose-phosphate 3-epimerase; RPI, ribose-5-phosphate isomerase; ACC, acetyl-CoA
    carboxylase; MAT, malonyltransferase; KAS, β-ketoacyl-ACP synthase; KAR, β-ketoacyl-ACP
    reductase; HAD, β-hydroxyoctanoyl-ACP-dehydrase; EAR, enoyl-ACP reductase; LACS,
    long-chain acyl-CoA synthetase; ACX, acyl-CoA oxidase. ACAT, acetyl-CoA C-acetyltransferase
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - VCAN
  - PAEP
  - PREP
  - PTPN22
  - PEPC
  - PGC
  - MAL
  - MRTFA
  - TIRAP
  - PUM3
  - RPE
  - PFKL
  - PFKM
  - PFKP
  - FOLR1
  - FOSL1
  - RABEP2
  - PDC
  - PNKD
  - ABCD2
  - ACSL1
  - MDH1
  - HSD17B12
  - HADH
  - CIT
  - DHDDS
  - ATP8A2
  - ACAT1
  - MAT1A
  - CES1
  - SOAT1
  - ACACA
  - BMS1
  - ACACB
  - UGP1
  - G1
  - PGM
  - PEP
  - PAS2
  - PEP
  - CO2
  - MAL
  - OAA
  - Glucose
  - Polysaccharides
  - PDC
  - Acetaldehyde
  - Acetate
  - Acetyl-CoA
  - Acyl-CoA
  - Fatty acids
  - CIT
  - trans-2-enoyl-CoA
  - Amino acids
  - ATP
  - Malonyl-CoA
  - Protein
  - ACC
---
