---
figid: PMC11042213__nihpp-2024.04.12.589130v1-f0001
figtitle: Environmental impacts on genotype-phenotype-fitness relationships
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11042213
filename: PMC11042213__nihpp-2024.04.12.589130v1-f0001.jpg
figlink: /pmc/articles/PMC11042213/figure/F1
number: F1
caption: '(A) The schematic shows that Genotype and Environment both influence the
  development of a Phenotype and that Phenotype and Environment together determine
  an organism’s relative Fitness. The grey dotted line connecting Fitness to Genotype
  reflects reproduction and potential evolutionary changes in allele frequencies from
  one generation to the next. (B) Allelic variation for a gene’s promoter (with arrowheads
  indicating the gene’s transcription start site) is shown under Genotype, leading
  to variation in the gene’s expression (shown with curvy lines representing differences
  in RNA abundance in budding yeast cells) shown under Phenotype, and variation in
  the abundance of different yeast genotypes resulting from their differences in gene
  expression shown under Fitness. Hypothetical differences in RNA abundance and relative
  fitness between two environments (Env. A and Env. B) are also shown. (C, D) Solid
  lines show the relationship between a focal gene’s expression and fitness in two
  different environments (green = environment A; purple = environment B). The black
  dot shows the expression and fitness of a “wild-type” promoter allele, with both
  values defined as 1. The green and purple dots show the expression and fitness of
  the same alternate promoter allele in environment A (green) and environment B (purple)
  with expression and fitness defined relative to the wild-type allele. The green
  and purple dotted arrows highlight the effects of genetic differences between the
  wild-type and alternate promoter alleles on gene expression and fitness in each
  environment. Panel C shows a case where the genetic difference between the wild-type
  and alternate allele (e.g., a new mutation) has the same effect on the focal gene’s
  expression in both environments (ExpA = ExpB; no phenotypic plasticity) but different
  effects on fitness (FitA ≠ FitB) due to differences in the shape of the fitness
  function between environments. Panel D shows a case where the genetic difference
  between the wild-type and alternate allele (e.g., a new mutation) has different
  effects on the focal gene’s expression in the two environments (ExpA ≠ ExpB; phenotypic
  plasticity) but the same effect on fitness (FitA ≠ FitB) due to differences in the
  shape of the fitness function between environments. Note that if the wild-type and
  alternate alleles are considered together, the case shown in C exhibits a gene-by-environment
  interaction for fitness because the alternate allele shows differences in fitness
  between the two environments but the wild-type allele does not; this case does not
  show a gene-by-environment interaction for gene expression because expression is
  the same for both alleles in the two environments (SI Appendix Fig. S1A,B). The
  opposite is true for the case shown in panel D: a gene-by-environment interaction
  is present for gene expression but not fitness (SI Appendix Fig. S1C,D). (E) The
  abbreviated metabolic pathway shown illustrates how the carbon sources used in this
  study (glucose, galactose, glycerol, and ethanol) relate to the glycolysis/gluconeogenesis
  pathway. The enzyme encoded by the TDH3 gene (TDH3p) catalyzes the interconversion
  of glyceraldehyde-3-phoshate (GAP) to 1,3-biphosphoglycerate (1,3BP), as shown by
  the double-headed black arrow. Gray arrows highlight other metabolic steps; multiple
  gray arrows indicate reactions catalyzed by two or more enzymes'
papertitle: Plasticity and environment-specific relationships between gene expression
  and fitness in Saccharomyces cerevisiae
reftext: Mohammad A. Siddiq, et al. bioRxiv. 2024 Apr 15;NA(NA).
year: '2024'
doi: 10.1101/2024.04.12.589130
journal_title: bioRxiv
journal_nlm_ta: bioRxiv
publisher_name: Cold Spring Harbor Laboratory
keywords: evolution | gene regulation | phenotypic plasticity | gene-by-environment
  interaction | mutations
automl_pathway: 0.7343672
figid_alias: PMC11042213__F1
figtype: Figure
redirect_from: /figures/PMC11042213__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11042213__nihpp-2024.04.12.589130v1-f0001.html
  '@type': Dataset
  description: '(A) The schematic shows that Genotype and Environment both influence
    the development of a Phenotype and that Phenotype and Environment together determine
    an organism’s relative Fitness. The grey dotted line connecting Fitness to Genotype
    reflects reproduction and potential evolutionary changes in allele frequencies
    from one generation to the next. (B) Allelic variation for a gene’s promoter (with
    arrowheads indicating the gene’s transcription start site) is shown under Genotype,
    leading to variation in the gene’s expression (shown with curvy lines representing
    differences in RNA abundance in budding yeast cells) shown under Phenotype, and
    variation in the abundance of different yeast genotypes resulting from their differences
    in gene expression shown under Fitness. Hypothetical differences in RNA abundance
    and relative fitness between two environments (Env. A and Env. B) are also shown.
    (C, D) Solid lines show the relationship between a focal gene’s expression and
    fitness in two different environments (green = environment A; purple = environment
    B). The black dot shows the expression and fitness of a “wild-type” promoter allele,
    with both values defined as 1. The green and purple dots show the expression and
    fitness of the same alternate promoter allele in environment A (green) and environment
    B (purple) with expression and fitness defined relative to the wild-type allele.
    The green and purple dotted arrows highlight the effects of genetic differences
    between the wild-type and alternate promoter alleles on gene expression and fitness
    in each environment. Panel C shows a case where the genetic difference between
    the wild-type and alternate allele (e.g., a new mutation) has the same effect
    on the focal gene’s expression in both environments (ExpA = ExpB; no phenotypic
    plasticity) but different effects on fitness (FitA ≠ FitB) due to differences
    in the shape of the fitness function between environments. Panel D shows a case
    where the genetic difference between the wild-type and alternate allele (e.g.,
    a new mutation) has different effects on the focal gene’s expression in the two
    environments (ExpA ≠ ExpB; phenotypic plasticity) but the same effect on fitness
    (FitA ≠ FitB) due to differences in the shape of the fitness function between
    environments. Note that if the wild-type and alternate alleles are considered
    together, the case shown in C exhibits a gene-by-environment interaction for fitness
    because the alternate allele shows differences in fitness between the two environments
    but the wild-type allele does not; this case does not show a gene-by-environment
    interaction for gene expression because expression is the same for both alleles
    in the two environments (SI Appendix Fig. S1A,B). The opposite is true for the
    case shown in panel D: a gene-by-environment interaction is present for gene expression
    but not fitness (SI Appendix Fig. S1C,D). (E) The abbreviated metabolic pathway
    shown illustrates how the carbon sources used in this study (glucose, galactose,
    glycerol, and ethanol) relate to the glycolysis/gluconeogenesis pathway. The enzyme
    encoded by the TDH3 gene (TDH3p) catalyzes the interconversion of glyceraldehyde-3-phoshate
    (GAP) to 1,3-biphosphoglycerate (1,3BP), as shown by the double-headed black arrow.
    Gray arrows highlight other metabolic steps; multiple gray arrows indicate reactions
    catalyzed by two or more enzymes'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ERVW-1
  - ERVK-20
  - ERVK-18
  - ERVK-32
  - MBNL1
  - XPO1
  - RASA1
  - RGS6
  - Glucose
  - Galactose
  - Glycerol
  - Pyruvate
  - Acetyl-CoA
  - Ethanol
---
