---
figid: PMC12219892__41540_2025_545_Fig2_HTML
figtitle: Scheme of the study design and workflow for transcriptomic and metabolomic
  analyses
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC12219892
filename: 41540_2025_545_Fig2_HTML.jpg
figlink: /pmc/articles/PMC12219892/figure/F2/
number: F2
caption: a RNA sequencing. The raw sequencing data from AC16 cells and publicly available
  datasets were aligned to the human reference genome. The public datasets included
  samples from normal tissues or cells treated with DOX. We then used the gene counts
  to perform RNA-seq analysis and metabolic model simulation. b RNA-seq analysis.
  We used the gene expression data to identify DEGs and subsequently performed functional
  and pathway analyses to identify the DOX-affected pathways. c GEM reconstruction
  and simulation. By integrating the RNA-seq data and a human metabolic model, we
  obtained a sample-specific GEM for each sample. These models were used to simulate
  metabolic fluxes and gene knockout effects, suggesting the occurrence of DOX-induced
  metabolic alterations between the groups. The figure was created using BioRender.com
papertitle: Temporal analysis of doxorubicin-induced cardiac toxicity and hypertrophy
reftext: Yu-Te Lin, et al. NPJ Syst Biol Appl. 2025;11(NA).
year: '2025'
doi: 10.1038/s41540-025-00545-7
journal_title: NPJ Systems Biology and Applications
journal_nlm_ta: NPJ Syst Biol Appl
publisher_name: Nature Publishing Group
keywords: Computational biology and bioinformatics | Systems biology
automl_pathway: 0.7409182
figid_alias: PMC12219892__F2
figtype: Figure
redirect_from: /figures/PMC12219892__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC12219892__41540_2025_545_Fig2_HTML.html
  '@type': Dataset
  description: a RNA sequencing. The raw sequencing data from AC16 cells and publicly
    available datasets were aligned to the human reference genome. The public datasets
    included samples from normal tissues or cells treated with DOX. We then used the
    gene counts to perform RNA-seq analysis and metabolic model simulation. b RNA-seq
    analysis. We used the gene expression data to identify DEGs and subsequently performed
    functional and pathway analyses to identify the DOX-affected pathways. c GEM reconstruction
    and simulation. By integrating the RNA-seq data and a human metabolic model, we
    obtained a sample-specific GEM for each sample. These models were used to simulate
    metabolic fluxes and gene knockout effects, suggesting the occurrence of DOX-induced
    metabolic alterations between the groups. The figure was created using BioRender.com
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TSC1
  - TSC2
  - INHBA
  - ATP5F1A
  - SCN2A
  - KRT90P
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - TP53
  - TP63
  - TP73
  - TNFRSF11A
  - SET
  - GEM
  - GMNN
  - ATP8A2
  - KCNK3
  - LAM
  - HBA
  - Glucose
  - ATP
---
