---
figid: PMC11001937__41467_2024_47182_Fig8_HTML
figtitle: Functional pathways in the three generations according to shotgun metagenomics
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11001937
filename: 41467_2024_47182_Fig8_HTML.jpg
figlink: /pmc/articles/PMC11001937/figure/F8
number: F8
caption: 'A Functional pathways from level B of the BRITE Hierarchies, grouped by
  their level A (“Metabolism”, “Genetic Information Processing”, “Environmental Information
  Processing” and “Cellular Processes”). Significant pathways are presented as box
  and whiskers plots (statistical test was linear mixed-effects model with the correction
  for multiple test comparisons, FDR p-value < 0.05). For the box and whiskers plots,
  the box ranged from the first to the third quartile, and the center the median value,
  while the whiskers extend from each quartile to the minimum or maximum values (n = 38,
  n = 43, n = 40 of biologically independent samples for Infants, Mothers, and Grandmothers
  groups, respectively). The graphs show pathway counts in the X axis. Complementary
  information is presented in Supplementary Data 8 for level B and Supplementary Data 9
  for level C pathways. B Selected KEGG Orthologues (KOs) grouped by pathways of interest
  are presented as box and whiskers plots (statistical test was linear mixed-effects
  model with the correction for multiple test comparisons, FDR p-value  <  0.05).
  For the box and whiskers plots, the box ranged from the first to the third quartile,
  and the center the median value, while the whiskers extend from each quartile to
  the minimum or maximum values (n = 38, n = 43, n = 40 of biologically independent
  samples for Infants, Mothers and Grandmothers groups, respectively). Complementary
  information is presented in Supplementary Data 7. EC: enzyme commission number.
  Source data are provided as a Source Data file. Significance: *p < 0.05; **p < 0.01;
  ***p < 0.001; n.s.: not significant; *p < 0.05; **p < 0.01; ***p < 0.001, exact
  p-values and FDR p-values are provided in Supplementary Data 7, 8, and 9'
papertitle: Comparative characterization of the infant gut microbiome and their maternal
  lineage by a multi-omics approach
reftext: Tomás Clive Barker-Tejeda, et al. Nat Commun. 2024;15(NA).
year: '2024'
doi: 10.1038/s41467-024-47182-y
journal_title: Nature Communications
journal_nlm_ta: Nat Commun
publisher_name: Nature Publishing Group UK
keywords: Metagenomics | Data integration | Paediatric research | Metabolomics | Microbiome
automl_pathway: 0.7795455
figid_alias: PMC11001937__F8
figtype: Figure
redirect_from: /figures/PMC11001937__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11001937__41467_2024_47182_Fig8_HTML.html
  '@type': Dataset
  description: 'A Functional pathways from level B of the BRITE Hierarchies, grouped
    by their level A (“Metabolism”, “Genetic Information Processing”, “Environmental
    Information Processing” and “Cellular Processes”). Significant pathways are presented
    as box and whiskers plots (statistical test was linear mixed-effects model with
    the correction for multiple test comparisons, FDR p-value < 0.05). For the box
    and whiskers plots, the box ranged from the first to the third quartile, and the
    center the median value, while the whiskers extend from each quartile to the minimum
    or maximum values (n = 38, n = 43, n = 40 of biologically independent samples
    for Infants, Mothers, and Grandmothers groups, respectively). The graphs show
    pathway counts in the X axis. Complementary information is presented in Supplementary
    Data 8 for level B and Supplementary Data 9 for level C pathways. B Selected KEGG
    Orthologues (KOs) grouped by pathways of interest are presented as box and whiskers
    plots (statistical test was linear mixed-effects model with the correction for
    multiple test comparisons, FDR p-value  <  0.05). For the box and whiskers plots,
    the box ranged from the first to the third quartile, and the center the median
    value, while the whiskers extend from each quartile to the minimum or maximum
    values (n = 38, n = 43, n = 40 of biologically independent samples for Infants,
    Mothers and Grandmothers groups, respectively). Complementary information is presented
    in Supplementary Data 7. EC: enzyme commission number. Source data are provided
    as a Source Data file. Significance: *p < 0.05; **p < 0.01; ***p < 0.001; n.s.:
    not significant; *p < 0.05; **p < 0.01; ***p < 0.001, exact p-values and FDR p-values
    are provided in Supplementary Data 7, 8, and 9'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CBLIF
  - CDK9
  - TRAP
  - ACP5
  - CD40LG
  - TRAF2
  - TRRAP
  - TDRD7
  - SCYL1
  - CAT
  - CRAT
  - GLYAT
  - Fatty acid
  - Trp
  - Polyamines
  - SCFA
  - BCFA
  - CAT
---
