---
figid: PMC10311107__gr5
figtitle: Metabolic and transcriptomic analyses elucidate a novel insight into the
  network for biosynthesis of carbohydrate and secondary metabolites in the stems
  of a medicinal orchid Dendrobium nobile
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC10311107
filename: gr5.jpg
figlink: /pmc/articles/PMC10311107/figure/fig5/
number: F5
caption: Expression changes of genes involved in carbohydrate biosynthesis in the
  stems of Dendrobium nobile. Red letters represent up-regulated expression (FDR were
  < 0.5 and fold change were ≥ 1.5), green letters represent down-regulated expression
  (FDR were < 0.5 and fold change were ≤ -1.5),and black letters represent no significant
  change in metabolite accumulation and gene expression. The orange box and letters
  indicate a pathway. Heatmaps show the level of increase and decrease of enzyme gene
  expression. Color scale on the top left represented normalized values of FPKM values
  and six boxes of each enzyme gene represented three biological replicates of DnS1
  and DnS2, respectively. SPS, sucrose-phosphate synthase; SPP, sucrose-6-phosphatase;
  SUS, sucrose synthase; UGP2, UTP-glucose-1-phosphate uridylyltransferase; INV, β-fructofuranosidase;
  BGLX, β-glucosidase; CELB, cellulase; CELS, cellulose synthase; GTP, glucose-1-phosphate
  guanylyltransferase; GAS, 1,3-beta-glucan synthase; EGLC, glucan endo-1,3-beta-D-glucosidase;
  PGT, phenol beta-glucosyltransferase; GLGC, glucose-1-phosphate adenylyltransferase;
  GLGA, starch synthase; GLGB, 1,4-alpha-glucan branching enzyme; AMYA, α-amylase;
  MGAM, maltase-glucoamylase; ISA, isoamylase; MAPA, maltose phosphorylase; OTSA,
  trehalose 6-phosphate synthase; OTSB, trehalose 6-phosphate phosphatase; TREP, alpha,
  alpha-trehalose phosphorylase; ALDO, fructose-bisphosphate aldolase; PFP, diphosphate-dependent
  phosphofructokinase; PFK9, 6-phosphofructokinase; TIGAR, fructose-2,6-bisphosphatase;
  FRUBP, fructose-2,6-bisphosphate 6-phosphatase; DFPP, D-fructose 2-phosphate phosphatase;
  SORD, L-iditol 2-dehydrogenase; AKR1B, aldehyde reductase; MI, mannose isomerase;
  GMUG, mannan endo-1,4-beta-mannosidase; GM, GDPmannose:mannan 1,4-beta-D-mannosyltransferase;
  GMPP, mannose-1-phosphate guanylyltransferase; PMM, phosphomannomutase; MANA, mannose-6-phosphate
  isomerase; MANX, mannose PTS system EIIAB; KHK, ketohexokinase; TKTB, transketolase;
  XFP, xylulose-5-phosphate; XYLB, xylulokinase; GALM, aldose 1-epimerase; GLPE, glucose-6-phosphate
  1-epimerase; GCK, glucokinase; PGM, phosphoglucomutase; GPI, glucose-6-phosphate
  isomerase; GAPDH, glyceraldehyde 3-phosphate dehydrogenase; PGK, phosphoglycerate
  kinase; PGAM, 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase; ENO, enolase;
  PYK, pyruvate kinase; ACEE, pyruvate dehydrogenase E1; PDC, pyruvate decarboxylase;
  ALDH, aldehyde dehydrogenase; ACDA, acetate-CoA ligase subunit α; DLD, dihydrolipoamide
  dehydrogenase; DLAT, pyruvate dehydrogenase E2; DAK, dihydroxyacetone kinase.
papertitle: Metabolic and transcriptomic analyses elucidate a novel insight into the
  network for biosynthesis of carbohydrate and secondary metabolites in the stems
  of a medicinal orchid Dendrobium nobile.
reftext: Yu-Wen Zhang, et al. Plant Divers. 2023 May;45(3):326-336.
year: '2023'
doi: 10.1016/j.pld.2022.10.004
journal_title: Plant Diversity
journal_nlm_ta: Plant Divers
publisher_name: Kunming Institute of Botany, Chinese Academy of Sciences
keywords: Dendrobium nobile | Transcriptome | Metabolome | Polysaccharides | Alkaloids
  | Secondary metabolite biosynthesis
automl_pathway: 0.7804205
figid_alias: PMC10311107__F5
figtype: Figure
redirect_from: /figures/PMC10311107__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10311107__gr5.html
  '@type': Dataset
  description: Expression changes of genes involved in carbohydrate biosynthesis in
    the stems of Dendrobium nobile. Red letters represent up-regulated expression
    (FDR were < 0.5 and fold change were ≥ 1.5), green letters represent down-regulated
    expression (FDR were < 0.5 and fold change were ≤ -1.5),and black letters represent
    no significant change in metabolite accumulation and gene expression. The orange
    box and letters indicate a pathway. Heatmaps show the level of increase and decrease
    of enzyme gene expression. Color scale on the top left represented normalized
    values of FPKM values and six boxes of each enzyme gene represented three biological
    replicates of DnS1 and DnS2, respectively. SPS, sucrose-phosphate synthase; SPP,
    sucrose-6-phosphatase; SUS, sucrose synthase; UGP2, UTP-glucose-1-phosphate uridylyltransferase;
    INV, β-fructofuranosidase; BGLX, β-glucosidase; CELB, cellulase; CELS, cellulose
    synthase; GTP, glucose-1-phosphate guanylyltransferase; GAS, 1,3-beta-glucan synthase;
    EGLC, glucan endo-1,3-beta-D-glucosidase; PGT, phenol beta-glucosyltransferase;
    GLGC, glucose-1-phosphate adenylyltransferase; GLGA, starch synthase; GLGB, 1,4-alpha-glucan
    branching enzyme; AMYA, α-amylase; MGAM, maltase-glucoamylase; ISA, isoamylase;
    MAPA, maltose phosphorylase; OTSA, trehalose 6-phosphate synthase; OTSB, trehalose
    6-phosphate phosphatase; TREP, alpha, alpha-trehalose phosphorylase; ALDO, fructose-bisphosphate
    aldolase; PFP, diphosphate-dependent phosphofructokinase; PFK9, 6-phosphofructokinase;
    TIGAR, fructose-2,6-bisphosphatase; FRUBP, fructose-2,6-bisphosphate 6-phosphatase;
    DFPP, D-fructose 2-phosphate phosphatase; SORD, L-iditol 2-dehydrogenase; AKR1B,
    aldehyde reductase; MI, mannose isomerase; GMUG, mannan endo-1,4-beta-mannosidase;
    GM, GDPmannose:mannan 1,4-beta-D-mannosyltransferase; GMPP, mannose-1-phosphate
    guanylyltransferase; PMM, phosphomannomutase; MANA, mannose-6-phosphate isomerase;
    MANX, mannose PTS system EIIAB; KHK, ketohexokinase; TKTB, transketolase; XFP,
    xylulose-5-phosphate; XYLB, xylulokinase; GALM, aldose 1-epimerase; GLPE, glucose-6-phosphate
    1-epimerase; GCK, glucokinase; PGM, phosphoglucomutase; GPI, glucose-6-phosphate
    isomerase; GAPDH, glyceraldehyde 3-phosphate dehydrogenase; PGK, phosphoglycerate
    kinase; PGAM, 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase; ENO,
    enolase; PYK, pyruvate kinase; ACEE, pyruvate dehydrogenase E1; PDC, pyruvate
    decarboxylase; ALDH, aldehyde dehydrogenase; ACDA, acetate-CoA ligase subunit
    α; DLD, dihydrolipoamide dehydrogenase; DLAT, pyruvate dehydrogenase E2; DAK,
    dihydroxyacetone kinase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - aceE
  - bglX
  - chbC
  - puuC
  - eno
  - galM
  - dld
  - glpE
  - pgk
  - pgm
  - otsA
  - glgC
  - manA
  - tktB
  - otsB
  - glgA
  - xylB
  - manX
  - amyA
  - glgB
  - Pyruvate
  - Phosphoenol-pyruvate
  - Glucose
  - 2-Hydroxy-ethyl-ThPP
  - Acetaldehyde
  - ENO
  - Sucrose
  - Glucoside
  - Acetate
  - GDP-Glucose
  - Dihydrolipoamide
  - Fructose
  - B-D-Glucose
  - PGT
  - S-Acetyl-dihydrolipoarride
  - Acetyl-CoA
  - Glycerate-3P
  - D-Fructose-6P
  - UDP
  - D-Glucose
  - Glyceraldehyde
  - Trehalose
  - Trehalose-6P
  - ADP
  - D-Fructose-2
  - D-Mannose-6P
  - D-Glyceraldehyde
  - D-Mannose-1P- GMPP
  - D-Fructose-2P
  - MAPA
  - D-Mannose
  - GDP-D-Mannose
  - Sorbitol
  - D-Fructose
  - 1,4-B-Mannan
  - Mannose
  - MI
---
