---
figid: PMC11048214__biomolecules-14-00411-g005
figtitle: IPA canonical pathway analysis in genes containing prioritised variants
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11048214
filename: biomolecules-14-00411-g005.jpg
figlink: /pmc/articles/PMC11048214/figure/F5
number: F5
caption: IPA canonical pathway analysis in genes containing prioritised variants.
  Red and green nodes represent the concordance score and show predicted deleterious
  and benign variants, respectively. We assigned a concordance score of +2 if predicted
  pathogenic by at least four pathogenicity predictors (red nodes); we assigned a
  concordance score of +1 if predicted pathogenic by up to three pathogenicity predictors
  (light red nodes). We assigned a concordance score of 0 if it was not predicted
  by any pathogenicity predictors or if there were conflicting interpretations of
  pathogenicity (grey nodes). A concordance score of -1 was assigned if classified
  as benign in up to three pathogenicity predictors (light green nodes) and -2 when
  all the predictors classified the variants as benign in at least four pathogenicity
  predictors (dark green nodes); edges are shown as grey. Sirtuin Signalling Pathway
  in the (a) WES dataset and in the (b) WGS dataset. Some genes containing variants
  were computed by IPA to be part of the Sirtuin Signalling Pathway other genes containing
  variants were not part of the pathway but were linked to it by direct interactions
  with the three mitochondrial sirtuins (SIRT3, SIRT4, and SIRT5). We used IPA’s “grow”
  functionality to add these genes to the pathway analysis. Some genes have been filled
  with more than one colour because they have different variants with different scores
  of pathogenicity concordance. * Genes containing more than one variant
papertitle: 'Mitochondrial and Nuclear DNA Variants in Amyotrophic Lateral Sclerosis:
  Enrichment in the Mitochondrial Control Region and Sirtuin Pathway Genes in Spinal
  Cord Tissue'
reftext: Sharon Natasha Cox, et al. Biomolecules. 2024 Apr;14(4).
year: '2024'
doi: 10.3390/biom14040411
journal_title: Biomolecules
journal_nlm_ta: Biomolecules
publisher_name: MDPI
keywords: Amyotrophic Lateral Sclerosis | bioinformatic pipeline | mtDNA | nDNA |
  WES | WGS | variants | heteroplasmy
automl_pathway: 0.7482663
figid_alias: PMC11048214__F5
figtype: Figure
redirect_from: /figures/PMC11048214__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11048214__biomolecules-14-00411-g005.html
  '@type': Dataset
  description: IPA canonical pathway analysis in genes containing prioritised variants.
    Red and green nodes represent the concordance score and show predicted deleterious
    and benign variants, respectively. We assigned a concordance score of +2 if predicted
    pathogenic by at least four pathogenicity predictors (red nodes); we assigned
    a concordance score of +1 if predicted pathogenic by up to three pathogenicity
    predictors (light red nodes). We assigned a concordance score of 0 if it was not
    predicted by any pathogenicity predictors or if there were conflicting interpretations
    of pathogenicity (grey nodes). A concordance score of -1 was assigned if classified
    as benign in up to three pathogenicity predictors (light green nodes) and -2 when
    all the predictors classified the variants as benign in at least four pathogenicity
    predictors (dark green nodes); edges are shown as grey. Sirtuin Signalling Pathway
    in the (a) WES dataset and in the (b) WGS dataset. Some genes containing variants
    were computed by IPA to be part of the Sirtuin Signalling Pathway other genes
    containing variants were not part of the pathway but were linked to it by direct
    interactions with the three mitochondrial sirtuins (SIRT3, SIRT4, and SIRT5).
    We used IPA’s “grow” functionality to add these genes to the pathway analysis.
    Some genes have been filled with more than one colour because they have different
    variants with different scores of pathogenicity concordance. * Genes containing
    more than one variant
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BAX
  - FOXO1
  - FOXO3
  - FOXO4
  - FOXO6
  - CASR
  - CXADR
  - PRKAR1A
  - SPG7
  - NR1I3
  - TRIM13
  - CXADRP1
  - ALPP
  - ATHS
  - SLPI
  - CCL27
  - ATRNL1
  - PDLIM3
  - NAT10
  - ASRGL1
  - AIP
  - AURKAIP1
  - LARS1
  - PLCL1
  - PRNP
  - NCOA6
  - MIF
  - AMH
  - S100A8
  - S100A9
  - EPHX2
  - CLPX
  - PCK2
  - FOXD1
  - FOXD2
  - FOXD3
  - FOXD4
  - FOXD4L1
  - FOXD4L3
  - FOXD4L4
  - FOXD4L5
  - FOXD4L6
  - CNDP2
  - SIK1
  - SIK2
  - SIK3
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - SARDH
  - SDHB
  - SORD
  - SDS
  - APP
  - YBEY
  - LAX1
  - MAL
  - MRTFA
  - TIRAP
  - GSTZ1
  - ATP8A2
  - CPT1B
  - DECR1
  - PDHX
  - PCCB
  - CDR2L
  - GLS
  - GLS2
  - SHMT2
  - CRYGD
  - AP1S2
  - DBT
  - OPAL
  - Alpha
  - NADH
  - NAD
  - ROS
  - Lipid
  - NAD+
  - ADH
  - App
  - Male
  - TCA
  - Resveratrol
  - Mal
  - ICA
  - ATP
  - NADPH
---
