---
figid: PMC11459742__KRNB_A_2408523_F0004_OC
figtitle: Modeling ncRNA interactions with signalling pathways using GO data
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11459742
filename: KRNB_A_2408523_F0004_OC.jpg
figlink: /pmc/articles/PMC11459742/figure/F4/
number: F4
caption: Modeling ncRNA interactions with signalling pathways using GO data. (A) A
  cytoscape.Js rendering of the FlyBase Drosophila Toll Signaling Pathway (FBgg0001059)
  network. ‘Core’ pathway components are shown in mauve, positive regulators in green,
  negative regulators in red, context-dependent regulators in white. The size of circular
  nodes (protein coding genes) is determined by the experimental data supporting pathway
  assignment [62]. The edges between nodes derived from physical interaction data.
  Regulatory ncRNAs are depicted by cream triangles (of uniform size, unrelated to
  support) and red lines connect them to their targets. (B) Intersection of ncRNAs
  with pathways in Drosophila based on pathway data curated by FlyBase using the GO.
  Signaling pathways are shown by orange triangles and ncRNAs by green circles. Multiple
  regulatory ncRNAs have been experimentally shown to interact and regulate each pathway.
  Some ncRNAs can target multiple pathways. (C) ‘Model of MiR-4691-3p inhibition of
  cGAS-sting signaling in the cytosol (human)’ gomodel:654d809000000802 in the noctua
  visual pathway Editor curation interface, showing the intersection of miR-4691-3p
  (RNAcentral:URS000012F9EC_9606) with the cGAS-sting pathway by the post-transcriptional
  gene silencing of STING1. The expanded box section shows the detailed annotations
  and evidence that is associated with the miR-4691-3p. (D) A simplified view of the
  GO-CAM shown in panel C rendered in cytoscape can be used to present the pathway
  constructed in the noctua curation interface. Currently these models can be viewed
  in AmiGO (https://amigo.geneontology.org/amigo/model/654d809000000802) and on Alliance
  gene page pathway tab (e.g. https://www.alliancegenome.org/gene/HGNC:21367#pathways)
papertitle: Representation of non-coding RNA-mediated regulation of gene expression
  using the Gene Ontology
reftext: Giulia Antonazzo, et al. RNA Biol. 2024 Oct 7;21(1).
year: '2024'
doi: 10.1080/15476286.2024.2408523
journal_title: RNA Biology
journal_nlm_ta: RNA Biol
publisher_name: Taylor & Francis
keywords: Gene Ontology | non-coding RNA | bioinformatics | biocuration | gene silencing
  | guidelines
automl_pathway: 0.6884959
figid_alias: PMC11459742__F4
figtype: Figure
redirect_from: /figures/PMC11459742__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11459742__KRNB_A_2408523_F0004_OC.html
  '@type': Dataset
  description: Modeling ncRNA interactions with signalling pathways using GO data.
    (A) A cytoscape.Js rendering of the FlyBase Drosophila Toll Signaling Pathway
    (FBgg0001059) network. ‘Core’ pathway components are shown in mauve, positive
    regulators in green, negative regulators in red, context-dependent regulators
    in white. The size of circular nodes (protein coding genes) is determined by the
    experimental data supporting pathway assignment [62]. The edges between nodes
    derived from physical interaction data. Regulatory ncRNAs are depicted by cream
    triangles (of uniform size, unrelated to support) and red lines connect them to
    their targets. (B) Intersection of ncRNAs with pathways in Drosophila based on
    pathway data curated by FlyBase using the GO. Signaling pathways are shown by
    orange triangles and ncRNAs by green circles. Multiple regulatory ncRNAs have
    been experimentally shown to interact and regulate each pathway. Some ncRNAs can
    target multiple pathways. (C) ‘Model of MiR-4691-3p inhibition of cGAS-sting signaling
    in the cytosol (human)’ gomodel:654d809000000802 in the noctua visual pathway
    Editor curation interface, showing the intersection of miR-4691-3p (RNAcentral:URS000012F9EC_9606)
    with the cGAS-sting pathway by the post-transcriptional gene silencing of STING1.
    The expanded box section shows the detailed annotations and evidence that is associated
    with the miR-4691-3p. (D) A simplified view of the GO-CAM shown in panel C rendered
    in cytoscape can be used to present the pathway constructed in the noctua curation
    interface. Currently these models can be viewed in AmiGO (https://amigo.geneontology.org/amigo/model/654d809000000802)
    and on Alliance gene page pathway tab (e.g. https://www.alliancegenome.org/gene/HGNC:21367#pathways)
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CGAS
  - BAG1
  - RARB
  - SAFB
  - RTN3
  - GAST
  - GALNS
  - PAGR1
  - STING1
  - TNF
  - SLC25A20
  - LILRB2
  - MIR1-2
  - DHH
  - IHH
  - SHH
  - SERPINA1
  - SERPINA2
  - S100A10
  - ENDOU
  - EXOSC6
  - MYD88
  - BMP1
  - BMP2
  - BMP3
  - BMP4
  - BMP5
  - BMP6
  - BMP7
  - BMP8A
  - BMP8B
  - BMP10
  - BMP15
  - GDF1
  - GDF2
  - GDF3
  - GDF5
  - GDF6
  - GDF7
  - EGFR
  - LILRB1
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - TLR4
  - MSH6
  - COL14A1
  - HAS1
  - NFKB1
  - NR1I2
  - JTB
  - BPIFA4P
  - GEN1
  - ACP1
  - TBK1
  - IRF3
  - pll
---
