---
figid: PMC11901428__KGMI_A_2476570_F0001_OC
figtitle: Host-microbiota interactome in IBD
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11901428
filename: KGMI_A_2476570_F0001_OC.jpg
figlink: /pmc/articles/PMC11901428/figure/F1/
number: F1
caption: 'Host-microbiota interactome in IBD. (a) Compositional and functional characteristics
  of the gut microbiota are both altered in IBD. Changes in the composition are reflected
  in a significant decrease in microbial abundance and diversity of certain commensal
  bacteria (e.g., Firmicutes and Bacteroides), with an enrichment in potentially pathogenic
  bacteria such as Proteobacteria. Functional changes, on the other hand, are reflected
  in alterations at different omics levels. The bacterial genome can significantly
  affect the host through variations in SNV, CNV, and SV. At the transcriptomic level,
  RNA can influence host’s nutrient metabolism, oxidative stress responses, and antimicrobial
  peptide production. (b) Proteins and metabolites can directly interact with intestinal
  epithelial cells. For example, membrane proteins can directly bind to the DC cell
  receptor DS-SIGN, leading to an increase in IL-10 secretion and a reduction in inflammation.
  Flagellar proteins can interact with the TLR5 receptor, activating the Nf-κB signaling
  pathway. Additionally, secreted proteins from B. vulgatus can damage intestinal
  epithelial cells. Small-molecule metabolites (e.g. SCFAs, SLs, BAs, TRPs) can act
  on receptors on epithelial cells, thereby influencing T cells (T regulatory cell,
  TH17 cell, TH1 cell) and macrophages to regulate the inflammatory state. Abbreviations:
  E. rectale: Eubacterium; F. prausnitzii, faecalibacterium prausnitzii; B. vulgatus,
  Bacteroides vulgatus; R. gnavus, ruminococcus gnavus; R. hominis, Roseburia hominis;
  B. brifidum, Bifidobacterium brifidum; AIEC, Adherent invasive E coli; SCFAs, short-chain
  fatty acids; SLs,Sphingolipids; BAs, bile acids; GPCR, G-protein coupled receptor;
  AhR, Aryl hydrocarbon receptor; S1PR, Sphingosine-1-phosphate receptor; TGR5, Transmembrane
  G protein-coupled receptor 5; TLR5, toll-like receptor 5; Nf-κB, nuclear factor-κB'
papertitle: Integrating multi-omics data to reveal the host-microbiota interactome
  in inflammatory bowel disease
reftext: Fengyuan Su, et al. Gut Microbes. 2025 Mar 10;17(1).
year: '2025'
doi: 10.1080/19490976.2025.2476570
journal_title: Gut Microbes
journal_nlm_ta: Gut Microbes
publisher_name: Taylor & Francis
keywords: IBD | gut microbiome | host-microbiota interactome | multi-omics | methodologies
automl_pathway: 0.9575904
figid_alias: PMC11901428__F1
figtype: Figure
redirect_from: /figures/PMC11901428__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11901428__KGMI_A_2476570_F0001_OC.html
  '@type': Dataset
  description: 'Host-microbiota interactome in IBD. (a) Compositional and functional
    characteristics of the gut microbiota are both altered in IBD. Changes in the
    composition are reflected in a significant decrease in microbial abundance and
    diversity of certain commensal bacteria (e.g., Firmicutes and Bacteroides), with
    an enrichment in potentially pathogenic bacteria such as Proteobacteria. Functional
    changes, on the other hand, are reflected in alterations at different omics levels.
    The bacterial genome can significantly affect the host through variations in SNV,
    CNV, and SV. At the transcriptomic level, RNA can influence host’s nutrient metabolism,
    oxidative stress responses, and antimicrobial peptide production. (b) Proteins
    and metabolites can directly interact with intestinal epithelial cells. For example,
    membrane proteins can directly bind to the DC cell receptor DS-SIGN, leading to
    an increase in IL-10 secretion and a reduction in inflammation. Flagellar proteins
    can interact with the TLR5 receptor, activating the Nf-κB signaling pathway. Additionally,
    secreted proteins from B. vulgatus can damage intestinal epithelial cells. Small-molecule
    metabolites (e.g. SCFAs, SLs, BAs, TRPs) can act on receptors on epithelial cells,
    thereby influencing T cells (T regulatory cell, TH17 cell, TH1 cell) and macrophages
    to regulate the inflammatory state. Abbreviations: E. rectale: Eubacterium; F.
    prausnitzii, faecalibacterium prausnitzii; B. vulgatus, Bacteroides vulgatus;
    R. gnavus, ruminococcus gnavus; R. hominis, Roseburia hominis; B. brifidum, Bifidobacterium
    brifidum; AIEC, Adherent invasive E coli; SCFAs, short-chain fatty acids; SLs,Sphingolipids;
    BAs, bile acids; GPCR, G-protein coupled receptor; AhR, Aryl hydrocarbon receptor;
    S1PR, Sphingosine-1-phosphate receptor; TGR5, Transmembrane G protein-coupled
    receptor 5; TLR5, toll-like receptor 5; Nf-κB, nuclear factor-κB'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FAM20C
  - ALDH3A2
  - BAS
  - GPR35
  - AHR
  - S1PR3
  - S1PR5
  - S1PR4
  - S1PR2
  - S1PR1
  - MAT2B
  - TXNRD3
  - ERVK-8
  - ERVK-10
  - ERVK-9
  - ERVK-21
  - ERVK-18
  - ERVK-25
  - ERVK-24
  - ERVK-19
  - TLR5
  - CD209
  - NFKB1
  - IL10
  - Tryptophan
  - ROS
  - RNS
  - SLS
  - SCFAs
  - BAS
---
