---
figid: PMC10040851__fgene-14-1105673-g006
figtitle: Single-cell atlas reveals shared and distinct immune responses and metabolic
  profiles in SARS-CoV-2 and HIV-1 infections
organisms:
- NA
pmcid: PMC10040851
filename: fgene-14-1105673-g006.jpg
figlink: /pmc/articles/PMC10040851/figure/F6/
number: F6
caption: IFN-I signaling is correlated with divergent biological functions in COVID-19
  versus HIV-1. (A) Network plot of genes related to IFN-I signaling that are differentially
  upregulated in COVID-19 (Right), HIV-1 (Left), or jointly upregulated compared to
  healthy controls. Genes are colored based on Log-2-fold change in expression in
  HIV-1+ versus COVID-19 patients. (B) Box plots of the average expression of HIV-1,
  COVID, and shared IFN-I module scores for each condition. p-values are computed
  with Wilcoxon signed-rank test with Holm-Bonferroni adjustment. (C) Normalized gene
  expression plots of IFN-I genes in COVID-19 (top) and HIV-1 (bottom). (D) Network
  plot of key pathways correlated with IFN-I signaling in COVID-19 (left) and HIV-1+
  patients (right). Size of each center corresponds to the number of genes present
  in the pathway. Genes are colored based on Log-2-fold change in expression in HIV-1+
  versus COVID-19 patients. (E) Box plots of the average expression of CD8+ T cell
  actin polymerization module scores for each disease condition (left). Box plots
  of the average expression of CD8+ effector, T/NK proliferating, and CD8+ Tex precursor
  scores for each disease condition (right). p-values are computed with Wilcoxon signed-rank
  test with Holm-Bonferroni adjustment. (F) Box plots of the average expression of
  CD8+ T cell TLR-signaling module scores for each disease condition (left). Box plots
  of the average expression of CD8+ effector, CD8+ effector memory and CD8+ naive
  TLR-signaling scores for each disease condition (right). p-values are computed with
  Wilcoxon signed-rank test with Holm-Bonferroni adjustment.
papertitle: A single-cell atlas reveals shared and distinct immune responses and metabolic
  profiles in SARS-CoV-2 and HIV-1 infections.
reftext: Tony Pan, et al. Front Genet. 2023;14:1105673.
year: '2023'
doi: 10.3389/fgene.2023.1105673
journal_title: Frontiers in Genetics
journal_nlm_ta: Front Genet
publisher_name: Frontiers Media S.A.
keywords: HIV | Single-cell RNA sequencing | SARS-CoV-2 | type 1 interferon | inflammation
  | metabolic signaling
automl_pathway: 0.7408242
figid_alias: PMC10040851__F6
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC10040851__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10040851__fgene-14-1105673-g006.html
  '@type': Dataset
  description: IFN-I signaling is correlated with divergent biological functions in
    COVID-19 versus HIV-1. (A) Network plot of genes related to IFN-I signaling that
    are differentially upregulated in COVID-19 (Right), HIV-1 (Left), or jointly upregulated
    compared to healthy controls. Genes are colored based on Log-2-fold change in
    expression in HIV-1+ versus COVID-19 patients. (B) Box plots of the average expression
    of HIV-1, COVID, and shared IFN-I module scores for each condition. p-values are
    computed with Wilcoxon signed-rank test with Holm-Bonferroni adjustment. (C) Normalized
    gene expression plots of IFN-I genes in COVID-19 (top) and HIV-1 (bottom). (D)
    Network plot of key pathways correlated with IFN-I signaling in COVID-19 (left)
    and HIV-1+ patients (right). Size of each center corresponds to the number of
    genes present in the pathway. Genes are colored based on Log-2-fold change in
    expression in HIV-1+ versus COVID-19 patients. (E) Box plots of the average expression
    of CD8+ T cell actin polymerization module scores for each disease condition (left).
    Box plots of the average expression of CD8+ effector, T/NK proliferating, and
    CD8+ Tex precursor scores for each disease condition (right). p-values are computed
    with Wilcoxon signed-rank test with Holm-Bonferroni adjustment. (F) Box plots
    of the average expression of CD8+ T cell TLR-signaling module scores for each
    disease condition (left). Box plots of the average expression of CD8+ effector,
    CD8+ effector memory and CD8+ naive TLR-signaling scores for each disease condition
    (right). p-values are computed with Wilcoxon signed-rank test with Holm-Bonferroni
    adjustment.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PSME1
  - HLA-DMA
  - PTPN2
  - IRF9
  - IFNAR2
  - IFIT2
  - HLA-DRB1
  - SRI
  - NAMPT
  - PPBP
  - SOD2
  - SOD2-OT1
  - PLSCR1
  - HLA-DQA1
  - PSME2
  - BST2
  - CD68
  - FITM2
  - IFITM3
  - HLA-F
  - POMP
  - NT5C3A
  - VAPB
  - TNFAIP3
  - CD86
  - HLA-C
  - TAF1
  - RIPK2
  - NMI
  - MYO1C
  - RASAL1
  - RASAL2
  - RASAL3
  - CSK
  - YTHDF3
  - CD74
  - FOLH1
  - SH2B1
  - FOLH1B
  - SLAMF7
  - SOCS3
  - ACSL1
  - SAMD9
  - RNF213
  - GRP
  - LSM4
  - LGALSL
  - UCMA
  - CMTR1
  - OGFR
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - MVP
  - MMVP1
  - PTPN1
  - HLA-B
  - CALM2
  - CALM3
  - PICALM
  - SNAP91
  - CALM1
  - IRF7
  - RHOG
  - AMACR
  - RND2
  - TRIM47
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - FYN
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - CD38
  - CNP
  - NPPC
  - HEY
  - CFLAR
  - ACTB
  - POTEF
  - ADAR
  - PARP12
  - PARP9
  - EIF4E3
  - PSMA2
  - IFNAR1
  - KIN
  - DEXI
  - WAS
  - GNAS-AS1
  - MAX
  - HSPA12A
  - HSPA12B
  - HSPA13
  - HSPA14
  - HSPA1A
  - HSPA1B
  - HSPA1L
  - HSPA2
  - HSPA4
  - HSPA4L
  - HSPA5
  - HSPA6
  - HSPA8
  - HSPA9
  - HSPH1
  - HYOU1
  - APC
  - PROC
  - PARP14
  - CSF2RB
  - GBP2
  - SELL
  - SLC25A28
  - C5
  - C6
  - C7
  - C8A
  - C8B
  - C8G
  - C9
  - NR1I2
  - JTB
  - KAT2B
  - FCGR1BP
  - FLOT1
  - FLOT2
  - ACTA1
  - ACTA2
  - ACTC1
  - APPL1
  - PPL
  - NPY6R
  - IFNA1
  - PRAME
  - LIAS
  - CNPY3
  - AN
  - PSAP
  - MTCH1
  - SFTPA1
  - SFTPA2
  - CMAS
  - STX7
  - LYN
  - CD163
  - SYK
  - TLR1
  - TLR2
  - TLR3
  - TLR4
  - TLR5
  - TLR6
  - TLR7
  - TLR8
  - TLR9
  - TLR10
---
