---
figid: PMC1636279__zjb0210661700007
figtitle: An integrated view of the metabolic pathway comparison between L
organisms:
- NA
pmcid: PMC1636279
filename: zjb0210661700007.jpg
figlink: /pmc/articles/PMC1636279/figure/f7/
number: F7
caption: 'An integrated view of the metabolic pathway comparison between L. monocytogenes
  EGD-e, L. innocua, and L. welshimeri. The map illustrates biochemical pathways for
  main energy productions, such as glycolysis, starch metabolism, pentose phosphate
  pathway, and tricarboxylic acid cycle, which are necessary to maintain the survival
  of the strains. Several amino acid pathways and nucleic acid metabolism are involved
  as well (pathway names are colored blue). The lipid and fatty acid pathways are
  omitted since they are incompletely present. Closed red arrowheads describe the
  corresponding enzymes/reactions absent in L. welshimeri; however, they exist in
  both L. monocytogenes and L. innocua, whereas open red arrowheads show the enzymes/functions
  specifically present in L. monocytogenes. Closed blue arrowheads indicate the reactions
  present in only L. welshimeri, not L. monocytogenes or L. innocua. The gray line
  means the reaction is believed to occur spontaneously. Each gene product with a
  predicted function in ion or solute transport is illustrated in the figure. Proteins
  are grouped by substrate specificity, with transporters for cations (green), anions
  (pink), carbohydrates and organic acids (orange), and amino acids and peptides (blue).
  Ion-coupled permeases are represented by ovals, while ABC transport systems are
  shown as ovals. This construction is concluded from the latest annotation data (NCBI)
  (C. Liang, unpublished data). The functional orthologs in silico are defined using
  a threshold of 50% identity and 75 to 125% alignment coverage rate. The resulting
  strain-specific genes were inspected using InterPro and ExPASy protein domain analysis
  platforms. AICAR, 5-amino-1-(5-phospho-d-ribosyl)imidazole-4-carboxamide; E4P, erythrose-4-phosphate;
  DHF, dihydrofolate; G-1-P, glucose-1-phosphate; Dahp, 3-deoxy-d-arabino-heptulosonate-7-phosphate;
  F-6-P, fructose-6-phosphate; G-6-P, glucose-6-phosphate; GAP, glyceraldehyde-3-phosphate;
  OAA, oxaloacetate; 6PGluco, 6-phosphogluconate; PRPP, 5-phosphoribosyl 1-pyrophosphate;
  R5P, ribose-5-phosphate; Ru5P, ribulose-5-phosphate; S7P, sedaheptulose-7-phosphate;
  TCA, tricarboxylic acid; TCOylcine, 3α,7α,12α-trihydroxy-5β-cholan-24-oylglycine;
  X5P, xylulose-5-phosphate; The enzymes encoded by the indicated genes are as follows:
  lwe0243, xylose isomerase; lwe0244, glycerol kinase; lmo1081, glucose-1-phosphate
  thymidyl transferase; lmo1082, dTDP-sugar epimerase; lmo1083, dTDP-d-glucose 4,6-dehydratase;
  lmo1084, DTDP-l-rhamnose synthetase; lmo2023, l-aspartate oxidase; lmo2067, conjugated
  bile acid hydrolase; lmo2143, mannose-6-phosphate isomerase; and lmo2848, l-rhamnose
  isomerase.'
papertitle: Whole-Genome Sequence of Listeria welshimeri Reveals Common Steps in Genome
  Reduction with Listeria innocua as Compared to Listeria monocytogenes  .
reftext: Torsten Hain, et al. J Bacteriol. 2006 Nov;188(21):7405-7415.
year: '2006'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6709136
figid_alias: PMC1636279__F7
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC1636279__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC1636279__zjb0210661700007.html
  '@type': Dataset
  description: 'An integrated view of the metabolic pathway comparison between L.
    monocytogenes EGD-e, L. innocua, and L. welshimeri. The map illustrates biochemical
    pathways for main energy productions, such as glycolysis, starch metabolism, pentose
    phosphate pathway, and tricarboxylic acid cycle, which are necessary to maintain
    the survival of the strains. Several amino acid pathways and nucleic acid metabolism
    are involved as well (pathway names are colored blue). The lipid and fatty acid
    pathways are omitted since they are incompletely present. Closed red arrowheads
    describe the corresponding enzymes/reactions absent in L. welshimeri; however,
    they exist in both L. monocytogenes and L. innocua, whereas open red arrowheads
    show the enzymes/functions specifically present in L. monocytogenes. Closed blue
    arrowheads indicate the reactions present in only L. welshimeri, not L. monocytogenes
    or L. innocua. The gray line means the reaction is believed to occur spontaneously.
    Each gene product with a predicted function in ion or solute transport is illustrated
    in the figure. Proteins are grouped by substrate specificity, with transporters
    for cations (green), anions (pink), carbohydrates and organic acids (orange),
    and amino acids and peptides (blue). Ion-coupled permeases are represented by
    ovals, while ABC transport systems are shown as ovals. This construction is concluded
    from the latest annotation data (NCBI) (C. Liang, unpublished data). The functional
    orthologs in silico are defined using a threshold of 50% identity and 75 to 125%
    alignment coverage rate. The resulting strain-specific genes were inspected using
    InterPro and ExPASy protein domain analysis platforms. AICAR, 5-amino-1-(5-phospho-d-ribosyl)imidazole-4-carboxamide;
    E4P, erythrose-4-phosphate; DHF, dihydrofolate; G-1-P, glucose-1-phosphate; Dahp,
    3-deoxy-d-arabino-heptulosonate-7-phosphate; F-6-P, fructose-6-phosphate; G-6-P,
    glucose-6-phosphate; GAP, glyceraldehyde-3-phosphate; OAA, oxaloacetate; 6PGluco,
    6-phosphogluconate; PRPP, 5-phosphoribosyl 1-pyrophosphate; R5P, ribose-5-phosphate;
    Ru5P, ribulose-5-phosphate; S7P, sedaheptulose-7-phosphate; TCA, tricarboxylic
    acid; TCOylcine, 3α,7α,12α-trihydroxy-5β-cholan-24-oylglycine; X5P, xylulose-5-phosphate;
    The enzymes encoded by the indicated genes are as follows: lwe0243, xylose isomerase;
    lwe0244, glycerol kinase; lmo1081, glucose-1-phosphate thymidyl transferase; lmo1082,
    dTDP-sugar epimerase; lmo1083, dTDP-d-glucose 4,6-dehydratase; lmo1084, DTDP-l-rhamnose
    synthetase; lmo2023, l-aspartate oxidase; lmo2067, conjugated bile acid hydrolase;
    lmo2143, mannose-6-phosphate isomerase; and lmo2848, l-rhamnose isomerase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - na
  - mn
  - E(spl)mgamma-HLH
  - cu
  - Gip
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - stp
  - Mhc
  - sine
  - pi
  - arg
  - aos
  - Coa
  - CycE
  - cyc
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - mo
  - G6P
  - Hf
  - Acetoacetyl-CoA
  - AICAR
  - Ribosylamine5P
  - Acetate
  - Alanine
  - Aspargine B-alanine
  - Arbutin6P
  - Arginine
  - Arg
  - Aspartate
  - L-Aspartate
  - carnitine
  - choline
  - L-Cysteine
  - Galactitol
  - Formate
  - Fruckone MarnoSorbitol-Fructose
  - Fructose
  - Glucose
  - Glucose Gluconate
  - Glucoside
  - Glutamine
  - Succinate OxoGlutarate
  - Glycerol
  - Glycine
  - glycine
  - Histidine Hidine
  - Isoleucine
  - Lactose
  - Leucine
  - Lysine
  - Mannose
  - Hydranyethyl L-Methionine
  - Nitrite
  - Pentose
  - Pentose phosphate
  - Proline
  - putrescine
  - Serine
  - L-Ser
  - Sorbitol
  - Spermidine
  - Threonine
  - Tryptophan
  - tyrosine
  - UDP
  - UMP-Pyrimidines
  - Valine
  - Glutamate
  - Lactate
  - Ammonium
---
