---
figid: PMC11076791__fcell-12-1356589-g004
figtitle: Proposed regulatory network
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11076791
filename: fcell-12-1356589-g004.jpg
figlink: /pmc/articles/PMC11076791/figure/F4
number: F4
caption: Proposed regulatory network. We integrated relevant regulatory interactions
  identified in experimental results (solid black lines; Figures 1–3), those predicted
  by bioinformatics (dashed lines; Table 1), and those described in the literature
  (solid blue lines) within the sea urchin embryo into a network visualized using
  Cytoscape (ver 3.10.1) (Shannon et al., 2003). The orange colored nodes represent
  components of the signaling pathway; green colored nodes represent transcription
  factors; blue colored genes represent miRNAs; and purple nodes represent specific
  miRNA target transcripts. The horizontal bars indicate known downregulation; the
  black arrows indicate known upregulation; the open arrows indicate predicted effects
  based on bioinformatics analyses. The numbers correspond to specific references
papertitle: Transcription of microRNAs is regulated by developmental signaling pathways
  and transcription factors
reftext: Malcolm Arnott, et al. Front Cell Dev Biol. 2024;12(NA).
year: '2024'
doi: 10.3389/fcell.2024.1356589
journal_title: Frontiers in Cell and Developmental Biology
journal_nlm_ta: Front Cell Dev Biol
publisher_name: Frontiers Media S.A.
keywords: post-transcriptional regulation | sea urchin | gene regulatory network |
  skeletogenesis | miR-1 | miR-31
automl_pathway: 0.8844815
figid_alias: PMC11076791__F4
figtype: Figure
redirect_from: /figures/PMC11076791__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11076791__fcell-12-1356589-g004.html
  '@type': Dataset
  description: Proposed regulatory network. We integrated relevant regulatory interactions
    identified in experimental results (solid black lines; Figures 1–3), those predicted
    by bioinformatics (dashed lines; Table 1), and those described in the literature
    (solid blue lines) within the sea urchin embryo into a network visualized using
    Cytoscape (ver 3.10.1) (Shannon et al., 2003). The orange colored nodes represent
    components of the signaling pathway; green colored nodes represent transcription
    factors; blue colored genes represent miRNAs; and purple nodes represent specific
    miRNA target transcripts. The horizontal bars indicate known downregulation; the
    black arrows indicate known upregulation; the open arrows indicate predicted effects
    based on bioinformatics analyses. The numbers correspond to specific references
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HNF4A
  - SNAI1
  - KIF15
  - FSD1
  - FSD1L
  - WNT1
  - ETS1
  - ETS2
  - MIR7-1
  - GFI1
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - NODAL
  - FOXA1
  - SHH
  - MIR31
  - ALX1
  - MIR9-2
  - ELF3
  - EIF3A
  - BMP1
  - PRCP
  - PGPEP1
  - ROCK1
  - ROCK2
  - HNF1A
  - EPHB2
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - SHH
---
