---
figid: PMC12190641__biomedicines-13-01349-g001
figtitle: Study flowchart
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC12190641
filename: biomedicines-13-01349-g001.jpg
figlink: /pmc/articles/PMC12190641/figure/F1/
number: F1
caption: Study flowchart. Utilizing whole-genome data for AD from the GEO database
  (GSE157194 and GSE193309 were merged for all subsequent analyses), we conducted
  differential analysis and applied WGCNA to identify differentially expressed genes
  associated with cuproptosis (Cu-DEGs). This was followed by an integration of GO/KEGG
  enrichment analysis, gene interaction network construction, and validation of key
  pathways associated with hallmark features, ROC curve evaluation, GSEA/GSVA analysis,
  and immune infiltration profiling. Through this comprehensive approach, we systematically
  untangled the regulatory mechanisms of Cu-DEGs in AD and assessed their diagnostic
  potential
papertitle: 'Decoding Cuproptosis-Sphingolipid-Immune Crosstalk in Atopic Dermatitis:
  A Multi-Omics Network Analysis'
reftext: Xiaowen Wen, et al. Biomedicines. 2025 Jun;13(6).
year: '2025'
doi: 10.3390/biomedicines13061349
journal_title: Biomedicines
journal_nlm_ta: Biomedicines
publisher_name: Multidisciplinary Digital Publishing Institute  (MDPI)
keywords: atopic dermatitis | cuproptosis | immune infiltration | bioinformatics |
  sphingolipid metabolism
automl_pathway: 0.9164448
figid_alias: PMC12190641__F1
figtype: Figure
redirect_from: /figures/PMC12190641__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC12190641__biomedicines-13-01349-g001.html
  '@type': Dataset
  description: Study flowchart. Utilizing whole-genome data for AD from the GEO database
    (GSE157194 and GSE193309 were merged for all subsequent analyses), we conducted
    differential analysis and applied WGCNA to identify differentially expressed genes
    associated with cuproptosis (Cu-DEGs). This was followed by an integration of
    GO/KEGG enrichment analysis, gene interaction network construction, and validation
    of key pathways associated with hallmark features, ROC curve evaluation, GSEA/GSVA
    analysis, and immune infiltration profiling. Through this comprehensive approach,
    we systematically untangled the regulatory mechanisms of Cu-DEGs in AD and assessed
    their diagnostic potential
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DEGS1
  - AMD1
  - NPR2
  - SPTLC2
  - IGSF3
---
