---
figid: PMC11528492__rsob.240022.f005
figtitle: Bioinformatic reconstruction of the L
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11528492
filename: rsob.240022.f005.jpg
figlink: /pmc/articles/PMC11528492/figure/F5/
number: F5
caption: Bioinformatic reconstruction of the L. plasmidifera plastid and its main
  metabolic pathways. The reconstruction is based on the data presented in electronic
  supplementary material, table S5. For simplicity, all the putative plastid-localized
  reactions are shown such as to taking place in the plastid stroma, but it is likely
  that some of them (e.g. the haem synthesis pathway, see main text) occur in the
  periplastidial space (between the second and third plastid envelope membrane). The
  pathways are distinguished by different colours of the intermediates and final products.
  Arrows with solid lines indicate a chemical transformation (reaction), with the
  respective enzymes catalysing the reaction indicated in ovals adjacent to the arrows
  (for the meaning of the abbreviations used see below). The crossed arrows in grey
  (to tryptophan, phenylalanine & tyrosine, galactolipids and cysteine) highlights
  the apparent lack of these common plastidial biosynthetic branches in L. plasmidifera,
  whereas the dotted arrow with a question mark leading from 5,10-methylene-THF to
  10-formyl-THF indicates that the metabolic link is expected but could not be reconstructed
  by our analyses. The green question mark at the synthesis of formylmethionyl-tRNA
  indicates that the process is expected to take place in the L. plasmidifera plastid
  but not directly supported by results of localization predictions for the respective
  enzyme (FMT). Arrows with dashed lines indicate transport of the particular compound
  between cellular compartments (for simplicity, the presumed import of diacylglycerol
  for SQDG biosynthesis from the ER is not illustrated). The complexes of the molecular
  machinery for the import of nuclear genome-encoded proteins into the plastid that
  have at least some of their subunits discernible in the transcriptome data from
  L. plasmidifera, are illustrated in association with the respective membrane bounding
  the plastid (SEC61, protein translocon at the ER; SELMA, translocon in the periplastidial
  membrane; TOC, translocon of the outer envelope membrane; TIC, translocon of the
  inner envelope membrane). Enzyme abbreviations are as follows. Haem synthesis—HemA,
  glutamyl tRNA reductase; HemL, glutamate-1-semialdehyde 2,1-aminomutase; HemB, porphobilinogen
  synthase; HemC, hydroxymethylbilane synthase; HemD, uroporphyrinogen III synthase;
  HemE, uroporphyrinogen decarboxylase; HemF, coproporphyrinogen III oxidase; HemY,
  protoporphyrinogen/coproporphyrinogen III oxidase; HemH, protoporphyrin/coproporphyrin
  ferrochelatase. Tocotrienol synthesis—PAH, phenylalanine-4-hydroxylase; TAT, tyrosine
  aminotransferase; HPD, 4-hydroxyphenylpyruvate dioxygenase; VTE2, homogentisate
  phytyltransferase/homogentisate geranylgeranyltransferase; VTE1, tocopherol cyclase;
  VTE3, MPBQ/MSBQ methytransferase. SQDG (sulphoquinovosyldiacylglycerol) synthesis—UGP2,
  UTP-glucose-1-phosphate uridylyltransferase; USP, UDP-sugar pyrophosphorylase; PHD1,
  plastid type UDP-glucose epimerase; PAPSS, 3'-phosphoadenosine 5'-phosphosulfate
  synthase; CysC, adenylylsulfate kinase; CysH, PAPS reductase; Trx, thioredoxin;
  TRR, thioredoxin reductase; SQD1, UDP-sulfoquinovose synthase; SQD2, sulfoquinovosyltransferase.
  Shikimate pathway—DAHP, 3-deoxy-D-arabino-heptulosonate 7-phosphate synthetase;
  DHQS, 3-dehydroquinate synthase; DHQD, 3-dehydroquinate dehydratase; SDH, shikimate
  dehydrogenase; SK, shikimate kinase); EPSP, 5-enolpyruvylshikimate-3-phosphate synthetase;
  CS, chorismate synthase. Folate synthesis—ADCS, 4-amino-4-deoxychorismate synthase;
  ADCL, 4-amino-4-deoxychorismate lyase; DHPS, dihydropteroate synthase (the activity
  conferred by the protein FolKP, which may in fact localize to the mitochondrion
  in L. plasmidifera), DHFS, dihydrofolate synthase (the activity conferred by the
  protein FolC1). Folate-based one-carbon metabolism—SHMT, serine hydroxymethyltransferase;
  FMT, methionyl tRNA formyltransferase. GluRS, glutamyl tRNA synthetase; MetRS, methionyl
  tRNA synthetase. Protein sequences corresponding to the enzymes and additional details
  are provided in electronic supplementary material, table S5
papertitle: A cryptic plastid and a novel mitochondrial plasmid in Leucomyxa plasmidifera
  gen. and sp. nov. (Ochrophyta) push the frontiers of organellar biology
reftext: Dovilė Barcytė, et al. Open Biol. 2024 Oct;14(10).
year: '2024'
doi: 10.1098/rsob.240022
journal_title: Open Biology
journal_nlm_ta: Open Biol
publisher_name: The Royal Society
keywords: Leukarachnion | mitochondrial plasmids | non-photosynthetic plastid | plastid
  evolution | plastid genome | stramenopiles
automl_pathway: 0.9206259
figid_alias: PMC11528492__F5
figtype: Figure
redirect_from: /figures/PMC11528492__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11528492__rsob.240022.f005.html
  '@type': Dataset
  description: Bioinformatic reconstruction of the L. plasmidifera plastid and its
    main metabolic pathways. The reconstruction is based on the data presented in
    electronic supplementary material, table S5. For simplicity, all the putative
    plastid-localized reactions are shown such as to taking place in the plastid stroma,
    but it is likely that some of them (e.g. the haem synthesis pathway, see main
    text) occur in the periplastidial space (between the second and third plastid
    envelope membrane). The pathways are distinguished by different colours of the
    intermediates and final products. Arrows with solid lines indicate a chemical
    transformation (reaction), with the respective enzymes catalysing the reaction
    indicated in ovals adjacent to the arrows (for the meaning of the abbreviations
    used see below). The crossed arrows in grey (to tryptophan, phenylalanine & tyrosine,
    galactolipids and cysteine) highlights the apparent lack of these common plastidial
    biosynthetic branches in L. plasmidifera, whereas the dotted arrow with a question
    mark leading from 5,10-methylene-THF to 10-formyl-THF indicates that the metabolic
    link is expected but could not be reconstructed by our analyses. The green question
    mark at the synthesis of formylmethionyl-tRNA indicates that the process is expected
    to take place in the L. plasmidifera plastid but not directly supported by results
    of localization predictions for the respective enzyme (FMT). Arrows with dashed
    lines indicate transport of the particular compound between cellular compartments
    (for simplicity, the presumed import of diacylglycerol for SQDG biosynthesis from
    the ER is not illustrated). The complexes of the molecular machinery for the import
    of nuclear genome-encoded proteins into the plastid that have at least some of
    their subunits discernible in the transcriptome data from L. plasmidifera, are
    illustrated in association with the respective membrane bounding the plastid (SEC61,
    protein translocon at the ER; SELMA, translocon in the periplastidial membrane;
    TOC, translocon of the outer envelope membrane; TIC, translocon of the inner envelope
    membrane). Enzyme abbreviations are as follows. Haem synthesis—HemA, glutamyl
    tRNA reductase; HemL, glutamate-1-semialdehyde 2,1-aminomutase; HemB, porphobilinogen
    synthase; HemC, hydroxymethylbilane synthase; HemD, uroporphyrinogen III synthase;
    HemE, uroporphyrinogen decarboxylase; HemF, coproporphyrinogen III oxidase; HemY,
    protoporphyrinogen/coproporphyrinogen III oxidase; HemH, protoporphyrin/coproporphyrin
    ferrochelatase. Tocotrienol synthesis—PAH, phenylalanine-4-hydroxylase; TAT, tyrosine
    aminotransferase; HPD, 4-hydroxyphenylpyruvate dioxygenase; VTE2, homogentisate
    phytyltransferase/homogentisate geranylgeranyltransferase; VTE1, tocopherol cyclase;
    VTE3, MPBQ/MSBQ methytransferase. SQDG (sulphoquinovosyldiacylglycerol) synthesis—UGP2,
    UTP-glucose-1-phosphate uridylyltransferase; USP, UDP-sugar pyrophosphorylase;
    PHD1, plastid type UDP-glucose epimerase; PAPSS, 3'-phosphoadenosine 5'-phosphosulfate
    synthase; CysC, adenylylsulfate kinase; CysH, PAPS reductase; Trx, thioredoxin;
    TRR, thioredoxin reductase; SQD1, UDP-sulfoquinovose synthase; SQD2, sulfoquinovosyltransferase.
    Shikimate pathway—DAHP, 3-deoxy-D-arabino-heptulosonate 7-phosphate synthetase;
    DHQS, 3-dehydroquinate synthase; DHQD, 3-dehydroquinate dehydratase; SDH, shikimate
    dehydrogenase; SK, shikimate kinase); EPSP, 5-enolpyruvylshikimate-3-phosphate
    synthetase; CS, chorismate synthase. Folate synthesis—ADCS, 4-amino-4-deoxychorismate
    synthase; ADCL, 4-amino-4-deoxychorismate lyase; DHPS, dihydropteroate synthase
    (the activity conferred by the protein FolKP, which may in fact localize to the
    mitochondrion in L. plasmidifera), DHFS, dihydrofolate synthase (the activity
    conferred by the protein FolC1). Folate-based one-carbon metabolism—SHMT, serine
    hydroxymethyltransferase; FMT, methionyl tRNA formyltransferase. GluRS, glutamyl
    tRNA synthetase; MetRS, methionyl tRNA synthetase. Protein sequences corresponding
    to the enzymes and additional details are provided in electronic supplementary
    material, table S5
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - C5
  - SHMT1
  - SHMT2
  - SARDH
  - SDHB
  - SORD
  - SDS
  - SEC61A1
  - RHBDF2
  - BMAL1
  - PSD4
  - F8
  - EARS2
  - TRNA
  - F9
  - HEMC
  - ETV6
  - CS
  - AZIN2
  - GADL1
  - DECR1
  - TRO
  - KLK3
  - SH2B2
  - HPD
  - USP1
  - USP2
  - USP3
  - USP4
  - USP5
  - USP6
  - USP7
  - USP8
  - USP10
  - USP11
  - USP12
  - USP13
  - USP14
  - USP15
  - USP16
  - USP18
  - USP19
  - USP20
  - USP21
  - USP22
  - USP24
  - USP25
  - USP26
  - USP27X
  - USP28
  - USP29
  - USP30
  - USP31
  - USP32
  - USP33
  - USP34
  - USP35
  - USP36
  - USP37
  - USP38
  - USP39
  - USP40
  - USP41P
  - USP42
  - USP43
  - USP44
  - USP45
  - USP46
  - USP47
  - USP48
  - USP49
  - USP50
  - USP51
  - PAN2
  - USP53
  - USP54
  - USP9X
  - USP9Y
  - USPL1
  - CYLD
  - USP17L1
  - USP17L2
  - UGP2
  - ATP8A2
  - PAH
  - Serine
  - Glycine
  - 5,10-Methylene-THF
  - D-Erythrose 4-phosphate
  - DAHP
  - 3-Dehydroshikimate
  - Shikimate
  - HemA
  - Glutamate-1-semialdehyde
  - tRNA
  - 10-Formyl-THF
  - Met
  - Porphobilinogen
  - Hydroxymethylbilane
  - HemE
  - Coproporphyrinogen
  - Hem
  - Protoporphyrin
  - TEL
  - TOCOTRIENOL
  - Phosphoenolpyruvate
  - Tryptophan
  - Chorismate
  - 4-Aminobenzoate
  - 7,8-Dihydropteroate
  - 7,8-Dihydrofolate
  - Sulfoquinovosyldiacylglycerol
  - Cystein
  - Protoheme
  - UDP-galactose
  - NADPH
  - UDP-glucose
  - PAPS
  - APS
  - HPD
  - PAPP
  - ATP
  - Tyrosine
  - PAH
  - Phenylalanine
---
