---
figid: PMC11722408__nihpp-2025.01.03.631250v2-f0002
figtitle: Mesenchymal Phenotypes Utilize Autocrine and Paracrine Signaling to Reinforce
  SCLC Subtypes
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11722408
filename: nihpp-2025.01.03.631250v2-f0002.jpg
figlink: /pmc/articles/PMC11722408/figure/F2/
number: F2
caption: 'A) Inferred active pathways in the SCLC RPM dataset. Dot size is representative
  of the relative interaction strength for a given time point. Blue circles are pathways
  known to be involved in EMT signaling. B) Inferred CellChat signaling of pathways
  known to be involved in EMT at TP7. Dot size is proportional to cell number. The
  legend on the right contains the cell type each dot represents. Line color represents
  the source of signal/ligand. Line width represents interaction strength. C) RPM
  TP7 SCLC-P/Y cell type inferred signaling network connecting both intercellular
  and intracellular signaling. The red box contains an illustrative summary of the
  network. The black box is a legend for the network. Node color represents the role
  in the signaling pathway. The shape of the ligand nodes represents the subtype source
  of the ligand. The edges connecting a ligand to a receptor vary in width corresponding
  to interaction strength. Average log2 fold-change of target gene expression is colored
  as indicated by the scale bar. Epithelial target genes are indicated by a yellow
  border around the node and mesenchymal genes have a black border. D) Proportion
  of lineage supporting genes included in (In Network) and excluded from (Out of Network)
  the network. Fisher’s exact test was used to calculate significance. P-values are
  labeled in red. Odds ratios from left to right: 126.99, inf, 54.08. E) Inferred
  CellChat signaling of EMT pathways from other datasets. Left: Human SCLC CDX. Subtypes
  are characterized through archetype analysis. Middle: HCT116 colon cancer. Subtype
  labels are Epi (Epithelial), Mes (Mesenchymal), and pEMT (Partial EMT) and they
  are from the original publication. Left: HER2 breast cancer mouse. Subtype labels
  are HS (Hormone-sensitive), HR-negative (Hormone-receptor negative), T (Transitional
  state), and EMT. The more mesenchymal states in these datasets are, Y, Mes, and
  EMT, respectively'
papertitle: Intercellular signaling reinforces single-cell level phenotypic transitions
  and facilitates robust re-equilibrium of heterogeneous cancer cell populations
reftext: Daniel Lopez, et al. bioRxiv. 2025 Mar 26;NA(NA).
year: '2025'
doi: 10.1101/2025.01.03.631250
journal_title: bioRxiv
journal_nlm_ta: bioRxiv
publisher_name: Cold Spring Harbor Laboratory Preprints
keywords: ''
automl_pathway: 0.7839192
figid_alias: PMC11722408__F2
figtype: Figure
redirect_from: /figures/PMC11722408__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11722408__nihpp-2025.01.03.631250v2-f0002.html
  '@type': Dataset
  description: 'A) Inferred active pathways in the SCLC RPM dataset. Dot size is representative
    of the relative interaction strength for a given time point. Blue circles are
    pathways known to be involved in EMT signaling. B) Inferred CellChat signaling
    of pathways known to be involved in EMT at TP7. Dot size is proportional to cell
    number. The legend on the right contains the cell type each dot represents. Line
    color represents the source of signal/ligand. Line width represents interaction
    strength. C) RPM TP7 SCLC-P/Y cell type inferred signaling network connecting
    both intercellular and intracellular signaling. The red box contains an illustrative
    summary of the network. The black box is a legend for the network. Node color
    represents the role in the signaling pathway. The shape of the ligand nodes represents
    the subtype source of the ligand. The edges connecting a ligand to a receptor
    vary in width corresponding to interaction strength. Average log2 fold-change
    of target gene expression is colored as indicated by the scale bar. Epithelial
    target genes are indicated by a yellow border around the node and mesenchymal
    genes have a black border. D) Proportion of lineage supporting genes included
    in (In Network) and excluded from (Out of Network) the network. Fisher’s exact
    test was used to calculate significance. P-values are labeled in red. Odds ratios
    from left to right: 126.99, inf, 54.08. E) Inferred CellChat signaling of EMT
    pathways from other datasets. Left: Human SCLC CDX. Subtypes are characterized
    through archetype analysis. Middle: HCT116 colon cancer. Subtype labels are Epi
    (Epithelial), Mes (Mesenchymal), and pEMT (Partial EMT) and they are from the
    original publication. Left: HER2 breast cancer mouse. Subtype labels are HS (Hormone-sensitive),
    HR-negative (Hormone-receptor negative), T (Transitional state), and EMT. The
    more mesenchymal states in these datasets are, Y, Mes, and EMT, respectively'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AGRN
  - ABCA4
  - GNA12
  - URI1
  - CD46
  - CDH1
  - FZR1
  - COL6A1
  - COL6A2
  - COL6A3
  - COL6A5
  - COL6A6
  - COL7A1
  - COL8A1
  - COL8A2
  - COL9A1
  - COL9A2
  - COL9A3
  - COL10A1
  - COL11A1
  - COL11A2
  - COL12A1
  - COL13A1
  - COL14A1
  - COL15A1
  - COL16A1
  - COL17A1
  - COL18A1
  - COL19A1
  - COL20A1
  - COL21A1
  - COL22A1
  - COL23A1
  - COL24A1
  - COL25A1
  - COL26A1
  - COL27A1
  - COL28A1
  - COL1A1
  - COL1A2
  - COL4A1
  - COL4A2
  - COL4A3
  - COL4A4
  - COL4A5
  - COL4A6
  - COL5A1
  - COL5A2
  - COL5A3
  - CSPG4
  - EGF
  - ENHO
  - EPHA1
  - EPHA2
  - EPHA3
  - EPHA4
  - EPHA5
  - EPHA6
  - EPHA7
  - EPHA8
  - EPHA10
  - EPHB1
  - EPHB2
  - EPHB3
  - EPHB4
  - EPHB6
  - FGF1
  - FGF2
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FGF10
  - FGF11
  - FGF12
  - FGF13
  - FGF14
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FN1
  - GAST
  - GALNS
  - PAGR1
  - PAEP
  - CORIN
  - CRNKL1
  - HSPG2
  - SDC2
  - F11R
  - KIT
  - L1CAM
  - LIFR
  - MPZ
  - NCAM1
  - ERBB2
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - NRG1
  - NRG2
  - NRG3
  - NRG4
  - OCLN
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - GPR162
  - PSD
  - SMAD1
  - SMAD4
  - SMAD5
  - SMAD6
  - SMAD7
  - SMAD9
  - SMAD2
  - SMAD3
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - JUN
  - JUNB
  - JUND
  - AN
  - JAG1
  - PTN
  - PTPRM
  - YBX3
  - TPD52
  - SP1
  - PSG1
  - DAND5
  - HES1
  - GATD3
  - TH
  - SEMA3F
  - SEMA3B
  - SPP1
  - CXXC1
  - SGPP1
  - TGFB1
  - TGFB2
  - TGFB3
  - CA6
  - AXL
  - THBS1
  - THBS2
  - THBS3
  - THBS4
  - COMP
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - PGF
  - NAMPT
  - ME1
  - MKS1
  - TCF23
  - PEMT
  - MUC1
  - TFPI
  - ITK
  - SLC22A3
  - RMP
  - CDH
  - COLLAGEN
  - EGF
  - CRN
  - PTN
  - Mes
---
