---
figid: PMC11800652__40793_2025_669_Fig5_HTML
figtitle: Metabolic maps of Sulfurovum species from Kebrit Deep BSI
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11800652
filename: 40793_2025_669_Fig5_HTML.jpg
figlink: /pmc/articles/PMC11800652/figure/F5/
number: F5
caption: Metabolic maps of Sulfurovum species from Kebrit Deep BSI. Overview of the
  metabolism inferred from the genome of Candidatus Sulfurovum kebritense sp. nov.
  (Kbt_02, upper cell) and Candidatus Sulfurovum profundilacunae sp. nov. (Kbt_03,
  lower cell). The models showcase the presence of genes involved in carbon, sulfur,
  nitrogen, and iron regulation pathways, hydrogenases, cytochromes, oxygen detoxification
  mechanisms, and transport or biosynthesis of compatible solutes. Abbreviations of
  genes are stated in the main text. Created with BioRender.com
papertitle: Metabolic redundancy and specialisation of novel sulfide-oxidizing Sulfurimonas
  and Sulfurovum along the brine-seawater interface of the Kebrit Deep
reftext: Rayyan Alamoudi, et al. Environ Microbiome. 2025;20(NA).
year: '2025'
doi: 10.1186/s40793-025-00669-7
journal_title: Environmental Microbiome
journal_nlm_ta: Environ Microbiome
publisher_name: BMC
keywords: Deep hypersaline anoxic basin (DHAB) | Sulfidic DHAB | Brine pool | Sulfur
  metabolism | Campylobacterota | Metabolic adaptation | Microbial networking | Metagenome
  | Red Sea
automl_pathway: 0.9588051
figid_alias: PMC11800652__F5
figtype: Figure
redirect_from: /figures/PMC11800652__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11800652__40793_2025_669_Fig5_HTML.html
  '@type': Dataset
  description: Metabolic maps of Sulfurovum species from Kebrit Deep BSI. Overview
    of the metabolism inferred from the genome of Candidatus Sulfurovum kebritense
    sp. nov. (Kbt_02, upper cell) and Candidatus Sulfurovum profundilacunae sp. nov.
    (Kbt_03, lower cell). The models showcase the presence of genes involved in carbon,
    sulfur, nitrogen, and iron regulation pathways, hydrogenases, cytochromes, oxygen
    detoxification mechanisms, and transport or biosynthesis of compatible solutes.
    Abbreviations of genes are stated in the main text. Created with BioRender.com
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - OPRD1
  - SQOR
  - SAT1
  - KLK3
  - SH2B2
  - SOD1
  - SOD2
  - SOD3
  - PAEP
  - PREP
  - PTPN22
  - GNPTAB
  - POR
  - VDAC2
  - MDH2
  - RTCA
  - PLA2G15
  - ACSS2
  - ACCS
  - OPRL1
  - GLUD1
  - UGDH
  - H6PD
  - CRYL1
  - VDAC1
  - FURIN
  - ATP8A2
  - WDTC1
  - DNAH8
  - SARDH
  - SDHB
  - SORD
  - SDS
  - NOS1
  - NANOS1
  - BRCA2
  - FANCD2
  - PSEN1
  - NOC2L
  - APRT
  - MFAP1
  - TNFRSF11A
  - SO2
  - Glucose
  - 2-oxoglutarate
  - Glutamine
  - APS
  - Cysteine
  - Glutamate
  - Ectoine
  - Oxaloacetate
  - Pyruvate
  - Acetyl-CoA
  - Malate
  - Proline
  - L-Glutamate
  - L-Glutamine
  - Acetate
  - Succinyl-CoA
  - Fumarate
  - Succinate
  - ATP
  - NADH
  - Cytochrome c
  - FAD
  - AMP
  - PEP
  - Citrate
---
