---
figid: PMC10142491__viruses-15-00911-g001
figtitle: 'Early-Phase Drive to the Precursor Pool: Chloroviruses Dive into the Deep
  End of Nucleotide Metabolism'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC10142491
filename: viruses-15-00911-g001.jpg
figlink: /pmc/articles/PMC10142491/figure/viruses-15-00911-f001/
number: F1
caption: 'KEGG pathway of the pyrimidine metabolism, annotated with C. variabilis
  NC64A and PBCV-1 genes. From genomic analyses of C. variabilis NC64A and the Chlorovirus
  PBCV-1, functional annotations of genes were mapped onto a KEGG (Kyoto Encyclopedia
  of Genes and Genomes) flow chart for pyrimidine metabolism. The chart indicates
  the (i) name of a metabolite, with the position indicated by “o”, (ii) enzyme function
  responsible for a metabolite conversion (annotated by the Enzyme Commission (E.C.)
  number), (iii) flux direction, indicated by a connecting arrow, and (iv) associated
  metabolic pathways that feed into or out of the pyrimidine metabolism pathway. Where
  the C. variabilis gene was identified, the E.C. number was color-coded green. Where
  the PBCV-1 homologous gene was identified, the E. C. number was color-coded red.
  If both the C. variabilis NC64A and PBCV-1 homologous gene was identified, the E.
  C. number was color-coded green and red. If no gene was identified, the E. C. number
  was left uncoded and appears as a white box. The information for the genomic analyses
  of the virus and host cell are found in , and the lists of genes used for the KEGG
  Mapper to the Pyrimidine Metabolism Pathway are found in . (A) Ribonucleoside-diphosphate
  reductase (E.C.1.17.4.1): augmenting enzyme of both purine and pyrimidine metabolism
  pathways. (B) Cytosine deaminase (E.C.3.5.4.1): complementing enzyme of pyrimidine
  metabolism. (C) Aspartate/ornithine carbamoyltransferase (E.C.2.1.3.2): complementing
  enzyme of the pyrimidine metabolism that is not sensitive to nucleotide feedback
  regulations. (D) dUTP pyrophosphatase (E.C.3.6.1.23): augmenting enzyme of the pyrimidine
  metabolism. (E) Deoxynucleoside kinase (E.C.2.7.4.14): augmenting enzyme of pyrimidine
  metabolism but not purine metabolism. (F) dCMP deaminase (E.C.3.5.4.12): augmenting
  enzyme of pyrimidine metabolism that is sensitive to dCTP positive feedback and
  dTTP negative feedback. (G) dCTP deaminase (E.C.3.5.4.13): complementing enzyme
  of pyrimidine metabolism that is sensitive to dTTP as a negative feedback. (H) Thymidylate
  synthase X (E.C.2.1.1.148): complementing enzyme of pyrimidine metabolism that may
  override the host-encoded ThyA; ThyX is FADH2-dependent.'
papertitle: 'Early-Phase Drive to the Precursor Pool: Chloroviruses Dive into the
  Deep End of Nucleotide Metabolism.'
reftext: David D. Dunigan, et al. Viruses. 2023 Apr;15(4):911.
year: '2023'
doi: 10.3390/v15040911
journal_title: Viruses
journal_nlm_ta: Viruses
publisher_name: MDPI
keywords: algae | virus | chlorovirus | giant virus | pyrimidine biosynthesis | nucleotide
  biosynthesis | metabolic reprogramming | genome-to-phenome | augmenting/auxiliary
  metabolic genes
automl_pathway: 0.7538056
figid_alias: PMC10142491__F1
figtype: Figure
redirect_from: /figures/PMC10142491__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10142491__viruses-15-00911-g001.html
  '@type': Dataset
  description: 'KEGG pathway of the pyrimidine metabolism, annotated with C. variabilis
    NC64A and PBCV-1 genes. From genomic analyses of C. variabilis NC64A and the Chlorovirus
    PBCV-1, functional annotations of genes were mapped onto a KEGG (Kyoto Encyclopedia
    of Genes and Genomes) flow chart for pyrimidine metabolism. The chart indicates
    the (i) name of a metabolite, with the position indicated by “o”, (ii) enzyme
    function responsible for a metabolite conversion (annotated by the Enzyme Commission
    (E.C.) number), (iii) flux direction, indicated by a connecting arrow, and (iv)
    associated metabolic pathways that feed into or out of the pyrimidine metabolism
    pathway. Where the C. variabilis gene was identified, the E.C. number was color-coded
    green. Where the PBCV-1 homologous gene was identified, the E. C. number was color-coded
    red. If both the C. variabilis NC64A and PBCV-1 homologous gene was identified,
    the E. C. number was color-coded green and red. If no gene was identified, the
    E. C. number was left uncoded and appears as a white box. The information for
    the genomic analyses of the virus and host cell are found in , and the lists of
    genes used for the KEGG Mapper to the Pyrimidine Metabolism Pathway are found
    in . (A) Ribonucleoside-diphosphate reductase (E.C.1.17.4.1): augmenting enzyme
    of both purine and pyrimidine metabolism pathways. (B) Cytosine deaminase (E.C.3.5.4.1):
    complementing enzyme of pyrimidine metabolism. (C) Aspartate/ornithine carbamoyltransferase
    (E.C.2.1.3.2): complementing enzyme of the pyrimidine metabolism that is not sensitive
    to nucleotide feedback regulations. (D) dUTP pyrophosphatase (E.C.3.6.1.23): augmenting
    enzyme of the pyrimidine metabolism. (E) Deoxynucleoside kinase (E.C.2.7.4.14):
    augmenting enzyme of pyrimidine metabolism but not purine metabolism. (F) dCMP
    deaminase (E.C.3.5.4.12): augmenting enzyme of pyrimidine metabolism that is sensitive
    to dCTP positive feedback and dTTP negative feedback. (G) dCTP deaminase (E.C.3.5.4.13):
    complementing enzyme of pyrimidine metabolism that is sensitive to dTTP as a negative
    feedback. (H) Thymidylate synthase X (E.C.2.1.1.148): complementing enzyme of
    pyrimidine metabolism that may override the host-encoded ThyA; ThyX is FADH2-dependent.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - ctp
  - epsilonCOP
  - Mp
  - cmp
  - Fs(3)Var
  - rut
  - SLC25A1
  - SLC15A3
  - COPE
  - VOPP1
  - MATN1
  - L-aspartate
  - 'N'
  - phosphate
  - L-Glutamine
  - Alanine
  - aspartate
  - glutamate
  - UTP
  - Oretidine
  - UDP
  - glucose
  - dTMP
  - Cytidine
  - Uridine
  - Deoxyridine
  - Deoxycytidine
  - Thymidine
  - Uracil
  - Pseudouridine
  - 3-phosphate
  - 2-Deasy-6-ribose-P
  - Dihydrouracil
  - propionate
  - Barbiturate
  - 3-Oxo-3-reid
  - Carbamate
  - 121-3-Uraidoacrylate
  - 21-3-Peroxy- Aminoacrylate
  - aminoacrylate
  - Malonate
  - semialdehyde
  - 3-Methylbarbiturate
  - 1-Oxo-3-ureida- Methylmalonate
  - isobutyrate
  - Dihydro (R)-3-Ureido- (R)-3-Amino
  - thymine
  - 3-Hydroxy
  - Valine
  - isoleucine
  - semialdehyde
---
