---
figid: PMC10142491__viruses-15-00911-g002
figtitle: 'Early-Phase Drive to the Precursor Pool: Chloroviruses Dive into the Deep
  End of Nucleotide Metabolism'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC10142491
filename: viruses-15-00911-g002.jpg
figlink: /pmc/articles/PMC10142491/figure/viruses-15-00911-f002/
number: F2
caption: KEGG pathway of the pyrimidine metabolism annotated with C. variabilis NC64A
  and PBCV-1 genes and comprehensive “roadway” predictions. C. variabilis NC64A genes
  are colored green, and PBCV-1 genes are red. This figure summarizes the virus-mediated
  metabolic fluxes where viral genes either (i) complement the pathway by providing
  a function not identified in the host genome, and we refer to this as a “roadway
  by-pass” (indicated by a blue arrow); or (ii) augment the pathway by providing a
  function that is present in the host genome, and we refer to this as a “roadway
  passing lane” (indicated by a yellow arrow). Metabolite-directed flux rates of virus-encoded
  enzymes are sensitive to either positive feedback regulators (indicated by a red
  solid line with a plus sign “+”) or negative feedback regulators (indicated by a
  red dashed line with a minus sign “−”).
papertitle: 'Early-Phase Drive to the Precursor Pool: Chloroviruses Dive into the
  Deep End of Nucleotide Metabolism.'
reftext: David D. Dunigan, et al. Viruses. 2023 Apr;15(4):911.
year: '2023'
doi: 10.3390/v15040911
journal_title: Viruses
journal_nlm_ta: Viruses
publisher_name: MDPI
keywords: algae | virus | chlorovirus | giant virus | pyrimidine biosynthesis | nucleotide
  biosynthesis | metabolic reprogramming | genome-to-phenome | augmenting/auxiliary
  metabolic genes
automl_pathway: 0.8671638
figid_alias: PMC10142491__F2
figtype: Figure
redirect_from: /figures/PMC10142491__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10142491__viruses-15-00911-g002.html
  '@type': Dataset
  description: KEGG pathway of the pyrimidine metabolism annotated with C. variabilis
    NC64A and PBCV-1 genes and comprehensive “roadway” predictions. C. variabilis
    NC64A genes are colored green, and PBCV-1 genes are red. This figure summarizes
    the virus-mediated metabolic fluxes where viral genes either (i) complement the
    pathway by providing a function not identified in the host genome, and we refer
    to this as a “roadway by-pass” (indicated by a blue arrow); or (ii) augment the
    pathway by providing a function that is present in the host genome, and we refer
    to this as a “roadway passing lane” (indicated by a yellow arrow). Metabolite-directed
    flux rates of virus-encoded enzymes are sensitive to either positive feedback
    regulators (indicated by a red solid line with a plus sign “+”) or negative feedback
    regulators (indicated by a red dashed line with a minus sign “−”).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - ctp
  - Mp
  - cmp
  - Egfr
  - Fs(3)Var
  - SLC25A1
  - SLC15A3
  - IL6
  - LIF
  - MATN1
  - THOP1
  - LINC00273
  - TOP1
  - TOP1MT
  - TOP3A
  - TOP3B
  - TOP2A
  - TOP2B
  - L-aspartate
  - Pentose phosphate
  - N-Carbamayl-4-352304
  - Argenine
  - L-Glutamine
  - Alanine
  - aspartate
  - UTP
  - Orotidine
  - ACDP
  - glucose
  - Uridine
  - Jeoxyridine
  - Cytosine
  - 2,3-Cyc
  - Thymidine
  - 3-Methylcytosine
  - D-ribose
  - 5-phosphate
  - pihydrouracil
  - Thymine
  - 3-Oxe-3-ureide
  - propanoate
  - O Pseudauridine
  - 3-Ureide
  - propionate
  - Carbamate
  - Malonate
  - (2)-3-Ureidoscrylate (2)-3-Perexy Aminoacrylate
  - aminoacrylate
  - urea
  - semialdehyde
  - 5-Methylbarbiturate
  - 3-Oxe-3-ureide Methylmalonate
  - isobutyrate
  - RJ-3-Ureide
  - R1-3-Amino
  - 3-Hydroxy
  - semialdehyde
---
