---
figid: PMC11969464__awae319f3
figtitle: Visualization analysis of pathways identified from normalized, significantly
  differentially-SUMO-enriched proteins
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11969464
filename: awae319f3.jpg
figlink: /pmc/articles/PMC11969464/figure/F3/
number: F3
caption: 'Visualization analysis of pathways identified from normalized, significantly
  differentially-SUMO-enriched proteins. (A) An ontology network, constructed from
  the protein interaction network extractor (PINE) tool, yielded enrichment-related
  relationships based on peptide-level fold-changes for Huntington’s disease (HD)
  versus non-transgenic (NT) conditions between the following Gene Ontology terms:
  regulation of synaptic plasticity; synaptic vesicle membrane; activation of NMDA
  receptors and postsynaptic events; neuron to neuron synapse; and activation of the
  mRNA on binding of cap-binding complex and eIFs and binding to 43S pathways. The
  fold-change of each peptide is projected onto its corresponding protein and is represented
  in red (for upregulated peptides) and/or blue (for downregulated peptides); the
  darker the shade, the greater the degree of fold-change between the HD versus NT
  conditions. For proteins with more than one significantly differentially expressed
  peptide (for example, SYNPO), the node is sectioned into multiple peptides, with
  each section depicting the fold-change associated with its corresponding peptide.
  Additionally, the central enrichment nodes (showing the various ontologies) are
  in orange or blue to depict overall up- or down-regulation, respectively, based
  on whether the majority of their associated peptides were up- or down-regulated.
  (B) The doughnut network depicts protein–protein interactions for proteins associated
  with the pathway terms in A. Each protein is shaped in the form of a doughnut, with
  the colour of the doughnut representing the associated pathway terms. For proteins
  associated with more than one pathway term (for example, ACTN2), the doughnut is
  sectioned into multiple colours, each representing a different pathway term'
papertitle: Dysregulation of protein SUMOylation networks in Huntington’s disease
  R6/2 mouse striatum
reftext: Marketta Kachemov, et al. Brain. 2025 Apr;148(4).
year: '2025'
doi: 10.1093/brain/awae319
journal_title: Brain
journal_nlm_ta: Brain
publisher_name: Oxford University Press
keywords: Huntington’s disease | SUMO | proteomics | PIAS1 | mGLUR7 | E3 SUMO ligase
  | synaptic
automl_pathway: 0.7964091
figid_alias: PMC11969464__F3
figtype: Figure
redirect_from: /figures/PMC11969464__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11969464__awae319f3.html
  '@type': Dataset
  description: 'Visualization analysis of pathways identified from normalized, significantly
    differentially-SUMO-enriched proteins. (A) An ontology network, constructed from
    the protein interaction network extractor (PINE) tool, yielded enrichment-related
    relationships based on peptide-level fold-changes for Huntington’s disease (HD)
    versus non-transgenic (NT) conditions between the following Gene Ontology terms:
    regulation of synaptic plasticity; synaptic vesicle membrane; activation of NMDA
    receptors and postsynaptic events; neuron to neuron synapse; and activation of
    the mRNA on binding of cap-binding complex and eIFs and binding to 43S pathways.
    The fold-change of each peptide is projected onto its corresponding protein and
    is represented in red (for upregulated peptides) and/or blue (for downregulated
    peptides); the darker the shade, the greater the degree of fold-change between
    the HD versus NT conditions. For proteins with more than one significantly differentially
    expressed peptide (for example, SYNPO), the node is sectioned into multiple peptides,
    with each section depicting the fold-change associated with its corresponding
    peptide. Additionally, the central enrichment nodes (showing the various ontologies)
    are in orange or blue to depict overall up- or down-regulation, respectively,
    based on whether the majority of their associated peptides were up- or down-regulated.
    (B) The doughnut network depicts protein–protein interactions for proteins associated
    with the pathway terms in A. Each protein is shaped in the form of a doughnut,
    with the colour of the doughnut representing the associated pathway terms. For
    proteins associated with more than one pathway term (for example, ACTN2), the
    doughnut is sectioned into multiple colours, each representing a different pathway
    term'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TIMM50
  - GSK3A
  - APOO
  - AGK
  - LETM1
  - BRSK1
  - RAC1
  - RNASE1
  - ACTN2
  - ARF1
  - ARF1P1
  - ERC2
  - GRM5
  - EPHA4
  - SYNPO
  - BSN
  - PTPN11
  - SYT7
  - AMPH
  - RAB3C
  - MAP4
  - DAP3
  - DLGAP3
  - SEPTIN11
  - RHEB
  - RHEBP1
  - RTN4
  - HOMER1
  - EIF3A
  - EIF6
  - EIF3J
  - EIF3K
  - CDKN3
  - NAPSA
  - SPTBN1
  - CTAA1
  - LNPEP
  - SERPINB6
  - HACD1
  - CAP1
  - SORBS1
  - BRD4
  - CAP2
---
