---
figid: PMC10960062__gr6
figtitle: Trans-omic analysis reveals opposite metabolic dysregulation between feeding
  and fasting in liver associated with obesity
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC10960062
filename: gr6.jpg
figlink: /pmc/articles/PMC10960062/figure/F6
number: F6
caption: Comparison of the differential regulatory trans-omic network and glycolysis/gluconeogenesis
  pathway between ad libitum feeding and 16 h fasting(A) Differential regulatory trans-omic
  network including differentially expressed molecules and differential regulations
  at ad libitum feeding and 16 h fasting (left).34 The percentages are calculated
  by dividing the total number of measured molecules or background molecules. The
  numbers of activating and inhibiting Allosteric Regulation (edge) regulated by Metabolite
  (Allosteric Regulator) (node) in ob/ob mice (right). The size of node and width
  of edge indicate the number of Metabolite (Allosteric Regulator) and Allosteric
  Regulation; we show the numbers around nodes and edges. See also Figures S8–S11.(B)
  Metabolic reactions, molecules, and regulations in Glycolysis/Gluconeogenesis pathway
  (mmu00010) extracted from the differential regulatory trans-omic network at ad libitum
  feeding (top left) and 16 h fasting (bottom left).34 Bar plots are shown on the
  corresponding nodes as the means and standard error of the mean of replicates in
  WT and ob/ob mice. For metabolites as substrates or products in Glycolysis/Gluconeogenesis
  pathway, we showed bar plots for all measured molecules. The colors of frame and
  edge indicate the changed direction. Diamond nodes indicate metabolic reactions.
  Black edges indicate the metabolic reactions between substrates and products. Black
  circles indicate the metabolites that were not measured. From metabolite to metabolic
  reaction, only allosteric regulations are colored. We counted the numbers of differentially
  expressed TF, Enzyme mRNA, Enzyme Protein, Metabolite, Metabolite (Allosteric Regulator),
  and differential regulations in Glycolysis/Gluconeogenesis pathway at ad libitum
  feeding (top right) and 16 h fasting (bottom right). We used the size of node and
  the width of edge to indicate them. For those with more than 30 or less than 2,
  we used one size respectively (>30, 30, <2, 2) in case too big or too small nodes
  and edges. Specific numbers were shown around nodes and edges. Note that the plotting
  scales are different from Figures 6A (right) and 6B (right). See Figures S12–S18
  for other metabolic pathways. Red, increased molecules or activating regulations
  in ob/ob mice; blue, decreased molecules or inhibiting regulations in ob/ob mice;
  gray, unchanged molecules between WT and ob/ob mice; magenta, metabolic reaction
  is both activated and inhibited in ob/ob mice
papertitle: Trans-omic analysis reveals opposite metabolic dysregulation between feeding
  and fasting in liver associated with obesity
reftext: Yunfan Bai, et al. iScience. 2024 Mar 15;27(3).
year: '2024'
doi: 10.1016/j.isci.2024.109121
journal_title: iScience
journal_nlm_ta: iScience
publisher_name: Elsevier
keywords: Biological sciences | Physiology | Proteomics | Metabolomics | Transcriptomics
automl_pathway: 0.7978996
figid_alias: PMC10960062__F6
figtype: Figure
redirect_from: /figures/PMC10960062__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10960062__gr6.html
  '@type': Dataset
  description: Comparison of the differential regulatory trans-omic network and glycolysis/gluconeogenesis
    pathway between ad libitum feeding and 16 h fasting(A) Differential regulatory
    trans-omic network including differentially expressed molecules and differential
    regulations at ad libitum feeding and 16 h fasting (left).34 The percentages are
    calculated by dividing the total number of measured molecules or background molecules.
    The numbers of activating and inhibiting Allosteric Regulation (edge) regulated
    by Metabolite (Allosteric Regulator) (node) in ob/ob mice (right). The size of
    node and width of edge indicate the number of Metabolite (Allosteric Regulator)
    and Allosteric Regulation; we show the numbers around nodes and edges. See also
    Figures S8–S11.(B) Metabolic reactions, molecules, and regulations in Glycolysis/Gluconeogenesis
    pathway (mmu00010) extracted from the differential regulatory trans-omic network
    at ad libitum feeding (top left) and 16 h fasting (bottom left).34 Bar plots are
    shown on the corresponding nodes as the means and standard error of the mean of
    replicates in WT and ob/ob mice. For metabolites as substrates or products in
    Glycolysis/Gluconeogenesis pathway, we showed bar plots for all measured molecules.
    The colors of frame and edge indicate the changed direction. Diamond nodes indicate
    metabolic reactions. Black edges indicate the metabolic reactions between substrates
    and products. Black circles indicate the metabolites that were not measured. From
    metabolite to metabolic reaction, only allosteric regulations are colored. We
    counted the numbers of differentially expressed TF, Enzyme mRNA, Enzyme Protein,
    Metabolite, Metabolite (Allosteric Regulator), and differential regulations in
    Glycolysis/Gluconeogenesis pathway at ad libitum feeding (top right) and 16 h
    fasting (bottom right). We used the size of node and the width of edge to indicate
    them. For those with more than 30 or less than 2, we used one size respectively
    (>30, 30, <2, 2) in case too big or too small nodes and edges. Specific numbers
    were shown around nodes and edges. Note that the plotting scales are different
    from Figures 6A (right) and 6B (right). See Figures S12–S18 for other metabolic
    pathways. Red, increased molecules or activating regulations in ob/ob mice; blue,
    decreased molecules or inhibiting regulations in ob/ob mice; gray, unchanged molecules
    between WT and ob/ob mice; magenta, metabolic reaction is both activated and inhibited
    in ob/ob mice
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MICE
  - TF
  - PPARG
  - BRD4
  - FOXA1
  - CREBBP
  - TAF3
  - RXRA
  - DUOXA1
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - G6PC1
  - PIGQ
  - ALDOB
  - FBP1
  - ALDOC
  - ALDOA
  - AGXT2
  - PKLR
  - GPT2
  - LDHA
  - ATP8A2
  - WDTC1
  - PGK1
  - GCK
  - MAP4K2
  - FAM20C
  - PDHB
  - GPT
  - DPAGT1
  - PAEP
  - PREP
  - PTPN22
  - NT5C2
  - GPS2
  - PROX1
  - EP300
  - XBP1
  - XBP1P1
  - RORA
  - STAG2
  - TBP
  - NELFA
  - NCOR2
  - NR1D1
  - PFKL
  - AGXT
  - PKM
  - TPI1
  - APRT
  - MFAP1
  - Metabolite
  - mRNA
  - DHAP
  - ATP
  - ADP
  - Alanine
  - PEP
  - Pyruvate
  - Lactate
  - Protein
---
