---
figid: PMC10418499__ijms-24-12387-g005
figtitle: Main metabolic pathways of Chao2R and RPY geng seedlings in response to
  saline–alkaline stress
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC10418499
filename: ijms-24-12387-g005.jpg
figlink: /pmc/articles/PMC10418499/figure/F5
number: F5
caption: Main metabolic pathways of Chao2R and RPY geng seedlings in response to saline–alkaline
  stress. Metabolites and genes were mapped to an integrated systemic metabolic pathway
  diagram based on combining the KEGG pathways of DAMs and DEGs. The first and second
  boxes indicate the log2 (fold change) of metabolites in Chao2R and RPY geng, respectively,
  after subjected to saline–alkaline treatment in contrast to untreated control, in
  which red indicates upregulation, and blue indicates downregulation. The metabolic
  pathways involvement in alkaloid, phenolic acid, lipid, amino acid, organic acid,
  saccharide, and plant signal transduction are marked in purple, green, blue, orange,
  black, light green, and red, respectively
papertitle: Integrated Transcriptomic and Metabolomic Analyses Uncover the Differential
  Mechanism in Saline–Alkaline Tolerance between Indica and Japonica Rice at the Seedling
  Stage
reftext: Jianyong Wang, et al. Int J Mol Sci. 2023 Aug;24(15).
year: '2023'
doi: 10.3390/ijms241512387
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: rice (Oryza sativa L.) | indica–japonica subspecies | saline–alkaline stress
  | transcriptome | metabolome
automl_pathway: 0.8360478
figid_alias: PMC10418499__F5
figtype: Figure
redirect_from: /figures/PMC10418499__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10418499__ijms-24-12387-g005.html
  '@type': Dataset
  description: Main metabolic pathways of Chao2R and RPY geng seedlings in response
    to saline–alkaline stress. Metabolites and genes were mapped to an integrated
    systemic metabolic pathway diagram based on combining the KEGG pathways of DAMs
    and DEGs. The first and second boxes indicate the log2 (fold change) of metabolites
    in Chao2R and RPY geng, respectively, after subjected to saline–alkaline treatment
    in contrast to untreated control, in which red indicates upregulation, and blue
    indicates downregulation. The metabolic pathways involvement in alkaloid, phenolic
    acid, lipid, amino acid, organic acid, saccharide, and plant signal transduction
    are marked in purple, green, blue, orange, black, light green, and red, respectively
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - KLK15
  - ANO1
  - ANO10
  - ANO2
  - ANO3
  - ANO4
  - ANO5
  - ANO6
  - ANO7
  - ANO8
  - ANO9
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - PDXP
  - CTSE
  - FCRL3
  - SULT1E1
  - MAL
  - MRTFA
  - TIRAP
  - ACSL1
  - ACSL3
  - ACSL4
  - ACSL5
  - ACSL6
  - PCOLCE
  - DES
  - LCT
  - ZNF346
  - LRWD1
  - DNA
  - alkaloid
  - DOPA
  - Rosmarinate
  - Prephenate
  - Chorismate
  - Glycine
  - Methyl
  - maleate
  - Acetyl-CoA
  - Ethylene
  - Glutathione
  - 5-Aminolevulinate
  - D-glucose
  - aldehyde
  - ethylamine
  - COM
  - alcohol
  - ferulic acid
  - feruloyl-CoA
  - succinate
  - Piperidine
  - putrescine
  - Gibberellin
  - Abscisic acid
  - Dimethylglycine
  - MAL
  - 3-propylmalate
  - Des
  - Phosphatidylcholine
  - acid
  - Jasmonic acid
  - Disease
---
