---
figid: PMC1976331__pgen.0030162.g004
figtitle: Linking Metabolic QTLs with Network and cis-eQTLs Controlling Biosynthetic
  Pathways
organisms:
- NA
organisms_ner:
- Arabidopsis thaliana
pmcid: PMC1976331
filename: pgen.0030162.g004.jpg
figlink: /pmc/articles/PMC1976331/figure/pgen-0030162-g004/
number: F4
caption: Results of ANOVA testing of all identified aliphatic glucosinolate QTLs for
  significant impact upon the accumulation of individual glucosinolate metabolites,
  transcript level of all biosynthetic genes, total aliphatic glucosinolate content,
  and the average expression of the aliphatic glucosinolate biosynthetic pathway is
  presented graphically. The genes and metabolites are shown with respect to the currently
  theorized biosynthetic pathway. Trait abbreviations are as listed in . Gene names
  are as listed in , and TAIR locus identifiers are used for gene families where there
  is no settled naming system.Cells within boxes represent aliphatic glucosinolate
  QTLs. The legend at the bottom right contains the QTL name. Cells representing QTLs
  significantly controlling the represented trait are colored to show the directionality
  of the allele substitution effect; a positive effect of the Bay-0 allele is blue,
  and a positive effect of the Sha allele is red. Dark red and dark blue show that
  the allele substitution at the given QTL led to greater than 50% phenotypic change
  in the trait, while the lighter colors represent QTLs of smaller phenotypic effect.
  Significant epistasis between the GSL.Aliph.AOP, GSL.Aliph.Elong, and GSL.Aliph.V.66
  QTLs are shown by black cross-hatching within the respectively labeled cell. For
  example, but-3-enyl is controlled by QTL at GSL.AOP, GSL.Elong, and GSL.I.20 with
  epistasis between the GSL.AOP and GSL.Elong loci. QTLs for gene expression are shown
  in smaller font with a smaller ANOVA box, while QTLs for metabolites are shown in
  bold larger font with a larger box.(A) QTLs for the whole pathway broken down into
  individual metabolites and transcripts.(B) QTLs for total aliphatic glucosinolate
  content and the mean z-score for the biosynthetic pathway.
papertitle: Linking Metabolic QTLs with Network and cis-eQTLs Controlling Biosynthetic
  Pathways.
reftext: Adam M Wentzell, et al. PLoS Genet. 2007 Sep;3(9):e162.
year: '2007'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8685243
figid_alias: PMC1976331__F4
figtype: Figure
redirect_from: /figures/PMC1976331__F4
ndex: f20067e3-deae-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC1976331__pgen.0030162.g004.html
  '@type': Dataset
  description: Results of ANOVA testing of all identified aliphatic glucosinolate
    QTLs for significant impact upon the accumulation of individual glucosinolate
    metabolites, transcript level of all biosynthetic genes, total aliphatic glucosinolate
    content, and the average expression of the aliphatic glucosinolate biosynthetic
    pathway is presented graphically. The genes and metabolites are shown with respect
    to the currently theorized biosynthetic pathway. Trait abbreviations are as listed
    in . Gene names are as listed in , and TAIR locus identifiers are used for gene
    families where there is no settled naming system.Cells within boxes represent
    aliphatic glucosinolate QTLs. The legend at the bottom right contains the QTL
    name. Cells representing QTLs significantly controlling the represented trait
    are colored to show the directionality of the allele substitution effect; a positive
    effect of the Bay-0 allele is blue, and a positive effect of the Sha allele is
    red. Dark red and dark blue show that the allele substitution at the given QTL
    led to greater than 50% phenotypic change in the trait, while the lighter colors
    represent QTLs of smaller phenotypic effect. Significant epistasis between the
    GSL.Aliph.AOP, GSL.Aliph.Elong, and GSL.Aliph.V.66 QTLs are shown by black cross-hatching
    within the respectively labeled cell. For example, but-3-enyl is controlled by
    QTL at GSL.AOP, GSL.Elong, and GSL.I.20 with epistasis between the GSL.AOP and
    GSL.Elong loci. QTLs for gene expression are shown in smaller font with a smaller
    ANOVA box, while QTLs for metabolites are shown in bold larger font with a larger
    box.(A) QTLs for the whole pathway broken down into individual metabolites and
    transcripts.(B) QTLs for total aliphatic glucosinolate content and the mean z-score
    for the biosynthetic pathway.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AT2G31790
  - BCAT4
  - SGT
  - GSTU24
  - FMO
  - IMS2
  - AOP3
  - AOP2
  - MAM1
  - AOP1
  - Methionine
---
