---
figid: PMC10140937__ijms-24-07545-g004
figtitle: Mitochondrial DNA Supplementation of Oocytes Has Downstream Effects on the
  Transcriptional Profiles of Sus scrofa Adult Tissues with High mtDNA Copy Number
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC10140937
filename: ijms-24-07545-g004.jpg
figlink: /pmc/articles/PMC10140937/figure/ijms-24-07545-f004/
number: F4
caption: Gene network and differential gene expression of functional pathways identified
  by enrichment analysis. (A) Interferon signalling pathway (R-HSA-913531) genes expressed
  in tissues were used to construct a protein interaction network by STRING. Levels
  of differential gene expression between control and mtDNA-supplemented-derived liver
  are shown as fold change relative to control by colour scale (right). (B) The tricarboxylic
  acid (TCA) cycle pathway (R-HSA-71403) genes expressed in tissues were used to construct
  a protein interaction network. Levels of differential gene expression between heterologous
  and autologous mtDNA-supplemented-derived brain are shown as fold change relative
  to autologous mtDNA-supplemented-derived brain by colour scale. The same colour
  scale was used as shown in (A). Genes displayed in grey boxes have no DEG data due
  to low or no expression in the corresponding tissues.
papertitle: Mitochondrial DNA Supplementation of Oocytes Has Downstream Effects on
  the Transcriptional Profiles of Sus scrofa Adult Tissues with High mtDNA Copy Number.
reftext: Takashi Okada, et al. Int J Mol Sci. 2023 Apr;24(8):7545.
year: '2023'
doi: 10.3390/ijms24087545
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: autologous | heterologous | glyoxylate metabolism | immune response | imprinted
  gene | mitochondrial DNA | mitochondrial DNA supplementation | oxidative phosphorylation
  | Sus scrofa | transcriptome analysis
automl_pathway: 0.8573118
figid_alias: PMC10140937__F4
figtype: Figure
redirect_from: /figures/PMC10140937__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10140937__ijms-24-07545-g004.html
  '@type': Dataset
  description: Gene network and differential gene expression of functional pathways
    identified by enrichment analysis. (A) Interferon signalling pathway (R-HSA-913531)
    genes expressed in tissues were used to construct a protein interaction network
    by STRING. Levels of differential gene expression between control and mtDNA-supplemented-derived
    liver are shown as fold change relative to control by colour scale (right). (B)
    The tricarboxylic acid (TCA) cycle pathway (R-HSA-71403) genes expressed in tissues
    were used to construct a protein interaction network. Levels of differential gene
    expression between heterologous and autologous mtDNA-supplemented-derived brain
    are shown as fold change relative to autologous mtDNA-supplemented-derived brain
    by colour scale. The same colour scale was used as shown in (A). Genes displayed
    in grey boxes have no DEG data due to low or no expression in the corresponding
    tissues.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TRAF3
  - ETF1
  - MTRF1
  - SCGB1A1
  - CAP2
  - SERPINB8
  - TMPRSS4
  - HLA-DRA
  - HLA-DQB1
  - GNAO1
  - HLA-DQA1
  - HLA-DQB2
  - SUCLG2
  - DLD
  - FAM3A
  - LDHD
  - OGDH
  - ME2
  - CELSR1
  - PCDHB11
  - SDHD
  - IDH2
  - IDH3G
  - MDH2
  - DLST
  - SDHC
  - LOC642502
  - FH
  - ME3
  - PCDHGA5
  - CS
  - NNY
---
