---
figid: PMC11822028__42003_2025_7640_Fig3_HTML
figtitle: Transcriptomic and proteomic profiles of G. subvermispora and T. versicolor
  of 4-HBA catabolic pathways, TCA cycle, and glyoxylate shunt under different cultivation
  conditions
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11822028
filename: 42003_2025_7640_Fig3_HTML.jpg
figlink: /pmc/articles/PMC11822028/figure/F3/
number: F3
caption: 'Transcriptomic and proteomic profiles of enzymes involved in (a) 4-HBA conversion
  and (b) the TCA cycle, and the glyoxylate shunt in G. subvermispora and T. versicolor.
  Heatmaps associated to each biochemical step display log2fold changes of relative
  enzyme (top row) and transcripts (bottom row) abundances in four pairwise comparisons
  (from left to right): AO vs. without AO (NAO) in agitation, AO vs NAO in static
  conditions, static vs. agitation with AO, and static vs. agitation without AO. Green
  is indicative of a negative log2fold change (negative induction) while dark blue
  is indicative of a positive log2fold change (positive induction) in the corresponding
  pairwise comparison. Gray boxes indicate that the protein or gene did not return
  triplicate transcripts or peptides in upstream analyses. Asterisks (*) within the
  heatmap boxes indicate significant differences in those pairwise comparisons, determined
  via Tukey’s Honest Significant Difference test90. Discontinuous arrow lines are
  non-validated biochemical reactions but included based on systems biology observations
  in del Cerro et al. (2021)6. Protein IDs (in blue) are also shown for each biochemical
  step abbreviated as (in alphabetic order): 2-AKD 2-alpha ketoglutarate dehydrogenase,
  AH aconitate hydratase, CAR carboxylic acid reductase, CS citrate synthase, FH fumarate
  hydratase, HL hydroxylase, ID isocitrate dehydrogenase, IDD intradiol dioxygenase,
  IL isocitrate lyase, MAR maleylacetate reductase, MDH malate dehydrogenase, MS malate
  synthase, OD oxidative decarboxylase, PCS peroxisomal citrate synthase, P450 cytochrome
  P450, SD succinate dehydrogenase. Other abbreviations: AO ascorbic acid and α-tocopherol
  antioxidants, NAO no antioxidants. Omics results originate from three biological
  replicates for each cultivation condition'
papertitle: Metabolic profiling of two white-rot fungi during 4-hydroxybenzoate conversion
  reveals biotechnologically relevant biosynthetic pathways
reftext: Lummy Maria Oliveira Monteiro, et al. Commun Biol. 2025;8(NA).
year: '2025'
doi: 10.1038/s42003-025-07640-9
journal_title: Communications Biology
journal_nlm_ta: Commun Biol
publisher_name: Nature Publishing Group
keywords: Systems analysis | Multicellular systems
automl_pathway: 0.8633943
figid_alias: PMC11822028__F3
figtype: Figure
redirect_from: /figures/PMC11822028__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11822028__42003_2025_7640_Fig3_HTML.html
  '@type': Dataset
  description: 'Transcriptomic and proteomic profiles of enzymes involved in (a) 4-HBA
    conversion and (b) the TCA cycle, and the glyoxylate shunt in G. subvermispora
    and T. versicolor. Heatmaps associated to each biochemical step display log2fold
    changes of relative enzyme (top row) and transcripts (bottom row) abundances in
    four pairwise comparisons (from left to right): AO vs. without AO (NAO) in agitation,
    AO vs NAO in static conditions, static vs. agitation with AO, and static vs. agitation
    without AO. Green is indicative of a negative log2fold change (negative induction)
    while dark blue is indicative of a positive log2fold change (positive induction)
    in the corresponding pairwise comparison. Gray boxes indicate that the protein
    or gene did not return triplicate transcripts or peptides in upstream analyses.
    Asterisks (*) within the heatmap boxes indicate significant differences in those
    pairwise comparisons, determined via Tukey’s Honest Significant Difference test90.
    Discontinuous arrow lines are non-validated biochemical reactions but included
    based on systems biology observations in del Cerro et al. (2021)6. Protein IDs
    (in blue) are also shown for each biochemical step abbreviated as (in alphabetic
    order): 2-AKD 2-alpha ketoglutarate dehydrogenase, AH aconitate hydratase, CAR
    carboxylic acid reductase, CS citrate synthase, FH fumarate hydratase, HL hydroxylase,
    ID isocitrate dehydrogenase, IDD intradiol dioxygenase, IL isocitrate lyase, MAR
    maleylacetate reductase, MDH malate dehydrogenase, MS malate synthase, OD oxidative
    decarboxylase, PCS peroxisomal citrate synthase, P450 cytochrome P450, SD succinate
    dehydrogenase. Other abbreviations: AO ascorbic acid and α-tocopherol antioxidants,
    NAO no antioxidants. Omics results originate from three biological replicates
    for each cultivation condition'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CYP2B6
  - CASR
  - CXADR
  - PRKAR1A
  - SPG7
  - NR1I3
  - TRIM13
  - CXADRP1
  - GNAS
  - GNAL
  - MDH2
  - FH
  - COL9A3
  - DGCR2
  - IRF1
  - CNTN3
  - CS
  - MS
  - 4-Hydroxybenzoate
  - 4-Hydroxybenzyl
  - alcohol
  - Hydroxyquinol
  - Maleylacetate
  - Glyoxylate
  - Succinyl-CoA
  - Acetyl-CoA
  - Oxaloacetate
  - Malate
  - Fumarate
  - Succinate
  - Citrate
---
