---
figid: PMC12167820__gr2
figtitle: Transcriptome analysis of S
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC12167820
filename: gr2.jpg
figlink: /pmc/articles/PMC12167820/figure/F2/
number: F2
caption: Transcriptome analysis of S. tsukubaensis according to growth(A) Growth curve
  of S. tsukubaensis in R5(−) medium. RNA was extracted at early rapid growth (E),
  transition (T), late rapid growth (L), and stationary (S) phases. Data are represented
  as mean +/− standard deviation (SD). n = 3 biological replicates.(B) RNA fold-changes
  of differentially expressed genes. mRNA fold-change of 3,990 genes showing significant
  changes in expression at two or more growth phases (fold-change >2 and p-value <0.05)
  are represented in a heatmap. Expression pattern of genes clustered into 16 clusters
  using K-mean clustering. Leftmost numbers are the number of K-means clusters. Clusters
  with up- and downregulated tendency are represented as orange and navy lines, respectively.(C)
  Transcriptome profile of central carbon metabolism and the FK506 biosynthetic pathway.
  Schematic representation of RNA fold-changes involved in primary metabolism and
  the FK506 biosynthetic pathway. Orange colored genes are members of cluster 1 or
  10, and red colored genes are FK506 BGC genes. Metabolites labeled with red boxes
  are precursors of FK506. DHCHC, 4,5-dihydroxycyclohex-1-enecarboxylic acid. T/E,
  fold-change between transition and early rapid growth phase; L/E, fold-change between
  late rapid growth and early rapid growth phase; and S/E, fold-change between stationary
  and early rapid growth phase. See also Method S2, Figures S3–S6, and S14, S15, and
  Table S2
papertitle: Regulatory orchestration of FK506 biosynthesis in Streptomyces tsukubaensis
  NRRL 18488 revealed through systematic analysis
reftext: Namil Lee, et al. iScience. 2025 Jun 20;28(6).
year: '2025'
doi: 10.1016/j.isci.2025.112698
journal_title: iScience
journal_nlm_ta: iScience
publisher_name: Elsevier
keywords: Microbial genomics | Microbial metabolism | Systems biology | genomics |
  Transcriptomics
automl_pathway: 0.6941721
figid_alias: PMC12167820__F2
figtype: Figure
redirect_from: /figures/PMC12167820__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC12167820__gr2.html
  '@type': Dataset
  description: Transcriptome analysis of S. tsukubaensis according to growth(A) Growth
    curve of S. tsukubaensis in R5(−) medium. RNA was extracted at early rapid growth
    (E), transition (T), late rapid growth (L), and stationary (S) phases. Data are
    represented as mean +/− standard deviation (SD). n = 3 biological replicates.(B)
    RNA fold-changes of differentially expressed genes. mRNA fold-change of 3,990
    genes showing significant changes in expression at two or more growth phases (fold-change
    >2 and p-value <0.05) are represented in a heatmap. Expression pattern of genes
    clustered into 16 clusters using K-mean clustering. Leftmost numbers are the number
    of K-means clusters. Clusters with up- and downregulated tendency are represented
    as orange and navy lines, respectively.(C) Transcriptome profile of central carbon
    metabolism and the FK506 biosynthetic pathway. Schematic representation of RNA
    fold-changes involved in primary metabolism and the FK506 biosynthetic pathway.
    Orange colored genes are members of cluster 1 or 10, and red colored genes are
    FK506 BGC genes. Metabolites labeled with red boxes are precursors of FK506. DHCHC,
    4,5-dihydroxycyclohex-1-enecarboxylic acid. T/E, fold-change between transition
    and early rapid growth phase; L/E, fold-change between late rapid growth and early
    rapid growth phase; and S/E, fold-change between stationary and early rapid growth
    phase. See also Method S2, Figures S3–S6, and S14, S15, and Table S2
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GPI
  - GNPDA1
  - PFKL
  - PFKM
  - PFKP
  - FBXO3
  - TPI1
  - GAPDH
  - GAPDHP44
  - PGAM1
  - PGAM2
  - GBA1LP
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - LAMB3
  - IGKV3D-20
  - PAEP
  - PREP
  - PTPN22
  - SP6
  - PCCB
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - PAAF1
  - DBF4
  - CS
  - ARSD
  - CHST3
  - KLK15
  - MDH2
  - IDH1
  - IDH2
  - FH
  - OPRK1
  - SARDH
  - SDHB
  - SORD
  - SDS
  - DAP
  - IAPP
  - DNPEP
  - TRIM54
  - Glucose
  - Methoxymalonyl-ACP
  - 2-PGA
  - PEP
  - allD
  - Shikimate
  - Chorismate
  - Phenylalanine
  - Tyrosin
  - Tryptophan
  - Pyruvate
  - amino acids
  - Acetoacetyl-CoA
  - Malonyl-CoA
  - Crotonyl-CoA
  - Methylmalonyl-CoA
  - Acetyl-CoA
  - Aspartate
  - Methionine
  - Citrate
  - Oxaloacetate
  - Threonine
  - Malate
  - Fumarate
  - Isoleucine
  - Succinyl-CoA
  - DAP
  - Peptidoglycan
  - FK506
  - Lysine
---
