---
figid: PMC1409726__7601023f2
figtitle: An overview of the eukaryotic transcriptome through examples of its products
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC1409726
filename: 7601023f2.jpg
figlink: /pmc/articles/PMC1409726/figure/f2/
number: F2
caption: An overview of the eukaryotic transcriptome through examples of its products.
  Transcription by RNA polymerase I of ribosomal RNA precursors is followed by removal
  of 5′, 3′ extensions and intervening sequences (blue thin lines) to generate ribosomal
  RNAs (blue rectangles), which are further modified by pseudouridynilation (Ψ) and
  ribose methylation (CH3) at specific residues and assemble onto ribosomal subunits
  (blue circle and ellipse), which are then exported to the cytoplasm, where they
  mediate protein synthesis. Transcription by RNA polymerase II of mRNA precursors
  follows the processing pathway depicted in . Exonic sequences can also be removed
  as part of an intron (green rectangle, in the example) to generate alternative mRNAs
  that can direct the synthesis of distinct protein isoforms. One class of transcripts
  that do not have extensive coding capacity (ncRNAs) are similarly generated and
  may play important cellular functions (in the example, by acting as a scaffold for
  the assembly of functionally connected protein factors, represented as polygons
  of various shapes). Some introns can generate additional transcripts with important
  functions in gene regulation. One example are snoRNAs, which assemble onto snoRNP
  RNP complexes and direct pseudouridynilation and ribose methylation of rRNA and
  other transcripts. Another example are precursors of miRNAs, which after cleavage
  by Drosha-type RNases in the nucleus and by Dicer-type enzymes in the cytoplasm,
  generate 20–28 ds miRNAs that assemble onto RNP complexes that repress translation
  by binding to 3′ UTRs of mRNAs and/or cause mRNA degradation, depending on the degree
  of complementarity with their target sequence. Other dsRNAs (e.g. siRNAs) can also
  trigger mRNA decay through the same mechanism. snoRNAs and miRNA precursors can
  also be generated from exonic sequences of dedicated transcripts. Small dsRNA fragments
  generated through bidirectional transcription—shRNAs (often from repetitive DNA—rasiRNAs)
  and cleavage can also induce transcriptional silencing through histone and DNA methylation.
papertitle: 'The expanding transcriptome: the genome as the ‘Book of Sand''.'
reftext: Luis M Mendes Soares, et al. EMBO J. 2006 Mar 8;25(5):923-931.
year: '2006'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9372623
figid_alias: PMC1409726__F2
figtype: Figure
redirect_from: /figures/PMC1409726__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC1409726__7601023f2.html
  '@type': Dataset
  description: An overview of the eukaryotic transcriptome through examples of its
    products. Transcription by RNA polymerase I of ribosomal RNA precursors is followed
    by removal of 5′, 3′ extensions and intervening sequences (blue thin lines) to
    generate ribosomal RNAs (blue rectangles), which are further modified by pseudouridynilation
    (Ψ) and ribose methylation (CH3) at specific residues and assemble onto ribosomal
    subunits (blue circle and ellipse), which are then exported to the cytoplasm,
    where they mediate protein synthesis. Transcription by RNA polymerase II of mRNA
    precursors follows the processing pathway depicted in . Exonic sequences can also
    be removed as part of an intron (green rectangle, in the example) to generate
    alternative mRNAs that can direct the synthesis of distinct protein isoforms.
    One class of transcripts that do not have extensive coding capacity (ncRNAs) are
    similarly generated and may play important cellular functions (in the example,
    by acting as a scaffold for the assembly of functionally connected protein factors,
    represented as polygons of various shapes). Some introns can generate additional
    transcripts with important functions in gene regulation. One example are snoRNAs,
    which assemble onto snoRNP RNP complexes and direct pseudouridynilation and ribose
    methylation of rRNA and other transcripts. Another example are precursors of miRNAs,
    which after cleavage by Drosha-type RNases in the nucleus and by Dicer-type enzymes
    in the cytoplasm, generate 20–28 ds miRNAs that assemble onto RNP complexes that
    repress translation by binding to 3′ UTRs of mRNAs and/or cause mRNA degradation,
    depending on the degree of complementarity with their target sequence. Other dsRNAs
    (e.g. siRNAs) can also trigger mRNA decay through the same mechanism. snoRNAs
    and miRNA precursors can also be generated from exonic sequences of dedicated
    transcripts. Small dsRNA fragments generated through bidirectional transcription—shRNAs
    (often from repetitive DNA—rasiRNAs) and cleavage can also induce transcriptional
    silencing through histone and DNA methylation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - cho
  - paaA
  - rna
---
