---
figid: PMC12007450__fuaf008fig2
figtitle: This figure illustrates the complex interactions between plant roots and
  beneficial microbes in the rhizosphere, focusing on microbial evasion of plant immune
  responses and the steps of rhizosphere colonization
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC12007450
filename: fuaf008fig2.jpg
figlink: /pmc/articles/PMC12007450/figure/F2/
number: F2
caption: 'This figure illustrates the complex interactions between plant roots and
  beneficial microbes in the rhizosphere, focusing on microbial evasion of plant immune
  responses and the steps of rhizosphere colonization. The image is divided into four
  panels (A–D) that highlight different aspects of microbial colonization and immune
  modulation mechanisms. (A) Microbial recognition and evasion: this panel demonstrates
  how beneficial microbes evade plant immune recognition. Pathogen-associated molecular
  patterns (PAMPs), such as flagellin and chitin, are detected by plant receptors
  (e.g. FLS2, EFR, and LYK5). Certain microbial proteins (e.g. Ecp6) can mask or evade
  these responses, helping microbes avoid immune activation. (B) Immune signaling
  modulation: beneficial microbes can alter plant immune signaling. Nod factors, flagellin,
  and other MAMPs are detected by receptors, activating signaling cascades like MAPK
  and CDPK. This leads to the expression of defense genes, such as those for camalexin
  and glucosinolate, and the production of ROS via calcium (Ca²⁺) channels and respiratory
  burst oxidase homolog (RBOHD). (C) Colonization process: this panel outlines the
  colonization stages for nonsymbiotic bacteria, including chemotaxis, root attachment,
  biofilm formation, and endophytic penetration. Key steps involve sensing root exudates,
  selecting infection sites, and overcoming plant immunity, with microbial competition
  for scarce elements like iron and phosphorus also shown. (D) Effector-mediated immune
  suppression: this panel illustrates how bacterial and fungal effectors suppress
  plant immune responses. Bacterial effectors (e.g. NopL) and fungal small secreted
  proteins (SSPs, such as MiSSP7) interfere with host signaling pathways, such as
  SIPK, MPK4, and CDPK. This leads to changes in transcriptional regulation, influencing
  defense-related gene expression and promoting root colonization'
papertitle: Plant exudates-driven microbiome recruitment and assembly facilitates
  plant health management
reftext: Chang-Xin Yang, et al. FEMS Microbiol Rev. 2025;49(NA).
year: '2025'
doi: 10.1093/femsre/fuaf008
journal_title: FEMS Microbiology Reviews
journal_nlm_ta: FEMS Microbiol Rev
publisher_name: Oxford University Press
keywords: plant exudate | plant–microbiome interaction | microbial community recruitment
  | plant health
automl_pathway: 0.9166279
figid_alias: PMC12007450__F2
figtype: Figure
redirect_from: /figures/PMC12007450__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC12007450__fuaf008fig2.html
  '@type': Dataset
  description: 'This figure illustrates the complex interactions between plant roots
    and beneficial microbes in the rhizosphere, focusing on microbial evasion of plant
    immune responses and the steps of rhizosphere colonization. The image is divided
    into four panels (A–D) that highlight different aspects of microbial colonization
    and immune modulation mechanisms. (A) Microbial recognition and evasion: this
    panel demonstrates how beneficial microbes evade plant immune recognition. Pathogen-associated
    molecular patterns (PAMPs), such as flagellin and chitin, are detected by plant
    receptors (e.g. FLS2, EFR, and LYK5). Certain microbial proteins (e.g. Ecp6) can
    mask or evade these responses, helping microbes avoid immune activation. (B) Immune
    signaling modulation: beneficial microbes can alter plant immune signaling. Nod
    factors, flagellin, and other MAMPs are detected by receptors, activating signaling
    cascades like MAPK and CDPK. This leads to the expression of defense genes, such
    as those for camalexin and glucosinolate, and the production of ROS via calcium
    (Ca²⁺) channels and respiratory burst oxidase homolog (RBOHD). (C) Colonization
    process: this panel outlines the colonization stages for nonsymbiotic bacteria,
    including chemotaxis, root attachment, biofilm formation, and endophytic penetration.
    Key steps involve sensing root exudates, selecting infection sites, and overcoming
    plant immunity, with microbial competition for scarce elements like iron and phosphorus
    also shown. (D) Effector-mediated immune suppression: this panel illustrates how
    bacterial and fungal effectors suppress plant immune responses. Bacterial effectors
    (e.g. NopL) and fungal small secreted proteins (SSPs, such as MiSSP7) interfere
    with host signaling pathways, such as SIPK, MPK4, and CDPK. This leads to changes
    in transcriptional regulation, influencing defense-related gene expression and
    promoting root colonization'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATN1
  - BAK1
  - STRADA
  - IRF6
  - NECTIN1
  - NR1I2
  - ATP6AP2
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - SP7
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - Amino
  - acids
  - Chitin
  - ROS
  - Ca2+
---
