---
figid: PMC10898159__12964_2023_1411_Fig1_HTML
figtitle: Gene and epigenetic regulations of KIT expression in GIST
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC10898159
filename: PMC10898159__12964_2023_1411_Fig1_HTML.jpg
figlink: /pmc/articles/PMC10898159/figure/F1
number: F1
caption: Gene and epigenetic regulations of KIT expression in GIST. KIT extracellular
  segment contains five immunoglobulin-like structural domains (D1-D5), D1-D2-D3 are
  stem cell factor (SCF) binding regions, and D4-D5 are important units for KIT dimer
  formation. In the absence of ligand stimulation, two adjacent KIT protein D4 structures
  remain independent due to charge repulsion. KIT mutations usually arise in the extracellular
  domains (exon 9), the juxtamembrane domain (exon 11) and in the kinase I and II
  domains (exons 13 and 17). The SCF binding to KIT and then changes the conformation
  of KIT and promote its dimerization, thereby activating tyrosine kinase activity
  in the intracellular segment by recruiting and phosphorylating substrates, thus
  forming signal transduction. But once KIT mutated, it disrupts its self-inhibition
  mechanism, resulting in continuous activation of signaling pathways such as Ras-Erk
  within the cell. FOXF1, ETV1 and HIC1 together form the core TF network in GIST
  binding enhancer and/or promoter and then promoting KIT expression. BRD4 (function
  as “readers”), HAND1 and BRAX1 positive regulate the core TF network. Transcription
  factor MITF and BIRC5 promote KIT transcription. MiRNA-20a, miRNA-17–92, miRNA200b-3p,
  miRNA-494, miRNA-148-3p are tumor suppressors downregulate KIT transcription and
  the majority of them directly bind to the 3’-UTR domain of relevant mRNA; the first
  three mentioned miRNAs can inhibit ETV1 mRNA levels, miRNA-494 can inhibit BIRC5
  expression suggesting a negative feedback mechanism. SH3BP2 promotes the expression
  of mutant KIT by up-regulation of the expression of MITF and ETV1. FTO, functions
  as a “eraser”, promoting m6A demethylation increases the expression of KIT. Normal
  KIT chromosomal has CTCF insulator creating a topological boundary. Once CTCF insulator
  displaced by DNA methylation, allowing the super-enhancer to contact and induce
  KIT
papertitle: 'KIT mutations and expression: current knowledge and new insights for
  overcoming IM resistance in GIST'
reftext: Shishan Zhou, et al. Cell Commun Signal. 2024;22(NA).
year: '2024'
doi: 10.1186/s12964-023-01411-x
journal_title: 'Cell Communication and Signaling : CCS'
journal_nlm_ta: Cell Commun Signal
publisher_name: BioMed Central
keywords: KIT mutations and expression | Gastrointestinal stromal tumor | Therapeutic
  targets | Imatinib resistance
automl_pathway: 0.9512618
figid_alias: PMC10898159__F1
figtype: Figure
redirect_from: /figures/PMC10898159__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10898159__12964_2023_1411_Fig1_HTML.html
  '@type': Dataset
  description: Gene and epigenetic regulations of KIT expression in GIST. KIT extracellular
    segment contains five immunoglobulin-like structural domains (D1-D5), D1-D2-D3
    are stem cell factor (SCF) binding regions, and D4-D5 are important units for
    KIT dimer formation. In the absence of ligand stimulation, two adjacent KIT protein
    D4 structures remain independent due to charge repulsion. KIT mutations usually
    arise in the extracellular domains (exon 9), the juxtamembrane domain (exon 11)
    and in the kinase I and II domains (exons 13 and 17). The SCF binding to KIT and
    then changes the conformation of KIT and promote its dimerization, thereby activating
    tyrosine kinase activity in the intracellular segment by recruiting and phosphorylating
    substrates, thus forming signal transduction. But once KIT mutated, it disrupts
    its self-inhibition mechanism, resulting in continuous activation of signaling
    pathways such as Ras-Erk within the cell. FOXF1, ETV1 and HIC1 together form the
    core TF network in GIST binding enhancer and/or promoter and then promoting KIT
    expression. BRD4 (function as “readers”), HAND1 and BRAX1 positive regulate the
    core TF network. Transcription factor MITF and BIRC5 promote KIT transcription.
    MiRNA-20a, miRNA-17–92, miRNA200b-3p, miRNA-494, miRNA-148-3p are tumor suppressors
    downregulate KIT transcription and the majority of them directly bind to the 3’-UTR
    domain of relevant mRNA; the first three mentioned miRNAs can inhibit ETV1 mRNA
    levels, miRNA-494 can inhibit BIRC5 expression suggesting a negative feedback
    mechanism. SH3BP2 promotes the expression of mutant KIT by up-regulation of the
    expression of MITF and ETV1. FTO, functions as a “eraser”, promoting m6A demethylation
    increases the expression of KIT. Normal KIT chromosomal has CTCF insulator creating
    a topological boundary. Once CTCF insulator displaced by DNA methylation, allowing
    the super-enhancer to contact and induce KIT
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DLL4
  - KIT
  - MAP2K7
  - MAP2K1
  - MAP2K2
  - ATP8A2
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - MIR494
  - FTO
  - HAND1
  - SH3BP2
  - BRD4
  - FOXF1
  - HIC1
  - ETV1
  - BIRC5
  - MITF
  - CTCF
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - MEK
  - Nucleus
  - Histone
---
