---
figid: PMC10581046__spectrum.01809-23.f007
figtitle: Metabolic pathway analysis of infection-associated module genes
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC10581046
filename: PMC10581046__spectrum.01809-23.f007.jpg
figlink: /pmc/articles/PMC10581046/figure/F7
number: F7
caption: Metabolic pathway analysis of infection-associated module genes. Module genes
  were mapped to yeast metabolic pathways and pathways in the Metacyc database and
  overrepresented pathways were selected for illustration. The partially activated
  pathways of fatty acid degradation (A), GTP synthesis (B), and glycolysis (C) in
  HB-Up subgroup were illustrated. The active pathways in cuticle penetration (CP-Up)
  were shown as indicated. (D) Glutamic acid degradation pathway. (E) Pyruvate metabolic
  pathway. (F) Tyrosine degradation pathway. (G) Coenzyme biosynthesis pathway. The
  known homologs of module genes in yeast were listed as indicated or as sole B. bassiana
  genes after being mapped to Metacyc. Abbreviations of chemicals listed were annotated
  in Metacyc databases. Multiple reactions in between substrates were linked with
  a dotted arrow, while reversible reactions were shown as double arrows.
papertitle: Fungal co-expression network analyses identify pathogen gene modules associated
  with host insect invasion
reftext: Shuaishuai Huang, et al. Microbiol Spectr. 2023;11(5):e01809-23.
year: '2023'
doi: 10.1128/spectrum.01809-23
journal_title: Microbiology Spectrum
journal_nlm_ta: Microbiol Spectr
publisher_name: American Society for Microbiology
keywords: transcriptome | co-expression | infection-associated modules | entomopathogenic
  fungi | Beauveria bassiana | host-pathogen interaction
automl_pathway: 0.9253998
figid_alias: PMC10581046__F7
figtype: Figure
redirect_from: /figures/PMC10581046__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC10581046__spectrum.01809-23.f007.html
  '@type': Dataset
  description: Metabolic pathway analysis of infection-associated module genes. Module
    genes were mapped to yeast metabolic pathways and pathways in the Metacyc database
    and overrepresented pathways were selected for illustration. The partially activated
    pathways of fatty acid degradation (A), GTP synthesis (B), and glycolysis (C)
    in HB-Up subgroup were illustrated. The active pathways in cuticle penetration
    (CP-Up) were shown as indicated. (D) Glutamic acid degradation pathway. (E) Pyruvate
    metabolic pathway. (F) Tyrosine degradation pathway. (G) Coenzyme biosynthesis
    pathway. The known homologs of module genes in yeast were listed as indicated
    or as sole B. bassiana genes after being mapped to Metacyc. Abbreviations of chemicals
    listed were annotated in Metacyc databases. Multiple reactions in between substrates
    were linked with a dotted arrow, while reversible reactions were shown as double
    arrows.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FANCA
  - MUSK
  - RAPSN
  - FLAD1
  - AAT2
  - HADHA
  - RBFOX2
  - IMPA1
  - BRAP
  - EMP2
  - PUM3
  - CD40LG
  - ACAD8
  - ACAD9
  - ACADL
  - ACADM
  - ACADS
  - ACADVL
  - ACAD10
  - ACAD11
  - ACADSB
  - GCDH
  - IVD
  - HCFC1
  - FXYD5
  - GLUD2
  - GPD2
  - NT5C2
  - MTG1
  - CD5
  - DLEU1
  - PDC
  - PNKD
  - FFA
  - Xanthine
  - Glycerol
  - Acyl-CoA
  - 2-Oxoglutarate
  - Nitrile
  - GMP
  - Succinate
  - GTP
  - DHAP
  - Glucose
  - Alanine
  - Acetaldehyde
  - Tyrosine
  - Homogentisate
  - CoA
---
