---
figid: PMC11520086__12864_2024_10923_Figc_HTML
figtitle: Metabolic pathway and enzyme gene expression difference of C1 compounds
  in the transcriptome group, Formaldehyde vs
organisms:
- NA
organisms_ner:
- Danio rerio
- Homo sapiens
pmcid: PMC11520086
filename: 12864_2024_10923_Figc_HTML.jpg
figlink: /pmc/articles/PMC11520086/figure/F3/
number: F3
caption: 'Metabolic pathway and enzyme gene expression difference of C1 compounds
  in the transcriptome group, Formaldehyde vs. Methanol. When formaldehyde is degraded,
  Major genes associated with methylotrophy, serine cycle, TCA cycle, and ethylmalonic
  acid pathway are downregulated. To compensate for this, the strain relies on partial
  upregulation of genes related to glycine, serine, and threonine metabolism, as well
  as ascorbic acid and aldoacid metabolism. These metabolic bypass pathways are interconnected
  through intermediate metabolites such as pyruvate and oxaloacetate. We propose the
  existence of uncharacterized pathway that bypass formaldehyde and formic acid intermediates,
  or minimize formic acid production, to assimilate methanol from formaldehyde dismutase
  into the enhanced bypass, constituting a novel auxiliary metabolic pathway. Red
  arrows: upregulation; Blue arrows: downregulation; Purple arrows: uncharacterized
  pathway, the newly identified methylotrophic genes are potential candidates for
  involvement; Green arrows: isozymes or enzyme clusters mediate a reaction that contains
  upregulated or downregulated genes simultaneously; Black arrows: no significant
  change; Black dashed arrows: unknown genes'
papertitle: Transcriptomic data reveals an auxiliary detoxification mechanism that
  alleviates formaldehyde stress in Methylobacterium sp. XJLW
reftext: Yunhai Shao, et al. BMC Genomics. 2024;25(NA).
year: '2024'
doi: 10.1186/s12864-024-10923-w
journal_title: BMC Genomics
journal_nlm_ta: BMC Genomics
publisher_name: BMC
keywords: Methylobacterium | Methyltransferase | Transcriptome | RS27765 | glyA |
  Novel methylotrophy gene | Metabolic bypass
automl_pathway: 0.9400147
figid_alias: PMC11520086__F3
figtype: Figure
redirect_from: /figures/PMC11520086__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11520086__12864_2024_10923_Figc_HTML.html
  '@type': Dataset
  description: 'Metabolic pathway and enzyme gene expression difference of C1 compounds
    in the transcriptome group, Formaldehyde vs. Methanol. When formaldehyde is degraded,
    Major genes associated with methylotrophy, serine cycle, TCA cycle, and ethylmalonic
    acid pathway are downregulated. To compensate for this, the strain relies on partial
    upregulation of genes related to glycine, serine, and threonine metabolism, as
    well as ascorbic acid and aldoacid metabolism. These metabolic bypass pathways
    are interconnected through intermediate metabolites such as pyruvate and oxaloacetate.
    We propose the existence of uncharacterized pathway that bypass formaldehyde and
    formic acid intermediates, or minimize formic acid production, to assimilate methanol
    from formaldehyde dismutase into the enhanced bypass, constituting a novel auxiliary
    metabolic pathway. Red arrows: upregulation; Blue arrows: downregulation; Purple
    arrows: uncharacterized pathway, the newly identified methylotrophic genes are
    potential candidates for involvement; Green arrows: isozymes or enzyme clusters
    mediate a reaction that contains upregulated or downregulated genes simultaneously;
    Black arrows: no significant change; Black dashed arrows: unknown genes'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - gar1
  - napbb
  - hyi
  - gck
  - ghra
  - sera
  - cita
  - phb
  - DECR1
  - GAR1
  - CNGB1
  - SLC6A1
  - HYI
  - WDTC1
  - ATP8A2
  - GCK
  - MAP4K2
  - FAM20C
  - PHGDH
  - SHMT2
  - PAEP
  - PREP
  - PTPN22
  - CIT
  - DHDDS
  - PHB1
  - NADPH
  - asp
  - D-Amino acid
  - 2,5-Dioxopentanoate
  - mgo
  - Arginine
  - Malate
  - D-Glucarate
  - Pyruvate
  - CoA
  - Malyl-CoA
  - 2-Hydroxy-3-oxopropanoate
  - Formaldehyde
  - Formic acid
  - formaldehyde
  - formic acid
  - L-Serine
  - Methanol
  - 3-phosphooxypyruvate
  - NAD+
  - NADH
  - 2-Oxoglutarate
  - L-Glutamate
  - ADP
  - ATP
  - Gly
  - PEP
  - Serine
  - Cit
  - TCA
  - Glyoxylate
  - PHB
  - Methylmalonyl-CoA
  - Crotonyl-CoA
  - propionyl-CoA
---
