---
figid: PMC12133565__41375_2025_2584_Fig6_HTML
figtitle: Loss of Hdac1 hyperactivates oncogenic transcription
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC12133565
filename: 41375_2025_2584_Fig6_HTML.jpg
figlink: /pmc/articles/PMC12133565/figure/F6/
number: F6
caption: A ATAC-seq tracks downloaded from the UCSC Genome Browser [65] depicting
  peaks, which represent chromatin accessibility in the Jdp2 gene. Biological replicates
  of NPM::ALK end-stage thymic tumors (n = 4, blue) and of NPM::ALK Hdac1KO end-stage
  thymic tumors (n = 4, red) are shown. The Gencode track (Gencode VM23 release) is
  displayed below the ATAC-seq tracks, indicating different transcripts of the Jdp2
  gene. Colored boxes on the bottom show ENCODE candidate Cis-Regulatory Elements
  (cCREs) combined from all available cell types (red promoter, orange proximal enhancer,
  yellow distal enhancer, blue CTCF binding sites). B VST normalized counts based
  on RNA-seq analysis for Jdp2 comparing NPM::ALK (blue) and NPM::ALK Hdac1KO (red)
  samples. C VST normalized counts based on RNA-seq analysis for Tnk2 comparing NPM::ALK
  (blue) and NPM::ALK Hdac1KO (red) samples. D ATAC-seq tracks for the Pdgfb gene
  as in A. E VST normalized counts based on RNA-seq analysis for Pdgfb comparing NPM::ALK
  (blue) and NPM::ALK Hdac1KO (red) samples. F Schematic representation of the PDGFRB/STAT5
  signaling pathway. Green boxes indicate genes upregulated in NPM::ALK Hdac1KO tumors
  as compared to NPM::ALK tumors based on RNA-seq data. G Immunoblot showing protein
  levels of PDGFRb, STAT5a, STAT5b and pSTAT5 in end-stage thymic tumors excised from
  NPM::ALK (n = 4), NPM::ALK Hdac1KO (n = 4) and NPM::ALK Hdac1KI mice (n = 4). Beta-actin
  was used as a loading control. The numbers on the left indicate the molecular weight
  of analyzed proteins in kiloDalton (kDa). H Schematic representation of the Ca2+
  signaling pathway. Green boxes indicate genes upregulated, red boxes indicate genes
  downregulated in NPM::ALK Hdac1KO tumors as compared to NPM::ALK tumors based on
  RNA-seq data. ER, endoplasmic reticulum. I Bar plot depicting the results of the
  Homer Motif analysis [66], indicating enrichment of the NFAT:AP1 motif in promoter
  peaks of NPM::ALK Hdac1KO samples (red) compared to NPM::ALK samples (blue) or control
  sequences (background) identified by ATAC-seq analysis. J Immunoblot showing protein
  levels of NFAT1 in end-stage thymic tumors excised from NPM::ALK (n = 4), NPM::ALK
  Hdac1KO (n = 4) and NPM::ALK Hdac1KI mice (n = 4). Beta-actin was used as a loading
  control. The numbers on the left indicate the molecular weight of analyzed proteins
  in kiloDalton (kDa)
papertitle: 'HDAC1 acts as a tumor suppressor in ALK-positive anaplastic large cell
  lymphoma: implications for HDAC inhibitor therapy'
reftext: Maša Zrimšek, et al. Leukemia. 2025 Apr 2;39(6).
year: '2025'
doi: 10.1038/s41375-025-02584-9
journal_title: Leukemia
journal_nlm_ta: Leukemia
publisher_name: .na.character
keywords: T-cell lymphoma | Cancer therapy
automl_pathway: 0.555885
figid_alias: PMC12133565__F6
figtype: Figure
redirect_from: /figures/PMC12133565__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC12133565__41375_2025_2584_Fig6_HTML.html
  '@type': Dataset
  description: A ATAC-seq tracks downloaded from the UCSC Genome Browser [65] depicting
    peaks, which represent chromatin accessibility in the Jdp2 gene. Biological replicates
    of NPM::ALK end-stage thymic tumors (n = 4, blue) and of NPM::ALK Hdac1KO end-stage
    thymic tumors (n = 4, red) are shown. The Gencode track (Gencode VM23 release)
    is displayed below the ATAC-seq tracks, indicating different transcripts of the
    Jdp2 gene. Colored boxes on the bottom show ENCODE candidate Cis-Regulatory Elements
    (cCREs) combined from all available cell types (red promoter, orange proximal
    enhancer, yellow distal enhancer, blue CTCF binding sites). B VST normalized counts
    based on RNA-seq analysis for Jdp2 comparing NPM::ALK (blue) and NPM::ALK Hdac1KO
    (red) samples. C VST normalized counts based on RNA-seq analysis for Tnk2 comparing
    NPM::ALK (blue) and NPM::ALK Hdac1KO (red) samples. D ATAC-seq tracks for the
    Pdgfb gene as in A. E VST normalized counts based on RNA-seq analysis for Pdgfb
    comparing NPM::ALK (blue) and NPM::ALK Hdac1KO (red) samples. F Schematic representation
    of the PDGFRB/STAT5 signaling pathway. Green boxes indicate genes upregulated
    in NPM::ALK Hdac1KO tumors as compared to NPM::ALK tumors based on RNA-seq data.
    G Immunoblot showing protein levels of PDGFRb, STAT5a, STAT5b and pSTAT5 in end-stage
    thymic tumors excised from NPM::ALK (n = 4), NPM::ALK Hdac1KO (n = 4) and NPM::ALK
    Hdac1KI mice (n = 4). Beta-actin was used as a loading control. The numbers on
    the left indicate the molecular weight of analyzed proteins in kiloDalton (kDa).
    H Schematic representation of the Ca2+ signaling pathway. Green boxes indicate
    genes upregulated, red boxes indicate genes downregulated in NPM::ALK Hdac1KO
    tumors as compared to NPM::ALK tumors based on RNA-seq data. ER, endoplasmic reticulum.
    I Bar plot depicting the results of the Homer Motif analysis [66], indicating
    enrichment of the NFAT:AP1 motif in promoter peaks of NPM::ALK Hdac1KO samples
    (red) compared to NPM::ALK samples (blue) or control sequences (background) identified
    by ATAC-seq analysis. J Immunoblot showing protein levels of NFAT1 in end-stage
    thymic tumors excised from NPM::ALK (n = 4), NPM::ALK Hdac1KO (n = 4) and NPM::ALK
    Hdac1KI mice (n = 4). Beta-actin was used as a loading control. The numbers on
    the left indicate the molecular weight of analyzed proteins in kiloDalton (kDa)
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - JDP2
  - PDGFB
  - TNK2
  - NPM1
  - ALK
  - PDGFRB
  - STAT5A
  - STAT5B
  - IL10
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - PLCL1
  - VEGFA
  - CACNA1E
  - CACNA1C
  - CACNA1D
  - CAMK1G
  - ITPR1
  - ITPR3
  - CALM2
  - CALM3
  - PICALM
  - SNAP91
  - CALM1
  - NOS2
  - NANOS2
  - NFATC2
  - ADCY1
  - PDE1C
  - CaN
---
