---
figid: PMC11612781__gr3
figtitle: 'From sampling to simulating: Single-cell multiomics in systems pathophysiological
  modeling'
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC11612781
filename: gr3.jpg
figlink: /pmc/articles/PMC11612781/figure/F3/
number: F3
caption: 'Computational models informed by experimental dataThe components, interactions,
  correlations, and patterns extracted from “Big” data (multi-omics data including
  transcriptomics, proteomics, metabolomics, and spatial omics) and the components,
  interactions and mechanisms extracted from “Not So Big” data (i.e., western blots,
  immuno-staining, and qPCR) can be utilized to generate and inform molecular signaling
  networks, putative cellular networks, and gene regulatory networks. For instance,
  while the MAP kinase pathway was discovered using “Not So Big” data sources (solid
  line) many “Big” data sources (dashed line) have confirmed and further explained
  and complemented these initial findings. Similarly, while gene regulatory networks
  have been mainly developed using “Big” data (solid line), “Not So Big” data (dashed
  line) can also be informative when generating such networks. For example, Park et al.,
  (2016) modeled neurons during the circadian cycle.13 First, five neuronal groups
  were identified according to their unique transcriptional landscapes with marker
  genes shown for each of the groups. A gene regulatory network was then developed
  based on the major molecular interactions between key neuropeptides (VIP, AVP, PROK2,
  and PACAP) and the neuronal groups.13 “Big” and “Not So Big” data (solid lines)
  have be analyzed in combination to identify putative cellular networks. Cell types
  can be identified within the “Big” data by using information from “Not So Big” data.
  Then, cell states within each cell type community can be determined by molecular
  markers. The cell types and states can then be used to infer cell state transitions,
  trajectories, and interactions. A greater influence of “Big” and “Not So Big” data
  on developing the various networks is shown with solid lines with lesser influence
  shown by dashed lines. GF: growth factor, GFR: growth factor receptor, VIP: Vasoactive
  Intestinal Peptide, AVP: Arginine Vasopressin, PROK2: Prokineticin 2, PACAP: Pituitary
  Adenylate Cyclase-Activating Polypeptide. Fig. adapted from Park et al., (2016).13
  Created using biorender.com'
papertitle: 'From sampling to simulating: Single-cell multiomics in systems pathophysiological
  modeling'
reftext: Alexandra Manchel, et al. iScience. 2024 Dec 20;27(12).
year: '2024'
doi: 10.1016/j.isci.2024.111322
journal_title: iScience
journal_nlm_ta: iScience
publisher_name: Elsevier
keywords: Systems biology | Data processing in systems biology | In silico biology
  | Biological constraints | Omics
automl_pathway: 0.7169847
figid_alias: PMC11612781__F3
figtype: Figure
redirect_from: /figures/PMC11612781__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC11612781__gr3.html
  '@type': Dataset
  description: 'Computational models informed by experimental dataThe components,
    interactions, correlations, and patterns extracted from “Big” data (multi-omics
    data including transcriptomics, proteomics, metabolomics, and spatial omics) and
    the components, interactions and mechanisms extracted from “Not So Big” data (i.e.,
    western blots, immuno-staining, and qPCR) can be utilized to generate and inform
    molecular signaling networks, putative cellular networks, and gene regulatory
    networks. For instance, while the MAP kinase pathway was discovered using “Not
    So Big” data sources (solid line) many “Big” data sources (dashed line) have confirmed
    and further explained and complemented these initial findings. Similarly, while
    gene regulatory networks have been mainly developed using “Big” data (solid line),
    “Not So Big” data (dashed line) can also be informative when generating such networks.
    For example, Park et al., (2016) modeled neurons during the circadian cycle.13
    First, five neuronal groups were identified according to their unique transcriptional
    landscapes with marker genes shown for each of the groups. A gene regulatory network
    was then developed based on the major molecular interactions between key neuropeptides
    (VIP, AVP, PROK2, and PACAP) and the neuronal groups.13 “Big” and “Not So Big”
    data (solid lines) have be analyzed in combination to identify putative cellular
    networks. Cell types can be identified within the “Big” data by using information
    from “Not So Big” data. Then, cell states within each cell type community can
    be determined by molecular markers. The cell types and states can then be used
    to infer cell state transitions, trajectories, and interactions. A greater influence
    of “Big” and “Not So Big” data on developing the various networks is shown with
    solid lines with lesser influence shown by dashed lines. GF: growth factor, GFR:
    growth factor receptor, VIP: Vasoactive Intestinal Peptide, AVP: Arginine Vasopressin,
    PROK2: Prokineticin 2, PACAP: Pituitary Adenylate Cyclase-Activating Polypeptide.
    Fig. adapted from Park et al., (2016).13 Created using biorender.com'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TBATA
  - PER1
  - VIP
  - CPAMD8
  - NR4A1
  - CRY1
  - GABRA1
  - AVP
  - IFNAR1
  - NLRP3
  - PROK2
  - ADCYAP1
  - MZB1
  - PROKR2
  - EGR2
  - RARA
  - RAB40B
  - RARB
  - RARG
  - EGR1
  - SGSM3
  - RAPGEF5
  - GRB2
  - KRAS
  - HRAS
  - NRAS
  - MAP2K1
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - EPHB2
  - MAPK1
  - MAPK3
  - AVP
  - Ras
---
