---
figid: PMC2878746__org0504_0177_fig001
figtitle: Ret gene structure and signaling
organisms:
- Homo sapiens
- Mus musculus
pmcid: PMC2878746
filename: org0504_0177_fig001.jpg
figlink: /pmc/articles/PMC2878746/figure/F1/
number: F1
caption: Ret gene structure and signaling. (A) Gene structure of RET. RET is located
  at chr 10q11.2 and consists of 20 exons (rectangular boxes). Depicted are two major
  RET isoforms, RET51 and RET9, generated by alternative splicing. (B) Diagram showing
  RET activation by glial cell line-derived neurotrophic factor (GDNF) family ligands
  (GFLs). A dimer of one of the four GFLs (GDNF; Neurturin, NRTN; Artemin, ARTN; Persephin,
  PSPN) associates with homodimers of one of the four GPi-linked corereceptor's (GFRα1–4)
  to form multimeric complex with a RET homodimer. This leads to activation of RET
  kinase domain in the cytoplasmic side by phosphorylation, and downstream signaling
  cascades that regulate the indicated biological responses. (c) Gross anatomy of
  RET. Different RET domains are shown, extracellular, transmembrane (TM) and cytoplasmic
  domain that harbors the kinase activity regions. Key RET tyrosine (Y) residues,
  the preferred intracellular adaptors that they bind to, and the downstream signaling
  cascades that are activated are shown. Y905 is a residue important for RET kinase
  activity. Note the extra Tyr Y1096 in the RET51 isoform directly binds to GRB2 to
  preferentially activate PI3K/AKT pathway. Thus, both RET-Y1062 and RET-Y1096 can
  activate PI3K through GRB2-GAB1/2.
papertitle: The many faces of RET dysfunction in kidney.
reftext: Sanjay Jain. Organogenesis. 2009 Oct-Dec;5(4):177-190.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8019539
figid_alias: PMC2878746__F1
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
redirect_from: /figures/PMC2878746__F1
ndex: a4ed0d48-df09-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2878746__org0504_0177_fig001.html
  '@type': Dataset
  description: Ret gene structure and signaling. (A) Gene structure of RET. RET is
    located at chr 10q11.2 and consists of 20 exons (rectangular boxes). Depicted
    are two major RET isoforms, RET51 and RET9, generated by alternative splicing.
    (B) Diagram showing RET activation by glial cell line-derived neurotrophic factor
    (GDNF) family ligands (GFLs). A dimer of one of the four GFLs (GDNF; Neurturin,
    NRTN; Artemin, ARTN; Persephin, PSPN) associates with homodimers of one of the
    four GPi-linked corereceptor's (GFRα1–4) to form multimeric complex with a RET
    homodimer. This leads to activation of RET kinase domain in the cytoplasmic side
    by phosphorylation, and downstream signaling cascades that regulate the indicated
    biological responses. (c) Gross anatomy of RET. Different RET domains are shown,
    extracellular, transmembrane (TM) and cytoplasmic domain that harbors the kinase
    activity regions. Key RET tyrosine (Y) residues, the preferred intracellular adaptors
    that they bind to, and the downstream signaling cascades that are activated are
    shown. Y905 is a residue important for RET kinase activity. Note the extra Tyr
    Y1096 in the RET51 isoform directly binds to GRB2 to preferentially activate PI3K/AKT
    pathway. Thus, both RET-Y1062 and RET-Y1096 can activate PI3K through GRB2-GAB1/2.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ret
  - Pspn
  - Artn
  - Nrtn
  - Ephb2
  - Mapk1
  - Frs2
  - Src
  - tm
  - PSPN
  - ARTN
  - NRTN
  - RET
  - EPHB2
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - FRS2
  - SRC
  - FGR
  - FYN
  - YES1
  - PLCG1
  - PLCG2
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - Cancer
---
