---
figid: PMC2874322__nihms159937f5
figtitle: 'An Update on the Genetics of Atopic Dermatitis: Scratching the Surface
  in 2009'
organisms:
- Human immunodeficiency virus 1
- Variola virus
- Molluscum contagiosum virus
- Variola minor virus
- Drosophila melanogaster
- Nicotiana tabacum
- Erythroxylum coca
- Pieris rapae
- Homo sapiens
- Mus musculus
- Canis lupus familiaris
- Staphylococcus aureus
organisms_ner:
- Mus musculus
- Homo sapiens
- Canis lupus familiaris
- Danio rerio
pmcid: PMC2874322
filename: nihms159937f5.jpg
figlink: /pmc/articles/PMC2874322/figure/F5/
number: F5
caption: In this summary of approaches used to date to identify candidate genes for
  AD, the first consideration is the complex pathology of AD (panel A), which includes
  defects and damage at the epithelial barrier, and penetration of allergens, microbes,
  pollutants and irritants into the epidermis and dermis, ultimately interacting with
  antigen presentation cells (i.e., Langerhans cells and dermal dendritic cells).
  The ‘brick wall-like’ structure of the stratum corneum is compromised, possibly
  by defects in genes residing in the epidermal differentiation complex (EDC), including
  FLG, LOR, LCEs, S100s, SPRRs, SCTE, and SCCE. Additional candidates include tight
  junctions (TJ), proteins that constitute the “gate” to the passage of water, ions
  and solutes through the paracellular pathway in the stratum granulosum (i.e., CLDN1).
  A defective innate immune response (involving, for example, the TLRs, CD14, NOD1,
  NOD2, DEFB1, and IRF2) may contribute to a heightened IgE-mediated, systemic Th2
  response. Combining a candidate gene approach with robust, genomewide gene expression
  profiling has the potential of both elucidating novel candidates and validating
  suspected candidates, as illustrated in panel B, in which genes in the EDC on chromosome
  1q21 are significantly differentially expressed in a human skin biopsies taken from
  AD patients compared to healthy nonatopic controls (courtesy L.A. Beck). Candidate
  genes are selected, and substitutions (i.e., single nucleotide polymorphisms, or
  SNPs) and simple structural variants (i.e., insertions/deletions, repeats) are genotyped
  in large populations selected for AD and tests for association are performed.
papertitle: 'An Update on the Genetics of Atopic Dermatitis: Scratching the Surface
  in 2009.'
reftext: Kathleen C. Barnes. J Allergy Clin Immunol. ;125(1):16-31.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5986761
figid_alias: PMC2874322__F5
figtype: Figure
redirect_from: /figures/PMC2874322__F5
ndex: 21e4c525-de97-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2874322__nihms159937f5.html
  '@type': Dataset
  description: In this summary of approaches used to date to identify candidate genes
    for AD, the first consideration is the complex pathology of AD (panel A), which
    includes defects and damage at the epithelial barrier, and penetration of allergens,
    microbes, pollutants and irritants into the epidermis and dermis, ultimately interacting
    with antigen presentation cells (i.e., Langerhans cells and dermal dendritic cells).
    The ‘brick wall-like’ structure of the stratum corneum is compromised, possibly
    by defects in genes residing in the epidermal differentiation complex (EDC), including
    FLG, LOR, LCEs, S100s, SPRRs, SCTE, and SCCE. Additional candidates include tight
    junctions (TJ), proteins that constitute the “gate” to the passage of water, ions
    and solutes through the paracellular pathway in the stratum granulosum (i.e.,
    CLDN1). A defective innate immune response (involving, for example, the TLRs,
    CD14, NOD1, NOD2, DEFB1, and IRF2) may contribute to a heightened IgE-mediated,
    systemic Th2 response. Combining a candidate gene approach with robust, genomewide
    gene expression profiling has the potential of both elucidating novel candidates
    and validating suspected candidates, as illustrated in panel B, in which genes
    in the EDC on chromosome 1q21 are significantly differentially expressed in a
    human skin biopsies taken from AD patients compared to healthy nonatopic controls
    (courtesy L.A. Beck). Candidate genes are selected, and substitutions (i.e., single
    nucleotide polymorphisms, or SNPs) and simple structural variants (i.e., insertions/deletions,
    repeats) are genotyped in large populations selected for AD and tests for association
    are performed.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - S100a1
  - Lor
  - Aass
  - Flg
  - Fgfr1
  - Klk5
  - Klk1b5
  - Klk7
  - Spink5
  - Cldn1
  - Tslp
  - Spg21
  - Spaca9
  - Hand2
  - Il13
  - Il4
  - Ikzf4
  - Prss33
  - Ccl11
  - Ccl5
  - Nod2
  - Nod1
  - Cd14
  - Defb1
  - Irf2
  - S100a11
  - Sprr2g
  - Sprr2e
  - Sprr1a
  - Sprr1b
  - Sprr2b
  - Sprr2a1
  - Sprr2a3
  - Sprr2a2
  - S100a9
  - S100a8
  - Sprr2f
  - S100a2
  - S100a16
  - Ivl
  - Sprr3
  - S100a14
  - S100A1
  - S100B
  - S100G
  - S100P
  - S100Z
  - HRNR
  - S100A10
  - S100A11
  - S100A12
  - S100A13
  - S100A14
  - S100A16
  - S100A2
  - S100A3
  - S100A4
  - S100A5
  - S100A6
  - S100A7
  - S100A7A
  - S100A8
  - S100A9
  - TCHHL1
  - LORICRIN
  - LOXL2
  - FLG
  - FGFR1
  - KLK5
  - KLK7
  - KLK6
  - CLDN1
  - TSLP
  - SPACA9
  - SPG21
  - IL13
  - IL4
  - EOS
  - IKZF4
  - PRSS33
  - CXCL10
  - CXCL11
  - CXCL12
  - CXCL13
  - CXCL14
  - CXCL9
  - PF4
  - CXCL1
  - CXCL2
  - CXCL3
  - CXCL5
  - CXCL6
  - CXCL8
  - PPBP
  - CCL11
  - CCL24
  - CCL26
  - CCL5
  - NOD2
  - NOD1
  - CD14
  - NDUFA2
  - DEFB1
  - IRF2
  - SPRR2G
  - SPRR2E
  - SPRR1A
  - SPRR1B
  - SPRR2B
  - SPRR2A
  - SPRR2F
  - IVL
  - SPRR3
  - S100A11P1
  - s100s
  - cldn1
  - th2
  - il13
  - il4
  - ikzf4
  - nod2
  - nod1
  - irf2b
  - s100a11
---
