---
figid: PMC2694487__1107fig5
figtitle: Human presenilin 1 and 2 (PSEN1 and PSEN2) subnetworks
organisms:
- Ciona intestinalis
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC2694487
filename: 1107fig5.jpg
figlink: /pmc/articles/PMC2694487/figure/F5/
number: F5
caption: 'Comparative network analysis in FunCoup. (A) Subnetworks in human (middle),
  mouse (top), and fly (left) were generated by submitting human presenilin 1 and
  2 (PSEN1 and PSEN2) to FunCoup, asking for one step of network expansion keeping
  the 20 strongest links with P > 0.5, and inclusion of orthologous subnetworks in
  mouse and fly. At the lower right are two newly predicted interactors of the gamma
  secretase complex, BET1 and LFNG. On the right, the color legend for the links is
  shown in terms of evidence type. Species are indicated by gene symbol prefixes:
  has, human; mmu, mouse; and dme, fly. Supplemental Figure 8 presents alternative
  views by species source or predicted class. The jSquid XML source to this figure
  is available as supplementary file psen_lfng_bet.xml. (B) Using FunCoup to identify
  novel candidate genes in Parkinson''s disease. We here employ orthologous networks
  in human (left) and yeast (right, squares). Novel candidates were extracted by looking
  for human genes that were coupled both to known PD genes and to orthologs of yeast
  alpha-synuclein toxicity modifiers. This resulted in 12 novel candidate PD genes
  (red circles). PD and alpha-synuclein toxicity modifier genes not connected to these
  12 genes were omitted from the network for clarity. Edge categories are shown in
  the figure. Node categories are as follows: (yellow) known human PD genes from the
  pathway KEGG05020 (triangles) and their yeast orthologs (squares); (blue) yeast
  modifiers of a-synuclein toxicity (squares) and their human orthologs (circles);
  (red) 12 novel human PD candidate genes (circles) and their yeast orthologs (squares);
  and larger shapes labeled “collapsed,” grouped genes—inparalogs, except for UBE2
  that represents four ubiquitin-conjugating enzymes with similar molecular function.
  For details on how the subnetwork was generated, see Supplemental Methods; for a
  large scale prospective of the presented subinteractome, see Supplemental Figure
  10.'
papertitle: Global networks of functional coupling in eukaryotes from comprehensive
  data integration.
reftext: Andrey Alexeyenko, et al. Genome Res. 2009 Jun;19(6):1107-1116.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6445499
figid_alias: PMC2694487__F5
figtype: Figure
redirect_from: /figures/PMC2694487__F5
ndex: a9b41640-de90-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2694487__1107fig5.html
  '@type': Dataset
  description: 'Comparative network analysis in FunCoup. (A) Subnetworks in human
    (middle), mouse (top), and fly (left) were generated by submitting human presenilin
    1 and 2 (PSEN1 and PSEN2) to FunCoup, asking for one step of network expansion
    keeping the 20 strongest links with P > 0.5, and inclusion of orthologous subnetworks
    in mouse and fly. At the lower right are two newly predicted interactors of the
    gamma secretase complex, BET1 and LFNG. On the right, the color legend for the
    links is shown in terms of evidence type. Species are indicated by gene symbol
    prefixes: has, human; mmu, mouse; and dme, fly. Supplemental Figure 8 presents
    alternative views by species source or predicted class. The jSquid XML source
    to this figure is available as supplementary file psen_lfng_bet.xml. (B) Using
    FunCoup to identify novel candidate genes in Parkinson''s disease. We here employ
    orthologous networks in human (left) and yeast (right, squares). Novel candidates
    were extracted by looking for human genes that were coupled both to known PD genes
    and to orthologs of yeast alpha-synuclein toxicity modifiers. This resulted in
    12 novel candidate PD genes (red circles). PD and alpha-synuclein toxicity modifier
    genes not connected to these 12 genes were omitted from the network for clarity.
    Edge categories are shown in the figure. Node categories are as follows: (yellow)
    known human PD genes from the pathway KEGG05020 (triangles) and their yeast orthologs
    (squares); (blue) yeast modifiers of a-synuclein toxicity (squares) and their
    human orthologs (circles); (red) 12 novel human PD candidate genes (circles) and
    their yeast orthologs (squares); and larger shapes labeled “collapsed,” grouped
    genes—inparalogs, except for UBE2 that represents four ubiquitin-conjugating enzymes
    with similar molecular function. For details on how the subnetwork was generated,
    see Supplemental Methods; for a large scale prospective of the presented subinteractome,
    see Supplemental Figure 10.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pr
  - bai
  - Nu
  - apt
  - ap
  - Apart
  - mu
  - cher
  - arm
  - Cpr
  - spn-A
  - La
  - anon-70Dc
  - Ubc7
  - Sce
  - park
  - ha
  - Rst(2)At
  - Uch
  - Uch-L5
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Prosbeta2
  - betaTub85D
  - nAChRbeta2
  - ss
  - EcR
  - pt
  - Ace
  - Acer
  - Ance
  - Ykt6
  - Dhps
  - PSENEN
  - ALB
  - RIOX2
  - PSEN2
  - LFNG
  - LYPLA2P1
  - ANXA13
  - AKR1A1
  - ARMC9
  - WDR20
  - UBA7
  - CDC34
  - UBE2A
  - UBE2B
  - UBE2C
  - UBE2D1
  - UBE2D2
  - UBE2D3
  - UBE2D4
  - UBE2E1
  - UBE2E2
  - UBE2E3
  - UBE2F
  - UBE2G1
  - UBE2G2
  - UBE2H
  - UBE2I
  - UBE2J1
  - UBE2J2
  - UBE2L3
  - UBE2L6
  - UBE2M
  - UBE2N
  - UBE2NL
  - UBE2Q1
  - UBE2R2
  - UBE2S
  - UBE2V1
  - UBE2V2
  - UBE2W
  - WARS1
  - SNCA
  - PRKN
  - GPR37
  - NEUROD1
  - TUBB4B
  - CALR
  - TRIM21
  - RAB1A
  - ACE
  - YKT6
  - DHPS
  - PIEZO1
---
