---
figid: PMC2702319__1471-2148-9-127-4
figtitle: Distributions of the orthologous gene groups containing duplicate genes
  that arose through 3R-WGD (shown as gray boxes) in the network diagrams
organisms:
- Gallus gallus
- Danio rerio
- Xenopus laevis
- teleost fish
- Oryzias latipes
- Dipnomorpha
- Xenopus tropicalis
- Takifugu rubripes
- Tetrapoda
- Tetraodon nigroviridis
- Gasterosteus aculeatus
- Gnathostomata
- Teleostei
- Latimeria chalumnae
- Callorhinchus callorynchus
- Latimeria menadoensis
- Drosophila melanogaster
- Paracentrotus lividus
- Ciona intestinalis
- Branchiostoma lanceolatum
- Paramecium tetraurelia
- Homo sapiens
- Saccharomyces cerevisiae
- Bos taurus
- Arabidopsis thaliana
- Canis lupus familiaris
- Pan troglodytes
- Mus musculus
organisms_ner:
- Drosophila melanogaster
- Gallus gallus
- Mus musculus
- Homo sapiens
- Xenopus tropicalis
- Bos taurus
- Canis lupus familiaris
- Pan troglodytes
- Latimeria chalumnae
- Danio rerio
- Oryzias latipes
- Takifugu rubripes
- Gasterosteus aculeatus
- Arabidopsis thaliana
- Xenopus laevis
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC2702319
filename: 1471-2148-9-127-4.jpg
figlink: /pmc/articles/PMC2702319/figure/F4/
number: F4
caption: 'Distributions of the orthologous gene groups containing duplicate genes
  that arose through 3R-WGD (shown as gray boxes) in the network diagrams. Panels
  A-D: long-term potentiation (LTP), taste transduction (TT), olfactory transduction
  (OT), and tricarboxylic acid cycle (TCA), respectively. White boxes denote orthologous
  gene groups with no 3R-WGD-derived duplicates (1:1 orthologous relationship between
  tetrapods and teleosts). Dashed boxes denote gene groups in which no clear orthologous
  relationship was identified between tetrapods and teleosts. The network schemes
  of the LTP, TT, OT, and TCA are based on knowledge in human summarized in KEGG pathway
  database [].'
papertitle: Temporal pattern of loss/persistence of duplicate genes involved in signal
  transduction and metabolic pathways after teleost-specific genome duplication.
reftext: Yukuto Sato, et al. BMC Evol Biol. 2009;9:127-127.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9062715
figid_alias: PMC2702319__F4
figtype: Figure
redirect_from: /figures/PMC2702319__F4
ndex: e460e255-dedb-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2702319__1471-2148-9-127-4.html
  '@type': Dataset
  description: 'Distributions of the orthologous gene groups containing duplicate
    genes that arose through 3R-WGD (shown as gray boxes) in the network diagrams.
    Panels A-D: long-term potentiation (LTP), taste transduction (TT), olfactory transduction
    (OT), and tricarboxylic acid cycle (TCA), respectively. White boxes denote orthologous
    gene groups with no 3R-WGD-derived duplicates (1:1 orthologous relationship between
    tetrapods and teleosts). Dashed boxes denote gene groups in which no clear orthologous
    relationship was identified between tetrapods and teleosts. The network schemes
    of the LTP, TT, OT, and TCA are based on knowledge in human summarized in KEGG
    pathway database [].'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pka-C1
  - Pka-R2
  - Pka-C3
  - Pka-R1
  - Pka-C2
  - Rap1
  - Pp1alpha-96A
  - flw
  - br
  - Pp4-19C
  - Pp1-13C
  - Pp1-87B
  - PpD6
  - Cam
  - CaMKII
  - Nmdar2
  - Nmdar1
  - inaC
  - Pkc98E
  - aPKC
  - Pkc53E
  - ras
  - Ras64B
  - Ras85D
  - Raf
  - S6kII
  - Tom
  - alphaTub67C
  - mGluR
  - ra
  - Rala
  - r-l
  - rai
  - CrebB
  - CrebA
  - CBP
  - nej
  - eIF4E1
  - norpA
  - oc
  - gg
  - Ggamma1
  - Ggamma30A
  - pck
  - ca
  - cs
  - Aco
  - Acox57D-d
  - grk
  - mch
  - ATPCL
  - Got2
  - ph-d
  - Hph
  - Pkg21D
  - for
  - fh
  - Ogdh
  - CG33791
  - ClC-a
  - CngA
  - sch
  - 'On'
  - TERF2IP
  - RPS6KA1
  - RARA
  - CAMK4
  - CS
  - ACLY
  - CLYBL
  - FH
  - OGDH
  - DLST
  - DLD
  - Rapgef3
  - Tir1
  - Terf2ip
  - Rap1a
  - Ppp1cc
  - Npy4r
  - Calm1
  - Tmem50a
  - Grin1
  - Itpr3
  - Hras
  - Kras
  - Rem1
  - Zhx2
  - Rps6ka1
  - Tpst2
  - Zfp410
  - Shc3
  - Rara
  - Camk4
  - Creb1
  - Crebbp
  - Pag1
  - Cytip
  - Plcb1
  - Cs
  - Grk4
  - Mch
  - Acly
  - Clybl
  - Dlst
  - Dld
  - Ldhd
  - Gpx6
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - RAPGEF3
  - KCNJ1
  - KCNJ10
  - KCNJ11
  - KCNJ12
  - KCNJ13
  - KCNJ14
  - KCNJ15
  - KCNJ16
  - KCNJ18
  - KCNJ2
  - KCNJ3
  - KCNJ4
  - KCNJ5
  - KCNJ6
  - KCNJ8
  - KCNJ9
  - KCNK1
  - KCNK10
  - KCNK12
  - KCNK13
  - KCNK15
  - KCNK16
  - KCNK17
  - KCNK18
  - KCNK2
  - KCNK3
  - KCNK4
  - KCNK5
  - KCNK6
  - KCNK7
  - KCNK9
  - KCNT1
  - KCNT2
  - KCNMA1
  - KCNN1
  - KCNN2
  - KCNN3
  - KCNN4
  - KCNU1
  - KCNA1
  - KCNA10
  - KCNA2
  - KCNA3
  - KCNA4
  - KCNA5
  - KCNA6
  - KCNA7
  - KCNB1
  - KCNB2
  - KCNC1
  - KCNC2
  - KCNC3
  - KCNC4
  - KCND1
  - KCND2
  - KCND3
  - KCNF1
  - KCNG1
  - KCNG2
  - KCNG3
  - KCNG4
  - KCNQ1
  - KCNQ2
  - KCNQ3
  - KCNQ4
  - KCNQ5
  - KCNS1
  - KCNS2
  - KCNS3
  - KCNV1
  - KCNV2
  - KCNH1
  - KCNH2
  - KCNH3
  - KCNH4
  - KCNH5
  - KCNH6
  - KCNH7
  - KCNH8
  - RAP1A
  - RABGEF1
  - RAP1B
  - CACNA1A
  - CACNA1B
  - CACNA1C
  - CACNA1D
  - CACNA1E
  - CACNA1F
  - CACNA1G
  - CACNA1H
  - CACNA1I
  - CACNA1S
  - CACNA2D1
  - CACNA2D2
  - CACNA2D3
  - CACNA2D4
  - CACNB1
  - CACNB2
  - CACNB3
  - CACNB4
  - CACNG1
  - CACNG2
  - CACNG3
  - CACNG4
  - CACNG5
  - CACNG6
  - CACNG7
  - CACNG8
  - PPA1
  - NPY4R
  - NPY4R2
  - CALM1
  - CALM2
  - CALM3
  - KRIT1
  - CAMKMT
  - GRIN1
  - GRIN2A
  - GRIN2B
  - GRIN2C
  - GRIN2D
  - GRIN3A
  - GRIN3B
  - ITPR3
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - KRAS
  - HRAS
  - NRAS
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - RPS6KA2
  - RPS6KA3
  - RPS6KA4
  - RPS6KA5
  - RPS6KA6
  - RPS6KB1
  - RPS6KB2
  - PRPF6
  - GRM1
  - GRM2
  - GRM3
  - GRM4
  - GRM5
  - GRM6
  - GRM7
  - GRM8
  - NRF1
  - ST8SIA6-AS1
  - ZNF410
  - RALA
  - RALB
  - GDF1
  - RNH1
  - PPP1R13L
  - SHC3
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - CREBBP
  - EIF4E
  - OPN1LW
  - PAG1
  - PCBP4
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - GNG10
  - GNG11
  - GNG12
  - GNG13
  - GNG2
  - GNG3
  - GNG4
  - GNG5
  - GNG7
  - GNG8
  - GNGT1
  - GNGT2
  - KLK15
  - GRK1
  - GRK4
  - GRK5
  - GRK6
  - GRK7
  - GRK2
  - GRK3
  - PMCH
  - GOT1
  - GOT2
  - PDC
  - PRKG1
  - CAMK2B
  - CAMK2A
  - CAMK2D
  - CAMK2G
  - ALPG
  - GUCA1A
  - CNGA1
  - CNGA2
  - CNGA3
  - CNGA4
  - CNGB1
  - CNGB3
  - HCN1
  - HCN2
  - HCN3
  - HCN4
  - NF2
  - FAM3A
  - LDHD
  - terf2ip
  - dnajc27
  - rap1a
  - npy4r
  - hras
  - braf
  - rps6ka1
  - rps6ka3
  - nrf1
  - znf410
  - rala
  - camk4
  - creb1
  - eif4e
  - crebbp
  - opn1lw
  - acox1
  - pmch
  - acly
  - got2
  - clybl
  - prkg1
  - ogdh
  - camk2b
  - camk2a
  - camk2d
  - camk2g
  - nf2
  - dlst
  - dld
  - parapinopsina
  - rab1ab
  - kita
  - ngfra
  - pcxb
  - pck2
  - aclya
  - ogdha
  - ldhd
  - si:ch73-60h1.1
  - ogdhb
  - dldh
  - TIR1
  - APA1
  - CLC-A
  - prickle1.S
  - dnajc27.L
  - terf2ip.L
  - rap1a.S
  - dnajc27.S
  - rap1a.L
  - npy4r.L
  - npy4r.S
  - calm2.L
  - raf1.S
  - rps6ka3.L
  - rps6ka1.L
  - rps6ka3.S
  - nrf1.S
  - nrf1.L
  - camk4.L
  - creb1.S
  - creb1.L
  - crebbp.S
  - crebbp.L
  - acox1.S
  - acox1.L
  - pmch.L
  - pmch.S
  - acly.S
  - got2.L
  - got2.S
  - clybl.L
  - prkg1.L
  - prkg1.S
  - ogdh.S
  - camk2g.S
  - camk2b.L
  - camk2d.L
  - camk2d.S
  - camk2a.S
  - camk2g.L
  - dlst.S
  - dld.S
  - patZ
  - creB
  - clcA
  - cyuP
  - Cancer
---
