---
figid: PMC2720375__pone.0006619.g007
figtitle: Genome-Wide Screen of Genes Required for Caffeine Tolerance in Fission Yeast
organisms:
- Saccharomyces cerevisiae
- Escherichia coli
- Pseudomonas aeruginosa
- Candida albicans
- Schizosaccharomyces pombe
- Escherichia coli IS1
- Homo sapiens
- Oryctolagus cuniculus
- His2 virus
pmcid: PMC2720375
filename: pone.0006619.g007.jpg
figlink: /pmc/articles/PMC2720375/figure/pone-0006619-g007/
number: F7
caption: 'The main detoxification mechanism for caffeine is extrusion by efflux pumps
  Hba2 and Caf5, which basal expression levels are dependent on the transcription
  factor (TF) Pap1. Pap1 is not activated by caffeine, but it shuttles under basal
  conditions between the nucleus (in pale grey, dotted line) and the cytosol, and
  regulates the basal expression of about 50 genes . Caffeine interferes with cellular
  activities and induces a number of toxic effects (in grey ovals): cell wall damage,
  protein traffic and general fitness impairment, cell cycle arrest and DNA damage.
  Most of the genes we have isolated in our screen code for proteins which can combat
  each one of those deleterious effects of caffeine, and have been grouped accordingly.
  The Sty1 pathway is activated in response to caffeine through an unknown mechanism.
  Activated Sty1 is then essential to activate changes in gene expression through
  the transcription factor Atf1, and these changes may modulate the cellular adaptation
  to caffeine. Chromatin remodelers and regulation of mRNA homeostasis may help regulate
  gene induction in response to caffeine as well. Lastly, proteins such as Moe1 may
  be essential to wild-type tolerance to caffeine due to its role in the regulation
  of Atf1 protein basal levels.'
papertitle: Genome-Wide Screen of Genes Required for Caffeine Tolerance in Fission
  Yeast.
reftext: Isabel A. Calvo, et al. PLoS One. 2009;4(8):e6619.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9267391
figid_alias: PMC2720375__F7
figtype: Figure
organisms_ner:
- Schizosaccharomyces pombe
- Saccharomyces cerevisiae S288C
- Homo sapiens
- Oryctolagus cuniculus
redirect_from: /figures/PMC2720375__F7
ndex: 886197bf-df0c-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2720375__pone.0006619.g007.html
  '@type': Dataset
  description: 'The main detoxification mechanism for caffeine is extrusion by efflux
    pumps Hba2 and Caf5, which basal expression levels are dependent on the transcription
    factor (TF) Pap1. Pap1 is not activated by caffeine, but it shuttles under basal
    conditions between the nucleus (in pale grey, dotted line) and the cytosol, and
    regulates the basal expression of about 50 genes . Caffeine interferes with cellular
    activities and induces a number of toxic effects (in grey ovals): cell wall damage,
    protein traffic and general fitness impairment, cell cycle arrest and DNA damage.
    Most of the genes we have isolated in our screen code for proteins which can combat
    each one of those deleterious effects of caffeine, and have been grouped accordingly.
    The Sty1 pathway is activated in response to caffeine through an unknown mechanism.
    Activated Sty1 is then essential to activate changes in gene expression through
    the transcription factor Atf1, and these changes may modulate the cellular adaptation
    to caffeine. Chromatin remodelers and regulation of mRNA homeostasis may help
    regulate gene induction in response to caffeine as well. Lastly, proteins such
    as Moe1 may be essential to wild-type tolerance to caffeine due to its role in
    the regulation of Atf1 protein basal levels.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - rhp54
  - rad3
  - rhp51
  - ssb3
  - cut8
  - pic1
  - hos2
  - dad2
  - spc34
  - mkh1
  - prz1
  - erg5
  - sre2
  - rho2
  - kin1
  - par1
  - rpm1
  - myo1
  - mfm2
  - mug24
  - sty1
  - SPCC594.04c
  - aps1
  - vas2
  - tpp1
  - atf1
  - moe1
  - cut12
  - ccr4
  - bfr1
  - pap1
  - caf5
  - RAD3
  - RAD51
  - HOS2
  - SPC34
  - MDM38
  - ERG5
  - RHO2
  - KIN1
  - YAP1
  - MYO1
  - APS1
  - TPP1
  - ATF1
  - STF1
  - SIN3
  - CCR4
  - PAP1
  - BAP3
  - SPSB3
  - SUMO1
  - MARK2
  - F2R
  - NR1I2
  - SLC52A2
  - PWAR1
  - MYO1A
  - MYO1B
  - MYO1C
  - MYO1D
  - MYO1E
  - MYO1F
  - MYO1G
  - MYO1H
  - CRYAB
  - AIRE
  - NUDT11
  - ACD
  - GDNF
  - TF
  - CSTA
  - PDX1
  - CNOT6
  - HBA2
  - REG3A
  - RP9
  - PDAP1
  - LPIN1
  - METTL9
  - HBA
  - Caffeine
  - Sterol
---
