---
figid: PMC2603471__nihms81687f1
figtitle: 'Ageing, oxidative stress and cancer: paradigms in parallax'
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC2603471
filename: nihms81687f1.jpg
figlink: /pmc/articles/PMC2603471/figure/F1/
number: F1
caption: Top-scoring gene networks, which are determined by ingenuity Pathways systems
  analysis of age and oxidative stress gene signatures in oestrogen receptor (ER)-positive
  breast cancers, commonly identify tumour necrosis factor (TNF) and transforming
  growth factor-β (TGFβ) pathway nodes. Ariadne Pathway studio was used to identify
  the respective TNF- and TGFβ- regulated (direct or indirect) targets from the 126-gene
  age signature (a) and the 62-gene oxidant–oestrogen+ER signature (intersection of
  oestrogen- and ER-regulated signature and oxidant-stressed signature; b). Regulatory
  directions were determined from primary references provided by Pathway studio, with
  positive (+) regulatory directions denoted by arrow-heads and negative regulatory
  directions denoted by bars (lines without either arrowheads or bars indicate ambiguous
  regulatory direction). Unigene symbols in red ovals indicate signature genes that
  are upregulated by oxidative stress or in early-onset breast cancer; those in green
  ovals indicate signature genes that are downregulated by oxidative stress or in
  early-onset breast cancer. At least 75% of the signature genes that have been shown
  to be regulated by TNF and TGFβ (a, b) contain nuclear factor κB (NFκB) or AP1 consensus
  element binding sites within their proximal promoters, as determined by EZRetrieve
  and TFSEARCH– (see TFSEARCH in Further information). ABCC3, ATP-binding cassette
  3; ANG, angiogenin; AREG, amphiregulin; CDC2, cell division cycle 2; CLDN1, claudin
  1; CTGF, connective tissue growth factor; CXCR4, chemokine (C-X-C motif) receptor
  4; CXCL13, chemokine (C-X-C motif) ligand 13; CYP1A1, cytochrome P450 1A1; EGR1,
  early growth response 1; DAB2, disabled homologue 2; EFNA1, ephrin A1; ESR1, oestrogen
  receptor 1; GAL, galanin prepropeptide; IGFBP5, insulin-like growth factor binding
  protein 5; ITPR1, inositol 1,4,5-triphosphate receptor 1; KRT17, kera-tin 17; LAMC2,
  laminin-γ 2; LCN2, lipocalin 2; LEF1, lymphoid enhancer-binding factor 1; LRP2,
  low density lipoprotein-related protein 2; MMP9, matrix metalloproteinase 9; PLN,
  phospholamban; PLIN, perilipin; PTHLH, parathyroid hormone-like hormone; PTGES,
  prostaglandin E synthase; SERPINA1, serpin peptidase inhibitor 1; SOD3, superoxide
  dismutase 3; TAGLN, transgelin; TNC, tenascin C; TNFSF10, tumour necrosis factor
  (ligand) superfamily 10.
papertitle: 'Ageing, oxidative stress and cancer: paradigms in parallax.'
reftext: Christopher C. Benz, et al. Nat Rev Cancer. ;8(11):875-879.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9356749
figid_alias: PMC2603471__F1
figtype: Figure
redirect_from: /figures/PMC2603471__F1
ndex: d2bd5d16-df04-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2603471__nihms81687f1.html
  '@type': Dataset
  description: Top-scoring gene networks, which are determined by ingenuity Pathways
    systems analysis of age and oxidative stress gene signatures in oestrogen receptor
    (ER)-positive breast cancers, commonly identify tumour necrosis factor (TNF) and
    transforming growth factor-β (TGFβ) pathway nodes. Ariadne Pathway studio was
    used to identify the respective TNF- and TGFβ- regulated (direct or indirect)
    targets from the 126-gene age signature (a) and the 62-gene oxidant–oestrogen+ER
    signature (intersection of oestrogen- and ER-regulated signature and oxidant-stressed
    signature; b). Regulatory directions were determined from primary references provided
    by Pathway studio, with positive (+) regulatory directions denoted by arrow-heads
    and negative regulatory directions denoted by bars (lines without either arrowheads
    or bars indicate ambiguous regulatory direction). Unigene symbols in red ovals
    indicate signature genes that are upregulated by oxidative stress or in early-onset
    breast cancer; those in green ovals indicate signature genes that are downregulated
    by oxidative stress or in early-onset breast cancer. At least 75% of the signature
    genes that have been shown to be regulated by TNF and TGFβ (a, b) contain nuclear
    factor κB (NFκB) or AP1 consensus element binding sites within their proximal
    promoters, as determined by EZRetrieve and TFSEARCH– (see TFSEARCH in Further
    information). ABCC3, ATP-binding cassette 3; ANG, angiogenin; AREG, amphiregulin;
    CDC2, cell division cycle 2; CLDN1, claudin 1; CTGF, connective tissue growth
    factor; CXCR4, chemokine (C-X-C motif) receptor 4; CXCL13, chemokine (C-X-C motif)
    ligand 13; CYP1A1, cytochrome P450 1A1; EGR1, early growth response 1; DAB2, disabled
    homologue 2; EFNA1, ephrin A1; ESR1, oestrogen receptor 1; GAL, galanin prepropeptide;
    IGFBP5, insulin-like growth factor binding protein 5; ITPR1, inositol 1,4,5-triphosphate
    receptor 1; KRT17, kera-tin 17; LAMC2, laminin-γ 2; LCN2, lipocalin 2; LEF1, lymphoid
    enhancer-binding factor 1; LRP2, low density lipoprotein-related protein 2; MMP9,
    matrix metalloproteinase 9; PLN, phospholamban; PLIN, perilipin; PTHLH, parathyroid
    hormone-like hormone; PTGES, prostaglandin E synthase; SERPINA1, serpin peptidase
    inhibitor 1; SOD3, superoxide dismutase 3; TAGLN, transgelin; TNC, tenascin C;
    TNFSF10, tumour necrosis factor (ligand) superfamily 10.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LCN2
  - SOD3
  - PLIN1
  - PTGES
  - IGFBP5
  - KRT17
  - TNFSF10
  - TNF
  - PEG3
  - TGFB1
  - PTHLH
  - MMP9
  - BCL2
  - ABCC3
  - CCN2
  - CDK1
  - POLD1
  - CXCL13
  - CXCR4
  - LRP2
  - GAL
  - GALP
  - DAB2
  - PLN
  - AREG
  - TAGLN
  - TNC
  - TNNC1
  - LAMC2
---
