---
figid: PMC2215060__nihms34233f1a
figtitle: 'WE GATHER TOGETHER: INSULATORS AND GENOME ORGANIZATION'
organisms:
- Mus musculus
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC2215060
filename: nihms34233f1a.jpg
figlink: /pmc/articles/PMC149053/figure/fig7/
number: F1a
caption: A. Action of CTCF at the mouse Igf2/H19 locus. On the maternally transmitted
  allele, CTCF binds to sites in the imprinted control region (ICR) and prevents downstream
  enhancers from activating Igf2 expression. The ICR of the paternal allele is methylated,
  CTCF does not bind, and the enhancer is no longer blocked [–].B. Loop formation
  stabilized by proteins and RNA bound to gypsy sites in Drosophila [] (Figure taken
  from []; see [] for review of a more complete version of this complex). For clarity,
  one set of interactions, leading to the formation of a single loop, is shown. Clusters
  of such sites form insulator bodies in vivo. (a) The proteins recruited to the site
  include the DNA-binding protein Su(Hw), CP190, and Mod(mdg4)2.2, as well as Topors,
  a ubiquitin ligase, which associates with the nuclear lamina and the insulator complex
  and is required for insulator function. (b,c) Loop formation and insulator activity
  also involve RNA and are dependent on members of the RNAi processing pathway. (d)
  Loop formation is interfered with by the RNA-binding protein, Rm62. The protein
  CP190 is additionally involved in insulator function at sites that do not contain
  Su(Hw).C. Contacts between distant CTCF-binding sites (red circles), which are also
  nuclease hypersensitive sites (HS), in embryonic day 12.5 mouse erythroid progenitor
  cells [,]. HS-62 is ~ 62kb upstream, and HS5 ~ 30kb upstream, of the globin gene
  Ey. 3′HS1 is downstream of the globin gene array. Other hypersensitive sites (yellow)
  mark locus control regions; not all such sites are shown. The globin genes (blue
  squares) are not expressed in these cells. Mutating the CTCF site at 3′HS1 destroys
  the interactions between it and the other two CTCF sites. At a later stage of erythroid
  development additional contacts are made between locus control region and other
  regulatory sites to form the ACH; this structure is independent of CTCF binding
  [].D. Loop domain models for the action of the ICR at the mouse Igf2/H19 locus,
  showing contacts involving the ICR, detected by 3C and other methods, that differ
  on the maternal and paternal alleles [,]. It is proposed that the conformation of
  the paternal allele allows contacts between Igf2 promoters and the enhancer that
  are blocked on the maternal allele. (a,b) The ICR on the maternal allele (a) makes
  contact with an imprinted site (DMR1) located upstream of the Igf2 promoters. On
  the paternal allele (b) the ICR contacts another such site, DMR2, downstream of
  Igf2. This should make the Igf2 promoters accessible to the enhancers located downstream
  of H19. For a more elaborate model based on additional data, see []. (c,d) A second
  set of 3C experiments [] shows that on the maternal allele (c) one of the downstream
  enhancers (mesodermal or endodermal specific, depending on cell type) contacts the
  ICR, effectively inhibiting contact between enhancer and Igf2 promoter. Other interactions,
  not shown, occur between the ICR and the Igf2 promoter. On the paternal allele (d),
  with CTCF not bound to the ICR, the enhancer makes contact with the appropriate
  subset of Igf2 promoters. Differences between the results shown in (a,b) and (c,d)
  may reflect different choices of anchor and target sites for the 3C analysis, and
  may not be mutually exclusive. Some of these interactions may be tissue-specific;
  which interactions are most important for establishing allele-specific expression
  is not yet clear.
papertitle: 'WE GATHER TOGETHER: INSULATORS AND GENOME ORGANIZATION.'
reftext: Julie A. Wallace, et al. Curr Opin Genet Dev. ;17(5):400-407.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9337959
figid_alias: PMC2215060__F1a
figtype: Figure
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
- Mus musculus
redirect_from: /figures/PMC2215060__F1a
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2215060__nihms34233f1a.html
  '@type': Dataset
  description: A. Action of CTCF at the mouse Igf2/H19 locus. On the maternally transmitted
    allele, CTCF binds to sites in the imprinted control region (ICR) and prevents
    downstream enhancers from activating Igf2 expression. The ICR of the paternal
    allele is methylated, CTCF does not bind, and the enhancer is no longer blocked
    [–].B. Loop formation stabilized by proteins and RNA bound to gypsy sites in Drosophila
    [] (Figure taken from []; see [] for review of a more complete version of this
    complex). For clarity, one set of interactions, leading to the formation of a
    single loop, is shown. Clusters of such sites form insulator bodies in vivo. (a)
    The proteins recruited to the site include the DNA-binding protein Su(Hw), CP190,
    and Mod(mdg4)2.2, as well as Topors, a ubiquitin ligase, which associates with
    the nuclear lamina and the insulator complex and is required for insulator function.
    (b,c) Loop formation and insulator activity also involve RNA and are dependent
    on members of the RNAi processing pathway. (d) Loop formation is interfered with
    by the RNA-binding protein, Rm62. The protein CP190 is additionally involved in
    insulator function at sites that do not contain Su(Hw).C. Contacts between distant
    CTCF-binding sites (red circles), which are also nuclease hypersensitive sites
    (HS), in embryonic day 12.5 mouse erythroid progenitor cells [,]. HS-62 is ~ 62kb
    upstream, and HS5 ~ 30kb upstream, of the globin gene Ey. 3′HS1 is downstream
    of the globin gene array. Other hypersensitive sites (yellow) mark locus control
    regions; not all such sites are shown. The globin genes (blue squares) are not
    expressed in these cells. Mutating the CTCF site at 3′HS1 destroys the interactions
    between it and the other two CTCF sites. At a later stage of erythroid development
    additional contacts are made between locus control region and other regulatory
    sites to form the ACH; this structure is independent of CTCF binding [].D. Loop
    domain models for the action of the ICR at the mouse Igf2/H19 locus, showing contacts
    involving the ICR, detected by 3C and other methods, that differ on the maternal
    and paternal alleles [,]. It is proposed that the conformation of the paternal
    allele allows contacts between Igf2 promoters and the enhancer that are blocked
    on the maternal allele. (a,b) The ICR on the maternal allele (a) makes contact
    with an imprinted site (DMR1) located upstream of the Igf2 promoters. On the paternal
    allele (b) the ICR contacts another such site, DMR2, downstream of Igf2. This
    should make the Igf2 promoters accessible to the enhancers located downstream
    of H19. For a more elaborate model based on additional data, see []. (c,d) A second
    set of 3C experiments [] shows that on the maternal allele (c) one of the downstream
    enhancers (mesodermal or endodermal specific, depending on cell type) contacts
    the ICR, effectively inhibiting contact between enhancer and Igf2 promoter. Other
    interactions, not shown, occur between the ICR and the Igf2 promoter. On the paternal
    allele (d), with CTCF not bound to the ICR, the enhancer makes contact with the
    appropriate subset of Igf2 promoters. Differences between the results shown in
    (a,b) and (c,d) may reflect different choices of anchor and target sites for the
    3C analysis, and may not be mutually exclusive. Some of these interactions may
    be tissue-specific; which interactions are most important for establishing allele-specific
    expression is not yet clear.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CTCF
  - IGF2
  - PERCC1
  - ICAM4-AS1
  - H19
  - H1-9P
  - Me
  - ee
  - Ctcf
  - Igf2
  - D7Zem1
  - D7Zem3
  - D7Zem2
  - Nckap1
  - H19h
---
