---
figid: PMC2856676__pone.0010065.g001
figtitle: Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon
  as a Model for the Temperate Grasses
organisms:
- Arabidopsis thaliana
- Zea mays
- Triticum aestivum
- Hordeum vulgare
- Brachypodium distachyon
- Triticum monococcum
- Homo sapiens
pmcid: PMC2856676
filename: pone.0010065.g001.jpg
figlink: /pmc/articles/PMC2856676/figure/pone-0010065-g001/
number: F1
caption: 'Arrows show promoting effects, T-bars show repressing effects. Environmental
  cues are shown as lower case letters in square boxes; v is extended cold (vernalization);
  ld is long days; sd is short days; am is ambient (non-vernalizing) temperature.
  Genes are shown in italics and proteins in non-italics in ovals. Proteins are clearly
  involved at all stages but we make this distinction only in cases where separate
  controls are known to exist for transcription and protein function or stability.
  # indicates inhibition of protein function. The pathways were compiled from the
  following reviews , , , , , , , , , , , , , , , ,  plus individual articles referenced
  in the text. Box A. In Arabidopsis genes assigned to pathways are shown in colour
  (photoperiod pathway in green, vernalization pathway in blue, autonomous pathway
  in light blue). Flowering pathway integrators are shown in red. Triple headed arrows
  indicate activation by red or blue light. Box B. Rice genes identified as homologues
  of those in Arabidopsis are shown in black. Ambiguous homology is indicated with
  a question mark. Genes with published roles in photoperiod regulated flowering are
  shown in green and orthologues of pathway integrators in red. Box C. In the temperate
  grasses box Brachypodium genes identified as homologues of those of Arabidopsis
  or rice are shown in black. The VRN2 and CO3 genes from barley and wheat are also
  included. Genes with published roles in photoperiod or vernalization regulated flowering
  in barley or wheat are shown in green or blue, respectively, and orthologues of
  pathway integrators in red. In B and C likely orthologues of Arabidopsis genes are
  shown in equivalent positions within the figure (e.g. Hd1 is the rice orthologue
  of CO and Hd3a and RFT1 are rice orthologues of FT).  and  gives information on
  gene names and synonyms. Square brackets in box B show three maize genes with known
  flowering time effects. Notes: *1 TOC1 represses PRR9, PRR7 and PRR5 . *2 SUF4 interacts
  with FRI and FRL1. LD blocks SUF4 action to reduce FLC expression . *3 Cerdan and
  Chory  identified PFT1 and proposed that it promotes flowering by a photoperiod
  independent pathway activating FT. The model here follows Wollenberg et al.  who
  propose an alternative in which PFT1 acts by repressing Pfr phytochrome signalling.
  *4 SWR1 is a protein complex including SWC6, ARP6 and PIE1. *5 CO expression from
  phloem-specific promoters complements the co mutation, but expression from meristem-specific
  promoters does not. CO triggers flowering from the phloem by the cell-autonomous
  activation of FT expression . *6 High temperatures (27°C vs 23°C) in short days
  induce flowering by a mechanism that requires FT and gibberellic acid and likely
  involves MAF1 (FLM). High temperature induction of flowering is independent of the
  photoperiod pathway but is repressed by FLC and by autonomous pathway mutations
  that increase FLC levels . Kumar and Wigge  showed that ambient temperature responses,
  including the activation of FT, involve the replacement of histone H2A.Z with H2A
  in nucleosomes. Mutation of ARP6 (part of the SWR1 complex) blocks this process.
  The late-flowering phenotype of dominant fwa mutants is caused by hypomethylation
  in the FWA locus leading to ectopic expression of a homeodomain leucine zipper (HD-ZIP)
  protein. The FWA protein interacts with FT protein and so ectopic expression of
  FWA delays flowering by impairing FT function . FWA is not expressed during the
  vegetative phase in wild type plants and normally has no role in flowering so homologues
  are not considered in this paper. *7 TWIN SISTER OF FT (TSF) is closely related
  to FT, is activated by CO, has a similar role in flowering and is repressed by SVP
  (; reviewed by , ). *8 Activation of AP1 and LFY requires two paralogous homeobox
  genes (PENNYWISE (PNY; aka BELLRINGER and REPLUMLESS) and POUND-FOOLISH (PNF)).
  Double null mutants express SOC1 and FUL but not AP1 or LFY. The double mutants
  can receive floral inductive signals but do not properly restructure the shoot apical
  meristem to make flowers , . *9 GI promotion of Hd1/CO is shown directly but could
  act through the FKF1/CDF route as in Arabidopsis. *10 Mayfield et al.  found that
  14-3-3 proteins physically interact with CO in Arabidopsis. *11 It is unclear if
  there is a direct connection between OsMADS50 and OsMADS14/15 as well as the connection
  via Edh1. *12 Ppd-H1 promotion of CO is shown directly but could act through the
  FKF1/CDF route as in Arabidopsis. *13 Downregulation of VRT2 by vernalization and
  a role in repressing VRN1 is proposed by Kane et al. ,  but is not supported by
  Trevaskis et al.  so VRT2 is shown in two positions. *14 Shitsukawa et al.  found
  that WSOC1 was expressed preferentially in leaves. Expression was not affected by
  photoperiod or vernalization but was upregulated in seedlings by a GA treatment.
  *15 X indicates an unknown target gene . Key to figures of phylogenetic trees A
  PRANK alignment of protein sequences (shown in the supplementary datasets) was used
  to construct phylogenetic trees using the neighbor-joining algorithm (subsequently
  NJ trees) with bootstrap support from 1000 iterations (values of ≥50% shown). Gene
  identifiers show the species as follows: At for Arabidopsis thaliana, Bradi for
  Brachypodium distachyon, Os for Orzya sativa (rice), Hv for Hordeum vulgare (barley),
  Ta for Triticum aestivum (bread wheat), Tm for Triticum monococcum (einkorn wheat),
  Sb for Sorghum bicolour (sorghum) and Zm for Zea mays (maize). Barley and wheat
  proteins are highly homologous and usually only one is shown. Gene names follow
  the identifiers where relevant to the text and genes with known flowering time roles
  are shown in colour following . In several instances we identified pairs of genes
  in grasses (paralogues) in segmental duplications (SegDup) which likely result from
  a grass whole genome duplication that predates the divergence of the rice, maize,
  sorghum, Brachypodium, barley and wheat lineages. These duplication events are shown
  as solid circles. Similarly, solid diamonds showduplication events in Arabidopsis;
  these are independent from the duplication event in the grasses. Boxed T shows tandem
  or closely arranged paralogues resulting from duplications within individual species.
  Where the protein belongs to a large family we show small clades containing genes
  of interest selected from analyses of whole families. The clades were identified
  either by using an online interrogatory tree of the transcription factor family
  using known flowering-time genes as query sequences  or from the family trees shown
  in the supplementary material. Key to figures of segmental duplications In these
  figures the size and spacing of genes in the respective genomic sequences is shown
  to the physical scale indicated in each figure. The components of each half of the
  duplication are grouped under the bars at the top of the figure. For example,  shows
  part of rice chromosome 3 which is colinear with part of Brachypodium chromosome
  1 and is related to part of rice chromosome 7 by segmental duplication. This section
  of rice 7 is colinear with a separate section of Brachypodium 1. Identifiers and
  names of flowering-time genes and their homologues are shown, with colours following
  . Gene identifiers are also shown at the top and bottom of each segment to allow
  orientation to the respective genome browsers. The segmental duplications are recognized
  by conserved genes (black boxes connected by black lines). Some figures have grey
  boxes which show genes that are not conserved across the duplication in all species.
  Within these regions, other genes are conserved between species but not across the
  duplication (red boxes connected by red lines). These are deletion/insertion events
  that occurred after the duplication arose but before the divergence of the respective
  species. Genes in yellow are not conserved across species and represent candidates
  for deletion/insertion events that occurred after species divergence. Hypothetical
  genes and sequences annotated as transposons are excluded. In most cases the segmental
  duplications extend further than shown as we only use those portions that are relevant
  to the relationships of the flowering-time genes discussed in the text.'
papertitle: Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon
  as a Model for the Temperate Grasses.
reftext: Janet A. Higgins, et al. PLoS One. 2010;5(4):e10065.
year: '2010'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8703857
figid_alias: PMC2856676__F1
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
- Homo sapiens
redirect_from: /figures/PMC2856676__F1
ndex: 1ec8baeb-df15-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2856676__pone.0010065.g001.html
  '@type': Dataset
  description: 'Arrows show promoting effects, T-bars show repressing effects. Environmental
    cues are shown as lower case letters in square boxes; v is extended cold (vernalization);
    ld is long days; sd is short days; am is ambient (non-vernalizing) temperature.
    Genes are shown in italics and proteins in non-italics in ovals. Proteins are
    clearly involved at all stages but we make this distinction only in cases where
    separate controls are known to exist for transcription and protein function or
    stability. # indicates inhibition of protein function. The pathways were compiled
    from the following reviews , , , , , , , , , , , , , , , ,  plus individual articles
    referenced in the text. Box A. In Arabidopsis genes assigned to pathways are shown
    in colour (photoperiod pathway in green, vernalization pathway in blue, autonomous
    pathway in light blue). Flowering pathway integrators are shown in red. Triple
    headed arrows indicate activation by red or blue light. Box B. Rice genes identified
    as homologues of those in Arabidopsis are shown in black. Ambiguous homology is
    indicated with a question mark. Genes with published roles in photoperiod regulated
    flowering are shown in green and orthologues of pathway integrators in red. Box
    C. In the temperate grasses box Brachypodium genes identified as homologues of
    those of Arabidopsis or rice are shown in black. The VRN2 and CO3 genes from barley
    and wheat are also included. Genes with published roles in photoperiod or vernalization
    regulated flowering in barley or wheat are shown in green or blue, respectively,
    and orthologues of pathway integrators in red. In B and C likely orthologues of
    Arabidopsis genes are shown in equivalent positions within the figure (e.g. Hd1
    is the rice orthologue of CO and Hd3a and RFT1 are rice orthologues of FT).  and  gives
    information on gene names and synonyms. Square brackets in box B show three maize
    genes with known flowering time effects. Notes: *1 TOC1 represses PRR9, PRR7 and
    PRR5 . *2 SUF4 interacts with FRI and FRL1. LD blocks SUF4 action to reduce FLC
    expression . *3 Cerdan and Chory  identified PFT1 and proposed that it promotes
    flowering by a photoperiod independent pathway activating FT. The model here follows
    Wollenberg et al.  who propose an alternative in which PFT1 acts by repressing
    Pfr phytochrome signalling. *4 SWR1 is a protein complex including SWC6, ARP6
    and PIE1. *5 CO expression from phloem-specific promoters complements the co mutation,
    but expression from meristem-specific promoters does not. CO triggers flowering
    from the phloem by the cell-autonomous activation of FT expression . *6 High temperatures
    (27°C vs 23°C) in short days induce flowering by a mechanism that requires FT
    and gibberellic acid and likely involves MAF1 (FLM). High temperature induction
    of flowering is independent of the photoperiod pathway but is repressed by FLC
    and by autonomous pathway mutations that increase FLC levels . Kumar and Wigge  showed
    that ambient temperature responses, including the activation of FT, involve the
    replacement of histone H2A.Z with H2A in nucleosomes. Mutation of ARP6 (part of
    the SWR1 complex) blocks this process. The late-flowering phenotype of dominant
    fwa mutants is caused by hypomethylation in the FWA locus leading to ectopic expression
    of a homeodomain leucine zipper (HD-ZIP) protein. The FWA protein interacts with
    FT protein and so ectopic expression of FWA delays flowering by impairing FT function
    . FWA is not expressed during the vegetative phase in wild type plants and normally
    has no role in flowering so homologues are not considered in this paper. *7 TWIN
    SISTER OF FT (TSF) is closely related to FT, is activated by CO, has a similar
    role in flowering and is repressed by SVP (; reviewed by , ). *8 Activation of
    AP1 and LFY requires two paralogous homeobox genes (PENNYWISE (PNY; aka BELLRINGER
    and REPLUMLESS) and POUND-FOOLISH (PNF)). Double null mutants express SOC1 and
    FUL but not AP1 or LFY. The double mutants can receive floral inductive signals
    but do not properly restructure the shoot apical meristem to make flowers , .
    *9 GI promotion of Hd1/CO is shown directly but could act through the FKF1/CDF
    route as in Arabidopsis. *10 Mayfield et al.  found that 14-3-3 proteins physically
    interact with CO in Arabidopsis. *11 It is unclear if there is a direct connection
    between OsMADS50 and OsMADS14/15 as well as the connection via Edh1. *12 Ppd-H1
    promotion of CO is shown directly but could act through the FKF1/CDF route as
    in Arabidopsis. *13 Downregulation of VRT2 by vernalization and a role in repressing
    VRN1 is proposed by Kane et al. ,  but is not supported by Trevaskis et al.  so
    VRT2 is shown in two positions. *14 Shitsukawa et al.  found that WSOC1 was expressed
    preferentially in leaves. Expression was not affected by photoperiod or vernalization
    but was upregulated in seedlings by a GA treatment. *15 X indicates an unknown
    target gene . Key to figures of phylogenetic trees A PRANK alignment of protein
    sequences (shown in the supplementary datasets) was used to construct phylogenetic
    trees using the neighbor-joining algorithm (subsequently NJ trees) with bootstrap
    support from 1000 iterations (values of ≥50% shown). Gene identifiers show the
    species as follows: At for Arabidopsis thaliana, Bradi for Brachypodium distachyon,
    Os for Orzya sativa (rice), Hv for Hordeum vulgare (barley), Ta for Triticum aestivum
    (bread wheat), Tm for Triticum monococcum (einkorn wheat), Sb for Sorghum bicolour
    (sorghum) and Zm for Zea mays (maize). Barley and wheat proteins are highly homologous
    and usually only one is shown. Gene names follow the identifiers where relevant
    to the text and genes with known flowering time roles are shown in colour following
    . In several instances we identified pairs of genes in grasses (paralogues) in
    segmental duplications (SegDup) which likely result from a grass whole genome
    duplication that predates the divergence of the rice, maize, sorghum, Brachypodium,
    barley and wheat lineages. These duplication events are shown as solid circles.
    Similarly, solid diamonds showduplication events in Arabidopsis; these are independent
    from the duplication event in the grasses. Boxed T shows tandem or closely arranged
    paralogues resulting from duplications within individual species. Where the protein
    belongs to a large family we show small clades containing genes of interest selected
    from analyses of whole families. The clades were identified either by using an
    online interrogatory tree of the transcription factor family using known flowering-time
    genes as query sequences  or from the family trees shown in the supplementary
    material. Key to figures of segmental duplications In these figures the size and
    spacing of genes in the respective genomic sequences is shown to the physical
    scale indicated in each figure. The components of each half of the duplication
    are grouped under the bars at the top of the figure. For example,  shows part
    of rice chromosome 3 which is colinear with part of Brachypodium chromosome 1
    and is related to part of rice chromosome 7 by segmental duplication. This section
    of rice 7 is colinear with a separate section of Brachypodium 1. Identifiers and
    names of flowering-time genes and their homologues are shown, with colours following
    . Gene identifiers are also shown at the top and bottom of each segment to allow
    orientation to the respective genome browsers. The segmental duplications are
    recognized by conserved genes (black boxes connected by black lines). Some figures
    have grey boxes which show genes that are not conserved across the duplication
    in all species. Within these regions, other genes are conserved between species
    but not across the duplication (red boxes connected by red lines). These are deletion/insertion
    events that occurred after the duplication arose but before the divergence of
    the respective species. Genes in yellow are not conserved across species and represent
    candidates for deletion/insertion events that occurred after species divergence.
    Hypothetical genes and sequences annotated as transposons are excluded. In most
    cases the segmental duplications extend further than shown as we only use those
    portions that are relevant to the relationships of the flowering-time genes discussed
    in the text.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PRR7
  - PRR9
  - PRR5
  - HD2B
  - PPD1
  - LHY
  - CCA1
  - ELF4
  - ZTL
  - PRR3
  - GI
  - FVE
  - FPA
  - FLK
  - LD
  - FLD
  - FY
  - FCA
  - PCL1
  - CLF
  - FIE
  - MSI1
  - SWN
  - VEL1
  - TCP11
  - VRN5
  - TOC1
  - CRY1
  - VIN3
  - VRN1
  - VRN2
  - COP1
  - ELF3
  - CRY2
  - FKF1
  - LKP2
  - CDF1
  - FRI
  - FRL1
  - SMT2
  - FRL2
  - FES1
  - CDF2
  - CDF3
  - LTP2
  - COL9
  - SUF4
  - EFS
  - MIR172b
  - MIR172a
  - HD1
  - AGL41
  - PHYB
  - SPA1
  - SPA3
  - SPA4
  - PEP
  - PAS2
  - FLC
  - HUA2
  - TFL2
  - SMZ
  - RAP2.7
  - PHYA
  - MAF1
  - TEM1
  - RAV2
  - EHD1
  - AGL19
  - AGL24
  - CIB1
  - SVP
  - AGL20
  - FT
  - FD
  - FT1
  - ELF9
  - SPL3
  - AGL8
  - AGL17
  - AP1
  - TFL1
  - LFY
  - C1orf210
  - HDAC2
  - DLL4
  - GLI1
  - TRNT1
  - NECTIN3
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - CRLF1
  - CRNKL1
  - CLQTL1
  - ACKR2
  - DEFA6
  - HDAC6
  - VIL1
  - CARD16
  - FMNL3
  - HSPBP1
  - PCLAF
  - HDAC1
  - PLEC
  - ID1
  - SIPA1
  - SFTPA1
  - CD5L
  - SFTPA2
  - PAEP
  - PREP
  - PTPN22
  - LARP1
  - TOE1
  - SIRT2
  - SIRT1
  - CD248
  - RASD2
  - AGL
  - TSC1
  - TSC2
  - SRCAP
  - AKTIP
  - RFT1
  - SLC19A1
  - SLC52A1
  - FOS
  - FOSB
  - JUN
  - JUNB
  - JUND
  - FOSL1
  - FOSL2
  - APEX1
---
