---
figid: PMC2739439__pcbi.1000523.g005
figtitle: 'Modular Organization of Protein Interactions in Escherichia coli '
organisms:
- Escherichia coli
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC2739439
filename: pcbi.1000523.g005.jpg
figlink: /pmc/articles/PMC2739439/figure/pcbi-1000523-g005/
number: F5
caption: 'A: Organization of LGT-derived proteins within the combined network. Each
  pie chart indicates a single protein, with the coloured arcs reflecting its phylogenetic
  profile (see inset key). The colour at the centre of each pie chart indicates module
  membership. Large coloured ovals grouping proteins define gene neighbourhoods (each
  gene is within 2000 bp of at least one other gene). Colours of links represent type
  of supporting evidence (GP = genome proximity; RS = Rosetta stone; PP = phylogenetic
  profiles; LT = literature curation; PD = pull down). The embedded colour key indicates
  the breakdown of taxonomic groups used to construct the phylogenetic profiles –
  numbers indicate the number of genomes associated with each group. B: Organization
  of LGT genes with the E. coli genome. The outer circle indicates the location of
  LGT genes. Grey lines indicate LGT genes not identified within our network. Coloured
  lines extending into the center indicate LGT genes identified within our network,
  organized into gene neighbourhoods. Coloured circles indicate the relationship between
  the gene neighbourhoods and their organization within the network shown in A. C:
  Network organization of proteins involved in hydrogenase biosynthesis. Two proteins
  associated with hydrogenase 3, hycE and hycG are thought to derive through LGT and
  are highlighted. Also present in the combined network are proteins associated with:
  hydrogenase 1 (hyaABDEF); hydrogenase 2 (hybCDFO); hydrogenase 4 (hyfBDFGI); hydrogenase
  maturation (hypBCDEF and slyD); and NADH∶ubiquinone biosynthesis (nuoBCEFGHILMN).
  D: Network organization of proteins involved in enterobactin synthesis and related
  processes. Again proteins thought to derive through LGT and are highlighted. Also
  shown are components of the tryptophan biosynthetic pathway responsible for production
  of the chorismate precursor of enterobactin (trpABCDE); and components of two other
  related iron transport systems – fhuABCDE and fecABCD, which uptake iron via hydroxamate
  and dicitrate respectively.'
papertitle: The Modular Organization of Protein Interactions in Escherichia coli .
reftext: José M. Peregrín-Alvarez, et al. PLoS Comput Biol. 2009 Oct;5(10):e1000523.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6891814
figid_alias: PMC2739439__F5
figtype: Figure
redirect_from: /figures/PMC2739439__F5
ndex: e25adfdf-df2d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2739439__pcbi.1000523.g005.html
  '@type': Dataset
  description: 'A: Organization of LGT-derived proteins within the combined network.
    Each pie chart indicates a single protein, with the coloured arcs reflecting its
    phylogenetic profile (see inset key). The colour at the centre of each pie chart
    indicates module membership. Large coloured ovals grouping proteins define gene
    neighbourhoods (each gene is within 2000 bp of at least one other gene). Colours
    of links represent type of supporting evidence (GP = genome proximity; RS = Rosetta
    stone; PP = phylogenetic profiles; LT = literature curation; PD = pull down).
    The embedded colour key indicates the breakdown of taxonomic groups used to construct
    the phylogenetic profiles – numbers indicate the number of genomes associated
    with each group. B: Organization of LGT genes with the E. coli genome. The outer
    circle indicates the location of LGT genes. Grey lines indicate LGT genes not
    identified within our network. Coloured lines extending into the center indicate
    LGT genes identified within our network, organized into gene neighbourhoods. Coloured
    circles indicate the relationship between the gene neighbourhoods and their organization
    within the network shown in A. C: Network organization of proteins involved in
    hydrogenase biosynthesis. Two proteins associated with hydrogenase 3, hycE and
    hycG are thought to derive through LGT and are highlighted. Also present in the
    combined network are proteins associated with: hydrogenase 1 (hyaABDEF); hydrogenase
    2 (hybCDFO); hydrogenase 4 (hyfBDFGI); hydrogenase maturation (hypBCDEF and slyD);
    and NADH∶ubiquinone biosynthesis (nuoBCEFGHILMN). D: Network organization of proteins
    involved in enterobactin synthesis and related processes. Again proteins thought
    to derive through LGT and are highlighted. Also shown are components of the tryptophan
    biosynthetic pathway responsible for production of the chorismate precursor of
    enterobactin (trpABCDE); and components of two other related iron transport systems
    – fhuABCDE and fecABCD, which uptake iron via hydroxamate and dicitrate respectively.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Mi-2
  - Prosbeta2
  - betaTub85D
  - nAChRbeta2
  - FER
  - dan
  - B4
  - Prosbeta4
  - Prosbeta7
  - betaTub97EF
  - La
  - bt
  - act
  - Act5C
  - .na.character
  - Act42A
  - Actbeta
  - key
  - fig
  - lgt
  - Moca-cyp
  - sc
  - gprs
  - rs
  - gp
  - lt
  - tap
  - P32
  - tapas
  - Ent1
  - fray
  - Scox
  - sna
  - op
  - pd
  - lot
  - Pen
  - peng
  - peo
  - PPO1
  - prg
  - prage
  - LBR
  - NEUROD1
  - TUBB4B
  - TUBB3
  - AP4B1
  - BHLHE23
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - DEPP1
  - GOPC
  - DOCK3
  - FLNB
  - USO1
  - NXF1
  - SEC14L2
  - TAP1
  - TAP2
  - ABCB9
  - SLC29A1
  - STMN1
  - CD74
  - DELYQ11
  - PUM3
  - PCSK1N
  - TSPAN33
  - PROA
  - PRB3
  - SRGN
  - RICTOR
  - Phosphonate
---
