---
figid: PMC2702500__emi0011-1038-f1
figtitle: Metabolic reconstruction of Db
organisms:
- Escherichia coli
- Staphylococcus aureus
- Escherichia coli K-12
- Pseudomonas putida
- Haemophilus influenzae
- Bacteroides fragilis
- Campylobacter jejuni
- Synechocystis sp. PCC 6803
- Fusobacterium nucleatum
- Borreliella burgdorferi
- Chlamydia trachomatis
- Clostridium acetobutylicum
- Mycobacterium tuberculosis H37Rv
- Sinorhizobium meliloti
- Zymomonas mobilis
- Xanthomonas campestris
- Pseudomonas putida KT2440
- Bacillus subtilis subsp. subtilis str. 168
- Caulobacter vibrioides
- Thermotoga maritima
- Methanocaldococcus jannaschii
- Corynebacterium glutamicum ATCC 13032
- Bradyrhizobium japonicum
- Deinococcus radiodurans
- Aquifex aeolicus
- Cupriavidus necator H16
- Archaeoglobus fulgidus
- Agrobacterium fabrum str. C58
- Acinetobacter baylyi ADP1
- Clostridium kluyveri
- Moorella thermoacetica
- Leptospira interrogans
- Desulfovibrio vulgaris
- Mesorhizobium loti
- Methanothermobacter thermautotrophicus
- Nostoc sp.
- Thermoplasma acidophilum
- Geobacter sulfurreducens
- Clostridium tetani
- Methanosarcina mazei
- Prochlorococcus marinus
- Nitrosomonas europaea
- Desulfovibrio vulgaris str. Hildenborough
- Desulfovibrio desulfuricans
- Methanopyrus kandleri
- Desulfitobacterium hafniense
- Methanococcus voltae
- Megalodesulfovibrio gigas
- Chlorobaculum tepidum TLS
- Cupriavidus pinatubonensis JMP134
- Methanothermobacter marburgensis
- Dehalococcoides mccartyi 195
- Bdellovibrio bacteriovorus
- Syntrophotalea carbinolica
- Methanosphaera stadtmanae
- Desulforapulum autotrophicum
- Rhodopseudomonas palustris CGA009
- Rhodopirellula baltica
- Aliivibrio fischeri ES114
- Helicobacter pylori J99
- Oceanobacillus iheyensis
- Archaeoglobus fulgidus DSM 4304
- Rhizobium etli CFN 42
- Methanothermobacter thermautotrophicus str. Delta H
- Oleidesulfovibrio alaskensis G20
- Sulfolobus acidocaldarius DSM 639
- Hyperthermus butylicus
- Archaeoglobus profundus
- Desulfitobacterium hafniense Y51
- Desulforapulum autotrophicum HRM2
- Burkholderia mallei ATCC 23344
- Geobacter sulfurreducens PCA
- Desulfovibrio desulfuricans ATCC 27774
- Desulfofarcimen acetoxidans
- Geobacter metallireducens GS-15
- Haloarcula marismortui ATCC 43049
- Anaeromyxobacter dehalogenans 2CP-C
- Desulfotalea psychrophila
- Desulfobacter postgatei
- Thiobacillus denitrificans ATCC 25259
- Syntrophus aciditrophicus SB
- Desulfosudis oleivorans Hxd3
- Desulfotalea psychrophila LSv54
- Clostridium tetani E88
- Moorella thermoacetica ATCC 39073
- Cutibacterium acnes KPA171202
- Salinibacter ruber DSM 13855
- Nitrobacter winogradskyi Nb-255
- Aromatoleum aromaticum EbN1
- Methanothermobacter marburgensis str. Marburg
- Oryza sativa
- Eucalyptus intertexta
- Homo sapiens
- stable RNAs
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC2702500
filename: emi0011-1038-f1.jpg
figlink: /pmc/articles/PMC2702500/figure/fig01/
number: F1
caption: 'Metabolic reconstruction of Db. autotrophicum HRM2 based on known growth
  substrates and metabolic capacities. Complete oxidation of organic substrates and
  CO2 fixation under autotrophic growth conditions proceeds via the Wood–Ljungdahl
  pathway (blue box). The sulfate reduction is given in a dark red box, heterodisulfide
  reduction is given in a dark orange box and electron transfer from Qmo to sulfate
  reduction is given in an orange box. All selenium-dependent proteins are marked
  by dark grey backgrounds and an asterisk (*). The cytoplasmic membrane is given
  in light grey. Arrows indicate metabolic flows; dashed lines indicate assumed or
  putative electron flows. For reasons of simplicity, cytochromes, multihaem proteins
  and the following proteins with one or more paralogues are displayed only once in
  the figure: Acs, CysAW, EtfAB, FdhAB, GlpAB, HdrA, HdrL, MvhD, PorABC, RnfA–E, Sat,
  Suc, SulP, TmcBCA. Abbreviations are: CM, cytoplasmic membrane; OM, outer membrane;
  Acs/CODH, acetyl-CoA synthase/CO dehydrogenase; Acs, acetyl-CoA synthetase; AcsF,
  CODH maturation factor; Adh, aldehyde dehydrogenase; AMP/ADP/ATP, adenosine-monophosphate/-diphosphate/-triphosphate;
  AprAB, adenylylsulfate reductase; AtpA–I, ATP synthase F0/F1; Cit, citrate synthase;
  CO2, carbon dioxide; Cdh1/2, carbon monoxide dehydrogenase; CdhA–E, bifunctional
  acetyl-CoA synthetase/carbon monoxide dehydrogenase; DctPQM, TRAP-type C4-dicarboxylate
  transporter; DsrABC, dissimilatory sulfite reductase complex; EtfABox/EtfABred,
  electron transfer flavoprotein, oxidized and reduced form; FabGH, 3-oxoacyl-acyl
  carrier synthase/reductase; FadDBA, long-chain fatty acid-CoA ligase; FdhABCD, formate
  dehydrogenase; Fdox/Fdred, ferredoxin, oxidized and reduced form; FdrABC, fumarate
  reductase; Fhs, formate-tetrahydrofolate ligase; Fdx, ferredoxin, 4Fe-4S cluster;
  Fhs, formate-tetrahydrofolate ligase; FolD, methylenetetrahydrofolate dehydrogenase;
  FumAC, fumarate hydratase; GlpAB, glycerol-3-phosphate dehydrogenase; HdrADFL, heterodisulfide
  reductase; HmeCDEP, Hdr-like menaquinol-oxidizing complex; HydA, [Fe]-only fusion
  hydrogenase; HyfBCDEFGI, hydrogenase Hyf homologue; HynAB, periplasmic [Ni/Fe] hydrogenase;
  HysAB, periplasmic [Ni/Fe/Se] hydrogenase; Idh, isocitrate dehydrogenase; LdhAB,
  lactate dehydrogenase; LldP, l-lactate permease; Mae, malic enzyme; MetF-ABC, methylenetetrahydrofolate
  reductase; MQ/MQ-H2, menaquinone pool, oxidized and reduced form; MvhADG, methylviologen
  non-reducing hydrogenase; NAD+, NADH/H+, nicotinamide-adenine dinucleotide, oxidized
  and reduced form; NADP+, NADPH/H+, nicotinamide-adenine dinucleotide phosphate,
  oxidized and reduced form; NqrA–F, Na+-translocating NADH-quinone reductase; PccB,
  propionyl-CoA carboxylase; Pfl, pyruvate formate lyase; PorABC, pyruvate:ferredoxin
  oxidoreductase; PPi, pyrophosphate; QmoABC, quinone-interacting membrane-bound oxidoreductase
  complex; RnfA–E, electron transport complex protein; Sat, sulfate adenylyl transferase;
  Sbm, methylmalonyl-CoA mutase; SdhABC, succinate dehydrogenase/fumarate reductase;
  SseA, thiosulfate sulfur transferase; SucCD, succinyl-CoA synthetase; SulP, high-affinity
  H+/SO42− symporter; TmcABC, acidic type II cytochrome complex; Tst, thiosulfate
  sulfur transferase; X/XH2, unknown carrier of reducing equivalent, oxidized and
  reduced form.'
papertitle: Genome sequence of Desulfobacterium autotrophicum HRM2, a marine sulfate
  reducer oxidizing organic carbon completely to carbon dioxide.
reftext: Axel W Strittmatter, et al. Environ Microbiol. 2009 May;11(5):1038-1055.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9473724
figid_alias: PMC2702500__F1
figtype: Figure
redirect_from: /figures/PMC2702500__F1
ndex: c66e8e46-def1-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2702500__emi0011-1038-f1.html
  '@type': Dataset
  description: 'Metabolic reconstruction of Db. autotrophicum HRM2 based on known
    growth substrates and metabolic capacities. Complete oxidation of organic substrates
    and CO2 fixation under autotrophic growth conditions proceeds via the Wood–Ljungdahl
    pathway (blue box). The sulfate reduction is given in a dark red box, heterodisulfide
    reduction is given in a dark orange box and electron transfer from Qmo to sulfate
    reduction is given in an orange box. All selenium-dependent proteins are marked
    by dark grey backgrounds and an asterisk (*). The cytoplasmic membrane is given
    in light grey. Arrows indicate metabolic flows; dashed lines indicate assumed
    or putative electron flows. For reasons of simplicity, cytochromes, multihaem
    proteins and the following proteins with one or more paralogues are displayed
    only once in the figure: Acs, CysAW, EtfAB, FdhAB, GlpAB, HdrA, HdrL, MvhD, PorABC,
    RnfA–E, Sat, Suc, SulP, TmcBCA. Abbreviations are: CM, cytoplasmic membrane; OM,
    outer membrane; Acs/CODH, acetyl-CoA synthase/CO dehydrogenase; Acs, acetyl-CoA
    synthetase; AcsF, CODH maturation factor; Adh, aldehyde dehydrogenase; AMP/ADP/ATP,
    adenosine-monophosphate/-diphosphate/-triphosphate; AprAB, adenylylsulfate reductase;
    AtpA–I, ATP synthase F0/F1; Cit, citrate synthase; CO2, carbon dioxide; Cdh1/2,
    carbon monoxide dehydrogenase; CdhA–E, bifunctional acetyl-CoA synthetase/carbon
    monoxide dehydrogenase; DctPQM, TRAP-type C4-dicarboxylate transporter; DsrABC,
    dissimilatory sulfite reductase complex; EtfABox/EtfABred, electron transfer flavoprotein,
    oxidized and reduced form; FabGH, 3-oxoacyl-acyl carrier synthase/reductase; FadDBA,
    long-chain fatty acid-CoA ligase; FdhABCD, formate dehydrogenase; Fdox/Fdred,
    ferredoxin, oxidized and reduced form; FdrABC, fumarate reductase; Fhs, formate-tetrahydrofolate
    ligase; Fdx, ferredoxin, 4Fe-4S cluster; Fhs, formate-tetrahydrofolate ligase;
    FolD, methylenetetrahydrofolate dehydrogenase; FumAC, fumarate hydratase; GlpAB,
    glycerol-3-phosphate dehydrogenase; HdrADFL, heterodisulfide reductase; HmeCDEP,
    Hdr-like menaquinol-oxidizing complex; HydA, [Fe]-only fusion hydrogenase; HyfBCDEFGI,
    hydrogenase Hyf homologue; HynAB, periplasmic [Ni/Fe] hydrogenase; HysAB, periplasmic
    [Ni/Fe/Se] hydrogenase; Idh, isocitrate dehydrogenase; LdhAB, lactate dehydrogenase;
    LldP, l-lactate permease; Mae, malic enzyme; MetF-ABC, methylenetetrahydrofolate
    reductase; MQ/MQ-H2, menaquinone pool, oxidized and reduced form; MvhADG, methylviologen
    non-reducing hydrogenase; NAD+, NADH/H+, nicotinamide-adenine dinucleotide, oxidized
    and reduced form; NADP+, NADPH/H+, nicotinamide-adenine dinucleotide phosphate,
    oxidized and reduced form; NqrA–F, Na+-translocating NADH-quinone reductase; PccB,
    propionyl-CoA carboxylase; Pfl, pyruvate formate lyase; PorABC, pyruvate:ferredoxin
    oxidoreductase; PPi, pyrophosphate; QmoABC, quinone-interacting membrane-bound
    oxidoreductase complex; RnfA–E, electron transport complex protein; Sat, sulfate
    adenylyl transferase; Sbm, methylmalonyl-CoA mutase; SdhABC, succinate dehydrogenase/fumarate
    reductase; SseA, thiosulfate sulfur transferase; SucCD, succinyl-CoA synthetase;
    SulP, high-affinity H+/SO42− symporter; TmcABC, acidic type II cytochrome complex;
    Tst, thiosulfate sulfur transferase; X/XH2, unknown carrier of reducing equivalent,
    oxidized and reduced form.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NAA15
  - BRD2
  - LDHA
  - DSG4
  - ATP8A2
  - KLK3
  - SH2B2
  - LRP1
  - PMAIP1
  - SUCO
  - AR
  - S100A6
  - MMP8
  - PLA2G15
  - ACSS2
  - ACCS
  - GATA3
  - C2
  - UMOD
  - F12
  - SART1
  - FH
  - TRA
  - THRA
  - TRAC
  - KDM5D
  - FDX1
  - WDTC1
  - om
  - cm
  - na
  - ATPsynbeta
  - Atpalpha
  - SA
  - sa
  - ca
  - Aps
  - su(N)
  - ar
  - pra
  - prg
  - AcCoAS
  - Acsl
  - aop
  - ss
  - Sh
  - chy
  - ty
  - hy
  - fo
  - haf
  - fh
  - tra
  - byn
  - Hr51
  - pi
  - adp
  - Pyruvate
  - Sulfate
---
