---
figid: PMC2242806__1471-2164-8-437-6
figtitle: Schematic presentation of the three L-arginine-degrading pathways in Synechocystis
  sp
organisms:
- Synechocystis sp. PCC 6803
- Synechocystis sp.
- Escherichia coli
- Pseudomonas aeruginosa
- Bacillus subtilis
- Corynebacterium glutamicum
- Pseudomonas putida
- Enterococcus faecalis
- Bacillus subtilis At2
- Escherichia coli str. K-12 substr. MG1655
- Pseudomonas fluorescens
- Pseudomonas aeruginosa PAO1
- Xanthomonas campestris
- Synechococcus elongatus PCC 6301
- Bacillus licheniformis
- Pseudomonas putida KT2440
- Bacillus subtilis subsp. subtilis str. 168
- Synechococcus elongatus PCC 7942  FACHB-805
- Streptomyces griseus
- Corynebacterium glutamicum ATCC 13032
- Nostoc sp. PCC 7120  FACHB-418
- Streptomyces microflavus
- Synechococcus elongatus
- Synechococcus sp.
- Pseudomonas protegens Pf-5
- Nostoc punctiforme PCC 73102
- Pseudomonas savastanoi pv. phaseolicola
- Brucella melitensis bv. 1 str. 16M
- Thermosynechococcus vestitus BP-1
- Gloeobacter violaceus PCC 7421
- Gloeobacter violaceus
- Trichormus variabilis ATCC 29413
- Pseudomonas fluorescens Pf0-1
- Ralstonia pseudosolanacearum GMI1000
- Prochlorococcus marinus subsp. pastoris str. CCMP1986
- Anabaena cylindrica
- Trichodesmium erythraeum IMS101
- Pseudomonas putida F1
- Halalkalibacterium halodurans C-125
- Pseudomonas syringae pv. syringae B728a
- Prochlorococcus marinus str. MIT 9313
- Crocosphaera watsonii
- Prochlorococcus marinus subsp. marinus str. CCMP1375
- Parasynechococcus marenigrum WH 8102
- Crocosphaera watsonii WH 8501
- Xanthomonas campestris pv. campestris str. ATCC 33913
- Geminocystis herdmanii PCC 6308
- Thermoplasma volcanium GSS1
- Synechococcus sp. CC9902
- Brucella melitensis bv. 1
- Thermoplasma acidophilum DSM 1728
- Prochlorococcus marinus str. NATL2A
- Alkalihalobacillus clausii KSM-K16
- Synechococcus sp. WH 5701
- Prochlorococcus marinus str. MIT 9312
- Prochlorococcus marinus str. MIT 9211
- Synechococcus sp. CC9605
- Synechocystis sp. PCC 6803 substr. Kazusa
- Synechococcus sp. RS9917
- Synechococcus sp. WH 7805
- Arabidopsis thaliana
- Glycine max
- Solanum lycopersicum
- Embryophyta
- Giardia intestinalis
- Trichomonas vaginalis
- Tritrichomonas suis
- Homo sapiens
- Rattus norvegicus
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC2242806
filename: 1471-2164-8-437-6.jpg
figlink: /pmc/articles/PMC2242806/figure/F6/
number: F6
caption: Schematic presentation of the three L-arginine-degrading pathways in Synechocystis
  sp. PCC 6803 with the corresponding enzymes, intermediates, cofactors, and final
  products. A). L-arginine decarboxylase pathway most likely only provides polyamines
  and ammonia. B) L-arginine deiminase pathway degrades L-arginine via L-citrulline
  to L-ornithine and carbamoyl phosphate. L-ornithine is further metabolized via glutamate
  semialdehyde to L-glutamate. Glutamate semialdehyde can also be converted to L-proline
  via Δ1pyrroline-5-carboxylate. Carbamoyl phosphate is further metabolized to ammonium
  and carbon dioxide. This enzymatic reaction is catalyzed by the enzyme carbamate
  kinase and is coupled to ATP synthesis. C) The L-arginine oxidase/dehydrogenase
  pathway converts L-arginine to succinate via 2-ketoarginine, 4-guanidinobutyrate,
  4-aminobutyrate, and succinate semialdehyde.
papertitle: Bioinformatic evaluation of L-arginine catabolic pathways in 24 cyanobacteria
  and transcriptional analysis of genes encoding enzymes of L-arginine catabolism
  in the cyanobacterium Synechocystis sp. PCC 6803.
reftext: Sarah Schriek, et al. BMC Genomics. 2007;8:437-437.
year: '2007'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9267004
figid_alias: PMC2242806__F6
figtype: Figure
redirect_from: /figures/PMC2242806__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2242806__1471-2164-8-437-6.html
  '@type': Dataset
  description: Schematic presentation of the three L-arginine-degrading pathways in
    Synechocystis sp. PCC 6803 with the corresponding enzymes, intermediates, cofactors,
    and final products. A). L-arginine decarboxylase pathway most likely only provides
    polyamines and ammonia. B) L-arginine deiminase pathway degrades L-arginine via
    L-citrulline to L-ornithine and carbamoyl phosphate. L-ornithine is further metabolized
    via glutamate semialdehyde to L-glutamate. Glutamate semialdehyde can also be
    converted to L-proline via Δ1pyrroline-5-carboxylate. Carbamoyl phosphate is further
    metabolized to ammonium and carbon dioxide. This enzymatic reaction is catalyzed
    by the enzyme carbamate kinase and is coupled to ATP synthesis. C) The L-arginine
    oxidase/dehydrogenase pathway converts L-arginine to succinate via 2-ketoarginine,
    4-guanidinobutyrate, 4-aminobutyrate, and succinate semialdehyde.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATPsynbeta
  - Atpalpha
  - adp
  - levy
  - COX6AL2
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Pyrroline-5-carboxylic acid
  - Agmatine
  - 4-Aminobutyrate
  - Arginine
  - L-Arginine
  - butyrate 4Guanidinobutyrase butyrate 2-KG
  - Carbamate
  - L-Proline
  - Proline oxidane
  - Putrescine
  - Succinate
  - Succinate semialdehyde
  - Glutamate
---
