---
figid: PMC2912833__1471-2172-11-39-3
figtitle: Generation of a biological network of genes of the most significant Canonical
  Pathways for the EF·CIF (Il10-/- vs
organisms:
- Escherichia coli
- Salmonella enterica subsp. enterica serovar Typhimurium
- Enterococcus faecalis
- Enterococcus faecium
- bacterium 1-7A
- Armoracia rusticana
- Matthiola incana
- Sequoia sempervirens
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Bos taurus
pmcid: PMC2912833
filename: 1471-2172-11-39-3.jpg
figlink: /pmc/articles/PMC2912833/figure/F3/
number: F3
caption: Generation of a biological network of genes of the most significant Canonical
  Pathways for the EF·CIF (Il10-/- vs. C57) comparison. The network was generated
  by IPA using all molecules from significantly affected Canonical Pathways (Fatty
  Acid Metabolism; LPS/IL-1 mediated inhibition of RXR function; Tryptophan Metabolism;
  β-Alanine Metabolism; Valine, Leucine and Isoleucine Degradation; Antigen Presentation
  Pathway; Interferon Signaling; IL-10 Signaling; Fatty Acid Elongation in Mitochondria).
  Connections were applied based on known interactions between these genes within
  the Ingenuity Pathways Knowledge Base. Central genes and their direct interactions
  were identified and were supported by published information. *Genes that are detected
  2 or more times on the array.Genes or gene products are represented as nodes, and
  the biological relationship between two nodes is represented as a line (i.e. an
  edge). All edges are supported by at least 1 reference from the literature. Red
  and green colored nodes indicate genes with up- and down-regulated expression, respectively.
  The intensity of the colors specifies the degree of up- or down-regulation. Greater
  intensity represents a higher level of differential expression. Nodes and edges
  are displayed with various shapes and labels that present the functional class of
  genes and the nature of the relationships between the nodes, as shown in legend
  below the figure.
papertitle: Changes in colon gene expression associated with increased colon inflammation
  in interleukin-10 gene-deficient mice inoculated with Enterococcus species.
reftext: Matthew PG Barnett, et al. BMC Immunol. 2010;11:39-39.
year: '2010'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.783402
figid_alias: PMC2912833__F3
figtype: Figure
organisms_ner:
- Bos taurus
- Homo sapiens
- Mus musculus
- Rattus norvegicus
redirect_from: /figures/PMC2912833__F3
ndex: d2c317e0-df2b-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2912833__1471-2172-11-39-3.html
  '@type': Dataset
  description: Generation of a biological network of genes of the most significant
    Canonical Pathways for the EF·CIF (Il10-/- vs. C57) comparison. The network was
    generated by IPA using all molecules from significantly affected Canonical Pathways
    (Fatty Acid Metabolism; LPS/IL-1 mediated inhibition of RXR function; Tryptophan
    Metabolism; β-Alanine Metabolism; Valine, Leucine and Isoleucine Degradation;
    Antigen Presentation Pathway; Interferon Signaling; IL-10 Signaling; Fatty Acid
    Elongation in Mitochondria). Connections were applied based on known interactions
    between these genes within the Ingenuity Pathways Knowledge Base. Central genes
    and their direct interactions were identified and were supported by published
    information. *Genes that are detected 2 or more times on the array.Genes or gene
    products are represented as nodes, and the biological relationship between two
    nodes is represented as a line (i.e. an edge). All edges are supported by at least
    1 reference from the literature. Red and green colored nodes indicate genes with
    up- and down-regulated expression, respectively. The intensity of the colors specifies
    the degree of up- or down-regulation. Greater intensity represents a higher level
    of differential expression. Nodes and edges are displayed with various shapes
    and labels that present the functional class of genes and the nature of the relationships
    between the nodes, as shown in legend below the figure.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FABP3
  - HMGCS2
  - ACADVL
  - FABP4
  - PPARA
  - MFNG
  - STA
  - IRF1
  - ALDH2
  - MAPK13
  - WARS1
  - CHPT1
  - DHDDS
  - HLA-DRB1
  - HLA-DRB4
  - TRNE
  - TRE-TTC3-1
  - EMD
  - GCY
  - HLA-DOA
  - HLA-DMA
  - CYP2C9
  - HLA-DMB
  - LONP1
  - Fabp3
  - Hmgcs2
  - Acadvl
  - Fabp4
  - Ppara
  - TrnE
  - Mfng
  - Sta
  - Emd
  - Irf1
  - Aldh2
  - Mapk13
  - Wars1
  - Lonp1
  - Aco1
  - Trne
  - Sult2a1
  - Aldh1a1
---
