---
figid: PMC2843790__nihms171328f2
figtitle: 'Switching cell fate: the remarkable rise of iPS cells and lineage reprogramming
  technologies'
organisms:
- NA
pmcid: PMC2843790
filename: nihms171328f2.jpg
figlink: /pmc/articles/PMC2843790/figure/F2/
number: F2
caption: 'Differentiated cells maintain their transcriptome with stable DNA and histone
  modifications for preserving cell-type-specific gene expression. Major epigenetic
  regulation of gene expression occurs through histone acetylation that enhances target
  gene expression, histone methylation that inhibits target gene expression and DNA
  modifications such as methylation of the promoter region that can prevent transcription
  factor binding. In a simplified model, reprogramming cell fate would require the
  initiation of gene expression specific to the target cell type that would most likely
  be within genomic loci that are silenced in a differentiated unrelated cell. Therefore,
  modifying cellular gene expression patterns would involve the regulation of histone
  deacetylases (HDACs), histone methyltransferases (HMTs) and DNA methyl transferases
  (DNMTs) that can modulate genome-wide DNA and histone modifications. In several
  studies, the reprogramming efficiency and generation of iPS cells could be enhanced
  using small molecules to inhibit these enzymes. In addition, factors inhibiting
  cell proliferation also affect epigenetic plasticity. In dividing cells, the chromatin
  is considered to be in an ‘open’ state and thereby more amenable to chemical modifications
  of histones and DNA. However, certain events can make a cell refractory to gene
  expression changes required for reprogramming to occur. Recent studies have shown
  that a tumor suppressor gene, p53, nicknamed the ‘guardian of the genome’ presents
  a barrier to epigenetic reprogramming. The p53 pathway is physiologically activated
  in conditions of DNA damage to induce cell cycle arrest, leading to global chromatin
  condensation. Suppression of p53 by either siRNA or induction of undifferentiated
  embryonic cell transcription factor 1 (UTF1) can significantly enhance reprogramming
  efficiency. Although the link between p53 activation and the reprogramming pathway
  is unclear, this physiological cellular response appears to ensure iPS cell genomic
  integrity. Chromatin-remodeling proteins such as Chd1 promote reprogramming by ensuring
  an open state of chromatin. Telomerase activity is also crucial for reprogramming:
  senescence associated with telomere attrition is inhibitory towards epigenetic reprogramming.'
papertitle: 'Switching cell fate: the remarkable rise of iPS cells and lineage reprogramming
  technologies.'
reftext: Vimal Selvaraj, et al. Trends Biotechnol. ;28(4):214-223.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9362401
figid_alias: PMC2843790__F2
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC2843790__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2843790__nihms171328f2.html
  '@type': Dataset
  description: 'Differentiated cells maintain their transcriptome with stable DNA
    and histone modifications for preserving cell-type-specific gene expression. Major
    epigenetic regulation of gene expression occurs through histone acetylation that
    enhances target gene expression, histone methylation that inhibits target gene
    expression and DNA modifications such as methylation of the promoter region that
    can prevent transcription factor binding. In a simplified model, reprogramming
    cell fate would require the initiation of gene expression specific to the target
    cell type that would most likely be within genomic loci that are silenced in a
    differentiated unrelated cell. Therefore, modifying cellular gene expression patterns
    would involve the regulation of histone deacetylases (HDACs), histone methyltransferases
    (HMTs) and DNA methyl transferases (DNMTs) that can modulate genome-wide DNA and
    histone modifications. In several studies, the reprogramming efficiency and generation
    of iPS cells could be enhanced using small molecules to inhibit these enzymes.
    In addition, factors inhibiting cell proliferation also affect epigenetic plasticity.
    In dividing cells, the chromatin is considered to be in an ‘open’ state and thereby
    more amenable to chemical modifications of histones and DNA. However, certain
    events can make a cell refractory to gene expression changes required for reprogramming
    to occur. Recent studies have shown that a tumor suppressor gene, p53, nicknamed
    the ‘guardian of the genome’ presents a barrier to epigenetic reprogramming. The
    p53 pathway is physiologically activated in conditions of DNA damage to induce
    cell cycle arrest, leading to global chromatin condensation. Suppression of p53
    by either siRNA or induction of undifferentiated embryonic cell transcription
    factor 1 (UTF1) can significantly enhance reprogramming efficiency. Although the
    link between p53 activation and the reprogramming pathway is unclear, this physiological
    cellular response appears to ensure iPS cell genomic integrity. Chromatin-remodeling
    proteins such as Chd1 promote reprogramming by ensuring an open state of chromatin.
    Telomerase activity is also crucial for reprogramming: senescence associated with
    telomere attrition is inhibitory towards epigenetic reprogramming.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BIX 01294
  - 5-aza cytidine
  - Hydroxamic Acid
  - Lysine
  - Valproic Acid
---
