---
figid: PMC2996932__1471-2164-11-404-4
figtitle: Transcriptional evidence for metabolic differences between sessile (biofilm)
  and swimming (planktonic) modes of life
organisms:
- Escherichia coli
- Pseudomonas aeruginosa
- Bacillus subtilis
- Listeria monocytogenes
- Escherichia coli K-12
- Lactiplantibacillus plantarum
- Xanthomonas campestris
- Acidithiobacillus ferrooxidans
- Salmonella sp.
- Acidithiobacillus caldus
- Acidithiobacillus thiooxidans
- Leptospirillum ferrodiazotrophum
- Leptospirillum rubarum
- Leptospirillum ferrooxidans
- Leptospirillum ferriphilum
- Bermanella marisrubri
- Acidimicrobium ferrooxidans
- Homo sapiens
- Hordeum vulgare
- environmental samples
- Octodon degus
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC2996932
filename: 1471-2164-11-404-4.jpg
figlink: /pmc/articles/PMC2996932/figure/F4/
number: F4
caption: 'Transcriptional evidence for metabolic differences between sessile (biofilm)
  and swimming (planktonic) modes of life. Genes encoding mixed acid fermentation
  enzymes are up-regulated in the biofilm (bottom part). As a result, two metabolites
  can be produced: acetyl-phosphate (acP) and acetate. The former can be used as a
  phospho-donor intermediate, while the later can be transformed to acetyl-CoA or
  released to the medium. They could be produced by fermentation of sugars via the
  Embden-Meyerhof-Parnas pathway to pyruvate, then to acetyl phosphate (acP) by pyruvate
  oxidase (PoxB), and finally to acetate by AckA. In contrast, genes encoding key
  TCA cycle enzymes are up-regulated in the swimming (planktonic) cells (top part;
  see also text for explanations). Circles indicate the name of the enzymes whose
  genes are being expressed in the Río Tinto samples (red, preferentially induced
  in biofilm; green, in planktonic cells; and yellow transcribed in both places).
  All of them have been identified in the L. ferrooxidans genome sequence draft (to
  be published elsewhere; see Additional file  Table S2, and Additional file ). POR,
  pyruvate oxidoreductase system; Pta, phosphotransacetylase; Idh, NAD+ dependent
  isocitrate dehydrogenase; Acn, Aconitase; Mdh, Malate dehydrogenase; FumC, fumarate
  hydratase component C; SdhA, succinate dehydrogenase component A; SucCD, Succinate
  dehydrogenase; PpsA, phosphoenolpyruvate synthase; AccACD, acetyl-CoA carboxylase.'
papertitle: Environmental transcriptome analysis reveals physiological differences
  between biofilm and planktonic modes of life of the iron oxidizing bacteria Leptospirillum
  spp. in their natural microbial community.
reftext: Mercedes Moreno-Paz, et al. BMC Genomics. 2010;11:404-404.
year: '2010'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.945357
figid_alias: PMC2996932__F4
figtype: Figure
redirect_from: /figures/PMC2996932__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2996932__1471-2164-11-404-4.html
  '@type': Dataset
  description: 'Transcriptional evidence for metabolic differences between sessile
    (biofilm) and swimming (planktonic) modes of life. Genes encoding mixed acid fermentation
    enzymes are up-regulated in the biofilm (bottom part). As a result, two metabolites
    can be produced: acetyl-phosphate (acP) and acetate. The former can be used as
    a phospho-donor intermediate, while the later can be transformed to acetyl-CoA
    or released to the medium. They could be produced by fermentation of sugars via
    the Embden-Meyerhof-Parnas pathway to pyruvate, then to acetyl phosphate (acP)
    by pyruvate oxidase (PoxB), and finally to acetate by AckA. In contrast, genes
    encoding key TCA cycle enzymes are up-regulated in the swimming (planktonic) cells
    (top part; see also text for explanations). Circles indicate the name of the enzymes
    whose genes are being expressed in the Río Tinto samples (red, preferentially
    induced in biofilm; green, in planktonic cells; and yellow transcribed in both
    places). All of them have been identified in the L. ferrooxidans genome sequence
    draft (to be published elsewhere; see Additional file  Table S2, and Additional
    file ). POR, pyruvate oxidoreductase system; Pta, phosphotransacetylase; Idh,
    NAD+ dependent isocitrate dehydrogenase; Acn, Aconitase; Mdh, Malate dehydrogenase;
    FumC, fumarate hydratase component C; SdhA, succinate dehydrogenase component
    A; SucCD, Succinate dehydrogenase; PpsA, phosphoenolpyruvate synthase; AccACD,
    acetyl-CoA carboxylase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - PREP
  - PTPN22
  - POR
  - VDAC2
  - SDHA
  - IDH1
  - IDH2
  - NF2
  - ACIN1
  - MDH2
  - F11
  - PTCRA
  - WDTC1
  - ACSS2
  - ATP8A2
  - RPS23
  - NDUFAB1
  - CPAT1
  - Pep
  - bgcn
  - por
  - SdhA
  - Idh
  - Idh3b
  - Acn
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - adp
  - ATPsynbeta
  - Atpalpha
  - pi
  - Acp26Ab
  - Acp26Aa
  - CG18284
  - Acp36DE
  - SP
  - Acp95EF
  - Acph-1
  - Succinyl-CoA
  - Acetate
  - Fatty acids
  - Glucose
  - Isocitrate
  - Malonyl-CoA
  - Pyruvate
  - Succinate
---
