---
figid: PMC2566006__zjb019088140002a
figtitle: 'A Defect in Menadione Biosynthesis Induces Global Changes in Gene Expression
  in Staphylococcus aureus  '
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC2566006
filename: zjb019088140002a.jpg
figlink: /pmc/articles/PMC149053/figure/fig7/
number: F2a
caption: 'Visualization of transcriptomic (DNA microarrays) and proteomic (2D gels)
  data for selected genes/proteins of S. aureus COL (wild type) compared to its isogenic
  menD mutant (ΔmenD) and for the hemB mutant (ΔhemB) compared to the menD mutant
  (ΔmenD) grown in TSB medium at the exponential growth phase and at early stationary
  phase. (A) Carbohydrate metabolism. (B) Amino acid metabolism. (C) Fermentation.
  (D) Nitrate/nitrite respiration. Induction ratios were given as logarithms to the
  base 2. Boxes ranging in color from light orange to dark orange indicate transcripts
  or proteins that were significantly downregulated in the menD mutant, and boxes
  ranging from light blue to dark blue indicate transcripts or proteins that were
  significantly upregulated in the menD mutant. White boxes indicate transcripts or
  proteins with no changes between the menD mutant and the wild type or the hemB mutant.
  Gray boxes represent proteins that were not identified on the gels. The IDs, symbols,
  and functions are based on TIGR annotation for S. aureus COL (http://www.tigr.org).
  Glycolysis: Eno, enolase; FbaA, fructose-bisphosphate aldolase; FdaB, fructose-bisphosphate
  aldolase (class I); FruK, 1-phosphofructokinase; GalM, aldose 1-epimerase; GapA1,
  glyceraldehyde-3-phosphate dehydrogenase; GapR, glycolytic operon regulator; Glk,
  glucokinase; IpdC, pyruvate decarboxylase; PfkA, 6-phosphofructokinase; Pgi, glucose-6-phosphate
  isomerase A; Pgk, phosphoglycerate kinase; Pgm, phosphoglycerate mutase; Pyk, pyruvate
  kinase; TpiA, triosephosphate isomerase; 2624, acetyl coenzyme A (acetyl-CoA) synthetase.
  TCA cycle: AcnA, aconitate hydratase; FumC, fumarate hydratase; GltA, citrate synthase;
  Icd, isocitrate dehyrogenase; Mqo1, malate:quinone oxidoreductase; SdhA, succinate
  dehydrogenase flavoprotein subunit; SdhB, succinate dehydrogenase iron-sulfur protein
  subunit; SdhC, succinate dehydrogenase cytochrome b-558; SucA, 2-oxoglutarate dehydrogenase
  E1; SucB, dihydrolipoamide succinyltransferase; SucCD, succinyl-CoA synthetase beta/alpha
  chain; 1308/1309, 2-oxoacid ferredoxin oxidoreductase, alpha/beta subunit; 1749,
  malate dehydrogenase homolog. Pyruvate metabolism: AckA, acetate kinase; PdhABCD,
  pyruvate dehydrogenase E1; Pta, phosphotransacetylase; 2553, putative pyruvate oxidase;
  2574, putative d-lactate dehydrogenase. Gluconeogenesis: Acs, acetyl-CoA synthetase;
  GapA2, glyceraldehyde 3-phosphate dehydrogenase 2; PckA, phosphoenolpyruvate carboxykinase;
  Pyc, pyruvate carboxylase; 2527, fructose-bisphosphatase. Pentose phosphate pathway:
  DeoC1, deoxyribose-phosphate aldolase; Gnd, phosphogluconate dehydrogenase; Tal,
  transaldolase; Zwf, glucose-6-phosphate 1-dehydrogenase. Sugar usage: CscA, sucrose-6-phosphate
  hydrolase; ManA2, mannose-6-phosphate isomerase; MtlD, mannitol-1-phosphate 5-dehydrogenase;
  0517, trehalose-6-phosphate hydrolase; 0598, l-ribulokinase; 0617, putative hexulose-6-phosphate
  synthase; 2028, putative fructokinase. Glycerol degradation: GlpD, glycerol-3-phosphate
  dehydrogenase; GlpF, glycerol uptake facilitator; GlpK, glycerol kinase; 1920, putative
  glycerate dehydrogenase. Acetoin usage: AcuA, acetoin utilization protein. Amino
  acid degradation: Ald1, alanine dehydrogenase; Ald2, alanine dehydrogenase; AldA1,
  aldehyde dehydrogenase; ArgB, N-acetylglutamate 5-phosphotransferase; Asd, aspartate
  semialdehyde dehydrogenase; DapA, dihydrodipicolinate synthase; Dat, d-alanine aminotransferase;
  GcvH, glycine cleavage system protein H homolog; GcvT, aminomethyltransferase; GluD,
  NAD-specific glutamate dehydrogenase; Hom, homoserine dehydrogenase; HutG, formiminoglutamase;
  HutI, imidazolonepropionase; HutU, urocanate hydratase; IlvA1, threonine deaminase
  IlvA homolog; ProC, pyrroline-5-carboxylate reductase; RocD2, ornithine aminotransferase;
  RocF, arginase; SdaAA, l-serine dehydratase; 1349, threonine aldolase; 1562, branched-chain
  alpha-keto acid dehydrogenase E1; 1561, branched-chain alpha-keto acid dehydrogenase
  E1; 1593, glycine dehydrogenase subunit 2 homolog; 1594, glycine dehydrogenase subunit
  1; 2569, 1-pyrroline-5-carboxylate dehydrogenase. Urea degradation: UreA, UreB,
  UreC, UreD, UreE, UreG, and UreF, urease subunits and urease accessory proteins;
  2279, urea transporter. Arginine deiminase pathway (ADI): ArcA, arginine deiminase;
  ArcB2, ornithine transcarbamoylase; ArcC2, carbamate kinase; ArcD, arginine/ornithine
  antiporter; 2653, ArcR protein; 2658, arginine repressor. Fermentation: BudB, acetolactate
  synthase large subunit; BudA1, alpha-acetolactate decarboxylase; FabG1, short-chain
  dehydrogenase; FdhD, formate dehydrogenase accessory protein; Ldh1, l-lactate dehydrogenase;
  Ldh2, l-lactate dehydrogenase; PflA, formate acetyltransferase activating enzyme;
  PflB, formate acetyltransferase; SrrA, staphylococcal respiratory response protein
  SrrA (response regulator); SrrB, staphylococcal respiratory response protein SrrB
  (sensor kinase); 0093, lactate transporter; 0111, acetoinreductase; 0135, alcohol
  dehydrogenase/acetaldehyde dehydrogenase; 0162, NAD-dependent formate dehydrogenase;
  0660, alcohol dehydrogenase I; 2178, putative alcohol dehydrogenase (zinc containing);
  2301, formate dehydrogenase homolog; 2363, l-lactate permease; 2535, d-specific
  d-2-hydroxyacid dehydrogenase. Nitrate/nitrite respiration: NarG, respiratory nitrate
  reductase alpha chain; NarH, nitrate reductase beta chain; NarI, nitrate reductase
  gamma chain; NarJ, nitrate reductase delta chain; NirB, nitrite reductase; NirD,
  nitrite reductase [NAD(P)H] (small subunit); NirR, transcriptional regulator; 0220,
  flavohemoprotein Hmp; 0301, formate/nitrite transporter; 2386, nitrite extrusion
  protein; 2388, two-component sensor histidine kinase NreB; 2389, regulator of two-component
  regulatory NreC.'
papertitle: A Defect in Menadione Biosynthesis Induces Global Changes in Gene Expression
  in Staphylococcus aureus  .
reftext: Christian Kohler, et al. J Bacteriol. 2008 Oct;190(19):6351-6364.
year: '2008'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5819974
figid_alias: PMC2566006__F2a
figtype: Figure
redirect_from: /figures/PMC2566006__F2a
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2566006__zjb019088140002a.html
  '@type': Dataset
  description: 'Visualization of transcriptomic (DNA microarrays) and proteomic (2D
    gels) data for selected genes/proteins of S. aureus COL (wild type) compared to
    its isogenic menD mutant (ΔmenD) and for the hemB mutant (ΔhemB) compared to the
    menD mutant (ΔmenD) grown in TSB medium at the exponential growth phase and at
    early stationary phase. (A) Carbohydrate metabolism. (B) Amino acid metabolism.
    (C) Fermentation. (D) Nitrate/nitrite respiration. Induction ratios were given
    as logarithms to the base 2. Boxes ranging in color from light orange to dark
    orange indicate transcripts or proteins that were significantly downregulated
    in the menD mutant, and boxes ranging from light blue to dark blue indicate transcripts
    or proteins that were significantly upregulated in the menD mutant. White boxes
    indicate transcripts or proteins with no changes between the menD mutant and the
    wild type or the hemB mutant. Gray boxes represent proteins that were not identified
    on the gels. The IDs, symbols, and functions are based on TIGR annotation for
    S. aureus COL (http://www.tigr.org). Glycolysis: Eno, enolase; FbaA, fructose-bisphosphate
    aldolase; FdaB, fructose-bisphosphate aldolase (class I); FruK, 1-phosphofructokinase;
    GalM, aldose 1-epimerase; GapA1, glyceraldehyde-3-phosphate dehydrogenase; GapR,
    glycolytic operon regulator; Glk, glucokinase; IpdC, pyruvate decarboxylase; PfkA,
    6-phosphofructokinase; Pgi, glucose-6-phosphate isomerase A; Pgk, phosphoglycerate
    kinase; Pgm, phosphoglycerate mutase; Pyk, pyruvate kinase; TpiA, triosephosphate
    isomerase; 2624, acetyl coenzyme A (acetyl-CoA) synthetase. TCA cycle: AcnA, aconitate
    hydratase; FumC, fumarate hydratase; GltA, citrate synthase; Icd, isocitrate dehyrogenase;
    Mqo1, malate:quinone oxidoreductase; SdhA, succinate dehydrogenase flavoprotein
    subunit; SdhB, succinate dehydrogenase iron-sulfur protein subunit; SdhC, succinate
    dehydrogenase cytochrome b-558; SucA, 2-oxoglutarate dehydrogenase E1; SucB, dihydrolipoamide
    succinyltransferase; SucCD, succinyl-CoA synthetase beta/alpha chain; 1308/1309,
    2-oxoacid ferredoxin oxidoreductase, alpha/beta subunit; 1749, malate dehydrogenase
    homolog. Pyruvate metabolism: AckA, acetate kinase; PdhABCD, pyruvate dehydrogenase
    E1; Pta, phosphotransacetylase; 2553, putative pyruvate oxidase; 2574, putative
    d-lactate dehydrogenase. Gluconeogenesis: Acs, acetyl-CoA synthetase; GapA2, glyceraldehyde
    3-phosphate dehydrogenase 2; PckA, phosphoenolpyruvate carboxykinase; Pyc, pyruvate
    carboxylase; 2527, fructose-bisphosphatase. Pentose phosphate pathway: DeoC1,
    deoxyribose-phosphate aldolase; Gnd, phosphogluconate dehydrogenase; Tal, transaldolase;
    Zwf, glucose-6-phosphate 1-dehydrogenase. Sugar usage: CscA, sucrose-6-phosphate
    hydrolase; ManA2, mannose-6-phosphate isomerase; MtlD, mannitol-1-phosphate 5-dehydrogenase;
    0517, trehalose-6-phosphate hydrolase; 0598, l-ribulokinase; 0617, putative hexulose-6-phosphate
    synthase; 2028, putative fructokinase. Glycerol degradation: GlpD, glycerol-3-phosphate
    dehydrogenase; GlpF, glycerol uptake facilitator; GlpK, glycerol kinase; 1920,
    putative glycerate dehydrogenase. Acetoin usage: AcuA, acetoin utilization protein.
    Amino acid degradation: Ald1, alanine dehydrogenase; Ald2, alanine dehydrogenase;
    AldA1, aldehyde dehydrogenase; ArgB, N-acetylglutamate 5-phosphotransferase; Asd,
    aspartate semialdehyde dehydrogenase; DapA, dihydrodipicolinate synthase; Dat,
    d-alanine aminotransferase; GcvH, glycine cleavage system protein H homolog; GcvT,
    aminomethyltransferase; GluD, NAD-specific glutamate dehydrogenase; Hom, homoserine
    dehydrogenase; HutG, formiminoglutamase; HutI, imidazolonepropionase; HutU, urocanate
    hydratase; IlvA1, threonine deaminase IlvA homolog; ProC, pyrroline-5-carboxylate
    reductase; RocD2, ornithine aminotransferase; RocF, arginase; SdaAA, l-serine
    dehydratase; 1349, threonine aldolase; 1562, branched-chain alpha-keto acid dehydrogenase
    E1; 1561, branched-chain alpha-keto acid dehydrogenase E1; 1593, glycine dehydrogenase
    subunit 2 homolog; 1594, glycine dehydrogenase subunit 1; 2569, 1-pyrroline-5-carboxylate
    dehydrogenase. Urea degradation: UreA, UreB, UreC, UreD, UreE, UreG, and UreF,
    urease subunits and urease accessory proteins; 2279, urea transporter. Arginine
    deiminase pathway (ADI): ArcA, arginine deiminase; ArcB2, ornithine transcarbamoylase;
    ArcC2, carbamate kinase; ArcD, arginine/ornithine antiporter; 2653, ArcR protein;
    2658, arginine repressor. Fermentation: BudB, acetolactate synthase large subunit;
    BudA1, alpha-acetolactate decarboxylase; FabG1, short-chain dehydrogenase; FdhD,
    formate dehydrogenase accessory protein; Ldh1, l-lactate dehydrogenase; Ldh2,
    l-lactate dehydrogenase; PflA, formate acetyltransferase activating enzyme; PflB,
    formate acetyltransferase; SrrA, staphylococcal respiratory response protein SrrA
    (response regulator); SrrB, staphylococcal respiratory response protein SrrB (sensor
    kinase); 0093, lactate transporter; 0111, acetoinreductase; 0135, alcohol dehydrogenase/acetaldehyde
    dehydrogenase; 0162, NAD-dependent formate dehydrogenase; 0660, alcohol dehydrogenase
    I; 2178, putative alcohol dehydrogenase (zinc containing); 2301, formate dehydrogenase
    homolog; 2363, l-lactate permease; 2535, d-specific d-2-hydroxyacid dehydrogenase.
    Nitrate/nitrite respiration: NarG, respiratory nitrate reductase alpha chain;
    NarH, nitrate reductase beta chain; NarI, nitrate reductase gamma chain; NarJ,
    nitrate reductase delta chain; NirB, nitrite reductase; NirD, nitrite reductase
    [NAD(P)H] (small subunit); NirR, transcriptional regulator; 0220, flavohemoprotein
    Hmp; 0301, formate/nitrite transporter; 2386, nitrite extrusion protein; 2388,
    two-component sensor histidine kinase NreB; 2389, regulator of two-component regulatory
    NreC.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - vs
  - Pgk
  - Eno
  - Pdha
  - Pka-C1
  - Pka-R2
  - Pka-C3
  - Pka-R1
  - Pka-C2
  - Pa
  - PEK
  - Pdh
  - Pdf
  - Pdhb
  - G6P
  - pygo
  - Pgi
  - PyK
  - AcCoAS
  - Acsl
  - da
  - nAChRalpha2
  - Aldh-III
  - SdhB
  - SdhC
  - .na.character
  - lds
  - SdhA
  - jus
  - CG46339
  - gnd
  - Suc
  - su(N)
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - pra
  - prg
  - pe
  - ps
  - mid
  - Pen
  - peng
  - peo
  - Tpi
  - Pep
  - bgcn
  - aa
  - LKRSDH
  - Sodh-2
  - tht
  - Coa
  - Ada
  - AP-2alpha
  - Med
  - so
  - ana
  - pg
  - Ald1
  - pk
  - Galphai
  - mthl5
  - lad
  - ScsbetaG
  - tRNA:Thr-TGT-1-1
  - tRNA:Thr-TGT-2-1
  - tRNA:Thr-TGT-2-2
  - tRNA:Thr-TGT-2-5
  - tRNA:Thr-TGT-2-4
  - tRNA:Thr-TGT-2-3
  - pana
  - wg
  - gl
  - Zw
  - Datp
  - Gpdh1
  - dlp
  - Ilp6
  - Ilp7
  - Daxx
  - Ilp1
  - Ilp2
  - Ilp3
  - Ilp4
  - Ilp8
  - Dys
  - Ilp5
  - Cndp2
  - DptA
  - Itpr
  - Nplp1
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - PDHA1
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - EIF2AK3
  - MAN2A1
  - SLC25A3
  - PDP1
  - DLAT
  - DLD
  - PDHA2
  - PDHB
  - PDHX
  - GLS
  - PAM
  - MYCBP2
  - GALM
  - GAPT
  - MUC1
  - BGN
  - GPI
  - PGPEP1
  - PHKA2
  - F11
  - PTCRA
  - EGFLAM
  - PLA2G15
  - ACSS2
  - ACCS
  - CDA
  - GNPTAB
  - SDHB
  - SDHC
  - LOC642502
  - SDHA
  - TKT
  - DDR2
  - SUCO
  - TALDO1
  - LRSAM1
  - TAAR1
  - S100A6
  - PUM3
  - PCSK1N
  - TSPAN33
  - PDC
  - PNKD
  - TPI1
  - PAEP
  - PREP
  - PTPN22
  - SMIM10L2A
  - SARDH
  - SORD
  - SDS
  - ADA
  - ADAD1
  - ADAD2
  - ADAR
  - ADARB1
  - ADARB2
  - ADAT1
  - ADAT2
  - ADAT3
  - COL9A1
  - COL9A2
  - COL9A3
  - COMP
  - SCN8A
  - CCNC
  - CDK8
  - CDK19
  - MED1
  - MED29
  - MED27
  - MED4
  - MED24
  - MED6
  - MED7
  - MED8
  - MED9
  - MED10
  - MED11
  - MED12
  - MED12L
  - MED13
  - MED13L
  - MED14
  - MED15
  - MED16
  - MED17
  - MED18
  - MED19
  - MED20
  - MED21
  - MED22
  - MED23
  - MED25
  - MED26
  - MED28
  - MED30
  - MED31
  - PFKM
  - MAN2C1
  - FBXO3
  - PAND1
  - PACC1
  - THADA
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - ITGB2
  - GLA
  - NAT8
  - ACKR1
  - GYPC
  - TXNL4B
  - DIP
  - TSC22D3
  - PIAS2
  - GRAMD4
  - NCKIPSD
  - acetate
  - fumarate
  - glucose
  - isocitrate
  - pyruvate
  - succinate suconyl
---
