---
figid: PMC2904328__nihms214685f4
figtitle: Pathways that generate miRNA-class regulatory RNAs from short hairpins in
  Drosophila
organisms:
- Drosophila melanogaster
- Drosophila simulans
- Drosophila mojavensis
- Saccharomyces cerevisiae
- Oryza sativa
- Homo sapiens
- Mus musculus
- Gallus gallus
- NA
pmcid: PMC2904328
filename: nihms214685f4.jpg
figlink: /pmc/articles/PMC2904328/figure/F4/
number: F4
caption: Pathways that generate miRNA-class regulatory RNAs from short hairpins in
  Drosophila. In the canonical pathway, the RNase III enzyme Drosha “crops” a primary
  miRNA transcript to release the pre-miRNA. In the conventional mirtron pathway,
  the splicing complex liberates short hairpin introns from protein-coding genes.
  Following their linearization by lariat debranching enzyme (Ldbr), these can fold
  into pre-miRNA mimics. In the 3′ tailed mirtron pathway, a 3′ tail separates the
  hairpin from the 3′ splice acceptor site. These require Ldbr and the RNA exosome
  to remove the 3′ tail up to the hairpin, yielding the pre-miRNA. In all cases, pre-miRNA
  hairpins are cleaved by the cytoplasmic RNase III Dicer-1 to yield miRNA/miRNA*
  duplexes. One strand is predominantly selected for incorporation into an AGO1 complex
  to repress seed-matched targets. At the bottom are additional examples of D. melanogaster
  tailed mirtrons whose inferred trimmed tails are highlighted green (see also ).
  Note that the precise terminal loop of the CG18815_in4/mir-3642 tailed mirtron was
  also cloned (yellow), providing additional evidence for Dicer cleavage.
papertitle: microRNA biogenesis via splicing and exosome-mediated trimming in Drosophila.
reftext: Alex S. Flynt, et al. Mol Cell. ;38(6):900-907.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6026552
figid_alias: PMC2904328__F4
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC2904328__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2904328__nihms214685f4.html
  '@type': Dataset
  description: Pathways that generate miRNA-class regulatory RNAs from short hairpins
    in Drosophila. In the canonical pathway, the RNase III enzyme Drosha “crops” a
    primary miRNA transcript to release the pre-miRNA. In the conventional mirtron
    pathway, the splicing complex liberates short hairpin introns from protein-coding
    genes. Following their linearization by lariat debranching enzyme (Ldbr), these
    can fold into pre-miRNA mimics. In the 3′ tailed mirtron pathway, a 3′ tail separates
    the hairpin from the 3′ splice acceptor site. These require Ldbr and the RNA exosome
    to remove the 3′ tail up to the hairpin, yielding the pre-miRNA. In all cases,
    pre-miRNA hairpins are cleaved by the cytoplasmic RNase III Dicer-1 to yield miRNA/miRNA*
    duplexes. One strand is predominantly selected for incorporation into an AGO1
    complex to repress seed-matched targets. At the bottom are additional examples
    of D. melanogaster tailed mirtrons whose inferred trimmed tails are highlighted
    green (see also ). Note that the precise terminal loop of the CG18815_in4/mir-3642
    tailed mirtron was also cloned (yellow), providing additional evidence for Dicer
    cleavage.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ldbr
  - drosha
  - Dcr-2
  - Dcr-1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - ago
  - AGO1
  - AGO2
  - mir-3642
  - Apt1
  - mir-3644
  - Kpc1
---
