---
figid: PMC2773089__pp1511221f06
figtitle: Representation of selected metabolic pathways induced in –P-stressed nodules
organisms:
- Rhizobium tropici
- Rhizobium tropici CIAT 899
- Homo sapiens
- Phaseolus vulgaris
pmcid: PMC2773089
filename: pp1511221f06.jpg
figlink: /pmc/articles/PMC2773089/figure/fig6/
number: F6
caption: 'Representation of selected metabolic pathways induced in –P-stressed nodules.
  Drawings are based on PathExpress outputs (; ) and contain two types of nodes: metabolites
  represented by ellipses, and enzymes (boxes) labeled with the enzyme name or abbreviation
  and/or the EC number. Enzymes coded by genes included in the PhvGI are shown. The
  color code indicates the induced gene expression of enzymes (; Supplemental Table
  S1) or increased metabolite pools (; green) or respective decrease (red). Other
  metabolites shown in each pathway (white ellipses) were not detected in our analysis.
  A, Glycerolipid metabolism. ADH, Alcohol dehydrogenase; ALDH, aldehyde dehydrogenase;
  MAGL, monoglyceride lipase. B, Glycolysis-gluconeogenesis-carbon fixation. ENO,
  Enolase 2; FBPA, Fru-bisphosphate aldolase; G3PD, glyceraldehyde-3-phosphate dehydrogenase;
  MDH, malate dehydrogenase; 3-PGK, phosphoglycerate kinase; TPI, triosephosphate
  isomerase. C, Phe metabolism. CA4H, trans-Cinnamate 4-monooxygenase; MPO, myeloperoxidase;
  PAL, Phe ammonia lyase.'
papertitle: Global Changes in the Transcript and Metabolic Profiles during Symbiotic
  Nitrogen Fixation in Phosphorus-Stressed Common Bean Plants.
reftext: Georgina Hernández, et al. Plant Physiol. 2009 Nov;151(3):1221-1238.
year: '2009'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9070596
figid_alias: PMC2773089__F6
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC2773089__F6
ndex: 3cbb5c29-df0a-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2773089__pp1511221f06.html
  '@type': Dataset
  description: 'Representation of selected metabolic pathways induced in –P-stressed
    nodules. Drawings are based on PathExpress outputs (; ) and contain two types
    of nodes: metabolites represented by ellipses, and enzymes (boxes) labeled with
    the enzyme name or abbreviation and/or the EC number. Enzymes coded by genes included
    in the PhvGI are shown. The color code indicates the induced gene expression of
    enzymes (; Supplemental Table S1) or increased metabolite pools (; green) or respective
    decrease (red). Other metabolites shown in each pathway (white ellipses) were
    not detected in our analysis. A, Glycerolipid metabolism. ADH, Alcohol dehydrogenase;
    ALDH, aldehyde dehydrogenase; MAGL, monoglyceride lipase. B, Glycolysis-gluconeogenesis-carbon
    fixation. ENO, Enolase 2; FBPA, Fru-bisphosphate aldolase; G3PD, glyceraldehyde-3-phosphate
    dehydrogenase; MDH, malate dehydrogenase; 3-PGK, phosphoglycerate kinase; TPI,
    triosephosphate isomerase. C, Phe metabolism. CA4H, trans-Cinnamate 4-monooxygenase;
    MPO, myeloperoxidase; PAL, Phe ammonia lyase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - diacylglycerol
  - Fructose
  - Fructose-6P
  - Glucose
  - Glyceraldehyde
  - Glycerol
  - glycerol
  - Malate
  - Phenylalanine
  - Phosphoenolpyruvate
  - phenyl cetamide
  - Triacylglycerol
  - Triglyceride
---
