---
figid: PMC2882257__jexboterq089f03_ht
figtitle: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
  symbiosis
organisms:
- Arabidopsis thaliana
- Nicotiana tabacum
- Zea mays
- Glycine max
- Solanum lycopersicum
- Solanum tuberosum
- Hordeum vulgare
- Cucumis sativus
- Medicago truncatula
- Botrytis cinerea
- Tracheophyta
- Nicotiana attenuata
- Phytophthora parasitica
- Valeriana edulis
- Funneliformis mosseae
- Rhizophagus intraradices
- Phytophthora parasitica var. nicotianae
- Serendipita vermifera
- Pseudomonas syringae
- Leptospira kmetyi
pmcid: PMC2882257
filename: jexboterq089f03_ht.jpg
figlink: /pmc/articles/PMC2882257/figure/fig3/
number: F3
caption: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
  symbiosis. Metabolic scheme of the oxylipin pathway including the 9-LOX and 13-LOX
  branches (modified after ). Shaded boxes show the metabolites analysed by LC-MS/MS
  in the present study. Thick arrows indicate the direction of the changes in expression
  levels compared with non-mycorrhizal control roots (up- or down-regulated) of the
  genes coding for the enzyme cited, as determined by the transcriptomic analysis;
  = indicates no changes in gene expression. LOX, lipoxygenase; AOS, allene oxide
  synthase; AOC, allene oxide cyclase; OPR3, oxo-phytodienoic acid reductase; JMT,
  jasmonic acid carboxyl methyltransferase; JAME, methyl jasmonate esterase; JAR1,
  jasmonate-amino synthetase; HPL, hydroperoxide lyase; and DES, divinyl ether synthase.
papertitle: Hormonal and transcriptional profiles highlight common and differential
  host responses to arbuscular mycorrhizal fungi and the regulation of the oxylipin
  pathway.
reftext: Juan A. López-Ráez, et al. J Exp Bot. 2010 Jun;61(10):2589-2601.
year: '2010'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9283642
figid_alias: PMC2882257__F3
figtype: Figure
organisms_ner:
- Solanum lycopersicum
- Zea mays
redirect_from: /figures/PMC2882257__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2882257__jexboterq089f03_ht.html
  '@type': Dataset
  description: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
    symbiosis. Metabolic scheme of the oxylipin pathway including the 9-LOX and 13-LOX
    branches (modified after ). Shaded boxes show the metabolites analysed by LC-MS/MS
    in the present study. Thick arrows indicate the direction of the changes in expression
    levels compared with non-mycorrhizal control roots (up- or down-regulated) of
    the genes coding for the enzyme cited, as determined by the transcriptomic analysis;
    = indicates no changes in gene expression. LOX, lipoxygenase; AOS, allene oxide
    synthase; AOC, allene oxide cyclase; OPR3, oxo-phytodienoic acid reductase; JMT,
    jasmonic acid carboxyl methyltransferase; JAME, methyl jasmonate esterase; JAR1,
    jasmonate-amino synthetase; HPL, hydroperoxide lyase; and DES, divinyl ether synthase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LOX1.1
  - loxD
  - DES
  - HPL
  - AOS2
  - AOS3
  - opr3
  - LOC542163
  - aos2
  - aos3
  - Divinyl ether
  - colnelenic acid
  - (3Z)-aldehydes
  - oxo-fatty acids
  - linolenic acid
---
