---
figid: PMC3180702__1471-2164-12-441-6
figtitle: Transcriptional complexity of the mesenchymal-epithelial transition network
organisms:
- Danio rerio
- Xenopus laevis
- Oryzias latipes
- Mus musculus
- Rattus norvegicus
- Homo sapiens
pmcid: PMC3180702
filename: 1471-2164-12-441-6.jpg
figlink: /pmc/articles/PMC3180702/figure/F6/
number: F6
caption: Transcriptional complexity of the mesenchymal-epithelial transition network.
  Transcriptional complexity associated with the 17 most characterized mesenchymal-epithelial
  transition pathway (MET) genes. Genes that have evidence of alternative splicing
  include alternate exon usage, alternate 5' and 3' exons highlighted with black circle.
  Genes with long 5' and/or 3' UTR signal are represented by white circles and antisense
  transcript in blue circles. Literature evidence of microRNA association is represented
  for Lhx1 (miR-30) and Hoxa11 (miR-181) along with other known transcriptional regulatory
  relationship (dotted arrows). Figure modified from Little et al [].
papertitle: Refining transcriptional programs in kidney development by integration
  of deep RNA-sequencing and array-based spatial profiling.
reftext: Rathi D Thiagarajan, et al. BMC Genomics. 2011;12:441-441.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.94962
figid_alias: PMC3180702__F6
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Danio rerio
- Oryzias latipes
redirect_from: /figures/PMC3180702__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3180702__1471-2164-12-441-6.html
  '@type': Dataset
  description: Transcriptional complexity of the mesenchymal-epithelial transition
    network. Transcriptional complexity associated with the 17 most characterized
    mesenchymal-epithelial transition pathway (MET) genes. Genes that have evidence
    of alternative splicing include alternate exon usage, alternate 5' and 3' exons
    highlighted with black circle. Genes with long 5' and/or 3' UTR signal are represented
    by white circles and antisense transcript in blue circles. Literature evidence
    of microRNA association is represented for Lhx1 (miR-30) and Hoxa11 (miR-181)
    along with other known transcriptional regulatory relationship (dotted arrows).
    Figure modified from Little et al [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Cm
  - Eya1
  - Pax2
  - Osr1
  - Oxsr1
  - Fgf2
  - Mir181a-2
  - Emx2
  - Fgf8
  - Lhx1
  - EYA1
  - PAX2
  - OSR1
  - OXSR1
  - OSER1
  - FGF2
  - FGF13
  - EMX2
  - FGF8
  - LHX1
  - ATOH7
  - Oser1
  - eya1
  - pax2a
  - osr1
  - fgf2
  - emx2
  - fgf8a
  - lhx1a
  - pax2b
  - mir18-1
---
