---
figid: PMC3901260__metabolites-03-00168-g003
figtitle: Overview of the CO2 fixation pathway and predicted enzymes for mannitol
  metabolism in E
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3901260
filename: metabolites-03-00168-g003.jpg
figlink: /pmc/articles/PMC3901260/figure/metabolites-03-00168-f003/
number: F3
caption: Overview of the CO2 fixation pathway and predicted enzymes for mannitol metabolism
  in E. huxleyi. Regular and bold italic indicate metabolites and enzymes or metabolic
  pathways, respectively. Metabolites in red were detected in the current study. Other
  metabolites except for pyruvate and mannose are hardly detected by GC-MS analysis.
  The proteins showing significant homology to known enzymes are listed close to the
  enzyme name. Protein IDs are taken from JGI Emiliania huxleyi CCMP1516 main genome
  assembly v1.0. Solid and dashed arrows show single and multiple reactions, respectively.
  The reactions colored by red, blue and purple are related to manntiol biosynthesis,
  catabolism and both pathways, respectively. Ru5P, riburose-5-phosphate; RuBP, riburose
  bisphosphate; 3PGA, 3-phosphoglycerate; DPG, 1,3-diphosphoglycerate; GAP, glyceraldehyd-3-phosphate;
  DHAP, dihydroxyacetone phosphate; F16BP, fructose-1,6-bisphosphate; F6P, fructose-6-phosphate;
  Pyr, pyruvate; OAA, oxaloacetate; BCAA, branched chain amino acid; PYC, pyruvate
  carboxylase; PMI, phosphomannose isomerase; PMred, phosphomannose reductase; M1PDH,
  mannitol-1-phosphate dehydrogenase; M1Pase, mannose-1-phosphatate; MDH, mannitol
  dehydrogenase; Hxk, hexokinase.
papertitle: Gas-Chromatography Mass-Spectrometry (GC-MS) Based Metabolite Profiling
  Reveals Mannitol as a Major Storage Carbohydrate in the Coccolithophorid Alga Emiliania
  huxleyi..
reftext: Toshihiro Obata, et al. Metabolites. 2013 Mar;3(1):168-184.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8991355
figid_alias: PMC3901260__F3
figtype: Figure
redirect_from: /figures/PMC3901260__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3901260__metabolites-03-00168-g003.html
  '@type': Dataset
  description: Overview of the CO2 fixation pathway and predicted enzymes for mannitol
    metabolism in E. huxleyi. Regular and bold italic indicate metabolites and enzymes
    or metabolic pathways, respectively. Metabolites in red were detected in the current
    study. Other metabolites except for pyruvate and mannose are hardly detected by
    GC-MS analysis. The proteins showing significant homology to known enzymes are
    listed close to the enzyme name. Protein IDs are taken from JGI Emiliania huxleyi
    CCMP1516 main genome assembly v1.0. Solid and dashed arrows show single and multiple
    reactions, respectively. The reactions colored by red, blue and purple are related
    to manntiol biosynthesis, catabolism and both pathways, respectively. Ru5P, riburose-5-phosphate;
    RuBP, riburose bisphosphate; 3PGA, 3-phosphoglycerate; DPG, 1,3-diphosphoglycerate;
    GAP, glyceraldehyd-3-phosphate; DHAP, dihydroxyacetone phosphate; F16BP, fructose-1,6-bisphosphate;
    F6P, fructose-6-phosphate; Pyr, pyruvate; OAA, oxaloacetate; BCAA, branched chain
    amino acid; PYC, pyruvate carboxylase; PMI, phosphomannose isomerase; PMred, phosphomannose
    reductase; M1PDH, mannitol-1-phosphate dehydrogenase; M1Pase, mannose-1-phosphatate;
    MDH, mannitol dehydrogenase; Hxk, hexokinase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pyr
  - EcR
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Aldh-III
  - CycE
  - cyc
  - Pmi
  - Tm1
  - pi
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - pl
  - ARID4B
  - RASA1
  - RGS6
  - MPI
  - PMM2
  - TMEM11
  - SNIP1
  - MDH2
  - Asp
  - Leu
  - Mannitol
  - Mannose
  - Ser
  - Val
---
