---
figid: PMC3266662__1471-2164-12-616-3
figtitle: Correlations of genes in central carbon metabolism with SPPR
organisms:
- Saccharomyces cerevisiae
- Diaporthe sclerotioides
- PX clade
- Schizosaccharomyces pombe
- Aspergillus fumigatus
- Agaricus bisporus
- Aspergillus nidulans
- Aspergillus niger
- Yarrowia lipolytica
- Aspergillus oryzae
- Pyricularia oryzae
- Fusarium graminearum
- Trichoderma reesei
- Aspergillus parasiticus
- Trichoderma harzianum
- Trichoderma atroviride
- Fusarium graminearum PH-1
- Coprinopsis cinerea
- Trichoderma reesei RUT C-30
- Terfezia morenoi
- Fusarium venenatum
- Trichoderma reesei QM9414
- Mycobacterium tuberculosis
- Bacillus velezensis At1
- Planktothrix pseudagardhii OF3
- Homo sapiens
- Mus musculus
- Plasmodium falciparum
- NA
organisms_ner:
- Mus musculus
- Homo sapiens
- Schizosaccharomyces pombe
- Saccharomyces cerevisiae S288C
pmcid: PMC3266662
filename: 1471-2164-12-616-3.jpg
figlink: /pmc/articles/PMC3266662/figure/F3/
number: F3
caption: Correlations of genes in central carbon metabolism with SPPR. Metabolic network
  of metabolites and genes (rectangles). Genes are colored according to correlation
  to SPPR. Gene names are as in S. cerevisiae, except that paralogs not found in yeast
  are distinguished by additional letters (a,b,..) and genes corresponding to many
  genes in S. cerevisiae have names extended with '-'. T. reesei gene names have been
  used when available (hxk1, glk1 [], lxr1 [], xdh1 [], xyl1 [], lad1 [], gld1, gfd1
  [], gal10, gal7 [], xdh1 [] ) and others when T. reesei nor S. cerevisiae name exists
  (acl1, acl2 []). Cellular locations are merely indicative. Genes detected by pathway
  detection are specified with a red dot.
papertitle: Correlation of gene expression and protein production rate - a system
  wide study.
reftext: Mikko Arvas, et al. BMC Genomics. 2011;12:616-616.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9186913
figid_alias: PMC3266662__F3
figtype: Figure
redirect_from: /figures/PMC3266662__F3
ndex: 2a613b4a-df0f-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3266662__1471-2164-12-616-3.html
  '@type': Dataset
  description: Correlations of genes in central carbon metabolism with SPPR. Metabolic
    network of metabolites and genes (rectangles). Genes are colored according to
    correlation to SPPR. Gene names are as in S. cerevisiae, except that paralogs
    not found in yeast are distinguished by additional letters (a,b,..) and genes
    corresponding to many genes in S. cerevisiae have names extended with '-'. T.
    reesei gene names have been used when available (hxk1, glk1 [], lxr1 [], xdh1
    [], xyl1 [], lad1 [], gld1, gfd1 [], gal10, gal7 [], xdh1 [] ) and others when
    T. reesei nor S. cerevisiae name exists (acl1, acl2 []). Cellular locations are
    merely indicative. Genes detected by pathway detection are specified with a red
    dot.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Decr1
  - Ctsj
  - Atp13a1
  - Rpe
  - Bgn
  - Gpi1
  - Vcan
  - Gip
  - Ctdsp1
  - Tal1
  - Fbxo3
  - Rap1gap
  - Rasa1
  - act
  - Fhl5
  - Acot7
  - ac
  - Cit
  - Dhdds
  - Pgk1
  - Lat
  - Prodh2
  - Eno1
  - Eno1b
  - Eno2
  - En1
  - En2
  - Atp5f1a
  - Mom6
  - Mom3
  - Mal
  - Tirap
  - Mrtfa
  - sc
  - Mtv7
  - Mtv13
  - Samsn1
  - Apoc4
  - UGP2
  - ATP8A2
  - DECR1
  - RPE
  - BGN
  - GPI
  - PGPEP1
  - VCAN
  - SMIM10L2A
  - GIP
  - GNAI2
  - C1QTNF1
  - MTG1
  - PFKM
  - PFKFB3
  - TAL1
  - FBXO3
  - RASA1
  - RGS6
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - KLK15
  - CIT
  - DHDDS
  - PGK1
  - REEP5
  - PDA1
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - LAT
  - ORC3
  - SPNS1
  - ENO1
  - ENO2
  - EN1
  - EN2
  - ATP5F1A
  - MAL
  - MRTFA
  - TIRAP
  - SAMSN1
  - SASH3
  - ACLY
  - pfk1
  - tal1
  - dak2
  - pgk1
  - oac1
  - pda1
  - adh1
  - eno101
  - eno102
  - GND1
  - PSA1
  - PFK1
  - PFK2
  - DAK2
  - OAC1
  - POX1
  - GPM2
  - GID7
  - MLS1
  - DAL7
  - Acetate
  - Galactitol
  - Fructose
  - Glycerol
  - Glyoxylate
  - GDP mannose
  - Pentose phosphate
  - Pyruvate
  - Sorbitol
  - Trehalose
  - UDP
  - Xylitol
---
