---
figid: PMC3971608__2049-2618-1-22-4
figtitle: Predicted membrane structure and proton motive force mechanisms from the
  three reconstructed Chloroflexi genomes
organisms:
- Pyrococcus furiosus
- phototrophic bacterium
- Chloroflexus aurantiacus
- Herpetosiphon aurantiacus
- Sulfurisphaera tokodaii str. 7
- Dehalococcoides mccartyi
- Dehalococcoides mccartyi CBDB1
- Cupriavidus metallidurans CH34
- Pelobacter acidigallici
- Thermomicrobium roseum
- Anaerolinea thermophila
- Caldilinea aerophila
- Syntrophotalea acetylenica
- Ktedonobacter racemifer
- Thermobaculum terrenum ATCC BAA-798
- Dehalogenimonas lykanthroporepellens
- Dehalogenimonas lykanthroporepellens BL-DC-9
- Desulfobacula toluolica Tol2
- Desulfitobacterium sp. PCE1
- Thermobaculum terrenum
- Sphaerobacter thermophilus
- Anaerolinea thermolimosa
- Leptolinea tardivitalis
- Levilinea saccharolytica
- Thermosporothrix hazakensis
- Anaerolinea thermophila UNI-1
- metagenome
- human metagenome
- metagenomes
- environmental samples
- uncultured bacterium
- uncultured Anaerolinea sp.
- Homo sapiens
- Mustela lutreola
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC3971608
filename: 2049-2618-1-22-4.jpg
figlink: /pmc/articles/PMC3971608/figure/F4/
number: F4
caption: 'Predicted membrane structure and proton motive force mechanisms from the
  three reconstructed Chloroflexi genomes. All three Chloroflexi are predicted to
  be Gram-negative, single membrane bound cells. RBG-9 is the only genome containing
  a predicted peptidoglycan pathway. Abbreviations: A0A1 - archaeal-type ATP synthase,
  Act - alternative complex III (ActABCDE), CM - cell membrane, Cu NIR - copper nitrite
  reductase, CYT - cytoplasm, CytC - cytochrome c, F1F0 - bacterial-type ATP synthase,
  Frd - succinate dehydrogenase/fumarate reductase (subunits ABC), hppA - membrane-bound
  proton-translocating pyrophosphatase, Hup - uptake hydrogenase, Hyd - Ni, Fe-hydrogenase,
  Mbh - membrane bound hydrogenase.'
papertitle: Community genomic analyses constrain the distribution of metabolic traits
  across the Chloroflexi phylum and indicate roles in sediment carbon cycling.
reftext: Laura A Hug, et al. Microbiome. 2013;1:22-22.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8710391
figid_alias: PMC3971608__F4
figtype: Figure
redirect_from: /figures/PMC3971608__F4
ndex: 7197ef6f-de94-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3971608__2049-2618-1-22-4.html
  '@type': Dataset
  description: 'Predicted membrane structure and proton motive force mechanisms from
    the three reconstructed Chloroflexi genomes. All three Chloroflexi are predicted
    to be Gram-negative, single membrane bound cells. RBG-9 is the only genome containing
    a predicted peptidoglycan pathway. Abbreviations: A0A1 - archaeal-type ATP synthase,
    Act - alternative complex III (ActABCDE), CM - cell membrane, Cu NIR - copper
    nitrite reductase, CYT - cytoplasm, CytC - cytochrome c, F1F0 - bacterial-type
    ATP synthase, Frd - succinate dehydrogenase/fumarate reductase (subunits ABC),
    hppA - membrane-bound proton-translocating pyrophosphatase, Hup - uptake hydrogenase,
    Hyd - Ni, Fe-hydrogenase, Mbh - membrane bound hydrogenase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ALPL
  - PAX7
  - AN
  - FER
  - ATP8A2
  - WDTC1
  - B3GNT2
  - B3GNTL1
  - ADRB2
  - NR1H4
  - BFAR
  - UBR5
  - NOC2L
  - IZUMO1R
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - an
  - dan
  - ATPsynbeta
  - Atpalpha
  - adp
  - egh
  - LanB1
  - betaTub56D
  - mys
  - Prosbeta1
  - B-H2
  - B-H1
  - hh
  - .na.character
  - fla
  - hyd
  - cu
  - Cyt-c-d
  - Cyt-c-p
  - Frd
  - act
  - Act5C
  - Act42A
  - Actbeta
  - RFeSP
  - ac
---
