---
figid: PMC3977653__nihms-567958-f0001
figtitle: 'At Loose Ends: Resecting a Double-Strand Break'
organisms:
- Saccharomyces cerevisiae
- Schizosaccharomyces pombe
- Homo sapiens
- Escherichia coli
pmcid: PMC3977653
filename: nihms-567958-f0001.jpg
figlink: /pmc/articles/PMC3977653/figure/F1/
number: F1
caption: 'Recent studies characterize a two-step mechanism for the processing of double-strand
  breaks (DSBs) at the 5′ ends to expose the 3′ single-stranded DNA (ssDNA) overhangs.
  Depending upon the cell-cycle phase and the type of DNA lesion, a DSB is processed
  by either the nonhomologous end joining pathway or the homologous recombination
  pathway. The names of the human homologs of the yeast proteins depicted are indicated
  in magenta. Following a DSB, the MRX/MRN complex is loaded onto the DNA ends at
  the break site. If cells are in the G1 phase of the cell cycle, nonhomologous end
  joining is used to repair the break. If the cells are in S/G2 phases, phosphorylation
  of Sae2/CtIP by cyclin-dependent kinases (CDKs) favors homologous recombination-mediated
  repair. During the first step of the homologous recombination repair pathway, the
  initial resection of the DSB is promoted by MRX/MRN and Sae2/CtIP, resulting in
  50–100 nucleotide ssDNA 3′ overhangs. In the second step, these fragments can serve
  as templates for long-range DNA end resection. This processive reaction can occur
  by two independent mechanisms: one that utilizes Sgs1/BLM and Dna2 (left) and the
  other using Exo1/hEXO1 (right). Following resection, the exposed ssDNA is coated
  by replication protein A (RPA), which recruits the Rad52 epistasis group of proteins
  (Rad52, Rad55, Rad57, Rad59, Rad54, Rdh54) to enable Rad51 filament formation. The
  DNA ends can be repaired using different mechanisms, such as single-strand annealing,
  break-induced replication, or gene conversion. DSB repair using the homologous chromosome
  is depicted here. After the search for homology, a joint DNA structure is formed.
  The resulting double Holliday junctions are then resolved and the lost bases are
  restored at the break site. The resolution of the junctions by the Sgs1-Top3-Rmi1/BLM-TOP3α-RMI1
  (BLAP75) complex depicted here leads to a non-crossover product of DSB repair by
  homologous recombination.'
papertitle: 'At Loose Ends: Resecting a Double-Strand Break.'
reftext: Kara A. Bernstein, et al. Cell. ;137(5):807-810.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9069553
figid_alias: PMC3977653__F1
figtype: Figure
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
redirect_from: /figures/PMC3977653__F1
ndex: 309ecf51-de9c-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3977653__nihms-567958-f0001.html
  '@type': Dataset
  description: 'Recent studies characterize a two-step mechanism for the processing
    of double-strand breaks (DSBs) at the 5′ ends to expose the 3′ single-stranded
    DNA (ssDNA) overhangs. Depending upon the cell-cycle phase and the type of DNA
    lesion, a DSB is processed by either the nonhomologous end joining pathway or
    the homologous recombination pathway. The names of the human homologs of the yeast
    proteins depicted are indicated in magenta. Following a DSB, the MRX/MRN complex
    is loaded onto the DNA ends at the break site. If cells are in the G1 phase of
    the cell cycle, nonhomologous end joining is used to repair the break. If the
    cells are in S/G2 phases, phosphorylation of Sae2/CtIP by cyclin-dependent kinases
    (CDKs) favors homologous recombination-mediated repair. During the first step
    of the homologous recombination repair pathway, the initial resection of the DSB
    is promoted by MRX/MRN and Sae2/CtIP, resulting in 50–100 nucleotide ssDNA 3′
    overhangs. In the second step, these fragments can serve as templates for long-range
    DNA end resection. This processive reaction can occur by two independent mechanisms:
    one that utilizes Sgs1/BLM and Dna2 (left) and the other using Exo1/hEXO1 (right).
    Following resection, the exposed ssDNA is coated by replication protein A (RPA),
    which recruits the Rad52 epistasis group of proteins (Rad52, Rad55, Rad57, Rad59,
    Rad54, Rdh54) to enable Rad51 filament formation. The DNA ends can be repaired
    using different mechanisms, such as single-strand annealing, break-induced replication,
    or gene conversion. DSB repair using the homologous chromosome is depicted here.
    After the search for homology, a joint DNA structure is formed. The resulting
    double Holliday junctions are then resolved and the lost bases are restored at
    the break site. The resolution of the junctions by the Sgs1-Top3-Rmi1/BLM-TOP3α-RMI1
    (BLAP75) complex depicted here leads to a non-crossover product of DSB repair
    by homologous recombination.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SAE2
  - EXO1
  - SGS1
  - DNA2
  - RAD51
  - DLG3
  - ABCG2
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - WARS1
  - CDKL1
  - CDKL2
  - CDKL3
  - CDKL4
  - CDKL5
  - CDK3
  - CDK4
  - CDK5
  - CDK6
  - CDK7
  - CDK8
  - CDK9
  - CDK10
  - CDK11A
  - CDK11B
  - CDK12
  - CDK13
  - CDK14
  - CDK15
  - CDK16
  - CDK17
  - CDK18
  - CDK19
  - CDK20
  - CDK1
  - CDK2
  - RBBP8
  - UBA2
  - RPA1
  - RPA2
  - RPA3
---
