---
figid: PMC3721115__ismej201345f3
figtitle: Energy metabolism pathways and levels of expression in Gamma- and ɛ-dominated
  metatranscriptomes
organisms:
- Escherichia coli
- Aliivibrio fischeri
- Buchnera aphidicola
- Sulfurimonas denitrificans
- endosymbiont of Riftia pachyptila
- Sulfurimonas autotrophica
- Sulfurimonas paralvinellae
- Sulfurimonas autotrophica DSM 16294
- Riftia pachyptila
- Olavius algarvensis
- Tevnia jerichonana
- Calyptogena magnifica
- Alviniconcha hessleri
- Lucinoma aequizonata
- Homo sapiens
- Bos taurus
- metagenomes
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC3721115
filename: ismej201345f3.jpg
figlink: /pmc/articles/PMC3721115/figure/fig3/
number: F3
caption: 'Energy metabolism pathways and levels of expression in γ- and ɛ-dominated
  metatranscriptomes. (a) Relative abundance of genes involved in sulfur oxidation,
  hydrogen oxidation, nitrogen reduction and assimilation, and aerobic respiration.
  Circle area reflects total normalized expression for each gene category for γ- or
  ɛ-dominated metatranscriptomes, respectively. Circles are divided according to relative
  contribution of each individual sample. ɛ-dominated metatranscriptomes are shaded
  yellow, while γ-dominated metatranscriptomes are shaded blue. (b) Energy metabolism
  models for γ- and ɛ-proteobacterial symbionts represented in a stylized cell. The
  upper, yellow half shows the model for ɛ-symbionts, while the blue lower half shows
  the model for γ-symbionts. Proteins and complexes are colored by metabolic category:
  yellow=sulfur metabolism; white=aerobic respiration; green=nitrogen metabolism;
  blue=hydrogen oxidation. Arrows show general direction of electron flux. Sox, Sox
  multienzyme complex; rDsr, reverse dissimilatory sulfur reduction pathway; Apr,
  adenylylsulfate reductase; Sat, sulfur adenylyltransferase; Sdh, sulfite dehydrogenase;
  Sqr, sulfide quinone (oxido)reductase; Psr, polysulfide reductase; Hyd, hydrogenase;
  Nap, periplasmic nitrate reductase; NarK, nitrate/nitrite transporter; NirA, ferredoxin-dependent
  nitrite reductase; NirBD, NADH-dependent siroheme nitrite reductase; NirS, membrane-bound
  respiratory nitrite reductase; Nor, nitric oxide reductase; Nos, nitrous oxide reductase;
  GS+GOGAT, glutamine synthetase+glutamate synthase; Amm. Trans., ammonium transporter;
  Q, quinone; b/c1, cytochrome bc1; c, cytochrome c; CytCO, cytochrome c oxidase.'
papertitle: Metatranscriptomics reveal differences in in situ energy and nitrogen
  metabolism among hydrothermal vent snail symbionts.
reftext: J G Sanders, et al. ISME J. 2013 Aug;7(8):1556-1567.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6023293
figid_alias: PMC3721115__F3
figtype: Figure
redirect_from: /figures/PMC3721115__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3721115__ismej201345f3.html
  '@type': Dataset
  description: 'Energy metabolism pathways and levels of expression in γ- and ɛ-dominated
    metatranscriptomes. (a) Relative abundance of genes involved in sulfur oxidation,
    hydrogen oxidation, nitrogen reduction and assimilation, and aerobic respiration.
    Circle area reflects total normalized expression for each gene category for γ-
    or ɛ-dominated metatranscriptomes, respectively. Circles are divided according
    to relative contribution of each individual sample. ɛ-dominated metatranscriptomes
    are shaded yellow, while γ-dominated metatranscriptomes are shaded blue. (b) Energy
    metabolism models for γ- and ɛ-proteobacterial symbionts represented in a stylized
    cell. The upper, yellow half shows the model for ɛ-symbionts, while the blue lower
    half shows the model for γ-symbionts. Proteins and complexes are colored by metabolic
    category: yellow=sulfur metabolism; white=aerobic respiration; green=nitrogen
    metabolism; blue=hydrogen oxidation. Arrows show general direction of electron
    flux. Sox, Sox multienzyme complex; rDsr, reverse dissimilatory sulfur reduction
    pathway; Apr, adenylylsulfate reductase; Sat, sulfur adenylyltransferase; Sdh,
    sulfite dehydrogenase; Sqr, sulfide quinone (oxido)reductase; Psr, polysulfide
    reductase; Hyd, hydrogenase; Nap, periplasmic nitrate reductase; NarK, nitrate/nitrite
    transporter; NirA, ferredoxin-dependent nitrite reductase; NirBD, NADH-dependent
    siroheme nitrite reductase; NirS, membrane-bound respiratory nitrite reductase;
    Nor, nitric oxide reductase; Nos, nitrous oxide reductase; GS+GOGAT, glutamine
    synthetase+glutamate synthase; Amm. Trans., ammonium transporter; Q, quinone;
    b/c1, cytochrome bc1; c, cytochrome c; CytCO, cytochrome c oxidase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CALM1
  - CALM2
  - CALM3
  - KRIT1
  - CAMKMT
  - SON
  - CTNNBL1
  - SQOR
  - LRP1
  - PMAIP1
  - SARDH
  - SRL
  - NR1I2
  - SAT1
  - KLK3
  - SH2B2
  - NBL1
  - C1QA
  - C1QB
  - C1QC
  - C1R
  - C1S
  - JMJD6
  - NOC2L
  - dsr
  - Cam
  - CaMKII
  - tow
  - Zw10
  - 'On'
  - Son
  - jigr1
  - mle
  - Hem
  - Atpalpha
  - BC1
  - Orc1
  - fru
  - Aps
  - so
  - cl
  - ci
  - Nitrate
  - Nitrite
---
