---
figid: PMC3234268__pone.0028319.g005
figtitle: 'Genome Characterization of the Oleaginous Fungus Mortierella alpina '
organisms:
- Mortierella alpina
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Solanum tuberosum
- Candida albicans
- Schizosaccharomyces pombe
- Aspergillus fumigatus
- Magnoliopsida
- Aspergillus nidulans
- Botrytis cinerea
- Cryptococcus neoformans
- Neurospora crassa
- Yarrowia lipolytica
- Saccharomyces cerevisiae S288C
- Fusarium oxysporum
- Aspergillus oryzae
- Fusarium graminearum
- Ustilago maydis
- Pyricularia grisea
- Fusarium verticillioides
- Sclerotinia sclerotiorum
- Rhizopus arrhizus
- Scheffersomyces stipitis
- Fusarium solani
- Mucor circinelloides
- Eremothecium gossypii
- Trichoderma atroviride
- Fusarium graminearum PH-1
- Coprinopsis cinerea
- Aspergillus fumigatus Af293
- Aspergillus oryzae RIB40
- Laccaria bicolor
- Candida albicans SC5314
- Schizosaccharomyces pombe 972h-
- Aspergillus niger ATCC 1015
- Aspergillus niger CBS 513.88
- Candida albicans 12C
- Aspergillus nidulans FGSC A4
- Aspergillus fischeri NRRL 181
- Yarrowia lipolytica CLIB122
- Nakaseomyces glabratus CBS 138
- Aspergillus clavatus NRRL 1
- Botrytis cinerea B05.10
- Neurospora crassa OR74A
- Trichoderma reesei QM6a
- Aspergillus fumigatus A1163
- Batrachochytrium dendrobatidis
- Ustilago maydis 521
- Phanerochaete chrysosporium RP-78
- Eremothecium gossypii ATCC 10895
- Mortierella alpina ATCC 32222
- Kluyveromyces lactis NRRL Y-1140
- Scheffersomyces stipitis CBS 6054
- Omphalotus olearius
- Pyricularia oryzae 70-15
- Debaryomyces hansenii CBS767
- Fusarium verticillioides 7600
- Laccaria bicolor S238N-H82
- Schizosaccharomyces octosporus
- Podospora anserina S mat
- Phytophthora infestans T30-4
- Trichoderma virens Gv29-8
- Malassezia globosa CBS 7966
- Fusarium oxysporum f. sp. conglutinans Fo5176
- Sclerotinia sclerotiorum 1980 UF-70
- Botrytis cinerea T4
- Cryptococcus neoformans var. neoformans B-3501A
- Schizosaccharomyces japonicus yFS275
- Vanderwaltozyma polyspora DSM 70294
- Schizosaccharomyces cryophilus OY26
- Batrachochytrium dendrobatidis JAM81
- Podila verticillata
- Homo sapiens
- Rattus norvegicus
- Agrobacterium tumefaciens
- NA
- NA
pmcid: PMC3234268
filename: pone.0028319.g005.jpg
figlink: /pmc/articles/PMC3234268/figure/pone-0028319-g005/
number: F5
caption: The glycolysis, pentose phosphate pathway, fatty acid synthesis, tricarboxylic
  acid cycle, malate/pyruvate cycle, sterol, glycerophospholipid, sphingolipid and
  glycerolipid synthesis pathways are outlined. Lipids detected by lipidomics are
  highlighted in yellow. Enzyme names are indicated and followed in parentheses by
  the number of genes encoding them. Enzymes conventionally thought to occur in a
  given pathway but for which homologs were not found in M. alpina are followed by
  a question mark. Hexokinase (HK, EC2.7.1.1), glucose-6-phosphate 1-dehydrogenase
  (G6PD, EC1.1.1.49), 6-phosphogluconolactonase (PGLS, EC3.1.1.31), phosphogluconate
  dehydrogenase (PGD, EC1.1.1.44), ribose 5-phosphate isomerase (RPIA, EC5.3.1.6),
  ribulose-5-phosphate-3-epimerase (RPE, EC5.1.3.1), transketolase (TKT, EC2.2.1.1),
  transaldolase (TALDO, EC2.2.1.2), glucose-6-phosphate isomerase (GPI, EC5.3.1.9),
  phosphofructokinase (PFK, EC2.7.1.11), fructose-bisphosphatase (FBP, EC3.1.3.11),
  aldolase A fructose-bisphosphate (ALDOA, EC4.1.2.13), glyceraldehyde 3-phosphate
  dehydrogenase (GAPDH, EC1.2.1.12), phosphoglycerate kinase (PGK, EC2.7.2.3), triose-phosphate
  isomerase (TPI, EC5.3.1.1), phosphoglycerate mutase (PGAM, EC5.4.2.1), enolase (ENO,
  EC4.2.1.11), pyruvate kinase (PK, EC2.7.1.40), lactate dehydrogenase (LDH, EC1.1.1.27),
  phosphoenolpyruvate carboxykinase (PCK, EC4.1.1.32 and EC4.1.1.49), acetyl-CoA carboxylase
  (ACC, EC6.4.1.2), malonyl-CoA decarboxylase (MLYCD, EC4.1.1.9), fatty acid synthase
  (FASN, EC2.3.1.86), acyl-CoA synthetase (ACSL, EC6.2.1.3), acyl-CoA thioesterase
  (ACOT, EC3.1.2.27), fatty acid elongase (ELOVL, EC2.3.1.-), fatty acid delta 9 desaturase
  (FADS9, EC1.14.19.1), fatty acid delta 12 desaturase (FADS12, EC1.14.19.6), fatty
  acid delta 15 desaturase (FADS15, EC1.14.19.-), fatty acid delta 6 desaturase (FADS6,
  EC1.14.19.3), fatty acid delta 5 desaturase (FADS5, EC1.14.19.-), malate dehydrogenase
  (MDH, EC1.1.1.37), malic enzyme (ME, EC1.1.1.40), pyruvate carboxylase (PC, EC6.4.1.1),
  ATP-citrate lyase (ACLY, EC2.3.3.8), pyruvate dehydrogenase (PDH, EC1.2.4.1), dihydrolipoamide
  acetyltransferase (DLAT, EC2.3.1.12), citrate synthase (CS, EC2.3.3.1), aconitase
  (ACO, EC4.2.1.3), isocitrate dehydrogenase (IDH, EC1.1.1.41 and EC1.1.1.42), glutamate
  dehydrogenase (GLUD, EC1.4.1.2 and EC1.4.1.3), 2-oxoglutarate dehydrogenase (OGDH,
  EC1.2.4.2), dihydrolipoamide succinyltransferase (DLST, EC2.3.1.61), succinyl-CoA
  ligase (SUCLG, EC6.2.1.4 and EC6.2.1.5), succinate dehydrogenase (SDH, EC1.3.5.1),
  fumarase (FH, EC4.2.1.2), acetyl-CoA acetyltransferase (ACAT, EC2.3.1.9), hydroxymethylglutaryl-CoA
  synthase (HMGCS, EC2.3.3.10), 3-hydroxy-3-methylglutaryl-CoA reductase (HMGCR, EC1.1.1.34),
  mevalonate kinase (MVK, EC2.7.1.36), phosphomevalonate kinase (PMVK, EC2.7.4.2),
  mevalonate pyrophosphate decarboxylase (MVD, EC4.1.1.33), isopentenyl diphosphate
  isomerase (IDI, EC5.3.3.2), farnesyl diphosphate synthase (FDPS, EC2.5.1.10), farnesyl-diphosphate
  farnesyltransferase (FDFT, EC2.5.1.21), squalene epoxidase (SQLE, EC1.14.99.7),
  cycloartenol synthase (CAS, EC5.4.99.8), delta24-sterol methyltransferase (SMT,
  EC2.1.1.41), cycloeucalenol cycloisomerase (CEI, EC5.5.1.9), sterol 14-demethylase/cytochrome
  p450 family 51 (CYP51, EC1.14.13.70), delta14-sterol reductase/(ERG24, EC1.3.1.70),
  cholestenol isomerase/emopamil binding protein (EBP, EC5.3.3.5), sterol C5 desaturase
  (SC5DL, EC1.14.21.6), 7-dehydrocholesterol reductase (DHCR7, EC1.3.1.21), lanosterol
  synthase (LSS, EC5.4.99.7), C4 sterol methyl oxidase (SMO, EC1.14.13.72), hydroxysteroid
  (3-beta) dehydrogenase (HSD3B, EC1.1.1.170), hydroxysteroid (17-beta) dehydrogenase
  (HSD17B, EC1.1.1.270), serine palmitoyltransferase (SPTLC, EC2.3.1.50), 3-ketodihydrosphingosine
  reductase (KDSR, EC1.1.1.102), alkaline ceramidase (ACER, EC3.5.1.23), ceramide
  synthetase (CERS, EC2.3.1.24), sphingolipid delta-4 desaturase/degenerative spermatocyte
  homolog (DEGS, EC1.14.-.-), phosphatic acid phosphatase (PPAP, EC3.1.3.4), ceramide
  kinase (CERK, EC2.7.1.138), sphingomyelin phosphodiesterase (SMPD, EC3.1.4.12),
  sphingomyelin synthase (SGMS, EC2.7.8.27), UDP-glucose ceramide glucosyltransferase
  (UGCG, EC2.4.1.80), glucosylceramidase (GLCM, EC3.2.1.45), sphingosine kinase (SPHK,
  EC2.7.1.91), sphingosine-1-phosphate lyase (SGPL, EC4.1.2.27), CDP-diacylglycerol
  synthase (CDS, EC2.7.7.41), CDP-diacylglycerol inositol 3-phosphatidyltransferase
  (CDIPT, EC2.7.8.11), phosphatidylserine synthase (PTDSS, EC2.7.8.8), phosphatidylethanolamine
  methyltransferase (PEMT, EC2.1.1.17), phosphatidylserine decarboxylase (PISD, EC4.1.1.65),
  choline/ethanolamine phosphotransferase (CEPT, EC2.7.8.1), phospholipase C (PLC,
  EC3.1.4.3), glycerate kinase (GLYCTK, EC2.7.1.31), aldehyde dehydrogenase (ALDH,
  EC1.2.1.3), alcohol dehydrogenase (ADH, EC1.1.1.2), glycerol kinase (GK, EC2.7.1.30),
  glycerol-3-phosphate dehydrogenase (GPD, EC1.1.1.8), glycero-3-phosphate acyltransferase
  (GPAT, EC2.3.1.15), 1-acylglycerol-3-phosphate acyltransferase (AGPAT, EC2.3.1.51),
  phospholipase D (PLD, EC3.1.4.4), diacylglycerol kinase (DGK, EC2.7.1.107), diacylglycerol
  acyltransferase (DGAT, EC2.3.1.20), phospholipid diacylglycerol acyltransferase
  (PDAT, EC2.3.1.158), triglyceride lipase (GL, EC3.1.1.3).
papertitle: Genome Characterization of the Oleaginous Fungus Mortierella alpina .
reftext: Lei Wang, et al. PLoS One. 2011;6(12):e28319.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9448338
figid_alias: PMC3234268__F5
figtype: Figure
organisms_ner:
- Homo sapiens
- Rattus norvegicus
- Arabidopsis thaliana
- Agrobacterium tumefaciens
redirect_from: /figures/PMC3234268__F5
ndex: e852e89d-ded1-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3234268__pone.0028319.g005.html
  '@type': Dataset
  description: The glycolysis, pentose phosphate pathway, fatty acid synthesis, tricarboxylic
    acid cycle, malate/pyruvate cycle, sterol, glycerophospholipid, sphingolipid and
    glycerolipid synthesis pathways are outlined. Lipids detected by lipidomics are
    highlighted in yellow. Enzyme names are indicated and followed in parentheses
    by the number of genes encoding them. Enzymes conventionally thought to occur
    in a given pathway but for which homologs were not found in M. alpina are followed
    by a question mark. Hexokinase (HK, EC2.7.1.1), glucose-6-phosphate 1-dehydrogenase
    (G6PD, EC1.1.1.49), 6-phosphogluconolactonase (PGLS, EC3.1.1.31), phosphogluconate
    dehydrogenase (PGD, EC1.1.1.44), ribose 5-phosphate isomerase (RPIA, EC5.3.1.6),
    ribulose-5-phosphate-3-epimerase (RPE, EC5.1.3.1), transketolase (TKT, EC2.2.1.1),
    transaldolase (TALDO, EC2.2.1.2), glucose-6-phosphate isomerase (GPI, EC5.3.1.9),
    phosphofructokinase (PFK, EC2.7.1.11), fructose-bisphosphatase (FBP, EC3.1.3.11),
    aldolase A fructose-bisphosphate (ALDOA, EC4.1.2.13), glyceraldehyde 3-phosphate
    dehydrogenase (GAPDH, EC1.2.1.12), phosphoglycerate kinase (PGK, EC2.7.2.3), triose-phosphate
    isomerase (TPI, EC5.3.1.1), phosphoglycerate mutase (PGAM, EC5.4.2.1), enolase
    (ENO, EC4.2.1.11), pyruvate kinase (PK, EC2.7.1.40), lactate dehydrogenase (LDH,
    EC1.1.1.27), phosphoenolpyruvate carboxykinase (PCK, EC4.1.1.32 and EC4.1.1.49),
    acetyl-CoA carboxylase (ACC, EC6.4.1.2), malonyl-CoA decarboxylase (MLYCD, EC4.1.1.9),
    fatty acid synthase (FASN, EC2.3.1.86), acyl-CoA synthetase (ACSL, EC6.2.1.3),
    acyl-CoA thioesterase (ACOT, EC3.1.2.27), fatty acid elongase (ELOVL, EC2.3.1.-),
    fatty acid delta 9 desaturase (FADS9, EC1.14.19.1), fatty acid delta 12 desaturase
    (FADS12, EC1.14.19.6), fatty acid delta 15 desaturase (FADS15, EC1.14.19.-), fatty
    acid delta 6 desaturase (FADS6, EC1.14.19.3), fatty acid delta 5 desaturase (FADS5,
    EC1.14.19.-), malate dehydrogenase (MDH, EC1.1.1.37), malic enzyme (ME, EC1.1.1.40),
    pyruvate carboxylase (PC, EC6.4.1.1), ATP-citrate lyase (ACLY, EC2.3.3.8), pyruvate
    dehydrogenase (PDH, EC1.2.4.1), dihydrolipoamide acetyltransferase (DLAT, EC2.3.1.12),
    citrate synthase (CS, EC2.3.3.1), aconitase (ACO, EC4.2.1.3), isocitrate dehydrogenase
    (IDH, EC1.1.1.41 and EC1.1.1.42), glutamate dehydrogenase (GLUD, EC1.4.1.2 and
    EC1.4.1.3), 2-oxoglutarate dehydrogenase (OGDH, EC1.2.4.2), dihydrolipoamide succinyltransferase
    (DLST, EC2.3.1.61), succinyl-CoA ligase (SUCLG, EC6.2.1.4 and EC6.2.1.5), succinate
    dehydrogenase (SDH, EC1.3.5.1), fumarase (FH, EC4.2.1.2), acetyl-CoA acetyltransferase
    (ACAT, EC2.3.1.9), hydroxymethylglutaryl-CoA synthase (HMGCS, EC2.3.3.10), 3-hydroxy-3-methylglutaryl-CoA
    reductase (HMGCR, EC1.1.1.34), mevalonate kinase (MVK, EC2.7.1.36), phosphomevalonate
    kinase (PMVK, EC2.7.4.2), mevalonate pyrophosphate decarboxylase (MVD, EC4.1.1.33),
    isopentenyl diphosphate isomerase (IDI, EC5.3.3.2), farnesyl diphosphate synthase
    (FDPS, EC2.5.1.10), farnesyl-diphosphate farnesyltransferase (FDFT, EC2.5.1.21),
    squalene epoxidase (SQLE, EC1.14.99.7), cycloartenol synthase (CAS, EC5.4.99.8),
    delta24-sterol methyltransferase (SMT, EC2.1.1.41), cycloeucalenol cycloisomerase
    (CEI, EC5.5.1.9), sterol 14-demethylase/cytochrome p450 family 51 (CYP51, EC1.14.13.70),
    delta14-sterol reductase/(ERG24, EC1.3.1.70), cholestenol isomerase/emopamil binding
    protein (EBP, EC5.3.3.5), sterol C5 desaturase (SC5DL, EC1.14.21.6), 7-dehydrocholesterol
    reductase (DHCR7, EC1.3.1.21), lanosterol synthase (LSS, EC5.4.99.7), C4 sterol
    methyl oxidase (SMO, EC1.14.13.72), hydroxysteroid (3-beta) dehydrogenase (HSD3B,
    EC1.1.1.170), hydroxysteroid (17-beta) dehydrogenase (HSD17B, EC1.1.1.270), serine
    palmitoyltransferase (SPTLC, EC2.3.1.50), 3-ketodihydrosphingosine reductase (KDSR,
    EC1.1.1.102), alkaline ceramidase (ACER, EC3.5.1.23), ceramide synthetase (CERS,
    EC2.3.1.24), sphingolipid delta-4 desaturase/degenerative spermatocyte homolog
    (DEGS, EC1.14.-.-), phosphatic acid phosphatase (PPAP, EC3.1.3.4), ceramide kinase
    (CERK, EC2.7.1.138), sphingomyelin phosphodiesterase (SMPD, EC3.1.4.12), sphingomyelin
    synthase (SGMS, EC2.7.8.27), UDP-glucose ceramide glucosyltransferase (UGCG, EC2.4.1.80),
    glucosylceramidase (GLCM, EC3.2.1.45), sphingosine kinase (SPHK, EC2.7.1.91),
    sphingosine-1-phosphate lyase (SGPL, EC4.1.2.27), CDP-diacylglycerol synthase
    (CDS, EC2.7.7.41), CDP-diacylglycerol inositol 3-phosphatidyltransferase (CDIPT,
    EC2.7.8.11), phosphatidylserine synthase (PTDSS, EC2.7.8.8), phosphatidylethanolamine
    methyltransferase (PEMT, EC2.1.1.17), phosphatidylserine decarboxylase (PISD,
    EC4.1.1.65), choline/ethanolamine phosphotransferase (CEPT, EC2.7.8.1), phospholipase
    C (PLC, EC3.1.4.3), glycerate kinase (GLYCTK, EC2.7.1.31), aldehyde dehydrogenase
    (ALDH, EC1.2.1.3), alcohol dehydrogenase (ADH, EC1.1.1.2), glycerol kinase (GK,
    EC2.7.1.30), glycerol-3-phosphate dehydrogenase (GPD, EC1.1.1.8), glycero-3-phosphate
    acyltransferase (GPAT, EC2.3.1.15), 1-acylglycerol-3-phosphate acyltransferase
    (AGPAT, EC2.3.1.51), phospholipase D (PLD, EC3.1.4.4), diacylglycerol kinase (DGK,
    EC2.7.1.107), diacylglycerol acyltransferase (DGAT, EC2.3.1.20), phospholipid
    diacylglycerol acyltransferase (PDAT, EC2.3.1.158), triglyceride lipase (GL, EC3.1.1.3).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACKR1
  - GYPC
  - RPIA
  - PGLS
  - PGD
  - PHGDH
  - RPE
  - DECR1
  - DGAT1
  - SLCO4A1
  - ALDOA
  - ALDOB
  - ALDOC
  - GLYCTK
  - BMP8B
  - BMP8A
  - GPI
  - GNPDA1
  - PFKL
  - PFKM
  - PFKP
  - ATM
  - PEMT
  - MUC1
  - CDIPT
  - ACLY
  - TSHZ1
  - PLA2G15
  - ACSS2
  - ACCS
  - ELOVL2
  - KLK15
  - MUSK
  - RAPSN
  - FLAD1
  - ACSL1
  - ACSL3
  - ACSL4
  - ACSL5
  - ACSL6
  - CERS4
  - ACOT2
  - ACD
  - DEGS1
  - IDH1
  - IDH2
  - PPARA
  - PPARD
  - PPARG
  - ELOVL1
  - ELOVL3
  - ELOVL4
  - ELOVL5
  - ELOVL6
  - ELOVL7
  - ADCY2
  - ACAD8
  - ACAD9
  - ACADL
  - ACADM
  - ACADS
  - ACADVL
  - ACAD10
  - ACAD11
  - ACADSB
  - GCDH
  - IVD
  - SLC5A3
  - ACAT1
  - CES1
  - SOAT1
  - ACAT2
  - SOAT2
  - HMGCS2
  - HMGCS1
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - LSS
  - LMBR1
  - SMO
  - SMOX
  - Rpia
  - Pgls
  - Pgd
  - Rpe
  - Got1
  - Dgat1
  - Glyctk
  - Gpi
  - Pgk1
  - Pemt
  - Cdipt
  - Acly
  - Acsl1
  - Pla2g15
  - Elovl2
  - Acox1
  - Ccne1
  - Cers4
  - Acot2
  - Acd
  - Degs1
  - Idh3g
  - Ppara
  - Adcy2
  - Slc5a3
  - Acat2
  - Soat2
  - Hmgcs2
  - Bcar1
  - Lss
  - Smo
  - ASP2
  - TT2
  - FBP
  - PGK
  - ACS
  - ALATS
  - ACI1
  - AAO3
  - CaS
  - FD
  - MK
  - rpiA
  - rpe
  - acs
  - Zymosterol 4-Metryl Ls 3-Kato-4-mety o 4-Methytrymonterol
  - Oleoyl-CoA
  - Stearidonic acid
  - Ceramide
  - sphingaine-IP ceramide
  - ethanolamine
  - Fatty acid
  - D-Fructose-P
  - Fructose
  - Isocitrate
  - Malate
  - Mevalonate
  - phosphate
  - Pyruvate
  - pyruvate
  - Ribose
  - Serine
  - Sphingolipid
  - Sterol
  - sterol
  - Succinate
  - Lactate
---
