---
figid: PMC3731064__nihms492698f1
figtitle: Chromatin and the genome integrity network
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Schizosaccharomyces pombe
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
- Xenopus laevis
- NA
pmcid: PMC3731064
filename: nihms492698f1.jpg
figlink: /pmc/articles/PMC3731064/figure/F1/
number: F1
caption: DNA damage (shown by the lightning bolt) can create double-strand breaks
  (DSBs) that are repaired by the appropriate pathway, depending on the cell cycle
  phase. DNA DSBs in G1 phase are preferably repaired by non-homologous end joining
  (NHEJ), whereas DSBs in S phase or in G2 phase are mainly repaired by homologous
  recombination. During S phase, replication forks that encounter DNA damage or that
  undergo other types of replication stress may induce DNA DSBs and/or lead to the
  formation of aberrant chromatin structures between chromosomes. If they are not
  resolved, these aberrant structures, which are derived from replication defects
  or incomplete homologous recombination, can lead to chromosome segregation failure
  in mitosis or to chromosomal breakage during cytokinesis. To ensure that the most
  appropriate response is activated, crosstalk between the genome stability pathways
  is essential. In addition, the genome stability network successfully connects the
  repair process with other pathways that regulate cell homeostasis. Chromatin has
  a major role in the different genome stability pathways (as depicted by the nucleosomes
  in boxes).
papertitle: Chromatin and the genome integrity network.
reftext: Manolis Papamichos-Chronakis, et al. Nat Rev Genet. ;14(1):62-75.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9427642
figid_alias: PMC3731064__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC3731064__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3731064__nihms492698f1.html
  '@type': Dataset
  description: DNA damage (shown by the lightning bolt) can create double-strand breaks
    (DSBs) that are repaired by the appropriate pathway, depending on the cell cycle
    phase. DNA DSBs in G1 phase are preferably repaired by non-homologous end joining
    (NHEJ), whereas DSBs in S phase or in G2 phase are mainly repaired by homologous
    recombination. During S phase, replication forks that encounter DNA damage or
    that undergo other types of replication stress may induce DNA DSBs and/or lead
    to the formation of aberrant chromatin structures between chromosomes. If they
    are not resolved, these aberrant structures, which are derived from replication
    defects or incomplete homologous recombination, can lead to chromosome segregation
    failure in mitosis or to chromosomal breakage during cytokinesis. To ensure that
    the most appropriate response is activated, crosstalk between the genome stability
    pathways is essential. In addition, the genome stability network successfully
    connects the repair process with other pathways that regulate cell homeostasis.
    Chromatin has a major role in the different genome stability pathways (as depicted
    by the nucleosomes in boxes).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - dsb
  - lds
  - LanB2
  - anon-70Db
  - anon-70Dc
  - stop
  - Dr
  - CycE
  - cyc
---
