---
figid: PMC3361414__pnas.1205990109fig05
figtitle: Schematic representation of the interplay of T7 gp5.5, gp3, E
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3361414
filename: pnas.1205990109fig05.jpg
figlink: /pmc/articles/PMC3361414/figure/fig05/
number: F5
caption: Schematic representation of the interplay of T7 gp5.5, gp3, E. coli H-NS,
  and tRNA in cells infected with phage T7. (A) After infection of E. coli by T7 the
  gene 3 endonuclease and gene 6 exonuclease degrade the E. coli genome to provide
  dNMPs for T7 DNA synthesis. T7 gp5 binds tightly to the abundant host thioredoxin
  to acquire processivity. The gp5/trx complex along with the gp4 helicase-primase
  and the gene 2.5 ssDNA binding protein form a replisome that mediates coordinated
  DNA synthesis. The degradation of the host genome releases the histone-like H-NS,
  which in turn can bind to the phage DNA, inhibiting DNA replication. However, the
  abundant gp5.5 binds to another abundant component in E. coli, tRNA, and acquires
  a high affinity for H-NS, resulting in a gp5.5/tRNA/H-NS complex. Thus, the excess
  H-NS is sequestered by gp5.5, eliminating its binding to T7 DNA. The short primers,
  synthesized by T7 primase, at the 5′ termini of Okazaki fragments are removed by
  gp6, and the Okazaki fragments are ligated. (B) In the absence of a functioning
  DNA primase an alternative pathway for priming of the lagging strand DNA polymerase
  is required. Suppressor mutations arise within gene 5.5 that deplete the pool of
  gp5.5. As a consequence, the pool of tRNA is not reduced. The similarity of the
  3′ terminus of many tRNAs to the tetraribonucleotides synthesized by the T7 primase
  allows them to function as primers. The tRNA primed DNA structure, however, is a
  target for gp3 endonuclease. Some suppressors therefore also contain mutations in
  gene 3 that reduce the activity of gp3.
papertitle: Gene 5.5 protein of bacteriophage T7 in complex with Escherichia coli
  nucleoid protein H-NS and transfer RNA masks transfer RNA priming in T7 DNA replication.
reftext: Bin Zhu, et al. Proc Natl Acad Sci U S A. 2012 May 22;109(21):8050-8055.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9345637
figid_alias: PMC3361414__F5
figtype: Figure
redirect_from: /figures/PMC3361414__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3361414__pnas.1205990109fig05.html
  '@type': Dataset
  description: Schematic representation of the interplay of T7 gp5.5, gp3, E. coli
    H-NS, and tRNA in cells infected with phage T7. (A) After infection of E. coli
    by T7 the gene 3 endonuclease and gene 6 exonuclease degrade the E. coli genome
    to provide dNMPs for T7 DNA synthesis. T7 gp5 binds tightly to the abundant host
    thioredoxin to acquire processivity. The gp5/trx complex along with the gp4 helicase-primase
    and the gene 2.5 ssDNA binding protein form a replisome that mediates coordinated
    DNA synthesis. The degradation of the host genome releases the histone-like H-NS,
    which in turn can bind to the phage DNA, inhibiting DNA replication. However,
    the abundant gp5.5 binds to another abundant component in E. coli, tRNA, and acquires
    a high affinity for H-NS, resulting in a gp5.5/tRNA/H-NS complex. Thus, the excess
    H-NS is sequestered by gp5.5, eliminating its binding to T7 DNA. The short primers,
    synthesized by T7 primase, at the 5′ termini of Okazaki fragments are removed
    by gp6, and the Okazaki fragments are ligated. (B) In the absence of a functioning
    DNA primase an alternative pathway for priming of the lagging strand DNA polymerase
    is required. Suppressor mutations arise within gene 5.5 that deplete the pool
    of gp5.5. As a consequence, the pool of tRNA is not reduced. The similarity of
    the 3′ terminus of many tRNAs to the tetraribonucleotides synthesized by the T7
    primase allows them to function as primers. The tRNA primed DNA structure, however,
    is a target for gp3 endonuclease. Some suppressors therefore also contain mutations
    in gene 3 that reduce the activity of gp3.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pri
  - Arr2
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - Hel25E
  - mtSSB
  - Exn
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - 'On'
  - trnA
  - tRNA:Asp-GTC-1-9
  - trx
  - Trxr-1
  - Trx-2
  - NPTN
  - GP6
  - RNPC3
  - NPC2
  - SSB
  - COX2
  - MTCO2P12
  - FUT3
  - TRNA
  - TXN
  - VAC14
  - CD36
  - HFM1
  - GP5
  - ERVK-9
  - ERVK-11
  - ERVK-19
---
