---
figid: PMC3092098__gm227-1
figtitle: The genes and pathways associated with Crohn's disease pathogenesis
organisms:
- Homo sapiens
- Mus musculus
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
- Danio rerio
pmcid: PMC3092098
filename: gm227-1.jpg
figlink: /pmc/articles/PMC3092098/figure/F1/
number: F1
caption: Schematic representation of the genes and pathways associated with Crohn's
  disease pathogenesis. The ongoing inflammatory response in the gastrointestinal
  tract in patients with Crohn's disease (CD) is thought to be caused by an aberrant
  immune response to commensal microflora in the gut. In patients with CD, defects
  in first defense mechanisms (that is, disrupted epithelial and mucosal barrier)
  contribute to increased bacterial penetration (MUC1 and MUC19). Genes involved in
  pattern recognition (NOD2, TLR4 and CARD9) suggest an increased response of antigen-presenting
  cells to commensal microbes. Consequently, the NF-кB cascade is activated (TNFSF15),
  leading to production of pro-inflammatory cytokines. Association of REL and UBE2L3
  suggest an impaired NF-кB negative feedback. Antigen-presenting cells migrate to
  Peyer's patches (intestinal mesenteric lymph nodes) (TNFSF11) to present antigens
  and stimulate T-cell proliferation (IL2RA and TAGAP) and differentiation. T cells
  of patients with CD, in turn, respond more intensely. Th0 cells are stimulated to
  differentiate into T-cell subtypes regulated by a variety of the produced cytokines
  and their receptors. Th17 cells are involved in many immune-related diseases, and
  they are activated through IL-23R, which, in turn, activates the JAK-STAT-TYK (Janus
  kinase- signal transducer and activator of transcription-tyrosine kinase) pathway
  that enhances pro-inflammatory cytokine production (JAK2, STAT3 and TYK2). Th1 and
  Th17 cells are pro-inflammatory, whereas Treg cells downregulate the immune response.
  Another major contribution to CD pathogenesis comes from autophagy. In autophagosomes,
  intracellular components, including phagocytosed microbes, are degraded, after which
  their antigens are presented to CD4+ cells. Autophagy is at least partly regulated
  by the CD risk genes ATG16L1, IRGM and VAMP3. The activation of CD4+ cells leads
  to the production of pro-inflammatory cytokines and the maintenance of the inflammation.
  All the displayed processes could finally lead to homing of leukocytes to inflammation
  sites (ICAM1,3, CCR cluster), and neutrophil recruitment. Consequently, chronic
  inflammation, ulceration and deeper microbial penetrance occur. The known associated
  genes are shown in red. Table 1 summarizes the associated loci shown here. CCL20,
  chemokine (C-C motif) ligand 20; ICOS, inducible T-cell co-stimulator; MDP, muramyl
  dipeptide; NF, nuclear factor; TCR, T-cell receptor; TGF, transforming growth factor;
  TGFBR, TGF β receptor; Th, T helper cell; TNF, tumor necrosis factor; Treg, regulatory
  T cell.
papertitle: The quest for genetic risk factors for Crohn's disease in the post-GWAS
  era.
reftext: Karin Fransen, et al. Genome Med. 2011;3(2):13-13.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9407139
figid_alias: PMC3092098__F1
figtype: Figure
redirect_from: /figures/PMC3092098__F1
ndex: 8e726244-dee1-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3092098__gm227-1.html
  '@type': Dataset
  description: Schematic representation of the genes and pathways associated with
    Crohn's disease pathogenesis. The ongoing inflammatory response in the gastrointestinal
    tract in patients with Crohn's disease (CD) is thought to be caused by an aberrant
    immune response to commensal microflora in the gut. In patients with CD, defects
    in first defense mechanisms (that is, disrupted epithelial and mucosal barrier)
    contribute to increased bacterial penetration (MUC1 and MUC19). Genes involved
    in pattern recognition (NOD2, TLR4 and CARD9) suggest an increased response of
    antigen-presenting cells to commensal microbes. Consequently, the NF-кB cascade
    is activated (TNFSF15), leading to production of pro-inflammatory cytokines. Association
    of REL and UBE2L3 suggest an impaired NF-кB negative feedback. Antigen-presenting
    cells migrate to Peyer's patches (intestinal mesenteric lymph nodes) (TNFSF11)
    to present antigens and stimulate T-cell proliferation (IL2RA and TAGAP) and differentiation.
    T cells of patients with CD, in turn, respond more intensely. Th0 cells are stimulated
    to differentiate into T-cell subtypes regulated by a variety of the produced cytokines
    and their receptors. Th17 cells are involved in many immune-related diseases,
    and they are activated through IL-23R, which, in turn, activates the JAK-STAT-TYK
    (Janus kinase- signal transducer and activator of transcription-tyrosine kinase)
    pathway that enhances pro-inflammatory cytokine production (JAK2, STAT3 and TYK2).
    Th1 and Th17 cells are pro-inflammatory, whereas Treg cells downregulate the immune
    response. Another major contribution to CD pathogenesis comes from autophagy.
    In autophagosomes, intracellular components, including phagocytosed microbes,
    are degraded, after which their antigens are presented to CD4+ cells. Autophagy
    is at least partly regulated by the CD risk genes ATG16L1, IRGM and VAMP3. The
    activation of CD4+ cells leads to the production of pro-inflammatory cytokines
    and the maintenance of the inflammation. All the displayed processes could finally
    lead to homing of leukocytes to inflammation sites (ICAM1,3, CCR cluster), and
    neutrophil recruitment. Consequently, chronic inflammation, ulceration and deeper
    microbial penetrance occur. The known associated genes are shown in red. Table
    1 summarizes the associated loci shown here. CCL20, chemokine (C-C motif) ligand
    20; ICOS, inducible T-cell co-stimulator; MDP, muramyl dipeptide; NF, nuclear
    factor; TCR, T-cell receptor; TGF, transforming growth factor; TGFBR, TGF β receptor;
    Th, T helper cell; TNF, tumor necrosis factor; Treg, regulatory T cell.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Card9
  - Irgm1
  - Rel
  - Muc19
  - Smgc
  - Trav6-3
  - Il10
  - Tht
  - Il12b
  - Hand1
  - Nelfcd
  - Ty
  - Icam5
  - Icam1
  - Il18
  - Nfkb1
  - Icosl
  - Il2ra
  - Icos
  - Ccl20
  - Il23a
  - Tagap
  - Il1
  - Tgfa
  - ap
  - tk
  - Tgfb1
  - Ltbp1
  - Il17a
  - Il21
  - Il22
  - DPEP1
  - CARD9
  - IRGM
  - REL
  - MUC19
  - TRBV20OR9-2
  - TRA
  - TRB
  - TRD
  - TRG
  - IFNK
  - IFNW1
  - IFNA1
  - IFNA2
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA17
  - IFNA21
  - IFNB1
  - IL10
  - IL12B
  - NELFCD
  - ICAM3
  - ICAM1
  - IL18
  - NFKB1
  - ICOSLG
  - IL2RA
  - ICOS
  - CCL20
  - IL37
  - IL23A
  - TAGAP
  - IL1B
  - TGFA
  - RPL23A
  - IGKV1D-43
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - TK1
  - TK2
  - TGFB1
  - TGFB2
  - TGFB3
  - IL17A
  - IL22
  - IL21
  - IL26
  - TGFBR1
  - TGFBR2
  - TGFBR3
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Syb
  - nSyb
  - Tcr
  - Def
  - tht
  - TH1
  - ty
  - Dif
  - dl
  - CR
  - JIL-1
  - RpL23A
  - Tehao
  - Stat92E
  - Tk
  - dpp
  - gbb
  - put
  - mav
  - card9
  - rel
  - il10
  - il12ba
  - icam3
  - ccl20b
  - tagapb
  - tgfa
  - thoc2
  - stat1b
  - stat4
  - il21
  - il22
  - il26
  - Defensin
  - Rheumatoid arthritis
---
