---
figid: PMC3852343__fig-3
figtitle: Oxygen-regulated COX subunit isoform switch in yeast and mammals
organisms:
- NA
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
pmcid: PMC3852343
filename: fig-3.jpg
figlink: /pmc/articles/PMC3852343/figure/f3/
number: F3
caption: Mechanisms of oxygen-regulated COX subunit isoform switch in yeast and mammals.
  (A) Transcriptional regulation of COX5a/COX5b in the yeast Saccharomyces cerevisiae.
  Normoxic (>0.5 μM O2) repression of COX5b is mediated through a heme-dependent pathway.
  In the presence of oxygen, heme is synthesized and binds to the transcriptional
  factor Hap1 leading to its activation. Hap1 induces the expression of a set of OXPHOS
  genes, including CYC1 and CYC7. It also activates the expression of the general
  transcription repressor Rox1, which binds to a consensus ATTGTTCTC box on the promoter
  region of anaerobic genes, such as COX5b, CYC7, and itself to repress their expression.
  Hap1 acts on CYC7 and ROX1 as an activator directly and also as a repressor through
  Rox1. This feedback regulatory loop has been proposed to confer mutational robustness
  in yeast transcription factor regulation (). Additional gene-specific transcriptional
  repressors may act, such as Ord1, in the case of COX5b. In normoxia and in a carbon
  source-dependent manner, the Hap2/3/4/5 transcriptional complex binds to consensus
  TNRTTGGT boxes and activates expression of aerobic genes, including most OXPHOS
  genes, such as COX5a and CYC1. In this way in normoxia, most COX contains the Cox5a
  isoform. In hypoxia, the heme-dependent Hap1 and Hap2/3/4/5 complexes are not activated.
  COX5a and CYC1 expression is not induced and COX5b and CYC7 expression is de-repressed.
  In consequence, the total amount of COX is decreased, but a higher proportion contains
  Cox5b. The hypoxic enzyme acts a nitric oxide (NO) synthase, releasing NO that act
  as a signaling molecule to further induce hypoxic gene expression. (B) Model for
  regulation of COX4-1/COX4-2 switch by the hypoxia inducing factor 1 (HIF-1) in mammalian
  cells. HIF-1 is a heterodimeric complex composed of a constitutively expressed HIF-1β
  subunit and an O2-regulated HIF-1α or HIF-2α subunit. In normoxia, HIF-1α is hydroxylated
  by prolyl hydroxylases (PHDs) and subjected to degradation. HIF-1α is also hydroxylated
  by factor inhibiting HIF-1 (FIH-1) at the asparagine residue, which inhibits its
  interaction with the p300 coactivator and inactivates HIF-1-mediated transcription
  of COXIV-2. In hypoxia/anoxia, when oxygen tension drops below 3%, HIF-1α is stabilized,
  translocates to the nucleus, and forms a heterodimer with HIF-1β on HIF-1-responsive
  element (HRE) sites to activate a plethora of hypoxic genes, including COXIV-2.
  HIF-1 also further activates mitochondrial LON protease expression that preferentially
  degrades COX4-1 in hypoxia (34), which facilitates the subunit switch of COX4-1
  to COX4-2.
papertitle: Redox and Reactive Oxygen Species Regulation of Mitochondrial Cytochrome
  c Oxidase Biogenesis.
reftext: Myriam Bourens, et al. Antioxid Redox Signal. 2013 Dec 1;19(16):1940-1952.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8277006
figid_alias: PMC3852343__F3
figtype: Figure
redirect_from: /figures/PMC3852343__F3
ndex: 2ba70cff-debb-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3852343__fig-3.html
  '@type': Dataset
  description: Mechanisms of oxygen-regulated COX subunit isoform switch in yeast
    and mammals. (A) Transcriptional regulation of COX5a/COX5b in the yeast Saccharomyces
    cerevisiae. Normoxic (>0.5 μM O2) repression of COX5b is mediated through a heme-dependent
    pathway. In the presence of oxygen, heme is synthesized and binds to the transcriptional
    factor Hap1 leading to its activation. Hap1 induces the expression of a set of
    OXPHOS genes, including CYC1 and CYC7. It also activates the expression of the
    general transcription repressor Rox1, which binds to a consensus ATTGTTCTC box
    on the promoter region of anaerobic genes, such as COX5b, CYC7, and itself to
    repress their expression. Hap1 acts on CYC7 and ROX1 as an activator directly
    and also as a repressor through Rox1. This feedback regulatory loop has been proposed
    to confer mutational robustness in yeast transcription factor regulation (). Additional
    gene-specific transcriptional repressors may act, such as Ord1, in the case of
    COX5b. In normoxia and in a carbon source-dependent manner, the Hap2/3/4/5 transcriptional
    complex binds to consensus TNRTTGGT boxes and activates expression of aerobic
    genes, including most OXPHOS genes, such as COX5a and CYC1. In this way in normoxia,
    most COX contains the Cox5a isoform. In hypoxia, the heme-dependent Hap1 and Hap2/3/4/5
    complexes are not activated. COX5a and CYC1 expression is not induced and COX5b
    and CYC7 expression is de-repressed. In consequence, the total amount of COX is
    decreased, but a higher proportion contains Cox5b. The hypoxic enzyme acts a nitric
    oxide (NO) synthase, releasing NO that act as a signaling molecule to further
    induce hypoxic gene expression. (B) Model for regulation of COX4-1/COX4-2 switch
    by the hypoxia inducing factor 1 (HIF-1) in mammalian cells. HIF-1 is a heterodimeric
    complex composed of a constitutively expressed HIF-1β subunit and an O2-regulated
    HIF-1α or HIF-2α subunit. In normoxia, HIF-1α is hydroxylated by prolyl hydroxylases
    (PHDs) and subjected to degradation. HIF-1α is also hydroxylated by factor inhibiting
    HIF-1 (FIH-1) at the asparagine residue, which inhibits its interaction with the
    p300 coactivator and inactivates HIF-1-mediated transcription of COXIV-2. In hypoxia/anoxia,
    when oxygen tension drops below 3%, HIF-1α is stabilized, translocates to the
    nucleus, and forms a heterodimer with HIF-1β on HIF-1-responsive element (HRE)
    sites to activate a plethora of hypoxic genes, including COXIV-2. HIF-1 also further
    activates mitochondrial LON protease expression that preferentially degrades COX4-1
    in hypoxia (34), which facilitates the subunit switch of COX4-1 to COX4-2.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SPE1
  - IXR1
  - ROX1
  - COX5A
  - CYC7
  - CYC1
  - COX5B
  - ILV1
  - PHD1
  - CCNI
  - CYCS
  - EIF1
  - SRR
  - COX8A
  - CPOX
  - COX6C
  - COX7C
  - .na.character
  - COX4I1
  - COX4I2
  - COX6A1
  - COX6A2
  - COX6B1
  - COX6B2
  - COX7A1
  - COX7A2
  - COX7B
  - COX7B2
  - COX8C
  - PTH
  - HIF1AN
  - EGLN2
  - RAB4B-EGLN2
  - EP300
  - HIF1A
  - ARNT
  - ARNT2
  - BMAL1
  - LONP1
---
