---
figid: PMC3391182__1741-7007-10-55-7
figlink: /pmc/articles/PMC3391182/figure/F7/
number: F7
caption: Working model of the yeast response to gene mistranslations. Mistranslated
  proteins are folding substrates of molecular chaperones that compete with wild type
  client substrates for folding/refolding. Gradual accumulation of mistranslated proteins
  shifts the HSP-client binding equilibrium to the subpopulation of misfolded proteins,
  releasing client substrates from HSPs. This activates or inactivates natural HSP-client
  substrates depending on whether the HSP-client interaction is positive or negative.
  HSP substrate release deregulates cellular processes mediated by the HSP client
  proteins and remodels chaperone-chaperone networks, which are critical for cellular
  homeostasis. Increased protein folding/refolding and degradation increase ATP consumption
  leading to up-regulation of mitochondrial metabolism and ROS accumulation. Accumulation
  of mistranslated proteins in mitochondria also increases mitochondrial stress and
  ROS production. Mistranslated proteins that enter the secretory pathway accumulate
  in the ER and up-regulate ER resident chaperones, activating the UPR, further increasing
  ROS production. This leads to a deficit in protein secretion with consequences for
  cell membranes and cell wall structure and function. Aggregation of mistranslated
  proteins exacerbates proteotoxic stress, increases ATP consumption and activates
  the general stress response (ESR) through the transcription factors Msn2/4p and
  Yaps. Mistranslations also down-regulate ribosome biosynthesis, translational factors
  and CLIPS, exacerbating the negative consequences of mistranslations due to the
  critical role of CLIPS in folding newly synthesized proteins.
pmcid: PMC3391182
papertitle: Low level genome mistranslations deregulate the transcriptome and translatome
  and generate proteotoxic stress in yeast.
reftext: Jo√£o A Paredes, et al. BMC Biol. 2012;10:55-55.
pmc_ranked_result_index: '180277'
pathway_score: 0.9682282
filename: 1741-7007-10-55-7.jpg
figtitle: Working model of the yeast response to gene mistranslations
year: '2012'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3391182__1741-7007-10-55-7.html
  '@type': Dataset
  description: Working model of the yeast response to gene mistranslations. Mistranslated
    proteins are folding substrates of molecular chaperones that compete with wild
    type client substrates for folding/refolding. Gradual accumulation of mistranslated
    proteins shifts the HSP-client binding equilibrium to the subpopulation of misfolded
    proteins, releasing client substrates from HSPs. This activates or inactivates
    natural HSP-client substrates depending on whether the HSP-client interaction
    is positive or negative. HSP substrate release deregulates cellular processes
    mediated by the HSP client proteins and remodels chaperone-chaperone networks,
    which are critical for cellular homeostasis. Increased protein folding/refolding
    and degradation increase ATP consumption leading to up-regulation of mitochondrial
    metabolism and ROS accumulation. Accumulation of mistranslated proteins in mitochondria
    also increases mitochondrial stress and ROS production. Mistranslated proteins
    that enter the secretory pathway accumulate in the ER and up-regulate ER resident
    chaperones, activating the UPR, further increasing ROS production. This leads
    to a deficit in protein secretion with consequences for cell membranes and cell
    wall structure and function. Aggregation of mistranslated proteins exacerbates
    proteotoxic stress, increases ATP consumption and activates the general stress
    response (ESR) through the transcription factors Msn2/4p and Yaps. Mistranslations
    also down-regulate ribosome biosynthesis, translational factors and CLIPS, exacerbating
    the negative consequences of mistranslations due to the critical role of CLIPS
    in folding newly synthesized proteins.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - YY1AP1
  - HMBS
  - ESR1
  - ESR2
genes:
- word: Yaps
  symbol: YAP
  source: hgnc_alias_symbol
  hgnc_symbol: YY1AP1
  entrez: '55249'
- word: UPS
  symbol: UPS
  source: hgnc_prev_symbol
  hgnc_symbol: HMBS
  entrez: '3145'
- word: ESR
  symbol: ESR
  source: bioentities_symbol
  hgnc_symbol: ESR1
  entrez: '2099'
- word: ESR
  symbol: ESR
  source: bioentities_symbol
  hgnc_symbol: ESR2
  entrez: '2100'
chemicals: []
diseases: []
figid_alias: PMC3391182__F7
redirect_from: /figures/PMC3391182__F7
figtype: Figure
---
