---
figid: PMC3067248__zam9991017810001
figtitle: Sulfur metabolism in B
organisms:
- NA
pmcid: PMC3067248
filename: zam9991017810001.jpg
figlink: /pmc/articles/PMC3067248/figure/f1/
number: F1
caption: Sulfur metabolism in B. aurantiacum. The BL numbers for B. aurantiacum genes
  correspond to those of Agmial (http://genome.jouy.inra.fr/∼vloux/publications/forquin08/)
  (). A systematic search for E. coli, B. subtilis, C. glutamicum, and M. tuberculosis
  genes involved in sulfur metabolism was performed. The high-GC-content Gram-positive
  bacteria C. glutamicum and M. tuberculosis are phylogenetically related to Brevibacterium
  strains (, ). The genes were renamed according to B. subtilis and C. glutamicum
  orthologues. “?” indicates a step or a pathway for which the gene proposed remains
  to be more clearly identified. (A) Transport of sulfur compounds. Methionine transporters
  (MetNIQ, MetPS), cyst(e)ine transporters (BL1249 to BL1251, BL1293 to BL1296), sulfate
  transporters and sulfonate transporters are indicated. (B) Central sulfur metabolism.
  ATP sulfurylase, cysDN; APS reductase, cysH; sulfite reductase, cysJI; serine O-acetyltransferase,
  cysE; OAS thiol-lyase, cysK; homoserine dehydrogenase, hom; homoserine acetyl-transferase,
  metX; OAH thiol-lyase, metY; cystathionine β-lyase, aecD; cystathionine γ-synthase,
  metB; methionine synthase, metE1-3; methionine γ-lyase, mgl; SAM synthetase, metK;
  SAH hydrolase, sahH; alkanesulfonate monooxygenase, ssuD1-2; flavin mononucleotide
  (FMN) reductase, ssuI; FMNH2-dependent alkanesulfonate monooxygenase, seuA; electron
  transfer flavoprotein, fixA; glycosyltransferase, mshA; MSH deacetylase, mshB; MSH
  ligase, mshC; MSH synthase, mshD; MSH disulfide reductase, mtr; NAD/mycothiol-dependent
  formaldehyde dehydrogenase/nitrosothiol reductase, mscR; MSH-S-conjugate amidase,
  mca; Ac-CoA, acetyl coenzyme A; APS, adenylyl sulfate; PPi, diphosphate; OAS, O-acetylserine;
  OAH, O-acetylhomoserine; MSH, mycothiol; MSSM, mycothiol disulfide; GlcN-INS, glucoaminonylinositol;
  THF, tetrahydrofolate; SAM, S-adenosylmethionine; SAH, S-adenosylhomocysteine; α-KB,
  α-ketobutyrate.
papertitle: Global Regulation of the Response to Sulfur Availability in the Cheese-Related
  Bacterium Brevibacterium aurantiacum  .
reftext: Marie-Pierre Forquin, et al. Appl Environ Microbiol. 2011 Feb;77(4):1449-1459.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8802016
figid_alias: PMC3067248__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Escherichia coli str. K-12 substr. MG1655
redirect_from: /figures/PMC3067248__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3067248__zam9991017810001.html
  '@type': Dataset
  description: Sulfur metabolism in B. aurantiacum. The BL numbers for B. aurantiacum
    genes correspond to those of Agmial (http://genome.jouy.inra.fr/∼vloux/publications/forquin08/)
    (). A systematic search for E. coli, B. subtilis, C. glutamicum, and M. tuberculosis
    genes involved in sulfur metabolism was performed. The high-GC-content Gram-positive
    bacteria C. glutamicum and M. tuberculosis are phylogenetically related to Brevibacterium
    strains (, ). The genes were renamed according to B. subtilis and C. glutamicum
    orthologues. “?” indicates a step or a pathway for which the gene proposed remains
    to be more clearly identified. (A) Transport of sulfur compounds. Methionine transporters
    (MetNIQ, MetPS), cyst(e)ine transporters (BL1249 to BL1251, BL1293 to BL1296),
    sulfate transporters and sulfonate transporters are indicated. (B) Central sulfur
    metabolism. ATP sulfurylase, cysDN; APS reductase, cysH; sulfite reductase, cysJI;
    serine O-acetyltransferase, cysE; OAS thiol-lyase, cysK; homoserine dehydrogenase,
    hom; homoserine acetyl-transferase, metX; OAH thiol-lyase, metY; cystathionine
    β-lyase, aecD; cystathionine γ-synthase, metB; methionine synthase, metE1-3; methionine
    γ-lyase, mgl; SAM synthetase, metK; SAH hydrolase, sahH; alkanesulfonate monooxygenase,
    ssuD1-2; flavin mononucleotide (FMN) reductase, ssuI; FMNH2-dependent alkanesulfonate
    monooxygenase, seuA; electron transfer flavoprotein, fixA; glycosyltransferase,
    mshA; MSH deacetylase, mshB; MSH ligase, mshC; MSH synthase, mshD; MSH disulfide
    reductase, mtr; NAD/mycothiol-dependent formaldehyde dehydrogenase/nitrosothiol
    reductase, mscR; MSH-S-conjugate amidase, mca; Ac-CoA, acetyl coenzyme A; APS,
    adenylyl sulfate; PPi, diphosphate; OAS, O-acetylserine; OAH, O-acetylhomoserine;
    MSH, mycothiol; MSSM, mycothiol disulfide; GlcN-INS, glucoaminonylinositol; THF,
    tetrahydrofolate; SAM, S-adenosylmethionine; SAH, S-adenosylhomocysteine; α-KB,
    α-ketobutyrate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - out
  - so
  - Aps
  - homer
  - Coa
  - Ahcy
  - sah
  - Dr
  - CG8517
  - Mtk
  - Sam-S
  - sam
  - qkr54B
  - sls
  - mgl
  - ea
  - cysN
  - cysE
  - cysH
  - cysI
  - cysJ
  - asd
  - metK
  - metV
  - cysK
  - metY
  - metB
  - mtr
  - mglA
  - mngB
  - methanethiol
  - acetate
  - mercapturic acid
  - aspartate
  - cystathionine
  - cysteine
  - homocysteine
  - methionine
  - Pyruvate
  - serine
  - sulfate
  - Glu
---
