---
figid: PMC3371005__pone.0038722.g006
figtitle: 'Genome-Wide Association for Sensitivity to Chronic Oxidative Stress in
  Drosophila melanogaster '
organisms:
- NA
pmcid: PMC3371005
filename: pone.0038722.g006.jpg
figlink: /pmc/articles/PMC3371005/figure/pone-0038722-g006/
number: F6
caption: '(A) Enriched cellular genetic pathway for candidate genes from all genome
  wide association analyses (gray squares), allowing one missing gene (white triangles)
  or compound (white circles). The border colors indicate the over-represented gene
  ontology categories (P<0.005): axon guidance (red), synapse organization (orange),
  protein phosphorylation (magenta), signal transduction (dark green), inositol phosphate
  metabolism (yellow), phagocytosis engulfment (dark blue), regulation of cell shape
  (light blue), actin cytoskeleton organization (light green), and potassium ion transport
  (pink). Drosophila genes with human homologs are indicated in red font. (B) Enriched
  network for human homologs of Drosophila candidate genes (gray squares) missing
  no more than two consecutive genes (white triangles) or compounds (white circles).
  The border colors indicate the over-represented gene ontology categories (P = 0.085):
  Rho GTPase signaling (orange), NGF signaling (red), EGFR signaling (olive green),
  synaptic transmission (light green), integrin cell surface interactions (dark green),
  DNA replication (magenta), inositol phosphate metabolism (yellow), integration of
  energy metabolism (light pink), metabolism of vitamins and cofactors (dark blue),
  glycerolipid metabolism (light blue), and metabolism of carbohydrates (light purple).'
papertitle: Genome-Wide Association for Sensitivity to Chronic Oxidative Stress in
  Drosophila melanogaster .
reftext: Katherine W. Jordan, et al. PLoS One. 2012;7(6):e38722.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8175545
figid_alias: PMC3371005__F6
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC3371005__F6
ndex: c3fa4c64-deec-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3371005__pone.0038722.g006.html
  '@type': Dataset
  description: '(A) Enriched cellular genetic pathway for candidate genes from all
    genome wide association analyses (gray squares), allowing one missing gene (white
    triangles) or compound (white circles). The border colors indicate the over-represented
    gene ontology categories (P<0.005): axon guidance (red), synapse organization
    (orange), protein phosphorylation (magenta), signal transduction (dark green),
    inositol phosphate metabolism (yellow), phagocytosis engulfment (dark blue), regulation
    of cell shape (light blue), actin cytoskeleton organization (light green), and
    potassium ion transport (pink). Drosophila genes with human homologs are indicated
    in red font. (B) Enriched network for human homologs of Drosophila candidate genes
    (gray squares) missing no more than two consecutive genes (white triangles) or
    compounds (white circles). The border colors indicate the over-represented gene
    ontology categories (P = 0.085): Rho GTPase signaling (orange), NGF signaling
    (red), EGFR signaling (olive green), synaptic transmission (light green), integrin
    cell surface interactions (dark green), DNA replication (magenta), inositol phosphate
    metabolism (yellow), integration of energy metabolism (light pink), metabolism
    of vitamins and cofactors (dark blue), glycerolipid metabolism (light blue), and
    metabolism of carbohydrates (light purple).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Grip
  - Grip128
  - GluRIA
  - SK
  - yip7
  - mAcon1
  - Mcr
  - sit
  - rasp
  - Mipp2
  - Graf
  - Moe
  - Pkc98E
  - RpS27
  - RpS27A
  - Pvf1
  - fra
  - kay
  - Pak
  - mbt
  - CG9784
  - norpA
  - Pka-C1
  - NetA
  - robo3
  - robo1
  - robo2
  - ena
  - dock
  - side-II
  - Ipp
  - CG9426
  - CG6805
  - Synj
  - INPP5E
  - 5PtaseI
  - Sos
  - KCNQ
  - Nrg
  - drk
  - Akt
  - side
  - Sidpn
  - SPE
  - lz
  - spen
  - kst
  - RapGAP1
  - S6k
  - side-VI
  - Treh
  - Pgk
  - Hk
  - nmo
  - Ank2
  - cm
  - ca
  - Cam
  - CaMKII
  - Shmt
  - PolE1
  - RPA3
  - RPA2
  - RPA1
  - Egfr
  - GRIP1
  - C5
  - C6
  - C7
  - C8A
  - C8B
  - C8G
  - C9
  - NR3C2
  - SIT1
  - ARHGAP26
  - RPS27
  - FOLR1
  - FOSL1
  - RABEP2
  - PAK1
  - PAK2
  - PAK3
  - PAK4
  - PAK5
  - PAK6
  - ROBO1
  - ROBO2
  - ROBO3
  - ENAH
  - IPP
  - XYLT2
  - SOS1
  - SOS2
  - NRG1
  - NRG2
  - NRG3
  - NRG4
  - AKT1
  - SERPINA4
  - RAP1GAP
  - RPS6KB1
  - TREH
  - PGK1
  - FOXM1
  - NLK
  - GPI
  - ANK2
  - CALM1
  - CALM2
  - CALM3
  - KRIT1
  - CAMKMT
  - PLCB1
  - SHMT1
  - SHMT2
  - ALDH9A1
  - NRCAM
  - PRKCE
  - PLCE1
  - RASGRP2
  - GRIP2
  - LIPG
  - GNA15
  - RPTOR
  - PTPRA
  - POLE
  - DGKZ
  - GNAQ
  - RASGRP1
  - YWHAB
  - LPL
  - LCP1
  - HGS
  - EGFR
  - SDC1
  - 3-Phosphoglycerate
  - Dihydrofolate
  - Fatty acid
  - Triglyceride
  - Glu
---
