---
figid: PMC3067333__zam9991019150005
figtitle: Schematic presentation of Nag PTS and parts of the central metabolism of
  R
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Arabidopsis thaliana
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
pmcid: PMC3067333
filename: zam9991019150005.jpg
figlink: /pmc/articles/PMC3067333/figure/f5/
number: F5
caption: 'Schematic presentation of Nag PTS and parts of the central metabolism of
  R. eutropha H16 that are relevant for this study. The reddish gene abbreviations
  (small font, italics) indicate genes being upregulated in mutant G+1 as reported
  by transcriptome experiments. The reddish protein abbreviations (large font) indicate
  proteins being increasingly synthesized in mutant G+1 in comparison to wild-type
  H16, as revealed by proteome analyses. The same applies to the greenish gene/protein
  abbreviations, but these indicate downregulation or reduced synthesis by mutant
  G+1 in comparison to wild-type H16. Bluish letters are common abbreviations of trivial
  names of the respective enzymes. Nag PTS components are as follows: nagC (outer-membrane
  protein (porin), nagE (sugar phosphotransferase system, membrane component EIICB),
  nagF (sugar phosphotransferase system, cytosolic component EIIA-HPr-EI). KDPG pathway
  and subsequent reactions leading to pyruvate are as follows: G6PD/Zwf1 (glucose-6-phosphate
  1-dehydrogenase), 6PGL (6-phosphogluconolactonase), EDD/Edd1 (phosphogluconate dehydratase),
  EDA (2-keto-3-deoxygluconate-6-phosphate aldolase), G3PD/CbbG1/cbbG1 (glyceraldehyde-3-phosphate
  dehydrogenase), PGM (phosphoglycerate mutase), ENO (enolase), PK (pyruvate kinase),
  PPS (phosphoenolpyruvate kinase). PDHC (pyruvate dehydrogenase multienzyme complex)
  includes pdhA1 (pyruvate dehydrogenase/decarboxylase [E1 of PDHC]), pdhB (dihydrolipoamide
  acetyltransferase [E2 of PDHC]), pdhL (dihydrolipoamide dehydrogenase [E3 of PDHC]).
  Involved in TCC and PHB metabolism are cisY (citrate synthase), PhaA (acetyl-CoA
  acetyltransferase), PhaB1 (acetoacetyl-CoA reductase), PhaC (PHB synthase), PhaP1/phaP1
  (phasin 1), phaP3 (phasin 3), PhaP4/phaP4 (phasin 4). NAD(P)-transhydrogenases include
  PntAa3,2,1/pntAa2,3 [NAD(P) transhydrogenase subunits alpha 1 of gene regions 1
  to 3], pntAb1,2,3 [NAD(P) transhydrogenase subunits alpha 2 of gene regions 1 to
  3], and pntB3 [NAD(P) transhydrogenase subunit beta of gene region 3].'
papertitle: Proteomic and Transcriptomic Elucidation of the Mutant Ralstonia eutropha
  G+1 with Regard to Glucose Utilization  .
reftext: Matthias Raberg, et al. Appl Environ Microbiol. 2011 Mar;77(6):2058-2070.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9640684
figid_alias: PMC3067333__F5
figtype: Figure
redirect_from: /figures/PMC3067333__F5
ndex: 8e6b2cca-ded5-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3067333__zam9991019150005.html
  '@type': Dataset
  description: 'Schematic presentation of Nag PTS and parts of the central metabolism
    of R. eutropha H16 that are relevant for this study. The reddish gene abbreviations
    (small font, italics) indicate genes being upregulated in mutant G+1 as reported
    by transcriptome experiments. The reddish protein abbreviations (large font) indicate
    proteins being increasingly synthesized in mutant G+1 in comparison to wild-type
    H16, as revealed by proteome analyses. The same applies to the greenish gene/protein
    abbreviations, but these indicate downregulation or reduced synthesis by mutant
    G+1 in comparison to wild-type H16. Bluish letters are common abbreviations of
    trivial names of the respective enzymes. Nag PTS components are as follows: nagC
    (outer-membrane protein (porin), nagE (sugar phosphotransferase system, membrane
    component EIICB), nagF (sugar phosphotransferase system, cytosolic component EIIA-HPr-EI).
    KDPG pathway and subsequent reactions leading to pyruvate are as follows: G6PD/Zwf1
    (glucose-6-phosphate 1-dehydrogenase), 6PGL (6-phosphogluconolactonase), EDD/Edd1
    (phosphogluconate dehydratase), EDA (2-keto-3-deoxygluconate-6-phosphate aldolase),
    G3PD/CbbG1/cbbG1 (glyceraldehyde-3-phosphate dehydrogenase), PGM (phosphoglycerate
    mutase), ENO (enolase), PK (pyruvate kinase), PPS (phosphoenolpyruvate kinase).
    PDHC (pyruvate dehydrogenase multienzyme complex) includes pdhA1 (pyruvate dehydrogenase/decarboxylase
    [E1 of PDHC]), pdhB (dihydrolipoamide acetyltransferase [E2 of PDHC]), pdhL (dihydrolipoamide
    dehydrogenase [E3 of PDHC]). Involved in TCC and PHB metabolism are cisY (citrate
    synthase), PhaA (acetyl-CoA acetyltransferase), PhaB1 (acetoacetyl-CoA reductase),
    PhaC (PHB synthase), PhaP1/phaP1 (phasin 1), phaP3 (phasin 3), PhaP4/phaP4 (phasin
    4). NAD(P)-transhydrogenases include PntAa3,2,1/pntAa2,3 [NAD(P) transhydrogenase
    subunits alpha 1 of gene regions 1 to 3], pntAb1,2,3 [NAD(P) transhydrogenase
    subunits alpha 2 of gene regions 1 to 3], and pntB3 [NAD(P) transhydrogenase subunit
    beta of gene region 3].'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - porin
  - pts
  - PheRS-m
  - Pep
  - bgcn
  - Zw
  - Pgls
  - Pgk
  - Pgm1
  - Eno
  - Pdha
  - Pdhb
  - pps
  - PANC
  - HPR
  - PEP
  - PAS2
  - PHB
  - EDD1
  - GI
  - PGK
  - PGM
  - ERD5
  - MAX2
  - nagC
  - nagE
  - hlyE
  - ptsH
  - edd
  - eda
  - pgk
  - pgm
  - eno
  - ppsA
  - VDAC1
  - PTS
  - PAEP
  - PREP
  - PTPN22
  - G6PD
  - UBL4A
  - PGLS
  - ANP32A
  - PHB1
  - UBR5
  - EDA
  - GAPDH
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - VCAN
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - SFXN1
  - PDHA1
  - PDHB
  - IRF6
  - INPP5K
  - Acetoacetyl-CoA
  - (R-)3-Hydroxybutyryl-CoA
  - 6-Phosphogluconolactone
  - Gluconate-6-0
  - Glucose
  - Pyruvate Glyceraldehyde-3-0
  - Pyruvate
---
