---
figid: PMC3734176__pone.0070630.g005
figtitle: Transcriptional Blood Signatures Distinguish Pulmonary Tuberculosis, Pulmonary
  Sarcoidosis, Pneumonias and Lung Cancers
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3734176
filename: pone.0070630.g005.jpg
figlink: /pmc/articles/PMC3734176/figure/pone-0070630-g005/
number: F5
caption: 'Differentially expressed genes were derived from the Training Set by comparing
  each disease to healthy controls matched for ethnicity and gender: TB = 2524, active
  sarcoidosis = 1391, pneumonia = 2801 and lung cancer = 1626 transcripts (≥1.5 fold
  change from the mean of the controls, Mann Whitney with Benjamini Hochberg FDR = 0.01).
  (A) IPA canonical pathways was used to determined the most significant pathways
  (i-iv) associated with each disease relative to the other diseases (Fisher’s exact
  with Benjamini Hochberg FDR = 0.05). The bottom x-axis and bars of each graph indicates
  the log (p-value) and the top x-axis and line indicates the percentage of genes
  present in the pathway. The genes in the EIF2 signalling pathway are predominately
  under-abundant genes however the genes in the other three pathways are predominantly
  over-abundant relative to the controls. Pathways above the blue dotted line are
  significant (p<0.05). (B) The IFN signalling IPA pathway is overlaid onto each disease
  group. Coloured genes are differentially expressed in that disease group compared
  to their matched controls (Fisher’s exact FDR = 0.05). Red genes represent over-abundance
  and green under-abundance. The pathway for TB is shown enlarged so the detail of
  the genes can be seen, it is also shown again in a much smaller scale besides the
  other diseases so that a visual comparison can be more easily made.'
papertitle: Transcriptional Blood Signatures Distinguish Pulmonary Tuberculosis, Pulmonary
  Sarcoidosis, Pneumonias and Lung Cancers.
reftext: Chloe I. Bloom, et al. PLoS One. 2013;8(8):e70630.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5947047
figid_alias: PMC3734176__F5
figtype: Figure
redirect_from: /figures/PMC3734176__F5
ndex: e4e6f6aa-df2f-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3734176__pone.0070630.g005.html
  '@type': Dataset
  description: 'Differentially expressed genes were derived from the Training Set
    by comparing each disease to healthy controls matched for ethnicity and gender:
    TB = 2524, active sarcoidosis = 1391, pneumonia = 2801 and lung cancer = 1626
    transcripts (≥1.5 fold change from the mean of the controls, Mann Whitney with
    Benjamini Hochberg FDR = 0.01). (A) IPA canonical pathways was used to determined
    the most significant pathways (i-iv) associated with each disease relative to
    the other diseases (Fisher’s exact with Benjamini Hochberg FDR = 0.05). The bottom
    x-axis and bars of each graph indicates the log (p-value) and the top x-axis and
    line indicates the percentage of genes present in the pathway. The genes in the
    EIF2 signalling pathway are predominately under-abundant genes however the genes
    in the other three pathways are predominantly over-abundant relative to the controls.
    Pathways above the blue dotted line are significant (p<0.05). (B) The IFN signalling
    IPA pathway is overlaid onto each disease group. Coloured genes are differentially
    expressed in that disease group compared to their matched controls (Fisher’s exact
    FDR = 0.05). Red genes represent over-abundance and green under-abundance. The
    pathway for TB is shown enlarged so the detail of the genes can be seen, it is
    also shown again in a much smaller scale besides the other diseases so that a
    visual comparison can be more easily made.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tb
  - eIF2alpha
  - eIF2beta
  - Stat92E
  - bh
  - kst
  - gas
  - Galphas
  - gd
  - Ne
  - EIF2S1
  - EIF2S3
  - EIF2S2
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - IFNA1
  - GAST
  - GALNS
  - PAGR1
  - FURIN
---
