---
figid: PMC3812160__pone.0078471.g007
figtitle: Transcriptome-Wide Mapping of Pea Seed Ageing Reveals a Pivotal Role for
  Genes Related to Oxidative Stress and Programmed Cell Death
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3812160
filename: pone.0078471.g007.jpg
figlink: /pmc/articles/PMC3812160/figure/pone-0078471-g007/
number: F7
caption: 'Differentially expressed PCD-related genes were mapped onto the model proposed
  by Kranner et al.  (solid arrows). ROS formed during ageing (1) are scavenged by
  GSH leading to an increase in E GSSG/2GSH (2), which causes the apoptosis signal-regulating
  kinase (ASK 1) to be split from thioredoxin (Trx; TRX-H; 3). ASK1 activates a MAPK
  cascade (4) leading to changes in gene expression (5). ROS may also participate
  directly in signalling pathways to activate MAP kinase cascades (dotted arrow).
  Ca2+ is released from the ER (6), which induces release of cyt c from the mitochondrion
  (7). Cyt c activates caspases (8) and a downstream proteolytic cascade (9) which
  leads to the breakdown of structural and nuclear proteins and activation of nucleases
  which cleave DNA (10) as part of the execution phase of PCD leading to cell death
  (11). ROS cause protein oxidation, and the damaged and mis-folded (due to ER stress)
  proteins (P) are degraded by the ubiquitin-proteasome pathway (dashed arrows). The
  ubiquitin-proteasome has been linked with activation of the caspase proteolytic
  cascade and some proteasome subunits have caspase-like activity, which may offer
  an alternative route to PCD. The coloured blocks on the left-hand side indicate
  the ratio of expression of genes involved in the model at three different ageing
  time points (from left to right: 8, 12 and 15 days) to expression in non-aged seeds.
  Expression values corresponding to a log2 ratio ≥1.0 or ≤−1.0 are shaded in light
  magenta (up-regulation) or light green (down-regulation), and those which were changed
  by a log2 ratio of ≥1.5 or ≤−1.5 are shaded in dark magenta (up-regulation) or dark
  green (down-regulation). White blocks indicate no change in expression between aged
  and non-aged seed.'
papertitle: Transcriptome-Wide Mapping of Pea Seed Ageing Reveals a Pivotal Role for
  Genes Related to Oxidative Stress and Programmed Cell Death.
reftext: Hongying Chen, et al. PLoS One. 2013;8(10):e78471.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7702861
figid_alias: PMC3812160__F7
figtype: Figure
redirect_from: /figures/PMC3812160__F7
ndex: 6d9328e2-df2b-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3812160__pone.0078471.g007.html
  '@type': Dataset
  description: 'Differentially expressed PCD-related genes were mapped onto the model
    proposed by Kranner et al.  (solid arrows). ROS formed during ageing (1) are scavenged
    by GSH leading to an increase in E GSSG/2GSH (2), which causes the apoptosis signal-regulating
    kinase (ASK 1) to be split from thioredoxin (Trx; TRX-H; 3). ASK1 activates a
    MAPK cascade (4) leading to changes in gene expression (5). ROS may also participate
    directly in signalling pathways to activate MAP kinase cascades (dotted arrow).
    Ca2+ is released from the ER (6), which induces release of cyt c from the mitochondrion
    (7). Cyt c activates caspases (8) and a downstream proteolytic cascade (9) which
    leads to the breakdown of structural and nuclear proteins and activation of nucleases
    which cleave DNA (10) as part of the execution phase of PCD leading to cell death
    (11). ROS cause protein oxidation, and the damaged and mis-folded (due to ER stress)
    proteins (P) are degraded by the ubiquitin-proteasome pathway (dashed arrows).
    The ubiquitin-proteasome has been linked with activation of the caspase proteolytic
    cascade and some proteasome subunits have caspase-like activity, which may offer
    an alternative route to PCD. The coloured blocks on the left-hand side indicate
    the ratio of expression of genes involved in the model at three different ageing
    time points (from left to right: 8, 12 and 15 days) to expression in non-aged
    seeds. Expression values corresponding to a log2 ratio ≥1.0 or ≤−1.0 are shaded
    in light magenta (up-regulation) or light green (down-regulation), and those which
    were changed by a log2 ratio of ≥1.5 or ≤−1.5 are shaded in dark magenta (up-regulation)
    or dark green (down-regulation). White blocks indicate no change in expression
    between aged and non-aged seed.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PHGPx
  - put
  - mit
  - Tom40
  - Sod2
  - RpS13
  - RpS15
  - Cp15
  - RpS15Aa
  - GstS1
  - GstE12
  - GstD1
  - GstD2
  - GstD3
  - GstD4
  - GstD5
  - GstD6
  - GstD7
  - GstD8
  - Jafrac1
  - ant
  - sesB
  - Antp
  - ems
  - Cyt-c-d
  - porin
  - dh
  - mAcon1
  - svr
  - Hk
  - Hex-A
  - su(sable)
  - trx
  - Trxr-1
  - Trx-2
  - Sh
  - Pcd
  - BI-1
  - ox
  - MKP-4
  - p38b
  - rl
  - scf
  - CG8188
  - Ubc10
  - Ubc84D
  - Ubc87F
  - E3
  - Skp2
  - CSN1a
  - bip
  - Hsc70-3
  - CG11700
  - Vha14-1
  - Taf3
  - GPX1
  - GPX2
  - GPX3
  - GPX4
  - GPX5
  - GPX6
  - GPX7
  - GPX8
  - SOD2
  - EFEMP1
  - RPS15
  - GLRX
  - SLCO6A1
  - GSTK1
  - GSTA1
  - GSTA2
  - GSTA3
  - GSTA4
  - GSTA5
  - GSTM1
  - GSTM2
  - GSTM3
  - GSTM4
  - GSTM5
  - GSTO1
  - GSTO2
  - GSTP1
  - GSTT1
  - GSTT2
  - GSTZ1
  - GSTT2B
  - PRX
  - PRDX6
  - ADH5
  - SLC25A4
  - SLC25A6
  - VDAC1
  - VDAC2
  - VDAC3
  - OMA1
  - PPID
  - PPIF
  - CWC27
  - PPIA
  - PPIB
  - PPIC
  - PPIE
  - PPIH
  - PPIL1
  - PPIL2
  - PPIL3
  - PPIL4
  - PPIL6
  - PPWD1
  - PPIAL4A
  - PPIAL4C
  - PPIAL4G
  - HK1
  - GCK
  - HK2
  - HK3
  - TXN
  - VAC14
  - DBF4
  - DNAH5
  - PCBD1
  - DNAI1
  - DNAAF3
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - TMBIM6
  - PNMA3
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - KITLG
  - UBC
  - RPS27A
  - LRSAM1
  - COPS8
  - UBE2K
  - GDF10
  - HSPA5
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - DNAH8
  - ERVK-8
  - ERVK-10
  - ERVK-9
  - ERVK-21
  - ERVK-18
  - ERVK-25
  - ERVK-24
  - ERVK-19
  - O 14-3-3
  - Pyruvate
  - O Sucrose
  - GSSG
---
