---
figid: PMC3313080__nihms-343162-f0002
figtitle: Endoplasmic Reticulum Stress, the Unfolded Protein Response, Autophagy,
  and the Integrated Regulation of Breast Cancer Cell Fate
organisms:
- NA
pmcid: PMC3313080
filename: nihms-343162-f0002.jpg
figlink: /pmc/articles/PMC3313080/figure/F2/
number: F2
caption: A. The UPR is an adaptive signaling pathway where the proximal activators
  of each of its three arms (PERK, ATF6, IRE1α) are normally activated following their
  dissociation from HSPA5 (GRP78; BiP). When released from HSPA5 the N-terminal luminal
  domains of two PERK proteins bind together (), the resulting dimer undergoing an
  activating autophosphorylation; phosphorylation of PERK tyrosine-615 is a key event
  (). PERK is a type I transmembrane protein that phosphorylates the eukaryotic translation
  initiation factor 2 α-subunit (eIF2α) (). Phosphorylation of eIF2α at serine-51
  blocks translational initiation () because, as a dominant negative inhibitor of
  eIF2β, the EI2F recycling required for further protein biosynthesis is blocked and
  the rate of protein biosynthesis is reduced (). Downstream events include the induction
  of ATF4 that then regulates expression of several genes including the proapoptotic
  DNA damage-inducible transcript 3 (DDI3; also known as CHOP or GADD153) (). The
  two mammalian ATF6 alleles (ATF6α; ATF6β) encode a type II transmembrane bZIP transcription
  factor. HSPA5 blocks two Golgi localization signals that are exposed upon its dissociation
  from ATF6 (). Following translocation to the Golgi, regulated intramembrane proteolysis
  by the site-1 (S1P) and -2 proteases (S2P) cleave ATF6α to its active p50 form (ATF6β
  plays only a minor role in the UPR). ATF6 p50 then enters the cytosol, translocates
  to the nucleus, and activates transcription in cooperation with the general transcription
  factor nuclear factor-Y (NF-Y) (,). Among the key genes regulated by ATF6 p50 are
  XBP1 (spliced in the IRE1α pathway) (), DDI3 (also induced in the PERK pathway)
  () and HSPA5 (regulates all three pathways ()). Activation of IRE1α and splicing
  of HAC1 (yeast) and XBP1, is the oldest and most conserved pathway for UPR signaling.
  Downstream targets of XBP1(S) include p58IPK and several UPR chaperones (). p58IPK
  represses PERK activity (). Thus, persistent XBP1(S) production in the face of continued
  EnR stress could shift UPR signaling from PERK to favor IRE1α and/or the integration
  of ATF6 (though increasing XBP1 transcription) and IRE1α (through increased XBP1
  splicing) signaling.B. Two stem-loop structures, each containing a highly conserved
  CNGNNG motif, are cleaved. How the two exons are ligated in mammalian cells remains
  unclear; this function likely being different from that described for yeast ().
  XBP1 splicing may not be exclusively cytosolic but this remains controversial ().
  Other nucleotide substrates may exist for IRE1α but none is known to possess the
  stem-loop structures evident in XBP1 and HAC1.C. UPR modulates crosstalk between
  autophagy and apoptosis through various mechanisms. Stimulation of UPR results in
  an increase in CHOP that promotes apoptosis. Moreover, IRE1α activation promotes
  apoptosis by phosphorylation of JNK, directly and indirectly inactivating anti-apoptotic
  BCL2 proteins. UPR release of EnR Ca2+ also directly promotes apoptosis. UPR signaling
  also stimulates autophagy. Activation of PERK and the resulting phosphorylation
  of eIF2α promote autophagy through ATF4-mediated Atg12 transcription. Furthermore,
  IRE1α-mediated activation of JNK and the subsequent phosphorylation of BCL2 results
  in dissociation of the BCL2/BECN1 complex, promoting autophagy.D. Low intracellular
  glucose concentrations result in the accumulation of unfolded proteins, stimulating
  the release of the three UPR signaling arms (PERK, IRE1, and ATF6) by HSPA5, and
  activating the UPR. UPR signaling can activate autophagy resulting in increased
  degradation of cellular material and release of peptides, amino-acids, and fatty
  acids. During metabolic stress, a major role of the peroxisome proliferator-activated
  receptors (PPARs) is likely their ability to ensure adequate turn-over of peroxisomes
  to manage a greater metabolic requirement for release of the energy stored in the
  longer chain fatty acids. Autophagy degradation byproducts (amino acids, carbohydrates
  and short chain fatty acids) promote the TCA cycle and the corresponding generation
  of ATP. Formation of ATP by mitochondria, using the raw material provided by autophagy,
  enables the cell to cope with the low glucose levels and promotes survival.
papertitle: Endoplasmic Reticulum Stress, the Unfolded Protein Response, Autophagy,
  and the Integrated Regulation of Breast Cancer Cell Fate.
reftext: Robert Clarke, et al. Cancer Res. ;72(6):1321-1331.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7380665
figid_alias: PMC3313080__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC3313080__F2
ndex: b4fd61b8-df30-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3313080__nihms-343162-f0002.html
  '@type': Dataset
  description: A. The UPR is an adaptive signaling pathway where the proximal activators
    of each of its three arms (PERK, ATF6, IRE1α) are normally activated following
    their dissociation from HSPA5 (GRP78; BiP). When released from HSPA5 the N-terminal
    luminal domains of two PERK proteins bind together (), the resulting dimer undergoing
    an activating autophosphorylation; phosphorylation of PERK tyrosine-615 is a key
    event (). PERK is a type I transmembrane protein that phosphorylates the eukaryotic
    translation initiation factor 2 α-subunit (eIF2α) (). Phosphorylation of eIF2α
    at serine-51 blocks translational initiation () because, as a dominant negative
    inhibitor of eIF2β, the EI2F recycling required for further protein biosynthesis
    is blocked and the rate of protein biosynthesis is reduced (). Downstream events
    include the induction of ATF4 that then regulates expression of several genes
    including the proapoptotic DNA damage-inducible transcript 3 (DDI3; also known
    as CHOP or GADD153) (). The two mammalian ATF6 alleles (ATF6α; ATF6β) encode a
    type II transmembrane bZIP transcription factor. HSPA5 blocks two Golgi localization
    signals that are exposed upon its dissociation from ATF6 (). Following translocation
    to the Golgi, regulated intramembrane proteolysis by the site-1 (S1P) and -2 proteases
    (S2P) cleave ATF6α to its active p50 form (ATF6β plays only a minor role in the
    UPR). ATF6 p50 then enters the cytosol, translocates to the nucleus, and activates
    transcription in cooperation with the general transcription factor nuclear factor-Y
    (NF-Y) (,). Among the key genes regulated by ATF6 p50 are XBP1 (spliced in the
    IRE1α pathway) (), DDI3 (also induced in the PERK pathway) () and HSPA5 (regulates
    all three pathways ()). Activation of IRE1α and splicing of HAC1 (yeast) and XBP1,
    is the oldest and most conserved pathway for UPR signaling. Downstream targets
    of XBP1(S) include p58IPK and several UPR chaperones (). p58IPK represses PERK
    activity (). Thus, persistent XBP1(S) production in the face of continued EnR
    stress could shift UPR signaling from PERK to favor IRE1α and/or the integration
    of ATF6 (though increasing XBP1 transcription) and IRE1α (through increased XBP1
    splicing) signaling.B. Two stem-loop structures, each containing a highly conserved
    CNGNNG motif, are cleaved. How the two exons are ligated in mammalian cells remains
    unclear; this function likely being different from that described for yeast ().
    XBP1 splicing may not be exclusively cytosolic but this remains controversial
    (). Other nucleotide substrates may exist for IRE1α but none is known to possess
    the stem-loop structures evident in XBP1 and HAC1.C. UPR modulates crosstalk between
    autophagy and apoptosis through various mechanisms. Stimulation of UPR results
    in an increase in CHOP that promotes apoptosis. Moreover, IRE1α activation promotes
    apoptosis by phosphorylation of JNK, directly and indirectly inactivating anti-apoptotic
    BCL2 proteins. UPR release of EnR Ca2+ also directly promotes apoptosis. UPR signaling
    also stimulates autophagy. Activation of PERK and the resulting phosphorylation
    of eIF2α promote autophagy through ATF4-mediated Atg12 transcription. Furthermore,
    IRE1α-mediated activation of JNK and the subsequent phosphorylation of BCL2 results
    in dissociation of the BCL2/BECN1 complex, promoting autophagy.D. Low intracellular
    glucose concentrations result in the accumulation of unfolded proteins, stimulating
    the release of the three UPR signaling arms (PERK, IRE1, and ATF6) by HSPA5, and
    activating the UPR. UPR signaling can activate autophagy resulting in increased
    degradation of cellular material and release of peptides, amino-acids, and fatty
    acids. During metabolic stress, a major role of the peroxisome proliferator-activated
    receptors (PPARs) is likely their ability to ensure adequate turn-over of peroxisomes
    to manage a greater metabolic requirement for release of the energy stored in
    the longer chain fatty acids. Autophagy degradation byproducts (amino acids, carbohydrates
    and short chain fatty acids) promote the TCA cycle and the corresponding generation
    of ATP. Formation of ATP by mitochondria, using the raw material provided by autophagy,
    enables the cell to cope with the low glucose levels and promotes survival.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ire1
  - crc
  - Atf6
  - PEK
  - rl
  - GC
  - Au
  - bsk
  - Low
  - Itpr
  - eIF2alpha
  - eIF2A
  - unk
  - Atg12
  - cop
  - Yif1
  - CycE
  - cyc
  - ATPsynbeta
  - Atpalpha
  - ERN1
  - ATF6
  - EIF2AK3
  - ATF4
  - MAPK8
  - MAPK9
  - MAPK10
  - IGF1
  - ITPR1
  - ITPR3
  - EIF2A
  - EIF2S1
  - UNK
  - LARGE1
  - ATG12
  - DEFB4A
  - DDIT3
  - ATP8A2
  - Carbohydrates
  - fatty acids
  - glucose
---
