---
figid: PMC3076503__ijbsv07p0308g05
figtitle: Bioenergenesis neurotransmitter pathways with differentially expressed genes
  between frontal cortex and hippocampus
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Middle East respiratory syndrome-related coronavirus
- Merluccius merluccius
pmcid: PMC3076503
filename: ijbsv07p0308g05.jpg
figlink: /pmc/articles/PMC3076503/figure/F4/
number: F4
caption: Bioenergenesis neurotransmitter pathways with differentially expressed genes
  between frontal cortex and hippocampus. A solid line with arrowhead indicates a
  direct reaction or transport. A dashed line with arrowhead indicates more than one
  steps involved in passages. The RNA levels of genes in red or green colors were
  expressed significantly higher or lower in the frontal cortex than in hippocampus,
  respectively. If the changes in RNA levels were not significant (p>0.05) or less
  than 1.25-fold, the gene symbols or enzymes are not shown in the flow chat. The
  mean expression levels, standard deviations, fold changes, p-values, false discovery
  rate (FDR) and the full name of genes are listed in Table . (A) Bioenergenesis neurotransmitter
  pathways include glycolysis, fatty acid synthesis, and neurotransmitter synthesis
  in cytoplasm and β-oxidation, TCA cycle and OXPHOS, REDOX, and neurotransmitter
  synthesis in mitochondria. Double dashed lines represent the inner and outer mitochondrial
  membranes, of which a portion was enlarged to illustrate OXPHOS complexes I, II,
  III, IV and V with differentially expressed genes in each complex. A proton (H+)
  flow and ATP synthesis are indicated. REDOX enzymes (GPX1, GSTZ1 and PRDX1) are
  involved in removal of ROS (e.g. O2-). Mitochondrial and cytoplasmic enzymes with
  RNA levels higher in the frontal cortex than in hippocampus for synthesis of neurotransmitters
  (gray ovals) include Glud1 for synthesis of the most abundant excitatory neurotransmitter
  L-glutamate, GAD1 and ABAT for the most abundant inhibitory neurotransmitter γ-aminobutyric
  acid [GABA], DDC for dopamine, histamine and serotonin, and AGXT for serine and
  glycine. SLC25A18 and SLC25A22 are involved in transport of L-glutamate across mitochondrial
  membranes. (B) Insulin signaling pathway. Insulin binding to its receptor results
  in the tyrosine phosphorylation of insulin receptor substrates. This leads to activation
  of phosphoinositide-3-kinase (PIK3CB) and growth factor receptor-bound protein (GRB2).
  These in turn lead to upregulation of insulin-responsive genes including hexokinase
  2 (HK2) and MAP kinase interacting serine/threonine kinase 1 (MKNK1). HK2 at the
  outer mitochondrial membrane phosphorylates glucose into glucose-6-phosphate, the
  first rate-limiting step of glycolysis pathway. MKNK1 in cytoplasm phosphorylates
  the eukaryotic translation initiation factor (EEIF4E) initiating protein synthesis,
  of which 18 of 25 genes displayed RNA levels higher in the frontal cortex than in
  hippocampus (Table ). MKNK1 also modifies proteins posttranslationally. ELK1 member
  of ETS oncogene family (ELK1) plays a critical role in mitogen growth factor signal
  transduction, and its downregulation suggests decreased activities in proliferation
  and differentiation. (C) PPARD signaling pathway. Peroxisome proliferator-activated
  receptor delta (PPARD) is a nuclear hormone receptor and transcriptional regulator.
  Retinoid X receptor alpha (RXRA) is a steroid and thyroid hormone receptor and transcriptional
  regulator. Both PPARD and RXRA are involved in fatty acid transport and oxidation
  via binding to DNA and regulating transcription.
papertitle: Rat Mitochondrion-Neuron Focused Microarray (rMNChip) and Bioinformatics
  Tools for Rapid Identification of Differential Pathways in Brain Tissues.
reftext: Yan A. Su, et al. Int J Biol Sci. 2011;7(3):308-322.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8737698
figid_alias: PMC3076503__F4
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
redirect_from: /figures/PMC3076503__F4
ndex: af7e8081-dea3-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3076503__ijbsv07p0308g05.html
  '@type': Dataset
  description: Bioenergenesis neurotransmitter pathways with differentially expressed
    genes between frontal cortex and hippocampus. A solid line with arrowhead indicates
    a direct reaction or transport. A dashed line with arrowhead indicates more than
    one steps involved in passages. The RNA levels of genes in red or green colors
    were expressed significantly higher or lower in the frontal cortex than in hippocampus,
    respectively. If the changes in RNA levels were not significant (p>0.05) or less
    than 1.25-fold, the gene symbols or enzymes are not shown in the flow chat. The
    mean expression levels, standard deviations, fold changes, p-values, false discovery
    rate (FDR) and the full name of genes are listed in Table . (A) Bioenergenesis
    neurotransmitter pathways include glycolysis, fatty acid synthesis, and neurotransmitter
    synthesis in cytoplasm and β-oxidation, TCA cycle and OXPHOS, REDOX, and neurotransmitter
    synthesis in mitochondria. Double dashed lines represent the inner and outer mitochondrial
    membranes, of which a portion was enlarged to illustrate OXPHOS complexes I, II,
    III, IV and V with differentially expressed genes in each complex. A proton (H+)
    flow and ATP synthesis are indicated. REDOX enzymes (GPX1, GSTZ1 and PRDX1) are
    involved in removal of ROS (e.g. O2-). Mitochondrial and cytoplasmic enzymes with
    RNA levels higher in the frontal cortex than in hippocampus for synthesis of neurotransmitters
    (gray ovals) include Glud1 for synthesis of the most abundant excitatory neurotransmitter
    L-glutamate, GAD1 and ABAT for the most abundant inhibitory neurotransmitter γ-aminobutyric
    acid [GABA], DDC for dopamine, histamine and serotonin, and AGXT for serine and
    glycine. SLC25A18 and SLC25A22 are involved in transport of L-glutamate across
    mitochondrial membranes. (B) Insulin signaling pathway. Insulin binding to its
    receptor results in the tyrosine phosphorylation of insulin receptor substrates.
    This leads to activation of phosphoinositide-3-kinase (PIK3CB) and growth factor
    receptor-bound protein (GRB2). These in turn lead to upregulation of insulin-responsive
    genes including hexokinase 2 (HK2) and MAP kinase interacting serine/threonine
    kinase 1 (MKNK1). HK2 at the outer mitochondrial membrane phosphorylates glucose
    into glucose-6-phosphate, the first rate-limiting step of glycolysis pathway.
    MKNK1 in cytoplasm phosphorylates the eukaryotic translation initiation factor
    (EEIF4E) initiating protein synthesis, of which 18 of 25 genes displayed RNA levels
    higher in the frontal cortex than in hippocampus (Table ). MKNK1 also modifies
    proteins posttranslationally. ELK1 member of ETS oncogene family (ELK1) plays
    a critical role in mitogen growth factor signal transduction, and its downregulation
    suggests decreased activities in proliferation and differentiation. (C) PPARD
    signaling pathway. Peroxisome proliferator-activated receptor delta (PPARD) is
    a nuclear hormone receptor and transcriptional regulator. Retinoid X receptor
    alpha (RXRA) is a steroid and thyroid hormone receptor and transcriptional regulator.
    Both PPARD and RXRA are involved in fatty acid transport and oxidation via binding
    to DNA and regulating transcription.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Wdtc1
  - Mdga2
  - Acsl3
  - Acsl6
  - Cpt1a
  - Eci2
  - Fancd2
  - Gapdh
  - Atp5f1e
  - Hps3
  - Acaa2
  - Gpx1
  - Gstz1
  - Mipep
  - Atp6v1e1
  - Atpaf2
  - Ppa2
  - Cs
  - tp
  - Cish
  - Cox6c
  - Cox7a2
  - Cox15
  - Gnat2
  - Abat
  - Uqcrfs1
  - Sdhb
  - Sdhc
  - Ndufa1
  - Ndufa4
  - Ndufa10
  - Ndufaf1
  - Ndufb10
  - Ndufs2
  - Sucla2
  - Ogdh
  - Gad1
  - Ntan1
  - Slc25a18
  - Ddc
  - Hk2
  - Pkm
  - Prickle3
  - Prkcb
  - Mknk1
  - Grb2
  - Elk1
  - Kcnh8
  - Agxt
  - Hbb-bt
  - ba
  - Baat
  - Ppard
  - Rxra
  - ATP8A2
  - WDTC1
  - ACSL3
  - ADH6
  - ACSL6
  - CPT1A
  - ECI2
  - BRCA2
  - FANCD2
  - PSEN1
  - GAPDH
  - GAPDHP44
  - ATP5F1E
  - ACAA2
  - GPX1
  - GSTZ1
  - MIPEP
  - ATP6V1E1
  - ATPAF2
  - PPA2
  - CS
  - CISH
  - COX6C
  - COX7A2
  - COX15
  - ABAT
  - UQCRFS1
  - SDHB
  - SDHC
  - LOC642502
  - NDUFA1
  - NDUFA4
  - NDUFA4P1
  - NDUFA10
  - NDUFAF1
  - NDUFB10
  - NDUFS2
  - MTG1
  - SUCLA2
  - OGDH
  - GAD1
  - NTAN1
  - SLC25A18
  - DDC
  - HK2
  - KCNA5
  - KIF2A
  - KLK2
  - HOOK2
  - PRICKLE3
  - PKM
  - PRKCB
  - MKNK1
  - GRB2
  - ELK1
  - KCNH4
  - KCNH8
  - AGXT
  - BAAT
  - PPARD
  - RXRA
  - Svs4
  - Adh6
  - Ccne1
  - Cox7a2l2
  - Ndufa10l1
  - Pck1
  - 3-phosphoglycerate -----> pyruvate
  - 1,3-biphosphoglycerate
  - Acetaldehyde
  - Acetal
  - Aconitate
  - Ethanol
  - Aldehyde
  - Dopamine
  - FAD
  - Fatty acid
  - Fructose
  - Fumarate
  - GABA
  - Glucose
  - Glyceraldehyde-3-P
  - Glycine
  - Glyoxylate
  - Histamine
  - L-Histidine
  - Isocitrate
  - L-DOPA
  - Oxaloacetate
  - Hexadecanoyl-CoA
  - Pyruvate
  - Serine
  - Serotonin
  - Succinate
  - L-Tryptophan
  - L-glutamate
  - GSSG
---
