---
figid: PMC3739039__958f3
figtitle: Integrated “omic” analysis reveals global metabolic changes
organisms:
- Mus musculus
- Homo sapiens
pmcid: PMC3739039
filename: 958f3.jpg
figlink: /pmc/articles/PMC3739039/figure/F3/
number: F3
caption: Integrated “omic” analysis reveals global metabolic changes. Kyoto Encyclopedia
  of Genes and Genomes “search&color pathway” analysis for phosphoproteins and metabolites
  overlaid on the human metabolic reference map (hsa01100). Phosphoproteomic data
  from one biological replicate of the SU-DHL-1 were used, whereas metabolomic data
  from 4 averaged biological replicates of SU-DHL-1 cells were used. Metabolites that
  changed (Student t test, P < .05) in response to CEP treatment were used. Green
  and red represent decrease and increase, respectively. Lines represent phosphoprotein
  and dots represent metabolites. Blue shading over glycolysis, TCA cycle, and nucleotide
  metabolism serve to highlight pathways that are highly represented.
papertitle: Integrated phosphoproteomic and metabolomic profiling reveals NPM-ALK–mediated
  phosphorylation of PKM2 and metabolic reprogramming in anaplastic large cell lymphoma.
reftext: Scott R. P. McDonnell, et al. Blood. 2013 Aug 8;122(6):958-968.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9112304
figid_alias: PMC3739039__F3
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
redirect_from: /figures/PMC3739039__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3739039__958f3.html
  '@type': Dataset
  description: Integrated “omic” analysis reveals global metabolic changes. Kyoto
    Encyclopedia of Genes and Genomes “search&color pathway” analysis for phosphoproteins
    and metabolites overlaid on the human metabolic reference map (hsa01100). Phosphoproteomic
    data from one biological replicate of the SU-DHL-1 were used, whereas metabolomic
    data from 4 averaged biological replicates of SU-DHL-1 cells were used. Metabolites
    that changed (Student t test, P < .05) in response to CEP treatment were used.
    Green and red represent decrease and increase, respectively. Lines represent phosphoprotein
    and dots represent metabolites. Blue shading over glycolysis, TCA cycle, and nucleotide
    metabolism serve to highlight pathways that are highly represented.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATIC
  - APRT
  - MFAP1
  - UMPS
  - MATN1
  - TKT
  - DDR2
  - TPI1
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - PKM
  - PKLR
  - PDHA1
  - LDHA
  - LDHB
  - LDHC
  - Atic
  - Umps
  - Matn1
  - Abcg5
  - Rps12-ps24
  - Tkt
  - Tpi1
  - Pkm
  - Gnat2
  - Adenosine
  - CMP Cytasine
  - Formate
  - Guanine
  - Glvcerone phosphate
  - Succinate
  - Taurine
  - Lactate
---
