---
figid: PMC3492890__nihms416467f2
figtitle: Metabolic Regulation in Pluripotent Stem Cells during Reprogramming and
  Self-Renewal
organisms:
- Homo sapiens
- Mus musculus
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC3492890
filename: nihms416467f2.jpg
figlink: /pmc/articles/PMC3492890/figure/F2/
number: F2
caption: Glucose enters glycolysis (orange panel) and is metabolized to lactate. The
  expression and activity of glycolytic regulating enzymes are controlled by mTOR
  and PI3K signaling pathways (crosstalk). Reprogramming factors c-MYC and LIN28 promote
  glucose metabolism and c-MYC also suppresses respiration (crosstalk). Alternatively,
  pyruvate can be converted into two molecules of acetyl-CoA and enter the TCA cycle.
  Acetyl-CoA can traffic to the nucleus as a substrate for acetyltransferase enzymes
  (crosstalk). TCA cycle intermediates can be siphoned into amino acid or lipid biosynthesis
  pathways, or can participate in the control of gene expression. For example, α-ketoglutarate
  (α-KG) is a cofactor for Jumonji-domain containing histone demethylases (crosstalk).
  Methionine adenosyltransferase, an enzyme in amino acid biosynthesis, also converts
  methionine and ATP into S-adenosylmethionine (SAM) for DNA methyltransferases and
  other transmethylation reactions (crosstalk). The NAD+/NADH ratio reflects cellular
  redox state and regulates deacetylase enzymes, such as the Sirtuins (crosstalk).
  Reactive oxygen species (ROS) are required for lineage-specific differentiation
  and are regulated by the p38 MAPK stress signaling pathway (crosstalk). Lipid biosynthesis
  supports membrane formation but also provides signaling molecules including arachidonic
  acid (AA) and docosahexanoic acid (DHA) (crosstalk). Altogether, metabolites, the
  redox environment, and metabolism-regulated signaling cascades communicate with
  the nucleus (grey panel) to directly impact chromatin structure and gene expression
  programs that may control PSC self-renewal, differentiation, and reprogramming.
  (Blue buttons indicate nucleus and signaling control of metabolic pathways; orange
  buttons indicate metabolic pathways that influence gene expression changes within
  the nucleus.)
papertitle: Metabolic Regulation in Pluripotent Stem Cells during Reprogramming and
  Self-Renewal.
reftext: Jin Zhang, et al. Cell Stem Cell. ;11(5):589-595.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8378292
figid_alias: PMC3492890__F2
figtype: Figure
redirect_from: /figures/PMC3492890__F2
ndex: a40ac5a3-df23-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3492890__nihms416467f2.html
  '@type': Dataset
  description: Glucose enters glycolysis (orange panel) and is metabolized to lactate.
    The expression and activity of glycolytic regulating enzymes are controlled by
    mTOR and PI3K signaling pathways (crosstalk). Reprogramming factors c-MYC and
    LIN28 promote glucose metabolism and c-MYC also suppresses respiration (crosstalk).
    Alternatively, pyruvate can be converted into two molecules of acetyl-CoA and
    enter the TCA cycle. Acetyl-CoA can traffic to the nucleus as a substrate for
    acetyltransferase enzymes (crosstalk). TCA cycle intermediates can be siphoned
    into amino acid or lipid biosynthesis pathways, or can participate in the control
    of gene expression. For example, α-ketoglutarate (α-KG) is a cofactor for Jumonji-domain
    containing histone demethylases (crosstalk). Methionine adenosyltransferase, an
    enzyme in amino acid biosynthesis, also converts methionine and ATP into S-adenosylmethionine
    (SAM) for DNA methyltransferases and other transmethylation reactions (crosstalk).
    The NAD+/NADH ratio reflects cellular redox state and regulates deacetylase enzymes,
    such as the Sirtuins (crosstalk). Reactive oxygen species (ROS) are required for
    lineage-specific differentiation and are regulated by the p38 MAPK stress signaling
    pathway (crosstalk). Lipid biosynthesis supports membrane formation but also provides
    signaling molecules including arachidonic acid (AA) and docosahexanoic acid (DHA)
    (crosstalk). Altogether, metabolites, the redox environment, and metabolism-regulated
    signaling cascades communicate with the nucleus (grey panel) to directly impact
    chromatin structure and gene expression programs that may control PSC self-renewal,
    differentiation, and reprogramming. (Blue buttons indicate nucleus and signaling
    control of metabolic pathways; orange buttons indicate metabolic pathways that
    influence gene expression changes within the nucleus.)
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ros1
  - Jarid2
  - Myc
  - Nol3
  - Lin28a
  - Snhg15
  - H2-Aa
  - Mtor
  - Pik3r1
  - Gnat2
  - Crk
  - Grap2
  - Syp
  - Mapk14
  - Ahsa1
  - BMPR2
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - ASH1L
  - DOT1L
  - EHMT1
  - EHMT2
  - EZH1
  - EZH2
  - KMT2A
  - KMT2B
  - KMT2C
  - KMT2D
  - KMT2E
  - KMT5A
  - KMT5B
  - KMT5C
  - MECOM
  - NSD1
  - NSD2
  - NSD3
  - PRDM16
  - PRDM2
  - PRDM6
  - PRDM8
  - PRDM9
  - SETD1A
  - SETD1B
  - SETD2
  - SETD7
  - SETDB1
  - SETDB2
  - SMYD1
  - SMYD2
  - SMYD3
  - SUV39H1
  - SUV39H2
  - DNMT1
  - DNMT3A
  - DNMT3B
  - DNMT3L
  - CARM1
  - PRMT1
  - PRMT2
  - PRMT3
  - PRMT5
  - PRMT6
  - PRMT7
  - PRMT8
  - PRMT9
  - ALKBH8
  - TRMT1
  - TRMT10A
  - TRMT10B
  - TRMT10C
  - TRMT11
  - TRMT112
  - TRMT12
  - TRMT13
  - TRMT1L
  - TRMT2A
  - TRMT2B
  - TRMT44
  - TRMT5
  - TRMT6
  - TRMT61A
  - TRMT61B
  - TRMT9B
  - MYC
  - LIN28A
  - MTOR
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - CRK
  - MAPK14
  - MAPK1
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK3
  - MAPK8
  - MAPK9
  - MAPK10
  - lin-28
  - sam
  - qkr54B
  - sls
  - Sam-S
  - G6P
  - aa
  - Tor
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - lic
  - p38b
  - Nurf-38
  - Ebp
  - p38a
  - Hrb87F
  - p38c
  - MKP-4
  - rl
---
