---
figid: PMC3294803__zjb9990912470001
figtitle: Reconstruction of AraR regulons in Clostridium species
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC3294803
filename: zjb9990912470001.jpg
figlink: /pmc/articles/PMC3294803/figure/F1/
number: F1
caption: Reconstruction of AraR regulons in Clostridium species. (A) Metabolic context
  of the reconstructed clostridial AraR regulons. Matching colors are used to mark
  components of three distinct pathways related to arabinose metabolism. The AraR
  regulon members predicted in this work are in red boxes. (B) Maximum likelihood
  phylogenetic tree and inferred DNA recognition motifs of the AraR regulator. The
  AraR regulon was reconstructed in this work for the nine Clostridium genomes marked
  in red. The known AraR-binding motif in B. subtilis is also shown. (C) Genomic organization
  of the araR-containing loci and AraR-regulated genes in Clostridium genomes. Genes
  (shown by arrows) from the same metabolic pathway are marked in matching colors.
  The araR genes and candidate AraR-binding sites are shown by red arrows and circles,
  respectively. The predicted araK gene is marked by asterisks.
papertitle: Ribulokinase and Transcriptional Regulation of Arabinose Metabolism in
  Clostridium acetobutylicum.
reftext: Lei Zhang, et al. J Bacteriol. 2012 Mar;194(5):1055-1064.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6772227
figid_alias: PMC3294803__F1
figtype: Figure
redirect_from: /figures/PMC3294803__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3294803__zjb9990912470001.html
  '@type': Dataset
  description: Reconstruction of AraR regulons in Clostridium species. (A) Metabolic
    context of the reconstructed clostridial AraR regulons. Matching colors are used
    to mark components of three distinct pathways related to arabinose metabolism.
    The AraR regulon members predicted in this work are in red boxes. (B) Maximum
    likelihood phylogenetic tree and inferred DNA recognition motifs of the AraR regulator.
    The AraR regulon was reconstructed in this work for the nine Clostridium genomes
    marked in red. The known AraR-binding motif in B. subtilis is also shown. (C)
    Genomic organization of the araR-containing loci and AraR-regulated genes in Clostridium
    genomes. Genes (shown by arrows) from the same metabolic pathway are marked in
    matching colors. The araR genes and candidate AraR-binding sites are shown by
    red arrows and circles, respectively. The predicted araK gene is marked by asterisks.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - araE
  - araJ
  - araA
  - araB
  - araD
  - tktA
  - L-Ribulose-5P
  - Ribulose
  - Arabinoside Arabinose
  - L-Arabinose
  - D-Fructose-6P
  - D-Glyceraldehyde-3P
  - phosphate
  - D-Ribose-5P
  - D-Xylulose-5P
---
