---
figid: PMC3919531__nihms534142f1
figtitle: The Intertwined Roles of Transcription and Repair Proteins
organisms:
- NA
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC3919531
filename: nihms534142f1.jpg
figlink: /pmc/articles/PMC3919531/figure/F1/
number: F1
caption: 'Exogenous and endogenous genotoxic agents (top) generate a variety of DNA
  damage, such as single and double strand breaks (SSBs, DSBs), insertions and deletions
  (indels). Lesions are detected and repaired by four major DNA repair pathways: base
  excision repair (BER, A), nucleotide excision repair (NER, B), mismatch repair (MMR,
  C) and recombinational repair (D). Mechanisms of BER, NER and recombinational repair
  are depicted. For each pathway, factors discussed in this review are in colors.A.
  Removal of uracil from DNA by BER. Thymine DNA glycosylase (TDG) removes the nitrogenous
  base and generates an abasic site (*). Apurinic/apyrimidinic (AP) endonuclease finalizes
  the nucleotide removal and creates a nick in the sugar-phosphate backbone. Poly
  ADP-ribose polymerase 1 (PARP-1) senses the SSB and recruits DNA polymerase and
  ligase to fill in the gap.B. Removal of large bulky adducts by NER. Transcribing
  RNA polymerase II (Pol II) stalls at DNA lesions and triggers transcription-coupled
  NER (TCR). TCR is initiated by the recruitment of Cockayne syndrome proteins A and
  B (CSA, CSB) to the arrested polymerase. DNA damage on the non-transcribed regions
  of the genome is repaired by global genome NER (GGR) instead. DNA lesion is recognized
  by the repair complex comprising Xeroderma pigmentosum C (XPC), RAD23B and Centrin
  2 (CETN2). Completion of TCR and GGR requires the recruitment of downstream NER
  factors (XPA, RPA, TFIIH, ERCC1-XPF, XPG). ERCC1-XPF and XPG endonucleases incise
  the damaged strand a few bases 5′ and 3′ to the DNA lesion, respectively. The gap
  is filled in by DNA polymerase and sealed by ligase.D. Removal of DSBs and inter-strand
  cross-links (ICL) by recombinational repair. 1. Repair of DSBs by non-homologous
  end joining (NHEJ). DSB sites are marked by Ataxia telangiectasia mutated (ATM)
  kinase-mediated phosphorylation of histone H2A variant X (γ-H2AX) (). Ku proteins
  direct the binding of the catalytic subunits of the DNA-dependent protein kinase
  (DNA-PKcs) to the exposed DNA ends. Autophosphorylation of DNA-PKcs facilitates
  DNA-ends processing and resealing.2. Repair of ICLs by Fanconi Anemia (FA), homologous
  recombination (HR) and NER pathways (see review ()). During S-phase, converging
  replication forks stall at ICLs and are sensed by FANCM protein, which recruits
  downstream FA proteins (FA core) and initiates ATR (ATM- and Rad3-related)-CHK1
  checkpoint response. The FA core complex ubiquitinates FANCD2 and FANCI. This facilitates
  the recruitment of endonucleases (SLX4, XPF/ERCC1, MUS81/EME1) and resection of
  the lesion from one of the two cross-linked strands. Translesion DNA synthesis proceeds
  through the uncut strand, generating the template for the homologous recombination
  machinery (MRN, BRCA2, RAD51) to complete DNA replication across the nicked DNA
  strand. NER removes the remaining adducts.'
papertitle: The Intertwined Roles of Transcription and Repair Proteins.
reftext: Yick W. Fong, et al. Mol Cell. ;52(3):291-302.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.530008
figid_alias: PMC3919531__F1
figtype: Figure
redirect_from: /figures/PMC3919531__F1
ndex: 6778c192-de92-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3919531__nihms534142f1.html
  '@type': Dataset
  description: 'Exogenous and endogenous genotoxic agents (top) generate a variety
    of DNA damage, such as single and double strand breaks (SSBs, DSBs), insertions
    and deletions (indels). Lesions are detected and repaired by four major DNA repair
    pathways: base excision repair (BER, A), nucleotide excision repair (NER, B),
    mismatch repair (MMR, C) and recombinational repair (D). Mechanisms of BER, NER
    and recombinational repair are depicted. For each pathway, factors discussed in
    this review are in colors.A. Removal of uracil from DNA by BER. Thymine DNA glycosylase
    (TDG) removes the nitrogenous base and generates an abasic site (*). Apurinic/apyrimidinic
    (AP) endonuclease finalizes the nucleotide removal and creates a nick in the sugar-phosphate
    backbone. Poly ADP-ribose polymerase 1 (PARP-1) senses the SSB and recruits DNA
    polymerase and ligase to fill in the gap.B. Removal of large bulky adducts by
    NER. Transcribing RNA polymerase II (Pol II) stalls at DNA lesions and triggers
    transcription-coupled NER (TCR). TCR is initiated by the recruitment of Cockayne
    syndrome proteins A and B (CSA, CSB) to the arrested polymerase. DNA damage on
    the non-transcribed regions of the genome is repaired by global genome NER (GGR)
    instead. DNA lesion is recognized by the repair complex comprising Xeroderma pigmentosum
    C (XPC), RAD23B and Centrin 2 (CETN2). Completion of TCR and GGR requires the
    recruitment of downstream NER factors (XPA, RPA, TFIIH, ERCC1-XPF, XPG). ERCC1-XPF
    and XPG endonucleases incise the damaged strand a few bases 5′ and 3′ to the DNA
    lesion, respectively. The gap is filled in by DNA polymerase and sealed by ligase.D.
    Removal of DSBs and inter-strand cross-links (ICL) by recombinational repair.
    1. Repair of DSBs by non-homologous end joining (NHEJ). DSB sites are marked by
    Ataxia telangiectasia mutated (ATM) kinase-mediated phosphorylation of histone
    H2A variant X (γ-H2AX) (). Ku proteins direct the binding of the catalytic subunits
    of the DNA-dependent protein kinase (DNA-PKcs) to the exposed DNA ends. Autophosphorylation
    of DNA-PKcs facilitates DNA-ends processing and resealing.2. Repair of ICLs by
    Fanconi Anemia (FA), homologous recombination (HR) and NER pathways (see review
    ()). During S-phase, converging replication forks stall at ICLs and are sensed
    by FANCM protein, which recruits downstream FA proteins (FA core) and initiates
    ATR (ATM- and Rad3-related)-CHK1 checkpoint response. The FA core complex ubiquitinates
    FANCD2 and FANCI. This facilitates the recruitment of endonucleases (SLX4, XPF/ERCC1,
    MUS81/EME1) and resection of the lesion from one of the two cross-linked strands.
    Translesion DNA synthesis proceeds through the uncut strand, generating the template
    for the homologous recombination machinery (MRN, BRCA2, RAD51) to complete DNA
    replication across the nicked DNA strand. NER removes the remaining adducts.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NR1H2
  - MRC1
  - BPIFA4P
  - TRBV20OR9-2
  - TRA
  - TRB
  - TRD
  - TRG
  - TDG
  - XRCC6
  - XRCC5
  - CSH2
  - ERCC6
  - ERCC8
  - CSH1
  - HSPA9
  - PRKDC
  - ATM
  - ERVW-4
  - XPC
  - CETN2
  - RAD23B
  - H2AX
  - H2BC21
  - GCGR
  - XPA
  - FANCD2
  - SLX4
  - ERCC1
  - ERCC4
  - FANCI
  - ERCC2
  - ERCC3
  - GTF2H1
  - GTF2H2
  - GTF2H3
  - GTF2H4
  - GTF2H5
  - ERCC5
  - ATR
  - ANTXR1
  - MMAB
  - SERPINA2
  - CHEK1
  - RPA1
  - RPA2
  - RPA3
  - MUS81
  - EME1
  - RASA1
  - RGS6
  - PARP1
  - PCNA
  - XRCC1
  - C1QA
  - C1QB
  - C1QC
  - C1R
  - C1S
  - RAD51
  - PRTN3
  - BRCA2
  - tdg.1
  - xrcc6
  - xrcc5
  - ercc6
  - ercc8
  - atm
  - c4
  - xpc
  - cetn2
  - cetn4
  - rad23b
  - h2ax
  - si:dkey-261m9.6
  - si:dkey-261m9.19
  - xpa
  - fancd2
  - slx4
  - ercc1
  - fanci
  - atr
  - rpa2
  - mus81
  - eme1
  - gap43
  - parp1
  - pcna
  - xrcc1
  - rad51
  - brca2
  - Uracil
  - Fanconi anemia
---
