---
figid: PMC3196718__1756-0500-4-370-4
figtitle: Network of upregulated genes in C2C12 cells treated with rhBMP2 and rhBMP7
  using the STRING 9.0 pathway analyzer
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC3196718
filename: 1756-0500-4-370-4.jpg
figlink: /pmc/articles/PMC3196718/figure/F4/
number: F4
caption: 'Network of upregulated genes in C2C12 cells treated with rhBMP2 and rhBMP7
  using the STRING 9.0 pathway analyzer. STRING 9.0 is a database of known and predicted
  protein interactions. The interactions may be physically direct or indirect (functional).
  The list of protein names are inserted as input and the organism to be analyzed
  is selected in a user-friendly web-browser interface. The network, composed of the
  uploaded proteins, is constructed based on four different sources: genomic context,
  high-throughput experiments, co-expression and scientific reports (http://string-db.org/).
  The upregulated genes found in each group (C2C12 cells treated with rhBMP7 or rhBMP2)
  were matched and classified according to a pre-described interaction (activation,
  repression, or predicted interaction). The circles represent the genes and the blue
  edges represent the interaction between them. The edges are based on proteins being
  co-mentioned in scientific publications as well as their association in curated
  databases. The edge thickness is directly proportional to the degree of evidence
  available, with lack of evidence for interactions in the literature being represented
  as no edges among the genes. Runx2: Runt Related Transcription factor 2; Dlx1/2/5:
  Distal-less homeobox 1/2 and 5;Id1/2/3: Inhibitor of DNA binding 1/2 and 3; Sp7,
  an alternative name for Osterix: Transcription factor Sp7; Hoxc8: Homeobox protein
  Hox-C8; Fkhr1: Forkhead protein FKHR1; Glis1/3: Gli homologous protein 1; Cfdp1:
  Craniofacial development protein 1; Lrp6: Low-density lipoprotein receptor-related
  protein 6; Segment polarity protein dishevelled homolog DVL-1; Ecsit: Evolutionarily
  conserved signaling intermediate in the Toll pathway, mitochondrial; PKCd: Protein
  kinase C, delta type; Ltbp2: Latent-transforming growth factor beta-binding protein
  2; Grn: Granulin; BMP1: Bone Morphogenetic Protein 1; Plod1: Procollagen-lysine,2-oxoglutarate
  5-dioxygenase 1; Postn: Periostin; Htra1: Serine protease HTRA1; Kazald1: Kazal-type
  serine protease inhibitor domain-containing protein 1 (IGFBP-rP10); Wnt3a: Protein
  Wnt-3a; Dkk1: Dickkopf-related protein 1 and Ctnn1b: Catenin beta-1.'
papertitle: Unveiling novel genes upregulated by both rhBMP2 and rhBMP7 during early
  osteoblastic transdifferentiation of C2C12 cells.
reftext: Juan C Bustos-Valenzuela, et al. BMC Res Notes. 2011;4:370-370.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6640656
figid_alias: PMC3196718__F4
figtype: Figure
redirect_from: /figures/PMC3196718__F4
ndex: 91e67bda-df15-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3196718__1756-0500-4-370-4.html
  '@type': Dataset
  description: 'Network of upregulated genes in C2C12 cells treated with rhBMP2 and
    rhBMP7 using the STRING 9.0 pathway analyzer. STRING 9.0 is a database of known
    and predicted protein interactions. The interactions may be physically direct
    or indirect (functional). The list of protein names are inserted as input and
    the organism to be analyzed is selected in a user-friendly web-browser interface.
    The network, composed of the uploaded proteins, is constructed based on four different
    sources: genomic context, high-throughput experiments, co-expression and scientific
    reports (http://string-db.org/). The upregulated genes found in each group (C2C12
    cells treated with rhBMP7 or rhBMP2) were matched and classified according to
    a pre-described interaction (activation, repression, or predicted interaction).
    The circles represent the genes and the blue edges represent the interaction between
    them. The edges are based on proteins being co-mentioned in scientific publications
    as well as their association in curated databases. The edge thickness is directly
    proportional to the degree of evidence available, with lack of evidence for interactions
    in the literature being represented as no edges among the genes. Runx2: Runt Related
    Transcription factor 2; Dlx1/2/5: Distal-less homeobox 1/2 and 5;Id1/2/3: Inhibitor
    of DNA binding 1/2 and 3; Sp7, an alternative name for Osterix: Transcription
    factor Sp7; Hoxc8: Homeobox protein Hox-C8; Fkhr1: Forkhead protein FKHR1; Glis1/3:
    Gli homologous protein 1; Cfdp1: Craniofacial development protein 1; Lrp6: Low-density
    lipoprotein receptor-related protein 6; Segment polarity protein dishevelled homolog
    DVL-1; Ecsit: Evolutionarily conserved signaling intermediate in the Toll pathway,
    mitochondrial; PKCd: Protein kinase C, delta type; Ltbp2: Latent-transforming
    growth factor beta-binding protein 2; Grn: Granulin; BMP1: Bone Morphogenetic
    Protein 1; Plod1: Procollagen-lysine,2-oxoglutarate 5-dioxygenase 1; Postn: Periostin;
    Htra1: Serine protease HTRA1; Kazald1: Kazal-type serine protease inhibitor domain-containing
    protein 1 (IGFBP-rP10); Wnt3a: Protein Wnt-3a; Dkk1: Dickkopf-related protein
    1 and Ctnn1b: Catenin beta-1.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BAD
  - SIRT1
  - KAZALD1
  - NOS3
  - NANOS3
  - FOXO4
  - GLIS1
  - HTRA1
  - ID3
  - AKT1
  - CREB1
  - CFDP1
  - NKD2
  - AXIN1
  - PLOD1
  - CTNNB1
  - ECSIT
  - DAAM1
  - SMAD1
  - GARS1
  - GLIS3
  - DKK1
  - LRP6
  - DURS1
  - VANGL2
  - ID2
  - WNT3A
  - GRN
  - ID1
  - POSTN
---
