---
figid: PMC3073932__pone.0018179.g007
figtitle: 'Regulatory Response to Carbon Starvation in Caulobacter crescentus '
organisms:
- Escherichia coli
- Bacillus subtilis
- Salmonella enterica subsp. enterica serovar Typhimurium
- Bacillus subtilis At2
- Sinorhizobium meliloti
- Caulobacter vibrioides
- Methylorubrum extorquens
- oligotrophic bacterium
- Caulobacter vibrioides CB15
- Caulobacter vibrioides NA1000
- Mesomycoplasma mobile
- Escherichia coli NC-7
- Saccharomyces cerevisiae
- Armoracia rusticana
- Prioria balsamifera
- Oryctolagus cuniculus
- Capra hircus
- Homo sapiens
- Vipera berus berus
pmcid: PMC3073932
filename: pone.0018179.g007.jpg
figlink: /pmc/articles/PMC3073932/figure/pone-0018179-g007/
number: F7
caption: 'Diagram of regulatory pathways involved in the response to carbon starvation,
  derived from the analysis of proteomic and gene expression profile changes. Ovals
  represent proteins and rectangles represent genes. Genes and proteins whose levels
  increase significantly upon starvation are represented in yellow, and downregulated
  genes and proteins in blue. For example, we indicate that of the 13 genes controlled
  by CtrA whose transcripts were downregulated in the absence of carbon (blue rectangle),
  we were able to detect the down-regulation of one of the corresponding protein products
  (blue oval). The same schematic representation was used for all genes shown. The
  genes and proteins that change significantly upon carbon starvation are listed in
  . Putative direct regulatory interactions are based on the presence of conserved
  promoter sequences, while an indirect regulation is proposed if there is no evidence
  of DNA binding or of the presence of a conserved promoter element. For the transcript
  changes, the 30 min time point was considered, while both the 30 and 60 min time
  points were considered for the protein changes. The CtrA regulon comprises direct
  CtrA targets, as previously determined , that changed upon 30 min of carbon starvation
  determined by microarray analysis (listed in ), as well as proteins encoded by direct
  CtrA targets that changed in the proteomic analysis at 30 and/or 60 min of carbon
  starvation (while most of those proteins were seen to change at the transcript level
  as well, SigU and LexA were observed to change only at the protein level). CtrA
  activation or repression was inferred by the direction of the change observed upon
  starvation: those genes with increased transcript levels upon starvation were inferred
  to be negatively regulated by CtrA (and consequently derepressed as CtrA protein
  levels decrease upon starvation); genes with decreased transcript levels were inferred
  to be positively regulated by CtrA. The SigT regulon includes the genes that showed
  a significant difference in transcript levels changes upon 15 min of carbon starvation
  in the sigT deletion strain with respect to wild type (listed in ). Genes with a
  SigT binding motif in their promoters are represented as direct targets, while those
  lacking the motif, as indirect targets. The levels of proteins encoded by a subset
  of these genes changed significantly upon 30 and/or 60 min of carbon starvation.
  The SigU-dependent gene, CC_3466, showed reduced transcript levels under carbon
  starvation in a sigU deletion strain with respect to wild type. The LexA (SOS) regulon
  comprises those genes belonging to the SOS regulon –as previously determined  -
  that changed significantly after 30 min of carbon starvation. The topology of the
  FixL-FixJ-FixK pathway is as determined by Crosson et al. . The FixK direct targets
  are those that have the cc_3 motif in their promoter, while the indirect targets
  were shown to be FixK-dependent by Crosson et al., but lack the cc_3 motif. The
  RpoD regulon comprises the genes with the cc_6 promoter motif whose transcripts
  levels were upregulated or downregulated upon 30 min of carbon starvation.'
papertitle: Regulatory Response to Carbon Starvation in Caulobacter crescentus .
reftext: Leticia Britos, et al. PLoS One. 2011;6(4):e18179.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8877777
figid_alias: PMC3073932__F7
figtype: Figure
organisms_ner:
- Caulobacter vibrioides NA1000
redirect_from: /figures/PMC3073932__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3073932__pone.0018179.g007.html
  '@type': Dataset
  description: 'Diagram of regulatory pathways involved in the response to carbon
    starvation, derived from the analysis of proteomic and gene expression profile
    changes. Ovals represent proteins and rectangles represent genes. Genes and proteins
    whose levels increase significantly upon starvation are represented in yellow,
    and downregulated genes and proteins in blue. For example, we indicate that of
    the 13 genes controlled by CtrA whose transcripts were downregulated in the absence
    of carbon (blue rectangle), we were able to detect the down-regulation of one
    of the corresponding protein products (blue oval). The same schematic representation
    was used for all genes shown. The genes and proteins that change significantly
    upon carbon starvation are listed in . Putative direct regulatory interactions
    are based on the presence of conserved promoter sequences, while an indirect regulation
    is proposed if there is no evidence of DNA binding or of the presence of a conserved
    promoter element. For the transcript changes, the 30 min time point was considered,
    while both the 30 and 60 min time points were considered for the protein changes.
    The CtrA regulon comprises direct CtrA targets, as previously determined , that
    changed upon 30 min of carbon starvation determined by microarray analysis (listed
    in ), as well as proteins encoded by direct CtrA targets that changed in the proteomic
    analysis at 30 and/or 60 min of carbon starvation (while most of those proteins
    were seen to change at the transcript level as well, SigU and LexA were observed
    to change only at the protein level). CtrA activation or repression was inferred
    by the direction of the change observed upon starvation: those genes with increased
    transcript levels upon starvation were inferred to be negatively regulated by
    CtrA (and consequently derepressed as CtrA protein levels decrease upon starvation);
    genes with decreased transcript levels were inferred to be positively regulated
    by CtrA. The SigT regulon includes the genes that showed a significant difference
    in transcript levels changes upon 15 min of carbon starvation in the sigT deletion
    strain with respect to wild type (listed in ). Genes with a SigT binding motif
    in their promoters are represented as direct targets, while those lacking the
    motif, as indirect targets. The levels of proteins encoded by a subset of these
    genes changed significantly upon 30 and/or 60 min of carbon starvation. The SigU-dependent
    gene, CC_3466, showed reduced transcript levels under carbon starvation in a sigU
    deletion strain with respect to wild type. The LexA (SOS) regulon comprises those
    genes belonging to the SOS regulon –as previously determined  - that changed significantly
    after 30 min of carbon starvation. The topology of the FixL-FixJ-FixK pathway
    is as determined by Crosson et al. . The FixK direct targets are those that have
    the cc_3 motif in their promoter, while the indirect targets were shown to be
    FixK-dependent by Crosson et al., but lack the cc_3 motif. The RpoD regulon comprises
    the genes with the cc_6 promoter motif whose transcripts levels were upregulated
    or downregulated upon 30 min of carbon starvation.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ctrA
  - sigU
  - sigT
  - lexA
  - fixL
  - fixJ
  - fixK
  - rpoD
---
