---
figid: PMC3744886__nihms139210f2
figtitle: The metabolic pathways controlled by lysine and glycine riboswitches
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC3744886
filename: nihms139210f2.jpg
figlink: /pmc/articles/PMC3744886/figure/F2/
number: F2
caption: The metabolic pathways controlled by lysine and glycine riboswitches. (A)
  Diaminopimelate pathway of lysine biosynthesis and lysine import pathways in various
  bacteria. Gene names are indicated in accordance to E. coli nomenclature. B. subtlis
  gene names are underlined. Putative lysine transport genes are boxed. Lysine riboswitches
  that repress expression of single genes or are located upstream of the first gene
  of the operon in any bacteria are shown with red lines. Most bacterial species,
  including E. coli, convert tetrahydrodipicolinate to diaminopimelate via succinylated
  DAP derivatives. Some species, including B. subtilis, use acetylated intermediates.
  Bacillus sphaericus perform conversion in one step using a unique DAP dehydrogenase
  encoded by the ddh gene. The figure is modified from refs. [, ]. (B) Simplified
  schematic of glycine metabolism and TCA of Cand. P. ubique. The glycine riboswitches
  that activate gene expression are shown by green arrows. The figure is adapted from
  ref. [].
papertitle: Amino acid recognition and gene regulation by riboswitches.
reftext: Alexander Serganov, et al. Biochim Biophys Acta. ;1789(0):592-611.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9576293
figid_alias: PMC3744886__F2
figtype: Figure
redirect_from: /figures/PMC3744886__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3744886__nihms139210f2.html
  '@type': Dataset
  description: The metabolic pathways controlled by lysine and glycine riboswitches.
    (A) Diaminopimelate pathway of lysine biosynthesis and lysine import pathways
    in various bacteria. Gene names are indicated in accordance to E. coli nomenclature.
    B. subtlis gene names are underlined. Putative lysine transport genes are boxed.
    Lysine riboswitches that repress expression of single genes or are located upstream
    of the first gene of the operon in any bacteria are shown with red lines. Most
    bacterial species, including E. coli, convert tetrahydrodipicolinate to diaminopimelate
    via succinylated DAP derivatives. Some species, including B. subtilis, use acetylated
    intermediates. Bacillus sphaericus perform conversion in one step using a unique
    DAP dehydrogenase encoded by the ddh gene. The figure is modified from refs. [,
    ]. (B) Simplified schematic of glycine metabolism and TCA of Cand. P. ubique.
    The glycine riboswitches that activate gene expression are shown by green arrows.
    The figure is adapted from ref. [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - lysC
  - thrA
  - metL
  - metB
  - metI
  - asd
  - gcvT
  - dapA
  - dapB
  - dapD
  - argD
  - patA
  - dapE
  - aceA
  - bamC
  - dapF
  - lysA
  - lysP
  - lysW
---
