---
figid: PMC3367986__pone.0038250.g002
figtitle: Comparative Genomic Analysis of Soybean Flowering Genes
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Glycine max
- Hordeum vulgare
- Medicago truncatula
- Brachypodium distachyon
- Lotus japonicus
- Glycine soja
- Arabidopsis lyrata
- Homo sapiens
- Pseudomonas phage PRR1
pmcid: PMC3367986
filename: pone.0038250.g002.jpg
figlink: /pmc/articles/PMC3367986/figure/pone-0038250-g002/
number: F2
caption: Arabidopsis flowering pathway depicted by Higgins et al. (2010)  was adapted
  for this study. Arabidopsis gene symbols are shown in upper box of each node along
  with the corresponding OG ID and the total number of soybean genes in the same OG.
  Only the soybean genes that are closer to the Arabidopsis flowering genes in each
  OG are listed in each node. Arabidopsis genes in grey shades are those that are
  not assigned with putative soybean orthologues. However, based on BLAST analysis
  these genes still have homologous soybean genes, the best matching soybean gene
  in BLAST analysis is shown below these genes. Dashed-lines were used for the arrows
  or T-bars that involve grey shades. Soybean genes marked with * are those that share
  the same clade with Arabidopsis genes but have not been investigated so far as flowering
  genes. OG IDs starting with ‘OG5_AT’ are arbitrarily generated in this study and
  do not exist in OrthoMCL 5.0 database. Other conventions are same as those used
  in the  by Higgins et al. (2010) . Arrows show promoting effects, T-bars show repressing
  effects. Environmental cues are shown as lower case letters in square boxes; ‘v’
  is extended cold (vernalization); ‘ld’ is long days; ‘sd’ is short days; ‘am’ is
  ambient (non-vernalizing) temperature. Genes are shown in italics and proteins in
  non-italics in ovals. ‘Pfr’ indicates Pfr phytochrome signaling. Arabidopsis genes
  assigned to specific pathways are color-coded (photoperiod pathway in green, vernalization
  in blue and autonomous pathway in purple). Flowering pathway integrators are shown
  in red. Triple headed arrows indicate activation by red or blue light.
papertitle: Comparative Genomic Analysis of Soybean Flowering Genes.
reftext: Chol-Hee Jung, et al. PLoS One. 2012;7(6):e38250.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5963153
figid_alias: PMC3367986__F2
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
- Homo sapiens
- Glycine max
redirect_from: /figures/PMC3367986__F2
ndex: af0cef2c-df1c-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3367986__pone.0038250.g002.html
  '@type': Dataset
  description: Arabidopsis flowering pathway depicted by Higgins et al. (2010)  was
    adapted for this study. Arabidopsis gene symbols are shown in upper box of each
    node along with the corresponding OG ID and the total number of soybean genes
    in the same OG. Only the soybean genes that are closer to the Arabidopsis flowering
    genes in each OG are listed in each node. Arabidopsis genes in grey shades are
    those that are not assigned with putative soybean orthologues. However, based
    on BLAST analysis these genes still have homologous soybean genes, the best matching
    soybean gene in BLAST analysis is shown below these genes. Dashed-lines were used
    for the arrows or T-bars that involve grey shades. Soybean genes marked with *
    are those that share the same clade with Arabidopsis genes but have not been investigated
    so far as flowering genes. OG IDs starting with ‘OG5_AT’ are arbitrarily generated
    in this study and do not exist in OrthoMCL 5.0 database. Other conventions are
    same as those used in the  by Higgins et al. (2010) . Arrows show promoting effects,
    T-bars show repressing effects. Environmental cues are shown as lower case letters
    in square boxes; ‘v’ is extended cold (vernalization); ‘ld’ is long days; ‘sd’
    is short days; ‘am’ is ambient (non-vernalizing) temperature. Genes are shown
    in italics and proteins in non-italics in ovals. ‘Pfr’ indicates Pfr phytochrome
    signaling. Arabidopsis genes assigned to specific pathways are color-coded (photoperiod
    pathway in green, vernalization in blue and autonomous pathway in purple). Flowering
    pathway integrators are shown in red. Triple headed arrows indicate activation
    by red or blue light.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PRR7
  - PRR9
  - FVE
  - FPA
  - FLK
  - LD
  - FLD
  - FY
  - FCA
  - LHY
  - CCA1
  - TCP11
  - CLF
  - SWN
  - FIE
  - MSI1
  - VEL1
  - VIN3
  - VRN1
  - VRN2
  - VRN5
  - PCL1
  - ELF4
  - ZTL
  - PRR3
  - GI
  - TOC1
  - CRY1
  - CRY2
  - COP1
  - ELF3
  - FRI
  - FKF1
  - FRL1
  - SMT2
  - FRL2
  - CDF1
  - CDF2
  - LKP2
  - FES1
  - SUF4
  - CDF3
  - LTP2
  - PAF1
  - EFS
  - PIE1
  - ARP6
  - MIR172b
  - MIR172a
  - PFT1
  - PEP
  - PAS2
  - PHYB
  - HUA2
  - FLC
  - SPA3
  - SPA4
  - RAP2.7
  - TFL2
  - SRT1
  - SRT2
  - SMZ
  - NF-YB1
  - PHYA
  - MAF1
  - RAV2
  - AGL24
  - SVP
  - CIB1
  - FT
  - TSF
  - FD
  - ELF9
  - AGL8
  - SPL3
  - AGL17
  - LFY
  - AP1
  - TFL1
  - C1orf210
  - TRNT1
  - CRLF1
  - CRNKL1
  - CLQTL1
  - NECTIN3
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - DAOA-AS1
  - CARD16
  - FMNL3
  - HSPBP1
  - PEX2
  - CDC73
  - CTR9
  - LEO1
  - RTF1
  - SKIC8
  - ACTR6
  - APOBEC3D
  - SRCAP
  - PAEP
  - PREP
  - PTPN22
  - AGXT
  - TOE1
  - LARP1
  - SIRT2
  - SIRT1
  - SMR3B
  - INMT
  - RASD2
  - AGL
  - FOS
  - FOSB
  - JUN
  - JUNB
  - JUND
  - FOSL1
  - FOSL2
  - LCL1
  - ELF3A
  - E4
  - SVPL
  - SOYAP1
---
