---
figid: PMC3621897__pone.0056598.g001
figlink: /pmc/articles/PMC3621897/figure/pone-0056598-g001/
number: F1
caption: The left-hand scale indicates carbon oxidation state. The chloromethane utilization
  cmu pathway (bold arrows) funnels the chloromethane-derived methyl group into central
  metabolism via methylene-H4F (CH2 = H4F), while the methanol (CH3OH) oxidation pathway
  operates with formaldehyde (HCHO) as a metabolic intermediate (grey arrows). H4F-
  and H4MPT-dependent enzyme-mediated steps are depicted in blue and pink, respectively.
  Carbon assimilation operates via the serine cycle (Ser) coupled with the ethylmalonyl-CoA
  pathway (EMCP) . Spontaneous condensation of HCHO with H4F or H4MPT, and formaldehyde
  oxidation to methylene-H4F are shown with broken line. In the cmu pathway, the methyl
  group enters a specific H4F-oxidation pathway for energy production driven by the
  FolD and PurU enzymes. Protein-encoded genes or genes located on plasmid pCMU01
  are shown in bold. Boxes and circles highlight proteins more abundant in chloromethane-
  and methanol grown-cultures, respectively. CmuA, methyltransferase/corrinoid-binding
  two-domain protein; CmuB, methylcobalamin:H4F methyltransferase; Fae, formaldehyde
  activating enzyme; Fch, methenyl-H4F cyclohydrolase; FDHs, formate dehydrogenases;
  Fhc, formyltransferase-hydrolase complex; FolD, bifunctional methylene-H4F dehydrogenase/cyclohydrolase;
  FtfL, formate-H4F ligase; Gck, glycerate kinase; GcvT, H4F-dependent aminomethyltransferase;
  HprA, hydroxypyruvate reductase; MDH, methanol dehydrogenase; MetF, methylene-H4F
  reductase; MtdA, bifunctional NAD(P)-dependant methylene-H4F and methylene-H4MPT
  dehydrogenase; MtdB, NAD(P)-dependent methylene-H4MPT dehydrogenase; Mch, methenyl-H4MPT
  cyclohydrolase; MtkA, malate thiokinase large subunit; MxaF, MDH alpha subunit,
  PurU, 10-formyl-H4F hydrolase; Sga, serine-glyoxylate aminotransferase . Plasmid
  pCMU01 encoded proteins with predicted functions include putative uncharacterized
  methyltransferases CmuC and CmuC2, the putative PaaE-like oxidoreductase, and the
  putative PQQ-linked dehydrogenase of unknown specificity XoxF2. GvcT may serve to
  transfer methyl groups from a wide range of substrates to H4F, as proposed for members
  that belong to the COG0354-related enzymes such as YgfZ .
pmcid: PMC3621897
papertitle: 'The 380 kb pCMU01 Plasmid Encodes Chloromethane Utilization Genes and
  Redundant Genes for Vitamin B12- and Tetrahydrofolate-Dependent Chloromethane Metabolism
  in Methylobacterium extorquens CM4: A Proteomic and Bioinformatics Study.'
reftext: Sandro Roselli, et al. PLoS One. 2013;8(4):e56598.
pmc_ranked_result_index: '34736'
pathway_score: 0.9114765
filename: pone.0056598.g001.jpg
figtitle: 'The 380 kb pCMU01 Plasmid Encodes Chloromethane Utilization Genes and Redundant
  Genes for Vitamin B12- and Tetrahydrofolate-Dependent Chloromethane Metabolism in
  Methylobacterium extorquens CM4: A Proteomic and Bioinformatics Study'
year: '2013'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3621897__pone.0056598.g001.html
  '@type': Dataset
  description: The left-hand scale indicates carbon oxidation state. The chloromethane
    utilization cmu pathway (bold arrows) funnels the chloromethane-derived methyl
    group into central metabolism via methylene-H4F (CH2 = H4F), while the methanol
    (CH3OH) oxidation pathway operates with formaldehyde (HCHO) as a metabolic intermediate
    (grey arrows). H4F- and H4MPT-dependent enzyme-mediated steps are depicted in
    blue and pink, respectively. Carbon assimilation operates via the serine cycle
    (Ser) coupled with the ethylmalonyl-CoA pathway (EMCP) . Spontaneous condensation
    of HCHO with H4F or H4MPT, and formaldehyde oxidation to methylene-H4F are shown
    with broken line. In the cmu pathway, the methyl group enters a specific H4F-oxidation
    pathway for energy production driven by the FolD and PurU enzymes. Protein-encoded
    genes or genes located on plasmid pCMU01 are shown in bold. Boxes and circles
    highlight proteins more abundant in chloromethane- and methanol grown-cultures,
    respectively. CmuA, methyltransferase/corrinoid-binding two-domain protein; CmuB,
    methylcobalamin:H4F methyltransferase; Fae, formaldehyde activating enzyme; Fch,
    methenyl-H4F cyclohydrolase; FDHs, formate dehydrogenases; Fhc, formyltransferase-hydrolase
    complex; FolD, bifunctional methylene-H4F dehydrogenase/cyclohydrolase; FtfL,
    formate-H4F ligase; Gck, glycerate kinase; GcvT, H4F-dependent aminomethyltransferase;
    HprA, hydroxypyruvate reductase; MDH, methanol dehydrogenase; MetF, methylene-H4F
    reductase; MtdA, bifunctional NAD(P)-dependant methylene-H4F and methylene-H4MPT
    dehydrogenase; MtdB, NAD(P)-dependent methylene-H4MPT dehydrogenase; Mch, methenyl-H4MPT
    cyclohydrolase; MtkA, malate thiokinase large subunit; MxaF, MDH alpha subunit,
    PurU, 10-formyl-H4F hydrolase; Sga, serine-glyoxylate aminotransferase . Plasmid
    pCMU01 encoded proteins with predicted functions include putative uncharacterized
    methyltransferases CmuC and CmuC2, the putative PaaE-like oxidoreductase, and
    the putative PQQ-linked dehydrogenase of unknown specificity XoxF2. GvcT may serve
    to transfer methyl groups from a wide range of substrates to H4F, as proposed
    for members that belong to the COG0354-related enzymes such as YgfZ .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FANCE
  - ADH5
  - PMCH
genes:
- word: Fae
  symbol: FAE
  source: hgnc_alias_symbol
  hgnc_symbol: FANCE
  entrez: '2178'
- word: FDH
  symbol: FDH
  source: hgnc_prev_symbol
  hgnc_symbol: ADH5
  entrez: '128'
- word: Mch
  symbol: MCH
  source: hgnc_alias_symbol
  hgnc_symbol: PMCH
  entrez: '5367'
chemicals: []
diseases: []
figid_alias: PMC3621897__F1
redirect_from: /figures/PMC3621897__F1
figtype: Figure
---
