---
figid: PMC3037835__nihms265254f5
figtitle: Major pathways of lipid metabolism in yeast
organisms:
- NA
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
pmcid: PMC3037835
filename: nihms265254f5.jpg
figlink: /pmc/articles/PMC3037835/figure/F5/
number: F5
caption: 'Major pathways of lipid metabolism in yeast. The pathways are color coded:
  (tan) glycerophospholipid pathway (violet) glycerol pathway, (yellow) phosphoinositides
  and inositol phosphates pathway, (blue) ceramide and sphingolipid pathway and (gray)
  sterol and TAG pathways. Solid arrows represent routes of metabolic conversion.
  Dotted lines represent multistep metabolic conversions. The names of structural
  genes (italicized) encoding enzymes catalyzing specific metabolic conversions are
  shown adjacent to arrows. An asterisk next to the gene name indicates genes regulated
  by inositol. Mutations in genes colored in green confer an overproduction of inositol
  (Opi−) phenotype due to elevated PA levels (). Mutations in genes colored in red
  confer an Ino− phenotype. An allele of the essential PIS1 gene (white letters on
  red background) has an Ino− phenotype. Genes enclosed in a rectangle are essential.
  Metabolites are indicated in plain text: DHAP dihydroxyacetone phosphate, DHA dihydroxyacetone,
  Gro-3-P glycerol 3-phosphate, G-6-P glucose 6-phosphate, I-3-P inositol 3-phosphate,
  lyso-PA lyso-phosphatidic acid, PA phosphatidic acid, DAG diacylglycerol, TAG triacylglycerol,
  CDP-DAG cytidine-diphosphate diacylglycerol, PS phosphatidylserine, PE phosphatidylethanolamine,
  SAM S-adenosyl methionine, PMME phosphatidyl-N-monomethylethanolamine, PDME phosphatidyl-N,N-dimethylethanolamine,
  PC phosphatidylcholine, Gro-PCho glycerophosphocholine, FFA free fatty acids, CDP-Cho
  cytidine-diphosphate-choline, Cho-P choline-phosphate, PI phosphatidylinositol,
  PI3P phosphatidylinositol 3-phosphate, PI4P phosphatidylinositol 4-phosphate, PI(3,5)P2,
  phosphatidylinositol 3,5-bisphosphate, PI(4,5)P2, phosphatidylinositol 4,5-bisphosphate,
  IP3 inositol 1,4,5-triphosphate, IP5 inositol 1,3,4,5,6-pentakisphosphate, IP6 inositol
  1,2,3,4,5,6-hexakisphosphate, IP7 diphosphoinositol pentakisphosphate, IP8 bis-diphosphoinositol
  tetrakisphosphate, 3-KDHS 3-ketodihydrosphingosine, DHS dihydrosphingosine, PHS
  phytosphingosine, DHSP dihydrosphingosine 1-phosphate, PHSP phytosphingosine 1-phosphate,
  E-P ethanolamine-phosphate, α-HPhyto-Cer alpha-hydroxyphytoceramide, IPC inositol-phosphoceramide,
  MIPC mannose-inositol-phosphoceramide, M(IP)2C mannosyl-diinositol-phosphorylceramide'
papertitle: Genome-wide screen for inositol auxotrophy in Saccharomyces cerevisiae
  implicates lipid metabolism in stress response signaling.
reftext: Manuel J. Villa-García, et al. Mol Genet Genomics. ;285(2):125-149.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9221134
figid_alias: PMC3037835__F5
figtype: Figure
redirect_from: /figures/PMC3037835__F5
ndex: 5c896078-df0c-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3037835__nihms265254f5.html
  '@type': Dataset
  description: 'Major pathways of lipid metabolism in yeast. The pathways are color
    coded: (tan) glycerophospholipid pathway (violet) glycerol pathway, (yellow) phosphoinositides
    and inositol phosphates pathway, (blue) ceramide and sphingolipid pathway and
    (gray) sterol and TAG pathways. Solid arrows represent routes of metabolic conversion.
    Dotted lines represent multistep metabolic conversions. The names of structural
    genes (italicized) encoding enzymes catalyzing specific metabolic conversions
    are shown adjacent to arrows. An asterisk next to the gene name indicates genes
    regulated by inositol. Mutations in genes colored in green confer an overproduction
    of inositol (Opi−) phenotype due to elevated PA levels (). Mutations in genes
    colored in red confer an Ino− phenotype. An allele of the essential PIS1 gene
    (white letters on red background) has an Ino− phenotype. Genes enclosed in a rectangle
    are essential. Metabolites are indicated in plain text: DHAP dihydroxyacetone
    phosphate, DHA dihydroxyacetone, Gro-3-P glycerol 3-phosphate, G-6-P glucose 6-phosphate,
    I-3-P inositol 3-phosphate, lyso-PA lyso-phosphatidic acid, PA phosphatidic acid,
    DAG diacylglycerol, TAG triacylglycerol, CDP-DAG cytidine-diphosphate diacylglycerol,
    PS phosphatidylserine, PE phosphatidylethanolamine, SAM S-adenosyl methionine,
    PMME phosphatidyl-N-monomethylethanolamine, PDME phosphatidyl-N,N-dimethylethanolamine,
    PC phosphatidylcholine, Gro-PCho glycerophosphocholine, FFA free fatty acids,
    CDP-Cho cytidine-diphosphate-choline, Cho-P choline-phosphate, PI phosphatidylinositol,
    PI3P phosphatidylinositol 3-phosphate, PI4P phosphatidylinositol 4-phosphate,
    PI(3,5)P2, phosphatidylinositol 3,5-bisphosphate, PI(4,5)P2, phosphatidylinositol
    4,5-bisphosphate, IP3 inositol 1,4,5-triphosphate, IP5 inositol 1,3,4,5,6-pentakisphosphate,
    IP6 inositol 1,2,3,4,5,6-hexakisphosphate, IP7 diphosphoinositol pentakisphosphate,
    IP8 bis-diphosphoinositol tetrakisphosphate, 3-KDHS 3-ketodihydrosphingosine,
    DHS dihydrosphingosine, PHS phytosphingosine, DHSP dihydrosphingosine 1-phosphate,
    PHSP phytosphingosine 1-phosphate, E-P ethanolamine-phosphate, α-HPhyto-Cer alpha-hydroxyphytoceramide,
    IPC inositol-phosphoceramide, MIPC mannose-inositol-phosphoceramide, M(IP)2C mannosyl-diinositol-phosphorylceramide'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ERG10
  - ERG13
  - HMG1
  - HMG2
  - LCB1
  - LCB2
  - ACB1
  - TSC3
  - ACC1
  - HXK2
  - INO1
  - DAK1
  - GUT2
  - GPD2
  - DAK2
  - ITR1
  - ELO2
  - RAD27
  - ELO3
  - PHS1
  - TSC13
  - LCB3
  - ITR2
  - TSC10
  - DPL1
  - ERG12
  - ERG8
  - MVD1
  - IDI1
  - ERG20
  - ERG9
  - ERG7
  - LCB4
  - LCB5
  - LAG1
  - SCT1
  - GPT2
  - SLC1
  - DYN2
  - ALE1
  - GRE3
  - YPR1
  - GPP1
  - ARA1
  - GCY1
  - CDS1
  - SUR2
  - YDC1
  - GPP2
  - PIS1
  - GUT1
  - YPC1
  - PSD1
  - PSD2
  - PAH1
  - DPP1
  - LPP1
  - STT4
  - PIK1
  - LSB6
  - ERG11
  - ERG24
  - ERG25
  - ERG26
  - ERG27
  - SCS7
  - AUR1
  - GUP1
  - FPS1
  - CHO2
  - MSS4
  - FAB1
  - ERG6
  - OPI3
  - TGL3
  - TGL4
  - TGL5
  - CSG2
  - CSH1
  - PLC1
  - ARG82
  - IPK1
  - ERG2
  - ERG3
  - ERG5
  - ERG4
  - OLE1
  - DGA1
  - ARE1
  - ARE2
  - ELO1
  - KCS1
  - DDP1
  - VIP1
  - SPO14
  - IPT1
  - FAS2
  - FAS1
  - CPT1
  - ISC1
  - NTE1
  - PLB1
  - PCT1
  - HNM1
  - CKI1
  - PLB2
  - PLB3
  - DGK1
  - VPS34
  - HMGB1
  - HMGB1P5
  - HMGB2
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - SPTLC1
  - SPTLC2
  - ACACA
  - BCL2A1
  - HK2
  - ISYNA1
  - CXCL3
  - FEN1
  - PTGS1
  - SPTLC3
  - TENM1
  - DPP6
  - EXT1
  - EXTL1
  - EXTL2
  - EXTL3
  - EXT2
  - FDFT1
  - GDF1
  - CCL4
  - CERS1
  - CCL4L1
  - DHPS
  - PIEZO1
  - MCHR1
  - SART1
  - CHEK2
  - MED23
  - ABCC9
  - GLI3
  - PCBD1
  - CDIPT
  - PSD
  - CTSC
  - PLPP1
  - VANGL2
  - MSMO1
  - FA2H
  - HHATL
  - EBP
  - RABIF
  - PIP5K1B
  - PIKFYVE
  - PI3
  - PLCG1
  - IPPK
  - KCNH6
  - ERG
  - SC5D
  - KCNH7
  - VPS52
  - PC
  - TBCE
  - TIMM8A
  - PPIP5K1
  - PLD1
  - PRKCSH
  - FAS
  - CPT1A
  - CPT2
  - CHPT1
  - MIP
  - TNPO1
  - MIPEP
  - MAFIP
  - TMED7
  - LINC01194
  - SMPD2
  - DDIT3
  - PIK3C3
  - Zymosterol
  - Fecosterol
  - Acyl-CoA
  - C16-aldehydes
  - Choline
  - CDP
  - Ergosterol
  - Steryl esters
  - Glucose
  - Glycerol
  - Inositol
  - Palmitoyl-CoA
  - Serine
---
