---
figid: PMC3598339__1758-5996-5-1-2
figtitle: Pathway analysis identifies a network of genes related to lipid metabolism
  in islets cultured at 5.6 mM glucose concentration
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Diaporthe sclerotioides
pmcid: PMC3598339
filename: 1758-5996-5-1-2.jpg
figlink: /pmc/articles/PMC3598339/figure/F2/
number: F2
caption: Pathway analysis identifies a network of genes related to lipid metabolism
  in islets cultured at 5.6 mM glucose concentration. The network is displayed graphically
  as nodes (genes/gene products) and edges (the biological relationships between the
  nodes). Nodes are displayed using various shapes that represent the functional class
  of the gene product. The node color indicates up (red)- or down (green)-regulation
  by Pioglitazone. Edges are displayed with various labels that describe the nature
  of the relationship between the nodes (A, activation; B, binding; E, expression;
  I, inhibition; P, phosphorylation; T, transcription). Edges without a label represent
  binding only. The node Srebf1 was identified by the pathway analysis as part of
  the network, and its differential gene expression was determined subsequently by
  qRT-PCR. A total of 15 differentially expressed focus genes were brought into this
  network with a highly significant score of 30.
papertitle: Apoptosis rate and transcriptional response of pancreatic islets exposed
  to the PPAR gamma agonist Pioglitazone.
reftext: Rodrigo N Lamounier, et al. Diabetol Metab Syndr. 2013;5:1-1.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.737707
figid_alias: PMC3598339__F2
figtype: Figure
organisms_ner:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
redirect_from: /figures/PMC3598339__F2
ndex: af9dad21-df2d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3598339__1758-5996-5-1-2.html
  '@type': Dataset
  description: Pathway analysis identifies a network of genes related to lipid metabolism
    in islets cultured at 5.6 mM glucose concentration. The network is displayed graphically
    as nodes (genes/gene products) and edges (the biological relationships between
    the nodes). Nodes are displayed using various shapes that represent the functional
    class of the gene product. The node color indicates up (red)- or down (green)-regulation
    by Pioglitazone. Edges are displayed with various labels that describe the nature
    of the relationship between the nodes (A, activation; B, binding; E, expression;
    I, inhibition; P, phosphorylation; T, transcription). Edges without a label represent
    binding only. The node Srebf1 was identified by the pathway analysis as part of
    the network, and its differential gene expression was determined subsequently
    by qRT-PCR. A total of 15 differentially expressed focus genes were brought into
    this network with a highly significant score of 30.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Atp6v1e1
  - Aldoc
  - Camk1d
  - Tmem97
  - Pcsk9
  - Stard4
  - Eci2
  - Srebf2
  - Ptn
  - Pten
  - ap
  - Acss2
  - Pmvk
  - Cdc34
  - Tnf
  - Nsdhl
  - Mpz
  - Snap91
  - Srebf1
  - Tom1l2
  - Scd1
  - Aacs
  - Ep300
  - Sgta
  - Insig1
  - Sc5d
  - Tcf12
  - Scd2
  - Insig2
  - Fabp4
  - Acta1
  - Wnt10b
  - Scd
  - Idh3g
  - ATP6V1E1
  - ALDOC
  - CAMK1D
  - TMEM97
  - PCSK9
  - STARD4
  - ECI2
  - IGKV3D-11
  - SREBF2
  - PTN
  - PTEN
  - ACSS2
  - PMVK
  - CDC34
  - TNF
  - NSDHL
  - MPZ
  - MPHOSPH6
  - SNAP91
  - SREBF1
  - SCD
  - SCD5
  - AACS
  - LGI1
  - FECH
  - EPX
  - EP300
  - SGTA
  - OMA1
  - INSIG1
  - IDH1
  - IDH2
  - SC5D
  - TCF12
  - INSIG2
  - FABP4
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - WNT10B
---
