---
figid: PMC3316891__ncomms1755-f8
figtitle: FAD-dependent lysine-specific demethylase-1 regulates cellular energy expenditure
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3316891
filename: ncomms1755-f8.jpg
figlink: /pmc/articles/PMC3316891/figure/f8/
number: F8
caption: Schematic model for the epigenetic regulation of energy metabolism by LSD1.
  FAD-dependent LSD1 facilitates metabolic changes through the repression of energy-expenditure
  genes via H3K4 demethylation. This pathway may be influenced by nutrients and/or
  fluctuating FAD level, implicating the link between energetic information and the
  epigenome.
papertitle: FAD-dependent lysine-specific demethylase-1 regulates cellular energy
  expenditure.
reftext: Shinjiro Hino, et al. Nat Commun. 2012 Mar 27;3:758.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9473058
figid_alias: PMC3316891__F8
figtype: Figure
redirect_from: /figures/PMC3316891__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3316891__ncomms1755-f8.html
  '@type': Dataset
  description: Schematic model for the epigenetic regulation of energy metabolism
    by LSD1. FAD-dependent LSD1 facilitates metabolic changes through the repression
    of energy-expenditure genes via H3K4 demethylation. This pathway may be influenced
    by nutrients and/or fluctuating FAD level, implicating the link between energetic
    information and the epigenome.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - Me
  - Lsd-1
  - Su(var)3-3
  - srl
  - BRCA2
  - FANCD2
  - PSEN1
  - KDM1A
  - PHF21A
  - PPARGC1A
  - FAD
---
