---
figid: PMC3296254__885fig5
figtitle: Arginine metabolic network
organisms:
- NA
pmcid: PMC3296254
filename: 885fig5.jpg
figlink: /pmc/articles/PMC3296254/figure/fig5/
number: F5
caption: Arginine metabolic network. Arginine is primarily transported into cells
  by the arginine permease Can1 (), and once internalized, the bulk of arginine is
  transported into the vacuole by the Vba2 transporter (). Cytoplasmic arginine exerts
  positive (green) and negative (red) effects on gene expression encoding enzymes
  required for arginine utilization and catabolism, respectively. Both positive and
  negative regulation relies on the ArgM/Mcm1 complex, which in an arginine-dependent
  manner participates in activating the expression of the genes in green and repressing
  the genes in red. (Arginine utilization; bottom) Arginine is degraded to form glutamate.
  Arginine is initially degraded in the cytoplasm to form proline; this requires the
  concerted action of arginase (CAR1) and ornithine aminotransferase (CAR2) to form
  glutamate γ-semialdehyde, which spontaneously converts to Δ1-pyrroline-5-carboxylate
  (P5C). P5C is converted to proline by the PRO3 gene product. Cytoplasmic proline
  is transported into the mitochondria where it is converted back to P5C by proline
  oxidase (PUT1). Finally, the mitochondrial P5C is converted to glutamate by the
  PUT2 gene product. Whereas CAR1 and CAR2 are positively regulated by the presence
  of arginine (discussed below), the expression of PUT1 and PUT2 is induced by proline
  (; ). Proline binds directly to the transcription factor Put3, a member of the well-studied
  Zn(II)2Cys6 binuclear cluster family of transcriptional regulators (). The activation
  of Put3 requires no additional components and can be induced by certain proline
  analogs with an unmodified pyrrolidine ring (). Detailed structural analysis indicates
  that proline directly controls the regulatory properties of transcriptional activator,
  providing a clear demonstration of how metabolite recognition and transcriptional
  control can be directly coupled (). (Arginine biosynthesis; top) The first five
  steps of biosynthesis take place in the mitochondria (ARG2, ARG5,-6, ARG8, ARG7)
  and result in the synthesis of ornithine. ARG5,-6 encode the enzymes that catalyze
  the second and third steps and are translated into a pre-protein that is imported
  into mitochondria, where it is cleaved, resulting in separate proteins, i.e., N-acetylglutamate
  kinase (Arg6) and N-acetylglutamyl-phosphate reductase (Arg5) (). The first two
  enzymes in the pathway, N-acetylglutamate synthase (Arg2) and N-acetylglutamate
  kinase (Arg6), bind each other, forming a complex that is necessary for their stability
  and for feedback inhibition by arginine (, ; ). The ornithine synthesized in mitochondria
  is transported to the cytoplasm via the mitochondrial carrier protein Ort1 (), and
  the remaining steps are carried out in the cytoplasm. Carbomoyl phosphate reacts
  with ornithine to form arginine in three steps (ARG3, ARG1, ARG4). Carbomoyl phosphate
  is synthesized from CO2, ATP, and the amide nitrogen of glutamine in a reaction
  catalyzed by the arginine-specific carbomoyl phosphate synthetase, a heterodimeric
  enzyme composed of a small regulatory subunit (CPA1) and a catalytic subunit (CPA2).
papertitle: Regulation of Amino Acid, Nucleotide, and Phosphate Metabolism in Saccharomyces
  cerevisiae.
reftext: Per O. Ljungdahl, et al. Genetics. 2012 Mar;190(3):885-929.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9205819
figid_alias: PMC3296254__F5
figtype: Figure
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
redirect_from: /figures/PMC3296254__F5
ndex: 0370bbca-df0d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3296254__885fig5.html
  '@type': Dataset
  description: Arginine metabolic network. Arginine is primarily transported into
    cells by the arginine permease Can1 (), and once internalized, the bulk of arginine
    is transported into the vacuole by the Vba2 transporter (). Cytoplasmic arginine
    exerts positive (green) and negative (red) effects on gene expression encoding
    enzymes required for arginine utilization and catabolism, respectively. Both positive
    and negative regulation relies on the ArgM/Mcm1 complex, which in an arginine-dependent
    manner participates in activating the expression of the genes in green and repressing
    the genes in red. (Arginine utilization; bottom) Arginine is degraded to form
    glutamate. Arginine is initially degraded in the cytoplasm to form proline; this
    requires the concerted action of arginase (CAR1) and ornithine aminotransferase
    (CAR2) to form glutamate γ-semialdehyde, which spontaneously converts to Δ1-pyrroline-5-carboxylate
    (P5C). P5C is converted to proline by the PRO3 gene product. Cytoplasmic proline
    is transported into the mitochondria where it is converted back to P5C by proline
    oxidase (PUT1). Finally, the mitochondrial P5C is converted to glutamate by the
    PUT2 gene product. Whereas CAR1 and CAR2 are positively regulated by the presence
    of arginine (discussed below), the expression of PUT1 and PUT2 is induced by proline
    (; ). Proline binds directly to the transcription factor Put3, a member of the
    well-studied Zn(II)2Cys6 binuclear cluster family of transcriptional regulators
    (). The activation of Put3 requires no additional components and can be induced
    by certain proline analogs with an unmodified pyrrolidine ring (). Detailed structural
    analysis indicates that proline directly controls the regulatory properties of
    transcriptional activator, providing a clear demonstration of how metabolite recognition
    and transcriptional control can be directly coupled (). (Arginine biosynthesis;
    top) The first five steps of biosynthesis take place in the mitochondria (ARG2,
    ARG5,-6, ARG8, ARG7) and result in the synthesis of ornithine. ARG5,-6 encode
    the enzymes that catalyze the second and third steps and are translated into a
    pre-protein that is imported into mitochondria, where it is cleaved, resulting
    in separate proteins, i.e., N-acetylglutamate kinase (Arg6) and N-acetylglutamyl-phosphate
    reductase (Arg5) (). The first two enzymes in the pathway, N-acetylglutamate synthase
    (Arg2) and N-acetylglutamate kinase (Arg6), bind each other, forming a complex
    that is necessary for their stability and for feedback inhibition by arginine
    (, ; ). The ornithine synthesized in mitochondria is transported to the cytoplasm
    via the mitochondrial carrier protein Ort1 (), and the remaining steps are carried
    out in the cytoplasm. Carbomoyl phosphate reacts with ornithine to form arginine
    in three steps (ARG3, ARG1, ARG4). Carbomoyl phosphate is synthesized from CO2,
    ATP, and the amide nitrogen of glutamine in a reaction catalyzed by the arginine-specific
    carbomoyl phosphate synthetase, a heterodimeric enzyme composed of a small regulatory
    subunit (CPA1) and a catalytic subunit (CPA2).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ARG56
  - ARG2
  - ARG8
  - ARG7
  - CPA1
  - CPA2
  - ARG3
  - ARG1
  - ARG4
  - CAN1
  - VBA2
  - CAR1
  - CAR2
  - PUT2
  - PUT1
  - PRO3
  - DECR1
  - WDTC1
  - ATP8A2
  - RCBTB1
  - EHMT1
  - GOLGA6A
  - TINAGL1
  - APRT
  - MFAP1
  - TCF23
  - CA1
  - NR1I3
  - CA2
  - NR1I4
  - PYCR1
  - BRCA2
  - FANCD2
  - PSEN1
  - KDM5B
  - pyrroline
  - arginine
  - aspartate
  - fumarate
  - glutamine
  - NADP proline
  - proline
  - glutamate
  - Glu
---
