---
figid: PMC3313874__1471-2180-12-22-2
figtitle: Networks built using differentially expressed genes in uninfected macrophages
  from C57BL/6 and CBA mice
organisms:
- Leishmania amazonensis
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC3313874
filename: 1471-2180-12-22-2.jpg
figlink: /pmc/articles/PMC3313874/figure/F2/
number: F2
caption: Networks built using differentially expressed genes in uninfected macrophages
  from C57BL/6 and CBA mice. C57BL/6 and CBA macrophages were cultured separately
  and then processed for microarray analysis as described in Materials and Methods.
  The cell death and lipid metabolism network (A) and the cell-cell signaling and
  interaction network (B) were modeled using Ingenuity Pathway Analysis software v8.8
  (IPA-Ingenuity Systems速). The above networks are displayed as a series of nodes
  (genes or gene products) and edges (or lines, corresponding to biological relationships
  between nodes). Nodes are displayed using shapes that represent the functional class
  of the gene product as indicated in the key. Nodes marked in green were found to
  be highly expressed in C57BL/6 macrophages in comparison to CBA. Nodes marked in
  red were found to be highly expressed in CBA macrophages compared to C57BL/6. The
  unmarked nodes were not identified in our samples; however, IPA速 added them to the
  networks due to their high probability of involvement in a given network. The node
  color intensity is an indication of the degree of up-(green) or down-(red) regulation
  of genes observed in the biological network analysis from uninfected C57BL/6 macrophages
  compared to CBA cells. Solid lines denote direct interactions, whereas dotted lines
  represent indirect interactions between the genes represented in this network.
papertitle: A comparison of two distinct murine macrophage gene expression profiles
  in response to Leishmania amazonensis infection.
reftext: Christian M Probst, et al. BMC Microbiol. 2012;12:22-22.
year: '2012'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8282827
figid_alias: PMC3313874__F2
figtype: Figure
redirect_from: /figures/PMC3313874__F2
ndex: 30d0690b-df25-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3313874__1471-2180-12-22-2.html
  '@type': Dataset
  description: Networks built using differentially expressed genes in uninfected macrophages
    from C57BL/6 and CBA mice. C57BL/6 and CBA macrophages were cultured separately
    and then processed for microarray analysis as described in Materials and Methods.
    The cell death and lipid metabolism network (A) and the cell-cell signaling and
    interaction network (B) were modeled using Ingenuity Pathway Analysis software
    v8.8 (IPA-Ingenuity Systems速). The above networks are displayed as a series of
    nodes (genes or gene products) and edges (or lines, corresponding to biological
    relationships between nodes). Nodes are displayed using shapes that represent
    the functional class of the gene product as indicated in the key. Nodes marked
    in green were found to be highly expressed in C57BL/6 macrophages in comparison
    to CBA. Nodes marked in red were found to be highly expressed in CBA macrophages
    compared to C57BL/6. The unmarked nodes were not identified in our samples; however,
    IPA速 added them to the networks due to their high probability of involvement in
    a given network. The node color intensity is an indication of the degree of up-(green)
    or down-(red) regulation of genes observed in the biological network analysis
    from uninfected C57BL/6 macrophages compared to CBA cells. Solid lines denote
    direct interactions, whereas dotted lines represent indirect interactions between
    the genes represented in this network.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EMB
  - XPO1
  - RSAD2
  - PTPN13
  - PROX2
  - GDF15
  - MT1E
  - FCGR1BP
  - IL10RB
  - ELOA
  - CD24
  - ERVK-1
  - STK26
  - ANKHD1
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - ERVK-16
  - EPHB2
  - MAPK1
  - MAPK3
  - AKT1
  - AKT2
  - AKT3
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - CTSC
  - AGTRAP
  - CADM1
  - ANGPT1
  - ANGPT2
  - ANGPT4
  - AREG
  - BDNF
  - CSF1
  - EFNA1
  - EFNA2
  - EFNA3
  - EFNA4
  - EFNA5
  - EGF
  - EREG
  - FGF1
  - FGF10
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF2
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FLT3LG
  - HGF
  - IGF1
  - IGF2
  - INS
  - KITLG
  - NGF
  - NTF3
  - NTF4
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - PGF
  - TGFA
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - NHERF1
  - LONP1
  - HLA-C
  - emb
  - COM
  - comt
  - compressed
  - lncRNA:flam
  - Rx
  - mask
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - Smurf
  - Erk7
  - rl
  - ha
  - Akt
  - Act5C
  - Act42A
  - Act57B
  - Act79B
  - Act88F
  - Act87E
  - Lon
  - schlank
---
