---
figid: PMC3110158__pntd.0001176.g001
figtitle: RNAi Effector Diversity in Nematodes
organisms:
- Drosophila melanogaster
- Caenorhabditis elegans
- Schistosoma mansoni
- Meloidogyne javanica
- Ascaris suum
- Haemonchus contortus
- Trichinella spiralis
- Meloidogyne incognita
- Caenorhabditis briggsae
- Brugia malayi
- Heligmosomoides polygyrus
- Pristionchus pacificus
- Paramecium tetraurelia
- Caenorhabditis remanei
- Ancylostoma caninum
- Globodera pallida
- Caenorhabditis brenneri
- Meloidogyne hapla
- Ostertagia ostertagi
- Caenorhabditis japonica
- Oesophagostomum dentatum
- Arabidopsis thaliana
- Neurospora crassa
- Cryptomeria japonica
- Homo sapiens
- bacterium 1-7A
- NA
organisms_ner:
- Caenorhabditis elegans
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
- Saccharomyces cerevisiae S288C
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC3110158
filename: pntd.0001176.g001.jpg
figlink: /pmc/articles/PMC3110158/figure/pntd-0001176-g001/
number: F1
caption: (1) Exogenously applied double-stranded RNA (dsRNA) and small interfering
  RNA (exo-siRNA) are thought to enter cells via SID (Systemic RNA Interference Defective)
  proteins SID-1/RSD-8 and SID-2. (2) Endogenous RNAi-based pathways begin in the
  nucleus; micro-interfering RNA (miRNA) synthesis begins with transcription of hairpin-looped
  primary miRNA (pri-miRNA) transcripts from intergenic, intronic or antisense regions.
  pri-miRNAs are processed by the DRSH-1/PASH-1 complex to pre-miRNA, which are exported
  from the nucleus by exportin proteins XPO-1, -2 and -3. Endogenous siRNAs (endo-siRNAs)
  are also produced from genomic regions, and exported by XPO-1, 2, and -3. (3) Both
  pre-miRNAs and exogenously applied dsRNA molecules are bound and cleaved by the
  dicer complex, which consists of the RNAse III-like nuclease DCR-1, the dsRNA-binding
  proteins RDE-1 and -4, the helicases DRH-1 and DRH-3/EKL-3, the RNA-dependent RNA-polymerase
  (RdRP) RRF-1, and the uncharacterized protein, AIN-1. Dicer cleaves dsRNA to produce
  siRNA molecules, and pre-miRNA to mature miRNA, both of which are substrates for
  the RNA-induced silencing complex (RISC). (4) Both siRNAs and miRNAs are the focus
  of a battery of inhibitors, which allow down-regulation of the RNAi response. (5)
  The RISC complex incorporates a single strand of miRNA or siRNA (termed the guide
  strand), and binds a complementary mRNA strand, eliciting gene silencing by either
  mRNA destruction or translational repression (6). The central catalytic component
  of RISC is an argonaute (AGO) protein, allied with the nuclease TSN-1, the RNA-binding
  protein VIG-1, and AIN-1. (7) The RNAi response may be amplified by the action of
  the RdRPs RRF-1 and -2, SMG-5, RDE-2/MUT-8 and MUT-7, which produce a population
  of single-stranded RNAs bearing N-terminal tri-phosphates from a target mRNA template.
  (8) These secondary siRNAs interact with Secondary-siRNA-specific AGOs (SAGO-1 and
  -2), terminating in down-regulation of target transcript. Secondary siRNAs can also
  spread between cells through RSD-2, -3 and -6, resulting in intercellular spread
  of the RNAi effect (9), and can be imported into the nucleus by NRDE-3, which elicits
  transcriptional silencing of nascent RNA transcripts as part of nuclear RISC (nucRISC)
  (10). siRNAs may also control aspects of nuclear RNAi, including histone methylation,
  chromatin formation and chromosome segregation (11). Dashed lines indicate miRNA-based
  pathways, solid lines indicate siRNA-based pathways.
papertitle: RNAi Effector Diversity in Nematodes.
reftext: Johnathan J. Dalzell, et al. PLoS Negl Trop Dis. 2011 Jun;5(6):e1176.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9083864
figid_alias: PMC3110158__F1
figtype: Figure
redirect_from: /figures/PMC3110158__F1
ndex: 2640fd85-df12-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3110158__pntd.0001176.g001.html
  '@type': Dataset
  description: (1) Exogenously applied double-stranded RNA (dsRNA) and small interfering
    RNA (exo-siRNA) are thought to enter cells via SID (Systemic RNA Interference
    Defective) proteins SID-1/RSD-8 and SID-2. (2) Endogenous RNAi-based pathways
    begin in the nucleus; micro-interfering RNA (miRNA) synthesis begins with transcription
    of hairpin-looped primary miRNA (pri-miRNA) transcripts from intergenic, intronic
    or antisense regions. pri-miRNAs are processed by the DRSH-1/PASH-1 complex to
    pre-miRNA, which are exported from the nucleus by exportin proteins XPO-1, -2
    and -3. Endogenous siRNAs (endo-siRNAs) are also produced from genomic regions,
    and exported by XPO-1, 2, and -3. (3) Both pre-miRNAs and exogenously applied
    dsRNA molecules are bound and cleaved by the dicer complex, which consists of
    the RNAse III-like nuclease DCR-1, the dsRNA-binding proteins RDE-1 and -4, the
    helicases DRH-1 and DRH-3/EKL-3, the RNA-dependent RNA-polymerase (RdRP) RRF-1,
    and the uncharacterized protein, AIN-1. Dicer cleaves dsRNA to produce siRNA molecules,
    and pre-miRNA to mature miRNA, both of which are substrates for the RNA-induced
    silencing complex (RISC). (4) Both siRNAs and miRNAs are the focus of a battery
    of inhibitors, which allow down-regulation of the RNAi response. (5) The RISC
    complex incorporates a single strand of miRNA or siRNA (termed the guide strand),
    and binds a complementary mRNA strand, eliciting gene silencing by either mRNA
    destruction or translational repression (6). The central catalytic component of
    RISC is an argonaute (AGO) protein, allied with the nuclease TSN-1, the RNA-binding
    protein VIG-1, and AIN-1. (7) The RNAi response may be amplified by the action
    of the RdRPs RRF-1 and -2, SMG-5, RDE-2/MUT-8 and MUT-7, which produce a population
    of single-stranded RNAs bearing N-terminal tri-phosphates from a target mRNA template.
    (8) These secondary siRNAs interact with Secondary-siRNA-specific AGOs (SAGO-1
    and -2), terminating in down-regulation of target transcript. Secondary siRNAs
    can also spread between cells through RSD-2, -3 and -6, resulting in intercellular
    spread of the RNAi effect (9), and can be imported into the nucleus by NRDE-3,
    which elicits transcriptional silencing of nascent RNA transcripts as part of
    nuclear RISC (nucRISC) (10). siRNAs may also control aspects of nuclear RNAi,
    including histone methylation, chromatin formation and chromosome segregation
    (11). Dashed lines indicate miRNA-based pathways, solid lines indicate siRNA-based
    pathways.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ain-1
  - vig-1
  - tsn-1
  - rrf-3
  - rrf-1
  - ego-1
  - rde-2
  - mut-7
  - adr-1
  - eri-1
  - eri-3
  - eri-5
  - eri-6
  - adr-2
  - eri-7
  - nrde-3
  - rde-1
  - drh-3
  - rde-4
  - dcr-1
  - drh-1
  - sid-1
  - sid-2
  - drsh-1
  - pash-1
  - Tudor-SN
  - amon
  - rsd
  - Act88F
  - eg-1
  - smg
  - nonC
  - SNRPG
  - ago
  - AGO1
  - AGO2
  - lin
  - drl
  - Dcr-1
  - ninaE
  - Dcr-2
  - Nuc
  - XRN4
  - Tudor1
  - ADR1
  - WRR4
  - WRNEXO
  - DRH1
  - NUC
  - MVP1
  - EDS5
  - EDS16
  - RSAD2
  - MED28
  - SAMD4A
  - EXD3
  - FBXW7
  - ERI1
  - ERI3
  - CLCN5
  - TNFRSF10C
  - DICER1
  - NUCB1
  - SCPEP1
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - SIDT1
  - RRF1
  - MEH1
  - ADH2
  - GID8
  - TAH11
  - rna
---
