---
figid: PMC3434495__zmb9991095470007
figlink: /pmc/articles/PMC3434495/figure/F7/
number: Fig 7
caption: Translational termination of deadenylated transcripts promotes an NMD-like
  response. Translational termination of poly(A) transcripts is known to involve recruitment
  of eRF1 and the GTPase eRF3 () (A). Critical to efficient ribosome dissociation
  is the interaction between eRF3 and poly(A) binding protein (PAB1) (oval with “AA”).
  The association of the 5′-end cap structure (thick gray zero in a circle) with PAB1
  facilitates reinitiation of translation () (B). During translation, the poly(A)
  tail is gradually shortened by deadenylase complexes (, , –, , ); in the case of
  A. nidulans, this has been shown to involve the CCR4-Not complex (, ). Under these
  circumstances, with a long poly(A) tail bound by PAB1 and an active deadenylase,
  CUTA/B-mediated 3′ tagging is inhibited (). After deadenylation has progressed to
  a critical point of ∼15A nucleotides, poly(A) binding protein dissociates from the
  tail and the processivity of the deadenylase complex is altered (C). In the absence
  of PAB1, we propose that the terminating ribosome will recruit Upf1 through its
  interaction with eRF3, as is known to occur during NMD (E and F), where a premature
  termination codon (PTC) is encountered sufficiently distant from the poly(A) tail
  and associated PAB1 (). Under these circumstances, ribosome dissociation is inhibited
  (, ) and the NMD pathway is triggered. Based on our data, efficient clearance of
  this termination complex requires 3′ tagging. Consistent with this, pyrimidine tagging
  generally occurs in transcripts with short poly(A) tails (D) () or in the case of
  NMD is poly(A) independent (F). The other well-characterized example is for the
  nonadenylated human histone mRNAs upon S-phase arrest (). In A. nidulans, 3′ tagging
  of short poly(A) tails is not completely lost in Δupf1 strains, suggesting that
  other factors are involved in facilitating the process, possibly the loss of PAB1
  and/or altered interactions with the deadenylase complex. Generally, 3′ tagging
  is a precursor to mRNA decapping; one possibility is that a pyrimidine tract enhances
  affinity for the LSM-PAT1 complex (, ), recruitment of which (D and E) is known
  to initiate a cascade of events, promoting transcript decapping and degradation
  as well as the inhibition of translational initiation (). We propose that recruitment
  of the LSM-PAT1 complex will also facilitate dissociation of the mRNA from the relatively
  stable termination complex which is formed in the absence of PAB1, facilitating
  efficient clearance from the polysomes.
pmcid: PMC3434495
papertitle: mRNA 3′ Tagging Is Induced by Nonsense-Mediated Decay and Promotes Ribosome
  Dissociation.
reftext: Igor Y. Morozov, et al. Mol Cell Biol. 2012 Jul;32(13):2585-2595.
pmc_ranked_result_index: '230978'
pathway_score: 0.9201409
filename: zmb9991095470007.jpg
figtitle: Translational termination of deadenylated transcripts promotes an NMD-like
  response
year: '2012'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3434495__zmb9991095470007.html
  '@type': Dataset
  description: Translational termination of deadenylated transcripts promotes an NMD-like
    response. Translational termination of poly(A) transcripts is known to involve
    recruitment of eRF1 and the GTPase eRF3 () (A). Critical to efficient ribosome
    dissociation is the interaction between eRF3 and poly(A) binding protein (PAB1)
    (oval with “AA”). The association of the 5′-end cap structure (thick gray zero
    in a circle) with PAB1 facilitates reinitiation of translation () (B). During
    translation, the poly(A) tail is gradually shortened by deadenylase complexes
    (, , –, , ); in the case of A. nidulans, this has been shown to involve the CCR4-Not
    complex (, ). Under these circumstances, with a long poly(A) tail bound by PAB1
    and an active deadenylase, CUTA/B-mediated 3′ tagging is inhibited (). After deadenylation
    has progressed to a critical point of ∼15A nucleotides, poly(A) binding protein
    dissociates from the tail and the processivity of the deadenylase complex is altered
    (C). In the absence of PAB1, we propose that the terminating ribosome will recruit
    Upf1 through its interaction with eRF3, as is known to occur during NMD (E and
    F), where a premature termination codon (PTC) is encountered sufficiently distant
    from the poly(A) tail and associated PAB1 (). Under these circumstances, ribosome
    dissociation is inhibited (, ) and the NMD pathway is triggered. Based on our
    data, efficient clearance of this termination complex requires 3′ tagging. Consistent
    with this, pyrimidine tagging generally occurs in transcripts with short poly(A)
    tails (D) () or in the case of NMD is poly(A) independent (F). The other well-characterized
    example is for the nonadenylated human histone mRNAs upon S-phase arrest (). In
    A. nidulans, 3′ tagging of short poly(A) tails is not completely lost in Δupf1
    strains, suggesting that other factors are involved in facilitating the process,
    possibly the loss of PAB1 and/or altered interactions with the deadenylase complex.
    Generally, 3′ tagging is a precursor to mRNA decapping; one possibility is that
    a pyrimidine tract enhances affinity for the LSM-PAT1 complex (, ), recruitment
    of which (D and E) is known to initiate a cascade of events, promoting transcript
    decapping and degradation as well as the inhibition of translational initiation
    (). We propose that recruitment of the LSM-PAT1 complex will also facilitate dissociation
    of the mRNA from the relatively stable termination complex which is formed in
    the absence of PAB1, facilitating efficient clearance from the polysomes.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CUTA
  - ETF1
  - TAS2R38
genes:
- word: CUTA/B
  symbol: CUTA
  source: hgnc_symbol
  hgnc_symbol: CUTA
  entrez: '51596'
- word: ERF1
  symbol: ERF1
  source: hgnc_prev_symbol
  hgnc_symbol: ETF1
  entrez: '2107'
- word: PTC
  symbol: PTC
  source: hgnc_prev_symbol
  hgnc_symbol: TAS2R38
  entrez: '5726'
chemicals: []
diseases: []
---
