---
figid: PMC3799303__pnas.1315623110fig01
figtitle: Simplified representation of the repair of double-strand breaks
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC3799303
filename: pnas.1315623110fig01.jpg
figlink: /pmc/articles/PMC3799303/figure/fig01/
number: F1
caption: Simplified representation of the repair of double-strand breaks. Sequence-specific
  endonucleases can be designed to cleave genome targets (represented by scissors).
  In general, cleaved sites can be rejoined to correct the double-strand break. This
  cycle can continue until the nuclease is no longer present or until an error occurs
  during repair that prevents recleavage. Exonuclease activity can truncate the free
  ends (bases that have been removed are shown in white). When both strands are truncated,
  deletions can be produced by nonhomologous end-joining (NHEJ). The change in local
  spacing produces a sequence that cannot be recleaved (Left pathway). If exonuclease
  activity removes bases from one strand, a gap can be created after NHEJ. The missing
  bases can be replaced by polymerase activity (Center pathway). Although this process
  generally proceeds with high fidelity, errors can occur (C > T error shown in red).
  If homologous DNA is present (shown in green), homology-directed repair (HDR) can
  result in the introduction of specific modifications (C > T modification shown in
  red). The result shown here (Right pathway) produces a product that is identical
  to the error shown for gap repair. After further processing through mismatch repair,
  the C > T substitution can be corrected or can retain the substitution. In either
  case, the resulting products may be recleaved. Larger modifications can be made
  than shown here. In addition, larger repair templates can be used to facilitate
  homologous recombination.
papertitle: Natural genotypes via genetic engineering.
reftext: Kevin D. Wells. Proc Natl Acad Sci U S A. 2013 Oct 8;110(41):16295-16296.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9242014
figid_alias: PMC3799303__F1
figtype: Figure
redirect_from: /figures/PMC3799303__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3799303__pnas.1315623110fig01.html
  '@type': Dataset
  description: Simplified representation of the repair of double-strand breaks. Sequence-specific
    endonucleases can be designed to cleave genome targets (represented by scissors).
    In general, cleaved sites can be rejoined to correct the double-strand break.
    This cycle can continue until the nuclease is no longer present or until an error
    occurs during repair that prevents recleavage. Exonuclease activity can truncate
    the free ends (bases that have been removed are shown in white). When both strands
    are truncated, deletions can be produced by nonhomologous end-joining (NHEJ).
    The change in local spacing produces a sequence that cannot be recleaved (Left
    pathway). If exonuclease activity removes bases from one strand, a gap can be
    created after NHEJ. The missing bases can be replaced by polymerase activity (Center
    pathway). Although this process generally proceeds with high fidelity, errors
    can occur (C > T error shown in red). If homologous DNA is present (shown in green),
    homology-directed repair (HDR) can result in the introduction of specific modifications
    (C > T modification shown in red). The result shown here (Right pathway) produces
    a product that is identical to the error shown for gap repair. After further processing
    through mismatch repair, the C > T substitution can be corrected or can retain
    the substitution. In either case, the resulting products may be recleaved. Larger
    modifications can be made than shown here. In addition, larger repair templates
    can be used to facilitate homologous recombination.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CycE
  - cyc
  - double
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - ct
---
