---
figid: PMC3064560__nihms-266333-f0005
figtitle: Analogue 3 also showed potent antiproliferative activity against the highly
  invasive non-small-cell lung cancer cell line CL1-5 with an ED50 value of 0.11 μM
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3064560
filename: nihms-266333-f0005.jpg
figlink: /pmc/articles/PMC3064560/figure/F4/
number: F3
caption: Analogue 3 also showed potent antiproliferative activity against the highly
  invasive non-small-cell lung cancer cell line CL1-5 with an ED50 value of 0.11 μM.
  To determine which genes were differentially expressed upon CL1-5 treatment with
  analogue 3, the genome-wide mRNA expression profiles of 3-treated cells and control
  cells were determined using Affymetrix human genome U133 plus 2.0 GeneChip according
  to the Manufacturer's protocols (Santa Clara, CA, http://www.affymetrix.com) by
  the Microarray Core Facility of National Research Program for Genomic Medicine of
  National Science Council in Taiwan as previously described. This Affymetrix GeneChip
  contains 54,675 probe sets to analyze the expression levels of 47,400 transcripts
  and variants, including 38,500 well-characterized human genes. GeneChips from the
  hybridization experiments were read by the Affymetrix GeneChip scanner 3000 7G,
  and raw data were processed using GC-RMA algorithm. The raw data were then analyzed
  by GeneSpring GX software version 11.01. 2.5 × 105 CL1-5 cells were treated for
  24 h with 3 at a concentration of 0.05 μg/mL, and then total RNA was extracted by
  TRI zol (Life Technologies, Gaithersburg, MD) RNA from non-treated CL1-5 cells was
  used as a control. A total of 2,838 genes showed at least two-fold changes in expression
  levels between the CL1-5 treated with 3 and CL1-5 DMSO control. Analogue 3 up-regulated
  the expression of 1,112 genes and down-regulate 1,726 genes. The differentially
  expressed genes were analyzed for GeneGo canonical pathway maps by using MetaCore
  Analytical Suite (GeneGo Inc., St Joseph, MI). The top ten pathways involved in
  analog 3 affected genes were shown in . Seven pathways are cell cycle-related pathways
  and three are DNA damage-related. For example, in the spindle assembly checkpoint
  (SAC) or chromosome segregation pathway, the genes altered by the treatment with
  3 encoded mitotic kinases (e.g., CDK1-cyclin B, Aurora A, Aurora B, and NEK2A),
  SAC proteins (e.g., MAD1, MAD2, securin, and separase), and motor proteins (e.g.,
  dynein1, dynein activator complex dynactin, and KNSL1) (). The cyclin dependent
  kinase, Aurora kinases, and NEK2A kinases are critical for mitotic progression,
  through phosphorylation of their numerous substrates. NEK2A or Aurora kinases are
  required for spindle formation at the onset of mitosis or chromosome segregation
  and cytokinesis, respectively. The SAC proteins, such as MAD2, activate spindle
  checkpoint and inhibit securin degradation, until all chromosomes are aligned at
  the metaphase plate. When chromosomes are aligned correctly, the E3 ubiquitin ligase
  anaphase-promoting complex/cyclosome (APC/C) inhibitor MAD2 is dissociated from
  the APC/C and removed from the attached kinetochore by dynein. Subsequently, APC/C
  is activated by CDC20 or CDH1. APC/C-CDC20 or -CDH1 recognizes substrates such as
  cyclins, NEK2A, and securin or Aurora kinases and cyclins, respectively. At the
  onset of anaphase, the separase inhibitor securin is poly-ubiquitinated by activated
  APC/C followed by digestion by the proteasome. Subsequently, activated separase
  cleaves thecohesin complex, resulting in separation of the sister chromatids. All
  of these proteins are expressed in a cell cycle-dependent manner. In our oligonucleotide
  microarray studies, genes encoding these proteins were up-regulated by the treatment
  with 3. The up-regulation of MAD2L1 transcript was confirmed by semi-quantitative
  RT-PCR (). Therefore, we assume that 3 may modulate SAC and chromosome separation,
  and conclude that 3 induces cell cycle arrest mainly in the G2/M-phase. Because
  oligonucleotide microarray data are quite complicated and can be contradictory,
  we will need to conduct additional experiments, such as real-time qPCR, to verify
  our results.
papertitle: 'Antitumor Agents 283. Further Elaboration of Desmosdumotin C Analogs
  as Potent Antitumor Agents: Activation of Spindle Assembly Checkpoint as Possible
  Mode of Action.'
reftext: Kyoko Nakagawa-Goto, et al. Bioorg Med Chem. ;19(5):1816-1822.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9477456
figid_alias: PMC3064560__F3
figtype: Figure
redirect_from: /figures/PMC3064560__F3
ndex: fd4be7da-dead-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3064560__nihms-266333-f0005.html
  '@type': Dataset
  description: Analogue 3 also showed potent antiproliferative activity against the
    highly invasive non-small-cell lung cancer cell line CL1-5 with an ED50 value
    of 0.11 μM. To determine which genes were differentially expressed upon CL1-5
    treatment with analogue 3, the genome-wide mRNA expression profiles of 3-treated
    cells and control cells were determined using Affymetrix human genome U133 plus
    2.0 GeneChip according to the Manufacturer's protocols (Santa Clara, CA, http://www.affymetrix.com)
    by the Microarray Core Facility of National Research Program for Genomic Medicine
    of National Science Council in Taiwan as previously described. This Affymetrix
    GeneChip contains 54,675 probe sets to analyze the expression levels of 47,400
    transcripts and variants, including 38,500 well-characterized human genes. GeneChips
    from the hybridization experiments were read by the Affymetrix GeneChip scanner
    3000 7G, and raw data were processed using GC-RMA algorithm. The raw data were
    then analyzed by GeneSpring GX software version 11.01. 2.5 × 105 CL1-5 cells were
    treated for 24 h with 3 at a concentration of 0.05 μg/mL, and then total RNA was
    extracted by TRI zol (Life Technologies, Gaithersburg, MD) RNA from non-treated
    CL1-5 cells was used as a control. A total of 2,838 genes showed at least two-fold
    changes in expression levels between the CL1-5 treated with 3 and CL1-5 DMSO control.
    Analogue 3 up-regulated the expression of 1,112 genes and down-regulate 1,726
    genes. The differentially expressed genes were analyzed for GeneGo canonical pathway
    maps by using MetaCore Analytical Suite (GeneGo Inc., St Joseph, MI). The top
    ten pathways involved in analog 3 affected genes were shown in . Seven pathways
    are cell cycle-related pathways and three are DNA damage-related. For example,
    in the spindle assembly checkpoint (SAC) or chromosome segregation pathway, the
    genes altered by the treatment with 3 encoded mitotic kinases (e.g., CDK1-cyclin
    B, Aurora A, Aurora B, and NEK2A), SAC proteins (e.g., MAD1, MAD2, securin, and
    separase), and motor proteins (e.g., dynein1, dynein activator complex dynactin,
    and KNSL1) (). The cyclin dependent kinase, Aurora kinases, and NEK2A kinases
    are critical for mitotic progression, through phosphorylation of their numerous
    substrates. NEK2A or Aurora kinases are required for spindle formation at the
    onset of mitosis or chromosome segregation and cytokinesis, respectively. The
    SAC proteins, such as MAD2, activate spindle checkpoint and inhibit securin degradation,
    until all chromosomes are aligned at the metaphase plate. When chromosomes are
    aligned correctly, the E3 ubiquitin ligase anaphase-promoting complex/cyclosome
    (APC/C) inhibitor MAD2 is dissociated from the APC/C and removed from the attached
    kinetochore by dynein. Subsequently, APC/C is activated by CDC20 or CDH1. APC/C-CDC20
    or -CDH1 recognizes substrates such as cyclins, NEK2A, and securin or Aurora kinases
    and cyclins, respectively. At the onset of anaphase, the separase inhibitor securin
    is poly-ubiquitinated by activated APC/C followed by digestion by the proteasome.
    Subsequently, activated separase cleaves thecohesin complex, resulting in separation
    of the sister chromatids. All of these proteins are expressed in a cell cycle-dependent
    manner. In our oligonucleotide microarray studies, genes encoding these proteins
    were up-regulated by the treatment with 3. The up-regulation of MAD2L1 transcript
    was confirmed by semi-quantitative RT-PCR (). Therefore, we assume that 3 may
    modulate SAC and chromosome separation, and conclude that 3 induces cell cycle
    arrest mainly in the G2/M-phase. Because oligonucleotide microarray data are quite
    complicated and can be contradictory, we will need to conduct additional experiments,
    such as real-time qPCR, to verify our results.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pr
  - alphaTub67C
  - alphaTub84B
  - alphaTub84D
  - alphaTub85E
  - CycB
  - Cdk1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Rcc1
  - ec
  - Zw10
  - Mad1
  - mad2
  - Ran
  - RanGAP
  - aurB
  - vtd
  - red
  - trc
  - Neurl4
  - CSE1L
  - AQP6
  - GJB2
  - KIF22
  - CCNB1
  - CCNB2
  - CCNB3
  - CDK1
  - TNFSF14
  - NEK2
  - RCC1
  - ZW10
  - MXD1
  - MAD1L1
  - MAD2L1
  - MXI1
  - RAN
  - TPX2
  - KIF11
  - RAD21
  - SMC3
  - IK
  - DYRK3
---
