---
figid: PMC3004842__1475-2859-9-94-2
figtitle: Flux-sum intensity in Z
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
pmcid: PMC3004842
filename: 1475-2859-9-94-2.jpg
figlink: /pmc/articles/PMC3004842/figure/F2/
number: F2
caption: Central metabolism and analysis of flux-sum intensity in Z. mobilis. (A)
  Central metabolism, including ED Pathway and TCA cycle in Z. mobilis based on annotated
  genes. Reactions for utilizing pentose sugars in Z. mobilis are introduced and represented
  to blue lines. The red lines indicate the missing reactions in Z. mobilis. (B) Flux-sum
  intensity of selected metabolites, including cofactors, amino acids and others,
  in central metabolism for Z. mobilis and E. coli, respectively, for three carbon
  sources (i.e. glucose, fructose, and sucrose). Uptake rate of each carbon source
  is fixed to 10 mmol/gDCW/h, and reaction for oxygen uptaking was deleted to describe
  anaerobic condition in both organisms. NGAME (i.e. non-growth associated maintenance
  energy) value for both organisms was eliminated. Upper constraints (i.e. limit the
  flux on fumaric acid, acetic acid, malic acid, acetoin, and acetaldehyde production)
  in Z. mobilis metabolic model for regulating the flux more realistically are relieved.
  Results are normalized by maximum value of each metabolite. Darker color indicates
  stronger flux-sum intensity.
papertitle: The genome-scale metabolic network analysis of Zymomonas mobilis ZM4 explains
  physiological features and suggests ethanol and succinic acid production strategies.
reftext: Kyung Yun Lee, et al. Microb Cell Fact. 2010;9:94-94.
year: '2010'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5248549
figid_alias: PMC3004842__F2
figtype: Figure
redirect_from: /figures/PMC3004842__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3004842__1475-2859-9-94-2.html
  '@type': Dataset
  description: Central metabolism and analysis of flux-sum intensity in Z. mobilis.
    (A) Central metabolism, including ED Pathway and TCA cycle in Z. mobilis based
    on annotated genes. Reactions for utilizing pentose sugars in Z. mobilis are introduced
    and represented to blue lines. The red lines indicate the missing reactions in
    Z. mobilis. (B) Flux-sum intensity of selected metabolites, including cofactors,
    amino acids and others, in central metabolism for Z. mobilis and E. coli, respectively,
    for three carbon sources (i.e. glucose, fructose, and sucrose). Uptake rate of
    each carbon source is fixed to 10 mmol/gDCW/h, and reaction for oxygen uptaking
    was deleted to describe anaerobic condition in both organisms. NGAME (i.e. non-growth
    associated maintenance energy) value for both organisms was eliminated. Upper
    constraints (i.e. limit the flux on fumaric acid, acetic acid, malic acid, acetoin,
    and acetaldehyde production) in Z. mobilis metabolic model for regulating the
    flux more realistically are relieved. Results are normalized by maximum value
    of each metabolite. Darker color indicates stronger flux-sum intensity.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATPsynbeta
  - Atpalpha
  - gr
  - adp
  - Pgi
  - aop
  - pg
  - Pep
  - bgcn
  - Ald1
  - pyr
  - ac
  - ad
  - lncRNA:acal
  - Eno
  - rasp
  - da
  - nAChRalpha2
  - sti
  - ld
  - PyK
  - Me
  - dnc
  - mal
  - bs
  - Mrtf
  - Fum1
  - Adh
  - arg
  - aos
  - Mad
  - CG2662
  - Mnt
  - ash2
  - tRNA:Gln-TTG-2-1
  - tRNA:Gln-TTG-1-1
  - tRNA:Gln-TTG-1-2
  - tRNA:Gln-TTG-1-3
  - cs
  - Cys
  - mmy
  - Csp
  - Acn
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - cof
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - pi
  - Nach
  - Suc
  - Coa
  - araH
  - zwf
  - pgi
  - pgl
  - fepA
  - fbp
  - edd
  - fbaA
  - eda
  - eno
  - sucC
  - carA
  - acnA
  - mdh
  - icd
  - ATP8A2
  - WDTC1
  - SART1
  - DECR1
  - BGN
  - GPI
  - PGPEP1
  - SDHD
  - GPC3
  - OTOR
  - TKT
  - UBR5
  - PAEP
  - PREP
  - PTPN22
  - FBXO3
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - EDA
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - CIT
  - DHDDS
  - PHKA2
  - ALDH7A1
  - PDE2A
  - PDE9A
  - PDE10A
  - PDE11A
  - PDE1A
  - PDE1B
  - PDE1C
  - PDE3A
  - PDE3B
  - PDE4A
  - PDE4B
  - PDE4C
  - PDE4D
  - PDE6A
  - PDE6B
  - PDE6C
  - PDE7A
  - PDE7B
  - PDE8A
  - PDE8B
  - MAL
  - MRTFA
  - TIRAP
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - ABL2
  - RERE
  - AMPD1
  - MXD1
  - TSHZ1
  - CS
  - NF2
  - ACIN1
  - MDH2
  - GNPTAB
  - SARDH
  - SDHB
  - SORD
  - SDS
  - BRCA2
  - FANCD2
  - PSEN1
  - Ribulose
  - Acetaldehyde
  - ALA
  - Ethanol
  - Arabinose
  - ARG
  - CYS
  - FAD
  - Fructose
  - Fructose-6P
  - Ribulose-5P Fructose
  - 6P-gluconate
  - D-Glucose
  - Glucose
  - Glyceraldehyde
  - GLY
  - Isocitrate
  - LEU
  - Oxaloacetate
  - PHE
  - phosphate
  - Phosphoenolpyruvate
  - PRO
  - Ser
  - Succinate
  - Sucrose
  - TYR
  - VAL
---
