---
figid: PMC3855486__pone.0082705.g001
figtitle: New Insights into the Phylogeny and Molecular Classification of Nicotinamide
  Mononucleotide Deamidases
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC3855486
filename: pone.0082705.g001.jpg
figlink: /pmc/articles/PMC3855486/figure/pone-0082705-g001/
number: F1
caption: 'The routes known to be functional across diverse bacterial species are shown
  by solid lines. Preiss-Handler pathway is shadowed. Dashed and hollow arrows relate
  to uptake and de novo NaMN synthesis, respectively. Blue arrow corresponds to nicotinamidase
  activity (PncA), Red arrow corresponds to nicotinamide mononucleotide deamidase
  activity (PncC) and green arrow corresponds to NAD+-dependent DNA ligase. Enzymes
  are indicated as the acronym used to identify the corresponding gene locus: NadA,
  quinolinate synthetase; NadB, L-aspartate oxidase; NadC, quinolinate phosphoribosyl
  transferase; NadD, NaMN adenylyltransferase; NadE, NAD synthetase; NadM, NMN adenyltransferase;
  NadRC, NmR kinase; NadRN, NMN adenylyltransferase; NadV, Nm phosphoribosyltransferase;
  PncA, Nam deamidase; PncB, Na phosphoribosyltransferase; PncC, nicotinamide mononucleotide
  deamidase; PncU, nucleoside permease. The gene name of the enzymes existing in O.
  iheyensis is highlighted in brown. Abbreviations: NAD, nicotinamide adenine dinucleotide;
  IA, α-iminosuccinate; Qa, quinolonic acid; Asp, aspartate; Trp, tryptophan; NAM,
  nicotinamide; NA: nicotinic acid; NMN, nicotinic acid mononucleotide; NaAD, nicotinic
  acid adenine dinucleotide; NmR, nicotinamide riboside; DNA, deoxyribonucleic acid.'
papertitle: New Insights into the Phylogeny and Molecular Classification of Nicotinamide
  Mononucleotide Deamidases.
reftext: Guiomar Sánchez-Carrón, et al. PLoS One. 2013;8(12):e82705.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9605158
figid_alias: PMC3855486__F1
figtype: Figure
redirect_from: /figures/PMC3855486__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3855486__pone.0082705.g001.html
  '@type': Dataset
  description: 'The routes known to be functional across diverse bacterial species
    are shown by solid lines. Preiss-Handler pathway is shadowed. Dashed and hollow
    arrows relate to uptake and de novo NaMN synthesis, respectively. Blue arrow corresponds
    to nicotinamidase activity (PncA), Red arrow corresponds to nicotinamide mononucleotide
    deamidase activity (PncC) and green arrow corresponds to NAD+-dependent DNA ligase.
    Enzymes are indicated as the acronym used to identify the corresponding gene locus:
    NadA, quinolinate synthetase; NadB, L-aspartate oxidase; NadC, quinolinate phosphoribosyl
    transferase; NadD, NaMN adenylyltransferase; NadE, NAD synthetase; NadM, NMN adenyltransferase;
    NadRC, NmR kinase; NadRN, NMN adenylyltransferase; NadV, Nm phosphoribosyltransferase;
    PncA, Nam deamidase; PncB, Na phosphoribosyltransferase; PncC, nicotinamide mononucleotide
    deamidase; PncU, nucleoside permease. The gene name of the enzymes existing in
    O. iheyensis is highlighted in brown. Abbreviations: NAD, nicotinamide adenine
    dinucleotide; IA, α-iminosuccinate; Qa, quinolonic acid; Asp, aspartate; Trp,
    tryptophan; NAM, nicotinamide; NA: nicotinic acid; NMN, nicotinic acid mononucleotide;
    NaAD, nicotinic acid adenine dinucleotide; NmR, nicotinamide riboside; DNA, deoxyribonucleic
    acid.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - nadB
  - pncC
  - nadA
  - nadC
  - nadD
  - pncB
  - nadE
  - pncA
  - nadR
  - pnuC
  - Asp
  - Trp
---
