---
figid: PMC3287586__1471-2164-12-S4-S3-2
figtitle: Preimplantation development pathway
organisms:
- Bos taurus
- Drosophila melanogaster
- Eutheria
- Eumetazoa
- Homo sapiens
- Mus musculus
- Syringa vulgaris
- Euteleostomi
organisms_ner:
- Mus musculus
- Homo sapiens
- Bos taurus
- Drosophila melanogaster
- Danio rerio
pmcid: PMC3287586
filename: 1471-2164-12-S4-S3-2.jpg
figlink: /pmc/articles/PMC3287586/figure/F2/
number: F2
caption: Preimplantation development pathway. The figure shows a pathway representation
  of the genes involved in the regulation of the preimplantation development and interactions
  between them. Some functions are also detailed in the grey rectangles. In the upper
  part of the figure are located genes involved in the early stages of development
  (until blastocyst formation) below these, the left part corresponds to the regulations
  that occur in the inner cell mass, the portion of cells that remains undifferentiated
  during a longer period of time, part of these cells will give rise to the primitive
  endoderm and the genes that regulate this process are shown in the bottom left.
  In the right are the genes involved in the development of the outer cells of the
  blastocyst, which differentiate to form the trophectoderm. The interactions are
  described in the text. KEGG Markup Language was used for pathway representation.
  The developmental stages figures were adapted from Yamanaka et al. 2006 []. The
  pathway genes are represented according to their ancestry based on the determination
  of their Last Common Ancestor. Genes considered recent are shown in green while
  genes of more ancient origin are shown in lilac. Genes that present an ortholog
  in D. melanogaster are marked (*). This will be further adressed in the text section
  “Pathway Ancestry”. DPPA1 and hRSCP are shown in grey due to the fact that the lack
  of corresponding SwissProt annotated gene product to be used as seed prevented their
  use in this analysis.
papertitle: Preimplantation development regulatory pathway construction through a
  text-mining approach.
reftext: Elisa Donnard, et al. BMC Genomics. 2011;12(Suppl 4):S3-S3.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8750802
figid_alias: PMC3287586__F2
figtype: Figure
redirect_from: /figures/PMC3287586__F2
ndex: 33cf2aaf-df1a-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3287586__1471-2164-12-S4-S3-2.html
  '@type': Dataset
  description: Preimplantation development pathway. The figure shows a pathway representation
    of the genes involved in the regulation of the preimplantation development and
    interactions between them. Some functions are also detailed in the grey rectangles.
    In the upper part of the figure are located genes involved in the early stages
    of development (until blastocyst formation) below these, the left part corresponds
    to the regulations that occur in the inner cell mass, the portion of cells that
    remains undifferentiated during a longer period of time, part of these cells will
    give rise to the primitive endoderm and the genes that regulate this process are
    shown in the bottom left. In the right are the genes involved in the development
    of the outer cells of the blastocyst, which differentiate to form the trophectoderm.
    The interactions are described in the text. KEGG Markup Language was used for
    pathway representation. The developmental stages figures were adapted from Yamanaka
    et al. 2006 []. The pathway genes are represented according to their ancestry
    based on the determination of their Last Common Ancestor. Genes considered recent
    are shown in green while genes of more ancient origin are shown in lilac. Genes
    that present an ortholog in D. melanogaster are marked (*). This will be further
    adressed in the text section “Pathway Ancestry”. DPPA1 and hRSCP are shown in
    grey due to the fact that the lack of corresponding SwissProt annotated gene product
    to be used as seed prevented their use in this analysis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Dnmt1
  - Ctcfl
  - Ctnnb1
  - Cdh1
  - Fzr1
  - Gjb3
  - Pou5f1
  - Stpg4
  - Rap1gap
  - Rasa1
  - Myc
  - Nol3
  - Pard3
  - Bmp4
  - Id2
  - te
  - Sap18
  - Cripto
  - Nr0b1
  - Hand1
  - Tgfb1
  - Ltbp1
  - Nle1
  - Dpp3
  - Smad1
  - Nfkb1
  - Upf1
  - Afp
  - Lamb3
  - Fgf4
  - Dob1
  - Lif
  - Lifr
  - Esrrb
  - Stat3
  - Gata2
  - Sox2
  - Pecam1
  - Nanog
  - Klf4
  - Rps14
  - Tead4
  - Yap1
  - Dppa5-ps7
  - Dppa5-ps6
  - Dppa5-ps5
  - Dppa5-ps4
  - Dppa5-ps3
  - Dppa5-ps2
  - Dppa5-ps1
  - Dppa5a
  - Mybl2
  - Gcm1
  - Gcm2
  - Tle2
  - Tcf3
  - Tcf7l1
  - Foxd3
  - Thy1
  - Lefty1
  - Eomes
  - Krt18
  - Lefty2
  - COX2
  - Ptgs2
  - Angptl6
  - Actr3
  - Apobec3
  - Cdx2
  - Wnt6
  - Snai1
  - Cer1
  - Acer1
  - Gata4
  - Dsc2
  - Fgf3
  - Hnf4a
  - Dab2
  - Sox17
  - Wnt9a
  - DNMT1
  - CTCFL
  - CTNNB1
  - CDH1
  - FZR1
  - GJB3
  - POU5F1
  - POU5F1P3
  - POU5F1P4
  - CTLA4
  - STPG4
  - CELIAC2
  - RASA1
  - RGS6
  - MYC
  - FBN1
  - PARD3
  - BMP4
  - ID2
  - SAP18
  - CRIPTO
  - NR0B1
  - HAND1
  - TGFB1
  - TGFB2
  - TGFB3
  - NLE1
  - DPP3
  - SMAD1
  - GARS1
  - NFKB1
  - RELA
  - RELB
  - REL
  - NFKB2
  - UPF1
  - AFP
  - TRIM26
  - GATA5
  - LAMB3
  - FGF4
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - MTREX
  - LIF
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - LIFR
  - ESRRB
  - STAT3
  - GATA2
  - YAE1
  - SOX2
  - NANOG
  - KLF4
  - RPS14
  - TEAD4
  - YAP1
  - TLE1
  - DPPA5
  - MYBL2
  - GCM1
  - TLE2
  - TCF3
  - TCF7L1
  - FOXD3
  - THY1
  - LEFTY1
  - EOMES
  - LATS1
  - LATS2
  - KRT18
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - LEFTY2
  - PTGS2
  - MTCO2P12
  - ACTR3
  - APOBEC3A
  - PSG2
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - CDX2
  - SNAI1
  - CER1
  - C3CER1
  - GATA4
  - DSC2
  - DSC3
  - FGF3
  - HNF4A
  - DAB2
  - SOX17
  - fzr
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - lds
  - aPKC
  - mas
  - myoblast-specific
  - Bin1
  - dpp
  - gbb
  - put
  - mav
  - Nle
  - Mad
  - Dif
  - dl
  - Rel
  - .na.character
  - hop
  - bsk
  - pnr
  - grn
  - RpS14a
  - RpS14b
  - CtBP
  - yki
  - gcm
  - hpo
  - hp
  - wts
  - Wnt5
  - Wnt4
  - wg
  - Wnt2
  - Arp3
  - l(3)psg2
  - Pka-C1
  - Pka-R2
  - Pka-C3
  - Pka-R1
  - Pka-C2
  - Sc2
  - Hnf4
  - dnmt1
  - ctnnb1
  - cdh1
  - si:busm1-180o5.1
  - si:busm1-180o5.2
  - gjb3
  - pou5f3
  - gap43
  - myca
  - mycbp2
  - pard3ab
  - bmp4
  - id2a
  - sap18
  - tdgf1
  - dpp3
  - smad1
  - upf1
  - lamb3
  - fgf4
  - fgf6a
  - m17
  - esrrb
  - stat3
  - gata2a
  - sox2
  - nanog
  - klf4
  - klf17
  - rps14
  - tle2a
  - tcf3a
  - tcf7l1a
  - foxd3
  - thy1
  - eomesa
  - lats1
  - krt18a.1
  - cdx1a
  - snai1a
  - gata4
  - fgf3
  - dab2
  - sox17
  - wnt9a
  - Retinol
  - Cancer
---
