---
figid: PMC3961715__nihms-537840-f0004
figtitle: Unrooted maximum-likelihood tree showing the relationships within a set
  of selected MoeH5 homologues found in primary and secondary metabolic pathways
organisms:
- Escherichia coli
- Staphylococcus aureus
- Pseudomonas aeruginosa
- Escherichia coli BW25113
- Streptomyces lividans
- Proteus mirabilis
- Streptomyces albus
- Vibrio vulnificus
- Streptomyces clavuligerus
- Shigella dysenteriae
- Streptomyces albus J1074
- Streptomyces lividans TK24
- Staphylococcus aureus Chi-4
- Providencia stuartii
- Streptomyces noursei ATCC 11455
- Streptomyces viridosporus ATCC 14672
- Streptomyces viridosporus
- Streptomyces prasinus
- Homo sapiens
- Gallus gallus
- Bombyx mori
- uncultured actinomycete
- NA
- NA
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC3961715
filename: nihms-537840-f0004.jpg
figlink: /pmc/articles/PMC3961715/figure/F4/
number: F4
caption: Unrooted maximum-likelihood tree showing the relationships within a set of
  selected MoeH5 homologues found in primary and secondary metabolic pathways. Complete
  amino acid (aa) sequences of the proteins were used for analysis. Double black triangles
  mark the proteins having truncated Ntn domains (see ), while single triangles mark
  those with complete Ntn domains that lack the invariable Cys1 residue. The tree
  was built with the help of phylogeny.fr v.2 server (Dereeper et al., 2008) using
  default parameters. Confidence indices aLRT () are shown on the nodes; the branches
  having aLRT values below stringent threshold level (0.75) were collapsed. The scale
  bar represents 2.0 aa substitution per aa position. Source data are given in .
papertitle: 'MoeH5: a natural glycorandomizer from the moenomycin biosynthetic pathway.'
reftext: Bohdan Ostash, et al. Mol Microbiol. ;90(6):1324-1338.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6310075
figid_alias: PMC3961715__F4
figtype: Figure
redirect_from: /figures/PMC3961715__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3961715__nihms-537840-f0004.html
  '@type': Dataset
  description: Unrooted maximum-likelihood tree showing the relationships within a
    set of selected MoeH5 homologues found in primary and secondary metabolic pathways.
    Complete amino acid (aa) sequences of the proteins were used for analysis. Double
    black triangles mark the proteins having truncated Ntn domains (see ), while single
    triangles mark those with complete Ntn domains that lack the invariable Cys1 residue.
    The tree was built with the help of phylogeny.fr v.2 server (Dereeper et al.,
    2008) using default parameters. Confidence indices aLRT () are shown on the nodes;
    the branches having aLRT values below stringent threshold level (0.75) were collapsed.
    The scale bar represents 2.0 aa substitution per aa position. Source data are
    given in .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TPI1
  - ARHGEF5
  - TIMELESS
  - HAVCR1
  - LIPA
  - CEL
  - SCGB1D1
  - SSPN
  - tbi
  - aa
  - tim
  - timeout
  - mag
  - Liprin-alpha
---
