---
figid: PMC3074020__nihms249503f2
figtitle: 'The 22q11.2 microdeletion: fifteen years of insights into the genetic and
  neural complexity of psychiatric disorders'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC3074020
filename: nihms249503f2.jpg
figlink: /pmc/articles/PMC3074020/figure/F2/
number: F2
caption: Both the Dgcr8 and mir-185 genes are located within the 1.5-Mb microdeletion
  region within chromosome 22q11.2 and the equivalent region of mouse chromosome 16.
  Hence, they are hemizygous in the microdeletion syndrome. This deficit results in
  a reduction in the levels of a subset of mature miRNAs (red, blue) in both PFC and
  HPC in Df(16)A+/− mice. mir-185 expression is particularly affected due to the combination
  of hemizygosity and reduced synthesis due to Dgcr8 hemizygosity, hence its down-regulation
  is stronger than other miRNAs (indicated by a thicker arrow). Partial reduction
  in miRNA levels can affect mRNA and protein levels of target genes. Moreover, since
  target mRNAs can be regulated by the combinational control of more than one miRNA,
  the level of repression achieved may be highly sensitive to the amount and types
  of available miRNA complexes. Therefore, miRNA dysregulation is a molecular pathway
  strongly affected in 22q11.2DS. For example, target genes may be upregulated due
  to reduction of miRNAs affected by the Dgcr8 deficiency (bottom left), due to reduction
  of miRNA genes removed by the 22q11.2 microdeletion (mir-185 within the 1.5-MB minimal
  region is shown as an example here, bottom right), or due to a combination of both
  (bottom middle). The alteration of target gene expression patterns could result
  in a number of phenotypic abnormalities controlled by these target genes.
papertitle: 'The 22q11.2 microdeletion: fifteen years of insights into the genetic
  and neural complexity of psychiatric disorders.'
reftext: Liam J. Drew, et al. Int J Dev Neurosci. ;29(3):259-281.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8425705
figid_alias: PMC3074020__F2
figtype: Figure
redirect_from: /figures/PMC3074020__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3074020__nihms249503f2.html
  '@type': Dataset
  description: Both the Dgcr8 and mir-185 genes are located within the 1.5-Mb microdeletion
    region within chromosome 22q11.2 and the equivalent region of mouse chromosome
    16. Hence, they are hemizygous in the microdeletion syndrome. This deficit results
    in a reduction in the levels of a subset of mature miRNAs (red, blue) in both
    PFC and HPC in Df(16)A+/− mice. mir-185 expression is particularly affected due
    to the combination of hemizygosity and reduced synthesis due to Dgcr8 hemizygosity,
    hence its down-regulation is stronger than other miRNAs (indicated by a thicker
    arrow). Partial reduction in miRNA levels can affect mRNA and protein levels of
    target genes. Moreover, since target mRNAs can be regulated by the combinational
    control of more than one miRNA, the level of repression achieved may be highly
    sensitive to the amount and types of available miRNA complexes. Therefore, miRNA
    dysregulation is a molecular pathway strongly affected in 22q11.2DS. For example,
    target genes may be upregulated due to reduction of miRNAs affected by the Dgcr8
    deficiency (bottom left), due to reduction of miRNA genes removed by the 22q11.2
    microdeletion (mir-185 within the 1.5-MB minimal region is shown as an example
    here, bottom right), or due to a combination of both (bottom middle). The alteration
    of target gene expression patterns could result in a number of phenotypic abnormalities
    controlled by these target genes.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - chr
  - Chro
  - RabX1
  - Dro
  - Drs
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Dcr-2
  - Dcr-1
---
