---
figid: PMC3787301__elife01102f004
figtitle: The physiological and metabolic landscape of Melainabacteria
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC3787301
filename: elife01102f004.jpg
figlink: /pmc/articles/PMC3787301/figure/fig4/
number: F4
caption: 'Metabolic predictions for Melainabacteria based on genes identified in .
  Genes in pathways detected in the genomes of the subsurface and at least one gut
  genome (white box), only in the subsurface genome (grey box), only in at least one
  gut genome (orange box), and genes missing from pathways in all genomes (red box).
  Glycolysis proceeds via the canonical Embden-Meyerhof-Parnas (EMP) pathway with
  the exception of fructose-6-phosphate 1-phosphotransferase (EC:2.7.1.90, gene 3).
  Names of pathways and fermentation end-products are bolded and ATP generated by
  substrate-level phosphorylation are noted. All Melainabacteria genomes sampled lack
  electron transport chain components (including cytochromes (Cyto), succinate dehydrogenase
  (sdh), flavins, quinones), terminal respiratory oxidases or reductases, and photosystem
  I or II (PS1, PS2). The genomes also lack a complete TCA cycle (absent enzymes noted
  by red boxes), with the TCA enzymes instead linked to the fermentation of amino
  acids and organic acids denoted (pathways, blue arrows). Ferredoxin (Fd, green text)
  is important for hydrogen (H2) production via hydrogenases (yellow background box).
  Proton translocation mechanisms (green background box) may be achieved by the activity
  of trimeric oxaloacetate (OAA) decarboxylase and sodium-hydrogen antiporter, pyrophosphate
  (PPi) hydrolysis with pyrophosphatases, 11 subunit NADH dehydrogenase, and an annotated
  NiFe hydrogenase (green enzyme). Annotations for the gene numbers are in . The complete
  metabolic comparison of the Melainabacteria can be accessed at http://ggkbase.berkeley.edu/genome_summaries/81-MEL-Metabolic-Overview-June2013.DOI:
  http://dx.doi.org/10.7554/eLife.01102.01010.7554/eLife.01102.011Figure 4—source
  data 1.Examination of enzymes (steps) in near-complete KEGG based modules shared
  among or unique to subsurface ACD20 and gut Melainabacteria genomes MEL.A1, MEL.B1,
  and MEL.B2.Analysis is based on the KEGG Module database ().DOI: http://dx.doi.org/10.7554/eLife.01102.01110.7554/eLife.01102.012Figure
  4—source data 2.Gene annotations corresponding to the numbers in .If the gene occurs
  in both the ACD20 and gut genomes, the reported annotation is based on ACD20.DOI:
  http://dx.doi.org/10.7554/eLife.01102.012'
papertitle: The human gut and groundwater harbor non-photosynthetic bacteria belonging
  to a new candidate phylum sibling to Cyanobacteria.
reftext: Sara C Di Rienzi, et al. eLife. 2013;2:e01102.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.901181
figid_alias: PMC3787301__F4
figtype: Figure
redirect_from: /figures/PMC3787301__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3787301__elife01102f004.html
  '@type': Dataset
  description: 'Metabolic predictions for Melainabacteria based on genes identified
    in . Genes in pathways detected in the genomes of the subsurface and at least
    one gut genome (white box), only in the subsurface genome (grey box), only in
    at least one gut genome (orange box), and genes missing from pathways in all genomes
    (red box). Glycolysis proceeds via the canonical Embden-Meyerhof-Parnas (EMP)
    pathway with the exception of fructose-6-phosphate 1-phosphotransferase (EC:2.7.1.90,
    gene 3). Names of pathways and fermentation end-products are bolded and ATP generated
    by substrate-level phosphorylation are noted. All Melainabacteria genomes sampled
    lack electron transport chain components (including cytochromes (Cyto), succinate
    dehydrogenase (sdh), flavins, quinones), terminal respiratory oxidases or reductases,
    and photosystem I or II (PS1, PS2). The genomes also lack a complete TCA cycle
    (absent enzymes noted by red boxes), with the TCA enzymes instead linked to the
    fermentation of amino acids and organic acids denoted (pathways, blue arrows).
    Ferredoxin (Fd, green text) is important for hydrogen (H2) production via hydrogenases
    (yellow background box). Proton translocation mechanisms (green background box)
    may be achieved by the activity of trimeric oxaloacetate (OAA) decarboxylase and
    sodium-hydrogen antiporter, pyrophosphate (PPi) hydrolysis with pyrophosphatases,
    11 subunit NADH dehydrogenase, and an annotated NiFe hydrogenase (green enzyme).
    Annotations for the gene numbers are in . The complete metabolic comparison of
    the Melainabacteria can be accessed at http://ggkbase.berkeley.edu/genome_summaries/81-MEL-Metabolic-Overview-June2013.DOI:
    http://dx.doi.org/10.7554/eLife.01102.01010.7554/eLife.01102.011Figure 4—source
    data 1.Examination of enzymes (steps) in near-complete KEGG based modules shared
    among or unique to subsurface ACD20 and gut Melainabacteria genomes MEL.A1, MEL.B1,
    and MEL.B2.Analysis is based on the KEGG Module database ().DOI: http://dx.doi.org/10.7554/eLife.01102.01110.7554/eLife.01102.012Figure
    4—source data 2.Gene annotations corresponding to the numbers in .If the gene
    occurs in both the ACD20 and gut genomes, the reported annotation is based on
    ACD20.DOI: http://dx.doi.org/10.7554/eLife.01102.012'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - emp
  - cup
  - t-cup
  - fd
  - anon-16Fb
  - ATPsynbeta
  - Atpalpha
  - adp
  - x2
  - ps
  - if
  - Fdh
  - sti
  - Coa
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - na
  - Fum1
  - mali
  - Vha14-1
  - MAEA
  - AIP
  - AURKAIP1
  - GLIS3
  - ATP8A2
  - WDTC1
  - SRC
  - PSEN2
  - TFF1
  - TAS2R64P
  - PDS5B
  - ADH5
  - ALDH1L1
  - CIT
  - DHDDS
  - SARDH
  - SDHB
  - SORD
  - SDS
  - MALL
  - DNAH8
  - Acetate
  - Ethanol
  - Butyrate
  - dicarboxylic acids
  - flavin
  - Formate
  - Fructose
  - Glucose
  - Hexose
  - Trehalose87 Maltose
  - Pentose phosphate
  - Pyruvate
  - Ribose
  - Sorbitol
  - Glu
  - Lactate
---
