---
figid: PMC3333776__gb-2011-12-10-r106-2
figtitle: Rhizosphere-induced genes in Rlv3841 metabolic pathways
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC3333776
filename: gb-2011-12-10-r106-2.jpg
figlink: /pmc/articles/PMC3333776/figure/F2/
number: F2
caption: 'Rhizosphere-induced genes in Rlv3841 metabolic pathways. Bold lines show
  reactions encoded by genes induced (by three-fold or more, P ≤ 0.05) and are color
  coded for the rhizospheres: black, all; green, pea-specific; red, alfalfa-specific;
  blue, legume-specific; purple, alfalfa and sugar beet. An induced gene is also considered
  to be elevated in further rhizospheres where expression was elevated by two-fold
  or more (Additional file ). Where more than one enzyme carries out a reaction the
  color reflects the gene elevated in most rhizospheres. Asterisks indicate compounds
  imported by a transport system whose expression was elevated (Figure 1). aGenes
  are part of an induced operon and elevated ≥ 1.3-fold; bgenes induced ≥ 3-fold,
  P ≤ 0.11; cgenes elevated ≥ 2.2-fold, P ≤ 0.05. Dotted lines and boxed genes show
  those down-regulated (≤ 0.3-fold, P ≤ 0.05) relative to glucose-grown cells. Expression
  is considered to be down-regulated in all rhizospheres if ≤ 0.5-fold. Reactions
  carried out by genes slightly down-regulated (0.4- to 0.8-fold; Additional file
  ) are shown by a thin grey line. The dichloroethane pathway shown in italics is
  speculative. Genes underlined have been mutated and results of their competitiveness
  in the rhizosphere are shown in Additional file . DHAP, dihydroxyacetone phosphate;
  ED, Entner-Doudoroff; TCA, tricarboxylic acid.'
papertitle: Adaptation of Rhizobium leguminosarum to pea, alfalfa and sugar beet rhizospheres
  investigated by comparative transcriptomics.
reftext: Vinoy K Ramachandran, et al. Genome Biol. 2011;12(10):R106-R106.
year: '2011'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.950057
figid_alias: PMC3333776__F2
figtype: Figure
redirect_from: /figures/PMC3333776__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3333776__gb-2011-12-10-r106-2.html
  '@type': Dataset
  description: 'Rhizosphere-induced genes in Rlv3841 metabolic pathways. Bold lines
    show reactions encoded by genes induced (by three-fold or more, P ≤ 0.05) and
    are color coded for the rhizospheres: black, all; green, pea-specific; red, alfalfa-specific;
    blue, legume-specific; purple, alfalfa and sugar beet. An induced gene is also
    considered to be elevated in further rhizospheres where expression was elevated
    by two-fold or more (Additional file ). Where more than one enzyme carries out
    a reaction the color reflects the gene elevated in most rhizospheres. Asterisks
    indicate compounds imported by a transport system whose expression was elevated
    (Figure 1). aGenes are part of an induced operon and elevated ≥ 1.3-fold; bgenes
    induced ≥ 3-fold, P ≤ 0.11; cgenes elevated ≥ 2.2-fold, P ≤ 0.05. Dotted lines
    and boxed genes show those down-regulated (≤ 0.3-fold, P ≤ 0.05) relative to glucose-grown
    cells. Expression is considered to be down-regulated in all rhizospheres if ≤
    0.5-fold. Reactions carried out by genes slightly down-regulated (0.4- to 0.8-fold;
    Additional file ) are shown by a thin grey line. The dichloroethane pathway shown
    in italics is speculative. Genes underlined have been mutated and results of their
    competitiveness in the rhizosphere are shown in Additional file . DHAP, dihydroxyacetone
    phosphate; ED, Entner-Doudoroff; TCA, tricarboxylic acid.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - tpiA
  - pgi
  - pgm
  - araA
  - dapE
  - araB
  - eda
  - ecpA
  - pck
  - matA
  - gltA
  - acnA
  - glcB
  - fumC
  - icd
  - glcF
  - sucA
  - sdhB
  - aldA
  - alkA
  - 4-hydroxyphenylpyruvate
  - Arabinan
  - D-gluconate-6P
  - APRL1206115
  - 4-hydroxybenzoate
  - Molybdate
  - B-Dglucose-6P L-ribulose-5
  - L-ribulose
  - 4-fumarylacetoacetate
  - Acetoacetate
  - Alanine
  - Arabinose
  - ethane
  - Formate
  - Fructose
  - Fumarate
  - fumarate
  - Glyoxylate
  - Isocitrate
  - Oxoglutarate
  - Lysine
  - Malonate
  - Mannitol
  - Oxaloacetate
  - Phenylalanine
  - phosphate
  - Proline
  - Pyruvate
  - pyruvate
  - Succinate
  - Tartrate
  - Tyrosine
  - Chloroacetate
---
