---
figid: PMC4256277__pgen.1004828.g002
figtitle: Epigenome-Guided Analysis of the Transcriptome of Plaque Macrophages during
  Atherosclerosis Regression Reveals Activation of the Wnt Signaling Pathway
organisms:
- Saccharomyces cerevisiae
- Candida dubliniensis
- Sequoia sempervirens
- Bos taurus
- Oryctolagus cuniculus
- Homo sapiens
- Mus musculus
- Venustaconcha ellipsiformis
- Mycobacterium tuberculosis SP5
organisms_ner:
- Mus musculus
- Bos taurus
- Homo sapiens
- Oryctolagus cuniculus
- Danio rerio
pmcid: PMC4256277
filename: pgen.1004828.g002.jpg
figlink: /pmc/articles/PMC4256277/figure/pgen-1004828-g002/
number: F2
caption: Yellow rectangles represent gene annotation categories. Ellipses represent
  genes that are differentially expressed between Mttp-inactivated and vehicle-treated
  sample groups in plaque macrophages in the Reversa mouse model of plaque regression
  (red = up in Mttp-inactivated, blue = down). Genes are labeled by official gene
  symbol. Color saturation indicates the magnitude of the fold-change (see ). Edges
  connect genes to the statistically enriched annotation categories with which they
  are associated (see ; enrichment P values are given in ). (A) A statistically significant
  cluster of upregulated genes is associated (by gene annotations) with the cytoskeleton.
  (B) A statistically significant cluster of downregulated genes is associated with
  the mitochondrion. (C) Human monocyte expression QTLs for genes that are differentially
  expressed in plaque CD68+ cells in regression are enriched for SNPs associated with
  risk of atherosclerosis or CAD, versus the number of atherosclerosis/CAD SNPs for
  monocyte eQTLs overall. The bars marked “expected” show the range of numbers of
  SNPs associated with risk of atherosclerosis or CAD that would be expected to occur
  by chance within the eQTLs of the set of 213 lipid lowering-responsive genes, based
  on the overall percentage of SNPs that are associated with risk of atherosclerosis/CAD
  (“random”) and based on the percentage of SNPs within all monocyte eQTLs that are
  associated with atherosclerosis/CAD (“monocytes”). Error bars indicate >90% confidence
  intervals for the number of atherosclerosis/CAD SNPs that would be expected by chance
  (see ). (*) P<0.05, Fisher's Exact Test (contingency table data are in ).
papertitle: Epigenome-Guided Analysis of the Transcriptome of Plaque Macrophages during
  Atherosclerosis Regression Reveals Activation of the Wnt Signaling Pathway.
reftext: Stephen A. Ramsey, et al. PLoS Genet. 2014 Dec;10(12):e1004828.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8648453
figid_alias: PMC4256277__F2
figtype: Figure
redirect_from: /figures/PMC4256277__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4256277__pgen.1004828.g002.html
  '@type': Dataset
  description: Yellow rectangles represent gene annotation categories. Ellipses represent
    genes that are differentially expressed between Mttp-inactivated and vehicle-treated
    sample groups in plaque macrophages in the Reversa mouse model of plaque regression
    (red = up in Mttp-inactivated, blue = down). Genes are labeled by official gene
    symbol. Color saturation indicates the magnitude of the fold-change (see ). Edges
    connect genes to the statistically enriched annotation categories with which they
    are associated (see ; enrichment P values are given in ). (A) A statistically
    significant cluster of upregulated genes is associated (by gene annotations) with
    the cytoskeleton. (B) A statistically significant cluster of downregulated genes
    is associated with the mitochondrion. (C) Human monocyte expression QTLs for genes
    that are differentially expressed in plaque CD68+ cells in regression are enriched
    for SNPs associated with risk of atherosclerosis or CAD, versus the number of
    atherosclerosis/CAD SNPs for monocyte eQTLs overall. The bars marked “expected”
    show the range of numbers of SNPs associated with risk of atherosclerosis or CAD
    that would be expected to occur by chance within the eQTLs of the set of 213 lipid
    lowering-responsive genes, based on the overall percentage of SNPs that are associated
    with risk of atherosclerosis/CAD (“random”) and based on the percentage of SNPs
    within all monocyte eQTLs that are associated with atherosclerosis/CAD (“monocytes”).
    Error bars indicate >90% confidence intervals for the number of atherosclerosis/CAD
    SNPs that would be expected by chance (see ). (*) P<0.05, Fisher's Exact Test
    (contingency table data are in ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Scrn1
  - Src
  - Fhdc1
  - Trim2
  - Baiap2
  - Psrc1
  - Chchd7
  - Pxmp2
  - Myo7a
  - Abcc1
  - Marcksl1
  - Mrps30
  - Rab34
  - Tatdn3
  - Cytip
  - Cyth1
  - Vps18
  - Parvg
  - Nipa2
  - Rpl35a
  - Mmgt2
  - Adhfe1
  - Hibch
  - Tfam
  - Cpt2
  - Rab40c
  - Mtif2
  - Ergic1
  - Ehd1
  - Coq3
  - Mtor
  - Aspscr1
  - Tmem109
  - Stard7
  - Mterf3
  - Arhgap22
  - Dolpp1
  - Slc8b1
  - Arhgap36
  - Rap1gap
  - Rasa1
  - Rho
  - Rhod
  - Ea2
  - Qsox1
  - Hsd3b7
  - Papolb
  - Stard8
  - Gmip
  - Degs1
  - Cad
  - Dffb
  - Acod1
  - Cgct
  - SCRN1
  - SRC
  - FHDC1
  - TRIM2
  - BAIAP2
  - PSRC1
  - CHCHD7
  - PXMP2
  - MYO7A
  - MRPS30
  - RAB34
  - TATDN3
  - CYTIP
  - VPS18
  - PARVG
  - NIPA2
  - RPL35A
  - ADHFE1
  - HIBCH
  - TFAM
  - CPT2
  - RAB40C
  - MTIF2
  - ERGIC1
  - EHD1
  - COQ3
  - MTOR
  - ASPSCR1
  - TMEM109
  - STARD7
  - MTERF3
  - ARHGAP22
  - DOLPP1
  - SLC8B1
  - RHO
  - QSOX1
  - HSD3B7
  - PAPOLB
  - STARD8
  - GMIP
  - CAD
  - FGR
  - FYN
  - YES1
  - MRPS7
  - MRPS2
  - MRPS5
  - MRPS6
  - MRPS9
  - MRPS10
  - MRPS11
  - MRPS12
  - MRPS14
  - MRPS15
  - MRPS16
  - MRPS17
  - MRPS18A
  - MRPS18B
  - MRPS18C
  - MRPS21
  - MRPS22
  - MRPS23
  - MRPS24
  - MRPS25
  - MRPS26
  - MRPS27
  - MRPS28
  - DAP3
  - MRPS31
  - MRPS33
  - MRPS34
  - MRPS35
  - MRPS36
  - PLIN4
  - ARHGAP1
  - RASA1
  - RGS6
  - RHOD
  - RHOA
  - RHOB
  - RHOC
  - DEGS1
  - DFFB
  - ACOD1
  - src
  - fhdc1
  - trim2b
  - trim2a
  - chchd7
  - pxmp2
  - myo7aa
  - abcc4
  - mrps30
  - rab34a
  - tatdn3
  - cytip
  - vps18
  - parvg
  - nipa2
  - rpl35a
  - adhfe1
  - hibch
  - tfam
  - cpt2
  - rab40c
  - mtif2
  - ergic1
  - ehd1b
  - coq3
  - mtor
  - aspscr1
  - stard7
  - mterf3
  - arhgap22
  - dolpp1
  - sec11a
  - gap43
  - rho
  - rnd2
  - qsox1
  - hsd3b7
  - papola
  - stard8
  - gmip
  - degs1
  - cad
  - dffb
  - cald1b
---
