---
figid: PMC4844212__kepi-10-11-1091145-g006
figtitle: A model of demethylation pathway selection in TET-induced DNA demethylation
organisms:
- NA
pmcid: PMC4844212
filename: kepi-10-11-1091145-g006.jpg
figlink: /pmc/articles/PMC4844212/figure/f0006/
number: F6
caption: A model of demethylation pathway selection in TET-induced DNA demethylation.
  When TET proteins globally oxidize 5mC in genomic DNA (upper part), the BER-based
  active pathway is triggered but quickly becomes saturated mainly due to the lack
  of BER genes adaptive upregulation in response to DNA damage. Only a few 5fC and
  5caC sites are cleaved and various types of DNA damage (e.g., AP-sites, SSBs) are
  produced as the end products. By contrast, through DNA replication, almost all 5hmC,
  5fC, and 5caC sites are efficiently converted into unmodified cytosine at no risk
  of DNA damage. Thus, the DNA-replication dependent passive pathway is selected as
  the predominant way for TET-induced global DNA demethylation. However, when TET
  proteins only oxidize 5mC in specific genomic loci (lower part), the TDG-BER capacity
  is enough to quickly complete the whole active demethylation process at these specific
  loci before DNA replication starts. Thus, the BER-based active demethylation pathway
  efficiently underlies TET-induced locus-specific DNA demethylation.
papertitle: Minimal role of base excision repair in TET-induced global DNA demethylation
  in HEK293T cells.
reftext: Chunlei Jin, et al. Epigenetics. 2015 Nov;10(11):1006-1013.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9204477
figid_alias: PMC4844212__F6
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC4844212__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4844212__kepi-10-11-1091145-g006.html
  '@type': Dataset
  description: A model of demethylation pathway selection in TET-induced DNA demethylation.
    When TET proteins globally oxidize 5mC in genomic DNA (upper part), the BER-based
    active pathway is triggered but quickly becomes saturated mainly due to the lack
    of BER genes adaptive upregulation in response to DNA damage. Only a few 5fC and
    5caC sites are cleaved and various types of DNA damage (e.g., AP-sites, SSBs)
    are produced as the end products. By contrast, through DNA replication, almost
    all 5hmC, 5fC, and 5caC sites are efficiently converted into unmodified cytosine
    at no risk of DNA damage. Thus, the DNA-replication dependent passive pathway
    is selected as the predominant way for TET-induced global DNA demethylation. However,
    when TET proteins only oxidize 5mC in specific genomic loci (lower part), the
    TDG-BER capacity is enough to quickly complete the whole active demethylation
    process at these specific loci before DNA replication starts. Thus, the BER-based
    active demethylation pathway efficiently underlies TET-induced locus-specific
    DNA demethylation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tet
  - Tdg
  - Rrp1
  - ber
  - TDG
  - APEX1
  - ERVK-9
  - ERVK-11
  - ERVK-19
---
