---
figid: PMC4308928__13073_2015_134_Fig1_HTML
figtitle: Regulation of DNA methylation and demethylation
organisms:
- Mus musculus
- Rattus norvegicus
- Bacteriophage sp.
- Tequatrovirus T4
- Homo sapiens
- Drosophila melanogaster
organisms_ner:
- Drosophila melanogaster
pmcid: PMC4308928
filename: 13073_2015_134_Fig1_HTML.jpg
figlink: /pmc/articles/PMC4308928/figure/Fig1/
number: F1
caption: Regulation of DNA methylation and demethylation. DNA demethylation can occur
  spontaneously via the DNMT enzymes that methylated the nucleotide cytosine (5-methylcytosine,
  5-mC) originally. A passive replication-dependent mechanism of DNA methylation is
  also possible. Several active demethylation pathways have been postulated. TET family
  proteins catalyze the oxidation of 5-mC into 5-hydroxymethylcytosine (5-hmC) and
  can further oxidize 5-hmC to 5-formylcytosine (5-fC) and 5-carboxycytosine (5-caC).
  5-hmC recognition and transformation into 5-hydroxymethyluracyl (5-hmU) by activation-induced
  deaminase (AID) to facilitate repair by DNA glycosylase and the base-excision repair
  (BER) pathway is still controversial. These latter activities are also thought to
  process 5-fC and 5-caC into unmodified cytosine . The decarboxylases involved in
  this process are still to be identified. APOBEC, apolipoprotein B mRNA editing enzyme;
  DNMT, DNA methyltransferase; T, thymine; TDG, thymine DNA glycosylase; TET, ten-eleven
  translocation.
papertitle: TET proteins and the control of cytosine demethylation in cancer.
reftext: Laurianne Scourzic, et al. Genome Med. 2015;7(1):9.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9413357
figid_alias: PMC4308928__F1
figtype: Figure
redirect_from: /figures/PMC4308928__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4308928__13073_2015_134_Fig1_HTML.html
  '@type': Dataset
  description: Regulation of DNA methylation and demethylation. DNA demethylation
    can occur spontaneously via the DNMT enzymes that methylated the nucleotide cytosine
    (5-methylcytosine, 5-mC) originally. A passive replication-dependent mechanism
    of DNA methylation is also possible. Several active demethylation pathways have
    been postulated. TET family proteins catalyze the oxidation of 5-mC into 5-hydroxymethylcytosine
    (5-hmC) and can further oxidize 5-hmC to 5-formylcytosine (5-fC) and 5-carboxycytosine
    (5-caC). 5-hmC recognition and transformation into 5-hydroxymethyluracyl (5-hmU)
    by activation-induced deaminase (AID) to facilitate repair by DNA glycosylase
    and the base-excision repair (BER) pathway is still controversial. These latter
    activities are also thought to process 5-fC and 5-caC into unmodified cytosine
    . The decarboxylases involved in this process are still to be identified. APOBEC,
    apolipoprotein B mRNA editing enzyme; DNMT, DNA methyltransferase; T, thymine;
    TDG, thymine DNA glycosylase; TET, ten-eleven translocation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Mt2
  - Nmt
  - Tet
  - Tdg
  - ber
---
