---
figid: PMC4554190__jnci.j_djv129_f0003
figtitle: Illustration of data-enhanced Bayesian network inference
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC4554190
filename: jnci.j_djv129_f0003.jpg
figlink: /pmc/articles/PMC4554190/figure/F3/
number: F3
caption: Illustration of data-enhanced Bayesian network inference. A) A signaling
  cascade in the Kyoto Encyclopedia of Genes and Genomes (KEGG) prostate cancer pathway.
  Nodes in the signaling cascade indicate proteins or protein families. Arrow with
  a solid line indicates activation. +p indicates phosphrylation. Arrow with a dashed
  line indicates indirect effect. The androgen receptor (AR) regulation of prostate
  specific antigen (PSA) is through binding to DNA and transcriptional activation
  of KLK3, which encodes PSA. B) Data-enhanced Posterior network inferred by Bayesian
  graphical model (BGM) analysis. One measurement feature is selected to represent
  each of the proteins or protein families in the signaling cascade (Supplementary
  Materials, section “Additional Results,” available online). The network includes
  only edges with the highest posterior probabilities. The numbers besides the edges
  are the posterior probabilities of including the edges inferred by BGM analysis
  (Supplementary Materials, section “Bayesian Graphical Models,” available online).
  All the other potential interaction edges have very low posterior probabilities
  (≤ 0.1) and not presented. Gene expression (GE) or protein expression (PE) in the
  parenthesis of each feature indicates gene expression or protein expression of the
  genes, respectively, which are used for analysis. Some of the protein expressions
  measure the abundances of proteins phosphrylated at specific positions. Positive
  interactions are indicated by green color and negative interactions are indicated
  by red color. Ser338, Ser217, and Ser221 indicate phosphorylations on serine-338,
  -217, and -221, respectively. GE = gene expression; PE = protein expression; Thr202
  = phosphorylation on threonine-202l; Tyr204 = phosphorylation on tyrosine-204.
papertitle: 'Zodiac: A Comprehensive Depiction of Genetic Interactions in Cancer by
  Integrating TCGA Data.'
reftext: Yitan Zhu, et al. J Natl Cancer Inst. 2015 Aug;107(8):djv129.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7379515
figid_alias: PMC4554190__F3
figtype: Figure
redirect_from: /figures/PMC4554190__F3
ndex: 0fdbecf7-dea6-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4554190__jnci.j_djv129_f0003.html
  '@type': Dataset
  description: Illustration of data-enhanced Bayesian network inference. A) A signaling
    cascade in the Kyoto Encyclopedia of Genes and Genomes (KEGG) prostate cancer
    pathway. Nodes in the signaling cascade indicate proteins or protein families.
    Arrow with a solid line indicates activation. +p indicates phosphrylation. Arrow
    with a dashed line indicates indirect effect. The androgen receptor (AR) regulation
    of prostate specific antigen (PSA) is through binding to DNA and transcriptional
    activation of KLK3, which encodes PSA. B) Data-enhanced Posterior network inferred
    by Bayesian graphical model (BGM) analysis. One measurement feature is selected
    to represent each of the proteins or protein families in the signaling cascade
    (Supplementary Materials, section “Additional Results,” available online). The
    network includes only edges with the highest posterior probabilities. The numbers
    besides the edges are the posterior probabilities of including the edges inferred
    by BGM analysis (Supplementary Materials, section “Bayesian Graphical Models,”
    available online). All the other potential interaction edges have very low posterior
    probabilities (≤ 0.1) and not presented. Gene expression (GE) or protein expression
    (PE) in the parenthesis of each feature indicates gene expression or protein expression
    of the genes, respectively, which are used for analysis. Some of the protein expressions
    measure the abundances of proteins phosphrylated at specific positions. Positive
    interactions are indicated by green color and negative interactions are indicated
    by red color. Ser338, Ser217, and Ser221 indicate phosphorylations on serine-338,
    -217, and -221, respectively. GE = gene expression; PE = protein expression; Thr202
    = phosphorylation on threonine-202l; Tyr204 = phosphorylation on tyrosine-204.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Sos
  - pe
  - ras
  - Ras64B
  - Ras85D
  - Raf
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Dsor1
  - Mtk
  - Erk7
  - rl
  - Act57B
  - ar
  - Psa
  - bgm
  - SOS1
  - XYLT2
  - SOS2
  - NRAS
  - KRAS
  - HRAS
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - RNASE3
  - MAP2K1
  - MAP2K7
  - MAP2K2
  - MAPK1
  - EPHB2
  - MAPK3
  - AR
  - KLK3
  - KLKB1
  - PLAG1
  - PROS1
  - NPEPPS
  - PSAT1
  - ACSBG1
  - Thr
  - Cancer
  - Cardiomyopathy
  - Noonan syndrome
---
