---
figid: PMC4794722__srep23118-f2
figtitle: Kcne2 deletion causes early-onset nonalcoholic fatty liver disease via iron
  deficiency anemia
organisms:
- NA
pmcid: PMC4794722
filename: srep23118-f2.jpg
figlink: /pmc/articles/PMC4794722/figure/f2/
number: F2
caption: (A) The liver DEG interacting networks (after microarray transcriptome analysis
  of liver tissue from global Kcne2─/─ versus Kcne2+/+ mice, n = 8 mice per group)
  with the highest consistency score included beta-oxidation of fatty acids, glucose
  concentration and hepatic steatosis, with predicted hierarchical control by PPARGC1A
  (IPA software, Qiagen). (B) Real-time qPCR does not detect Kcne2 transcript expression
  in Kcne2+/+ mouse liver. Kcne2+/+ stomach tissue was used as a positive control,
  Kcne2─/─tissue as a negative control; n = 3–4 mice per group, each quantified in
  triplicate. (C) The first-ranked disease process/biological function gene set altered
  by Kcne2 deletion was Inborn error of lipid metabolism, identified by Ingenuity
  pathway analysis of hepatic transcriptome changes quantified by microarray analysis
  as in panel A (P = 5.1 × 10−7, n = 8 per genotype).
papertitle: Kcne2 deletion causes early-onset nonalcoholic fatty liver disease via
  iron deficiency anemia.
reftext: Soo Min Lee, et al. Sci Rep. 2016;6:23118.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.913395
figid_alias: PMC4794722__F2
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC4794722__F2
ndex: 42901d3a-deb1-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4794722__srep23118-f2.html
  '@type': Dataset
  description: (A) The liver DEG interacting networks (after microarray transcriptome
    analysis of liver tissue from global Kcne2─/─ versus Kcne2+/+ mice, n = 8 mice
    per group) with the highest consistency score included beta-oxidation of fatty
    acids, glucose concentration and hepatic steatosis, with predicted hierarchical
    control by PPARGC1A (IPA software, Qiagen). (B) Real-time qPCR does not detect
    Kcne2 transcript expression in Kcne2+/+ mouse liver. Kcne2+/+ stomach tissue was
    used as a positive control, Kcne2─/─tissue as a negative control; n = 3–4 mice
    per group, each quantified in triplicate. (C) The first-ranked disease process/biological
    function gene set altered by Kcne2 deletion was Inborn error of lipid metabolism,
    identified by Ingenuity pathway analysis of hepatic transcriptome changes quantified
    by microarray analysis as in panel A (P = 5.1 × 10−7, n = 8 per genotype).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PPARGC1A
  - LIPA
  - CEL
  - SCGB1D1
  - CYP4A11
  - ACADVL
  - INSIG2
  - HADHA
  - ACADM
  - PLIN5
  - PPARA
  - HADH
  - NR0B2
  - ESRRA
  - KCNE2
  - APOB
  - CPT2
  - ETFA
  - FUT3
  - ACADS
  - ETFB
  - ETFDH
  - PSAP
  - MTCH1
  - SFTPA1
  - SFTPA2
  - OXCT1
  - PEX26
  - fatty acid
  - D-glucose
---
