---
figid: PMC4985183__nihms-802379-f0004
figtitle: Phosphoproteome Integration Reveals Patient-Specific Networks in Prostate
  Cancer
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC4985183
filename: nihms-802379-f0004.jpg
figlink: /pmc/articles/PMC4985183/figure/F3/
number: F3
caption: Enriched cancer hallmarks generated by dataset integration using TieDIE after
  inclusion of the phosphoproteomic and gene expression data relative to gene expression
  data alone (A). Several cancer hallmarks were enriched after inclusion of the phosphoproteomic
  data including the cell cycle pathway (B, red nodes), DNA repair pathway (D, yellow
  nodes), AKT/mTOR/MAPK pathway (F, blue nodes), and the nuclear receptor pathway
  (H, green nodes). Detailed analysis of each of these pathways revealed several common
  and unique players with high connectivity. Assessment of a select number of kinases
  and phosphoproteins from each network confirmed their elevated phosphorylation state
  (C, E, G, I) including some with direct phosphorylation on their enzymatic active
  residue (C, E). This supports the activation state of the networks observed. Black
  arrow represents phosphoresidues that result in enzymatic activity of the given
  protein. These defined subnetworks only contain genes that fall within both the
  curated hallmark gene sets and the previously generated scaffold network, with colored
  nodes corresponding to genes that are members of a hallmark and exclusive to the
  integrated network solution containing the phosphoproteomic data; grey genes are
  other scaffold members in the surrounding region. T-test was performed to calculate
  significance. *=p<0.05, **=p<0.01, ***=p<0.001, ****=p<0.0001. See also .
papertitle: Phosphoproteome Integration Reveals Patient-Specific Networks in Prostate
  Cancer.
reftext: Justin M. Drake, et al. Cell. ;166(4):1041-1054.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.936072
figid_alias: PMC4985183__F3
figtype: Figure
redirect_from: /figures/PMC4985183__F3
ndex: ee5e2169-de8e-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4985183__nihms-802379-f0004.html
  '@type': Dataset
  description: Enriched cancer hallmarks generated by dataset integration using TieDIE
    after inclusion of the phosphoproteomic and gene expression data relative to gene
    expression data alone (A). Several cancer hallmarks were enriched after inclusion
    of the phosphoproteomic data including the cell cycle pathway (B, red nodes),
    DNA repair pathway (D, yellow nodes), AKT/mTOR/MAPK pathway (F, blue nodes), and
    the nuclear receptor pathway (H, green nodes). Detailed analysis of each of these
    pathways revealed several common and unique players with high connectivity. Assessment
    of a select number of kinases and phosphoproteins from each network confirmed
    their elevated phosphorylation state (C, E, G, I) including some with direct phosphorylation
    on their enzymatic active residue (C, E). This supports the activation state of
    the networks observed. Black arrow represents phosphoresidues that result in enzymatic
    activity of the given protein. These defined subnetworks only contain genes that
    fall within both the curated hallmark gene sets and the previously generated scaffold
    network, with colored nodes corresponding to genes that are members of a hallmark
    and exclusive to the integrated network solution containing the phosphoproteomic
    data; grey genes are other scaffold members in the surrounding region. T-test
    was performed to calculate significance. *=p<0.05, **=p<0.01, ***=p<0.001, ****=p<0.0001.
    See also .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - CTNNB1
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - MYC
  - DHH
  - IHH
  - SHH
  - KRAS
  - NRAS
  - E2F1
  - E2F2
  - E2F3
  - E2F4
  - E2F5
  - E2F6
  - E2F7
  - E2F8
  - TP53
  - TP63
  - TP73
  - EGFR
  - ATM
  - APC
  - PROC
  - DMD
  - TSC1
  - CCL26
  - AKT1
  - DNMT1
  - MAPK14
  - NIPBL
  - PRKDC
  - TK2
  - TK1
  - DAXX
  - PTK2
  - POLE
  - MYBL2
  - CBX3
  - TRIM33
  - CHAF1A
  - MDC1
  - EZH2
  - RB1
  - RBM45
  - IGF1R
  - NEK3
  - SNAPC4
  - PARP1
  - ZCCHC8
  - TRIM24
  - CHD4
  - XPC
  - WRN
  - ERCC2
  - XRCC5
  - AR
  - REL
  - DNMT3A
  - PML
  - TRIM28
  - SUPT16H
  - RAD21
  - SUV39H1
  - MCM7
  - SNW1
  - EIF2AK2
  - PROM2
  - PHB1
  - CREBBP
  - NUP214
  - PELP1
  - RPA1
  - POLR1A
  - MCM2
  - LATS1
  - KMT2A
  - CDK1
  - RPS27
  - RPS6KA4
  - RPS18
  - RPN1
  - PSMD2
  - PRKAA2
  - ARID4A
  - NCOA4
  - HSF1
  - NR5A1
  - CCND1
  - FKBP5
  - PTPN11
  - STK39
  - POU2F1
  - PIAS1
  - MCM6
  - PSMD8
  - RPS14
  - THRSP
  - Cholesterol
  - Fatty Acid
  - Cancer
---
