---
figid: PMC4331812__1471-2164-16-S3-S9-5
figtitle: Metabolic flux map of central carbon metabolism in cotyledons at early seed
  fill
organisms:
- NA
pmcid: PMC4331812
filename: 1471-2164-16-S3-S9-5.jpg
figlink: /pmc/articles/PMC4331812/figure/F5/
number: F5
caption: 'Metabolic flux map of central carbon metabolism in cotyledons at early seed
  fill. Glycine max, var. Evans plants were grown in a growth chamber. Seeds were
  harvested at 21 DAF and cultured in medium containing 13C-sucrose until 26 DAF.
  Widths of flux arrows are drawn in proportion to relative fluxes; arrow directions
  indicate net flux. Each flux is represented by the metabolic reaction, with a superscript
  indicating subcellular localization (if known): c, cytosol; p, plastid; and m, mitochondrion.
  Oval fill colors: intracellular metabolites (white); "sink" metabolites such as
  proteinogenic amino acids or polysaccharides (gray); metabolites absorbed from/discharged
  into the medium (blue). A. Metabolic flux map. Flux arrow and name colors: glycolysis
  and sucrose metabolism (dark red); pentose phosphate pathway (pale blue); TCA cycle
  (orange); pyruvate dehydrogenase (purple-gray); anaplerotic reactions (mauve); glyoxylate
  shunt (green yellow); glucose metabolism, GABA shunt and related transmembrane fluxes
  (brown); fluxes towards synthesis of biomass (gray); all other transmembrane fluxes
  (black). B. Integration of metabolic flux with transcriptomics data. Flux arrow
  and font colors: reactions that have transcript support for encoded proteins predicted
  to be in the same subcellular localization as the reaction (dark green); transcripts
  for encoded proteins of undetermined subcellular location (light green); no transcript
  identified from the soybean genome and/or and no transcript on the Affymetrix soybean
  genome chip, and/or multi-step reactions (grey).'
papertitle: A systems biology approach toward understanding seed composition in soybean.
reftext: Ling Li, et al. BMC Genomics. 2015;16(Suppl 3):S9-S9.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9392799
figid_alias: PMC4331812__F5
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC4331812__F5
ndex: d258c530-def8-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4331812__1471-2164-16-S3-S9-5.html
  '@type': Dataset
  description: 'Metabolic flux map of central carbon metabolism in cotyledons at early
    seed fill. Glycine max, var. Evans plants were grown in a growth chamber. Seeds
    were harvested at 21 DAF and cultured in medium containing 13C-sucrose until 26
    DAF. Widths of flux arrows are drawn in proportion to relative fluxes; arrow directions
    indicate net flux. Each flux is represented by the metabolic reaction, with a
    superscript indicating subcellular localization (if known): c, cytosol; p, plastid;
    and m, mitochondrion. Oval fill colors: intracellular metabolites (white); "sink"
    metabolites such as proteinogenic amino acids or polysaccharides (gray); metabolites
    absorbed from/discharged into the medium (blue). A. Metabolic flux map. Flux arrow
    and name colors: glycolysis and sucrose metabolism (dark red); pentose phosphate
    pathway (pale blue); TCA cycle (orange); pyruvate dehydrogenase (purple-gray);
    anaplerotic reactions (mauve); glyoxylate shunt (green yellow); glucose metabolism,
    GABA shunt and related transmembrane fluxes (brown); fluxes towards synthesis
    of biomass (gray); all other transmembrane fluxes (black). B. Integration of metabolic
    flux with transcriptomics data. Flux arrow and font colors: reactions that have
    transcript support for encoded proteins predicted to be in the same subcellular
    localization as the reaction (dark green); transcripts for encoded proteins of
    undetermined subcellular location (light green); no transcript identified from
    the soybean genome and/or and no transcript on the Affymetrix soybean genome chip,
    and/or multi-step reactions (grey).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - sta
  - G6P
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - Suc
  - pk
  - Nuc
  - Cys
  - mmy
  - Csp
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Eno
  - Pep
  - bgcn
  - PyK
  - pyr
  - Pdi
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - Me
  - Asx
  - ascutex
  - pyx
  - arg
  - aos
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - Osi23
  - Ms
  - mal
  - bs
  - Mrtf
  - sah
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - Gat
  - .na.character
  - Pitslre
  - gate
  - GABA-B-R1
  - Rdl
  - Gabat
  - Adh
  - ss
  - Hn
  - Blos1
  - mah
  - edl
  - Scn
  - sub
  - Gad1
  - IL32
  - EMD
  - GCY
  - TALDO1
  - LRSAM1
  - TAAR1
  - MSMB
  - PSPN
  - PSPH
  - REG1A
  - STXBP3
  - RIDA
  - BPIFA2
  - TKT
  - DDR2
  - PRF1
  - NUCB1
  - RASA1
  - RGS6
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - PAEP
  - PREP
  - PTPN22
  - PHKA2
  - P4HB
  - PADI1
  - PDIA2
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - ABL2
  - RERE
  - MDH2
  - HAO1
  - GLIS3
  - MS
  - GNE
  - MAL
  - MRTFA
  - TIRAP
  - PNMA1
  - GSTZ1
  - BIRC3
  - ACSM3
  - SARDH
  - SDHB
  - SORD
  - SDS
  - GLYAT
  - LINC02154
  - TRIM35
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - WAS
  - CALR
  - TRIM21
  - GBA1
  - GBA1LP
  - GBA3
  - PAH
  - BLOC1S1
  - PACRG
  - SLC6A1
  - SRI
  - SCN10A
  - SCN11A
  - SCN1A
  - SCN2A
  - SCN3A
  - SCN4A
  - SCN5A
  - SCN8A
  - SCN9A
  - SCN1B
  - SCN2B
  - SCN3B
  - SCN4B
  - GAD1
  - GAD2
  - Ala
  - Arg
  - Cys
  - fructose
  - GABA
  - glucose
  - His
  - Leu
  - Lys
  - Met
  - Phe
  - Pro
  - Ser
  - Thr
  - Trp
  - Tyr
  - Val
  - Glu
---
