---
figid: PMC4345372__zam9991160980006
figtitle: Proposed pathway for reutericyclin biosynthesis
organisms:
- Lactiplantibacillus plantarum
- Limosilactobacillus reuteri
- Streptococcus mutans
organisms_ner:
- Drosophila melanogaster
pmcid: PMC4345372
filename: zam9991160980006.jpg
figlink: /pmc/articles/PMC4345372/figure/F6/
number: F6
caption: Proposed pathway for reutericyclin biosynthesis. The A domain of RtcN selects
  for d-leucine (), but RtcN has no epimerase domain. Most Gram-positive bacteria
  are capable of synthesizing d-Ala and d-Glu as building blocks for peptidoglycan
  (), but the synthesis of other d-amino acids is less common. An isoleucine 2-epimerase
  with leucine epimerase activity has been characterized in lactobacilli (); a corresponding
  epimerase is conserved in all four strains of L. reuteri used in this study (WP_011953381.1),
  and strains of L. reuteri have been reported to produce d-leucine (). It is therefore
  highly likely that the isoleucine 2-epimerase homologues are responsible for d-leucine
  synthesis in L. reuteri. d-Amino acids can be directly activated by bacterial NRPS
  (). Once activated, the leucine aminoacyl adenylate is likely loaded onto the phosphopantetheinyl
  arm of the RtcN thiolation (T) domain. Then the C domain of RtcN may acylate the
  amino group of the NRPS-bound leucine with ACP- or coenzyme A-activated 2-decenoic
  acid. The reutericyclin genomic island does not code for enzymes related to lipid
  metabolism; the C10 fatty acid in reutericyclin thus likely originates from the
  general metabolism. Cross talk between polyketide synthases and fatty acid synthases
  has been reported in Escherichia coli (). In bacteria, the 2-decenoic acid component
  can be synthesized by β-hydroxydecanoyl-ACP dehydratase (FabA) or a more common
  enzyme, hydroxyacyl-ACP dehydratase (FabZ). Two fabZ homologues are present in the
  genomes of reutericyclin producers (L. reuteri TMW1.656 genome locus tags HQ33_05470
  and HQ33_05510). Sequence analysis of RtcK suggests the presence of KS, ACP, and
  TE domains. However, no acyltransferase (AT) domain is found. The amino acid sequence
  between the KS and ACP domains shows homology to the region N-terminal to AT domains
  found in other bacterial PKS. This is similar to what is found in the AT domain-less
  type I PKS, where the remnant of the AT domain sequence acts as a docking domain
  for a distinct AT enzyme (). We propose that the KS domain of RtcK catalyzes a decarboxylative
  Claisen condensation. Once the enzyme-bound malonate decarboxylates, the resulting
  enolate attacks the leucine thioester loaded on the T domain of RtcN. The TE domain
  of RtcK may then catalyze the cyclization and offloading of the reutericyclin precursor
  via the nucleophilic attack of the amide nitrogen on the thioester carbonyl. Finally,
  we propose that the resulting tetramic acid is acetylated by RtcABC, thus forming
  reutericyclin. RtcABC show sequence homology to PhlABC, proteins involved in the
  biosynthesis of 2,4-diacetylphloroglucinol (). Biosynthetic studies suggest that
  PhlABC are responsible for acetylating phloroglucinol, yielding 2-acetylphloroglucinol
  and 2,4-diacetylphloroglucinol (). This acetylation is highly analogous to the predicted
  role in reutericyclin biosynthesis.
papertitle: Genetic Determinants of Reutericyclin Biosynthesis in Lactobacillus reuteri.
reftext: Xiaoxi B. Lin, et al. Appl Environ Microbiol. 2015 Mar;81(6):2032-2041.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.906881
figid_alias: PMC4345372__F6
figtype: Figure
redirect_from: /figures/PMC4345372__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4345372__zam9991160980006.html
  '@type': Dataset
  description: Proposed pathway for reutericyclin biosynthesis. The A domain of RtcN
    selects for d-leucine (), but RtcN has no epimerase domain. Most Gram-positive
    bacteria are capable of synthesizing d-Ala and d-Glu as building blocks for peptidoglycan
    (), but the synthesis of other d-amino acids is less common. An isoleucine 2-epimerase
    with leucine epimerase activity has been characterized in lactobacilli (); a corresponding
    epimerase is conserved in all four strains of L. reuteri used in this study (WP_011953381.1),
    and strains of L. reuteri have been reported to produce d-leucine (). It is therefore
    highly likely that the isoleucine 2-epimerase homologues are responsible for d-leucine
    synthesis in L. reuteri. d-Amino acids can be directly activated by bacterial
    NRPS (). Once activated, the leucine aminoacyl adenylate is likely loaded onto
    the phosphopantetheinyl arm of the RtcN thiolation (T) domain. Then the C domain
    of RtcN may acylate the amino group of the NRPS-bound leucine with ACP- or coenzyme
    A-activated 2-decenoic acid. The reutericyclin genomic island does not code for
    enzymes related to lipid metabolism; the C10 fatty acid in reutericyclin thus
    likely originates from the general metabolism. Cross talk between polyketide synthases
    and fatty acid synthases has been reported in Escherichia coli (). In bacteria,
    the 2-decenoic acid component can be synthesized by β-hydroxydecanoyl-ACP dehydratase
    (FabA) or a more common enzyme, hydroxyacyl-ACP dehydratase (FabZ). Two fabZ homologues
    are present in the genomes of reutericyclin producers (L. reuteri TMW1.656 genome
    locus tags HQ33_05470 and HQ33_05510). Sequence analysis of RtcK suggests the
    presence of KS, ACP, and TE domains. However, no acyltransferase (AT) domain is
    found. The amino acid sequence between the KS and ACP domains shows homology to
    the region N-terminal to AT domains found in other bacterial PKS. This is similar
    to what is found in the AT domain-less type I PKS, where the remnant of the AT
    domain sequence acts as a docking domain for a distinct AT enzyme (). We propose
    that the KS domain of RtcK catalyzes a decarboxylative Claisen condensation. Once
    the enzyme-bound malonate decarboxylates, the resulting enolate attacks the leucine
    thioester loaded on the T domain of RtcN. The TE domain of RtcK may then catalyze
    the cyclization and offloading of the reutericyclin precursor via the nucleophilic
    attack of the amide nitrogen on the thioester carbonyl. Finally, we propose that
    the resulting tetramic acid is acetylated by RtcABC, thus forming reutericyclin.
    RtcABC show sequence homology to PhlABC, proteins involved in the biosynthesis
    of 2,4-diacetylphloroglucinol (). Biosynthetic studies suggest that PhlABC are
    responsible for acetylating phloroglucinol, yielding 2-acetylphloroglucinol and
    2,4-diacetylphloroglucinol (). This acetylation is highly analogous to the predicted
    role in reutericyclin biosynthesis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Sh
  - ATPsynbeta
  - Atpalpha
  - Acp26Ab
  - Acp26Aa
  - CG18284
  - Acp36DE
  - SP
  - Acp95EF
  - Acph-1
  - te
  - Reutericyclin
---
