---
figid: PMC4826392__733f02
figtitle: Pathways of DNA demethylation mediated by TET enzymes
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC4826392
filename: 733f02.jpg
figlink: /pmc/articles/PMC4826392/figure/RASMUSSENGAD276568F2/
number: F2
caption: Pathways of DNA demethylation mediated by TET enzymes. (A) Model of passive
  replication-dependent DNA demethylation. The diagrams illustrate the two opposite
  fates of a hemihydroxymethylated CpG dinucleotide through two rounds of DNA replication.
  The left panel shows the replication-dependent DNA demethylation, in which DNA methylation
  is lost on the DNA strand opposite to 5hmC. The right panel shows the reverse outcome,
  in which maintenance of DNA methylation is achieved by DNMT1 in complex with UHRF1/2
  or DNMT3A/B. The relative extent of these two opposing outcomes of 5hmC deposition
  in vivo remains to be determined and is likely influenced by global as well as locus-specific
  factors. In both cases, newly replicated DNA dilutes 5hmC during cell division.
  (B) Model of active DNA demethylation by a TET/TDG (thymine–DNA–glycosylase)/BER
  (base excision repair)-dependent pathway. A cytosine base can be methylated by the
  DNA methylation machinery (DNMT1 or DNMT3A/B) to form 5mC, which in turn can be
  iteratively oxidized by TET enzymes to produce 5hmC, 5fC, and 5caC. TDG then recognizes
  5fC and 5caC, and the oxidized cytosine base is excised. This yields an abasic site
  that is repaired by BER and results in restoration of the unmodified cytosine state.
  Additional pathways of active DNA demethylation have been suggested (for review,
  see ). (SAH) S-adenosyl-homocysteine; (SAM) S-adenosyl-methionine.
papertitle: Role of TET enzymes in DNA methylation, development, and cancer.
reftext: Kasper Dindler Rasmussen, et al. Genes Dev. 2016 Apr 1;30(7):733-750.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6960821
figid_alias: PMC4826392__F2
figtype: Figure
redirect_from: /figures/PMC4826392__F2
ndex: 0e098db0-df31-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4826392__733f02.html
  '@type': Dataset
  description: Pathways of DNA demethylation mediated by TET enzymes. (A) Model of
    passive replication-dependent DNA demethylation. The diagrams illustrate the two
    opposite fates of a hemihydroxymethylated CpG dinucleotide through two rounds
    of DNA replication. The left panel shows the replication-dependent DNA demethylation,
    in which DNA methylation is lost on the DNA strand opposite to 5hmC. The right
    panel shows the reverse outcome, in which maintenance of DNA methylation is achieved
    by DNMT1 in complex with UHRF1/2 or DNMT3A/B. The relative extent of these two
    opposing outcomes of 5hmC deposition in vivo remains to be determined and is likely
    influenced by global as well as locus-specific factors. In both cases, newly replicated
    DNA dilutes 5hmC during cell division. (B) Model of active DNA demethylation by
    a TET/TDG (thymine–DNA–glycosylase)/BER (base excision repair)-dependent pathway.
    A cytosine base can be methylated by the DNA methylation machinery (DNMT1 or DNMT3A/B)
    to form 5mC, which in turn can be iteratively oxidized by TET enzymes to produce
    5hmC, 5fC, and 5caC. TDG then recognizes 5fC and 5caC, and the oxidized cytosine
    base is excised. This yields an abasic site that is repaired by BER and results
    in restoration of the unmodified cytosine state. Additional pathways of active
    DNA demethylation have been suggested (for review, see ). (SAH) S-adenosyl-homocysteine;
    (SAM) S-adenosyl-methionine.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GC
  - TET1
  - TET3
  - TET2
  - UHRF1
  - DNMT1
  - DNMT3A
  - DNMT3B
  - DNMT3L
  - TDG
  - ACSM3
  - Cytosine
  - Succinate
---
