---
figid: PMC4320441__13104_2014_3461_Fig3_HTML
figtitle: 49 genes whose functional domains were changed by DAS exons
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC4320441
filename: 13104_2014_3461_Fig3_HTML.jpg
figlink: /pmc/articles/PMC4320441/figure/Fig3/
number: F3
caption: Pathway analysis of 49 genes whose functional domains were changed by DAS
  exons. The EGFR/mTOR signaling pathway was analyzed using 49 genes, where the functionalities
  of the changed domains were related to gene annotations. Thus, representative functions
  of these genes may be regulated by alternative splicing events. Seven of them were
  shown with larger fonts with an inner color. The lines indicate the intermolecular
  relationships. Medium fonts indicate differentially expressed genes suggested by
  a previous analysis []. Outer colors indicate the elevation or reduction of their
  expressions. Smaller fonts indicate closely related upstream or downstream proteins.
  Blue or red indicate successfully validated gain of function and elevation of expression
  or loss of function and reduction of expression, respectively. Success of the validation
  by semi-quantitative RT-PCR was determined by an Fc value > 1.5 between undifferentiated
  P19 cells (day 0) and the cells at the neuronal stage (day 7) (see also Figure ).
  Light blue or orange colors indicate unsuccessful validation, even though these
  genes were suggested by a previous microarray experiment. The violet inner color
  of Kitlg indicates that its domains were suggested to change but the gain or loss
  of function was difficult to determine. Tsc2 was not one of the extracted 49 genes,
  but was one of 262 DAS exon genes. Indeed, its alternative isoforms have no difference
  in their functional protein domains. Additionally, the transcript of p53 (TP53)
  was not listed as a differentially expressed gene; however, its protein level was
  likely to be upregulated, as described in a previous report [].
papertitle: Computational extraction of a neural molecular network through alternative
  splicing.
reftext: Shafiul Alam, et al. BMC Res Notes. 2014;7:934.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.937713
figid_alias: PMC4320441__F3
figtype: Figure
redirect_from: /figures/PMC4320441__F3
ndex: 0b4e6333-defd-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4320441__13104_2014_3461_Fig3_HTML.html
  '@type': Dataset
  description: Pathway analysis of 49 genes whose functional domains were changed
    by DAS exons. The EGFR/mTOR signaling pathway was analyzed using 49 genes, where
    the functionalities of the changed domains were related to gene annotations. Thus,
    representative functions of these genes may be regulated by alternative splicing
    events. Seven of them were shown with larger fonts with an inner color. The lines
    indicate the intermolecular relationships. Medium fonts indicate differentially
    expressed genes suggested by a previous analysis []. Outer colors indicate the
    elevation or reduction of their expressions. Smaller fonts indicate closely related
    upstream or downstream proteins. Blue or red indicate successfully validated gain
    of function and elevation of expression or loss of function and reduction of expression,
    respectively. Success of the validation by semi-quantitative RT-PCR was determined
    by an Fc value > 1.5 between undifferentiated P19 cells (day 0) and the cells
    at the neuronal stage (day 7) (see also Figure ). Light blue or orange colors
    indicate unsuccessful validation, even though these genes were suggested by a
    previous microarray experiment. The violet inner color of Kitlg indicates that
    its domains were suggested to change but the gain or loss of function was difficult
    to determine. Tsc2 was not one of the extracted 49 genes, but was one of 262 DAS
    exon genes. Indeed, its alternative isoforms have no difference in their functional
    protein domains. Additionally, the transcript of p53 (TP53) was not listed as
    a differentially expressed gene; however, its protein level was likely to be upregulated,
    as described in a previous report [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EP400
  - HBEGF
  - ARAP1
  - EGF
  - TGFA
  - ARHGEF12
  - EGFR
  - KITLG
  - SHC1
  - KIT
  - GRB2
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - CDKN2A
  - AKT1
  - TSC1
  - CCL26
  - MDM2
  - TSC2
  - TP53
  - MTOR
  - RHEB
  - RHEBP1
  - Cancer
---
