---
figid: PMC4948178__nihms799309f3
figtitle: The expanding world of small RNAs in plants
organisms:
- NA
organisms_ner:
- Arabidopsis thaliana
pmcid: PMC4948178
filename: nihms799309f3.jpg
figlink: /pmc/articles/PMC4948178/figure/F3/
number: F3
caption: a. Plant microRNAs (miRNAs) target transcripts for cleavage or translational
  repression, and also trigger the production of secondary small-interfering RNAs
  (siRNAs) from mRNAs, non-coding RNAs and transposable elements. The most accepted
  mechanism for the biogenesis of trans-acting siRNA (tasiRNA), phased siRNA (phasiRNA)
  and epigenetically activated siRNA (easiRNA) relies on two distinct pathways. One
  consists a two-hit system utilizing two 21-nucleotide (nt) miRNAs per transcript,
  and requires the activity of an RNA-inducing silencing complex comprising ARGONAUTE
  7 (AGO7). The second pathway consists of a one-hit system that usually involves
  a 22-nt miRNA loaded on AGO1, or 22-nt miRNA variants that are produced from monouridylation
  of 21-nt miRNAs (see ). Both pathways are routed towards RNA-DEPENDENT RNA POLYMERASE
  6 (RDR6)-mediated dsRNA synthesis aided by SUPPRESSOR OF GENE SILENCING 3 (SGS3),
  and processing of 21 and 22-nucleotide siRNAs by DICER-LIKE 4 (DCL4) and DCL2, respectively.
  RNA Polymerase II (Pol II)-derived transcripts might also produce miRNA-independent
  secondary siRNA via interactions with other RNA processing machineries such as the
  spliceosome, or during RNA decay,, but these pathways are not fully understood.
  b. An additional phasiRNA biogenesis pathway was found in monocot plants such as
  maize and rice, which involves the transcription of non-coding PHAS transcripts
  form intergenic loci. Two miRNAs (miR2118 and miR2275) were found involved in cleavage
  of PHAS transcripts by an unknown AGO protein. These cleavage products are converted
  into dsRNA by RDR6 and SGS3, and processed into 21- and 24-nucleotide phasiRNAs
  by DCL4 and DCL5, respectively (reviewed in ).
papertitle: The expanding world of small RNAs in plants.
reftext: Filipe Borges, et al. Nat Rev Mol Cell Biol. ;16(12):727-741.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9414576
figid_alias: PMC4948178__F3
figtype: Figure
redirect_from: /figures/PMC4948178__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4948178__nihms799309f3.html
  '@type': Dataset
  description: a. Plant microRNAs (miRNAs) target transcripts for cleavage or translational
    repression, and also trigger the production of secondary small-interfering RNAs
    (siRNAs) from mRNAs, non-coding RNAs and transposable elements. The most accepted
    mechanism for the biogenesis of trans-acting siRNA (tasiRNA), phased siRNA (phasiRNA)
    and epigenetically activated siRNA (easiRNA) relies on two distinct pathways.
    One consists a two-hit system utilizing two 21-nucleotide (nt) miRNAs per transcript,
    and requires the activity of an RNA-inducing silencing complex comprising ARGONAUTE
    7 (AGO7). The second pathway consists of a one-hit system that usually involves
    a 22-nt miRNA loaded on AGO1, or 22-nt miRNA variants that are produced from monouridylation
    of 21-nt miRNAs (see ). Both pathways are routed towards RNA-DEPENDENT RNA POLYMERASE
    6 (RDR6)-mediated dsRNA synthesis aided by SUPPRESSOR OF GENE SILENCING 3 (SGS3),
    and processing of 21 and 22-nucleotide siRNAs by DICER-LIKE 4 (DCL4) and DCL2,
    respectively. RNA Polymerase II (Pol II)-derived transcripts might also produce
    miRNA-independent secondary siRNA via interactions with other RNA processing machineries
    such as the spliceosome, or during RNA decay,, but these pathways are not fully
    understood. b. An additional phasiRNA biogenesis pathway was found in monocot
    plants such as maize and rice, which involves the transcription of non-coding
    PHAS transcripts form intergenic loci. Two miRNAs (miR2118 and miR2275) were found
    involved in cleavage of PHAS transcripts by an unknown AGO protein. These cleavage
    products are converted into dsRNA by RDR6 and SGS3, and processed into 21- and
    24-nucleotide phasiRNAs by DCL4 and DCL5, respectively (reviewed in ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AGO1
  - AGO7
  - RDR6
  - SGS3
  - DCL4
  - DCL2
---
