---
figid: PMC4820109__pone.0152834.g001
figtitle: 'Association of MicroRNA-196a2 Variant with Response to Short-Acting B2-Agonist
  in COPD: An Egyptian Pilot Study'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC4820109
filename: pone.0152834.g001.jpg
figlink: /pmc/articles/PMC4820109/figure/pone.0152834.g001/
number: F1
caption: Gene and microRNA sequence and structure were retrieved from miRBase database
  (http://microrna.sanger.ac.uk/targets/v3/). Multiple computational prediction tools
  were employed to identify miR-196a2 target genes (in 3'URT, 5'UTR, and CDS) as miRDB,
  miRNAMap, TargetScanHuman v6.2, miRTarBase v18 and DIANA-microT-CDS v5.0 databases.
  Result intersection, statistical validation, and filtration of the putative miRNA
  targets were applied to reduce the false positive prediction rate. The predicted
  miRNA target genes were analyzed for gene ontology (GO) terms and KEGG enrichment
  pathway analysis using DIANA-miRPath v2.0 web-server (http://diana.imis.athenainnovation.gr/DianaTools/index.php?r=mirpath/index)
  and miRTar Human tool (http://miRTar.mbc.nctu.edu.tw/). MiRNA-196a2-disease association
  was explored using a miRPub server (http://www.microrna.gr/mirpub/). Gene variations
  and frequencies in various populations were obtained from Ensembl (http://www.ensembl.org/)
  and miRdSNP databases (http://mirdsnp.ccr.buffalo.edu/). The Impact of the SNP on
  secondary structure was predicted based on the minimum free energy using miRNAMap
  2.0 and RNAfold server. Comparative functional analysis between predicted target
  gene sets in wild and mutant variants were performed using miRmut2Go. SNP identification
  in the study groups aimed to assess its association with disease risk, severity
  and pulmonary function.
papertitle: 'Association of MicroRNA-196a2 Variant with Response to Short-Acting Î²2-Agonist
  in COPD: An Egyptian Pilot Study.'
reftext: Manal S. Fawzy, et al. PLoS One. 2016;11(4):e0152834.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7290275
figid_alias: PMC4820109__F1
figtype: Figure
redirect_from: /figures/PMC4820109__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4820109__pone.0152834.g001.html
  '@type': Dataset
  description: Gene and microRNA sequence and structure were retrieved from miRBase
    database (http://microrna.sanger.ac.uk/targets/v3/). Multiple computational prediction
    tools were employed to identify miR-196a2 target genes (in 3'URT, 5'UTR, and CDS)
    as miRDB, miRNAMap, TargetScanHuman v6.2, miRTarBase v18 and DIANA-microT-CDS
    v5.0 databases. Result intersection, statistical validation, and filtration of
    the putative miRNA targets were applied to reduce the false positive prediction
    rate. The predicted miRNA target genes were analyzed for gene ontology (GO) terms
    and KEGG enrichment pathway analysis using DIANA-miRPath v2.0 web-server (http://diana.imis.athenainnovation.gr/DianaTools/index.php?r=mirpath/index)
    and miRTar Human tool (http://miRTar.mbc.nctu.edu.tw/). MiRNA-196a2-disease association
    was explored using a miRPub server (http://www.microrna.gr/mirpub/). Gene variations
    and frequencies in various populations were obtained from Ensembl (http://www.ensembl.org/)
    and miRdSNP databases (http://mirdsnp.ccr.buffalo.edu/). The Impact of the SNP
    on secondary structure was predicted based on the minimum free energy using miRNAMap
    2.0 and RNAfold server. Comparative functional analysis between predicted target
    gene sets in wild and mutant variants were performed using miRmut2Go. SNP identification
    in the study groups aimed to assess its association with disease risk, severity
    and pulmonary function.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ag
  - I-3
  - tRNA:Leu-AAG-1-4
  - tRNA:Leu-AAG-1-1
  - tRNA:Leu-AAG-1-3
  - tRNA:Leu-AAG-1-2
  - yip2
  - capt
  - tRNA:Thr-AGT-2-1
  - tRNA:Thr-AGT-1-7
  - tRNA:Thr-AGT-1-2
  - tRNA:Thr-AGT-1-3
  - tRNA:Thr-AGT-1-4
  - tRNA:Thr-AGT-1-6
  - tRNA:Thr-AGT-1-5
  - tRNA:Thr-AGT-1-1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Snp
  - Ts
  - SLC29A1
  - ACAA1
  - AGA
  - DDX18
  - GUCY2D
  - MIR196A2
  - IMPACT
---
