---
figid: PMC4887206__elife-13550-fig6
figtitle: Drosophila and vertebrate patched orthologs share a similar cis-regulatory
  architecture and gene regulatory strategy
organisms:
- NA
pmcid: PMC4887206
filename: elife-13550-fig6.jpg
figlink: /pmc/articles/PMC4887206/figure/fig6/
number: F6
caption: '(A) X-gal-stained E10.5 mouse embryos carrying novel Ptch1 enhancers as
  transient-transgenic lacZ reporters () are shown above a map of the mouse Ptch1
  region. Known Ptch1 enhancers (; ) are shown as cartoons. Red boxes along the gene
  locus schematic represent Gli1 occupancy in neural progenitors near Ptch1 (as identified
  by ChIP-seq; ). (B) ChIP-seq of Gli2 and Gli3 near Ptch1. Red arrowheads highlight
  tissue-specific Gli binding. (C) ChIP-seq shows H3K27me3 is enriched at the proximal
  promoter but reduced upon pathway stimulation, consistent with relief of Polycomb-mediated
  repression. (D) Left, summary of the ptc response to Hh in the Drosophila embryonic
  ectoderm. Multiple Hh-responsive enhancers combine Ci/Gli sites (red bands) and
  tissue-specific inputs (colored bands), generating overlapping subsets of the embryonic
  ptc response which are integrated and amplified by the promoter-proximal region,
  resulting in a complete segment-polarity stripe pattern (top). The promoter-proximal
  region also generates the basal signal-independent ptc expression (bottom, purple).
  Right, summary of the Hh response of ptc in the wing disc. A large battery of tissue-specific
  Gli-responsive enhancers, driving wing stripes of different widths, intensities,
  and dorso-ventral extents, interact with a Polycomb/Trithorax-bound promoter-proximal
  region to provide a complete wing stripe pattern (top). The PRE-containing promoter-proximal
  region also produces signal-independent ptc expression in the anterior (bottom,
  purple). Identification of flanking CTCF insulator sites for both fly and mouse
  patched were determined using ChIP-seq and ChIP-CHIP analyses available from the
  UCSC genome browser and modENCODE databases ().DOI: http://dx.doi.org/10.7554/eLife.13550.01810.7554/eLife.13550.019Figure
  6—source data 1.Mouse Ptch1 enhancer coordinates.Sequence coordinates, corresponding
  to the mm9 build of the Mus musculus genome, are provided for all enhancers tested
  as lacZ reporters in E10.5 mouse embryos in . Enhancers are named for their position
  relative to the Ptch1 promoter. References are provided for previously published
  enhancers.DOI: http://dx.doi.org/10.7554/eLife.13550.019'
papertitle: An ancient yet flexible cis-regulatory architecture allows localized Hedgehog
  tuning by patched/Ptch1.
reftext: David S Lorberbaum, et al. eLife. 2016;5:e13550.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6674185
figid_alias: PMC4887206__F6
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC4887206__F6
ndex: 75b4304b-dea3-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4887206__elife-13550-fig6.html
  '@type': Dataset
  description: '(A) X-gal-stained E10.5 mouse embryos carrying novel Ptch1 enhancers
    as transient-transgenic lacZ reporters () are shown above a map of the mouse Ptch1
    region. Known Ptch1 enhancers (; ) are shown as cartoons. Red boxes along the
    gene locus schematic represent Gli1 occupancy in neural progenitors near Ptch1
    (as identified by ChIP-seq; ). (B) ChIP-seq of Gli2 and Gli3 near Ptch1. Red arrowheads
    highlight tissue-specific Gli binding. (C) ChIP-seq shows H3K27me3 is enriched
    at the proximal promoter but reduced upon pathway stimulation, consistent with
    relief of Polycomb-mediated repression. (D) Left, summary of the ptc response
    to Hh in the Drosophila embryonic ectoderm. Multiple Hh-responsive enhancers combine
    Ci/Gli sites (red bands) and tissue-specific inputs (colored bands), generating
    overlapping subsets of the embryonic ptc response which are integrated and amplified
    by the promoter-proximal region, resulting in a complete segment-polarity stripe
    pattern (top). The promoter-proximal region also generates the basal signal-independent
    ptc expression (bottom, purple). Right, summary of the Hh response of ptc in the
    wing disc. A large battery of tissue-specific Gli-responsive enhancers, driving
    wing stripes of different widths, intensities, and dorso-ventral extents, interact
    with a Polycomb/Trithorax-bound promoter-proximal region to provide a complete
    wing stripe pattern (top). The PRE-containing promoter-proximal region also produces
    signal-independent ptc expression in the anterior (bottom, purple). Identification
    of flanking CTCF insulator sites for both fly and mouse patched were determined
    using ChIP-seq and ChIP-CHIP analyses available from the UCSC genome browser and
    modENCODE databases ().DOI: http://dx.doi.org/10.7554/eLife.13550.01810.7554/eLife.13550.019Figure
    6—source data 1.Mouse Ptch1 enhancer coordinates.Sequence coordinates, corresponding
    to the mm9 build of the Mus musculus genome, are provided for all enhancers tested
    as lacZ reporters in E10.5 mouse embryos in . Enhancers are named for their position
    relative to the Ptch1 promoter. References are provided for previously published
    enhancers.DOI: http://dx.doi.org/10.7554/eLife.13550.019'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Me
  - mb
  - E(spl)mbeta-HLH
  - Ne
  - .na.character
  - ha
  - CTCF
  - ps
  - cnk
  - ptc
  - Pat1
  - dl
  - pre
  - pal
  - Pal2
  - Pa1
  - Low
  - en
  - pe
  - ad
  - par
  - para
  - par-6
  - alphaTub84B
  - br
  - pho
  - pall
  - Pal1
  - sdt
  - MB
  - SLC14A1
  - FANCE
  - GLI2
  - GLI3
  - SAG
  - DMBT1
  - RNF7
  - F9
  - TAS2R38
  - PTCH1
  - RET
  - CCDC6
  - PATE2
  - APPBP2
  - SLC36A1
  - PTCH2
  - PAM
  - LRIT1
  - SHCBP1
  - PAGR1
  - SERPINE1
  - SERPINB2
  - NR1I2
  - JTB
  - NMRK2
  - GFRAL
---
