---
figid: PMC4973819__PMIC-15-3508-g003
figlink: /pmc/articles/PMC4973819/figure/pmic12095-fig-0003/
number: F3
caption: 'Reconstruction of the C1 metabolism at the organismal level in the microbial
  communities inhabiting petroleum‐polluted marine sediments based on the proteome
  analysis. Panels A, B and C, represent the active pathways in HAV, MES and PRI,
  respectively. The complete C1 metabolism that included the metabolic coupling of
  CH4 and CH3OH metabolism and the Wood‐Ljungdahl pathway is shown in Panel (D). As
  shown, only a small portion of the reactions within the entire C1 metabolism (D)
  was identified as being active in HAV, MES and PRI (A–C). The presence of enzymes
  for each transformation and the taxonomic affiliation of polypeptides are shown.
  Circles in A–C represent the relative number of proteins (referred to the total
  number proteins assigned to these pathways) in a sample assigned to each taxonomic
  group (the total number of proteins of each is indicated in the circles). Solid
  lines (HAV, blue; MES, red; PRI, green) in A–C display the syntrophy between different
  members as they participate in contiguous reactions, as described in panel (D).
  The relative number of proteins assigned to these syntrophic reactions is presented
  by the line thickness. In panels (A–C), the grey colour indicates transformations
  for which no proteins in the proteome were identified, whereas the black colour
  represents transformations for which proteins were found (putatively active reactions).
  Transformations in the CH4 and CH3OH metabolism for which no proteome evidence were
  found (e.g., the glutathione (GSH) pathway connecting methanol with CO2) are not
  indicated. Note: the representation of CO2 in a circle (bottom of each panel) represents
  the presence of a carbon dioxide concentrating carboxysome shell protein, a structural
  protein involved in CO2 accumulation within the cells, but with no catalytic function.'
pmcid: PMC4973819
papertitle: Metaproteomics and metabolomics analyses of chronically petroleum‐polluted
  sites reveal the importance of general anaerobic processes uncoupled with degradation.
reftext: Rafael Bargiela, et al. Proteomics. 2015 Oct;15(20):3508-3520.
pmc_ranked_result_index: '157103'
pathway_score: 0.6659684
filename: PMIC-15-3508-g003.jpg
figtitle: Reconstruction of the C1 metabolism at the organismal level in the microbial
  communities inhabiting petroleum‐polluted marine sediments based on the proteome
  analysis
year: '2015'
organisms:
- Thalassolituus oleivorans MIL-1
- unidentified marine bacterioplankton
- Desulforapulum autotrophicum HRM2
- Thalassolituus oleivorans
- Oleispira antarctica
- H4N4 subtype
- metagenome
- Pseudomonas putida
- Chlamydomonas reinhardtii
- activated sludge metagenome
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4973819__PMIC-15-3508-g003.html
  '@type': Dataset
  description: 'Reconstruction of the C1 metabolism at the organismal level in the
    microbial communities inhabiting petroleum‐polluted marine sediments based on
    the proteome analysis. Panels A, B and C, represent the active pathways in HAV,
    MES and PRI, respectively. The complete C1 metabolism that included the metabolic
    coupling of CH4 and CH3OH metabolism and the Wood‐Ljungdahl pathway is shown in
    Panel (D). As shown, only a small portion of the reactions within the entire C1
    metabolism (D) was identified as being active in HAV, MES and PRI (A–C). The presence
    of enzymes for each transformation and the taxonomic affiliation of polypeptides
    are shown. Circles in A–C represent the relative number of proteins (referred
    to the total number proteins assigned to these pathways) in a sample assigned
    to each taxonomic group (the total number of proteins of each is indicated in
    the circles). Solid lines (HAV, blue; MES, red; PRI, green) in A–C display the
    syntrophy between different members as they participate in contiguous reactions,
    as described in panel (D). The relative number of proteins assigned to these syntrophic
    reactions is presented by the line thickness. In panels (A–C), the grey colour
    indicates transformations for which no proteins in the proteome were identified,
    whereas the black colour represents transformations for which proteins were found
    (putatively active reactions). Transformations in the CH4 and CH3OH metabolism
    for which no proteome evidence were found (e.g., the glutathione (GSH) pathway
    connecting methanol with CO2) are not indicated. Note: the representation of CO2
    in a circle (bottom of each panel) represents the presence of a carbon dioxide
    concentrating carboxysome shell protein, a structural protein involved in CO2
    accumulation within the cells, but with no catalytic function.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - 5,10-Methylene THMPT
  - Formaldehyde THMPT
  - THMPT
  - Dimethylamine
  - Formaldehyde
  - Formaldehyde CH
  - OH Formaldehyde
  - Formate
genes: []
chemicals:
- word: 5,10-Methylene THMPT
  source: MESH
  identifier: C054095
- word: Formaldehyde THMPT
  source: MESH
  identifier: C054095
- word: THMPT
  source: MESH
  identifier: C054095
- word: Dimethylamine
  source: MESH
  identifier: D004123
- word: Formaldehyde
  source: MESH
  identifier: D005557
- word: Formaldehyde CH
  source: MESH
  identifier: D005557
- word: OH Formaldehyde
  source: MESH
  identifier: D005557
- word: Formate
  source: MESH
  identifier: D005561
diseases: []
figid_alias: PMC4973819__F3
redirect_from: /figures/PMC4973819__F3
figtype: Figure
---
