---
figid: PMC4008262__1471-2164-14-868-8
figtitle: Metabolic pathway map for the alloplasmic genotype TH237 (H
organisms:
- NA
organisms_ner:
- NA
pmcid: PMC4008262
filename: 1471-2164-14-868-8.jpg
figlink: /pmc/articles/PMC4008262/figure/F8/
number: F8
caption: 'Metabolic pathway map for the alloplasmic genotype TH237 (H. chilense cytoplasm)
  subjected to high light treatment. Each data point is the fold change of the alloplasmic
  genotype on the euplasmic control. Circles and rectangles denote measured metabolites
  and metabolic genes, respectively. Blue, red and grey points indicate significant
  (p ≤ 0.05) increases, decreases and no change of metabolite and enzyme levels, respectively.
  Metabolic genes measured are mapped and indicated with enzyme EC numbers: arginine
  decarboxylase (4.1.1.19), asparaginase (3.5.1.1), glutamine synthetase (6.3.1.3),
  aspartyl-phosphate (2.7.2.4), threonine synthase (4.2.3.1), sucrose synthase (2.4.1.13),
  phospho-glucomutase (5.4.2.2), and β-fructofuranosidase (3.2.1.26).'
papertitle: Cytoplasmic genome substitution in wheat affects the nuclear-cytoplasmic
  cross-talk leading to transcript and metabolite alterations.
reftext: Cristina Crosatti, et al. BMC Genomics. 2013;14:868-868.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9369748
figid_alias: PMC4008262__F8
figtype: Figure
redirect_from: /figures/PMC4008262__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4008262__1471-2164-14-868-8.html
  '@type': Dataset
  description: 'Metabolic pathway map for the alloplasmic genotype TH237 (H. chilense
    cytoplasm) subjected to high light treatment. Each data point is the fold change
    of the alloplasmic genotype on the euplasmic control. Circles and rectangles denote
    measured metabolites and metabolic genes, respectively. Blue, red and grey points
    indicate significant (p ≤ 0.05) increases, decreases and no change of metabolite
    and enzyme levels, respectively. Metabolic genes measured are mapped and indicated
    with enzyme EC numbers: arginine decarboxylase (4.1.1.19), asparaginase (3.5.1.1),
    glutamine synthetase (6.3.1.3), aspartyl-phosphate (2.7.2.4), threonine synthase
    (4.2.3.1), sucrose synthase (2.4.1.13), phospho-glucomutase (5.4.2.2), and β-fructofuranosidase
    (3.2.1.26).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Itaconate
  - Galactinol
  - Agmatine
  - Alanine
  - Arginine
  - Asparagine
  - Aspartate
  - Fructose
  - Fumarate
  - GABA
  - Galactose
  - Glucose
  - Glutamine
  - Glyceraldehyde
  - Glycine
  - O Isocitrate
  - Isoleucine
  - Lactose
  - Leucine
  - Malonate
  - O Mannitol
  - Oxaloacetate
  - Phenylalanine
  - Proline
  - Putrescine
  - Pyroglutamate
  - Pyruvate
  - Serine
  - Sorbitol
  - Succinate
  - Sucrose
  - Threonine
  - Tryptophan
  - Tyrosine
  - UDP
  - Valine
  - Glutamate
---
