---
figid: PMC4342492__zbc0131510360003
figtitle: Reconstruction of the metabolic pathways utilized for de novo syntheses
  of amino acids as derived from isotopologue profiling with [U-13C6]glucose and [1,2-13C2]glucose
organisms:
- NA
pmcid: PMC4342492
filename: zbc0131510360003.jpg
figlink: /pmc/articles/PMC4342492/figure/F3/
number: F3
caption: Reconstruction of the metabolic pathways utilized for de novo syntheses of
  amino acids as derived from isotopologue profiling with [U-13C6]glucose and [1,2-13C2]glucose.
  13C patterns of amino acid isotopologues are indicated by red and green lines, respectively,
  as deduced from labeling experiments performed with [U-13C6]glucose and [1,2-13C2]glucose.
  [1,2-13C2]Glucose-derived isotopologue patterns are shown for Ala, Asp, Thr, biosynthesis
  pathways, and the formation of Gly from Ser only. The three-letter abbreviation
  of de novo synthesized amino acids is highlighted in red, whereas that of unlabeled
  amino acids remains dark blue. Simplified metabolic reactions being part of the
  EMP pathway and PPP are highlighted by yellow and light blue shading, respectively.
  Note that in the case of [1,2-13C2]glucose catabolization via PPP, the formation
  of unlabeled GAP molecules is shown solely. Full metabolic fluxes refer to the Kyoto
  Encyclopedia of Genes and Genomes database, and arrows indicate the number of metabolic
  steps needed for biosynthesis of amino acids. Corresponding gene annotations are
  given in Table S9. Glu-6-P, glucose-6-phosphate; 6-P-glc-1,5-lac, 6-phospho-d-glucono-1,5-lactone;
  Ery-4-P, erythrose-4-phosphate; DHAP, dihydroxy-acetone-phosphate; PYR, pyruvate;
  α-KG, α-ketoglutarate.
papertitle: Characterization of the Pivotal Carbon Metabolism of Streptococcus suis
  Serotype 2 under ex Vivo and Chemically Defined in Vitro Conditions by Isotopologue
  Profiling.
reftext: Jörg Willenborg, et al. J Biol Chem. 2015 Feb 27;290(9):5840-5854.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9488435
figid_alias: PMC4342492__F3
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
redirect_from: /figures/PMC4342492__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4342492__zbc0131510360003.html
  '@type': Dataset
  description: Reconstruction of the metabolic pathways utilized for de novo syntheses
    of amino acids as derived from isotopologue profiling with [U-13C6]glucose and
    [1,2-13C2]glucose. 13C patterns of amino acid isotopologues are indicated by red
    and green lines, respectively, as deduced from labeling experiments performed
    with [U-13C6]glucose and [1,2-13C2]glucose. [1,2-13C2]Glucose-derived isotopologue
    patterns are shown for Ala, Asp, Thr, biosynthesis pathways, and the formation
    of Gly from Ser only. The three-letter abbreviation of de novo synthesized amino
    acids is highlighted in red, whereas that of unlabeled amino acids remains dark
    blue. Simplified metabolic reactions being part of the EMP pathway and PPP are
    highlighted by yellow and light blue shading, respectively. Note that in the case
    of [1,2-13C2]glucose catabolization via PPP, the formation of unlabeled GAP molecules
    is shown solely. Full metabolic fluxes refer to the Kyoto Encyclopedia of Genes
    and Genomes database, and arrows indicate the number of metabolic steps needed
    for biosynthesis of amino acids. Corresponding gene annotations are given in Table
    S9. Glu-6-P, glucose-6-phosphate; 6-P-glc-1,5-lac, 6-phospho-d-glucono-1,5-lactone;
    Ery-4-P, erythrose-4-phosphate; DHAP, dihydroxy-acetone-phosphate; PYR, pyruvate;
    α-KG, α-ketoglutarate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - zwf
  - aroC
  - glyA
  - ppc
  - alaT
  - aspC
  - 2-Oxoisovalerate
  - 2-Oxobutanoate
  - (S)-2-Acetolactate
  - Ala
  - Asp
  - glucose
  - Gly
  - Phe
  - Ser
  - Thr
  - Tyr
  - Glu
---
