---
figid: PMC4242313__mch071f1
figtitle: Post‐genomic Approach to Understanding Sphingolipid Metabolism in Arabidopsis
  thaliana
organisms:
- NA
organisms_ner:
- Saccharomyces cerevisiae S288C
- Arabidopsis thaliana
- Homo sapiens
pmcid: PMC4242313
filename: mch071f1.jpg
figlink: /pmc/articles/PMC4242313/figure/MCH071F1/
number: F1
caption: 'Fig. 1. A representation of the metabolic pathway displaying the enzymatic
  interconversions and structures of plant sphingolipids. Known yeast gene orthologues
  are listed in parentheses following abbreviated enzyme designations. Plant genes
  predicted to be involved in sphingolipid metabolism are not shown but are described
  in the text. Enzymatic steps modifying long‐chain bases and amide‐linked acyl chains
  via hydroxylation and/or desaturation are thought to utilize ceramide, glucosylceramide
  or inositolphosphorylceramide as substrate, but acyl‐chain elongation precedes ceramide
  formation, and the hydroxylation of free sphinganine has been demonstrated. Inositolphosphorylceramide
  is thought to be the precursor to complex glycophosphosphingolipids but the glycosyltransferases
  presumably required have not been identified. In addition to free sphinganine (d18:0)
  and 4‐hydroxysphinganine (t18:0, phytosphingosine), other long‐chain bases present
  in plants include cis and trans isomers of 8‐sphingenine (d18:1Δ8), 4,8‐sphingadienine
  (d18:2Δ4,8) and 4‐hydroxy‐8‐sphingenine (t18:1Δ8), shown here as constituents of
  ceramide, glucosylceramide and inositolphosphorylceramide, respectively. Sphingosine
  (d18:1Δ4trans, 4‐sphingenine) is shown as the phosphorylated derivative. The acyl
  chain amide‐linked in the ceramide shown is a nonhydroxy saturated C24 chain (lignocerate),
  whereas that of inositolphosphorylceramide and glucosylceramide is the α‐hydroxylated
  counterpart. Enzyme abbreviations: CERase, ceramidase; GCase, glucosylceramidase;
  GCS, glucosylceramide synthase; IPCS, inositolphophorylceramide synthase; KSR, 3‐ketosphinganine
  reductase; LCBK, long‐chain base kinase; LCBPL; long‐chain base‐phosphate lyase;
  LCBPP, long‐chain base‐phosphate phosphatase; PLC, phospholipase C; SAT, sphinganine
  acyltransferase; SH, sphinganine hydroxylase; SPT, serine palmitoyltransferase.'
papertitle: A Post‐genomic Approach to Understanding Sphingolipid Metabolism in Arabidopsis
  thaliana.
reftext: TERESA M. DUNN, et al. Ann Bot. 2004 May;93(5):483-497.
year: '2004'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9454881
figid_alias: PMC4242313__F1
figtype: Figure
redirect_from: /figures/PMC4242313__F1
ndex: c2441b5b-debd-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4242313__mch071f1.html
  '@type': Dataset
  description: 'Fig. 1. A representation of the metabolic pathway displaying the enzymatic
    interconversions and structures of plant sphingolipids. Known yeast gene orthologues
    are listed in parentheses following abbreviated enzyme designations. Plant genes
    predicted to be involved in sphingolipid metabolism are not shown but are described
    in the text. Enzymatic steps modifying long‐chain bases and amide‐linked acyl
    chains via hydroxylation and/or desaturation are thought to utilize ceramide,
    glucosylceramide or inositolphosphorylceramide as substrate, but acyl‐chain elongation
    precedes ceramide formation, and the hydroxylation of free sphinganine has been
    demonstrated. Inositolphosphorylceramide is thought to be the precursor to complex
    glycophosphosphingolipids but the glycosyltransferases presumably required have
    not been identified. In addition to free sphinganine (d18:0) and 4‐hydroxysphinganine
    (t18:0, phytosphingosine), other long‐chain bases present in plants include cis
    and trans isomers of 8‐sphingenine (d18:1Δ8), 4,8‐sphingadienine (d18:2Δ4,8) and
    4‐hydroxy‐8‐sphingenine (t18:1Δ8), shown here as constituents of ceramide, glucosylceramide
    and inositolphosphorylceramide, respectively. Sphingosine (d18:1Δ4trans, 4‐sphingenine)
    is shown as the phosphorylated derivative. The acyl chain amide‐linked in the
    ceramide shown is a nonhydroxy saturated C24 chain (lignocerate), whereas that
    of inositolphosphorylceramide and glucosylceramide is the α‐hydroxylated counterpart.
    Enzyme abbreviations: CERase, ceramidase; GCase, glucosylceramidase; GCS, glucosylceramide
    synthase; IPCS, inositolphophorylceramide synthase; KSR, 3‐ketosphinganine reductase;
    LCBK, long‐chain base kinase; LCBPL; long‐chain base‐phosphate lyase; LCBPP, long‐chain
    base‐phosphate phosphatase; PLC, phospholipase C; SAT, sphinganine acyltransferase;
    SH, sphinganine hydroxylase; SPT, serine palmitoyltransferase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LCB2
  - TSC3
  - TSC10
  - LCB4
  - LCB5
  - SUR2
  - LCB3
  - YSR3
  - TSC13
  - ELO2
  - ELO3
  - SCS7
  - SPT
  - CYP83B1
  - MNP
  - CER10
  - ABO1
  - AGXT
  - SPTLC2
  - KSR1
  - KSR2
  - MED23
  - ABCC9
  - SPTLC3
  - AN
  - SAT1
  - TFPI
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - GCLC
  - UGCG
  - BPIFA4P
  - FA2H
  - 3-ketosphinganine
  - ethanolamine phosphate
  - sphinganine
  - inositolphosphorylceramide
  - aldehyde
  - ceramide
  - glucosylceramide
  - serine
---
