---
figid: PMC4065383__nihms583137f2
figtitle: Origins and removal of ribonucleotides in DNA
organisms:
- NA
pmcid: PMC4065383
filename: nihms583137f2.jpg
figlink: /pmc/articles/PMC4065383/figure/F2/
number: F2
caption: Origins and removal of ribonucleotides in DNA. (A) Incorporation and removal
  of ribonucleotides incorporated by RNA primase during OF synthesis and maturation.
  Two of the pathways by which the RNA primer synthesized by Pol α-primase is removed
  are displayed. RNase H2 is able to digest and remove the RNA primer up until the
  terminal 3′ ribonucleotide that is removed by FEN1. Alternatively, Pol δ can perform
  strand displacement synthesis to generate a flap that is degraded by FEN1. (B) Proofreading
  efficiency is determined by the balance between excision and extension. The polymerase
  exonuclease activity can excise ribonucleotides from the 3′ terminus in primer-template
  DNA, an event that is more efficient on a mismatched 3′ terminal dNMP primer than
  it is on an rNMP. The kinetics of proofreading (both excision and extension) by
  a replicative DNA polymerase in the presence of a terminal rNMP remain to be biochemically
  determined. (C) The ribonucleotide excision repair (RER) pathway is initiated by
  incision 5′ to the ribonucleotide by RNase H2. Strand displacement synthesis by
  the polymerase is followed by nucleolytic flap processing and ligation. There are
  redundant functions of the polymerases (Pol δ and Polε) as well as the nucleases
  (FEN1 and Exo1; Adapted from Fig. 4 in []). (D) Loss of RNase H2 activity due to
  deletion of RNH201 or RNase H2-deficiency in AGS leaves unrepaired genomic ribonucleotides
  that may become targets for Top1-cleavage and removal. The resulting unligatable
  DNA ends possess a 2′–3′-cyclic phosphate and a 5′-hydroxyl that may promote the
  formation of DNA breaks and/or recombination and cause genome instability. (Adapted
  from Fig. 4 in []). (E) The pathway by which RNase H2 cleavage at ribonucleotides
  can generate abortive ligation intermediates (adenylated 5′-RNA–DNA junctions) that
  require APTXHNT3 for resolution. Failure of this pathway due to deletion of HNT3
  or APTX deficiency in AOA1 causes phenotypes that may reflect the persistence of
  adenylated RNA–DNA junctions (Adapted from Fig. 2E in [].
papertitle: 'Ribonucleotides in DNA: Origins, repair and consequences.'
reftext: Jessica S. Williams, et al. DNA Repair (Amst). ;19:27-37.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8059683
figid_alias: PMC4065383__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC4065383__F2
ndex: 059bf853-df1b-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4065383__nihms583137f2.html
  '@type': Dataset
  description: Origins and removal of ribonucleotides in DNA. (A) Incorporation and
    removal of ribonucleotides incorporated by RNA primase during OF synthesis and
    maturation. Two of the pathways by which the RNA primer synthesized by Pol α-primase
    is removed are displayed. RNase H2 is able to digest and remove the RNA primer
    up until the terminal 3′ ribonucleotide that is removed by FEN1. Alternatively,
    Pol δ can perform strand displacement synthesis to generate a flap that is degraded
    by FEN1. (B) Proofreading efficiency is determined by the balance between excision
    and extension. The polymerase exonuclease activity can excise ribonucleotides
    from the 3′ terminus in primer-template DNA, an event that is more efficient on
    a mismatched 3′ terminal dNMP primer than it is on an rNMP. The kinetics of proofreading
    (both excision and extension) by a replicative DNA polymerase in the presence
    of a terminal rNMP remain to be biochemically determined. (C) The ribonucleotide
    excision repair (RER) pathway is initiated by incision 5′ to the ribonucleotide
    by RNase H2. Strand displacement synthesis by the polymerase is followed by nucleolytic
    flap processing and ligation. There are redundant functions of the polymerases
    (Pol δ and Polε) as well as the nucleases (FEN1 and Exo1; Adapted from Fig. 4
    in []). (D) Loss of RNase H2 activity due to deletion of RNH201 or RNase H2-deficiency
    in AGS leaves unrepaired genomic ribonucleotides that may become targets for Top1-cleavage
    and removal. The resulting unligatable DNA ends possess a 2′–3′-cyclic phosphate
    and a 5′-hydroxyl that may promote the formation of DNA breaks and/or recombination
    and cause genome instability. (Adapted from Fig. 4 in []). (E) The pathway by
    which RNase H2 cleavage at ribonucleotides can generate abortive ligation intermediates
    (adenylated 5′-RNA–DNA junctions) that require APTXHNT3 for resolution. Failure
    of this pathway due to deletion of HNT3 or APTX deficiency in AOA1 causes phenotypes
    that may reflect the persistence of adenylated RNA–DNA junctions (Adapted from
    Fig. 2E in [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Fen1
  - Polr2A
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - Dg
  - AP-1gamma
  - slow
  - gt
  - ra
  - Rst(2)At
  - Top1
  - PCNA
  - Rf-c
  - CG8142
  - RfC4
  - Gnf1
  - RPA3
  - RPA2
  - RPA1
  - flap
  - Dna2
  - min
  - mnb
  - .na.character
  - inc
  - insc
  - InR
  - FEN1
  - ERVW-4
  - POLA1
  - FASTK
  - FOXD3-AS1
  - C5
  - JAG1
  - RNASEH1
  - RNASEH1P1
  - TOP1
  - RFC1
  - SLC19A1
  - RFC2
  - RFC3
  - RFC4
  - RFC5
  - APTX
  - ALOX5AP
  - EXO1
  - DNA2
  - GHRH
  - INS
---
