---
figid: PMC4597147__zjw0101551410004
figtitle: Regulated protein networks in GM-BMMs versus M-BMMs
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC4597147
filename: zjw0101551410004.jpg
figlink: /pmc/articles/PMC4597147/figure/F4/
number: F4
caption: Regulated protein networks in GM-BMMs versus M-BMMs. Protein abundance and
  phosphorylation data were analyzed with the Ingenuity Pathway Analysis (IPA) using
  145 differentially expressed proteins and phosphoproteins involved in differentially
  regulated biological process between GM-BMMs and M-BMMs. Glucose metabolism, lipid
  metabolism, amino acid metabolism, DNA replication, endocytosis, and Rho/GTPase
  signaling were the subnetworks most significantly affected. Proteins/nodes are grouped
  according to their function. Red color indicates GM-BMMs up-regulation and green
  is M-BMMs up-regulation. Circle means DEP, diamond means DPP. White circles are
  predicted interacting signaling molecules as well as transcription factors. All
  indicated by gene symbol.
papertitle: Proteomic Analysis Reveals Distinct Metabolic Differences Between Granulocyte-Macrophage
  Colony Stimulating Factor (GM-CSF) and Macrophage Colony Stimulating Factor (M-CSF)
  Grown Macrophages Derived from Murine Bone Marrow Cells .
reftext: Yi Rang Na, et al. Mol Cell Proteomics. 2015 Oct;14(10):2722-2732.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7353283
figid_alias: PMC4597147__F4
figtype: Figure
redirect_from: /figures/PMC4597147__F4
ndex: 5f16d8c5-df28-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4597147__zjw0101551410004.html
  '@type': Dataset
  description: Regulated protein networks in GM-BMMs versus M-BMMs. Protein abundance
    and phosphorylation data were analyzed with the Ingenuity Pathway Analysis (IPA)
    using 145 differentially expressed proteins and phosphoproteins involved in differentially
    regulated biological process between GM-BMMs and M-BMMs. Glucose metabolism, lipid
    metabolism, amino acid metabolism, DNA replication, endocytosis, and Rho/GTPase
    signaling were the subnetworks most significantly affected. Proteins/nodes are
    grouped according to their function. Red color indicates GM-BMMs up-regulation
    and green is M-BMMs up-regulation. Circle means DEP, diamond means DPP. White
    circles are predicted interacting signaling molecules as well as transcription
    factors. All indicated by gene symbol.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - ITGA1
  - ITGA10
  - ITGA11
  - ITGA2
  - ITGA2B
  - ITGA3
  - ITGA4
  - ITGA5
  - ITGA6
  - ITGA7
  - ITGA8
  - ITGA9
  - ITGAD
  - ITGAE
  - ITGAL
  - ITGAM
  - ITGAV
  - ITGAX
  - ITGB1
  - ITGB2
  - ITGB3
  - ITGB4
  - ITGB5
  - ITGB6
  - ITGB7
  - ITGB8
  - ITGBL1
  - CD19
  - CD36
  - CXCL2
  - CD74
  - SLC2A4
  - SLC7A2
  - NHERF1
  - PLCB2
  - CXCR2
  - LAT
  - ORC3
  - SPNS1
  - CD22
  - GHR
  - IFNB1
  - ZAP70
  - LCAT
  - HBB
  - KRT89P
  - APOE
  - TNF
  - ANXA4
  - MS4A1
  - GSTM5
  - CTNNB1
  - DLG4
  - LLGL1
  - ATP6V0D2
  - PAG1
  - STX7
  - CSN2
  - COPS2
  - EHD1
  - VAMP4
  - FYN
  - RAC1
  - RNASE1
  - BIRC5
  - PLCG1
  - PLCG2
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - NOS2
  - NANOS2
  - ITSN1
  - CAT
  - CRAT
  - GLYAT
  - TFAM
  - HMGCS1
  - HCK
  - CSK
  - ELMO1
  - DOK1
  - ART4
  - AACS
  - LYN
  - HSPB1
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - DOCK2
  - PRDX6
  - PRDX5
  - MVD
  - GPX4
  - PRKCH
  - STMN1
  - PML
  - CIDEC
  - MAPK8IP1
  - ACLY
  - GRB2
  - IDI1
  - CDC42
  - CUX1
  - USO1
  - LPL
  - LCP1
  - CIDEA
  - RHO
  - RHOD
  - RHOA
  - RHOB
  - RHOC
  - ASS1
  - NQO1
  - FNBP1L
  - FABP5
  - FABP5P1
  - FDPS
  - KEAP1
  - CUL3
  - PITPNC1
  - PTPN11
  - TSC22D1
  - FASN
  - GM2A
  - CYP2A13
  - SREBF2
  - LPIN1
  - NLK
  - GPI
  - NFE2L2
  - ALDOC
  - ENO1
  - FBXW7
  - CBL
  - PARP1
  - COL11A2
  - PARP2
  - PARP3
  - PARP4
  - PARP6
  - PARP8
  - PARP9
  - PARP10
  - PARP11
  - PARP12
  - PARP14
  - PARP15
  - PARP16
  - TNKS
  - TNKS2
  - TIPARP
  - PFKP
  - PYGL
  - LDHB
  - RPIA
  - UBC
  - RPS27A
  - ZFP36
  - RBBP7
  - NCOA6
  - ELAVL1
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - AEBP1
  - RBBP4
  - STAT3
  - HIF1A
  - ATM
  - TP53
  - TOP2A
  - DNMT1
  - CHD7
  - POLD3
  - MED13
  - NR2C1
  - NR1H3
  - HNF4A
  - HNF1A
  - FOXO1
  - RFC1
  - SLC19A1
  - RRM1
  - FOXA2
  - NR2C2
  - PRIM1
  - NR1H2
  - MYC
  - ATF4
  - PCNA
  - Glucose
---
