---
figid: PMC4496299__nihms672876f4
figtitle: Epigenomic profiling in visceral white adipose tissue of offspring of mice
  exposed to late gestational sleep fragmentation
organisms:
- Mus musculus
- Rattus norvegicus
- Oryctolagus cuniculus
- Diaporthe sclerotioides
- Equus caballus
- Eucalyptus intertexta
- Homo sapiens
pmcid: PMC4496299
filename: nihms672876f4.jpg
figlink: /pmc/articles/PMC4496299/figure/F4/
number: F4
caption: Gene network corresponding to genes associated to DMRs exhibiting higher
  DNA methylation in LG-SF (Panel A) and LG-SC (Panel B) groups. Networks were built
  using a bioinformatic tool (Ingenuity Pathway Analysis) based on databases relying
  on experimental evidence of gene functions. Molecules with a reported function in
  lipid metabolism and immune response are highlighted. Canonical pathways (CP) related
  to those functions are indicated with labels. Molecules in main nodes are colored
  in red and blue (for higher methylation in LG-SF and LG-SC, respectively) and indicated
  by a colored circle.
papertitle: Epigenomic profiling in visceral white adipose tissue of offspring of
  mice exposed to late gestational sleep fragmentation.
reftext: Rene Cortese, et al. Int J Obes (Lond). 2015 Jul;39(7):1135-1142.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6793982
figid_alias: PMC4496299__F4
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Equus caballus
- Oryctolagus cuniculus
redirect_from: /figures/PMC4496299__F4
ndex: 44887b7d-df34-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4496299__nihms672876f4.html
  '@type': Dataset
  description: Gene network corresponding to genes associated to DMRs exhibiting higher
    DNA methylation in LG-SF (Panel A) and LG-SC (Panel B) groups. Networks were built
    using a bioinformatic tool (Ingenuity Pathway Analysis) based on databases relying
    on experimental evidence of gene functions. Molecules with a reported function
    in lipid metabolism and immune response are highlighted. Canonical pathways (CP)
    related to those functions are indicated with labels. Molecules in main nodes
    are colored in red and blue (for higher methylation in LG-SF and LG-SC, respectively)
    and indicated by a colored circle.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - KCNN2
  - PAPSS2
  - SPHK2
  - MYL1
  - CP
  - IL12A
  - IL12B
  - TNS1
  - LGI1
  - RBP4
  - POLR2D
  - ATP2A3
  - FFAR4
  - CHRFAM7A
  - TNNI3
  - PHLPP1
  - BMP5
  - EIF2S1
  - MAP1A
  - METAP1
  - RBM38
  - IRS1
  - APOE
  - ARID1A
  - NEU3
  - NR1H2
  - NR1H3
  - RXRA
  - RXRB
  - RXRG
  - ACADS
  - DDT
  - DDTL
  - GSK3A
  - GSK3B
  - PIK3R1
  - PIK3R2
  - ARHGEF7
  - CBFA2T2
  - PPP1R13B
  - PPP1R12C
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - SHC1
  - SHC2
  - SHC3
  - SHC4
  - PCSK9
  - AKT1
  - AKT2
  - AKT3
  - UCP1
  - HSD11B1
  - ALB
  - APOA1
  - APOA2
  - CLU
  - ANGPTL4
  - IRF6
  - IL1B
  - NEDD8
  - KCNB1
  - PROM1
  - MAGI2
  - ACADL
  - PPARA
  - ACAT2
  - SOAT2
  - RNF40
  - SSTR1
  - RPE65
  - Papss2
  - Sphk2
  - Skn2
  - Kcnn2
  - Myl1
  - Cp
  - Tns1
  - Pycard
  - Lgi1
  - Rbp4
  - Polr2d
  - Atp2a3
  - Ffar4
  - Tnni3
  - Phlpp1
  - Bmp5
  - Eif2s1
  - Map1a
  - Rbm38
  - Irs1
  - Apoe
  - Arid1a
  - Neu3
  - Nr1h3
  - Nr1h2
  - Acads
  - Ddt
  - Gsk3b
  - Ecm1
  - Ppp1r13b
  - Pik3r1
  - Shc1
  - Pcsk9
  - Akt1
  - Ucp1
  - Angptl4
  - Tlr4
  - Il1
  - Nedd8
  - Kcnb1
  - Pik3r6
  - Prom1
  - Magi2
  - Acadl
  - Rxra
  - Ppara
  - Acat2
  - Soat2
  - Rnf40
  - Sstr1
  - Rpe65
  - Slk
  - Il12b
  - Pik3cg
  - ATP2A1
  - cholesterol
  - Diabetes mellitus
---
