---
figid: PMC4357382__pbio.1002109.g002
figtitle: 'Bistability in a Metabolic Network Underpins the De Novo Evolution of Colony
  Switching in Pseudomonas fluorescens '
organisms:
- NA
pmcid: PMC4357382
filename: pbio.1002109.g002.jpg
figlink: /pmc/articles/PMC4357382/figure/pbio.1002109.g002/
number: F2
caption: The pyrimidine biosynthetic pathway is a molecular link between the mutant
  carB genotype and the CAP capsule phenotype. The switch-causing carB mutation reduces
  concentrations of intermediates in the pyrimidine biosynthesis pathway (shown in
  black), exposing a decision point at which uridine triphosphate (UTP) is used either
  by PyrG for nucleotide biosynthesis (leading to the Cap- phenotype, components in
  red), or by GalU for polymer biosynthesis (generating Cap+ phenotype, components
  in blue). Metabolic pathways related to UTP biosynthesis are shown in grey. High-flux
  reactions recycling nucleotides are shown by thick arrows, and nucleotide biosynthetic
  pathways are shown by thin arrows. Figure based on amino acid homology with established
  Escherichia coli pathways [].
papertitle: Bistability in a Metabolic Network Underpins the De Novo Evolution of
  Colony Switching in Pseudomonas fluorescens .
reftext: Jenna Gallie, et al. PLoS Biol. 2015 Mar;13(3):e1002109.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9346371
figid_alias: PMC4357382__F2
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
redirect_from: /figures/PMC4357382__F2
ndex: 598e4660-df08-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4357382__pbio.1002109.g002.html
  '@type': Dataset
  description: The pyrimidine biosynthetic pathway is a molecular link between the
    mutant carB genotype and the CAP capsule phenotype. The switch-causing carB mutation
    reduces concentrations of intermediates in the pyrimidine biosynthesis pathway
    (shown in black), exposing a decision point at which uridine triphosphate (UTP)
    is used either by PyrG for nucleotide biosynthesis (leading to the Cap- phenotype,
    components in red), or by GalU for polymer biosynthesis (generating Cap+ phenotype,
    components in blue). Metabolic pathways related to UTP biosynthesis are shown
    in grey. High-flux reactions recycling nucleotides are shown by thick arrows,
    and nucleotide biosynthetic pathways are shown by thin arrows. Figure based on
    amino acid homology with established Escherichia coli pathways [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - rna
  - cmk
  - ndk
  - pyrG
  - dcd
  - pyrH
  - lptD
  - udp
  - galU
  - crp
  - carA
  - purU
  - purT
  - purN
  - Arginine
  - Arg
  - CDP
  - CMP
  - Guanine
  - Met
  - Ser
  - Uracil
  - UDP
  - UTP
---
