---
figid: PMC4859290__JAH3-5-e003170-g007
figtitle: Analysis of signaling pathways specific to base:apex comparison
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC4859290
filename: JAH3-5-e003170-g007.jpg
figlink: /pmc/articles/PMC4859290/figure/jah31433-fig-0007/
number: F7
caption: 'Analysis of signaling pathways specific to base:apex comparison. A, Venn
  diagram comparing overlap of genes identified in the top 3 canonical pathways: eIF2
  signaling, mTOR signaling, and regulation of eIF4 and p70S6K signaling. In total,
  18 genes were identified in common. B, Schematic of translation initiation with
  genes/complexes associated with the DEG list identified. Shaded in green are genes
  that are upregulated in the apex. C, Venn diagram of genes of mitochondrial dysfunction
  and oxidative phosphorylation. No genes were unique to oxidative phosphorylation.
  D, Schematic of oxidative phosphorylation. Genes identified in each complex are
  listed. For (B and D), red indicates genes/complexes that are downregulated, green
  indicates genes/complexes that are upregulated in the LV apex, and white indicates
  genes/complexes that are not represented on the DEG list but are important to pathway.
  DEG indicates differentially expressed gene; EEC, endocardial endothelial cell;
  eIF, eukaryotic initiation factor; LV, left ventricle; mTOR, mammalian target of
  rapamycin.'
papertitle: Integrated Regional Cardiac Hemodynamic Imaging and RNA Sequencing Reveal
  Corresponding Heterogeneity of Ventricular Wall Shear Stress and Endocardial Transcriptome.
reftext: Margaret E. McCormick, et al. J Am Heart Assoc. 2016 Apr;5(4):e003170.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6545511
figid_alias: PMC4859290__F7
figtype: Figure
redirect_from: /figures/PMC4859290__F7
ndex: 76530108-dea6-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4859290__JAH3-5-e003170-g007.html
  '@type': Dataset
  description: 'Analysis of signaling pathways specific to base:apex comparison. A,
    Venn diagram comparing overlap of genes identified in the top 3 canonical pathways:
    eIF2 signaling, mTOR signaling, and regulation of eIF4 and p70S6K signaling. In
    total, 18 genes were identified in common. B, Schematic of translation initiation
    with genes/complexes associated with the DEG list identified. Shaded in green
    are genes that are upregulated in the apex. C, Venn diagram of genes of mitochondrial
    dysfunction and oxidative phosphorylation. No genes were unique to oxidative phosphorylation.
    D, Schematic of oxidative phosphorylation. Genes identified in each complex are
    listed. For (B and D), red indicates genes/complexes that are downregulated, green
    indicates genes/complexes that are upregulated in the LV apex, and white indicates
    genes/complexes that are not represented on the DEG list but are important to
    pathway. DEG indicates differentially expressed gene; EEC, endocardial endothelial
    cell; eIF, eukaryotic initiation factor; LV, left ventricle; mTOR, mammalian target
    of rapamycin.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MTOR
  - EIF2S1
  - EIF2S3
  - EIF2S2
  - PABPC1
  - EIF2AK3
  - TRNA
  - ELF2
  - ELF3
  - EIF3A
  - RPS6
  - ELF4
  - EIF4G1
  - EIF4G2
  - EIF4G3
  - EIF4E
  - EIF4E2
  - EIF4E3
  - EIF4A1
  - EIF4A2
  - EIF4A3
  - RPS6KB1
  - RPS6KB2
  - RPS23
  - RPS3
  - RPS16
  - RPS17
  - RPS27
  - RPS25
  - RPS21
  - RPS26
  - RPS26P32
  - RPS26P47
  - EEC1
  - APEX1
  - SDHB
  - NDUFA1
  - SDHD
  - COX7B
  - COX7B2
  - UQCRQ
  - NDUFA11
  - NDUFS2
  - ATP5F1B
  - NDUFS5
  - WDTC1
  - ATP8A2
  - Met
---
