---
figid: PMC4978736__fpls-07-01138-g0006
figlink: /pmc/articles/PMC4978736/figure/F6/
number: F6
caption: 'Omics and C4pathway analysis of two subtypes distinguished according to
  the decarboxylating enzyme in mature and immature tissues. (A) NADP-ME, NADP requiring
  malic enzyme. (B) NAD-ME, NAD requiring malic enzyme. Gene expression and proteomics
  data are provided as log2FC but for clarity of the figure only provided if either
  proteome or transcriptome changed by more than logFC of 0.5. Metabolite data is
  provided as concentration fold change. Omics data are highlighted in green, red,
  and yellow. Green: Increase in gene expression, protein accumulation, or in metabolite
  concentration in mature tissues compared to immature tissues. Red: Decrease in gene
  expression, in protein accumulation, or in metabolite concentration in mature tissues
  compared to immature tissues. Yellow: Up and down regulation of genes/enzymes that
  characterize isoforms in different tissues and organelles. Gray: no statistically
  significant difference between tissues. Numbers refer to enzymes. (1) PEP carboxylase,
  (2) NADP-malate dehydrogenase, (3) NADP-malic enzyme, (4) Pyruvate-Pi dikinase,
  (5) Rubisco (carboxylation), (6) glucose-1-phosphate adenylyltransferase, (7) alpha
  /beta amylase, (8) pyruvate dehydrogenase complex (dihydrolipoamide S-acetyltransferase),
  (9) fatty acid synthases (saturated fatty acids), (10) aspartate aminotrasferase,
  (11) NAD-malate dehydrogenase, (12) NAD-malic enzyme, (13) alanine amino transferase.
  Some steps were omitted for the sake of simplicity.'
pmcid: PMC4978736
papertitle: 'Metabolic Reconstruction of Setaria italica: A Systems Biology Approach
  for Integrating Tissue-Specific Omics and Pathway Analysis of Bioenergy Grasses.'
reftext: Cristiana G. de Oliveira Dal'Molin, et al. Front Plant Sci. 2016;7:1138.
pmc_ranked_result_index: '200533'
pathway_score: 0.9051699
filename: fpls-07-01138-g0006.jpg
figtitle: Omics and C4pathway analysis of two subtypes distinguished according to
  the decarboxylating enzyme in mature and immature tissues
year: '2016'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4978736__fpls-07-01138-g0006.html
  '@type': Dataset
  description: 'Omics and C4pathway analysis of two subtypes distinguished according
    to the decarboxylating enzyme in mature and immature tissues. (A) NADP-ME, NADP
    requiring malic enzyme. (B) NAD-ME, NAD requiring malic enzyme. Gene expression
    and proteomics data are provided as log2FC but for clarity of the figure only
    provided if either proteome or transcriptome changed by more than logFC of 0.5.
    Metabolite data is provided as concentration fold change. Omics data are highlighted
    in green, red, and yellow. Green: Increase in gene expression, protein accumulation,
    or in metabolite concentration in mature tissues compared to immature tissues.
    Red: Decrease in gene expression, in protein accumulation, or in metabolite concentration
    in mature tissues compared to immature tissues. Yellow: Up and down regulation
    of genes/enzymes that characterize isoforms in different tissues and organelles.
    Gray: no statistically significant difference between tissues. Numbers refer to
    enzymes. (1) PEP carboxylase, (2) NADP-malate dehydrogenase, (3) NADP-malic enzyme,
    (4) Pyruvate-Pi dikinase, (5) Rubisco (carboxylation), (6) glucose-1-phosphate
    adenylyltransferase, (7) alpha /beta amylase, (8) pyruvate dehydrogenase complex
    (dihydrolipoamide S-acetyltransferase), (9) fatty acid synthases (saturated fatty
    acids), (10) aspartate aminotrasferase, (11) NAD-malate dehydrogenase, (12) NAD-malic
    enzyme, (13) alanine amino transferase. Some steps were omitted for the sake of
    simplicity.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - MFAP1
  - Alanine
  - Aspartate
  - Fatty acids
  - Glucose
  - Sucrose
  - Glutamate
genes:
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: AMP
  symbol: AMP
  source: hgnc_alias_symbol
  hgnc_symbol: MFAP1
  entrez: '4236'
chemicals:
- word: Alanine
  source: MESH
  identifier: D000409
- word: Aspartate
  source: MESH
  identifier: D001224
- word: Fatty acids
  source: MESH
  identifier: D005227
- word: Glucose
  source: MESH
  identifier: D005947
- word: Sucrose
  source: MESH
  identifier: D013395
- word: Glutamate
  source: MESH
  identifier: D018698
diseases: []
figid_alias: PMC4978736__F6
redirect_from: /figures/PMC4978736__F6
figtype: Figure
---
