---
figid: PMC4158947__nihms622983f3
figtitle: Mechanism of converting epigenetic marks
organisms:
- Nicotiana tabacum
- Brassica napus
- Cannabis sativa
- Mus musculus
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC4158947
filename: nihms622983f3.jpg
figlink: /pmc/articles/PMC4158947/figure/F3/
number: F3
caption: Mechanism of converting epigenetic marks. Each aversive environmental factor
  take specific entry to alter the epigenetics, e.g. through altering DNA methyltransferase
  (DNMT) and changing methyl donor availability to affect DNA methylation, changing
  histone deacetylase (HDAC) or histone acetyltrasnferase (HAT) to alter histone code,
  or through altering DNA methylation on regulatory regions of transposon element
  (TE) or on miRNA to affect TE and miRNA. To this date, there are missing information
  how environmental factors might directly affect the TE or miRNA. The RE1 silencing
  transcription factor (REST), which regulates the expression of miRNA, may be a potential
  pathway. Finally, the genomic polymorphism or C-T mutation resulted single nucleotide
  variant (SNV) can also affect the CpG methylation.
papertitle: Environmental Alterations of Epigenetics Prior to the Birth.
reftext: Chiao-Ling Lo, et al. Int Rev Neurobiol. ;115:1-49.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8863804
figid_alias: PMC4158947__F3
figtype: Figure
redirect_from: /figures/PMC4158947__F3
ndex: fc695de6-deb2-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4158947__nihms622983f3.html
  '@type': Dataset
  description: Mechanism of converting epigenetic marks. Each aversive environmental
    factor take specific entry to alter the epigenetics, e.g. through altering DNA
    methyltransferase (DNMT) and changing methyl donor availability to affect DNA
    methylation, changing histone deacetylase (HDAC) or histone acetyltrasnferase
    (HAT) to alter histone code, or through altering DNA methylation on regulatory
    regions of transposon element (TE) or on miRNA to affect TE and miRNA. To this
    date, there are missing information how environmental factors might directly affect
    the TE or miRNA. The RE1 silencing transcription factor (REST), which regulates
    the expression of miRNA, may be a potential pathway. Finally, the genomic polymorphism
    or C-T mutation resulted single nucleotide variant (SNV) can also affect the CpG
    methylation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Mt2
  - Nmt
  - te
  - HDAC4
  - HDAC6
  - HDAC1
  - HDAC3
  - hat
  - Elp3
  - CG1894
  - chm
  - Folic acid
  - Nicotine
---
