---
figid: PMC4223488__zjw0101448490004
figtitle: Pathways involved in carbon source assimilation
organisms:
- NA
pmcid: PMC4223488
filename: zjw0101448490004.jpg
figlink: /pmc/articles/PMC4223488/figure/F4/
number: F4
caption: 'Pathways involved in carbon source assimilation. Highlighting indicates
  Embden-Meyerhof-Parnas pathway and gluconeogenic pathway (orange), pentose phosphate
  pathway (blue), Entner-Doudoroff pathway (yellow), glycerate pathway (green), methylcitrate
  and methylmalonate pathways (pink), tricarboxylic acid cycle and glyoxylate shunt
  (gray). Enzyme accession numbers without the “MSMEG_” prefix are adjacent to arrows
  indicating their associated reactions. Differentially abundant proteins are in bold
  green type with boxes labeled as follows: G (abundance in glucose compared with
  both acetate and propionate); A (abundance in acetate compared with glucose); and
  P (abundance in propionate compared with glucose). Color coding in each box indicates
  higher abundance (red), lower abundance (blue), or no difference (white). Proteins
  that show different trends in acetate versus glucose and propionate versus glucose
  are indicated as “no difference” in the G boxes. Only the quantified proteins are
  indicated. Metabolic pathways have been drawn according to Kyoto encyclopedia of
  genes and genomes pathway maps.'
papertitle: Quantitative Mass Spectrometry Reveals Plasticity of Metabolic Networks
  in Mycobacterium smegmatis  .
reftext: Tarun Chopra, et al. Mol Cell Proteomics. 2014 Nov;13(11):3014-3028.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9210455
figid_alias: PMC4223488__F4
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC4223488__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4223488__zjw0101448490004.html
  '@type': Dataset
  description: 'Pathways involved in carbon source assimilation. Highlighting indicates
    Embden-Meyerhof-Parnas pathway and gluconeogenic pathway (orange), pentose phosphate
    pathway (blue), Entner-Doudoroff pathway (yellow), glycerate pathway (green),
    methylcitrate and methylmalonate pathways (pink), tricarboxylic acid cycle and
    glyoxylate shunt (gray). Enzyme accession numbers without the “MSMEG_” prefix
    are adjacent to arrows indicating their associated reactions. Differentially abundant
    proteins are in bold green type with boxes labeled as follows: G (abundance in
    glucose compared with both acetate and propionate); A (abundance in acetate compared
    with glucose); and P (abundance in propionate compared with glucose). Color coding
    in each box indicates higher abundance (red), lower abundance (blue), or no difference
    (white). Proteins that show different trends in acetate versus glucose and propionate
    versus glucose are indicated as “no difference” in the G boxes. Only the quantified
    proteins are indicated. Metabolic pathways have been drawn according to Kyoto
    encyclopedia of genes and genomes pathway maps.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - ap
  - Coa
  - Sos
  - Propionyl CoA
  - 5525 Succinyl-CoA
  - (S)-Methylmalonyl CoA
  - Methylmalonyl CoA
  - Erythrose-4-Phosphate
  - Fructose-6-Phosphate
  - Ribulose
  - Acetate
  - acetone Phosphate
  - Arabinose
  - Gluconate
  - gluconate
  - phospho-gluconate
  - 2118 Glucose
  - Glucose
  - Keto-glutarate
  - Glyceraldehyde-3-Phosphate
  - Glycerol
  - Glyoxylate
  - 2-Methyl isocitrate
  - Isocitrate
  - Oxaloacetate
  - 6-Phosphate
  - 7-Phosphate
  - Phosphate
  - Propionate
  - EPhosphoenol pyruvate
  - Pyruvate
  - Ribose
  - Succinate
  - Tartronate
  - Xylose
  - Xylulose-5-4
  - Glutamate
  - Glucose-6-Phosphate
---
