---
figid: PMC4451040__peerj-03-968-g002
figtitle: Back from the dead; the curious tale of the predatory cyanobacterium Vampirovibrio
  chlorellavorus
organisms:
- Vampirovibrio chlorellavorus
- Escherichia coli
- Agrobacterium tumefaciens
- Enterococcus faecalis
- Synechocystis sp. PCC 6803
- Synechocystis sp.
- Acinetobacter baumannii
- Neisseria meningitidis
- Caulobacter vibrioides
- Pasteurella multocida
- Staphylococcus aureus 1-B-1
- Xanthomonas campestris pv. campestris
- Nostoc punctiforme
- Herpetosiphon aurantiacus
- Bdellovibrio bacteriovorus
- Candidatus Obscuribacter phosphatis
- Nitrosomonas eutropha
- Agromyces ramosus
- Micavibrio aeruginosavorus
- Saprospira grandis str. Lewin
- PX clade
- metagenome
- environmental samples
- Chlorella vulgaris
- Homo sapiens
- Chlorella virus
- NA
organisms_ner:
- Homo sapiens
- Agrobacterium tumefaciens
- Neisseria meningitidis
- Pasteurella multocida
- Drosophila melanogaster
pmcid: PMC4451040
filename: peerj-03-968-g002.jpg
figlink: /pmc/articles/PMC4451040/figure/fig-2/
number: F2
caption: Metabolic predictions for V. chlorellavorus based on genes annotated by IMG/ER
  (). Solid and dashed lines represent single or multiple steps in a pathway respectively.
  Black ovals indicate substrates that enter the glycolysis pathway. Fermentation
  end-products are indicated as black rectangles. V. chlorellavorus is capable of
  oxidative phosphorylation as it contains a complete TCA cycle and electron transport
  chain. Biosynthetic products are shown in green (amino acids), red (co-factors and
  vitamins), purple (nucleotides), and orange (non-mevalonate pathway products). Serine
  (highlighted in blue) is not able to be synthesised and is presumably transported
  into the cell. ATP-binding cassette transporters are highlighted in yellow and permeases,
  pumps and transporters are highlighted in orange. The direction of substrate transport
  across the membrane is shown with arrows. Putatively highly expressed genes and
  complexes are bolded. V. chlorellavorus is missing all recognised photosynthesis
  genes including those for Photosystems I and II, chlorophyll and antennae proteins.
papertitle: Back from the dead; the curious tale of the predatory cyanobacterium Vampirovibrio
  chlorellavorus.
reftext: Rochelle M. Soo, et al. PeerJ. 2015;3:e968.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7907073
figid_alias: PMC4451040__F2
figtype: Figure
redirect_from: /figures/PMC4451040__F2
ndex: b7a41cd2-df1e-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4451040__peerj-03-968-g002.html
  '@type': Dataset
  description: Metabolic predictions for V. chlorellavorus based on genes annotated
    by IMG/ER (). Solid and dashed lines represent single or multiple steps in a pathway
    respectively. Black ovals indicate substrates that enter the glycolysis pathway.
    Fermentation end-products are indicated as black rectangles. V. chlorellavorus
    is capable of oxidative phosphorylation as it contains a complete TCA cycle and
    electron transport chain. Biosynthetic products are shown in green (amino acids),
    red (co-factors and vitamins), purple (nucleotides), and orange (non-mevalonate
    pathway products). Serine (highlighted in blue) is not able to be synthesised
    and is presumably transported into the cell. ATP-binding cassette transporters
    are highlighted in yellow and permeases, pumps and transporters are highlighted
    in orange. The direction of substrate transport across the membrane is shown with
    arrows. Putatively highly expressed genes and complexes are bolded. V. chlorellavorus
    is missing all recognised photosynthesis genes including those for Photosystems
    I and II, chlorophyll and antennae proteins.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CPA1
  - APC
  - PROC
  - RASA1
  - RGS6
  - UQCRFS1
  - C6
  - RBM14
  - KANK2
  - CACYBP
  - TP53INP1
  - TSACC
  - RBM14-RBM4
  - SERPINA2
  - SERPINA1
  - PAEP
  - PREP
  - PTPN22
  - IPP
  - CS
  - AEN
  - CDK13
  - OVOL2
  - SLC4A11
  - CHEK1
  - CHEK2
  - SARDH
  - SDHB
  - SORD
  - SDS
  - OGDH
  - TWIST1
  - CAPG
  - CD46
  - DNAH8
  - gap
  - hemE
  - fumC
  - cheD
  - ade
  - cpa
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Apc
  - Apc2
  - Axn
  - Fs(3)Apc
  - G6P
  - tipE
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - RFeSP
  - fi
  - Dsor1
  - gar
  - plo
  - Pep
  - bgcn
  - che
  - Ace
  - Ipp
  - CG9784
  - CG9426
  - CG6805
  - Synj
  - INPP5E
  - 5PtaseI
  - cs
  - Mrtf
  - rempA
  - pi
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - Coa
  - sls
  - Abd-B
  - Vha14-1
  - na
  - Pyrimidine
  - Pentane Phasyuhate
  - DMAPP
  - Acetate
  - acetate
  - Alanine
  - Arabinose
  - Asparagine
  - Aspartate Oxalo
  - Carbohydrate
  - Cysteine
  - Fatty Acid
  - Glutamine
  - Glycerol
  - Isocitrate
  - Isoleucine
  - Oxoglutarate
  - Leucine
  - Lysine
  - Mannose  acid-palyamine
  - Methionine
  - Nicotinate
  - Phosphate
  - Proline
  - Purine
  - Riboflavin
  - Serine
  - Succinate
  - Threonine
  - Tryptophan
  - Valine
  - Glutamate
  - Ammonium
---
