---
figid: PMC4612132__elife07966fs003
figtitle: Metabolic reconstruction of the Bathymodiolus symbiont
organisms:
- Escherichia coli
- Pseudomonas aeruginosa
- Bacillus subtilis
- Salmonella enterica subsp. enterica serovar Typhimurium
- Vibrio cholerae
- Bacillus subtilis At2
- Synechocystis sp. PCC 6803
- Synechocystis sp.
- Yersinia pestis
- Aliivibrio fischeri
- Vibrio vulnificus
- Rhizobium leguminosarum
- Bordetella pertussis
- Sodalis glossinidius
- Bordetella bronchiseptica
- Candidatus Hamiltonella defensa
- sulfur-oxidizing symbionts
- Mycetohabitans rhizoxinica
- endosymbiont of Riftia pachyptila
- Mycetohabitans endofungorum
- Sinorhizobium sp.
- Candidatus Ruthturnera calyptogenae
- Paraburkholderia phenoliruptrix BR3459a
- Chlorochromatium aggregatum
- Xenorhabdus poinarii
- Candidatus Endonucleobacter bathymodioli
- Pinna nobilis
- Mytilus edulis
- Euprymna scolopes
- Pinctada fucata
- Riftia pachyptila
- Sipunculus nudus
- Bathymodiolus azoricus
- Dreissena polymorpha
- Tevnia jerichonana
- Calyptogena magnifica
- Solemya velum
- Bocageia sp.
- Bathymodiolus puteoserpentis
- Bathymodiolus sp.
- Toxolasma parvum
- Heteractis magnifica
- Perna perna
- Armoracia rusticana
- Oryza sativa
- Erythrina poeppigiana
- Rhizopus microsporus
- Trachipleistophora hominis
- Mimosa flocculosa
- metagenome
- metagenomes
- Homo sapiens
- Ovis aries
- Neacomys sp.
- NA
pmcid: PMC4612132
filename: elife07966fs003.jpg
figlink: /pmc/articles/PMC4612132/figure/fig2s3/
number: SF2_3
caption: 'Key metabolic pathways were inferred from genomic information using Pathway
  Tools (). Red stars indicate that the gene was not found in the B. sp. symbiont
  genome, and blue stars indicate that the gene was not found in BazSymB, but was
  found in BazSymA, both symbionts of B. azoricus. Red arrow indicates a missing enzyme
  that could be replaced with an alternative reaction. Green arrow indicates an inorganic
  pyrophosphate-dependent step in the modified version of the Calvin cycle. Nar =
  nitrate reductase; Nir = nitrite reductase; Nor = nitric oxide reductase; Hup =
  membrane-bound hydrogenase; SOX = sulfur oxidation; rDSR = reverse dissimilatory
  sulfite reductase; Sqr = sulfide-quinone reductase; Apr = adenylsulfate reductase;
  SAT = sulfate adenyltransferase; P = phosphate; BP = biphosphate; COX = cytochrome
  c oxidase; Gln = glutamine; Arg = arginine; Pro = proline; Met = methionine; Lys
  = lysine; Thr = threonine; Ile = isoleucine; PPi = inorganic pyrophosphate; PPase
  = soluble pyrophosphatase; SS = secretion system.DOI: http://dx.doi.org/10.7554/eLife.07966.008'
papertitle: Abundant toxin-related genes in the genomes of beneficial symbionts from
  deep-sea hydrothermal vent mussels.
reftext: Lizbeth Sayavedra, et al. eLife. 2015;4:e07966.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.953178
figid_alias: PMC4612132__SF2_3
figtype: Supplemental figure
organisms_ner:
- NA
redirect_from: /figures/PMC4612132__SF2_3
ndex: 2883568d-deee-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4612132__elife07966fs003.html
  '@type': Dataset
  description: 'Key metabolic pathways were inferred from genomic information using
    Pathway Tools (). Red stars indicate that the gene was not found in the B. sp.
    symbiont genome, and blue stars indicate that the gene was not found in BazSymB,
    but was found in BazSymA, both symbionts of B. azoricus. Red arrow indicates a
    missing enzyme that could be replaced with an alternative reaction. Green arrow
    indicates an inorganic pyrophosphate-dependent step in the modified version of
    the Calvin cycle. Nar = nitrate reductase; Nir = nitrite reductase; Nor = nitric
    oxide reductase; Hup = membrane-bound hydrogenase; SOX = sulfur oxidation; rDSR
    = reverse dissimilatory sulfite reductase; Sqr = sulfide-quinone reductase; Apr
    = adenylsulfate reductase; SAT = sulfate adenyltransferase; P = phosphate; BP
    = biphosphate; COX = cytochrome c oxidase; Gln = glutamine; Arg = arginine; Pro
    = proline; Met = methionine; Lys = lysine; Thr = threonine; Ile = isoleucine;
    PPi = inorganic pyrophosphate; PPase = soluble pyrophosphatase; SS = secretion
    system.DOI: http://dx.doi.org/10.7554/eLife.07966.008'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - 3-phosphoglycerate
  - 1,3-biphosphoglycerate
  - Arg
  - L-aspartate
  - L-aspartate Lys
  - dihydroxyacetone
  - fructose
  - fructose-1,6-BP
  - fumarate
  - glucose-6-P
  - glyceraldehyde
  - glyceraldehyde-3-P
  - isocitrate
  - Met
  - oxaloacetate
  - 4-phosphate
  - phosphoenolpyruvate
  - Pyruvate
  - succinate
  - L-glutamate
---
