---
figid: PMC4287430__12864_2014_6788_Fig1_HTML
figtitle: Predicted model of the S
organisms:
- Solemya velum
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC4287430
filename: 12864_2014_6788_Fig1_HTML.jpg
figlink: /pmc/articles/PMC4287430/figure/Fig1/
number: F1
caption: 'Predicted model of the S. velum symbiont cell. The diagram, based on the
  gene annotation of the symbiont genome, depicts key functional systems and metabolic
  pathways: sulfur oxidation, electron transport, ATP synthases, CO2-fixation via
  the Calvin Cycle, gluconeogenesis, polyglucose synthesis, glycolysis, TCA and glyoxylate
  cycles, synthesis of amino acids, fatty acids, lipids, isoprenoids via non-mevalonate
  pathway, and the cell wall, solute transporters, protein secretion systems, and
  the type IV pilus. Different protein categories are color-coded and the individual
  subunits indicated by shape symbols. The direction of substrate transport across
  the membrane is shown with arrows. Components of the electron transport chain are
  arranged from the lowest to the highest electronegativity of the electron donors
  (blue) and acceptors (red). The corresponding electronegativity values are listed
  next to the respective enzymes. Enzymes shared between glycolysis, gluconeogenesis,
  and the Calvin cycle are designated in green. Enzymes unique to these pathways are
  designated in red. Enzymes shared between the Calvin cycle and the pentose phosphate
  cycle are designated in blue. Amino acids which may be essential for the host are
  designated in red. Speculated pathways are designated with a question mark. The
  abbreviations used, the respective full gene product names, and the corresponding
  NCBI protein ID references are listed in Additional file : Table S3.'
papertitle: 'The genome of the intracellular bacterium of the coastal bivalve, Solemya
  velum: a blueprint for thriving in and out of symbiosis.'
reftext: Oleg Dmytrenko, et al. BMC Genomics. 2014;15:924.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9386604
figid_alias: PMC4287430__F1
figtype: Figure
redirect_from: /figures/PMC4287430__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4287430__12864_2014_6788_Fig1_HTML.html
  '@type': Dataset
  description: 'Predicted model of the S. velum symbiont cell. The diagram, based
    on the gene annotation of the symbiont genome, depicts key functional systems
    and metabolic pathways: sulfur oxidation, electron transport, ATP synthases, CO2-fixation
    via the Calvin Cycle, gluconeogenesis, polyglucose synthesis, glycolysis, TCA
    and glyoxylate cycles, synthesis of amino acids, fatty acids, lipids, isoprenoids
    via non-mevalonate pathway, and the cell wall, solute transporters, protein secretion
    systems, and the type IV pilus. Different protein categories are color-coded and
    the individual subunits indicated by shape symbols. The direction of substrate
    transport across the membrane is shown with arrows. Components of the electron
    transport chain are arranged from the lowest to the highest electronegativity
    of the electron donors (blue) and acceptors (red). The corresponding electronegativity
    values are listed next to the respective enzymes. Enzymes shared between glycolysis,
    gluconeogenesis, and the Calvin cycle are designated in green. Enzymes unique
    to these pathways are designated in red. Enzymes shared between the Calvin cycle
    and the pentose phosphate cycle are designated in blue. Amino acids which may
    be essential for the host are designated in red. Speculated pathways are designated
    with a question mark. The abbreviations used, the respective full gene product
    names, and the corresponding NCBI protein ID references are listed in Additional
    file : Table S3.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATP8A2
  - LAT
  - ORC3
  - SPNS1
  - ETV3
  - CELA2A
  - VCAN
  - PYGL
  - GPI
  - GNPDA1
  - RPIA
  - RPE
  - LPA
  - TPI1
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - NT5E
  - PNPLA6
  - DGCR2
  - ABCB6
  - CS
  - HCCS
  - TRIM54
  - KLK15
  - FH
  - OGDH
  - TWIST1
  - ACACA
  - BMS1
  - ACACB
  - NDUFAB1
  - CPAT1
  - DURS1
  - HSD17B8
  - FDPS
  - ELDR
  - PIKFYVE
  - CDIPT
  - PTDSS1
  - PEE1
  - PSD
  - PISD
  - FBXL15
  - OMP
  - ADHFE1
  - BRCA2
  - FANCD2
  - PSEN1
  - TRAP
  - ACP5
  - CD40LG
  - TRAF2
  - TRRAP
  - TDRD7
  - SCYL1
  - DCT
  - TALDO1
  - LRSAM1
  - TAAR1
  - GPR3
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - SLC6A1
  - ATPsynbeta
  - Atpalpha
  - ke
  - ef
  - et
  - Orc3
  - pel
  - ca
  - ine
  - tei
  - Pgm1
  - gt
  - Pgi
  - Rpi
  - CycE
  - cyc
  - Rpe
  - Hk
  - Tpi
  - Pgk
  - Cys
  - mmy
  - Csp
  - I-t
  - UbcE2H
  - Eno
  - pk
  - LanA
  - Me
  - cs
  - mle
  - Aco
  - Acox57D-d
  - fh
  - mura
  - Ogdh
  - CG33791
  - Kdm5
  - ACC
  - tacc
  - acclinal-wing
  - Acp26Ab
  - Acp26Aa
  - CG18284
  - Acp36DE
  - SP
  - Acp95EF
  - Acph-1
  - Fpps
  - fab1
  - Pis
  - Cds
  - psd
  - Pisd
  - opm
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - pana
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - aop
  - .na.character
  - tant
  - run
  - Sy
  - grh
  - Rheb
  - E(spl)malpha-BFM
  - ac
  - edl
  - phosphane
  - amine
  - Cys
  - FATTY ACIDS
  - Leu
  - Met
  - UDP
---
