---
figid: PMC4279084__elife04265fs017
figtitle: A more detailed genetic model for roles of CED-3 caspase in regulating the
  heterochronic pathway and for showing other genes from our genomic screen in this
  pathway
organisms:
- Homo sapiens
- Mus musculus
- Oryctolagus cuniculus
- Caenorhabditis elegans
- Triportheus paranensis
pmcid: PMC4279084
filename: elife04265fs017.jpg
figlink: /pmc/articles/PMC4279084/figure/fig10s1/
number: SF10_1
caption: 'Modified from the discussion provided by a recent review (), this model
  is still simplified given the complex and dynamic processes (; ; ; ; ; ). Black
  arrows and blocks indicate aspects of the pathway that were previously known. Red
  blocks and font indicate the major findings of this study. Blue font indicates factors
  known to function in this pathway that were also identified in our enhancer screen
  () as genetic interactors of GW182 (ain-1 and/or ain-2). The blue asterisk associated
  with let-7 is to indicate that we also identified the let-7 family miRNAs in our
  miRNA-ced-3 enhancer screen () and that ced-3(lf) was able to enhance their specific
  temporal cell fate patterning phenotypes as seen with adult-specific alae defects
  (). The blue asterisks and dashed red lines for miR-1 and miR-246 indicate additional
  miRNAs we identified in our miRNA-ced-3 enhancer screen (). Both miR-1 and miR-246
  are predicted to target lin-14 mRNA (See ‘Materials and methods’) but have not been
  experimentally demonstrated to do so here. They failed to show enhanced specific
  developmental timing defects and were not pursued further. Dashed arrows and blocks
  are intended to indicate a combination of complex direct and indirect effects ultimately
  affecting developmental fate and timing. ** Previous works have indicated DAF-12
  functions on several components of the pathway (; ), providing the basis for us
  to have observed the enhanced developmental defects associated with ain-1(lf) and
  daf-12(RNAi) (). The exact mechanism of LIN-66 function is still unclear (dashed
  block) though it is able to negatively regulate LIN-28 function (). MSI-1 (Musashi-1)
  was identified as an RNA-binding protein that may be important for LIN-28-mediated
  regulation of pre-miRNA processing as well as translational regulation (; ). Interestingly,
  we also identified the C. elegans musashi-1 ortholog, msi-1, as an enhancer in our
  screen (). Following binding by LIN-28, pre-let-7 is poly-uridylated by a TUTase
  (poly-U-polymerase) to mark it for destruction. PUP-2 is indicated here as a C.
  elegans ortholog for TUTase and it was also identified in our screen (). Recently,
  the RNAse, Dis3l2 (Dis-3-like RNAse gene 2), was identified as the 3′-5′ exonuclease
  that degrades poly-uridylated-pre-let-7 miRNA and is the RNAse mutated in Perlman
  syndrome (; ; ). We suggest that the C. elegans gene F48E8.6 is the C. elegans ortholog
  of Dis3l2 and have therefore named it DISL-2 (Dis-3-like RNAse gene 2) (See ). The
  intricate network of regulatory factors shown here emphasizes the importance of
  this pathway and how CED-3 caspase cooperates with the miRISC and numerous other
  factors to control temporal cell fate patterning.DOI: http://dx.doi.org/10.7554/eLife.04265.036'
papertitle: CED-3 caspase acts with miRNAs to regulate non-apoptotic gene expression
  dynamics for robust development in C. elegans.
reftext: Benjamin P Weaver, et al. eLife. 2014;3:e04265.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.957699
figid_alias: PMC4279084__SF10_1
figtype: Supplemental figure
organisms_ner:
- Caenorhabditis elegans
redirect_from: /figures/PMC4279084__SF10_1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4279084__elife04265fs017.html
  '@type': Dataset
  description: 'Modified from the discussion provided by a recent review (), this
    model is still simplified given the complex and dynamic processes (; ; ; ; ; ).
    Black arrows and blocks indicate aspects of the pathway that were previously known.
    Red blocks and font indicate the major findings of this study. Blue font indicates
    factors known to function in this pathway that were also identified in our enhancer
    screen () as genetic interactors of GW182 (ain-1 and/or ain-2). The blue asterisk
    associated with let-7 is to indicate that we also identified the let-7 family
    miRNAs in our miRNA-ced-3 enhancer screen () and that ced-3(lf) was able to enhance
    their specific temporal cell fate patterning phenotypes as seen with adult-specific
    alae defects (). The blue asterisks and dashed red lines for miR-1 and miR-246
    indicate additional miRNAs we identified in our miRNA-ced-3 enhancer screen ().
    Both miR-1 and miR-246 are predicted to target lin-14 mRNA (See ‘Materials and
    methods’) but have not been experimentally demonstrated to do so here. They failed
    to show enhanced specific developmental timing defects and were not pursued further.
    Dashed arrows and blocks are intended to indicate a combination of complex direct
    and indirect effects ultimately affecting developmental fate and timing. ** Previous
    works have indicated DAF-12 functions on several components of the pathway (;
    ), providing the basis for us to have observed the enhanced developmental defects
    associated with ain-1(lf) and daf-12(RNAi) (). The exact mechanism of LIN-66 function
    is still unclear (dashed block) though it is able to negatively regulate LIN-28
    function (). MSI-1 (Musashi-1) was identified as an RNA-binding protein that may
    be important for LIN-28-mediated regulation of pre-miRNA processing as well as
    translational regulation (; ). Interestingly, we also identified the C. elegans
    musashi-1 ortholog, msi-1, as an enhancer in our screen (). Following binding
    by LIN-28, pre-let-7 is poly-uridylated by a TUTase (poly-U-polymerase) to mark
    it for destruction. PUP-2 is indicated here as a C. elegans ortholog for TUTase
    and it was also identified in our screen (). Recently, the RNAse, Dis3l2 (Dis-3-like
    RNAse gene 2), was identified as the 3′-5′ exonuclease that degrades poly-uridylated-pre-let-7
    miRNA and is the RNAse mutated in Perlman syndrome (; ; ). We suggest that the
    C. elegans gene F48E8.6 is the C. elegans ortholog of Dis3l2 and have therefore
    named it DISL-2 (Dis-3-like RNAse gene 2) (See ). The intricate network of regulatory
    factors shown here emphasizes the importance of this pathway and how CED-3 caspase
    cooperates with the miRISC and numerous other factors to control temporal cell
    fate patterning.DOI: http://dx.doi.org/10.7554/eLife.04265.036'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ced-3
  - daf-12
  - lin-4
  - lin-66
  - lin-46
  - mir-1
  - hbl-1
  - lin-14
  - mir-246
  - lin-28
  - msi-1
  - let-7
  - lin-41
  - lin-29
  - pup-2
  - disl-2
---
