---
figid: PMC4039369__msb201321-f1
figtitle: Yeast pathway map of nucleotide degradation and ribose salvage
organisms:
- Escherichia coli
- Salmonella enterica subsp. enterica serovar Typhimurium
- Escherichia coli IS1
- Saccharomyces cerevisiae
- Saccharomyces cerevisiae S288C
- Homo sapiens
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
- Arabidopsis thaliana
pmcid: PMC4039369
filename: msb201321-f1.jpg
figlink: /pmc/articles/PMC4039369/figure/f1/
number: F1
caption: Yeast pathway map of nucleotide degradation and ribose salvage. The pathway
  begins with the degradation of RNA. In starvation, this is triggered by macroautophagy
  (ribophagy). The resulting nucleotides are degraded into nucleosides except for
  AMP, which is first converted to IMP (). Purine nucleosides are then converted into
  ribose-1-phosphate and base, whereas pyrimidines are hydrolyzed into ribose and
  base. Both ribose and ribose-1-phosphate are converted into ribose-5-phosphate,
  which enters the nonoxidative branch of PPP. Gene annotations that are taken from
  definitive prior literature are shown in black; those confirmed by experiments in
  this study are shown in blue, and those newly assigned in this study are shown in
  red. Incorrect and missing annotations in the current genome scale metabolic model
  () (http://www.comp-sys-bio.org/yeastnet/) are listed in . The enzyme converting
  ribose to R5P remains unclear. G6P, glucose-6-phosphate; FBP, fructose-1,6-bisphosphate;
  GAP, glyceraldehyde-3-phosphate; DHAP, dihydroxyacetone phosphate; R5P, ribose-5-phosphate;
  X5P, xylulose-5-phosphate; Ru5P, ribulose-5-phosphate; E4P, erythrose-4-phosphate.
papertitle: Nucleotide degradation and ribose salvage in yeast.
reftext: Yi-Fan Xu, et al. Mol Syst Biol. 2013;9:665-665.
year: '2013'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9392058
figid_alias: PMC4039369__F1
figtype: Figure
redirect_from: /figures/PMC4039369__F1
ndex: b7cf06a9-df05-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4039369__msb201321-f1.html
  '@type': Dataset
  description: Yeast pathway map of nucleotide degradation and ribose salvage. The
    pathway begins with the degradation of RNA. In starvation, this is triggered by
    macroautophagy (ribophagy). The resulting nucleotides are degraded into nucleosides
    except for AMP, which is first converted to IMP (). Purine nucleosides are then
    converted into ribose-1-phosphate and base, whereas pyrimidines are hydrolyzed
    into ribose and base. Both ribose and ribose-1-phosphate are converted into ribose-5-phosphate,
    which enters the nonoxidative branch of PPP. Gene annotations that are taken from
    definitive prior literature are shown in black; those confirmed by experiments
    in this study are shown in blue, and those newly assigned in this study are shown
    in red. Incorrect and missing annotations in the current genome scale metabolic
    model () (http://www.comp-sys-bio.org/yeastnet/) are listed in . The enzyme converting
    ribose to R5P remains unclear. G6P, glucose-6-phosphate; FBP, fructose-1,6-bisphosphate;
    GAP, glyceraldehyde-3-phosphate; DHAP, dihydroxyacetone phosphate; R5P, ribose-5-phosphate;
    X5P, xylulose-5-phosphate; Ru5P, ribulose-5-phosphate; E4P, erythrose-4-phosphate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATG7
  - ISN1
  - PHM8
  - CDD1
  - URH1
  - PNP1
  - GUD1
  - FCY1
  - RKI1
  - RPE1
  - PRM15
  - TKL1
  - TKL2
  - PFK1
  - FBA1
  - NQM1
  - TAL1
  - TPI1
  - TDH1
  - TDH3
  - TDH2
  - APRT
  - MFAP1
  - IMPA1
  - BRAP
  - NT5C2
  - MATN1
  - PTPN13
  - PGM3
  - TK2
  - TK1
  - PFKM
  - SHBG
  - CCHCR1
  - RASA1
  - RGS6
  - SLC5A5
  - APG7
  - PI
  - AT3G60750
  - AT2G45290
  - TKI1
---
