---
figid: PMC4893057__424_2016_1828_Fig1_HTML
figtitle: Sequence elements and protein components involved in the formation of poly(A)
  tails
organisms:
- Drosophila melanogaster
- Caenorhabditis elegans
- Autographa gamma
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Saccharomyces cerevisiae
- unidentified influenza virus
pmcid: PMC4893057
filename: 424_2016_1828_Fig1_HTML.jpg
figlink: /pmc/articles/PMC4893057/figure/Fig1/
number: F1
caption: Sequence elements and protein components involved in the formation of poly(A)
  tails. a The specificity and efficiency of cleavage and polyadenylation are determined
  by the binding of multi-protein complexes to specific elements at the 3′ end of
  the pre-mRNA. Most pre-mRNAs contain two core elements. The canonical polyadenylation
  signal AAUAAA (or less frequently AUUAAA) upstream of the cleavage site is recognized
  by the multimeric cleavage and polyadenylation specificity factor (CPSF) consisting
  of at least six subunits (CPSF 160, CPSF 100, CPSF 73, CPSF 30, hFip1, and WDR33).
  This RNA-protein interaction determines the site of cleavage 10–30 nt downstream,
  preferentially immediately 3′ of a CA dinucleotide. The second canonical sequence
  element is characterized by a high density of G/U or U residues which is located
  up to 30 nt downstream of the cleavage site. This downstream sequence element (DSE)
  is bound by the 64-kDa subunit of the hetero-trimeric cleavage stimulating factor
  (CSTF) that promotes the efficiency of 3′ end processing. Furthermore, multimers
  of a UGUA motif are localized at variable distances upstream of the cleavage site
  to recruit the heterodimeric cleavage factor CFIm [, ]. Finally, accessory sequences
  can function as upstream sequence elements (USE) [, , , , , , , , , , ] to facilitate
  3′ end processing by recruiting canonical 3′ end factors directly [, ] or by serving
  as an additional anchor for the (canonical) 3′ end processing machinery [, ]. After
  assembly of the basal 3′ end processing machinery, the endonucleolytic cleavage
  reaction is catalyzed by CPSF-73, and the cleaved mRNA is polyadenylated by nuclear
  poly(A)-polymerase (PAP). The binding of PABPN1 to the poly(A) tail is unstable,
  and upon nuclear export, PABPN1 is replaced by the cytosolic poly(A) binding protein
  (PABPC), which interacts with the translation initiation factor eIF4G, stimulating
  translation and regulating mRNA stability. Of note, in addition to histone 3′ end
  processing that follows a different pathway [, ], other “non-canonical” mechanisms
  of 3′ end processing exist []. b Transcription initiation, elongation, and termination
  (circular arrow) are tightly coupled to mRNA processing steps such as capping, splicing,
  and 3′ end processing (inner circle). Appropriate 3′ end processing is functionally
  interconnected with transcription and mRNA capping and splicing and impacts on post-transcriptional
  mechanisms (mRNA release, export, abundance, and translation). Loss- or gain-of-function
  of 3′ end processing thus critically interferes with other gene expression steps
  (modified after []). c Co-transcriptional mRNA processing promotes packaging of
  the nascent RNA transcript (formation of an “inert” RNP particle) and thus to prevent
  the accumulation of co-transcriptional R-loops (lower panel), which can lead to
  DNA-double-strand breaks and chromosomal rearrangements. Disruption of co-transcriptional
  RNA-processing can thus lead to genomic instability (modified after [])
papertitle: 'Processing and transcriptome expansion at the mRNA 3′ end in health and
  disease: finding the right end.'
reftext: Anton Ogorodnikov, et al. Pflugers Arch. 2016;468:993-1012.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6713031
figid_alias: PMC4893057__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Homo sapiens
- Rattus norvegicus
redirect_from: /figures/PMC4893057__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4893057__424_2016_1828_Fig1_HTML.html
  '@type': Dataset
  description: Sequence elements and protein components involved in the formation
    of poly(A) tails. a The specificity and efficiency of cleavage and polyadenylation
    are determined by the binding of multi-protein complexes to specific elements
    at the 3′ end of the pre-mRNA. Most pre-mRNAs contain two core elements. The canonical
    polyadenylation signal AAUAAA (or less frequently AUUAAA) upstream of the cleavage
    site is recognized by the multimeric cleavage and polyadenylation specificity
    factor (CPSF) consisting of at least six subunits (CPSF 160, CPSF 100, CPSF 73,
    CPSF 30, hFip1, and WDR33). This RNA-protein interaction determines the site of
    cleavage 10–30 nt downstream, preferentially immediately 3′ of a CA dinucleotide.
    The second canonical sequence element is characterized by a high density of G/U
    or U residues which is located up to 30 nt downstream of the cleavage site. This
    downstream sequence element (DSE) is bound by the 64-kDa subunit of the hetero-trimeric
    cleavage stimulating factor (CSTF) that promotes the efficiency of 3′ end processing.
    Furthermore, multimers of a UGUA motif are localized at variable distances upstream
    of the cleavage site to recruit the heterodimeric cleavage factor CFIm [, ]. Finally,
    accessory sequences can function as upstream sequence elements (USE) [, , , ,
    , , , , , , ] to facilitate 3′ end processing by recruiting canonical 3′ end factors
    directly [, ] or by serving as an additional anchor for the (canonical) 3′ end
    processing machinery [, ]. After assembly of the basal 3′ end processing machinery,
    the endonucleolytic cleavage reaction is catalyzed by CPSF-73, and the cleaved
    mRNA is polyadenylated by nuclear poly(A)-polymerase (PAP). The binding of PABPN1
    to the poly(A) tail is unstable, and upon nuclear export, PABPN1 is replaced by
    the cytosolic poly(A) binding protein (PABPC), which interacts with the translation
    initiation factor eIF4G, stimulating translation and regulating mRNA stability.
    Of note, in addition to histone 3′ end processing that follows a different pathway
    [, ], other “non-canonical” mechanisms of 3′ end processing exist []. b Transcription
    initiation, elongation, and termination (circular arrow) are tightly coupled to
    mRNA processing steps such as capping, splicing, and 3′ end processing (inner
    circle). Appropriate 3′ end processing is functionally interconnected with transcription
    and mRNA capping and splicing and impacts on post-transcriptional mechanisms (mRNA
    release, export, abundance, and translation). Loss- or gain-of-function of 3′
    end processing thus critically interferes with other gene expression steps (modified
    after []). c Co-transcriptional mRNA processing promotes packaging of the nascent
    RNA transcript (formation of an “inert” RNP particle) and thus to prevent the
    accumulation of co-transcriptional R-loops (lower panel), which can lead to DNA-double-strand
    breaks and chromosomal rearrangements. Disruption of co-transcriptional RNA-processing
    can thus lead to genomic instability (modified after [])
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ca
  - cp
  - Cpsf160
  - wisp
  - PAPLA1
  - Pli
  - Pdf
  - skd
  - hrg
  - PolI
  - PolE1
  - sip2
  - SP
  - nt
  - Cp
  - Cpsf2
  - Cpsf1
  - Asap1
  - Reg3b
  - Papola
  - Acp3
  - Papolb
  - Mrps30
  - Tusc2
  - Asap2
  - Pdap1
  - Poli
  - Pole
  - CP
  - CPSF1
  - REG3A
  - ASAP2
  - MRPS30
  - PAPOLA
  - PDAP1
  - TUSC2
  - ASAP1
  - POLI
  - RNPC3
  - PARTICL
  - POLE
  - USF2
  - GLI3
---
