---
figid: PMC4872805__emss-52495-f002
figtitle: Sirtuins as regulators of metabolism and healthspan
organisms:
- Saccharomyces cerevisiae
- Arachis hypogaea
- Homo sapiens
- Mus musculus
- Caenorhabditis elegans
- Chenuda virus
pmcid: PMC4872805
filename: emss-52495-f002.jpg
figlink: /pmc/articles/PMC4872805/figure/F1/
number: F1
caption: a | Various transcription factors regulate sirtuin expression. FOXO1, PPARα,
  PPARβ/δ and CREB enhance SIRT1 (in green), whereas PPARγ, ChREBP, PARP2 and HIC1
  repress SIRT1 expression (in red). Only CREB, ChREBP and PARP2 were also shown to
  possess this activity in vivo. Sirt1 expression is also repressed by the miRNAs
  miR-34a and miR-199a. b | Reversible post-translational modifications affect SIRT1
  activity. The cyclinB/Cdk1 complex phosphorylates (P) SIRT1 thereby allowing cell
  cycle progression. Activation of JNK by reactive oxygen species (ROS) results in
  SIRT1 phosphorylation and subsequent deacetylation of histone H3, but not p53. Genotoxic
  stress, for instance by UV light or H2O2 exposure, results in desumoylation of SIRT1,
  inactiving it. Sumoylation (sumo) by a yet unidentified enzyme activates SIRT1.
  c | Complex formation with other proteins influences SIRT1 enzymatic activity. A
  complex of NCoR1 and SIRT1 blocks the transcriptional activity of PPARγ. Genotoxic
  and metabolic (high-fat diet) stress induce DBC1-SIRT1 complex formation, by which
  DBC1 inactivates SIRT1. Fasting relieves this inactivation. The LSD1-SIRT1 complex
  represses Notch target gene expression by demethylation and deacetylation of specific
  histones, but is derepressed by activation of the Notch pathway. Complex formation
  with AROS activates SIRT1. d | Controlling the levels of the cofactor NAD+ governs
  SIRT1 function. NAD+ levels are increased by providing precursors nicotinic acid
  (NA), nicotinamide (NAM) or nicotinamide riboside (NR), inhibiting its breakdown
  by means of PARP or CD38 inhibition, or by AMPK activation following energy stress
  or treatment with the AMPK activator resveratrol. Increased NAD+ levels subsequently
  lead to sirtuin activation.
papertitle: Sirtuins as regulators of metabolism and healthspan.
reftext: Riekelt H. Houtkooper, et al. Nat Rev Mol Cell Biol. ;13(4):225-238.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9383245
figid_alias: PMC4872805__F1
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Caenorhabditis elegans
redirect_from: /figures/PMC4872805__F1
ndex: 91659932-df00-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4872805__emss-52495-f002.html
  '@type': Dataset
  description: a | Various transcription factors regulate sirtuin expression. FOXO1,
    PPARα, PPARβ/δ and CREB enhance SIRT1 (in green), whereas PPARγ, ChREBP, PARP2
    and HIC1 repress SIRT1 expression (in red). Only CREB, ChREBP and PARP2 were also
    shown to possess this activity in vivo. Sirt1 expression is also repressed by
    the miRNAs miR-34a and miR-199a. b | Reversible post-translational modifications
    affect SIRT1 activity. The cyclinB/Cdk1 complex phosphorylates (P) SIRT1 thereby
    allowing cell cycle progression. Activation of JNK by reactive oxygen species
    (ROS) results in SIRT1 phosphorylation and subsequent deacetylation of histone
    H3, but not p53. Genotoxic stress, for instance by UV light or H2O2 exposure,
    results in desumoylation of SIRT1, inactiving it. Sumoylation (sumo) by a yet
    unidentified enzyme activates SIRT1. c | Complex formation with other proteins
    influences SIRT1 enzymatic activity. A complex of NCoR1 and SIRT1 blocks the transcriptional
    activity of PPARγ. Genotoxic and metabolic (high-fat diet) stress induce DBC1-SIRT1
    complex formation, by which DBC1 inactivates SIRT1. Fasting relieves this inactivation.
    The LSD1-SIRT1 complex represses Notch target gene expression by demethylation
    and deacetylation of specific histones, but is derepressed by activation of the
    Notch pathway. Complex formation with AROS activates SIRT1. d | Controlling the
    levels of the cofactor NAD+ governs SIRT1 function. NAD+ levels are increased
    by providing precursors nicotinic acid (NA), nicotinamide (NAM) or nicotinamide
    riboside (NR), inhibiting its breakdown by means of PARP or CD38 inhibition, or
    by AMPK activation following energy stress or treatment with the AMPK activator
    resveratrol. Increased NAD+ levels subsequently lead to sirtuin activation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Sirt1
  - Cdk1
  - Ncor1
  - Ncor2
  - Foxo1
  - Ppara
  - Ppard
  - nr
  - Cd38
  - Creb1
  - Mapk8
  - Brinp1
  - Ccar2
  - H3
  - Dyrk1a
  - Trp53
  - Kdm1a
  - Parp2
  - Hic1
  - Mir34a
  - SIRT1
  - CDK1
  - CCNB1
  - CCNB2
  - CCNB3
  - NCOR1
  - NCOR2
  - FOXO1
  - PRKAA1
  - PRKAA2
  - PRKAB1
  - PRKAB2
  - PRKAG1
  - PRKAG2
  - PRKAG3
  - PPARA
  - PPARD
  - CD38
  - STAC3
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - MAPK8
  - MAPK9
  - MAPK10
  - BRINP1
  - CCAR2
  - DYRK1A
  - DYRK1B
  - DYRK2
  - DYRK3
  - DYRK4
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - TP53
  - TP63
  - TP73
  - KDM1A
  - PARP2
  - HIC1
  - MIR34A
  - SENP1
  - SENP2
  - SENP3
  - SENP5
  - SENP6
  - SENP7
  - SENP8
  - cdk-1
  - lsd-1
  - parp-2
  - hic-1
  - Cancer
---
