---
figid: PMC4628106__fmicb-06-01205-g004
figtitle: Predicted geochemical cycling in the FRC subsurface based on metabolic profiles
  of pristine (FW301, left) and contaminated (FW106, right)
organisms:
- metagenome
- metagenomes
- Pseudomonas syringae
- Pseudomonas fluorescens
- Cupriavidus necator
- Rhodobacter capsulatus
- Ralstonia solanacearum
- Bradyrhizobium japonicum
- Stutzerimonas stutzeri
- Shewanella oneidensis
- Azotobacter vinelandii
- Archaeoglobus fulgidus
- Methanosarcina barkeri
- Chromobacterium violaceum
- Clostridium ljungdahlii
- Desulfovibrio vulgaris
- Allochromatium vinosum
- Rhizobium etli
- Geobacter sulfurreducens
- Acidithiobacillus ferrooxidans
- Methylococcus capsulatus
- Clostridium carboxidivorans
- Cupriavidus metallidurans
- Ralstonia pickettii
- Chlorobium limicola
- Candidatus Kuenenia stuttgartiensis
- Candidatus Accumulibacter phosphatis
- Methylosinus trichosporium
- Anaeromyxobacter dehalogenans
- Methylobacter tundripaludum
- Nitrobacter winogradskyi
- Thiothrix nivea
- Afipia sp.
- Sinorhizobium sp.
- Nitrosomonas eutropha
- Desulfosporosinus sp.
- Beggiatoa sp.
- Rhodanobacter sp.
- Methanohalobium evestigatum
- Rhodanobacter thiooxydans
- Desulforamulus reducens
- groundwater metagenome
- Homo sapiens
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC4628106
filename: fmicb-06-01205-g004.jpg
figlink: /pmc/articles/PMC4628106/figure/F4/
number: F4
caption: Predicted geochemical cycling in the FRC subsurface based on metabolic profiles
  of pristine (FW301, left) and contaminated (FW106, right). The denitrification pathway
  genes are labeled in bold in the FW106 figure to indicate their overabundance in
  this metagenome compared to FW301. Dotted lines indicate either that the pathway
  genes were not detected in the metagenomes, detected in very low abundance, or that
  bacteria known to implement these pathways were not detected or in extremely low
  abundance (<1%). Contaminants present in high concentrations at the FW106 site are
  shown in white boxes. Partitions indicate different geochemical and electrochemical
  environments that may transiently exist at the OR-IFRC sites and do not necessarily
  represent the specific environmental partitioning present in these environments.
papertitle: Comparative metagenomics reveals impact of contaminants on groundwater
  microbiomes.
reftext: Christopher L. Hemme, et al. Front Microbiol. 2015;6:1205.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9614098
figid_alias: PMC4628106__F4
figtype: Figure
redirect_from: /figures/PMC4628106__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4628106__fmicb-06-01205-g004.html
  '@type': Dataset
  description: Predicted geochemical cycling in the FRC subsurface based on metabolic
    profiles of pristine (FW301, left) and contaminated (FW106, right). The denitrification
    pathway genes are labeled in bold in the FW106 figure to indicate their overabundance
    in this metagenome compared to FW301. Dotted lines indicate either that the pathway
    genes were not detected in the metagenomes, detected in very low abundance, or
    that bacteria known to implement these pathways were not detected or in extremely
    low abundance (<1%). Contaminants present in high concentrations at the FW106
    site are shown in white boxes. Partitions indicate different geochemical and electrochemical
    environments that may transiently exist at the OR-IFRC sites and do not necessarily
    represent the specific environmental partitioning present in these environments.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - hv
  - dock
  - Doc3
  - Doc2
  - Doc1
  - Yif1
  - so
  - Eno
  - sno
  - Snoo
  - CG43051
  - Hn
  - Polyphosphate
---
