---
figid: PMC4905194__kmco-01-02-957039-g002
figtitle: Mechanisms of gap filling
organisms:
- Gallus gallus
- Xenopus laevis
- Homo sapiens
- Mus musculus
- Saccharomyces cerevisiae
- Escherichia coli
pmcid: PMC4905194
filename: kmco-01-02-957039-g002.jpg
figlink: /pmc/articles/PMC4905194/figure/f0002/
number: F2
caption: Mechanisms of gap filling. A replication fork block leads to Rad6/Rad18-dependent
  PCNA monoubiquitination (A), which targets a specialized polymerase able to insert
  a correct or incorrect nucleotide opposite the lesion (translesion synthesis) (B).
  Alternatively, Rad51 and Rad5 can promote ssDNA gap filling by a mechanism that
  employs the intact sister chromatid to circumvent the lesion through the formation
  and subsequent dissolution of hemicatenanes (C–H). Hemicatenane formation requires
  the ssDNA binding complex replication protein A (RPA), the recombination proteins
  Rad51, Rad52, Rad55, Rad57, the Shu complex, Rad53-dependent phosphorylation of
  Rad55, Rad54, the nuclease Exo1, the polyubiquitin ligase activity of Rad5, the
  DNA bending activity of the high mobility group protein Hmo1, the cyclin-dependent
  kinase Cdc28, and Pol δ, but neither Pol ε nor the TLS polymerases. Dissolution
  of hemicatenanes requires the Sgs1/Top3/Rmi1 complex together with Ubc9/Mms21-dependent
  sumoylation of the DNA repair Smc5-6 complex. An alternative, Rad5-independent,
  Rad17/Mec3/Ddc1-dependent salvage HR pathway, which is inhibited in S phase by sumoPCNA-dependent
  recruitment of Srs2, might be operating through the formation and further resolution
  of HJs (C–I). See text for more details. * It should be noted that Rad51-strand
  invasion does not require PCNA monoubiquitination.
papertitle: Homologous recombination maintenance of genome integrity during DNA damage
  tolerance.
reftext: Félix Prado. Mol Cell Oncol. 2014 Apr-Jun;1(2):e957039.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9176836
figid_alias: PMC4905194__F2
figtype: Figure
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
- Mus musculus
- Gallus gallus
redirect_from: /figures/PMC4905194__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4905194__kmco-01-02-957039-g002.html
  '@type': Dataset
  description: Mechanisms of gap filling. A replication fork block leads to Rad6/Rad18-dependent
    PCNA monoubiquitination (A), which targets a specialized polymerase able to insert
    a correct or incorrect nucleotide opposite the lesion (translesion synthesis)
    (B). Alternatively, Rad51 and Rad5 can promote ssDNA gap filling by a mechanism
    that employs the intact sister chromatid to circumvent the lesion through the
    formation and subsequent dissolution of hemicatenanes (C–H). Hemicatenane formation
    requires the ssDNA binding complex replication protein A (RPA), the recombination
    proteins Rad51, Rad52, Rad55, Rad57, the Shu complex, Rad53-dependent phosphorylation
    of Rad55, Rad54, the nuclease Exo1, the polyubiquitin ligase activity of Rad5,
    the DNA bending activity of the high mobility group protein Hmo1, the cyclin-dependent
    kinase Cdc28, and Pol δ, but neither Pol ε nor the TLS polymerases. Dissolution
    of hemicatenanes requires the Sgs1/Top3/Rmi1 complex together with Ubc9/Mms21-dependent
    sumoylation of the DNA repair Smc5-6 complex. An alternative, Rad5-independent,
    Rad17/Mec3/Ddc1-dependent salvage HR pathway, which is inhibited in S phase by
    sumoPCNA-dependent recruitment of Srs2, might be operating through the formation
    and further resolution of HJs (C–I). See text for more details. * It should be
    noted that Rad51-strand invasion does not require PCNA monoubiquitination.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RAD6
  - RAD18
  - RAD51
  - SRS2
  - TOP3
  - RMI1
  - SGS1
  - MUS81
  - MMS4
  - YEN1
  - PCNA
  - POLD1
  - ERVW-4
  - POLE
  - ERVK-9
  - ERVK-11
  - ERVK-19
  - FUS
  - WARS1
  - TOP3A
  - Pcna
  - Rad18
  - Mela
  - Gm42368
  - Pole
  - Fus
  - Rad51
  - Xrcc2
  - Gabrg2
  - Rmi1
  - Mus81
  - Ms4
  - BRAF
---
