---
figid: PMC4586477__vaccines-03-00771-g002
figtitle: TAA-specific CD8+ T cells often lack co-stimulation for an immune response
  against tumors
organisms:
- Sequoia sempervirens
- Streptococcus pyogenes
- Serratia marcescens
- Galatella linosyris
- Homo sapiens
- Mus musculus
- NA
pmcid: PMC4586477
filename: vaccines-03-00771-g002.jpg
figlink: /pmc/articles/PMC4586477/figure/vaccines-03-00771-f002/
number: F2
caption: 'TAA-specific CD8+ T cells often lack co-stimulation for an immune response
  against tumors. CD28 signaling was reflected in a top canonical pathway predicted
  by genome-wide mRNA expression profiling to trend towards less activation in TAA-specific
  CD8+ T cells from the tumor compared to those in circulation (p-value of overlap
  = 3.2E-4, right-tailed Fisher Exact Test, z-score = −0.816). Many signaling pathways
  overlap between T cell subsets, so it is not surprising that a CD4+ T cell signaling
  pathway was associated with the gene expression of CD8+ T cells used to generate
  the figure above. Differential gene expression corresponding to molecules enriched
  in this pathway are outlined in purple with red fill representing overexpression
  and green fill representing decreased expression in T cells from the tumor relative
  to the periphery, and color intensity corresponds to the extent of expression difference.
  CD28 was predicted to be a key upstream regulator that is less active in TAA-specific
  CD8+ T cells from the tumor (p-value of overlap = 1.3E-5, right-tailed Fisher exact
  test, activation z-score = −0.696). CTLA-4 was also predicted to be an upstream
  regulator of differential gene expression (p-value of overlap = 9.7E-5, right-tailed
  Fisher exact test). Differentially-expressed genes and corresponding fold-changes
  have been previously published and were reanalyzed through the use of QIAGEN’s Ingenuity®Pathway
  Analysis (IPA®, QIAGEN, Redwood City, CA, USA, www.qiagen.com/ingenuity) []. Both
  direct and indirect relationships were assessed in the Ingenuity knowledge base
  reference set with a confidence threshold of previous experimental observation in
  T cells. Note: The connecting arrow between SHP and T cell activation was altered
  to a line to better reflect inhibition downstream of CTLA-4.'
papertitle: Targeting Transcriptional Regulators of CD8+ T Cell Dysfunction to Boost
  Anti-Tumor Immunity.
reftext: Katherine A. Waugh, et al. Vaccines (Basel). 2015 Sep;3(3):771-802.
year: '2015'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8659679
figid_alias: PMC4586477__F2
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
redirect_from: /figures/PMC4586477__F2
ndex: 684d6694-df1d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4586477__vaccines-03-00771-g002.html
  '@type': Dataset
  description: 'TAA-specific CD8+ T cells often lack co-stimulation for an immune
    response against tumors. CD28 signaling was reflected in a top canonical pathway
    predicted by genome-wide mRNA expression profiling to trend towards less activation
    in TAA-specific CD8+ T cells from the tumor compared to those in circulation (p-value
    of overlap = 3.2E-4, right-tailed Fisher Exact Test, z-score = −0.816). Many signaling
    pathways overlap between T cell subsets, so it is not surprising that a CD4+ T
    cell signaling pathway was associated with the gene expression of CD8+ T cells
    used to generate the figure above. Differential gene expression corresponding
    to molecules enriched in this pathway are outlined in purple with red fill representing
    overexpression and green fill representing decreased expression in T cells from
    the tumor relative to the periphery, and color intensity corresponds to the extent
    of expression difference. CD28 was predicted to be a key upstream regulator that
    is less active in TAA-specific CD8+ T cells from the tumor (p-value of overlap
    = 1.3E-5, right-tailed Fisher exact test, activation z-score = −0.696). CTLA-4
    was also predicted to be an upstream regulator of differential gene expression
    (p-value of overlap = 9.7E-5, right-tailed Fisher exact test). Differentially-expressed
    genes and corresponding fold-changes have been previously published and were reanalyzed
    through the use of QIAGEN’s Ingenuity®Pathway Analysis (IPA®, QIAGEN, Redwood
    City, CA, USA, www.qiagen.com/ingenuity) []. Both direct and indirect relationships
    were assessed in the Ingenuity knowledge base reference set with a confidence
    threshold of previous experimental observation in T cells. Note: The connecting
    arrow between SHP and T cell activation was altered to a line to better reflect
    inhibition downstream of CTLA-4.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Cd28
  - Cd86
  - Cd80
  - ag
  - H2-T18
  - Ptprc
  - Cd4
  - Syk
  - Lat
  - Grb2
  - Fyn
  - Grap2
  - Lcp2
  - Csk
  - Akt1
  - Zap70
  - Vav1
  - Cdc42
  - Itk
  - Was
  - Pak1
  - Pkn1
  - Map2k1
  - Map3k1
  - Arp
  - Rere
  - Calm1
  - Picalm
  - Mdk
  - Nfkbib
  - Nfkb1
  - Jun
  - Fos
  - Fancd2
  - Mapk8
  - Il2
  - CD28
  - CD86
  - CD80
  - PTPRC
  - CD4
  - SYK
  - LAT
  - ORC3
  - SPNS1
  - GRB2
  - FYN
  - GRAP2
  - LCP2
  - CSK
  - AKT1
  - RAC1
  - RAC2
  - RAC3
  - RHOG
  - ZAP70
  - VAV1
  - CDC42
  - ITK
  - WAS
  - PAK1
  - PKN1
  - MAP3K1
  - PDK1
  - PDK2
  - PDK3
  - PDK4
  - CRISP1
  - RERE
  - MANF
  - ARFRP1
  - CALM2
  - CALM3
  - PICALM
  - SNAP91
  - CALM1
  - MAP2K7
  - MAP2K1
  - MAP2K2
  - NFKB1
  - RELA
  - RELB
  - REL
  - NFKB2
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - JUN
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - BRCA2
  - FANCD2
  - PSEN1
  - MAPK8
  - MAPK9
  - MAPK10
  - IL2
  - FAD
---
