---
figid: PMC5833380__41419_2017_250_Fig3_HTML
figtitle: ROS-independent ER stress-mediated NRF2 activation promotes warburg effect
  to maintain stemness-associated properties of cancer-initiating cells
organisms:
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
organisms_ner:
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
- Danio rerio
pmcid: PMC5833380
filename: 41419_2017_250_Fig3_HTML.jpg
figlink: /pmc/articles/PMC5833380/figure/Fig3/
number: F3
caption: a (Upper left panel) Venn diagram depicting the overlap genes (Group A; purple)
  between the NRF2-bound genes (ChIP-Seq analyses) and genes with significantly altered
  expression in ROSLow cells (blue). Otherwise, the Venn diagram depicts the overlap
  genes (Group B; orange) between NRF2-bound genes and genes with significantly altered
  expression in ROSHigh cells. (Right panel) NRF2 mediated metabolic genes, involved
  in glycolysis and the non-oxidative PPP pathway in ROSLow cells (Group A), or involved
  in the oxidative PPP and glutamine pathway in ROSHigh cells (Group B). (Lower left
  panel) Summary of NRF2 regulating distinct metabolic pathways in ROSLow cells (quiescent
  CICs) and ROSHigh cells (proliferating CICs). b ChIP-Seq tracks of NRF2 in SAS sphere
  cells, assessing the binding of glycolysis and PPP. Location of ChIP-Seq peaks that
  were identified by using Strand NGS software. RNA-Seq traces (UCSC genome browser)
  for each gene in ROSLow cells and ROSHigh cells
papertitle: ROS-independent ER stress-mediated NRF2 activation promotes warburg effect
  to maintain stemness-associated properties of cancer-initiating cells.
reftext: Ching-Wen Chang, et al. Cell Death Dis. 2018 Feb;9(2):194.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6409255
figid_alias: PMC5833380__F3
figtype: Figure
redirect_from: /figures/PMC5833380__F3
ndex: 0a6ca314-de90-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5833380__41419_2017_250_Fig3_HTML.html
  '@type': Dataset
  description: a (Upper left panel) Venn diagram depicting the overlap genes (Group
    A; purple) between the NRF2-bound genes (ChIP-Seq analyses) and genes with significantly
    altered expression in ROSLow cells (blue). Otherwise, the Venn diagram depicts
    the overlap genes (Group B; orange) between NRF2-bound genes and genes with significantly
    altered expression in ROSHigh cells. (Right panel) NRF2 mediated metabolic genes,
    involved in glycolysis and the non-oxidative PPP pathway in ROSLow cells (Group
    A), or involved in the oxidative PPP and glutamine pathway in ROSHigh cells (Group
    B). (Lower left panel) Summary of NRF2 regulating distinct metabolic pathways
    in ROSLow cells (quiescent CICs) and ROSHigh cells (proliferating CICs). b ChIP-Seq
    tracks of NRF2 in SAS sphere cells, assessing the binding of glycolysis and PPP.
    Location of ChIP-Seq peaks that were identified by using Strand NGS software.
    RNA-Seq traces (UCSC genome browser) for each gene in ROSLow cells and ROSHigh
    cells
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HK2
  - KCNA5
  - KIF2A
  - KLK2
  - HOOK2
  - PFKFB3
  - ENO2
  - ENO1
  - PGK1
  - LDHA
  - PDK1
  - PDPK1
  - HSPA5
  - SLC2A3
  - PGLS
  - G6PD
  - UBL4A
  - GLS
  - GLS2
  - F2
  - PFKL
  - PGD
  - PHGDH
  - TKT
  - DDR2
  - RASA1
  - RGS6
  - GAPDH
  - GAPDHP44
  - IMPA1
  - BRAP
  - PAEP
  - PREP
  - PTPN22
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - TALDO1
  - SLC2A1
  - GABPA
  - NFE2L2
  - ANMA
  - AN
  - GCLM
  - Hk2
  - Pfkfb3
  - Eno2
  - Eno1
  - Eno1b
  - Pgk1
  - Ldha
  - Pdk1
  - Pdpk1
  - Hspa5
  - Slc2a3
  - Pgls
  - G6pdx
  - Gls
  - Gls2
  - Pfkl
  - Pgd
  - Ptgdr
  - Phgdh
  - Tkt
  - Rap1gap
  - Rasa1
  - Gapdh
  - Brap
  - Prep
  - Ptpn22
  - Fndc5
  - Taldo1
  - Slc2a1
  - Hps3
  - Nfe2l2
  - Poo
  - Gclm
  - Eno
  - Pgk
  - Hsc70-3
  - Glut3
  - Zw
  - amon
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Gpdh1
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - Pep
  - bgcn
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - cic
  - Coa
  - Keap1
  - cnc
  - en
  - an
  - E(spl)malpha-BFM
  - hk2
  - pfkfb3
  - eno2
  - eno1a
  - pgk1
  - ldha
  - pdk1
  - hspa5
  - pgls
  - g6pd
  - glsb
  - f2
  - pgd
  - tktb
  - gap43
  - gapdh
  - gapdhs
  - taldo1
  - slc2a1a
  - napbb
  - nfe2l2a
  - gclm
  - ENO1
  - ENO2
  - Glucose
  - NADP GSH
  - NADPHGSSG
  - ENOT 2-PG
  - acetyl-CoA
  - Oxalacetate
  - Citrate
  - Lactate
  - TCA
  - Isocitrate
  - Glutamine
  - Succinate
  - Succinyl4
  - Glutamate
---
