---
figid: PMC5085640__ijms-17-01607-g008
figtitle: Regulatory changes in the pathway of glycolysis/gluconeogenesis
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Glycine max
- Triticum aestivum
- Hordeum vulgare
- Medicago sativa
- Citrus sinensis
- Triticum turgidum subsp. durum
- Jasminum officinale
- Hordeum vulgare subsp. spontaneum
- Boehmeria nivea
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Arabidopsis thaliana
- Glycine max
pmcid: PMC5085640
filename: ijms-17-01607-g008.jpg
figlink: /pmc/articles/PMC5085640/figure/ijms-17-01607-f008/
number: F8
caption: 'Regulatory changes in the pathway of glycolysis/gluconeogenesis. Colored
  circle nodes correspond with the ramie genes detected in the isobaric tags for relative
  and absolute quantitation (iTRAQ) data. (a) upregulated or downregulated genes in
  leaves; (b) upregulated or downregulated genes in roots. The above network model
  is generated with a cytoscape web application, based on information gained from
  up to four levels of functional analysis: fold change of gene/protein, protein-protein
  interaction, Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment,
  and biological process enrichment. Circle nodes: genes/proteins; rectangle nodes:
  KEGG pathway or biological process. Pathways are colored in a gradient color from
  yellow to blue; yellow indicates a lower p-value, and blue indicates a higher p-value.
  Biological processes are colored in red. In the case of fold change analysis, genes/proteins
  are colored in red (upregulation) and green (downregulation). A default confidence
  cutoff of 400 was used: interactions with a higher confident score are shown as
  solid lines between genes/proteins; dashed lines indicate otherwise.'
papertitle: Isobaric Tags for Relative and Absolute Quantitation (iTRAQ)-Based Comparative
  Proteome Analysis of the Response of Ramie under Drought Stress.
reftext: Xia An, et al. Int J Mol Sci. 2016 Oct;17(10):1607.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.685782
figid_alias: PMC5085640__F8
figtype: Figure
redirect_from: /figures/PMC5085640__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5085640__ijms-17-01607-g008.html
  '@type': Dataset
  description: 'Regulatory changes in the pathway of glycolysis/gluconeogenesis. Colored
    circle nodes correspond with the ramie genes detected in the isobaric tags for
    relative and absolute quantitation (iTRAQ) data. (a) upregulated or downregulated
    genes in leaves; (b) upregulated or downregulated genes in roots. The above network
    model is generated with a cytoscape web application, based on information gained
    from up to four levels of functional analysis: fold change of gene/protein, protein-protein
    interaction, Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment,
    and biological process enrichment. Circle nodes: genes/proteins; rectangle nodes:
    KEGG pathway or biological process. Pathways are colored in a gradient color from
    yellow to blue; yellow indicates a lower p-value, and blue indicates a higher
    p-value. Biological processes are colored in red. In the case of fold change analysis,
    genes/proteins are colored in red (upregulation) and green (downregulation). A
    default confidence cutoff of 400 was used: interactions with a higher confident
    score are shown as solid lines between genes/proteins; dashed lines indicate otherwise.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDE345
  - FBA2
  - AT1G56190
  - ALDH2B7
  - HOT5
  - ALDH2B4
  - PDH2
  - ADH2
  - TYRA-A
---
