---
figid: PMC5735158__41598_2017_16627_Fig6_HTML
figlink: /pmc/articles/PMC5735158/figure/Fig6/
number: Figure 6
caption: Dynamic enrichment analysis identifies context relevant higher level SCPs
  emerging from interactions of level-3 SCPs. Three case studies using experimental
  high throughput datasets from literature were used to demonstrate the ability of
  the MBC Ontology to identify SCPs relationships that give rise to the whole cell
  function that was studied in each case. SCP units were generated by merging any
  two or three level-3 SCPs that were related to each other based on the top 25% predicted
  SCP relationships and contained at least one experimentally perturbed gene. The
  merged SCPs were added to the original level-3 SCPs and the experimental data was
  analyzed for enrichment analysis using the extended set of level-3 SCPs and Fisher’s
  exact test. All level-3 SCPs (blue rectangles and diamonds) that were among the
  top 5 predictions (either as a single SCP or as an SCP unit) were connected with
  each other based on all predicted SCP-SCP relationships (dashed black lines). The
  largest SCP network was kept and assigned to a context specific higher level SCP
  (open [unspecified level] or light red [level-2] octagon). Annotated level-1 grandparents
  (dark red rectangles) and level-2 parents (light red rectangles) were added to demonstrate
  that the combined level-3 SCPs belong to different SCP families. Arrows connect
  parent SCPs with their children SCPs. Blue lines encircle level-3 SCPs that were
  identified as part of a new merged SCP unit. Squares indicate level-1 to level-3
  SCPs that were identified by the original taxonomy. Diamonds indicate level-1 to
  level-3 SCPs that were only identified by the dynamic enrichment approach that enables
  extension of the original taxonomy. (A) Dynamic enrichment analysis identifies the
  inferred context relevant higher-level SCP Protein proofreading and degradation
  as the major disease mechanism that reduces CFTR activity at the plasma membrane
  in cystic fibrosis. (B) The two major parts of the Secretory pathway were identified
  as a major requirement for secretion. (C) Transcription associated DNA re-arrangements
  were identified as a major mechanism by which erlotinib might increase sensitivity
  of breast cancer cells to doxycyclin. In this case including the top 26% (instead
  of 25%) predicted SCP relationships increases the size of the SCP-network that is
  related to the identified whole-cell function (at a 25% threshold the network contained
  the SCP Focal adhesion organization that was connected to the SCP Thyroid hormone
  receptor signaling instead of the SCPs Histone methylation and demythelation and
  Transcription regulation by chromatin remodeling). (D) Cell cortical actin filament
  dynamics were predicted to be responsible for reduction in colony formation activity
  of erlotinib treated breast cancer cells. See also Supplementary Fig.  D/E, 22 D/E
  and 23 D/H/I.
pmcid: PMC5735158
papertitle: A flexible ontology for inference of emergent whole cell function from
  relationships between subcellular processes.
reftext: Jens Hansen, et al. Sci Rep. 2017;7:17689.
pmc_ranked_result_index: '187793'
pathway_score: 0.5432072
filename: 41598_2017_16627_Fig6_HTML.jpg
figtitle: Dynamic enrichment analysis identifies context relevant higher level SCPs
  emerging from interactions of level-3 SCPs
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5735158__41598_2017_16627_Fig6_HTML.html
  '@type': Dataset
  description: Dynamic enrichment analysis identifies context relevant higher level
    SCPs emerging from interactions of level-3 SCPs. Three case studies using experimental
    high throughput datasets from literature were used to demonstrate the ability
    of the MBC Ontology to identify SCPs relationships that give rise to the whole
    cell function that was studied in each case. SCP units were generated by merging
    any two or three level-3 SCPs that were related to each other based on the top
    25% predicted SCP relationships and contained at least one experimentally perturbed
    gene. The merged SCPs were added to the original level-3 SCPs and the experimental
    data was analyzed for enrichment analysis using the extended set of level-3 SCPs
    and Fisher’s exact test. All level-3 SCPs (blue rectangles and diamonds) that
    were among the top 5 predictions (either as a single SCP or as an SCP unit) were
    connected with each other based on all predicted SCP-SCP relationships (dashed
    black lines). The largest SCP network was kept and assigned to a context specific
    higher level SCP (open [unspecified level] or light red [level-2] octagon). Annotated
    level-1 grandparents (dark red rectangles) and level-2 parents (light red rectangles)
    were added to demonstrate that the combined level-3 SCPs belong to different SCP
    families. Arrows connect parent SCPs with their children SCPs. Blue lines encircle
    level-3 SCPs that were identified as part of a new merged SCP unit. Squares indicate
    level-1 to level-3 SCPs that were identified by the original taxonomy. Diamonds
    indicate level-1 to level-3 SCPs that were only identified by the dynamic enrichment
    approach that enables extension of the original taxonomy. (A) Dynamic enrichment
    analysis identifies the inferred context relevant higher-level SCP Protein proofreading
    and degradation as the major disease mechanism that reduces CFTR activity at the
    plasma membrane in cystic fibrosis. (B) The two major parts of the Secretory pathway
    were identified as a major requirement for secretion. (C) Transcription associated
    DNA re-arrangements were identified as a major mechanism by which erlotinib might
    increase sensitivity of breast cancer cells to doxycyclin. In this case including
    the top 26% (instead of 25%) predicted SCP relationships increases the size of
    the SCP-network that is related to the identified whole-cell function (at a 25%
    threshold the network contained the SCP Focal adhesion organization that was connected
    to the SCP Thyroid hormone receptor signaling instead of the SCPs Histone methylation
    and demythelation and Transcription regulation by chromatin remodeling). (D) Cell
    cortical actin filament dynamics were predicted to be responsible for reduction
    in colony formation activity of erlotinib treated breast cancer cells. See also
    Supplementary Fig.  D/E, 22 D/E and 23 D/H/I.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACTG1
  - ACTA2
  - ACTG2
  - ACTB
  - ACTC1
  - ACTA1
  - TGFB2
  - TGFB1
  - TGFB3
genes:
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG1
  entrez: '71'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA2
  entrez: '59'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG2
  entrez: '72'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTB
  entrez: '60'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTC1
  entrez: '70'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA1
  entrez: '58'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA1
  entrez: '58'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG2
  entrez: '72'
- word: TGF-beta
  symbol: TGFB
  source: bioentities_symbol
  hgnc_symbol: TGFB2
  entrez: '7042'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG1
  entrez: '71'
- word: TGF-beta
  symbol: TGFB
  source: bioentities_symbol
  hgnc_symbol: TGFB1
  entrez: '7040'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA2
  entrez: '59'
- word: TGF-beta
  symbol: TGFB
  source: bioentities_symbol
  hgnc_symbol: TGFB3
  entrez: '7043'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTC1
  entrez: '70'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTB
  entrez: '60'
chemicals: []
diseases: []
---
