---
figid: PMC5088520__12864_2016_3066_Fig1_HTML
figtitle: 'Path2enet: generation of human pathway-derived networks in an expression
  specific context'
organisms:
- Homo sapiens
- Mus musculus
organisms_ner:
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC5088520
filename: 12864_2016_3066_Fig1_HTML.jpg
figlink: /pmc/articles/PMC5088520/figure/Fig1/
number: F1
caption: 'Figure showing the NOTCH Signaling Pathway and the transformation into a
  network using Path2enet: (a) the canonical map included in the KEGG database (hsa:04330),
  showing all the distinct proteins that are included in each node of this pathway
  (total 48 proteins); (b) transfromation of the canonical pathway to a network done
  with Path2enet, that produces a new view incorporating all the information about
  the gene products that are expressed (i.e., active) based on the pathway and on
  the expression data sets of lymphocytes that are incorporated (163 samples: 32 B
  cells and 131 T cells). Path2enet uses the gene expression Barcode algorithm to
  evaluate if a gene is expressed (ON, yellow) in the network or not (OFF, white).
  The genes in blue correspond to NA (not assigned) since the tool could not assign
  them because they are not present in the expression platform used. The panels inside
  the figure indicate: "Edges" the characteristics of the links/relations that connect
  each node (blue, activation; red, inhibition; green, expression; black, other type
  of link); "Nodes" the expression level assigned to each node in the sample type
  studied that in this case were lymphocytes (white nodes, when their expression is
  below the threshold ≤0.4; yellow nodes, when their expression is above the threshold
  >0.4; blue nodes, when the expression level is not assigned)'
papertitle: 'Path2enet: generation of human pathway-derived networks in an expression
  specific context.'
reftext: Conrad Droste, et al. BMC Genomics. 2016;17(Suppl 8):731.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8712874
figid_alias: PMC5088520__F1
figtype: Figure
redirect_from: /figures/PMC5088520__F1
ndex: 67d7a212-dea0-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5088520__12864_2016_3066_Fig1_HTML.html
  '@type': Dataset
  description: 'Figure showing the NOTCH Signaling Pathway and the transformation
    into a network using Path2enet: (a) the canonical map included in the KEGG database
    (hsa:04330), showing all the distinct proteins that are included in each node
    of this pathway (total 48 proteins); (b) transfromation of the canonical pathway
    to a network done with Path2enet, that produces a new view incorporating all the
    information about the gene products that are expressed (i.e., active) based on
    the pathway and on the expression data sets of lymphocytes that are incorporated
    (163 samples: 32 B cells and 131 T cells). Path2enet uses the gene expression
    Barcode algorithm to evaluate if a gene is expressed (ON, yellow) in the network
    or not (OFF, white). The genes in blue correspond to NA (not assigned) since the
    tool could not assign them because they are not present in the expression platform
    used. The panels inside the figure indicate: "Edges" the characteristics of the
    links/relations that connect each node (blue, activation; red, inhibition; green,
    expression; black, other type of link); "Nodes" the expression level assigned
    to each node in the sample type studied that in this case were lymphocytes (white
    nodes, when their expression is below the threshold ≤0.4; yellow nodes, when their
    expression is above the threshold >0.4; blue nodes, when the expression level
    is not assigned)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Lfng
  - Meng
  - Rfng
  - Dvl1
  - Dvl2
  - Dvl3
  - Maml1
  - Maml2
  - Numb
  - Numbl
  - Crebbp
  - Dtx1
  - Dtx2
  - Dtx3l
  - Maml3
  - Notch1
  - Notch2
  - Notch3
  - Snw1
  - Dll1
  - Dll3
  - Dll4
  - Gm14207
  - Hes1
  - Hesx1
  - Inpp5k
  - Mapk8ip1
  - Sphkap
  - Dtx3
  - Notch4
  - Kat2b
  - Dtx4
  - Hes5
  - Csl
  - Smpx
  - Rbpjl
  - Rbpj
  - Ptcra
  - Psen1
  - Psenen
  - Psen2
  - Ncstn
  - Aph1a
  - Ncor2
  - Jag1
  - Ctbp1
  - Jag2
  - Pag1
  - Cytip
  - Ctbp2
  - cir
  - Kcnj5
  - Cir1
  - Adam17
  - Hdac1
  - Hdac1-ps
  - Hdac2
  - ras
  - Hras
  - Kras
  - Rem1
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - LFNG
  - RFNG
  - DVL1
  - DVL1P1
  - DVL2
  - DVL3
  - MAML1
  - MAML2
  - CRTC1
  - NUMB
  - NUMBL
  - CREBBP
  - DTX1
  - DTX2
  - DTX3L
  - MAML3
  - IFT81
  - TMPRSS11D
  - SNW1
  - DLL1
  - DLL3
  - DLL4
  - HES1
  - GATD3
  - PLEKHM2
  - INPP5K
  - SPHKAP
  - DTX3
  - KAT2B
  - DTX4
  - HES5
  - PSMB6
  - YY1
  - CSHL1
  - RBPJ
  - SMPX
  - RBPJL
  - PTCRA
  - PSEN1
  - PSENEN
  - PSEN2
  - NCSTN
  - APH1A
  - NCOR2
  - JAG1
  - CTBP1
  - JAG2
  - EIF4E
  - OPN1LW
  - PAG1
  - PCBP4
  - CTBP2
  - KCNJ5
  - CIR1
  - VIM2P
  - UBE2V1
  - ADAM17
  - HDAC9
  - HDAC1
  - HDAC2
  - HDAC3
  - HDAC8
  - HDAC4
  - HDAC5
  - HDAC7
  - HDAC6
  - HDAC10
  - SIRT1
  - SIRT2
  - SIRT3
  - SIRT4
  - SIRT5
  - SIRT6
  - SIRT7
  - HDAC11
  - KRAS
  - HRAS
  - NRAS
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - lfng
  - rfng
  - dvl1b
  - dvl2
  - dvl3
  - dvl3a
  - maml1
  - numb
  - numbl
  - crebbp
  - crebbpa
  - dtx1
  - dtx4a
  - dtx2
  - maml3
  - notch2
  - notch3
  - snw1
  - dll4
  - dtx3
  - kat2b
  - dtx4b
  - her15.1
  - rbpjl
  - rbpja
  - psen1
  - psenen
  - psen2
  - ncstn
  - aph1b
  - ncor2
  - jag1a
  - ctbp1
  - jag2b
  - ctbp2a
  - cir1
  - adam17b
  - hdac1
  - rab1ab
---
