---
figid: PMC5796629__bmb-51-014f2
figtitle: ROS generation metabolomics, transcriptomics, and metabotranscriptomics
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Escherichia coli
- Caenorhabditis elegans
- Danio rerio
organisms_ner:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Danio rerio
- Caenorhabditis elegans
- Drosophila melanogaster
pmcid: PMC5796629
filename: bmb-51-014f2.jpg
figlink: /pmc/articles/PMC5796629/figure/f2-bmb-51-014/
number: F2
caption: Bioinformatics of ROS generation using ingenuity pathway analysis (IPA),
  (A) metabolomics, (B) transcriptomics, and (C) metabotranscriptomics based on a
  previous report (). Red and green areas indicate up- and downregulated metabolites,
  respectively, in cells treated with MNPs@SiO2(RITC) compared with control cells.
  Differentially regulated metabolites obtained from the metabolic profile (more than
  a ± 20% change) and microarray data (genes with a > 3-fold change) are shown. In
  the representation of the genetic networks, the red and green colors indicate up-
  and down-regulated genes, respectively. Network shape indicates categorization of
  molecules and function. Information pertaining to the corresponding genes can be
  found in NCBI (https://www.ncbi.nlm.nih.gov/).
papertitle: Integration of metabolomics and transcriptomics in nanotoxicity studies.
reftext: Tae Hwan Shin, et al. BMB Rep. 2018 Jan;51(1):14-20.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7322368
figid_alias: PMC5796629__F2
figtype: Figure
redirect_from: /figures/PMC5796629__F2
ndex: 03e98042-de91-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5796629__bmb-51-014f2.html
  '@type': Dataset
  description: Bioinformatics of ROS generation using ingenuity pathway analysis (IPA),
    (A) metabolomics, (B) transcriptomics, and (C) metabotranscriptomics based on
    a previous report (). Red and green areas indicate up- and downregulated metabolites,
    respectively, in cells treated with MNPs@SiO2(RITC) compared with control cells.
    Differentially regulated metabolites obtained from the metabolic profile (more
    than a ± 20% change) and microarray data (genes with a > 3-fold change) are shown.
    In the representation of the genetic networks, the red and green colors indicate
    up- and down-regulated genes, respectively. Network shape indicates categorization
    of molecules and function. Information pertaining to the corresponding genes can
    be found in NCBI (https://www.ncbi.nlm.nih.gov/).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Epo
  - Epx
  - Fgf2
  - Cxcl2
  - Htr2a
  - Igf1
  - Calb1
  - Apoe
  - Mapk14
  - Alb
  - Mapt
  - Ucp1
  - Nos2
  - Nanos2
  - Tnfrsf13b
  - Nox1
  - Prkdc
  - Rhoa
  - Egfr
  - Fn1
  - Gsn
  - Cbs
  - Ikbkb
  - Il18
  - Becn1
  - Il4
  - Bcl2
  - Mapk1
  - Mapk8
  - Acod1
  - Lonp1
  - Vip
  - Mpo
  - Nos1
  - Nanos1
  - Tlr4
  - Tac1
  - Src
  - Sod2
  - Nr4a1
  - Snca
  - P2rx7
  - Pde2a
  - Rb1
  - Pyy
  - Ppard
  - nt
  - Adipoq
  - Psen1
  - Prkce
  - Rela
  - Mapk3
  - Mageb16
  - EPO
  - TIMP1
  - EPX
  - FGF2
  - FGF13
  - CXCL2
  - HTR2A
  - IGF1
  - CALB1
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - APOE
  - ALB
  - FBF1
  - MAPT
  - UCP1
  - NOS2
  - NANOS2
  - TNFRSF13B
  - NOX1
  - RHOA
  - EGFR
  - FN1
  - GSN
  - CBS
  - IKBKB
  - IL18
  - BECN1
  - ANGPT1
  - ANGPT2
  - ANGPT4
  - AREG
  - BDNF
  - CSF1
  - EFNA1
  - EFNA2
  - EFNA3
  - EFNA4
  - EFNA5
  - EGF
  - EREG
  - FGF1
  - FGF10
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FLT3LG
  - HGF
  - IGF2
  - INS
  - KITLG
  - NGF
  - NTF3
  - NTF4
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - PGF
  - TGFA
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - IL4
  - BCL2
  - ACOD1
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - LONP1
  - VIP
  - CPAMD8
  - MPO
  - NOS1
  - NANOS1
  - TLR4
  - OMA1
  - TAC1
  - SRC
  - FGR
  - FYN
  - YES1
  - SOD2
  - SOD2-OT1
  - NR4A1
  - SNCA
  - P2RX7
  - PDE2A
  - RB1
  - RBM45
  - PYY
  - PPARD
  - CYP3A4
  - PSEN1
  - PRKCE
  - epoa
  - fgf2
  - htr2aa
  - igf1
  - calb1
  - apoeb
  - mapk14a
  - mapk14b
  - ucp1
  - nos2a
  - nox1
  - rhoaa
  - rhoab
  - egfra
  - fn1a
  - gsna
  - scinla
  - cbsb
  - ikbkb
  - becn1
  - il4
  - il13
  - bcl2a
  - mapk1
  - mapk8b
  - acod1
  - kita
  - ngfra
  - vip
  - mpx
  - pak2a
  - itpka
  - nos1
  - nanos3
  - tac1
  - src
  - sod2
  - nr4a1
  - p2rx7
  - pde2a
  - rb1
  - pyya
  - psen1
  - prkcea
  - prkceb
  - nos-2
  - nos-1
  - tac-1
  - sod-2
  - bnl
  - MKP-4
  - p38b
  - rl
  - alb
  - Mpc1
  - Nos
  - Nox
  - Rho1
  - cbs
  - caboose
  - Atg6
  - Debcl
  - Lon
  - schlank
  - Toll-4
  - Src42A
  - Csk
  - Src64B
  - Rbf
  - RpS2
  - Arpc1
  - isoleucine
  - glutame acid
  - lysine
  - phenylalanine
  - proline
  - aspartic acid
  - tyrosine
  - alanine
  - valine
  - 3-hydroxybutynic acid
  - acetoacetic acid
  - alpha-ketoglutarie acid
  - MAPT
  - cis-aconitic acid
  - MPC
  - Succinic acid
  - citric acid
  - pyruvic acid
  - fumanc acid
  - isocitric acid
  - oxalacelic acid
  - glutamic acid
  - glycine
  - O
  - malic acid
  - Pyruvic acid
  - oxalacetic acid
  - fumaric acid
  - pyroglutamic acid
  - O O
  - CBS
---
