---
figid: PMC5904076__12192_2017_843_Fig6_HTML
figtitle: HSPs and double strand break repair
organisms:
- NA
pmcid: PMC5904076
filename: 12192_2017_843_Fig6_HTML.jpg
figlink: /pmc/articles/PMC5904076/figure/Fig6/
number: F6
caption: 'HSPs and double strand break repair. Double strand breaks (DSB) are repaired
  by two major mechanisms: homologous recombination repair (HR) and non-homologous
  end joining (NHEJ). HR operates after replication, when a second identical DNA copy
  is available. The pathway begins with the recruitment of the MRN complex (MRE11-NBS1-RAD50)
  to the DSB. ATM, ATR, and the MRN complex act as damage sensors. MRE11 has DNA exonuclease
  and DNA unwinding activity. RAD50 contains motifs for nucleotide binding. BRCA1
  interacts with p53, BRCA2, RAD51, MRN complex, p21, and cyclin B to form multisubunit
  complexes. hSNM1B/Apollo is a DNA 5′ exonuclease with a preference for single-stranded
  substrates. The resection of 5′ DNA on either side of the DSB is accomplished by
  a BRCA1-dependent process, resulting in the exposure of two regions of single-stranded
  DNA (ssDNA). BRCA2 localizes the DNA recombinase RAD51 to the exposed ssDNA regions.
  RAD51 forms a nucleoprotein filament that can invade the DNA double helix and pair
  with undamaged homologous sequences. DNA polymerases δ/ε use the homologous DNA
  sequence as a template and synthesize new DNA. After DNA synthesis occurs, recombination
  between chromatids can be resolved by endonucleases and the nicks sealed by DNA
  ligase 1. Meanwhile, the NHEJ system ligates two broken ends without a sequence
  homology and when DSB is affecting a short region (1 to 6 bp). The first step consists
  in the DNA binding of the Ku70/Ku80 heterodimer, protecting the DNA from exonuclease
  digestion. The Ku heterodimer then binds to the catalytic subunit of DNA-PK (DNA-dependent
  protein kinase), activating the enzyme, and Artemis stabilizes the ends. After juxtaposition
  of the two DNA ends, DNA-PK autophophorylates and DNA ligation is performed by the
  DNA ligase 4 (LIG4)-XRCC4. When end processing is required, DNA polymerase μ (Pol
  μ) and DNA polymerase λ (Pol λ) can fill in 5′-single-strand extensions, which are
  sealed by the LIG4-XRCC4 complex. The participation of HSPs is indicated with dotted
  lines'
papertitle: 'Heat shock proteins and DNA repair mechanisms: an updated overview.'
reftext: Mayra L. Sottile, et al. Cell Stress Chaperones. 2018 May;23(3):303-315.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9166071
figid_alias: PMC5904076__F6
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC5904076__F6
ndex: 8f8d3031-deab-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5904076__12192_2017_843_Fig6_HTML.html
  '@type': Dataset
  description: 'HSPs and double strand break repair. Double strand breaks (DSB) are
    repaired by two major mechanisms: homologous recombination repair (HR) and non-homologous
    end joining (NHEJ). HR operates after replication, when a second identical DNA
    copy is available. The pathway begins with the recruitment of the MRN complex
    (MRE11-NBS1-RAD50) to the DSB. ATM, ATR, and the MRN complex act as damage sensors.
    MRE11 has DNA exonuclease and DNA unwinding activity. RAD50 contains motifs for
    nucleotide binding. BRCA1 interacts with p53, BRCA2, RAD51, MRN complex, p21,
    and cyclin B to form multisubunit complexes. hSNM1B/Apollo is a DNA 5′ exonuclease
    with a preference for single-stranded substrates. The resection of 5′ DNA on either
    side of the DSB is accomplished by a BRCA1-dependent process, resulting in the
    exposure of two regions of single-stranded DNA (ssDNA). BRCA2 localizes the DNA
    recombinase RAD51 to the exposed ssDNA regions. RAD51 forms a nucleoprotein filament
    that can invade the DNA double helix and pair with undamaged homologous sequences.
    DNA polymerases δ/ε use the homologous DNA sequence as a template and synthesize
    new DNA. After DNA synthesis occurs, recombination between chromatids can be resolved
    by endonucleases and the nicks sealed by DNA ligase 1. Meanwhile, the NHEJ system
    ligates two broken ends without a sequence homology and when DSB is affecting
    a short region (1 to 6 bp). The first step consists in the DNA binding of the
    Ku70/Ku80 heterodimer, protecting the DNA from exonuclease digestion. The Ku heterodimer
    then binds to the catalytic subunit of DNA-PK (DNA-dependent protein kinase),
    activating the enzyme, and Artemis stabilizes the ends. After juxtaposition of
    the two DNA ends, DNA-PK autophophorylates and DNA ligation is performed by the
    DNA ligase 4 (LIG4)-XRCC4. When end processing is required, DNA polymerase μ (Pol
    μ) and DNA polymerase λ (Pol λ) can fill in 5′-single-strand extensions, which
    are sealed by the LIG4-XRCC4 complex. The participation of HSPs is indicated with
    dotted lines'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HSP90AA1
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - XRCC6
  - HSPA12A
  - HSPA12B
  - HSPA13
  - HSPA14
  - HSPA1A
  - HSPA1B
  - HSPA1L
  - HSPA2
  - HSPA4
  - HSPA4L
  - HSPA5
  - HSPA6
  - HSPA8
  - HSPA9
  - HSPH1
  - HYOU1
  - XRCC5
  - DCLRE1B
  - HSPB1
  - BRCA1
  - HSP90AB1
  - BRCA2
  - RAD51
  - PRKDC
  - HSF1
  - NR5A1
  - PCNA
  - ERVW-4
  - POLL
  - POLI
  - LIG4
  - XRCC4
  - LIG1
  - LRIG1
---
