---
figid: PMC5572252__fnmol-10-00263-g0001
figlink: /pmc/articles/PMC5572252/figure/F1/
number: F1
caption: 'Cross-talk between autophagy and other pathogenic mechanisms in amyotrophic
  lateral sclerosis (ALS) (A) endoplasmic reticulum (ER) stress and unfolded protein
  response (UPR) is triggered by mutations in superoxide dismutase 1 (mSOD1), TAR
  DNA-binding protein 43 (TDP-43) and fused in sarcoma (FUS). Accumulation of misfolded
  mutant proteins activates either one or all three UPR pathways—protein kinase RNA-like
  endoplasmic reticulum kinase (PERK), endoplasmic reticulum to nucleus signaling
  1 (ERN1) and activating transcription factor 6 (ATF6). Each of the pathways stimulate
  autophagy through downstream activators. ERN1 activates autophagy through three
  pathways—(i) the AMPK pathway phosphorylates and activates two proteins—ULK1 and
  tuberous sclerosis protein 1/2 (TSC1/2)—both of which inhibit mammalian target of
  rapamycin complex 1 (mTORC1) which leads to initiation of autophagy; (ii) ERN1 activates
  mitogen-activated protein kinase 8 (MAPK8), which uses its kinase activity to phosphorylate
  B-cell lymphoma 2 (BCL2) which is in a complex with Beclin-1 (BECN1). This phosphorylation
  disrupts the complex and releases BECN1, which in turn activates autophagy through
  mTORC1 inhibition; (iii) ERN1 also acts an endoribonuclease to splice XBP1. The
  spliced version of XBP1 (XBP1s) promotes expression of BECN1. The UPR activator
  ATF6 activates the transcription factor CCAAT/enhancer-binding protein beta (CEBPB)
  that promotes expression of the kinase death-associated protein kinase 1 (DAPK1).
  DAPK1 phosphorylates BECN1 and thus breaks it from its complex with BCL2. The free
  BECN1 now activates autophagy through mTORC1 inhibition. The third UPR activator,
  PERK, activates the expression of two stress response genes—activating transcription
  factor 4 (ATF4) and DNA damage inducible transcript 3 (DDIT3). These genes promote
  the transcription of autophagy-related genes (ATGs) that activate the process. (B)
  Oxidative stress and mitochondrial dysfunction are complimentary mechanisms that
  are triggered by mutations in optineurin (mOPTN), TDP-43, SQSTM1/p62, SOD1 and C9orf72
  GGGGCC expansion. Mitochondrial dysfunction is the primary source of reactive oxygen
  and reactive nitrogen species (ROS and RNS) that are directly implicated in activating
  autophagy. An alternative mechanism is through the Kelch-like ECH-associated protein
  1 (KEAP1)-nuclear factor erythroid 2-related factor 2 (NRF2) pathways. Oxidative
  stress KEAP1-NRF2 complex is disrupted. This leaves NRF2 free to translocate to
  the nucleus where it induces the expression of several autophagy-related genes.
  Autophagy also acts to clear damaged mitochondria through an organelle-specific
  mechanism called mitophagy. (C) RNA metabolism defect is a major pathological outcome
  of mutations in RNA-binding proteins, especially FUS and TDP-43. These proteins
  are major components of stress granules and mutations in low complexity domains
  of these proteins alter the phase of stress granules to give them a complex liquid
  state. This alters the dynamics of stress granules, in that, they are not cleared
  and potentially act as primers for protein aggregates. Autophagy is a crucial mechanism
  involved in the clearance of stress granules under normal conditions and dysfunction
  in autophagy could be linked to altered stress granule dynamics. (D) DNA damage,
  either caused as a direct result of pathogenic mutation or triggered by oxidative
  stress, is a common pathological feature of mutations in FUS and C9orf72 GGGGCC
  expansion. DNA damage activates autophagy through activation of AMPK pathway. The
  AMPK pathway phosphorylates and activates two proteins, ULK1 and TSC1/2, both of
  which independently act as mTORC1 inhibitors. DNA damage activates the AMPK pathways
  through: (i) Hyperactivation of PARP1, which leads to a depletion of NAD+ reserves
  in the cell. This, in turn, depletes ATP and thus activates AMPK; or (ii) FOXOa3
  dissociation from DNA that allows it to interact with ATM. Activation of ATM by
  phosphorylation leads to downstream activation of AMPK pathway. Alternatively, FOXOa3
  also acts as a transcription factor that regulates expression of autophagy-related
  genes.'
pmcid: PMC5572252
papertitle: 'Autophagy Dysregulation in ALS: When Protein Aggregates Get Out of Hand.'
reftext: Nandini Ramesh, et al. Front Mol Neurosci. 2017;10:263.
pmc_ranked_result_index: '228542'
pathway_score: 0.9444356
filename: fnmol-10-00263-g0001.jpg
figtitle: 'Autophagy Dysregulation in ALS: When Protein Aggregates Get Out of Hand.'
year: '2017'
organisms: Homo sapiens
ndex: 92b714e1-deff-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5572252__fnmol-10-00263-g0001.html
  '@type': Dataset
  description: 'Cross-talk between autophagy and other pathogenic mechanisms in amyotrophic
    lateral sclerosis (ALS) (A) endoplasmic reticulum (ER) stress and unfolded protein
    response (UPR) is triggered by mutations in superoxide dismutase 1 (mSOD1), TAR
    DNA-binding protein 43 (TDP-43) and fused in sarcoma (FUS). Accumulation of misfolded
    mutant proteins activates either one or all three UPR pathways—protein kinase
    RNA-like endoplasmic reticulum kinase (PERK), endoplasmic reticulum to nucleus
    signaling 1 (ERN1) and activating transcription factor 6 (ATF6). Each of the pathways
    stimulate autophagy through downstream activators. ERN1 activates autophagy through
    three pathways—(i) the AMPK pathway phosphorylates and activates two proteins—ULK1
    and tuberous sclerosis protein 1/2 (TSC1/2)—both of which inhibit mammalian target
    of rapamycin complex 1 (mTORC1) which leads to initiation of autophagy; (ii) ERN1
    activates mitogen-activated protein kinase 8 (MAPK8), which uses its kinase activity
    to phosphorylate B-cell lymphoma 2 (BCL2) which is in a complex with Beclin-1
    (BECN1). This phosphorylation disrupts the complex and releases BECN1, which in
    turn activates autophagy through mTORC1 inhibition; (iii) ERN1 also acts an endoribonuclease
    to splice XBP1. The spliced version of XBP1 (XBP1s) promotes expression of BECN1.
    The UPR activator ATF6 activates the transcription factor CCAAT/enhancer-binding
    protein beta (CEBPB) that promotes expression of the kinase death-associated protein
    kinase 1 (DAPK1). DAPK1 phosphorylates BECN1 and thus breaks it from its complex
    with BCL2. The free BECN1 now activates autophagy through mTORC1 inhibition. The
    third UPR activator, PERK, activates the expression of two stress response genes—activating
    transcription factor 4 (ATF4) and DNA damage inducible transcript 3 (DDIT3). These
    genes promote the transcription of autophagy-related genes (ATGs) that activate
    the process. (B) Oxidative stress and mitochondrial dysfunction are complimentary
    mechanisms that are triggered by mutations in optineurin (mOPTN), TDP-43, SQSTM1/p62,
    SOD1 and C9orf72 GGGGCC expansion. Mitochondrial dysfunction is the primary source
    of reactive oxygen and reactive nitrogen species (ROS and RNS) that are directly
    implicated in activating autophagy. An alternative mechanism is through the Kelch-like
    ECH-associated protein 1 (KEAP1)-nuclear factor erythroid 2-related factor 2 (NRF2)
    pathways. Oxidative stress KEAP1-NRF2 complex is disrupted. This leaves NRF2 free
    to translocate to the nucleus where it induces the expression of several autophagy-related
    genes. Autophagy also acts to clear damaged mitochondria through an organelle-specific
    mechanism called mitophagy. (C) RNA metabolism defect is a major pathological
    outcome of mutations in RNA-binding proteins, especially FUS and TDP-43. These
    proteins are major components of stress granules and mutations in low complexity
    domains of these proteins alter the phase of stress granules to give them a complex
    liquid state. This alters the dynamics of stress granules, in that, they are not
    cleared and potentially act as primers for protein aggregates. Autophagy is a
    crucial mechanism involved in the clearance of stress granules under normal conditions
    and dysfunction in autophagy could be linked to altered stress granule dynamics.
    (D) DNA damage, either caused as a direct result of pathogenic mutation or triggered
    by oxidative stress, is a common pathological feature of mutations in FUS and
    C9orf72 GGGGCC expansion. DNA damage activates autophagy through activation of
    AMPK pathway. The AMPK pathway phosphorylates and activates two proteins, ULK1
    and TSC1/2, both of which independently act as mTORC1 inhibitors. DNA damage activates
    the AMPK pathways through: (i) Hyperactivation of PARP1, which leads to a depletion
    of NAD+ reserves in the cell. This, in turn, depletes ATP and thus activates AMPK;
    or (ii) FOXOa3 dissociation from DNA that allows it to interact with ATM. Activation
    of ATM by phosphorylation leads to downstream activation of AMPK pathway. Alternatively,
    FOXOa3 also acts as a transcription factor that regulates expression of autophagy-related
    genes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PARP3
  - TIPARP
  - PARP4
  - TNKS2
  - PARP1
  - PARP10
  - ATM
  - TNKS
  - PARP9
  - PRKAG3
  - PARP15
  - PARP8
  - PARP2
  - PARP6
  - TSC2
  - PRKAG2
  - PRKAG1
  - PARP14
  - PRKAA1
  - PARP12
  - PRKAB2
  - PARP11
  - PRKAB1
  - PARP16
  - PRKAA2
genes:
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP3
  entrez: '10039'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TIPARP
  entrez: '25976'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP4
  entrez: '143'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS2
  entrez: '80351'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP1
  entrez: '142'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP10
  entrez: '84875'
- word: ATA
  symbol: ATA
  source: hgnc_prev_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS
  entrez: '8658'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP9
  entrez: '83666'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG3
  entrez: '53632'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP15
  entrez: '165631'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP8
  entrez: '79668'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP2
  entrez: '10038'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP6
  entrez: '56965'
- word: TSC2
  symbol: TSC2
  source: hgnc_symbol
  hgnc_symbol: TSC2
  entrez: '7249'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG2
  entrez: '51422'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG1
  entrez: '5571'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP14
  entrez: '54625'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA1
  entrez: '5562'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP12
  entrez: '64761'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB2
  entrez: '5565'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP11
  entrez: '57097'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB1
  entrez: '5564'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP16
  entrez: '54956'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA2
  entrez: '5563'
chemicals: []
diseases: []
figid_alias: PMC5572252__F1
redirect_from: /figures/PMC5572252__F1
figtype: Figure
---
