---
figid: PMC523366__pp1363043f05
figtitle: Metabolic flux map of primary and intermediate metabolic pathways in developing
  soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln
organisms:
- Glycine max
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC523366
filename: pp1363043f05.jpg
figlink: /pmc/articles/PMC523366/figure/fig5/
number: F5
caption: 'Metabolic flux map of primary and intermediate metabolic pathways in developing
  soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln. Fluxes are proportional
  to arrow widths. (Fluxes less than 0.4 μmol d−1 cotyledon−1 are shown with lines
  of lowest visible thickness.) Arrows indicate the direction of net flux. A complete
  numerical listing of estimated flux values and reaction reversibilities is provided
  in . Intracellular metabolites are shown in white ovals, and gray ovals show sink
  metabolites (proteinogenic amino acids, polysaccharides, etc.). Metabolites taken
  up from/secreted into the medium are shown in blue ovals. Metabolic pathways are
  color coded as follows: dark red, glycolysis and Suc metabolism; pale blue, pentose
  phosphate pathway; orange, TCA cycle; blue-gray, pyruvate dehydrogenase link; mauve,
  anaplerotic fluxes; dark yellow, glyoxylate shunt; green, Glu metabolism, GABA shunt,
  and associated intercompartmental transport fluxes; gray, fluxes toward biomass
  synthesis; and black, all intercompartmental transport fluxes except those involved
  in Gln metabolism and GABA shunt. Abbreviations for intracellular metabolites not
  defined in the text are as follows: P5P, pentose-5-P; S7P, sedoheptulose-7-P; E4P,
  erythrose-4-P; acetCoA, acetyl CoA; iCit, isocitrate; aKG, α-ketoglutarate; Scn,
  succinate; SSA, succinic semi-aldehyde; and GOx, glyoxylate. Sink metabolite abbreviations
  are as follows: Psac, polysaccharides; Nuc, carbon skeleton of nucleotides; and
  Sta, starch. Asp and Asn are denoted together as Asx. Glu and Gln are denoted together
  as Glx. F6P and T3P appear at two different locations each in the cytosol and plastid,
  to avoid confusing intersections of lines. Each flux is assigned a short name based
  on the name of the gene encoding one of the metabolic reactions represented by it.
  Intracellular metabolites and fluxes with a superscript are located in specific
  subcellular compartments: c, cytosol; p, plastid; and m, mitochondrion. If a flux
  has no superscript, its compartmentation could not be unambiguously determined (such
  as gap, eno, and pyk, and some fluxes toward biosynthesis).'
papertitle: Quantification of Compartmented Metabolic Fluxes in Developing Soybean
  Embryos by Employing Biosynthetically Directed Fractional 13C Labeling, Two-Dimensional
  [13C, 1H] Nuclear Magnetic Resonance, and Comprehensive Isotopomer Balancing.
reftext: Ganesh Sriram, et al. Plant Physiol. 2004 Oct;136(2):3043-3057.
year: '2004'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9391505
figid_alias: PMC523366__F5
figtype: Figure
redirect_from: /figures/PMC523366__F5
ndex: e5e2bbc5-dee5-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC523366__pp1363043f05.html
  '@type': Dataset
  description: 'Metabolic flux map of primary and intermediate metabolic pathways
    in developing soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln. Fluxes
    are proportional to arrow widths. (Fluxes less than 0.4 μmol d−1 cotyledon−1 are
    shown with lines of lowest visible thickness.) Arrows indicate the direction of
    net flux. A complete numerical listing of estimated flux values and reaction reversibilities
    is provided in . Intracellular metabolites are shown in white ovals, and gray
    ovals show sink metabolites (proteinogenic amino acids, polysaccharides, etc.).
    Metabolites taken up from/secreted into the medium are shown in blue ovals. Metabolic
    pathways are color coded as follows: dark red, glycolysis and Suc metabolism;
    pale blue, pentose phosphate pathway; orange, TCA cycle; blue-gray, pyruvate dehydrogenase
    link; mauve, anaplerotic fluxes; dark yellow, glyoxylate shunt; green, Glu metabolism,
    GABA shunt, and associated intercompartmental transport fluxes; gray, fluxes toward
    biomass synthesis; and black, all intercompartmental transport fluxes except those
    involved in Gln metabolism and GABA shunt. Abbreviations for intracellular metabolites
    not defined in the text are as follows: P5P, pentose-5-P; S7P, sedoheptulose-7-P;
    E4P, erythrose-4-P; acetCoA, acetyl CoA; iCit, isocitrate; aKG, α-ketoglutarate;
    Scn, succinate; SSA, succinic semi-aldehyde; and GOx, glyoxylate. Sink metabolite
    abbreviations are as follows: Psac, polysaccharides; Nuc, carbon skeleton of nucleotides;
    and Sta, starch. Asp and Asn are denoted together as Asx. Glu and Gln are denoted
    together as Glx. F6P and T3P appear at two different locations each in the cytosol
    and plastid, to avoid confusing intersections of lines. Each flux is assigned
    a short name based on the name of the gene encoding one of the metabolic reactions
    represented by it. Intracellular metabolites and fluxes with a superscript are
    located in specific subcellular compartments: c, cytosol; p, plastid; and m, mitochondrion.
    If a flux has no superscript, its compartmentation could not be unambiguously
    determined (such as gap, eno, and pyk, and some fluxes toward biosynthesis).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Psa
  - sta
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - pot
  - Suc
  - Nuc
  - pk
  - Cys
  - mmy
  - Csp
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Blos2
  - Eno
  - Pep
  - bgcn
  - PyK
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - Me
  - Asx
  - ascutex
  - arg
  - aos
  - pyx
  - mah
  - mth
  - pyr
  - mal
  - bs
  - Mrtf
  - sah
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - Gdh
  - Gpdh1
  - Gld
  - Gat
  - .na.character
  - Pitslre
  - gate
  - SA
  - sa
  - GABA-B-R1
  - Rdl
  - Gabat
  - KLK3
  - PLAG1
  - PROS1
  - NPEPPS
  - PSAT1
  - IL32
  - EMD
  - GCY
  - PRG4
  - TALDO1
  - LRSAM1
  - TAAR1
  - NUCB1
  - RASA1
  - RGS6
  - BLOC1S2
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - PAEP
  - PREP
  - PTPN22
  - PHKA2
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - ABL2
  - RERE
  - MAL
  - MRTFA
  - TIRAP
  - PNMA1
  - GSTZ1
  - PACRG
  - ACSM3
  - SARDH
  - SDHB
  - SORD
  - SDS
  - BIRC3
  - GLUD1
  - UGDH
  - H6PD
  - CRYL1
  - GLYAT
  - GBA1
  - GBA1LP
  - GBA3
  - Ala
  - Arg
  - Cys
  - fructose F6PEso
  - GABA
  - Leu
  - Lys
  - Met
  - Phe
  - Ser
  - Thr
  - Trp
  - Tyr
  - Val
---
