---
figid: PMC5541666__12974_2017_928_Fig3_HTML
figtitle: Systematic analysis of key factors in the mitochondria-associated innate
  immunity pathway, using acute starvation of human SH-SY5Y neuroblastoma cells to
  reveal the transcriptional regulation of stress responses and their dependence on
  PINK1
organisms:
- Hepatitis B virus
- Human betaherpesvirus 5
- DNA viruses
- Enterovirus
- Mus musculus
- Rattus norvegicus
- Drosophila melanogaster
- Caenorhabditis elegans
- Homo sapiens
- Gallus gallus
- Ovis aries
- prion
- Sisyrinchium striatum
pmcid: PMC5541666
filename: 12974_2017_928_Fig3_HTML.jpg
figlink: /pmc/articles/PMC5541666/figure/Fig3/
number: F3
caption: 'Systematic analysis of key factors in the mitochondria-associated innate
  immunity pathway, using acute starvation of human SH-SY5Y neuroblastoma cells to
  reveal the transcriptional regulation of stress responses and their dependence on
  PINK1. Transcript levels were documented in control non-target knock-down (NT-KD)
  versus PINK1-knock-down (PINK1-KD) cells for the following inflammatory factors.
  a Summary scheme on the detection of pathogen DNA/RNA in the cytosol and the mitochondria-associated
  triggering of innate immunity, as altered during starvation by PINK1. Viral RNA
  (ssRNA or dsRNA) is recognized in the cytosol by helicase RIG-I and MDA5. This triggers
  the dimerization of MAVS in the outer mitochondrial membrane (OMM) which recruits
  transactivators (such as TRADD, TRAF, IKK family) leading to nuclear translocation
  of phosphorylated IRF3, IRF7, and NF-κB and promoting the transcription of IFN stimulated
  genes and pro-inflammatory cytokines, respectively. Effector molecules such as IFIT1,
  IFIT2, IFIT3, and RSAD2 (viperin) inhibit virus DNA/RNA replication. IFIT3 also
  functions as a scaffold to facilitate interaction between MAVS and TBK1 and represents
  a positive feedback of DDX58 (RIG-I) signaling through MAVS. Low ΔΨm or decreased
  ROS inhibit MAVS-mediated signaling. Mitochondria cooperate with the endoplasmic
  reticulum (ER) to regulate lipid synthesis and antiviral signaling at the mitochondria-associated
  membranes (MAM), possibly by interactions of MAVS/MFN1 with TMEM173 (STING). MFN1
  leads to the redistribution of MAVS along mitochondria and a fusion of the mitochondrial
  network that promotes the interaction between MAVS and STING. Low energy induces
  the localization of PINK1 to the OMM and recruitment of PARKIN from the cytosol,
  which is the signal for dysfunctional mitochondria to be digested in the autophagosome.
  Abbreviations: 5′ ppp 5′ triphosphate, ΔΨm mitochondrial membrane potential, IκB
  inhibitor of κ light polypeptide gene enhancer in B cells, IKK IκB Kinase, LC3II
  phosphatidylethanolamine conjugate of the autophagy-related protein LC3 (MAP1LC3),
  MDA5 melanoma differentiation-associated gene 5, NF-κB nuclear factor κB, p62 sequestosome-1,
  adaptor between polyubiquitinated substrates and autophagic machinery, PARKIN ubiquitin
  ligase, its loss-of-function leads to the PARK2 variant of Parkinson’s disease,
  RIG-1 (DDX58) retinoic acid-inducible gene-1 (DExD/H-Box RNA Helicase 58), ROS reactive
  oxygen species, ssRNA/dsRNA single-stranded/double-stranded RNA, TMEM173 (STING)
  transmembrane Protein 173, TRADD tumor necrosis factor receptor type 1 associated
  death domain protein, TRAF TNF-receptor-associated factor. b PINK1 (PTEN induced
  kinase 1) as a known determinant of selective mitophagy and autosomal recessive
  Parkinson’s disease; HPRT1 (Hypoxanthine Phosphoribosyltransferase 1) as a loading
  control. c RSAD2 (= viperin, radical S-adenosyl methionine domain containing 2)
  as an interferon-inducible lipid-droplet associated virus inhibitory factor. d HEBP1
  (heme binding protein 1) that contains a natural ligand for formyl peptide receptor-like
  receptor 2. e TBK1 (TANK-binding kinase 1) that phosphorylates interferon regulatory
  factors in response to toll-like receptor activation. f IRF3 (interferon regulatory
  factor 3) as a regulator of type I Interferon gene transcription. g MFN1 (mitofusin
  1) as PARKIN-dependent factor in mitochondrial dynamics and mitochondria-associated
  anti-microbial signaling. h IFIT3 (interferon-induced protein with tetratricopeptide
  repeats 3) as a detector of pathogen DNA/RNA. i IFIT1 (interferon-induced protein
  with tetratricopeptide repeats 3) as a detector of pathogen DNA/RNA. j MAVS (mitochondria-associated
  viral sensor) as inducer of interferon-dependent long-term expression of defense
  factors. k–m The levels of known autophagy factors during this starvation time course
  exhibited progressive consumption of p62 in spite of its transcriptional induction,
  in parallel to a PINK1-modulated transcriptional induction of LRRK2. Four independent
  experiments compared their expression during a nutrient and trophic deprivation
  time course triggered by a culture switch from RPMI growth medium to HBSS starvation
  medium. The bar graphs show mean and standard error of the mean, illustrating the
  significance with asterisks (*p < 0.05, **p < 0.01, ***p < 0.001, ****p < 0.0001)'
papertitle: Progression of pathology in PINK1-deficient mouse brain from splicing
  via ubiquitination, ER stress, and mitophagy changes to neuroinflammation.
reftext: Sylvia Torres-Odio, et al. J Neuroinflammation. 2017;14:154.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5945432
figid_alias: PMC5541666__F3
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Ovis aries
- Gallus gallus
- Drosophila melanogaster
redirect_from: /figures/PMC5541666__F3
ndex: 1c576c59-de9b-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5541666__12974_2017_928_Fig3_HTML.html
  '@type': Dataset
  description: 'Systematic analysis of key factors in the mitochondria-associated
    innate immunity pathway, using acute starvation of human SH-SY5Y neuroblastoma
    cells to reveal the transcriptional regulation of stress responses and their dependence
    on PINK1. Transcript levels were documented in control non-target knock-down (NT-KD)
    versus PINK1-knock-down (PINK1-KD) cells for the following inflammatory factors.
    a Summary scheme on the detection of pathogen DNA/RNA in the cytosol and the mitochondria-associated
    triggering of innate immunity, as altered during starvation by PINK1. Viral RNA
    (ssRNA or dsRNA) is recognized in the cytosol by helicase RIG-I and MDA5. This
    triggers the dimerization of MAVS in the outer mitochondrial membrane (OMM) which
    recruits transactivators (such as TRADD, TRAF, IKK family) leading to nuclear
    translocation of phosphorylated IRF3, IRF7, and NF-κB and promoting the transcription
    of IFN stimulated genes and pro-inflammatory cytokines, respectively. Effector
    molecules such as IFIT1, IFIT2, IFIT3, and RSAD2 (viperin) inhibit virus DNA/RNA
    replication. IFIT3 also functions as a scaffold to facilitate interaction between
    MAVS and TBK1 and represents a positive feedback of DDX58 (RIG-I) signaling through
    MAVS. Low ΔΨm or decreased ROS inhibit MAVS-mediated signaling. Mitochondria cooperate
    with the endoplasmic reticulum (ER) to regulate lipid synthesis and antiviral
    signaling at the mitochondria-associated membranes (MAM), possibly by interactions
    of MAVS/MFN1 with TMEM173 (STING). MFN1 leads to the redistribution of MAVS along
    mitochondria and a fusion of the mitochondrial network that promotes the interaction
    between MAVS and STING. Low energy induces the localization of PINK1 to the OMM
    and recruitment of PARKIN from the cytosol, which is the signal for dysfunctional
    mitochondria to be digested in the autophagosome. Abbreviations: 5′ ppp 5′ triphosphate,
    ΔΨm mitochondrial membrane potential, IκB inhibitor of κ light polypeptide gene
    enhancer in B cells, IKK IκB Kinase, LC3II phosphatidylethanolamine conjugate
    of the autophagy-related protein LC3 (MAP1LC3), MDA5 melanoma differentiation-associated
    gene 5, NF-κB nuclear factor κB, p62 sequestosome-1, adaptor between polyubiquitinated
    substrates and autophagic machinery, PARKIN ubiquitin ligase, its loss-of-function
    leads to the PARK2 variant of Parkinson’s disease, RIG-1 (DDX58) retinoic acid-inducible
    gene-1 (DExD/H-Box RNA Helicase 58), ROS reactive oxygen species, ssRNA/dsRNA
    single-stranded/double-stranded RNA, TMEM173 (STING) transmembrane Protein 173,
    TRADD tumor necrosis factor receptor type 1 associated death domain protein, TRAF
    TNF-receptor-associated factor. b PINK1 (PTEN induced kinase 1) as a known determinant
    of selective mitophagy and autosomal recessive Parkinson’s disease; HPRT1 (Hypoxanthine
    Phosphoribosyltransferase 1) as a loading control. c RSAD2 (= viperin, radical
    S-adenosyl methionine domain containing 2) as an interferon-inducible lipid-droplet
    associated virus inhibitory factor. d HEBP1 (heme binding protein 1) that contains
    a natural ligand for formyl peptide receptor-like receptor 2. e TBK1 (TANK-binding
    kinase 1) that phosphorylates interferon regulatory factors in response to toll-like
    receptor activation. f IRF3 (interferon regulatory factor 3) as a regulator of
    type I Interferon gene transcription. g MFN1 (mitofusin 1) as PARKIN-dependent
    factor in mitochondrial dynamics and mitochondria-associated anti-microbial signaling.
    h IFIT3 (interferon-induced protein with tetratricopeptide repeats 3) as a detector
    of pathogen DNA/RNA. i IFIT1 (interferon-induced protein with tetratricopeptide
    repeats 3) as a detector of pathogen DNA/RNA. j MAVS (mitochondria-associated
    viral sensor) as inducer of interferon-dependent long-term expression of defense
    factors. k–m The levels of known autophagy factors during this starvation time
    course exhibited progressive consumption of p62 in spite of its transcriptional
    induction, in parallel to a PINK1-modulated transcriptional induction of LRRK2.
    Four independent experiments compared their expression during a nutrient and trophic
    deprivation time course triggered by a culture switch from RPMI growth medium
    to HBSS starvation medium. The bar graphs show mean and standard error of the
    mean, illustrating the significance with asterisks (*p < 0.05, **p < 0.01, ***p < 0.001,
    ****p < 0.0001)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GTF2H1
  - SQSTM1
  - KHDRBS1
  - NUP62
  - DCTN4
  - PINK1
  - MAVS
  - RIGI
  - IFIH1
  - SGCG
  - STING1
  - TRADD
  - CHUK
  - IKBKB
  - IKBKG
  - IKBKE
  - TBK1
  - TRAF1
  - TRAF2
  - TRAF3
  - TRAF4
  - TRAF5
  - TRAF6
  - TRAF7
  - MFN1
  - IFIT3
  - IFIT1
  - IFIT2
  - NFKB1
  - IRF3
  - IRF7
  - RSAD2
  - IFNK
  - IFNW1
  - IFNA1
  - IFNA2
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA17
  - IFNA21
  - IFNB1
  - TRIM63
  - ACTG2
  - ME1
  - MKS1
  - .na.character
  - Gtf2h1
  - Nup62
  - Sqstm1
  - Khdrbs1
  - Dctn4
  - Pink1
  - Mavs
  - Rigi
  - Ifih1
  - Pzp
  - Sting1
  - Tradd
  - Mfn1
  - Ifit3
  - Ifit1
  - Ifit2
  - Nfkbib
  - Nfkb1
  - Irf3
  - Irf7
  - Rsad2
  - Ptch1
  - A2m
  - NFKB2
  - PPY
  - TMEM173
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Low
  - park
  - ref(2)P
  - DCTN4-p62
  - Tfb1
  - Taf6
  - how
  - amon
  - mam
  - Sting
  - aub
  - IKKepsilon
  - key
  - IKKbeta
  - Traf6
  - Traf4
  - fzo
  - Dif
  - dl
  - Rel
  - Irp-1B
  - Irp-1A
  - as
---
