---
figid: PMC5076483__fig-4
figtitle: Disturbed Flow-Induced Endothelial Proatherogenic Signaling Via Regulating
  Post-Translational Modifications and Epigenetic Events
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC5076483
filename: fig-4.jpg
figlink: /pmc/articles/PMC5076483/figure/f4/
number: F4
caption: DNA methylation/demethylation enzymes. Methylation of the promoter regions
  of genes significantly suppresses transcription by direct inhibition of transcription
  factor binding and recruitment of methyl-CpG-binding proteins within their recognition
  sites of transcription factors. DNA methylation occurs at carbon 5 of cytosine (5-methylcytosine
  [5mC]) in CpG dinucleotides. DNMT1 maintains DNA methylation patterns during cell
  proliferation via methylation of a hemimethylated nascent DNA strand. DNMT3A and
  DNMT3B are required for genome-wide de novo methylation and play crucial roles in
  the establishment of DNA methylation patterns. Methylation by DNMTs is counterbalanced
  by DNA demethylation. TET oxidizes 5mC to 5-hydroxymethylcytosine (5hmC) and subsequently
  to 5-formyl cytosine (5fC) and 5-carboxy cytosine (5caC). The carboxyl group of
  5caC is excised by TDG to restore cytosine. An active demethylation pathway through
  consecutive oxidation of 5-methylcytosine (5mC) mediated by TET proteins and subsequent
  BER in mammalian system DNA methylation dynamics. BER, base excision repair; CpG,
  cytosine-phosphate-guanine; DNMT1, DNA (cytosine-5-)-methyltransferase 1; TDG, thymine
  DNA glycolase; TET, ten-eleven translocation. To see this illustration in color,
  the reader is referred to the web version of this article at www.liebertpub.com/ars
papertitle: Disturbed Flow-Induced Endothelial Proatherogenic Signaling Via Regulating
  Post-Translational Modifications and Epigenetic Events.
reftext: Kyung-Sun Heo, et al. Antioxid Redox Signal. 2016 Sep 1;25(7):435-450.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8883849
figid_alias: PMC5076483__F4
figtype: Figure
redirect_from: /figures/PMC5076483__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5076483__fig-4.html
  '@type': Dataset
  description: DNA methylation/demethylation enzymes. Methylation of the promoter
    regions of genes significantly suppresses transcription by direct inhibition of
    transcription factor binding and recruitment of methyl-CpG-binding proteins within
    their recognition sites of transcription factors. DNA methylation occurs at carbon
    5 of cytosine (5-methylcytosine [5mC]) in CpG dinucleotides. DNMT1 maintains DNA
    methylation patterns during cell proliferation via methylation of a hemimethylated
    nascent DNA strand. DNMT3A and DNMT3B are required for genome-wide de novo methylation
    and play crucial roles in the establishment of DNA methylation patterns. Methylation
    by DNMTs is counterbalanced by DNA demethylation. TET oxidizes 5mC to 5-hydroxymethylcytosine
    (5hmC) and subsequently to 5-formyl cytosine (5fC) and 5-carboxy cytosine (5caC).
    The carboxyl group of 5caC is excised by TDG to restore cytosine. An active demethylation
    pathway through consecutive oxidation of 5-methylcytosine (5mC) mediated by TET
    proteins and subsequent BER in mammalian system DNA methylation dynamics. BER,
    base excision repair; CpG, cytosine-phosphate-guanine; DNMT1, DNA (cytosine-5-)-methyltransferase
    1; TDG, thymine DNA glycolase; TET, ten-eleven translocation. To see this illustration
    in color, the reader is referred to the web version of this article at www.liebertpub.com/ars
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DNMT1
  - DNMT3B
  - DNMT3A
  - TDG
---
