---
figid: PMC5037390__srep33956-f6
figtitle: Hypothetical mechanistic network derived from predicted upstream regulators
  for genes found to be differentially expression in chondrocytes in transition from
  monolayer to alginate bead cultures (redifferentiation)
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC5037390
filename: srep33956-f6.jpg
figlink: /pmc/articles/PMC5037390/figure/f6/
number: F6
caption: (a) Hypothetical mechanistic network derived from predicted upstream regulators
  for genes found to be differentially expression in chondrocytes in transition from
  monolayer to alginate bead cultures (redifferentiation). Genes showing lower expression
  in monolayer relative to alginate beads (Hif1a, Nfe2l2, Atf3) are coloured green
  (i.e. indicating higher expression in alginate cultures); genes in orange are predicted.
  Genes more highly expressed in monolayer are not shown (absence of red-coloured
  genes). In monolayer chondrocytes PDGF BB mediated effects are predicted to be inactive
  for the observed expression profile, i.e. activated in alginate cultures. Ingenuity®
  canonical pathways significantly enriched for this expression profile included ‘NRF2-mediated
  oxidative stress’, ‘PI3K/AKT signalling’ and ‘the role of osteoblasts, osteoclasts
  and chondrocytes in rheumatoid arthritis’. The physiological function terms ‘differentiation
  of cells’ (p = 1.6e-14) and ‘apoptosis’ (p = 4.9e-12) were highly enriched in this
  subnetwork. Nodes with a pink border were enriched for ‘condensation of cartilage
  tissue’, p = 2.07e-7. The small molecule inhibitor of the PI3K/AKT pathway ‘LY294004’
  was predicted to be activated based upon the supplied gene expression profile (z-score = 4.58,
  overlap p-value = 3.3e-15, 219 gene interactions). This mechanistic network indicates
  that PI3K signalling and PDGF BB are likely to be active in the redifferentiation
  phenotype in alginate cultures; the PI3K inhibitor LY294004 describes the inverse
  for monolayer cells. (b) Using the same elements a protein-protein association network
  consisting of nodes (proteins) and edges (evidence of associations) indicates a
  shared function between selected nodes, but not necessarily physical interactions.
  Sources for associations are defined in the network legend. Elements determined
  to influence differentiation status of chondrocytes and tenocytes in culture are
  shown to have significant enrichment for protein-protein interactions (p < 0.0001)
  indicating that as a group they are biologically connected. Some elements did not
  demonstrate evidence of association in this network (GPNMB, CHI3L).
papertitle: A systems biology approach to defining regulatory mechanisms for cartilage
  and tendon cell phenotypes.
reftext: A. J. Mueller, et al. Sci Rep. 2016;6:33956.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.715465
figid_alias: PMC5037390__F6
figtype: Figure
redirect_from: /figures/PMC5037390__F6
ndex: 0c9d0fbf-df32-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5037390__srep33956-f6.html
  '@type': Dataset
  description: (a) Hypothetical mechanistic network derived from predicted upstream
    regulators for genes found to be differentially expression in chondrocytes in
    transition from monolayer to alginate bead cultures (redifferentiation). Genes
    showing lower expression in monolayer relative to alginate beads (Hif1a, Nfe2l2,
    Atf3) are coloured green (i.e. indicating higher expression in alginate cultures);
    genes in orange are predicted. Genes more highly expressed in monolayer are not
    shown (absence of red-coloured genes). In monolayer chondrocytes PDGF BB mediated
    effects are predicted to be inactive for the observed expression profile, i.e.
    activated in alginate cultures. Ingenuity® canonical pathways significantly enriched
    for this expression profile included ‘NRF2-mediated oxidative stress’, ‘PI3K/AKT
    signalling’ and ‘the role of osteoblasts, osteoclasts and chondrocytes in rheumatoid
    arthritis’. The physiological function terms ‘differentiation of cells’ (p = 1.6e-14)
    and ‘apoptosis’ (p = 4.9e-12) were highly enriched in this subnetwork. Nodes with
    a pink border were enriched for ‘condensation of cartilage tissue’, p = 2.07e-7.
    The small molecule inhibitor of the PI3K/AKT pathway ‘LY294004’ was predicted
    to be activated based upon the supplied gene expression profile (z-score = 4.58,
    overlap p-value = 3.3e-15, 219 gene interactions). This mechanistic network indicates
    that PI3K signalling and PDGF BB are likely to be active in the redifferentiation
    phenotype in alginate cultures; the PI3K inhibitor LY294004 describes the inverse
    for monolayer cells. (b) Using the same elements a protein-protein association
    network consisting of nodes (proteins) and edges (evidence of associations) indicates
    a shared function between selected nodes, but not necessarily physical interactions.
    Sources for associations are defined in the network legend. Elements determined
    to influence differentiation status of chondrocytes and tenocytes in culture are
    shown to have significant enrichment for protein-protein interactions (p < 0.0001)
    indicating that as a group they are biologically connected. Some elements did
    not demonstrate evidence of association in this network (GPNMB, CHI3L).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FUT3
  - PDGFB
  - PDGFA
  - PDGFC
  - PDGFD
  - TGFB2
  - APOE
  - JUN
  - JUNB
  - JUND
  - ATM
  - ATF4
  - NFE2L2
  - NFKBIA
  - SOX9
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - ANGPT1
  - ANGPT2
  - ANGPT4
  - AREG
  - BDNF
  - CSF1
  - EFNA1
  - EFNA2
  - EFNA3
  - EFNA4
  - EFNA5
  - EGF
  - EREG
  - FGF1
  - FGF10
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF2
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FLT3LG
  - HGF
  - IGF1
  - IGF2
  - INS
  - KITLG
  - NGF
  - NTF3
  - NTF4
  - PGF
  - TGFA
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - OMA1
  - CXCL2
  - BHLHE40
  - EGR1
  - ATF3
  - COL2A1
  - HIF1A
  - GPNMB
---
