---
figid: PMC5662193__nihms912225f6
figtitle: Cytotoxic Vibrio T3SS1 Rewires Host Gene Expression to Subvert Cell Death
  Signaling and Activate Cell Survival Networks
organisms:
- Escherichia coli
- Pseudomonas aeruginosa
- Helicobacter pylori
- Vibrio parahaemolyticus
- Clostridium botulinum
- Vibrio vulnificus
- Helicobacter pylori SS1
- Bos taurus
- Oryctolagus cuniculus
- Equus asinus
- Saccharomyces cerevisiae
- Sequoia sempervirens
- Homo sapiens
- Mus musculus
- uncultured marine bacterium
pmcid: PMC5662193
filename: nihms912225f6.jpg
figlink: /pmc/articles/PMC5662193/figure/F6/
number: F6
caption: Illustration of the action key POR3-differentially expressed genes in the
  cell survival network generated using Ingenuity®Pathway Analysis. The subcellular
  localization of the protein products of each gene and differential expression status
  (yellow, increased; blue, decreased) are depicted. Dotted lines denote indirect
  interactions. Orange lines indicate a gene expression state consistent with activation
  of cell survival; yellow lines indicate an expression state inconsistent with cell
  survival activation.
papertitle: Cytotoxic Vibrio T3SS1 Rewires Host Gene Expression to Subvert Cell Death
  Signaling and Activate Cell Survival Networks.
reftext: Nicole J. De Nisco, et al. Sci Signal. ;10(479):eaal4501.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9361365
figid_alias: PMC5662193__F6
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
- Bos taurus
- Oryctolagus cuniculus
- Equus asinus
redirect_from: /figures/PMC5662193__F6
ndex: 99e43081-deef-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5662193__nihms912225f6.html
  '@type': Dataset
  description: Illustration of the action key POR3-differentially expressed genes
    in the cell survival network generated using Ingenuity®Pathway Analysis. The subcellular
    localization of the protein products of each gene and differential expression
    status (yellow, increased; blue, decreased) are depicted. Dotted lines denote
    indirect interactions. Orange lines indicate a gene expression state consistent
    with activation of cell survival; yellow lines indicate an expression state inconsistent
    with cell survival activation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BMP4
  - ADAMTS1
  - CXCL8
  - CXCL1
  - CXCL2
  - PLAU
  - CCL2
  - IL6
  - FGF18
  - STC1
  - CSF2
  - CSF3
  - TNFSF9
  - LIF
  - IL21R
  - AMIGO2
  - THBD
  - ICAM1
  - KIF1C
  - NFKBIA
  - HSPB8
  - BCL2L2
  - SGK1
  - INSIG1
  - RFK
  - C2
  - SOD2
  - SOD2-OT1
  - XIAP
  - C1QA
  - C1QB
  - C1QC
  - C1R
  - C1S
  - F3
  - SOCS1
  - PTGS2
  - IER3
  - PRDM1
  - ZFP36
  - AKT1
  - PTK2B
  - PMAIP1
  - RCAN1
  - CEBPD
  - IKZF2
  - CEBPB
  - ID2
  - NR4A1
  - KLF6
  - ATF3
  - TSC22D3
  - NR4A2
  - DUSP1
  - DUSP12
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - JUND
  - NR4A3
  - DUSP10
  - BHLHE40
  - KLF4
  - KLF2
  - JUN
  - JUNB
  - SMAD6
  - CDKN1B
  - ZNF100
  - KLF5
  - PLK2
  - FUT3
  - TWIST1
  - REST
  - NFIL3
  - EGR1
  - DESI1
  - SLC35G1
  - Bmp4
  - Adamts1
  - Adamts5
  - Cxcl1
  - Cxcl2
  - Plau
  - Ccl2
  - Il6
  - Fgf18
  - Stc1
  - Csf2
  - Csf3
  - Tnfsf9
  - Lif
  - Lifr
  - Il21r
  - Amigo2
  - Thbd
  - Icam1
  - Kif1c
  - Nfkbia
  - Hspb8
  - Bcl2l2
  - Sgk1
  - Insig1
  - Rfk
  - Sod2
  - Xiap
  - Pfdn4
  - Oog1
  - Socs1
  - Ptgs2
  - Ier3
  - Cor
  - Prdm1
  - Atp6ap1
  - Zfp36
  - Rnf141
  - Akt1
  - Akt2
  - Pmaip1
  - Rcan1
  - Cebpd
  - Ikzf2
  - Cebpb
  - Id2
  - Nr4a1
  - Klf6
  - Atf3
  - Tsc22d3
  - Nr4a2
  - Dusp1
  - Fos
  - Jund
  - Nr4a3
  - Dusp10
  - Bhlhe40
  - Klf4
  - Klf2
  - Jun
  - Smad6
  - Cdkn1b
  - Klf5
  - Plk2
  - Twist1
  - Rest
  - Nfil3
  - Egr1
  - Minute
  - GRO1
  - RESTB
  - ICAM-1
  - SOD-2
---
