---
figid: PMC5487774__jvms-79-1003-g002
figtitle: Subcellular location of genes in the canonical pathway, termed as “Signaling
  by Rho Family GTPases”, as determined via IPA analysis to compare transcriptomes
  in the APs of pre- and post-ovulation heifers
organisms:
- Mus musculus
- Rattus norvegicus
- Danio rerio
- Cricetulus griseus
- Carassius auratus
- Anas platyrhynchos
- Homo sapiens
- Bos taurus
- Caenorhabditis elegans
- Lolium multiflorum
pmcid: PMC5487774
filename: jvms-79-1003-g002.jpg
figlink: /pmc/articles/PMC5487774/figure/fig_002/
number: F2
caption: The subcellular location of genes in the canonical pathway, termed as “Signaling
  by Rho Family GTPases”, as determined via IPA analysis to compare transcriptomes
  in the APs of pre- and post-ovulation heifers. Note that the IPA software shows
  gene names as normal font and not italics. The solid lines and dotted line indicate
  direct or indirect biological relationship between genes, respectively. The bold
  grey lines indicate the border between the extracellular space, cytoplasm, and nucleus.
  The purple indicates the upregulated genes identified in this experiment. This figure
  includes the results of molecule activity predictor (MAP), and the predicted stimulated
  genes are shown in orange, and the predicted inhibited genes are shown in blue.
  In total, 236 genes constituted the canonical pathway, and 9 of these genes (ACTA2,
  CDH10, ELK1, GNA14, LIMK1, LIMK2, RHOC, SEPT5, and WIPF1) were identified in the
  bovine APs used in this study. In this figure, F-actin represents ACTA2, Cadherin
  represents CDH10, Gα represents GNA14, LIMK represents both LIMK1 and LIMK2, Rho
  represents RHOC, Septin5 represents SEPT5, and WIP represents WIPF1.
papertitle: Deep sequencing of the transcriptome in the anterior pituitary of heifers
  before and after ovulation.
reftext: Kiran PANDEY, et al. J Vet Med Sci. 2017 Jun;79(6):1003-1012.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.893188
figid_alias: PMC5487774__F2
figtype: Figure
organisms_ner:
- Bos taurus
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Cricetulus griseus
- Anas platyrhynchos
- Danio rerio
- Carassius auratus
redirect_from: /figures/PMC5487774__F2
ndex: a748b4f8-deb8-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5487774__jvms-79-1003-g002.html
  '@type': Dataset
  description: The subcellular location of genes in the canonical pathway, termed
    as “Signaling by Rho Family GTPases”, as determined via IPA analysis to compare
    transcriptomes in the APs of pre- and post-ovulation heifers. Note that the IPA
    software shows gene names as normal font and not italics. The solid lines and
    dotted line indicate direct or indirect biological relationship between genes,
    respectively. The bold grey lines indicate the border between the extracellular
    space, cytoplasm, and nucleus. The purple indicates the upregulated genes identified
    in this experiment. This figure includes the results of molecule activity predictor
    (MAP), and the predicted stimulated genes are shown in orange, and the predicted
    inhibited genes are shown in blue. In total, 236 genes constituted the canonical
    pathway, and 9 of these genes (ACTA2, CDH10, ELK1, GNA14, LIMK1, LIMK2, RHOC,
    SEPT5, and WIPF1) were identified in the bovine APs used in this study. In this
    figure, F-actin represents ACTA2, Cadherin represents CDH10, Gα represents GNA14,
    LIMK represents both LIMK1 and LIMK2, Rho represents RHOC, Septin5 represents
    SEPT5, and WIP represents WIPF1.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AR
  - GFAP
  - MLC
  - FOS
  - JUN
  - Gpbar1
  - Gy
  - Ar
  - Gm45924
  - Etv5
  - Wasl
  - Was
  - Zhx2
  - Musk
  - Trp53
  - Decr1
  - Gfap
  - Cd74
  - Pomc
  - Wasf1
  - Arp
  - Rere
  - Ephb2
  - Mapk1
  - Mapk8
  - Mlc1
  - Jun
  - Fos
  - Kcnh8
  - Ephb1
  - FZD4
  - NMUR1
  - LPAR3
  - LGR6
  - MRGPRX3
  - MRGPRX4
  - GPR151
  - OXER1
  - GPRC6A
  - MRGPRX1
  - VN1R17P
  - GPR166P
  - GRM1
  - GRM2
  - GRM3
  - GRM4
  - GRM5
  - GRM6
  - GRM7
  - GRM8
  - CASR
  - GABBR1
  - GABBR2
  - GPR156
  - GPR158
  - GPR179
  - GPRC5A
  - GPRC5B
  - GPRC5C
  - GPRC5D
  - PARP1
  - EPRS1
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - ETV5
  - EZR
  - RDX
  - MSN
  - WASL
  - WAS
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - MAP3K13
  - IARS1
  - IRS1
  - IRS2
  - IRS4
  - TP53
  - TP63
  - TP73
  - DECR1
  - CD74
  - POMC
  - CLIP1
  - CLIP2
  - FUT3
  - WASF1
  - CRISP1
  - RERE
  - MANF
  - ARFRP1
  - EPHB2
  - MAPK1
  - MAPK3
  - MAPK8
  - MAPK9
  - MAPK10
  - MLC1
  - FOSB
  - JUNB
  - JUND
  - FOSL1
  - FOSL2
  - EPHB1
  - KCNH8
  - OGDH
  - Tas2r134
  - Rxfp2
  - Prkcg
  - Tp53
  - Smr3b
  - kita
  - ngfra
  - ar
  - etv5b
  - tp53
  - gfap
  - rplp0
  - mapk8b
  - fosab
  - jun
  - pomca
  - Cancer
---
