---
figid: PMC520969__tpc1602749f07
figtitle: Molecular Phenotyping of the pal1 and pal2 Mutants of Arabidopsis thaliana
  Reveals Far-Reaching Consequences on Phenylpropanoid, Amino Acid, and Carbohydrate
  Metabolism
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
pmcid: PMC520969
filename: tpc1602749f07.jpg
figlink: /pmc/articles/PMC520969/figure/fig7/
number: F7
caption: 'Integration of the Metabolic Pathways Affected by a Decreased Carbon Flux
  into the Phenylpropanoid Pathway, as Evidenced by Molecular Alterations in pal Mutants.Results
  of the transcriptome analyses (RT-PCR, cDNA-AFLP, and microarray) and the biochemical
  analyses (extractable phenolics, amino acids, and lignin) are incorporated. A dashed
  line indicates transport into another compartment; right and left harpoons mark
  a reversible enzyme reaction and arrows an irreversible enzyme reaction, respectively;
  question mark denotes a not fully established path. Arrows pointed up or down indicate
  the upregulation or downregulation of the gene or metabolite, respectively. The
  following color code is used: red, transcripts and metabolites linked with the pal1
  but not the pal2 mutation; green, transcripts and metabolites linked with the pal2
  but not the pal1 mutation; orange, transcripts and metabolites linked with the pal1
  and pal2 mutations; orange with asterisk, transcripts and metabolites occurring
  only in the pal1 pal2 double mutant. Abbreviations are listed per pathway. Carbohydrate
  metabolism: 1,3-BisPGA, 1,3-bisphosphoglycerate; BXL, β-xylosidase; CESA, cellulose
  synthase; DHAP, dihydroxyacetone phosphate; Ery4P, erythrose 4-phosphate; F1,6-BisP,
  fructose 1,6-bisphosphate; F6P, fructose 6-phosphate; G1P, glucose 1-phosphate;
  G6P, glucose 6-phosphate; GAP, glyceraldehyde 3-phosphate; INV, invertase; PEP,
  phospho-enol-pyruvate; 2-PGA, 2-phosphoglycerate; 3-PGA, 3-phosphoglycerate; 3-PGAkin,
  3-phosphoglycerate kinase; R1,5-BisP, ribulose 1,5-bisphosphate; RubAct, Rubisco
  activase; StP, starch phosphorylase; StS, starch synthase; TK, transketolase; TPI,
  triose-phosphate isomerase; TPS, trehalose 6-phosphate synthase. Jasmonate biosynthesis:
  AOS, allene oxide synthase; 3FAD, ω-3 fatty acid desaturase; JA, jasmonate. Methionine
  metabolism: CysS, cystathionine β-lyase (Cys synthase); MetS, Met synthase. Shikimate
  pathway: DAHP, 3-deoxy-d-arabino-heptulosonate 7-phosphate; Ery4P, erythrose 4-phosphate;
  4-HBA, para-hydroxybenzoate; 4-HBald, para-hydroxybenzaldehyde; PEP, phospho-enol-pyruvate;
  UbiRed, NADH-ubiquinone oxidoreductase. Recycling of NH4+1: ASN3, Asn synthase 3.
  Phenylpropanoid pathway: ATR3, NADPH-ferrihemoprotein reductase 3; CCoAOMT, caffeoyl-CoA
  3-O-methyltransferase; 4-HBA, para-hydroxybenzoate. Monolignol biosynthesis: G lignin,
  guaiacyl lignin monomers; S lignin, syringyl lignin monomers. Flavonoid biosynthesis:
  CHS, chalcone synthase.'
papertitle: Molecular Phenotyping of the pal1 and pal2 Mutants of Arabidopsis thaliana
  Reveals Far-Reaching Consequences on Phenylpropanoid, Amino Acid, and Carbohydrate
  Metabolism.
reftext: Antje Rohde, et al. Plant Cell. 2004 Oct;16(10):2749-2771.
year: '2004'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9331057
figid_alias: PMC520969__F7
figtype: Figure
redirect_from: /figures/PMC520969__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC520969__tpc1602749f07.html
  '@type': Dataset
  description: 'Integration of the Metabolic Pathways Affected by a Decreased Carbon
    Flux into the Phenylpropanoid Pathway, as Evidenced by Molecular Alterations in
    pal Mutants.Results of the transcriptome analyses (RT-PCR, cDNA-AFLP, and microarray)
    and the biochemical analyses (extractable phenolics, amino acids, and lignin)
    are incorporated. A dashed line indicates transport into another compartment;
    right and left harpoons mark a reversible enzyme reaction and arrows an irreversible
    enzyme reaction, respectively; question mark denotes a not fully established path.
    Arrows pointed up or down indicate the upregulation or downregulation of the gene
    or metabolite, respectively. The following color code is used: red, transcripts
    and metabolites linked with the pal1 but not the pal2 mutation; green, transcripts
    and metabolites linked with the pal2 but not the pal1 mutation; orange, transcripts
    and metabolites linked with the pal1 and pal2 mutations; orange with asterisk,
    transcripts and metabolites occurring only in the pal1 pal2 double mutant. Abbreviations
    are listed per pathway. Carbohydrate metabolism: 1,3-BisPGA, 1,3-bisphosphoglycerate;
    BXL, β-xylosidase; CESA, cellulose synthase; DHAP, dihydroxyacetone phosphate;
    Ery4P, erythrose 4-phosphate; F1,6-BisP, fructose 1,6-bisphosphate; F6P, fructose
    6-phosphate; G1P, glucose 1-phosphate; G6P, glucose 6-phosphate; GAP, glyceraldehyde
    3-phosphate; INV, invertase; PEP, phospho-enol-pyruvate; 2-PGA, 2-phosphoglycerate;
    3-PGA, 3-phosphoglycerate; 3-PGAkin, 3-phosphoglycerate kinase; R1,5-BisP, ribulose
    1,5-bisphosphate; RubAct, Rubisco activase; StP, starch phosphorylase; StS, starch
    synthase; TK, transketolase; TPI, triose-phosphate isomerase; TPS, trehalose 6-phosphate
    synthase. Jasmonate biosynthesis: AOS, allene oxide synthase; 3FAD, ω-3 fatty
    acid desaturase; JA, jasmonate. Methionine metabolism: CysS, cystathionine β-lyase
    (Cys synthase); MetS, Met synthase. Shikimate pathway: DAHP, 3-deoxy-d-arabino-heptulosonate
    7-phosphate; Ery4P, erythrose 4-phosphate; 4-HBA, para-hydroxybenzoate; 4-HBald,
    para-hydroxybenzaldehyde; PEP, phospho-enol-pyruvate; UbiRed, NADH-ubiquinone
    oxidoreductase. Recycling of NH4+1: ASN3, Asn synthase 3. Phenylpropanoid pathway:
    ATR3, NADPH-ferrihemoprotein reductase 3; CCoAOMT, caffeoyl-CoA 3-O-methyltransferase;
    4-HBA, para-hydroxybenzoate. Monolignol biosynthesis: G lignin, guaiacyl lignin
    monomers; S lignin, syringyl lignin monomers. Flavonoid biosynthesis: CHS, chalcone
    synthase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ubx
  - Tps1
  - G6P
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Aldh-III
  - Aos1
  - Tpi
  - Tk
  - sam
  - qkr54B
  - sls
  - Sam-S
  - Pep
  - bgcn
  - cmt
  - ppk
  - vvl
  - TPS1
  - TPI
  - PEP
  - PAS2
  - TSB1
  - CMT1
  - DFR
  - TPSAB1
  - TBX19
  - RASA1
  - RGS6
  - SAE1
  - ARHGAP31
  - TPI1
  - TK1
  - TK2
  - PAEP
  - PREP
  - PTPN22
  - SPINK1
  - MPZ
  - RNMT
  - ABCC1
  - Dihydroquercetin
  - Ubiquinol
  - Naringenin
  - Prephenate
  - Coniferyl alcohol
  - 4-HydroxyphenylpyruvateTyrosinet
  - Anthranilate
  - Kaempferol Leucopelargonidin Leucodelphinidin Leucocyanidin Quercetin
  - Phenylpyruvate Phenylalaninet
  - Caffeoyl-CoA
  - Coniferaldehyde
  - Sinapaldehyde
  - Sinapoylmalate
  - Dihydrokaempferol
  - Sinapoylglucose
  - 4-HBA
  - Dihydromyricetin
  - Sinapyl alcohol
  - ADP-glucose
  - Anthocyanin
  - Anthocyanin-glutathione
  - Anthocyanins
  - Asparagine
  - Aspartate
  - benzaldehyde
  - Benzoate
  - Carbohydrate
  - Cinnamate
  - Cystathionine
  - Cysteine
  - Flavonoid
  - glucose
  - Homocysteine
  - Methionine
  - 3-PGA
  - Pyruvate
  - Quercetin glycosides
  - Salicylate
  - Sucrose
  - Trehalose
  - Triose
  - Ubiquinone
  - UDP
  - Linolenate
  - Linoleate
  - Kaempferol
---
