---
figid: PMC5505665__nihms872426f2
figtitle: Metabolic and epigenetic coordination of T cell and Macrophage immunity
organisms:
- Saccharomyces cerevisiae
- Candida albicans
- Homo sapiens
- Mus musculus
- Influenza A virus
pmcid: PMC5505665
filename: nihms872426f2.jpg
figlink: /pmc/articles/PMC5505665/figure/F2/
number: F2
caption: Recognition of pathogen by immune cells initiates a genetic program that
  promotes use of numerous metabolic pathways for biogenesis (i.e. nucleotide synthesis)
  and energy production (i.e. glycolysis, TCA cycle). Metabolites involved in glycolysis
  (teal), TCA cycle (orange), and one-carbon metabolism (light-green) can play critical
  roles as substrates such as acetyl-CoA and SAM, for histone acetylation and methylation
  respectively, or cofactors such as 2-oxoglutarate and NAD+, for demethylation of
  DNA and histone deacetylation respectively. Epigenetic enzymes (blue) catalyze the
  addition (HAT, HMT or DNMT) and removal (HDAC, JHDM, or TET) of epigenetic marks
  and can be impacted by multiple metabolic pathways. Metabolites shown to be controlled
  by pathways relevant in T cells or macrophages (bold) are critical for energy production
  (i.e. lactate, acetyl-CoA) and proliferation (i.e. serine). Flexibility in metabolic
  pathway usage is reflected by the small network depicted here; for example, 3-phosphoglcerate
  can be converted to serine for use in one-carbon metabolism. Clear links between
  activity of epigenome modifying enzymes and metabolites relevant in T cells and
  macrophages that will impact the epigenetic landscape of activated cells can be
  drawn.
papertitle: Metabolic and epigenetic coordination of T cell and Macrophage immunity.
reftext: Anthony T. Phan, et al. Immunity. ;46(5):714-729.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9459195
figid_alias: PMC5505665__F2
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC5505665__F2
ndex: 2c6cf9c7-defe-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5505665__nihms872426f2.html
  '@type': Dataset
  description: Recognition of pathogen by immune cells initiates a genetic program
    that promotes use of numerous metabolic pathways for biogenesis (i.e. nucleotide
    synthesis) and energy production (i.e. glycolysis, TCA cycle). Metabolites involved
    in glycolysis (teal), TCA cycle (orange), and one-carbon metabolism (light-green)
    can play critical roles as substrates such as acetyl-CoA and SAM, for histone
    acetylation and methylation respectively, or cofactors such as 2-oxoglutarate
    and NAD+, for demethylation of DNA and histone deacetylation respectively. Epigenetic
    enzymes (blue) catalyze the addition (HAT, HMT or DNMT) and removal (HDAC, JHDM,
    or TET) of epigenetic marks and can be impacted by multiple metabolic pathways.
    Metabolites shown to be controlled by pathways relevant in T cells or macrophages
    (bold) are critical for energy production (i.e. lactate, acetyl-CoA) and proliferation
    (i.e. serine). Flexibility in metabolic pathway usage is reflected by the small
    network depicted here; for example, 3-phosphoglcerate can be converted to serine
    for use in one-carbon metabolism. Clear links between activity of epigenome modifying
    enzymes and metabolites relevant in T cells and macrophages that will impact the
    epigenetic landscape of activated cells can be drawn.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - 3-Phosphoglycerate
  - Acyl-CoA
  - Folate
  - Fumarate
  - Glucose
  - Glycine
  - homocysteine
  - Isocitrate
  - Methionine
  - Phosphoenolpyruvate
  - Pyruvate
  - Serine
  - Succinate
  - Lactate
  - Lactate Butyrate
---
