---
figid: PMC5445305__13068_2017_820_Fig7_HTML
figtitle: Characterization of central metabolic model in developing L
organisms:
- Lindera glauca
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC5445305
filename: 13068_2017_820_Fig7_HTML.jpg
figlink: /pmc/articles/PMC5445305/figure/Fig7/
number: F7
caption: 'Characterization of central metabolic model in developing L. glauca fruits
  for the regulation of carbon partitioning and energy supply for oil synthesis. The
  identified metabolic routes of carbon flux allocation and energy provision for FA
  synthesis are based on the transcriptome data from Illumina and 454 sequencing,
  and temporal transcript pattern analysis by qRT-PCR. Purple arrows indicate carbon
  flux drains into oil synthesis, brown arrows represent the transports of metabolite
  and energy across intracellular membrane by specific transporters, and green curved
  arrows represent all reactions for generation and consumption of energy in FA biosynthetic
  pathway, oxidative pentose phosphate pathway (OPPP), glycolysis, and tricarboxylic
  acid (TCA) cycle. All transporters and enzymes involved in carbon flux allocation
  and energy provision for FA synthesis are shown in red. Abbreviations for the enzymes,
  metabolites, and transporters are as follows: AAC ATP/ADP carrier, ACC acetyl-CoA
  carboxylase, ACLB ATP-citrate lyase subunit B, ACO aconitate hydratase, ADNT adenine
  nucleotide carrier, CS citrate synthase, DGAT diacylglycerol (DAG) acyltransferase,
  DIC dicarboxylate carrier, DTC dicarboxylate/tricarboxylate carrier, EAR enoyl-ACP
  reductase, ENO enolase, ER endoplasmic reticulum, FA fatty acid, FAD FA desaturase,
  FATA fatty acyl-ACP thioesterase A, FATB fatty acyl-ACP thioesterase B, FBA fructose-bisphosphate
  (F1,6P) aldolase, FK fructokinase, FUM fumarase, G6PDH glucose-6-phosphate (G6P)
  dehydrogenase, GAPA/C glyceraldehyde 3-phosphate (GAP) dehydrogenase subunit A/C,
  GLT glycolipid transporter, GPAT G3P acyltransferase, GPI G6P isomerase, GPT G6P
  transporter, HXK hexokinase, IDH isocitrate dehydrogenase, INV invertase, KAR ketoacyl-ACP
  reductase, LACS long-chain acyl-CoA synthase, LPAAT lysoPA acyltransferases, LPCAT
  lysoPC acyltransferases, LSC succinyl-CoA synthetase, MDH malate (Mal) dehydrogenase,
  ME malic enzyme, NTT nucleoside triphosphate (NTP) transporter, OGDHC 2-oxoglutarate
  dehydrogenase, 6PGDH 6-phosphogluconate (6PG) dehydrogenase, PA phosphatidic acid,
  PAP PA phosphohydrolases, PC phosphatidylcholine, PDAT DAG acyltransferase, PDC
  pyruvate (PYR) dehydrogenase complex, PEPC phosphoenolpyruvate (PEP) carboxylase,
  PFK ATP-dependent phosphofructokinase, PFP pyrophosphate phosphofructokinase, PGK
  phosphoglycerate kinase, PGLS 6-phosphogluconolactonase, PGM phosphoglycerate mutase,
  PHT phosphate (Pi) transporter, PK PYR kinase, PPT PEP transporter, PRK phosphoribulokinase,
  RBCS ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco) small subunit, RPE
  ribulose-5-phosphate (Ru5P) epimerase, RPI ribose 5-phosphate isomerase, SDH succinate
  dehydrogenase, SUC sucrose transporter, SUS sucrose synthase, TA transaldolase,
  TAG triacylglycerol, TK transketolase, TPI triose phosphate isomerase, TPT triose
  phosphate transporters, UGP UDP-glucose (UDPG) pyrophosphorylase'
papertitle: Integrated analysis of 454 and Illumina transcriptomic sequencing characterizes
  carbon flux and energy source for fatty acid synthesis in developing Lindera glauca
  fruits for woody biodiesel.
reftext: Zixin Lin, et al. Biotechnol Biofuels. 2017;10:134.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.894439
figid_alias: PMC5445305__F7
figtype: Figure
redirect_from: /figures/PMC5445305__F7
ndex: 740e243d-defc-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5445305__13068_2017_820_Fig7_HTML.html
  '@type': Dataset
  description: 'Characterization of central metabolic model in developing L. glauca
    fruits for the regulation of carbon partitioning and energy supply for oil synthesis.
    The identified metabolic routes of carbon flux allocation and energy provision
    for FA synthesis are based on the transcriptome data from Illumina and 454 sequencing,
    and temporal transcript pattern analysis by qRT-PCR. Purple arrows indicate carbon
    flux drains into oil synthesis, brown arrows represent the transports of metabolite
    and energy across intracellular membrane by specific transporters, and green curved
    arrows represent all reactions for generation and consumption of energy in FA
    biosynthetic pathway, oxidative pentose phosphate pathway (OPPP), glycolysis,
    and tricarboxylic acid (TCA) cycle. All transporters and enzymes involved in carbon
    flux allocation and energy provision for FA synthesis are shown in red. Abbreviations
    for the enzymes, metabolites, and transporters are as follows: AAC ATP/ADP carrier,
    ACC acetyl-CoA carboxylase, ACLB ATP-citrate lyase subunit B, ACO aconitate hydratase,
    ADNT adenine nucleotide carrier, CS citrate synthase, DGAT diacylglycerol (DAG)
    acyltransferase, DIC dicarboxylate carrier, DTC dicarboxylate/tricarboxylate carrier,
    EAR enoyl-ACP reductase, ENO enolase, ER endoplasmic reticulum, FA fatty acid,
    FAD FA desaturase, FATA fatty acyl-ACP thioesterase A, FATB fatty acyl-ACP thioesterase
    B, FBA fructose-bisphosphate (F1,6P) aldolase, FK fructokinase, FUM fumarase,
    G6PDH glucose-6-phosphate (G6P) dehydrogenase, GAPA/C glyceraldehyde 3-phosphate
    (GAP) dehydrogenase subunit A/C, GLT glycolipid transporter, GPAT G3P acyltransferase,
    GPI G6P isomerase, GPT G6P transporter, HXK hexokinase, IDH isocitrate dehydrogenase,
    INV invertase, KAR ketoacyl-ACP reductase, LACS long-chain acyl-CoA synthase,
    LPAAT lysoPA acyltransferases, LPCAT lysoPC acyltransferases, LSC succinyl-CoA
    synthetase, MDH malate (Mal) dehydrogenase, ME malic enzyme, NTT nucleoside triphosphate
    (NTP) transporter, OGDHC 2-oxoglutarate dehydrogenase, 6PGDH 6-phosphogluconate
    (6PG) dehydrogenase, PA phosphatidic acid, PAP PA phosphohydrolases, PC phosphatidylcholine,
    PDAT DAG acyltransferase, PDC pyruvate (PYR) dehydrogenase complex, PEPC phosphoenolpyruvate
    (PEP) carboxylase, PFK ATP-dependent phosphofructokinase, PFP pyrophosphate phosphofructokinase,
    PGK phosphoglycerate kinase, PGLS 6-phosphogluconolactonase, PGM phosphoglycerate
    mutase, PHT phosphate (Pi) transporter, PK PYR kinase, PPT PEP transporter, PRK
    phosphoribulokinase, RBCS ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco)
    small subunit, RPE ribulose-5-phosphate (Ru5P) epimerase, RPI ribose 5-phosphate
    isomerase, SDH succinate dehydrogenase, SUC sucrose transporter, SUS sucrose synthase,
    TA transaldolase, TAG triacylglycerol, TK transketolase, TPI triose phosphate
    isomerase, TPT triose phosphate transporters, UGP UDP-glucose (UDPG) pyrophosphorylase'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATPsynbeta
  - Atpalpha
  - Pgls
  - Pgd
  - Gapdh2
  - Gpdh1
  - Gapdh1
  - adp
  - fru
  - G6P
  - Pgi
  - UGP
  - Gip
  - Pfk
  - CG15812
  - fi
  - Ald1
  - Aldh-III
  - Tk
  - CdGAPr
  - RasGAP1
  - ArfGAP1
  - Zw
  - Pgk
  - Rpe
  - CycE
  - cyc
  - Rpi
  - pg
  - Pep
  - bgcn
  - kar
  - spidey
  - Fs(3)Kar
  - pk
  - ear
  - pyr
  - ACC
  - tacc
  - acclinal-wing
  - pi
  - chal
  - Adar
  - Aco
  - Acox57D-d
  - Idh
  - Idh3b
  - LPCAT
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - mal
  - bs
  - Mrtf
  - Pc
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - Suc
  - l(1)sc
  - Pa
  - CG11438
  - CG11426
  - Amph
  - Dark
  - Fum1
  - cha
  - ChAT
  - INVS
  - ATP8A2
  - PGLS
  - GAPDH
  - GAPDHP44
  - WDTC1
  - ZBTB22
  - DECR1
  - GPI
  - GNPDA1
  - PFKFB1
  - GIP
  - GNAI2
  - C1QTNF1
  - UGP2
  - PRF1
  - PFKL
  - PFKM
  - PFKP
  - FBXO3
  - TK1
  - TK2
  - RASA1
  - RGS6
  - ANTXR1
  - H6PD
  - RPE
  - RPIA
  - ENO3
  - PAEP
  - PREP
  - PTPN22
  - MDH1
  - HSD17B12
  - OXSM
  - PDC
  - PNKD
  - ACACA
  - BMS1
  - ACACB
  - DNMT1
  - PEPC
  - PGC
  - ACSL4
  - AKR1B1
  - FDXR
  - KLK15
  - IDH1
  - IDH2
  - LPCAT3
  - LPCAT1
  - MBOAT2
  - MDH2
  - MAL
  - MRTFA
  - TIRAP
  - PNMA1
  - GSTZ1
  - PC
  - PCSK7
  - BRCA2
  - FANCD2
  - PSEN1
  - SARDH
  - SDHB
  - SORD
  - SDS
  - ARHGEF1
  - LPA
  - PLPP1
  - MYBBP1A
  - PLPPR5
  - TMED7
  - LINC01194
  - PLK3
  - APRT
  - MFAP1
  - TCFL5
  - FAD
  - Sucrose
  - Glu
---
