---
figid: PMC5182094__nihms829395f2
figtitle: Biosynthetic pathway connects cryptic ribosomally synthesized posttranslationally
  modified peptide genes with pyrroloquinoline alkaloids
organisms:
- Escherichia coli
- Mycobacterium tuberculosis
- Streptomyces coelicolor
- Escherichia coli BW25113
- Myxococcus xanthus
- Salinispora tropica
- Halalkalibacterium halodurans
- marine bacterium SO3
- Halalkalibacterium halodurans C-125
- Streptomyces rugosporus
- Streptomyces aurantiacus
- Kibdelosporangium sp. MJ126-NF4
- Amycolatopsis japonica
- Streptomyces sp. CNT302
- Saccharothrix sp. NRRL B-16348
- Nocardiopsis sp. NRRL B-16309
- Nocardia brevicatena
- Streptomyces xanthophaeus
- Amycolatopsis thermoflava
- Nocardia vinacea
- Desmospora sp. 8437
- Saccharomyces cerevisiae
- Matthiola incana
- uncultured actinomycete
- uncultured marine bacterium
- NA
- Homo sapiens
pmcid: PMC5182094
filename: nihms829395f2.jpg
figlink: /pmc/articles/PMC5182094/figure/F2/
number: F2
caption: 'A. Alignment of the ammosamide (amm) and lymphostin (lym) biosynthetic gene
  clusters. Related orphaned gene cluster (xan) from S. xanthophaeus also included.
  ORFs of high sequence homology are connected with a dotted line. TAR cloning sites
  for amm are indicated with “X”. Conserved genes are indicated with brackets and
  tailoring genes are outline in black. B. For ammosamide B and lymphostin, structural
  features derived from tailoring genes in their respective pathways are highlighted
  in grey. C. Aligned translated sequences of the short ORFs in amm, lym, and xan
  pathways. FNLD motif is in bold. Abbreviations: H = hypothetical protein; LD = lantibiotic
  dehydratase; P = peptidase; O = oxidoreductase; Amt = amido transferase; M = methyl
  transferase; Hal = halogenase. Pairs of ‡ * ¤ ¥ indicate homologues.'
papertitle: Biosynthetic pathway connects cryptic ribosomally synthesized posttranslationally
  modified peptide genes with pyrroloquinoline alkaloids.
reftext: Peter A. Jordan, et al. Cell Chem Biol. 2016 Dec 22;23(12):1504-1514.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7238564
figid_alias: PMC5182094__F2
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC5182094__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5182094__nihms829395f2.html
  '@type': Dataset
  description: 'A. Alignment of the ammosamide (amm) and lymphostin (lym) biosynthetic
    gene clusters. Related orphaned gene cluster (xan) from S. xanthophaeus also included.
    ORFs of high sequence homology are connected with a dotted line. TAR cloning sites
    for amm are indicated with “X”. Conserved genes are indicated with brackets and
    tailoring genes are outline in black. B. For ammosamide B and lymphostin, structural
    features derived from tailoring genes in their respective pathways are highlighted
    in grey. C. Aligned translated sequences of the short ORFs in amm, lym, and xan
    pathways. FNLD motif is in bold. Abbreviations: H = hypothetical protein; LD =
    lantibiotic dehydratase; P = peptidase; O = oxidoreductase; Amt = amido transferase;
    M = methyl transferase; Hal = halogenase. Pairs of ‡ * ¤ ¥ indicate homologues.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BAX
  - HAL
  - KRT31
  - ARHGAP45
  - SPINT1
  - ST14
  - AMT
  - Ammosamide B
---
