---
figid: PMC5519968__rccm.201607-1525OC_f3
figtitle: Network of canonical signaling pathways that are individually associated
  with CpG-ODN responsiveness, host gene coexpression modules, and microbial community
  features
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC5519968
filename: rccm.201607-1525OC_f3.jpg
figlink: /pmc/articles/PMC5519968/figure/fig3/
number: F3
caption: 'Network of canonical signaling pathways that are individually associated
  with CpG-ODN responsiveness, host gene coexpression modules, and microbial community
  features. An integrated approach was used to identify common canonical pathways
  (light blue circles) enriched from the following three paired datasets obtained
  from patients with IPF in the COMET-IPF study: (1) fibroblast response to CpG-ODN,
  (2) microbial diversity indices and OTUs, and (3) WGCNA host gene coexpression modules.
  Up-regulated genes between cases with CpG-ODN response and CpG-ODN nonresponse were
  subjected to Ingenuity Pathway Analysis with a criterion of FDR less than 0.01 to
  identify 22 significantly CpG-ODN response–associated canonical pathways. These
  22 pathways were then matched to the canonical pathways associated with 10 host
  gene coexpression modules (gray squares) from PBMC microarray and microbial diversity
  indices and OTUs (purple diamonds) from bacterial 16S ribosomal RNA sequencing (Tables
  E2 and E4, respectively). Fourteen pathways (light blue circles) were shared among
  CpG-ODN response (yellow hexagon), microbial features (purple diamonds), and host
  gene coexpression modules (gray squares). The red and green edges represent positive
  and negative associations between nodes, respectively. Gray edges represent both
  positive and negative associations between nodes. The width of the edge is proportional
  to the value of −log P value (i.e., thicker edge is more significant). COMET-IPF = Correlating
  Outcomes with Biochemical Markers to Estimate Time-progression in Idiopathic Pulmonary
  Fibrosis; FDR = false discovery rate; iNOS = inducible nitric oxide synthase; IPF = idiopathic
  pulmonary fibrosis; NF-κB = nuclear factor-κB; ODN = oligodeoxynucleotide; OTU = operational
  taxonomic unit; PBMC = peripheral blood mononuclear cell; PKR = protein kinase R;
  PPAR = peroxisome proliferator–activated receptor; TNFR2 = tumor necrosis factor
  receptor 2; WGCNA = weighted gene coexpression network analysis R software package.'
papertitle: Microbes Are Associated with Host Innate Immune Response in Idiopathic
  Pulmonary Fibrosis.
reftext: Yong Huang, et al. Am J Respir Crit Care Med. 2017 Jul 15;196(2):208-219.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7867538
figid_alias: PMC5519968__F3
figtype: Figure
redirect_from: /figures/PMC5519968__F3
ndex: 61396660-deb5-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5519968__rccm.201607-1525OC_f3.html
  '@type': Dataset
  description: 'Network of canonical signaling pathways that are individually associated
    with CpG-ODN responsiveness, host gene coexpression modules, and microbial community
    features. An integrated approach was used to identify common canonical pathways
    (light blue circles) enriched from the following three paired datasets obtained
    from patients with IPF in the COMET-IPF study: (1) fibroblast response to CpG-ODN,
    (2) microbial diversity indices and OTUs, and (3) WGCNA host gene coexpression
    modules. Up-regulated genes between cases with CpG-ODN response and CpG-ODN nonresponse
    were subjected to Ingenuity Pathway Analysis with a criterion of FDR less than
    0.01 to identify 22 significantly CpG-ODN response–associated canonical pathways.
    These 22 pathways were then matched to the canonical pathways associated with
    10 host gene coexpression modules (gray squares) from PBMC microarray and microbial
    diversity indices and OTUs (purple diamonds) from bacterial 16S ribosomal RNA
    sequencing (Tables E2 and E4, respectively). Fourteen pathways (light blue circles)
    were shared among CpG-ODN response (yellow hexagon), microbial features (purple
    diamonds), and host gene coexpression modules (gray squares). The red and green
    edges represent positive and negative associations between nodes, respectively.
    Gray edges represent both positive and negative associations between nodes. The
    width of the edge is proportional to the value of −log P value (i.e., thicker
    edge is more significant). COMET-IPF = Correlating Outcomes with Biochemical Markers
    to Estimate Time-progression in Idiopathic Pulmonary Fibrosis; FDR = false discovery
    rate; iNOS = inducible nitric oxide synthase; IPF = idiopathic pulmonary fibrosis;
    NF-κB = nuclear factor-κB; ODN = oligodeoxynucleotide; OTU = operational taxonomic
    unit; PBMC = peripheral blood mononuclear cell; PKR = protein kinase R; PPAR = peroxisome
    proliferator–activated receptor; TNFR2 = tumor necrosis factor receptor 2; WGCNA = weighted
    gene coexpression network analysis R software package.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Dif
  - dl
  - Rel
  - red
  - trc
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Spp
  - pps
  - Inos
  - Nos
  - Neurl4
  - .na.character
  - Pink1
  - IL10
  - IL6
  - IL17A
  - NFKB1
  - TNFRSF1B
  - IK
  - DYRK3
  - IFNK
  - IFNW1
  - IFNA1
  - IFNA2
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA17
  - IFNA21
  - IFNB1
  - EIF2AK2
  - HM13
  - NOS2
  - ISYNA1
  - PPARA
  - PPARD
  - PPARG
  - Aryl Hydrocarbon
---
