---
figid: PMC5033206__kgmc-05-04-947827-g006
figtitle: Illustration of the metabolic pathways and genes that were differentially
  affected in siliques between B
organisms:
- NA
pmcid: PMC5033206
filename: kgmc-05-04-947827-g006.jpg
figlink: /pmc/articles/PMC5033206/figure/f0006/
number: F6
caption: 'Illustration of the metabolic pathways and genes that were differentially
  affected in siliques between B. juncea and B. napus. Unabbreviated gene names and
  AGI numbers are listed in respective pathway- and Tables S6 to S11. Other abbreviations
  for each pathway are as follows. IAA biosynthesis: IAA, indole-3-acetic acid; IBA,
  indole-3-butyric acid. Shikimate pathway: PEP, phospho-enol-pyruvate; E4P, erythrose
  4-phosphate. Isoprenoid biosynthesis: MVP, mevalonate pathway; IPP, isopentenyl
  diphosphate; MEP, methyl erythritol phosphate pathway; G3P, D-glyceraldehyde-3-phosphate;
  Pyr, pyruvate; GGP, geranylgeranyl diphosphate. Carotenoid biosynthesis: GGP, geranylgeranyl
  diphosphate. Jasmonates biosynthesis: 18:1-18:1-PC, 1,2-dioleoylphosphatidylcholine;
  18:3-18:3-PC, 1,2-dilinolenoylphosphatidylcholine; JA, jasmonic acid; MeJA, methyl
  jasmonate; JA-Ile, JA-isoleucine. Ethylene biosynthesis: SAM, S-adenosyl methionine.'
papertitle: A comprehensive transcriptome analysis of silique development and dehiscence
  in Arabidopsis and Brassica integrating genotypic, interspecies and developmental
  comparisons.
reftext: Masrur R Jaradat, et al. GM Crops Food. 2014 Oct-Dec;5(4):302-320.
year: '2014'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9066571
figid_alias: PMC5033206__F6
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC5033206__F6
ndex: 9a48b831-de9d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5033206__kgmc-05-04-947827-g006.html
  '@type': Dataset
  description: 'Illustration of the metabolic pathways and genes that were differentially
    affected in siliques between B. juncea and B. napus. Unabbreviated gene names
    and AGI numbers are listed in respective pathway- and Tables S6 to S11. Other
    abbreviations for each pathway are as follows. IAA biosynthesis: IAA, indole-3-acetic
    acid; IBA, indole-3-butyric acid. Shikimate pathway: PEP, phospho-enol-pyruvate;
    E4P, erythrose 4-phosphate. Isoprenoid biosynthesis: MVP, mevalonate pathway;
    IPP, isopentenyl diphosphate; MEP, methyl erythritol phosphate pathway; G3P, D-glyceraldehyde-3-phosphate;
    Pyr, pyruvate; GGP, geranylgeranyl diphosphate. Carotenoid biosynthesis: GGP,
    geranylgeranyl diphosphate. Jasmonates biosynthesis: 18:1-18:1-PC, 1,2-dioleoylphosphatidylcholine;
    18:3-18:3-PC, 1,2-dilinolenoylphosphatidylcholine; JA, jasmonic acid; MeJA, methyl
    jasmonate; JA-Ile, JA-isoleucine. Ethylene biosynthesis: SAM, S-adenosyl methionine.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EDS16
  - ICS2
  - PEP
  - PAS2
  - DHS2
  - NIT3
  - NIT4
  - PAI1
  - PAI2
  - PAL1
  - PAL2
  - PAL4
  - 4CL1
  - 4CL3
  - ILL6
  - C4H
  - HCT
  - CCoAOMT1
  - IBR3
  - SNG1
  - ATCAD4
  - CAD6
  - CAD5
  - PSY
  - ABA4
  - ABC4
  - FPS1
  - GGPS1
  - CYP707A4
  - CYP707A1
  - ABA1
  - IPP1
  - GGPS2
  - FPS2
  - NCED3
  - ACS2
  - ACS4
  - ACS6
  - ACS7
  - ACS8
  - LOX1
  - JAR1
  - ACO1
  - FAD3
  - LOX2
  - LOX3
  - AOC1
  - AOC2
  - SAM1
  - EFE
  - JMT
  - OPR3
  - AOC3
  - Violaxanthin
  - Coniferyl alcohol
  - trans-Neoxanthin
  - Coniferaldehyde
  - Sinapaldehyde
  - Sinapoyl malate
  - Sinapyl alcohol
  - Carotene
  - Coumarate Cinnamate
  - DDE1  -MeJA
  - Ethylene
  - FAD3
  - glucose
  - Methionine
  - Phe
  - Pyruvate
  - Isoprenoid
  - Trp
  - Tyr
---
