---
figid: PMC5437397__12864_2017_3769_Fig7_HTML
figtitle: RNA interference (RNAi) pathway members in malacostracans
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC5437397
filename: 12864_2017_3769_Fig7_HTML.jpg
figlink: /pmc/articles/PMC5437397/figure/Fig7/
number: F7
caption: 'RNA interference (RNAi) pathway members in malacostracans. a Exogenous viral
  dsRNA is imported into the cytoplasm and is cleaved by the endoribonuclease from
  the RNase III family, Dicer. Cleaved fragments of dsRNA are known as small interfering
  RNAs (siRNAs). Dicer contains both Piwi/Argonaute/Zwille (PAZ) and helicase domains.
  Dicer activates the RNA-induced silencing complex (RISC), which is comprised of
  Argonaute-2 and the transactivating response RNA-binding protein (TRBP). Single-stranded
  siRNAs are incorporated into a RISC complex, upon which the siRNAs form complementary
  base-pairing to target mRNA and mRNA cleavage ensues. Other RISC-associated proteins
  include R2D2 and Loquacious in D. melanogaster. Dicer is also involved in microRNA
  (miRNA) biogenesis. Encoded by the genome, miRNAs are involved in the regulation
  of gene expression in the RNAi pathway. Transcribed by RNA polymerase II, Pri-miRNA
  is a long primary transcript of miRNAs and is processed into a stem-loop containing
  pre-miRNA by the microprocessor complex consisting of a ribonuclease III enzyme
  Drosha and Pasha, which is also known as DGCR8, protein. Pre-miRNA enters the cytoplasm
  and is cleaved by Dicer to generate a mature miRNA that is then integrated into
  the RISC complex. The miRNA-targeted transcript is either degraded or silenced.
  Phylogenetic trees of b Dicer, c Drosha, d TRBP and e Pasha are constructed using
  the maximum-likelihood method from an amino acid multiple sequence alignment. Taxa
  labels are depicted as their respective colour codes. Bootstrap support values (n = 1000)
  for all trees can be found in Additional file : Figure S14. Scale bar represents
  substitution per site'
papertitle: Comparative genomic analysis of innate immunity reveals novel and conserved
  components in crustacean food crop species.
reftext: Alvina G. Lai, et al. BMC Genomics. 2017;18:389.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7714329
figid_alias: PMC5437397__F7
figtype: Figure
redirect_from: /figures/PMC5437397__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5437397__12864_2017_3769_Fig7_HTML.html
  '@type': Dataset
  description: 'RNA interference (RNAi) pathway members in malacostracans. a Exogenous
    viral dsRNA is imported into the cytoplasm and is cleaved by the endoribonuclease
    from the RNase III family, Dicer. Cleaved fragments of dsRNA are known as small
    interfering RNAs (siRNAs). Dicer contains both Piwi/Argonaute/Zwille (PAZ) and
    helicase domains. Dicer activates the RNA-induced silencing complex (RISC), which
    is comprised of Argonaute-2 and the transactivating response RNA-binding protein
    (TRBP). Single-stranded siRNAs are incorporated into a RISC complex, upon which
    the siRNAs form complementary base-pairing to target mRNA and mRNA cleavage ensues.
    Other RISC-associated proteins include R2D2 and Loquacious in D. melanogaster.
    Dicer is also involved in microRNA (miRNA) biogenesis. Encoded by the genome,
    miRNAs are involved in the regulation of gene expression in the RNAi pathway.
    Transcribed by RNA polymerase II, Pri-miRNA is a long primary transcript of miRNAs
    and is processed into a stem-loop containing pre-miRNA by the microprocessor complex
    consisting of a ribonuclease III enzyme Drosha and Pasha, which is also known
    as DGCR8, protein. Pre-miRNA enters the cytoplasm and is cleaved by Dicer to generate
    a mature miRNA that is then integrated into the RISC complex. The miRNA-targeted
    transcript is either degraded or silenced. Phylogenetic trees of b Dicer, c Drosha,
    d TRBP and e Pasha are constructed using the maximum-likelihood method from an
    amino acid multiple sequence alignment. Taxa labels are depicted as their respective
    colour codes. Bootstrap support values (n = 1000) for all trees can be found in
    Additional file : Figure S14. Scale bar represents substitution per site'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Dcr-2
  - Dcr-1
  - de
  - der
  - Egfr
  - loqs
  - da
  - nAChRalpha2
  - dep
  - DptA
  - dve
  - Ate1
  - AGO2
  - AGO1
  - Sec23
  - AstA-R2
  - Dop1R2
  - Dora
  - dor
  - DOR
  - dl
  - Ts
  - pan
  - Panx
  - Art1
  - Me
  - bap
  - mel
  - Men
  - l(1)16Fa
  - mei-W68
  - pr
  - per
  - Dera
  - drosha
  - tRNA:Lys-TTT-1-1
  - tRNA:Lys-TTT-2-4
  - tRNA:Lys-TTT-2-1
  - tRNA:Lys-TTT-2-2
  - tRNA:Lys-TTT-2-5
  - tRNA:Lys-TTT-2-3
  - gra
  - grn
  - grape
  - Ude
  - ida
  - pasha
  - tral
  - cue
  - bel
  - pain
  - Kap-alpha1
  - Dop1R1
  - nAChRalpha1
  - Doa
  - den
  - peo
  - anne
  - sno
  - Snoo
  - CG43051
  - DICER1
  - TARBP2
  - TARBP2P1
  - NCOA6
  - SEPTIN4
  - ATM
  - ARSI
  - KDM5D
  - IGSF6
  - OPRD1
  - TP53INP2
  - ADA2
  - RAB8A
  - MAP2K1
  - ME1
  - PCDHB16
  - PER1
  - DERA
  - DROSHA
  - APP
  - AAAS
  - AAA1
  - WT1
  - IDUA
  - DGCR8
  - PATE1
  - ADRA1D
  - ATP2A2
  - TPM2
  - IGHD4-11
  - POLG
  - TWNK
  - LRBA
  - SKIL
  - SBNO2
  - SBNO1
---
