---
figid: PMC5739026__nihms862609f1
figtitle: Three main classes of regulatory small RNAs
organisms:
- Drosophila melanogaster
- Caenorhabditis elegans
- Plasmodium falciparum
- Bombyx mori
- Aedes aegypti
- Anopheles gambiae
- Plasmodium berghei
- Tribolium castaneum
- Aedes albopictus
- Acyrthosiphon pisum
- Nilaparvata lugens
- Culex quinquefasciatus
- Plutella xylostella
- Anopheles stephensi
- Culex pipiens
- Blattella germanica
- Heliconius melpomene
- Episyrphus balteatus
- Megaselia abdita
- Dengue virus
- West Nile virus
- Sindbis virus
- Chikungunya virus
- Dengue virus type 2
- Semliki Forest virus
- Rift Valley fever virus
- dsRNA viruses
- Flock House virus
- La Crosse virus
- mosquito-borne viruses
- Yellow fever virus group
- Wolbachia pipientis
- Wolbachia sp. wMelPop
- Homo sapiens
- Mus musculus
- Solanum melongena
- NA
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC5739026
filename: nihms862609f1.jpg
figlink: /pmc/articles/PMC5739026/figure/F1/
number: F1
caption: Three main classes of regulatory small RNAs. (A) miRNAs are transcribed by
  RNA polymerase II (RNA pol) to yield a primary miRNA (pri-miRNA) transcript, which
  is cleaved by Drosha to produce the precursor miRNA (pre-miRNA). These pre-miRNAs
  are exported from the nucleus to the cytoplasm, where they are further processed
  by Dcr-1 to generate an miRNA and miRNA* duplex. Once loaded into the RNA-induced
  silencing complex (RISC), predominately Ago-1-associated RISC, the miRNA strand
  guides translational repression and/or mRNA degradation of target mRNAs. (B) The
  siRNA pathway is subdivided into an endogenous and exogenous pathway, depending
  on the origin of the dsRNA trigger. Endo-siRNAs are derived from the convergent
  transcription of overlapping genes and structural loci; exo-siRNAs are processed
  from the dsRNA byproducts of virus replication or other foreign genetic material.
  dsRNAs derived from these sources are cleaved by the RNase III enzyme, Dcr-2, generating
  siRNA duplexes. These duplexes are then incorporated into a RISC. After removal
  of the passenger strand from the activated RISC, the remaining guide strand directs
  Ago-2 to complementary RNAs in the cell, resulting in their degradation. (C) piRNAs
  are transcribed from genomic loci termed piRNA clusters. Cleavage of these ss transcripts
  by Zucchini (Zuc) generates an initial pool of primary piRNAs, which are loaded
  into Piwi. Secondary piRNAs are generated in a slicer-dependent amplification loop.
  Piwi- or Aub-bound piRNAs direct the cleavage of target strands, generating piRNAs
  that will be bound by Ago-3 and vice versa.
papertitle: 'Small RNAs: a new frontier in mosquito biology.'
reftext: Keira J. Lucas, et al. Trends Parasitol. ;29(6):295-303.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9586685
figid_alias: PMC5739026__F1
figtype: Figure
redirect_from: /figures/PMC5739026__F1
ndex: 73f5f4e9-debf-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5739026__nihms862609f1.html
  '@type': Dataset
  description: Three main classes of regulatory small RNAs. (A) miRNAs are transcribed
    by RNA polymerase II (RNA pol) to yield a primary miRNA (pri-miRNA) transcript,
    which is cleaved by Drosha to produce the precursor miRNA (pre-miRNA). These pre-miRNAs
    are exported from the nucleus to the cytoplasm, where they are further processed
    by Dcr-1 to generate an miRNA and miRNA* duplex. Once loaded into the RNA-induced
    silencing complex (RISC), predominately Ago-1-associated RISC, the miRNA strand
    guides translational repression and/or mRNA degradation of target mRNAs. (B) The
    siRNA pathway is subdivided into an endogenous and exogenous pathway, depending
    on the origin of the dsRNA trigger. Endo-siRNAs are derived from the convergent
    transcription of overlapping genes and structural loci; exo-siRNAs are processed
    from the dsRNA byproducts of virus replication or other foreign genetic material.
    dsRNAs derived from these sources are cleaved by the RNase III enzyme, Dcr-2,
    generating siRNA duplexes. These duplexes are then incorporated into a RISC. After
    removal of the passenger strand from the activated RISC, the remaining guide strand
    directs Ago-2 to complementary RNAs in the cell, resulting in their degradation.
    (C) piRNAs are transcribed from genomic loci termed piRNA clusters. Cleavage of
    these ss transcripts by Zucchini (Zuc) generates an initial pool of primary piRNAs,
    which are loaded into Piwi. Secondary piRNAs are generated in a slicer-dependent
    amplification loop. Piwi- or Aub-bound piRNAs direct the cleavage of target strands,
    generating piRNAs that will be bound by Ago-3 and vice versa.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Polr2A
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - drosha
  - pasha
  - Ranbp21
  - piwi
  - zuc
  - Dcr-1
  - Dcr-2
  - aub
  - ago
  - AGO1
  - AGO2
  - AGO3
  - SCAP
  - bmm
  - Mela
  - Gm42368
  - Drosha
  - Xpo5
  - Pld6
  - Dicer1
  - nt
  - Scpep1
  - Fbxw7
  - Ago2
  - Ago1
  - Ago3
  - Scap
  - ERVW-4
  - POLR1A
  - POLR1B
  - POLR1C
  - POLR1D
  - DROSHA
  - DGCR8
  - XPO5
  - PIWIL1
  - PLD6
  - DICER1
  - SCPEP1
  - FBXW7
  - SH2D2A
  - dicer1
  - ago2
  - ago1
  - ago3b
  - scap
---
