---
figid: PMC5441870__elife-18990-fig3
figtitle: Central metabolism of T
organisms:
- Thermovibrio ammonificans
- Escherichia coli
- Aquifex aeolicus
- Pyrococcus abyssi
- Desulfovibrio desulfuricans
- Hydrogenobacter thermophilus
- Aquifex pyrophilus
- Desulfobacter hydrogenophilus
- Terrisporobacter glycolicus
- Ferroglobus placidus
- endosymbiont of Riftia pachyptila
- Caminibacter mediatlanticus
- Persephonella marina
- Hydrogenobacter thermophilus TK-6
- Thermosulfidibacter takaii ABI70S6
- Hydrogenothermus marinus
- Aquifex aeolicus VF5
- Sulfurihydrogenibium azorense
- Hydrogenobaculum acidophilum
- Hydrogenobacter hydrogenophilus
- Hydrogenobaculum sp. Y04AAS1
- Nitratiruptor sp. SB155-2
- bacterium sp. OF1
- Sulfurihydrogenibium sp. YO3AOP1
- Desulfurobacterium thermolithotrophum
- Sulfurihydrogenibium subterraneum DSM 15120
- Desulfurobacterium sp. TC5-1
- Persephonella sp. IF05-L8
- Persephonella marina EX-H1
- Magnetococcus marinus MC-1
- Sulfurihydrogenibium subterraneum
- Persephonella guaymasensis
- Desulfurobacterium indicum
- Desulfurobacterium sp.
- Sulfurihydrogenibium azorense Az-Fu1
- Sulfurihydrogenibium sp.
- Balnearium lithotrophicum
- Hydrogenivirga caldilitoris
- Thermocrinis minervae
- Hydrogenivirga sp. 128-5
- Caminibacter mediatlanticus TB-2
- Hydrogenivirga sp. 128-5-R1-6
- Hydrogenivirga sp. 128-5-R1-1
- Venenivibrio stagnispumantis
- Thermosulfidibacter takaii
- Ammonifex degensii
- Sulfurihydrogenibium yellowstonense SS-5
- Thermocrinis albus DSM 14484
- Thermovibrio ammonificans HB-1
- Thermocrinis ruber
- Desulfurobacterium thermolithotrophum DSM 11699
- Thermocrinis albus DSM 12173
- Phorcysia thermohydrogeniphila
- Arabidopsis thaliana
- Carica papaya
- uncultured bacterium
- Trachipleistophora hominis
- uncultured bacterium Of23
- uncultured bacterium yo3
- Riftia pachyptila
- Alvinella pompejana
- Homo sapiens
- NA
pmcid: PMC5441870
filename: elife-18990-fig3.jpg
figlink: /pmc/articles/PMC5441870/figure/fig3/
number: F3
caption: 'Enzyme names are reported together with the gene locus number (Theam_number).
  Primary compounds involved in reactions were also reported, however visualized reactions
  are not complete. Pathways were arbitrarily color coded according to their reconstructed
  function and are consistent throughout the paper. Solid shapes represent genes for
  which the enzyme was found in the proteome, while outlined shapes were only identified
  in the genome. Circled numbers 1 to 6 represent reaction numbers of the putative
  reductive acetyl-CoA pathway as described in the text. Abbreviations: NITRATE AMMONIFICATION:
  NapCMADGH – Periplasmic nitrate reductase complex; NirA – Putative nitrite reductase;
  AmtB – ammonia transporter; GlnA – L-Glutamine synthetase; GltA – Glutamate synthetase.
  HYDROGEN OXIDATION: HynABC – Ni-Fe Membrane bound hydrogenase; FrhACB – Cytosolic
  Ni-Fe hydrogenase/putative coenzyme F420 hydrogenase; HupD – Cyrosolic Ni-Fe hydrogenase
  maturation protease; HypA-F – Hydrogenases expression/synthesis accessory proteins;
  EchABCEF – Ech membrane bound hydrogenase complex; HydAB – Cytosolic Ni-Fe hydrogenases
  potentially involved in ferredoxin reduction; Hdr? – Missing heterodisulfide reductase
  CoB-CoM; FdhABC – Formate dehydrogenase. ENERGY PRODUCTION: NADH dehydrogenase and
  ATP synthetase are reported without the names of the single units; Nhe – Sodium/hydrogen
  symporter. SULFUR REDUCTION: Sqr – Putative sulfate quinone reductase involved in
  sulfur respiration; Nsr? – FAD/NAD nucleotide-disulphide oxidoreductase; Aps – Sulfate
  adenylyl transferase and kinase involved in assimilation of sulfur. FLAGELLAR COMPLEX:
  for simplicity single unit names are not reported. REDUCTIVE ACETYL-CoA: Fhs – Proposed
  reverse formyl-THF deformylase; PurHT – Formyltransferases (phosphoribosylaminoimidazolecarboxamide
  and phosphoribosylglycinamide) FolD – Methenyl-THF cyclohydrolase and dehydrogenase;
  MetVF – Methylene-THF reductase; MetR – Putative methyl-transferase; CodH – Carbon
  monoxyde dehydrogenase; AcsA – Acetyl-CoA ligase/synthase; AccABCD – Acetyl-CoA
  carboxylase. REDUCTIVE CITRIC ACID CYCLE: AclAB – ATP-citrate lyase; Mdh – Malate
  dehydrogenase; FumAB – Fumarate hydratase; FrdAB – Fumarate reductase; SucCD – Succinyl-CoA
  synthetase; OorABCD – 2-Oxoglutarate synthase; Idh2 – Isocitrate dehydrogenase/2-oxoglutarate
  carboxylase; AcnB – Aconitate hydratase; PorABDG – Pyruvate synthase; PycAB - Pyruvate
  carboxylase; PpsA – Phosphoenolpyruvate synthase water dikinase; PyK – Pyruvate:water
  dikinase. GLUCONEOGENESIS: Eno – Enolase; Pgm – Phosphoglycerate mutase; Pgk – Phosphoglycerate
  kinase; Gapor – Glyceraldehyde-3-phosphate dehydrogenase; Gapdh – Glyceraldehyde
  3-phosphate dehydrogenase; Fba – Predicted fructose-bisphosphate aldolase; Tim –
  Triosephosphate isomerase; Fbp – Fructose-1,6-bisphosphatase I; Pgi – Phosphoglucose
  isomerase; Pgm – Phosphoglucomutase.DOI: http://dx.doi.org/10.7554/eLife.18990.00710.7554/eLife.18990.008Figure
  3—source data 1.List of the proteins identified in the proteome of T. ammonificans
  grown under nitrate reducing conditions.DOI: http://dx.doi.org/10.7554/eLife.18990.008'
papertitle: Insight into the evolution of microbial metabolism from the deep-branching
  bacterium, Thermovibrio ammonificans.
reftext: Donato Giovannelli, et al. eLife. 2017;6:e18990.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8993864
figid_alias: PMC5441870__F3
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
- Arabidopsis thaliana
redirect_from: /figures/PMC5441870__F3
ndex: 0085450e-dea7-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5441870__elife-18990-fig3.html
  '@type': Dataset
  description: 'Enzyme names are reported together with the gene locus number (Theam_number).
    Primary compounds involved in reactions were also reported, however visualized
    reactions are not complete. Pathways were arbitrarily color coded according to
    their reconstructed function and are consistent throughout the paper. Solid shapes
    represent genes for which the enzyme was found in the proteome, while outlined
    shapes were only identified in the genome. Circled numbers 1 to 6 represent reaction
    numbers of the putative reductive acetyl-CoA pathway as described in the text.
    Abbreviations: NITRATE AMMONIFICATION: NapCMADGH – Periplasmic nitrate reductase
    complex; NirA – Putative nitrite reductase; AmtB – ammonia transporter; GlnA –
    L-Glutamine synthetase; GltA – Glutamate synthetase. HYDROGEN OXIDATION: HynABC
    – Ni-Fe Membrane bound hydrogenase; FrhACB – Cytosolic Ni-Fe hydrogenase/putative
    coenzyme F420 hydrogenase; HupD – Cyrosolic Ni-Fe hydrogenase maturation protease;
    HypA-F – Hydrogenases expression/synthesis accessory proteins; EchABCEF – Ech
    membrane bound hydrogenase complex; HydAB – Cytosolic Ni-Fe hydrogenases potentially
    involved in ferredoxin reduction; Hdr? – Missing heterodisulfide reductase CoB-CoM;
    FdhABC – Formate dehydrogenase. ENERGY PRODUCTION: NADH dehydrogenase and ATP
    synthetase are reported without the names of the single units; Nhe – Sodium/hydrogen
    symporter. SULFUR REDUCTION: Sqr – Putative sulfate quinone reductase involved
    in sulfur respiration; Nsr? – FAD/NAD nucleotide-disulphide oxidoreductase; Aps
    – Sulfate adenylyl transferase and kinase involved in assimilation of sulfur.
    FLAGELLAR COMPLEX: for simplicity single unit names are not reported. REDUCTIVE
    ACETYL-CoA: Fhs – Proposed reverse formyl-THF deformylase; PurHT – Formyltransferases
    (phosphoribosylaminoimidazolecarboxamide and phosphoribosylglycinamide) FolD –
    Methenyl-THF cyclohydrolase and dehydrogenase; MetVF – Methylene-THF reductase;
    MetR – Putative methyl-transferase; CodH – Carbon monoxyde dehydrogenase; AcsA
    – Acetyl-CoA ligase/synthase; AccABCD – Acetyl-CoA carboxylase. REDUCTIVE CITRIC
    ACID CYCLE: AclAB – ATP-citrate lyase; Mdh – Malate dehydrogenase; FumAB – Fumarate
    hydratase; FrdAB – Fumarate reductase; SucCD – Succinyl-CoA synthetase; OorABCD
    – 2-Oxoglutarate synthase; Idh2 – Isocitrate dehydrogenase/2-oxoglutarate carboxylase;
    AcnB – Aconitate hydratase; PorABDG – Pyruvate synthase; PycAB - Pyruvate carboxylase;
    PpsA – Phosphoenolpyruvate synthase water dikinase; PyK – Pyruvate:water dikinase.
    GLUCONEOGENESIS: Eno – Enolase; Pgm – Phosphoglycerate mutase; Pgk – Phosphoglycerate
    kinase; Gapor – Glyceraldehyde-3-phosphate dehydrogenase; Gapdh – Glyceraldehyde
    3-phosphate dehydrogenase; Fba – Predicted fructose-bisphosphate aldolase; Tim
    – Triosephosphate isomerase; Fbp – Fructose-1,6-bisphosphatase I; Pgi – Phosphoglucose
    isomerase; Pgm – Phosphoglucomutase.DOI: http://dx.doi.org/10.7554/eLife.18990.00710.7554/eLife.18990.008Figure
    3—source data 1.List of the proteins identified in the proteome of T. ammonificans
    grown under nitrate reducing conditions.DOI: http://dx.doi.org/10.7554/eLife.18990.008'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - metV
  - metR
  - hupB
  - eno
  - ppsA
  - pgm
  - napH
  - ompC
  - napC
  - ecpD
  - rtcA
  - napA
  - ATP8A2
  - HADH
  - HADHA
  - IRF1
  - CPSF4
  - ETHE1
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - PAK1
  - ZNF420
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - VCAN
  - CTNNBL1
  - NR1I2
  - JTB
  - PMCH
  - RTCA
  - NAPA
  - NAPSA
  - FANCA
  - TPI1
  - ARHGEF5
  - TIMELESS
  - HAVCR1
  - NDP
  - CHI
  - PGM
  - NAP
  - TIM
  - Nitrate
---
