---
figid: PMC5377334__srep45557-f3
figtitle: Systems approach to characterize the metabolism of liver cancer stem cells
  expressing CD133
organisms:
- Mus musculus
- Rattus norvegicus
- Danio rerio
- Corvus corax
- Homo sapiens
- Bos taurus
- Escherichia coli
- Hepatitis B virus
- NA
pmcid: PMC5377334
filename: srep45557-f3.jpg
figlink: /pmc/articles/PMC5377334/figure/f3/
number: F3
caption: (a) Reconstruction process of LCSC and non-LCSC GEMs. Transcriptome and metabolome
  data were integrated with a generic human GEM, HMR 2.0. During the GEM reconstruction
  process, draft GEMs were examined and validated by simulating the predefined 56
  metabolic tasks under the cultivation condition of RPMI-1640 medium (). The GrowMatch
  gap-filling algorithm was performed to enable the cell type-specific GEMs to demonstrate
  growth. Reactions involved in lipid and amino acid metabolisms were more included
  in LCSC GEMs, whereas keratan sulfate biosynthetic pathway (largely mediated by
  MAN1A1) was more represented in non-LCSC GEMs. (b) Scheme of metabolic simulation
  using flux response analysis of the LCSC and non-LCSC GEMs to predict transcription
  factors regulating biomass generation (i.e., cell growth rate). The mean flux value
  (purple line) of each reaction was used when calculating the correlation between
  each reaction’s flux value and cell growth rate. In the right panel, upon prediction
  of reactions with flux values positively correlated with cell growth rate (red lines).
  Transcription factors (TF) that are known to bind to and regulate genes of the predicted
  reactions were next searched from RegNetwork. (c) Transcription factors that potentially
  regulate genes responsible for the metabolic reactions with flux values positively
  correlated with cell growth rate. Names of the top 10 transcription factors (out
  of 184 transcription factors) are presented.
papertitle: Systems approach to characterize the metabolism of liver cancer stem cells
  expressing CD133.
reftext: Wonhee Hur, et al. Sci Rep. 2017;7:45557.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.564702
figid_alias: PMC5377334__F3
figtype: Figure
organisms_ner:
- Rattus norvegicus
- Homo sapiens
- Bos taurus
- Mus musculus
- Danio rerio
redirect_from: /figures/PMC5377334__F3
ndex: bfef1a3f-df35-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5377334__srep45557-f3.html
  '@type': Dataset
  description: (a) Reconstruction process of LCSC and non-LCSC GEMs. Transcriptome
    and metabolome data were integrated with a generic human GEM, HMR 2.0. During
    the GEM reconstruction process, draft GEMs were examined and validated by simulating
    the predefined 56 metabolic tasks under the cultivation condition of RPMI-1640
    medium (). The GrowMatch gap-filling algorithm was performed to enable the cell
    type-specific GEMs to demonstrate growth. Reactions involved in lipid and amino
    acid metabolisms were more included in LCSC GEMs, whereas keratan sulfate biosynthetic
    pathway (largely mediated by MAN1A1) was more represented in non-LCSC GEMs. (b)
    Scheme of metabolic simulation using flux response analysis of the LCSC and non-LCSC
    GEMs to predict transcription factors regulating biomass generation (i.e., cell
    growth rate). The mean flux value (purple line) of each reaction was used when
    calculating the correlation between each reaction’s flux value and cell growth
    rate. In the right panel, upon prediction of reactions with flux values positively
    correlated with cell growth rate (red lines). Transcription factors (TF) that
    are known to bind to and regulate genes of the predicted reactions were next searched
    from RegNetwork. (c) Transcription factors that potentially regulate genes responsible
    for the metabolic reactions with flux values positively correlated with cell growth
    rate. Names of the top 10 transcription factors (out of 184 transcription factors)
    are presented.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Gem
  - Nr4a1
  - Tf
  - Myc
  - Tfap2a
  - E2f1
  - Rxra
  - Sp1
  - Ar
  - Ppara
  - Th
  - GEM
  - GMNN
  - NR4A1
  - MRC1
  - TF
  - MYC
  - TFAP2A
  - E2F1
  - UBE2L3
  - RXRA
  - SP1
  - PSG1
  - DAND5
  - AR
  - KCNK3
  - PPARA
  - PPARD
  - PPARG
  - Gmnn
  - Nol3
  - Kcnk3
  - gem
  - rem1
  - myca
  - mycbp2
  - tfap2a
  - e2f1
  - rxrga
  - rxrba
  - rxrab
  - sp1
  - ar
  - sulfate
---
