---
figid: PMC5788505__elife-31097-fig4-figsupp2
figtitle: Comparative genomic reconstruction of L-gulonate and related uronic acid
  catabolic pathways and regulons in gammaproteobacteria
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC5788505
filename: elife-31097-fig4-figsupp2.jpg
figlink: /pmc/articles/PMC5788505/figure/fig4s2/
number: F4
caption: (A) Genomic context of L-gulonate, D-glucuronate, and L-galactonate utilization
  genes and regulons in 45 bacteria from the Pasteurellales lineage. The L-gulonate
  catabolic pathway from H. influenzae Rd KW20 was projected onto the pathogenic gammaproteobacteria,
  using the subsystems approach in the SEED genomic platform (). Analysis of the adjacent
  metabolic pathways for use of hexuronates (D-glucuronate, D-galacturonate) and another
  6-carbon aldonic acid (L-galactonate) revealed that the signature enzymes for the
  L-gulonate and the D-glucuronate (GlcA) pathways are the L-gulonate dehydrogenase;
  the GulD and the GlcA isomerases; and UxaC, respectively. Predicted sugar utilization
  phenotypes are given in parentheses. Among 21 analyzed strains of H. influenzae,
  17 strains possess the uronate catabolic genes. Genes with the same functional roles
  are marked in matching colors. Pairwise similarity between orthologs in different
  H. influenzae strains is highlighted by light green (>95% protein identity), yellow
  (>50%), and pink (<50%). Tripartite TRAP transporters are shown by magenta arrows.
  Experimentally defined specificities of TRAP SBPs are indicated for gulP/uxuP/lgoP
  (LGul, L-gulonate, GlcA, D-glucuronate, LGal, L-galactonate), these genes are outlined
  in red and their locus tags are given below. Transcriptional regulators are shown
  by black arrows. Predicted DNA-binding sites of GulR and UxuR regulators are shown
  by black pins, and their common DNA motif is shown as a logo. For reconstruction
  of UxaR and GulR regulons, we used an established comparative genomics approach,
  based on identification of candidate regulator-binding sites using RegPredict tool
  (). LGul pathway gene organization is conserved in 8 out of 21 hr. influenza strains
  into a single genetic locus containing two divergently transcribed operons, gulD-gulPQM-kdgK-uxuB-kdgA
  and gulR-uxuA, that are controlled by candidate GulR-binding sites in their common
  promoter region. Glucuronide hydrolases from different families are shown by white
  arrows. (B) Genomic context of D-glucuronate and D-galacturonate utilization genes
  and regulons in E. coli K-12. (C) Reconstructed metabolic pathways for utilization
  of hexuronic and aldonic acids. Solid and broken arrows indicate enzymatic reactions
  and transport, respectively. The TRAP and MFS transporter families are indicated
  by red stars and squares, respectively. (D) Genomic context of GulD orthologs in
  Enterobacteria. The uncharacterized genes gulT and gulR-II encode a novel MFS-transporter
  and a GntR-family regulator that are likely specific to L-gulonate. (E) Phylogenetic
  tree for selected zinc-dependent dehydrogenases from the COG1063 family. The analyzed
  GulD orthologs from the Pasteurellales and Enterobacteria are 62% identical to each
  other, while within the Pasteurellales lineage all GulDs are >70 identical. The
  previously characterized L-gulonate-specific dehydrogenases from E. coli K-12 and
  Halomonas spp. () belong to the RspB branch, which is distinct from the GulD branch
  established in this work.
papertitle: Prediction of enzymatic pathways by integrative pathway mapping.
reftext: Sara Calhoun, et al. eLife. 2018;7:e31097.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.516447
figid_alias: PMC5788505__F4
figtype: Figure
redirect_from: /figures/PMC5788505__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5788505__elife-31097-fig4-figsupp2.html
  '@type': Dataset
  description: (A) Genomic context of L-gulonate, D-glucuronate, and L-galactonate
    utilization genes and regulons in 45 bacteria from the Pasteurellales lineage.
    The L-gulonate catabolic pathway from H. influenzae Rd KW20 was projected onto
    the pathogenic gammaproteobacteria, using the subsystems approach in the SEED
    genomic platform (). Analysis of the adjacent metabolic pathways for use of hexuronates
    (D-glucuronate, D-galacturonate) and another 6-carbon aldonic acid (L-galactonate)
    revealed that the signature enzymes for the L-gulonate and the D-glucuronate (GlcA)
    pathways are the L-gulonate dehydrogenase; the GulD and the GlcA isomerases; and
    UxaC, respectively. Predicted sugar utilization phenotypes are given in parentheses.
    Among 21 analyzed strains of H. influenzae, 17 strains possess the uronate catabolic
    genes. Genes with the same functional roles are marked in matching colors. Pairwise
    similarity between orthologs in different H. influenzae strains is highlighted
    by light green (>95% protein identity), yellow (>50%), and pink (<50%). Tripartite
    TRAP transporters are shown by magenta arrows. Experimentally defined specificities
    of TRAP SBPs are indicated for gulP/uxuP/lgoP (LGul, L-gulonate, GlcA, D-glucuronate,
    LGal, L-galactonate), these genes are outlined in red and their locus tags are
    given below. Transcriptional regulators are shown by black arrows. Predicted DNA-binding
    sites of GulR and UxuR regulators are shown by black pins, and their common DNA
    motif is shown as a logo. For reconstruction of UxaR and GulR regulons, we used
    an established comparative genomics approach, based on identification of candidate
    regulator-binding sites using RegPredict tool (). LGul pathway gene organization
    is conserved in 8 out of 21 hr. influenza strains into a single genetic locus
    containing two divergently transcribed operons, gulD-gulPQM-kdgK-uxuB-kdgA and
    gulR-uxuA, that are controlled by candidate GulR-binding sites in their common
    promoter region. Glucuronide hydrolases from different families are shown by white
    arrows. (B) Genomic context of D-glucuronate and D-galacturonate utilization genes
    and regulons in E. coli K-12. (C) Reconstructed metabolic pathways for utilization
    of hexuronic and aldonic acids. Solid and broken arrows indicate enzymatic reactions
    and transport, respectively. The TRAP and MFS transporter families are indicated
    by red stars and squares, respectively. (D) Genomic context of GulD orthologs
    in Enterobacteria. The uncharacterized genes gulT and gulR-II encode a novel MFS-transporter
    and a GntR-family regulator that are likely specific to L-gulonate. (E) Phylogenetic
    tree for selected zinc-dependent dehydrogenases from the COG1063 family. The analyzed
    GulD orthologs from the Pasteurellales and Enterobacteria are 62% identical to
    each other, while within the Pasteurellales lineage all GulDs are >70 identical.
    The previously characterized L-gulonate-specific dehydrogenases from E. coli K-12
    and Halomonas spp. () belong to the RspB branch, which is distinct from the GulD
    branch established in this work.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - uxuA
  - kdgK
  - eda
  - uxuB
  - uxaC
  - uxuR
  - uidB
  - uidA
  - uxaA
  - uxaB
  - exuT
  - exuR
  - lgoD
  - rspB
  - F3031
  - L-gulonate
  - D-glucuronate
  - L-gulonate, B-Glucuronides
  - F0397
  - L-galactonate
  - D-galacturonate
  - C Glucuronides
  - D-Glucuronate
  - L-Gulonate
  - L-Galactonate
  - D-Glucuronate L-Gulonate
  - HexA
  - D-Mannonate
  - Gulonate
  - 2-keto-3-deoxy-D-gluconate
  - KDG
  - Glyceraldehyde
  - Pyruvate
  - TRAP
  - Mannheimia haemolytica
  - Haemophilus somnus
  - Mannheimia
  - parahaemolyticus, H.paraphrohaemolyticus
---
