---
figid: PMC5603023__13104_2017_2719_Fig5_HTML
figtitle: Two networks diagrams generated by ingenuity pathway analysis with the 85
  genes
organisms:
- NA
pmcid: PMC5603023
filename: 13104_2017_2719_Fig5_HTML.jpg
figlink: /pmc/articles/PMC5603023/figure/Fig5/
number: F5
caption: Two networks diagrams generated by ingenuity pathway analysis with the 85
  genes. The associated network functions of network 1 (a) were cell-mediated immune
  response, cellular development, and cellular function and maintenance and included
  19 of the 85 genes, with a score of 47, and those of network 2 (b) were gene expression,
  cellular development, and cellular growth and proliferation, and included 15 of
  the 85 genes, with a score of 31. The networks are based on known protein–protein
  interactions and functional relationships. The intensity of gene (node) color in
  the networks indicates the degree of hypermethylation (red) or hypomethylation (gray).
  Nodes are displayed using various shapes that represent the functional class of
  gene products
papertitle: Changes in DNA methylation in naïve T helper cells regulate the pathophysiological
  state in minimal-change nephrotic syndrome.
reftext: Yasuko Kobayashi, et al. BMC Res Notes. 2017;10:480.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8150616
figid_alias: PMC5603023__F5
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
redirect_from: /figures/PMC5603023__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5603023__13104_2017_2719_Fig5_HTML.html
  '@type': Dataset
  description: Two networks diagrams generated by ingenuity pathway analysis with
    the 85 genes. The associated network functions of network 1 (a) were cell-mediated
    immune response, cellular development, and cellular function and maintenance and
    included 19 of the 85 genes, with a score of 47, and those of network 2 (b) were
    gene expression, cellular development, and cellular growth and proliferation,
    and included 15 of the 85 genes, with a score of 31. The networks are based on
    known protein–protein interactions and functional relationships. The intensity
    of gene (node) color in the networks indicates the degree of hypermethylation
    (red) or hypomethylation (gray). Nodes are displayed using various shapes that
    represent the functional class of gene products
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - kay
  - kto
  - oo
  - 'On'
  - act
  - Act5C
  - .na.character
  - Act42A
  - Actbeta
  - Lon
  - schlank
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - galR
  - recE
  - actS
  - lon
  - TRX-CAT1-2
  - GJA8
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - MED12
  - EOMES
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - ANGPT1
  - ANGPT2
  - ANGPT4
  - AREG
  - BDNF
  - CSF1
  - EFNA1
  - EFNA2
  - EFNA3
  - EFNA4
  - EFNA5
  - EGF
  - EREG
  - FGF1
  - FGF10
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF2
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FLT3LG
  - HGF
  - IGF1
  - IGF2
  - INS
  - KITLG
  - NGF
  - NTF3
  - NTF4
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - PGF
  - TGFA
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - LONP1
  - FUT3
---
