---
figid: PMC5578105__ijms-18-01715-g001
figlink: /pmc/articles/PMC5578105/figure/ijms-18-01715-f001/
number: F1
caption: Ubiquitin signaling in both sub-pathways of nucleotide excision repair (NER),
  global genomic (GG-) and Transcription coupled NER (TC-NER), and double-strand break
  (DSB) repair. (A) Ubiquitin signaling at DSB sites. DSBs are recognized by the MRN
  (MRE11–RAD50–NBS1) complex, which recruits and activates the ATM kinase through
  association with its NBS1 subunit. ATM phosphorylates H2AX at S139 (γH2AX), thereby
  generating a binding platform for MDC1 (mediator of DNA damage checkpoint protein
  1). The protein Casein kinase 2 (CK2) (not shown for clarity) constitutively phosphorylates
  residues within MDC1’s SDT domain. Phosphorylated MDC1 is recognized by NBS1, a
  subunit of the aforementioned MRN complex. The MDC1-bound MRN subunit NSB1 attracts
  the ATM kinase, thereby promoting the spreading of γH2AX along the DNA. Phosphorylated
  MDC1 subsequently recruits the ubiquitin E3 ligase RNF8, which promotes K63-linked
  poly-ubiquitylation of histone H1 recognized by RNF168. RNF168, in concert with
  RNF8, ubiquitylates histones H2A and H2AX at K13 and K15 generating recruitment
  platforms for a range of ubiquitin-binding factors. Additionally, RNF168 can also
  recognize ubiquitylated forms of H2A, further supporting the efficient spreading
  of the DNA damage signal along the chromatin. Furthermore, the BMI1-RNF2 (shown
  as RNF2) module regulates the early initiation steps in the DNA damage response
  through association with H2AX and stimulating H2AX mono-ubiquitylation which serves
  as binding platform for ATM; (B) The UV-RING1B complex causes mono-ubiquitylation
  of histone H2A at lysine residue 119 in proximity to the DNA damage site (red star).
  ZRF1 is localized to the damage site both by interaction with XPC and by binding
  to H2A-ubiquitin via its ubiquitin binding domain. Furthermore, ZRF1 mediates remodeling
  of the UV RING1B E3 ligase complex by facilitating the release of the CUL4B-RING1B
  module and the incorporation of the CUL4A-RBX1 module. The newly-formed DDB-CUL4A
  E3 ligase complex (green) catalyzes the poly-ubiquitylation of XPC, which is thereby
  stabilized at the damage site; (C) Ubiquitin signaling events during transcription
  coupled-NER (TC-NER) which either lead to the formation of a pre-incision TC-NER
  complex (upper part), or mediate proteasomal degradation of RNA pol II (lower part).
  Elongating RNA polymerase II is stalled after encountering a DNA lesion (indicated
  as red star). The stalled polymerase serves as a recruiting signal for the Cockayne
  Syndrome B (CSB) protein. CSB further recruits Cockayne Syndrome WD repeat protein
  A (CSA) which is part of a multiprotein complex consisting of DNA damage binding
  protein 1 (DDB1), the scaffold protein CUL4A or CUL4B, respectively, and the E3
  ligase RBX1. The removal of CSB initiated by its poly-ubiquitylation is counteracted
  by the action of the deubiquitinase USP7 which is recruited to stalled RNA pol II
  by UV-stimulated scaffold protein A (UVSSA) and catalyzes the degradation of the
  poly-ubiquitin chain on CSB. An alternative pathway initiates subsequent proteasomal
  degradation of RNA pol II in case TC-NER fails to repair the DNA lesion. The E3
  ubiquitin ligase NEDD4 mediates K63 linked poly-ubiquitylation of the RNA pol II
  component, termed the RNA pol II subunit B1 (RPB1). Poly-ubiquitylated RPB1 is further
  modified by deubiquitinases and by the elongin A (ELOA)-ELOB-ELOC-CUL5-RING-box
  protein 2 (RBX2) E3 ligase complex (not shown), which trigger its extraction by
  the valosin-containing protein (VCP)/p97 ATPase complex followed by proteasomal
  degradation.
pmcid: PMC5578105
papertitle: 'Regulation of DNA Repair Mechanisms: How the Chromatin Environment Regulates
  the DNA Damage Response.'
reftext: Jens Stadler, et al. Int J Mol Sci. 2017 Aug;18(8):1715.
pmc_ranked_result_index: '227620'
pathway_score: 0.7389445
filename: ijms-18-01715-g001.jpg
figtitle: Ubiquitin signaling in both sub-pathways of nucleotide excision repair (NER),
  global genomic (GG-) and Transcription coupled NER (TC-NER), and double-strand break
  (DSB) repair
year: '2017'
organisms: Homo sapiens
ndex: 5a37ed7d-deab-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5578105__ijms-18-01715-g001.html
  '@type': Dataset
  description: Ubiquitin signaling in both sub-pathways of nucleotide excision repair
    (NER), global genomic (GG-) and Transcription coupled NER (TC-NER), and double-strand
    break (DSB) repair. (A) Ubiquitin signaling at DSB sites. DSBs are recognized
    by the MRN (MRE11–RAD50–NBS1) complex, which recruits and activates the ATM kinase
    through association with its NBS1 subunit. ATM phosphorylates H2AX at S139 (γH2AX),
    thereby generating a binding platform for MDC1 (mediator of DNA damage checkpoint
    protein 1). The protein Casein kinase 2 (CK2) (not shown for clarity) constitutively
    phosphorylates residues within MDC1’s SDT domain. Phosphorylated MDC1 is recognized
    by NBS1, a subunit of the aforementioned MRN complex. The MDC1-bound MRN subunit
    NSB1 attracts the ATM kinase, thereby promoting the spreading of γH2AX along the
    DNA. Phosphorylated MDC1 subsequently recruits the ubiquitin E3 ligase RNF8, which
    promotes K63-linked poly-ubiquitylation of histone H1 recognized by RNF168. RNF168,
    in concert with RNF8, ubiquitylates histones H2A and H2AX at K13 and K15 generating
    recruitment platforms for a range of ubiquitin-binding factors. Additionally,
    RNF168 can also recognize ubiquitylated forms of H2A, further supporting the efficient
    spreading of the DNA damage signal along the chromatin. Furthermore, the BMI1-RNF2
    (shown as RNF2) module regulates the early initiation steps in the DNA damage
    response through association with H2AX and stimulating H2AX mono-ubiquitylation
    which serves as binding platform for ATM; (B) The UV-RING1B complex causes mono-ubiquitylation
    of histone H2A at lysine residue 119 in proximity to the DNA damage site (red
    star). ZRF1 is localized to the damage site both by interaction with XPC and by
    binding to H2A-ubiquitin via its ubiquitin binding domain. Furthermore, ZRF1 mediates
    remodeling of the UV RING1B E3 ligase complex by facilitating the release of the
    CUL4B-RING1B module and the incorporation of the CUL4A-RBX1 module. The newly-formed
    DDB-CUL4A E3 ligase complex (green) catalyzes the poly-ubiquitylation of XPC,
    which is thereby stabilized at the damage site; (C) Ubiquitin signaling events
    during transcription coupled-NER (TC-NER) which either lead to the formation of
    a pre-incision TC-NER complex (upper part), or mediate proteasomal degradation
    of RNA pol II (lower part). Elongating RNA polymerase II is stalled after encountering
    a DNA lesion (indicated as red star). The stalled polymerase serves as a recruiting
    signal for the Cockayne Syndrome B (CSB) protein. CSB further recruits Cockayne
    Syndrome WD repeat protein A (CSA) which is part of a multiprotein complex consisting
    of DNA damage binding protein 1 (DDB1), the scaffold protein CUL4A or CUL4B, respectively,
    and the E3 ligase RBX1. The removal of CSB initiated by its poly-ubiquitylation
    is counteracted by the action of the deubiquitinase USP7 which is recruited to
    stalled RNA pol II by UV-stimulated scaffold protein A (UVSSA) and catalyzes the
    degradation of the poly-ubiquitin chain on CSB. An alternative pathway initiates
    subsequent proteasomal degradation of RNA pol II in case TC-NER fails to repair
    the DNA lesion. The E3 ubiquitin ligase NEDD4 mediates K63 linked poly-ubiquitylation
    of the RNA pol II component, termed the RNA pol II subunit B1 (RPB1). Poly-ubiquitylated
    RPB1 is further modified by deubiquitinases and by the elongin A (ELOA)-ELOB-ELOC-CUL5-RING-box
    protein 2 (RBX2) E3 ligase complex (not shown), which trigger its extraction by
    the valosin-containing protein (VCP)/p97 ATPase complex followed by proteasomal
    degradation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DDB2
  - NEDD4
  - DDB1
  - RNF2
  - CUL4B
  - MDC1
  - RBX1
  - DNAJC2
  - RNF168
  - CUL4A
  - USP7
  - UVSSA
  - XPC
  - RNF8
  - POLR2A
  - ATM
genes:
- word: DDB2)
  symbol: DDB2
  source: hgnc_symbol
  hgnc_symbol: DDB2
  entrez: '1643'
- word: NEDD4
  symbol: NEDD4
  source: hgnc_symbol
  hgnc_symbol: NEDD4
  entrez: '4734'
- word: DDB1
  symbol: DDB1
  source: hgnc_symbol
  hgnc_symbol: DDB1
  entrez: '1642'
- word: RING1B
  symbol: RING1B
  source: hgnc_alias_symbol
  hgnc_symbol: RNF2
  entrez: '6045'
- word: (CUL4B
  symbol: CUL4B
  source: hgnc_symbol
  hgnc_symbol: CUL4B
  entrez: '8450'
- word: MDC1)
  symbol: MDC1
  source: hgnc_symbol
  hgnc_symbol: MDC1
  entrez: '9656'
- word: RBX1
  symbol: RBX1
  source: hgnc_symbol
  hgnc_symbol: RBX1
  entrez: '9978'
- word: ZRF1
  symbol: ZRF1
  source: hgnc_prev_symbol
  hgnc_symbol: DNAJC2
  entrez: '27000'
- word: RNF168
  symbol: RNF168
  source: hgnc_symbol
  hgnc_symbol: RNF168
  entrez: '165918'
- word: RNF2
  symbol: RNF2
  source: hgnc_symbol
  hgnc_symbol: RNF2
  entrez: '6045'
- word: CUL4A
  symbol: CUL4A
  source: hgnc_symbol
  hgnc_symbol: CUL4A
  entrez: '8451'
- word: USP7
  symbol: USP7
  source: hgnc_symbol
  hgnc_symbol: USP7
  entrez: '7874'
- word: UVSSA
  symbol: UVSSA
  source: hgnc_symbol
  hgnc_symbol: UVSSA
  entrez: '57654'
- word: XPC
  symbol: XPC
  source: hgnc_symbol
  hgnc_symbol: XPC
  entrez: '7508'
- word: RNF8
  symbol: RNF8
  source: hgnc_symbol
  hgnc_symbol: RNF8
  entrez: '9025'
- word: RPB1
  symbol: RPB1
  source: hgnc_alias_symbol
  hgnc_symbol: POLR2A
  entrez: '5430'
- word: ATM
  symbol: ATM
  source: hgnc_symbol
  hgnc_symbol: ATM
  entrez: '472'
chemicals: []
diseases: []
figid_alias: PMC5578105__F1
redirect_from: /figures/PMC5578105__F1
figtype: Figure
---
