---
figid: PMC5127835__fphar-07-00449-g0002
figtitle: Functional Genomics Identifies Tis21-Dependent Mechanisms and Putative Cancer
  Drug Targets Underlying Medulloblastoma Shh-Type Development
organisms:
- Mus musculus
- Homo sapiens
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC5127835
filename: fphar-07-00449-g0002.jpg
figlink: /pmc/articles/PMC5127835/figure/F2/
number: F2
caption: 'A molecular overview of our Shh-deregulated mouse model, characterized by
  the genetic disruption of hedgehog signaling resulting from a heterozygous mutation
  in the negative regulators Ptch1, whose effects are enhanced by the homozygous deletion
  of the Tis21 tumor suppressor gene. The figure shows the proteins mainly involved
  in neural developmental pathways, encoded by the differentially expressed genes
  of set B and set D and shown in their subcellular compartments. Each differentially
  expressed pathway object is labeled with a thermometer that indicates the gene expression
  changes: downward thermometers have a blue color and indicate down-regulated expression,
  whereas upward thermometers have a red color and indicate up-regulated expression.
  The thermometer number 1 is related to the pairwise comparison Ptch1+/−/Tis21+/+
  vs. wild type Set B, while the thermometer number 2 is related to the pairwise comparison
  Ptch1+/−/Tis21−/− vs. wild type Set D. Signaling cross-talks between developmental
  pathways involved in pre-neoplastic GCPs development are initiated by different
  growth factors and cytokines, the most part of which interacts with G-protein coupled
  receptors. This is supported by a strong up-regulation of many members of heterotrimeric
  G-proteins, their target molecules and regulators (data not shown in figure). Among
  those reported in literature as involved in MB (Roussel and Hatten, ), developmental
  signaling pathways appear to be up-reregulated in our experimental data, according
  to the far fewer percentages of negative signature genes (Chen et al., ) and the
  over-representation of WNT and axonal guidance genes present in human MB Shh-type
  (Northcott et al., ). Interestingly, a β-Catenin-Gli1 balanced interaction has been
  recently reported to regulate Shh-driven MB tumor growth in Ptch1 heterozygous mice
  in vitro (Zinke et al., ), while Sox7, a transcription factor that is known to reduce
  Wnt/β-Catenin stimulated transcription in a dose-dependent manner (Takash et al.,
  ), is up-regulated in Set B and Tis21 ablation enhances its up-regulation in Set
  D (data not shown in figure). Furthermore, Smo-dependent non-canonical Shh pathways
  (Jenkins, ), which have been reported to modulate cytoskeleton-dependent processes
  and fluctuation of Ca2+ through the plasma membrane in mammalian neurons (Brennan
  et al., ) and suggested in possible association with MB (Briscoe and Therond, ),
  are put in light here for the first time as related to the MB Shh-type mouse model.
  Evidences of a deregulated Slit-Robo pathway, which is implicated in neuronal migration
  (Wong et al., ; Marillat et al., ), are present in our data with the up-regulation
  of the axon guidance receptor Robo4. The ligand of Robo4, Slit2, has been linked
  to the inhibition of MB cell invasion (Werbowetski-Ogilvie et al., ). Proteins belonging
  to the ubiquitin-dependent degradation of GCPs cell cycle regulators [24] have their
  genes up-regulated in our model, in particular a number of ubiquitin-conjugating
  enzymes and some constituents of the SCF (Skip1, Cullin1, F-box)-E3 ubiquitin ligase
  complex. Among them, a substrate recognition component of the SCF-type E3 ubiquitin
  ligase, the F-box protein Fbw7, which has been linked to a premature migration of
  GCPs in conditional Fbw7-knockout mice [30]. An up-regulation genes coding for proteins
  involved in palmitoylation (i.e., HHAT) and transport of Shh (i.e., DISP1) is noticed
  in in Set D, where Ptch1 sterol-sensing domain seems to control Smoothened activity
  through Ptch1 vesicular trafficking [34]. Retinoblastoma-associated protein (Rb),
  as well as its downstream effectors E2F3 and E2F5, has its correspondent gene up-regulated
  in set D, where the deregulation of the Rb/E2F tumor suppressor complex in MB Shh-driven
  has been already associated to the E2F1-dependent regulation of lipogenic enzymes
  in primary cerebellar granule neuron precursors (Bhatia et al., ). Figure  below
  shows the set of symbols whereby network objects and interactions between objects
  are indicated in this figure.'
papertitle: Functional Genomics Identifies Tis21-Dependent Mechanisms and Putative
  Cancer Drug Targets Underlying Medulloblastoma Shh-Type Development.
reftext: Giulia Gentile, et al. Front Pharmacol. 2016;7:449.
year: '2016'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9363644
figid_alias: PMC5127835__F2
figtype: Figure
redirect_from: /figures/PMC5127835__F2
ndex: 3a0e3a1a-deb1-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5127835__fphar-07-00449-g0002.html
  '@type': Dataset
  description: 'A molecular overview of our Shh-deregulated mouse model, characterized
    by the genetic disruption of hedgehog signaling resulting from a heterozygous
    mutation in the negative regulators Ptch1, whose effects are enhanced by the homozygous
    deletion of the Tis21 tumor suppressor gene. The figure shows the proteins mainly
    involved in neural developmental pathways, encoded by the differentially expressed
    genes of set B and set D and shown in their subcellular compartments. Each differentially
    expressed pathway object is labeled with a thermometer that indicates the gene
    expression changes: downward thermometers have a blue color and indicate down-regulated
    expression, whereas upward thermometers have a red color and indicate up-regulated
    expression. The thermometer number 1 is related to the pairwise comparison Ptch1+/−/Tis21+/+
    vs. wild type Set B, while the thermometer number 2 is related to the pairwise
    comparison Ptch1+/−/Tis21−/− vs. wild type Set D. Signaling cross-talks between
    developmental pathways involved in pre-neoplastic GCPs development are initiated
    by different growth factors and cytokines, the most part of which interacts with
    G-protein coupled receptors. This is supported by a strong up-regulation of many
    members of heterotrimeric G-proteins, their target molecules and regulators (data
    not shown in figure). Among those reported in literature as involved in MB (Roussel
    and Hatten, ), developmental signaling pathways appear to be up-reregulated in
    our experimental data, according to the far fewer percentages of negative signature
    genes (Chen et al., ) and the over-representation of WNT and axonal guidance genes
    present in human MB Shh-type (Northcott et al., ). Interestingly, a β-Catenin-Gli1
    balanced interaction has been recently reported to regulate Shh-driven MB tumor
    growth in Ptch1 heterozygous mice in vitro (Zinke et al., ), while Sox7, a transcription
    factor that is known to reduce Wnt/β-Catenin stimulated transcription in a dose-dependent
    manner (Takash et al., ), is up-regulated in Set B and Tis21 ablation enhances
    its up-regulation in Set D (data not shown in figure). Furthermore, Smo-dependent
    non-canonical Shh pathways (Jenkins, ), which have been reported to modulate cytoskeleton-dependent
    processes and fluctuation of Ca2+ through the plasma membrane in mammalian neurons
    (Brennan et al., ) and suggested in possible association with MB (Briscoe and
    Therond, ), are put in light here for the first time as related to the MB Shh-type
    mouse model. Evidences of a deregulated Slit-Robo pathway, which is implicated
    in neuronal migration (Wong et al., ; Marillat et al., ), are present in our data
    with the up-regulation of the axon guidance receptor Robo4. The ligand of Robo4,
    Slit2, has been linked to the inhibition of MB cell invasion (Werbowetski-Ogilvie
    et al., ). Proteins belonging to the ubiquitin-dependent degradation of GCPs cell
    cycle regulators [24] have their genes up-regulated in our model, in particular
    a number of ubiquitin-conjugating enzymes and some constituents of the SCF (Skip1,
    Cullin1, F-box)-E3 ubiquitin ligase complex. Among them, a substrate recognition
    component of the SCF-type E3 ubiquitin ligase, the F-box protein Fbw7, which has
    been linked to a premature migration of GCPs in conditional Fbw7-knockout mice
    [30]. An up-regulation genes coding for proteins involved in palmitoylation (i.e.,
    HHAT) and transport of Shh (i.e., DISP1) is noticed in in Set D, where Ptch1 sterol-sensing
    domain seems to control Smoothened activity through Ptch1 vesicular trafficking
    [34]. Retinoblastoma-associated protein (Rb), as well as its downstream effectors
    E2F3 and E2F5, has its correspondent gene up-regulated in set D, where the deregulation
    of the Rb/E2F tumor suppressor complex in MB Shh-driven has been already associated
    to the E2F1-dependent regulation of lipogenic enzymes in primary cerebellar granule
    neuron precursors (Bhatia et al., ). Figure  below shows the set of symbols whereby
    network objects and interactions between objects are indicated in this figure.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Mp
  - Dmp1
  - Apela
  - Tnf
  - Saa
  - Trac
  - Abcg5
  - Notch1
  - te
  - Pla2g7
  - Ptk6
  - ap
  - Nr1i3
  - Casr
  - Cxadr
  - Rcvrn
  - Trim13
  - Arr3
  - Creb3
  - Par2
  - F2rl1
  - Slc52a2
  - Apo
  - Aopep
  - Ptk2
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - APELA
  - TNF
  - LINC-ROR
  - ROR1
  - ROR2
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - SAA1
  - SAA2
  - SAA4
  - TRAC
  - NCOR2
  - PAFAH1B1
  - PLA2G7
  - PTK6
  - CASR
  - CXADR
  - PRKAR1A
  - SPG7
  - NR1I3
  - TRIM13
  - CXADRP1
  - CREB3
  - F2RL1
  - NR1I2
  - SLC52A1
  - AOPEP
  - PTK2
  - EPHB6
  - DNLZ
  - Stat92E
  - mo
  - sni
  - egr
  - Ror
  - .na.character
  - Wnt5
  - Wnt4
  - wg
  - Wnt2
  - sn
  - ca
  - stt
  - Stt3A
  - Stt3B
  - be
  - Dr
  - Me
  - Pa
  - brk
  - car
  - apo
  - Fak
  - hep
  - Sce
---
