---
figid: PMC5424241__zcd9990954730005
figtitle: Leukocyte extravasation signaling pathway
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC5424241
filename: zcd9990954730005.jpg
figlink: /pmc/articles/PMC5424241/figure/F5/
number: F5
caption: Leukocyte extravasation signaling pathway. For all genes, the absolute LFC
  was >1 and the FDR was <0.05. (A) Gene expression heat map of genes in the leukocyte
  extravasation signaling pathway generated by contrasting the challenged and nonchallenged
  birds within each time and line. The contrasts, listed atop each column, were clustered
  by similarity on the basis of genes (blue text) that were upregulated (red), that
  were downregulated (green), and that had no change (gray). Genes that were not DE
  in any of the comparisons were removed from the graph. This pathway was activated
  in all contrasts (orange), as predicted by IPA. (B) The leukocyte extravasation
  signaling pathway in the Fayoumis at 2 dpi. Genes were upregulated (red), downregulated
  (green), or not DE (gray) in the challenged relative to the nonchallenged chickens.
  Genes in white were not found in the data set because they were removed during normalization,
  unrecognized by IPA, or not found in the chicken genome. The shapes of the genes
  represent their function. The genes within a parent node that were DE in any of
  the six contrasts are shown individually next to the parental node and are denoted
  with smaller molecules and labels.
papertitle: Novel Mechanisms Revealed in the Trachea Transcriptome of Resistant and
  Susceptible Chicken Lines following Infection with Newcastle Disease Virus.
reftext: Melissa S. Deist, et al. Clin Vaccine Immunol. 2017 May;24(5):e00027-17.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7261648
figid_alias: PMC5424241__F5
figtype: Figure
redirect_from: /figures/PMC5424241__F5
ndex: ca74187c-df13-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5424241__zcd9990954730005.html
  '@type': Dataset
  description: Leukocyte extravasation signaling pathway. For all genes, the absolute
    LFC was >1 and the FDR was <0.05. (A) Gene expression heat map of genes in the
    leukocyte extravasation signaling pathway generated by contrasting the challenged
    and nonchallenged birds within each time and line. The contrasts, listed atop
    each column, were clustered by similarity on the basis of genes (blue text) that
    were upregulated (red), that were downregulated (green), and that had no change
    (gray). Genes that were not DE in any of the comparisons were removed from the
    graph. This pathway was activated in all contrasts (orange), as predicted by IPA.
    (B) The leukocyte extravasation signaling pathway in the Fayoumis at 2 dpi. Genes
    were upregulated (red), downregulated (green), or not DE (gray) in the challenged
    relative to the nonchallenged chickens. Genes in white were not found in the data
    set because they were removed during normalization, unrecognized by IPA, or not
    found in the chicken genome. The shapes of the genes represent their function.
    The genes within a parent node that were DE in any of the six contrasts are shown
    individually next to the parental node and are denoted with smaller molecules
    and labels.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - WAS
  - VAV1
  - VAV2
  - VAV3
  - TECR
  - TEC
  - MMP7
  - PRKCB
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - RASGRP1
  - RASSF5
  - ITK
  - TUFM
  - AIMP1
  - H3P31
  - PHKA2
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - RAP1A
  - RABGEF1
  - TERF2IP
  - RAP1B
  - RAP1GAP
  - LRPAP1
  - RASA1
  - RGS6
  - ETV5
  - EZR
  - RDX
  - MSN
  - MMP9
  - AGXT
  - CD44
  - RAPGEF3
  - RAPGEF4
  - SIPA1
  - SFTPA1
  - RHOH
  - NCF1
  - ITGAL
  - MGAM
  - ITGA4
  - PPBP
  - CXCR4
  - PRKCE
  - BTK
  - RAC2
  - VCL
  - VASP
  - F11R
  - ITGB2
  - ARHGAP1
  - PPBPP1
  - AFDN
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - ACTN1
  - ACTN2
  - ACTN3
  - ACTN4
  - TXK
  - CTNNB1
  - NCF4
  - JAM2
  - JAM3
  - NOX1
  - FER
  - MAP3K4
  - UNK
  - TIMP1
  - NCF2
  - MMP13
  - CD99
  - PTPN11
  - NR0B2
  - PRRT2
  - PRKCA
  - PRKCD
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - ABL1
  - MTTP
  - DSP-AS1
  - ABL2
  - PXN
  - PXDN
  - MMP11
  - PTK2
  - ICAM1
  - ICAM2
  - ICAM3
  - ICAM4
  - ICAM5
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - FANCA
  - SNU13
  - DLK1
  - MVP
  - MMVP1
  - MMP16
  - CRK
  - MMP2
  - MAPK14
  - MAPK1
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK3
  - MAPK8
  - MAPK9
  - MAPK10
  - EDIL3
  - PAX3
  - CLSTN1
  - CTNNA1
  - CTNNA2
  - CTNNA3
  - CLDN11
  - CALM1
  - CALM2
  - CALM3
  - KRIT1
  - CAMKMT
  - POTEF
  - Cardiomyopathy
---
