---
figid: PMC5496883__41598_2017_4027_Fig3_HTML
figtitle: Signalling by Rho GTPases (upper panel) and Neuronal system pathway (bottom
  panel)
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC5496883
filename: 41598_2017_4027_Fig3_HTML.jpg
figlink: /pmc/articles/PMC5496883/figure/Fig3/
number: F3
caption: Signalling by Rho GTPases (upper panel) and Neuronal system pathway (bottom
  panel). These Reactome pathways show statistically significant enrichment (Fisher
  Exact test) of DEGs both in “Night” (a, upper panel left, q-value < 0.004; a, bottom
  panel left, q-value = 0.007) and “Pooled” (b, upper panel right, q-value < 0.001;
  b, bottom panel right, q-value < 0.001) conditions analysed with Graphite web tool.
  Coloured nodes represent the differentially expressed genes. The colour of the nodes
  is proportional to their expression levels represented as log2 (dSur KD/control).
  Raw p-values were adjusted using Benjamini-Hockberg method (q-value).
papertitle: 'Identifying pathways modulating sleep duration: from genomics to transcriptomics.'
reftext: Karla V. Allebrandt, et al. Sci Rep. 2017;7:4555.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7671785
figid_alias: PMC5496883__F3
figtype: Figure
redirect_from: /figures/PMC5496883__F3
ndex: 073143cf-df2e-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5496883__41598_2017_4027_Fig3_HTML.html
  '@type': Dataset
  description: Signalling by Rho GTPases (upper panel) and Neuronal system pathway
    (bottom panel). These Reactome pathways show statistically significant enrichment
    (Fisher Exact test) of DEGs both in “Night” (a, upper panel left, q-value < 0.004;
    a, bottom panel left, q-value = 0.007) and “Pooled” (b, upper panel right, q-value < 0.001;
    b, bottom panel right, q-value < 0.001) conditions analysed with Graphite web
    tool. Coloured nodes represent the differentially expressed genes. The colour
    of the nodes is proportional to their expression levels represented as log2 (dSur
    KD/control). Raw p-values were adjusted using Benjamini-Hockberg method (q-value).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RhoGAP92B
  - GEFmeso
  - RhoGAPp190
  - Rlip
  - RhoGDI
  - tum
  - hop
  - RhoGAP54D
  - pbl
  - nan
  - Oct-TyrR
  - RhoGEF2
  - Rho1
  - rho
  - Dap160
  - Mtl
  - RhoU
  - Graf
  - Vav
  - RhoGAP5A
  - RhoGAP68F
  - RhoGAP93B
  - RhoGAP1A
  - RhoGAP100F
  - cv-c
  - Rac2
  - RhoBTB
  - Cdc42
  - Rac1
  - Akt
  - CG42674
  - Sos
  - conu
  - RhoGAP102A
  - rin
  - CG30440
  - CG43102
  - Exn
  - sif
  - SIFa
  - Cip4
  - Shaw
  - GABA-B-R1
  - Shal
  - se
  - DCTN5-p25
  - GABA-B-R2
  - Rdl
  - Gabat
  - Sh
  - eag
  - eg
  - Hk
  - Shab
  - AGER
  - ARHGDIA
  - ARHGDIB
  - ARHGDIG
  - SMR3A
  - PBRM1
  - APOBEC3C
  - MT1A
  - MT1B
  - MT1E
  - MT1F
  - MT1G
  - MT1H
  - MT1JP
  - MT1M
  - MT1L
  - MT1X
  - MTNR1A
  - ALG1
  - MT1IP
  - ARHGAP26
  - VAV1
  - VAV2
  - VAV3
  - ARHGAP1
  - RAC2
  - CDC42
  - AKT1
  - RAC1
  - RAC3
  - RHOG
  - XYLT2
  - SOS1
  - SOS2
  - RIT2
  - TRIP10
  - GABBR1
  - GABBR2
  - TRK-CTT2-3
  - KCNH1
  - KCNQ5
  - ABCC9
  - KCNJ6
  - KCNO
  - GABA
---
