---
figid: PMC5931277__jove-132-56926-2
figlink: pmc/articles/PMC5931277/figure/F3/
number: F3
caption: Decarboxylation is a common reaction in metabolism. In particular, amino
  acid DCs (EC 4.1.1) are cofactor-free (pyruvoyl-dependent) or PLP-dependent enzymes,
  and catalyze the decarboxylation of amino acids into the corresponding polyamines
  in bacteria and higher organisms. The mono- and dihydroxy compounds () 4-7, 10-11
  correspond to hydroxylated cadaverine, the diamine obtained by decarboxylation of
  L-lysine. Cadaverine is a key building block for the chemical industry, specifically
  it is a component of polyamide and polyurethane polymers. Therefore, bio-based production
  of this diamine from renewable resources has attracted attention as an alternative
  to the petroleum-based route, and various microorganisms have been engineered for
  this purpose. In these metabolic pathways, lysine DC (LDC) is the key enzyme. LDC
  is a PLP-dependent enzyme belonging to the alanine racemase (AR) structural family.
  The PLP-dependent DCs (PLP-DCs) are known to be highly substrate-specific. However,
  a few enzymes own the capability of slight promiscuity, being active towards both
  L-ornithine and L-lysine amino acids, as for example the LDC from Selenomonas rumirantium
  (LDCSrum), which has similar kinetic constants for lysine and ornithine decarboxylation.
  This extended substrate specificity makes this enzyme a good candidate for the decarboxylation
  of mono- and di-hydroxy-L-lysine. In addition, to find DCs active towards the hydroxyl
  derivatives of lysine, we examined the genomic context of the genes encoding the
  αKAO enzymes. Indeed, in prokaryotic genomes the genes encoding enzymes involved
  in the same biosynthetic pathway are generally co-localized in gene clusters. The
  KDO2 (from Chitinophaga pinensis) gene was found co-localized with a gene encoding
  putative PLP-DC (). In contrast, no gene encoding for DC has been found when analyzing
  the genomic context of the KDO1 dioxygenase. The PLP-DC protein from C. pinensis
  (DCCpin) was therefore selected as a promising candidate to catalyze the decarboxylation
  step of the cascade reaction.
pmcid: PMC5931277
papertitle: Enzymatic Cascade Reactions for the Synthesis of Chiral Amino Alcohols
  from L-lysine.
reftext: Aurélie Fossey-Jouenne, et al. J Vis Exp. 2018;(132):56926.
pmc_ranked_result_index: '44407'
pathway_score: 0.7458519
filename: jove-132-56926-2.jpg
figtitle: Enzymatic Cascade Reactions for the Synthesis of Chiral Amino Alcohols from
  L-lysine
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5931277__jove-132-56926-2.html
  '@type': Dataset
  description: Decarboxylation is a common reaction in metabolism. In particular,
    amino acid DCs (EC 4.1.1) are cofactor-free (pyruvoyl-dependent) or PLP-dependent
    enzymes, and catalyze the decarboxylation of amino acids into the corresponding
    polyamines in bacteria and higher organisms. The mono- and dihydroxy compounds
    () 4-7, 10-11 correspond to hydroxylated cadaverine, the diamine obtained by decarboxylation
    of L-lysine. Cadaverine is a key building block for the chemical industry, specifically
    it is a component of polyamide and polyurethane polymers. Therefore, bio-based
    production of this diamine from renewable resources has attracted attention as
    an alternative to the petroleum-based route, and various microorganisms have been
    engineered for this purpose. In these metabolic pathways, lysine DC (LDC) is the
    key enzyme. LDC is a PLP-dependent enzyme belonging to the alanine racemase (AR)
    structural family. The PLP-dependent DCs (PLP-DCs) are known to be highly substrate-specific.
    However, a few enzymes own the capability of slight promiscuity, being active
    towards both L-ornithine and L-lysine amino acids, as for example the LDC from
    Selenomonas rumirantium (LDCSrum), which has similar kinetic constants for lysine
    and ornithine decarboxylation. This extended substrate specificity makes this
    enzyme a good candidate for the decarboxylation of mono- and di-hydroxy-L-lysine.
    In addition, to find DCs active towards the hydroxyl derivatives of lysine, we
    examined the genomic context of the genes encoding the αKAO enzymes. Indeed, in
    prokaryotic genomes the genes encoding enzymes involved in the same biosynthetic
    pathway are generally co-localized in gene clusters. The KDO2 (from Chitinophaga
    pinensis) gene was found co-localized with a gene encoding putative PLP-DC ().
    In contrast, no gene encoding for DC has been found when analyzing the genomic
    context of the KDO1 dioxygenase. The PLP-DC protein from C. pinensis (DCCpin)
    was therefore selected as a promising candidate to catalyze the decarboxylation
    step of the cascade reaction.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SR-hydroxy-L-lysine
  - lysine
  - (SR)-hydroxy-L-lysine
  - L-lysine
genes: []
chemicals:
- word: SR-hydroxy-L-lysine
  source: MESH
  identifier: C114808
- word: lysine
  source: MESH
  identifier: C114808
- word: (SR)-hydroxy-L-lysine
  source: MESH
  identifier: C114808
- word: L-lysine
  source: MESH
  identifier: C114808
diseases: []
figid_alias: PMC5931277__F3
redirect_from: /figures/PMC5931277__F3
figtype: Figure
---
