---
figid: PMC5714215__gkx860fig3
figtitle: Changes in expression of genes involved in nitrogen assimilation and carbon
  metabolism
organisms:
- NA
pmcid: PMC5714215
filename: gkx860fig3.jpg
figlink: /pmc/articles/PMC5714215/figure/F3/
number: F3
caption: Changes in expression of genes involved in nitrogen assimilation and carbon
  metabolism. This diagram shows metabolic pathways and metabolites of the TCA and
  GS/glutamine oxoglutarate amidotransferase (GOGAT) cycle, glycolysis, gluconeogenesis,
  pentose phosphate pathway (OPP) and glycogen metabolism. These pathways were predicted
  from the KEGG database (www.genome.jp/kegg/pathway.html). NtcA target genes obtained
  from ChIP-seq analysis are shown for −N (green square) or NH4 (red square) treatments.
  Genes encoding metabolic enzymes are shown below; narB, nitrate reductase; nirA,
  nitrite reductase, gifA, GS-inactivating factor; gifB, GS-inactivating factor; glnA,
  GS type I; glnN, GS type III; gltD, glutamate synthase small subunit; gltB, GOGAT;
  glsF, ferredoxin-dependent glutamate synthase, gdhA, glutamate dehydrogenase; icd,
  isocitrate dehydrogenase; acnB, aconitate hydratase; citH, malate dehydrogenase;
  gltA, citrate synthase; odhB, pyruvate dehydrogenase; pyk1–2, pyruvate kinase; eno,
  enolase; ppc, phosphoenolpyruvate (PEP) carboxylase; gpmB, phosphoglycerate mutase;
  pgk phosphoglycerate kinase; gap1, glyceraldehyde-3-phosphate dehydrogenase (catabolic
  reaction); gap2, glyceraldehyde-3-phosphate dehydrogenase (anabolic reaction); tpi,
  glycogen isoamylase; cbbA, fructose-bisphosphate aldolase; fbp, fructose-1,6-bisphosphatase;
  pgl, 6-phosphogluconolactonase; pgm, phosphoglucomutase; glgC glucose-1-phosphate
  adenylyltransferase; glgP, glycogen phosphorylase; glgX, glycogen isoamylase; glgA,
  glycogen synthase; glgB, 1,4-alpha-glucan branching enzyme; zwf, glucose-6-phosphate
  dehydrogenase; gnd, 6-phosphogluconate dehydrogenase; tktA, transketolase; rpiA
  or B, ribose-5-phosphate isomerase; rpe, pentose-5-phosphate-3-epimerase; prk, phosphoribulokinase;
  rbcL, ribulose bisphosphate carboxylase large subunit; rbcS, ribulose bisphosphate
  carboxylase small subunit.
papertitle: Identification of the direct regulon of NtcA during early acclimation
  to nitrogen starvation in the cyanobacterium Synechocystis sp. PCC 6803.
reftext: Joaquín Giner-Lamia, et al. Nucleic Acids Res. 2017 Nov 16;45(20):11800-11820.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8855349
figid_alias: PMC5714215__F3
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
redirect_from: /figures/PMC5714215__F3
ndex: 85efabbb-df17-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5714215__gkx860fig3.html
  '@type': Dataset
  description: Changes in expression of genes involved in nitrogen assimilation and
    carbon metabolism. This diagram shows metabolic pathways and metabolites of the
    TCA and GS/glutamine oxoglutarate amidotransferase (GOGAT) cycle, glycolysis,
    gluconeogenesis, pentose phosphate pathway (OPP) and glycogen metabolism. These
    pathways were predicted from the KEGG database (www.genome.jp/kegg/pathway.html).
    NtcA target genes obtained from ChIP-seq analysis are shown for −N (green square)
    or NH4 (red square) treatments. Genes encoding metabolic enzymes are shown below;
    narB, nitrate reductase; nirA, nitrite reductase, gifA, GS-inactivating factor;
    gifB, GS-inactivating factor; glnA, GS type I; glnN, GS type III; gltD, glutamate
    synthase small subunit; gltB, GOGAT; glsF, ferredoxin-dependent glutamate synthase,
    gdhA, glutamate dehydrogenase; icd, isocitrate dehydrogenase; acnB, aconitate
    hydratase; citH, malate dehydrogenase; gltA, citrate synthase; odhB, pyruvate
    dehydrogenase; pyk1–2, pyruvate kinase; eno, enolase; ppc, phosphoenolpyruvate
    (PEP) carboxylase; gpmB, phosphoglycerate mutase; pgk phosphoglycerate kinase;
    gap1, glyceraldehyde-3-phosphate dehydrogenase (catabolic reaction); gap2, glyceraldehyde-3-phosphate
    dehydrogenase (anabolic reaction); tpi, glycogen isoamylase; cbbA, fructose-bisphosphate
    aldolase; fbp, fructose-1,6-bisphosphatase; pgl, 6-phosphogluconolactonase; pgm,
    phosphoglucomutase; glgC glucose-1-phosphate adenylyltransferase; glgP, glycogen
    phosphorylase; glgX, glycogen isoamylase; glgA, glycogen synthase; glgB, 1,4-alpha-glucan
    branching enzyme; zwf, glucose-6-phosphate dehydrogenase; gnd, 6-phosphogluconate
    dehydrogenase; tktA, transketolase; rpiA or B, ribose-5-phosphate isomerase; rpe,
    pentose-5-phosphate-3-epimerase; prk, phosphoribulokinase; rbcL, ribulose bisphosphate
    carboxylase large subunit; rbcS, ribulose bisphosphate carboxylase small subunit.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pgk
  - eno
  - tktA
  - tpiA
  - ppc
  - gnd
  - fbp
  - pgl
  - pgi
  - gltA
  - zwf
  - pgm
  - icd
  - glgC
  - gltB
  - gltD
  - glgP
  - glgX
  - glgA
  - glgB
  - fnr
  - mobA
  - Ribulose
  - Ribulose-1,5P
  - Dihydroxyacetone
  - Fructose
  - Fructose-6P
  - Glucose
  - Glutamine
  - Glyceraldehyde-3P
  - isocitrate
  - Phosphoenolpyruvate
  - Pyruvate
  - Ribose-5P
  - Glutamate
---
