---
figid: PMC5811630__sys0011821800007
figtitle: Iron starvation responses
organisms:
- Chlamydia trachomatis
- Homo sapiens
organisms_ner:
- Homo sapiens
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC5811630
filename: sys0011821800007.jpg
figlink: /pmc/articles/PMC5811630/figure/fig7/
number: F7
caption: Pathway analysis of iron starvation responses. (A and B) Association networks
  for differentially expressed genes, with a P value of ≤0.05 and a minimum of 10
  mapped reads, were generated for the early-cycle response (6 + 3 h BPDL) (A) and
  midcycle response (12 + 3 h BPDL) (B) using STRING-db v.10.5. The thickness of the
  lines connecting the nodes (genes) correlates with confidence of gene association,
  with a minimum confidence cutoff value of 0.7. (C and D) Midcycle (12 + 3 h BPDL
  or 12 + 6 h BPDL) clustered genes were mapped to nucleotide metabolism (C) and acetyl-CoA
  synthesis (D) pathways using KEGGMapper v.2.8. Data corresponding to upregulated
  and downregulated genes in panels C and D are shown with green and red backgrounds,
  respectively. Data corresponding to unchanged genes have a white background.
papertitle: Genomewide Transcriptional Responses of Iron-Starved Chlamydia trachomatis
  Reveal Prioritization of Metabolic Precursor Synthesis over Protein Translation.
reftext: Amanda J. Brinkworth, et al. mSystems. 2018 Jan-Feb;3(1):e00184-17.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8878221
figid_alias: PMC5811630__F7
figtype: Figure
redirect_from: /figures/PMC5811630__F7
ndex: 4154a282-def9-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5811630__sys0011821800007.html
  '@type': Dataset
  description: Pathway analysis of iron starvation responses. (A and B) Association
    networks for differentially expressed genes, with a P value of ≤0.05 and a minimum
    of 10 mapped reads, were generated for the early-cycle response (6 + 3 h BPDL)
    (A) and midcycle response (12 + 3 h BPDL) (B) using STRING-db v.10.5. The thickness
    of the lines connecting the nodes (genes) correlates with confidence of gene association,
    with a minimum confidence cutoff value of 0.7. (C and D) Midcycle (12 + 3 h BPDL
    or 12 + 6 h BPDL) clustered genes were mapped to nucleotide metabolism (C) and
    acetyl-CoA synthesis (D) pathways using KEGGMapper v.2.8. Data corresponding to
    upregulated and downregulated genes in panels C and D are shown with green and
    red backgrounds, respectively. Data corresponding to unchanged genes have a white
    background.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CMPK2
  - PNP
  - PPY
  - DCD
  - SLC25A1
  - CUX1
  - NPPA
  - CXCL9
  - CMPK1
  - MATN1
  - AATF
  - DUT
  - TTPA
  - UEVLD
  - PDHA1
  - PDHB
  - HMGA2
  - LPA
  - PAEP
  - PREP
  - PTPN22
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - TRNA
  - OFCC1
  - ADA2
  - SPAG9
  - ndk
  - udp
  - pyrH
  - pnp
  - dcd
  - rna
  - cmk
  - nrdA
  - nrdB
  - dut
  - dnaQ
  - tmk
  - pykF
  - eno
  - glnU
  - dnaA
  - polA
  - pheT
  - UTP
  - UDP
  - CDP
  - CMP
  - dCDP
  - DUDP
  - Acetyl-CoA
  - S-Acotyldihidro
  - ethyl-THPP
  - Pyruvate
  - amide
  - Oxaloacetato
  - lysine
  - LPA
  - octanoyl
  - TCA CYCLE
  - 2-Phosphate
  - AMINO-ACYL
---
