---
figid: PMC5264521__fmicb-08-00040-g0004
figtitle: Hydrogenophaga b174 cell diagram representing selected catabolic and energy
  conservation pathways active during the study
organisms:
- metagenome
- Homo sapiens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC5264521
filename: fmicb-08-00040-g0004.jpg
figlink: /pmc/articles/PMC5264521/figure/F4/
number: F4
caption: 'Hydrogenophaga b174 cell diagram representing selected catabolic and energy
  conservation pathways active during the study. This reconstruction is a composite
  of pathways that were active at any measured point during the 20-day study. Genes
  that were not expressed, or for which the gene was not identified in the metagenome
  and therefore no expression level could be assigned, are represented in the diagram
  as white and black, respectively. Protein complexes (e.g., ETC and hydrogenases)
  are represented in the diagram with a color gradient if not all subunit genes were
  expressed. Denitrification: 1, nitrate/nitrite transporters (narK1K2); 2, nitrate
  reductase (narGH); 3, nitrite reductase (nirS); 4, nitric oxide reductase (norB);
  5, nitrous oxide reductase (nosZ); 6, assimilatory nitrate reductase (nasAB); 7,
  assimilatory nitrite reductase (nirB); 8, anaerobic nitric oxide reductase flavoredoxin
  (norV); Amino acid degradation: 9, amino acid transporters; 10, arginine deiminase;
  11, ornithine carbamoyltransferase; 12, carbamate kinase; 13, arginine decarboxylase;
  14, agmatinase; 15, arginase; 16, urease (ureABC); 17, ornithine cyclodeaminase;
  18, bifunctional proline dehydrogenase/L-glutamate gamma-semialdehyde dehydrogenase;
  19, histidine ammonia-lyase; 20, urocanase; 21 imidazolonepropionase; 22, formimidoylglutamase;
  23, glutamate dehydrogenase; 24, aspartate aminotransferase; β-oxidation: 25, Fatty
  acid transporters; 26, acyl-CoA dehydrogenase; 27, enoyl-CoA hydratase; 28, 3-hydroxyacyl-CoA
  dehydrogenase; 29; acetyl-CoA acetyltransferase (thiolase); Plant biomass degradation:
  30, sugar transporters; 31, xylose isomerase; 32, xylulokinase; 33, tannase and
  feruloyl esterase; 34, feruloyl-CoA synthase; 35, vanillin synthase /trans-feruloyl-CoA
  hydratase; 36, vanillin dehydrogenase; 37, pyrogallol hydroxytransferase large subunit;
  38, gallate dioxygenase; 39, 2-pyrone-4,6-dicarboxylate hydrolase; 40, 4-oxalomesaconate
  hydratase; 41, 4-carboxy-4-hydroxy-2-oxoadipate aldolase/oxaloacetate decarboxylase;
  Glycolysis: 42, glucokinase; 43, glucose-6-phosphate isomerase; 44, 6-phosphofructokinase;
  45, fructose-1,6-bisphosphate aldolase; 46, triose-phosphate isomerase; 47, type
  I glyceraldehyde-3-phosphate dehydrogenase; 48, phosphoglycerate kinase; 49, phosphoglycerate
  mutase; 50, enolase; 51, pyruvate kinase; Entner-Doudoroff Pathway: 52, glucose-6-phosphate
  dehydrogenase; 53, 6-phosphogluconolactonase; 54, phosphogluconate dehydratase;
  55, 2-keto-3-deoxy-6-phosphogluconate aldolase; TCA Cycle: 56, pyruvate dehydrogenase
  (acetyl-transferring) (aceE); 57, citrate (Si)-synthase (gltA); 58, aconitate hydratase
  (acnAB); 59, isocitrate dehydrogenase (NADP+) (icdA); 60, 2-oxoglutarate dehydrogenase
  subunit E1 (sucA); 61, succinate-CoA ligase subunit, beta (sucCD); 62, succinate
  dehydrogenase flavoprotein subunit (sdhABCD); 63, fumarate hydratase (fumCA); 64,
  malate dehydrogenase (mdh); CBB Cycle: 65, bicarbonate transporters; 66, ribulose-1,5-bisphosphate
  carboxylase/oxygenase (RubisCO) (rbcLSQO); 67, phosphoglycerate kinase; 68, glyceraldehyde-3-phosphate
  dehydrogenase; 69, triose-phosphate isomerase; 70, fructose-bisphosphate aldolase;
  71, fructose-1,6-bisphosphatase class 1; 72, transketolase; 73, ribulose-phosphate
  3-epimerase; 74, phosphoribulokinase; 75, “aerobic” carbon monoxide dehydrogenase
  (coxLSM); S oxidation: 76, S-compound transporters; 77, rhodanese; 78, sulfite dehydrogenase;
  79, S oxidation genes (soxCD, cytochrome c552, cytochrome c551, soxYZAXBR); H2oxidation:
  80, [NiFe] hydrogenase (hupLU) and accessory/maturation proteins; electron transport
  chain (ETC)/oxidative phosphorylation: 81, ETC complex I—NADH:ubiquinone oxidoreductase
  (nuoABCDEFGHIJKLMN); 82, ETC complex II—succinate dehydrogenase (sdhCD, sdhAB);
  83, ETC complex III—cytochrome bc1-type ubiquinol oxidoreductase (petAB); 84, ETC
  complex IV—cbb3-type cytochrome c oxidase (ccoPQON); 85, ETC complex V—ATP synthase
  (atpCDIBEFHAG).'
papertitle: Metatranscriptomic Analysis Reveals Unexpectedly Diverse Microbial Metabolism
  in a Biogeochemical Hot Spot in an Alluvial Aquifer.
reftext: Talia N. M. Jewell, et al. Front Microbiol. 2017;8:40.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.880886
figid_alias: PMC5264521__F4
figtype: Figure
redirect_from: /figures/PMC5264521__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5264521__fmicb-08-00040-g0004.html
  '@type': Dataset
  description: 'Hydrogenophaga b174 cell diagram representing selected catabolic and
    energy conservation pathways active during the study. This reconstruction is a
    composite of pathways that were active at any measured point during the 20-day
    study. Genes that were not expressed, or for which the gene was not identified
    in the metagenome and therefore no expression level could be assigned, are represented
    in the diagram as white and black, respectively. Protein complexes (e.g., ETC
    and hydrogenases) are represented in the diagram with a color gradient if not
    all subunit genes were expressed. Denitrification: 1, nitrate/nitrite transporters
    (narK1K2); 2, nitrate reductase (narGH); 3, nitrite reductase (nirS); 4, nitric
    oxide reductase (norB); 5, nitrous oxide reductase (nosZ); 6, assimilatory nitrate
    reductase (nasAB); 7, assimilatory nitrite reductase (nirB); 8, anaerobic nitric
    oxide reductase flavoredoxin (norV); Amino acid degradation: 9, amino acid transporters;
    10, arginine deiminase; 11, ornithine carbamoyltransferase; 12, carbamate kinase;
    13, arginine decarboxylase; 14, agmatinase; 15, arginase; 16, urease (ureABC);
    17, ornithine cyclodeaminase; 18, bifunctional proline dehydrogenase/L-glutamate
    gamma-semialdehyde dehydrogenase; 19, histidine ammonia-lyase; 20, urocanase;
    21 imidazolonepropionase; 22, formimidoylglutamase; 23, glutamate dehydrogenase;
    24, aspartate aminotransferase; β-oxidation: 25, Fatty acid transporters; 26,
    acyl-CoA dehydrogenase; 27, enoyl-CoA hydratase; 28, 3-hydroxyacyl-CoA dehydrogenase;
    29; acetyl-CoA acetyltransferase (thiolase); Plant biomass degradation: 30, sugar
    transporters; 31, xylose isomerase; 32, xylulokinase; 33, tannase and feruloyl
    esterase; 34, feruloyl-CoA synthase; 35, vanillin synthase /trans-feruloyl-CoA
    hydratase; 36, vanillin dehydrogenase; 37, pyrogallol hydroxytransferase large
    subunit; 38, gallate dioxygenase; 39, 2-pyrone-4,6-dicarboxylate hydrolase; 40,
    4-oxalomesaconate hydratase; 41, 4-carboxy-4-hydroxy-2-oxoadipate aldolase/oxaloacetate
    decarboxylase; Glycolysis: 42, glucokinase; 43, glucose-6-phosphate isomerase;
    44, 6-phosphofructokinase; 45, fructose-1,6-bisphosphate aldolase; 46, triose-phosphate
    isomerase; 47, type I glyceraldehyde-3-phosphate dehydrogenase; 48, phosphoglycerate
    kinase; 49, phosphoglycerate mutase; 50, enolase; 51, pyruvate kinase; Entner-Doudoroff
    Pathway: 52, glucose-6-phosphate dehydrogenase; 53, 6-phosphogluconolactonase;
    54, phosphogluconate dehydratase; 55, 2-keto-3-deoxy-6-phosphogluconate aldolase;
    TCA Cycle: 56, pyruvate dehydrogenase (acetyl-transferring) (aceE); 57, citrate
    (Si)-synthase (gltA); 58, aconitate hydratase (acnAB); 59, isocitrate dehydrogenase
    (NADP+) (icdA); 60, 2-oxoglutarate dehydrogenase subunit E1 (sucA); 61, succinate-CoA
    ligase subunit, beta (sucCD); 62, succinate dehydrogenase flavoprotein subunit
    (sdhABCD); 63, fumarate hydratase (fumCA); 64, malate dehydrogenase (mdh); CBB
    Cycle: 65, bicarbonate transporters; 66, ribulose-1,5-bisphosphate carboxylase/oxygenase
    (RubisCO) (rbcLSQO); 67, phosphoglycerate kinase; 68, glyceraldehyde-3-phosphate
    dehydrogenase; 69, triose-phosphate isomerase; 70, fructose-bisphosphate aldolase;
    71, fructose-1,6-bisphosphatase class 1; 72, transketolase; 73, ribulose-phosphate
    3-epimerase; 74, phosphoribulokinase; 75, “aerobic” carbon monoxide dehydrogenase
    (coxLSM); S oxidation: 76, S-compound transporters; 77, rhodanese; 78, sulfite
    dehydrogenase; 79, S oxidation genes (soxCD, cytochrome c552, cytochrome c551,
    soxYZAXBR); H2oxidation: 80, [NiFe] hydrogenase (hupLU) and accessory/maturation
    proteins; electron transport chain (ETC)/oxidative phosphorylation: 81, ETC complex
    I—NADH:ubiquinone oxidoreductase (nuoABCDEFGHIJKLMN); 82, ETC complex II—succinate
    dehydrogenase (sdhCD, sdhAB); 83, ETC complex III—cytochrome bc1-type ubiquinol
    oxidoreductase (petAB); 84, ETC complex IV—cbb3-type cytochrome c oxidase (ccoPQON);
    85, ETC complex V—ATP synthase (atpCDIBEFHAG).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - CycE
  - cyc
  - so
  - Coa
  - vanillin
  - 2-pyrone-4,6-dicarboxylate
  - L-arginine
  - L-aspartate
  - Bicarbonate
  - fatty acids
  - glucose
  - L-histidine
  - Nitrate
  - phosphate
  - L-proline
  - putrescine
  - pyrogallol
  - pyruvate
  - urea
  - L-glutamate
---
