---
figid: PMC5408482__12870_2017_1020_Fig4_HTML
figtitle: Metabolite and transcript pathway network analysis of soybean responses
  to R
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Glycine max
- Solanum lycopersicum
- Embryophyta
- Solanum tuberosum
- Triticum aestivum
- Hordeum vulgare
- Vitis vinifera
- Phytophthora infestans
- Ustilago maydis
- Panicum miliaceum
- Sclerotinia sclerotiorum
- Rhizoctonia solani
- Fulvia fulva
- Blumeria graminis
- Vitis labrusca
- Zymoseptoria tritici
- Phytophthora ramorum
- Colletotrichum lindemuthianum
- Salix purpurea
- Rhizoctonia solani AG-1
- Rhizoctonia solani AG-1 IA
- Homo sapiens
- Venustaconcha ellipsiformis
- Pseudomonas syringae
pmcid: PMC5408482
filename: 12870_2017_1020_Fig4_HTML.jpg
figlink: /pmc/articles/PMC5408482/figure/Fig4/
number: F4
caption: Metabolite and transcript pathway network analysis of soybean responses to
  R. solani AG1-IA. Gene-to-metabolite and metabolite-to-gene network fluctuations
  in soybean leaves in response to R. solani AG1-IA infection 12 and 24 h post-inoculation
  (h.p.i.). Metabolite fold changes are represented in bar graphs, and significant
  changes based on PLS-DA loading coefficients are indicated by an asterisk (P < 0.05).
  Transcripts are represented as arrows connecting metabolites. Transcript fold-changes
  (1.5 fold threshold; P < 0.05) based on qRT-PCR results are represented by boxes
  with 12 h.p.i. and 24 h.p.i. time-points represented on the left and right, respectively.
  Red illustrates an increase in transcript abundance in response to R. solani AG1-IA
  infection, while hatched green represents a decrease (P < 0.05 and fold change >1.5
  or <−1.5). Capital lettering beside the boxes indicates gene name abbreviations
  (see Table ). Black arrows without boxes represent transcripts detected in RNAseq,
  but not significantly affected by infection. Solid arrows represent single-step
  reactions, while hashed arrows represent multi-step reactions. Stars represent transcripts
  and metabolite bins that were identified as significantly altered using O2PLS analytical
  methods
papertitle: An integrated RNAseq-1H NMR metabolomics approach to understand soybean
  primary metabolism regulation in response to Rhizoctonia foliar blight disease.
reftext: Tanya R. Copley, et al. BMC Plant Biol. 2017;17:84.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9456624
figid_alias: PMC5408482__F4
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC5408482__F4
ndex: dee50d05-def8-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5408482__12870_2017_1020_Fig4_HTML.html
  '@type': Dataset
  description: Metabolite and transcript pathway network analysis of soybean responses
    to R. solani AG1-IA. Gene-to-metabolite and metabolite-to-gene network fluctuations
    in soybean leaves in response to R. solani AG1-IA infection 12 and 24 h post-inoculation
    (h.p.i.). Metabolite fold changes are represented in bar graphs, and significant
    changes based on PLS-DA loading coefficients are indicated by an asterisk (P < 0.05).
    Transcripts are represented as arrows connecting metabolites. Transcript fold-changes
    (1.5 fold threshold; P < 0.05) based on qRT-PCR results are represented by boxes
    with 12 h.p.i. and 24 h.p.i. time-points represented on the left and right, respectively.
    Red illustrates an increase in transcript abundance in response to R. solani AG1-IA
    infection, while hatched green represents a decrease (P < 0.05 and fold change
    >1.5 or <−1.5). Capital lettering beside the boxes indicates gene name abbreviations
    (see Table ). Black arrows without boxes represent transcripts detected in RNAseq,
    but not significantly affected by infection. Solid arrows represent single-step
    reactions, while hashed arrows represent multi-step reactions. Stars represent
    transcripts and metabolite bins that were identified as significantly altered
    using O2PLS analytical methods
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAL1
  - FDH
  - KCS10
  - AGT
  - PEP
  - PAS2
  - MLS
  - GSTU24
  - Succinyl-CoA
  - Acetaldehyde
  - Acetate
  - Alanine
  - Ethanol
  - Threonine L-asparagine
  - Aspartate
  - L-aspartate
  - Cinnamate
  - Cysteine
  - Flavonoids
  - Formate
  - Fructose
  - GABA
  - Glucose
  - Glutamine
  - Glutathione
  - R-S-Glutathione
  - Glycine
  - Isocitrate
  - Isoleucine
  - Leucine
  - Lysine
  - Malate
  - Oxaloacetate
  - Phenylalanine
  - Proline
  - Pyruvate
  - Serine
  - Succinate
  - Sucrose
  - Tryptophan
  - Tyrosine
  - Valine
  - Glutamate
---
