---
figid: PMC5423858__nihms829667f1
figtitle: MicroRNA function in Drosophila memory formation
organisms:
- Drosophila melanogaster
- Caenorhabditis elegans
- Homo sapiens
- Mus musculus
- Agaricus bisporus
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC5423858
filename: nihms829667f1.jpg
figlink: /pmc/articles/PMC5423858/figure/F1/
number: F1
caption: MiR gene expression is regulated in ways similar to protein coding genes.
  Neuronal activity (depolarization, neurotrophins, sensory stimuli, etc.) can induce
  miR expression [,,]. MiR-132, for example, possesses a CRE sequence in its promoter
  allowing CREB to regulate its activity. The NMDA receptor, CaMKII and the MAPK-ERK
  pathway are known to control miR-132 expression although different upstream signaling
  pathways that may regulate other miR genes []. There are two miR biogenesis pathways
  in the nucleus, the canonical and the ‘miRtrons’ pathway. In the canonical pathway,
  RNA polymerase II transcribes miR genes into pri-miRs that are further processed
  into pre-miRs by the Pasha/Drosha microprocessor complex. In the miRtron pathway,
  miRs are spliced from introns by the spliceosome to pre-miRs. Exportin5 along with
  Ran-GTP actively transports pre-miRs to the cytoplasm. Dicer/Loquacious complex
  further processes the pre-miRs in the cytoplasm to mature duplexes - with a guide
  strand (miR) and its passenger (miR*). A member of the Argonaute family of proteins
  (Ago1 or Ago2 depending on the pathway) loads one of the strands into the miRISC
  complex (lower right corner). The complex includes the Gw182 protein and other auxiliary
  proteins such as dFmr1. This outcome leads to mRNA degradation or translational
  inhibition depending on the match quality of miR/mRNA hybrid. In rare cases (upper
  right corner), duplexes with a very high level of complementarity between strands
  are sorted to an alternative pathway using Dicer2, leading to mRNA degradation by
  Ago2 and inducing competition between the processing pathways []. Neural activity,
  genetic manipulations or disorders have been shown to positively or negatively influence
  the processing enzymes involved in miRNA biogenesis (grey squares with red and green
  arrows) [,,].
papertitle: MicroRNA function in Drosophila memory formation.
reftext: Germain U. Busto, et al. Curr Opin Neurobiol. ;43:15-24.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9574236
figid_alias: PMC5423858__F1
figtype: Figure
redirect_from: /figures/PMC5423858__F1
ndex: 69af6cfa-dee6-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5423858__nihms829667f1.html
  '@type': Dataset
  description: MiR gene expression is regulated in ways similar to protein coding
    genes. Neuronal activity (depolarization, neurotrophins, sensory stimuli, etc.)
    can induce miR expression [,,]. MiR-132, for example, possesses a CRE sequence
    in its promoter allowing CREB to regulate its activity. The NMDA receptor, CaMKII
    and the MAPK-ERK pathway are known to control miR-132 expression although different
    upstream signaling pathways that may regulate other miR genes []. There are two
    miR biogenesis pathways in the nucleus, the canonical and the ‘miRtrons’ pathway.
    In the canonical pathway, RNA polymerase II transcribes miR genes into pri-miRs
    that are further processed into pre-miRs by the Pasha/Drosha microprocessor complex.
    In the miRtron pathway, miRs are spliced from introns by the spliceosome to pre-miRs.
    Exportin5 along with Ran-GTP actively transports pre-miRs to the cytoplasm. Dicer/Loquacious
    complex further processes the pre-miRs in the cytoplasm to mature duplexes - with
    a guide strand (miR) and its passenger (miR*). A member of the Argonaute family
    of proteins (Ago1 or Ago2 depending on the pathway) loads one of the strands into
    the miRISC complex (lower right corner). The complex includes the Gw182 protein
    and other auxiliary proteins such as dFmr1. This outcome leads to mRNA degradation
    or translational inhibition depending on the match quality of miR/mRNA hybrid.
    In rare cases (upper right corner), duplexes with a very high level of complementarity
    between strands are sorted to an alternative pathway using Dicer2, leading to
    mRNA degradation by Ago2 and inducing competition between the processing pathways
    []. Neural activity, genetic manipulations or disorders have been shown to positively
    or negatively influence the processing enzymes involved in miRNA biogenesis (grey
    squares with red and green arrows) [,,].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Nmdar1
  - ca
  - CaMKII
  - Dcr-2
  - ago
  - AGO1
  - AGO2
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Parp
  - MKP-4
  - p38b
  - rl
  - Dcr-1
  - Erk7
  - lncRNA:let7C
  - mir-let7
  - CrebB
  - CrebA
  - PolI
  - pasha
  - E(spl)mgamma-HLH
  - mirr
  - gw
  - Ran
  - RanGAP
  - Camk2a
  - Camk2b
  - Mecp2
  - Fbxw7
  - Parp1
  - Ephb2
  - Mapk1
  - Creb1
  - Poll
  - Pol1
  - Poli
  - Tnrc6a
  - Rasl2-9
  - CAMK2G
  - MECP2
  - FBXW7
  - PARP1
  - COL11A2
  - PARP2
  - PARP3
  - PARP4
  - PARP6
  - PARP8
  - PARP9
  - PARP10
  - PARP11
  - PARP12
  - PARP14
  - PARP15
  - PARP16
  - TNKS
  - TNKS2
  - TIPARP
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - DICER1
  - EPHB2
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - POLL
  - POLI
  - DGCR8
  - TNRC6A
  - RAN
  - kita
  - ngfra
  - mecp2
  - parp14rs2.1
  - poll
  - poli
  - ran
  - AMAA
  - Glutamate
---
