---
figid: PMC5428490__41598_2017_438_Fig2_HTML
figtitle: Transcriptome pattern differs between two oil biosynthesis tissues in oil
  palm
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC5428490
filename: 41598_2017_438_Fig2_HTML.jpg
figlink: /pmc/articles/PMC5428490/figure/Fig2/
number: F2
caption: Transcriptome pattern differs between two oil biosynthesis tissues in oil
  palm. (A) The ratio of up- to down-regulated genes when compared the transcription
  level of Tenera against Dura in mesocarp and endosperm. The gene expression level
  of their respective key stage (2.5 MAF for endosperm, 3.5 MAF for mesocarp) were
  used to compare. Assumed at least 2 folds change as up- or down-regulated genes.
  Whole transcriptome and some specific categorized pathways are listed out. Plastid
  FAS refers to the pathway of fatty acid synthesis in plastid. Noticed that genes
  of lipid related or plastid FAS are highly upregulated in mesocarp and endosperm
  of Tenera respectively. (B) Transcripts patterns for enzymes involved in glysolysis
  reactions for Dura and Tenera. Values in red indicate the fold increase in reads
  of mesocarp of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the fold
  increase in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples collected
  from different tissues for Dura, Pisifera and Tenera (#53) were used for transcriptome
  analysis. Orange arrow indicates the cytosolic glycolytic process is the major carbon
  flow increased in Tenera compared with Dura both in mesocarp and endosperm, as deduced
  by the fold increase in reads. Light yellow arrows indicate the major increased
  cytosol-to-plastid carbon flow for mesocarp. Light blue arrow indicates the major
  increased cytosol-to-plastid carbon flow for endosperm. (C) Transcripts patterns
  for enzymes involved in plastidial and extraplasidial reactions for Dura and Tenera.
  Values in red indicate the fold increase in reads of mesocarp of Tenera to that
  of Dura at 3.5 MAF. Values in blue indicate the fold increase in reads of endosperm
  of Tenera to that of Dura at 2.5 MAF. Samples collected from different tissues for
  Dura, Pisifera and Tenera (#53) were used for transcriptome analysis. Reads for
  enzymes with multiple isoforms were summed. For details on abbreviations, annotations,
  and transcriptome levels at each samples, see Dataset S1. 16:0, palmitic acid; 18:0,
  stearic acid; 18:1, oleic acid, 18:2, Linoleic acid; 18:3, linolenic acid. For details
  on abbreviations, annotations, and transcription levels at each stage, see Supplemental
  Dataset .
papertitle: Transcriptome and functional analysis reveals hybrid vigor for oil biosynthesis
  in oil palm.
reftext: Jingjing Jin, et al. Sci Rep. 2017;7:439.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9093914
figid_alias: PMC5428490__F2
figtype: Figure
redirect_from: /figures/PMC5428490__F2
ndex: 6c82d621-defc-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5428490__41598_2017_438_Fig2_HTML.html
  '@type': Dataset
  description: Transcriptome pattern differs between two oil biosynthesis tissues
    in oil palm. (A) The ratio of up- to down-regulated genes when compared the transcription
    level of Tenera against Dura in mesocarp and endosperm. The gene expression level
    of their respective key stage (2.5 MAF for endosperm, 3.5 MAF for mesocarp) were
    used to compare. Assumed at least 2 folds change as up- or down-regulated genes.
    Whole transcriptome and some specific categorized pathways are listed out. Plastid
    FAS refers to the pathway of fatty acid synthesis in plastid. Noticed that genes
    of lipid related or plastid FAS are highly upregulated in mesocarp and endosperm
    of Tenera respectively. (B) Transcripts patterns for enzymes involved in glysolysis
    reactions for Dura and Tenera. Values in red indicate the fold increase in reads
    of mesocarp of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the
    fold increase in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples
    collected from different tissues for Dura, Pisifera and Tenera (#53) were used
    for transcriptome analysis. Orange arrow indicates the cytosolic glycolytic process
    is the major carbon flow increased in Tenera compared with Dura both in mesocarp
    and endosperm, as deduced by the fold increase in reads. Light yellow arrows indicate
    the major increased cytosol-to-plastid carbon flow for mesocarp. Light blue arrow
    indicates the major increased cytosol-to-plastid carbon flow for endosperm. (C)
    Transcripts patterns for enzymes involved in plastidial and extraplasidial reactions
    for Dura and Tenera. Values in red indicate the fold increase in reads of mesocarp
    of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the fold increase
    in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples collected
    from different tissues for Dura, Pisifera and Tenera (#53) were used for transcriptome
    analysis. Reads for enzymes with multiple isoforms were summed. For details on
    abbreviations, annotations, and transcriptome levels at each samples, see Dataset
    S1. 16:0, palmitic acid; 18:0, stearic acid; 18:1, oleic acid, 18:2, Linoleic
    acid; 18:3, linolenic acid. For details on abbreviations, annotations, and transcription
    levels at each stage, see Supplemental Dataset .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pgi
  - pyr
  - Ald1
  - Gapdh2
  - Gpdh1
  - Gapdh1
  - Pfk
  - CG15812
  - Pglym78
  - Pglym87
  - Pgk
  - Eno
  - pk
  - Tpt
  - Ppt1
  - Aldh-III
  - Tpi
  - fru
  - Pu
  - Suc
  - Pgm1
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - PPO1
  - Aco
  - Acox57D-d
  - Fas2
  - FASN1
  - Fas3
  - fand
  - tei
  - rg
  - FASN2
  - FASN3
  - cta
  - ACC
  - tacc
  - acclinal-wing
  - LPCAT
  - mat
  - Mad
  - mdy
  - Pc
  - Acp26Ab
  - Acp26Aa
  - CG18284
  - Acp36DE
  - SP
  - Acp95EF
  - Acph-1
  - Fad2
  - et
  - ear
  - kar
  - spidey
  - Fs(3)Kar
  - wisp
  - PAPLA1
  - Pli
  - skd
  - hrg
  - Pa
  - sad
  - Smox
  - nAChRalpha2
  - CG15450
  - Gpat4
  - Gnpat
  - GPI
  - GNPDA1
  - FBXO3
  - GAPDH
  - GAPDHP44
  - PFKL
  - PFKM
  - PFKP
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - GPT2
  - SHH
  - PDSS1
  - LMBR1
  - PPT1
  - TPI1
  - ZBTB22
  - VCAN
  - PRF1
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - KLK15
  - FAS
  - FASN
  - PCYT1A
  - CERNA3
  - ACACA
  - BMS1
  - ACACB
  - CTBS
  - PCYT1B
  - TMED7
  - LINC01194
  - PCSK7
  - LPCAT3
  - LPCAT1
  - MBOAT2
  - ACAT1
  - MAT1A
  - OXSM
  - DGAT1
  - PC
  - NDUFAB1
  - CPAT1
  - FANCD2
  - MDH1
  - HSD17B12
  - REG3A
  - ASAP2
  - MRPS30
  - PAPOLA
  - PDAP1
  - TUSC2
  - ASAP1
  - AGPAT1
  - LPA
  - PPAT
  - GPAM
  - Acyl-CoA
  - Glycerol
  - Malonyl-CoA
  - Pyruvate
  - Succinate
---
