---
figid: PMC6060558__13046_2018_843_Fig4_HTML
figtitle: Enriched GO network groups using ClueGO/CluePedia-based enrichment
organisms:
- herbal medicine
- Curcuma longa
- Citrus limon
- Bos taurus
- Oryctolagus cuniculus
- Homo sapiens
- Mus musculus
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC6060558
filename: 13046_2018_843_Fig4_HTML.jpg
figlink: /pmc/articles/PMC6060558/figure/Fig4/
number: F4
caption: Enriched GO network groups using ClueGO/CluePedia-based enrichment. a Network
  view for KEGG pathway of CurcuUp-Regulated proteins and b Biological Process of
  CurcuDown-Regulated proteins. Terms (each represented as node) are functionally
  grouped based on shared genes (kappa score ≥ 0.4) and are shown with different colors.
  The specific players (proteins) of each node are highlighted with the respective
  gene name Node color represents the class that they belong. The size of the nodes
  indicates the degree of significance. Within each class, the most significant term
  (indicated with colored and bold characters) defines the name of the group. Ungrouped
  terms are not shown
papertitle: SWATH-MS based quantitative proteomics analysis reveals that curcumin
  alters the metabolic enzyme profile of CML cells by affecting the activity of miR-22/IPO7/HIF-1α
  axis.
reftext: Francesca Monteleone, et al. J Exp Clin Cancer Res. 2018;37:170.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6906593
figid_alias: PMC6060558__F4
figtype: Figure
redirect_from: /figures/PMC6060558__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6060558__13046_2018_843_Fig4_HTML.html
  '@type': Dataset
  description: Enriched GO network groups using ClueGO/CluePedia-based enrichment.
    a Network view for KEGG pathway of CurcuUp-Regulated proteins and b Biological
    Process of CurcuDown-Regulated proteins. Terms (each represented as node) are
    functionally grouped based on shared genes (kappa score ≥ 0.4) and are shown with
    different colors. The specific players (proteins) of each node are highlighted
    with the respective gene name Node color represents the class that they belong.
    The size of the nodes indicates the degree of significance. Within each class,
    the most significant term (indicated with colored and bold characters) defines
    the name of the group. Ungrouped terms are not shown
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ar
  - Cox6a1
  - Rap1gap
  - Rasa1
  - hf
  - Mos
  - Mocos
  - tu
  - STK38
  - HEMGN
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - LINC01191
  - AR
  - COL1A1
  - COL1A2
  - RASA1
  - RGS6
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - MOS
  - MOCOS
  - trc
  - vin
  - Vinc
  - ar
  - 'On'
  - we
  - tur
  - Col4a1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Act5C
  - Act42A
  - Act57B
  - Act79B
  - Act88F
  - Act87E
  - Hf
  - fu
  - Rep
  - rept
  - ec
  - Tu
  - Rpt6
  - PMO
  - purine
  - ribonucleoside
  - Aminoacyl
  - Pyruvate
  - methionine
  - Purine
  - Glutathione
  - phosphate
  - Amino sugar
  - nucleotide
  - sugar
  - AT
  - Vial carcinogenesis
  - HF gaing etabolism
  - cancer
---
