---
figid: PMC6787556__nihms-1053646-f0001
figtitle: Nicotinamide Adenine Dinucleotide (NAD+) Production and Catabolism in Mammalian
  Cells
organisms:
- Mus musculus
- Rattus norvegicus
- Heterocephalus glaber
- Homo sapiens
- Saccharomyces cerevisiae
- Drosophila melanogaster
- Caenorhabditis elegans
- Macaca mulatta
- Bikinia letestui
- Xenopus laevis
- gut metagenome
- prion
- Akkermansia muciniphila
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Macaca mulatta
- Heterocephalus glaber
- Drosophila melanogaster
- Xenopus laevis
pmcid: PMC6787556
filename: nihms-1053646-f0001.jpg
figlink: pmc/articles/PMC6787556/figure/F1/
number: F1
caption: '(A) NAD+ is produced via three major pathways in mammals. The first is the
  de novo biosynthesis from tryptophan (Trp), also called the kynurenine pathway.
  Trp enters the cell via the transporters SLC7A5 and SLC36A4. Within the cell, Trp
  is converted to formylkynurenine (FK), which is further converted to kynurenine.
  Kynurenine can be converted to kynurenic acid (via kynurenine aminotransferases/KATs)
  and finally quinaldic acid. Additionally, kynurenine can be converted to 3-hydroxykynurenine
  (3-HK) by kynurenine 3-monooxygenase (KMO), and further to 3-hydroxyanthranilic
  acid (3-HAA) by tryptophan 2,3-dioxygenase (KYNU). The next step is performed by
  3-hydroxyanthranilic acid oxygenase (3HAO) to produce α-amino-β-carboxymuconate-ϵ-semialdehyde
  (ACMS). Via a spontaneous reaction, ACMS converts to quinolinic acid, which further
  formulates to NAMN by quinolinate phosphoribosyltransferase (QPRT), and to nicotinic
  acid adenine dinucleotide (NAAD), and finally to NAD+. 3-HK, 3-HAA, and quinolinic
  acid are neurotoxic (denoted with red asterisk), whereas kynurenic acid and picolinic
  acid are neuroprotective (marked with green asterisk). The second pathway is the
  Preiss-Handler pathway during which nicotinic acid (NA) is used as an NAD+ precursor.
  NA enters the cell via SLC5A8 or SLC22A3 transporters. The Preiss-Handler pathway
  is initiated by the conversion of NA to NAMN by NA phosphoribosyl-transferase (NAPRT).
  NAMN, an intermediate in both kynurenine pathway and the Preiss-Handler pathway,
  is converted to form NAAD by NAM mononucleotide transferases (NMNATs). Lastly, NAAD
  is converted to NAD+ by NAD+ synthase (NADS). The third pathway is the salvage pathway
  with the cells generating NAD+ from nicotinamide riboside (NR) and recycling nicotinamide
  (NAM) back to NAD+ via nicotinamide mononucleotide (NMN). Extracellularly, NAD+
  or NAM can be converted to NMN, which is in turn dephosphorylated to NR, possibly
  by CD73. NR is transported into the cell via an unknown nucleoside transport. Intracellularly,
  NR forms NMN via NRK1 or NRK2 in a tissue-specific manner. NMN is then converted
  to NAD+ by NMNATs. The enzyme NAM N-methyltransferase (NNMT) methylates NAM, using
  S-adenosyl methionine (SAM) as a methyl donor. This removes NAM from recycling and
  indirectly affects NAD+ levels.(B) The four major NAD+-consuming enzymes. From left:
  poly(ADP-ribose) polymerases (PARPs), especially PARP1 and PARP2, use NAD+ as a
  co-substrate to PARylate target proteins, generating NAM as a by-product. The deacetylation
  activity of sirtuin (SIRT)1, SIRT3, and SIRT6 depends on NAD+, generating NAM as
  a by-product, with NAM at high cellular levels inhibiting the activity of SIRTs.
  The NADases or cyclic ADP-ribose synthases (cADPRSs) CD38 and CD157 hydrolyze NAD+
  to NAM, generating ADPR and cADPR; in addition, CD38 can degrade NMN to NAM, removing
  NMN from NAD+synthesis. Sterile alpha and TIR motif-containing 1 (SARM1) was recently
  identified as a NADase, which cleaves NAD+ to NAM, cADPR, and ADPR.(C) The chemical
  structures of NAD+ and the NAD+ precursors. From left: nicotinic acid (NA), nicotinamide
  (NAM), nicotinamide riboside (NR), nicotinamide mononucleotide (NMN), and oxidized
  form of nicotinamide adenine dinucleotide (NAD+).Note that CD38, CD73, and CD157
  are membrane proteins. For simplicity, we did not attach them to the membrane in
  this schematic model.'
papertitle: NAD+ in Brain Aging and Neurodegenerative Disorders.
reftext: Sofie Lautrup, et al. Cell Metab. ;30(4):630-655.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8206832
figid_alias: PMC6787556__F1
figtype: Figure
redirect_from: /figures/PMC6787556__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6787556__nihms-1053646-f0001.html
  '@type': Dataset
  description: '(A) NAD+ is produced via three major pathways in mammals. The first
    is the de novo biosynthesis from tryptophan (Trp), also called the kynurenine
    pathway. Trp enters the cell via the transporters SLC7A5 and SLC36A4. Within the
    cell, Trp is converted to formylkynurenine (FK), which is further converted to
    kynurenine. Kynurenine can be converted to kynurenic acid (via kynurenine aminotransferases/KATs)
    and finally quinaldic acid. Additionally, kynurenine can be converted to 3-hydroxykynurenine
    (3-HK) by kynurenine 3-monooxygenase (KMO), and further to 3-hydroxyanthranilic
    acid (3-HAA) by tryptophan 2,3-dioxygenase (KYNU). The next step is performed
    by 3-hydroxyanthranilic acid oxygenase (3HAO) to produce α-amino-β-carboxymuconate-ϵ-semialdehyde
    (ACMS). Via a spontaneous reaction, ACMS converts to quinolinic acid, which further
    formulates to NAMN by quinolinate phosphoribosyltransferase (QPRT), and to nicotinic
    acid adenine dinucleotide (NAAD), and finally to NAD+. 3-HK, 3-HAA, and quinolinic
    acid are neurotoxic (denoted with red asterisk), whereas kynurenic acid and picolinic
    acid are neuroprotective (marked with green asterisk). The second pathway is the
    Preiss-Handler pathway during which nicotinic acid (NA) is used as an NAD+ precursor.
    NA enters the cell via SLC5A8 or SLC22A3 transporters. The Preiss-Handler pathway
    is initiated by the conversion of NA to NAMN by NA phosphoribosyl-transferase
    (NAPRT). NAMN, an intermediate in both kynurenine pathway and the Preiss-Handler
    pathway, is converted to form NAAD by NAM mononucleotide transferases (NMNATs).
    Lastly, NAAD is converted to NAD+ by NAD+ synthase (NADS). The third pathway is
    the salvage pathway with the cells generating NAD+ from nicotinamide riboside
    (NR) and recycling nicotinamide (NAM) back to NAD+ via nicotinamide mononucleotide
    (NMN). Extracellularly, NAD+ or NAM can be converted to NMN, which is in turn
    dephosphorylated to NR, possibly by CD73. NR is transported into the cell via
    an unknown nucleoside transport. Intracellularly, NR forms NMN via NRK1 or NRK2
    in a tissue-specific manner. NMN is then converted to NAD+ by NMNATs. The enzyme
    NAM N-methyltransferase (NNMT) methylates NAM, using S-adenosyl methionine (SAM)
    as a methyl donor. This removes NAM from recycling and indirectly affects NAD+
    levels.(B) The four major NAD+-consuming enzymes. From left: poly(ADP-ribose)
    polymerases (PARPs), especially PARP1 and PARP2, use NAD+ as a co-substrate to
    PARylate target proteins, generating NAM as a by-product. The deacetylation activity
    of sirtuin (SIRT)1, SIRT3, and SIRT6 depends on NAD+, generating NAM as a by-product,
    with NAM at high cellular levels inhibiting the activity of SIRTs. The NADases
    or cyclic ADP-ribose synthases (cADPRSs) CD38 and CD157 hydrolyze NAD+ to NAM,
    generating ADPR and cADPR; in addition, CD38 can degrade NMN to NAM, removing
    NMN from NAD+synthesis. Sterile alpha and TIR motif-containing 1 (SARM1) was recently
    identified as a NADase, which cleaves NAD+ to NAM, cADPR, and ADPR.(C) The chemical
    structures of NAD+ and the NAD+ precursors. From left: nicotinic acid (NA), nicotinamide
    (NAM), nicotinamide riboside (NR), nicotinamide mononucleotide (NMN), and oxidized
    form of nicotinamide adenine dinucleotide (NAD+).Note that CD38, CD73, and CD157
    are membrane proteins. For simplicity, we did not attach them to the membrane
    in this schematic model.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Cd38
  - Bst1
  - Nt5e
  - nr
  - Slc36a4
  - Slc12a8
  - Slc22a3
  - Tdo2
  - Ido1
  - Fk
  - Naprt
  - Nmrk1
  - Decr1
  - Qprt
  - Nmnat1
  - Nmnat3
  - Nmnat2
  - Kynu
  - Nampt
  - Nadk
  - Snhg15
  - Nnmt
  - Asah1
  - Sarm1
  - ac
  - oh
  - CD38
  - BST1
  - NT5E
  - STAC3
  - SLC36A4
  - SLC12A8
  - SLC22A3
  - DLX3
  - TDO2
  - IDO1
  - NAPRT
  - NMRK1
  - NEK4
  - NMRK2
  - DECR1
  - QPRT
  - NMNAT1
  - NMNAT3
  - NMNAT2
  - KYNU
  - NAMPT
  - NADK
  - NNMT
  - ASAH1
  - SARM1
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - de
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - na
  - .na.character
  - Nmnat
  - CG33156
  - sam
  - qkr54B
  - sls
  - Sam-S
  - Nadsyn
  - Sarm
  - hh
  - cd38.L
  - bst1.L
  - nt5e.S
  - nt5e.L
  - slc36a4.L
  - slc12a8.L
  - dlx3.S
  - dlx3.L
  - ido1.L
  - qprt.L
  - nmnat1.L
  - nampt.S
  - nadk.S
  - nnmt.L
  - Kynurenine
  - NAD
  - Tryptophan
  - Trp
  - Nicotinic acid
  - Nicotinamide riboside
  - NADPH
  - NADP
  - KYNU
  - 3-NAA
  - ZNAO
  - NAAD
  - NADS
  - Quinolinic
  - NADH
  - MeNAM
  - Ac
  - Nicotinamide
  - Nicotinamide mononucleotide
  - Nicotinamide adenine dinucleotide
---
