---
figid: PMC6493774__ppat.1007729.g005
figtitle: Niche-specific metabolic adaptation in biotrophic and necrotrophic oomycetes
  is manifested in differential use of nutrients, variation in gene content, and enzyme
  evolution
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Zea mays
- Solanum lycopersicum
- Solanum tuberosum
- Hordeum vulgare
- Aspergillus nidulans
- Spinacia oleracea
- Pisum sativum
- Aspergillus niger
- Phaseolus vulgaris
- Neurospora crassa
- Fusarium graminearum
- Phytophthora infestans
- Pyricularia grisea
- Nitzschia alba
- Peronospora farinosa
- Phytophthora capsici
- Hyaloperonospora arabidopsidis
- Phytophthora ramorum
- Skeletonema costatum
- Mirabilis jalapa
- Phytophthora palmivora
- Phytophthora megasperma
- Plasmopara halstedii
- Phytophthora fragariae
- Globisporangium ultimum var. ultimum
- Phytophthora mirabilis
- Phytopythium vexans
- Oryctolagus cuniculus
- Phascolarctos cinereus
- Homo sapiens
- Escherichia coli
- Drosophila melanogaster
- metagenome
- NA
organisms_ner:
- NA
pmcid: PMC6493774
filename: ppat.1007729.g005.jpg
figlink: /pmc/articles/PMC6493774/figure/ppat.1007729.g005/
number: F5
caption: (A) Relative mRNA levels of genes involved in glycerolipid metabolism, starting
  from the cleavage of a triacylglycerol into glycerol and fatty acids. Enzyme Commission
  protein 2.3.1.21 is part of the carnitine shuttle that moves cytoplasmic acyl-CoA
  into the mitochondria. (B) Genes involved in the assimilation of inorganic sulfate.
  In both panels, the samples in the bar graphs are indicated in the key, and continue
  the color scheme and sample orders used in . The y-axis in each bar graph represents
  relative mRNA levels, per-gene normalized to an average of 1.0 for all four samples,
  as indicated in the key. Indicated in the pathway is each metabolite and the Enzyme
  Commission (EC) number for each enzyme.
papertitle: Niche-specific metabolic adaptation in biotrophic and necrotrophic oomycetes
  is manifested in differential use of nutrients, variation in gene content, and enzyme
  evolution.
reftext: Audrey M. V. Ah-Fong, et al. PLoS Pathog. 2019 Apr;15(4):e1007729.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7996701
figid_alias: PMC6493774__F5
figtype: Figure
redirect_from: /figures/PMC6493774__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6493774__ppat.1007729.g005.html
  '@type': Dataset
  description: (A) Relative mRNA levels of genes involved in glycerolipid metabolism,
    starting from the cleavage of a triacylglycerol into glycerol and fatty acids.
    Enzyme Commission protein 2.3.1.21 is part of the carnitine shuttle that moves
    cytoplasmic acyl-CoA into the mitochondria. (B) Genes involved in the assimilation
    of inorganic sulfate. In both panels, the samples in the bar graphs are indicated
    in the key, and continue the color scheme and sample orders used in . The y-axis
    in each bar graph represents relative mRNA levels, per-gene normalized to an average
    of 1.0 for all four samples, as indicated in the key. Indicated in the pathway
    is each metabolite and the Enzyme Commission (EC) number for each enzyme.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Sulfate
  - triacylglycerol
  - sulfate
  - ATP
  - fatty acid
  - adenylyl sulfate
  - glycerol
  - 3-phosphoadenylyl sulfate
  - PAPS
  - glycerol-3-P
  - glyceraldehyde
  - sulfite
  - enoyl-CoA
  - dihydroxyacetone
  - phosphate
  - glycerate
  - hydroxyacyl-CoA
  - sulfide
  - O-acetylserine
  - 3-phosphoglycerate
  - 3-ketoacyl-CoA
  - cysteine
  - acyl-CoA
  - acyl-carnitine
  - acetyl-CoA
  - APS
---
