---
figid: PMC6377297__nihms-1512412-f0001
figtitle: 'Shift work: snapshot of cellular activities across the day and night'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC6377297
filename: nihms-1512412-f0001.jpg
figlink: pmc/articles/PMC6377297/figure/F1/
number: F1
caption: 'A) A representation of an S. elongatus cell with an overview of the major
  metabolic pathways important for day-night physiology. B) Reactions of glycogen
  metabolism, TCA pathway, and central metabolism that are present in S. elongatus
  are provided in more detail. Green arrows in panel B indicate reactions where the
  respective enzyme has a detectable light/dark-dependent redox modification in Synechocystis
  sp. PCC 6803 []. Data about the peak expression time of circadian genes is indicated
  using data collected from S. elongatus PCC 7942 []. Genes colored in red peak in
  expression in the subjective morning, genes colored in blue peak in expression in
  the subjective evening, and genes colored in black have no detectable circadian
  rhythm in S. elongatus. Essential genes and genes that cause light-dark sensitive
  phenotypes when mutated are indicated by full yellow and half yellow-half dark grey
  circles, respectively. Abbreviations: PBS, phycobilisome; PSII, photosystem II;
  cyt b6f, cytochrome b6f; PSI, photosystem I; Fd(red), ferredoxin (reduced); Ru1,5P,
  ribulose-1,5-bisphosphate; 3PG, 3-phosphoglycerate; 1,3-BPG, 1,3-bisphosphoglycerate;
  GAP, glyceraldehyde-3-phosphate; F6P, fructose-6-phosphate; G6P, glucose-6-phosphate;
  G1P, glucose-1-phosphate; 6PGL, 6-phosphogluconolactone; 6PG, 6-phosphogluconate;
  Ru5P, ribulose-5-phosphate; ac-CoA, acetyl-CoA; aKG, a-ketoglutarate; Glu, glutamate;
  Gln, glutamine.'
papertitle: 'A Hard Dayâ€™s Night: Cyanobacteria in Diel Cycles.'
reftext: David G. Welkie, et al. Trends Microbiol. ;27(3):231-242.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7292426
figid_alias: PMC6377297__F1
figtype: Figure
redirect_from: /figures/PMC6377297__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6377297__nihms-1512412-f0001.html
  '@type': Dataset
  description: 'A) A representation of an S. elongatus cell with an overview of the
    major metabolic pathways important for day-night physiology. B) Reactions of glycogen
    metabolism, TCA pathway, and central metabolism that are present in S. elongatus
    are provided in more detail. Green arrows in panel B indicate reactions where
    the respective enzyme has a detectable light/dark-dependent redox modification
    in Synechocystis sp. PCC 6803 []. Data about the peak expression time of circadian
    genes is indicated using data collected from S. elongatus PCC 7942 []. Genes colored
    in red peak in expression in the subjective morning, genes colored in blue peak
    in expression in the subjective evening, and genes colored in black have no detectable
    circadian rhythm in S. elongatus. Essential genes and genes that cause light-dark
    sensitive phenotypes when mutated are indicated by full yellow and half yellow-half
    dark grey circles, respectively. Abbreviations: PBS, phycobilisome; PSII, photosystem
    II; cyt b6f, cytochrome b6f; PSI, photosystem I; Fd(red), ferredoxin (reduced);
    Ru1,5P, ribulose-1,5-bisphosphate; 3PG, 3-phosphoglycerate; 1,3-BPG, 1,3-bisphosphoglycerate;
    GAP, glyceraldehyde-3-phosphate; F6P, fructose-6-phosphate; G6P, glucose-6-phosphate;
    G1P, glucose-1-phosphate; 6PGL, 6-phosphogluconolactone; 6PG, 6-phosphogluconate;
    Ru5P, ribulose-5-phosphate; ac-CoA, acetyl-CoA; aKG, a-ketoglutarate; Glu, glutamate;
    Gln, glutamine.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - bef
  - ATPsynbeta
  - Atpalpha
  - Psi
  - Ald1
  - Pfk
  - Pgi
  - G6P
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Aldh-III
  - fbp
  - Is
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - LanB2
  - anon-70Db
  - Pgk
  - PyK
  - Pep
  - bgcn
  - gnd
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - Me
  - gs
  - Pgm1
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - sps
  - Spp
  - pps
  - inv
  - Invadolysin
  - su(sable)
  - SFMBT1
  - DECR1
  - ALYREF
  - ATP8A2
  - FBXO3
  - PFKM
  - BGN
  - GPI
  - PGPEP1
  - RASA1
  - RGS6
  - ATP2B3
  - FBP1
  - FOLR1
  - FOLR2
  - ECB2
  - FUBP1
  - RAB14
  - HAS1
  - LUM
  - TALDO1
  - LRSAM1
  - TAAR1
  - CYP1A2
  - PHKA2
  - PAEP
  - PREP
  - PTPN22
  - SDHD
  - RPIA
  - RN7SL263P
  - PLK3
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - GNPTAB
  - GNAS
  - GNAL
  - VCAN
  - SARDH
  - SDHB
  - SORD
  - SDS
  - SMS
  - TBX4
  - SEPHS1
  - PDSS1
  - HM13
  - INVS
  - LMLN
  - Nitrogen
  - Ru1,5P
  - NADPH
  - GinTNH
  - ATP
  - DHAP
  - O
  - NADH
  - TCA
  - Amino acid
  - Nucleotide
  - Pyruvate
  - Ru5P
  - Acetyl-CoA Glutamine
  - Sucrose
  - Citrate
  - ADP
  - Isocitrate
  - Fructose
  - Succinate
  - 2-OG
  - Glutamate
  - Glucose
---
