---
figid: PMC6191511__fnmol-11-00375-g001
figtitle: Functional analysis of genes differentially expressed in response to L-lactate
  after 1 h
organisms:
- NA
pmcid: PMC6191511
filename: fnmol-11-00375-g001.jpg
figlink: /pmc/articles/PMC6191511/figure/F1/
number: F1
caption: Functional analysis of genes differentially expressed in response to L-lactate
  after 1 h. (A) GO plots showing enrichment of biological processes and molecular
  functions for the genes differentially expressed after 1 h stimulation with L-lactate
  (135 genes, q-value < 0.1, three biological replicates, n = 3 independent cultures).
  Statistical significance for the enriched terms is based on the multiple testing-corrected
  p-value, which is color-coded. The number of genes associated with each respective
  term is indicated by the dot size. (B) A detailed KEGG graph of the MAPK signaling
  pathway with genes modulated by L-lactate at 1 h highlighted in green indicating
  that they are up-regulated. Dusp10 and Gadd45g are not indicated on the pathway
  map. Genes highlighted in red are genes down-regulated in the co-treatment condition
  with MK801 and L-lactate. (C) Bar plot showing fold enrichment of specific transcription
  factor binding sites in the promoters of genes modulated by L-lactate at 1 h (FDR
  < 0.001). Z-score is the main enrichment indicator and is obtained by dividing the
  rate of occurrence of a TFBS in the target set of genes to the expected rate estimated
  from the pre-computed background. The Fisher score is calculated as the negative
  natural logarithm of the one-tailed Fisher exact probability.
papertitle: 'L-Lactate Regulates the Expression of Synaptic Plasticity and Neuroprotection
  Genes in Cortical Neurons: A Transcriptome Analysis.'
reftext: Michael B. Margineanu, et al. Front Mol Neurosci. 2018;11:375.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5456278
figid_alias: PMC6191511__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC6191511__F1
ndex: 3995605d-de97-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6191511__fnmol-11-00375-g001.html
  '@type': Dataset
  description: Functional analysis of genes differentially expressed in response to
    L-lactate after 1 h. (A) GO plots showing enrichment of biological processes and
    molecular functions for the genes differentially expressed after 1 h stimulation
    with L-lactate (135 genes, q-value < 0.1, three biological replicates, n = 3 independent
    cultures). Statistical significance for the enriched terms is based on the multiple
    testing-corrected p-value, which is color-coded. The number of genes associated
    with each respective term is indicated by the dot size. (B) A detailed KEGG graph
    of the MAPK signaling pathway with genes modulated by L-lactate at 1 h highlighted
    in green indicating that they are up-regulated. Dusp10 and Gadd45g are not indicated
    on the pathway map. Genes highlighted in red are genes down-regulated in the co-treatment
    condition with MK801 and L-lactate. (C) Bar plot showing fold enrichment of specific
    transcription factor binding sites in the promoters of genes modulated by L-lactate
    at 1 h (FDR < 0.001). Z-score is the main enrichment indicator and is obtained
    by dividing the rate of occurrence of a TFBS in the target set of genes to the
    expected rate estimated from the pre-computed background. The Fisher score is
    calculated as the negative natural logarithm of the one-tailed Fisher exact probability.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - mapP
  - MKP-4
  - p38b
  - rl
  - Amph
  - Mp
  - so
  - AttC
  - AttA
  - AttB
  - AttD
  - DPCoAC
  - att-ORFB
  - ap
  - tra
  - byn
  - Max
  - mof
  - Tab2
  - Abd-B
  - tun
  - Zasp52
  - Stat92E
  - en
  - gra
  - grn
  - grape
  - Srf
  - mal
  - bs
  - ca
  - fray
  - du
  - E(spl)malpha-BFM
  - da
  - nAChRalpha2
  - se
  - CG42366
  - mle
  - de
  - ERVK-9
  - ERVK-11
  - ERVK-19
  - SGSM3
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - APRT
  - MFAP1
  - PTF1A
  - MAPT
  - ATM
  - GLYATL1
  - MECOM
  - PHGDH
  - TRA
  - THRA
  - TRAC
  - MAX
  - TAB1
  - TAB2
  - TAB3
  - CA14
  - MAPK7
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - SRF
  - NFKBIA
  - MYCN
  - CREB1
  - STMN1
  - CD74
  - TH
  - MAK
  - ALPK3
  - NFKB1
  - NFKB2
  - tyrosineserinethreonine
  - Ca
  - Ta
  - Se
  - Guo
---
