---
figid: PMC6515235__ijms-20-02020-g001
figtitle: The l-leucine biosynthesis pathway of Corynebacterium glutamicum and the
  metabolic engineering steps performed in this study
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC6515235
filename: ijms-20-02020-g001.jpg
figlink: /pmc/articles/PMC6515235/figure/ijms-20-02020-f001/
number: F1
caption: 'The l-leucine biosynthesis pathway of Corynebacterium glutamicum and the
  metabolic engineering steps performed in this study. Enzymes and corresponding genes
  are shown. An encircled minus indicates repression of gene expression (solid lines).
  Deletion of the gene encoding LtbR is indicated by green “×”. Red thick arrows indicate
  increased metabolic fluxes. Introduction of enzymatic reactions are highlighted
  in blue. NADPH and NADP+ are highlighted in grey shaded boxes. Abbreviations: AHAS—acetotydroxyacid
  synthetase; AHAIR—acetohydroxyacid isomeroreductase; DHAD—dihydroxyacid dehydratase;
  TA—branched-chain amino acid transaminase; IPMS—α-isopropylmalate synthase; IPMI—α-isopropylmalate
  isomerase; IPMD—β-isopropylmalate dehydrogenase; LeuDH—leucine dehydrogenase; PEP—phosphoenolpyruvate;
  OAA—oxaloacetic acid; TCA—tricarboxylic acid cycle; GDH and RocG—glutamate dehydrogenase.'
papertitle: Improvement of l-Leucine Production in Corynebacterium glutamicum by Altering
  the Redox Flux.
reftext: Ying-Yu Wang, et al. Int J Mol Sci. 2019 Apr;20(8):2020.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9453287
figid_alias: PMC6515235__F1
figtype: Figure
redirect_from: /figures/PMC6515235__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6515235__ijms-20-02020-g001.html
  '@type': Dataset
  description: 'The l-leucine biosynthesis pathway of Corynebacterium glutamicum and
    the metabolic engineering steps performed in this study. Enzymes and corresponding
    genes are shown. An encircled minus indicates repression of gene expression (solid
    lines). Deletion of the gene encoding LtbR is indicated by green “×”. Red thick
    arrows indicate increased metabolic fluxes. Introduction of enzymatic reactions
    are highlighted in blue. NADPH and NADP+ are highlighted in grey shaded boxes.
    Abbreviations: AHAS—acetotydroxyacid synthetase; AHAIR—acetohydroxyacid isomeroreductase;
    DHAD—dihydroxyacid dehydratase; TA—branched-chain amino acid transaminase; IPMS—α-isopropylmalate
    synthase; IPMI—α-isopropylmalate isomerase; IPMD—β-isopropylmalate dehydrogenase;
    LeuDH—leucine dehydrogenase; PEP—phosphoenolpyruvate; OAA—oxaloacetic acid; TCA—tricarboxylic
    acid cycle; GDH and RocG—glutamate dehydrogenase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - alaT
  - aer
  - avtA
  - ilvC
  - ilvD
  - leuA
  - leuB
  - gltA
  - acrA
  - Glucose
  - NAD
  - NADH
  - L-alanine
  - L-valine
  - keto ketosovalerate
  - NADPH
  - NADP
  - L-glutamate
  - isovalerate
  - Pyruvate
  - acetolactate
  - ketoisovalerate
  - Acetyl-CoA
  - isopropylmalate
  - ketobutyrate
  - L-isoleucine
  - Citrate
  - DHAD
  - B-isopropylmalate
  - TSA
  - leuB
  - ketoglutarate
  - ketoisocaproate
  - L-leucine
---
