---
figid: PMC6105390__nihms959254f4
figtitle: Dietary fat and fiber interact to uniquely modify global histone post-translational
  epigenetic programming in a rat colon cancer progression model
organisms:
- Mus musculus
- Rattus norvegicus
- Oryctolagus cuniculus
- Capra hircus
- Zea mays
- Sequoia sempervirens
- Homo sapiens
- Drosophila melanogaster
- gut metagenome
- Simian endogenous retrovirus
pmcid: PMC6105390
filename: nihms959254f4.jpg
figlink: /pmc/articles/PMC6105390/figure/F4/
number: F4
caption: A) Representative networks of genes transcriptionally regulated in fish oil
  and pectin fed rats following carcinogen exposure by top ranked upstream key regulators
  (URs). Yellow fill represents the projected increase in UR activity. Blue fill indicates
  decreased gene activity (DE) and orange fill indicates increased gene activity (DE),
  deeper color hue indicates genes with greater |log2(FoldChange)|, and genes with
  FDR<0.1 are bolded. Solid lines represent direct and dashed lines represent indirect
  gene interaction. P-values and activity (z-scores) of URs were determined by Ingenuity
  Pathway Analysis of genes with p< 0.01. B) Fish oil + pectin diet transcriptionally
  enhanced beta-oxidation and decreased lipid metabolism associated genes during colon
  cancer progression. The dashed lines indicate aspects of lipid metabolism associated
  with a specific gene and the activation/inhibition of a biological function. Only
  a small subset of these genes were differentially transcribed in fish oil + cellulose
  fed rats following carcinogen injection, as labeled by yellow ovals.
papertitle: Dietary fat and fiber interact to uniquely modify global histone post-translational
  epigenetic programming in a rat colon cancer progression model.
reftext: Karen Triff, et al. Int J Cancer. 2018 Sep 15;143(6):1402-1415.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8060759
figid_alias: PMC6105390__F4
figtype: Figure
organisms_ner:
- Homo sapiens
- Rattus norvegicus
- Mus musculus
- Capra hircus
- Oryctolagus cuniculus
- Drosophila melanogaster
redirect_from: /figures/PMC6105390__F4
ndex: 3fed348b-df27-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6105390__nihms959254f4.html
  '@type': Dataset
  description: A) Representative networks of genes transcriptionally regulated in
    fish oil and pectin fed rats following carcinogen exposure by top ranked upstream
    key regulators (URs). Yellow fill represents the projected increase in UR activity.
    Blue fill indicates decreased gene activity (DE) and orange fill indicates increased
    gene activity (DE), deeper color hue indicates genes with greater |log2(FoldChange)|,
    and genes with FDR<0.1 are bolded. Solid lines represent direct and dashed lines
    represent indirect gene interaction. P-values and activity (z-scores) of URs were
    determined by Ingenuity Pathway Analysis of genes with p< 0.01. B) Fish oil +
    pectin diet transcriptionally enhanced beta-oxidation and decreased lipid metabolism
    associated genes during colon cancer progression. The dashed lines indicate aspects
    of lipid metabolism associated with a specific gene and the activation/inhibition
    of a biological function. Only a small subset of these genes were differentially
    transcribed in fish oil + cellulose fed rats following carcinogen injection, as
    labeled by yellow ovals.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BMP6
  - CXCL10
  - CCL5
  - GJB2
  - SLC13A1
  - ABCB1
  - VLDLR
  - HLA-E
  - MGLL
  - LPCAT3
  - HLA-DQA1
  - SLC51A
  - EMP1
  - SLC22A3
  - ABCG2
  - ADTRP
  - AQP8
  - EPB41L4B
  - PEA15
  - MTTP
  - TRNP
  - FABP1
  - SLC25A20
  - ARG2
  - ALDH3A2
  - CTH
  - VSIG2
  - PDK4
  - SLC27A2
  - ABCD3
  - CX3CL1
  - SCD
  - SCD5
  - GZMA
  - ACADM
  - DGAT2
  - CYP4B1
  - ACOT1
  - PEX11A
  - CPT1A
  - CAT
  - CRAT
  - GLYAT
  - CPT2
  - PRDX6
  - PRDX5
  - AXIN2
  - CYP1A1
  - HADHA
  - DIO1
  - DIDO1
  - RBP7
  - PYGL
  - DBI
  - TSPO
  - OMA1
  - PPARA
  - PPARD
  - HNF4A
  - HNF1A
  - NR3C1
  - PPARG
  - NR1H4
  - NR1I2
  - NR1H2
  - NR1H3
  - PLK2
  - NDRG1
  - ACTN2
  - NFKBIZ
  - ARNT2
  - BHLHE40
  - THEM4
  - PPARGC1A
  - ACSBG1
  - CYBB
  - LBP
  - EHHADH
  - RPSA
  - SLC26A3
  - FBN2
  - CADM1
  - PAM
  - MYCBP2
  - NCEH1
  - IL1R1
  - CYP4F12
  - SRD5A1
  - FA2H
  - ACADL
  - SLC1A1
  - SLC23A2
  - SLC23A1
  - SLC9A3
  - ENTPD5
  - MAP2K1
  - SSTR2
  - GNAI1
  - IFNGR1
  - PLA1A
  - CYP2C18
  - Bmp6
  - Cxcl10
  - Ccl5
  - Gjb2
  - Slc13a1
  - Abcb1b
  - Abcb1a
  - Vldlr
  - Mgll
  - Clec10a
  - Lpcat3
  - Slc51a
  - Emp1
  - Slc22a3
  - Abcg2
  - Adtrp
  - Aqp8
  - Epb41l4b
  - Pea15
  - Mttp
  - Trnp
  - Fabp1
  - Slc25a20
  - Arg2
  - Aldh3a2
  - Cth
  - Pdk4
  - Slc27a2
  - Abcd3
  - Scd
  - Scd2
  - Gzma
  - Acadm
  - Dgat2
  - Cyp4b1
  - Acot1
  - Pex11a
  - Cpt1a
  - Cat
  - Cpt2
  - Prdx6
  - Axin2
  - Hadha
  - Dio1
  - Rbp7
  - Pygl
  - Dbi
  - Ppara
  - Ppard
  - Hnf4a
  - Nr3c1
  - Pparg
  - Nr1h4
  - Nr1i2
  - Plk2
  - Ndrg1
  - Actn2
  - Nfkbiz
  - Arnt2
  - Bhlhe40
  - Them4
  - Ppargc1a
  - Acsbg1
  - Cybb
  - Lbp
  - Ehhadh
  - Slc26a3
  - Cadm1
  - Pam
  - Mycbp2
  - Nceh1
  - Il1r1
  - Cyp1a1
  - Srd5a1
  - Fa2h
  - Acadl
  - Slc1a1
  - Slc23a2
  - Slc9a3
  - Entpd5
  - Map2k1
  - Sstr2
  - Gnai1
  - Ifngr1
  - Pla1a
  - Lypla1
  - Pea15a
  - Cx3cl1
  - Scd1
  - cac
  - Mip
  - Trpv6
  - Glyat
  - Prdx5
  - Dido1
  - Polr2g
  - Nr1h3
  - Nr1h2
  - Slc23a1
  - GOT2
  - Fatp2
  - scd
  - Hsc70-4
  - Pex11ab
  - Ca-alpha1T
  - ChAT
  - sv
  - Mtpalpha
  - Hrb27C
  - Acbp2
  - Hnf4
  - Fmr1
  - fzr
  - tRNA:Pro-TGG-1-1
  - tRNA:Pro-TGG-1-4
  - tRNA:Pro-TGG-1-3
  - tRNA:Pro-TGG-1-5
  - tRNA:Pro-TGG-1-2
  - fa2h
  - PEA15
  - fatty acid
  - bile acid
  - SCD
---
