---
figid: PMC6160413__41598_2018_32792_Fig6_HTML
figtitle: Impact of osmotic stress on central carbon metabolism
organisms:
- NA
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
pmcid: PMC6160413
filename: 41598_2018_32792_Fig6_HTML.jpg
figlink: /pmc/articles/PMC6160413/figure/Fig6/
number: F6
caption: Impact of osmotic stress on central carbon metabolism. (a) Pathway annotation
  illustrating those enzymes that were detected, and their fold changes in response
  to the osmotic stress (see scale, top left). Any enzyme not detected in these experiments
  is coloured grey. The changes in selected representative proteins are illustrated
  in panel (b). For some proteins, it was possible to compare the proteome data with
  transcriptomic data generated by Enjalbert et al. in terms of relative expression
  of specific proteins compared to the cognate transcripts (c).
papertitle: Specificity of the osmotic stress response in Candida albicans highlighted
  by quantitative proteomics.
reftext: Mette D. Jacobsen, et al. Sci Rep. 2018;8:14492.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5540685
figid_alias: PMC6160413__F6
figtype: Figure
redirect_from: /figures/PMC6160413__F6
ndex: 82037688-dea0-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6160413__41598_2018_32792_Fig6_HTML.html
  '@type': Dataset
  description: Impact of osmotic stress on central carbon metabolism. (a) Pathway
    annotation illustrating those enzymes that were detected, and their fold changes
    in response to the osmotic stress (see scale, top left). Any enzyme not detected
    in these experiments is coloured grey. The changes in selected representative
    proteins are illustrated in panel (b). For some proteins, it was possible to compare
    the proteome data with transcriptomic data generated by Enjalbert et al. in terms
    of relative expression of specific proteins compared to the cognate transcripts
    (c).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FBP1
  - PFK2
  - RPE1
  - XKS1
  - TPI1
  - GPD1
  - GUT2
  - ENO2
  - PBA1
  - MDH2
  - PYC2
  - ADH2
  - ASC1
  - ADH4
  - ACO1
  - ACO2
  - SOH1
  - SDH2
  - SUA7
  - IDP2
  - LSC2
  - GLK1
  - IDP1
  - ICL1
  - GPP1
  - GPD2
  - LSC1
  - MDH3
  - PDA1
  - SOL3
  - ZWF1
  - TDH3
  - GPM1
  - GID7
  - CIT2
  - MIA2
  - PFKFB3
  - TDH
  - SDSL
  - ADH1B
  - CIT
  - DHDDS
  - RPL3
  - TRIP4
  - SLC7A10
  - CXCR1
  - ADH7
  - AVP
  - ADH1A
  - ADH1C
  - ADH5
  - ADH6
  - ADHFE1
  - MED31
  - SDHA
  - SDHB
  - PSMD1
  - H3P21
  - TIA1
  - CSGALNACT2
  - Glucose
  - 6P-gluconolactone
  - Glucose-6-phosphate
  - 6P-gluconolactate
  - Ribulose
  - Fructose-6-phosphate
  - Ribose
  - Fructose-1,6-bisphosphate
  - Arabitol
  - Dirydroxyacetone
  - phosphate
  - Glyceraldehyde
  - 3-phosphate
  - Glycerol-3-phosphate
  - 1,3-Bisphosphoglycerate
  - Giycerol
  - 3-Phosphoglycerate
  - 2-Phosphoglycerate
  - ENO2
  - Phosphoenolpyruvate
  - Oxaloacetate
  - Pyruvate
  - Acetaldehyde
  - EPyruvate
  - Ac
  - Citrate
  - Acetate
  - Ethanol
  - TCA
  - Isocitrate
  - Succinate
  - Succinyl CoA
---
