---
figid: PMC6425516__zh40081722900001
figtitle: Epigenetic regulation of peroxisome proliferator-activated receptorA (PPARA)
  in fatty acid oxidation (FAO) genes (A–C) and PPARA signaling to the FAO metabolome
  (D)
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC6425516
filename: zh40081722900001.jpg
figlink: /pmc/articles/PMC6425516/figure/F0001/
number: F1
caption: 'Schematic representation of epigenetic regulation of peroxisome proliferator-activated
  receptor-α (PPARα) in fatty acid oxidation (FAO) genes (A–C) and PPARα signaling
  to the FAO metabolome (D). A: PPARα heterodimerizes with retinoid X receptor-α (RXRα)
  and binds to the specific region of DNA sequence known as the peroxisome proliferator
  receptor response element (PPRE)/direct repeat 1 (DR1) located in the promoters
  of PPARα target genes, which leads to initiation of transcription by recruiting
  RNA polymerase II (RNA POL II) and other transcription factors. Thus, PPARα positively
  regulates expression of FAO genes throughout the pathway. B: during pressure overload
  (PO), PPARα also interacts with sirtuin 1 (Sirt1), which is a class III histone
  deacetylase (HDAC). PPARα recruits Sirt1 to the promoter regions of a subset of
  PPARα target genes that contain an “imperfect” PPRE/DR1, leading to transcriptional
  repression, presumably through deacetylation of histones. It is hypothesized that
  PPARα also interacts with other epigenetic modulators, which possibly affect the
  methylation (Me) and acetylation (Ac) of histones surrounding the PPARα-binding
  region. (“Me” in pink, associated with gene activation, e.g., H3K4; Me in gray,
  associated with gene repression, e.g., H3K9). C: promoters of genes that are upregulated
  by PPARα principally harbor the “perfect” PPRE/DR1, which contains the consensus
  sequence of two typical hexads with no more than one nucleotide mismatch (yellow).
  In contrast, the imperfect PPRE/DR1 contains one typical hexad and one nonconserved
  (atypical) hexad [red, bottom left, modified from Oka et al. ()] and is found in
  genes involved in β-oxidation, the tricarboxylic acid (TCA) cycle, and the electron
  transport chain (ETC) in the FAO downstream pathway. Thus, the PPARα-Sirt1 complex
  preferentially targets the FAO downstream pathway in the diseased heart, and the
  unique DNA sequences of PPREs allow PPARα to differentially regulate its target
  genes during PO. D: epigenetic modifications regulated by PPARα ultimately determine
  the metabolic phenotype of the diseased heart, which includes the FAO metabolome
  (i.e., free fatty acids, fatty acyl-CoA, carnitine, acylcarnitines, acetyl-CoA),
  fatty acid (FA) storage (triglycerides), and the redox status (cellular levels of
  NAD+, NADH, and FADH2). Free fatty acids are byproducts of lipolysis from triglycerides
  and are the ligands of PPARα. The altered myocardial contents of fatty acids and
  carnitine/acylcarnitine species (bold) are the signature of the metabolomic phenotype
  of the failing heart. OCTN2, organic cation/carnitine transporter 2; FABP, fatty
  acid-binding protein; FACS, fatty acyl-CoA synthetase; MCAD, medium-chain acyl-CoA
  dehydrogenase; PPi, inorganic pyrophosphate.'
papertitle: 'Metabolic reprogramming via PPARα signaling in cardiac hypertrophy and
  failure: From metabolomics to epigenetics.'
reftext: Junco Shibayama Warren, et al. Am J Physiol Heart Circ Physiol. 2017 Sep
  1;313(3):H584-H596.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9339321
figid_alias: PMC6425516__F1
figtype: Figure
redirect_from: /figures/PMC6425516__F1
ndex: b2a15a5a-dee4-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6425516__zh40081722900001.html
  '@type': Dataset
  description: 'Schematic representation of epigenetic regulation of peroxisome proliferator-activated
    receptor-α (PPARα) in fatty acid oxidation (FAO) genes (A–C) and PPARα signaling
    to the FAO metabolome (D). A: PPARα heterodimerizes with retinoid X receptor-α
    (RXRα) and binds to the specific region of DNA sequence known as the peroxisome
    proliferator receptor response element (PPRE)/direct repeat 1 (DR1) located in
    the promoters of PPARα target genes, which leads to initiation of transcription
    by recruiting RNA polymerase II (RNA POL II) and other transcription factors.
    Thus, PPARα positively regulates expression of FAO genes throughout the pathway.
    B: during pressure overload (PO), PPARα also interacts with sirtuin 1 (Sirt1),
    which is a class III histone deacetylase (HDAC). PPARα recruits Sirt1 to the promoter
    regions of a subset of PPARα target genes that contain an “imperfect” PPRE/DR1,
    leading to transcriptional repression, presumably through deacetylation of histones.
    It is hypothesized that PPARα also interacts with other epigenetic modulators,
    which possibly affect the methylation (Me) and acetylation (Ac) of histones surrounding
    the PPARα-binding region. (“Me” in pink, associated with gene activation, e.g.,
    H3K4; Me in gray, associated with gene repression, e.g., H3K9). C: promoters of
    genes that are upregulated by PPARα principally harbor the “perfect” PPRE/DR1,
    which contains the consensus sequence of two typical hexads with no more than
    one nucleotide mismatch (yellow). In contrast, the imperfect PPRE/DR1 contains
    one typical hexad and one nonconserved (atypical) hexad [red, bottom left, modified
    from Oka et al. ()] and is found in genes involved in β-oxidation, the tricarboxylic
    acid (TCA) cycle, and the electron transport chain (ETC) in the FAO downstream
    pathway. Thus, the PPARα-Sirt1 complex preferentially targets the FAO downstream
    pathway in the diseased heart, and the unique DNA sequences of PPREs allow PPARα
    to differentially regulate its target genes during PO. D: epigenetic modifications
    regulated by PPARα ultimately determine the metabolic phenotype of the diseased
    heart, which includes the FAO metabolome (i.e., free fatty acids, fatty acyl-CoA,
    carnitine, acylcarnitines, acetyl-CoA), fatty acid (FA) storage (triglycerides),
    and the redox status (cellular levels of NAD+, NADH, and FADH2). Free fatty acids
    are byproducts of lipolysis from triglycerides and are the ligands of PPARα. The
    altered myocardial contents of fatty acids and carnitine/acylcarnitine species
    (bold) are the signature of the metabolomic phenotype of the failing heart. OCTN2,
    organic cation/carnitine transporter 2; FABP, fatty acid-binding protein; FACS,
    fatty acyl-CoA synthetase; MCAD, medium-chain acyl-CoA dehydrogenase; PPi, inorganic
    pyrophosphate.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CG6356
  - crq
  - fabp
  - Polr2A
  - Polr2C
  - Polr2B
  - Polr2I
  - Polr2F
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - Coa
  - NC2beta
  - Dop1R1
  - ATPsynbeta
  - Atpalpha
  - Acsl
  - Amph
  - whd
  - Sirt1
  - CPT2
  - Me
  - Mcad
  - CycE
  - cyc
  - cal
  - Cam
  - PPARA
  - PPARD
  - PPARG
  - RXRA
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - SLC22A5
  - CD36
  - CRABP1
  - CRABP2
  - FABP1
  - FABP12
  - FABP2
  - FABP3
  - FABP4
  - FABP5
  - FABP6
  - FABP7
  - FABP9
  - PMP2
  - RBP1
  - RBP2
  - RBP5
  - RBP7
  - ERVW-4
  - DR1
  - ATP8A2
  - APRT
  - MFAP1
  - CPT1A
  - CHPT1
  - SIRT1
  - ACADM
  - CDH15
  - FBLIM1
  - GOPC
  - CA1
---
