---
figid: PMC6409899__genes-10-00170-g001
figlink: /pmc/articles/PMC6409899/figure/genes-10-00170-f001/
number: F1
caption: 'Models of replication-coupled ICL repair. In replicating cells, ICLs are
  efficiently repaired by cooperative action of FA pathway proteins and the proteins
  involved in the NER, TLS, and HR pathways. Other than the classical ICL repair model
  of single fork convergence, there are different proposed mechanisms by which crosslinks
  can be repaired in a replication-dependent manner, enabling forks stalled at ICLs
  to resume DNA synthesis. (A) Dual fork convergence model. Crosslink repair is triggered
  when two replication forks converge at an ICL. The leading strands of two converging
  forks initially stall at ~20 nucleotides away from the lesion due to the steric
  hindrance imposed by the CMG complex (Cdc45/MCM2-7/GINS). Ubiquitin signaling promotes
  chromatin unloading of the CMG complex, which allows one of the leading strands
  on either side of the ICL to approach to within one nucleotide of the lesion. The
  FA pathway is simultaneously activated through the monoubiquitination of the FANCI‒FANCD2
  complex. Ubiquitylated FANCI‒FANCD2 in turn promotes recruitment of SLX4 (FANCP),
  which acts as a molecular scaffold to recruit structure-specific endonucleases SLX1,
  XPF (FANCQ)-ERCC1, and MUS81 at the ICL site. Nucleolytic incision and subsequent
  unhooking of the ICL by the coordinated actions of these enzymes creates a DSB in
  one sister chromatid. FAN1, another endonuclease, is also implicated in the incision
  step. TLS polymerases, Pol ζ and REV1 subsequently extend the leading strand past
  the unhooked ICL and generate an intact DNA template suitable for HR repair. The
  DSB is finally repaired by HR and the unhooked ICL is removed by NER process. (B)
  Replication traverse model. In living cells, the majority (~60%) of replication
  forks encountering ICLs bypass the lesions without unhooking them in a manner dependent
  on the translocase activity of FANCM/MHF complex. The “X-shaped” DNA structures
  generated through the traverse process are subsequently processed by endonucleases
  followed by lesion bypass and post-replication repair as in dual fork convergence
  model. (C) Replication fork reversal at ICL. Following CMG complex unloading, one
  of the two converging forks undergoes reversal. The opposite fork is subsequently
  incised by the endonucleases leading to ICL unhooking. The lesion is then bypassed
  by the action of TLS polymerases and DSB intermediate generated during the incision
  process is finally repaired by HR. ICL: Interstrand cross-link; FA: Fanconi anemia;
  NER: Nucleotide excision repair; TLS: Translesion; HR: Homologous recombination.'
pmcid: PMC6409899
papertitle: Holding All the Cards—How Fanconi Anemia Proteins Deal with Replication
  Stress and Preserve Genomic Stability.
reftext: Arindam Datta, et al. Genes (Basel). 2019 Feb;10(2):170.
pmc_ranked_result_index: '160950'
pathway_score: 0.5406148
filename: genes-10-00170-g001.jpg
figtitle: Models of replication-coupled ICL repair
year: '2019'
organisms: Homo sapiens
ndex: 43485371-df37-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6409899__genes-10-00170-g001.html
  '@type': Dataset
  description: 'Models of replication-coupled ICL repair. In replicating cells, ICLs
    are efficiently repaired by cooperative action of FA pathway proteins and the
    proteins involved in the NER, TLS, and HR pathways. Other than the classical ICL
    repair model of single fork convergence, there are different proposed mechanisms
    by which crosslinks can be repaired in a replication-dependent manner, enabling
    forks stalled at ICLs to resume DNA synthesis. (A) Dual fork convergence model.
    Crosslink repair is triggered when two replication forks converge at an ICL. The
    leading strands of two converging forks initially stall at ~20 nucleotides away
    from the lesion due to the steric hindrance imposed by the CMG complex (Cdc45/MCM2-7/GINS).
    Ubiquitin signaling promotes chromatin unloading of the CMG complex, which allows
    one of the leading strands on either side of the ICL to approach to within one
    nucleotide of the lesion. The FA pathway is simultaneously activated through the
    monoubiquitination of the FANCI‒FANCD2 complex. Ubiquitylated FANCI‒FANCD2 in
    turn promotes recruitment of SLX4 (FANCP), which acts as a molecular scaffold
    to recruit structure-specific endonucleases SLX1, XPF (FANCQ)-ERCC1, and MUS81
    at the ICL site. Nucleolytic incision and subsequent unhooking of the ICL by the
    coordinated actions of these enzymes creates a DSB in one sister chromatid. FAN1,
    another endonuclease, is also implicated in the incision step. TLS polymerases,
    Pol ζ and REV1 subsequently extend the leading strand past the unhooked ICL and
    generate an intact DNA template suitable for HR repair. The DSB is finally repaired
    by HR and the unhooked ICL is removed by NER process. (B) Replication traverse
    model. In living cells, the majority (~60%) of replication forks encountering
    ICLs bypass the lesions without unhooking them in a manner dependent on the translocase
    activity of FANCM/MHF complex. The “X-shaped” DNA structures generated through
    the traverse process are subsequently processed by endonucleases followed by lesion
    bypass and post-replication repair as in dual fork convergence model. (C) Replication
    fork reversal at ICL. Following CMG complex unloading, one of the two converging
    forks undergoes reversal. The opposite fork is subsequently incised by the endonucleases
    leading to ICL unhooking. The lesion is then bypassed by the action of TLS polymerases
    and DSB intermediate generated during the incision process is finally repaired
    by HR. ICL: Interstrand cross-link; FA: Fanconi anemia; NER: Nucleotide excision
    repair; TLS: Translesion; HR: Homologous recombination.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MCM2
  - MCM4
  - MCM5
  - FANCI
  - FANCM
  - SLX4
  - FANCD2
  - MCM3
  - MCM7
  - NR1H2
  - MCM6
  - MHP
genes:
- word: MCM2-7
  symbol: MCM2
  source: hgnc_symbol
  hgnc_symbol: MCM2
  entrez: '4171'
- word: MCM2-7
  symbol: MCM4
  source: hgnc_symbol
  hgnc_symbol: MCM4
  entrez: '4173'
- word: MCM2-7
  symbol: MCM5
  source: hgnc_symbol
  hgnc_symbol: MCM5
  entrez: '4174'
- word: FANCI
  symbol: FANCI
  source: hgnc_symbol
  hgnc_symbol: FANCI
  entrez: '55215'
- word: FANCM
  symbol: FANCM
  source: hgnc_symbol
  hgnc_symbol: FANCM
  entrez: '57697'
- word: SLX4
  symbol: SLX4
  source: hgnc_symbol
  hgnc_symbol: SLX4
  entrez: '84464'
- word: FANCD2
  symbol: FANCD2
  source: hgnc_symbol
  hgnc_symbol: FANCD2
  entrez: '2177'
- word: MCM2-7
  symbol: MCM3
  source: hgnc_symbol
  hgnc_symbol: MCM3
  entrez: '4172'
- word: MCM2-7
  symbol: MCM7
  source: hgnc_symbol
  hgnc_symbol: MCM7
  entrez: '4176'
- word: NER
  symbol: NER
  source: hgnc_alias_symbol
  hgnc_symbol: NR1H2
  entrez: '7376'
- word: MCM2-7
  symbol: MCM6
  source: hgnc_symbol
  hgnc_symbol: MCM6
  entrez: '4175'
chemicals: []
diseases:
- word: MHP
  source: ''
  identifier: ''
figid_alias: PMC6409899__F1
redirect_from: /figures/PMC6409899__F1
figtype: Figure
---
