---
figid: PMC6193562__nihms963425f4
figtitle: Applications of Metabolomics to Study Cancer Metabolism
organisms:
- NA
pmcid: PMC6193562
filename: nihms963425f4.jpg
figlink: /pmc/articles/PMC6193562/figure/F4/
number: F4
caption: Metabolic changes define many phenotypic aspects of genetically-determined
  diseases. These diseases generally originate with genomic mutations and are executed
  through changes in the transcriptome, proteome and metabolome. Recent work has emphasized
  the importance of signaling effects caused by perturbed metabolic states, resulting
  in changes in protein function, transcription, and other effects. Examples include
  post-translational protein modification or regulation of these modifications by
  2-HG, Acetyl-CoA, and UDP-GlcNac, all of which can impact cell signaling. Other
  metabolites regulate epigenetic control of the transcriptome or promote further
  genomic alterations. Systems biology provides systematic techniques to interrogate
  the complex interaction of genes and proteins with metabolites. Broadly, high throughput
  data generated from multiple compartments can be integrated with metabolomics using
  three different approaches. Concordance analysis uses direct information from the
  transcript/protein expression of enzymes and levels of product and substrate of
  the reaction. As an example, high levels of glucose and glucose 6-phosphate (Glucose-6-P)
  correlate with elevated hexokinase 1 (HK1) expression. Pathway based enrichment
  analysis uses statistical tests, such as Fisher’s exact test, to determine the likelihood
  of observing alterations in groups of metabolites/genes associated with specific
  metabolic pathways. In the corresponding figure, node size represents the number
  of metabolites in a pathway, and enrichment score represents directionality of enriched
  pathways based on composite score of differential metabolites. Network based integration
  uses interaction information about genes, proteins and metabolites as well as stoichiometry
  information of reactomes to design networks to test enrichment of metabolic pathways
  using several mathematical models. In the figure, node size corresponds to number
  of metabolites in a pathway, and interaction between pathways and directionality
  of flux are represented by arrows of varying width.
papertitle: Applications of Metabolomics to Study Cancer Metabolism.
reftext: Akash K. Kaushik, et al. Biochim Biophys Acta Rev Cancer. 2018 Aug;1870(1):2-14.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7755839
figid_alias: PMC6193562__F4
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC6193562__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6193562__nihms963425f4.html
  '@type': Dataset
  description: Metabolic changes define many phenotypic aspects of genetically-determined
    diseases. These diseases generally originate with genomic mutations and are executed
    through changes in the transcriptome, proteome and metabolome. Recent work has
    emphasized the importance of signaling effects caused by perturbed metabolic states,
    resulting in changes in protein function, transcription, and other effects. Examples
    include post-translational protein modification or regulation of these modifications
    by 2-HG, Acetyl-CoA, and UDP-GlcNac, all of which can impact cell signaling. Other
    metabolites regulate epigenetic control of the transcriptome or promote further
    genomic alterations. Systems biology provides systematic techniques to interrogate
    the complex interaction of genes and proteins with metabolites. Broadly, high
    throughput data generated from multiple compartments can be integrated with metabolomics
    using three different approaches. Concordance analysis uses direct information
    from the transcript/protein expression of enzymes and levels of product and substrate
    of the reaction. As an example, high levels of glucose and glucose 6-phosphate
    (Glucose-6-P) correlate with elevated hexokinase 1 (HK1) expression. Pathway based
    enrichment analysis uses statistical tests, such as Fisher’s exact test, to determine
    the likelihood of observing alterations in groups of metabolites/genes associated
    with specific metabolic pathways. In the corresponding figure, node size represents
    the number of metabolites in a pathway, and enrichment score represents directionality
    of enriched pathways based on composite score of differential metabolites. Network
    based integration uses interaction information about genes, proteins and metabolites
    as well as stoichiometry information of reactomes to design networks to test enrichment
    of metabolic pathways using several mathematical models. In the figure, node size
    corresponds to number of metabolites in a pathway, and interaction between pathways
    and directionality of flux are represented by arrows of varying width.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Glucose
  - Acetyl-CoA
  - UDP
  - Glucose  Glucose-6-P
---
