---
figid: PMC6167743__nihms-952486-f0002
figtitle: Schematic model of a MurNAc transport and utilization pathway
organisms:
- Tannerella forsythia
pmcid: PMC6167743
filename: nihms-952486-f0002.jpg
figlink: /pmc/articles/PMC6167743/figure/F2/
number: F2
caption: '(A) Schematic model of a MurNAc transport and utilization pathway in E.
  coli and T. forsythia (adapted from Rruscitto et al 2016). Cytoplasmic Memb, cytoplasmic
  membrane; PEP, phosphoenolpyruvate; E1, enzyme E1; HPR, histidine protein; EIIA,
  enzyme IIA. Predicted orthologs are sown in parenthesis (gene IDs) and dashed line
  arrows connecting intermediates indicate enzyme orthologs involved are not detected.(B)
  Schematic representation (to scale) of MurTKQ containing loci from a range of human
  dwelling Bacteroidetes. Key: XthA, predicted Xanthan lyase; Ser/Thr, kinase, predicted
  serine/threonine kinase; Gtf, contain predicted glycosyltransferase domains; LytB,
  predicted amidase enhancer; AmpG, predicted muropeptide transporter; Hyp, no known
  homologies found; Ferredoxin, contains 4Fe4s cluster, FAD-binding motif and homology
  to ferredoxin proteins; AfuC, predicted α-L-fucosidase; β-lactamase, predicted b-lactamase;
  MurT, predicted homolog of TfMurT homolog; MurK, predicted homolog of TfMurK; MurQ,
  predicted homolog of MurQ etherase. Gene annotations are predicted from PATRIC/Refseq
  annotations and use of BlastP.'
papertitle: 'Peptidoglycan synthesis in Tannerella forsythia: scavenging is the modus
  operandi.'
reftext: Angela Ruscitto, et al. Mol Oral Microbiol. 2018 Apr;33(2):125-132.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.662774
figid_alias: PMC6167743__F2
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC6167743__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6167743__nihms-952486-f0002.html
  '@type': Dataset
  description: '(A) Schematic model of a MurNAc transport and utilization pathway
    in E. coli and T. forsythia (adapted from Rruscitto et al 2016). Cytoplasmic Memb,
    cytoplasmic membrane; PEP, phosphoenolpyruvate; E1, enzyme E1; HPR, histidine
    protein; EIIA, enzyme IIA. Predicted orthologs are sown in parenthesis (gene IDs)
    and dashed line arrows connecting intermediates indicate enzyme orthologs involved
    are not detected.(B) Schematic representation (to scale) of MurTKQ containing
    loci from a range of human dwelling Bacteroidetes. Key: XthA, predicted Xanthan
    lyase; Ser/Thr, kinase, predicted serine/threonine kinase; Gtf, contain predicted
    glycosyltransferase domains; LytB, predicted amidase enhancer; AmpG, predicted
    muropeptide transporter; Hyp, no known homologies found; Ferredoxin, contains
    4Fe4s cluster, FAD-binding motif and homology to ferredoxin proteins; AfuC, predicted
    α-L-fucosidase; β-lactamase, predicted b-lactamase; MurT, predicted homolog of
    TfMurT homolog; MurK, predicted homolog of TfMurK; MurQ, predicted homolog of
    MurQ etherase. Gene annotations are predicted from PATRIC/Refseq annotations and
    use of BlastP.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MurNAC
  - UDP
  - enolpyruvate
  - ADP
  - MurNAc-6-P
  - lactate
  - UTP
  - GlcN-6-P
  - Fructose-6-P
  - Ser
  - Thr
---
