---
figid: PMC6921603__12870_2019_2195_Fig6_HTML
figtitle: Schematic representation of magnoflorine biosynthetic pathway and the expression
  profile of those unigenes of the C
organisms:
- NA
pmcid: PMC6921603
filename: 12870_2019_2195_Fig6_HTML.jpg
figlink: /pmc/articles/PMC6921603/figure/Fig6/
number: F6
caption: 'Schematic representation of magnoflorine biosynthetic pathway and the expression
  profile of those unigenes of the C. draco species which coding for the enzymes involved.
  In the figure, the molecular structure of each proposed intermediate is represented,
  followed by the enzymes (acronyms shown in red) and their corresponding orthologs
  identified in the C. draco transcriptome. Expression profiles representing the abundance
  of the transcripts (FPKM) in each organ analysed are also shown for each of the
  C. draco unigenes identified as orthologs of the reference enzymes. Acronyms: TYDC
  (tyrosine decarboxylase); NCS ((S)-norcoclaurine synthase); 6OMT ((S)-norcoclaurine
  6-O-methyltransferase); CNMT ((S)-coclaurine N-methyltransferase); NMCH ((S)-N-methylcoclaurine
  3′-hydroxylase, CYP82B subfamily); 4’OMT ((S)3′-hydroxy N-methylcoclaurine 4′-O-methyltransferase);
  CTS ((S)-corytuberine synthase, CYP80G2)'
papertitle: Novel findings to the biosynthetic pathway of magnoflorine and taspine
  through transcriptomic and metabolomic analysis of Croton draco (Euphorbiaceae).
reftext: Anahí Canedo-Téxon, et al. BMC Plant Biol. 2019;19:560.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.875283
figid_alias: PMC6921603__F6
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC6921603__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6921603__12870_2019_2195_Fig6_HTML.html
  '@type': Dataset
  description: 'Schematic representation of magnoflorine biosynthetic pathway and
    the expression profile of those unigenes of the C. draco species which coding
    for the enzymes involved. In the figure, the molecular structure of each proposed
    intermediate is represented, followed by the enzymes (acronyms shown in red) and
    their corresponding orthologs identified in the C. draco transcriptome. Expression
    profiles representing the abundance of the transcripts (FPKM) in each organ analysed
    are also shown for each of the C. draco unigenes identified as orthologs of the
    reference enzymes. Acronyms: TYDC (tyrosine decarboxylase); NCS ((S)-norcoclaurine
    synthase); 6OMT ((S)-norcoclaurine 6-O-methyltransferase); CNMT ((S)-coclaurine
    N-methyltransferase); NMCH ((S)-N-methylcoclaurine 3′-hydroxylase, CYP82B subfamily);
    4’OMT ((S)3′-hydroxy N-methylcoclaurine 4′-O-methyltransferase); CTS ((S)-corytuberine
    synthase, CYP80G2)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Root
  - as
  - thi
  - thiv
  - be
  - step
  - L-tyrosine
  - (S)3-hydroxy-N
  - methylcoclaurine
  - 4OMT
  - para-hydroxyphenyl
  - dopamine
  - acetaldehyde
  - (S)-norcoclaurine
  - 6OMT
  - Aporphine
  - UniGene103967
  - (S)-coclaurine UniGene103967
  - magnoflorine
  - (S)-N-methylcoclaurine
  - 2-oxoglutarate
  - (S)-coclaurine
  - Coptis japonica
---
