---
figid: PMC6080592__c8mo00042e-f3
figtitle: 'Developments in toxicogenomics: understanding and predicting compound-induced
  toxicity from gene expression data'
organisms:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Drosophila melanogaster
- Caenorhabditis elegans
- Gorilla gorilla
- Streptococcus pneumoniae
pmcid: PMC6080592
filename: c8mo00042e-f3.jpg
figlink: /pmc/articles/PMC6080592/figure/fig3/
number: F3
caption: '. An example network of enriched KEGG pathways of compound-induced differentially
  expressed genes relating to liver fibrosis. Of the 15 pathways shown, down-regulated
  pathways were predominantly metabolism related; whereas up-regulated pathways were
  related to processes associated with liver fibrosis, for example, the focal adhesion
  pathway is pathway 3 (P3) and also immune related pathways are depicted (P7-antigen
  processing or P9-chemokine signaling). The authors hypothesised that the metabolic
  pathways could be related to external factors (e.g. altered food intake) or an indication
  of reduced liver function. Genes with an average fold change >1.5 are in red, <0.75
  are in green, and the remaining in grey. PX represents pathway number X and is represented
  in a blue circle. DOI: ; 10.1371/journal.pone.0112193.g003.'
papertitle: 'Developments in toxicogenomics: understanding and predicting compound-induced
  toxicity from gene expression data.'
reftext: Benjamin Alexander-Dann, et al. Mol Omics. 2018 Aug 1;14(4):218-236.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6142771
figid_alias: PMC6080592__F3
figtype: Figure
organisms_ner:
- Rattus norvegicus
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
- Caenorhabditis elegans
redirect_from: /figures/PMC6080592__F3
ndex: 3b6129b2-df37-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6080592__c8mo00042e-f3.html
  '@type': Dataset
  description: '. An example network of enriched KEGG pathways of compound-induced
    differentially expressed genes relating to liver fibrosis. Of the 15 pathways
    shown, down-regulated pathways were predominantly metabolism related; whereas
    up-regulated pathways were related to processes associated with liver fibrosis,
    for example, the focal adhesion pathway is pathway 3 (P3) and also immune related
    pathways are depicted (P7-antigen processing or P9-chemokine signaling). The authors
    hypothesised that the metabolic pathways could be related to external factors
    (e.g. altered food intake) or an indication of reduced liver function. Genes with
    an average fold change >1.5 are in red, <0.75 are in green, and the remaining
    in grey. PX represents pathway number X and is represented in a blue circle. DOI:
    ; 10.1371/journal.pone.0112193.g003.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Dao
  - Cxcl9
  - Bax
  - Prnp
  - A2m
  - Tkfc
  - Inpp5d
  - Otc
  - Plau
  - Fabp1
  - Oat
  - Aldh1b1
  - Jak2
  - Casp12
  - Ptpre
  - Prodh
  - Prodh2
  - F2r
  - Thbd
  - Plat
  - Cdkn2b
  - Lpl
  - Cxcl10
  - Plpp1
  - Gngt2
  - Vav1
  - Gsn
  - Rac2
  - Cxcl16
  - Gnai2
  - Rhoc
  - Cd86
  - COX1
  - Pkm
  - RT1-Db1
  - Hagh
  - Mdh1
  - Hspa1b
  - Hspa1a
  - Fabp2
  - Lgmn
  - Cyba
  - Acox2
  - Tapbp
  - Cd74
  - Mmp2
  - RT1-Da
  - Ppara
  - Cybb
  - Mpeg1
  - RT1-DMa
  - Limk2
  - Ctss
  - Chdh
  - Fyn
  - RT1-Ba
  - Ezr
  - RT1-M3-1
  - Itgb2
  - Itgal
  - Itga1
  - Col3a1
  - Itga2b
  - Vasp
  - Itgb3
  - S100a10
  - Pdgfra
  - Egf
  - Birc3
  - C1qb
  - Cxcl12
  - Ldhb
  - Fabp7
  - Bcar1
  - Orm1
  - Prkcb
  - RT1-DMb
  - Pde1a
  - Pin1
  - fc
  - Aoc1
  - Nkx3-1
  - Pzp
  - Pcolce
  - S100a9
  - Lamtor2
  - Fabp4
  - Sub1
  - Mrpl28
  - Atcay
  - Lp1
  - Hapln1
  - Zbtb17
  - Me1
  - Atg7
  - Fmnl2
  - Cdk2ap1
  - Spink1
  - Pold4
  - Col1a2
  - Lncpint
  - Birc2
  - Ps
  - Cse1l
  - CXCL10
  - CXCL11
  - CXCL12
  - CXCL13
  - CXCL14
  - CXCL9
  - PF4
  - CXCL1
  - CXCL2
  - CXCL3
  - CXCL5
  - CXCL6
  - CXCL8
  - PPBP
  - DAO
  - AOC1
  - BAX
  - PRNP
  - A2M
  - IGHA2
  - TKFC
  - INPP5D
  - OTC
  - PLAU
  - CDKN2A
  - S100A9
  - CDK2AP2
  - SUB1
  - RPP14
  - LAMTOR2
  - SF3B6
  - H3P8
  - OAT
  - ALDH1B1
  - JAK2
  - CASP12
  - PTPRE
  - PRODH
  - PRODH2
  - S100A10
  - EXOSC3
  - F2R
  - THBD
  - PLAT
  - CDKN2B
  - MRPL28
  - NXT1
  - H3P9
  - COL25A1
  - ATCAY
  - LPL
  - LCP1
  - RPLP1
  - SLC7A7
  - PLPP1
  - GNGT2
  - VAV1
  - GSN
  - RAC2
  - CXCL16
  - GNAI2
  - RHOC
  - CD86
  - SOD1
  - SOD2
  - SOD3
  - TRAPPC2
  - PKM
  - PKLR
  - HAGH
  - MDH1
  - HSPA1B
  - EXOSC1
  - H3P6
  - FABP2
  - PRTN3
  - LGMN
  - CYBA
  - ACOX2
  - TAPBP
  - CD74
  - MMP2
  - PPARA
  - PPARD
  - PPARG
  - CA6
  - CYBB
  - MPEG1
  - LIMK2
  - CTSS
  - PAX3
  - FYN
  - EZR
  - ITGB2
  - ITGAL
  - ITGA1
  - COL3A1
  - ITGA2B
  - VASP
  - ITGB3
  - PML
  - MYL1
  - MYL3
  - MYL4
  - MYL6
  - MYL6B
  - MYL10
  - MYL12A
  - MYL12B
  - MYL2
  - MYL5
  - MYL7
  - MYL9
  - MYL11
  - POLE4
  - POLD4
  - H3P5
  - ENDOU
  - EXOSC6
  - LINC-PINT
  - PDGFRA
  - EGF
  - BIRC3
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - C1QB
  - GEN1
  - LDHB
  - FABP7
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - ATP5MK
  - PRKCB
  - PDE1A
  - PIN1
  - PDZD2
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Sh
  - AP-1gamma
  - gammaTry
  - E(spl)mgamma-HLH
  - dao
  - Daao1
  - slgA
  - elB
  - dap
  - Nxt1
  - RpLP1
  - so
  - Sod2
  - Sod1
  - coi
  - Curl
  - vib
  - P13
  - man
  - alpha-Man-IIa
  - ena
  - Ssb-c31a
  - pll
  - Hrb87F
  - Egfr
  - Act5C
  - Act42A
  - Act57B
  - Act79B
  - Act88F
  - Act87E
  - kn
  - Gen
  - ps
  - cas
  - caz
  - Cse1
  - cass
  - cash
  - pe
  - dod
  - vav-1
  - rac-2
  - mdh-1
  - Arginine
  - proline
  - Cd
  - Pyruvate
  - erythematosus
  - Viral myocarditis
---
