---
figid: PMC6255705__ery32509
figtitle: Schematic representation of metabolic reprogramming caused by the DAPAT
  mutation during diurnal cycle
organisms:
- Arabidopsis thaliana
- Nicotiana tabacum
- Oryza sativa
- Zea mays
- Solanum lycopersicum
- Chlamydomonas reinhardtii
- Petunia x hybrida
- Lolium perenne
- Pseudomonas syringae
- Rhodospirillum rubrum
- Homo sapiens
pmcid: PMC6255705
filename: ery32509.jpg
figlink: /pmc/articles/PMC6255705/figure/F9/
number: F9
caption: 'Schematic representation of metabolic reprogramming caused by the DAPAT
  mutation during diurnal cycle. Despite lower stomatal conductance, biochemical changes
  were responsible for lower photosynthesis, including an impact on carbohydrate metabolism
  resulting in putative energy deprivation and further accumulation of protein and
  amino acids, leading to C/N imbalance, which seems to be associated with the dwarfism
  phenotype in dapat plants. In order to obtain a metabolic adjustment, we postulate
  a hypothetical model in which malate is hyper-accumulated at the end of day (ED).
  Malate is further oxidized during the night by an alternative pathway associated
  with the cytosolic malic enzyme 2 to generated pyruvate and, thus, to sustain mitochondrial
  dark respiration at similar level when compared with WT plants. Furthermore, higher
  turnover of both protein and amino acids in dapat plants acts as a source for alternative
  respiration mediated by the ETF–ETFQO complex and dehydrogenases such as IVDH and
  D2HGDH. Data obtained from the transcriptome, proteome, and metabolome were integrated
  to build the mechanism described here. Rectangles represent transcript data; circles
  represent protein data, and underlined names represent metabolite data. Increased
  and reduced levels from the data obtained are shown in black uppercase and grey
  lowercase letters, respectively. Abbreviations: Cb6f, complex b6/f; CS, citrate
  synthase; cME2, cytosolic malic enzyme 2; D2HGDH, D-2-hydroxyglutarate dehydrogenase;
  DAPAT, L-L-diaminopimelate aminotransferase; ETF–ETFQO, electron-transfer flavoprotein–electron-transfer
  flavoprotein: ubiquinone oxidoredutase; GAD1, glutamate decarboxylase; GADPH, glyceraldehyde-3-phosphate
  dehydrogenase; IVDH, isovaleryl-CoA dehydrogenase; PRK, phosphoribulokinase; PSI,
  photosystem I; PSII, photosystem 2; TRX, thioredoxin; WT, wild type.'
papertitle: An L,L-diaminopimelate aminotransferase mutation leads to metabolic shifts
  and growth inhibition in Arabidopsis.
reftext: João Henrique F Cavalcanti, et al. J Exp Bot. 2018 Dec 1;69(22):5489-5506.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9395195
figid_alias: PMC6255705__F9
figtype: Figure
organisms_ner:
- Homo sapiens
- Arabidopsis thaliana
redirect_from: /figures/PMC6255705__F9
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6255705__ery32509.html
  '@type': Dataset
  description: 'Schematic representation of metabolic reprogramming caused by the
    DAPAT mutation during diurnal cycle. Despite lower stomatal conductance, biochemical
    changes were responsible for lower photosynthesis, including an impact on carbohydrate
    metabolism resulting in putative energy deprivation and further accumulation of
    protein and amino acids, leading to C/N imbalance, which seems to be associated
    with the dwarfism phenotype in dapat plants. In order to obtain a metabolic adjustment,
    we postulate a hypothetical model in which malate is hyper-accumulated at the
    end of day (ED). Malate is further oxidized during the night by an alternative
    pathway associated with the cytosolic malic enzyme 2 to generated pyruvate and,
    thus, to sustain mitochondrial dark respiration at similar level when compared
    with WT plants. Furthermore, higher turnover of both protein and amino acids in
    dapat plants acts as a source for alternative respiration mediated by the ETF–ETFQO
    complex and dehydrogenases such as IVDH and D2HGDH. Data obtained from the transcriptome,
    proteome, and metabolome were integrated to build the mechanism described here.
    Rectangles represent transcript data; circles represent protein data, and underlined
    names represent metabolite data. Increased and reduced levels from the data obtained
    are shown in black uppercase and grey lowercase letters, respectively. Abbreviations:
    Cb6f, complex b6/f; CS, citrate synthase; cME2, cytosolic malic enzyme 2; D2HGDH,
    D-2-hydroxyglutarate dehydrogenase; DAPAT, L-L-diaminopimelate aminotransferase;
    ETF–ETFQO, electron-transfer flavoprotein–electron-transfer flavoprotein: ubiquinone
    oxidoredutase; GAD1, glutamate decarboxylase; GADPH, glyceraldehyde-3-phosphate
    dehydrogenase; IVDH, isovaleryl-CoA dehydrogenase; PRK, phosphoribulokinase; PSI,
    photosystem I; PSII, photosystem 2; TRX, thioredoxin; WT, wild type.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GNPAT
  - SPPL2B
  - DECR1
  - TXN
  - VAC14
  - CP
  - PLK3
  - GAD1
  - CS
  - D2HGDH
  - IVD
  - CRT3
  - AT4G11320
  - PRK
  - GAD
  - NADP NADPH
  - GADPH
  - AMINO ACIDS
  - pyruvate
  - AMINO
  - citrate
  - TCA
  - Tryptophan
  - ETF-ETFQO
---
