---
figid: PMC6899527__gr3
figtitle: Single-Cell Sequencing of ChAT/vAChT-Positive Cortical Cells and Analysis
  of Their Expressed TranscriptsExpression values were normalized (0–1) for each dataset
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Nicotiana tabacum
- Canis lupus familiaris
pmcid: PMC6899527
filename: gr3.jpg
figlink: /pmc/articles/PMC6899527/figure/fig3/
number: F3
caption: Single-Cell Sequencing of ChAT/vAChT-Positive Cortical Cells and Analysis
  of Their Expressed TranscriptsExpression values were normalized (0–1) for each dataset.
  The order of genes in each heatmap reflects transcript clustering rather than level
  differences. Columns represent individual samples from original data (see column
  names in ); cholinergic genes are denoted in blue and neurokine receptors in orange.(A)
  Clustered single-cell sequences of the transgenic mouse somatosensory cortex and
  hippocampus ().(B) Clustered single-cell sequences from the transgenic mouse visual
  cortex ().(C) Single-nucleus sequencing of adult mouse hippocampus ().(D) Single-cell
  sequencing of the human developing neocortex ().(E) Permutation analysis of miRNA
  and TF targeting data of genes expressed in cholinergic cells (via miRNet) identified
  putative cholinergic/neurokine co-regulators (∗p < 0.05, ∗∗p < 0.001). Implicated
  TFs are regulated by a subset of miRNAs targeting cholinergic genes, indicating
  nested regulation (details in ).(F–I) Gp130-family neurokine, cholinergic, and circadian
  signaling pathways are controlled by primate-specific and evolutionarily conserved
  miRNAs. Components expressed in cholinergic neurons (F), processes following receptor
  binding (G) and STAT phosphorylation (H), and components of cholinoceptive neurons
  (I) are shown. miRNA targeting of individual genes (indicated by colored symbols)
  yields complex transcriptional interactions. Several miRNAs directly targeting the
  cholinergic pathway also target TFs controlling this pathway (circles and triangles).
papertitle: Integrative Transcriptomics Reveals Sexually Dimorphic Control of the
  Cholinergic/Neurokine Interface in Schizophrenia and Bipolar Disorder.
reftext: Sebastian Lobentanzer, et al. Cell Rep. 2019 Oct 15;29(3):764-777.e5.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8000042
figid_alias: PMC6899527__F3
figtype: Figure
organisms_ner:
- Rattus norvegicus
- Mus musculus
- Homo sapiens
- Canis lupus familiaris
redirect_from: /figures/PMC6899527__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6899527__gr3.html
  '@type': Dataset
  description: Single-Cell Sequencing of ChAT/vAChT-Positive Cortical Cells and Analysis
    of Their Expressed TranscriptsExpression values were normalized (0–1) for each
    dataset. The order of genes in each heatmap reflects transcript clustering rather
    than level differences. Columns represent individual samples from original data
    (see column names in ); cholinergic genes are denoted in blue and neurokine receptors
    in orange.(A) Clustered single-cell sequences of the transgenic mouse somatosensory
    cortex and hippocampus ().(B) Clustered single-cell sequences from the transgenic
    mouse visual cortex ().(C) Single-nucleus sequencing of adult mouse hippocampus
    ().(D) Single-cell sequencing of the human developing neocortex ().(E) Permutation
    analysis of miRNA and TF targeting data of genes expressed in cholinergic cells
    (via miRNet) identified putative cholinergic/neurokine co-regulators (∗p < 0.05,
    ∗∗p < 0.001). Implicated TFs are regulated by a subset of miRNAs targeting cholinergic
    genes, indicating nested regulation (details in ).(F–I) Gp130-family neurokine,
    cholinergic, and circadian signaling pathways are controlled by primate-specific
    and evolutionarily conserved miRNAs. Components expressed in cholinergic neurons
    (F), processes following receptor binding (G) and STAT phosphorylation (H), and
    components of cholinoceptive neurons (I) are shown. miRNA targeting of individual
    genes (indicated by colored symbols) yields complex transcriptional interactions.
    Several miRNAs directly targeting the cholinergic pathway also target TFs controlling
    this pathway (circles and triangles).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tf
  - Chrna4
  - Ntrk2
  - Rbfox3
  - Gad1
  - Vip
  - Ache
  - Chrm3
  - Chrm1
  - Chrnb2
  - Chat
  - Cntfr
  - Lifr
  - Chrna7
  - Sst
  - Lhx6
  - Chrm2
  - Bche
  - Chrna5
  - Gfap
  - Chrna3
  - Chrm4
  - Olig1
  - Prima1
  - Ntrk1
  - Nkx2-1
  - Chrna6
  - Chrnb3
  - Colq
  - Clps
  - Slc18a3
  - Chrm5
  - Chrnb4
  - Aif1
  - Gbx1
  - Lhx8
  - Gbx2
  - Isl1
  - Chrna10
  - Chrna2
  - Ngfr
  - Slc5a7
  - Il6r
  - Il6st
  - Cntf
  - Lif
  - Il6
  - Jak1
  - Jak2
  - Tyk2
  - Clock
  - Bmal1
  - Stat1
  - Stat5a
  - Stat3
  - Ad
  - Sh2d3c
  - Lmx1a
  - Alpi
  - Il6ra
  - Gm44505
  - Il11ra1
  - Lrpprc
  - H2-Aa
  - Hacu
  - ao
  - Gnao1
  - TF
  - CHRNA4
  - NTRK2
  - RBFOX3
  - GAD1
  - VIP
  - CPAMD8
  - ACHE
  - CHRM3
  - CHRM1
  - CHRNB2
  - CHAT
  - SH2D3C
  - CNTFR
  - LIFR
  - CHRNA7
  - CHRFAM7A
  - SST
  - LHX6
  - CHRM2
  - BCHE
  - CHRNA5
  - GFAP
  - CHRNA3
  - CHRM4
  - OLIG1
  - PRIMA1
  - NTRK1
  - NKX2-1
  - CHRNA6
  - CHRNB3
  - COLQ
  - SLC18A3
  - CHRM5
  - CHRNB4
  - AIF1
  - GBX1
  - LHX8
  - GBX2
  - ISL1
  - CHRNA10
  - CHRNA2
  - NGFR
  - SLC5A7
  - IGKV2-36
  - IL6R
  - IL6ST
  - ASPH
  - IL6
  - CNTF
  - LIF
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - NM
  - LRPPRC
  - JAK1
  - JAK2
  - TYK2
  - CLOCK
  - BMAL1
  - LOC608697
  - ARNTL
  - CHAT
---
