---
figid: PMC6260860__12864_2018_5238_Fig3_HTML
figtitle: Different gene expression profiles in metabolic pathways of E
organisms:
- Escherichia coli
- Bacillus subtilis At2
- Escherichia coli str. K-12 substr. MG1655
- Moorella thermoacetica
- Clostridium ljungdahlii
- Acetobacterium woodii
- Clostridium autoethanogenum
- Eubacterium limosum
- Clostridium ljungdahlii DSM 13528
- Carboxydothermus hydrogenoformans
- Clostridium aceticum
- Eubacterium limosum KIST612
- Homo sapiens
- Saccharomyces cerevisiae
- Bos taurus
organisms_ner:
- Bos taurus
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
- Moorella thermoacetica
- Drosophila melanogaster
pmcid: PMC6260860
filename: 12864_2018_5238_Fig3_HTML.jpg
figlink: /pmc/articles/PMC6260860/figure/Fig3/
number: F3
caption: 'Different gene expression profiles in metabolic pathways of E. limosum ATCC
  8486. (a) Embden-Meyerhof-Parnas glycolytic pathway (blue circle), incomplete tricarboxylic
  acid cycle (brown circle), Wood-Ljungdahl pathway (green circle), and pentose phosphate
  pathway (purple circle) are shown. (b) Ethanol producing pathway (red circle), hydrogenase
  complex (grey circle), Rnf complex (dark circle), ETF complex (blue circle), and
  ATP synthase complex (orange circle) are shown. Metabolites are shown as circles
  and reactions are shown as arrows. All gene numbers and enzyme reactions are provided
  next to the arrows. Transcriptional dynamics of E. limosum ATCC 8486 are described
  in the heatmap with three columns; the left and middle boxes indicate normalized
  RNA reads from heterotrophic and autotrophic conditions, respectively. The right
  box indicates RNA fold change, as autotrophic expression over heterotrophic expression.
  In addition, gene numbers responsible for the expression are located under the bottom
  box of the heatmap. A full list and expression data can be found in Additional file
  : Table S1'
papertitle: Genome-scale analysis of syngas fermenting acetogenic bacteria reveals
  the translational regulation for its autotrophic growth.
reftext: Yoseb Song, et al. BMC Genomics. 2018;19:837.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5573007
figid_alias: PMC6260860__F3
figtype: Figure
redirect_from: /figures/PMC6260860__F3
ndex: eeb133e4-df36-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6260860__12864_2018_5238_Fig3_HTML.html
  '@type': Dataset
  description: 'Different gene expression profiles in metabolic pathways of E. limosum
    ATCC 8486. (a) Embden-Meyerhof-Parnas glycolytic pathway (blue circle), incomplete
    tricarboxylic acid cycle (brown circle), Wood-Ljungdahl pathway (green circle),
    and pentose phosphate pathway (purple circle) are shown. (b) Ethanol producing
    pathway (red circle), hydrogenase complex (grey circle), Rnf complex (dark circle),
    ETF complex (blue circle), and ATP synthase complex (orange circle) are shown.
    Metabolites are shown as circles and reactions are shown as arrows. All gene numbers
    and enzyme reactions are provided next to the arrows. Transcriptional dynamics
    of E. limosum ATCC 8486 are described in the heatmap with three columns; the left
    and middle boxes indicate normalized RNA reads from heterotrophic and autotrophic
    conditions, respectively. The right box indicates RNA fold change, as autotrophic
    expression over heterotrophic expression. In addition, gene numbers responsible
    for the expression are located under the bottom box of the heatmap. A full list
    and expression data can be found in Additional file : Table S1'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PTS
  - FOLR3
  - GAPDH
  - TKT
  - RPE
  - MTHFR
  - ATR
  - MMAB
  - PC
  - PLA2G15
  - ACCS
  - CS
  - ACE
  - TNK2
  - glk
  - pgi
  - fbp
  - glcB
  - pfkA
  - fbaA
  - tpiA
  - tktA
  - pgk
  - rpe
  - eno
  - acs
  - mdh
  - acnA
  - folD
  - pta
  - GCK
  - MAP4K3
  - BGN
  - GPI
  - PGPEP1
  - FBP1
  - FOLR1
  - FOLR2
  - ECB2
  - FUBP1
  - RAB14
  - ADH5
  - ALDH1L1
  - IK
  - DYRK3
  - ATP8A2
  - WDTC1
  - PFKM
  - FBXO3
  - TALDO1
  - LRSAM1
  - TPI1
  - SULT1A1
  - TRIP10
  - GAPDHP44
  - MTHFD1
  - DDR2
  - RPIA
  - MSMB
  - PGAM1
  - PGAM2
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - ANTXR1
  - SERPINA2
  - ACSS2
  - MDH2
  - NF2
  - ACIN1
  - IDH1
  - IDH2
  - OPRK1
  - TRIM31
  - TEAD2
  - ATP5F1A
  - ATP5F1B
  - ATP5F1C
  - ATP5F1D
  - ATP5F1E
  - ATP5MK
  - ATP5ME
  - ATP5MF
  - ATP5MG
  - ATP5MJ
  - .na.character
  - ATP5PB
  - ATP5PD
  - ATP5PF
  - ATP5PO
  - ATP5IF1
  - ATP5MC1
  - ATP5MC2
  - ATP5MC3
  - TCF23
  - F11
  - PTCRA
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH6
  - ADH7
  - ADHFE1
  - PAEP
  - PREP
  - PTPN22
  - fba
  - ppdK
  - pts
  - PheRS-m
  - Pgi
  - Fdh
  - fd
  - anon-16Fb
  - red
  - trc
  - GluClalpha
  - betaGlu
  - ATPsynbeta
  - Atpalpha
  - adp
  - G6P
  - Ald1
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - Aldh-III
  - Tpi
  - stp
  - Mhc
  - Gapdh2
  - Gpdh1
  - Gapdh1
  - Pgk
  - Rpi
  - Rpe
  - pug
  - Prps
  - Pglym78
  - Eno
  - pe
  - pk
  - mei-41
  - put
  - RYa-R
  - Gug
  - pyr
  - Pc
  - Coa
  - pd
  - AcCoAS
  - Acsl
  - cs
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - Acn
  - Fum1
  - Idh
  - Idh3b
  - fc
  - Sh
  - na
  - wal
  - out
  - Ace
  - Acer
  - Ance
  - Ack
  - aor
  - Adh
  - elm
  - Pep
  - bgcn
  - pi
  - DTO
  - Fd
  - ATP
  - ADP
  - ATP ADP
  - O ELIM c2267
  - Formate
  - THF
  - DHAP
  - Formyl
  - NAD
  - NADH
  - BPG
  - Methenyl
  - O
  - 3PG
  - Methylene
  - Ru5P
  - ENO
  - Methyl
  - CoFeS-P
  - CO
  - HCO
  - OAc
  - Oxaloacetate
  - O Citrate
  - O Malate
  - Isocitrate
  - 2-oxoglutarate
  - Succinyl-CoA O
  - Na
  - NADH NAD
  - Fd Fd
  - Ac
  - Acetyl
  - Acetate
  - Ethanol
  - PD
---
