---
figid: PMC6959606__pntd.0007945.g004
figtitle: Gene expression profiling of Trypanosoma cruzi in the presence of heme points
  to glycosomal metabolic adaptation of epimastigotes inside the vector
organisms:
- NA
pmcid: PMC6959606
filename: pntd.0007945.g004.jpg
figlink: pmc/articles/PMC6959606/figure/pntd.0007945.g004/
number: F4
caption: 'Trypanosoma cruzi epimastigotes metabolic pathway illustrating differentially
  expressed genes in response to heme treatment involved in energy metabolism measured
  by RNAseq analysis. Values inbox are expressed as fold change of heme treated parasites
  in relation to control sample. Enzymes reactions were presented as known (black
  solid arrow), downregulated suggested-routes (black dashed arrow) and downregulated
  (gray solid arrow) routes by RNAseq data. Yellow circles indicate pyridoxal 5-phosphate
  binding enzymes. The enzymes of each proposed pathway are presented as specific
  colors. Glucose metabolism (blue): ST: sugar transporter; GALKg: galactokinase,
  glycosomal; HKg: hexokinase, glycosomal; PMIg: phosphomannose isomerase, glycosomal;
  PFKg: phosphofructokinase, glycosomal; ALDg: fructose bisphosphate aldolase, glycosomal;
  ENOc: enolase, cytosolic. Fermentation process (green): PPDKg: pyruvate phosphate
  dikinase, glycosomal; PEPCKg: phosphoenolpyruvate carboxykinase, glycosomal; MDHg:
  malate dehydrogenase, glycosomal; FRDg*: NADH-dependent fumarate reductase, glycosomal.
  Amino acids metabolism (purple): KBL: 2-amino-3-ketobutyrate CoA ligase; P5CDH:
  δ-1-pyrroline-5-carboxylate dehydrogenase; ASAT: aspartate aminotransferase; AS:
  asparagine synthetase A. AHADH: aromatic L-2-hydroxyacid dehydrogenase; Malate metabolism
  (orange): ME: malic enzyme. Mitochondrial metabolism (pink): COMPLEX I: NADH-ubiquinone
  oxidoreductase; SDH: succinate dehydrogenase subunit. Other metabolic pathways (gray):
  PdxK: pyridoxal kinase; ABCt: ABC transporter. Metabolytes: G-6-P: glucose-6-phosphate;
  F-6-P: fructose-6-phosphate; F-1,6-P: fructose-1,6-biphosphate; DHAP: dihydroxyacetone
  phosphate; GA3P: glyceraldehyde-3-phosphate; 1,3-BPGA: 1,3-bisphosphoglycerate;
  3-PGA: 3-phosphoglycerate; 2-PGA: 2-phosphoglycerate; PEP: phosphoenolpyruvate;
  OXA: oxaloacetate; Glu: glutamate; Gln: glutamine; Phe: phenylalanine; Trp: tryptophan;
  Tyr: tyrosine; P5C: δ-1-pyrroline-5-carboxylate; AOB: amino-oxobutyrate.'
papertitle: Gene expression profiling of Trypanosoma cruzi in the presence of heme
  points to glycosomal metabolic adaptation of epimastigotes inside the vector.
reftext: Marcia C. Paes, et al. PLoS Negl Trop Dis. 2020 Jan;14(1):e0007945.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9347332
figid_alias: PMC6959606__F4
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC6959606__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6959606__pntd.0007945.g004.html
  '@type': Dataset
  description: 'Trypanosoma cruzi epimastigotes metabolic pathway illustrating differentially
    expressed genes in response to heme treatment involved in energy metabolism measured
    by RNAseq analysis. Values inbox are expressed as fold change of heme treated
    parasites in relation to control sample. Enzymes reactions were presented as known
    (black solid arrow), downregulated suggested-routes (black dashed arrow) and downregulated
    (gray solid arrow) routes by RNAseq data. Yellow circles indicate pyridoxal 5-phosphate
    binding enzymes. The enzymes of each proposed pathway are presented as specific
    colors. Glucose metabolism (blue): ST: sugar transporter; GALKg: galactokinase,
    glycosomal; HKg: hexokinase, glycosomal; PMIg: phosphomannose isomerase, glycosomal;
    PFKg: phosphofructokinase, glycosomal; ALDg: fructose bisphosphate aldolase, glycosomal;
    ENOc: enolase, cytosolic. Fermentation process (green): PPDKg: pyruvate phosphate
    dikinase, glycosomal; PEPCKg: phosphoenolpyruvate carboxykinase, glycosomal; MDHg:
    malate dehydrogenase, glycosomal; FRDg*: NADH-dependent fumarate reductase, glycosomal.
    Amino acids metabolism (purple): KBL: 2-amino-3-ketobutyrate CoA ligase; P5CDH:
    δ-1-pyrroline-5-carboxylate dehydrogenase; ASAT: aspartate aminotransferase; AS:
    asparagine synthetase A. AHADH: aromatic L-2-hydroxyacid dehydrogenase; Malate
    metabolism (orange): ME: malic enzyme. Mitochondrial metabolism (pink): COMPLEX
    I: NADH-ubiquinone oxidoreductase; SDH: succinate dehydrogenase subunit. Other
    metabolic pathways (gray): PdxK: pyridoxal kinase; ABCt: ABC transporter. Metabolytes:
    G-6-P: glucose-6-phosphate; F-6-P: fructose-6-phosphate; F-1,6-P: fructose-1,6-biphosphate;
    DHAP: dihydroxyacetone phosphate; GA3P: glyceraldehyde-3-phosphate; 1,3-BPGA:
    1,3-bisphosphoglycerate; 3-PGA: 3-phosphoglycerate; 2-PGA: 2-phosphoglycerate;
    PEP: phosphoenolpyruvate; OXA: oxaloacetate; Glu: glutamate; Gln: glutamine; Phe:
    phenylalanine; Trp: tryptophan; Tyr: tyrosine; P5C: δ-1-pyrroline-5-carboxylate;
    AOB: amino-oxobutyrate.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - st
  - ATPsynGL
  - ATPsynG
  - adp
  - ATPsynbeta
  - Atpalpha
  - G6P
  - Aldh-III
  - Pep
  - bgcn
  - Pepck2
  - Pepck1
  - as
  - Pdxk
  - pe
  - Ids
  - Me
  - P5CDh1
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - LKRSDH
  - SdhB
  - SdhA
  - Sodh-2
  - SdhC
  - hh
  - WDTC1
  - ATP8A2
  - PAEP
  - PREP
  - PTPN22
  - PCK2
  - PEPC
  - PGC
  - PDXK
  - ABCB7
  - IDS
  - DECR1
  - HADH
  - HADHA
  - GCAT
  - ALDH4A1
  - PYCR1
  - BRCA2
  - FANCD2
  - PSEN1
  - EIF2B5
  - RTRAF
  - SARDH
  - SDHB
  - SORD
  - SDS
  - Galactose
  - Glucose
  - ADP
  - Galactose-1-P ATP
  - UDP
  - Glu
  - Mannose
  - ATP
  - Succinate
  - NAD
  - Mannose-6-P
  - NADH
  - F-6-P
  - Alanine
  - DHAP
  - Pyruvate
  - Asparagine
  - 1,3-BPGA 3-PGA
  - Phe
  - Trp
  - Aspartate L-Tyr
  - Pyridoxal
  - Pyridoxal-5-P
  - HCO
  - NADPH
  - NADP
  - Aromatic 2-hydroxi acids
  - NAD NADH COA-SH Glycine
  - L-threonine
  - Thr
  - Acetyl
  - NAD(P)H
  - L-proline
  - TCA
  - FADH
  - FAD
---
