---
figid: PMC6992476__nihms-1049429-f0002
figtitle: Accelerating Medicines Partnership – Organizational Structure and Preliminary
  Data from the Phase 1 Studies of Lupus Nephritis
organisms:
- Homo sapiens
- Mus musculus
- Eucalyptus intertexta
organisms_ner:
- Mus musculus
- Drosophila melanogaster
pmcid: PMC6992476
filename: nihms-1049429-f0002.jpg
figlink: pmc/articles/PMC6992476/figure/F2/
number: F2
caption: Samples and clinical data are collected at point of care (red box). Patient
  data is loaded into RedCap and samples are processed according to optimized protocols
  and shipped to the Sample and Tissue Repository for distribution to the technical
  sites. Proteomic analyses, CYTOF and scRNAseq are each performed at different technical
  sites (blue box) and data analyses and integration (green box) are performed by
  the scientific groups. Examples of analyses and integration methods are shown in
  the bottom panel. Profiled cells are grouped into clusters sharing similar gene
  expression patterns (Step 1) (). The dimensionality of the expression data of these
  genes is reduced using PCA (principal component analysis) and the resulting low-dimensional
  data analyzed using graph-based clustering (Step 2a). Further subclustering reveals
  cell subtypes (Step 2b). Cluster labeling is performed by taking into account the
  distribution of known lineage markers across clusters; the identity of genes specifically
  upregulated in each cluster; and by comparing the gene expression data of each cluster
  to those of published datasets of reference samples (Step 2c). Individual gene expression
  (violin plot) in each cell subtype can be generated (Step 3). Pathway enrichment
  analysis using curated databases and gene ontology analysis provides information
  about which genes are active in each pathway (Step 4). Developmental trajectories
  are constructed by linking cell types to progenitor populations (Step 5) and regulatory
  relationships can be inferred between genes using cellular interdependency networks
  (Step 6). Cluster analysis of CyTOF data-displayed as a tSNE plot (Box). Phase 2
  will integrate multimodal data to address the goals shown in  and to generate hypotheses.
papertitle: The Accelerating Medicines Partnership – Organizational Structure and
  Preliminary Data from the Phase 1 Studies of Lupus Nephritis.
reftext: Paul Hoover, et al. Arthritis Care Res (Hoboken). ;72(2):233-242.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5936999
figid_alias: PMC6992476__F2
figtype: Figure
redirect_from: /figures/PMC6992476__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6992476__nihms-1049429-f0002.html
  '@type': Dataset
  description: Samples and clinical data are collected at point of care (red box).
    Patient data is loaded into RedCap and samples are processed according to optimized
    protocols and shipped to the Sample and Tissue Repository for distribution to
    the technical sites. Proteomic analyses, CYTOF and scRNAseq are each performed
    at different technical sites (blue box) and data analyses and integration (green
    box) are performed by the scientific groups. Examples of analyses and integration
    methods are shown in the bottom panel. Profiled cells are grouped into clusters
    sharing similar gene expression patterns (Step 1) (). The dimensionality of the
    expression data of these genes is reduced using PCA (principal component analysis)
    and the resulting low-dimensional data analyzed using graph-based clustering (Step
    2a). Further subclustering reveals cell subtypes (Step 2b). Cluster labeling is
    performed by taking into account the distribution of known lineage markers across
    clusters; the identity of genes specifically upregulated in each cluster; and
    by comparing the gene expression data of each cluster to those of published datasets
    of reference samples (Step 2c). Individual gene expression (violin plot) in each
    cell subtype can be generated (Step 3). Pathway enrichment analysis using curated
    databases and gene ontology analysis provides information about which genes are
    active in each pathway (Step 4). Developmental trajectories are constructed by
    linking cell types to progenitor populations (Step 5) and regulatory relationships
    can be inferred between genes using cellular interdependency networks (Step 6).
    Cluster analysis of CyTOF data-displayed as a tSNE plot (Box). Phase 2 will integrate
    multimodal data to address the goals shown in  and to generate hypotheses.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Lat2
  - H2-Ab1
  - Ptprc
  - Pou2f3
  - Fam91a1
  - Pfdn4
  - Oog1
  - Ptpn5
  - 'On'
  - lab
  - vnd
  - Dhc64C
  - abd-A
  - Abd-B
  - lncRNA:iab4
  - lncRNA:iab8
  - step
  - CCT3
---
