---
figid: PMC6962170__41598_2019_57142_Fig4_HTML
figtitle: Integrated metabolic, proteomic and transcriptomic functional analysis
organisms:
- NA
pmcid: PMC6962170
filename: 41598_2019_57142_Fig4_HTML.jpg
figlink: pmc/articles/PMC6962170/figure/Fig4/
number: F4
caption: 'Integrated metabolic, proteomic and transcriptomic functional analysis.
  The network represents the results from a KEGG pathway analysis of the DEGs (red
  and blue gene names) and modified proteins (orange gene names) included in Fig. ,
  as well as the metabolites listed in Fig. . Only pathways (nodes) including at least
  one metabolite and five hits are shown. The complete list of KEGG results are shown
  in Supplementary Table . Gene names are codified as in Fig. : blue and red gene
  names correspond to genes the transcripts of which appear as under- or over-represented
  in AA-treated samples, respectively; orange gene names indicate proteins for which
  at least one peptide showed more than 20% of its Cys residues as propionamide adducts.
  KEGG codes (in magenta) represent the metabolites listed in Fig. ; only natural
  metabolites could be included in the KEGG analysis, and all of them were found underrepresented
  in the AA-treated metabolome (Fig. ). KEGG codes are the following: C00003, NAD+;
  C00025, L-Glutamate; C00041, L-Alanine; C00049, L-Aspartate; C00051, GSH; C00127,
  GSSG; C00386, Carnosine; C00719 Betaine. Ovals show the position of relevant pathways.
  The central position of L-Glutamate, which participated in most identified pathways,
  is marked with a red frame.'
papertitle: 'Targeting redox metabolism: the perfect storm induced by acrylamide poisoning
  in the brain.'
reftext: Demetrio Raldúa, et al. Sci Rep. 2020;10:312.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8457225
figid_alias: PMC6962170__F4
figtype: Figure
organisms_ner:
- Danio rerio
- Homo sapiens
redirect_from: /figures/PMC6962170__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6962170__41598_2019_57142_Fig4_HTML.html
  '@type': Dataset
  description: 'Integrated metabolic, proteomic and transcriptomic functional analysis.
    The network represents the results from a KEGG pathway analysis of the DEGs (red
    and blue gene names) and modified proteins (orange gene names) included in Fig. ,
    as well as the metabolites listed in Fig. . Only pathways (nodes) including at
    least one metabolite and five hits are shown. The complete list of KEGG results
    are shown in Supplementary Table . Gene names are codified as in Fig. : blue and
    red gene names correspond to genes the transcripts of which appear as under- or
    over-represented in AA-treated samples, respectively; orange gene names indicate
    proteins for which at least one peptide showed more than 20% of its Cys residues
    as propionamide adducts. KEGG codes (in magenta) represent the metabolites listed
    in Fig. ; only natural metabolites could be included in the KEGG analysis, and
    all of them were found underrepresented in the AA-treated metabolome (Fig. ).
    KEGG codes are the following: C00003, NAD+; C00025, L-Glutamate; C00041, L-Alanine;
    C00049, L-Aspartate; C00051, GSH; C00127, GSSG; C00386, Carnosine; C00719 Betaine.
    Ovals show the position of relevant pathways. The central position of L-Glutamate,
    which participated in most identified pathways, is marked with a red frame.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - cdaa
  - nme2b.1
  - gabra1
  - thraa
  - kita
  - ngfra
  - htr1ab
  - chac1
  - hrh3
  - gap43
  - rars2
  - ahcyl1
  - abca1a
  - mtap
  - tuba1b
  - mat2b
  - sufu
  - aldoca
  - aspa
  - smox
  - foxg1a
  - prmt1
  - tdh
  - alas1
  - stat3
  - GABRA1
  - GOT1
  - SLC17A5
  - CHAC1
  - HRH3
  - RASA1
  - RGS6
  - RARS2
  - AHCYL1
  - MTAP
  - TUBA1B
  - MAT2B
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - ASPA
  - TMT1B
  - SMOX
  - RICTOR
  - FOXG1
  - PRMT1
  - TDH
  - SDSL
  - ALAS1
  - STAT3
  - FOXO1
  - FOXO3
  - FOXO4
  - FOXO6
  - Glutathione
  - Amino Acid
  - Carbon
---
