---
figid: PMC6391848__fgene-10-00091-g004
figtitle: Enrichment analysis of differentially expressed mRNAs
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC6391848
filename: fgene-10-00091-g004.jpg
figlink: /pmc/articles/PMC6391848/figure/F4/
number: F4
caption: Enrichment analysis of differentially expressed mRNAs. (A) Advanced bubble
  chart of GO-BP. Y-axis label represents terms name, and X-axis label typifies rich
  factor which is defined as the percentage of target genes per term. Size and color
  of the bubble are measured as count and –log10qvalue to represent the amount of
  differentially expressed mRNAs enriched in these terms and the enrichment significance,
  respectively, as well as three shapes of SampleGroup represent three functional
  clusters. (B) KEGG interaction network. Terms are represented as nodes based on
  the kappa score level (= 0.4), and node size indicates the term enrichment significant.
  Edges represent shared genes, the wider the edge, the larger the overlap is. The
  colors indicating different functional groups are shown at the right top. (C) Bos
  taurus Wnt signaling pathway map (). Upregulated genes in our network are marked
  in red.
papertitle: Characterization of lncRNA–miRNA–mRNA Network to Reveal Potential Functional
  ceRNAs in Bovine Skeletal Muscle.
reftext: Binglin Yue, et al. Front Genet. 2019;10:91.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6352031
figid_alias: PMC6391848__F4
figtype: Figure
redirect_from: /figures/PMC6391848__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6391848__fgene-10-00091-g004.html
  '@type': Dataset
  description: Enrichment analysis of differentially expressed mRNAs. (A) Advanced
    bubble chart of GO-BP. Y-axis label represents terms name, and X-axis label typifies
    rich factor which is defined as the percentage of target genes per term. Size
    and color of the bubble are measured as count and –log10qvalue to represent the
    amount of differentially expressed mRNAs enriched in these terms and the enrichment
    significance, respectively, as well as three shapes of SampleGroup represent three
    functional clusters. (B) KEGG interaction network. Terms are represented as nodes
    based on the kappa score level (= 0.4), and node size indicates the term enrichment
    significant. Edges represent shared genes, the wider the edge, the larger the
    overlap is. The colors indicating different functional groups are shown at the
    right top. (C) Bos taurus Wnt signaling pathway map (). Upregulated genes in our
    network are marked in red.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - toe
  - be
  - sie
  - sosie
  - cas
  - longevity
  - sah
  - Pdfr
  - Gr5a
  - Polr3E
  - CycE
  - Proc
  - E(spl)malpha-BFM
  - Ama-1
  - na
  - Eye
  - ey
  - cl
  - ci
  - ads
  - ot
  - ProtA
  - ty
  - ais
  - Pen
  - alphaTub85E
  - Prosalpha2
  - nAChRalpha2
  - A3
  - nAChRalpha3
  - Prosalpha3
  - alphaTub84D
  - term
  - se
  - Past1
  - Rbf
  - lt
  - Rich
  - CTNNB1
  - HNRNPM
  - FLNB
  - GSC-DT
  - SFRP2
  - ACSM3
  - EFS
  - POLR3E
  - PROC
  - LYVE1
  - C1QA
  - C1QB
  - C1QC
  - C1R
  - C1S
  - AR
  - TP63
  - IS1
  - GPR162
  - EHD1
  - RB1
  - RBM45
  - HSD11B2
  - GO-BP
  - chramcsone
  - thymocyne
  - proteomal ubijulirdependefi
  - Colorectal cancer
  - SAH
  - mitoea rucieer
---
