---
figid: PMC6659691__41598_2019_47381_Fig1_HTML
figtitle: Computational Models Accurately Predict Multi-Cell Biomarker Profiles in
  Inflammation and Cancer
organisms:
- NA
pmcid: PMC6659691
filename: 41598_2019_47381_Fig1_HTML.jpg
figlink: /pmc/articles/PMC6659691/figure/Fig1/
number: F1
caption: An illustration shows the general process for creating single cell and multi-cell
  computational models that were validated directly by comparing the predicted results
  against the chemokine and cytokine profiles produced from the same cells grown in
  single cell and multi-cell cultures. (A) Cell-specific computational models were
  created in a series of successive steps. Information from high quality genomic,
  transcriptomic, proteomic, and metabolomic datasets were downloaded into the computational
  network library and used to create computational models for single GE KER, DC, and
  HTL. We used these individual models to create GE KER + DC + HTL multi-cell computational
  models to predict the effects of inflammatory events. The predicted responses were
  validated against the observed responses of the same cells grown in multi-cell cultures.
  (B) In multi-cell cultures, each cell type reacts to both the presence of an added
  LPS agonist as well as the presence of chemokines and cytokines produced from adjacent
  cells, and vice versa. (C) The autocrine and paracrine responses of the multi-cell
  interaction were modeled and the IFNG pathway is included as an example. The predictive
  responses were calculated using Michaelis Menton kinetics; mass action kinetics;
  as ordinary differential equations (ODEs). ODEs contained the reaction; enzyme;
  initial concentrations of protein intermediate reactants; and the Ka, Km, kcat,
  and Vmax parameters of the reactants at each step of the signaling process.
papertitle: Computational Models Accurately Predict Multi-Cell Biomarker Profiles
  in Inflammation and Cancer.
reftext: Carol L. Fischer, et al. Sci Rep. 2019;9:10877.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8700723
figid_alias: PMC6659691__F1
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6659691__F1
ndex: 1ea2060e-df1a-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6659691__41598_2019_47381_Fig1_HTML.html
  '@type': Dataset
  description: An illustration shows the general process for creating single cell
    and multi-cell computational models that were validated directly by comparing
    the predicted results against the chemokine and cytokine profiles produced from
    the same cells grown in single cell and multi-cell cultures. (A) Cell-specific
    computational models were created in a series of successive steps. Information
    from high quality genomic, transcriptomic, proteomic, and metabolomic datasets
    were downloaded into the computational network library and used to create computational
    models for single GE KER, DC, and HTL. We used these individual models to create
    GE KER + DC + HTL multi-cell computational models to predict the effects of inflammatory
    events. The predicted responses were validated against the observed responses
    of the same cells grown in multi-cell cultures. (B) In multi-cell cultures, each
    cell type reacts to both the presence of an added LPS agonist as well as the presence
    of chemokines and cytokines produced from adjacent cells, and vice versa. (C)
    The autocrine and paracrine responses of the multi-cell interaction were modeled
    and the IFNG pathway is included as an example. The predictive responses were
    calculated using Michaelis Menton kinetics; mass action kinetics; as ordinary
    differential equations (ODEs). ODEs contained the reaction; enzyme; initial concentrations
    of protein intermediate reactants; and the Ka, Km, kcat, and Vmax parameters of
    the reactants at each step of the signaling process.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - IFNG
  - IL4
  - IL13
  - IL10
  - CSF2
  - IFNGR2
  - IL18
  - IL22
  - JAK2
  - IL17A
  - IFNGR1
  - IL12A
  - IL12B
  - IL2RA
  - IL2RB
  - IL2RG
  - TNFRSF1A
  - RORC
  - CD28
  - CD40LG
  - IL2
  - CD86
  - CD80
  - TLR4
  - CD40
  - TNF
  - VEGFA
  - IRF6
  - IL17RA
  - IL22RA1
  - CXCL10
  - S100A1
  - S100B
  - S100G
  - S100P
  - S100Z
  - HRNR
  - S100A10
  - S100A11
  - S100A12
  - S100A13
  - S100A14
  - S100A16
  - S100A2
  - S100A3
  - S100A4
  - S100A5
  - S100A6
  - S100A7
  - S100A7A
  - S100A8
  - S100A9
  - TCHHL1
  - CCL20
  - COL1A1
  - COL1A2
  - HLA-C
  - EMP1
  - Co80
  - LPS
---
