---
figid: PMC6833135__12934_2019_1244_Fig3_HTML
figtitle: Gene expression levels of central carbon metabolic pathways
organisms:
- NA
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
pmcid: PMC6833135
filename: 12934_2019_1244_Fig3_HTML.jpg
figlink: /pmc/articles/PMC6833135/figure/Fig3/
number: F3
caption: Gene expression levels of central carbon metabolic pathways. Tricarboxylic
  acid (TCA) cycle, glyoxylate pathway, gluconeogenesis, glycogenesis and pentose
  phosphate pathway (PPP) are presented. The comparative analysis includes the log2
  fold-change (log2FC) xylose/glucose under carbon limitation conditions. The green
  label indicates overexpressed enzymes, “fbr” indicates feedback-resistant
papertitle: Metabolic engineering and transcriptomic analysis of Saccharomyces cerevisiae
  producing p-coumaric acid from xylose.
reftext: Gheorghe M. Borja, et al. Microb Cell Fact. 2019;18:191.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9251561
figid_alias: PMC6833135__F3
figtype: Figure
redirect_from: /figures/PMC6833135__F3
ndex: a94e16b1-df0c-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6833135__12934_2019_1244_Fig3_HTML.html
  '@type': Dataset
  description: Gene expression levels of central carbon metabolic pathways. Tricarboxylic
    acid (TCA) cycle, glyoxylate pathway, gluconeogenesis, glycogenesis and pentose
    phosphate pathway (PPP) are presented. The comparative analysis includes the log2
    fold-change (log2FC) xylose/glucose under carbon limitation conditions. The green
    label indicates overexpressed enzymes, “fbr” indicates feedback-resistant
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HXT2
  - HXT5
  - ATH1
  - TPS3
  - GND1
  - GSY2
  - GND2
  - UGP1
  - PGM1
  - PGM2
  - RKI1
  - FBP26
  - GPH1
  - PFK26
  - PFK27
  - PFK1
  - PFK2
  - GUT2
  - GPP2
  - GPP1
  - TDH1
  - TDH2
  - TDH3
  - ADH4
  - ADH2
  - ADH6
  - SFA1
  - CDC19
  - PYK2
  - PDC6
  - ADY2
  - PYC2
  - ACS1
  - ACS2
  - LAT1
  - MDH2
  - CIT2
  - MDH3
  - MAE1
  - CIT3
  - ACO1
  - DAL7
  - SDH1
  - DIF1
  - SOR1
  - SDH2
  - SDH3
  - SDH4
  - IDP2
  - IDH1
  - IDP3
  - IDH2
  - FRD1
  - KGD2
  - LSC2
  - FPS1
  - TPI1
  - ADH1
  - ENO2
  - ADH5
  - PDC5
  - ACO2
  - KGD1
  - ATOH1
  - UGP2
  - PGLS
  - PFKM
  - PFKFB3
  - RASA1
  - RGS6
  - SLC5A5
  - ADH7
  - ADH1B
  - CD151
  - PTK2B
  - WDTC1
  - ATP8A2
  - NATP
  - ACSL1
  - ACSL6
  - ACSL5
  - SLC7A5
  - LAT
  - DECR1
  - BRCA2
  - FANCD2
  - PSEN1
  - MAL
  - MRTFA
  - TIRAP
  - CIT
  - DHDDS
  - SDHA
  - SDHB
  - SDHC
  - LOC642502
  - SDHD
  - DLST
  - TH
  - ADH1A
  - PAEP
  - PREP
  - PTPN22
  - OGDH
  - Glucose
  - Trehalose
  - O GSY2
  - UDPG
  - O PGM2
  - O PFK26
  - O PFK27
  - O PFK1
  - O PFK2
  - FBP1O
  - Glycerol
  - DHAP
  - Xylitol
  - xyloselglucose
  - Ethanol
  - ENO1O
  - ADH2 O
  - ENO2O
  - SFA1 O
  - PDC5O
  - PDC6 O
  - Acetate
  - ADP
  - ATP
  - NAD
  - NADH
  - NADP
  - NADPH
  - FAD
  - FADH
  - O FADH
  - O ACO1
  - ACO20
  - ICI
  - O IDP3
---
