---
figid: PMC6386537__IAI.00613-18-f003a
figtitle: Immunologic Pathways in Protective versus Maladaptive Host Responses
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC6386537
filename: IAI.00613-18-f003a.jpg
figlink: /pmc/articles/PMC149053/figure/fig7/
number: F3
caption: Immune pathways highlighting differential expression of genes following exposure
  to Rlow and LAVs. Pathways include cytokine, cytokine receptor interaction (A),
  MAPK signaling (B), NOD-like receptor (C), Jak-STAT signaling (D), and Toll-like
  receptor (E). DEGs identified in pathways are indicated for each infection group
  by red stars (control versus Rlow), blue stars (control versus Mg7), and yellow
  stars (control versus GT5) as identified by DAVID bioinformatics software and the
  KEGG database (). Green shading represents the completeness of the pathway identified
  within the Gallus gallus genome. TLR-15 and -21 were added to the original output,
  which is based on mammalian pathways. Copyright, Kanehisa Laboratories; reproduced
  with permission.
papertitle: Immunologic Pathways in Protective versus Maladaptive Host Responses to
  Attenuated and Pathogenic Strains of Mycoplasma gallisepticum.
reftext: Jessica Beaudet, et al. Infect Immun. 2019 Mar;87(3):e00613-18.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9238684
figid_alias: PMC6386537__F3
figtype: Figure
redirect_from: /figures/PMC6386537__F3
ndex: 2eac7d97-de8e-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6386537__IAI.00613-18-f003a.html
  '@type': Dataset
  description: Immune pathways highlighting differential expression of genes following
    exposure to Rlow and LAVs. Pathways include cytokine, cytokine receptor interaction
    (A), MAPK signaling (B), NOD-like receptor (C), Jak-STAT signaling (D), and Toll-like
    receptor (E). DEGs identified in pathways are indicated for each infection group
    by red stars (control versus Rlow), blue stars (control versus Mg7), and yellow
    stars (control versus GT5) as identified by DAVID bioinformatics software and
    the KEGG database (). Green shading represents the completeness of the pathway
    identified within the Gallus gallus genome. TLR-15 and -21 were added to the original
    output, which is based on mammalian pathways. Copyright, Kanehisa Laboratories;
    reproduced with permission.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - TNF
  - TNFRSF1A
  - CXCL2
  - CXCL3
  - IL3RA
  - IL2
  - ACVR2A
  - ACHE
  - CCL25
  - INHBA
  - INHBE
  - OSM
  - CCM2
  - CCR4
  - CNOT6
  - ABCC1
  - MARCKSL1
  - WLS
  - ZBTB20
  - LIFR
  - CCR7
  - EGF
  - ADM2
  - AMH
  - CNTFR
  - CCL2
  - CNTF
  - CCR2
  - RBL2
  - NOLC1
  - RAB3GAP1
  - FLT3LG
  - TNFRSF9
  - LTA
  - LTBR
  - BMP2
  - IFN1@
  - IFNA1
  - IFNA2
  - IFNA17
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA21
  - LTB
  - CRYGC
  - CRYGEP
  - CCL1
  - CCL3L1
  - CCL3L3
  - CCL4L1
  - CCL4L2
  - CCL5
  - CCL7
  - CCL8
  - CCL11
  - CCL13
  - CCL14
  - CCL15
  - CCL16
  - CCL17
  - CCL18
  - CCL19
  - CCL20
  - CCL21
  - CCL22
  - CCL24
  - CCL26
  - CCL27
  - CCL28
  - CXCL1
  - CXCL5
  - CXCL6
  - CXCL8
  - CXCL9
  - CXCL10
  - CXCL11
  - CXCL12
  - CXCL13
  - CXCL14
  - CXCL16
  - CXCL17
  - XCL1
  - XCL2
  - CCL3
  - CCL4
  - GOT1
  - SLC17A5
  - CPSF4
  - IFNK
  - BMP1
  - BMP3
  - BMP4
  - BMP5
  - BMP6
  - BMP7
  - BMP8A
  - BMP8B
  - BMP10
  - BMP15
  - GDF1
  - GDF2
  - GDF3
  - GDF5
  - GDF6
  - GDF7
  - FASLG
  - LEP
  - EREG
  - IL21
  - IL22
  - HAND1
  - TSLP
  - CRLF2
  - IL24
  - SAGE1
  - EDA
  - LRRC15
  - EDAR
  - GHR
  - GH2
  - RPL12
  - RSL1D1
  - IGKV1-5
  - EDA2R
  - SIPA1
  - SFTPA1
  - IL1RN
  - LNPEP
  - PRL
  - TRO
  - IL23R
  - IFNL1
  - MPL
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - MAP3K14
  - MAP4K4
  - CHUK
  - IKBKB
  - IKBKG
  - IKBKE
  - TBK1
  - DAO
  - AOC1
  - EIF2B5
  - ACOT12
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - THAS
  - MRPL49
  - BDNF
  - BDNF-AS
  - RAP1A
  - RABGEF1
  - TERF2IP
  - RAP1B
  - PLA2G1B
  - PLA2G2A
  - PLA2G6
  - PLA2G3
  - PLA2G5
  - PLA2G7
  - PLA2G10
  - PLA2G12A
  - PLA2G12B
  - PLA2G2C
  - PLA2G2D
  - PLA2G2E
  - PLA2G2F
  - PLA2G4A
  - PLA2G4C
  - PLA2G4D
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - RPS6KA3
  - MPI
  - SLPI
  - ELK1
  - KCNH4
  - KCNH8
  - POR
  - VDAC2
  - MAP2K2
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - ZNF143
  - EPHB2
  - S100A4
  - CAP1
  - PRSS8
  - TRAF3
  - MOS
  - MOCOS
  - NEFL
  - MYC
  - FLNA
  - ARRB1
  - ARRB2
  - JUN
  - JUNB
  - JUND
  - GCK
  - MAP4K3
  - ATF2
  - GDNF
  - MAP3K13
  - EPB41
  - CUX1
  - CYTIP
  - CRTAP
  - AKT1
  - AKT2
  - AKT3
  - MAP3K12
  - SLC25A3
  - REG1A
  - PTPRU
  - TRAF2
  - TANK
  - MAP4K2
  - FAM20C
  - CD300C
  - MAP3K6
  - FAS
  - FASN
  - MAP2K3
  - DAXX
  - MAX
  - ELF4
  - MEFV
  - TAB2
  - CD14
  - NDUFA2
  - IRF6
  - INPP5K
  - TRAF6
  - PPM1B
  - PPP2CB
  - ECSIT
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - NLK
  - GPI
  - RPS6KB2
  - ZAP70
  - RPL18
  - IGKV1D-13
  - IGKV1-13
  - CXCR1
  - MAP3K8
  - SULT1E1
  - ITPR1
  - INHBB
  - APP
  - SUCLA2
  - KDR
  - LIF
  - GPR182
  - CXCR3
  - ADGRL2
  - CCDC80
  - DCLK2
  - ERVW-5
  - CRLS1
  - CCR1
  - SF1
  - NR5A1
  - IL19
  - IFNL3
  - IL18
  - TPO
  - THPO
  - IL26
  - LRPAP1
  - SUCLG2
  - GNB1
  - GNB2
  - GNB3
  - GNB4
  - GNB5
  - GRB2
  - GNA12
  - GNA13
  - CRK
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAP3K11
  - CTSA
  - CECR7
  - CYC1
  - CCNI
  - TP53
  - TP63
  - TP73
  - PPP6R3
  - TAB3
  - HSPB1
  - HSPB2
  - IFNO
  - OHI
  - PEC
  - O
  - Cr
  - CHD
  - Phoephatidylinoeito
  - MEF
  - LPS
---
