---
figid: PMC6160310__JCTH-6-310-g001
figtitle: Hepatitis E Virus life cycle
organisms:
- Human immunodeficiency virus
- Human papillomavirus
- unidentified influenza virus
- Dengue virus
- Vaccinia virus
- West Nile virus
- Japanese encephalitis virus
- Dengue virus type 2
- SARS coronavirus NS-1
- Homo sapiens
- Mus musculus
- NA
- NA
- NA
pmcid: PMC6160310
filename: JCTH-6-310-g001.jpg
figlink: /pmc/articles/PMC6160310/figure/f01/
number: F1
caption: HEV enters a permissive cell supposedly through a receptor-dependent process,
  aided by heparan sulfate proteoglycans (HSPGs) and other unknown factors. The viral
  genome is released, ORF1 gets translated and processed into different functional
  domains, followed by replication. Multiple copies of capped (green box) genomic
  (g)RNA and subgenomic (sg)RNAs are thus produced. SgRNAs synthesize viral capsid
  (ORF2) and ORF3 proteins. ORF2, gRNA and other viral and/or host factors mediate
  assembly of new virions, which are released out of the cell through an endosomal
  sorting complex required for transport (ESCRT)-dependent process involving the viral
  ORF3 protein. The green asterik indicates the steps that can be targeted for antiviral
  development. A, B, C and D represent the unknown factors present in the viral replication
  complex. Note that ORF4 is present only in the case of genotype 1 HEV. Known antivirals
  have been indicated at the appropriate steps. The mode of pegylated-interferon-alpha
  (PEG-IFN-α) action is represented through the illustration of the inteferon-alpha
  (IFN-α) signaling pathway. PEG-IFN-α induces the production of interferon-stimulated
  proteins (ISGs) and interferon-inducible transmemebrane proteins (IFITM), which
  activate the canonical antiviral signaling pathways that results in the inhibition
  of HEV entry and/or replication.
papertitle: Recent Advances Towards the Development of a Potent Antiviral Against
  the Hepatitis E Virus.
reftext: Saumya Anang, et al. J Clin Transl Hepatol. 2018 Sep 28;6(3):310-316.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9318405
figid_alias: PMC6160310__F1
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
redirect_from: /figures/PMC6160310__F1
ndex: 4e2e100f-df0a-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6160310__JCTH-6-310-g001.html
  '@type': Dataset
  description: HEV enters a permissive cell supposedly through a receptor-dependent
    process, aided by heparan sulfate proteoglycans (HSPGs) and other unknown factors.
    The viral genome is released, ORF1 gets translated and processed into different
    functional domains, followed by replication. Multiple copies of capped (green
    box) genomic (g)RNA and subgenomic (sg)RNAs are thus produced. SgRNAs synthesize
    viral capsid (ORF2) and ORF3 proteins. ORF2, gRNA and other viral and/or host
    factors mediate assembly of new virions, which are released out of the cell through
    an endosomal sorting complex required for transport (ESCRT)-dependent process
    involving the viral ORF3 protein. The green asterik indicates the steps that can
    be targeted for antiviral development. A, B, C and D represent the unknown factors
    present in the viral replication complex. Note that ORF4 is present only in the
    case of genotype 1 HEV. Known antivirals have been indicated at the appropriate
    steps. The mode of pegylated-interferon-alpha (PEG-IFN-α) action is represented
    through the illustration of the inteferon-alpha (IFN-α) signaling pathway. PEG-IFN-α
    induces the production of interferon-stimulated proteins (ISGs) and interferon-inducible
    transmemebrane proteins (IFITM), which activate the canonical antiviral signaling
    pathways that results in the inhibition of HEV entry and/or replication.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - STAT1
  - IFITM1
  - IFITM2
  - IFITM3
  - IFITM5
  - IFITM10
  - IRF9
  - TNF
  - CXCL8
  - IFNG
  - IL12A
  - IL12B
  - SMOC1
  - EIF2AK2
  - IRF7
  - ISG15
  - NCKIPSD
  - AHI1
  - DEFA1
  - CBX5
  - DEFA1B
  - NPC2
  - BMP1
  - PRCP
  - PGPEP1
  - ASZ1
  - APP
  - SUCLA2
  - EIF4E
  - EIF4E2
  - EIF4E3
  - ACTG1
  - EIF4G1
  - EIF4G2
  - EIF4G3
  - TUBB
  - TUBB2A
  - TUBB1
  - TUBB2B
  - TUBB3
  - TUBB4A
  - TUBB4B
  - TUBB6
  - EIF3A
  - EIF6
  - EIF3J
  - CTTNBP2
  - IFNGR1
  - IFNAR2
  - EEF1A1
  - TSG101
  - EIF4A2
  - Stat1
  - Irf9
  - Tnf
  - Cxcl15
  - Ifng
  - Eif2ak2
  - Irf7
  - Isg15
  - L1Md-Tf30
  - Med14
  - Maged2
  - Nckipsd
  - Cbx5
  - Hp
  - Npc2
  - pcp
  - Bmp1
  - Pgpep1
  - Prcp
  - Emc7
  - Asz1
  - Ofd1
  - App
  - H2-Ab1
  - Eif4e
  - Actg1
  - Tubb4a
  - Eif3a
  - Eif3j1
  - Cttnbp2
  - Ifngr1
  - Ifnar2
  - Eef1a1
  - Eef1a1-ps1
  - Tsg101
  - Eif4a2
  - Eif4b
  - PEG
  - Met
  - Zinc
  - Nucleotide
  - Ribavirin
  - MPA
  - Mycophenolate
  - Leflunomide
  - 6-azauracil
  - Nucleoside
  - Sofosbuvir
---
