---
figid: PMC6327307__nihms-999516-f0003
figtitle: Epigenetic control of CD8+ T cell differentiation
organisms:
- NA
pmcid: PMC6327307
filename: nihms-999516-f0003.jpg
figlink: /pmc/articles/PMC6327307/figure/F2/
number: F2
caption: a | The DNA methylation cycle of cytosine nucleotide bases is depicted. DNA
  (cyto-sine-5)-methyltransferase 3A (DNMT3A) and DNMT3B add methylation (m) modifications
  to unmodified cytosines in a de novo manner, whereas DNMT1 acts to maintain established
  patterns during DNA replication. Passive demethylation results from a lack of maintenance
  methylation and is replication dependent, whereas active demethylation is directed
  by Ten-eleven translocation (TET) proteins and can be either replication dependent
  or independent. TET proteins mediate the serial oxidation of methylated cytosines
  (red bases), resulting in multiple intermediary bases, including 5-hydroxymethylcytosine
  (green bases). Eventually, modified bases are returned to an unmodified state via
  the base excision repair (BER) pathway. b | Nucleosomes consist of two copies each
  of the histone proteins H2A, H2B, H3 and H4 encircled by DNA. The amino-terminal
  tails of histone proteins protrude from the nucleosome structure and can be post-translationally
  modified. The amino acid sequence of the histone H3 N-terminal tail is shown along
  with the position of select lysines (K) that are subject to methylation (me) and/or
  acetylation (ac); however, this is not an exhaustive list of all H3 post-translational
  modifications. c | Chromatin can be broadly categorized as either euchromatin or
  heterochromatin on the basis of its accessibility level. Euchromatin is characterized
  by an open chromatin conformation that is less compact and more accessible by regulatory
  proteins (yellow oval). Heterochromatin is categorized as either facultative or
  constitutive. Both are more compact and inaccessible relative to euchromatin; however,
  constitutive heterochromatin is generally more compacted and is associated with
  gene-poor repetitive regions that largely remain in a closed state. Facultative
  heterochromatin, by contrast, is associated with regions that often transition to
  an open conformation as transcriptional requirements change. Each chromatin state
  has specific histone post-translational modifications that are associated with it,
  as depicted.
papertitle: Epigenetic control of CD8+ T cell differentiation.
reftext: Amanda N. Henning, et al. Nat Rev Immunol. 2018 May;18(5):340-356.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8803681
figid_alias: PMC6327307__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC6327307__F2
ndex: 6b0ec4d5-de93-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6327307__nihms-999516-f0003.html
  '@type': Dataset
  description: a | The DNA methylation cycle of cytosine nucleotide bases is depicted.
    DNA (cyto-sine-5)-methyltransferase 3A (DNMT3A) and DNMT3B add methylation (m)
    modifications to unmodified cytosines in a de novo manner, whereas DNMT1 acts
    to maintain established patterns during DNA replication. Passive demethylation
    results from a lack of maintenance methylation and is replication dependent, whereas
    active demethylation is directed by Ten-eleven translocation (TET) proteins and
    can be either replication dependent or independent. TET proteins mediate the serial
    oxidation of methylated cytosines (red bases), resulting in multiple intermediary
    bases, including 5-hydroxymethylcytosine (green bases). Eventually, modified bases
    are returned to an unmodified state via the base excision repair (BER) pathway.
    b | Nucleosomes consist of two copies each of the histone proteins H2A, H2B, H3
    and H4 encircled by DNA. The amino-terminal tails of histone proteins protrude
    from the nucleosome structure and can be post-translationally modified. The amino
    acid sequence of the histone H3 N-terminal tail is shown along with the position
    of select lysines (K) that are subject to methylation (me) and/or acetylation
    (ac); however, this is not an exhaustive list of all H3 post-translational modifications.
    c | Chromatin can be broadly categorized as either euchromatin or heterochromatin
    on the basis of its accessibility level. Euchromatin is characterized by an open
    chromatin conformation that is less compact and more accessible by regulatory
    proteins (yellow oval). Heterochromatin is categorized as either facultative or
    constitutive. Both are more compact and inaccessible relative to euchromatin;
    however, constitutive heterochromatin is generally more compacted and is associated
    with gene-poor repetitive regions that largely remain in a closed state. Facultative
    heterochromatin, by contrast, is associated with regions that often transition
    to an open conformation as transcriptional requirements change. Each chromatin
    state has specific histone post-translational modifications that are associated
    with it, as depicted.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GC
  - Tet
  - ber
  - cg
  - Tailor
  - Me
  - ac
  - His2Av
  - egr
  - DNMT3A
  - DNMT3B
  - DNMT1
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - TUBA4A
  - H2AC18
  - TUBA3D
  - H3 H2B
  - 'N'
  - H3K4me3
  - H3K9me2
  - H3K9me3
---
