---
figid: PMC6801973__ijms-20-04679-g001
figtitle: Overview of sulfate metabolism functionally linked to sulfate activation
organisms:
- Toxoplasma gondii
- Dictyostelium discoideum
- Entamoeba histolytica
- Amoebozoa
- Entamoeba invadens
- Entamoeba dispar
- Mastigamoeba balamuthi
- Mycobacterium tuberculosis
- Pseudomonas aeruginosa
- Bacteroides sp.
- Deltaproteobacteria
- gut metagenome
- PX clade
- human gut metagenome
- Euglena gracilis
- Homo sapiens
pmcid: PMC6801973
filename: ijms-20-04679-g001.jpg
figlink: /pmc/articles/PMC6801973/figure/ijms-20-04679-f001/
number: F1
caption: 'Overview of sulfate metabolism functionally linked to sulfate activation.
  An overall metabolic pathway is depicted, in which information from bacterial to
  human studies is integrated. Note that Entamoeba does not possess genes encoding
  APS and PAPS reductases (AmoebaDB; KEGG). The enzyme responsible for each step is
  shown by a filled oval. Abbreviations used: APS, adenosine 5′-phosphosulfate; APSK,
  APS kinase; AS, ATP sulfurylase; IPP, inorganic pyrophosphatase; PAP, adenosine
  3′,5′-bisphosphate; PAPase, 3′(2′),5′-bisphosphate nucleotidase; PAPS, 3′-phosphoadenosine
  5′-phosphosulfate; Pi, inorganic phosphate; PPi, inorganic pyrophosphate; SF, sulfatase;
  and SULT, sulfotransferase. Enzymatic reactions are indicated by arrows. Multiple
  steps for synthesizing sulfur-containing biomolecules is simplified by dashed arrow.
  Fine dotted lines indicate molecular transfer of APS and PAPS.'
papertitle: Unique Features of Entamoeba Sulfur Metabolism; Compartmentalization,
  Physiological Roles of Terminal Products, Evolution and Pharmaceutical Exploitation.
reftext: Fumika Mi-ichi, et al. Int J Mol Sci. 2019 Oct;20(19):4679.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9572372
figid_alias: PMC6801973__F1
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6801973__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6801973__ijms-20-04679-g001.html
  '@type': Dataset
  description: 'Overview of sulfate metabolism functionally linked to sulfate activation.
    An overall metabolic pathway is depicted, in which information from bacterial
    to human studies is integrated. Note that Entamoeba does not possess genes encoding
    APS and PAPS reductases (AmoebaDB; KEGG). The enzyme responsible for each step
    is shown by a filled oval. Abbreviations used: APS, adenosine 5′-phosphosulfate;
    APSK, APS kinase; AS, ATP sulfurylase; IPP, inorganic pyrophosphatase; PAP, adenosine
    3′,5′-bisphosphate; PAPase, 3′(2′),5′-bisphosphate nucleotidase; PAPS, 3′-phosphoadenosine
    5′-phosphosulfate; Pi, inorganic phosphate; PPi, inorganic pyrophosphate; SF,
    sulfatase; and SULT, sulfotransferase. Enzymatic reactions are indicated by arrows.
    Multiple steps for synthesizing sulfur-containing biomolecules is simplified by
    dashed arrow. Fine dotted lines indicate molecular transfer of APS and PAPS.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATP8A2
  - KLK3
  - SH2B2
  - WDTC1
  - REG3A
  - ASAP2
  - MRPS30
  - PAPOLA
  - PDAP1
  - TUSC2
  - ASAP1
  - FES
  - SF
  - Sulfate
  - ATP
  - Sulfur
  - ADP
  - sulfite
  - SO
  - sulfide
  - S2
  - PAP
  - AMP
  - sugar
  - cysteine
  - methionine
  - Fe-S
  - thiamine
  - CoA
  - Desulfated
---
