---
figid: PMC6307301__40168_2018_613_Fig6_HTML
figtitle: Comparative genomics analysis of correlations between Marinilabiliales microbes
  and the microorganisms enriched during enrichment culturing
organisms:
- Escherichia coli
- Mycobacterium tuberculosis
- Pseudomonas aeruginosa
- Bacillus subtilis
- Vibrio cholerae
- Bacillus subtilis At2
- Escherichia coli O157H7
- Bacteroides sp.
- Clostridium pasteurianum
- Nitrosomonas europaea
- Bradymonas sediminis
- Draconibacterium orientale
- Salinivirga cyanobacteriivorans
- Ancylomarina subtilis
- Halomonas denitrificans
- Marinobacter xestospongiae
- gut metagenome
- metagenome
- human metagenome
- uncultured bacterium
- enrichment cultures
- enrichment culture
- unidentified marine bacterioplankton
- Saccharomyces cerevisiae
- Diaporthe sclerotioides
- Populus deltoides
- Homo sapiens
- mixed culture
organisms_ner:
- Vibrio cholerae
- Clostridium pasteurianum
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC6307301
filename: 40168_2018_613_Fig6_HTML.jpg
figlink: /pmc/articles/PMC6307301/figure/Fig6/
number: F6
caption: 'Comparative genomics analysis of correlations between Marinilabiliales microbes
  and the microorganisms enriched during enrichment culturing. Overview of biotin
  biosynthesis and precursor salvage. Biosynthetic enzymes: BioH, pimeloyl-[acyl-carrier
  protein] methyl ester esterase; BioC, malonyl-CoA O-methyltransferase; BioF, 8-amino-7-oxononanoate
  synthase; BioA, adenosylmethionine-8-amino-7-oxononanoate aminotransferase; BioD,
  dethiobiotin synthetase; BioB, biotin synthase; and BirA, biotin-[acetyl-CoA-carboxylase]
  ligase. Biotin translocases include the BioYM ECF-type transporter. The auxotrophic
  groups, which mainly contain Marinilabiliales bacteria, are unable to synthesize
  biotin de novo and are missing most of the related pathway genes. Phototrophic groups,
  which mainly contain Vibrionaceae, Clostridiaceae, Desulfobulbaceae, Desulfovibrionaceae,
  and Desulfobacteraceae bacteria, can synthesize biotin de novo. The Desulfobacteraceae
  node has no correlations with the Prolixibacteraceae node and the Marinilabiliaceae
  node; however, the bacteria within this family were highly enriched and became the
  dominant family during enrichment culturing. We also analyzed the genomes of this
  family’s members. Pie slices indicate the total fragments per kilobase of transcript
  per million fragments mapped (FPKM) of the corresponding gene in each sediment sample'
papertitle: Metatranscriptomic and comparative genomic insights into resuscitation
  mechanisms during enrichment culturing.
reftext: Da-Shuai Mu, et al. Microbiome. 2018;6:230.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9122996
figid_alias: PMC6307301__F6
figtype: Figure
redirect_from: /figures/PMC6307301__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6307301__40168_2018_613_Fig6_HTML.html
  '@type': Dataset
  description: 'Comparative genomics analysis of correlations between Marinilabiliales
    microbes and the microorganisms enriched during enrichment culturing. Overview
    of biotin biosynthesis and precursor salvage. Biosynthetic enzymes: BioH, pimeloyl-[acyl-carrier
    protein] methyl ester esterase; BioC, malonyl-CoA O-methyltransferase; BioF, 8-amino-7-oxononanoate
    synthase; BioA, adenosylmethionine-8-amino-7-oxononanoate aminotransferase; BioD,
    dethiobiotin synthetase; BioB, biotin synthase; and BirA, biotin-[acetyl-CoA-carboxylase]
    ligase. Biotin translocases include the BioYM ECF-type transporter. The auxotrophic
    groups, which mainly contain Marinilabiliales bacteria, are unable to synthesize
    biotin de novo and are missing most of the related pathway genes. Phototrophic
    groups, which mainly contain Vibrionaceae, Clostridiaceae, Desulfobulbaceae, Desulfovibrionaceae,
    and Desulfobacteraceae bacteria, can synthesize biotin de novo. The Desulfobacteraceae
    node has no correlations with the Prolixibacteraceae node and the Marinilabiliaceae
    node; however, the bacteria within this family were highly enriched and became
    the dominant family during enrichment culturing. We also analyzed the genomes
    of this family’s members. Pie slices indicate the total fragments per kilobase
    of transcript per million fragments mapped (FPKM) of the corresponding gene in
    each sediment sample'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - bioH
  - bioC
  - bioF
  - bioA
  - bioD
  - bioB
  - birA
  - Biotin
  - pimeloyl
  - methyl ester
  - pimyeloyl-lacp
  - fatty acids
  - pimyeloyl
  - 8-amino-7-oxononanoate
  - 8-dlaminononanoate
  - 8-diaminononanoate
  - dethiobiotin
  - biotin
  - biotinyl
  - Coenzyme A
  - JCM 21150
  - O DOO
---
