---
figid: PMC6666404__41586_2019_1236_Fig10_ESM
figtitle: Longitudinal multi-omics of host–microbe dynamics in prediabetes
organisms:
- gut metagenome
- metagenome
- human gut metagenome
- human metagenome
- Respiratory syncytial virus
- Enterovirus
- Human rhinovirus sp.
- Homo sapiens
- Mus musculus
- Clarkia concinna
- unclassified Butyricicoccus
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC6666404
filename: 41586_2019_1236_Fig10_ESM.jpg
figlink: /pmc/articles/PMC6666404/figure/Fig10/
number: F4
caption: a, Comparison of AUC test (two-sided) performance with LR analysis with the
  time covariate. Left, distribution plot of P values resulting from the AUC test.
  Right, distribution plot of P values resulting from the LR analysis. The pie chart
  shows the number of differentially expressed genes (q < 0.1) for both methods and
  the overlapping number. b, Top, comparison of results from paired t-test (two-sided)
  with DESeq2 method (Wald test) for stage pairwise analyses (for example, EE versus
  personal healthy baseline). Seventy-one per cent of transcripts identified by DESeq2
  method overlap with paired t-test results. Bottom, P value distribution for paired
  t-test (left) and DESeq2 (right). Both methods show a homogenous distribution of
  P values with enrichment for significant ones. Supplementary Table 39 lists molecules
  found significant by paired t-test or DESeq2 method. c, Illustration of IL10 pathway
  enriched in response to RVI. Multi-omic molecules that changed significantly during
  the course of RVI by AUC test are highlighted in red (upregulated) or green (downregulated).
  d, An example of enriched neuroinflammation signalling showing molecules significantly
  upregulated (red) or downregulated (green) in response to RVI. Simplified pathway
  graph is shown in Fig.  (bottom). e, Illustration of insulin signalling (left) and
  cardiac hypertrophy (right) pathways, which were enriched in response to RVI. Molecules
  that changed significantly during the course of RVI by AUC test are highlighted
  in dark pink (upregulated) or green (downregulated).
papertitle: Longitudinal multi-omics of host–microbe dynamics in prediabetes.
reftext: Wenyu Zhou, et al. Nature. 2019 May;569(7758):663-671.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5047616
figid_alias: PMC6666404__F4
figtype: Figure
redirect_from: /figures/PMC6666404__F4
ndex: 5cb93798-de9f-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6666404__41586_2019_1236_Fig10_ESM.html
  '@type': Dataset
  description: a, Comparison of AUC test (two-sided) performance with LR analysis
    with the time covariate. Left, distribution plot of P values resulting from the
    AUC test. Right, distribution plot of P values resulting from the LR analysis.
    The pie chart shows the number of differentially expressed genes (q < 0.1) for
    both methods and the overlapping number. b, Top, comparison of results from paired
    t-test (two-sided) with DESeq2 method (Wald test) for stage pairwise analyses
    (for example, EE versus personal healthy baseline). Seventy-one per cent of transcripts
    identified by DESeq2 method overlap with paired t-test results. Bottom, P value
    distribution for paired t-test (left) and DESeq2 (right). Both methods show a
    homogenous distribution of P values with enrichment for significant ones. Supplementary
    Table 39 lists molecules found significant by paired t-test or DESeq2 method.
    c, Illustration of IL10 pathway enriched in response to RVI. Multi-omic molecules
    that changed significantly during the course of RVI by AUC test are highlighted
    in red (upregulated) or green (downregulated). d, An example of enriched neuroinflammation
    signalling showing molecules significantly upregulated (red) or downregulated
    (green) in response to RVI. Simplified pathway graph is shown in Fig.  (bottom).
    e, Illustration of insulin signalling (left) and cardiac hypertrophy (right) pathways,
    which were enriched in response to RVI. Molecules that changed significantly during
    the course of RVI by AUC test are highlighted in dark pink (upregulated) or green
    (downregulated).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Des
  - Il10
  - Was
  - Wdr7
  - tor
  - Rorc
  - Sta
  - Emd
  - Gnat1
  - Ar
  - tu
  - S100a11
  - Fblim1
  - Gopc
  - Car1
  - Pdia4
  - App
  - Snca
  - Tnf
  - pn
  - Rusc1
  - Nes
  - Gls
  - Gls2
  - Sri
  - Ntrk1
  - Uchl1
  - Penk
  - Rf
  - Glyat
  - Tg
  - Nt5e
  - da
  - Mapt
  - Prph2
  - Furin
  - Ssr3
  - st
  - Os
  - nr
  - Spin2g
  - Pagr1a
  - Mst1r
  - Nos1
  - DES
  - IL10
  - WAS
  - WDR7
  - RBM8A
  - RORC
  - EMD
  - GCY
  - TRA
  - THRA
  - TRAC
  - AR
  - FBLIM1
  - GOPC
  - CA1
  - APP
  - SNCA
  - ATP8A2
  - TNF
  - EYCL1
  - RUSC1
  - NES
  - GLS
  - GLS2
  - NTRK1
  - TPM3
  - GAD1
  - GAD2
  - GLYAT
  - TG
  - NT5E
  - GJA8
  - MAPT
  - PRPH2
  - TST
  - FURIN
  - SSR3
  - CTSB
  - PAM
  - LRIT1
  - SHCBP1
  - PAGR1
  - SERPINE1
  - SERPINB2
  - MST1R
  - NOS1
  - NOS2
  - NOS3
  - ifc
  - wac
  - tar
  - Fs(2)Tar
  - wa
  - Tor
  - Egfr
  - sta
  - tra
  - byn
  - ar
  - Tu
  - cal
  - Cam
  - app
  - Appl
  - ApepP
  - APP-BP1
  - ATPsynbeta
  - Atpalpha
  - egr
  - ca
  - nes
  - Ae2
  - ot
  - Me
  - Dsor1
  - trk
  - Gad1
  - Ppa
  - Pa
  - cd
  - GABA-B-R1
  - Rdl
  - Gabat
  - rf
  - AP-1gamma
  - Gat
  - .na.character
  - Pitslre
  - gate
  - ry
  - de
  - Orai
  - nAChRalpha2
  - La
  - abb
  - br
  - stas
  - sbe
  - spi
  - pal
  - Pal2
  - Pa1
  - Ne
  - Nos
  - nos
  - so
  - Bilverdin
  - ATP
  - GABA
  - Ca
  - Cardiac hypertrophy
---
