---
figid: PMC6616015__nihms-1518317-f0002
figtitle: Gene-gene interaction networks
organisms:
- Mus musculus
- Rattus norvegicus
- Sequoia sempervirens
- Nepeta menthoides
- Homo sapiens
- vectors
pmcid: PMC6616015
filename: nihms-1518317-f0002.jpg
figlink: pmc/articles/PMC6616015/figure/F2/
number: F2
caption: Gene-gene interaction networks. Genes associated with the differentially
  methylated sites were uploaded to Ingenuity pathway analysis. Based on p-value cutoffs
  of 10−8, three networks were identified. Two of them were similar in function with
  several overlapping molecules. Hence, two of the different networks are presented
  here. The network in panel A is associated with Cell Signaling, Molecular Transport,
  Vitamin and Mineral Metabolism. It had a p-score of 33 and 14 focus molecules (including
  CACNA1A, CACNA1C, Calmodulin, ERK1/2, Histone h3, Histone h4, IkB-NfkB, NFkB (complex),
  miR-9–3p). The network in Panel B is associated with Neurological Disease, Organismal
  Injury and Abnormalities, Connective Tissue Development and Function; with a p-score
  of 12, and 6 focus molecules (including ESR1, KCNK6, PRIM2, TNF).
papertitle: Enrichment of genomic pathways based on differential DNA methylation associated
  with chronic postsurgical pain and anxiety in children – a prospective, pilot study.
reftext: Vidya Chidambaran, et al. J Pain. ;20(7):771-785.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.772442
figid_alias: PMC6616015__F2
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
redirect_from: /figures/PMC6616015__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6616015__nihms-1518317-f0002.html
  '@type': Dataset
  description: Gene-gene interaction networks. Genes associated with the differentially
    methylated sites were uploaded to Ingenuity pathway analysis. Based on p-value
    cutoffs of 10−8, three networks were identified. Two of them were similar in function
    with several overlapping molecules. Hence, two of the different networks are presented
    here. The network in panel A is associated with Cell Signaling, Molecular Transport,
    Vitamin and Mineral Metabolism. It had a p-score of 33 and 14 focus molecules
    (including CACNA1A, CACNA1C, Calmodulin, ERK1/2, Histone h3, Histone h4, IkB-NfkB,
    NFkB (complex), miR-9–3p). The network in Panel B is associated with Neurological
    Disease, Organismal Injury and Abnormalities, Connective Tissue Development and
    Function; with a p-score of 12, and 6 focus molecules (including ESR1, KCNK6,
    PRIM2, TNF).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TRIM27
  - CBX1
  - KCTD9
  - CARD14
  - SMN1
  - GALNT2
  - ANGPT4
  - CENPM
  - RIMBP2
  - FGL1
  - HPS1
  - AP3B1
  - SCUBE2
  - CHST2
  - EPHX4
  - ZBED4
  - OGDHL
  - ARHGAP18
  - RADIL
  - ADGRG6
  - MALL
  - CCNA1
  - CCNA2
  - CDK2
  - REL
  - NRARP
  - PRIM2
  - KLHL40
  - BCR
  - IGKV3D-20
  - Trim27
  - Cbx1
  - Kctd9
  - Card14
  - Smn1
  - Galnt2
  - Angpt4
  - Cenpm
  - Rimbp2
  - Hps1
  - Scube2
  - Adamts1
  - Chst2
  - Ephx4
  - Zbed4
  - Ogdhl
  - Arhgap18
  - Radil
  - Adgrg6
  - Mall
  - Cdk2
  - Rel
  - Nrarp
  - Prim2
  - Klhl40
  - Efna5
  - Foxn1
  - phosphate
  - beta-hydroxyinovaleric acid
  - fucosyllactose
---
