---
figid: PMC6786970__41564_2018_190_Fig4_HTML
figtitle: Host-linked soil viral ecology along a permafrost thaw gradient
organisms:
- Hordeum vulgare
- Cyamopsis tetragonoloba
- Tamarindus indica
- Ceratonia siliqua
- Escherichia coli
- Cellulophaga baltica
- DNA viruses
- Bacteriophage sp.
- metagenomes
pmcid: PMC6786970
filename: 41564_2018_190_Fig4_HTML.jpg
figlink: /pmc/articles/PMC6786970/figure/Fig4/
number: F4
caption: Schematic overview of the carbon cycle, with light orange labels highlighting
  the potential viral contributions. Plant polymers that have the potential to be
  degraded by viral glycoside hydrolases are along the top, with examples of each
  below. Individual sugars are colour coded (see monomers beneath the black and orange
  arrows). The predicted enzymatic functions from computational protein models are
  listed in red, and example polymer cleavage points are indicated by the red arrows.
  Smaller molecules (monosaccharides and small oligosaccharides) cleaved from the
  polymers are shown at the ends of the black arrows, indicating the potential enzymatic
  conversion encoded by the viral glycoside hydrolase (the orange arrow indicates
  cleavage that was functionally confirmed by enzymatic assay), with the PHYRE2-modelled
  structure of each viral glycoside hydrolase in red. The asterisks indicate that
  multiple viral genes were recovered with the indicated predicted function. In those
  cases, the viral protein structure containing the most catalytic residues is shown
  (see Supplementary Table  for details). The grey line indicates a cleavage pathway
  of small oligosaccharides to monomers (non-viral). β-Gal, β-galactosidase; endogluc,
  endoglucanase; RG-I, rhamnogalacturonan I; black monomer, rhamnose; dark grey monomer,
  galacturonic acid.
papertitle: Host-linked soil viral ecology along a permafrost thaw gradient.
reftext: Joanne B. Emerson, et al. Nat Microbiol. 2018;3(8):870-880.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7932958
figid_alias: PMC6786970__F4
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC6786970__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6786970__41564_2018_190_Fig4_HTML.html
  '@type': Dataset
  description: Schematic overview of the carbon cycle, with light orange labels highlighting
    the potential viral contributions. Plant polymers that have the potential to be
    degraded by viral glycoside hydrolases are along the top, with examples of each
    below. Individual sugars are colour coded (see monomers beneath the black and
    orange arrows). The predicted enzymatic functions from computational protein models
    are listed in red, and example polymer cleavage points are indicated by the red
    arrows. Smaller molecules (monosaccharides and small oligosaccharides) cleaved
    from the polymers are shown at the ends of the black arrows, indicating the potential
    enzymatic conversion encoded by the viral glycoside hydrolase (the orange arrow
    indicates cleavage that was functionally confirmed by enzymatic assay), with the
    PHYRE2-modelled structure of each viral glycoside hydrolase in red. The asterisks
    indicate that multiple viral genes were recovered with the indicated predicted
    function. In those cases, the viral protein structure containing the most catalytic
    residues is shown (see Supplementary Table  for details). The grey line indicates
    a cleavage pathway of small oligosaccharides to monomers (non-viral). β-Gal, β-galactosidase;
    endogluc, endoglucanase; RG-I, rhamnogalacturonan I; black monomer, rhamnose;
    dark grey monomer, galacturonic acid.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Galactomannan
  - Fucose Galactose Mannose
  - Arabinose Galactose
  - Glucose
  - alcohols
  - Acetate
  - Methane
  - glycoside
---
