---
figid: PMC6442683__nihms-1005382-f0003
figtitle: Dendritic structural plasticity and neuropsychiatric disease
organisms:
- Homo sapiens
pmcid: PMC6442683
filename: nihms-1005382-f0003.jpg
figlink: /pmc/articles/PMC6442683/figure/F3/
number: F3
caption: 'a |Schematic illustration of a systems biology approach for identifying
  novel genes likely to affect spine plasticity in neuropsychiatric disorders. Exome
  sequencing studies of de novo variants are particularly useful for uncovering genetic
  risk networks involved in different disorders. Although many genes discovered in
  exome sequencing studies are individually nonsignificant, as a whole, they can reveal
  a mutational spectrum for each disorder and provide clues for the underlying pathways
  and networks involved in pathogenesis. In particular, genes with de novo mutations
  for which the protein products localize to the postsynaptic density (PSD) are excellent
  candidates for future studies on the dysfunctional pathways altering structural
  plasticity in neuropsychiatric disorders. To highlight such genes, de novo mutations
  affecting protein coding for each disorder were intersected with proteins present
  in the human PSD proteome (a proxy for the dendritic spine). The number of genes
  in each overlap is indicated (top panels). All of the de novo data were obtained
  from denovo-db (see ‘’) accessed on 21 September 2017. Additional studies for schizophrenia–
  that were not in the denovo-db were obtained from supplementary data compiled in
  REF. Only de novo variants affecting protein-coding regions or splice sites were
  included (that is, missense, frameshift and splice site mutations, altered stop
  codons, altered start codons, insertions and deletions). Human PSD proteome data
  were obtained from REF. . Gene ontologies (GOs) were then used to classify PSD-associated
  mutations into biological processes relevant to spine structure and function. To
  do so, GO categories from each data set were defined in DAVID (see ‘’) v6.8, accessed
  on 11 November 2017, using the functional annotation chart. Homo sapiens was used
  as a background, and the ‘biological process’ ontology (GOTERM_BP_FAT) was used
  to annotate gene lists. The following ontologies were used to cluster genes into
  functional pathways: cell adhesion (GO:0007155-cell adhesion), GluR signalling (GO:0007215-glutamate
  signalling pathway), calcium (GO:0006816-calcium ion transport) and GTPase (G00043087-regulation
  of GTPase activity). This analysis shows that the de novo mutations affecting PSD
  proteins cluster into distinct biological pathways to varying degrees, providing
  an overview of the contribution of each pathway to each disorder (bottom panels).
  The percentage of genes in each GO category is depicted on the y-axis. Exome sequencing
  data for bipolar disorder are currently insufficient to classify mutations into
  categories. b | Illustration of a dendritic spine containing different functional
  groups of neuropsychiatric-disorder-associated risk factors () that regulate spine
  structure. Functional groups are colour coded. ANK3, ankyrin 3; ARHGEF9, Rho guanine
  nucleotide exchange factor 9; ASD, autism spectrum disorder; CACNA1C, voltage-gated
  calcium channel subunit-α1Cav1.2; CACNG2, voltage-dependent calcium channel-γ2 subunit;
  CACNB4, calcium channel voltage-dependent subunit-β4; CAMK2A/B, calcium/calmodulin-dependent
  protein kinase type II subunit-α/β; CASK, calcium/calmodulin-dependent serine protein
  kinase; CNTNAP2, contactin-associated protein-like 2; DLG4, disks large homologue
  4; DLGAP1, disks large-associated protein 1; GRIA1/2, glutamate receptor ionotropic,
  AMPA1/2; GRIN2A/B, glutamate receptor ionotropic NMDA2A/B; KALRN, kalirin; NF1,
  neurofibromin 1; NLGN3, neuroligin 3; NRXN1, neurexin 1; PRRT2, proline-rich transmembrane
  protein 2; RAC1, RAS-related C3 botulinum toxin substrate 1; SHANK3, SH3 and multiple
  ankyrin repeat domains protein 3; SYNGAP1, RAS/RAP GTPase-activating protein SynGAP
  1; TRIO, triple functional domain protein.'
papertitle: Dendritic structural plasticity and neuropsychiatric disease.
reftext: Marc P. Forrest, et al. Nat Rev Neurosci. 2018 Mar 16;19(4):215-234.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7478532
figid_alias: PMC6442683__F3
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6442683__F3
ndex: 617a18ce-df2f-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6442683__nihms-1005382-f0003.html
  '@type': Dataset
  description: 'a |Schematic illustration of a systems biology approach for identifying
    novel genes likely to affect spine plasticity in neuropsychiatric disorders. Exome
    sequencing studies of de novo variants are particularly useful for uncovering
    genetic risk networks involved in different disorders. Although many genes discovered
    in exome sequencing studies are individually nonsignificant, as a whole, they
    can reveal a mutational spectrum for each disorder and provide clues for the underlying
    pathways and networks involved in pathogenesis. In particular, genes with de novo
    mutations for which the protein products localize to the postsynaptic density
    (PSD) are excellent candidates for future studies on the dysfunctional pathways
    altering structural plasticity in neuropsychiatric disorders. To highlight such
    genes, de novo mutations affecting protein coding for each disorder were intersected
    with proteins present in the human PSD proteome (a proxy for the dendritic spine).
    The number of genes in each overlap is indicated (top panels). All of the de novo
    data were obtained from denovo-db (see ‘’) accessed on 21 September 2017. Additional
    studies for schizophrenia– that were not in the denovo-db were obtained from supplementary
    data compiled in REF. Only de novo variants affecting protein-coding regions or
    splice sites were included (that is, missense, frameshift and splice site mutations,
    altered stop codons, altered start codons, insertions and deletions). Human PSD
    proteome data were obtained from REF. . Gene ontologies (GOs) were then used to
    classify PSD-associated mutations into biological processes relevant to spine
    structure and function. To do so, GO categories from each data set were defined
    in DAVID (see ‘’) v6.8, accessed on 11 November 2017, using the functional annotation
    chart. Homo sapiens was used as a background, and the ‘biological process’ ontology
    (GOTERM_BP_FAT) was used to annotate gene lists. The following ontologies were
    used to cluster genes into functional pathways: cell adhesion (GO:0007155-cell
    adhesion), GluR signalling (GO:0007215-glutamate signalling pathway), calcium
    (GO:0006816-calcium ion transport) and GTPase (G00043087-regulation of GTPase
    activity). This analysis shows that the de novo mutations affecting PSD proteins
    cluster into distinct biological pathways to varying degrees, providing an overview
    of the contribution of each pathway to each disorder (bottom panels). The percentage
    of genes in each GO category is depicted on the y-axis. Exome sequencing data
    for bipolar disorder are currently insufficient to classify mutations into categories.
    b | Illustration of a dendritic spine containing different functional groups of
    neuropsychiatric-disorder-associated risk factors () that regulate spine structure.
    Functional groups are colour coded. ANK3, ankyrin 3; ARHGEF9, Rho guanine nucleotide
    exchange factor 9; ASD, autism spectrum disorder; CACNA1C, voltage-gated calcium
    channel subunit-α1Cav1.2; CACNG2, voltage-dependent calcium channel-γ2 subunit;
    CACNB4, calcium channel voltage-dependent subunit-β4; CAMK2A/B, calcium/calmodulin-dependent
    protein kinase type II subunit-α/β; CASK, calcium/calmodulin-dependent serine
    protein kinase; CNTNAP2, contactin-associated protein-like 2; DLG4, disks large
    homologue 4; DLGAP1, disks large-associated protein 1; GRIA1/2, glutamate receptor
    ionotropic, AMPA1/2; GRIN2A/B, glutamate receptor ionotropic NMDA2A/B; KALRN,
    kalirin; NF1, neurofibromin 1; NLGN3, neuroligin 3; NRXN1, neurexin 1; PRRT2,
    proline-rich transmembrane protein 2; RAC1, RAS-related C3 botulinum toxin substrate
    1; SHANK3, SH3 and multiple ankyrin repeat domains protein 3; SYNGAP1, RAS/RAP
    GTPase-activating protein SynGAP 1; TRIO, triple functional domain protein.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ARSD
  - PSD
  - PISD
  - FBXL15
  - NRXN1
  - CACNA1C
  - NLGN3
  - CACNG2
  - CNTNAP2
  - GRIA1
  - CASK
  - ANK3
  - CACNB4
  - DLG4
  - LLGL1
  - PRRT2
  - GRIN2A
  - DLGAP1
  - SHANK3
  - SYNGAP1
  - CAMK2A
  - NF1
  - CAMK2B
  - ARHGEF6
  - TRIO
  - RAC1
  - RNASE1
  - KALRN
  - Glutamate
  - Calcium
  - AMPA
  - NMDA
  - calcium
  - Epilepsy
  - ASD
  - Schizophrenia  Bipolar disorder
  - Dendritic spine proteins
  - Dendritic branch
---
