---
figid: PMC6095600__pone.0202530.g003
figtitle: Orthologous proteins of experimental de- and remyelination are differentially
  regulated in the CSF proteome of multiple sclerosis subtypes
organisms:
- NA
pmcid: PMC6095600
filename: pone.0202530.g003.jpg
figlink: /pmc/articles/PMC6095600/figure/pone.0202530.g003/
number: F3
caption: 'Upstream regulators of up- and down-regulated orthologous genes were determined
  by Ingenuity Pathway Analysis. The experimentally observed gene expression tendencies
  are shown by the bottom line of nodes of the graphs (red increased, green decreased
  expression), while the networks above these genes displays the predicted regulatory
  networks might be responsible for the detected transcription effect. Orange color
  in the regulatory network indicates positive and blue color indicates negative regulatory
  effects. Marks on the graph edges shows the type of functional interactions (E:
  expression, LO: localization, A: activation, TR: translocation, T: transcription,
  RB: regulation of binding, PD: protein-DNA binding, P: phosphorylation) and the
  number of the supporting evidences in Ingenuity Knowledge Base.'
papertitle: Orthologous proteins of experimental de- and remyelination are differentially
  regulated in the CSF proteome of multiple sclerosis subtypes.
reftext: Nellie A. Martin, et al. PLoS One. 2018;13(8):e0202530.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.950799
figid_alias: PMC6095600__F3
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6095600__F3
ndex: 4291908f-dea4-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6095600__pone.0202530.g003.html
  '@type': Dataset
  description: 'Upstream regulators of up- and down-regulated orthologous genes were
    determined by Ingenuity Pathway Analysis. The experimentally observed gene expression
    tendencies are shown by the bottom line of nodes of the graphs (red increased,
    green decreased expression), while the networks above these genes displays the
    predicted regulatory networks might be responsible for the detected transcription
    effect. Orange color in the regulatory network indicates positive and blue color
    indicates negative regulatory effects. Marks on the graph edges shows the type
    of functional interactions (E: expression, LO: localization, A: activation, TR:
    translocation, T: transcription, RB: regulation of binding, PD: protein-DNA binding,
    P: phosphorylation) and the number of the supporting evidences in Ingenuity Knowledge
    Base.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RPL15
  - IGKV1D-16
  - CCDC34
  - TNF
  - RB1
  - RBM45
  - FOXM1
  - SP3
  - AZF1
  - IFNG
  - IL18
  - STAT3
  - SLC7A5
  - RNU105C
  - PPARA
  - TIMP1
  - CD74
  - MAF
  - MAFA
  - MAFB
  - MAFF
  - MAFG
  - MAFK
  - PSMB9
  - IFI35
  - ANGPTL4
  - SCAP
  - SH2D2A
  - RBL2
  - FUT3
  - DHCR7
  - ELOVL6
  - ACSL1
  - MVD
  - SMR3A
  - PBRM1
  - SMR3B
  - NPRL3
  - SREBF2
  - Pb
---
