---
figid: PMC6753514__10.1177_1178626419875089-fig3
figtitle: Peptide/amino acid transport and metabolism and cofactor synthesis in Gs
  and Au cells
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC6753514
filename: 10.1177_1178626419875089-fig3.jpg
figlink: /pmc/articles/PMC6753514/figure/fig3-1178626419875089/
number: F3
caption: 'Peptide/amino acid transport and metabolism and cofactor synthesis in Gs
  and Au cells. The schematic representation contains protein names and gene loci
  (if protein short names were not provided for ORFs based on conserved sequences)
  in red. Each gene locus contains only the last 4 and 5 numbers of the Au and Gs
  gene accession terms, respectively. Details on the proteins (quantities and descriptions)
  are provided in datasets of the Supplemental File S2. Metabolite names are depicted
  in black. Blue arrows illustrate an enzymatic activity or pathway step, while black
  arrows indicate a transport process. The darker the color of the circle behind each
  protein name, the higher its average abundance in CB datasets. Blue: Gs proteins;
  green: Au proteins; gray: protein not detected in the proteomes. Cofactors, in light
  green script, are depicted next to enzymes where applicable: Me2+ (metal ion), Zn2+
  (zinc), py (pyridoxal-5′-phosphate). DHA indicates dihydroxyacetone; “P,” phosphate;
  PPP, pentose-phosphate pathway; “?” indicates that a gene is predicted to catalyze
  an enzymatic step based on a domain with a predicted function or its gene neighborhood.'
papertitle: Aerococcus urinae and Globicatella sanguinis Persist in Polymicrobial
  Urethral Catheter Biofilms Examined in Longitudinal Profiles at the Proteomic Level.
reftext: Yanbao Yu, et al. Biochem Insights. 2019;12:1178626419875089.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7323254
figid_alias: PMC6753514__F3
figtype: Figure
redirect_from: /figures/PMC6753514__F3
ndex: 5116eaa3-df2d-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6753514__10.1177_1178626419875089-fig3.html
  '@type': Dataset
  description: 'Peptide/amino acid transport and metabolism and cofactor synthesis
    in Gs and Au cells. The schematic representation contains protein names and gene
    loci (if protein short names were not provided for ORFs based on conserved sequences)
    in red. Each gene locus contains only the last 4 and 5 numbers of the Au and Gs
    gene accession terms, respectively. Details on the proteins (quantities and descriptions)
    are provided in datasets of the Supplemental File S2. Metabolite names are depicted
    in black. Blue arrows illustrate an enzymatic activity or pathway step, while
    black arrows indicate a transport process. The darker the color of the circle
    behind each protein name, the higher its average abundance in CB datasets. Blue:
    Gs proteins; green: Au proteins; gray: protein not detected in the proteomes.
    Cofactors, in light green script, are depicted next to enzymes where applicable:
    Me2+ (metal ion), Zn2+ (zinc), py (pyridoxal-5′-phosphate). DHA indicates dihydroxyacetone;
    “P,” phosphate; PPP, pentose-phosphate pathway; “?” indicates that a gene is predicted
    to catalyze an enzymatic step based on a domain with a predicted function or its
    gene neighborhood.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - gnd
  - metN
  - livJ
  - rpe
  - tktA
  - glyA
  - dapA
  - dapB
  - pepT
  - pepA
  - glnA
  - gltD
  - gltB
  - ilvE
  - epd
  - fsaA
  - thrA
  - ilvD
  - thrC
  - gdhA
  - aldA
  - fbaA
  - serC
  - pdxJ
  - cysK
  - gcvH
  - metE
  - iadA
  - tmcA
  - panE
  - pdxH
  - panC
  - coaA
  - map
  - amino acid
  - Au
  - D-gluconate
  - amino acids
  - O
  - 04830 O
  - OG 04135 GINHO
  - G011550 G 03780
  - Me2
  - D-ribulose 5-P
  - D-ribose 5-P
  - Amino acid
  - Pyridoxal phosphate
  - Lys
  - Met
  - Thr
  - Mn2
  - D-glyceraldehyde 3-P
  - AspB
  - D-fructofuranose 6-P
  - Zn2
  - aldehydo-D-ribose 5-P
  - 4-PD-erythronate
  - Pip  Me2
  - ThrA
  - 3-hydroxy 2-oxo 4-P oxybutanoate
  - ThrC
  - L-glutamine
  - SerC
  - DHA
  - 4-phospho-oxy-L-threonine
  - 3-amino-1-hydroxyacetone
  - pyridoxal 5-P
  - ketoisovalerate
  - pyridoxine 5-P
  - pyridoxal
  - (R)-pantothenate
  - (R)-4-phosphopantothenate
---
