---
figid: PMC6638128__MPP-19-1267-g001
figtitle: General mechanisms of NLR transcript regulation described in this review
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
pmcid: PMC6638128
filename: MPP-19-1267-g001.jpg
figlink: /pmc/articles/PMC6638128/figure/mpp12607-fig-0001/
number: F1
caption: General mechanisms of NLR transcript regulation described in this review.
  (A) General flow of genetic information from chromatin‐associated genomic DNA through
  transcription and transcript processing to mature mRNAs, and from mRNAs via translation
  to NLR proteins. CC, coiled coil; LRR, leucine‐rich repeat; NB, nucleotide‐binding
  site; TIR, Toll Interleukin‐1 Receptor domain. (B) Post‐translational histone modifications
  associated with active transcription. H3K4me3, H3K36me2, H3K36me3 and H2ub1 levels
  positively correlate with transcript levels of some NLRs. In Arabidopsis, the H3K4
  and H3K36 methyltransferase ATXR7/MOS9 and the H2 ubiquitylases HUB1 and HUB2 are
  known to promote high levels of these epigenetic marks. Although not shown for NLR
  loci so far, some post‐translational histone modifications, such as H3K4me3, can
  be inducible during pathogen defence. Post‐translational modifications shown in
  this figure are not meant to be present at the exact location indicated, but rather
  in the genomic area of the NLR gene. (C) Transposon‐associated silencing marks.
  Transposon insertions can recruit epigenetic marks silencing their expression into
  the context of NLR genes, where these marks can have suppressive effects on NLR
  transcription (or other expression‐related processes; see G). At some NLR loci,
  the levels of these silencing marks, such as 5mC or H3K9me2, decrease after defence
  induction. In Arabidopsis, H3K9me2 is established by the SUVH (Suppressor of variegation
  homologue) 4, 5 and 6 H3 methyltransferases, whereas 5mC is established by the RNA‐directed
  DNA methylation (RdDM) pathway and maintained by the MET1, CMT2 or CMT3 DNA methyltransferases.
  How these marks are removed during defence induction is not known. (D) Regulatory
  transcription factors (TFs), such as WRKYs, can affect the transcript levels of
  certain NLRs. A broad role of WRKYs in NLR transcription is likely, as their binding
  sites are enriched in the promoters of Arabidopsis NLRs. Many WRKY TFs are known
  to be controlled by defence signals and can play activating or suppressing roles
  in defence gene expression. (E, F) Expression of a large number of NLRs is known
  to be affected by alternative splicing. This process can result in the inclusion
  of alternative exons (E) or retained introns (F) in the respective mRNAs. Either
  one of these processes can lead to the inclusion of in‐frame premature termination
  codons (PTCs), which may result in truncated NLR proteins and/or recruit the respective
  transcripts to the nonsense‐mediated decay (NMD) RNA degradation pathway. In at
  least some cases, truncated NLRs generated by alternative splicing are probably
  functional by associating with full‐length NLRs or acting as downstream signalling
  components. Defence‐associated suppression of NMD can lead to increased abundance
  of some NLR isoforms, probably resulting in heightened states of immunity. (G) An
  increasing number of NLRs are found to be regulated by alternative polyadenylation.
  In the case of Arabidopsis RPP7, a transposon recruited the silencing mark H3K9me2
  into the first intron of this NLR gene. High levels of H3K9me2 in this area suppress
  the use of a proximal polyadenylation site and promote the production of full‐length
  RPP7 coding mRNAs. Reduced levels of H3K9me2 caused by defence induction or mutations
  in the EDM2 gene lead to enhanced use of the proximal polyadenylation site, increased
  levels of a prematurely terminated non‐coding transcript and, consequently, reduced
  levels of full‐length RPP7 coding transcripts. Alternative polyadenylation can be
  combined with alternative splicing by the differential inclusion of alternative
  terminal exons, as in RPP7. (A–G) Basic graphical elements used in these panels
  are defined in (A). Features specifically important or unique for the respective
  mechanisms are shown in red. TE, transposable element; alt, alternative exon; Defence,
  defence‐inducing signals; thin pointed arrows, activating interactions; thin blunt
  arrows, suppressing interactions. (H) MicroRNA (miRNA)‐triggered cascades of secondary
  phased small interfering RNAs (phasiRNAs). In several cases, miRNAs targeting conserved
  regions in NLR transcripts were shown to serve as initial triggers (short red horizontal
  lines) of secondary phasiRNA (short blue horizontal lines) cascades, which comprehensively
  mediate post‐transcriptional co‐silencing of NLR gene families. On association with
  primary target NLR transcripts (e.g. NLRs a, b and c in the figure), these miRNAs
  trigger the production of phasiRNAs which amplify and diversify the initial trigger
  and reinforce the silencing of the primary NLR targets (e.g. NLRs a, b and c), but
  also secondary NLR targets (e.g. NLRs x, y and z). Consequently, the secondary phasiRNAs
  target a wider set of NLR transcripts than the initial miRNA trigger. Such NLR silencing
  cascades can be suppressed by microbial RNA silencing suppressors resulting in enhanced
  immunity.
papertitle: Transcript‐level expression control of plant NLR genes.
reftext: Yan Lai, et al. Mol Plant Pathol. 2018 May;19(5):1267-1281.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9069495
figid_alias: PMC6638128__F1
figtype: Figure
redirect_from: /figures/PMC6638128__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6638128__MPP-19-1267-g001.html
  '@type': Dataset
  description: General mechanisms of NLR transcript regulation described in this review.
    (A) General flow of genetic information from chromatin‐associated genomic DNA
    through transcription and transcript processing to mature mRNAs, and from mRNAs
    via translation to NLR proteins. CC, coiled coil; LRR, leucine‐rich repeat; NB,
    nucleotide‐binding site; TIR, Toll Interleukin‐1 Receptor domain. (B) Post‐translational
    histone modifications associated with active transcription. H3K4me3, H3K36me2,
    H3K36me3 and H2ub1 levels positively correlate with transcript levels of some
    NLRs. In Arabidopsis, the H3K4 and H3K36 methyltransferase ATXR7/MOS9 and the
    H2 ubiquitylases HUB1 and HUB2 are known to promote high levels of these epigenetic
    marks. Although not shown for NLR loci so far, some post‐translational histone
    modifications, such as H3K4me3, can be inducible during pathogen defence. Post‐translational
    modifications shown in this figure are not meant to be present at the exact location
    indicated, but rather in the genomic area of the NLR gene. (C) Transposon‐associated
    silencing marks. Transposon insertions can recruit epigenetic marks silencing
    their expression into the context of NLR genes, where these marks can have suppressive
    effects on NLR transcription (or other expression‐related processes; see G). At
    some NLR loci, the levels of these silencing marks, such as 5mC or H3K9me2, decrease
    after defence induction. In Arabidopsis, H3K9me2 is established by the SUVH (Suppressor
    of variegation homologue) 4, 5 and 6 H3 methyltransferases, whereas 5mC is established
    by the RNA‐directed DNA methylation (RdDM) pathway and maintained by the MET1,
    CMT2 or CMT3 DNA methyltransferases. How these marks are removed during defence
    induction is not known. (D) Regulatory transcription factors (TFs), such as WRKYs,
    can affect the transcript levels of certain NLRs. A broad role of WRKYs in NLR
    transcription is likely, as their binding sites are enriched in the promoters
    of Arabidopsis NLRs. Many WRKY TFs are known to be controlled by defence signals
    and can play activating or suppressing roles in defence gene expression. (E, F)
    Expression of a large number of NLRs is known to be affected by alternative splicing.
    This process can result in the inclusion of alternative exons (E) or retained
    introns (F) in the respective mRNAs. Either one of these processes can lead to
    the inclusion of in‐frame premature termination codons (PTCs), which may result
    in truncated NLR proteins and/or recruit the respective transcripts to the nonsense‐mediated
    decay (NMD) RNA degradation pathway. In at least some cases, truncated NLRs generated
    by alternative splicing are probably functional by associating with full‐length
    NLRs or acting as downstream signalling components. Defence‐associated suppression
    of NMD can lead to increased abundance of some NLR isoforms, probably resulting
    in heightened states of immunity. (G) An increasing number of NLRs are found to
    be regulated by alternative polyadenylation. In the case of Arabidopsis RPP7,
    a transposon recruited the silencing mark H3K9me2 into the first intron of this
    NLR gene. High levels of H3K9me2 in this area suppress the use of a proximal polyadenylation
    site and promote the production of full‐length RPP7 coding mRNAs. Reduced levels
    of H3K9me2 caused by defence induction or mutations in the EDM2 gene lead to enhanced
    use of the proximal polyadenylation site, increased levels of a prematurely terminated
    non‐coding transcript and, consequently, reduced levels of full‐length RPP7 coding
    transcripts. Alternative polyadenylation can be combined with alternative splicing
    by the differential inclusion of alternative terminal exons, as in RPP7. (A–G)
    Basic graphical elements used in these panels are defined in (A). Features specifically
    important or unique for the respective mechanisms are shown in red. TE, transposable
    element; alt, alternative exon; Defence, defence‐inducing signals; thin pointed
    arrows, activating interactions; thin blunt arrows, suppressing interactions.
    (H) MicroRNA (miRNA)‐triggered cascades of secondary phased small interfering
    RNAs (phasiRNAs). In several cases, miRNAs targeting conserved regions in NLR
    transcripts were shown to serve as initial triggers (short red horizontal lines)
    of secondary phasiRNA (short blue horizontal lines) cascades, which comprehensively
    mediate post‐transcriptional co‐silencing of NLR gene families. On association
    with primary target NLR transcripts (e.g. NLRs a, b and c in the figure), these
    miRNAs trigger the production of phasiRNAs which amplify and diversify the initial
    trigger and reinforce the silencing of the primary NLR targets (e.g. NLRs a, b
    and c), but also secondary NLR targets (e.g. NLRs x, y and z). Consequently, the
    secondary phasiRNAs target a wider set of NLR transcripts than the initial miRNA
    trigger. Such NLR silencing cascades can be suppressed by microbial RNA silencing
    suppressors resulting in enhanced immunity.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - N-b
  - LRR
  - CAP
  - SMC3
  - Cpn
  - Capa
  - capt
  - bel
  - SERCA
  - Cadps
  - Start1
  - SCAP
  - bmm
  - te
  - ptc
  - nmd
  - SCAR
  - Atg1
  - squ
  - alt
  - CG1640
  - Ca-alpha1T
  - alphaTub84B
  - sca
  - sc
  - TIR
  - SDG25
  - AT1G12530
  - HUB1
  - HUB2
  - CNGC4
  - MET1
  - CMT2
  - CMT3
  - SUVH4
  - SUVH5
  - SUVH6
  - EDM2
  - CXCR1
  - CTAA1
  - LNPEP
  - SERPINB6
  - HACD1
  - CAP1
  - SORBS1
  - BRD4
  - CAP2
  - ZNF282
  - UBL5
  - GZMM
  - SH2D2A
  - MAD2L1BP
  - KIF1B
  - AAA3
  - F9
  - TAS2R38
  - PTCH1
  - RET
  - CCDC6
  - RPS4X
  - AGER
  - CD14
  - CD163
  - CD163L1
  - CD207
  - CD209
  - CD36
  - CD68
  - CLEC7A
  - COLEC12
  - CXCL16
  - LY75
  - MARCO
  - MEGF10
  - MRC1
  - MSR1
  - OLR1
  - SCARA3
  - SCARA5
  - SCARB1
  - SCARB2
  - SCARF1
  - SCARF2
  - SSC4D
  - SSC5D
  - STAB1
  - STAB2
  - SLC1A5
  - SLC1A7
  - COL9A2
  - MAN2C1
  - GPT
  - SLC5A8
  - H3K4me3
  - H3K9me2
  - PtC
  - EDM2
---
