---
figid: PMC6789908__pnas.1913493116fig01
figtitle: Rationale and workflow of CRISPR-BEST
organisms:
- NA
pmcid: PMC6789908
filename: pnas.1913493116fig01.jpg
figlink: /pmc/articles/PMC6789908/figure/fig01/
number: F1
caption: Rationale and workflow of CRISPR-BEST. (A) The CRISPR-BEST plasmid is a pSG5
  replicon-based, temperature-sensitive E. coli–Streptomyces shuttle plasmid. The
  displayed plasmid map is CRISPR-cBEST, in which the core component is the fusion
  protein of S. coelicolor A3(2) codon-optimized rAPOBEC1, Cas9n (D10A), and UGI.
  The key component of CRISPR-aBEST is a fusion protein of S. coelicolor A3(2) codon-optimized
  ecTadA, and Cas9n (D10A). Expression of the fusion proteins is controlled by a promoter
  PtipA. In our tests, the leakage expression is enough to carry out the base editing
  functionality; stronger expression can be induced by adding thiostrepton. (B) The
  sgRNA cassette is under control of the promoter PermE*. A PCR-free, 1-step ssDNA
  bridging approach can be applied for the 20-bp spacer cloning. (C and D) Overview
  of the base editing strategies for CRISPR-cBEST and CRISPR-aBEST, respectively.
  The target nucleotide within the editing window is indicated in red and the possible
  active domains in each step is shown in a brighter color. First, sgRNA (purple)
  binds to D10A Cas9n (blue), ending up with Cas9n:sgRNA complex. Second, the Cas9n:sgRNA
  complex finds and binds its target DNA, which mediates the separation of the double-stranded
  DNA to form the R-loop structure. Third (C), for C-to-T editing, a tethered Streptomyces-optimized
  cytidine deaminase rAPOBEC1 (green) converts the target C in the nontargeted strand
  to a U by cytidine deamination. Due to the inhibition of the nucleotide excision
  repair (NER) pathway by UGI, the cellular mismatch repair (MMR) becomes the dominant
  DNA repair pathway. It preferentially repairs the mismatch in a nicked strand. Therefore,
  the G in the targeted strand, which is nicked by D10A Cas9n, is going to be efficiently
  replaced by A and, in the next replication cycle, repaired to a T:A base pair. (D)
  For A-to-G editing, a tethered Streptomyces-optimized adenosine deaminase ecTadA
  heterodimer (cyan-yellow) converts the target A in the nontargeted strand to an
  I by deamination. As I is read as G by DNA polymerase, the resulting I:T heteroduplex
  is permanently converted to a G:C base pair during DNA replication. (E) Representation
  of the possible amino acid exchanges resulting from CRISPR-BEST. Blue lines indicate
  that the edited nucleotide is in the coding strand, while red lines indicate that
  the edited nucleotide is in the noncoding strand. The thickness of the lines indicates
  the number of possible routes that can end up with the same amino acid exchange
  by CRISPR-BEST.
papertitle: Highly efficient DSB-free base editing for streptomycetes with CRISPR-BEST.
reftext: Yaojun Tong, et al. Proc Natl Acad Sci U S A. 2019 Oct 8;116(41):20366-20375.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5337803
figid_alias: PMC6789908__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC6789908__F1
ndex: e3152ab6-df34-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6789908__pnas.1913493116fig01.html
  '@type': Dataset
  description: Rationale and workflow of CRISPR-BEST. (A) The CRISPR-BEST plasmid
    is a pSG5 replicon-based, temperature-sensitive E. coli–Streptomyces shuttle plasmid.
    The displayed plasmid map is CRISPR-cBEST, in which the core component is the
    fusion protein of S. coelicolor A3(2) codon-optimized rAPOBEC1, Cas9n (D10A),
    and UGI. The key component of CRISPR-aBEST is a fusion protein of S. coelicolor
    A3(2) codon-optimized ecTadA, and Cas9n (D10A). Expression of the fusion proteins
    is controlled by a promoter PtipA. In our tests, the leakage expression is enough
    to carry out the base editing functionality; stronger expression can be induced
    by adding thiostrepton. (B) The sgRNA cassette is under control of the promoter
    PermE*. A PCR-free, 1-step ssDNA bridging approach can be applied for the 20-bp
    spacer cloning. (C and D) Overview of the base editing strategies for CRISPR-cBEST
    and CRISPR-aBEST, respectively. The target nucleotide within the editing window
    is indicated in red and the possible active domains in each step is shown in a
    brighter color. First, sgRNA (purple) binds to D10A Cas9n (blue), ending up with
    Cas9n:sgRNA complex. Second, the Cas9n:sgRNA complex finds and binds its target
    DNA, which mediates the separation of the double-stranded DNA to form the R-loop
    structure. Third (C), for C-to-T editing, a tethered Streptomyces-optimized cytidine
    deaminase rAPOBEC1 (green) converts the target C in the nontargeted strand to
    a U by cytidine deamination. Due to the inhibition of the nucleotide excision
    repair (NER) pathway by UGI, the cellular mismatch repair (MMR) becomes the dominant
    DNA repair pathway. It preferentially repairs the mismatch in a nicked strand.
    Therefore, the G in the targeted strand, which is nicked by D10A Cas9n, is going
    to be efficiently replaced by A and, in the next replication cycle, repaired to
    a T:A base pair. (D) For A-to-G editing, a tethered Streptomyces-optimized adenosine
    deaminase ecTadA heterodimer (cyan-yellow) converts the target A in the nontargeted
    strand to an I by deamination. As I is read as G by DNA polymerase, the resulting
    I:T heteroduplex is permanently converted to a G:C base pair during DNA replication.
    (E) Representation of the possible amino acid exchanges resulting from CRISPR-BEST.
    Blue lines indicate that the edited nucleotide is in the coding strand, while
    red lines indicate that the edited nucleotide is in the noncoding strand. The
    thickness of the lines indicates the number of possible routes that can end up
    with the same amino acid exchange by CRISPR-BEST.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - cg
  - GC
  - MED24
  - EcR
  - arg
  - aos
  - arm
  - Cys
  - mmy
  - Csp
  - stop
  - as
  - uri
  - ha
  - Hn
  - PSG5
  - ABL2
  - RERE
  - AKR1A1
  - ARMC9
  - MAP6
  - PAM
  - MYCBP2
  - COX2
  - MTCO2P12
  - KY
  - Ala
  - Thr Leu
  - Phe
  - Leu
  - Ala Thr
  - Val
  - Met
  - Ser
  - Glu
  - Asn
  - Lys
  - Asp
  - Arg
  - Tyr
  - Trp
  - Cys Tyr
  - Cytidine
  - Uridine
  - Adenosine
  - Inosine
---
