---
figid: PMC6425966__nihms-999818-f0001
figtitle: Overview of the construction, computational prioritization, and validation
  of the drug repurposing network
organisms:
- Homo sapiens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC6425966
filename: nihms-999818-f0001.jpg
figlink: /pmc/articles/PMC6425966/figure/F1/
number: F1
caption: A) ITS computation. We applied ITS to compute the similarity between GO-BPs,
  SNPs, and genes through a cascade process as described in Methods –. This began
  construction of the biomolecular network layer. B) Integration of multiscale biomolecular
  associations using GWAS diseases, SNPs, and eGenes as nodes. The associations (edges)
  between nodes were obtained by extracting GWAS disease-to-SNP, and SNP-to-eGene
  (SNPeG) relationships from the database resources described (). The biomolecular
  network was then filtered to remove SNP-SNP pairs not meeting the introduced criteria
  (Edge Legend, ). ITSSNP-SNP is computed as in  considering all the eGenes extracted
  from eQTL data and the network was further refined to include only significantly
  similar eGene-eGene pairs, i.e. ITSSNP-SNP and ITSeGENE-eGENE () False Discovery
  Rate (FDR) <0.05. Drug-eGene and Drug-indication associations extracted from Drugbank
  (drug information layer) are included to obtain the final drug-repurposing network.
  C) Network validation. The drug repurposing network is validated by querying if
  the network predicted a significantly high number of gold standard treatments for
  GWAS diseases. Two conditions of validation are proposed, one stringent and one
  more relaxed (**). D) Drug repurposing patterns. We extracted GWAS disease pairs
  and the related convergent mechanisms where at least a gold standard treatment was
  predicted for one of the two GWAS diseases. The approach predicts new candidate
  therapies by repositioning drugs across these GWAS disease pairs.
papertitle: Precision drug repurposing via convergent eQTL-based molecules and pathway
  targeting independent disease-associated polymorphisms.
reftext: Francesca Vitali, et al. Pac Symp Biocomput. 2019;24:308-319.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7944297
figid_alias: PMC6425966__F1
figtype: Figure
redirect_from: /figures/PMC6425966__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6425966__nihms-999818-f0001.html
  '@type': Dataset
  description: A) ITS computation. We applied ITS to compute the similarity between
    GO-BPs, SNPs, and genes through a cascade process as described in Methods –. This
    began construction of the biomolecular network layer. B) Integration of multiscale
    biomolecular associations using GWAS diseases, SNPs, and eGenes as nodes. The
    associations (edges) between nodes were obtained by extracting GWAS disease-to-SNP,
    and SNP-to-eGene (SNPeG) relationships from the database resources described ().
    The biomolecular network was then filtered to remove SNP-SNP pairs not meeting
    the introduced criteria (Edge Legend, ). ITSSNP-SNP is computed as in  considering
    all the eGenes extracted from eQTL data and the network was further refined to
    include only significantly similar eGene-eGene pairs, i.e. ITSSNP-SNP and ITSeGENE-eGENE
    () False Discovery Rate (FDR) <0.05. Drug-eGene and Drug-indication associations
    extracted from Drugbank (drug information layer) are included to obtain the final
    drug-repurposing network. C) Network validation. The drug repurposing network
    is validated by querying if the network predicted a significantly high number
    of gold standard treatments for GWAS diseases. Two conditions of validation are
    proposed, one stringent and one more relaxed (**). D) Drug repurposing patterns.
    We extracted GWAS disease pairs and the related convergent mechanisms where at
    least a gold standard treatment was predicted for one of the two GWAS diseases.
    The approach predicts new candidate therapies by repositioning drugs across these
    GWAS disease pairs.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - eg-1
  - Go
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Snp
  - Mhc
  - zip
  - ITSGO-GO
  - NODE
  - GWD
---
