---
figid: PMC6966572__cancers-11-01980-g001
figtitle: Action of DNA damage repair in normal healthy cells and with PARP inhibitors
organisms:
- Homo sapiens
- Mus musculus
organisms_ner:
- Drosophila melanogaster
pmcid: PMC6966572
filename: cancers-11-01980-g001.jpg
figlink: pmc/articles/PMC6966572/figure/cancers-11-01980-f001/
number: F1
caption: 'Mechanism of action of DNA damage repair in normal healthy cells and with
  PARP inhibitors. Abbreviations: SSB = single strand break, DSB = double strand break,
  DDR = DNA damage repair, HRR = homologous recombination repair, PARP = poly(ADP-ribose)
  polymerase, PARPi= PARP inhibitor, BER = base excision repair. (A) Normal DNA damage
  repair process: (a) In healthy cells, SSB mainly by BER pathway mediated by a family
  of enzymes known as PARP. (b) PARP-1 detects SSBs and binds to the DNA damage site
  via zinc motif fingers at the DNA binding domain. (c) PARP DNA binding activates
  its catalytic activity and utilization of NAD+ to synthesize poly ADP-ribose (pADPr)
  polymer formation on itself (autoPARylation) and other histone proteins. The pADPr
  polymers recruit DNA repair proteins, including XRCC1. (d) PARylation also reduces
  the affinity of PARP-1 for DNA binding, releasing it from the site of DDR. The pADPr
  polymers are lysed from PARP by enzymes such as PARG and ARH3, restoring its ability
  to detect and bind to DNA damage sites. (e) PARP removal from the site of DNA damage
  allows DDR effector proteins to bind at the site of damage leading to successful
  repair of DNA as depicted in (f). (B): (a) SSB in the presence of PARPi. (b) PARPi
  bind with the catalytic domain of PARP enzyme and inhibit the synthesis of pADPr
  formation and recruitment of DNA repair proteins. (c) PARPi also trap PARP-1 on
  the damaged DNA site, prevent its release and accumulate cytotoxic DNA complexes.
  (d) This eventually can culminate in cell death. (e) When BER mechanism is dysfunctional,
  unrepaired SSBs stall the replication fork leading to DSBs formation. (f–i) The
  concept of synthetic lethality. (f,h) In BRCA 1 or 2 heterozygous (BRCA+/−) or BRCA
  wild-type (BRCA+/+) cells, the DDR occurs through the error-free HRR pathway averting
  the cell death. (g) In the presence of BRCA1 or BRCA2 mutations (BRCA−/−) and other
  mutations with a similar phenotype of defective HRR, the cells are unable to repair
  DSBs through HR pathway. (i) Repair alternatively occurs through the error-prone
  pathway of NHEJ leading to cell cycle arrest, genomic instability and lethality.'
papertitle: 'Poly(ADP-Ribose) Polymerase Inhibitors in Pancreatic Cancer: A New Treatment
  Paradigms and Future Implications.'
reftext: Medhavi Gupta, et al. Cancers (Basel). 2019 Dec;11(12):1980.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9319654
figid_alias: PMC6966572__F1
figtype: Figure
redirect_from: /figures/PMC6966572__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6966572__cancers-11-01980-g001.html
  '@type': Dataset
  description: 'Mechanism of action of DNA damage repair in normal healthy cells and
    with PARP inhibitors. Abbreviations: SSB = single strand break, DSB = double strand
    break, DDR = DNA damage repair, HRR = homologous recombination repair, PARP =
    poly(ADP-ribose) polymerase, PARPi= PARP inhibitor, BER = base excision repair.
    (A) Normal DNA damage repair process: (a) In healthy cells, SSB mainly by BER
    pathway mediated by a family of enzymes known as PARP. (b) PARP-1 detects SSBs
    and binds to the DNA damage site via zinc motif fingers at the DNA binding domain.
    (c) PARP DNA binding activates its catalytic activity and utilization of NAD+
    to synthesize poly ADP-ribose (pADPr) polymer formation on itself (autoPARylation)
    and other histone proteins. The pADPr polymers recruit DNA repair proteins, including
    XRCC1. (d) PARylation also reduces the affinity of PARP-1 for DNA binding, releasing
    it from the site of DDR. The pADPr polymers are lysed from PARP by enzymes such
    as PARG and ARH3, restoring its ability to detect and bind to DNA damage sites.
    (e) PARP removal from the site of DNA damage allows DDR effector proteins to bind
    at the site of damage leading to successful repair of DNA as depicted in (f).
    (B): (a) SSB in the presence of PARPi. (b) PARPi bind with the catalytic domain
    of PARP enzyme and inhibit the synthesis of pADPr formation and recruitment of
    DNA repair proteins. (c) PARPi also trap PARP-1 on the damaged DNA site, prevent
    its release and accumulate cytotoxic DNA complexes. (d) This eventually can culminate
    in cell death. (e) When BER mechanism is dysfunctional, unrepaired SSBs stall
    the replication fork leading to DSBs formation. (f–i) The concept of synthetic
    lethality. (f,h) In BRCA 1 or 2 heterozygous (BRCA+/−) or BRCA wild-type (BRCA+/+)
    cells, the DDR occurs through the error-free HRR pathway averting the cell death.
    (g) In the presence of BRCA1 or BRCA2 mutations (BRCA−/−) and other mutations
    with a similar phenotype of defective HRR, the cells are unable to repair DSBs
    through HR pathway. (i) Repair alternatively occurs through the error-prone pathway
    of NHEJ leading to cell cycle arrest, genomic instability and lethality.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - mtSSB
  - Parp
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - side
  - da
  - nAChRalpha2
  - double
  - AP-2alpha
  - Ada
  - NAD
  - Nicotinamide
  - Poly(ADP
---
