---
figid: PMC6982389__40168_2020_786_Fig5_HTML
figtitle: Representative metabolic maps of bacterial groups in the bioreactor
organisms:
- metagenome
- metagenomes
- activated sludge metagenome
- Pannonibacter phragmitetus
- Campylobacterota
- Candidatus Brocadia sapporoensis
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC6982389
filename: 40168_2020_786_Fig5_HTML.jpg
figlink: pmc/articles/PMC6982389/figure/Fig5/
number: F5
caption: Representative metabolic maps of bacterial groups in the bioreactor. To prevent
  redundancy, the metabolism is presented in a nested approach with each panel showing
  only paths unique to the relevant metabolic group. Two exceptions are Group β (all
  detected paths are shown), and Group δ. The latter is not presented here since it
  shares all paths with Group ɣ and only differs by auxotrophies. (A) Metabolic map
  of paths that are common to all bacteria in the bioreactor (except Microgenomates
  and Brocadia sp.). The vast majority of bacteria in the bioreactor are heterotrophs,
  capable of carbohydrate-based metabolism (glycolysis, pentose phosphate pathway),
  and amino acid-based metabolism. Some bacteria can respire oxygen and can also ferment
  (acetate/alanine). (B) Paths unique to Group ɑ. These bacteria have genes for hydrogen
  oxidation, supporting anaerobic growth, as well as genes for oxidative phosphorylation
  with cytochrome BD complex. These bacteria have a cassette of extracellular proteases
  and decarboxylases, paired with a wide array of transporters. They are also potentially
  capable of synthesizing long chain isoprenoids. (C) Paths found in Gram (–) bacteria
  (Groups ɣ, δ, and ε). Most paths are related to fatty acid and lipid synthesis.
  Several important precursors (chorismate and IMP) can potentially be synthesized
  by these bacteria. Motility is also a common feature in these bacteria (via a flagellar
  motor). (D) Unique paths of Group ε (Proteobacteria). This group has the potential
  to synthesize multiple vitamins and cofactors (biotin, pyridoxal, glutathione, etc.),
  as well as several amino acids (tyrosine, phenylalanine, proline). Another unique
  feature is the multiple secretion systems present in the bacteria. (E) Metabolic
  profile of CPR bacteria (Microgenomates). These bacteria are obligate anaerobes
  that ferment pyruvate. They can only utilize carbohydrates as their carbon source.
  Some of the bacteria in this group might also be able to synthesize long chain isoprenoids,
  in the same path as Group ɑ
papertitle: Increased replication of dissimilatory nitrate-reducing bacteria leads
  to decreased anammox bioreactor performance.
reftext: Ray Keren, et al. Microbiome. 2020;8:7.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7460945
figid_alias: PMC6982389__F5
figtype: Figure
redirect_from: /figures/PMC6982389__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6982389__40168_2020_786_Fig5_HTML.html
  '@type': Dataset
  description: Representative metabolic maps of bacterial groups in the bioreactor.
    To prevent redundancy, the metabolism is presented in a nested approach with each
    panel showing only paths unique to the relevant metabolic group. Two exceptions
    are Group β (all detected paths are shown), and Group δ. The latter is not presented
    here since it shares all paths with Group ɣ and only differs by auxotrophies.
    (A) Metabolic map of paths that are common to all bacteria in the bioreactor (except
    Microgenomates and Brocadia sp.). The vast majority of bacteria in the bioreactor
    are heterotrophs, capable of carbohydrate-based metabolism (glycolysis, pentose
    phosphate pathway), and amino acid-based metabolism. Some bacteria can respire
    oxygen and can also ferment (acetate/alanine). (B) Paths unique to Group ɑ. These
    bacteria have genes for hydrogen oxidation, supporting anaerobic growth, as well
    as genes for oxidative phosphorylation with cytochrome BD complex. These bacteria
    have a cassette of extracellular proteases and decarboxylases, paired with a wide
    array of transporters. They are also potentially capable of synthesizing long
    chain isoprenoids. (C) Paths found in Gram (–) bacteria (Groups ɣ, δ, and ε).
    Most paths are related to fatty acid and lipid synthesis. Several important precursors
    (chorismate and IMP) can potentially be synthesized by these bacteria. Motility
    is also a common feature in these bacteria (via a flagellar motor). (D) Unique
    paths of Group ε (Proteobacteria). This group has the potential to synthesize
    multiple vitamins and cofactors (biotin, pyridoxal, glutathione, etc.), as well
    as several amino acids (tyrosine, phenylalanine, proline). Another unique feature
    is the multiple secretion systems present in the bacteria. (E) Metabolic profile
    of CPR bacteria (Microgenomates). These bacteria are obligate anaerobes that ferment
    pyruvate. They can only utilize carbohydrates as their carbon source. Some of
    the bacteria in this group might also be able to synthesize long chain isoprenoids,
    in the same path as Group ɑ
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Vha14-1
  - path
  - ali
  - alpha-Est7
  - gus
  - pb
  - Spn88Eb
  - pim
  - RFeSP
  - amon
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - mn
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - wo
  - Ipp
  - CG9784
  - CG9426
  - CG6805
  - Synj
  - INPP5E
  - 5PtaseI
  - Coa
  - pe
  - gb
  - Gbeta5
  - Gbeta13F
  - Gbeta76C
  - ca
  - na
  - mol
  - aa
  - porin
  - CycE
  - cyc
  - so
  - E(spl)mgamma-HLH
  - Pa
  - ps
  - arg
  - aos
  - Pep
  - bgcn
  - Nak
  - Atpalpha
  - plo
  - cu
  - Cpr
  - mo
  - be
  - Camta
  - DNAH8
  - BCR
  - IGKV3D-20
  - SMR3B
  - SERPINB1
  - SERPINB2
  - SERPINB3
  - SERPINB4
  - SERPINB5
  - SERPINB6
  - SERPINB7
  - SERPINB8
  - SERPINB9
  - SERPINB10
  - SERPINB11
  - SERPINB12
  - SERPINB13
  - UQCRFS1
  - IMPA1
  - BRAP
  - SLC6A6
  - RASA1
  - RGS6
  - IPP
  - SUCLG2
  - GNB1
  - GNB2
  - GNB3
  - GNB4
  - GNB5
  - GIPC1
  - DUOXA1
  - CRPPA
  - NDUFB3
  - TNFAIP1
  - VDAC1
  - ABL2
  - RERE
  - PAEP
  - PREP
  - PTPN22
  - MCEE
  - TBK1
  - SERPINA2
  - SERPINA1
  - POR
  - sulfate
  - NADH
  - Succinate
  - pyridoxal
  - NAOH NAD
  - Pyrimidine
  - Gluconate
  - Succinyl
  - Pomehaloselmaltose
  - Arginine
  - Glutamine
  - Histidine NH
  - Orithine
  - Mn
  - Fe
  - 1Thiamine
  - Purine
  - Raffinose
  - Glutamate
  - Oxaloglutarate
  - Fumarate
  - Glycine
  - Propanoyl
  - Threonine
  - Homoserine
  - Aspartate
  - NAD NADH
  - Citrate
  - spermidine
  - putrescine
  - Oxaloacetate
  - 2-Oxoglutarate
  - Palmitate
  - Acetyl-CoA
  - NAD
  - Ethanol
  - Sucrose
  - sugar
  - Acetaldehyde
  - 2-onoisovalorate
  - Ca
  - Na
  - Acetoacetate
  - Valine
  - Tryptophan
  - Acetate
  - NUCO
  - Zn
  - malonyl-ACP pimeloyl-ACP
  - Ni
  - Urea
  - Iron
  - Polyamine
  - Pyruvate
  - TCA
  - Acetyl-CoA Ce C12
  - phosphoenolpyruvate
  - Mg
  - Phosphatidylethanolamine
  - Sedoneptulose-7P DRibulose
  - ADP
  - L-glycerb-D-manno-heptose CMP-KDO
  - Glycerol-2P
  - Acyl CoA
  - NaH
  - KDO2-lipid A
  - UDP
  - N-acetylglucosamine
  - Methionine
  - Triacylglyoerol
  - Giutamine
  - Serine
  - lysine
  - Arg
  - Galactose
  - fatty acids
  - DMPP
  - lactate
  - Lindane
  - Cu
  - ZnCd
  - Oxalate
  - Homoprotocatechuate2-axohept-3-enedioate
  - O-acetylserinelcysteine
  - Homocerine-lactone
  - Leucine
  - Acetyl-COA
  - Benzoyl-CoA
  - 3-hydroxypimeloyl-CoA
  - Formate
  - Glutathione
  - Arogenate
  - Ubiquinone
  - Prephenate Pretyrosine Tyrosine
  - Phenylpyruvate Phenylalanine
  - Lysine Saccharopine Acetoacetyl-CoA
  - Benzoate
  - acyl-ACP
  - Pimeloyl-ACP
  - Tricarboxlate
  - KAPA Biotin
  - Proline
  - Pimelate Pimeloyl-CoA
  - sn-glycerol 3-phosphate
  - Erythrose-4P Pyridoxal
  - ZnVMn
  - NH
---
