---
figid: PMC6617847__12870_2019_1793_Fig5_HTML
figtitle: Schematic outline of the salinity-induced response pathway based on proteome,
  gene expression, and physiological changes in chickpea leaves
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Glycine max
- Solanum lycopersicum
- Diaporthe sclerotioides
- Triticum aestivum
- Hordeum vulgare
- Brassica napus
- Medicago truncatula
- Spinacia oleracea
- Pisum sativum
- Medicago sativa
- Phaseolus vulgaris
- Prunus persica
- Cicer arietinum
- Vicia faba
- Brachypodium distachyon
- Lotus japonicus
- Vigna unguiculata
- Chrysanthemum x morifolium
- Cajanus cajan
- Glycine soja
- Cynodon dactylon
- Cucurbita pepo
- Gossypium raimondii
- Suaeda salsa
- Zostera marina
- Robinia pseudoacacia
- Solanum chacoense
- Lolium rigidum
- Acanthus ebracteatus
- Cynodon x magennisii
- Kengyilia thoroldiana
- Trifolium aureum
- Halogeton glomeratus
- Androctonus crassicauda
- Heterometrus longimanus
- Homo sapiens
organisms_ner:
- Homo sapiens
- Glycine max
- Arabidopsis thaliana
- Cicer arietinum
- Drosophila melanogaster
pmcid: PMC6617847
filename: 12870_2019_1793_Fig5_HTML.jpg
figlink: /pmc/articles/PMC6617847/figure/Fig5/
number: F5
caption: Schematic outline of the salinity-induced response pathway based on proteome,
  gene expression, and physiological changes in chickpea leaves. Numbers in yellow
  circles are corresponding to the protein(s) mentioned in Table . Blue arrows indicate
  induction or increase; red arrows indicate repression or decrease, and dotted arrows
  indicate possible or reversible alterations. Salinity activates several signaling
  cascades, regulates gene expression, and promotes regulatory and functional protein
  biosynthesis, in which the main role of ROS, antioxidants, and chaperones under
  saline condition is presented. Tolerant seedlings inhibited or decreased major metabolic
  pathways (i.e., photosynthesis, photorespiration, TCA cycle, and amino acid metabolism)
  less than susceptible seedlings to provide more energy and other compatible metabolites.
  Importantly, salt tolerance relies on higher transporter activities to remove H2O2,
  Na+, etc., and enhance osmotic regulation, and membrane and cell wall re-modulation
  to cope with the stress
papertitle: Comparative proteomics and gene expression analyses revealed responsive
  proteins and mechanisms for salt tolerance in chickpea genotypes.
reftext: Mohammad Arefian, et al. BMC Plant Biol. 2019;19:300.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.933715
figid_alias: PMC6617847__F5
figtype: Figure
redirect_from: /figures/PMC6617847__F5
ndex: 531d4c2c-df09-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6617847__12870_2019_1793_Fig5_HTML.html
  '@type': Dataset
  description: Schematic outline of the salinity-induced response pathway based on
    proteome, gene expression, and physiological changes in chickpea leaves. Numbers
    in yellow circles are corresponding to the protein(s) mentioned in Table . Blue
    arrows indicate induction or increase; red arrows indicate repression or decrease,
    and dotted arrows indicate possible or reversible alterations. Salinity activates
    several signaling cascades, regulates gene expression, and promotes regulatory
    and functional protein biosynthesis, in which the main role of ROS, antioxidants,
    and chaperones under saline condition is presented. Tolerant seedlings inhibited
    or decreased major metabolic pathways (i.e., photosynthesis, photorespiration,
    TCA cycle, and amino acid metabolism) less than susceptible seedlings to provide
    more energy and other compatible metabolites. Importantly, salt tolerance relies
    on higher transporter activities to remove H2O2, Na+, etc., and enhance osmotic
    regulation, and membrane and cell wall re-modulation to cope with the stress
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TNFSF14
  - FABP4
  - GTF3A
  - TFAP2A
  - MYB
  - XK
  - NLRP1
  - MYC
  - DECR1
  - AVP
  - IFNAR1
  - NLRP3
  - DNAH8
  - XYLT2
  - SOS1
  - SOS2
  - AOC1
  - SHBG
  - SUCLG2
  - GNB1
  - GNB2
  - GNB3
  - GNB4
  - GNB5
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - YWHAQ
  - GRP
  - LSM4
  - LGALSL
  - UCMA
  - ATP8A2
  - DREB
  - WRKY14
  - CDPK
  - N2
  - LBA
  - AP2
  - bZIP
  - ABP1
  - LEA
  - ECP63
  - BZIP
  - 14-3-3
  - na
  - et
  - SA
  - sa
  - ca
  - Low
  - lm
  - pr
  - CycE
  - cyc
  - AP-2alpha
  - TfAP-2
  - AP-2sigma
  - Myb
  - nac
  - Nacalpha
  - Myc
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Vha14-1
  - Sos
  - grh
  - Rheb
  - gb
  - Gbeta5
  - Gbeta13F
  - Gbeta76C
  - MKP-4
  - p38b
  - rl
  - robo2
  - 14-3-3zeta
  - 14-3-3epsilon
  - grp
  - Gagr
  - gpp
  - ATPsynbeta
  - Atpalpha
  - Na
  - Ca
  - CO
  - Sugar
  - Pentose Phosphate
  - TCA
  - NAC
  - NADPH
  - Polyols
  - sugars
  - Fatty acids
  - Met
  - Glu
  - polyamine
  - ethylene
  - Amino acid
  - callose
---
