---
figid: PMC6246609__41396_2018_238_Fig7_HTML
figtitle: Stimuli responses of EET-related metabolic pathways of four abundantly shown
  Geobacteraceae OCSs
organisms:
- Escherichia coli
- Shewanella oneidensis
- Geobacter sulfurreducens
- Geobacter metallireducens
- Geobacter sp.
- Syntrophotalea carbinolica
- Geotalea uraniireducens
- Pelobacter propionicus
- Geobacter sulfurreducens PCA
- Methylophilus sp.
- Desulfurivibrio alkaliphilus
- Methanocorpusculum parvum
- Acetobacterium submarinus
- Methanocorpusculum bavaricum
- Anaerosinus glycerini
- Anaeroarcus burkinensis
- Methanocorpusculum sinense
- human metagenome
- uncultured bacterium
- uncultured bacterium Of14
- Homo sapiens
- Triportheus paranensis
pmcid: PMC6246609
filename: 41396_2018_238_Fig7_HTML.jpg
figlink: /pmc/articles/PMC6246609/figure/Fig7/
number: F7
caption: Stimuli responses of EET-related metabolic pathways of four abundantly shown
  Geobacteraceae OCSs. EET-related metabolic pathway map constructed by gene expression
  profiles of selected marker genes (refer to Supplementary Fig. ) in the OCS H1geo(pan)
  (a), H2/M4geo (b), L1/M1geo(pan) (c), and M3ppro (d). The operational variable-associated
  metabolic flows and electron (e-) flow are analyzed by the wCCA analysis (Supplementary
  Fig. -). Sucrose-consuming EET (Sucrose EET) pathway was shown by yellow arrows,
  acetate-consuming EET (Acetate EET) pathway was shown by red arrows, and open circuit
  responses were shown by blue arrows with the detailed description about OC responses
  for each OCS in the blue box. Genes coding MH-cytC proteins for EET reactions are
  shown as oval (Names are described inside)
papertitle: Comparative metatranscriptomics reveals extracellular electron transfer
  pathways conferring microbial adaptivity to surface redox potential changes.
reftext: Shun’ichi Ishii, et al. ISME J. 2018 Dec;12(12):2844-2863.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8319133
figid_alias: PMC6246609__F7
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6246609__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6246609__41396_2018_238_Fig7_HTML.html
  '@type': Dataset
  description: Stimuli responses of EET-related metabolic pathways of four abundantly
    shown Geobacteraceae OCSs. EET-related metabolic pathway map constructed by gene
    expression profiles of selected marker genes (refer to Supplementary Fig. ) in
    the OCS H1geo(pan) (a), H2/M4geo (b), L1/M1geo(pan) (c), and M3ppro (d). The operational
    variable-associated metabolic flows and electron (e-) flow are analyzed by the
    wCCA analysis (Supplementary Fig. -). Sucrose-consuming EET (Sucrose EET) pathway
    was shown by yellow arrows, acetate-consuming EET (Acetate EET) pathway was shown
    by red arrows, and open circuit responses were shown by blue arrows with the detailed
    description about OC responses for each OCS in the blue box. Genes coding MH-cytC
    proteins for EET reactions are shown as oval (Names are described inside)
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CTSA
  - PNPLA6
  - PTGER3
  - SFTA3
  - SHE
  - DCLK1
  - ADGRL1
  - CYC1
  - CCNI
  - USF2
  - SGPL1
  - Sucrose
  - Formate
  - Acetate
  - Lactate
  - Ethanol
  - Propionate
  - TCA
  - NADH
  - TEA
  - Sucrose EET EET
  - Sucrose EET
  - Acetate EET
---
