---
figid: PMC6247912__nihms-986371-f0002
figtitle: Somatic mutations of the canonical signal transduction and tumor suppressor
  pathways in HCC
organisms:
- Iris germanica
- Phleum pratense
- Homo sapiens
- Bos taurus
pmcid: PMC6247912
filename: nihms-986371-f0002.jpg
figlink: /pmc/articles/PMC6247912/figure/F2/
number: F2
caption: (A) RAS/RAF/MAPK and PI3K/AKT/mTOR pathway mutations occurred in 55% of tumors,
  and the incidence of mutation in each component of the pathway is indicated. Oncogenic
  alterations are shown in red; tumor suppressor alterations are shown in blue. All
  alterations were somatic mutations, with the exception of amplification events for
  RHEB and BRAF and deletion events for NF1. (B) DNA damage and DNA repair pathways
  were mutated in 38% of tumors, and the incidence of mutation in each component of
  the pathway is indicated. Gene mutations involved in DNA damage are shown in blue.
  Mutations in DNA repair pathways occurred in homologous recombination (orange),
  nucleotide excision repair (green), mismatch repair (brown), and DNA strand cross-link
  repair (red). (C) Epigenetic-modifying gene mutations occurred in 33 tumors (59%),
  by either chromatin modification (55%) or DNA modification (9%); the incidence of
  mutation in each component of the pathway is indicated. Chromatin-modifying mutations
  occurred in chromatin-modifying complexes, such as the SWI/SNF complex (red), histone
  acetyltransferases (green), methyltransferases (blue), histone deacetylases (purple),
  and demethylases (brown), as well as in histones themselves. DNAmodifying mutations
  occurred in DNA methyltransferases (yellow) and demethylases (orange).
papertitle: Integrated genomic analysis of HÃ¼rthle cell cancer reveals oncogenic drivers,
  recurrent mitochondrial mutations, and unique chromosomal landscapes.
reftext: Ian Ganly, et al. Cancer Cell. 2018 Aug 13;34(2):256-270.e5.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7215222
figid_alias: PMC6247912__F2
figtype: Figure
organisms_ner:
- Homo sapiens
- Bos taurus
redirect_from: /figures/PMC6247912__F2
ndex: e3bf69ab-df0e-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6247912__nihms-986371-f0002.html
  '@type': Dataset
  description: (A) RAS/RAF/MAPK and PI3K/AKT/mTOR pathway mutations occurred in 55%
    of tumors, and the incidence of mutation in each component of the pathway is indicated.
    Oncogenic alterations are shown in red; tumor suppressor alterations are shown
    in blue. All alterations were somatic mutations, with the exception of amplification
    events for RHEB and BRAF and deletion events for NF1. (B) DNA damage and DNA repair
    pathways were mutated in 38% of tumors, and the incidence of mutation in each
    component of the pathway is indicated. Gene mutations involved in DNA damage are
    shown in blue. Mutations in DNA repair pathways occurred in homologous recombination
    (orange), nucleotide excision repair (green), mismatch repair (brown), and DNA
    strand cross-link repair (red). (C) Epigenetic-modifying gene mutations occurred
    in 33 tumors (59%), by either chromatin modification (55%) or DNA modification
    (9%); the incidence of mutation in each component of the pathway is indicated.
    Chromatin-modifying mutations occurred in chromatin-modifying complexes, such
    as the SWI/SNF complex (red), histone acetyltransferases (green), methyltransferases
    (blue), histone deacetylases (purple), and demethylases (brown), as well as in
    histones themselves. DNAmodifying mutations occurred in DNA methyltransferases
    (yellow) and demethylases (orange).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ERBB2
  - EGFR
  - PDGFRB
  - PDGFRA
  - TSHR
  - RET
  - XRCC3
  - BRCA1
  - XPC
  - ERCC5
  - ATM
  - HRAS
  - NF1
  - KRAS
  - NRAS
  - PTEN
  - BRAF
  - CHEK2
  - TP53
  - FANCD2
  - FANCB
  - CHEK1
  - MSH2
  - MSH3
  - TSC1
  - CCL26
  - TSC2
  - EPHB2
  - MAPK1
  - MAPK3
  - POLE
  - PMS2
  - E2F1
  - UBE2L3
  - PML
  - CDKN1A
  - TCEAL1
  - NSG1
  - H3P16
  - TLK1
  - RHEB
  - RHEBP1
  - RPS6KB1
  - RPS6KB2
  - MTOR
  - EIF4B
  - EIF1AX
  - EIF4A1
  - EIF4A2
  - EIF4A3
  - EIF4G1
  - EIF4G2
  - EIF4G3
  - EIF4H
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - PGF
  - EIF4EBP1
  - EIF4E
  - EIF4E2
  - EIF4E3
  - SMARCA1
  - INO80
  - SMARCA5
  - ATRX
  - DNMT1
  - DNMT3A
  - SRCAP
  - ERCC6
  - TET1
  - TET2
  - CHD9
  - CHD2
  - DSCAM
  - CHDH
  - CHD8
  - CHD5
  - GET1
  - KALRN
  - CHD6
  - CHD4
  - SMARCAD1
  - SMARCD2
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - BRD2
  - CREBBP
  - SETD1B
  - SETD2
  - SETMAR
  - NSD1
  - BRD7
  - SMYD4
  - KMT2A
  - KMT2D
  - KMT2C
  - KAT8
  - EZH1
  - SIRT6
  - KDM4C
  - KDM2B
  - KDM5C
  - JMJD1C
  - ARID1A
  - PHF2
  - PHF1
  - AIR
  - AIRN
  - PIK3CG
  - Nucleotide
  - INO80
  - KDM2B KDM4C KDM5C
  - CHD
---
