---
figid: PMC6921965__fonc-09-01388-g0001
figtitle: DSB repair pathways in mammalian cells
organisms:
- NA
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC6921965
filename: fonc-09-01388-g0001.jpg
figlink: /pmc/articles/PMC6921965/figure/F1/
number: F1
caption: 'DSB repair pathways in mammalian cells. Two-ended DSBs are preferably repaired
  by two major competing pathways: classical non-homologous end joining (c-NHEJ) and
  homologous recombination (HR). In addition, DSBs can be subjected to alternative
  end joining [a-EJ, also referred to as DNA polymerase theta-mediated end joining
  (TMEJ)] or single-strand annealing (SSA). BRCA1 and 53BP1 are placed at the center
  of DSB repair pathway choice. Whereas, chromatin recruitment of 53BP1 drives c-NHEJ,
  BRCA1 antagonizes 53BP1 to channel DSB repair into HR. (i) C-NHEJ begins with Ku70-Ku80
  (Ku) binding to DSB ends, followed by the recruitment of DNA-dependent protein kinase
  catalytic subunit (DNA-PKcs), forming the DNA-PK holoenzyme implicated in DNA synapsis.
  If necessary, DNA-PK coordinates limited processing of incompatible or chemically
  modified DNA ends by nucleases (e.g., Artemis) and other enzymes. The DNA ligase
  IV (LigIV)-XRCC4-XLF-PAXX complex executes the final ligation step. DNA end resection
  interferes with the default engagement of c-NHEJ by removing Ku from DNA ends, which
  is a critical step for initiating HR (ii). First, the MRE11-RAD50-NBS1 (MRN) complex
  senses the DSB and with the help of BRCA1 and CtIP promotes limited resection of
  the 5′ strand. Next, more extensive 5′-3′ resection by exonuclease 1 (EXO1), or
  by the Bloom''s syndrome (BLM) helicase together with the DNA2 nuclease, generates
  long 3′ ssDNA overhangs that become rapidly coated with the RPA heterotrimer. The
  BRCA1-PALB2-BRCA2 complex disassembles RAD51 heptamers and loads monomeric RAD51
  onto ssDNA, promoting RAD51 filament assembly. Template-dependent strand extension
  is followed by “synthesis-dependent strand annealing” (SDSA), resulting in a non-crossover
  gene conversion. Alternatively, capture of the second ssDNA by the D-loop forms
  a double Holliday junction intermediate, which can be resolved either as a non-crossover
  or as a crossover. (iii) SSA requires at least 20–25 base pairs (bp) of DNA sequence
  homology, which are typically found between repetitive elements (indicated as green
  boxes) in the genome. Subsequently, RAD52 promotes annealing of complementary ssDNA
  and leftover non-homologous flaps of the 3′ overhangs are cleaved by XPF-ERCC1.
  The factors that promote gap filling and ligation during SSA remain largely elusive.
  (iv) In contrast to SSA, a-EJ (or TMEJ) utilizes short microhomologies (MHs) of
  2–20 bp (indicated as red boxes) to join the two DNA strands. PARP1 has been implicated
  in promoting DNA end synapsis and recruiting the specialized DNA polymerase θ (Polθ)
  to DSBs. Polθ stabilizes MH-mediated joints between the two DNA ends serving as
  primers for fill-in synthesis. 3′ flaps extending from the joints are removed by
  XPF/ERCC1. Flap endonuclease 1 (FEN1) has recently been implicated in the removal
  of 5′ flaps generated by Polθ-mediated strand displacement, while the DNA Ligase
  III (LigIII)-XRCC1 complex is essential for the final ligation step. Inset, bottom
  left: DSB repair pathway-specific inhibitors. Inhibition of c-NHEJ has so far been
  mainly achieved by targeting DNA-PK using different small molecule inhibitors. Strategies
  to inhibit a-EJ and SSA focus on targeting their respective DNA annealing factors
  Polθ and RAD52, while the primary target to disrupt HR is RAD51 (see text for more
  details).'
papertitle: Harnessing DNA Double-Strand Break Repair for Cancer Treatment.
reftext: Anika Trenner, et al. Front Oncol. 2019;9:1388.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9269689
figid_alias: PMC6921965__F1
figtype: Figure
redirect_from: /figures/PMC6921965__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6921965__fonc-09-01388-g0001.html
  '@type': Dataset
  description: 'DSB repair pathways in mammalian cells. Two-ended DSBs are preferably
    repaired by two major competing pathways: classical non-homologous end joining
    (c-NHEJ) and homologous recombination (HR). In addition, DSBs can be subjected
    to alternative end joining [a-EJ, also referred to as DNA polymerase theta-mediated
    end joining (TMEJ)] or single-strand annealing (SSA). BRCA1 and 53BP1 are placed
    at the center of DSB repair pathway choice. Whereas, chromatin recruitment of
    53BP1 drives c-NHEJ, BRCA1 antagonizes 53BP1 to channel DSB repair into HR. (i)
    C-NHEJ begins with Ku70-Ku80 (Ku) binding to DSB ends, followed by the recruitment
    of DNA-dependent protein kinase catalytic subunit (DNA-PKcs), forming the DNA-PK
    holoenzyme implicated in DNA synapsis. If necessary, DNA-PK coordinates limited
    processing of incompatible or chemically modified DNA ends by nucleases (e.g.,
    Artemis) and other enzymes. The DNA ligase IV (LigIV)-XRCC4-XLF-PAXX complex executes
    the final ligation step. DNA end resection interferes with the default engagement
    of c-NHEJ by removing Ku from DNA ends, which is a critical step for initiating
    HR (ii). First, the MRE11-RAD50-NBS1 (MRN) complex senses the DSB and with the
    help of BRCA1 and CtIP promotes limited resection of the 5′ strand. Next, more
    extensive 5′-3′ resection by exonuclease 1 (EXO1), or by the Bloom''s syndrome
    (BLM) helicase together with the DNA2 nuclease, generates long 3′ ssDNA overhangs
    that become rapidly coated with the RPA heterotrimer. The BRCA1-PALB2-BRCA2 complex
    disassembles RAD51 heptamers and loads monomeric RAD51 onto ssDNA, promoting RAD51
    filament assembly. Template-dependent strand extension is followed by “synthesis-dependent
    strand annealing” (SDSA), resulting in a non-crossover gene conversion. Alternatively,
    capture of the second ssDNA by the D-loop forms a double Holliday junction intermediate,
    which can be resolved either as a non-crossover or as a crossover. (iii) SSA requires
    at least 20–25 base pairs (bp) of DNA sequence homology, which are typically found
    between repetitive elements (indicated as green boxes) in the genome. Subsequently,
    RAD52 promotes annealing of complementary ssDNA and leftover non-homologous flaps
    of the 3′ overhangs are cleaved by XPF-ERCC1. The factors that promote gap filling
    and ligation during SSA remain largely elusive. (iv) In contrast to SSA, a-EJ
    (or TMEJ) utilizes short microhomologies (MHs) of 2–20 bp (indicated as red boxes)
    to join the two DNA strands. PARP1 has been implicated in promoting DNA end synapsis
    and recruiting the specialized DNA polymerase θ (Polθ) to DSBs. Polθ stabilizes
    MH-mediated joints between the two DNA ends serving as primers for fill-in synthesis.
    3′ flaps extending from the joints are removed by XPF/ERCC1. Flap endonuclease
    1 (FEN1) has recently been implicated in the removal of 5′ flaps generated by
    Polθ-mediated strand displacement, while the DNA Ligase III (LigIII)-XRCC1 complex
    is essential for the final ligation step. Inset, bottom left: DSB repair pathway-specific
    inhibitors. Inhibition of c-NHEJ has so far been mainly achieved by targeting
    DNA-PK using different small molecule inhibitors. Strategies to inhibit a-EJ and
    SSA focus on targeting their respective DNA annealing factors Polθ and RAD52,
    while the primary target to disrupt HR is RAD51 (see text for more details).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PRKDC
  - BRCA1
  - MLIP
  - AK6
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - BRCA2
  - EXO1
  - NHEJ1
  - ALOX5AP
  - RASA1
  - RGS6
  - RPA1
  - RPA2
  - RPA3
  - POLE
  - FEN1
  - RAD51
  - RAD52
  - CALR
  - TRIM21
  - exo1
  - gap43
  - rpa2
  - pole
  - fen1
  - rad51
  - rad52
  - brca2
  - CIP
---
