---
figid: PMC6813780__nihms-867730-f0002
figtitle: Genomics of Apicomplexan Parasites
organisms:
- Plasmodium falciparum
- Venustaconcha ellipsiformis
- Toxoplasma gondii
- Trypanosoma brucei
- Plasmodium berghei
- Plasmodium vivax
- Cryptosporidium parvum
- Plasmodium yoelii
- Plasmodium chabaudi
- Alveolata
- Plasmodium knowlesi
- Neospora caninum
- Eimeria tenella
- Theileria parva
- Theileria annulata
- Cryptosporidium hominis
- Plasmodium falciparum HB3
- Cryptosporidium muris
- Plasmodium cynomolgi
- Hammondia hammondi
- Sarcocystis neurona
- Sarcocystis hominis
- Escherichia coli
- Mycobacterium tuberculosis
- Campylobacter jejuni
- Saccharomyces cerevisiae
- PX clade
- prion
- transposons
- Homo sapiens
- Mus musculus
- Bos taurus
pmcid: PMC6813780
filename: nihms-867730-f0002.jpg
figlink: /pmc/articles/PMC6813780/figure/F2/
number: F2
caption: The venn diagram represents the distribution of enzymes common to all apicomplexans,
  shared between various clades, and unique to each clade. The distribution of conserved
  pathways and pathways with clade-specific enzymes (≥2 in a clade) is represented
  as a network. The network represents KEGG pathways (nodes) connected by number of
  shared metabolites, with pathways belonging to a superclass (same border colours)
  grouped together wherever possible. Each node is represented as a circos chart depicting
  the number of unique enzymes present in each major clade. Conserved pathways are
  indicated as empty circles. The core of the network, enclosed in a dashed circle,
  mainly encompasses pathways from amino acid, carbohydrate, energy, and nucleotide
  metabolism, with quite a few conserved pathways, as well as several clade-specific
  pathways, especially from coccidia. The abbreviated pathway names are expanded for
  those in the core, and the pathways with unique enzymes, in the periphery.
papertitle: Genomics of Apicomplexan Parasites.
reftext: Lakshmipuram S. Swapna, et al. Crit Rev Biochem Mol Biol. 2017 Jun;52(3):254-273.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.3290693
figid_alias: PMC6813780__F2
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Bos taurus
redirect_from: /figures/PMC6813780__F2
ndex: 301a794b-df15-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6813780__nihms-867730-f0002.html
  '@type': Dataset
  description: The venn diagram represents the distribution of enzymes common to all
    apicomplexans, shared between various clades, and unique to each clade. The distribution
    of conserved pathways and pathways with clade-specific enzymes (≥2 in a clade)
    is represented as a network. The network represents KEGG pathways (nodes) connected
    by number of shared metabolites, with pathways belonging to a superclass (same
    border colours) grouped together wherever possible. Each node is represented as
    a circos chart depicting the number of unique enzymes present in each major clade.
    Conserved pathways are indicated as empty circles. The core of the network, enclosed
    in a dashed circle, mainly encompasses pathways from amino acid, carbohydrate,
    energy, and nucleotide metabolism, with quite a few conserved pathways, as well
    as several clade-specific pathways, especially from coccidia. The abbreviated
    pathway names are expanded for those in the core, and the pathways with unique
    enzymes, in the periphery.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tlr4
  - tip
  - Mettl8
  - Gnat2
  - Vim
  - fm
  - Gal
  - Ggta1
  - H2-Aa
  - Prl7d1
  - Prnp
  - Prh1
  - Prdx2
  - Hspa9
  - Ercc8
  - Il31ra
  - Fancd2
  - Jak3
  - Elovl6
  - Pym1
  - Fxyd6
  - Phpt1
  - Mgst1
  - Cym
  - Arp
  - Rere
  - stm
  - Cd5
  - Cacna1c
  - Dock2
  - Aqp2
  - Cpe
  - Grid2
  - Por
  - Cpb2
  - Aatf
  - Heph
  - Gabrb3
  - Cp2
  - IRF6
  - KAT5
  - METTL8
  - ITFG1
  - TIPRL
  - VIM
  - SDHD
  - GAL
  - GALP
  - FANCB
  - C4BPA
  - PRNP
  - PRDX2
  - ABCB6
  - CARD14
  - ERCC8
  - CSH1
  - HSPA9
  - IL31RA
  - BRCA2
  - FANCD2
  - PSEN1
  - FANCE
  - ELOVL6
  - SLC14A2
  - MORN4
  - POMC
  - PYM1
  - ASAH1
  - SOX3
  - PHPT1
  - SLCO6A1
  - GSTK1
  - GSTA1
  - GSTA2
  - GSTA3
  - GSTA4
  - GSTA5
  - GSTM1
  - GSTM2
  - GSTM3
  - GSTM4
  - GSTM5
  - GSTO1
  - GSTO2
  - GSTP1
  - GSTT1
  - GSTT2
  - GSTZ1
  - GSTT2B
  - CARD16
  - CRISP1
  - RERE
  - MANF
  - ARFRP1
  - SULT1A3
  - SULT1A4
  - SLX1A-SULT1A3
  - CPE
  - POR
  - TRB
  - TRBV20OR9-2
  - THRB
  - TXNRD2
  - HEPH
  - CYM
  - hypotaurine
  - Pyrimidine
  - Isoquinoline alkaloid
  - Ala
  - Amino sugar
  - Arginine
  - Asp
  - Benzoate
  - Caprolactam
  - Carbohydrate
  - Cys
  - Fatty acid
  - unsaturated fatty acids
  - Fructose
  - Galactose
  - Glucosinolate
  - Glutathione
  - Histidine
  - Inositol phosphate
  - Leu
  - Lysine
  - mannose
  - Methane
  - Met
  - Monobactam
  - Pentose phosphate
  - Phenylalanine
  - Phe
  - Proline
  - Purine
  - Pyruvate
  - Ser
  - Taurine
  - Terpenoid
  - Terpenoids
  - Thr
  - Tryptophan
  - Trp
  - Tyrosine
  - Tyr
  - Val
  - vancomycin
  - Glu
  - Citric acid
  - Phosphonate
---
