---
figid: PMC6546361__krnb-16-04-1582974-g004
figtitle: Applications of RNA-guided Cas9
organisms:
- NA
pmcid: PMC6546361
filename: krnb-16-04-1582974-g004.jpg
figlink: /pmc/articles/PMC6546361/figure/F0004/
number: F4
caption: 'Applications of RNA-guided Cas9. Cas9 (in blue) from S. pyogenes and the
  guide RNAs have been engineered and repurposed for a variety of biotechnological
  applications. The construction of the sgRNA (in red), which mimics the tracRNA:crRNA
  mature duplex, allows targeting of Cas9 to the desired sequence simply by changing
  the sequence that corresponds to the spacer (yellow). Using the wild type (WT) Cas9
  allows generating double strand breaks (DSBs) in virtually any desired sequence
  (as long as it is adjacent to the NGG PAM, in green). If, instead of the WT Cas9,
  a mutant in one of the nuclease domains is used (nickase Cas9 or nCas9), a nick
  will be generated on the target DNA. A D10A substitution inactivates the RuvC domain
  while a H840A mutation inactivates the HNH domain, in both cases without affecting
  its affinity for DNA (domain inactivation is represented by the stop sign on the
  scissors). Using paired nCas9 (two D10A nCas) to create two nicks instead of a DSB
  improves the specificity. Inactivating both nuclease domains generates a dead Cas9
  (dCas9), which specifically binds to the target DNA without affecting its integrity.
  Genome editing. DSBs or nicks produced by Cas9 at specific sites can be repaired
  by the non-homologous end-joining (NHEJ) or the homology-directed repair (HDR) systems.
  While the NHEJ pathway generates an insertion and/or deletion, HDR leads to the
  replacement of the specific sequence with a sequence from the DNA donor template.
  Targeting Cas9 to a gene of interest can be used to generate deletion mutants or
  to replace one gene with, for example, a different allele. Gene regulation. Catalytically
  inactive dCas9 can be fused to various effector domains like: 1. A transcriptional
  activation domain (green) or a transcriptional repression domain (red) to control
  the transcription of target genes or 2. epigenetic effectors (in purple) to regulate
  epigenetic modifications (e.g. DNA methylation, histone modifications) of the target
  loci. Imaging. dCas9 can also be fused with the Green Fluorescent Protein (GFP in
  green) to allow the visualization of specific loci in cells. Genome screening. Eukaryotic
  cells expressing Cas9 are first infected with a sgRNA library. Each cell will contain
  one sgRNA targeting a specific sequence (each color corresponds to a different spacer).
  The cells are then treated with a particular drug or compound and sgRNAs of the
  survivor cells are sequenced and compared to the ones of the initial population.
  The cells where a sgRNA targets an essential gene for survival, under the studied
  condition, are killed by the specific Cas9 DNA cleavage. This leads to the identification
  of genes that are indispensable for survival in the response to a certain challenge.'
papertitle: CRISPR-Cas in Streptococcus pyogenes.
reftext: Ana√Øs Le Rhun, et al. RNA Biol. 2019 Apr;16(4):380-389.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7747118
figid_alias: PMC6546361__F4
figtype: Figure
organisms_ner:
- Danio rerio
redirect_from: /figures/PMC6546361__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6546361__krnb-16-04-1582974-g004.html
  '@type': Dataset
  description: 'Applications of RNA-guided Cas9. Cas9 (in blue) from S. pyogenes and
    the guide RNAs have been engineered and repurposed for a variety of biotechnological
    applications. The construction of the sgRNA (in red), which mimics the tracRNA:crRNA
    mature duplex, allows targeting of Cas9 to the desired sequence simply by changing
    the sequence that corresponds to the spacer (yellow). Using the wild type (WT)
    Cas9 allows generating double strand breaks (DSBs) in virtually any desired sequence
    (as long as it is adjacent to the NGG PAM, in green). If, instead of the WT Cas9,
    a mutant in one of the nuclease domains is used (nickase Cas9 or nCas9), a nick
    will be generated on the target DNA. A D10A substitution inactivates the RuvC
    domain while a H840A mutation inactivates the HNH domain, in both cases without
    affecting its affinity for DNA (domain inactivation is represented by the stop
    sign on the scissors). Using paired nCas9 (two D10A nCas) to create two nicks
    instead of a DSB improves the specificity. Inactivating both nuclease domains
    generates a dead Cas9 (dCas9), which specifically binds to the target DNA without
    affecting its integrity. Genome editing. DSBs or nicks produced by Cas9 at specific
    sites can be repaired by the non-homologous end-joining (NHEJ) or the homology-directed
    repair (HDR) systems. While the NHEJ pathway generates an insertion and/or deletion,
    HDR leads to the replacement of the specific sequence with a sequence from the
    DNA donor template. Targeting Cas9 to a gene of interest can be used to generate
    deletion mutants or to replace one gene with, for example, a different allele.
    Gene regulation. Catalytically inactive dCas9 can be fused to various effector
    domains like: 1. A transcriptional activation domain (green) or a transcriptional
    repression domain (red) to control the transcription of target genes or 2. epigenetic
    effectors (in purple) to regulate epigenetic modifications (e.g. DNA methylation,
    histone modifications) of the target loci. Imaging. dCas9 can also be fused with
    the Green Fluorescent Protein (GFP in green) to allow the visualization of specific
    loci in cells. Genome screening. Eukaryotic cells expressing Cas9 are first infected
    with a sgRNA library. Each cell will contain one sgRNA targeting a specific sequence
    (each color corresponds to a different spacer). The cells are then treated with
    a particular drug or compound and sgRNAs of the survivor cells are sequenced and
    compared to the ones of the initial population. The cells where a sgRNA targets
    an essential gene for survival, under the studied condition, are killed by the
    specific Cas9 DNA cleavage. This leads to the identification of genes that are
    indispensable for survival in the response to a certain challenge.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pam
  - mycbp2
  - puf60b
  - hdr
---
