---
figid: PMC6941526__JB.00584-19-f0002
figtitle: Predicted cobamide biosynthetic and cobamide-dependent pathways in C
organisms:
- Clostridioides difficile
- Clostridioides difficile 630
- Homo sapiens
- human gut metagenome
organisms_ner:
- NA
pmcid: PMC6941526
filename: JB.00584-19-f0002.jpg
figlink: pmc/articles/PMC6941526/figure/F2/
number: F2
caption: 'Predicted cobamide biosynthetic and cobamide-dependent pathways in C. difficile
  630 Δerm. The enzymes of the cobamide biosynthesis pathway are shown in blue text,
  homologs of cobalamin-dependent enzymes are in magenta text, cobalamin-independent
  isozymes are in green text, and the transporter BtuFCD is shown as a black rectangle.
  Abbreviations: Cba, cobamide; Cbi, cobinamide; ALA, 5-aminolevulinic acid; rSAM,
  radical S-adenosylmethionine; NDPs, ribonucleoside diphosphates; NTPs, ribonucleoside
  triphosphates; dNDPs, deoxyribonucleoside diphosphates; dNTPs, deoxyribonucleoside
  triphosphates; DAP, 2,4-diaminopentanoate. Standard amino acid abbreviations are
  used. Enzymes: MetH, cobalamin-dependent methionine synthase; NrdEF, cobalamin-independent,
  aerobic (oxygen-requiring) (class I) ribonucleotide reductase (RNR); NrdDG, cobalamin-independent,
  anaerobic (oxygen-sensitive) (class III) RNR; NrdJ, cobalamin-dependent (class II)
  RNR; QueG, epoxyqueuosine reductase; EutBC, ethanolamine ammonia lyase; CFeSP, corrinoid
  iron-sulfur protein; OraSE, d-ornithine 4,5-aminomutase.'
papertitle: Flexible Cobamide Metabolism in Clostridioides (Clostridium) difficile
  630 Δerm.
reftext: Amanda N. Shelton, et al. J Bacteriol. 2020 Jan;202(2):e00584-19.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7659391
figid_alias: PMC6941526__F2
figtype: Figure
redirect_from: /figures/PMC6941526__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6941526__JB.00584-19-f0002.html
  '@type': Dataset
  description: 'Predicted cobamide biosynthetic and cobamide-dependent pathways in
    C. difficile 630 Δerm. The enzymes of the cobamide biosynthesis pathway are shown
    in blue text, homologs of cobalamin-dependent enzymes are in magenta text, cobalamin-independent
    isozymes are in green text, and the transporter BtuFCD is shown as a black rectangle.
    Abbreviations: Cba, cobamide; Cbi, cobinamide; ALA, 5-aminolevulinic acid; rSAM,
    radical S-adenosylmethionine; NDPs, ribonucleoside diphosphates; NTPs, ribonucleoside
    triphosphates; dNDPs, deoxyribonucleoside diphosphates; dNTPs, deoxyribonucleoside
    triphosphates; DAP, 2,4-diaminopentanoate. Standard amino acid abbreviations are
    used. Enzymes: MetH, cobalamin-dependent methionine synthase; NrdEF, cobalamin-independent,
    aerobic (oxygen-requiring) (class I) ribonucleotide reductase (RNR); NrdDG, cobalamin-independent,
    anaerobic (oxygen-sensitive) (class III) RNR; NrdJ, cobalamin-dependent (class
    II) RNR; QueG, epoxyqueuosine reductase; EutBC, ethanolamine ammonia lyase; CFeSP,
    corrinoid iron-sulfur protein; OraSE, d-ornithine 4,5-aminomutase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Cobamide
  - cobyrinate
  - Met
  - Carbon
  - Arg
  - L-citrulline
  - homocysteine
  - L-ornithine
  - ethanolamine
  - D-ornithine
  - acetyl-CoA
  - epoxyqueuosine
  - 5-amino
  - D-Ala
  - pentanoate
  - queuosine
---
