---
figid: PMC6498147__nihms-1020432-f0002
figtitle: Abnormal RNA stability in neurodegenerative disease
organisms:
- Hepatitis delta virus
- Human endogenous retroviruses
- Human endogenous retrovirus
- Human endogenous retrovirus K
- Saccharomyces cerevisiae
- Schizosaccharomyces pombe
- Neurospora crassa
- Homo sapiens
- Xenopus laevis
- Triportheus paranensis
- Macaca fascicularis
- Mus musculus
- Bos taurus
- Drosophila melanogaster
- prion
pmcid: PMC6498147
filename: nihms-1020432-f0002.jpg
figlink: /pmc/articles/PMC6498147/figure/F2/
number: F2
caption: 'Abnormal RNA stability in neurodegenerative disease. Here we compare how
  normal pathways (left column) are disrupted in disease (right column). RNA Sequestration:
  There is constant flux between pools of RNA transcripts that are actively being
  translated (the polysome), those sequestered in stress granules, and those associated
  with P-bodies. In disease states, increased stress granule formation or reduced
  stress granule dissociation disrupts the equilibrium, resulting in fewer transcripts
  undergoing translation. Repeat Expansions and RNA Foci: Transcripts containing repeat
  expansions form secondary structures such as hairpin loops and G-quadruplexes that
  are often stabilized in nuclear foci, which also sequester RNA-binding proteins
  (green circles). These transcripts also generate proteins via RAN translation that
  can disrupt membraneless organelles involved in RNA splicing and processing. RNA
  Transport and the Exosome: Mutations in THO, Gle1, and other components of the RNA
  export pathway result in nuclear RNA retention and degradation via the exosome complex.
  Mutations in exosome components can inhibit RNA turnover and further disrupt RNA
  homeostasis. Alternative Splicing: Mutations that disrupt splice sites, or dysfunction
  of splicing regulators such as TDP43, result in the inclusion of unannotated or
  “cryptic” exons (pink). These transcripts are often targeted for nonsense-mediated
  decay. Retrotransposons: These transposable elements insert themselves into the
  genome, often disrupting open reading frames or splice sites. The transcripts that
  are transcribed from these regions are often faulty, and are targeted for RNA decay.'
papertitle: RNA Degradation in Neurodegenerative Disease.
reftext: Kaitlin Weskamp, et al. Adv Neurobiol. 2018;20:103-142.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9510454
figid_alias: PMC6498147__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Mus musculus
redirect_from: /figures/PMC6498147__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6498147__nihms-1020432-f0002.html
  '@type': Dataset
  description: 'Abnormal RNA stability in neurodegenerative disease. Here we compare
    how normal pathways (left column) are disrupted in disease (right column). RNA
    Sequestration: There is constant flux between pools of RNA transcripts that are
    actively being translated (the polysome), those sequestered in stress granules,
    and those associated with P-bodies. In disease states, increased stress granule
    formation or reduced stress granule dissociation disrupts the equilibrium, resulting
    in fewer transcripts undergoing translation. Repeat Expansions and RNA Foci: Transcripts
    containing repeat expansions form secondary structures such as hairpin loops and
    G-quadruplexes that are often stabilized in nuclear foci, which also sequester
    RNA-binding proteins (green circles). These transcripts also generate proteins
    via RAN translation that can disrupt membraneless organelles involved in RNA splicing
    and processing. RNA Transport and the Exosome: Mutations in THO, Gle1, and other
    components of the RNA export pathway result in nuclear RNA retention and degradation
    via the exosome complex. Mutations in exosome components can inhibit RNA turnover
    and further disrupt RNA homeostasis. Alternative Splicing: Mutations that disrupt
    splice sites, or dysfunction of splicing regulators such as TDP43, result in the
    inclusion of unannotated or “cryptic” exons (pink). These transcripts are often
    targeted for nonsense-mediated decay. Retrotransposons: These transposable elements
    insert themselves into the genome, often disrupting open reading frames or splice
    sites. The transcripts that are transcribed from these regions are often faulty,
    and are targeted for RNA decay.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Ran
  - RanGAP
  - Gle1
  - Tehao
  - Decay
  - tum
  - mid
  - Fs(2)Ket
  - en
  - nmd
  - Xport-A
  - Rasl2-9
  - Ighmbp2
---
