---
figid: PMC6176597__pnas.1808567115fig05
figtitle: Application of the screened knockdown gene targets for the increased production
  of phenylpropanoids
organisms:
- Escherichia coli
- Corynebacterium glutamicum
- Pseudomonas putida
- Saccharomyces cerevisiae
- Homo sapiens
pmcid: PMC6176597
filename: pnas.1808567115fig05.jpg
figlink: /pmc/articles/PMC6176597/figure/fig05/
number: F5
caption: 'Application of the screened knockdown gene targets for the increased production
  of phenylpropanoids. (A) Effects of introducing each of the 14 synthetic sRNAs to
  the initial resveratrol producer (BTY5 harboring pTY13-HisTAL and pTacCDF-VvSTS-At4CL1m)
  on resveratrol production. The gene names written along the x axis represent the
  knockdown gene targets. CT, control (without sRNA), is noted as the gray bar. Six
  knockdown gene targets displayed as deeper red bars are selected for further experiments
  according to the set threshold (2.5-fold increase in titer compared with the control
  strain without sRNA). **P < 0.01, ***P < 0.001, determined by two-tailed Student’s
  t test. (B) The naringenin biosynthetic pathway from glycerol. A red X indicates
  that the corresponding gene was knocked out. A blue X (along with gene written in
  bold letters surrounded by yellow box) indicates that the corresponding gene was
  knocked down. Gray dotted lines indicate omitted pathways. Thick black arrows along
  with genes written in bold letters denote overexpressed metabolic fluxes. Red dotted
  lines (along with − signs in circles) indicate transcriptional repression. Black
  dotted lines (along with + signs in circles) indicate transcriptional activation.
  The upper box describes the introduced plasmids. The genetic construct shown at
  the lower left is on the chromosome. bla, β-lactamase gene; CDF, replication origin;
  ColE1, replication origin; kanR, kanamycin-resistance gene; p15A, replication origin;
  PBAD, arabinose-inducible promoter; PR, PR promoter; Ptac, tac promoter; Ptrc, trc
  promoter; rrnB, rrnBT1T2 terminator; spcR, spectinomycin-resistance gene; T1/TE,
  terminator. Each gene encodes the following: aroA, 3-phosphoshikimate 1-carboxyvinyltransferase;
  aroC, chorismate synthase; aroG, 3-deoxy-7-phosphoheptulonate synthase; aroL, shikimate
  kinase; aspC, aspartate aminotransferase; fabA, β-hydroxydecanoyl thioester dehydrase;
  fadR, negative regulator for fad regulon and positive regulator of fabA; fbr, feedback
  resistant; tyrA, chorismate mutase/prephenate dehydrogenase; tyrB, aromatic-amino
  acid transaminase; tyrP, tyrosine-specific transport protein; tyrR, transcriptional
  regulator of aroF, aroG and tyrA; xapA, xanthosine phosphorylase; xapB, xanthosine
  transporter; xapR, transcriptional activator of xapAB. (C) Effects of introducing
  each of the 14 synthetic sRNAs to the initial naringenin producer (BTY5 harboring
  pTY13-HisTAL and pTrcCDF-At4CL1m-AtCHI-PhCHS) on naringenin production. The gene
  names written along the x axis represent the knockdown gene targets. CT, control
  (without sRNA). Three knockdown gene targets displayed as deeper red bars are selected
  for further experiments according to the set threshold (15% increase in titer compared
  with the control strain without sRNA). *P < 0.05, determined by two-tailed Student’s
  t test. (D) Combinatorial double knockdown of the three gene targets selected from
  C for naringenin production. Control (without sRNA) is noted as the gray bar. The
  genes corresponding to the filled squares under each bar denote knocked down genes.
  All error bars, mean ± SD (n = 3). **P < 0.01, determined by two-tailed Student’s
  t test. Abbreviations: 1,3BPG, 1,3-bisphosphoglycerate; 2PG, 2-phosphoglycerate;
  3PG, 3-phosphoglycerate; AcCoA, acetyl-CoA; CHOR, chorismate; COU, p-coumaric acid;
  CouCoA, p-coumaroyl-CoA; DAHP, 3-deoxy-d-arabinoheptulosonate 7-phosphate; DHA,
  dihydroxyacetone; DHAP, dihydroxyacetone phosphate; E4P, d-erythrose 4-phosphate;
  FOR, formate; G3P, glyceraldehyde 3-phosphate; Gly, glycerol; Gly-3P, glycerol 3-phosphate;
  HPP, 4-hydroxyphenylpyruvate; MalCoA, malonyl-CoA; OAA, oxaloacetate; PEP, phosphoenolpyruvate;
  PPHN, prephenate; PYR, pyruvate; SHIK, shikimate; TYR, l-tyrosine.'
papertitle: Repurposing type III polyketide synthase as a malonyl-CoA biosensor for
  metabolic engineering in bacteria.
reftext: Dongsoo Yang, et al. Proc Natl Acad Sci U S A. 2018 Oct 2;115(40):9835-9844.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5195016
figid_alias: PMC6176597__F5
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
redirect_from: /figures/PMC6176597__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6176597__pnas.1808567115fig05.html
  '@type': Dataset
  description: 'Application of the screened knockdown gene targets for the increased
    production of phenylpropanoids. (A) Effects of introducing each of the 14 synthetic
    sRNAs to the initial resveratrol producer (BTY5 harboring pTY13-HisTAL and pTacCDF-VvSTS-At4CL1m)
    on resveratrol production. The gene names written along the x axis represent the
    knockdown gene targets. CT, control (without sRNA), is noted as the gray bar.
    Six knockdown gene targets displayed as deeper red bars are selected for further
    experiments according to the set threshold (2.5-fold increase in titer compared
    with the control strain without sRNA). **P < 0.01, ***P < 0.001, determined by
    two-tailed Student’s t test. (B) The naringenin biosynthetic pathway from glycerol.
    A red X indicates that the corresponding gene was knocked out. A blue X (along
    with gene written in bold letters surrounded by yellow box) indicates that the
    corresponding gene was knocked down. Gray dotted lines indicate omitted pathways.
    Thick black arrows along with genes written in bold letters denote overexpressed
    metabolic fluxes. Red dotted lines (along with − signs in circles) indicate transcriptional
    repression. Black dotted lines (along with + signs in circles) indicate transcriptional
    activation. The upper box describes the introduced plasmids. The genetic construct
    shown at the lower left is on the chromosome. bla, β-lactamase gene; CDF, replication
    origin; ColE1, replication origin; kanR, kanamycin-resistance gene; p15A, replication
    origin; PBAD, arabinose-inducible promoter; PR, PR promoter; Ptac, tac promoter;
    Ptrc, trc promoter; rrnB, rrnBT1T2 terminator; spcR, spectinomycin-resistance
    gene; T1/TE, terminator. Each gene encodes the following: aroA, 3-phosphoshikimate
    1-carboxyvinyltransferase; aroC, chorismate synthase; aroG, 3-deoxy-7-phosphoheptulonate
    synthase; aroL, shikimate kinase; aspC, aspartate aminotransferase; fabA, β-hydroxydecanoyl
    thioester dehydrase; fadR, negative regulator for fad regulon and positive regulator
    of fabA; fbr, feedback resistant; tyrA, chorismate mutase/prephenate dehydrogenase;
    tyrB, aromatic-amino acid transaminase; tyrP, tyrosine-specific transport protein;
    tyrR, transcriptional regulator of aroF, aroG and tyrA; xapA, xanthosine phosphorylase;
    xapB, xanthosine transporter; xapR, transcriptional activator of xapAB. (C) Effects
    of introducing each of the 14 synthetic sRNAs to the initial naringenin producer
    (BTY5 harboring pTY13-HisTAL and pTrcCDF-At4CL1m-AtCHI-PhCHS) on naringenin production.
    The gene names written along the x axis represent the knockdown gene targets.
    CT, control (without sRNA). Three knockdown gene targets displayed as deeper red
    bars are selected for further experiments according to the set threshold (15%
    increase in titer compared with the control strain without sRNA). *P < 0.05, determined
    by two-tailed Student’s t test. (D) Combinatorial double knockdown of the three
    gene targets selected from C for naringenin production. Control (without sRNA)
    is noted as the gray bar. The genes corresponding to the filled squares under
    each bar denote knocked down genes. All error bars, mean ± SD (n = 3). **P < 0.01,
    determined by two-tailed Student’s t test. Abbreviations: 1,3BPG, 1,3-bisphosphoglycerate;
    2PG, 2-phosphoglycerate; 3PG, 3-phosphoglycerate; AcCoA, acetyl-CoA; CHOR, chorismate;
    COU, p-coumaric acid; CouCoA, p-coumaroyl-CoA; DAHP, 3-deoxy-d-arabinoheptulosonate
    7-phosphate; DHA, dihydroxyacetone; DHAP, dihydroxyacetone phosphate; E4P, d-erythrose
    4-phosphate; FOR, formate; G3P, glyceraldehyde 3-phosphate; Gly, glycerol; Gly-3P,
    glycerol 3-phosphate; HPP, 4-hydroxyphenylpyruvate; MalCoA, malonyl-CoA; OAA,
    oxaloacetate; PEP, phosphoenolpyruvate; PPHN, prephenate; PYR, pyruvate; SHIK,
    shikimate; TYR, l-tyrosine.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - aroL
  - araC
  - micC
  - fmt
  - aroC
  - aroA
  - tyrA
  - aspC
  - tyrB
  - gyrB
  - fabA
  - fadR
  - tyrP
  - pabA
  - grcA
  - purB
  - fabH
  - fabF
  - araA
  - hycI
  - cytR
  - pyrF
  - tyrR
  - xapR
  - xapA
  - ytiD
  - IL6
  - LIF
  - CYP19A1
  - MICC
  - PAEP
  - PREP
  - PTPN22
  - SPTA1
  - HPP1
  - STK40
  - TYRP1
  - PURB
  - CCDC88C
  - FKBP10
  - SP7
  - DHA
  - DHAP
  - COU
  - TYR
  - TCA
  - Fatty acids
  - Tetraketide
  - Naringenin
  - chalcone
  - Tyrosine
  - Resveratrol
---
