---
figid: PMC6852473__EMI-21-3627-g005
figtitle: Comparison of central metabolic features of the Rifle comammox Nitrospira
organisms:
- Helicobacter pylori
- Thermus thermophilus
- Enterococcus hirae
- Nitrosomonas europaea
- Rhodococcus rhodochrous
- Nitrospira sp.
- Nitrospina gracilis
- Hydrogenobacter thermophilus TK-6
- Candidatus Nitrospira inopinata
- Nitrospira moscoviensis
- Nitrospira lenta
- Nitrosomonas nitrosa
- metagenome
- metagenomes
- activated sludge metagenome
- Homo sapiens
- Neoglyphea inopinata
pmcid: PMC6852473
filename: EMI-21-3627-g005.jpg
figlink: /pmc/articles/PMC6852473/figure/emi14691-fig-0005/
number: F5
caption: Comparison of central metabolic features of the Rifle comammox Nitrospira.
  AMO, ammonia monooxygenase; CAT, catalase; CLD, chlorite dismutase; FDH, formate
  dehydrogenase; HAO, hydroxylamine dehydrogenase; HURM, hydroxylamine‐ubiquinone
  reaction module; HYD, group 3b bifunctional NAD(P) hydrogenase; Mnh, multisubunit
  Na+/H+ antiporter; NHase, nitrile hydratase; NirK, Cu‐dependent nitrite reductase;
  SulP, sulfate permease. Asterisks indicate missing genes that potentially result
  from low‐quality assemblies. The question mark indicates that the exact pathway
  of nitrite formation from NO is uncertain.
papertitle: Metagenomic recovery of two distinct comammox Nitrospira from the terrestrial
  subsurface.
reftext: Lianna Poghosyan, et al. Environ Microbiol. 2019 Oct;21(10):3627-3637.
year: '2019'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8533842
figid_alias: PMC6852473__F5
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC6852473__F5
ndex: 5246d57e-df21-11ea-99da-0ac135e8bacf
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6852473__EMI-21-3627-g005.html
  '@type': Dataset
  description: Comparison of central metabolic features of the Rifle comammox Nitrospira.
    AMO, ammonia monooxygenase; CAT, catalase; CLD, chlorite dismutase; FDH, formate
    dehydrogenase; HAO, hydroxylamine dehydrogenase; HURM, hydroxylamine‐ubiquinone
    reaction module; HYD, group 3b bifunctional NAD(P) hydrogenase; Mnh, multisubunit
    Na+/H+ antiporter; NHase, nitrile hydratase; NirK, Cu‐dependent nitrite reductase;
    SulP, sulfate permease. Asterisks indicate missing genes that potentially result
    from low‐quality assemblies. The question mark indicates that the exact pathway
    of nitrite formation from NO is uncertain.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GSN
  - DSTN
  - ATP8A2
  - IK
  - DYRK3
  - TRPM6
  - RCN1
  - UBR5
  - DECR1
  - ADH5
  - ALDH1L1
  - CAT
  - CRAT
  - GLYAT
  - SOD1
  - SOD2
  - SOD3
  - WDTC1
  - VV
  - SLC26A3
  - NONO
  - ABCB6
  - LRSAM1
  - HAAO
  - NAD NADH
  - H
  - Succinate
  - Fumarate
  - TCA
  - Pentose phosphate
  - NAD(P)H
  - NAD
  - NAD HCOOH
  - Urea
  - Fe
  - NADH
  - CO
  - Fe-Mn
  - ADP
  - ATP
  - SulP
  - Cu-Zn
  - Na
  - Formate
  - Quinol
  - Quinone
  - Nitrospira RCB
---
