---
figid: PMC6591730__nihms-1037106-f0003
figtitle: Causes and Consequences of Enzyme-Induced Cytosine Deamination
organisms:
- NA
pmcid: PMC6591730
filename: nihms-1037106-f0003.jpg
figlink: /pmc/articles/PMC6591730/figure/F3/
number: F3
caption: (A) ssDNA exposed during replication fork progression is a substrate for
  cytosine deamination. Active replication forks expose ssDNA on the lagging strand.
  Stalled replication forks expose ssDNA on both the leading and lagging strand, which
  activates ATR to stop replication and stabilize the fork through controlled fork
  reversal. Endonuclease cleavage causes replication fork collapse into one-ended
  DNA DSBs, which must be resected before repair by HR.(B) ssDNA exposed during transcription
  is a substrate for cytosine deamination. RNA polymerase (RNAP) exposes ssDNA while
  synthesizing nascent RNA (red) at the transcription bubble and behind the RNAP as
  a result of negative supercoiling (left). Transcription stress induces the formation
  of R-loops (middle), which exposes long stretches of ssDNA on the non-template strand.
  R-loops may be resolved in DSBs through cleavage by TC-NER components XPF and XPG
  (right).(C) Mechanism of enzyme-induced somatic hypermutation (SHM). Cytosine deamination
  generates a U:G mismatch, which is recognized by uracil glycosylase UNG to generate
  an abasic site opposite the G. Apurinic/apyrimidinic endodeoxyribonuclease (APE)
  excises the abasic nucleotide which is repaired by POLb. If the abasic site is replicated
  over prior to repair, this can give rise to transition or transversion mutations
  (N). If the U:G mismatch is replicated prior to repair, C>T substitution occurs.
  Mutations at neighboring A:T base pairs can occur if the U:G pair is recognized
  by the MMR pathway and repaired by error prone polymerase eta (Polη).(D) Mechanisms
  of enzyme-induced rearrangements and class-switch recombination (CSR). If cytosine
  deamination occurs nearby on opposite strands, repair through BER or MMR will form
  DNA DSBs, leading to chromosomal rearrangements or CSR.
papertitle: Endogenous DNA Damage as a Source of Genomic Instability in Cancer.
reftext: Anthony Tubbs, et al. Cell. 2017 Feb 9;168(4):644-656.
year: '2017'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9677986
figid_alias: PMC6591730__F3
figtype: Figure
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
redirect_from: /figures/PMC6591730__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6591730__nihms-1037106-f0003.html
  '@type': Dataset
  description: (A) ssDNA exposed during replication fork progression is a substrate
    for cytosine deamination. Active replication forks expose ssDNA on the lagging
    strand. Stalled replication forks expose ssDNA on both the leading and lagging
    strand, which activates ATR to stop replication and stabilize the fork through
    controlled fork reversal. Endonuclease cleavage causes replication fork collapse
    into one-ended DNA DSBs, which must be resected before repair by HR.(B) ssDNA
    exposed during transcription is a substrate for cytosine deamination. RNA polymerase
    (RNAP) exposes ssDNA while synthesizing nascent RNA (red) at the transcription
    bubble and behind the RNAP as a result of negative supercoiling (left). Transcription
    stress induces the formation of R-loops (middle), which exposes long stretches
    of ssDNA on the non-template strand. R-loops may be resolved in DSBs through cleavage
    by TC-NER components XPF and XPG (right).(C) Mechanism of enzyme-induced somatic
    hypermutation (SHM). Cytosine deamination generates a U:G mismatch, which is recognized
    by uracil glycosylase UNG to generate an abasic site opposite the G. Apurinic/apyrimidinic
    endodeoxyribonuclease (APE) excises the abasic nucleotide which is repaired by
    POLb. If the abasic site is replicated over prior to repair, this can give rise
    to transition or transversion mutations (N). If the U:G mismatch is replicated
    prior to repair, C>T substitution occurs. Mutations at neighboring A:T base pairs
    can occur if the U:G pair is recognized by the MMR pathway and repaired by error
    prone polymerase eta (Polη).(D) Mechanisms of enzyme-induced rearrangements and
    class-switch recombination (CSR). If cytosine deamination occurs nearby on opposite
    strands, repair through BER or MMR will form DNA DSBs, leading to chromosomal
    rearrangements or CSR.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATR
  - ANTXR1
  - MMAB
  - SERPINA2
  - ERCC4
  - ERCC5
  - GRHL3
  - POLB
  - POLG2
  - APEX1
  - CCDC88A
  - UNG
  - MSH2
  - MRC1
  - MSH6
  - EXO1
  - POLN
  - mei-41
  - put
  - RYa-R
  - dsb
  - mus201
  - som
  - RpLP0
  - ung
  - spel1
  - tin
  - ber
  - Msh6
  - ct
  - cn
  - Cytosine
---
