---
figid: PMC6226483__41598_2018_34730_Fig4_HTML
figtitle: The complexity of the carbon assimilation metabolic pathway of cassava based
  on reaction-enzyme association
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Solanum tuberosum
- Triticum aestivum
- PX clade
- Candida albicans
- Yarrowia lipolytica
- Manihot esculenta
- Aspergillus oryzae
- Brassica rapa
- Ricinus communis
- Rhizopus arrhizus
- Jatropha curcas
- Colletotrichum gloeosporioides
- Hevea brasiliensis
- Brassica rapa subsp. rapa
- Mucor circinelloides
- Eremothecium gossypii
- Escherichia coli
- Halobacterium salinarum
- Haloarcula marismortui
- Natronomonas pharaonis
- Thermoproteus tenax
- Haloquadratum walsbyi
- Alligator sinensis
- Homo sapiens
pmcid: PMC6226483
filename: 41598_2018_34730_Fig4_HTML.jpg
figlink: /pmc/articles/PMC6226483/figure/Fig4/
number: F4
caption: The complexity of the carbon assimilation metabolic pathway of cassava based
  on reaction-enzyme association. (a) The schematic of relationship among annotated
  enzymatic proteins (P), enzymes (EC), enzymatic reactions (RXN), and metabolites
  (MET); (b) Examples of the reaction-enzyme association in MeRecon; (c) The relationship
  between enzymes (EC numbers) and reactions in MeRecon, represented as nodes and
  edges. Nodes consist of enzymes (circles) and enzymatic reactions (rectangles),
  and the edges between the nodes represent reaction-enzyme association of each metabolic
  reaction. Size of enzyme nodes represents node degree, and the number of enzymatic
  reactions catalyzed by that enzyme. AMI - amino acid biosynthesis, CAL - Calvin
  cycle, CEL - cell wall biosynthesis, FAT - fatty acid biosynthesis, NUC - nucleotide
  biosynthesis, RES – respiration, STA - starch biosynthesis, and SUC - sucrose biosynthesis.
papertitle: Unlocking conserved and diverged metabolic characteristics in cassava
  carbon assimilation via comparative genomics approach.
reftext: Wanatsanan Siriwat, et al. Sci Rep. 2018;8:16593.
year: '2018'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7659873
figid_alias: PMC6226483__F4
figtype: Figure
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
redirect_from: /figures/PMC6226483__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6226483__41598_2018_34730_Fig4_HTML.html
  '@type': Dataset
  description: The complexity of the carbon assimilation metabolic pathway of cassava
    based on reaction-enzyme association. (a) The schematic of relationship among
    annotated enzymatic proteins (P), enzymes (EC), enzymatic reactions (RXN), and
    metabolites (MET); (b) Examples of the reaction-enzyme association in MeRecon;
    (c) The relationship between enzymes (EC numbers) and reactions in MeRecon, represented
    as nodes and edges. Nodes consist of enzymes (circles) and enzymatic reactions
    (rectangles), and the edges between the nodes represent reaction-enzyme association
    of each metabolic reaction. Size of enzyme nodes represents node degree, and the
    number of enzymatic reactions catalyzed by that enzyme. AMI - amino acid biosynthesis,
    CAL - Calvin cycle, CEL - cell wall biosynthesis, FAT - fatty acid biosynthesis,
    NUC - nucleotide biosynthesis, RES – respiration, STA - starch biosynthesis, and
    SUC - sucrose biosynthesis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - metB
  - metC
  - metQ
  - metI
  - metN
  - manY
  - metE
  - lptD
  - RO0430
  - MET B
  - F01858
  - O0000
  - O000
  - AMI O CAL OCEL OFAT ONUC ORES OSTA Osuc O
  - O0000 00000 O000 AMI
---
