---
figid: PMC7146499__nutrients-12-00670-g002
figtitle: Dehydration alters hippocampal transcriptional networks
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Danio rerio
pmcid: PMC7146499
filename: nutrients-12-00670-g002.jpg
figlink: pmc/articles/PMC7146499/figure/nutrients-12-00670-f002/
number: F2
caption: Dehydration alters hippocampal transcriptional networks. A hippocampal transcriptional
  network analysis was performed, comprising next-generation RNA sequencing and bioinformatic
  analyses. (a) Genes that were differentially regulated between the control (CON)
  and dehydrated (DEH) mice were classified into functional categories via an Ingenuity
  Pathway Analysis (IPA). Significantly changed biological functions are indicated
  by p-values calculated using a Fisher’s exact test, and the number of distinctively
  modulated molecules in each functional category is indicated. (b–d) Transcriptional
  networks related to altered glucose metabolism, synaptogenic cell-adhesion, and
  synaptic plasticity were differentially regulated in response to long-term dehydration.
  (e) Genes associated with long-term potentiation (LTP) signaling transduction were
  differentially up- (red) or downregulated (green) in the DEH compared to the CON
  mice.
papertitle: Dehydration Impairs Physical Growth and Cognitive Development in Young
  Mice.
reftext: Chong-Su Kim, et al. Nutrients. 2020 Mar;12(3):670.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5780229
figid_alias: PMC7146499__F2
figtype: Figure
redirect_from: /figures/PMC7146499__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7146499__nutrients-12-00670-g002.html
  '@type': Dataset
  description: Dehydration alters hippocampal transcriptional networks. A hippocampal
    transcriptional network analysis was performed, comprising next-generation RNA
    sequencing and bioinformatic analyses. (a) Genes that were differentially regulated
    between the control (CON) and dehydrated (DEH) mice were classified into functional
    categories via an Ingenuity Pathway Analysis (IPA). Significantly changed biological
    functions are indicated by p-values calculated using a Fisher’s exact test, and
    the number of distinctively modulated molecules in each functional category is
    indicated. (b–d) Transcriptional networks related to altered glucose metabolism,
    synaptogenic cell-adhesion, and synaptic plasticity were differentially regulated
    in response to long-term dehydration. (e) Genes associated with long-term potentiation
    (LTP) signaling transduction were differentially up- (red) or downregulated (green)
    in the DEH compared to the CON mice.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BRD2
  - GRIA2
  - GRIN1
  - GRIN2A
  - GRIN2B
  - GRIN2C
  - GRIN2D
  - GRIN3A
  - GRIN3B
  - ATP8A2
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - MTMR11
  - LUC7L3
  - CAMK2B
  - CAMK2A
  - CAMK2D
  - CAMK2G
  - CAMK4
  - HK1
  - KCNA4
  - KLK1
  - HOOK1
  - TAC4
  - GALE
  - LDHA
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - CASR
  - CXADR
  - PRKAR1A
  - SPG7
  - NR1I3
  - TRIM13
  - CXADRP1
  - GAA
  - RELN
  - PFKL
  - PYGM
  - ITIH1
  - MBP
  - MBL2
  - PRG2
  - ZFPM1
  - DUSP1
  - DUSP12
  - ARPP21
  - PER2
  - HSD17B12
  - DNAJB5
  - CSRNP1
  - TH
  - GAD2
  - SLC50A1
  - BDNF
  - BDNF-AS
  - LYPD1
  - SIAH2
  - NR4A1
  - BHLHE40
  - FSCN1
  - DNAJC21
  - P4HA1
  - GPD2
  - ARC
  - NOL3
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - MIDN
  - SYNGAP1
  - DLGAP1
  - MAG
  - DLG1
  - IRAK1
  - JUNB
  - RNASEH2A
  - APOE
  - VEGFA
  - CYP51A1
  - SDCBP
  - SRC
  - FGR
  - FYN
  - YES1
  - AVP
  - IFNAR1
  - NLRP3
  - DLG4
  - LLGL1
  - DLG2
  - MPP2
  - CASK
  - BTG2
  - EGR4
  - PICK1
  - NRXN1
  - PPP1R9A
  - CIART
  - SIK1
  - AFDN
  - AFD1
  - NLGN3
  - KALRN
  - Brd2
  - Scn11a
  - Gria2
  - Grin1
  - Mela
  - Sgip1
  - Hspg2
  - Mtmr11
  - Camk4
  - Hk1
  - Gale
  - Ldha
  - Creb1
  - Nr1i3
  - Casr
  - Cxadr
  - Rcvrn
  - Trim13
  - Arr3
  - Gaa
  - Reln
  - Pfkl
  - Pygm
  - Mbp
  - Prg2
  - Zfpm1
  - Dusp1
  - Arpp21
  - Per2
  - Hsd17b12
  - Dnajb5
  - Csrnp1
  - Th
  - Gad2
  - Slc50a1
  - Ugt2b5
  - Bdnf
  - Lypd1
  - Siah2
  - Nr4a1
  - Bhlhe40
  - Fscn1
  - Dnajc21
  - P4ha1
  - Gpd2
  - G6pd2
  - Arc
  - Nol3
  - Fos
  - Midn
  - Syngap1
  - Dlgap1
  - Mag
  - Mtag2
  - Crem
  - Dlg1
  - Irak1
  - Junb
  - Apoe
  - Vegfa
  - Cyp51
  - Sdcbp
  - Src
  - Avp
  - Dlg4
  - Dlg2
  - Mpp2
  - Cask
  - Btg2
  - Egr4
  - Pick1
  - Nrxn1
  - Ppp1r9a
  - Ciart
  - Sik1
  - Afdn
  - Nlgn3
  - Kalrn
  - Nat1
  - Camk2d
  - Errfi1
  - camk4
  - hk1
  - gale
  - ldha
  - cxadr
  - carmil1
  - gaa
  - reln
  - mbpa
  - zfpm1
  - dusp1
  - arpp21
  - per2
  - dnajb5
  - th
  - gad2
  - slc50a1
  - bdnf
  - nr4a1
  - bhlhe40
  - fscn1a
  - dnajc21
  - gpd2
  - fosab
  - midn
  - dlgap1a
  - dlgap1b
  - mag
  - creb1a
  - dlg1a
  - irak1
  - junba
  - apoeb
  - vegfaa
  - sdcbp
  - sdcbp2
  - src
  - avp
  - dlg4
  - dlg4b
  - dlg2
  - caska
  - btg2
  - egr4
  - pick1
  - ppp1r9a
  - ciarta
  - sik1
  - Ca2+
  - ATP
  - C-Ra
  - Glucose
  - Neurological disease
  - Neurodegeneration
  - Cognitive impairment
  - SIAH2
---
