---
figid: PMC7080263__pone.0229513.g005
figtitle: RNA-seq transcriptomic analysis reveals genes mediating salt tolerance through
  rapid triggering of ion transporters in a mutant barley
organisms:
- Arabidopsis thaliana
- Nicotiana tabacum
- Oryza sativa
- Zea mays
- Glycine max
- Hordeum vulgare
- Cucumis sativus
- Magnoliopsida
- Pisum sativum
- Saccharum officinarum
- Tracheophyta
- Marchantiophyta
- Mimosa pudica
- Hordeum vulgare subsp. spontaneum
- Sporobolus alterniflorus
- Picea asperata
- Hordeum marinum
- Plantago coronopus
- Allenrolfea occidentalis
pmcid: PMC7080263
filename: pone.0229513.g005.jpg
figlink: pmc/articles/PMC7080263/figure/pone.0229513.g005/
number: F5
caption: It represents genes involved in salinity-stress signaling pathways and tolerant
  mechanisms in the salt-tolerant mutant genotype in response to 300 mM salt (six
  hours of exposure) in comparison to the wild-type genotype. After entrance of Na+
  to cell cytoplasm, the activity of RBOH NADPH oxidase was increased to promote ROS
  formation and cytosolic Ca2+ content []. Ca2+ activated SOS3 and then SOS2, leading
  to SOS1-mediated Na+ extrusion []. On the other hand, NHX1 was activated by SOS3/SOS2
  complex via HVA and HVP pumps, providing the driving force to sequester Na+ from
  cytosol to vacuole []. Moreover, TPK1/KCO1 is an important vacuolar permeable channel
  for K+ extrusion from vacuole to cytoplasm contributing to homeostasis regulation.
  Salt stress magnifies the activity of protein kinases in MAP kinase pathway to produce
  ethylene []. Ethylene can promote K+ retention through K+ channels, such as HAK
  and HKT to regulate Na+ and K+ homeostasis [].
papertitle: The RNA-seq transcriptomic analysis reveals genes mediating salt tolerance
  through rapid triggering of ion transporters in a mutant barley.
reftext: Sareh Yousefirad, et al. PLoS One. 2020;15(3):e0229513.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9430409
figid_alias: PMC7080263__F5
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
- Glycine max
redirect_from: /figures/PMC7080263__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7080263__pone.0229513.g005.html
  '@type': Dataset
  description: It represents genes involved in salinity-stress signaling pathways
    and tolerant mechanisms in the salt-tolerant mutant genotype in response to 300
    mM salt (six hours of exposure) in comparison to the wild-type genotype. After
    entrance of Na+ to cell cytoplasm, the activity of RBOH NADPH oxidase was increased
    to promote ROS formation and cytosolic Ca2+ content []. Ca2+ activated SOS3 and
    then SOS2, leading to SOS1-mediated Na+ extrusion []. On the other hand, NHX1
    was activated by SOS3/SOS2 complex via HVA and HVP pumps, providing the driving
    force to sequester Na+ from cytosol to vacuole []. Moreover, TPK1/KCO1 is an important
    vacuolar permeable channel for K+ extrusion from vacuole to cytoplasm contributing
    to homeostasis regulation. Salt stress magnifies the activity of protein kinases
    in MAP kinase pathway to produce ethylene []. Ethylene can promote K+ retention
    through K+ channels, such as HAK and HKT to regulate Na+ and K+ homeostasis [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SOS2
  - SOS3
  - NHX1
  - ERF13
  - AP2
  - TPK1
  - KCO1
  - AGO7
  - WRKY14
  - CDPK
  - HAK
  - NADPH
  - NADP+H
  - Na
  - Ethylene
  - NAC
---
