---
figid: PMC7356990__microorganisms-08-00866-g001
figtitle: N-methylanthranilate (NMA) biosynthesis (A) embedded into aromatic amino
  acid metabolism of engineered C
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC7356990
filename: microorganisms-08-00866-g001.jpg
figlink: pmc/articles/PMC7356990/figure/microorganisms-08-00866-f001/
number: F1
caption: Schematic representation of N-methylanthranilate (NMA) biosynthesis (A) embedded
  into aromatic amino acid metabolism of engineered C. glutamicum (B). Continuous
  arrows indicate single reactions, dashed arrows indicate multiple reactions. Green
  arrows and gene names indicate genome-based overexpression, yellow arrows and gene
  names indicate vector-based expression, crossed arrows and red gene names indicate
  gene deletion. (A) N-methylation of anthranilate by N-methylanthranilate transferase
  (ANMT) from R. graveolens under consumption of S-adenosylmethionine (SAM). The SAM
  regeneration cycle is depicted with overexpression of sahH, S-adenosylhomocysteine
  hydrolase. SAH, S-adenosylhomocysteine; HCYS, l-homocysteine; MET, l-methionine;
  metE/metH, methionine synthase; metK, methionine adenosyltransferase. (B) Strain
  engineering towards production of NMA. Grey sugR indicates reversion of deleted
  sugR back to wild type sugR. PEP, phosphoenolpyruvate; TCA, tricarboxylic acid;
  PPP, pentose phosphate pathway; E4P, erythrose-4-phosphate; DAHP, 3-deoxy-d-arabinoheptulosonate-7-phosphate;
  3DHQ, 3-dehydroquinate; 3DHS, 3-dehydroshikimic acid; PCA, protocatechuic acid;
  iolR, transcriptional regulator; sugR, transcriptional regulator; ppc, phosphoenolpyruvate
  carboxylase; ldhA, lactate dehydrogenase; tkt, transketolase; aroF, DAHP synthase;
  aroGFBR, feedback-resistant DAHP synthase from Escherichia coli; aroB, 3-dehydroquinate
  synthase; qsuC, 3-dehydroquinate dehydratase; qsuB, 3-dehydroshikimate dehydratase;
  qsuD, shikimate dehydrogenase; aroE, shikimate dehydrogenase; qsuA, putative shikimate
  importer; aroK, shikimate kinase; aroA, 5-enolpyruvylshikimate-3-phosphate synthase;
  aroC, chorismate synthase; csm, chorismate mutase; trpEFBR, feedback-resistant anthranilate
  synthase from E. coli.
papertitle: Fermentative N-Methylanthranilate Production by Engineered Corynebacterium
  glutamicum.
reftext: Tatjana Walter, et al. Microorganisms. 2020 Jun;8(6):866.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9467492
figid_alias: PMC7356990__F1
figtype: Figure
redirect_from: /figures/PMC7356990__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7356990__microorganisms-08-00866-g001.html
  '@type': Dataset
  description: Schematic representation of N-methylanthranilate (NMA) biosynthesis
    (A) embedded into aromatic amino acid metabolism of engineered C. glutamicum (B).
    Continuous arrows indicate single reactions, dashed arrows indicate multiple reactions.
    Green arrows and gene names indicate genome-based overexpression, yellow arrows
    and gene names indicate vector-based expression, crossed arrows and red gene names
    indicate gene deletion. (A) N-methylation of anthranilate by N-methylanthranilate
    transferase (ANMT) from R. graveolens under consumption of S-adenosylmethionine
    (SAM). The SAM regeneration cycle is depicted with overexpression of sahH, S-adenosylhomocysteine
    hydrolase. SAH, S-adenosylhomocysteine; HCYS, l-homocysteine; MET, l-methionine;
    metE/metH, methionine synthase; metK, methionine adenosyltransferase. (B) Strain
    engineering towards production of NMA. Grey sugR indicates reversion of deleted
    sugR back to wild type sugR. PEP, phosphoenolpyruvate; TCA, tricarboxylic acid;
    PPP, pentose phosphate pathway; E4P, erythrose-4-phosphate; DAHP, 3-deoxy-d-arabinoheptulosonate-7-phosphate;
    3DHQ, 3-dehydroquinate; 3DHS, 3-dehydroshikimic acid; PCA, protocatechuic acid;
    iolR, transcriptional regulator; sugR, transcriptional regulator; ppc, phosphoenolpyruvate
    carboxylase; ldhA, lactate dehydrogenase; tkt, transketolase; aroF, DAHP synthase;
    aroGFBR, feedback-resistant DAHP synthase from Escherichia coli; aroB, 3-dehydroquinate
    synthase; qsuC, 3-dehydroquinate dehydratase; qsuB, 3-dehydroshikimate dehydratase;
    qsuD, shikimate dehydrogenase; aroE, shikimate dehydrogenase; qsuA, putative shikimate
    importer; aroK, shikimate kinase; aroA, 5-enolpyruvylshikimate-3-phosphate synthase;
    aroC, chorismate synthase; csm, chorismate mutase; trpEFBR, feedback-resistant
    anthranilate synthase from E. coli.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - metK
  - metH
  - metE
  - aroF
  - aroB
  - tktA
  - aroE
  - aroK
  - aroA
  - ppc
  - aroC
  - crp
  - yciE
  - Rg
  - Anthranilate
  - N-methylanthranilate
  - Sugars
  - Pyruvate
  - Lactate
  - Oxaloacetate
  - Prephenate
  - Tryptophan
  - TCA
  - Tyrosine
  - Phenylalanine
  - SAH
---
