---
figid: PMC7784690__41396_2020_722_Fig5_HTML
figtitle: Termite mounds contain soil-derived methanotroph communities kinetically
  adapted to elevated methane concentrations
organisms:
- Erebia epipsodea
- Termitoidae
- Macrotermes falciger
- Tumulitermes pastinator
- Microcerotermes nervosus
- Macrognathotermes sunteri
- Methylosinus trichosporium OB3b
- Methylacidiphilum fumariolicum SolV
- Methylocapsa acidiphila
- Methylocystis sp. SC2
- Methylocapsa acidiphila B2
- metagenome
- metagenomes
- environmental samples
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC7784690
filename: 41396_2020_722_Fig5_HTML.jpg
figlink: pmc/articles/PMC7784690/figure/Fig5/
number: F5
caption: H4MPT tetrahydromethanopterin pathway, pMMO particulate methane monooxygenase,
  MDH methanol dehydrogenase, FDH formate dehydrogenase, NiR copper-containing nitrite
  reductase, I NADH dehydrogenase (complex I), II succinate dehydrogenase (complex
  II), IV cytochrome aa3 oxidase and cytochrome bo3 oxidase (complex IV). Genes detected
  are shown in italics. Dashed black lines indicate diffusion. Dashed grey lines indicate
  reactions performed by three enzymes that were not detected, methanol dehydrogenase,
  hydroxylamine dehydrogenase and serine hydroxymethyltransferase. It is assumed that
  these enzymes are encoded by the bacterium, but were not detected given the moderate
  predicted completeness of the MAG (73%).
papertitle: Termite mounds contain soil-derived methanotroph communities kinetically
  adapted to elevated methane concentrations.
reftext: Eleonora Chiri, et al. ISME J. 2020 Nov;14(11):2715-2731.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8841308
figid_alias: PMC7784690__F5
figtype: Figure
redirect_from: /figures/PMC7784690__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7784690__41396_2020_722_Fig5_HTML.html
  '@type': Dataset
  description: H4MPT tetrahydromethanopterin pathway, pMMO particulate methane monooxygenase,
    MDH methanol dehydrogenase, FDH formate dehydrogenase, NiR copper-containing nitrite
    reductase, I NADH dehydrogenase (complex I), II succinate dehydrogenase (complex
    II), IV cytochrome aa3 oxidase and cytochrome bo3 oxidase (complex IV). Genes
    detected are shown in italics. Dashed black lines indicate diffusion. Dashed grey
    lines indicate reactions performed by three enzymes that were not detected, methanol
    dehydrogenase, hydroxylamine dehydrogenase and serine hydroxymethyltransferase.
    It is assumed that these enzymes are encoded by the bacterium, but were not detected
    given the moderate predicted completeness of the MAG (73%).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NOC2L
  - PMCH
  - DECR1
  - F11
  - PTCRA
  - TNK2
  - ADH5
  - ALDH1L1
  - AGA
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - ATP8A2
  - WDTC1
  - RAB8A
  - MAP2K1
  - ME1
  - PCDHB16
  - mch
  - Ack
  - Fdh
  - CycE
  - cyc
  - tRNA:Arg-TCT-3-1
  - tRNA:Arg-TCT-2-1
  - tRNA:Arg-TCT-1-1
  - Eno
  - ATPsynbeta
  - Atpalpha
  - adp
  - mel
  - Men
  - l(1)16Fa
  - mei-W68
  - mah
  - CH
  - OH
  - Nitrite
  - NADH
  - H
  - NAD
  - Formaldehyde
  - NADP
  - NADPH
  - Formate
  - Serine
  - Acetate
  - Acetyl-CoA
  - ATP
  - ADP
  - P
  - Succinate Fumarate
  - MOH
---
