---
figid: PMC7454314__bbaa173f1
figtitle: Overall experimental workflow applied in this study
organisms:
- Severe acute respiratory syndrome coronavirus 2
- Influenza A virus
- Coronaviridae
- Human immunodeficiency virus
- Severe acute respiratory syndrome-related coronavirus
- Middle East respiratory syndrome-related coronavirus
- Respiratory syncytial virus
- Orthomyxoviridae
- H1N1 subtype
- Vesicular stomatitis virus
- Vaccinia virus
- H5N1 subtype
- Betacoronavirus
- unidentified human coronavirus
- Human coronavirus 229E
- H3N2 subtype
- Porcine reproductive and respiratory syndrome virus
- Human coronavirus OC43
- Suid alphaherpesvirus 1
- H7N9 subtype
- Human coronavirus HKU1
- Bat coronavirus
- Bat SARS-like coronavirus
- Macropodid alphaherpesvirus 1
- Murine hepatitis virus strain 1
- Canis lupus familiaris
- Felis catus
- Mustela putorius furo
- Homo sapiens
- Saccharomyces cerevisiae
- Sar
- NA
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC7454314
filename: bbaa173f1.jpg
figlink: pmc/articles/PMC7454314/figure/f1/
number: F1
caption: The overall experimental workflow applied in this study. Following establishing
  SARS-CoV as the closest relative of 2019-nCoV through whole-genome alignment (1000
  bootstrap tests), the microarray datasets related to SARS-CoV infection and four
  COVID-19 complications (i.e. SARS, PNA, SHOB and DRA) were retrieved and analyzed
  to identify differentially expressed genes (DEGs). Herein, Limma R package was employed
  where logFC≥|1| was used to distinguish differential expression, and adjusted P-value
  ≤0.05 was set as cutoff for statistical significance. Then, the DEGs from SARS-CoV
  infection were subjected to validation with gold-standard databases (i.e. OMIM disease,
  OMIM expanded and dbGaP) for comorbidity and risk factor assessment followed by
  comparative analysis with the disease datasets. Finally, we used the shared DEGs
  for network-based interactome analysis and functional enrichments in terms of protein–protein
  interactions (PPI confidence cutoff = 980), gene–biomarker associations (DC≥5 and
  BC≥35) and protein–chemical interactions (DC≥20 and BC≥20). For pathway analysis,
  we considered BioCarta (2016), Reactome (2016), KEGG (human; 2019) and WikiPathways
  (human; 2019) databases, while Biological Process (2018), Molecular Function (2018)
  and Human Phenotype Ontology databases were used for gene ontological evaluation.
  In both cases, adjusted P-value ≤0.05 was considered as significant.
papertitle: Pathogenetic profiling of COVID-19 and SARS-like viruses.
reftext: Zulkar Nain, et al. Brief Bioinform. 2020 Aug 11 :bbaa173.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.515482
figid_alias: PMC7454314__F1
figtype: Figure
redirect_from: /figures/PMC7454314__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7454314__bbaa173f1.html
  '@type': Dataset
  description: The overall experimental workflow applied in this study. Following
    establishing SARS-CoV as the closest relative of 2019-nCoV through whole-genome
    alignment (1000 bootstrap tests), the microarray datasets related to SARS-CoV
    infection and four COVID-19 complications (i.e. SARS, PNA, SHOB and DRA) were
    retrieved and analyzed to identify differentially expressed genes (DEGs). Herein,
    Limma R package was employed where logFC≥|1| was used to distinguish differential
    expression, and adjusted P-value ≤0.05 was set as cutoff for statistical significance.
    Then, the DEGs from SARS-CoV infection were subjected to validation with gold-standard
    databases (i.e. OMIM disease, OMIM expanded and dbGaP) for comorbidity and risk
    factor assessment followed by comparative analysis with the disease datasets.
    Finally, we used the shared DEGs for network-based interactome analysis and functional
    enrichments in terms of protein–protein interactions (PPI confidence cutoff =
    980), gene–biomarker associations (DC≥5 and BC≥35) and protein–chemical interactions
    (DC≥20 and BC≥20). For pathway analysis, we considered BioCarta (2016), Reactome
    (2016), KEGG (human; 2019) and WikiPathways (human; 2019) databases, while Biological
    Process (2018), Molecular Function (2018) and Human Phenotype Ontology databases
    were used for gene ontological evaluation. In both cases, adjusted P-value ≤0.05
    was considered as significant.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Col4a1
  - Ca-alpha1D
  - alphaTub84B
  - nAChRalpha1
  - wb
  - Prosalpha1
  - CG4456
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Go
  - d4
  - 2019-nCov
  - COVID-19
  - CoV
---
