---
figid: PMC7144542__fimmu-11-00502-g0001
figtitle: NFKB and MAPK signaling pathway in lumpfish
organisms:
- Cyclopterus lumpus
- Homo sapiens
organisms_ner:
- Homo sapiens
- Cyclopterus lumpus
- Drosophila melanogaster
pmcid: PMC7144542
filename: fimmu-11-00502-g0001.jpg
figlink: pmc/articles/PMC7144542/figure/F1/
number: F1
caption: Overview of NF-κB and MAPK signaling pathway in lumpfish. (A) NF-κB signaling
  pathway (modified from KEGG map 04064) and (B) MAPK signaling pathway (modified
  from KEGG map 04010). The colors of the boxes refers to the respective gene's differential
  expression upon bacterial exposure and whether or not they are present in the lumpfish
  transcriptome. Gray—not present, a scale ranging from deep blue (very negative,
  log2 fold change = −10), through white (neutral, log2 fold change = 0), to brown-red
  (very positive, log2 fold change = 10) refers to the differentially expressed genes
  at 24 h. All genes that are identified in lumpfish, but not significant (adjusted
  p > 0.05) regulated is shown as white boxes. Differently regulated transcripts of
  the same gene are represented with horizontal bars within the respective gene-box.
  The 11 most regulated genes at 24 h are shown in the tables in the figure. See ,  for
  a full list of DEG at 6 and 24 h post bacterial exposure.
papertitle: 'Interleukin-1 Ligands and Receptors in Lumpfish (Cyclopterus lumpus L.):
  Molecular Characterization, Phylogeny, Gene Expression, and Transcriptome Analyses.'
reftext: Håvard Ø. Eggestøl, et al. Front Immunol. 2020;11:502.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8927081
figid_alias: PMC7144542__F1
figtype: Figure
redirect_from: /figures/PMC7144542__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7144542__fimmu-11-00502-g0001.html
  '@type': Dataset
  description: Overview of NF-κB and MAPK signaling pathway in lumpfish. (A) NF-κB
    signaling pathway (modified from KEGG map 04064) and (B) MAPK signaling pathway
    (modified from KEGG map 04010). The colors of the boxes refers to the respective
    gene's differential expression upon bacterial exposure and whether or not they
    are present in the lumpfish transcriptome. Gray—not present, a scale ranging from
    deep blue (very negative, log2 fold change = −10), through white (neutral, log2
    fold change = 0), to brown-red (very positive, log2 fold change = 10) refers to
    the differentially expressed genes at 24 h. All genes that are identified in lumpfish,
    but not significant (adjusted p > 0.05) regulated is shown as white boxes. Differently
    regulated transcripts of the same gene are represented with horizontal bars within
    the respective gene-box. The 11 most regulated genes at 24 h are shown in the
    tables in the figure. See ,  for a full list of DEG at 6 and 24 h post bacterial
    exposure.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - IL1B
  - CXCL8
  - TNF
  - COX2
  - PTGS2
  - MTCO2P12
  - BPI
  - LTBR
  - CD40
  - AN
  - PERCC1
  - ICAM4-AS1
  - ATM
  - IKBKG
  - BLNK
  - MEMO1
  - RPIA
  - MALT1
  - TNFRSF1A
  - TAB1
  - TAB2
  - TAB3
  - LINC01191
  - RIGI
  - LY6E
  - TMF1
  - CLEC3B
  - IRF6
  - CD14
  - NDUFA2
  - TRAP
  - ACP5
  - CD40LG
  - TRAF2
  - TRRAP
  - TDRD7
  - SCYL1
  - BTK
  - CNTF
  - NGF
  - EGF
  - TRAM1
  - TICAM2
  - TMED7-TICAM2
  - KRT10
  - FUT3
  - RELA
  - TRAF1
  - TRAF3
  - TRAF4
  - TRAF5
  - TRAF6
  - TRAF7
  - ERBB2
  - TANK
  - APELA
  - AZU1
  - CELA1
  - CELA2A
  - CELA2B
  - CELA3A
  - CELA3B
  - CTRC
  - ELANE
  - PRTN3
  - LTB
  - CXCL12
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - GEM
  - GMNN
  - SGSM3
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - BDNF
  - BDNF-AS
  - NFATC3
  - ACAT1
  - MAT1A
  - LRPAP1
  - KRAS
  - HRAS
  - NRAS
  - MUC4
  - PPRC1
  - NT5E
  - PNPLA6
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - MOS
  - MOCOS
  - DUS1L
  - GNA12
  - GNA13
  - PDGFB
  - TGFB3
  - CRK
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - PPIA
  - CUX1
  - CYTIP
  - CRTAP
  - TP53
  - TP63
  - TP73
  - AKT1
  - AKT2
  - AKT3
  - PSS
  - CDSN
  - NR1H2
  - MAX
  - MEF2C
  - C4A
  - C4B
  - WDR7
  - NLK
  - GPI
  - cd40
  - atm
  - blnk
  - malt1
  - btk
  - egf
  - rela
  - traf3
  - gem
  - bdnf
  - mos
  - tgfb3
  - max
  - tab2
  - egr
  - Pas
  - shakB
  - La
  - an
  - tefu
  - nmo
  - CR
  - Rpi
  - Ets97D
  - Tab2
  - Abd-B
  - vin
  - Vinc
  - neo
  - Cow
  - tna
  - Btk
  - Vm26Ab
  - Egfr
  - TRAM
  - cm
  - Ne
  - Rel
  - Traf6
  - Traf4
  - elA
  - Elal
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Traf-like
  - elB
  - MKP-4
  - p38b
  - rl
  - ee
  - geminin
  - dally
  - mapP
  - Pka-C1
  - Pka-R2
  - Pka-C3
  - Pka-R1
  - Pka-C2
  - mat
  - Mad
  - ca
  - Rap1
  - fzr
  - ras
  - Ras64B
  - Ras85D
  - prc
  - CrebB
  - CrebA
  - Mos
  - Mp
  - bsk
  - lic
  - Nurf-38
  - Ebp
  - p38a
  - Hrb87F
  - p38c
  - casp
  - p53
  - betaTub60D
  - hth
  - Akt
  - PTP-ER
  - Pss
  - wgn
  - Max
  - mof
  - COM
  - comt
  - compressed
  - lncRNA:flam
  - Mef2
  - wnd
  - Dsor1
  - hep
  - Calcium
  - Ba
  - TRAM
  - Phosphasidyinostol
  - IPI
  - Ca
  - LPS
  - HR
  - MAT
---
