---
figid: PMC7859603__mejdd-12-257-g001
figtitle: DisGeNET contains a list of diseases associated genes that have been collected
  based on the presence of genetic overlaps between diseases
organisms:
- Homo sapiens
- Macaca mulatta
- Mus musculus
- NA
pmcid: PMC7859603
filename: mejdd-12-257-g001.jpg
figlink: pmc/articles/PMC7859603/figure/F1/
number: F1
caption: DisGeNET contains a list of diseases associated genes that have been collected
  based on the presence of genetic overlaps between diseases. In this list, among
  the 4,753,986 potential disease associations, 13,064 diseases were found to share
  at least one gene with other diseases. Among these associations, CeD was significantly
  associated (combined score at p< 0.05) with atherosclerosis, multiple sclerosis,
  and hepatitis C, while in IBD, the significant associations were seen with stomach
  neoplasms, osteoporosis, postmenopausal syndrome, and sepsis. The combined score
  is calculated by taking the log of the p values generated by the Fisher exact test
  and multiplying that by the z-score of the deviation from the expected rank. Both
  CeD and IBD, with a partial similar genetic background, target the gut, which is
  usually followed up by chronic inflammation. Since the ubiquity of CeD in patients
  with IBD remains unclear, we, therefore, were interested in mining CeD-IBD correlations
  at the transcriptome level to better understand the underlying reasons for this
  overlap. Only CeD-IBD associated genes retrieved from DisGeNET database with at
  least one evidence from a database of protein families (Pfam) were chosen for further
  analysis. Out of 325 CeD and 880 IBD associated genes, 314 and 851 genes, respectively,
  were selected from which 149 genes were common between CeD and IBD. As mentioned
  earlier, GeneMANIA server was used to establish a common network of co-expressed
  genes and pathways in CeD-IBD by utilizing the common identified 149 genes. Afterward,
  the GeneMANIA network was further clustered into the highly interconnected gene
  clusters by MCODE, which totally resulted in the eight sub-networks (). The MCODE
  scores varied from 2.8-28.8. According to the previous studies, immune response
  associated transcription factors (NFKB1, IRF2, VDR), interleukins (IL10, IL6, IL2,
  and IL2RA), epidermal growth factor receptor (EGFR), and interferon-gamma (IFNG)
  were more interesting. The genes within the identified sub-networks were enriched
  in the inflammatory response (FAS), apoptotic pathway (CD14, IRF1), and signaling
  pathways (EFGR, CCL5, CCR5). In pathway analysis, signaling pathways like inflammation
  mediated by cytokine and chemokines, interleukin, toll receptor, and apoptosis seemed
  to be more biologically associated with CeD and IBD (). We additionally investigated
  the 149 common genes to discover disease-associated motifs and master regulators
  by iRegulon and TRRUST (). Among the eight identified sub-networks except for sub-network
  1 in which IRF2 was found as the most probable regulator and NFKB1 placed as the
  second regulator, NFKB1 was found to be the global regulator of 149 common genes
  in other sub-networks. We also examined if the gene-transcription factor links from
  iRegulon were also present as experimentally validated links in TRRUST. As mentioned
  in the method section, finally, the differential expressed genes from these 149
  common genes were determined using Gemma server. Only 39 genes were detected as
  differentially expressed in CeD-oriented experiments in Gemma in different cell
  types () while all of the 149 genes were identified as differentially expressed
  in IBD experiments under different time points, genetic modifications, phenotypes,
  cell types, and medical procedures. The expression of these genes did not change
  in different diets and clinical treatments of patients with IBD ().
papertitle: Network Analysis of Common Genes and Transcriptional Factors between Celiac
  Disease and Inflammatory Bowel Diseases.
reftext: Fereshteh Izadi, et al. Middle East J Dig Dis. 2020 Oct;12(4):257-264.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6656182
figid_alias: PMC7859603__F1
figtype: Figure
organisms_ner:
- Homo sapiens
- Macaca mulatta
- Mus musculus
redirect_from: /figures/PMC7859603__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7859603__mejdd-12-257-g001.html
  '@type': Dataset
  description: DisGeNET contains a list of diseases associated genes that have been
    collected based on the presence of genetic overlaps between diseases. In this
    list, among the 4,753,986 potential disease associations, 13,064 diseases were
    found to share at least one gene with other diseases. Among these associations,
    CeD was significantly associated (combined score at p< 0.05) with atherosclerosis,
    multiple sclerosis, and hepatitis C, while in IBD, the significant associations
    were seen with stomach neoplasms, osteoporosis, postmenopausal syndrome, and sepsis.
    The combined score is calculated by taking the log of the p values generated by
    the Fisher exact test and multiplying that by the z-score of the deviation from
    the expected rank. Both CeD and IBD, with a partial similar genetic background,
    target the gut, which is usually followed up by chronic inflammation. Since the
    ubiquity of CeD in patients with IBD remains unclear, we, therefore, were interested
    in mining CeD-IBD correlations at the transcriptome level to better understand
    the underlying reasons for this overlap. Only CeD-IBD associated genes retrieved
    from DisGeNET database with at least one evidence from a database of protein families
    (Pfam) were chosen for further analysis. Out of 325 CeD and 880 IBD associated
    genes, 314 and 851 genes, respectively, were selected from which 149 genes were
    common between CeD and IBD. As mentioned earlier, GeneMANIA server was used to
    establish a common network of co-expressed genes and pathways in CeD-IBD by utilizing
    the common identified 149 genes. Afterward, the GeneMANIA network was further
    clustered into the highly interconnected gene clusters by MCODE, which totally
    resulted in the eight sub-networks (). The MCODE scores varied from 2.8-28.8.
    According to the previous studies, immune response associated transcription factors
    (NFKB1, IRF2, VDR), interleukins (IL10, IL6, IL2, and IL2RA), epidermal growth
    factor receptor (EGFR), and interferon-gamma (IFNG) were more interesting. The
    genes within the identified sub-networks were enriched in the inflammatory response
    (FAS), apoptotic pathway (CD14, IRF1), and signaling pathways (EFGR, CCL5, CCR5).
    In pathway analysis, signaling pathways like inflammation mediated by cytokine
    and chemokines, interleukin, toll receptor, and apoptosis seemed to be more biologically
    associated with CeD and IBD (). We additionally investigated the 149 common genes
    to discover disease-associated motifs and master regulators by iRegulon and TRRUST
    (). Among the eight identified sub-networks except for sub-network 1 in which
    IRF2 was found as the most probable regulator and NFKB1 placed as the second regulator,
    NFKB1 was found to be the global regulator of 149 common genes in other sub-networks.
    We also examined if the gene-transcription factor links from iRegulon were also
    present as experimentally validated links in TRRUST. As mentioned in the method
    section, finally, the differential expressed genes from these 149 common genes
    were determined using Gemma server. Only 39 genes were detected as differentially
    expressed in CeD-oriented experiments in Gemma in different cell types () while
    all of the 149 genes were identified as differentially expressed in IBD experiments
    under different time points, genetic modifications, phenotypes, cell types, and
    medical procedures. The expression of these genes did not change in different
    diets and clinical treatments of patients with IBD ().
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MMP3
  - IL3
  - QRSL1
  - GATA1
  - GATA2
  - GATA3
  - GATA4
  - GATA5
  - GATA6
  - CCL2
  - TGFBI
  - CD14
  - NDUFA2
  - FAS
  - FASN
  - IL10
  - CCN2
  - IL6
  - IL2
  - RELA
  - NFKBIA
  - JUN
  - JUNB
  - JUND
  - VDR
  - CYP27B1
  - BCL2
  - TBX2
  - STAT3
  - HDAC1
  - HDAC2
  - HDAC3
  - HDAC8
  - FOXP1
  - FOXP2
  - FOXP3
  - FOXP4
  - TNF
  - CEBPB
  - TG
  - EGFR
  - IFNG
  - HLA-G
  - PPARG
  - IL2RA
  - APOB
  - MYB
  - IL17A
  - IL17F
  - TNFRSF9
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - SELE
  - NOS2
  - NANOS2
  - HLA-DRA
  - ABCB1
  - CIITA
  - IFNA17
  - IFNA21
  - APC
  - PROC
  - NLRP3
  - IL6ST
  - MAMU-DRA
  - Mmp3
  - Il3
  - Qrsl1
  - Ccl2
  - Tgfbi
  - Cd14
  - Fas
  - Fasn
  - Spin2g
  - Il10
  - Ccn2
  - Il6
  - Il2
  - Rela
  - Nfkbia
  - Jun
  - Vdr
  - Cyp27b1
  - Bcl2
  - Hps3
  - Tbx2
  - Stat3
  - Tnf
  - Cebpb
  - Tg
  - Egfr
  - Ifng
  - Pparg
  - Il2ra
  - Apob
  - Myb
  - Il17a
  - Tnfrsf9
  - Creb1
  - Sele
  - Nos2
  - Nanos2
  - Abcb1b
  - Ciita
  - Foxp3
  - Apc
  - Nlrp3
  - TBX2
  - APC
---
