---
figid: PMC7611882__EMS134114-f009
figtitle: Non-canonical glutamine transamination sustains efferocytosis by coupling
  redox buffering to oxidative phosphorylation
organisms:
- Mus musculus
- Rattus norvegicus
- Escherichia coli
- Drosophila melanogaster
- Escherichia coli K-12
- Ciona intestinalis
- Homo sapiens
- Armoracia rusticana
organisms_ner:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Drosophila melanogaster
- Danio rerio
pmcid: PMC7611882
filename: EMS134114-f009.jpg
figlink: /pmc/articles/PMC7611882/figure/F9/
number: F4
caption: To elucidate the origin of the mitochondrial reprogramming in Gls1-deficient
  macrophage (), we performed an RNA sequencing (RNA-seq) analysis of PCMs stimulated
  by IL-4 in vivo. Metabolic pathway enrichment analysis highlighted down-regulation
  of oxidative phosphorylation and GSH synthesis KEGG pathways (i.e, NADP, folate
  and GSH metabolism) in resting and reparative MacΔGls1 PCMs (). Thus, to dissect
  the interplay between the mitochondrial metabolic repurposing and the perturbation
  in antioxidant metabolism in MacΔGls1 macrophages, we next performed topological
  analyses using CoMBI-T profiling, and compared predictions to DreamBio that allows
  direct computational KEGG pathway mapping. CoMBI-T and DreamBio profiling analyses
  featured a link between MacΔGls1 PCM mitochondrial metabolic reprogramming and lower
  expression of TCA cycle and aspartate-arginino-succinate (AAS) shunt genes. These
  genes are interconnected to metabolism of glutamine per se, carbohydrates, hexosamine
  and fatty acids (). CoMBI-T predictions also revealed that the redox status of reparative
  MacΔGls1 PCMs may connect to the one carbon cycle-centered module known to support
  the transsulfuration pathway (i.e, GSH synthesis) (). KEGG mapping with DreamBio
  offered an alternative visualization of the perturbed transcriptomic pathways in
  reparative Gls1-deficient macrophages and highlighted additional reactions that
  were not yet predicted (). For instance, in contrast to CoMBI-T, DreamBio analysis
  predicted downregulation of phosphogluconate dehydrogenase (Pgd) and malic enzyme
  (Me1) in reparative Gls1-deficient macrophages. These are two genes enconding enzymes
  that contribute to GSH salvage pathway through NADPH generation to efficiently supply
  reduced glutathione and maintain macrophage redox status (). Altogether, this unique
  approach helped to pinpoint the metabolic origin on how GLS1-dependent glutamine
  metabolism can integrate canonical mitochondrial reprogramming (i.e, TCA and AAS
  shunt) and non-canonical redox status (i.e, GSH synthesis and GSH salvage pathway)
  to support macrophage reparative function.
papertitle: Non-canonical glutamine transamination sustains efferocytosis by coupling
  redox buffering to oxidative phosphorylation.
reftext: Johanna Merlin, et al. Nat Metab. ;3(10):1313-1326.
year: '2021'
doi: 10.1038/s42255-021-00471-y
journal_title: Nature metabolism
journal_nlm_ta: Nat Metab
publisher_name: ''
keywords: Macrophage | Glutaminolysis | Efferocytosis | Interleukin-4 and Atherosclerosis
automl_pathway: 0.9186487
figid_alias: PMC7611882__F4
figtype: Figure
redirect_from: /figures/PMC7611882__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7611882__EMS134114-f009.html
  '@type': Dataset
  description: To elucidate the origin of the mitochondrial reprogramming in Gls1-deficient
    macrophage (), we performed an RNA sequencing (RNA-seq) analysis of PCMs stimulated
    by IL-4 in vivo. Metabolic pathway enrichment analysis highlighted down-regulation
    of oxidative phosphorylation and GSH synthesis KEGG pathways (i.e, NADP, folate
    and GSH metabolism) in resting and reparative MacΔGls1 PCMs (). Thus, to dissect
    the interplay between the mitochondrial metabolic repurposing and the perturbation
    in antioxidant metabolism in MacΔGls1 macrophages, we next performed topological
    analyses using CoMBI-T profiling, and compared predictions to DreamBio that allows
    direct computational KEGG pathway mapping. CoMBI-T and DreamBio profiling analyses
    featured a link between MacΔGls1 PCM mitochondrial metabolic reprogramming and
    lower expression of TCA cycle and aspartate-arginino-succinate (AAS) shunt genes.
    These genes are interconnected to metabolism of glutamine per se, carbohydrates,
    hexosamine and fatty acids (). CoMBI-T predictions also revealed that the redox
    status of reparative MacΔGls1 PCMs may connect to the one carbon cycle-centered
    module known to support the transsulfuration pathway (i.e, GSH synthesis) ().
    KEGG mapping with DreamBio offered an alternative visualization of the perturbed
    transcriptomic pathways in reparative Gls1-deficient macrophages and highlighted
    additional reactions that were not yet predicted (). For instance, in contrast
    to CoMBI-T, DreamBio analysis predicted downregulation of phosphogluconate dehydrogenase
    (Pgd) and malic enzyme (Me1) in reparative Gls1-deficient macrophages. These are
    two genes enconding enzymes that contribute to GSH salvage pathway through NADPH
    generation to efficiently supply reduced glutathione and maintain macrophage redox
    status (). Altogether, this unique approach helped to pinpoint the metabolic origin
    on how GLS1-dependent glutamine metabolism can integrate canonical mitochondrial
    reprogramming (i.e, TCA and AAS shunt) and non-canonical redox status (i.e, GSH
    synthesis and GSH salvage pathway) to support macrophage reparative function.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ad
  - Gatm
  - Odc1
  - Odc-rs1
  - Cyp26b1
  - B3gnt5
  - St3gal2
  - Chkb
  - Asah2
  - Cers6
  - Cad
  - Dffb
  - Acod1
  - Cgct
  - Pisd
  - Dpm3
  - Plat
  - Asph
  - Ext1
  - Arsb
  - Slurp1
  - Pik3c3
  - Lias
  - Hacd4
  - Cers2
  - Elovl7
  - Oxct2a
  - Oxct1
  - Hps3
  - Aldoa
  - Nagk
  - fm
  - da
  - Gnat2
  - Kdm5d
  - Hya
  - Gale
  - Epha3
  - Esd
  - Mthfd1
  - Suclg1
  - Hk1
  - Pgam1
  - Paics
  - Aco2
  - Echs1
  - Hexb
  - Dgkg
  - Aldh2
  - Uchl1
  - Mela
  - Gm42368
  - Pola1
  - Hal
  - Krt31
  - Arhgap45
  - Acad8
  - Dbt
  - Dpm1
  - Alg2
  - Pdcd6
  - Acadm
  - Adpgk
  - Adal
  - Adsl
  - Ads1
  - Pold3
  - Pnp
  - Gast
  - Gnas
  - Por
  - Arfip2
  - Bckdhb
  - Guk1
  - Pat
  - Chd7
  - Cbr1
  - Gart
  - Arg1
  - Tinagl1
  - Coasy
  - Psat1
  - Bst1
  - Cd38
  - Ccne1
  - Slc25a21
  - Orm1
  - Lrit1
  - Pagr1
  - Aldh1a1
  - Agt
  - GATM
  - ODC1
  - SLC25A21
  - CYP26B1
  - B3GNT5
  - ST3GAL2
  - CHKB
  - ASAH2
  - CERS6
  - FGD1
  - ND3
  - CAD
  - DFFB
  - ACOD1
  - PISD
  - SFI1
  - DPM3
  - PLAT
  - ASPH
  - EXT1
  - ARSB
  - SLURP1
  - PIK3C3
  - LIAS
  - HACD4
  - CERS2
  - ELOVL7
  - OXCT1
  - ALDOA
  - NAGK
  - DYSF
  - LY86
  - MMD
  - GALE
  - REG3A
  - ESD
  - MTHFD1
  - SUCLG1
  - ALDOB
  - ALDOC
  - HK1
  - KCNA4
  - KLK1
  - HOOK1
  - TAC4
  - PGAM1
  - PGAM4
  - LDHA
  - LDHB
  - LDHC
  - CTSC
  - GART
  - PAICS
  - ACO2
  - ECHS1
  - HEXB
  - DGKG
  - ALDH2
  - GAD1
  - GAD2
  - POLR2A
  - ERVW-4
  - PFKM
  - POLA1
  - HAL
  - KRT31
  - ARHGAP45
  - SPINT1
  - ST14
  - ACAD8
  - DBT
  - DPM1
  - ALG2
  - PDCD6
  - ACADM
  - ADPGK
  - ADAL
  - ADA
  - TADA1
  - ENAM
  - ADSL
  - POLD3
  - PNP
  - PPY
  - GAST
  - GALNS
  - PAGR1
  - POR
  - VDAC2
  - BCKDHB
  - GUK1
  - CBR1
  - ARG1
  - TINAGL1
  - COASY
  - PSAT1
  - BST1
  - PGAP1
  - CD38
  - CycE
  - cyc
  - ad
  - cad
  - .na.character
  - Drep4
  - Ca-alpha1D
  - shg
  - Dok
  - ttv
  - Me
  - Coa
  - Ald1
  - fand
  - nAChRalpha2
  - Sh
  - hy
  - Alp1
  - Ldh
  - sdt
  - Gad1
  - be
  - Polr2A
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - Pfk
  - Ote
  - hall
  - dco
  - Alg-2
  - Ada1-1
  - Ada1-2
  - AdSL
  - PolD3
  - gas
  - Galphas
  - gd
  - Galphai
  - por
  - pat
  - ptc
  - obt
  - sm
  - Pcp
  - pho
  - slc47a5
  - gatm
  - odc1
  - si:ch211-220m17.5
  - cyp26b1
  - b3gnt5a
  - st3gal2
  - st3gal1
  - chkb
  - asah2
  - cers6
  - dffb
  - cald1b
  - pisd
  - dpm3
  - plat
  - asph
  - arsb
  - pik3c3
  - lias
  - hacd4
  - elovl7a
  - napbb
  - aldoaa
  - nagk
  - gale
  - esd
  - mthfd1b
  - suclg1
  - hk1
  - pgam1a
  - serpine1
  - aco2
  - echs1
  - hexb
  - pmt
  - aldh2.2
  - aldh2.1
  - pola1
  - hal
  - prkci
  - acad8
  - dbt
  - dpm1
  - gmppb
  - alg2
  - acadm
  - adpgk
  - adal
  - adsl
  - pold3
  - pora
  - bckdhb
  - guk1b
  - cbr1
  - gart
  - arg1
  - coasy
  - psat1
---
