---
figid: PMC7986493__12934_2021_1563_Fig10_HTML
figtitle: Glycine, serine, and threonine metabolism pathways of L
organisms:
- Lactobacillus gasseri
- Homo sapiens
pmcid: PMC7986493
filename: 12934_2021_1563_Fig10_HTML.jpg
figlink: pmc/articles/PMC7986493/figure/Fig10/
number: F10
caption: 'Glycine, serine, and threonine metabolism pathways of L. gasseri HHuMIN
  D. Solid arrows indicate that the metabolism pathways of glycine, serine, and threonine
  are effective and dashed arrows indicate invalid: sdaA/sdaB/tdcG L-serine dehydratase,
  glyA/SHMT glycine hydroxymethyltransferase, AGXT alanine-glyoxylate transaminase,
  glxK/garK glycerate 2-kinase, serA/PHGDH D-3-phosphoglycerate dehydrogenase, serC/PSAT1
  phosphoserine aminotransferase, serB/PSPH phosphoserine phosphatase, thrH/phosphoserine
  homoserine phosphotransferase, CBS cystathionine beta-synthase, CHO1/pssA CDP-diacylglycerol–-serine
  O-phosphatidyltransferase, trpA tryptophan synthase alpha chain; and trpB, tryptophan
  synthase beta chain. Whole genome sequencing of L. gasseri HHuMIN D was performed
  by Chunlab, Inc. (Seoul, Korea), which used PacBio Sequel Systems (Pacific Biosciences,
  Menlo Park, CA, USA) and analyzed using CLgenomics™ and EZBioCloud Apps programs
  (Chunlab, Seoul, Korea). Biosynthetic pathways of glycine, serine, and threonine
  were analyzed by matching with the KEGG pathway using complete genome sequencing
  of L. gasseri HHuMIN D via CL Genomics™ (ChunLab, Seoul, South Korea, 2004). The
  activities of related genes and substances produced can be inferred through the
  biosynthetic pathways provided by KEGG'
papertitle: Oral probiotic activities and biosafety of Lactobacillus gasseri HHuMIN
  D.
reftext: Soyon Mann, et al. Microb Cell Fact. 2021;20:75.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9520121
figid_alias: PMC7986493__F10
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC7986493__F10
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7986493__12934_2021_1563_Fig10_HTML.html
  '@type': Dataset
  description: 'Glycine, serine, and threonine metabolism pathways of L. gasseri HHuMIN
    D. Solid arrows indicate that the metabolism pathways of glycine, serine, and
    threonine are effective and dashed arrows indicate invalid: sdaA/sdaB/tdcG L-serine
    dehydratase, glyA/SHMT glycine hydroxymethyltransferase, AGXT alanine-glyoxylate
    transaminase, glxK/garK glycerate 2-kinase, serA/PHGDH D-3-phosphoglycerate dehydrogenase,
    serC/PSAT1 phosphoserine aminotransferase, serB/PSPH phosphoserine phosphatase,
    thrH/phosphoserine homoserine phosphotransferase, CBS cystathionine beta-synthase,
    CHO1/pssA CDP-diacylglycerol–-serine O-phosphatidyltransferase, trpA tryptophan
    synthase alpha chain; and trpB, tryptophan synthase beta chain. Whole genome sequencing
    of L. gasseri HHuMIN D was performed by Chunlab, Inc. (Seoul, Korea), which used
    PacBio Sequel Systems (Pacific Biosciences, Menlo Park, CA, USA) and analyzed
    using CLgenomics™ and EZBioCloud Apps programs (Chunlab, Seoul, Korea). Biosynthetic
    pathways of glycine, serine, and threonine were analyzed by matching with the
    KEGG pathway using complete genome sequencing of L. gasseri HHuMIN D via CL Genomics™
    (ChunLab, Seoul, South Korea, 2004). The activities of related genes and substances
    produced can be inferred through the biosynthetic pathways provided by KEGG'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AGXT
  - PHGDH
  - PSPH
  - PSAT1
  - CBS
  - TPSG1
  - PTDSS1
  - KIF23
  - SHMT2
  - SHMT1
  - Glycine
  - Pyruvate
  - Hydroxy-pyruvate
  - serB
  - Serine
  - Phosphoserine
  - 3P-Hydroxypyruvat
  - L-tryptophane
  - O-Phophatidiyl L-serine
---
