---
figid: PMC7693813__metabolites-10-00425-g004
figtitle: MapMan and KEGG analysis of the DEGs in the mutants
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Solanum lycopersicum
- Triticum aestivum
- Pyricularia oryzae
organisms_ner:
- Arabidopsis thaliana
- Solanum lycopersicum
- Drosophila melanogaster
pmcid: PMC7693813
filename: metabolites-10-00425-g004.jpg
figlink: pmc/articles/PMC7693813/figure/metabolites-10-00425-f004/
number: F4
caption: MapMan and KEGG analysis of the DEGs in the mutants. (A) Kyoto Encyclopedia
  of Genes and Genomes (KEGG) analysis of the differentially expressed genes (DEGs)
  in d10 vs. WT and d14 vs. WT. (B) Overview of the secondary metabolism-related DEGs
  in the mutants. (C) Enrichment of DEGs into biotic stress pathways in d10 (blue
  rectangle) and d14 (red rectangle). In the heat maps in B and C, red and blue indicate
  upregulation or downregulation in the mutants compared with the WT, respectively.
  Colored boxes in each region represent multiple variations in gene expression. (D)
  qRT-PCR based expression levels of WRKY45 and ST1 in d10, d14, and WT. The data
  are represented as mean ± SD of two biological replicates. The Student’s t-test
  analysis indicates a significant difference (compared with WT, * p < 0.05, ** p
  < 0.01).
papertitle: Integrative Metabolomic and Transcriptomic Analyses Reveal Metabolic Changes
  and Its Molecular Basis in Rice Mutants of the Strigolactone Pathway.
reftext: Xiujuan Zhou, et al. Metabolites. 2020 Nov;10(11):425.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7826645
figid_alias: PMC7693813__F4
figtype: Figure
redirect_from: /figures/PMC7693813__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7693813__metabolites-10-00425-g004.html
  '@type': Dataset
  description: MapMan and KEGG analysis of the DEGs in the mutants. (A) Kyoto Encyclopedia
    of Genes and Genomes (KEGG) analysis of the differentially expressed genes (DEGs)
    in d10 vs. WT and d14 vs. WT. (B) Overview of the secondary metabolism-related
    DEGs in the mutants. (C) Enrichment of DEGs into biotic stress pathways in d10
    (blue rectangle) and d14 (red rectangle). In the heat maps in B and C, red and
    blue indicate upregulation or downregulation in the mutants compared with the
    WT, respectively. Colored boxes in each region represent multiple variations in
    gene expression. (D) qRT-PCR based expression levels of WRKY45 and ST1 in d10,
    d14, and WT. The data are represented as mean ± SD of two biological replicates.
    The Student’s t-test analysis indicates a significant difference (compared with
    WT, * p < 0.05, ** p < 0.01).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AT3G03990
  - APK
  - FCA
  - CNGC4
  - bZIP
  - WRKY45
  - SOT12
  - MST1
  - CHI
  - MYB
  - TMP1
  - ST1
  - Tak1
  - CanA1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Rich
  - Myb
  - br
  - Nplp1
  - SA
  - sa
  - beta-Spec
  - betaTub56D
  - betaTry
  - anon-70Da
  - dor
  - stumps
  - St1
  - Diedel
  - sucrose
  - Amino sugar
  - nucleotide sugar
  - Diterpenoid
  - Phenylpropanoid
  - Glutathione
  - Cysteine
  - methionine
  - leucine
  - S
  - Terpenes
  - Flavonoids
  - Ethylene
  - Li
---
