---
figid: PMC7380581__mSystems.00651-19-f0005
figtitle: Metabolic reconstruction of “Ca
organisms:
- Archaeoglobus fulgidus
- Methanosarcina barkeri
- Geobacter metallireducens
- Methanopyrus kandleri
- Archaeoglobus fulgidus DSM 4304
- Geoglobus acetivorans
- Archaeoglobus profundus
- Desulforapulum autotrophicum HRM2
- Ferroglobus placidus
- Candidatus Methanoplasma termitum
- Geoglobus ahangari
- Candidatus Methanodesulfokores washburnensis
- metagenome
- metagenomes
- uncultured bacterium
- uncultured archaeon
- Eucalyptus intertexta
- transposons
- Ficus racemosa
- integrons
organisms_ner:
- Geoglobus ahangari
- Archaeoglobus fulgidus DSM 4304
- Geoglobus acetivorans
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC7380581
filename: mSystems.00651-19-f0005.jpg
figlink: pmc/articles/PMC7380581/figure/fig5/
number: F5
caption: Metabolic reconstruction of “Ca. Methanomixophus.” The genes used to construct
  this metabolic network are listed in . Pathways for methanogenesis are depicted
  in blue, pathways for anaerobic methane oxidation coupled with sulfate reduction
  are depicted in red, and other pathways are depicted in black. The sulfate reduction
  pathway was found only in “Ca. Methanomixophus dualitatem,” whereas other pathways
  were found to be conserved in all “Ca. Methanomixophus” MAGs. FPKM values are represented
  as red bars close to the gene names, and an average FPKM value representing the
  transcribed genes was used for enzymes comprising multiple subunits. Complexes without
  proteomic evidence are depicted with dashed lines. (See  legend for definitions
  of abbreviations.)
papertitle: Genomic and Transcriptomic Evidence Supports Methane Metabolism in Archaeoglobi.
reftext: Yi-Fan Liu, et al. mSystems. 2020 Mar-Apr;5(2):e00651-19.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9690393
figid_alias: PMC7380581__F5
figtype: Figure
redirect_from: /figures/PMC7380581__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7380581__mSystems.00651-19-f0005.html
  '@type': Dataset
  description: Metabolic reconstruction of “Ca. Methanomixophus.” The genes used to
    construct this metabolic network are listed in . Pathways for methanogenesis are
    depicted in blue, pathways for anaerobic methane oxidation coupled with sulfate
    reduction are depicted in red, and other pathways are depicted in black. The sulfate
    reduction pathway was found only in “Ca. Methanomixophus dualitatem,” whereas
    other pathways were found to be conserved in all “Ca. Methanomixophus” MAGs. FPKM
    values are represented as red bars close to the gene names, and an average FPKM
    value representing the transcribed genes was used for enzymes comprising multiple
    subunits. Complexes without proteomic evidence are depicted with dashed lines.
    (See  legend for definitions of abbreviations.)
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - mch
  - mer
  - sat
  - ca
  - CSN4
  - mo
  - fo
  - ss
  - fwd
  - feo
  - fd
  - anon-16Fb
  - red
  - trc
  - Mcr
  - Src42A
  - Stat92E
  - Src64B
  - hh
  - dsr
  - mtd
  - VhaAC39-1
  - Aldh
  - Aldh-III
  - MTA1-like
  - MtnA
  - Mer
  - Aps
  - Orc1
  - fru
  - ATPsynbeta
  - Atpalpha
  - Vha14-1
  - cd
  - por
  - did
  - poly
  - FGFR1
  - TNFRSF11A
  - IK
  - DYRK3
  - PMCH
  - HAS1
  - NR3C2
  - PIGP
  - MT1E
  - ALDH1A1
  - ALDH1A2
  - ALDH1A3
  - ALDH2
  - ALDH3A1
  - ALDH3A2
  - ALDH3B1
  - ALDH5A1
  - ALDH7A1
  - ALDH9A1
  - LRP1
  - PMAIP1
  - GPER1
  - MERTK
  - KLK3
  - SH2B2
  - SAT1
  - ATP8A2
  - DNAH8
  - POR
  - VDAC2
  - Methane
  - Sulfate
  - Ca
  - CoB-SH
  - CHO
  - Fd
  - CH
  - 2Na
  - CHI-HMPT
  - ATP
  - Ethanol
  - Acetyl-CoA
  - ADP
  - Fatty acid
  - 2-keto acids
  - Acetate
  - Pyruvate
  - Amino acids
  - D-lactate
---
