---
figid: PMC7809420__41598_2020_80635_Fig5_HTML
figtitle: Differentially expressed lncRNAs in liver tissues of TX mice with hepatolenticular
  degeneration
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Drosophila melanogaster
pmcid: PMC7809420
filename: 41598_2020_80635_Fig5_HTML.jpg
figlink: pmc/articles/PMC7809420/figure/Fig5/
number: F5
caption: As shown in (a), ENSMUST00000181536 (Snhg19) correlated with 222 protein-coding
  genes, and the enrichment analysis showed that Snhg18 may be related to leukocyte
  transendothelial migration, phagosome, glutathione metabolism, regulation of actin
  cytoskeleton, natural killer cell mediated cytotoxicity, focal adhesion, Fc epsilon
  RI signalling pathway, cell adhesion molecules, apoptosis, and Fc gamma R-mediated
  phagocytosis. As shown in (b), ENSMUST00000150851 (Meg3) correlated with 200 protein-coding
  genes, and KEGG enrichment analysis pointed out that it may participate in sphingolipid
  metabolism, regulation of actin cytoskeleton, glycosaminoglycan degradation, complement
  and coagulation cascades, adherens junction, leukocyte transendothelial migration,
  galactose metabolism, haematopoietic cell lineage, chemokine signalling pathway,
  p53 signalling pathway, B cell receptor signalling pathway, Fc epsilon RI signalling
  pathway, ABC transporters, mitogenactivated protein kinase (MAPK) signalling pathway,
  and apoptosis. As shown in (c), ENSMUST00000129245 (Meg3) correlated with 169 protein-coding
  genes, that enriched to dozens of KEGG pathway, such as gap junction, ABC transporters,
  peroxisome, glutathione metabolism, GnRH signalling pathway, complement and coagulation
  cascades, peroxisome proliferator-activated receptor (PPAR) signalling pathway,
  metabolism of xenobiotics by cytochrome P450, phagosome, tryptophan metabolism,
  fatty acid degradation, carbohydrate digestion and absorption, Notch signalling
  pathway, ribosome, and toll-like receptor signalling pathway. KEGG analysis revealed
  that two lncRNA transcripts were both related to ABC transporters and complement
  and coagulation cascades, although these two transcripts did not share the same
  gene of interaction. As shown in (d), ENSMUST00000136359 (H19) correlated with 169
  protein-coding genes, including vitamin digestion and absorption, galactose metabolism,
  carbohydrate digestion and absorption, retinol metabolism, phagosome, gap junction,
  arachidonic acid metabolism, butanoate metabolism, complement and coagulation cascades,
  antigen processing and presentation, PPAR signalling pathway, ABC transporters,
  Notch signalling pathway, bile secretion, and MAPK signalling pathway.
papertitle: Differentially expressed lncRNAs in liver tissues of TX mice with hepatolenticular
  degeneration.
reftext: Juan Zhang, et al. Sci Rep. 2021;11:1377.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7539433
figid_alias: PMC7809420__F5
figtype: Figure
redirect_from: /figures/PMC7809420__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7809420__41598_2020_80635_Fig5_HTML.html
  '@type': Dataset
  description: As shown in (a), ENSMUST00000181536 (Snhg19) correlated with 222 protein-coding
    genes, and the enrichment analysis showed that Snhg18 may be related to leukocyte
    transendothelial migration, phagosome, glutathione metabolism, regulation of actin
    cytoskeleton, natural killer cell mediated cytotoxicity, focal adhesion, Fc epsilon
    RI signalling pathway, cell adhesion molecules, apoptosis, and Fc gamma R-mediated
    phagocytosis. As shown in (b), ENSMUST00000150851 (Meg3) correlated with 200 protein-coding
    genes, and KEGG enrichment analysis pointed out that it may participate in sphingolipid
    metabolism, regulation of actin cytoskeleton, glycosaminoglycan degradation, complement
    and coagulation cascades, adherens junction, leukocyte transendothelial migration,
    galactose metabolism, haematopoietic cell lineage, chemokine signalling pathway,
    p53 signalling pathway, B cell receptor signalling pathway, Fc epsilon RI signalling
    pathway, ABC transporters, mitogenactivated protein kinase (MAPK) signalling pathway,
    and apoptosis. As shown in (c), ENSMUST00000129245 (Meg3) correlated with 169
    protein-coding genes, that enriched to dozens of KEGG pathway, such as gap junction,
    ABC transporters, peroxisome, glutathione metabolism, GnRH signalling pathway,
    complement and coagulation cascades, peroxisome proliferator-activated receptor
    (PPAR) signalling pathway, metabolism of xenobiotics by cytochrome P450, phagosome,
    tryptophan metabolism, fatty acid degradation, carbohydrate digestion and absorption,
    Notch signalling pathway, ribosome, and toll-like receptor signalling pathway.
    KEGG analysis revealed that two lncRNA transcripts were both related to ABC transporters
    and complement and coagulation cascades, although these two transcripts did not
    share the same gene of interaction. As shown in (d), ENSMUST00000136359 (H19)
    correlated with 169 protein-coding genes, including vitamin digestion and absorption,
    galactose metabolism, carbohydrate digestion and absorption, retinol metabolism,
    phagosome, gap junction, arachidonic acid metabolism, butanoate metabolism, complement
    and coagulation cascades, antigen processing and presentation, PPAR signalling
    pathway, ABC transporters, Notch signalling pathway, bile secretion, and MAPK
    signalling pathway.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Adam10
  - Grk4
  - Dcaf13
  - Tinag
  - Cdipt
  - Emilin1
  - Glyat
  - Myl12a
  - Cartpt
  - Bid
  - Tenm3
  - Sta
  - Emd
  - Hpse
  - Vav1
  - Bcar1
  - Cse1l
  - Hspg2
  - Per1
  - Lum
  - Pt
  - Ptprs
  - Cda
  - Pdcl
  - Pdcd1
  - Cyba
  - Caf
  - Gsta13
  - Cat
  - cac
  - Mip
  - Trpv6
  - Myh14
  - Myh7b
  - Ngf
  - Smyd4
  - Fgfr2
  - Get1
  - Edem2
  - ap
  - Gpc1
  - Pam
  - Mycbp2
  - Efnb2
  - Vasp
  - Vim
  - Rhob
  - Fancd2
  - Afg3l2
  - Etv3
  - Large1
  - Ehhadh
  - Dusp1
  - Bax
  - Nkx3-1
  - Aanat
  - Mmp27
  - Tgm2
  - ur
  - tip
  - Mettl8
  - Faf1
  - Fgf1
  - Set
  - Abl2
  - Rere
  - Albfm1
  - H20
  - Chd1
  - Lrch1
  - Lrat
  - Gem
  - Gmnn
  - Gsta1
  - H2
  - Rerg
  - Jun
  - Mmp14
  - Acsl1
  - Acss2
  - Pla2g15
  - di
  - Lnx2
  - Cerk
  - Abca3
  - Mgam
  - Serpina1e
  - Dnmbp
  - Meta
  - Thbd
  - pr
  - Rgs2
  - Tuba1a
  - Gla
  - Gnai2
  - Hamp
  - Fndc10
  - Lbp
  - Cmip
  - Tubb4a
  - Aspg
  - Adamts1
  - Adamts5
  - Ppig
  - Mal
  - Tirap
  - Mrtfa
  - Sowahb
  - Alkbh1
  - Ar
  - Procr
  - Dgat1
  - Dgat2
  - Awat2
  - Re
  - Hand2
  - ADAM10
  - GRK1
  - GRK4
  - GRK5
  - GRK6
  - GRK7
  - GRK2
  - GRK3
  - DCAF13
  - TINAG
  - CDIPT
  - EMILIN1
  - GLYAT
  - MYL12A
  - CARTPT
  - CMA1
  - BID
  - TENM3
  - CAPN1
  - CAPNS1
  - FAM111B
  - EMD
  - GCY
  - HPSE
  - CCDC85B
  - COMA
  - VAV1
  - VAV2
  - VAV3
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - PER1
  - KLK15
  - GSTM2
  - DUSP5
  - DUSP1
  - DUSP2
  - DUSP3
  - DUSP4
  - DUSP6
  - LUM
  - ATM
  - PTPRS
  - CDA
  - PDCL
  - CYBA
  - ACAN
  - ASGR1
  - ASGR2
  - ATRN
  - ATRNL1
  - BCAN
  - CD207
  - CD209
  - CD248
  - CD302
  - CD69
  - CD72
  - CD93
  - CHODL
  - CLEC10A
  - CLEC11A
  - CLEC12A
  - CLEC12B
  - CLEC14A
  - CLEC16A
  - CLEC17A
  - CLEC18A
  - CLEC18B
  - CLEC18C
  - CLEC19A
  - CLEC1A
  - CLEC1B
  - CLEC20A
  - CLEC2A
  - CLEC2B
  - CLEC2D
  - CLEC2L
  - CLEC3A
  - CLEC3B
  - CLEC4A
  - CLEC4C
  - CLEC4D
  - CLEC4E
  - CLEC4F
  - CLEC4G
  - CLEC5A
  - CLEC6A
  - CLEC7A
  - CLEC9A
  - COLEC10
  - COLEC11
  - COLEC12
  - DGCR2
  - FCER2
  - FREM1
  - KLRB1
  - KLRC1
  - KLRC2
  - KLRC3
  - KLRD1
  - KLRF1
  - KLRF2
  - KLRG1
  - KLRG2
  - KLRK1
  - LAYN
  - LY75
  - NCAN
  - OLR1
  - PKD1
  - PKD1L2
  - PKD1L3
  - PLA2R1
  - PRG2
  - PRG3
  - REG1A
  - REG1B
  - REG3A
  - REG3G
  - REG4
  - SFTPA1
  - SFTPA2
  - SFTPD
  - THBD
  - VCAN
  - MRC1
  - MRC2
  - SELE
  - SELL
  - SELP
  - KAT2B
  - ACTA1
  - CAT
  - CRAT
  - MYH14
  - MYH7B
  - NGF
  - SMYD4
  - FGFR2
  - GET1
  - MOB3C
  - EDEM2
  - GPC1
  - PAM
  - MYCBP2
  - EFNB2
  - TYRP1
  - VASP
  - SEPTIN4
  - VIM
  - RHOB
  - FUT1
  - FUT2
  - FUT3
  - FUT4
  - FUT5
  - FUT6
  - FUT7
  - FUT8
  - FUT9
  - FUT10
  - FUT11
  - POFUT1
  - POFUT2
  - KRT126P
  - AKT1
  - BRCA2
  - FANCD2
  - PSEN1
  - NR1I2
  - JTB
  - ETV3
  - CELA2A
  - LARGE1
  - EHHADH
  - DUSP12
  - BAX
  - AANAT
  - UGT1A
  - UGT1A10
  - UGT1A8
  - UGT1A7
  - UGT1A6
  - UGT1A4
  - UGT1A1
  - MMP27
  - TGM2
  - KAT5
  - METTL8
  - ITFG1
  - TIPRL
  - USP9X
  - SET
  - GSN
  - ABL2
  - RERE
  - CHD1
  - LRCH1
  - LRAT
  - GEM
  - GMNN
  - GSTA1
  - HSPA12A
  - HSPA12B
  - HSPA13
  - HSPA14
  - HSPA1A
  - HSPA1B
  - HSPA1L
  - HSPA2
  - HSPA4
  - HSPA4L
  - HSPA5
  - HSPA6
  - HSPA8
  - HSPA9
  - HSPH1
  - HYOU1
  - SLC25A4
  - SLC25A6
  - RERG
  - JUN
  - JUNB
  - JUND
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - MMP14
  - PLA2G15
  - ACSS2
  - ACCS
  - LNX2
  - CERK
  - DURS1
  - ABCA3
  - ABCC1
  - MGAM
  - DNMBP
  - TUBA1A
  - TUBA1B
  - TUBA1C
  - TUBA3C
  - TUBA3D
  - TUBA3E
  - TUBA4A
  - TUBA8
  - TUBAL3
  - RGS2
  - RIEG2
  - GLA
  - NAT8
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - HAMP
  - FNDC10
  - LBP
  - RPSA
  - CMIP
  - TUBB
  - TUBB2A
  - TUBB1
  - TUBB2B
  - TUBB3
  - TUBB4A
  - TUBB4B
  - TUBB6
  - ASPG
  - ADAMTS1
  - PPIG
  - CYP17A1
  - CYP19A1
  - CYP20A1
  - CYP21A2
  - CYP24A1
  - CYP39A1
  - TBXAS1
  - CYP46A1
  - CYP51A1
  - CYP1A1
  - CYP1A2
  - CYP1B1
  - CYP11A1
  - CYP11B1
  - CYP11B2
  - CYP2A13
  - CYP2A6
  - CYP2A7
  - CYP2B6
  - CYP2C18
  - CYP2C19
  - CYP2C8
  - CYP2C9
  - CYP2D6
  - CYP2D7
  - CYP2E1
  - CYP2F1
  - CYP2J2
  - CYP2R1
  - CYP2S1
  - CYP2U1
  - CYP2W1
  - CYP26A1
  - CYP26B1
  - CYP26C1
  - CYP27A1
  - CYP27B1
  - CYP27C1
  - CYP3A4
  - CYP3A43
  - CYP3A5
  - CYP3A7
  - CYP4A11
  - CYP4A22
  - CYP4B1
  - CYP4F11
  - CYP4F12
  - CYP4F2
  - CYP4F22
  - CYP4F3
  - CYP4F8
  - CYP4V2
  - CYP4X1
  - CYP4Z1
  - CYP7A1
  - CYP7B1
  - CYP8B1
  - PTGIS
  - MAL
  - MRTFA
  - TIRAP
  - PNMA1
  - GSTZ1
  - SOWAHB
  - ALKBH1
  - AR
  - PROCR
  - DGAT1
  - DGAT2
  - AWAT2
  - SGSM3
  - HAND2
  - Chdh
  - Sult2a1
  - Sult1e1
  - Acox1
  - Dnaaf4
  - Plaur
  - Rnase3
  - Setl1
  - Setsip
  - Gsta2
  - Mgam2
  - Tuba1b
  - Tubb5
  - Cyp3a23-3a1
  - Pnma1
  - Ddn
  - kuz
  - Kul
  - grk
  - ra
  - ca
  - gm
  - Cen
  - cnn
  - lin
  - drl
  - Gat
  - .na.character
  - Pitslre
  - gate
  - Me
  - rm
  - sng
  - CarT
  - sta
  - DIP-alpha
  - Cndp2
  - DptA
  - Vav
  - cass
  - per
  - Aco
  - Acox57D-d
  - usp
  - pt
  - oc
  - Acf
  - Ca-alpha1T
  - ChAT
  - sv
  - Ae
  - Smyd4-2
  - htl
  - CG32022
  - Myt1
  - CG9314
  - ena
  - kra
  - Past1
  - Kr
  - rho
  - Mad
  - rasp
  - Rac1
  - mr
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - par
  - para
  - par-6
  - ear
  - pel
  - et
  - Nup154
  - insc
  - sktl
  - I-2
  - Pa
  - arg
  - aos
  - gem
  - geminin
  - dally
  - de
  - ant
  - sesB
  - Antp
  - ems
  - Jra
  - MKP-4
  - p38b
  - rl
  - AcCoAS
  - Acsl
  - Dad
  - se
  - alphaTub84B
  - wg
  - gl
  - Galphai
  - mthl5
  - mle
  - Serinc
  - Mtpalpha
  - dib
  - mal
  - bs
  - Mrtf
  - ar
  - Proc-R
  - Kdm2
  - den
  - Men
  - mapP
  - CdB4
  - Ca
  - Pt
  - Kr
  - Co
  - Se
  - Coma
---
