---
figid: PMC7479627__gr2
figtitle: 'Applications of CRISPR/Cas to Improve Crop Disease Resistance: Beyond Inactivation
  of Susceptibility Factors'
organisms:
- Arabidopsis thaliana
- Nicotiana tabacum
- Oryza sativa
- Zea mays
- Glycine max
- Solanum lycopersicum
- Solanum tuberosum
- Triticum aestivum
- Hordeum vulgare
- Malus domestica
- Cucumis sativus
- Brassica napus
- Sorghum bicolor
- Fragaria x ananassa
- Nicotiana benthamiana
- Chlamydomonas reinhardtii
- Brassica oleracea
- Botrytis cinerea
- eudicotyledons
- Saccharum officinarum
- Musa acuminata
- Gossypium hirsutum
- Manihot esculenta
- Fusarium oxysporum
- Citrus x paradisi
- Brassica rapa
- Pyricularia oryzae
- Citrus sinensis
- Liliopsida
- Fusarium graminearum
- Phytophthora infestans
- Petunia x hybrida
- Citrullus lanatus subsp. vulgaris
- Sclerotinia sclerotiorum
- Peronospora farinosa
- Verticillium dahliae
- Blumeria graminis
- Phytophthora capsici
- Phytophthora parasitica
- Verticillium longisporum
- Blumeria graminis f. sp. tritici
- Erysiphe neolycopersici
- Exserohilum turcicum
- Musa sp.
- Phytophthora tropicalis
- Turnip mosaic virus
- Papaya ringspot virus
- Potato virus Y
- Tomato yellow leaf curl virus
- Soybean mosaic virus
- Cotton leaf curl Multan virus
- Wheat dwarf virus
- Zucchini yellow mosaic virus
- Rice tungro spherical virus
- Banana streak virus
- Cucumber vein yellowing virus
- Cassava brown streak virus
- Agrobacterium tumefaciens
- Pseudomonas syringae
- Streptococcus pyogenes
- Xanthomonas campestris
- Pseudomonas syringae pv. tomato str. DC3000
- Pseudomonas syringae pv. tomato
- Xanthomonas oryzae pv. oryzae
- Xanthomonas oryzae
- Erwinia amylovora
- Xanthomonas citri
- Nilaparvata lugens
- Chilo suppressalis
- Heterodera avenae
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC7479627
filename: gr2.jpg
figlink: pmc/articles/PMC7479627/figure/fig2/
number: F2
caption: Potential CRISPR/Cas Applications to Increase Pathogen Resistance(A–E) Depending
  on the repair mechanism the CRISPR/Cas-induced double-strand breaks (DSBs) can lead
  to simple mutation of a certain sequence (via the error-prone non-homologous end
  joining [NHEJ] pathway), resulting often in small insertions or deletions (InDels)
  causing frameshift mutations and thereby loss of function (A). Alternatively, in
  the presence of a repair template, the DSB can be repaired via homology-directed
  repair (HDR) allowing the insertion of larger foreign DNA in a genomic locus of
  choice (B) or modification of a target region by introducing mutations of choice
  (C). Single-nucleotide mutations can further be induced by deploying cytosine base
  editors (CBEs) with cytidine deaminase or adenine base editors (ABEs) with adenine
  deaminase activity (D). Larger defined mutations can be induced by prime editing,
  which is based on nCas9 fused with reverse transcriptase depending on prime editing
  gRNA instead of normal sgRNAs (E). For details see text.
papertitle: 'Applications of CRISPR/Cas to Improve Crop Disease Resistance: Beyond
  Inactivation of Susceptibility Factors.'
reftext: Dirk Schenke, et al. iScience. 2020 Sep 25;23(9):101478.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.5657157
figid_alias: PMC7479627__F2
figtype: Figure
redirect_from: /figures/PMC7479627__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7479627__gr2.html
  '@type': Dataset
  description: Potential CRISPR/Cas Applications to Increase Pathogen Resistance(A–E)
    Depending on the repair mechanism the CRISPR/Cas-induced double-strand breaks
    (DSBs) can lead to simple mutation of a certain sequence (via the error-prone
    non-homologous end joining [NHEJ] pathway), resulting often in small insertions
    or deletions (InDels) causing frameshift mutations and thereby loss of function
    (A). Alternatively, in the presence of a repair template, the DSB can be repaired
    via homology-directed repair (HDR) allowing the insertion of larger foreign DNA
    in a genomic locus of choice (B) or modification of a target region by introducing
    mutations of choice (C). Single-nucleotide mutations can further be induced by
    deploying cytosine base editors (CBEs) with cytidine deaminase or adenine base
    editors (ABEs) with adenine deaminase activity (D). Larger defined mutations can
    be induced by prime editing, which is based on nCas9 fused with reverse transcriptase
    depending on prime editing gRNA instead of normal sgRNAs (E). For details see
    text.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - CEACAM4
  - CEACAM6
  - SEMA6A
  - GATA3
  - BPIFA4P
  - CST8
  - HAS1
  - cas
  - caz
  - Cse1
  - cass
  - cash
  - Nca
  - dsb
  - levy
  - COX6AL2
  - if
  - as
  - be
  - cytidine
  - adenine
  - HIGS
---
