---
figid: PMC7097452__41581_2019_199_Fig2_HTML
figtitle: Prognostic and predictive enrichment in sepsis
organisms:
- Homo sapiens
- Mus musculus
pmcid: PMC7097452
filename: 41581_2019_199_Fig2_HTML.jpg
figlink: pmc/articles/PMC7097452/figure/Fig2/
number: F2
caption: 'The community approach to mortality risk prediction in a sepsis study identified
  65 risk-predicting genes based on publicly available gene expression data from adults,
  children and neonate with sepsis. Similarly, during the initial discovery phase
  of the PERSEVERE biomarkers used for prognostic enrichment, 80 genes associated
  with poor outcome were selected based on transcriptomic data from a cohort of paediatric
  patients with septic shock. An analysis of the genes identified in these two studies
  revealed 11 genes in common: adhesion G protein-coupled receptor E3 (ADGRE3), CD24
  molecule (CD24), carcinoembryonic antigen-related cell adhesion molecule 8 (CEACAM8),
  CDC28 protein kinase regulatory subunit 2 (CKS2), C-X3-C motif chemokine receptor
  1 (CX3CR1), DNA damage inducible transcript 4 (DDIT4), G0/G1 switch 2 (G0S2), C-X-C
  motif chemokine ligand 8 (CXCL8, also known as IL-8), MAF bZIP transcription factor
  F (MAFF), regulator of G protein signalling 1 (RGS1) and transforming growth factor
  beta induced (TGFBI). The network was generated by uploading these 11 genes to the
  Ingenuity Pathway Analysis platform. Solid lines represent direct interactions and
  dashed lines represent indirect interactions; solid arrowheads can represent activation,
  modification and phosphorylation among others (for more details see Ingenuity Pathway
  Analysis legend), open arrowhead represents translocation and bar-headed arrow indicates
  that the gene inhibits and acts on its target. The gene network contains three highly
  connected central nodes relevant to the pathobiology of sepsis and inflammation:
  tumour necrosis factor (TNF), CXCL8 and tumour protein p53 (TP53). APOBEC3B, apolipoprotein
  B mRNA editing enzyme catalytic subunit 3B; CCRL2, C-C motif chemokine receptor
  like 2; CD300C, CMRF35-like molecule 6; DCSTAMP, dendrocyte expressed seven transmembrane
  protein; EHD1, EH domain containing 1; FFAR2, free fatty acid receptor 2; HRG, histidine-rich
  glycoprotein; HTR4, 5-hydroxytryptamine receptor 4; IL36RN, interleukin 36 receptor
  antagonist; LAT2, linker for activation of T cells family member 2; LILRA5, leukocyte
  immunoglobulin-like receptor A5; LSS, lanosterol synthase; MIR142, microRNA 142;
  NINJ1, ninjurin 1; PRIM1, DNA primase subunit 1; RGS13, regulator of G protein signalling
  13; SCO2, SCO cytochrome c oxidase assembly protein 2; SQLE, squalene epoxidase;
  TGFB1, transforming growth factor beta 1; TGM2, transglutaminase 2; TREM1, triggering
  receptor expressed on myeloid cells 1. aRefers to the mature sequence miR-125b-2–3p,
  encoded in the micro RNA-125b-2 (MIR125B2) stem loop sequence, and includes other
  microRNAs with the seed sequence CAAGUCA.'
papertitle: Prognostic and predictive enrichment in sepsis.
reftext: Natalja L. Stanski, et al. Nat Rev Nephrol. 2020;16(1):20-31.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8986049
figid_alias: PMC7097452__F2
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
redirect_from: /figures/PMC7097452__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7097452__41581_2019_199_Fig2_HTML.html
  '@type': Dataset
  description: 'The community approach to mortality risk prediction in a sepsis study
    identified 65 risk-predicting genes based on publicly available gene expression
    data from adults, children and neonate with sepsis. Similarly, during the initial
    discovery phase of the PERSEVERE biomarkers used for prognostic enrichment, 80
    genes associated with poor outcome were selected based on transcriptomic data
    from a cohort of paediatric patients with septic shock. An analysis of the genes
    identified in these two studies revealed 11 genes in common: adhesion G protein-coupled
    receptor E3 (ADGRE3), CD24 molecule (CD24), carcinoembryonic antigen-related cell
    adhesion molecule 8 (CEACAM8), CDC28 protein kinase regulatory subunit 2 (CKS2),
    C-X3-C motif chemokine receptor 1 (CX3CR1), DNA damage inducible transcript 4
    (DDIT4), G0/G1 switch 2 (G0S2), C-X-C motif chemokine ligand 8 (CXCL8, also known
    as IL-8), MAF bZIP transcription factor F (MAFF), regulator of G protein signalling
    1 (RGS1) and transforming growth factor beta induced (TGFBI). The network was
    generated by uploading these 11 genes to the Ingenuity Pathway Analysis platform.
    Solid lines represent direct interactions and dashed lines represent indirect
    interactions; solid arrowheads can represent activation, modification and phosphorylation
    among others (for more details see Ingenuity Pathway Analysis legend), open arrowhead
    represents translocation and bar-headed arrow indicates that the gene inhibits
    and acts on its target. The gene network contains three highly connected central
    nodes relevant to the pathobiology of sepsis and inflammation: tumour necrosis
    factor (TNF), CXCL8 and tumour protein p53 (TP53). APOBEC3B, apolipoprotein B
    mRNA editing enzyme catalytic subunit 3B; CCRL2, C-C motif chemokine receptor
    like 2; CD300C, CMRF35-like molecule 6; DCSTAMP, dendrocyte expressed seven transmembrane
    protein; EHD1, EH domain containing 1; FFAR2, free fatty acid receptor 2; HRG,
    histidine-rich glycoprotein; HTR4, 5-hydroxytryptamine receptor 4; IL36RN, interleukin
    36 receptor antagonist; LAT2, linker for activation of T cells family member 2;
    LILRA5, leukocyte immunoglobulin-like receptor A5; LSS, lanosterol synthase; MIR142,
    microRNA 142; NINJ1, ninjurin 1; PRIM1, DNA primase subunit 1; RGS13, regulator
    of G protein signalling 13; SCO2, SCO cytochrome c oxidase assembly protein 2;
    SQLE, squalene epoxidase; TGFB1, transforming growth factor beta 1; TGM2, transglutaminase
    2; TREM1, triggering receptor expressed on myeloid cells 1. aRefers to the mature
    sequence miR-125b-2–3p, encoded in the micro RNA-125b-2 (MIR125B2) stem loop sequence,
    and includes other microRNAs with the seed sequence CAAGUCA.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CX3CR1
  - ADGRE3
  - CD300C
  - FFAR2
  - TGM2
  - CCRL2
  - TGFB1
  - TGFBI
  - HTR4
  - HDAC1
  - PLEC
  - CEACAM8
  - CXCL8
  - IL36RN
  - G0S2
  - TNF
  - DCSTAMP
  - NINJ1
  - HRG
  - NRG1
  - SLC48A1
  - CKS2
  - RGS1
  - PRIM1
  - MAFF
  - TREM1
  - MIR125B2
  - TP53
  - CD24
  - LAT2
  - SLC7A6
  - SLC7A8
  - DDIT4
  - DELYQ11
  - MIR142
  - RGS13
  - RGS18
  - LSS
  - LMBR1
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - Cx3cr1
  - Cd300c
  - Cd300c2
  - Ffar2
  - Tgm2
  - Ccrl2
  - Tgfb1
  - Tgfbi
  - Htr4
  - Hdac1-ps
  - H3b
  - Hdac1
  - Il36rn
  - G0s2
  - Tnf
  - Dcstamp
  - Ninj1
  - Hrg
  - Nrg1
  - Cks2
  - Rgs1
  - Prim1
  - Maff
  - Trem1
  - Mir125b-2
  - Trp53
  - Cd24a
  - Lat2
  - Slc7a8
  - Slc7a6
  - Ddit4
  - Mir142
  - Rgs13
  - Lss
---
