---
figid: PMC7862443__41418_2020_676_Fig2_HTML
figtitle: Met1-Ub Machinery
organisms:
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC7862443
filename: 41418_2020_676_Fig2_HTML.jpg
figlink: /pmc/articles/PMC7862443/figure/Fig2/
number: F2
caption: A The proteins involved in ‘writing’ (assembling), ‘reading’ (binding), and
  ‘erasing’ (disassembling) Met1-Ub signals. LUBAC, together with E1 and E2 enzymes,
  assembles Met1-Ub in an ATP-dependent process. The first Ub conjugated to a substrate
  (shown in grey) does not have a defined linkage as the lysine residue (K) is unique
  to the substrate. Several proteins can bind Met1-Ub signals, and these are involved
  in ‘interpreting’ the biochemical modification of the substrate and funnelling it
  into the correct cellular pathway or response. OTULIN and CYLD (in complex with
  SPATA2) are the best-described DUBs that disassemble Met1-Ub. OTULIN exclusively
  disassembles Met1-Ub whereas CYLD-SPATA2 is less specific and also disassembles
  Lys63-Ub and some other linkages in addition to Met1-Ub (this activity is not shown).
  B Domain architecture of LUBAC and associated Met1-Ub DUBs. Domain interactions
  are indicated by arrows. CAP-Gly, cytoskeleton-associated protein glycine-rich;
  IBR, in-between-RING; LDD, linear ubiquitin chain-determining domain; LTM, LUBAC-tethering
  motif; NZF, Npl4 zinc finger; OTU, ovarian tumour; PH, pleckstrin homology; PIM,
  peptide:N-glycanase/ubiquitin-associated UBA-containing or UBX-containing protein-interacting
  motif; PUB, peptide:N-glycanase/ubiquitin-associated UBA-containing or UBX-containing
  protein; RING, Really Interesting New Gene; UBA, ubiquitin-associated; UBL, ubiquitin-like;
  USP, Ubiquitin specific protease; Znf, Zinc finger.
papertitle: 'Met1-linked ubiquitin signalling in health and disease: inflammation,
  immunity, cancer, and beyond.'
reftext: Akhee Sabiha Jahan, et al. Cell Death Differ. 2021 Feb;28(2):473-492.
year: '2021'
doi: 10.1038/s41418-020-00676-w
journal_title: Cell Death and Differentiation
journal_nlm_ta: Cell Death Differ
publisher_name: Nature Publishing Group UK
keywords: Cancer | Cell death and immune response | Immune cell death | Immunological
  disorders | Signal transduction
automl_pathway: 0.955688
figid_alias: PMC7862443__F2
figtype: Figure
redirect_from: /figures/PMC7862443__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7862443__41418_2020_676_Fig2_HTML.html
  '@type': Dataset
  description: A The proteins involved in ‘writing’ (assembling), ‘reading’ (binding),
    and ‘erasing’ (disassembling) Met1-Ub signals. LUBAC, together with E1 and E2
    enzymes, assembles Met1-Ub in an ATP-dependent process. The first Ub conjugated
    to a substrate (shown in grey) does not have a defined linkage as the lysine residue
    (K) is unique to the substrate. Several proteins can bind Met1-Ub signals, and
    these are involved in ‘interpreting’ the biochemical modification of the substrate
    and funnelling it into the correct cellular pathway or response. OTULIN and CYLD
    (in complex with SPATA2) are the best-described DUBs that disassemble Met1-Ub.
    OTULIN exclusively disassembles Met1-Ub whereas CYLD-SPATA2 is less specific and
    also disassembles Lys63-Ub and some other linkages in addition to Met1-Ub (this
    activity is not shown). B Domain architecture of LUBAC and associated Met1-Ub
    DUBs. Domain interactions are indicated by arrows. CAP-Gly, cytoskeleton-associated
    protein glycine-rich; IBR, in-between-RING; LDD, linear ubiquitin chain-determining
    domain; LTM, LUBAC-tethering motif; NZF, Npl4 zinc finger; OTU, ovarian tumour;
    PH, pleckstrin homology; PIM, peptide:N-glycanase/ubiquitin-associated UBA-containing
    or UBX-containing protein-interacting motif; PUB, peptide:N-glycanase/ubiquitin-associated
    UBA-containing or UBX-containing protein; RING, Really Interesting New Gene; UBA,
    ubiquitin-associated; UBL, ubiquitin-like; USP, Ubiquitin specific protease; Znf,
    Zinc finger.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - OTULIN
  - RBCK1
  - RNF31
  - SHARPIN
  - ATP8A2
  - UBD
  - ZUP1
  - TNIP3
  - TNIP1
  - TNIP2
  - TNFAIP3
  - IGKV1-27
  - PIM1
  - PIM2
  - PIM3
  - CYLD
  - PHF20
  - ZNF629
  - ZNF44
  - ZNF436
  - ZNF501
  - ZNF569
  - ZNF763
  - BBOX1
  - ALG13
  - OTULINL
  - VCPIP1
  - ZRANB1
  - OTUB1
  - OTUB2
  - OTUD1
  - YOD1
  - OTUD3
  - OTUD4
  - OTUD5
  - OTUD6A
  - OTUD6B
  - OTUD7A
  - OTUD7B
  - SPATA2
  - RNF2
  - HSD17B8
  - RING1
  - USP1
  - USP2
  - USP3
  - USP4
  - USP5
  - USP6
  - USP7
  - USP8
  - USP10
  - USP11
  - USP12
  - USP13
  - USP14
  - USP15
  - USP16
  - USP18
  - USP19
  - USP20
  - USP21
  - USP22
  - USP24
  - USP25
  - USP26
  - USP27X
  - USP28
  - USP29
  - USP30
  - USP31
  - USP32
  - USP33
  - USP34
  - USP35
  - USP36
  - USP37
  - USP38
  - USP39
  - USP40
  - USP41P
  - USP42
  - USP43
  - USP44
  - USP45
  - USP46
  - USP47
  - USP48
  - USP49
  - USP50
  - USP51
  - PAN2
  - USP53
  - USP54
  - USP9X
  - USP9Y
  - USPL1
  - USP17L1
  - USP17L2
  - hoip
  - ATPsynbeta
  - Atpalpha
  - sub
  - neo
  - pim
  - Pub
  - otu
  - Uba1
  - ubl
  - Polr2A
  - Uch
  - Sce
  - usp
  - EcR
---
