---
figid: PMC7300384__gr2
figtitle: Epigenetic and epitranscriptomic pathways depend on metabolism
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC7300384
filename: gr2.jpg
figlink: pmc/articles/PMC7300384/figure/fig2/
number: F2
caption: 'Epigenetic and epitranscriptomic pathways depend on metabolism. Crosstalk
  between the mitochondria and nucleus is regulated by the translocation of metabolic
  enzymes, metabolites, RNAs, and epigenetic regulators. Epigenetic gene regulation
  is established by enzymes that write, read, or erase post-translational modifications
  on histones, in particular acetylation (ac) and methylation (me), and regulate the
  methylation of DNA. In a similar fashion, modifications of RNA molecules have the
  potential to regulate all RNA-dependent functions and activities. Acetyl-CoA, the
  donor of acetyl groups used by HATs, is generated in distinct pools within the cell.
  Three main pathways in the mitochondria generate acetyl-CoA: (i) fatty acid breakdown
  via the β-oxidation pathway, (ii) PDC acting on pyruvate derived from glucose through
  glycolysis, and (iii) ACSS1 acting on acetate. Mitochondrial acetyl-CoA is subsequently
  used to acetylate mitochondrial proteins or channelled into the TCA cycle to generate
  energy. In the cytoplasm, two main pathways generate acetyl-CoA: (i) by ACLY acting
  on citrate, which in turn can be generated from the TCA cycle and shuttled between
  the mitochondria and cytoplasm and (ii) by ACSS2 on acetate. The three main precursors
  of acetyl-CoA (pyruvate, acetate, and citrate) can also diffuse into the nucleus,
  where they are processed into acetyl-CoA by nuclear pools of PDC, ACSS2, and ACLY,
  respectively. Distinct pools of acetyl-CoA regulate acetylation of mitochondrial,
  cytoplasmic, and nuclear proteins, including histones. HAT-mediated histone acetylation
  results in chromatin de-condensation and active transcription (active chromatin).
  Nuclear HATs MOF and GCN5 also localise to mitochondria. For more information on
  nuclear epigenetic factors localising to mitochondria (blue), see the main text
  and . α-KG is generated in the TCA cycle in mitochondria from isocitrate by IDH2.
  Isocitrate can diffuse into the cytosol where it is converted by IDH1 to α-KG. α-KG
  diffuses into the nucleus, where it is used as a cofactor by JMJD histone demethylases,
  TET DNA demethylases, or is converted by α-KGDH into succinyl-CoA, a cofactor for
  HATs. Mitochondria regulate the redox levels of FAD, a cofactor for LSDs. During
  β-oxidation and the TCA cycle, FAD is reduced to FADH2 and returned to its oxidised
  form during OXPHOS. FAD can then diffuse into the cytosol and nucleus. SAM is the
  donor of methyl groups utilised by HMTs and DNMTs in the nucleus. SAM is generated
  through the coupling of the folate and methionine cycles in the cytosol, which in
  turn is sustained by 1-C metabolism in mitochondria or is generated via the methionine
  salvage pathway. SAM can also enter the mitochondria. Methylation of DNA and lysine
  (K) 9 of histone H3 (K9me) together with histone deacetylation are associated with
  chromatin compaction and gene repression (inactive chromatin). Sirtuin histone deacetylates
  use NAD+ as a cofactor, which is generated de novo from the amino acid tryptophan
  or via the NAD+ salvage pathway, which uses NAD+ precursors such as NAM directly
  from the diet or recycled from intracellular reactions. A nuclear pool of NAD+ is
  either generated via a nuclear NAD+ salvage pathway or through passive diffusion
  of cytoplasmic NAD+ through nuclear pores. All classes of nuclear RNAs (mRNAs, miRNAs,
  lncRNAs, rRNAs, and tRNAs) are transcribed from active chromatin and actively transported
  into the cytoplasm. miRNAs are processed in the cytoplasm and can be transported
  into the mitochondria, while mRNAs are translated at cytoplasmic ribosomes. Transcription
  in the mitochondria of RNAs (mRNAs, rRNAs, tRNAs, and perhaps miRNAs) also occurs
  with translation of the mRNAs at mito-ribosomes. Throughout the cell, all classes
  of RNAs are targeted by specific RNA-modifying enzymes (orange). Proteins synthesised
  by cytoplasmic ribosomes either reside in the cytoplasm, are imported into mitochondria
  and/or into the nucleus. During mitochondrial stress, mitochondrial GPS2 rapidly
  translocates to the nucleus where it collaborates with H3K9 demethylases to activate
  gene expression facilitating mitochondrial stress response. Also, certain nuclear
  proteins translocate to the mitochondria only during cellular stress. For example,
  both histone H3 and H1.2 are released from chromatin and translocate to the mitochondria
  during apoptosis. This figure was adapted from []. Abbreviations: 1-C: one-carbon,
  Acetyl-CoA: acetyl-coenzyme A, ACLY: ATP-citrate lyase, ACSS1: acyl-CoA synthetase
  short-chain family member 1, α-KG: α-ketoglutarate, α-KGDH: α-ketoglutarate dehydrogenase,
  ATP: adenosine triphosphate, dcSAM: decarboxylated SAM, DNMT: DNA methyltransferase,
  FAD: flavin adenine dinucleotide, GCN5: lysine acetyltransferase 2A, GPS2: G-protein
  pathway suppressor 2, HAT: histone acetyltransferase, HDAC: histone lysine deacetylase,
  HMT: histone methyltransferase, IDH2/3: isocitrate dehydrogenase 2/3, JMJD: Jumonji
  C domain-containing histone demethylase, K9: lysine 9, LSDs: lysine-specific histone
  demethylases, MOF: lysine acetyltransferase 8, MTA: 5′-methylthioadenosine, NAD+:
  nicotinamide adenine dinucleotide, NAM: nicotinamide, NAMPT: nicotinamide phosphoribosyl
  transferase, NMN: nicotinamide mononucleotide, NMNAT1/2: nicotinamide mononucleotide
  adenylyltransferase 1/2/3, OAA: oxaloacetate, OXPHOS: oxidative phosphorylation,
  PDC: pyruvate dehydrogenase complex, SAM: S-adenosyl methionine, SIRT: sirtuin deacetylase,
  TCA: tricarboxylic acid, TET: ten-eleven translocation DNA demethylase. (For interpretation
  of the references to colour in this figure legend, the reader is referred to the
  Web version of this article).'
papertitle: 'Two genomes, one cell: Mitochondrial-nuclear coordination via epigenetic
  pathways.'
reftext: Meike Wiese, et al. Mol Metab. 2020 Aug;38:100942.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9494036
figid_alias: PMC7300384__F2
figtype: Figure
redirect_from: /figures/PMC7300384__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7300384__gr2.html
  '@type': Dataset
  description: 'Epigenetic and epitranscriptomic pathways depend on metabolism. Crosstalk
    between the mitochondria and nucleus is regulated by the translocation of metabolic
    enzymes, metabolites, RNAs, and epigenetic regulators. Epigenetic gene regulation
    is established by enzymes that write, read, or erase post-translational modifications
    on histones, in particular acetylation (ac) and methylation (me), and regulate
    the methylation of DNA. In a similar fashion, modifications of RNA molecules have
    the potential to regulate all RNA-dependent functions and activities. Acetyl-CoA,
    the donor of acetyl groups used by HATs, is generated in distinct pools within
    the cell. Three main pathways in the mitochondria generate acetyl-CoA: (i) fatty
    acid breakdown via the β-oxidation pathway, (ii) PDC acting on pyruvate derived
    from glucose through glycolysis, and (iii) ACSS1 acting on acetate. Mitochondrial
    acetyl-CoA is subsequently used to acetylate mitochondrial proteins or channelled
    into the TCA cycle to generate energy. In the cytoplasm, two main pathways generate
    acetyl-CoA: (i) by ACLY acting on citrate, which in turn can be generated from
    the TCA cycle and shuttled between the mitochondria and cytoplasm and (ii) by
    ACSS2 on acetate. The three main precursors of acetyl-CoA (pyruvate, acetate,
    and citrate) can also diffuse into the nucleus, where they are processed into
    acetyl-CoA by nuclear pools of PDC, ACSS2, and ACLY, respectively. Distinct pools
    of acetyl-CoA regulate acetylation of mitochondrial, cytoplasmic, and nuclear
    proteins, including histones. HAT-mediated histone acetylation results in chromatin
    de-condensation and active transcription (active chromatin). Nuclear HATs MOF
    and GCN5 also localise to mitochondria. For more information on nuclear epigenetic
    factors localising to mitochondria (blue), see the main text and . α-KG is generated
    in the TCA cycle in mitochondria from isocitrate by IDH2. Isocitrate can diffuse
    into the cytosol where it is converted by IDH1 to α-KG. α-KG diffuses into the
    nucleus, where it is used as a cofactor by JMJD histone demethylases, TET DNA
    demethylases, or is converted by α-KGDH into succinyl-CoA, a cofactor for HATs.
    Mitochondria regulate the redox levels of FAD, a cofactor for LSDs. During β-oxidation
    and the TCA cycle, FAD is reduced to FADH2 and returned to its oxidised form during
    OXPHOS. FAD can then diffuse into the cytosol and nucleus. SAM is the donor of
    methyl groups utilised by HMTs and DNMTs in the nucleus. SAM is generated through
    the coupling of the folate and methionine cycles in the cytosol, which in turn
    is sustained by 1-C metabolism in mitochondria or is generated via the methionine
    salvage pathway. SAM can also enter the mitochondria. Methylation of DNA and lysine
    (K) 9 of histone H3 (K9me) together with histone deacetylation are associated
    with chromatin compaction and gene repression (inactive chromatin). Sirtuin histone
    deacetylates use NAD+ as a cofactor, which is generated de novo from the amino
    acid tryptophan or via the NAD+ salvage pathway, which uses NAD+ precursors such
    as NAM directly from the diet or recycled from intracellular reactions. A nuclear
    pool of NAD+ is either generated via a nuclear NAD+ salvage pathway or through
    passive diffusion of cytoplasmic NAD+ through nuclear pores. All classes of nuclear
    RNAs (mRNAs, miRNAs, lncRNAs, rRNAs, and tRNAs) are transcribed from active chromatin
    and actively transported into the cytoplasm. miRNAs are processed in the cytoplasm
    and can be transported into the mitochondria, while mRNAs are translated at cytoplasmic
    ribosomes. Transcription in the mitochondria of RNAs (mRNAs, rRNAs, tRNAs, and
    perhaps miRNAs) also occurs with translation of the mRNAs at mito-ribosomes. Throughout
    the cell, all classes of RNAs are targeted by specific RNA-modifying enzymes (orange).
    Proteins synthesised by cytoplasmic ribosomes either reside in the cytoplasm,
    are imported into mitochondria and/or into the nucleus. During mitochondrial stress,
    mitochondrial GPS2 rapidly translocates to the nucleus where it collaborates with
    H3K9 demethylases to activate gene expression facilitating mitochondrial stress
    response. Also, certain nuclear proteins translocate to the mitochondria only
    during cellular stress. For example, both histone H3 and H1.2 are released from
    chromatin and translocate to the mitochondria during apoptosis. This figure was
    adapted from []. Abbreviations: 1-C: one-carbon, Acetyl-CoA: acetyl-coenzyme A,
    ACLY: ATP-citrate lyase, ACSS1: acyl-CoA synthetase short-chain family member
    1, α-KG: α-ketoglutarate, α-KGDH: α-ketoglutarate dehydrogenase, ATP: adenosine
    triphosphate, dcSAM: decarboxylated SAM, DNMT: DNA methyltransferase, FAD: flavin
    adenine dinucleotide, GCN5: lysine acetyltransferase 2A, GPS2: G-protein pathway
    suppressor 2, HAT: histone acetyltransferase, HDAC: histone lysine deacetylase,
    HMT: histone methyltransferase, IDH2/3: isocitrate dehydrogenase 2/3, JMJD: Jumonji
    C domain-containing histone demethylase, K9: lysine 9, LSDs: lysine-specific histone
    demethylases, MOF: lysine acetyltransferase 8, MTA: 5′-methylthioadenosine, NAD+:
    nicotinamide adenine dinucleotide, NAM: nicotinamide, NAMPT: nicotinamide phosphoribosyl
    transferase, NMN: nicotinamide mononucleotide, NMNAT1/2: nicotinamide mononucleotide
    adenylyltransferase 1/2/3, OAA: oxaloacetate, OXPHOS: oxidative phosphorylation,
    PDC: pyruvate dehydrogenase complex, SAM: S-adenosyl methionine, SIRT: sirtuin
    deacetylase, TCA: tricarboxylic acid, TET: ten-eleven translocation DNA demethylase.
    (For interpretation of the references to colour in this figure legend, the reader
    is referred to the Web version of this article).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - KAT8
  - TET2
  - DNMT1
  - SIRT4
  - SELENBP1
  - HDAC1
  - SIRT3
  - HDAC7
  - HDAC9
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - ACSS2
  - GPS2
  - APCDD1
  - PDC
  - PNKD
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - POMC
  - ACLY
  - IDH1
  - OGDH
  - STAC3
  - IDH2
  - NAMPT
  - BRCA2
  - FANCD2
  - PSEN1
  - NMNAT2
  - TET
  - acetyl
  - acetyl-CoA
  - NAD
  - Glucose
  - Acetate
  - NATS
  - Pyruvate
  - POC acetyl-CoA
  - Citrate
  - Isocitrate
  - Succinyl-CoA
  - TCA
  - NAMPT
  - Polyamines
  - Methionine
  - Succinate
  - Met
  - Folate
---
