---
figid: PMC7284592__nutrients-12-01420-g001
figtitle: Two-sample Mendelian randomization analysis (2SMR) using Bonder microbiome
  and Dubois celiac disease (CeD) genome-wide association study (GWAS) as exposure
  and outcome datasets, respectively
organisms:
- gut metagenome
- metagenome
- human gut metagenome
- Homo sapiens
- Mus musculus
- Neisseria flavescens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC7284592
filename: nutrients-12-01420-g001.jpg
figlink: pmc/articles/PMC7284592/figure/nutrients-12-01420-f001/
number: F1
caption: 'Schematic representation of two-sample Mendelian randomization analysis
  (2SMR) using Bonder microbiome and Dubois celiac disease (CeD) genome-wide association
  study (GWAS) as exposure and outcome datasets, respectively. (a) Flowchart of the
  step-by-step analysis: after preparing exposure data (Step 1), outcome data is extracted
  (Step 2), both datasets are harmonized (Step 3), and 2SMR analysis is performed
  (Step 4); finally, significant hits are selected based on their false discovery
  rate (FDR) (Step 5); (b) Diagram, representing the number of single nucleotide polymorphisms
  (SNPs) selected in each category (taxa, pathway, gene ontology (GO)) after performing
  each step of the analysis.'
papertitle: A Two-Sample Mendelian Randomization Analysis Investigates Associations
  Between Gut Microbiota and Celiac Disease.
reftext: Iraia Garc√≠a-Santisteban, et al. Nutrients. 2020 May;12(5):1420.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9090961
figid_alias: PMC7284592__F1
figtype: Figure
redirect_from: /figures/PMC7284592__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7284592__nutrients-12-01420-g001.html
  '@type': Dataset
  description: 'Schematic representation of two-sample Mendelian randomization analysis
    (2SMR) using Bonder microbiome and Dubois celiac disease (CeD) genome-wide association
    study (GWAS) as exposure and outcome datasets, respectively. (a) Flowchart of
    the step-by-step analysis: after preparing exposure data (Step 1), outcome data
    is extracted (Step 2), both datasets are harmonized (Step 3), and 2SMR analysis
    is performed (Step 4); finally, significant hits are selected based on their false
    discovery rate (FDR) (Step 5); (b) Diagram, representing the number of single
    nucleotide polymorphisms (SNPs) selected in each category (taxa, pathway, gene
    ontology (GO)) after performing each step of the analysis.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Go
  - ld
  - 'On'
  - Snp
---
