---
figid: PMC7522539__gr1
figtitle: Conceptual systems framework in NPDs (BD/SZ/MD), and lithium treatment amelioration
  of mitochondrial (mt) functions in distinct pathways
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC7522539
filename: gr1.jpg
figlink: pmc/articles/PMC7522539/figure/f0005/
number: F1
caption: Conceptual systems framework in NPDs (BD/SZ/MD), and lithium treatment amelioration
  of mitochondrial (mt) functions in distinct pathways. (A) Genetic variants from
  GWAS will allow the discovery of NPD-associated mt risk genes. A CRISPR-based chemical-genetic
  interaction screening can unveil NPD-relevant mt targets inhibited by a small molecule.
  Here, knockouts (or knockdowns) of the interacting gene of NPD-associated mt risk
  factors is lethal when cells are treated with small molecule. Artificial intelligence
  (AI) methods, such as deep learning can be employed towards de novo drug discovery
  and combinatorial drug screening. By utilizing iPSC-derived neurons or 3D brain
  organoids from NPD patients vs. healthy subjects, loss (dotted lines)-or gain (thick
  lines) of protein–protein interactions (PPIs), gene expression in distinct cell
  states (OPC, oligodendrocytes; NPCs, Neural progenitor cells), and genetic interaction
  networks (i.e. two functionally linked single gene knockouts show lethal phenotype)
  can be mapped to pinpoint specific or common pathological macromolecular complexes
  (identified from PPI networks) or pathways involved in the pathophysiology of BD/SZ/MD.
  Integration of PPIs with co-expression profiles from scRNA-seq should uncover functional
  modules and driver genes of NPDs. Notably, a gene knockout that show chemical-genetic
  interaction with a small molecule should display lethal with a mutation in the drug
  target gene. Thus, comparing genetic interactions with chemical-genetic interaction
  profiles will identify mt pathway targets modulated by small molecule. Collectively,
  integration of these orthogonal datasets in various combinations should generate
  a higher order pathway-level understanding of NPD-related mt genes and in the discovery
  of disease biomarkers and drug targets. (B) Lithium inhibits GSK-3 pathway, allowing
  the translocation of β-catenin to the nucleus and transcription of pro-survival
  factors. Lithium prevents mt translocation of GSK-3 and improves oxidative phosphorylation
  (OXPHOS). (C) Lithium impedes the phosphoinositol cycle, reducing myoinositol levels
  and, consequently, blocking the activation of PKC pathway. Inhibition of this pathway
  prevents destabilization of neuronal membranes, increased long-term potentiation,
  and induction of apoptosis. (D) BCL-2 and BDNF factors are improved via ERK activation.
  Increased expression of these two proteins ameliorates neurotransmission, avoids
  mt Ca2+ overload and prevents cytochrome c release.
papertitle: 'Mitochondria under the spotlight: On the implications of mitochondrial
  dysfunction and its connectivity to neuropsychiatric disorders.'
reftext: Mara Zilocchi, et al. Comput Struct Biotechnol J. 2020;18:2535-2546.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8387482
figid_alias: PMC7522539__F1
figtype: Figure
redirect_from: /figures/PMC7522539__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7522539__gr1.html
  '@type': Dataset
  description: Conceptual systems framework in NPDs (BD/SZ/MD), and lithium treatment
    amelioration of mitochondrial (mt) functions in distinct pathways. (A) Genetic
    variants from GWAS will allow the discovery of NPD-associated mt risk genes. A
    CRISPR-based chemical-genetic interaction screening can unveil NPD-relevant mt
    targets inhibited by a small molecule. Here, knockouts (or knockdowns) of the
    interacting gene of NPD-associated mt risk factors is lethal when cells are treated
    with small molecule. Artificial intelligence (AI) methods, such as deep learning
    can be employed towards de novo drug discovery and combinatorial drug screening.
    By utilizing iPSC-derived neurons or 3D brain organoids from NPD patients vs.
    healthy subjects, loss (dotted lines)-or gain (thick lines) of protein–protein
    interactions (PPIs), gene expression in distinct cell states (OPC, oligodendrocytes;
    NPCs, Neural progenitor cells), and genetic interaction networks (i.e. two functionally
    linked single gene knockouts show lethal phenotype) can be mapped to pinpoint
    specific or common pathological macromolecular complexes (identified from PPI
    networks) or pathways involved in the pathophysiology of BD/SZ/MD. Integration
    of PPIs with co-expression profiles from scRNA-seq should uncover functional modules
    and driver genes of NPDs. Notably, a gene knockout that show chemical-genetic
    interaction with a small molecule should display lethal with a mutation in the
    drug target gene. Thus, comparing genetic interactions with chemical-genetic interaction
    profiles will identify mt pathway targets modulated by small molecule. Collectively,
    integration of these orthogonal datasets in various combinations should generate
    a higher order pathway-level understanding of NPD-related mt genes and in the
    discovery of disease biomarkers and drug targets. (B) Lithium inhibits GSK-3 pathway,
    allowing the translocation of β-catenin to the nucleus and transcription of pro-survival
    factors. Lithium prevents mt translocation of GSK-3 and improves oxidative phosphorylation
    (OXPHOS). (C) Lithium impedes the phosphoinositol cycle, reducing myoinositol
    levels and, consequently, blocking the activation of PKC pathway. Inhibition of
    this pathway prevents destabilization of neuronal membranes, increased long-term
    potentiation, and induction of apoptosis. (D) BCL-2 and BDNF factors are improved
    via ERK activation. Increased expression of these two proteins ameliorates neurotransmission,
    avoids mt Ca2+ overload and prevents cytochrome c release.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Akt
  - bd
  - md
  - E(spl)mdelta-HLH
  - sgg
  - CrebB
  - CrebA
  - arm
  - vs
  - de
  - ca
  - norpA
  - sl
  - Plc21C
  - inaC
  - Pkc98E
  - aPKC
  - Pkc53E
  - Las
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Erk7
  - rl
  - Debcl
  - AKT1
  - AKT2
  - AKT3
  - GSK3A
  - GSK3B
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - CTNNB1
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - MARCKS
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - LIAS
  - EPHB2
  - MAPK1
  - MAPK3
  - BDNF
  - BDNF-AS
  - BCL2
  - Lithium
  - Al
  - Ca
  - Myoinositol
  - O Gene
  - calcium
---
