---
figid: PMC7915224__biomolecules-11-00236-g006a
figtitle: Ins and Outs of RAS Effector Complexes
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC7915224
filename: biomolecules-11-00236-g006a.jpg
figlink: .na.character
number: F6a
caption: Signaling capacities of known RAS effector pathway components at different
  HRAS localizations superimposed on diagrams of these pathways. Signaling pathways
  activated when HRASV12 is located at the (A) disordered membrane (DM); (B) lipid
  rafts (LR); (C) endoplasmic reticulum (ER); and (D) Golgi apparatus (GA). The nodes
  displayed in bright colors are present in the corresponding network. The red circle
  adjacent to each node shows the number of interactions it participates in. In cases
  where a node represents more than one protein (isoforms or homologs), the signaling
  capacity of the node is represented by the total number of interactions involving
  all proteins corresponding to the node. The enriched GO terms of the HRAS localization
  specific transcriptome are shown at bottom of each pathway diagram. This figure
  is from the supplementary data of [].
papertitle: The Ins and Outs of RAS Effector Complexes.
reftext: Christina Kiel, et al. Biomolecules. 2021 Feb;11(2):236.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8473491
figid_alias: PMC7915224__F6a
figtype: Figure
redirect_from: /figures/PMC7915224__F6a
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7915224__biomolecules-11-00236-g006a.html
  '@type': Dataset
  description: Signaling capacities of known RAS effector pathway components at different
    HRAS localizations superimposed on diagrams of these pathways. Signaling pathways
    activated when HRASV12 is located at the (A) disordered membrane (DM); (B) lipid
    rafts (LR); (C) endoplasmic reticulum (ER); and (D) Golgi apparatus (GA). The
    nodes displayed in bright colors are present in the corresponding network. The
    red circle adjacent to each node shows the number of interactions it participates
    in. In cases where a node represents more than one protein (isoforms or homologs),
    the signaling capacity of the node is represented by the total number of interactions
    involving all proteins corresponding to the node. The enriched GO terms of the
    HRAS localization specific transcriptome are shown at bottom of each pathway diagram.
    This figure is from the supplementary data of [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ras
  - Ras64B
  - Ras85D
  - Rac1
  - Rac2
  - Akt
  - Mtl
  - Pak
  - mbt
  - Erk7
  - rl
  - Rgl
  - Raf
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - inaC
  - Pkc98E
  - aPKC
  - Pkc53E
  - lic
  - ksr
  - Abl
  - Rab4
  - Rab5
  - Dsor1
  - Mtk
  - cno
  - wts
  - p38b
  - Nurf-38
  - Ebp
  - AIMP2
  - p38a
  - Hrb87F
  - p38c
  - Mtor
  - Tor
  - yki
  - Myc
  - Stat92E
  - foxo
  - IKKepsilon
  - key
  - IKKbeta
  - RtGEF
  - RhoGEF2
  - RhoGEF3
  - RhoGEF64C
  - RhoGEF4
  - sif
  - dap
  - DCTN6-p27
  - CG9588
  - tefu
  - Dif
  - dl
  - Rel
  - mei-41
  - put
  - RYa-R
  - p53
  - betaTub60D
  - hth
  - Fas2
  - FASN1
  - Fas3
  - fand
  - tei
  - rg
  - FASN2
  - FASN3
  - sgg
  - Axn
  - gskt
  - Cdk2
  - Cdk4
  - Fadd
  - lok
  - rb
  - Traf6
  - Traf4
  - E2f1
  - E2f2
  - unk
  - sip2
  - grp
  - Rap1
  - norpA
  - sl
  - Plc21C
  - levy
  - COX6AL2
  - Rala
  - r-l
  - rai
  - rho
  - Bx
  - Rho1
  - Lcp65Ab1
  - CycE
  - cyc
  - hpo
  - Mst75Ca
  - Mst75Cb
  - KRAS
  - HRAS
  - NRAS
  - AKT1
  - RAC1
  - RAC2
  - RAC3
  - RHOG
  - PAK1
  - PAK2
  - PAK3
  - PAK4
  - PAK5
  - PAK6
  - AKT2
  - AKT3
  - EPHB2
  - MAPK1
  - MAPK3
  - RALGDS
  - MAP3K11
  - SHC4
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - IMPA1
  - BRAP
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - MAP2K3
  - MAP2K6
  - KSR1
  - KSR2
  - ABL1
  - MTTP
  - DSP-AS1
  - ABL2
  - RAB4A
  - RAB5A
  - MAP2K7
  - MAP2K1
  - MAP2K2
  - AFDN
  - LATS1
  - LATS2
  - CRK
  - MAPK14
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAPK11
  - MAPK12
  - MAPK13
  - MTOR
  - YAP1
  - MYC
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - FOXO4
  - CHUK
  - IKBKB
  - IKBKG
  - IKBKE
  - TBK1
  - CDKN1A
  - TCEAL1
  - NSG1
  - H3P16
  - IFI27
  - PSMD9
  - ZNRD2
  - DCTN6
  - TMED7
  - H3P23
  - ATM
  - FASLG
  - NFKB1
  - RELA
  - RELB
  - REL
  - NFKB2
  - ATR
  - ANTXR1
  - MMAB
  - SERPINA2
  - TP53
  - TP63
  - TP73
  - FAS
  - FASN
  - GSK3B
  - CDK2
  - CDK6
  - CDK4
  - FADD
  - CHEK2
  - TRAF1
  - TRAF2
  - TRAF3
  - TRAF4
  - TRAF5
  - TRAF6
  - TRAF7
  - E2F4
  - E2F3
  - E2F7
  - E2F5
  - E2F1
  - UBE2L3
  - E2F2
  - E2F6
  - UNK
  - USF2
  - ERC1
  - CHEK1
  - RAP1A
  - RABGEF1
  - TERF2IP
  - RAP1B
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - SEMA6A
  - RALA
  - RALB
  - GDF1
  - RNH1
  - PPP1R13L
  - SHC3
  - RHO
  - RHOD
  - RHOA
  - RHOB
  - RHOC
  - STK3
  - O
  - tyrosine
  - DM
---
