---
figid: PMC7265486__41540_2020_136_Fig1_HTML
figtitle: Stratification and prediction of drug synergy based on target functional
  similarity
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC7265486
filename: 41540_2020_136_Fig1_HTML.jpg
figlink: pmc/articles/PMC7265486/figure/Fig1/
number: F1
caption: a (i) First, we compute activities scores for 11 pathways from gene expression
  of cancer cell lines. It consists in multiplying the transcriptomics data by a loading
  matrix using PROGENy. (ii) We then use the Macau algorithm to predict multiple drugs’
  responses simultaneously by uncovering the common (latent) features that can benefit
  each individual learning task. We use as input features (side information) the PROGENy
  pathway scores) for the cell lines and the nominal target for the drugs. Each side
  information matrix is transformed into a matrix of L latent dimensions by a link
  matrix. Drug response is then computed by a matrix multiplication of the two latent
  matrices. (iii) Concurrently to drug response prediction, we derive the interactions
  between drug features (targets) and cell line features (pathway activity), by multiplying
  the two link matrices. An association between protein X and pathway Y means that
  activation of pathway Y correlates with drug sensitivity when targeting protein
  X. In case of causality, we can say that activation of pathway Y confers sensitivity
  to any drug targeting protein X. The approach behind steps (i–iii) is described
  in detail in Yang et al. (iv) These interactions allow us to define the functional
  similarity between two target proteins. In this example of breast tissue, the functional
  similarity between proteins EGFR and AKT1 is the correlation of their interaction
  values with the 11 PROGENy pathways. As the final step of the synergy prediction
  workflow, the derived target functional similarity informs us about the likelihood
  of synergy. (v) We use the Target functional similarity metric for compound prioritization.
  b For synergy stratification workflow, we start with target pairs already known
  to be synergistic. The value of the functional similarity between the target proteins
  reflects different synergy mechanisms. If the similarity is close to 1, synergy
  occurs by targeting the same signaling pathways. A similarity close to −1 suggests
  a synergy induced by compensation of escape mechanism. We build specific synergy
  models for each case to predict synergy scores of cancer cell lines. Long arrows
  denote transition to a new step and short arrows indicate direction of matrix multiplications.
papertitle: Stratification and prediction of drug synergy based on target functional
  similarity.
reftext: Mi Yang, et al. NPJ Syst Biol Appl. 2020;6:16.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6695093
figid_alias: PMC7265486__F1
figtype: Figure
redirect_from: /figures/PMC7265486__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7265486__41540_2020_136_Fig1_HTML.html
  '@type': Dataset
  description: a (i) First, we compute activities scores for 11 pathways from gene
    expression of cancer cell lines. It consists in multiplying the transcriptomics
    data by a loading matrix using PROGENy. (ii) We then use the Macau algorithm to
    predict multiple drugs’ responses simultaneously by uncovering the common (latent)
    features that can benefit each individual learning task. We use as input features
    (side information) the PROGENy pathway scores) for the cell lines and the nominal
    target for the drugs. Each side information matrix is transformed into a matrix
    of L latent dimensions by a link matrix. Drug response is then computed by a matrix
    multiplication of the two latent matrices. (iii) Concurrently to drug response
    prediction, we derive the interactions between drug features (targets) and cell
    line features (pathway activity), by multiplying the two link matrices. An association
    between protein X and pathway Y means that activation of pathway Y correlates
    with drug sensitivity when targeting protein X. In case of causality, we can say
    that activation of pathway Y confers sensitivity to any drug targeting protein
    X. The approach behind steps (i–iii) is described in detail in Yang et al. (iv)
    These interactions allow us to define the functional similarity between two target
    proteins. In this example of breast tissue, the functional similarity between
    proteins EGFR and AKT1 is the correlation of their interaction values with the
    11 PROGENy pathways. As the final step of the synergy prediction workflow, the
    derived target functional similarity informs us about the likelihood of synergy.
    (v) We use the Target functional similarity metric for compound prioritization.
    b For synergy stratification workflow, we start with target pairs already known
    to be synergistic. The value of the functional similarity between the target proteins
    reflects different synergy mechanisms. If the similarity is close to 1, synergy
    occurs by targeting the same signaling pathways. A similarity close to −1 suggests
    a synergy induced by compensation of escape mechanism. We build specific synergy
    models for each case to predict synergy scores of cancer cell lines. Long arrows
    denote transition to a new step and short arrows indicate direction of matrix
    multiplications.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - anon-70Dc
  - gm
  - pn
  - Ssb-c31a
  - pll
  - Hrb87F
  - 'On'
  - CG17528
  - pk
  - Hug
  - Egfr
  - MKP-4
  - p38b
  - rl
  - Dif
  - dl
  - Rel
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - dpp
  - gbb
  - put
  - mav
  - egr
  - tra2
  - Pvf1
  - Pvf2
  - Pvf3
  - Pvr
  - p53
  - betaTub60D
  - hth
  - link
  - be
  - Akt
  - bo
  - tral
  - Nipped-A
  - Stat92E
  - hop
  - bsk
  - dpa
  - Dl
  - .na.character
  - Tapdelta
  - deltaTry
  - Pa
  - WARS1
  - S100A10
  - ENDOU
  - EXOSC6
  - MPEG1
  - PKLR
  - PROK1
  - PROK2
  - PKN1
  - PKN2
  - PKN3
  - EGFR
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - NFKB1
  - RELA
  - RELB
  - REL
  - NFKB2
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - TGFB1
  - TGFB2
  - TGFB3
  - TNF
  - TNFSF10
  - VEGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - PGF
  - TP53
  - TP63
  - TP73
  - AKT1
  - AKT2
  - BOC
  - THRA
  - HSP90B1
  - TRRAP
  - PLSCR4
  - SOAT1
  - STAT1
  - STAT2
  - STAT3
  - STAT4
  - STAT6
  - STAT5A
  - STAT5B
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - PSMB6
  - YY1
  - DLL1
  - THOP1
  - LINC00273
  - TOP1
  - TOP1MT
  - TOP3A
  - TOP3B
  - TOP2A
  - TOP2B
  - Be
---
