---
figid: PMC7901476__mSystems.00786-20-f006
figtitle: Exemplary metabolic sketch of BSC Rubrobacteraceae species based on the
  most abundant MAG
organisms:
- Escherichia coli
- Streptomyces coelicolor
- Myxococcus xanthus
- phototrophic bacterium
- Deinococcus radiodurans
- Actinomycetia bacterium
- Gluconacetobacter diazotrophicus
- Rubrobacter xylanophilus
- Rubrobacter radiotolerans
- Microvirga vignae
- Rubrobacter calidifluminis
- Rubrobacter naiadicus
- Rubrobacter taiwanensis
- Rubrobacter indicoceani
- Rubrobacter aplysinae
- metagenome
- metagenomes
- uncultured bacterium
- human oral metagenome
- uncultured Planctomycetota bacterium
- Matthiola incana
- Vigna unguiculata
- Bryophyta
- Erythrina poeppigiana
- Homo sapiens
- Epirrhoe alternata
pmcid: PMC7901476
filename: mSystems.00786-20-f006.jpg
figlink: pmc/articles/PMC7901476/figure/fig6/
number: F6
caption: Exemplary metabolic sketch of BSC Rubrobacteraceae species based on the most
  abundant MAG. The sketch illustrates basic energy generation and C acquisition mechanisms,
  as well as some stress resistance and survival mechanisms. Rubrobacteraceae encode
  genetic potential for a mixotrophic lifestyle with rhodopsin and a high-affinity
  hydrogenase to generate proton motive force. The exact functions of CO dehydrogenase
  and propane monooxygenase homologs are yet to be determined. The C metabolism is
  optimized for efficient C utilization by using phosphotransferase sugar import systems
  (PTS) and glyoxylate bypass of the TCA cycle. It is important to note the absence
  of terminal cytochrome c oxidase in the respiratory chain and exclusive reliance
  on the manganese-based catalase for combating reactive oxygen species (ROS). DNA-binding
  rubrerythrin might also contribute to preventing ROS damage to DNA. Also note the
  presence of an additional homolog of PolA polymerase (COG0749), key in double-stranded
  DNA break repair. For pathway evaluation, RAST annotations were analyzed with Pathway
  Tools v.24 (), and missing enzymes, usually due to the absence of an EC number in
  the annotation or the use of an uncommon synonymous enzyme name, were searched manually
  in RAST and other annotations of the MAGs. The depicted pathways are simplified
  versions of pathway depictions in the MetaCyc database () used by Pathway Tools
  software.
papertitle: Distribution of Mixotrophy and Desiccation Survival Mechanisms across
  Microbial Genomes in an Arid Biological Soil Crust Community.
reftext: Dimitri V. Meier, et al. mSystems. 2021 Jan-Feb;6(1):e00786-20.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9279211
figid_alias: PMC7901476__F6
figtype: Figure
organisms_ner:
- NA
redirect_from: /figures/PMC7901476__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7901476__mSystems.00786-20-f006.html
  '@type': Dataset
  description: Exemplary metabolic sketch of BSC Rubrobacteraceae species based on
    the most abundant MAG. The sketch illustrates basic energy generation and C acquisition
    mechanisms, as well as some stress resistance and survival mechanisms. Rubrobacteraceae
    encode genetic potential for a mixotrophic lifestyle with rhodopsin and a high-affinity
    hydrogenase to generate proton motive force. The exact functions of CO dehydrogenase
    and propane monooxygenase homologs are yet to be determined. The C metabolism
    is optimized for efficient C utilization by using phosphotransferase sugar import
    systems (PTS) and glyoxylate bypass of the TCA cycle. It is important to note
    the absence of terminal cytochrome c oxidase in the respiratory chain and exclusive
    reliance on the manganese-based catalase for combating reactive oxygen species
    (ROS). DNA-binding rubrerythrin might also contribute to preventing ROS damage
    to DNA. Also note the presence of an additional homolog of PolA polymerase (COG0749),
    key in double-stranded DNA break repair. For pathway evaluation, RAST annotations
    were analyzed with Pathway Tools v.24 (), and missing enzymes, usually due to
    the absence of an EC number in the annotation or the use of an uncommon synonymous
    enzyme name, were searched manually in RAST and other annotations of the MAGs.
    The depicted pathways are simplified versions of pathway depictions in the MetaCyc
    database () used by Pathway Tools software.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATP
  - ubiquinol
  - trehalose
  - Quinol
  - ADP
  - NADH 2 NAD
  - sugar
  - pyruvate
  - Acetyl-CoA
  - Phosphate
  - NADH
  - NAD
  - Oxygen
  - amino acids
  - TCA
  - glyoxylate
  - L-malate
  - Manganese
  - isocitrate
  - Mn
  - NADP
  - ANADPH
  - Sucoinyl-CaA
  - Propanoyl
  - HCO
---
