---
figid: PMC7572606__nihms-1625709-f0002
figtitle: MYC transcriptionally upregulates the glutamine metabolism pathway
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Danio rerio
pmcid: PMC7572606
filename: nihms-1625709-f0002.jpg
figlink: pmc/articles/PMC7572606/figure/F1/
number: F1
caption: (A) Analysis of MYC ChIP-seq data identifies the highly enriched MYC consensus
  binding motif as CACGTG. DNA sequences of 200 bp flanking each MYC ChIP-seq peak
  were extracted from ChIP-seq data of OVCAR3 and MCF7 cells. (B) Relative mRNA expression
  of MYC and MYC target genes in OVCAR3 following MYC knockdown using two independent
  siRNAs. Transcription levels of each gene were quantified by RT-qPCR, and data were
  normalized to a reference gene, APP. Relative expression was further normalized
  to data obtained from scramble control RNA-treated cells. *p < 0.05; **p < 0.01;
  ***p < 0.001; n=3, Student’s t-test. (C) Schema showing MYC-regulated enzymes that
  are involved in metabolic pathways. TCA cycle - tricarboxylic acid cycle; OAA -
  oxaloacetate; GSH - glutathione; UMP - Uridine monophosphate; CTP - cytidine triphosphate;
  AICAR – aminoimidazole carboxamide ribotide, IMP - Inosine 5'-monophosphate, XMP
  - xanthine monophosphate; dGTP - Deoxyguanosine triphosphate. (D) Ingenuity® Pathway
  Analysis performed on MYC target genes identified by overlaying genes differentially
  expressed in MYC knockdown cells with genes whose promoters and regulatory regions
  were bound by MYC. Plots show the top three upstream network hubs and molecules
  in each network. Molecules indicated in green and red correspond to downregulated
  and upregulated genes following MYC knockdown, respectively. (E) Analysis of IPA
  data of canonical pathways showing enrichment of MYC target genes. The x-axis represents
  the negative log p value. Only the top 10 pathways with the most significant negative
  values are shown. Blue, red, and gray bars indicate pathway down-, up- or un-regulated
  in response to MYC knockdown.
papertitle: Inhibition of the MYC-regulated glutaminase metabolic axis is an effective
  synthetic lethal approach for treating chemoresistant cancers.
reftext: Yao-An Shen, et al. Cancer Res. ;80(20):4514-4526.
year: ''
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.514468
figid_alias: PMC7572606__F1
figtype: Figure
redirect_from: /figures/PMC7572606__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7572606__nihms-1625709-f0002.html
  '@type': Dataset
  description: (A) Analysis of MYC ChIP-seq data identifies the highly enriched MYC
    consensus binding motif as CACGTG. DNA sequences of 200 bp flanking each MYC ChIP-seq
    peak were extracted from ChIP-seq data of OVCAR3 and MCF7 cells. (B) Relative
    mRNA expression of MYC and MYC target genes in OVCAR3 following MYC knockdown
    using two independent siRNAs. Transcription levels of each gene were quantified
    by RT-qPCR, and data were normalized to a reference gene, APP. Relative expression
    was further normalized to data obtained from scramble control RNA-treated cells.
    *p < 0.05; **p < 0.01; ***p < 0.001; n=3, Student’s t-test. (C) Schema showing
    MYC-regulated enzymes that are involved in metabolic pathways. TCA cycle - tricarboxylic
    acid cycle; OAA - oxaloacetate; GSH - glutathione; UMP - Uridine monophosphate;
    CTP - cytidine triphosphate; AICAR – aminoimidazole carboxamide ribotide, IMP
    - Inosine 5'-monophosphate, XMP - xanthine monophosphate; dGTP - Deoxyguanosine
    triphosphate. (D) Ingenuity® Pathway Analysis performed on MYC target genes identified
    by overlaying genes differentially expressed in MYC knockdown cells with genes
    whose promoters and regulatory regions were bound by MYC. Plots show the top three
    upstream network hubs and molecules in each network. Molecules indicated in green
    and red correspond to downregulated and upregulated genes following MYC knockdown,
    respectively. (E) Analysis of IPA data of canonical pathways showing enrichment
    of MYC target genes. The x-axis represents the negative log p value. Only the
    top 10 pathways with the most significant negative values are shown. Blue, red,
    and gray bars indicate pathway down-, up- or un-regulated in response to MYC knockdown.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ripk1
  - Myc
  - Nol3
  - Tnfrsf11a
  - Dob2
  - Ubd
  - Gsr
  - Gusr
  - Sae1
  - Myd88
  - Jun
  - Sumo3
  - Pias3
  - Idh2
  - Lamp3
  - Ctcfl
  - Fbl
  - Nfkbia
  - Stat2
  - Vamp3
  - Plaur
  - Oas3
  - Tph1
  - Parp12
  - gp
  - Zc3hav1
  - Adar
  - Mvp
  - Fos
  - Csde1
  - Acaca
  - Sp1
  - Hk2
  - Tsku
  - Nop56
  - Smarcc1
  - Elk1
  - Kcnh8
  - Uba2
  - Abca3
  - Tff1
  - Atf1
  - Txnip
  - Cad
  - Dffb
  - Acod1
  - Cgct
  - Phb1
  - Psme2
  - Btnl4
  - Samhd1
  - Trim25
  - Bst2
  - Krt17
  - Ogfr
  - Eif2s2
  - Mtor
  - Elf4
  - Eif4a1
  - Eif4a2
  - Rps6kb1
  - Atic
  - Rrm2
  - Slc25a10
  - Slc25a11
  - Gls
  - Gls2
  - Shmt2
  - Paics
  - Nme1
  - Adsl
  - oh
  - Pfas
  - Gnat2
  - Brap
  - Emp2
  - Slc25a1
  - Slc25a39
  - RIPK1
  - MYC
  - HOMER1
  - HOMER2
  - HOMER3
  - TNFRSF11A
  - UBD
  - GSR
  - SAE1
  - MYD88
  - FOS
  - FOSB
  - JUN
  - JUNB
  - JUND
  - SUMO3
  - SUMO2
  - PIAS3
  - IDH2
  - LAMP3
  - CTCFL
  - FBL
  - NFKBIA
  - STAT2
  - VAMP3
  - PLAUR
  - OAS3
  - PARP12
  - ZC3HAV1
  - ADAR
  - NKRF
  - MVP
  - MMVP1
  - HOTAIR
  - FOSL1
  - FOSL2
  - CSDE1
  - ACACA
  - SP1
  - PSG1
  - DAND5
  - HK2
  - KCNA5
  - KIF2A
  - KLK2
  - HOOK2
  - TSKU
  - NOP56
  - SMARCC1
  - ELK1
  - KCNH4
  - KCNH8
  - UBA2
  - ABCA3
  - TFF1
  - ATF1
  - GDNF
  - TXNIP
  - CAD
  - DFFB
  - ACOD1
  - PHB1
  - PSME2
  - HLA-E
  - BTN3A3
  - SAMHD1
  - TRIM25
  - BST2
  - KRT17
  - OGFR
  - EIF2S1
  - EIF2S3
  - EIF2S2
  - MTOR
  - ELF4
  - EIF4G1
  - EIF4G2
  - EIF4G3
  - EIF4E
  - EIF4E2
  - EIF4E3
  - EIF4A1
  - EIF4A2
  - EIF4A3
  - RPS6KB1
  - RPS6KB2
  - ATIC
  - RRM2
  - SLC25A10
  - SLC25A11
  - GLS
  - GLS2
  - SHMT2
  - PAICS
  - NME1
  - RMRP
  - IFNK
  - IFNW1
  - IFNA1
  - IFNA2
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA17
  - IFNA21
  - IFNB1
  - ADSL
  - TRIM63
  - PFAS
  - IMPA1
  - BRAP
  - EMP2
  - SLC25A1
  - SLC25A39
  - Ippk
  - Ctse
  - Fnta
  - Ccne1
  - Pcyt1a
  - myca
  - mycbp2
  - tnfrsf1b
  - gsr
  - sae1
  - myd88
  - sumo3a
  - sumo3b
  - idh2
  - lamp3
  - fbl
  - nfkbiab
  - stat2
  - jun
  - tph1a
  - adar
  - nkrf
  - mvp
  - fosab
  - csde1
  - acaca
  - sp1
  - hk2
  - tsku
  - nop56
  - smarcc1a
  - elk1
  - uba2
  - atf1
  - txnipa
  - cad
  - dffb
  - cald1b
  - phb
  - psme2
  - samhd1
  - trim25
  - ftr34
  - krt17
  - ogfr
  - ogfr2
  - eif2s1b
  - mtor
  - atic
  - rrm2
  - rrm2b
  - slc25a10b
  - slc25a11
  - glsb
  - shmt2
  - paics
  - nme2b.1
  - adsl
  - pfas
  - cmpk
  - slc25a39
  - PHB
  - Glutamine
  - Purine
  - Ribose-5-phosphate
  - 5-phopho-ribosy-N
  - formylglycineamide
  - Glutamate
  - Purine Nucleotides
  - Citrate
  - KG a-KG
  - pSer
  - Glycine GSH
  - Iso-citrate
  - TCA
  - Pyrimidine
  - Aspartate
  - Cystine
  - Dihydroorotate
  - succinate
  - FITS
  - TSKU
---
