---
figid: PMC7140926__PP_201901118DR1_f6
figtitle: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization
organisms:
- NA
pmcid: PMC7140926
filename: PP_201901118DR1_f6.jpg
figlink: pmc/articles/PMC7140926/figure/fig6/
number: F6
caption: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization.
  Integrated metabolomic and transcriptomic network analysis of metabolite amounts
  and transcript abundance between Osgs1;1 mutant roots and wild type at the third-leaf
  stage. The enriched metabolomic/transcriptomic network highlights the most regulated
  subnetwork in the mutants in comparison with wild type, based on the GAM web application
  (; https://artyomovlab.wustl.edu/shiny/gam/). Metabolites and mRNAs whose levels
  were increased/upregulated (log2 FC > 0) or decreased/downregulated (log2 FC < 0)
  in the mutants versus wild type are represented as red (log2 FC > 0) or green (log2
  FC < 0) nodes (metabolites) and links (reactions), respectively. Metabolite names
  with italic characters represent metabolites that were undetectable by gas chromatography-mass
  spectrometry-based metabolite profiling. Larger nodes and wider links indicate lower
  FDR values. Solid lines represent direct enzyme reactions between metabolites, whereas
  dashed lines indicate “main” reaction formula in one-to-one relationships of substrate-product
  pairs in the KEGG pathway maps for Arabidopsis. Gene names and metabolite names
  are listed in Supplemental Tables S8 and S9. We used transcript profiles of mutant
  NC2373 to represent Osgs1;1 mutant for the network analysis.
papertitle: Cytosolic GLUTAMINE SYNTHETASE1;1 Modulates Metabolism and Chloroplast
  Development in Roots.
reftext: Miyako Kusano, et al. Plant Physiol. 2020 Apr;182(4):1894-1909.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6672875
figid_alias: PMC7140926__F6
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC7140926__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7140926__PP_201901118DR1_f6.html
  '@type': Dataset
  description: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization.
    Integrated metabolomic and transcriptomic network analysis of metabolite amounts
    and transcript abundance between Osgs1;1 mutant roots and wild type at the third-leaf
    stage. The enriched metabolomic/transcriptomic network highlights the most regulated
    subnetwork in the mutants in comparison with wild type, based on the GAM web application
    (; https://artyomovlab.wustl.edu/shiny/gam/). Metabolites and mRNAs whose levels
    were increased/upregulated (log2 FC > 0) or decreased/downregulated (log2 FC <
    0) in the mutants versus wild type are represented as red (log2 FC > 0) or green
    (log2 FC < 0) nodes (metabolites) and links (reactions), respectively. Metabolite
    names with italic characters represent metabolites that were undetectable by gas
    chromatography-mass spectrometry-based metabolite profiling. Larger nodes and
    wider links indicate lower FDR values. Solid lines represent direct enzyme reactions
    between metabolites, whereas dashed lines indicate “main” reaction formula in
    one-to-one relationships of substrate-product pairs in the KEGG pathway maps for
    Arabidopsis. Gene names and metabolite names are listed in Supplemental Tables
    S8 and S9. We used transcript profiles of mutant NC2373 to represent Osgs1;1 mutant
    for the network analysis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PLPB
  - FTA
  - RSR4
  - NIT4
  - AAT
  - TSB1
  - PGLP1
  - GLC
  - HXK2
  - CSY2
  - HCEF1
  - ACO1
  - FBA2
  - ATTPS6
  - ALDH11A3
  - ICL
  - IDH1
  - TPP
  - AGT2
  - Asn
  - Glu
  - Asp
  - Trp
  - Ser
  - TPP
---
