---
figid: PMC7494788__fbioe-08-00914-g0001
figtitle: Schematic of data and resource driven strategy for microalgal bioengineering
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Embryophyta
- PX clade
- Medicago truncatula
- Helianthus annuus
- Spinacia oleracea
- Chlamydomonas reinhardtii
- Chlorophyta
- Lotus japonicus
- Phaeodactylum tricornutum
- Nitzschia alba
- Chlorella vulgaris
- Haematococcus lacustris
- Thalassiosira pseudonana
- Ostreococcus tauri
- Dunaliella salina
- Chlorella sorokiniana
- Cyanidioschyzon merolae
- Emiliania huxleyi
- Volvox carteri
- Galdieria sulphuraria
- Chlorella sp.
- Tetradesmus obliquus
- Chlorella variabilis
- Ettlia oleoabundans
- Chromochloris zofingiensis
- Nannochloropsis oculata
- Nannochloropsis gaditana
- Auxenochlorella pyrenoidosa
- Nannochloropsis oceanica
- Auxenochlorella protothecoides
- Botryococcus braunii
- Breviolum minutum
- Karenia brevis
- Fistulifera solaris
- Parachlorella kessleri
- Fragilariopsis cylindrus
- Pseudochlorella pringsheimii
- Coccomyxa subellipsoidea
- Micromonas commoda
- Cylindrotheca fusiformis
- Cyclotella cryptica
- Thalassiosira pseudonana CCMP1335
- Guillardia theta
- Symbiodinium microadriaticum
- Phaeodactylum tricornutum CCAP 10551
- Monoraphidium neglectum
- Chaetoceros gracilis
- Cyanidioschyzon merolae strain 10D
- Cylindrotheca closterium
- Neodesmus pupukensis
- Klebsormidium flaccidum
- Scrippsiella trochoidea
- Emiliania huxleyi CCMP1516
- Gonium pectorale
- Micromonas pusilla CCMP1545
- Coccomyxa subellipsoidea C-169
- Klebsormidium nitens
- Nannochloropsis oceanica strain IMET1
- Helicosporidium sp. ATCC 50920
- Fragilariopsis cylindrus CCMP1102
- Botryococcus braunii Showa
- Nannochloropsis limnetica
- Tetraselmis striata
- Trebouxia gelatinosa
- Thalassiosira oceanica CCMP1005
- Bathycoccus prasinos RCC1105
- Neodesmus sp. UTEX 2219-4
- Nannochloropsis oculata CCMP525
- Breviolum minutum Mf 1.05b.01
- Auxenochlorella protothecoides sp 0710
- Picochlorum sp. SENEW3
- Coccomyxa sp. LA000219
- Micromonas sp. ASP10-01a
- Coccomyxa sp. Carn
- Chloroidium sp. CF
- Scenedesmus sp. ARA3
- Fistulifera sp.
- Coccomyxa sp. KJ
- Monoraphidium convolutum
- Ostreococcus lucimarinus CCE9901
- Edaphochlamys debaryana
- Yamagishiella unicocca
- Guillardia theta CCMP2712
- Escherichia coli
- Staphylococcus aureus
- Agrobacterium tumefaciens
- Streptococcus pyogenes
- Rhodobacter capsulatus
- Cauliflower mosaic virus
- Chlorella virus
- Emiliania huxleyi virus sp.
- Homo sapiens
- Hylobates sp.
- Coregonus peled
- Pyrrhula pyrrhula
- Mus musculus
- uncultured cyanobacterium
- Bigelowiella natans CCMP2755
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC7494788
filename: fbioe-08-00914-g0001.jpg
figlink: pmc/articles/PMC7494788/figure/F1/
number: F1
caption: 'The schematic of data and resource driven strategy for microalgal bioengineering.
  (A) Resource generation and enrichment: The high-throughput technologies, intense
  computation and bioinformatic analysis, and the extensive research interest on microalgae
  can generate high-quality curated data. The genomic and transcriptomic data of model
  organisms provides a basic understanding of the biosynthetic pathway. This imperative
  information is aided by proteomics and metabolomics that offers functional insights
  for bioproduct discovery in microalgae. Also, the metabolomic data can be implemented
  to novel microbial isolates with limited genomic and transcriptomic information.
  (B) Strain development and resource refinement: The leads from metabolic models
  and the use of state-of-the-art technologies, such as genome-editing and high-throughput
  variant selection can be used for microalgae strain development. Often the metabolic
  flux shifts of the mutants implies an organism''s evolution to optimize flux rearrangement.
  The objective of the flux balance shift can be biomass production or enhanced production
  of desired product. Moreover, the information obtained from fine-tuned modeling
  and genomic-editing experiments create resource avenues for further discoveries.'
papertitle: 'Bioengineering of Microalgae: Recent Advances, Perspectives, and Regulatory
  Challenges for Industrial Application.'
reftext: Gulshan Kumar, et al. Front Bioeng Biotechnol. 2020;8:914.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6824041
figid_alias: PMC7494788__F1
figtype: Figure
redirect_from: /figures/PMC7494788__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7494788__fbioe-08-00914-g0001.html
  '@type': Dataset
  description: 'The schematic of data and resource driven strategy for microalgal
    bioengineering. (A) Resource generation and enrichment: The high-throughput technologies,
    intense computation and bioinformatic analysis, and the extensive research interest
    on microalgae can generate high-quality curated data. The genomic and transcriptomic
    data of model organisms provides a basic understanding of the biosynthetic pathway.
    This imperative information is aided by proteomics and metabolomics that offers
    functional insights for bioproduct discovery in microalgae. Also, the metabolomic
    data can be implemented to novel microbial isolates with limited genomic and transcriptomic
    information. (B) Strain development and resource refinement: The leads from metabolic
    models and the use of state-of-the-art technologies, such as genome-editing and
    high-throughput variant selection can be used for microalgae strain development.
    Often the metabolic flux shifts of the mutants implies an organism''s evolution
    to optimize flux rearrangement. The objective of the flux balance shift can be
    biomass production or enhanced production of desired product. Moreover, the information
    obtained from fine-tuned modeling and genomic-editing experiments create resource
    avenues for further discoveries.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - lwr
---
