---
figid: PMC7966505__fimmu-12-587119-g003
figtitle: Graphical demonstration of identified tryptophan pathway gene differences
  relative to generated metabolites in axSpA and HC
organisms:
- Escherichia coli
- Clostridioides difficile
- Bacteroides fragilis
- Akkermansia muciniphila
- Bacteroides thetaiotaomicron
- Bifidobacterium adolescentis
- Ruminococcus gnavus
- Roseburia intestinalis
- Streptococcus anginosus
- Bacteroides ovatus
- Ruminococcus bromii
- Phocaeicola dorei
- Alistipes finegoldii
- Prevotella bivia
- Alistipes shahii
- Ruminococcus lactaris
- Bacteroides eggerthii
- Campylobacter hominis
- Corynebacterium aurimucosum
- Dialister invisus
- Hoylesella buccalis
- Sutterella wadsworthensis
- Porphyromonas somerae
- Porphyromonas bennonis
- gut metagenome
- metagenome
- metagenomes
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Cerobasis alpha
organisms_ner:
- NA
pmcid: PMC7966505
filename: fimmu-12-587119-g003.jpg
figlink: pmc/articles/PMC7966505/figure/f3/
number: F3
caption: Graphical demonstration of identified tryptophan pathway gene differences
  relative to generated metabolites in axSpA and HC. Genes encoding enzymes in the
  tryptophan synthesis/metabolism pathway were identified using metagenomics () and
  are represented visually here. Numerous pathways towards tryptophan synthesis were
  identified as significantly higher in HC compared to axSpA and shown with black
  arrows and listed KO names. Indolepyruvate decarboxylase, which is responsible for
  tryptophan metabolism towards IAA and I3Ald, was found to be significantly increased
  in genetic abundance in axSpA compared to HC. This gene and the resulting metabolites
  that are increased in axSpA are identified in red.
papertitle: Multi â€˜Omics Analysis of Intestinal Tissue in Ankylosing Spondylitis Identifies
  Alterations in the Tryptophan Metabolism Pathway.
reftext: Adam J. Berlinberg, et al. Front Immunol. 2021;12:587119.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9318369
figid_alias: PMC7966505__F3
figtype: Figure
redirect_from: /figures/PMC7966505__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7966505__fimmu-12-587119-g003.html
  '@type': Dataset
  description: Graphical demonstration of identified tryptophan pathway gene differences
    relative to generated metabolites in axSpA and HC. Genes encoding enzymes in the
    tryptophan synthesis/metabolism pathway were identified using metagenomics ()
    and are represented visually here. Numerous pathways towards tryptophan synthesis
    were identified as significantly higher in HC compared to axSpA and shown with
    black arrows and listed KO names. Indolepyruvate decarboxylase, which is responsible
    for tryptophan metabolism towards IAA and I3Ald, was found to be significantly
    increased in genetic abundance in axSpA compared to HC. This gene and the resulting
    metabolites that are increased in axSpA are identified in red.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Carboxyphenylamino-deoxyribulose-5-phosphate
  - Indole-3-glycerol phosphate
  - Indole
  - Tryptophan
  - K01695
  - Indole-3-pyruvate
  - N-methyl-L-tryptophan
  - Indolepyruvate
  - K04103
  - Indole-3-acetaldehyde
  - Indole-3-acetate
---
