---
figid: PMC7140033__cancers-12-00722-g001
figtitle: Bioinformatics pipeline
organisms:
- Homo sapiens
- Mus musculus
- Mouse mammary tumor virus
organisms_ner:
- Drosophila melanogaster
pmcid: PMC7140033
filename: cancers-12-00722-g001.jpg
figlink: pmc/articles/PMC7140033/figure/cancers-12-00722-f001/
number: F1
caption: Bioinformatics pipeline. Schematics showing eight patterns of interest (including
  early changes, early sustained changes, and late changes) that were used as criteria
  for identifying meaningful changes in (A) miRNAs and in (C) mRNAs and pathways that
  define breast cancer progression across a panel of cell lines comprising normal-like
  (MCF10A.P), preneoplastic (MCF10.AT1), DCIS (MCF10.DCIS), and invasive cancer (MCF10.CA1D)
  cells. The red lines (A,C) indicate that miRNA/mRNA expression levels and pathway
  scores are higher in progressive stages and the green lines indicate that miRNA/mRNA
  expression levels and pathway scores are lower in progressive stages compared to
  MCF10A.P. (B) Flowchart showing the bioinformatics pipeline for processing of miRNA-
  and RNA-sequencing datasets to derive target genes and pathways. First, statistical
  significance analysis (as described in Methods) was used to identify meaningful
  miRNAs and RNAs across the breast cancer progression steps, and parametric analysis
  of gene set enrichment (PAGE) was used to calculate scores of the pathways. Then,
  a k-mean clustering analysis was used to group the significant miRNAs, RNAs, or
  pathways for the patterns of interest. TargetScan was applied to determine miRNAs’
  targets, which were used to determine the target pathways, if the target genes are
  among the pathway gene membership. At the last, networks were drawn to visualize
  the relationships between miRNA, target genes, and pathways. None of the miRNA conforming
  to the expression pattern of G8 were identified, therefore G8 is missing in the
  miRNA set.
papertitle: 'Integrative Analyses of Multilevel Omics Reveal Preneoplastic Breast
  to Possess a Molecular Landscape That is Globally Shared with Invasive Basal-Like
  Breast Cancer (Running Title: Molecular Landscape of Basal-Like Breast Cancer Progression).'
reftext: Zhenlin Ju, et al. Cancers (Basel). 2020 Mar;12(3):722.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.7055516
figid_alias: PMC7140033__F1
figtype: Figure
redirect_from: /figures/PMC7140033__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7140033__cancers-12-00722-g001.html
  '@type': Dataset
  description: Bioinformatics pipeline. Schematics showing eight patterns of interest
    (including early changes, early sustained changes, and late changes) that were
    used as criteria for identifying meaningful changes in (A) miRNAs and in (C) mRNAs
    and pathways that define breast cancer progression across a panel of cell lines
    comprising normal-like (MCF10A.P), preneoplastic (MCF10.AT1), DCIS (MCF10.DCIS),
    and invasive cancer (MCF10.CA1D) cells. The red lines (A,C) indicate that miRNA/mRNA
    expression levels and pathway scores are higher in progressive stages and the
    green lines indicate that miRNA/mRNA expression levels and pathway scores are
    lower in progressive stages compared to MCF10A.P. (B) Flowchart showing the bioinformatics
    pipeline for processing of miRNA- and RNA-sequencing datasets to derive target
    genes and pathways. First, statistical significance analysis (as described in
    Methods) was used to identify meaningful miRNAs and RNAs across the breast cancer
    progression steps, and parametric analysis of gene set enrichment (PAGE) was used
    to calculate scores of the pathways. Then, a k-mean clustering analysis was used
    to group the significant miRNAs, RNAs, or pathways for the patterns of interest.
    TargetScan was applied to determine miRNAs’ targets, which were used to determine
    the target pathways, if the target genes are among the pathway gene membership.
    At the last, networks were drawn to visualize the relationships between miRNA,
    target genes, and pathways. None of the miRNA conforming to the expression pattern
    of G8 were identified, therefore G8 is missing in the miRNA set.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LanB2
  - anon-70Db
  - Set
  - I-2
  - anon-70Dc
  - g5
  - tt
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
---
