---
figid: PMC7404231__889f2
figtitle: Stress-response mechanisms
organisms:
- Homo sapiens
- Mus musculus
- Saccharomyces cerevisiae
- Caenorhabditis elegans
- Pseudomonas aeruginosa
- prion
organisms_ner:
- Caenorhabditis elegans
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
pmcid: PMC7404231
filename: 889f2.jpg
figlink: pmc/articles/PMC7404231/figure/fig2/
number: F2
caption: 'Stress-response mechanisms. The accumulation of protein aggregates in different
  cellular organelles activates organelle-specific stress-response mechanisms that
  modulate gene expression to support the restoration of proteostasis. (A) Heat exposure
  leads to the accumulation of aggregated proteins within the cytosol and to the activation
  of the heat-shock response (HSR). A cytosolic chaperone, a member of the HSP70 family,
  recognizes protein aggregates (A-I) and mediates the translocation of the transcription
  factor HSF-1 into the nucleus (A-II), where it trimerizes and regulates the expression
  of its target genes (A-III). The resulting transcripts migrate to the cytosol (A-IV)
  and the translated proteins, mostly chaperones (A-V), support proteostasis maintenance
  (A-VI). (B) A similar mechanism, known as the mitochondrial unfolded protein response
  (UPRmt), is caused by metabolic impairment such as oxidative stress or malfunction
  of the electron transport chain in mitochondria. Increased levels of protein aggregates
  are detected by the HSP70 chaperone, HSP-6 (B-I), which induces the migration of
  the transcription factor ATFS-1 into the nucleus (B-II). In the nucleus, ATFS-1
  teams up with UBL-5 and the chromatin modulator DVE-1 to change the expression levels
  of its target genes (B-III). The resulting messenger RNA transcripts are exported
  into the cytosol (B-IV) where they are translated (B-V) and transported into mitochondria
  to restore proteostasis. (C) The ER has three canonical unfolded protein response
  mechanisms (UPRER) that are activated upon accumulation of aggregated proteins within
  the ER lumen. Similarly to the HSR and the UPRmt, a member of the HSP70 family named
  BiP identifies misfolded proteins and activates IRE-1 (C-I), which initiates a process
  of splicing and translation of the transcription factor XBP-1 (C-II). XBP-1 migrates
  to the nucleus (C-III) where it activates the expression of genes that encode for
  ER chaperones and components of the ER-associated protein degradation (ERAD) pathway.
  The protein aggregation-mediated ER stress further activates the kinase PERK, which
  has two key functions (C-V): it promotes the migration of ATF-4 into the nucleus
  (C-VI) where it regulates its target gene networks and phosphorylates eIF2α. The
  phosphorylation of eIF2α inhibits the translation of proteins that require the assistance
  of chaperones to fold properly, thereby easing the workload of the ER-resident proteostasis
  network. The third UPRER mechanism that is activated upon ER stress involves ER
  membrane-bound ATF-6 (C-VII), which is shuttled to the Golgi apparatus, where it
  is proteolytically cleaved (C-VIII) enabling shuttling into the nucleus (C-IX).
  In the nucleus, it enhances the expression of xbp-1 and additional target genes
  encoding ERAD components. The transcripts that are formed as a result of the activities
  of XBP-1, ATF-4, and ATF-6(N) are exported to the cytosol (C-X) and translated by
  the ribosome. The resulting ER-resident chaperones as well as ERAD components are
  transported into the ER to maintain proteostasis, either by refolding of aggregated
  proteins (C-XI) or proteasomal degradation of terminally misfolded proteins (C-XII).'
papertitle: Organismal Protein Homeostasis Mechanisms.
reftext: Thorsten Hoppe, et al. Genetics. 2020 Aug;215(4):889-901.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9511347
figid_alias: PMC7404231__F2
figtype: Figure
redirect_from: /figures/PMC7404231__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7404231__889f2.html
  '@type': Dataset
  description: 'Stress-response mechanisms. The accumulation of protein aggregates
    in different cellular organelles activates organelle-specific stress-response
    mechanisms that modulate gene expression to support the restoration of proteostasis.
    (A) Heat exposure leads to the accumulation of aggregated proteins within the
    cytosol and to the activation of the heat-shock response (HSR). A cytosolic chaperone,
    a member of the HSP70 family, recognizes protein aggregates (A-I) and mediates
    the translocation of the transcription factor HSF-1 into the nucleus (A-II), where
    it trimerizes and regulates the expression of its target genes (A-III). The resulting
    transcripts migrate to the cytosol (A-IV) and the translated proteins, mostly
    chaperones (A-V), support proteostasis maintenance (A-VI). (B) A similar mechanism,
    known as the mitochondrial unfolded protein response (UPRmt), is caused by metabolic
    impairment such as oxidative stress or malfunction of the electron transport chain
    in mitochondria. Increased levels of protein aggregates are detected by the HSP70
    chaperone, HSP-6 (B-I), which induces the migration of the transcription factor
    ATFS-1 into the nucleus (B-II). In the nucleus, ATFS-1 teams up with UBL-5 and
    the chromatin modulator DVE-1 to change the expression levels of its target genes
    (B-III). The resulting messenger RNA transcripts are exported into the cytosol
    (B-IV) where they are translated (B-V) and transported into mitochondria to restore
    proteostasis. (C) The ER has three canonical unfolded protein response mechanisms
    (UPRER) that are activated upon accumulation of aggregated proteins within the
    ER lumen. Similarly to the HSR and the UPRmt, a member of the HSP70 family named
    BiP identifies misfolded proteins and activates IRE-1 (C-I), which initiates a
    process of splicing and translation of the transcription factor XBP-1 (C-II).
    XBP-1 migrates to the nucleus (C-III) where it activates the expression of genes
    that encode for ER chaperones and components of the ER-associated protein degradation
    (ERAD) pathway. The protein aggregation-mediated ER stress further activates the
    kinase PERK, which has two key functions (C-V): it promotes the migration of ATF-4
    into the nucleus (C-VI) where it regulates its target gene networks and phosphorylates
    eIF2α. The phosphorylation of eIF2α inhibits the translation of proteins that
    require the assistance of chaperones to fold properly, thereby easing the workload
    of the ER-resident proteostasis network. The third UPRER mechanism that is activated
    upon ER stress involves ER membrane-bound ATF-6 (C-VII), which is shuttled to
    the Golgi apparatus, where it is proteolytically cleaved (C-VIII) enabling shuttling
    into the nucleus (C-IX). In the nucleus, it enhances the expression of xbp-1 and
    additional target genes encoding ERAD components. The transcripts that are formed
    as a result of the activities of XBP-1, ATF-4, and ATF-6(N) are exported to the
    cytosol (C-X) and translated by the ribosome. The resulting ER-resident chaperones
    as well as ERAD components are transported into the ER to maintain proteostasis,
    either by refolding of aggregated proteins (C-XI) or proteasomal degradation of
    terminally misfolded proteins (C-XII).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - hsp-70
  - hsf-1
  - hsp-6
  - atfs-1
  - ubl-5
  - dve-1
  - xbp-1
  - atf-4
  - ire-1
  - atf-6
  - eif-2alpha
  - eif-2A
  - HSR
  - HSPA1A
  - HSPA4
  - GPR162
  - IL6
  - NR5A1
  - CACNA1E
  - NEUROD1
  - TUBB4B
  - EEF1B2P2
  - UBL5
  - PIN1
  - XBP1
  - DDIT3
  - ATF4
  - ERN1
  - GDF10
  - HSPA5
  - EIF2AK3
  - IL17C
  - DPYSL5
  - BMPER
  - ATF6
  - COX8A
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - EIF2A
  - EIF2S1
  - Csprs
  - Hspa1b
  - Gpr162
  - Hsf1
  - Anxa4
  - Cacna1e
  - Hbb-b2
  - Neurod1
  - Cacna1b
  - Cacnb3
  - Gabrb3
  - Bhlhe22
  - Ubl5
  - Xbp1
  - Ddit3
  - Atf4
  - Ern2
  - Hspa5
  - Eif2ak3
  - Lamp1
  - Bmper
  - Atf6
  - Eif2a
  - Eif2s1
  - Prosalpha3
  - Hsp70Ab
  - Hsp70Ba
  - Hsp70Aa
  - Hsp70Bb
  - Hsp70Bc
  - Hsp70Bbb
  - Prosalpha2
  - Hsf
  - bi
  - Prosbeta2
  - Prosbeta3
  - bv
  - Itgbn
  - crc
  - ix
  - vi
  - Sh
  - Ire1
  - bip
  - Hsc70-3
  - PEK
  - rl
  - l(1)14Ai
  - l(3)65CEc
  - l(2)46Df
  - anon-48Aj
  - anon-XII
  - ple
  - Eig71Ee
  - l(2)46Da
  - anon-48Ae
  - TER94
  - COX8
  - rip
  - l(1)13Fc
  - anon-48Af
  - anon-VIII
  - eIF2alpha
  - eIF2A
  - 'N'
---
