---
figid: PMC7763666__cells-09-02668-g005
figtitle: Wingless-type (WNT) signaling pathway
organisms:
- Homo sapiens
- Mus musculus
pmcid: PMC7763666
filename: cells-09-02668-g005.jpg
figlink: pmc/articles/PMC7763666/figure/cells-09-02668-f005/
number: F5
caption: Wingless-type (WNT) signaling pathway. To investigate possible interactions
  of genes located in chromosomal regions with gains or losses, as determined by array-comparative
  genomic hybridization (CGH), ingenuity pathway analysis (IPA) has been performed.
  One of the most highly rated networks in IPA analysis was the WNT signaling pathway.
  The chromosomal regions coding for genes that are shaded were determined to be significant
  from the statistical analysis. The genes shaded red are located in chromosomal regions
  with gains and those that are green are located in chromosomal regions with losses.
  White shaded genes relate to the pathway but were not found to be altered in the
  actual analysis. The intensity of the shading shows to what degree an amplification
  or loss of the specific chromosomal region was detected. A solid line represents
  a direct interaction between the two gene products, arrows show activating interactions,
  bar-headed lines inactivation.
papertitle: Establishment and Characterization of a Sclerosing Spindle Cell Rhabdomyosarcoma
  Cell Line with a Complex Genomic Profile.
reftext: Sabine Schleicher, et al. Cells. 2020 Dec;9(12):2668.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8387741
figid_alias: PMC7763666__F5
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
redirect_from: /figures/PMC7763666__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7763666__cells-09-02668-g005.html
  '@type': Dataset
  description: Wingless-type (WNT) signaling pathway. To investigate possible interactions
    of genes located in chromosomal regions with gains or losses, as determined by
    array-comparative genomic hybridization (CGH), ingenuity pathway analysis (IPA)
    has been performed. One of the most highly rated networks in IPA analysis was
    the WNT signaling pathway. The chromosomal regions coding for genes that are shaded
    were determined to be significant from the statistical analysis. The genes shaded
    red are located in chromosomal regions with gains and those that are green are
    located in chromosomal regions with losses. White shaded genes relate to the pathway
    but were not found to be altered in the actual analysis. The intensity of the
    shading shows to what degree an amplification or loss of the specific chromosomal
    region was detected. A solid line represents a direct interaction between the
    two gene products, arrows show activating interactions, bar-headed lines inactivation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tgfb1
  - Ltbp1
  - Lrp1
  - Mvp
  - Txlng
  - Kremen1
  - Lgals3
  - Pias1
  - Shh
  - Mark2
  - Src
  - Map4k1
  - Ilk
  - Gm12892
  - Tab1
  - Apc
  - Gsk3b
  - Nr2c2
  - Map3k7
  - Axin1
  - Btrc
  - Mdm2
  - Ppp2ca
  - Ppp2r1a
  - Nlk
  - Gpi1
  - Qsox1
  - Rara
  - Rab40b
  - Crebbp
  - Pag1
  - Cytip
  - Bc1
  - Bc1-ps1
  - Igkv1-136
  - Hdac1
  - Hdac1-ps
  - Jun
  - Sfrp1
  - Dkk1
  - Pou5f1
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - TGFB1
  - TGFB2
  - TGFB3
  - CDH1
  - CDH10
  - CDH11
  - CDH12
  - CDH13
  - CDH15
  - CDH16
  - CDH17
  - CDH18
  - CDH19
  - CDH2
  - CDH20
  - CDH22
  - CDH24
  - CDH3
  - CDH4
  - CDH5
  - CDH6
  - CDH7
  - CDH8
  - CDH9
  - RPSA
  - LRP1
  - PTPRA
  - MVP
  - KREMEN1
  - TGFBR1
  - TGFBR2
  - TGFBR3
  - HADHA
  - LGALS1
  - PIAS1
  - TMEM132A
  - ADAR
  - DVL1P1
  - MARK2
  - F2R
  - PWAR1
  - SRC
  - FGR
  - FYN
  - YES1
  - MAP4K1
  - ILK
  - CSNK1A1
  - KRT1
  - CSNK1A1L
  - TAB1
  - APC
  - PROC
  - GSK3A
  - GSK3B
  - MAP3K7
  - NR2C2
  - AXIN1
  - AXIN2
  - BTRC
  - MDM2
  - PTPA
  - NLK
  - GPI
  - RARA
  - RAB40B
  - RARB
  - RARG
  - RUVBL2
  - APPL1
  - CREBBP
  - EIF4E
  - OPN1LW
  - PAG1
  - PCBP4
  - CHMP2A
  - HDAC1
  - HNF4A
  - JUN
  - JUNB
  - JUND
  - CDKN2A
  - SFRP1
  - DKK1
  - POU5F1
  - POU5F1P3
  - POU5F1P4
---
