---
figid: PMC7047088__ars.2019.7976_figure3
figtitle: HIF-mediated gene expression in metabolic reprogramming
organisms:
- NA
pmcid: PMC7047088
filename: ars.2019.7976_figure3.jpg
figlink: pmc/articles/PMC7047088/figure/f3/
number: F3
caption: HIF-mediated gene expression in metabolic reprogramming. (A) Metabolic reprogramming
  during embryogenesis results in a shift in bioenergetic metabolism (ATP demand away
  from mitochondria OXPHOS to a dependence on glucose utilization through glycolysis
  pathways). These reprogramming changes are driven by physical isolation from atmospheric
  O2 and the resultant activation of heterodimeric O2-sensing bHLH-PAS nuclear transcription
  factors belonging to the family of HIFs. During conditions of normoxia, O2 is able
  to bind to the enzyme prolyl hydroxylase (PHD), which then hydroxylates HIF-1α at
  two positions, allowing the binding of von Hippel–Lindau tumor suppressor protein
  (VHL) and activation of ubiquitin-mediated HIF-1α protein degradation. This action
  prevents HIF-mediated gene expression by removing the transcription factor from
  the cytosol. Under conditions of hypoxia, HIF-1α is allowed to accumulate and it
  is translocated to the nucleus where it combines with its heterodimerization partner,
  HIF-1β/ARNT, and binds to the HRE (5′-ACGTG-3′) to induce the expression of hundreds
  of genes. (B) The metabolic reprogramming that occurs in developing cells and tissues
  in response to moderate/severe hypoxia allows the developing conceptus to thrive
  in normally hostile conditions. Induced expression of HIF target genes (shown in
  orange/red fonts) shows that the transition to glycolysis dominated bioenergetics
  along with several other beneficial consequences. With metabolic reprogramming comes
  increases in glucose uptake/transport, upregulation of all 10 enzymes in the glycolysis
  pathway, increased pentose phosphate pathway activity and NADPH synthesis, inhibition
  of TCA cycle and mitochondrial ETC function, altered transcriptional controls, and
  the modulation of many other pathways, including hexosamine biosynthesis, phospholipid
  biosynthesis, nucleotide synthesis, and one-carbon metabolism. This representation
  of HIF-upregulated genes in metabolic reprogramming is not complete and should not
  be viewed as a comprehensive list [Figure based, in part, from Dengler et al. ()].
  ARNT, aryl hydrocarbon receptor nuclear translocator; bHLH, basic helix loop helix;
  ETC, electron transport chain; HIF, hypoxia-inducible factor; HRE, hypoxia response
  element; OXPHOS, oxidative phosphorylation; PAS, Per-Arnt-Sim; PHD, prolyl hydroxylase.
  Color images are available online.
papertitle: The Redox Theory of Development.
reftext: Jason M. Hansen, et al. Antioxid Redox Signal.  April 1, 2020;32(10):715-740.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9680983
figid_alias: PMC7047088__F3
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC7047088__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7047088__ars.2019.7976_figure3.html
  '@type': Dataset
  description: HIF-mediated gene expression in metabolic reprogramming. (A) Metabolic
    reprogramming during embryogenesis results in a shift in bioenergetic metabolism
    (ATP demand away from mitochondria OXPHOS to a dependence on glucose utilization
    through glycolysis pathways). These reprogramming changes are driven by physical
    isolation from atmospheric O2 and the resultant activation of heterodimeric O2-sensing
    bHLH-PAS nuclear transcription factors belonging to the family of HIFs. During
    conditions of normoxia, O2 is able to bind to the enzyme prolyl hydroxylase (PHD),
    which then hydroxylates HIF-1α at two positions, allowing the binding of von Hippel–Lindau
    tumor suppressor protein (VHL) and activation of ubiquitin-mediated HIF-1α protein
    degradation. This action prevents HIF-mediated gene expression by removing the
    transcription factor from the cytosol. Under conditions of hypoxia, HIF-1α is
    allowed to accumulate and it is translocated to the nucleus where it combines
    with its heterodimerization partner, HIF-1β/ARNT, and binds to the HRE (5′-ACGTG-3′)
    to induce the expression of hundreds of genes. (B) The metabolic reprogramming
    that occurs in developing cells and tissues in response to moderate/severe hypoxia
    allows the developing conceptus to thrive in normally hostile conditions. Induced
    expression of HIF target genes (shown in orange/red fonts) shows that the transition
    to glycolysis dominated bioenergetics along with several other beneficial consequences.
    With metabolic reprogramming comes increases in glucose uptake/transport, upregulation
    of all 10 enzymes in the glycolysis pathway, increased pentose phosphate pathway
    activity and NADPH synthesis, inhibition of TCA cycle and mitochondrial ETC function,
    altered transcriptional controls, and the modulation of many other pathways, including
    hexosamine biosynthesis, phospholipid biosynthesis, nucleotide synthesis, and
    one-carbon metabolism. This representation of HIF-upregulated genes in metabolic
    reprogramming is not complete and should not be viewed as a comprehensive list
    [Figure based, in part, from Dengler et al. ()]. ARNT, aryl hydrocarbon receptor
    nuclear translocator; bHLH, basic helix loop helix; ETC, electron transport chain;
    HIF, hypoxia-inducible factor; HRE, hypoxia response element; OXPHOS, oxidative
    phosphorylation; PAS, Per-Arnt-Sim; PHD, prolyl hydroxylase. Color images are
    available online.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HIF1A
  - CA9
  - SLC2A1
  - SLC2A3
  - SLC16A4
  - SLC16A3
  - SLC9A1
  - PDC
  - HK1
  - KCNA4
  - KLK1
  - HOOK1
  - TAC4
  - HK2
  - KCNA5
  - KIF2A
  - KLK2
  - HOOK2
  - LDHA
  - BNIP3L
  - MXI1
  - BNIP3
  - GTPBP1
  - PFKFB3
  - PFKFB4
  - PFKFB1
  - PFKFB2
  - PFKL
  - PDK1
  - PDPK1
  - ALDOA
  - LONP1
  - ALDOC
  - TKT
  - DDR2
  - TPI1
  - ARNT
  - GAPDH
  - GAPDHP44
  - TKTL2
  - ATP8A2
  - WDTC1
  - PGK1
  - VCAN
  - ENO1
  - ETS1
  - BHLHE40
  - DELEC1
  - CITED2
  - PKM
  - PKLR
  - BHLHE41
  - ID2
  - Glucose
  - Glucose-6-phosphate
  - Lactate
  - NAD
  - TCA
  - Pyruvate
  - NADH
  - Acetyl CoA
  - ADP
  - ATP
  - Serine
  - Glycine
  - HIF-MEDIATED
  - HIF-1a  HIF-1B
---
