---
figid: PMC7045390__mSphere.00913-19-f0006
figtitle: Correlation analysis of the hypoxic transcriptome and metabolome data
organisms:
- NA
pmcid: PMC7045390
filename: mSphere.00913-19-f0006.jpg
figlink: pmc/articles/PMC7045390/figure/fig6/
number: F6
caption: Correlation analysis of the hypoxic transcriptome and metabolome data. The
  different altered oxygen-sensitive metabolic routes and the dynamic of transcripts
  that were significantly altered during the corresponding time points are summarized.
  (A to N) Differential expression of metabolic genes associated with glycolysis (A),
  amino acid biosynthesis (histidine [B] serine [D], glycine and cysteine [G], valine
  [I], leucine, tryptophan, and phenylalanine [J]; aspartate, asparagine, methionine,
  threonine, and isoleucine [L], and glutamine, glutamate, proline, arginine, and
  lysine [N]), the pentose phosphate pathway (C), lipids (glycerolipids [E], sphingolipids
  [F], ergosterol [K]), chitin synthesis (H), and the TCA cycle (M) are shown as clustergrams.
  Upregulated and downregulated genes are indicated by red and blue, respectively.
  The 10- to 60-min transcriptional data were extracted from a study by Sellam et
  al. (). The full list of differentially expressed transcripts at 180 min of hypoxia
  is available in . Aa, amino acid.
papertitle: Metabolic Reprogramming in the Opportunistic Yeast Candida albicans in
  Response to Hypoxia.
reftext: Anaïs Burgain, et al. mSphere. 2020 Jan-Feb;5(1):e00913-19.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8626198
figid_alias: PMC7045390__F6
figtype: Figure
organisms_ner:
- Saccharomyces cerevisiae S288C
- Homo sapiens
redirect_from: /figures/PMC7045390__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7045390__mSphere.00913-19-f0006.html
  '@type': Dataset
  description: Correlation analysis of the hypoxic transcriptome and metabolome data.
    The different altered oxygen-sensitive metabolic routes and the dynamic of transcripts
    that were significantly altered during the corresponding time points are summarized.
    (A to N) Differential expression of metabolic genes associated with glycolysis
    (A), amino acid biosynthesis (histidine [B] serine [D], glycine and cysteine [G],
    valine [I], leucine, tryptophan, and phenylalanine [J]; aspartate, asparagine,
    methionine, threonine, and isoleucine [L], and glutamine, glutamate, proline,
    arginine, and lysine [N]), the pentose phosphate pathway (C), lipids (glycerolipids
    [E], sphingolipids [F], ergosterol [K]), chitin synthesis (H), and the TCA cycle
    (M) are shown as clustergrams. Upregulated and downregulated genes are indicated
    by red and blue, respectively. The 10- to 60-min transcriptional data were extracted
    from a study by Sellam et al. (). The full list of differentially expressed transcripts
    at 180 min of hypoxia is available in . Aa, amino acid.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HIS1
  - HIS4
  - HIS7
  - SNO1
  - ZWF1
  - SOL3
  - RKI1
  - TKL1
  - TAL1
  - SER1
  - SER33
  - SER2
  - SHM2
  - CYS4
  - STR2
  - GLK1
  - PGM2
  - PGI1
  - PFK2
  - FBA1
  - TPI1
  - TDH3
  - PGK1
  - GPM1
  - ENO1
  - CDC19
  - TYE7
  - SLC1
  - DYN2
  - DPP1
  - DAP2
  - PSD2
  - CHO1
  - CHO2
  - PFK1
  - EPT1
  - LCB1
  - LCB2
  - SUR2
  - SUR1
  - LAG1
  - AUR1
  - CSG2
  - PSD1
  - GNA1
  - QRI1
  - CHS2
  - CHS1
  - CHS3
  - LAC1
  - GFA1
  - CYS3
  - TRP4
  - TRP5
  - TRP2
  - TRP3
  - ARO7
  - LEU1
  - LEU2
  - LEU4
  - ERG13
  - ERG10
  - ERG9
  - ERG24
  - ERG26
  - ILV5
  - ILV2
  - ILV3
  - BAT2
  - AAT1
  - ACO1
  - ACO2
  - HMG1
  - ERG1
  - ERG7
  - ERG11
  - ERG6
  - ERG3
  - ARO9
  - ARO8
  - ASN1
  - MET2
  - MET6
  - CIT1
  - THR4
  - THR1
  - MDH1
  - ILV1
  - CHA1
  - IDH2
  - IDH1
  - HOM2
  - HOM3
  - MET17
  - ERG2
  - ERG5
  - ERG25
  - SDH1
  - DIF1
  - SOR1
  - SDH4
  - KGD2
  - KGD1
  - GDH3
  - GDH2
  - LSC1
  - LSC2
  - GLN1
  - GLT1
  - PYC2
  - LYS1
  - LYS4
  - LYS21
  - LYS9
  - ARG1
  - ARG3
  - ARG8
  - PRO1
  - PRO2
  - ARG56
  - HTN1
  - VIS1
  - HEXIM1
  - JAG2
  - PKLR
  - PROK1
  - PFKFB3
  - MCM2
  - MCHR1
  - CTSC
  - DPP7
  - KIF23
  - EBP
  - PFKM
  - TMED7
  - LINC01194
  - DPP3
  - SELENOI
  - SPTLC1
  - SPTLC2
  - MED23
  - ABCC9
  - ABCC8
  - GDF1
  - CCL4
  - CERS1
  - CCL4L1
  - PSD
  - GNPNAT1
  - UAP1
  - LYST
  - VPS13B
  - PAEP
  - PREP
  - PTPN22
  - TRPC4
  - TRPC5
  - DCT
  - RANBP2
  - TRP-AGG2-6
  - TRP-TGG3-1
  - TRPC3
  - CD5
  - DLEU1
  - CD8A
  - CD8B
  - DLEU2
  - FDFT1
  - PRRC2A
  - GPT
  - CFAP91
  - ADCY2
  - HMGB1
  - HMGB1P5
  - KCNH2
  - CUZD1
  - ERG
  - SC5D
  - KCNH7
  - COPG2IT1
  - THRB
  - SRR
  - KCNH6
  - MSMO1
  - SDHA
  - SDHB
  - SDHD
  - DLST
  - OGDH
  - GLUD2
  - GPD2
  - SLC1A2
  - TINAGL1
  - LMNA
  - ZMPSTE24
  - ABL2
  - RERE
  - CYS4
  - Histidine
  - Glucose
  - Ribulose 5PRibose 5P Sedoheptulose
  - Glycerol
  - Glucosamine
  - N-Acetylglucosamine-6P
  - TAG
  - DHAP G3P
  - Serine
  - ENO11
  - PtdSer
  - Phytosphingosine
  - Glycine
  - Tyrosine
  - Phenylalanine
  - Tryptophane
  - LEU1,2,4,42 Leucine
  - Pyruvate Acetyl
  - Mevalonate
  - Squalene
  - Ergosterol
  - Alanine
  - Valine
  - Aspartate
  - Citrate
  - Asparagine
  - oxaloacetate
  - cis-aconitate
  - Methionine
  - Homoserine
  - Threonine
  - TCA
  - ketoglutarate
  - Isoleucine
  - Succinate
  - Succinyl
  - Glutamate
  - Glutamine
  - Proline Arginine
  - Lysine
  - Asp
  - Asn
  - Met
  - Thr
  - His
  - Glu
  - Arg
  - Lys
  - Sphingolipids
  - Hypoxia
  - H Hypoxia
  - N Hypoxia
---
