---
figid: PMC7483256__nihms-1576048-f0002
figtitle: Small-molecule drug repurposing to target DNA damage repair and response
  pathways
organisms:
- NA
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC7483256
filename: nihms-1576048-f0002.jpg
figlink: /pmc/articles/PMC7483256/figure/F2/
number: F2
caption: 'Targeting DNA base damage and single strand break repair pathways. Through
  oncogenic stress, chemotherapy and radiation can induce misincorporation, base damage
  and single-strand breaks (SSBs); these are also common lesions that arise due to
  normal cell metabolism and proliferation. The anticipated toxicity of agents that
  block base, nucleotide, or single-strand repair pathways may limit repurposing opportunities,
  but some examples are reported. A. The mismatch repair (MMR) pathway finds and corrects
  misincorporation errors. After detection by Mut proteins, the leading strand is
  excised by Exo1. DNA polymerase delta and Ligase 1 complete the repair. Baicalein
  and triamterene bind to mismatch sites, leading to frustrated repair and DNA double
  strand breaks (DSBs). B. Nucleotide excision repair (NER) removes bulky adducts
  in DNA. This pathway is divided in two arms: global genome nucleotide excision repair
  (GG-BER), which recognizes lesions anywhere in the genome requiring XPC-Rad23 complexes
  and the DNA damage binding proteins (DDB1/2), while transcription coupled nucleotide
  excision repair (TC-NER) recognizes lesions in actively transcribed genes initiated
  via stalled RNA polymerase and the CS proteins, CSA and CSB. These two pathways
  differ in damage recognition but share the same process in lesion incision, repair,
  and ligation. Next, transcription factor II H (TFIH) and its subunit are recruited
  to the damage site and unwind DNA. Dual incision via the XPF-ERCC1 heterodimer removes
  damage containing single strand oligonucleotides. DNA polymerases repair the damaged
  DNA using the undamaged strand as a template then DNA ligase seals the nicks to
  complete NER. Spironolactone is able to direct TFIH to degradation and thus inhibit
  NER. C. Interstrand crosslinks (ICLs) in quiescent cells can be recognized and repaired
  by NER, directly leading to DSBs that may be repaired by end-joining pathways. ICLs
  in S phase lead to stalled replication forks. Binding of the FANCM protein complex
  recruits and activates the Fanconi anemia (FA) core complex. EME1 and FAN1 endonucleases
  mediate ICL incision, promoting DSB repair by homologous recombination (HR). Defects
  in ICL repair displayed by repurposed drugs are linked to targeting the FA pathway.'
papertitle: Small-molecule drug repurposing to target DNA damage repair and response
  pathways.
reftext: Jacqueline A. Brinkman, et al. Semin Cancer Biol. ;68:230-241.
year: '2020'
doi: 10.1016/j.semcancer.2020.02.013
journal_title: Seminars in cancer biology
journal_nlm_ta: Semin Cancer Biol
publisher_name: ''
keywords: NHEJ | Homologous recombination | Apoptosis | Senescence
automl_pathway: 0.908416
figid_alias: PMC7483256__F2
figtype: Figure
redirect_from: /figures/PMC7483256__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7483256__nihms-1576048-f0002.html
  '@type': Dataset
  description: 'Targeting DNA base damage and single strand break repair pathways.
    Through oncogenic stress, chemotherapy and radiation can induce misincorporation,
    base damage and single-strand breaks (SSBs); these are also common lesions that
    arise due to normal cell metabolism and proliferation. The anticipated toxicity
    of agents that block base, nucleotide, or single-strand repair pathways may limit
    repurposing opportunities, but some examples are reported. A. The mismatch repair
    (MMR) pathway finds and corrects misincorporation errors. After detection by Mut
    proteins, the leading strand is excised by Exo1. DNA polymerase delta and Ligase
    1 complete the repair. Baicalein and triamterene bind to mismatch sites, leading
    to frustrated repair and DNA double strand breaks (DSBs). B. Nucleotide excision
    repair (NER) removes bulky adducts in DNA. This pathway is divided in two arms:
    global genome nucleotide excision repair (GG-BER), which recognizes lesions anywhere
    in the genome requiring XPC-Rad23 complexes and the DNA damage binding proteins
    (DDB1/2), while transcription coupled nucleotide excision repair (TC-NER) recognizes
    lesions in actively transcribed genes initiated via stalled RNA polymerase and
    the CS proteins, CSA and CSB. These two pathways differ in damage recognition
    but share the same process in lesion incision, repair, and ligation. Next, transcription
    factor II H (TFIH) and its subunit are recruited to the damage site and unwind
    DNA. Dual incision via the XPF-ERCC1 heterodimer removes damage containing single
    strand oligonucleotides. DNA polymerases repair the damaged DNA using the undamaged
    strand as a template then DNA ligase seals the nicks to complete NER. Spironolactone
    is able to direct TFIH to degradation and thus inhibit NER. C. Interstrand crosslinks
    (ICLs) in quiescent cells can be recognized and repaired by NER, directly leading
    to DSBs that may be repaired by end-joining pathways. ICLs in S phase lead to
    stalled replication forks. Binding of the FANCM protein complex recruits and activates
    the Fanconi anemia (FA) core complex. EME1 and FAN1 endonucleases mediate ICL
    incision, promoting DSB repair by homologous recombination (HR). Defects in ICL
    repair displayed by repurposed drugs are linked to targeting the FA pathway.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MRC1
  - MSH2
  - MSH6
  - MLH1
  - PMS2
  - LIG1
  - LRIG1
  - EXO1
  - PCNA
  - RPA1
  - RPA2
  - RPA3
  - XPC
  - DDB2
  - DDB1
  - ERCC2
  - ERCC1
  - ERCC4
  - XPA
  - POLE
  - NR1H2
  - ERVW-4
  - POLR1A
  - POLR1B
  - POLR1C
  - POLR1D
  - ERCC5
  - LIG3
  - LRIG3
  - CSH2
  - ERCC6
  - ACSS2
  - FANCM
  - FAAP24
  - BRCA2
  - FANCD2
  - FAN1
  - FSCN1
  - FANCI
  - RAD51
  - MUS81
  - msh2
  - msh6
  - mlh1
  - pms2
  - lig1
  - exo1
  - pcna
  - rpa2
  - xpc
  - ddb2
  - ddb1
  - ercc2
  - ercc1
  - xpa
  - pole
  - lig3
  - ercc6
  - fancm
  - faap24
  - fan1
  - fanci
  - rad51
  - mus81
---
