---
figid: PMC7326039__fonc-10-00812-g0001
figtitle: Work flow for identification of circular RNA candidates in context of non-muscle-invasive
  bladder cancer subtypes
organisms:
- Homo sapiens
- Gorilla gorilla
organisms_ner:
- Drosophila melanogaster
pmcid: PMC7326039
filename: fonc-10-00812-g0001.jpg
figlink: pmc/articles/PMC7326039/figure/F1/
number: F1
caption: Work flow for identification of circular RNA candidates in context of non-muscle-invasive
  bladder cancer subtypes. Using 58 patient tumor tissue samples across G1, G2, and
  G3 tumor grades for NMIBC and additionally 6 MIBC samples, total RNA-sequencing
  was performed using Illumina platform. Coordinate boundaries for circular RNA candidates
  were in-silico reconstructed using DCC algorithm. For each circular RNA candidate,
  simultaneous evidence for back-spicing and linear splicing (circular RNA count and
  linear RNA count, respectively) were quantified which were then used to calculate
  the circular-to-linear ratio for each candidate. These ratios were then subjected
  to differential expression statistics using the limma BioConductor package. The
  differentially expressed circular RNAs across the tumor grade subtypes were then
  used for pathway analysis based on the host gene name. Further the circular RNA
  sequence body was checked for harboring potential miRNA target sites using TargetScan
  database.
papertitle: Back-Splicing Transcript Isoforms (Circular RNAs) Affect Biologically
  Relevant Pathways and Offer an Additional Layer of Information to Stratify NMIBC
  Patients.
reftext: Anshita Goel, et al. Front Oncol. 2020;10:812.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8928337
figid_alias: PMC7326039__F1
figtype: Figure
redirect_from: /figures/PMC7326039__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7326039__fonc-10-00812-g0001.html
  '@type': Dataset
  description: Work flow for identification of circular RNA candidates in context
    of non-muscle-invasive bladder cancer subtypes. Using 58 patient tumor tissue
    samples across G1, G2, and G3 tumor grades for NMIBC and additionally 6 MIBC samples,
    total RNA-sequencing was performed using Illumina platform. Coordinate boundaries
    for circular RNA candidates were in-silico reconstructed using DCC algorithm.
    For each circular RNA candidate, simultaneous evidence for back-spicing and linear
    splicing (circular RNA count and linear RNA count, respectively) were quantified
    which were then used to calculate the circular-to-linear ratio for each candidate.
    These ratios were then subjected to differential expression statistics using the
    limma BioConductor package. The differentially expressed circular RNAs across
    the tumor grade subtypes were then used for pathway analysis based on the host
    gene name. Further the circular RNA sequence body was checked for harboring potential
    miRNA target sites using TargetScan database.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - eor
  - l(1)N30
  - LanB2
  - anon-70Db
  - anon-70Dc
  - l(1)N17
  - .na.character
  - fra
  - vs
---
