---
figid: PMC7793809__fmicb-11-596882-g007
figtitle: Differences in the metabolic pathway for Phenylalanine, tyrosine, and tryptophan
  biosynthesis (A), Valine, leucine, and isoleucine biosynthesis (B), and Biotin metabolism
  (C) between the mucosa-associated microbiota in the small and large intestines
organisms:
- Corynebacterium glutamicum
- Lactiplantibacillus plantarum
- Akkermansia muciniphila
- Ligilactobacillus salivarius
- Lactobacillus jensenii
- Limosilactobacillus mucosae
- Bacteroidales bacterium Bact_22
- Ligilactobacillus hayakitensis
- Limosilactobacillus equigenerosi
- bacterium Lincoln Park 3
- Bos taurus
- Capra hircus
- Sus scrofa
- Ovis aries
- Equus caballus
- Equus asinus
- Equus kiang
- Mus musculus
- Rattus norvegicus
- gut metagenome
- metagenome
- Homo sapiens
- Glycine max
organisms_ner:
- Akkermansia muciniphila
- Lactiplantibacillus plantarum
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC7793809
filename: fmicb-11-596882-g007.jpg
figlink: pmc/articles/PMC7793809/figure/F7/
number: F7
caption: Differences in the metabolic pathway for Phenylalanine, tyrosine, and tryptophan
  biosynthesis (A), Valine, leucine, and isoleucine biosynthesis (B), and Biotin metabolism
  (C) between the mucosa-associated microbiota in the small and large intestines.
  Red font indicates that the abundance of this KO genes is higher in the large intestine
  compared with the small intestine.
papertitle: Unveiling the Biogeography and Potential Functions of the Intestinal Digesta-
  and Mucosa-Associated Microbiome of Donkeys.
reftext: Ruiyang Zhang, et al. Front Microbiol. 2020;11:596882.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.8310502
figid_alias: PMC7793809__F7
figtype: Figure
redirect_from: /figures/PMC7793809__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7793809__fmicb-11-596882-g007.html
  '@type': Dataset
  description: Differences in the metabolic pathway for Phenylalanine, tyrosine, and
    tryptophan biosynthesis (A), Valine, leucine, and isoleucine biosynthesis (B),
    and Biotin metabolism (C) between the mucosa-associated microbiota in the small
    and large intestines. Red font indicates that the abundance of this KO genes is
    higher in the large intestine compared with the small intestine.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - leuB
  - fabF
  - ilvC
  - ilvD
  - bioA
  - bioD
  - bioB
  - trpB
  - aroE
  - ais
  - trpD
  - tnaA
  - leuA
  - bioH
  - Methylmalate
  - 2-Oxobutanoate
  - Malonyl-[acp] methylester
  - 3-phosphate
  - methylester
  - (S)-2-Aceto-2
  - hydroxybutanoate (S)-2-Acetolactate
  - 5-0-1-Carboxyvinyl
  - 3-phosphoshikimate
  - Glutaryl
  - (R)-3-Hydroxy-3- 3-Hydroxy-3-methyl
  - Prephenate
  - 3-Ketopimeloyl
  - methyl-2
  - 2-oxobutanoate
  - Anthranilate
  - Pimeloyl
  - 4-Nydroxy
  - phenylpyruvate
  - 2,3-Dihydroxy-3
  - methylpentanoate
  - methylbutanoate
  - 5-Phospho
  - anthranilate
  - Isopropylmalate
  - 8-Amino-7-xononanoate
  - (S)-3-Methyl-2- 2-Oxoisovalerate
  - oxopentanoate
  - 1-(2-Carboxyphenylamino)-1-deoxy
  - 5-phosphate
  - 8-Diamino-nonanoate
  - Tyrosine
  - 3-Indoyl
  - glycerolphosphate
  - Isoleucine
  - Leucine
  - Biotin
  - Valine
  - Indole
  - Tryptophan
---
