---
figid: PMC7613831__EMS156770-f004
figtitle: DMT induces a transcriptomic signature promoting FAO and opposing fibrosis
  in the renal cortex of ZDF and ZDSD rats
organisms:
- Mus musculus
- Rattus norvegicus
- Armoracia rusticana
- gut metagenome
- Homo sapiens
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Drosophila melanogaster
pmcid: PMC7613831
filename: EMS156770-f004.jpg
figlink: /pmc/articles/PMC7613831/figure/F4/
number: F4
caption: 'A-B: Principal component analysis of rlog gene expression counts in the
  ZDF (A) and ZDSD (B) experiments (ZDF: n=6 fa/+, n=6 SHAM, n=9 DMT; ZDSD: n=6 SD,
  n=7 SHAM, n=6 DMT). C: Venn diagram enumeration of the number of DEGs in the ZDF
  and ZDSD experiments, as well as the number of DEGs which were commonly changed
  in the DMT vs SHAM differential expression analysis in both studies. Transcripts
  with an absolute fold-change value ≥1.3 and adjusted p-value <0.05 were considered
  as differentially expressed. D: Dotplot of Reactome pathway over-representation
  analysis illustrating pathways commonly changed in the ZDF and ZDSD experiments,
  for both the SHAM vs Healthy and DMT vs SHAM comparisons [, ]. Pathways pertaining
  to fatty acid metabolism are bolded. E: Network plot of 5 fatty acid metabolism
  pathways changed by DMT in both animal models (bolded in panel D). Smaller gene
  nodes, changed by DMT relative to SHAM in either or both animal models, are connected
  to the larger Reactome pathway nodes to which they belong. Node size of pathways
  is scaled by the number of genes changed in that pathway (lower right legend). Pathway
  and gene nodes are coloured according to whether they are changed in the DMT vs
  SHAM differential expression analysis in one or both animal models. F: Bubble plot
  of predicted upstream regulators of DMT vs SHAM gene expression changes in both
  animal models, identified using IPA software []. Upstream regulator changes are
  presented for the DMT vs SHAM comparison in the ZDF and ZDSD models. The predicted
  status (inhibited or activated) of upstream regulators is plotted according to z-score.
  Dot size is scaled to reflect the statistical significance of z-score changes. Upstream
  regulators commonly changed in both models are bolded. G: qRT-PCR validation of
  expression changes in PPARα-responsive transcripts, both peroxisomal (Acox1, Ehhadh)
  and mitochondrial (Acaa2, Pdk4), in renal cortical tissue from the ZDF and ZDSD
  experiments (ZDF: n=6 fa/+, n=6 SHAM, n=9 DMT; ZDSD: n=6 SD, n=7 SHAM, n=6 DMT).
  Statistical significance of between-group differences derived from multiplicity-corrected
  Wilcoxon rank-sum tests is denoted as follows: ns=not significant; *=p<0.05; **=p<0.01;
  ***=p<0.001; ****=p<0.0001. DEG, differentially expressed gene; DMT, dietary restriction
  plus medical therapy; FAO, fatty acid oxidation; PC, principal component; RQ mRNA,
  relative quantification of messenger ribonucleic acid; VLCFA, very-long-chain fatty
  acid; ZDF, Zucker Diabetic Fatty; ZDSD, Zucker Diabetic Sprague Dawley.'
papertitle: Dietary Restriction and Medical Therapy Drives PPARα-Regulated Improvements
  in Early Diabetic Kidney Disease in Male Rats.
reftext: William P. Martin, et al. Clin Sci (Lond). ;136(21):1485-1511.
year: '2023'
doi: 10.1042/CS20220205
journal_title: 'Clinical science (London, England : 1979)'
journal_nlm_ta: Clin Sci (Lond)
publisher_name: ''
keywords: diabetic kidney disease | fatty acid oxidation | metabolome | mitochondria
  | obesity | peroxisome | PPARα | preclinical models | transcriptome | weight loss
automl_pathway: 0.7884516
figid_alias: PMC7613831__F4
figtype: Figure
redirect_from: /figures/PMC7613831__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7613831__EMS156770-f004.html
  '@type': Dataset
  description: 'A-B: Principal component analysis of rlog gene expression counts in
    the ZDF (A) and ZDSD (B) experiments (ZDF: n=6 fa/+, n=6 SHAM, n=9 DMT; ZDSD:
    n=6 SD, n=7 SHAM, n=6 DMT). C: Venn diagram enumeration of the number of DEGs
    in the ZDF and ZDSD experiments, as well as the number of DEGs which were commonly
    changed in the DMT vs SHAM differential expression analysis in both studies. Transcripts
    with an absolute fold-change value ≥1.3 and adjusted p-value <0.05 were considered
    as differentially expressed. D: Dotplot of Reactome pathway over-representation
    analysis illustrating pathways commonly changed in the ZDF and ZDSD experiments,
    for both the SHAM vs Healthy and DMT vs SHAM comparisons [, ]. Pathways pertaining
    to fatty acid metabolism are bolded. E: Network plot of 5 fatty acid metabolism
    pathways changed by DMT in both animal models (bolded in panel D). Smaller gene
    nodes, changed by DMT relative to SHAM in either or both animal models, are connected
    to the larger Reactome pathway nodes to which they belong. Node size of pathways
    is scaled by the number of genes changed in that pathway (lower right legend).
    Pathway and gene nodes are coloured according to whether they are changed in the
    DMT vs SHAM differential expression analysis in one or both animal models. F:
    Bubble plot of predicted upstream regulators of DMT vs SHAM gene expression changes
    in both animal models, identified using IPA software []. Upstream regulator changes
    are presented for the DMT vs SHAM comparison in the ZDF and ZDSD models. The predicted
    status (inhibited or activated) of upstream regulators is plotted according to
    z-score. Dot size is scaled to reflect the statistical significance of z-score
    changes. Upstream regulators commonly changed in both models are bolded. G: qRT-PCR
    validation of expression changes in PPARα-responsive transcripts, both peroxisomal
    (Acox1, Ehhadh) and mitochondrial (Acaa2, Pdk4), in renal cortical tissue from
    the ZDF and ZDSD experiments (ZDF: n=6 fa/+, n=6 SHAM, n=9 DMT; ZDSD: n=6 SD,
    n=7 SHAM, n=6 DMT). Statistical significance of between-group differences derived
    from multiplicity-corrected Wilcoxon rank-sum tests is denoted as follows: ns=not
    significant; *=p<0.05; **=p<0.01; ***=p<0.001; ****=p<0.0001. DEG, differentially
    expressed gene; DMT, dietary restriction plus medical therapy; FAO, fatty acid
    oxidation; PC, principal component; RQ mRNA, relative quantification of messenger
    ribonucleic acid; VLCFA, very-long-chain fatty acid; ZDF, Zucker Diabetic Fatty;
    ZDSD, Zucker Diabetic Sprague Dawley.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ht
  - pc
  - Ms6hm
  - Pcsk1
  - Enpp1
  - Pkd1
  - Acan
  - Degs1
  - Glyat
  - Zfp157
  - Ccl28
  - Myod1
  - Ppig
  - act
  - Fhl5
  - Acot7
  - Serpina5
  - Larp1
  - Agt
  - Agxt
  - Mgmt
  - Gm128
  - Scap
  - Ppara
  - Phlpp1
  - Pparg
  - Myog
  - Synpo2
  - Cited2
  - Acox1
  - Ehhadh
  - Slc6a3
  - Rr149356
  - HT
  - PC
  - KRT16
  - PCSK1
  - ENPP1
  - PKD1
  - TPD52
  - DMTN
  - ACAN
  - DEGS1
  - GLYAT
  - CCL28
  - MYOD1
  - ACTA1
  - PPIG
  - CYP17A1
  - CYP19A1
  - CYP20A1
  - CYP21A2
  - CYP24A1
  - CYP39A1
  - TBXAS1
  - CYP46A1
  - CYP51A1
  - CYP1A1
  - CYP1A2
  - CYP1B1
  - CYP11A1
  - CYP11B1
  - CYP11B2
  - CYP2A13
  - CYP2A6
  - CYP2A7
  - CYP2B6
  - CYP2C18
  - CYP2C19
  - CYP2C8
  - CYP2C9
  - CYP2D6
  - CYP2D7
  - CYP2E1
  - CYP2F1
  - CYP2J2
  - CYP2R1
  - CYP2S1
  - CYP2U1
  - CYP2W1
  - CYP26A1
  - CYP26B1
  - CYP26C1
  - CYP27A1
  - CYP27B1
  - CYP27C1
  - CYP3A4
  - CYP3A43
  - CYP3A5
  - CYP3A7
  - CYP4A11
  - CYP4A22
  - CYP4B1
  - CYP4F11
  - CYP4F12
  - CYP4F2
  - CYP4F22
  - CYP4F3
  - CYP4F8
  - CYP4V2
  - CYP4X1
  - CYP4Z1
  - CYP7A1
  - CYP7B1
  - CYP8B1
  - PTGIS
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - SERPINA5
  - SEPTIN1
  - LARP1
  - AGT
  - AGXT
  - KLK15
  - IFNK
  - IFNW1
  - IFNA1
  - IFNA2
  - IFNA4
  - IFNA5
  - IFNA6
  - IFNA7
  - IFNA8
  - IFNA10
  - IFNA13
  - IFNA14
  - IFNA16
  - IFNA17
  - IFNA21
  - IFNB1
  - C1orf56
  - SCAP
  - SH2D2A
  - PPARA
  - PHLPP1
  - PPARG
  - GOSR2
  - CITED2
  - ACOX1
  - EHHADH
  - SLC6A3
  - GEN1
  - Pc
  - Cyp3a23-3a1
  - C2h1orf56
  - dmt
  - tRNA:Thr-TGT-1-1
  - tRNA:Thr-TGT-2-1
  - tRNA:Thr-TGT-2-2
  - tRNA:Thr-TGT-2-5
  - tRNA:Thr-TGT-2-4
  - tRNA:Thr-TGT-2-3
  - GlyRS
  - vs
  - Rab1
  - ha
  - Gat
  - Pitslre
  - gate
  - ca
  - Uba1
  - nau
  - dib
  - be
  - Act5C
  - Act42A
  - Actbeta
  - 'On'
  - larp
  - agt
  - Aco
  - Acox57D-d
  - AP-2alpha
  - alpha-Spec
  - bam
  - alphaTry
  - bmm
  - alpha-Cat
  - myo
  - sqh
  - Mhc
  - zip
  - wake
  - lolal
  - mir-ban
  - DAT
  - speck
  - Gen
  - pe
  - eb
  - OMT
  - Be
  - Fatty acid
---
