---
figid: PMC7570223__plants-09-01108-g001
figtitle: Transcriptomic reconfiguration of pathways involved in energy metabolism
  in response to O2 deficiency
organisms:
- NA
pmcid: PMC7570223
filename: plants-09-01108-g001.jpg
figlink: pmc/articles/PMC7570223/figure/plants-09-01108-f001/
number: F1
caption: 'Transcriptomic reconfiguration of pathways involved in energy metabolism
  in response to O2 deficiency. Through a comparative analysis between two genotypes
  of Prunus rootstocks, the hypoxia-tolerant ‘Mariana 2624’ (‘M26′) and the hypoxia-sensitive
  ‘Mazzard F12/1’ (‘F12′), the changes in the transcript levels of genes belonging
  to the routes typically related to energy metabolism (starch/sucrose, glycolysis/gluconeogenesis
  and lactic/ethanolic fermentations) and that are expressed in a contrasting way
  between the genotypes are described. Along with these routes, their connections
  to TCA cycle, GABA shunt and the GS/GOGAT cycle are presented. The differential
  transcript levels of the genes of the last two pathways show clear differences that
  point to the favoring of one or another metabolic pathway in a genotype-dependent
  manner. In addition, the steps where the regeneration of NAD+, consumption of excess
  H+ (regulation of cytoplasmic pH) and assimilation of NH4+ take place are shown.
  The data included in this re-analysis were obtained through a de novo approach via
  RNAseq data retrieved from NCBI BioProjects PRJNA215360 and PRJNA215068 [], corresponding
  to P. cerasifera × P. munsoniana ‘Mariana 2624’ and P. avium ‘Mazzard F12/1’, respectively.
  Quality control of the libraries was performed with FastQC and the AfterQC tool
  []. Consecutively, all libraries were merged into one unique file to perform a de
  novo assembly with Trinity []. The resulting fasta file was depurated to obtain
  unigenes using Transdecoder with the pFam database [] and CD-HIT-EST [,]. This output
  was used as a reference to perform an alignment using Hisat2 []. Transcript assembly
  was performed through StringTie []. DEGs were obtained using EdgeR from Bioconductor
  []. Genes with adjusted p-value < 0.05 and LogFC > 2 & < −2 were used for further
  analysis. The annotation was performed using GO FEAT [] and KEGG []. DEGs included
  in this figure are indicated with their homolog loci from P. persica: SuSy—(1) Prupe.7G192300.4;
  (2) Prupe.8G264300.1; (3) Prupe.1G131700.1; (4) Prupe.7G192300.1. Invertase—(1)
  Prupe.2G277900.1; (2) Prupe.2G277900.1; (3) Prupe.6G122600.1; (4) Prupe.6G122600.2;
  (5) Prupe.1G111800.1. Hexokinase 3—(1) Prupe.1G366000. SnRK1,1—(1) Prupe.3G262900.1.
  Phosphoglucomutase—(1) Prupe.2G286900.1; (2) Prupe.1G330700.1. PEPCK (ATP)—(1) Prupe.6G211000.1;
  (2) Prupe.1G541200.7. PEPCK (GTP)—(1) Prupe.1G541200.4; (2) Prupe.6G210900.1; (3)
  Prupe.6G211000.1; (4) Prupe.1G541200.7; (5) Prupe.4G166400.4. DHLTA—(1) Prupe.1G309100.1;
  (2) Prupe.8G056000.1. Glutamate synthase—(1) Prupe.2G311700.2. Glutamine syntethase—(1)
  Prupe.1G148700.1; (2) Prupe.3G166500.3; (3) Prupe.1G346600.1; (4) Prupe.5G236300.1;
  (5) Prupe.3G166500.2. GDH—(1) Prupe.7G004100.1; (2) Prupe.2G269800.1. GAD—(1) Prupe.1G339900.1;
  (2) Prupe.7G252300.1. For more detailed results of the transcriptional patterns
  of LDH, PDC and ADH (Quantitative Reverse Transcription (qRT)-PCR) and alanine levels
  (HPLC-DAD) inserted in this figure, refer to [,].'
papertitle: 'Keep Calm and Survive: Adaptation Strategies to Energy Crisis in Fruit
  Trees under Root Hypoxia.'
reftext: Ariel Salvatierra, et al. Plants (Basel). 2020 Sep;9(9):1108.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9029781
figid_alias: PMC7570223__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC7570223__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7570223__plants-09-01108-g001.html
  '@type': Dataset
  description: 'Transcriptomic reconfiguration of pathways involved in energy metabolism
    in response to O2 deficiency. Through a comparative analysis between two genotypes
    of Prunus rootstocks, the hypoxia-tolerant ‘Mariana 2624’ (‘M26′) and the hypoxia-sensitive
    ‘Mazzard F12/1’ (‘F12′), the changes in the transcript levels of genes belonging
    to the routes typically related to energy metabolism (starch/sucrose, glycolysis/gluconeogenesis
    and lactic/ethanolic fermentations) and that are expressed in a contrasting way
    between the genotypes are described. Along with these routes, their connections
    to TCA cycle, GABA shunt and the GS/GOGAT cycle are presented. The differential
    transcript levels of the genes of the last two pathways show clear differences
    that point to the favoring of one or another metabolic pathway in a genotype-dependent
    manner. In addition, the steps where the regeneration of NAD+, consumption of
    excess H+ (regulation of cytoplasmic pH) and assimilation of NH4+ take place are
    shown. The data included in this re-analysis were obtained through a de novo approach
    via RNAseq data retrieved from NCBI BioProjects PRJNA215360 and PRJNA215068 [],
    corresponding to P. cerasifera × P. munsoniana ‘Mariana 2624’ and P. avium ‘Mazzard
    F12/1’, respectively. Quality control of the libraries was performed with FastQC
    and the AfterQC tool []. Consecutively, all libraries were merged into one unique
    file to perform a de novo assembly with Trinity []. The resulting fasta file was
    depurated to obtain unigenes using Transdecoder with the pFam database [] and
    CD-HIT-EST [,]. This output was used as a reference to perform an alignment using
    Hisat2 []. Transcript assembly was performed through StringTie []. DEGs were obtained
    using EdgeR from Bioconductor []. Genes with adjusted p-value < 0.05 and LogFC
    > 2 & < −2 were used for further analysis. The annotation was performed using
    GO FEAT [] and KEGG []. DEGs included in this figure are indicated with their
    homolog loci from P. persica: SuSy—(1) Prupe.7G192300.4; (2) Prupe.8G264300.1;
    (3) Prupe.1G131700.1; (4) Prupe.7G192300.1. Invertase—(1) Prupe.2G277900.1; (2)
    Prupe.2G277900.1; (3) Prupe.6G122600.1; (4) Prupe.6G122600.2; (5) Prupe.1G111800.1.
    Hexokinase 3—(1) Prupe.1G366000. SnRK1,1—(1) Prupe.3G262900.1. Phosphoglucomutase—(1)
    Prupe.2G286900.1; (2) Prupe.1G330700.1. PEPCK (ATP)—(1) Prupe.6G211000.1; (2)
    Prupe.1G541200.7. PEPCK (GTP)—(1) Prupe.1G541200.4; (2) Prupe.6G210900.1; (3)
    Prupe.6G211000.1; (4) Prupe.1G541200.7; (5) Prupe.4G166400.4. DHLTA—(1) Prupe.1G309100.1;
    (2) Prupe.8G056000.1. Glutamate synthase—(1) Prupe.2G311700.2. Glutamine syntethase—(1)
    Prupe.1G148700.1; (2) Prupe.3G166500.3; (3) Prupe.1G346600.1; (4) Prupe.5G236300.1;
    (5) Prupe.3G166500.2. GDH—(1) Prupe.7G004100.1; (2) Prupe.2G269800.1. GAD—(1)
    Prupe.1G339900.1; (2) Prupe.7G252300.1. For more detailed results of the transcriptional
    patterns of LDH, PDC and ADH (Quantitative Reverse Transcription (qRT)-PCR) and
    alanine levels (HPLC-DAD) inserted in this figure, refer to [,].'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Hex-A
  - Su(osk)M26
  - Pep
  - bgcn
  - Pepck2
  - Pepck1
  - Ldh
  - gs
  - Coa
  - Adh
  - CycE
  - cyc
  - Gdh
  - Gpdh1
  - Gld
  - GABA-B-R1
  - Rdl
  - Gabat
  - Gad1
  - ss
  - E(spl)mgamma-HLH
  - fw
  - HK1
  - GCK
  - HK2
  - HK3
  - F12
  - PEPC
  - PGC
  - PAEP
  - PREP
  - PTPN22
  - PCK2
  - MTG1
  - LDHA
  - LDHB
  - LDHC
  - GNAS
  - GNAL
  - PDC
  - PNKD
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - GLUD1
  - UGDH
  - H6PD
  - CRYL1
  - GAD1
  - GAD2
  - CALR
  - TRIM21
  - sucrose
  - GTP
  - NAD
  - lactate
  - alanine
  - Glutamate
  - Alanine
  - ethanol
  - acetyl CoA
  - Glutamine
  - TCA
  - 2-oxoglutarate
  - glutamate
  - succinate
  - GABA
  - glutamine
  - GS
---
