---
figid: PMC7504089__ijms-21-06111-g007
figtitle: Differential gene expression in apoptosis and death receptor signaling pathway
  in ‘ECACC vs
organisms:
- Adenoviridae
- Influenza A virus
- unidentified influenza virus
- Orthomyxoviridae
- H1N1 subtype
- H3N2 subtype
- Human adenovirus sp.
- Canine mastadenovirus A
- Canine adenovirus 2
- Bos taurus
- Canis lupus familiaris
- Homo sapiens
- Rattus norvegicus
- Gallus gallus
- Matthiola incana
organisms_ner:
- Canis lupus familiaris
- Rattus norvegicus
- Bos taurus
- Homo sapiens
- Gallus gallus
- Danio rerio
pmcid: PMC7504089
filename: ijms-21-06111-g007.jpg
figlink: pmc/articles/PMC7504089/figure/ijms-21-06111-f007/
number: F7
caption: 'Differential gene expression in apoptosis and death receptor signaling pathway
  in ‘ECACC vs. ATCC’ cells. Differential expression patterns for pairwise comparison
  of ‘ECACC vs. ATCC cells’ matching culture conditions and infection status. Differential
  expression values of 2-fold occurring in, at least one, of the pairwise comparisons
  are indicated in dark red and dark blue. Since pathway analysis is based on orchestrated
  changes, even if small, more modest expression changes (between 1.5-fold and 2-fold)
  are also showing in light red (up-regulation) and light blue (down-regulation),
  although these modest changes were not used as a differential expression cut-off
  for running pathway analysis. Dashed genes indicated genes for which no probes were
  found in the array. Map downloaded from Pathway Central (SABiosciences, now part
  of Qiagen) and manually curated and adapted. Absolute gene expression values and
  gene expression fold-change for each gene are given in . For a matter of simplicity,
  only gene symbols are shown: the official full name of each gene was omitted and
  can be found in .'
papertitle: Cell Bank Origin of MDCK Parental Cells Shapes Adaptation to Serum-Free
  Suspension Culture and Canine Adenoviral Vector Production.
reftext: Ana Filipa Rodrigues, et al. Int J Mol Sci. 2020 Sep;21(17):6111.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9714244
figid_alias: PMC7504089__F7
figtype: Figure
redirect_from: /figures/PMC7504089__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7504089__ijms-21-06111-g007.html
  '@type': Dataset
  description: 'Differential gene expression in apoptosis and death receptor signaling
    pathway in ‘ECACC vs. ATCC’ cells. Differential expression patterns for pairwise
    comparison of ‘ECACC vs. ATCC cells’ matching culture conditions and infection
    status. Differential expression values of 2-fold occurring in, at least one, of
    the pairwise comparisons are indicated in dark red and dark blue. Since pathway
    analysis is based on orchestrated changes, even if small, more modest expression
    changes (between 1.5-fold and 2-fold) are also showing in light red (up-regulation)
    and light blue (down-regulation), although these modest changes were not used
    as a differential expression cut-off for running pathway analysis. Dashed genes
    indicated genes for which no probes were found in the array. Map downloaded from
    Pathway Central (SABiosciences, now part of Qiagen) and manually curated and adapted.
    Absolute gene expression values and gene expression fold-change for each gene
    are given in . For a matter of simplicity, only gene symbols are shown: the official
    full name of each gene was omitted and can be found in .'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TNF
  - TNFSF12
  - TNFSF10
  - APP
  - FASLG
  - TNFRSF1A
  - NGF
  - RIPK1
  - CRADD
  - CASP2
  - TRADD
  - FADD
  - DAXX
  - TRAF2
  - CASP10
  - MAP2K4
  - MAP2K7
  - CFLAR
  - IKBKB
  - CHUK
  - BIRC2
  - BIRC3
  - IKBKG
  - NFKBIB
  - CASP12
  - NFKB1
  - CASP9
  - CASP7
  - BID
  - CASP6
  - CASP3
  - ERN1
  - DIABLO
  - APAF1
  - HTRA2
  - CALM2
  - BCL2
  - ARHGDIB
  - DFFA
  - GAS2
  - CASP4
  - DAPK1
  - CYCS
  - BAD
  - BAX
  - DFFB
  - FOSB
  - FOS
  - JUN
  - FOSL1
  - FOSL2
  - Tnf
  - Tnfsf12
  - Tnfsf10
  - App
  - Faslg
  - Tnfrsf1a
  - Ngf
  - Ripk1
  - Ippk
  - Cradd
  - Casp2
  - Tradd
  - Fadd
  - Daxx
  - Traf2
  - Mapk8
  - Nfkb1
  - Map2k4
  - Map2k7
  - Cflar
  - Ikbkb
  - Chuk
  - Birc2
  - Birc3
  - Ikbkg
  - Nfkbib
  - Casp12
  - Casp9
  - Casp7
  - Bid
  - Casp6
  - Casp3
  - Ern1
  - Diablo
  - Apaf1
  - Htra2
  - Bcl2
  - Arhgdib
  - Dffa
  - Gas2
  - Gas2l2
  - Casp1
  - Dapk1
  - Cycs
  - Bad
  - Bax
  - Dffb
  - Fosb
  - Fos
  - Jun
  - Fosl1
  - Fosl2
  - RNPEP
  - CASP1
  - DAPK2
  - DAPK3
  - STK17A
  - STK17B
  - TANK
  - CASP5
  - CASP8
  - CASP14
  - MAPK8
  - MAPK9
  - MAPK10
  - NFKB2
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - JUNB
  - JUND
  - PPY
  - tnfrsfa
  - tnfb
  - tnfsf12
  - tnfsf10
  - appa
  - faslg
  - tnfrsf1a
  - ngfb
  - cradd
  - casp2
  - tradd
  - fadd
  - daxx
  - casp10
  - mapk8b
  - map2k4a
  - map2k7
  - cflara
  - ikbkb
  - chuk
  - birc2
  - ikbkg
  - nfkbib
  - nfkb1
  - casp9
  - casp7
  - bida
  - casp6a
  - casp3a
  - ern1
  - diabloa
  - apaf1
  - bcl2a
  - arhgdig
  - dffa
  - gas2
  - gas2b
  - caspa
  - dapk1
  - krt18a.1
  - cycsa
  - badb
  - baxa
  - dffb
  - fosb
  - fosab
  - jun
  - fosl1a
  - fosl2
---
