---
figid: PMC7327986__gr5
figtitle: NRF2 signaling pathway in the context of environmental pollutants induced
  redox toxicity and its regulation
organisms:
- Saccharomyces cerevisiae
- Nicotiana tabacum
- Oryza sativa
- Artemisia campestris
- Bathysa cuspidata
- Danio rerio
- Ctenopharyngodon idella
- Geositta cunicularia
- Mus musculus
- Rattus norvegicus
- gut metagenome
- uncultured marine bacterium
- Homo sapiens
- Bos taurus
- Drosophila melanogaster
- Aliivibrio fischeri
pmcid: PMC7327986
filename: gr5.jpg
figlink: pmc/articles/PMC7327986/figure/fig5/
number: F5
caption: NRF2 signaling pathway in the context of environmental pollutants induced
  redox toxicity and its regulation. Exposure of environmental toxicants such as atmospheric
  pollutants, pesticides and heavy metals could cause oxidative stress, including
  ROS and electrophiles. Besides the well-known PTP1B/EGFR, PTEN/AKT and HSP90/HSF1
  pathways, NRF2 can be activated. This NRF2 activation can be induced by the negative
  regulation of KEAP1 through Cys residues or by the inhibition of KEAP1/NRF2 complex
  stability via p21, p62 and PKC. Upon the dissociation of KEAP1, NRF2 no longer proceeds
  to the ubiquitination and degradation, resulting in the translocation to the nucleus.
  The translocated NRF2 heterodimerizes with other nuclear factors such as sMAF and
  binds to AREs, driven the activation of detoxing enzymes and antioxidants. This
  process is also epigenetically regulated. For instance, the hypermethylation of
  KEAP1 promoter results in NRF2 nuclear translocation, while the hypermethylation
  of NRF2 promoter represses NRF2. Environmental pollutants could modulate histone
  acetylation through histone acetyltransferases (HATs) or histone deacetylases (HDACs).
  Finally, miRNAs and LncRNAs also involve in this regulation process.
papertitle: Redox toxicology of environmental chemicals causing oxidative stress.
reftext: Fuli Zheng, et al. Redox Biol. 2020 Jul;34:101475.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9350295
figid_alias: PMC7327986__F5
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Bos taurus
- Saccharomyces cerevisiae S288C
- Danio rerio
- Ctenopharyngodon idella
redirect_from: /figures/PMC7327986__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7327986__gr5.html
  '@type': Dataset
  description: NRF2 signaling pathway in the context of environmental pollutants induced
    redox toxicity and its regulation. Exposure of environmental toxicants such as
    atmospheric pollutants, pesticides and heavy metals could cause oxidative stress,
    including ROS and electrophiles. Besides the well-known PTP1B/EGFR, PTEN/AKT and
    HSP90/HSF1 pathways, NRF2 can be activated. This NRF2 activation can be induced
    by the negative regulation of KEAP1 through Cys residues or by the inhibition
    of KEAP1/NRF2 complex stability via p21, p62 and PKC. Upon the dissociation of
    KEAP1, NRF2 no longer proceeds to the ubiquitination and degradation, resulting
    in the translocation to the nucleus. The translocated NRF2 heterodimerizes with
    other nuclear factors such as sMAF and binds to AREs, driven the activation of
    detoxing enzymes and antioxidants. This process is also epigenetically regulated.
    For instance, the hypermethylation of KEAP1 promoter results in NRF2 nuclear translocation,
    while the hypermethylation of NRF2 promoter represses NRF2. Environmental pollutants
    could modulate histone acetylation through histone acetyltransferases (HATs) or
    histone deacetylases (HDACs). Finally, miRNAs and LncRNAs also involve in this
    regulation process.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - mn
  - ar
  - cd
  - dap
  - Egfr
  - Cys
  - mmy
  - Csp
  - ref(2)P
  - DCTN4-p62
  - Tfb1
  - Nup62
  - Taf6
  - how
  - Pten
  - Akt
  - Keap1
  - Hsp83
  - Gp93
  - Hsf
  - cnc
  - HDAC4
  - HDAC6
  - HDAC1
  - HDAC3
  - sgg
  - hat
  - Elp3
  - CG1894
  - chm
  - inaC
  - Pkc98E
  - aPKC
  - Pkc53E
  - Ho
  - dm
  - Ar
  - Cdkn1a
  - Nsg1
  - Tpt1
  - Tceal1
  - Gtf2h1
  - Sqstm1
  - Khdrbs1
  - Dctn4
  - Akt1
  - Hsp90ab1
  - Hsp90aa1
  - Hsp84-2
  - Hsp84-3
  - Hsp86-ps2
  - Hsp86-ps1
  - Hsf1
  - Nfe2l2
  - Gsk3b
  - Hmox1
  - Alpha-re
  - AR
  - CDKN1A
  - TCEAL1
  - NSG1
  - H3P16
  - EGFR
  - GTF2H1
  - SQSTM1
  - KHDRBS1
  - NUP62
  - DCTN4
  - PTEN
  - AKT1
  - AKT2
  - AKT3
  - KEAP1
  - HSP90AA1
  - HSP90B1
  - TRAP1
  - HSP90AB1
  - HSF1
  - NR5A1
  - GABPA
  - NFE2L2
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - HDAC9
  - HDAC2
  - HDAC8
  - HDAC5
  - HDAC7
  - HDAC10
  - SIRT1
  - SIRT2
  - SIRT3
  - SIRT4
  - SIRT5
  - SIRT6
  - SIRT7
  - HDAC11
  - GSK3A
  - GSK3B
  - TMPRSS11D
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - HMOX1
  - Kras
  - H2bc1
  - Prkcg
  - PTP1
  - HSC82
  - HSP82
  - RIM11
  - cdkn1a
  - egfra
  - khdrbs1a
  - sqstm1
  - ptenb
  - keap1a
  - hsp90aa1.1
  - hsf1
  - nfe2l2a
  - gsk3ba
  - gsk3ab
  - hsp90ab1
  - hsp90b1
  - hmox1a
  - MeHg
  - Mn
  - quinones
  - Cd
  - Cys
  - DM
---
