---
figid: PMC7848718__pnas.2023236118fig04
figtitle: Longitudinal shear stress response in human endothelial cells to atheroprone
  and atheroprotective conditions
organisms:
- Homo sapiens
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC7848718
filename: pnas.2023236118fig04.jpg
figlink: /pmc/articles/PMC7848718/figure/fig04/
number: F4
caption: 'Comparison of OS vs. PS differential expression response of (A) HAECs and
  (B) HUVECs for Oxidative Stress pathway (WikiPathways)-related genes. Color scale
  of log2 fold changes: −1 (blue) to 0 (white) to +1 (red). The overall response at
  the pathway level is similar for the two cell types, although some genes show stronger
  fold changes in HAECs vs. HUVECs (e.g., NOX4) or weaker fold changes in HAECs vs.
  HUVECs (e.g., CYP1A1 or SOD2). For some genes, e.g., JUNB, the difference is temporal.'
papertitle: Longitudinal shear stress response in human endothelial cells to atheroprone
  and atheroprotective conditions.
reftext: Mano R. Maurya, et al. Proc Natl Acad Sci U S A. 2021 Jan 26;118(4):e2023236118.
year: '2021'
doi: 10.1073/pnas.2023236118
journal_title: Proceedings of the National Academy of Sciences of the United States
  of America
journal_nlm_ta: Proc Natl Acad Sci U S A
publisher_name: National Academy of Sciences
keywords: atherosclerosis | endothelial cells | systems biology | temporal analysis
  of flow response | transcriptomics
automl_pathway: 0.8448606
figid_alias: PMC7848718__F4
figtype: Figure
redirect_from: /figures/PMC7848718__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7848718__pnas.2023236118fig04.html
  '@type': Dataset
  description: 'Comparison of OS vs. PS differential expression response of (A) HAECs
    and (B) HUVECs for Oxidative Stress pathway (WikiPathways)-related genes. Color
    scale of log2 fold changes: −1 (blue) to 0 (white) to +1 (red). The overall response
    at the pathway level is similar for the two cell types, although some genes show
    stronger fold changes in HAECs vs. HUVECs (e.g., NOX4) or weaker fold changes
    in HAECs vs. HUVECs (e.g., CYP1A1 or SOD2). For some genes, e.g., JUNB, the difference
    is temporal.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SGSM3
  - TF
  - NFE2L2
  - NFKB1
  - NFKB2
  - SP1
  - PSG1
  - DAND5
  - MAPK10
  - MAPK14
  - CAT
  - CRAT
  - GLYAT
  - TXN2
  - TXNRD1
  - TXNRD2
  - GCLC
  - MT1X
  - NOX1
  - CYBB
  - NOX3
  - HMOX1
  - GPX1
  - GPX3
  - NQO1
  - UGT1A6
  - JUNB
  - RNASEH2A
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - XDH
  - SORD
  - MAOA
  - GSR
  - GSTT2
  - MGST1
  - SOD1
  - SOD2
  - SOD2-OT1
  - SOD3
  - NFIX
  - CYP1A1
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK8
  - MAPK9
  - MT1A
  - MT1B
  - MT1E
  - MT1F
  - MT1G
  - MT1H
  - MT1JP
  - MT1M
  - MT1L
  - MTNR1A
  - ALG1
  - MT1IP
  - NOX4
  - pak2a
  - itpka
  - nfe2l2a
  - nfkb1
  - sp1
  - mapk10
  - mapk14a
  - mapk14b
  - cat
  - txn2
  - txnrd3
  - txnrd2.2
  - gclc
  - nox1
  - cybb
  - hmox1a
  - gpx1a
  - gpx3
  - nqo1
  - ugt1a6
  - junba
  - fosab
  - xdh
  - aox5
  - gsr
  - gstt2
  - gstt1b
  - mgst1.1
  - sod1
  - sod2
  - sod3b
  - nfixa
  - cyp1a
  - mt
  - mtnr1aa
  - mt2
  - nox4
---
