---
figid: PMC7716236__cells-09-02423-g003
figtitle: Metabolic variations in the course of NB4 differentiation along the myeloid
  lineage after treatment with ATRA
organisms:
- NA
pmcid: PMC7716236
filename: cells-09-02423-g003.jpg
figlink: pmc/articles/PMC7716236/figure/cells-09-02423-f003/
number: F3
caption: Metabolic variations in the course of NB4 differentiation along the myeloid
  lineage after treatment with ATRA. (A) WikiPathways analysis of the glycolytic pathway
  enriched in ATRA-regulated genes (FC|1.5|; FDR < 5%). The Transcriptome Analysis
  Console (TAC) 4.0.2 (Affymetrix, Inc.) allowed to observe that the “metabolic reprogramming
  pathway” was significantly enriched (p < 0.001; Fisher’s exact test) in ATRA-regulated
  genes (nine upregulated genes, and one downregulated gene). Red and green boxes
  represent up and downregulated genes, respectively. (B) WikiPathways analysis of
  OXPHOS system enriched in ATRA-regulated genes (FC|1.5|; FDR < 5%). The “metabolic
  reprogramming pathway” was found significantly enriched (p < 0.001; Fisher’s exact
  test) in ATRA regulated genes (10 upregulated gene, and 7 downregulated genes).
  Red and green boxes represent up and downregulated genes, respectively. (C) Validation
  of some differentially expressed glycolytic and OXPHOS enzymes in NB4 cells treated
  with 1 μM ATRA for 120 h in comparison with untreated NB4 cells. Representative
  immunoblot analysis of Aldolase C, LDHB, NDUFB6, PGAM1/4, and SDHA proteins. GAPDH
  and β-Actin were used as loading controls. (D) Quantification of immunoblot experiments.
  Data are reported as mean ± SD of at least three independent experiments (Student’s
  t-test, ** p < 0.01; *** p < 0.001, with respect to relative controls; a.d.u., arbitrary
  densitometric unit).
papertitle: Transcriptional and Metabolic Dissection of ATRA-Induced Granulocytic
  Differentiation in NB4 Acute Promyelocytic Leukemia Cells.
reftext: Jacopo Albanesi, et al. Cells. 2020 Nov;9(11):2423.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.750118
figid_alias: PMC7716236__F3
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC7716236__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7716236__cells-09-02423-g003.html
  '@type': Dataset
  description: Metabolic variations in the course of NB4 differentiation along the
    myeloid lineage after treatment with ATRA. (A) WikiPathways analysis of the glycolytic
    pathway enriched in ATRA-regulated genes (FC|1.5|; FDR < 5%). The Transcriptome
    Analysis Console (TAC) 4.0.2 (Affymetrix, Inc.) allowed to observe that the “metabolic
    reprogramming pathway” was significantly enriched (p < 0.001; Fisher’s exact test)
    in ATRA-regulated genes (nine upregulated genes, and one downregulated gene).
    Red and green boxes represent up and downregulated genes, respectively. (B) WikiPathways
    analysis of OXPHOS system enriched in ATRA-regulated genes (FC|1.5|; FDR < 5%).
    The “metabolic reprogramming pathway” was found significantly enriched (p < 0.001;
    Fisher’s exact test) in ATRA regulated genes (10 upregulated gene, and 7 downregulated
    genes). Red and green boxes represent up and downregulated genes, respectively.
    (C) Validation of some differentially expressed glycolytic and OXPHOS enzymes
    in NB4 cells treated with 1 μM ATRA for 120 h in comparison with untreated NB4
    cells. Representative immunoblot analysis of Aldolase C, LDHB, NDUFB6, PGAM1/4,
    and SDHA proteins. GAPDH and β-Actin were used as loading controls. (D) Quantification
    of immunoblot experiments. Data are reported as mean ± SD of at least three independent
    experiments (Student’s t-test, ** p < 0.01; *** p < 0.001, with respect to relative
    controls; a.d.u., arbitrary densitometric unit).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TCF23
  - CTRL
  - CALCR
  - SLC31A1
  - CCDC174
  - LDHB
  - ALDOC
  - GAPDH
  - GAPDHP44
  - AMPD1
  - LMNA
  - PGAM1
  - PGAM4
  - NDUFB6
  - SDHA
  - ACTA1
  - ACTA2
  - ACTB
  - ACTC1
  - ACTG1
  - ACTG2
  - ALDOA
  - LDHA
  - LDHC
  - ATP5F1A
  - ATP5F1B
  - ATP5F1C
  - ATP5F1D
  - ATP5F1E
  - ATP5MK
  - ATP5ME
  - ATP5MF
  - ATP5MG
  - ATP5MJ
  - .na.character
  - ATP5PB
  - ATP5PD
  - ATP5PF
  - ATP5PO
  - ATP5IF1
  - ATP5MC1
  - ATP5MC2
  - ATP5MC3
  - ATP8A2
  - GSTZ1
  - CCL21
  - UQCR10
  - BRCA2
  - FANCD2
  - PSEN1
  - COX8A
  - CPOX
  - COX5A
  - COX5B
  - COX6C
  - COX7C
  - COX4I1
  - COX4I2
  - COX6A1
  - COX6A2
  - COX6B1
  - COX6B2
  - COX7A1
  - COX7A2
  - COX7B
  - COX7B2
  - COX8C
  - DAP
  - IAPP
  - DNPEP
  - DNER
  - ATRA
  - Pentose Phesphate
  - Triglyceride
  - TCA
  - ATP
  - NADH-Uiquinone
  - Suceinate-Ubiquinone
  - Ubiquinol
  - Adenine
  - NOUFA
  - NAD
  - FAD
  - O
  - NOUFAO
---
