---
figid: PMC7604625__EMBJ-39-e101767-g005
figtitle: Glycolysis regulates Hedgehog signalling via the plasma membrane potential
organisms:
- Saccharomyces cerevisiae
- Drosophila melanogaster
- Caenorhabditis elegans
- Schizosaccharomyces pombe
- Matthiola incana
- Beta vulgaris subsp. vulgaris
- Patella vulgata
- Major minor
- Mus musculus
- Cavia porcellus
- Homo sapiens
- Gallus gallus
- unidentified
organisms_ner:
- Gallus gallus
- Drosophila melanogaster
- Mus musculus
- Homo sapiens
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC7604625
filename: EMBJ-39-e101767-g005.jpg
figlink: pmc/articles/PMC7604625/figure/embj2019101767-fig-0002ev/
number: EV2
caption: AOverview of the glycolytic pathway and the entry into the tricarboxylic
  acid (TCA) cycle. Drosophila gene symbols of enzymes are given. PDC stands for pyruvate
  dehydrogenase complex.B–B″Two Glyceraldehyde‐3-phosphate dehydrogenase 2 (Gapdh2)
  transcripts are annotated in the Drosophila genome. Grey boxes indicate 5′ and 3′
  untranslated regions (UTR). Exons are indicated by beige boxes and introns by the
  black line (adapted from FlyBase). Red arrowheads mark the position of primers used
  to detect fully spliced cDNA; orange arrowheads mark the position of primers used
  to detect unspliced cDNA. (B′) Representative result of semi‐quantitative RT–PCR
  analysis of Gapdh2 transcripts upon knock‐down of ecd in the whole wing disc using
  C765‐GAL4 (C765>ecd RNAi, see  for the expression pattern of C765‐GAL4). The expression
  of the translation initiation factor eIF‐4a serves as an internal normalization
  control. (B″) Levels of spliced (red) and unspliced (orange) Gapdh2 mRNAs upon knock‐down
  of ecd, shown as a fold change relative to control. Levels of Gapdh2 transcripts
  were normalized to the levels of eIF‐4a. Error bars indicate ± SD, n = 6. t‐test,
  **P ≤ 0.01, ***P ≤ 0.001. Loss of Ecd reduces the levels of spliced Gapdh2 mRNA
  but increases the levels of unspliced Gapdh2 mRNA.C–C″Two Pyruvate kinase (PyK)
  transcripts are annotated in the Drosophila genome (adapted from FlyBase). (C′)
  Representative result of semi‐quantitative RT–PCR analysis of PyK transcripts upon
  knock‐down of ecd. (C″). Levels of spliced (red) and unspliced (orange) PyK mRNAs
  upon knock‐down of ecd, shown as a fold change relative to control. Levels of PyK
  transcripts were normalized to the levels of eIF‐4a. Error bars indicate ± SD, n = 6
  (red, spliced), n = 4 (orange, unspliced). t‐test, ns =  not significant, ***P ≤ 0.001.
  Loss of Ecd increases unspliced PyK mRNA.D–D″Four Pyruvate dehydrogenase kinase
  (Pdk) transcripts are annotated in the Drosophila genome (adapted from FlyBase).
  (D′) Representative result of semi‐quantitative RT–PCR analysis for Pdk transcripts
  upon knock‐down of ecd. (D″). Levels of spliced (red) and unspliced (orange) Pdk
  mRNAs upon knock‐down of ecd, shown as a fold change relative to control. Levels
  of Pdk transcripts were normalized to the levels of eIF‐4a. Error bars indicate ± SD,
  n = 3. t‐test, *P ≤ 0.05. Loss of Ecd reduces Pdk mRNA levels, consistent with results
  from mammalian cell culture (Dey et al, ).  Source data are available online for
  this figure.
papertitle: Glycolysis regulates Hedgehog signalling via the plasma membrane potential.
reftext: Stephanie Spannl, et al. EMBO J. 2020 Nov 2;39(21):e101767.
year: '2020'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.6289743
figid_alias: PMC7604625__EV2
figtype: Extended view
redirect_from: /figures/PMC7604625__EV2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7604625__EMBJ-39-e101767-g005.html
  '@type': Dataset
  description: AOverview of the glycolytic pathway and the entry into the tricarboxylic
    acid (TCA) cycle. Drosophila gene symbols of enzymes are given. PDC stands for
    pyruvate dehydrogenase complex.B–B″Two Glyceraldehyde‐3-phosphate dehydrogenase
    2 (Gapdh2) transcripts are annotated in the Drosophila genome. Grey boxes indicate
    5′ and 3′ untranslated regions (UTR). Exons are indicated by beige boxes and introns
    by the black line (adapted from FlyBase). Red arrowheads mark the position of
    primers used to detect fully spliced cDNA; orange arrowheads mark the position
    of primers used to detect unspliced cDNA. (B′) Representative result of semi‐quantitative
    RT–PCR analysis of Gapdh2 transcripts upon knock‐down of ecd in the whole wing
    disc using C765‐GAL4 (C765>ecd RNAi, see  for the expression pattern of C765‐GAL4).
    The expression of the translation initiation factor eIF‐4a serves as an internal
    normalization control. (B″) Levels of spliced (red) and unspliced (orange) Gapdh2
    mRNAs upon knock‐down of ecd, shown as a fold change relative to control. Levels
    of Gapdh2 transcripts were normalized to the levels of eIF‐4a. Error bars indicate ± SD,
    n = 6. t‐test, **P ≤ 0.01, ***P ≤ 0.001. Loss of Ecd reduces the levels of spliced
    Gapdh2 mRNA but increases the levels of unspliced Gapdh2 mRNA.C–C″Two Pyruvate
    kinase (PyK) transcripts are annotated in the Drosophila genome (adapted from
    FlyBase). (C′) Representative result of semi‐quantitative RT–PCR analysis of PyK
    transcripts upon knock‐down of ecd. (C″). Levels of spliced (red) and unspliced
    (orange) PyK mRNAs upon knock‐down of ecd, shown as a fold change relative to
    control. Levels of PyK transcripts were normalized to the levels of eIF‐4a. Error
    bars indicate ± SD, n = 6 (red, spliced), n = 4 (orange, unspliced). t‐test, ns = 
    not significant, ***P ≤ 0.001. Loss of Ecd increases unspliced PyK mRNA.D–D″Four
    Pyruvate dehydrogenase kinase (Pdk) transcripts are annotated in the Drosophila
    genome (adapted from FlyBase). (D′) Representative result of semi‐quantitative
    RT–PCR analysis for Pdk transcripts upon knock‐down of ecd. (D″). Levels of spliced
    (red) and unspliced (orange) Pdk mRNAs upon knock‐down of ecd, shown as a fold
    change relative to control. Levels of Pdk transcripts were normalized to the levels
    of eIF‐4a. Error bars indicate ± SD, n = 3. t‐test, *P ≤ 0.05. Loss of Ecd reduces
    Pdk mRNA levels, consistent with results from mammalian cell culture (Dey et al,
    ).  Source data are available online for this figure.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HA
  - PGK2
  - F10
  - CS
  - ha
  - Gapdh2
  - l(1)N15
  - Pgi
  - Pfk
  - CG15812
  - CG9932
  - eIF4A
  - Mps1
  - Ald1
  - Ald2
  - Ca-alpha1D
  - Gapdh1
  - be
  - Pgk
  - Pglym78
  - Eno
  - tRNA:Lys-TTT-1-1
  - tRNA:Lys-TTT-2-4
  - tRNA:Lys-TTT-2-1
  - tRNA:Lys-TTT-2-2
  - tRNA:Lys-TTT-2-5
  - tRNA:Lys-TTT-2-3
  - Pdk
  - Pdk1
  - PyK
  - Ldh
  - cs
  - CycE
  - cyc
  - pk
  - Bgn
  - Gpi1
  - Abcd1
  - Soat1
  - Bcl2a1d
  - Aaas
  - Cs
  - Gnat2
  - oh
  - GAPDHS
  - BGN
  - GPI
  - PGPEP1
  - PFKL
  - PFKM
  - PFKP
  - EIF4A1
  - ABCD1
  - PCID2
  - FAM110A
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - APP
  - AAAS
  - AAA1
  - PDK1
  - PDK2
  - PDK3
  - PDK4
  - PHKA2
  - LDHA
  - LDHB
  - LDHC
  - pgi
  - aldA
  - fbaA
  - pgk
  - eno
  - dld
  - glucose
  - glucose-6-phosphate
  - fructose-6-phosphate
  - fructose-1,6-biphosphate
  - glyceraldehyde-3-phosphate
  - 1,3-biphosphoglycerate
  - 2-phosphoglycerate
  - phosphoenolpyruvate
  - lactate pyruvate
  - PDCH
  - acetyl-CoA
  - TCA
---
