---
figid: PMC8138124__fmolb-08-668821-g001
figtitle: 'New Faces of old Friends: Emerging new Roles of RNA-Binding Proteins in
  the DNA Double-Strand Break Response'
organisms:
- NA
pmcid: PMC8138124
filename: fmolb-08-668821-g001.jpg
figlink: /pmc/articles/PMC8138124/figure/F1/
number: F1
caption: Overview of the chromatin-based DSB response. (A) DSB signaling is initiated
  by the MRN complex, which senses DSBs and recruits the ATM kinase to the damaged
  chromatin. Here, ATM phosphorylates the histone H2A variant H2A.X to form γ-H2AX,
  which in turn is recognized by MDC1. MDC1 is constitutively phosphorylated by CK2
  (not shown) and activates a positive feedback loop by recruiting more MRN and ATM
  to the break site. In addition, MDC1 is phosphorylated by ATM, and this phosphorylation
  event triggers the recruitment of the E3 ubiquitin ligase RNF8. RNF8 in turn ubiquitylates
  histone H1, which is then bound by a second E3 ubiquitin ligase called RNF168. RNF168
  ubiquitylates histone H2A at residues K13/K15 (H2AubK13/15). The resulting ubiquitin
  signal is bound by multiple proteins, including RNF168 itself (to create a second
  positive feedback loop that locally amplifies the DNA damage signal) and the DSB
  repair pathway choice proteins BRCA1 and 53BP1. BRCA1 recognizes ubiquitylated chromatin
  via its interaction partner RAP80 and promotes HR-mediated DSB repair. In contrast,
  53BP1is enriched at damaged chromatin by simultaneously binding to RN168-ubiquitylated
  H2A and to constitutively methylated histone H4K20me2, and promotes NHEJ-mediated
  DSB repair. CIRBP, WRAP53, and FUS are examples of RBPs that modulate DSB signaling.
  CIRBP enhances the accumulation of MRN-ATM at the DSB site, while WRAP53 facilitates
  RNF8 recruitment by stabilizing the MDC1-RNF8 interaction. FUS likely promotes the
  retention of signaling and repair factors by participating in phase separation around
  the break DSB site. (B) NHEJ-mediated DSB repair is initiated when DSB ends are
  recognized by the KU proteins, followed by recruitment of the kinase DNA-PKcs. KU-DNA-PKcs
  mostly act as a scaffold to recruit numerous DNA end-processing factors (including
  TdT and many others such as DNA polymerases λ and μ, Artemis, PNKP, and TDP1; not
  shown) that create DNA ends compatible for re-ligation. Following DNA end-processing,
  the DNA ligation complex XLF-XRCC4-DNA ligase IV is recruited to the DSB to re-join
  the broken DNA ends. Numerous RBPs promote NHEJ. For example, FUS enhances NHEJ
  by recruiting the histone deacetylase HDAC1 and as such enhancing local chromatin
  remodeling. RBMX binds to DSB ends to prevent DNA end-resection. PRPF19 recruits
  the histone methyltransferase SETMAR, which enhances the recruitment of NHEJ factors
  such as KU, and also binds to the DNA end-processing factor TdT. TDP43, RBM14, and
  SFPQ-NONO act as scaffolds that stabilize the assembly of the DNA ligation complex.
  (C) HR-directed DSB repair is initiated by extensive DNA end-resection by the nucleases
  CtIP, MRE11 (which is part of the MRN complex), EXO1 and DNA2-BLM. DNA end-resection
  generates long single-stranded DNA overhangs that are rapidly bound by RPA. Next,
  with the help of numerous mediator complexes, including the BRCA2-PALB2-BRCA1 and
  the RAD51 paralog complexes, RPA is exchanged for the DNA recombinase RAD51. The
  resulting RAD51 nucleoprotein filament then searches for homologous sequences elsewhere
  in the genome (usually the sister chromatid), where it then catalyzes strand invasion
  to form a DNA crossover called a Holliday junction. This generates a primer for
  DNA synthesis (dashed lines), which is extended by branch migration away from the
  crossover site. After DNA repair, the joint DNA molecules are resolved by cleavage
  of the crossed or non-crossed DNA strands (black arrowheads). The RBPs hnRNP UL
  1 and 2 enhance DNA end-resection by stimulating BLM recruitment. hnRNPC, GEMIN2
  and SFPQ support the formation of the RAD51 nucleoprotein filament. In addition,
  GEMIN2, SFPQ, and FUS also stimulate strand exchange. Note that DSB signaling and
  repair reaction require many additional factors, posttranslational modifications
  and various species of local non-coding RNAs that are not depicted. Selected RBPs
  that participate in the chromatin-based response are highlighted in red in all three
  panels. Additional RBPs that participate in DSB signal transduction and repair are
  listed in .
papertitle: 'New Faces of old Friends: Emerging new Roles of RNA-Binding Proteins
  in the DNA Double-Strand Break Response.'
reftext: Julie A. Klaric, et al. Front Mol Biosci. 2021;8:668821.
year: '2021'
doi: 10.3389/fmolb.2021.668821
journal_title: Frontiers in Molecular Biosciences
journal_nlm_ta: Front Mol Biosci
publisher_name: Frontiers Media S.A.
keywords: DNA double strand break | RNA-binding protein | genome stability | DNA repair
  | non-coding RNA | phase separation | DSB response | DNA damage
automl_pathway: 0.9560671
figid_alias: PMC8138124__F1
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC8138124__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8138124__fmolb-08-668821-g001.html
  '@type': Dataset
  description: Overview of the chromatin-based DSB response. (A) DSB signaling is
    initiated by the MRN complex, which senses DSBs and recruits the ATM kinase to
    the damaged chromatin. Here, ATM phosphorylates the histone H2A variant H2A.X
    to form γ-H2AX, which in turn is recognized by MDC1. MDC1 is constitutively phosphorylated
    by CK2 (not shown) and activates a positive feedback loop by recruiting more MRN
    and ATM to the break site. In addition, MDC1 is phosphorylated by ATM, and this
    phosphorylation event triggers the recruitment of the E3 ubiquitin ligase RNF8.
    RNF8 in turn ubiquitylates histone H1, which is then bound by a second E3 ubiquitin
    ligase called RNF168. RNF168 ubiquitylates histone H2A at residues K13/K15 (H2AubK13/15).
    The resulting ubiquitin signal is bound by multiple proteins, including RNF168
    itself (to create a second positive feedback loop that locally amplifies the DNA
    damage signal) and the DSB repair pathway choice proteins BRCA1 and 53BP1. BRCA1
    recognizes ubiquitylated chromatin via its interaction partner RAP80 and promotes
    HR-mediated DSB repair. In contrast, 53BP1is enriched at damaged chromatin by
    simultaneously binding to RN168-ubiquitylated H2A and to constitutively methylated
    histone H4K20me2, and promotes NHEJ-mediated DSB repair. CIRBP, WRAP53, and FUS
    are examples of RBPs that modulate DSB signaling. CIRBP enhances the accumulation
    of MRN-ATM at the DSB site, while WRAP53 facilitates RNF8 recruitment by stabilizing
    the MDC1-RNF8 interaction. FUS likely promotes the retention of signaling and
    repair factors by participating in phase separation around the break DSB site.
    (B) NHEJ-mediated DSB repair is initiated when DSB ends are recognized by the
    KU proteins, followed by recruitment of the kinase DNA-PKcs. KU-DNA-PKcs mostly
    act as a scaffold to recruit numerous DNA end-processing factors (including TdT
    and many others such as DNA polymerases λ and μ, Artemis, PNKP, and TDP1; not
    shown) that create DNA ends compatible for re-ligation. Following DNA end-processing,
    the DNA ligation complex XLF-XRCC4-DNA ligase IV is recruited to the DSB to re-join
    the broken DNA ends. Numerous RBPs promote NHEJ. For example, FUS enhances NHEJ
    by recruiting the histone deacetylase HDAC1 and as such enhancing local chromatin
    remodeling. RBMX binds to DSB ends to prevent DNA end-resection. PRPF19 recruits
    the histone methyltransferase SETMAR, which enhances the recruitment of NHEJ factors
    such as KU, and also binds to the DNA end-processing factor TdT. TDP43, RBM14,
    and SFPQ-NONO act as scaffolds that stabilize the assembly of the DNA ligation
    complex. (C) HR-directed DSB repair is initiated by extensive DNA end-resection
    by the nucleases CtIP, MRE11 (which is part of the MRN complex), EXO1 and DNA2-BLM.
    DNA end-resection generates long single-stranded DNA overhangs that are rapidly
    bound by RPA. Next, with the help of numerous mediator complexes, including the
    BRCA2-PALB2-BRCA1 and the RAD51 paralog complexes, RPA is exchanged for the DNA
    recombinase RAD51. The resulting RAD51 nucleoprotein filament then searches for
    homologous sequences elsewhere in the genome (usually the sister chromatid), where
    it then catalyzes strand invasion to form a DNA crossover called a Holliday junction.
    This generates a primer for DNA synthesis (dashed lines), which is extended by
    branch migration away from the crossover site. After DNA repair, the joint DNA
    molecules are resolved by cleavage of the crossed or non-crossed DNA strands (black
    arrowheads). The RBPs hnRNP UL 1 and 2 enhance DNA end-resection by stimulating
    BLM recruitment. hnRNPC, GEMIN2 and SFPQ support the formation of the RAD51 nucleoprotein
    filament. In addition, GEMIN2, SFPQ, and FUS also stimulate strand exchange. Note
    that DSB signaling and repair reaction require many additional factors, posttranslational
    modifications and various species of local non-coding RNAs that are not depicted.
    Selected RBPs that participate in the chromatin-based response are highlighted
    in red in all three panels. Additional RBPs that participate in DSB signal transduction
    and repair are listed in .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - ATM
  - MDC1
  - NR1I2
  - JTB
  - CIRBP
  - FUS
  - PARP1
  - WRAP53
  - RNF8
  - BRCA1
  - UIMC1
  - RNF168
  - TP53BP1
  - ALPK2
  - HDAC1
  - SETMAR
  - PRPF19
  - PRKDC
  - MATR3
  - RBMX
  - DNTT
  - RBM14
  - RBM14-RBM4
  - NONO
  - SFPQ
  - LOC654780
  - TARDBP
  - NHEJ1
  - HNRNPC
  - HNRNPDL
  - RBBP8
  - AN
  - EXO1
  - DNA2
  - RNPC3
  - BLM
  - PALB2
  - BRCA2
  - GEMIN2
  - CD74
  - POMC
  - CLIP1
  - CLIP2
  - RAD51
---
