---
figid: PMC8999811__ijms-23-03945-g002
figtitle: Jasmonate Signaling Pathway Modulates Plant Defense, Growth, and Their Trade-Offs
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Glycine max
- Solanum lycopersicum
- Embryophyta
- Malus domestica
- Salvia miltiorrhiza
- Botrytis cinerea
- Artemisia annua
- Fusarium oxysporum
- Vicia faba
- Pyricularia oryzae
- Tracheophyta
- Marchantiophyta
- Nicotiana attenuata
- Alternaria brassicicola
- Marchantia polymorpha
- Artemisia vulgaris
- Rosa chinensis
- Plectosphaerella cucumerina
- Globisporangium irregulare
- Artemisia absinthium
- Manduca sexta
- Nilaparvata lugens
- Helicoverpa armigera
- Tetranychus urticae
- Meloidogyne javanica
- Spodoptera littoralis
- Trichoplusia ni
- Bemisia tabaci
- Frankliniella occidentalis
- Laodelphax striatellus
- Meloidogyne graminicola
- Alticini
- Cnaphalocrocis medinalis
- Pseudomonas syringae
- Pseudomonas syringae pv. tomato
- Xanthomonas oryzae pv. oryzae
- Homo sapiens
- Rice black streaked dwarf virus
- NA
pmcid: PMC8999811
filename: ijms-23-03945-g002.jpg
figlink: /pmc/articles/PMC8999811/figure/ijms-23-03945-f002/
number: F2
caption: 'The terminal regulation mechanism of jasmonic acid (JA) signaling: (A) the
  dynamic banlance of JAs. Metabolic and catabolic enzymes maintain a dynamic balance
  between JA and JA-Ile. Methyl jasmonate (MeJA) is formed by JA catalyzed by jasmonic
  acid carboxyl methyltransferase (JMT). JAR1 transforms JA into JA-Ile. JA-Ile is
  continuously oxidized twice by CYP94 P450 family enzymes to generate 12-OH-JA-Ile
  and 12-COOH-JA-Ile in turn. The amide bond of JA-Ile/12-OH-JA-Ile is broken by amide
  hydrolases (IAR3/ILL5/ILL6) to generate JA/12-OH-JA, respectively. JA can be directly
  hydroxylated to 12-OH-JA and further sulfated to form 12-HSO4-JA; (B) COI1 degradation
  pattern. Jasmonate receptor COI1 forms the SCFCOI1 complex with ASK1 and CUL1. Excessive
  COI1 protein is recruited and degraded by ubiquitination through the 26S proteasome
  pathway; (C) the inert and alternative splicing of JAZ-mediated JA attenuation pattern.
  JA signal suppressor JAZ protein can bind to NINJA protein and directly or indirectly
  recruit co-inhibitor TPL to inhibit the transcription of MYC2, thereby inhibiting
  the expression of JA-responsive genes. Inert JAZs (JAZ7, JAZ8, and JAZ13) have a
  conserved EAR domain that directly interacts with TPL. JAZ10.4, a variable splice
  of JAZ10, lacks the Jas motif and mediates desensitization to JA. MED25-recruited
  splicing factors PRP39a and PRP40a promote the correct splicing of JAZ genes and
  prevent the overproduction of splicing variants, thereby regulating the activation
  of JA signal; (D) bHLH-like protein-mediated JA attenuation pattern. bHLH IIId (JAM1/2/3,
  bHLH14) proteins compete with MYC-like TFs to bind the G-box element on the promoter
  of the JA response gene, inhibit the formation of the MYC2-MED25 complex, and negatively
  regulate the JA response. Arrows: activations; bar-headed arrows: repressions.'
papertitle: Jasmonate Signaling Pathway Modulates Plant Defense, Growth, and Their
  Trade-Offs.
reftext: Cong Li, et al. Int J Mol Sci. 2022 Apr;23(7):3945.
year: '2022'
doi: 10.3390/ijms23073945
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: jasmonate | evolutionary origin | termination mechanism | growth | defense
  | growth-defense trade-off
automl_pathway: 0.8868711
figid_alias: PMC8999811__F2
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
- Homo sapiens
redirect_from: /figures/PMC8999811__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8999811__ijms-23-03945-g002.html
  '@type': Dataset
  description: 'The terminal regulation mechanism of jasmonic acid (JA) signaling:
    (A) the dynamic banlance of JAs. Metabolic and catabolic enzymes maintain a dynamic
    balance between JA and JA-Ile. Methyl jasmonate (MeJA) is formed by JA catalyzed
    by jasmonic acid carboxyl methyltransferase (JMT). JAR1 transforms JA into JA-Ile.
    JA-Ile is continuously oxidized twice by CYP94 P450 family enzymes to generate
    12-OH-JA-Ile and 12-COOH-JA-Ile in turn. The amide bond of JA-Ile/12-OH-JA-Ile
    is broken by amide hydrolases (IAR3/ILL5/ILL6) to generate JA/12-OH-JA, respectively.
    JA can be directly hydroxylated to 12-OH-JA and further sulfated to form 12-HSO4-JA;
    (B) COI1 degradation pattern. Jasmonate receptor COI1 forms the SCFCOI1 complex
    with ASK1 and CUL1. Excessive COI1 protein is recruited and degraded by ubiquitination
    through the 26S proteasome pathway; (C) the inert and alternative splicing of
    JAZ-mediated JA attenuation pattern. JA signal suppressor JAZ protein can bind
    to NINJA protein and directly or indirectly recruit co-inhibitor TPL to inhibit
    the transcription of MYC2, thereby inhibiting the expression of JA-responsive
    genes. Inert JAZs (JAZ7, JAZ8, and JAZ13) have a conserved EAR domain that directly
    interacts with TPL. JAZ10.4, a variable splice of JAZ10, lacks the Jas motif and
    mediates desensitization to JA. MED25-recruited splicing factors PRP39a and PRP40a
    promote the correct splicing of JAZ genes and prevent the overproduction of splicing
    variants, thereby regulating the activation of JA signal; (D) bHLH-like protein-mediated
    JA attenuation pattern. bHLH IIId (JAM1/2/3, bHLH14) proteins compete with MYC-like
    TFs to bind the G-box element on the promoter of the JA response gene, inhibit
    the formation of the MYC2-MED25 complex, and negatively regulate the JA response.
    Arrows: activations; bar-headed arrows: repressions.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SNRK2.4
  - SKP1
  - JMT
  - CUL1
  - PFT1
  - JAR1
  - ILL5
  - ILL6
  - IAR3
  - CYP94B3
  - ST2A
  - CYP94C1
  - PRP40A
  - NINJA
  - TPL
  - MAP3K5
  - MED25
  - MED9
  - MYC
  - MARS1
---
