---
figid: PMC8284505__nihms-1718924-f0008
figtitle: 'Molecular epigenetics: Chemical biology tools come of age'
organisms:
- Homo sapiens
- Mus musculus
- Saccharomyces cerevisiae
- Drosophila melanogaster
- Adenoviridae
organisms_ner:
- Drosophila melanogaster
- Mus musculus
pmcid: PMC8284505
filename: nihms-1718924-f0008.jpg
figlink: /pmc/articles/PMC8284505/figure/F8/
number: F8
caption: Oxidized 5-methylcytosine modifications as epigenetic regulators. (a) DNA
  modifications at the C5 position of cytosine. Successive steps of oxidation of 5mC
  serve as an active demethylation pathway, however, oxidized 5mC derivatives have
  been shown to serve distinct epigenetic regulatory functions in eukaryotic cells.
  (b) An oligo probe for the detection of 5fC at specific sites within eukaryotic
  DNA. Positioning a hydroxylamine-modified uracil four base pairs to the 5’-direction
  relative to 5fC exhibited highly efficient and selective reactions. Diaminobenzene
  is used as a catalyst to increase the rate of reaction. (c) Affinity purification
  of cytosine reader proteins using modified oligonucleotides as bait. Interacting
  proteins were identified by quantitative proteomics using the stable isotope labelling
  with amino acids in cell culture (SILAC) approach. (d) Generation of 5fC modifications
  by incorporation of photolabile 5fC precursors into nucleosome DNA. Irradiation
  produced the 5fC marks in situ, demonstrating that DNA-histone crosslinks can form
  within the folded nucleosome structure. PDB, 1kx5.
papertitle: 'Molecular epigenetics: Chemical biology tools come of age.'
reftext: John D. Bagert, et al. Annu Rev Biochem. ;90:287-320.
year: '2021'
doi: 10.1146/annurev-biochem-080120-021109
journal_title: Annual review of biochemistry
journal_nlm_ta: Annu Rev Biochem
publisher_name: ''
keywords: Chemical biology | Chromatin | Epigenetics | Chemical tools
automl_pathway: 0.8714588
figid_alias: PMC8284505__F8
figtype: Figure
redirect_from: /figures/PMC8284505__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8284505__nihms-1718924-f0008.html
  '@type': Dataset
  description: Oxidized 5-methylcytosine modifications as epigenetic regulators. (a)
    DNA modifications at the C5 position of cytosine. Successive steps of oxidation
    of 5mC serve as an active demethylation pathway, however, oxidized 5mC derivatives
    have been shown to serve distinct epigenetic regulatory functions in eukaryotic
    cells. (b) An oligo probe for the detection of 5fC at specific sites within eukaryotic
    DNA. Positioning a hydroxylamine-modified uracil four base pairs to the 5’-direction
    relative to 5fC exhibited highly efficient and selective reactions. Diaminobenzene
    is used as a catalyst to increase the rate of reaction. (c) Affinity purification
    of cytosine reader proteins using modified oligonucleotides as bait. Interacting
    proteins were identified by quantitative proteomics using the stable isotope labelling
    with amino acids in cell culture (SILAC) approach. (d) Generation of 5fC modifications
    by incorporation of photolabile 5fC precursors into nucleosome DNA. Irradiation
    produced the 5fC marks in situ, demonstrating that DNA-histone crosslinks can
    form within the folded nucleosome structure. PDB, 1kx5.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tdg
  - ap
  - ber
  - Mt2
  - Nmt
  - Ms
  - bi
  - fc
  - Sh
  - ru
  - levy
  - COX6AL2
  - hv
  - oh
  - Tnfsf14
  - Dnmt1
  - Sema6a
---
