---
figid: PMC8467246__cells-10-02378-g001
figtitle: Landscape of IFN/ISG Signaling in HIV-1-Infected Macrophages and Its Possible
  Role in the HIV-1 Latency
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC8467246
filename: cells-10-02378-g001.jpg
figlink: /pmc/articles/PMC8467246/figure/cells-10-02378-f001/
number: F1
caption: Classical and non-classical mechanisms of regulation of IFN/ISG signaling
  network. Classical regulatory mechanisms (Left). The type I IFN may conduce to activation
  of the transcriptional complex ISGF3, which is translocated to the nucleus where
  induces the expression of hundreds of ISGs such as Mx1/2, Viperin, CXCL10, Tetherin,
  APOBEC3G, RIG-1, MDA5, STAT1/2, SOCS, and USP18 []. SOCS and USP18 act inhibiting
  the IFN/ISG signaling pathway, in addition to the phosphorylation (Ser287) of STAT2
  and the downregulation of cell surface IFNAR (red arrows (−)). Another classic mechanism
  occurs at the transcriptional level, it is constituted by the interaction of ISGF3
  with coactivators and/or corepressors that recognize ISG response elements (ISRE),
  modulating the expression of ISGs. Non-classical regulatory mechanisms (Right).
  One of these non-classical mechanisms is performed by ADAR1, which is an ISG that
  can control IFN/ISG signaling by prevent RIG-1 and MDA5 sensing of viral RNAs (blue
  arrow (+)). Another non-classical mechanism is performed by miR-132-3p, which negatively
  regulates the IFN/ISG signaling interfering with the gene expression of IRF1 (red
  arrows (−)). Another example of non-classical regulation of IFN/ISG signaling is
  performed by an epigenetic mechanism, called “innate memory”, which favors an accelerated
  recruitment of RNA polymerase II and transcription/chromatin factors associated
  via H3K36me3, promoting mRNA transcription of associated molecules of the IFN/ISG
  signaling pathway. Finally, a recently described non-classical mechanism of regulation
  of IFN/ISG signaling pathway, involves the epitranscriptomic regulation through
  of the m6A-machinery, controlling the fate of IFN and ISG mRNAs for degradation
  or translation [,,,]. Figure created with Biorender.com (accessed on 15 July 2021).
papertitle: The Landscape of IFN/ISG Signaling in HIV-1-Infected Macrophages and Its
  Possible Role in the HIV-1 Latency.
reftext: Masyelly Rojas, et al. Cells. 2021 Sep;10(9):2378.
year: '2021'
doi: 10.3390/cells10092378
journal_title: Cells
journal_nlm_ta: Cells
publisher_name: MDPI
keywords: HIV | latent HIV-1 reservoir | macrophages | IFN/ISG response | epitranscriptomic
  regulation
automl_pathway: 0.9269608
figid_alias: PMC8467246__F1
figtype: Figure
redirect_from: /figures/PMC8467246__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8467246__cells-10-02378-g001.html
  '@type': Dataset
  description: Classical and non-classical mechanisms of regulation of IFN/ISG signaling
    network. Classical regulatory mechanisms (Left). The type I IFN may conduce to
    activation of the transcriptional complex ISGF3, which is translocated to the
    nucleus where induces the expression of hundreds of ISGs such as Mx1/2, Viperin,
    CXCL10, Tetherin, APOBEC3G, RIG-1, MDA5, STAT1/2, SOCS, and USP18 []. SOCS and
    USP18 act inhibiting the IFN/ISG signaling pathway, in addition to the phosphorylation
    (Ser287) of STAT2 and the downregulation of cell surface IFNAR (red arrows (−)).
    Another classic mechanism occurs at the transcriptional level, it is constituted
    by the interaction of ISGF3 with coactivators and/or corepressors that recognize
    ISG response elements (ISRE), modulating the expression of ISGs. Non-classical
    regulatory mechanisms (Right). One of these non-classical mechanisms is performed
    by ADAR1, which is an ISG that can control IFN/ISG signaling by prevent RIG-1
    and MDA5 sensing of viral RNAs (blue arrow (+)). Another non-classical mechanism
    is performed by miR-132-3p, which negatively regulates the IFN/ISG signaling interfering
    with the gene expression of IRF1 (red arrows (−)). Another example of non-classical
    regulation of IFN/ISG signaling is performed by an epigenetic mechanism, called
    “innate memory”, which favors an accelerated recruitment of RNA polymerase II
    and transcription/chromatin factors associated via H3K36me3, promoting mRNA transcription
    of associated molecules of the IFN/ISG signaling pathway. Finally, a recently
    described non-classical mechanism of regulation of IFN/ISG signaling pathway,
    involves the epitranscriptomic regulation through of the m6A-machinery, controlling
    the fate of IFN and ISG mRNAs for degradation or translation [,,,]. Figure created
    with Biorender.com (accessed on 15 July 2021).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - IFNAR1
  - IFNAR2
  - STAT1
  - STAT2
  - IRF9
  - IFNB1
  - IFNA1
  - MX1
  - MX2
  - CXCL10
  - BST2
  - ISG15
  - APOBEC3G
  - PLAAT4
  - RIGI
  - ROBO3
  - IFIH1
  - CISH
  - USP18
  - ADAR
  - IRF1
  - YTHDF2
  - METTL3
  - ISG20
  - METTL14
  - ALKBH5
  - GPM6A
---
