---
figid: PMC8532513__MSB-17-e10427-g008
figtitle: The framework used in this work to explore how new traits evolved across
  yeast subphylum by combining systematic evolution analysis with the metabolic network
  reconstruction and simulation
organisms:
- Saccharomyces cerevisiae
- Candida albicans
- Schizosaccharomyces pombe
- Aspergillus nidulans
- Komagataella pastoris
- Neurospora crassa
- Yarrowia lipolytica
- Saccharomyces cerevisiae S288C
- Fusarium graminearum
- Nakaseomyces glabratus
- Kluyveromyces lactis
- Sclerotinia sclerotiorum
- Kluyveromyces marxianus
- Candida auris
- Eremothecium gossypii
- Parastagonospora nodorum
- Komagataella phaffii
- Zygosaccharomyces rouxii
- Ogataea polymorpha
- Torulaspora delbrueckii
- Naumovozyma castellii
- Ogataea parapolymorpha
- Saccharomycetales
- Lachancea thermotolerans
- Brettanomyces bruxellensis
- Lachancea fermentati
- Saccharomyces eubayanus
- Tetrapisispora phaffii
- Naumovozyma dairenensis
- Tetrapisispora blattae
- Kazachstania africana
- Kluyveromyces dobzhanskii
- Xylona heveae
- Eremothecium coryli
- Eremothecium sinecaudum
- Nadsonia fulvescens
- Escherichia coli
- Streptomyces coelicolor
- Brucella abortus 2308
- unidentified
- other sequences
- Homo sapiens
- Corvus corax
- NA
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8532513
filename: MSB-17-e10427-g008.jpg
figlink: /pmc/articles/PMC8532513/figure/msb202110427-fig-0001/
number: F1
caption: AReconstruction of 343 species‐specific GEMs by leveraging the pan‐GEM expanded
  from the S. cerevisiae GEM Yeast8 (left); and different kinds of sequence‐based
  evolution analysis conducted in this work (right). During the evolution analysis,
  the gene ortholog group, protein 3D structure data and functional site annotation
  (blue part) were used as the basis for other more in‐depth calculations (red part).BPotential
  mechanisms underlying trait diversity of yeast. Pathway expansion by distinct enzymes
  could enable novel substrate utilization. Divergent sequence evolution could further
  change the specialty of enzymes by increasing its activity. Lastly, long‐term site‐wise
  selection will fix useful mutations to increase the cellular fitness under specific
  niches.CVarious trait data collected for yeast species, including the ability to
  grow on 32 carbon and nitrogen sources, and the complex traits of thermotolerance
  and Crabtree effect. All the trait data could be found in Dataset .
papertitle: Yeast metabolic innovations emerged via expanded metabolic network and
  gene positive selection.
reftext: Hongzhong Lu, et al. Mol Syst Biol. 2021 Oct;17(10):e10427.
year: '2021'
doi: 10.15252/msb.202110427
journal_title: Molecular Systems Biology
journal_nlm_ta: Mol Syst Biol
publisher_name: John Wiley and Sons Inc.
keywords: genome analysis | genome‐scale metabolic models | metabolic innovation |
  systems biology | Evolution & Ecology | Metabolism
automl_pathway: 0.8664362
figid_alias: PMC8532513__F1
figtype: Figure
redirect_from: /figures/PMC8532513__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8532513__MSB-17-e10427-g008.html
  '@type': Dataset
  description: AReconstruction of 343 species‐specific GEMs by leveraging the pan‐GEM
    expanded from the S. cerevisiae GEM Yeast8 (left); and different kinds of sequence‐based
    evolution analysis conducted in this work (right). During the evolution analysis,
    the gene ortholog group, protein 3D structure data and functional site annotation
    (blue part) were used as the basis for other more in‐depth calculations (red part).BPotential
    mechanisms underlying trait diversity of yeast. Pathway expansion by distinct
    enzymes could enable novel substrate utilization. Divergent sequence evolution
    could further change the specialty of enzymes by increasing its activity. Lastly,
    long‐term site‐wise selection will fix useful mutations to increase the cellular
    fitness under specific niches.CVarious trait data collected for yeast species,
    including the ability to grow on 32 carbon and nitrogen sources, and the complex
    traits of thermotolerance and Crabtree effect. All the trait data could be found
    in Dataset .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RASA1
  - RGS6
  - IL36RN
  - IL36B
  - IL37
  - IL36A
  - GPX4
  - EAF2
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Fili
  - Gem2
  - Gem3
  - gprs
---
