---
figid: PMC8593415__fonc-11-723162-g008
figtitle: Multidimensional Mutational Profiling of the Indian HNSCC Sub-Population
  Provides IRAK1, a Novel Driver Gene and Potential Druggable Target
organisms:
- NA
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC8593415
filename: fonc-11-723162-g008.jpg
figlink: /pmc/articles/PMC8593415/figure/f8/
number: F8
caption: (A) A flowchart explaining the pipeline followed for prediction of novel
  deleterious variants from in-house data. (B) This diagram shows the classification
  of 114 novel deleterious variants into 11 Tumor suppressor genes, 35 oncogenes and
  2 DNA damage and Repair genes. (C) This figure shows the distribution of the 114
  novel deleterious variants into 3 significant pathway interaction networks, namely
  Collagen Biosynthesis and Degradation, mTOR signaling and ECM Signaling. (D) A bar
  graph showing all the significantly mutated pathways by the number of genes mutated
  with the most significant pathway being receptor tyrosine kinase signaling.
papertitle: Multidimensional Mutational Profiling of the Indian HNSCC Sub-Population
  Provides IRAK1, a Novel Driver Gene and Potential Druggable Target.
reftext: Sagar Sanjiv Desai, et al. Front Oncol. 2021;11:723162.
year: '2021'
doi: 10.3389/fonc.2021.723162
journal_title: Frontiers in Oncology
journal_nlm_ta: Front Oncol
publisher_name: Frontiers Media S.A.
keywords: HNSCC | IRAK1 | survival | driver gene | LASSO | ROC | whole exome sequencing
automl_pathway: 0.5466822
figid_alias: PMC8593415__F8
figtype: Figure
redirect_from: /figures/PMC8593415__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8593415__fonc-11-723162-g008.html
  '@type': Dataset
  description: (A) A flowchart explaining the pipeline followed for prediction of
    novel deleterious variants from in-house data. (B) This diagram shows the classification
    of 114 novel deleterious variants into 11 Tumor suppressor genes, 35 oncogenes
    and 2 DNA damage and Repair genes. (C) This figure shows the distribution of the
    114 novel deleterious variants into 3 significant pathway interaction networks,
    namely Collagen Biosynthesis and Degradation, mTOR signaling and ECM Signaling.
    (D) A bar graph showing all the significantly mutated pathways by the number of
    genes mutated with the most significant pathway being receptor tyrosine kinase
    signaling.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BCR
  - IGKV3D-20
  - TTF2
  - FOXE1
  - TYMS
  - SMC2
  - RAB33B
  - MTOR
  - COL6A1
  - COL6A2
  - COL6A3
  - COL6A5
  - COL6A6
  - COL7A1
  - COL8A1
  - COL8A2
  - COL9A1
  - COL9A2
  - COL9A3
  - COL10A1
  - COL11A1
  - COL11A2
  - COL12A1
  - COL13A1
  - COL14A1
  - COL15A1
  - COL16A1
  - COL17A1
  - COL18A1
  - COL19A1
  - COL20A1
  - COL21A1
  - COL22A1
  - COL23A1
  - COL24A1
  - COL25A1
  - COL26A1
  - COL27A1
  - COL28A1
  - COL1A1
  - COL1A2
  - COL4A1
  - COL4A2
  - COL4A3
  - COL4A4
  - COL4A5
  - COL4A6
  - COL5A1
  - COL5A2
  - COL5A3
  - LMO1
  - RRAS2
  - AKT1
  - VLDLR
  - SMAD7
  - RBM20
  - LAMA2
  - MYH7
  - FBLN1
  - TRIM47
  - efna5b
  - ttf2
  - tyms
  - smc2
  - mtor
  - lmo1
  - rras2
  - akt1
  - vldlr
  - smad7
  - rbm20
  - lama2
  - myh7
  - myh7bb
  - si:ch211-24n20.3
  - smyhc2
  - fbln1
  - rab1ab
---
