---
figid: PMC8452524__gr1a_lrg
figtitle: 'Cytokine storm in the pathophysiology of COVID-19: Possible functional
  disturbances of miRNAs'
organisms:
- Severe acute respiratory syndrome coronavirus 2
- Homo sapiens
- Hepatitis B virus
- Coronaviridae
- Severe acute respiratory syndrome-related coronavirus
- Middle East respiratory syndrome-related coronavirus
- Mus musculus
- Rattus norvegicus
- Macaca mulatta
- Helicobacter pylori
- Camellia sinensis
- Sar
- teleost fish
- NA
organisms_ner:
- Rattus norvegicus
- Macaca mulatta
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC8452524
filename: gr1a_lrg.jpg
figlink: ''
number: F1a
caption: The most effective pro-inflammatory pathways on cytokine storm development
  in severe/critical COVID-19. a) Following the SARS-CoV-2 entry, its genome is recognized
  by endosomal TLRs and cytosolic sensors. Multiple adaptor proteins are then recruited.
  These lead to the nuclear translocation of IRF3, IRF7, and NF-κB transcription factors
  accompanied by the production of IFNs-I, pro-inflammatory mediators, and miRNAs.
  A group of proteins, encoded by the virus or host, impede the activation of IFNs-I
  generation pathways. Among them, viral nsp13 and 6 inhibit the TANK-binding kinase
  1 (TBK1) and IRF3, respectively. SARS-CoV-2 ORF6 and host PGE2 also prevent the
  nuclear translocation of IRF3 and IRF7. Moreover, viral nsp15 hinders the identification
  of this pathogen genome by cytosolic sensors, and its N protein inhibits them. Ang
  II/AT1R interaction underlies the activation of pro-inflammatory pathways, nuclear
  translocation of NF-κB as well as AP-1 transcription factors, and ultimately high-level
  production of pro-inflammatory mediators on the one hand, and causes activation
  of ADAM17 molecule, conversion of the mIL-6R into sIL-6R, and induction of this
  receptor trans-signaling process on the other. Several pro-inflammatory pathways
  are also activated downstream of IL-6R resulting in the excessive production of
  the pro-inflammatory mediators and IL-6 Amp cycle. The cPLA2 produces arachidonic
  acid (AA) and LPLs via decomposition of membrane phospholipids and the latter are
  predisposing factors for virus increased replication. The enzymes COX-I, -II, mPGES-1,
  lipoxygenases (LOXs), etc. participate in the generation pathways of eicosanoids
  and these lipid mediators intensify the pro-inflammatory reactions. It seems that
  Apelin/APJ axis undergoes some disruptions in this infection, in a way that it does
  not respectively induce the elevated and reduced expression of ACE2 and AT1R leading
  to the cytokine storm progression. b) Produced IFNs-I bind to the related receptors
  and launching their signaling pathways results in the nuclear translocation of STAT1/STAT2
  heterodimers, expression of ISGs and some miRNAs. However, several viral proteins
  including nsp1, 6, 13, ORF3a, 7a, and M inhibit STAT1 and 2 transcription factors
  and decrease the activity of this pivotal antiviral pathway. In addition, SARS-CoV-2
  ORF6 impedes the nuclear translocation of STAT1/STAT2 heterodimers and host PGE2
  hampers the ISGs. Activation of multiple pathways and molecules including activity
  of nicotinamide adenine dinucleotide phosphate (NADPH) oxidases (NOXs) in the endosome,
  the enzymes involved in the eicosanoids generation pathway, in the downstream of
  RAGE, NF-κB and AP-1 ones, and also as a consequence of mitochondrial membrane depolarization
  by elevated Ang II provides the basis for high-level production of ROS. This molecule
  provokes the activity of the NLRP3-Inflammasome complex and NF-κB pathway, both
  of which lead to the production of pro-inflammatory mediators and pyroptosis. The
  host PGE2 also stimulates the activity of the mentioned complex. The Ang II/AT1R
  interaction causes trans-activation of the cytoplasmic tail of RAGE and launching
  its downstream pathway on the one hand and activates ADAM17 on the other. This molecule
  underlies the accumulation of Ang II through ACE2 cleavage. Furthermore, ADAM17
  activity is followed by high-level procreation of MMP-9 and consequently more complications.
  Production of mature TNF-α is the other result of ADAM17 activation. Ligation of
  RAGE, TNFR and TLR4 with their correspondent ligands (e.g. advanced glycation end
  products (AGEs), TNF-α, and lipopolysaccharide (LPS)) also contributes to the generation
  of pro-inflammatory mediators. Nuclear translocation of IRF3 and gene transcription
  of IFNs-I occur in the MyD88-independent pathway of TLR4 signaling. Nevertheless,
  SARS-CoV-2 nsp13 and 6 respectively inhibit the TBK1 and IRF3. Its ORF6 and host
  PGE2 also impede the IRF3 nuclear translocation resulting in the limitation of IFNs-I
  production in this pathway.
papertitle: 'Cytokine storm in the pathophysiology of COVID-19: Possible functional
  disturbances of miRNAs.'
reftext: Mona Aslani, et al. Int Immunopharmacol. 2021 Dec;101:108172-108172.
year: '2021'
doi: 10.1016/j.intimp.2021.108172
journal_title: International Immunopharmacology
journal_nlm_ta: Int Immunopharmacol
publisher_name: Elsevier B.V.
keywords: SARS-CoV-2 | COVID-19 | miRNAs | Gene regulators | Cytokine storm | miRNAs-based
  therapy | 2019-nCoV, 2019 novel coronavirus | 3′-UTR, 3′-untranslated region | AA,
  Arachidonic acid | ACE, Angiotensin-converting enzyme | ADAM17, A disintegrin and
  metalloproteinase 17 | AGEs, Advanced glycation end products | AGO, Argonaute |
  ALI, Acute lung injury | Amp, Amplifier | Ang, Angiotensin | ANRIL, Antisense noncoding
  RNA in the INK4 locus | AP-1, Activator protein 1 | APJ, Apelin receptor | ARDS,
  Acute respiratory distress syndrome | ASOs, Antisense oligonucleotides | AT1R, Ang
  II receptor type 1 | BBB, Blood-brain barrier | CatB/L, Cathepsin B/L | CCL, C-C
  motif chemokine ligand | CD, Cluster of differentiation | ceRNAs, Competing endogenous
  RNAs | CHD, Coronary heart disease | circRNAs, Circular RNAs | CNS, Central nervous
  system | COVID-19, Coronavirus disease 2019 | COX, Cyclooxygenase | cPLA2, Cytoplasmic
  phospholipase A2 | CRISPR, Cas13 family of clustered regularly interspaced short
  palindromic repeats | CSF, Colony-stimulating factor | CXCL, C-X-C motif chemokine
  ligand | CYLD, Cylindromatosis | DAMPs, Damage-associated molecular patterns | DCs,
  Dendritic cells | DIC, Disseminated intravascular coagulation | DMD, Dense matted
  deposits | DMVs, Double membrane vesicles | E protein, Envelope protein | EAE, Experimental
  autoimmune encephalomyelitis | ER, Endoplasmic reticulum | ERK, Extracellular signal-regulated
  kinase | FOXO3, Forkhead box O3 | G-CSF, Granulocyte CSF | GM-CSF, Granulocyte-macrophage
  CSF | gp130, Glycoprotein 130 | H3, Histone 3 | HMGB1, High mobility group box protein
  1 | hsa, Homo sapiens | HSP, Heat shock protein | ICU, Intensive care unit | IFN,
  Interferon | IFNs-I, Type I IFNs | Ig, Immunoglobulin | IκB-ζ, Inhibitor of nuclear
  factor kappa B zeta | IL, Interleukin | IRF, IFN regulatory factor | ISGs, IFN-stimulated
  genes | JAK, Janus kinase | K, Lysine | KCNQ1OT1, KCNQ1 overlapping transcript 1
  | lncRNAs, Long non-coding RNAs | LOXs, Lipoxygenases | LPLs, Lysophospholipids
  | LPS, Lipopolysaccharide | LTs, Leukotrienes | M protein, Membrane protein | MAPK,
  Mitogen-activated protein kinase | MCs, Mast cells | M−CSF, Macrophage CSF | MDA5,
  Melanoma differentiation-associated protein 5 | me2, Dimethylation | me3, Trimethylation
  | MEG3, Maternally expressed gene 3 | MERS-CoV, Middle East respiratory syndrome
  coronavirus | mIL-6R, Membrane IL-6 receptor | miRISC, MicroRNA-induced silencing
  complex | miRNAs, MicroRNAs | MMP, Matrix metalloproteinase | MODS, Multiple organ
  dysfunction syndrome | mPGES-1, Microsomal prostaglandin E synthase-1 | mRNA, Messenger
  RNA | MyD88, Myeloid differentiation factor 88 | N protein, Nucleocapsid protein
  | NF-κB, Nuclear factor kappa-light-chain-enhancer of activated B cells | NLRP3,
  NOD-, LRR- and pyrin domain-containing protein 3 | NOXs, Nicotinamide adenine dinucleotide
  phosphate (NADPH) oxidases | nsp, Non-structural protein | ORF, Open reading frame
  | OxPLs, Oxidized phospholipids | PaO2/FiO2, Pressure of arterial oxygen to fractional
  inspired oxygen concentration | PBMCs, Peripheral blood mononuclear cells | PG,
  Prostaglandin | PKCα, Protein kinase C alpha | PPP2CA, Protein phosphatase 2 catalytic
  subunit alpha | pre-miRNAs, Precursor miRNAs | pri-miRNAs, Primary miRNAs | PRR,
  Pattern recognition receptor | RAGE, Receptor for advanced glycation end products
  | RAS, Renin-Angiotensin system | RBCs, Red blood cells | RGMB-AS1, RGMB antisense
  RNA 1 | RhoB, Ras homolog gene family member B | RIG-I, Retinoic acid-inducible
  gene I | RNAi, RNA interference | RNA pol, RNA polymerase | ROS, Reactive oxygen
  species | S100A9, S100 calcium-binding protein A9 | S protein, Spike protein | SARS-CoV-2,
  Severe acute respiratory syndrome coronavirus 2 | sIL-6R, Soluble IL-6R | siRNAs,
  Small interfering RNAs | SOCS3, Suppressor of cytokine signaling 3 | SOFA, Sequential
  organ failure assessment score | Sry, Sex-determining region Y | STAT, Signal transducer
  and activator of transcription | TAB, Transforming growth factor β-activated kinase-1
  (TAK1)-binding protein | TBK1, TANK-binding kinase 1 | Th cells, T helper cells
  | TIMP-I, Tissue inhibitors of matrix metalloproteinases-I | TLRs, Toll-like receptors
  | TMPRSS2, Transmembrane serine protease 2 | TNF-α, Tumor necrosis factor alpha
  | TNFR, TNF receptor | TRIM27, Tripartite motif-containing protein 27 | TXs, Thromboxanes
  | v-miRNA, Viral miRNA | XPO5, Exportin-5
automl_pathway: 0.9149827
figid_alias: PMC8452524__F1a
figtype: Figure
redirect_from: /figures/PMC8452524__F1a
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8452524__gr1a_lrg.html
  '@type': Dataset
  description: The most effective pro-inflammatory pathways on cytokine storm development
    in severe/critical COVID-19. a) Following the SARS-CoV-2 entry, its genome is
    recognized by endosomal TLRs and cytosolic sensors. Multiple adaptor proteins
    are then recruited. These lead to the nuclear translocation of IRF3, IRF7, and
    NF-κB transcription factors accompanied by the production of IFNs-I, pro-inflammatory
    mediators, and miRNAs. A group of proteins, encoded by the virus or host, impede
    the activation of IFNs-I generation pathways. Among them, viral nsp13 and 6 inhibit
    the TANK-binding kinase 1 (TBK1) and IRF3, respectively. SARS-CoV-2 ORF6 and host
    PGE2 also prevent the nuclear translocation of IRF3 and IRF7. Moreover, viral
    nsp15 hinders the identification of this pathogen genome by cytosolic sensors,
    and its N protein inhibits them. Ang II/AT1R interaction underlies the activation
    of pro-inflammatory pathways, nuclear translocation of NF-κB as well as AP-1 transcription
    factors, and ultimately high-level production of pro-inflammatory mediators on
    the one hand, and causes activation of ADAM17 molecule, conversion of the mIL-6R
    into sIL-6R, and induction of this receptor trans-signaling process on the other.
    Several pro-inflammatory pathways are also activated downstream of IL-6R resulting
    in the excessive production of the pro-inflammatory mediators and IL-6 Amp cycle.
    The cPLA2 produces arachidonic acid (AA) and LPLs via decomposition of membrane
    phospholipids and the latter are predisposing factors for virus increased replication.
    The enzymes COX-I, -II, mPGES-1, lipoxygenases (LOXs), etc. participate in the
    generation pathways of eicosanoids and these lipid mediators intensify the pro-inflammatory
    reactions. It seems that Apelin/APJ axis undergoes some disruptions in this infection,
    in a way that it does not respectively induce the elevated and reduced expression
    of ACE2 and AT1R leading to the cytokine storm progression. b) Produced IFNs-I
    bind to the related receptors and launching their signaling pathways results in
    the nuclear translocation of STAT1/STAT2 heterodimers, expression of ISGs and
    some miRNAs. However, several viral proteins including nsp1, 6, 13, ORF3a, 7a,
    and M inhibit STAT1 and 2 transcription factors and decrease the activity of this
    pivotal antiviral pathway. In addition, SARS-CoV-2 ORF6 impedes the nuclear translocation
    of STAT1/STAT2 heterodimers and host PGE2 hampers the ISGs. Activation of multiple
    pathways and molecules including activity of nicotinamide adenine dinucleotide
    phosphate (NADPH) oxidases (NOXs) in the endosome, the enzymes involved in the
    eicosanoids generation pathway, in the downstream of RAGE, NF-κB and AP-1 ones,
    and also as a consequence of mitochondrial membrane depolarization by elevated
    Ang II provides the basis for high-level production of ROS. This molecule provokes
    the activity of the NLRP3-Inflammasome complex and NF-κB pathway, both of which
    lead to the production of pro-inflammatory mediators and pyroptosis. The host
    PGE2 also stimulates the activity of the mentioned complex. The Ang II/AT1R interaction
    causes trans-activation of the cytoplasmic tail of RAGE and launching its downstream
    pathway on the one hand and activates ADAM17 on the other. This molecule underlies
    the accumulation of Ang II through ACE2 cleavage. Furthermore, ADAM17 activity
    is followed by high-level procreation of MMP-9 and consequently more complications.
    Production of mature TNF-α is the other result of ADAM17 activation. Ligation
    of RAGE, TNFR and TLR4 with their correspondent ligands (e.g. advanced glycation
    end products (AGEs), TNF-α, and lipopolysaccharide (LPS)) also contributes to
    the generation of pro-inflammatory mediators. Nuclear translocation of IRF3 and
    gene transcription of IFNs-I occur in the MyD88-independent pathway of TLR4 signaling.
    Nevertheless, SARS-CoV-2 nsp13 and 6 respectively inhibit the TBK1 and IRF3. Its
    ORF6 and host PGE2 also impede the IRF3 nuclear translocation resulting in the
    limitation of IFNs-I production in this pathway.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ace2
  - Tbk1
  - Rnf138
  - Traf3
  - Irf3
  - Tlr3
  - Tlr7
  - Tmprss2
  - Furin
  - Tlr8
  - Myd88
  - Irak4
  - Irak1
  - Irak2
  - Traf6
  - Chuk
  - Irf7
  - Rig1
  - Mavs
  - Nfkb1
  - Angpt1
  - Ang
  - Agt
  - Ifnb1
  - Mapk8
  - Prkcg
  - Mapk14
  - Nox4
  - Akt1
  - Stat3
  - Mapk3
  - Jun
  - Adam17
  - Il6
  - Nlrp3
  - Ptgs2
  - Nos2
  - Socs3
  - Lox
  - Pla2g4a
  - Tmprss5
  - Ccne1
  - Tnf
  - ACE2
  - TBK1
  - TRAF3
  - IRF3
  - TLR3
  - TLR7
  - TMPRSS2
  - FURIN
  - TLR8
  - MYD88
  - IRAK4
  - IRAK1
  - IRAK2
  - TRAF6
  - CHUK
  - IRF7
  - MAVS
  - NOX4
  - STAT3
  - JUN
  - ADAM17
  - IL6ST
  - NLRP3
  - COX2
  - NOS2
  - SOCS3
  - LOX
  - TNF
  - Znhit2
  - Trim69
  - Ticam1
  - Ticam2
  - Ikbke
  - Robo3
  - Ang3
  - Tm7sf2
  - Ifna
  - Nfkbib
  - Crk
  - Grap2
  - Syp
  - Ahsa1
  - COX1
  - H2-Aa
  - Ptgir
  - Kat5
  - TRIM69
  - TICAM1
  - IKBKE
  - PLAAT4
  - RIGI
  - ROBO3
  - NFKB1
  - ANGPT1
  - TM7SF2
  - ANGPT2
  - VPS51
  - IFNA1
  - IFNB1
  - MAPK8
  - MAPK9
  - MAPK10
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - CRK
  - MAPK14
  - MAPK1
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAPK11
  - MAPK12
  - MAPK13
  - AKT1
  - AKT2
  - AKT3
  - MAPK3
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - FOS
  - FOSB
  - JUNB
  - JUND
  - RPS19BP1
  - IL6
  - PTGS2
  - ISYNA1
  - PLA2G4A
  - KAT5
  - APRT
  - MFAP1
  - MMP1
  - MMP2
  - MMP3
  - MMP7
  - MMP8
  - MMP9
  - MMP10
  - MMP11
  - MMP12
  - MMP13
  - MMP14
  - MMP15
  - MMP16
  - MMP17
  - MMP19
  - MMP20
  - MMP21
  - MMP23B
  - MMP24
  - MMP25
  - MMP26
  - MMP27
  - MMP28
  - ace2
  - tbk1
  - ticam1
  - traf3
  - irf3
  - tlr3
  - tlr7
  - tmprss2
  - furina
  - myd88
  - irak4
  - irak1
  - traf6
  - irf7
  - mavs
  - angpt1
  - angpt2b
  - mapk8b
  - mapk14a
  - nox4
  - stat3
  - mapk3
  - jun
  - adam17b
  - il6
  - nlrp3
  - socs3a
  - gpib
  - loxa
  - pla2g4aa
  - tnfb
---
