---
figid: PMC8122967__cancers-13-02042-g003
figtitle: PARP Enzyme Family and the Hallmarks of Cancer Part 1. Cell Intrinsic Hallmarks
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC8122967
filename: cancers-13-02042-g003.jpg
figlink: /pmc/articles/PMC8122967/figure/cancers-13-02042-f003/
number: F3
caption: The involvement of PARPs in metabolic reprogramming in cancer. The rewired
  metabolism in cancer is established by oncogenic signaling and transcriptional programs
  switched on in response to changes in the internal milieu of cancer cells and the
  tumor microenvironment. PARPs exert a multifaceted influence on metabolism either
  through stimulating these transcriptional programs or via direct PAR/MARylation
  of metabolic proteins. By enhancing HIF1/2 target gene expression through HIF stabilization
  and coactivation, PARP-1 and PARP-2 increase glycolytic flux at the levels of glucose
  entry into the cell and into the glycolytic pathway and at the level of pyruvate
  removal by LDHA. PARP-1 induces c-MYC and E2F expressions and is a coactivator of
  NRF2. NRF2, c-MYC, and E2F responsive genes include enzymes in the pentose phosphate,
  serine/glycine, and C1-metabolic pathways, which channel away glycolytic intermediates
  for the synthesis of nucleotides, nucleic acids, and NADPH that are required for
  biomass production. Inhibition of GSK3β and GAPDH by PARP10 and of PKM2 by PARP14
  also divert intermediates to anabolic pathways branching off from glycolysis. The
  uptake of metabolic precursors, such as glucose, glutamine, and fatty acids, are
  stimulated by PARP1/2-enhanced HIF1/2, c-Myc, and E2F-dependent expressions of their
  respective transporters. NRF2 enhances glutamine metabolism, glutathione synthesis
  and reduction, and NADPH-producing enzyme expression (ME1, IDH2) for elevated oxidative
  stress tolerance. This leads to anaplerotic recircuiting of the TCA cycle. A major
  factor in the downregulation of mitochondrial function and oxidative phosphorylation
  is the inactivation of the pyruvate dehydrogenase complex by PDK1, which is also
  a target gene of HIF1/2. The electron transport chain complexes may be inhibited
  by PARylation, although the identity(ies) of the mitochondrial PARP(s) is/are uncertain.
  PARP1, PARP2, and PARP10 expressions may also negatively influence mitochondrial
  biogenesis and performance through SIRT1 and PGC1α. The PI3K/Akt-pathway is a major
  stimulator of cell survival and growth through glucose transporter expression, protein
  synthesis, and the protection of mitochondria. Under pronounced genotoxic stress,
  the consumption of cytoplasmic NAD+ by activated PARP-1 antagonizes the pathway
  due to the inhibition of NAD+-dependent deacetylation of Akt by SIRT1. Additionally,
  Rictor, the regulatory subunit of mTORC2, is an activator of Akt. Rictor is inhibited
  by noncovalently bound PAR. The inhibition of the PI3K-Akt-mTOR axis, the activation
  of AMPK, and the activation of the HIF transcriptional response are all mechanisms
  through which PARP-1 supports nutrient acquisition by autophagy, another feature
  of tumor metabolism. Functional p53 in cancers exerts a negative regulatory effect
  on the expression of several GFR–PI3K–Akt–mTOR pathway components, glycolytic flux
  determining enzymes, and PDK1, which may be derepressed by PARP-1. Fatty acids produced
  in cancer cells by upregulated FASN expression induce PARP-1 gene expression through
  the downregulation of NFκB and the upregulation of Sp1, buttressing the tumor cell’s
  tolerance of DNA damage.
papertitle: The PARP Enzyme Family and the Hallmarks of Cancer Part 1. Cell Intrinsic
  Hallmarks.
reftext: Máté A. Demény, et al. Cancers (Basel). 2021 May;13(9):2042.
year: '2021'
doi: 10.3390/cancers13092042
journal_title: Cancers
journal_nlm_ta: Cancers (Basel)
publisher_name: MDPI
keywords: hallmarks of cancer | poly (ADP-ribose) polymerase | oncogenes | tumor suppressors
  | cell death | replicative immortality | metabolic reprogramming
automl_pathway: 0.8763292
figid_alias: PMC8122967__F3
figtype: Figure
redirect_from: /figures/PMC8122967__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8122967__cancers-13-02042-g003.html
  '@type': Dataset
  description: The involvement of PARPs in metabolic reprogramming in cancer. The
    rewired metabolism in cancer is established by oncogenic signaling and transcriptional
    programs switched on in response to changes in the internal milieu of cancer cells
    and the tumor microenvironment. PARPs exert a multifaceted influence on metabolism
    either through stimulating these transcriptional programs or via direct PAR/MARylation
    of metabolic proteins. By enhancing HIF1/2 target gene expression through HIF
    stabilization and coactivation, PARP-1 and PARP-2 increase glycolytic flux at
    the levels of glucose entry into the cell and into the glycolytic pathway and
    at the level of pyruvate removal by LDHA. PARP-1 induces c-MYC and E2F expressions
    and is a coactivator of NRF2. NRF2, c-MYC, and E2F responsive genes include enzymes
    in the pentose phosphate, serine/glycine, and C1-metabolic pathways, which channel
    away glycolytic intermediates for the synthesis of nucleotides, nucleic acids,
    and NADPH that are required for biomass production. Inhibition of GSK3β and GAPDH
    by PARP10 and of PKM2 by PARP14 also divert intermediates to anabolic pathways
    branching off from glycolysis. The uptake of metabolic precursors, such as glucose,
    glutamine, and fatty acids, are stimulated by PARP1/2-enhanced HIF1/2, c-Myc,
    and E2F-dependent expressions of their respective transporters. NRF2 enhances
    glutamine metabolism, glutathione synthesis and reduction, and NADPH-producing
    enzyme expression (ME1, IDH2) for elevated oxidative stress tolerance. This leads
    to anaplerotic recircuiting of the TCA cycle. A major factor in the downregulation
    of mitochondrial function and oxidative phosphorylation is the inactivation of
    the pyruvate dehydrogenase complex by PDK1, which is also a target gene of HIF1/2.
    The electron transport chain complexes may be inhibited by PARylation, although
    the identity(ies) of the mitochondrial PARP(s) is/are uncertain. PARP1, PARP2,
    and PARP10 expressions may also negatively influence mitochondrial biogenesis
    and performance through SIRT1 and PGC1α. The PI3K/Akt-pathway is a major stimulator
    of cell survival and growth through glucose transporter expression, protein synthesis,
    and the protection of mitochondria. Under pronounced genotoxic stress, the consumption
    of cytoplasmic NAD+ by activated PARP-1 antagonizes the pathway due to the inhibition
    of NAD+-dependent deacetylation of Akt by SIRT1. Additionally, Rictor, the regulatory
    subunit of mTORC2, is an activator of Akt. Rictor is inhibited by noncovalently
    bound PAR. The inhibition of the PI3K-Akt-mTOR axis, the activation of AMPK, and
    the activation of the HIF transcriptional response are all mechanisms through
    which PARP-1 supports nutrient acquisition by autophagy, another feature of tumor
    metabolism. Functional p53 in cancers exerts a negative regulatory effect on the
    expression of several GFR–PI3K–Akt–mTOR pathway components, glycolytic flux determining
    enzymes, and PDK1, which may be derepressed by PARP-1. Fatty acids produced in
    cancer cells by upregulated FASN expression induce PARP-1 gene expression through
    the downregulation of NFκB and the upregulation of Sp1, buttressing the tumor
    cell’s tolerance of DNA damage.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SLC2A4
  - FASN
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - SLC2A1
  - TNKS
  - RELA
  - SYT1
  - GORASP1
  - WNK1
  - TP53
  - TP63
  - TP73
  - CD40
  - ARHGEF7
  - ASCC1
  - H3P40
  - PARG
  - PTEN
  - SIRT1
  - TRIM31
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - PARP1
  - PRKAA1
  - PRKAA2
  - PARP2
  - TNKS2
  - PARP10
  - HIF1A
  - EPAS1
  - HIF3A
  - ARNT
  - ARNT2
  - BMAL1
  - CRABP1
  - CRABP2
  - FABP1
  - FABP12
  - FABP2
  - FABP3
  - FABP4
  - FABP5
  - FABP6
  - FABP7
  - FABP9
  - PMP2
  - RBP1
  - RBP2
  - RBP5
  - RBP7
  - PLIN2
  - PDK1
  - PDPK1
  - AKT1
  - AKT2
  - AKT3
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - SLC1A5
  - E2F1
  - E2F2
  - E2F3
  - E2F4
  - E2F5
  - E2F6
  - E2F7
  - E2F8
  - PGD
  - PHGDH
  - G6PD
  - UBL4A
  - IDH1
  - MAFG
  - ADHFE1
  - ME1
  - PCDHB16
  - TSC2
  - TSC1
  - CCL26
  - PKM
  - PKLR
  - STAC3
  - RPS6KB1
  - PARP14
  - APRT
  - MFAP1
  - NR1I2
  - JTB
  - HK1
  - KCNA4
  - KLK1
  - HOOK1
  - TAC4
  - WDTC1
  - RICTOR
  - RHEB
  - RHEBP1
  - MTOR
  - VHL
  - ULK1
  - GABPA
  - NFE2L2
  - MAPK8
  - MAPK9
  - MAPK10
  - DECR1
  - ATP8A2
  - COL11A2
  - PARP3
  - PARP4
  - PARP6
  - PARP8
  - PARP9
  - PARP11
  - PARP12
  - PARP15
  - PARP16
  - TIPARP
  - SLC25A4
  - SLC25A6
  - VDAC1
  - VDAC2
  - VDAC3
  - SETD2
  - HK2
  - KCNA5
  - KIF2A
  - KLK2
  - HOOK2
  - IL1A
  - IL1B
  - PFKM
  - LDHA
  - GAPDH
  - GAPDHP44
  - IRF1
  - IDH2
  - ACLY
  - GLS
  - GLS2
  - ACACB
  - BCL2A1
  - MYC
  - SLC16A1
  - MCAT
  - MCPH1
  - SLC38A5
---
