---
figid: PMC8661833__439_2021_2397_Fig2_HTML
figtitle: Common, low-frequency, rare, and ultra-rare coding variants contribute to
  COVID-19 severity
organisms:
- Severe acute respiratory syndrome coronavirus 2
- Coronaviridae
- Enterovirus A71
- Homo sapiens
- Mus musculus
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8661833
filename: 439_2021_2397_Fig2_HTML.jpg
figlink: /pmc/articles/PMC8661833/figure/Fig2/
number: F2
caption: 'Biological impact of ultra-rare, rare, low-frequency, and common features.
  Examples of ultra-rare (A), rare (B), low-frequency (C), and common (D) features
  are illustrated in panel A–D. The complete list of features is presented in Supplementary
  Tables 3–6.  = contributing to COVID-19 severity;  = contributing to COVID-19 mildness.
  Pink faces = contributing to females only; blue faces = contributing to males only;
  pink/blue faces = contribution in both sexes. In parentheses: AD = autosomal dominant
  inheritance; AR = autosomal recessive inheritance; XL = X-linked recessive inheritance.
  A Ultra-rare mutations in the RNA sensor TLR7, TLR3, and TICAM1 (encoding TRIF protein),
  already reported associated with XL, AR and AD inheritance (Zhang et al. ; Van der
  Made et al. ; Fallerini et al. ; Solanich et al. ) impair interferon (IFNs) production
  in innate immune system cells. Mutations in TLR8, as well as of the signal transducer
  IRAK1 also impair interferon production. The specific location of TLR7/8 and IRAK1
  (on the X chromosome) as well as X-inactivation escaping are responsible for opposite
  effects in males and females. Mutation in RNASEL impair the antiviral effect of
  the gene. In lung epithelial cells, ACE2 ultra-rare variants (on the X chromosome)
  exert protective effects (probably) due to lowering virus entrance, while ultra-rare
  variants in ADAM17 (might) reduce the shedding of ACE2 and induce a severe outcome.
  The same is true for CFTR and SCNN1A (encoding ENaCA protein and involved in a CFTR-related
  physiological pathway), and the lipid transporter ABCA3 (Baldassarri et al. ).Mutations
  of ADAMTS13 in vessels reduce the cleavage of the multimeric von Willebrand Factor
  (VWF), leading to thrombosis; B) Rare variants of the estrogen regulated TLR5 are
  associated with severity in females. Rare variants of the CFTR-related SLC26A9 are
  associated with severity in both sexes. This ion transporter has three discrete
  physiological modes: nCl(–)-HCO(3)(–) exchanger, Cl(–) channel, and Na(+)-anion
  cotransporter. Other examples of rare mutations associated with severity are the
  NK and T cell receptor FCRL6, IFN signal transducer IRAK2, and the actin depolymerization
  MICAL2; C low-frequency variants in another CFTR-related gene, SCNN1D (encoding
  for ENaCD protein) are associated with mildness, while rare variants in the following
  genes are associated with severity: cargo protein SPMA6, vesicle formation PEX1,
  inflammatory protein NOD2 (CARD15); D A number of coding polymorphisms, indicated
  with an asterisk, are in LD with genomic SNPs already associated with COVID-19 (The
  complete list is presented in Supplementary Table 11) (Severe Covid-19 GWAS Group
  (; Pairo-Castineira et al. ). In some cases, such as the case of SFTDP, the genomic
  SNP is the coding polymorphism itself. Of note are the genes of surfactant proteins
  associated with severe disease: SFTDP gene encoding for SP-D protein and SFTPA1
  gene encoding for SP-A protein; the signal transducer, PPP1R15A gene encoding for
  GADD34 protein. OAS1 and OAS3 related to RNA clearance of RNASEL (reported in panel
  A as having ultra-rare mutations; included here should also be the already reported
  TLR3412 (Croci et al. ); the already reported SELP603 related to thrombosis (Fallerini
  et al. ). Note: OAS1 haplotype A = c.1039-1G>A (Wickenhagen et al. ), (p.(Gly162Ser)),
  (p.(Ala352Thr)), (p.(Arg361Thr)), (p.(Gly397Arg)), (p.(Thr358Profs*26)). OAS1 haplotype
  B = haplotype without the variant combination in haplotype A'
papertitle: Common, low-frequency, rare, and ultra-rare coding variants contribute
  to COVID-19 severity.
reftext: Chiara Fallerini, et al. Hum Genet. 2022;141(1):147-173.
year: '2022'
doi: 10.1007/s00439-021-02397-7
journal_title: Human Genetics
journal_nlm_ta: Hum Genet
publisher_name: Springer Berlin Heidelberg
keywords: ''
automl_pathway: 0.9496157
figid_alias: PMC8661833__F2
figtype: Figure
redirect_from: /figures/PMC8661833__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8661833__439_2021_2397_Fig2_HTML.html
  '@type': Dataset
  description: 'Biological impact of ultra-rare, rare, low-frequency, and common features.
    Examples of ultra-rare (A), rare (B), low-frequency (C), and common (D) features
    are illustrated in panel A–D. The complete list of features is presented in Supplementary
    Tables 3–6.  = contributing to COVID-19 severity;  = contributing to COVID-19
    mildness. Pink faces = contributing to females only; blue faces = contributing
    to males only; pink/blue faces = contribution in both sexes. In parentheses: AD = autosomal
    dominant inheritance; AR = autosomal recessive inheritance; XL = X-linked recessive
    inheritance. A Ultra-rare mutations in the RNA sensor TLR7, TLR3, and TICAM1 (encoding
    TRIF protein), already reported associated with XL, AR and AD inheritance (Zhang
    et al. ; Van der Made et al. ; Fallerini et al. ; Solanich et al. ) impair interferon
    (IFNs) production in innate immune system cells. Mutations in TLR8, as well as
    of the signal transducer IRAK1 also impair interferon production. The specific
    location of TLR7/8 and IRAK1 (on the X chromosome) as well as X-inactivation escaping
    are responsible for opposite effects in males and females. Mutation in RNASEL
    impair the antiviral effect of the gene. In lung epithelial cells, ACE2 ultra-rare
    variants (on the X chromosome) exert protective effects (probably) due to lowering
    virus entrance, while ultra-rare variants in ADAM17 (might) reduce the shedding
    of ACE2 and induce a severe outcome. The same is true for CFTR and SCNN1A (encoding
    ENaCA protein and involved in a CFTR-related physiological pathway), and the lipid
    transporter ABCA3 (Baldassarri et al. ).Mutations of ADAMTS13 in vessels reduce
    the cleavage of the multimeric von Willebrand Factor (VWF), leading to thrombosis;
    B) Rare variants of the estrogen regulated TLR5 are associated with severity in
    females. Rare variants of the CFTR-related SLC26A9 are associated with severity
    in both sexes. This ion transporter has three discrete physiological modes: nCl(–)-HCO(3)(–)
    exchanger, Cl(–) channel, and Na(+)-anion cotransporter. Other examples of rare
    mutations associated with severity are the NK and T cell receptor FCRL6, IFN signal
    transducer IRAK2, and the actin depolymerization MICAL2; C low-frequency variants
    in another CFTR-related gene, SCNN1D (encoding for ENaCD protein) are associated
    with mildness, while rare variants in the following genes are associated with
    severity: cargo protein SPMA6, vesicle formation PEX1, inflammatory protein NOD2
    (CARD15); D A number of coding polymorphisms, indicated with an asterisk, are
    in LD with genomic SNPs already associated with COVID-19 (The complete list is
    presented in Supplementary Table 11) (Severe Covid-19 GWAS Group (; Pairo-Castineira
    et al. ). In some cases, such as the case of SFTDP, the genomic SNP is the coding
    polymorphism itself. Of note are the genes of surfactant proteins associated with
    severe disease: SFTDP gene encoding for SP-D protein and SFTPA1 gene encoding
    for SP-A protein; the signal transducer, PPP1R15A gene encoding for GADD34 protein.
    OAS1 and OAS3 related to RNA clearance of RNASEL (reported in panel A as having
    ultra-rare mutations; included here should also be the already reported TLR3412
    (Croci et al. ); the already reported SELP603 related to thrombosis (Fallerini
    et al. ). Note: OAS1 haplotype A = c.1039-1G>A (Wickenhagen et al. ), (p.(Gly162Ser)),
    (p.(Ala352Thr)), (p.(Arg361Thr)), (p.(Gly397Arg)), (p.(Thr358Profs*26)). OAS1
    haplotype B = haplotype without the variant combination in haplotype A'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ar
  - am
  - Calm3
  - H2-T18
  - Impact
  - Ace
  - sar
  - Srl
  - Ad
  - Mical1
  - Bcar1
  - Cse1l
  - App
  - H2-Ab1
  - ac
  - Igfals
  - Gla
  - Gnai2
  - Gpc4
  - Tlr3
  - Selp
  - Selenop
  - AR
  - FOLH1
  - IMPACT
  - ACE
  - SARDH
  - SRL
  - NR1I2
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - MICAL1
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - SUCLA2
  - SLC29A1
  - IGFALS
  - SOD1
  - TP63
  - IS1
  - GLA
  - NAT8
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GPC4
  - RPL32
  - RAX2
  - TLR3
  - SELP
  - SELENOP
  - ar
  - ss
  - ale
  - Acer
  - Ance
  - ad
  - hop
  - bsk
  - l(1)TK412
  - Mical
  - cas
  - caz
  - Cse1
  - cass
  - cash
  - ab
  - Appl
  - Tehao
  - Toll-9
  - als
  - conv
  - nAChRalpha1
  - ais
  - wg
  - gl
  - Galphai
  - mthl5
  - RpL32
  - mRpL32
  - l(1)14Ai
  - l(3)65CEc
  - l(2)46Df
  - anon-48Aj
  - anon-XII
---
