---
figid: PMC8401276__microorganisms-09-01557-g004
figtitle: Genome-Wide Analysis of Nutrient Signaling Pathways Conserved in Arbuscular
  Mycorrhizal Fungi
organisms:
- Saccharomyces cerevisiae
- Oryza sativa
- Zea mays
- Solanum lycopersicum
- Candida albicans
- Schizosaccharomyces pombe
- Medicago truncatula
- Helianthus annuus
- Pyricularia oryzae
- Fusarium graminearum
- Ustilago maydis
- Tracheophyta
- Petunia x hybrida
- Pyricularia grisea
- Citrus reticulata
- Alternaria alternata
- Rhizopus arrhizus
- Trifolium repens
- Cryptococcus neoformans var. grubii
- Mucor circinelloides
- Zymoseptoria tritici
- Fusarium graminearum PH-1
- Serendipita indica
- Aspergillus fumigatus Af293
- Laccaria bicolor
- Funneliformis mosseae
- Rhizophagus intraradices
- Colletotrichum higginsianum
- Rhizophagus irregularis
- Cryptococcus neoformans var. neoformans JEC21
- Sporisorium scitamineum
- Gigaspora margarita
- Neurospora crassa OR74A
- Tuber melanosporum
- Astragalus sinicus
- Gigaspora rosea
- Hebeloma cylindrosporum
- Glomus sp.
- Ustilago maydis 521
- Diversispora versiformis
- Rhizophagus irregularis DAOM 181602DAOM 197198
- Tulasnella calospora MUT 4182
- Serendipita indica DSM 11827
- Mucor lusitanicus
- Zymoseptoria tritici IPO323
- Laccaria bicolor S238N-H82
- Verticillium dahliae VdLs.17
- Geosiphon pyriformis
- Diversispora epigaea
- Rhizopus delemar RA 99-880
- Trebouxia jamesii
- Tuber melanosporum Mel28
- Glomus cerebriforme
- Hebeloma cylindrosporum h7
- Colletotrichum higginsianum IMI 349063
- Pisolithus tinctorius Marx 270
- Rhizophagus clarus
- Escherichia coli
- Rhodothermus clarus
- Homo sapiens
- NA
organisms_ner:
- Schizosaccharomyces pombe
- Zymoseptoria tritici IPO323
- Homo sapiens
- Saccharomyces cerevisiae S288C
pmcid: PMC8401276
filename: microorganisms-09-01557-g004.jpg
figlink: /pmc/articles/PMC8401276/figure/microorganisms-09-01557-f004/
number: F4
caption: TOR signaling pathways in yeast (A) and AM fungi (B). The amino acids are
  sensed by Ssy1 and transported by the amino acid permeases (AAPs), while the ammonium
  is sensed and transported by AMTs/Meps. The increase in amino acids levels also
  activates TORC1 via the EGO complex, which is composed of Gtr1, Gtr2 and other unknown
  proteins. TORC1 is localized to the vacuolar membrane, consisting of Tor2, Kog1
  and Lst8. The activated TORC1 phosphorylates the downstream targets Sch9 and Tap42.
  Tap42 associates with phosphatase PP2A (Pph22, Tpd3 and Cdc55/Rts1) and PP2A-related
  protein phosphatases (Sit4/Ppg1 and Sap190) complexes, along with the regulatory
  protein Rrd1/2. TORC1 enhances the hyperphosphorylation and association of Gln3/Gat1
  and Ure2, preventing the nuclear localization of Gln3/Gat1 via the Tap42-PP2A complex,
  resulting in the suppression of genes related to nitrogen catabolite repression
  (NCR) and use of less preferred nitrogen sources. Tip41 interacts with Tap42 to
  negatively regulate the TOR pathway. Rapamycin binds to its receptor FKBP12 encoded
  by FPR1 to form a complex, which can bind Tor2, thereby inhibiting TORC1. TORC1
  promotes Sch9 activity that can phosphorylate Maf1 to activate RNA polymerase III-transcribed
  genes. Tap42 effector positively regulated by TORC1 prevents the translation of
  Gcn4, a transcriptional activator needed for amino acid biosynthesis. Tor2 also
  can bind to Avo1, Sit4 and Bit61 to form the TORC2 complex. The proteins Tco89,
  Ego1 and Ego3 are not found in the genomes of AM fungi. The red lines with the arrows
  indicate facilitation, whereas the blue lines with the dots indicate inhibition
  and the black arrows represent translocation or transformation.
papertitle: Genome-Wide Analysis of Nutrient Signaling Pathways Conserved in Arbuscular
  Mycorrhizal Fungi.
reftext: Xiaoqin Zhou, et al. Microorganisms. 2021 Aug;9(8):1557.
year: '2021'
doi: 10.3390/microorganisms9081557
journal_title: Microorganisms
journal_nlm_ta: Microorganisms
publisher_name: MDPI
keywords: arbuscular mycorrhizal fungi | nutrient signaling pathways | cAMP/PKA |
  TOR | SNF1 | PHO | appressorium formation | arbuscule longevity
automl_pathway: 0.9349574
figid_alias: PMC8401276__F4
figtype: Figure
redirect_from: /figures/PMC8401276__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8401276__microorganisms-09-01557-g004.html
  '@type': Dataset
  description: TOR signaling pathways in yeast (A) and AM fungi (B). The amino acids
    are sensed by Ssy1 and transported by the amino acid permeases (AAPs), while the
    ammonium is sensed and transported by AMTs/Meps. The increase in amino acids levels
    also activates TORC1 via the EGO complex, which is composed of Gtr1, Gtr2 and
    other unknown proteins. TORC1 is localized to the vacuolar membrane, consisting
    of Tor2, Kog1 and Lst8. The activated TORC1 phosphorylates the downstream targets
    Sch9 and Tap42. Tap42 associates with phosphatase PP2A (Pph22, Tpd3 and Cdc55/Rts1)
    and PP2A-related protein phosphatases (Sit4/Ppg1 and Sap190) complexes, along
    with the regulatory protein Rrd1/2. TORC1 enhances the hyperphosphorylation and
    association of Gln3/Gat1 and Ure2, preventing the nuclear localization of Gln3/Gat1
    via the Tap42-PP2A complex, resulting in the suppression of genes related to nitrogen
    catabolite repression (NCR) and use of less preferred nitrogen sources. Tip41
    interacts with Tap42 to negatively regulate the TOR pathway. Rapamycin binds to
    its receptor FKBP12 encoded by FPR1 to form a complex, which can bind Tor2, thereby
    inhibiting TORC1. TORC1 promotes Sch9 activity that can phosphorylate Maf1 to
    activate RNA polymerase III-transcribed genes. Tap42 effector positively regulated
    by TORC1 prevents the translation of Gcn4, a transcriptional activator needed
    for amino acid biosynthesis. Tor2 also can bind to Avo1, Sit4 and Bit61 to form
    the TORC2 complex. The proteins Tco89, Ego1 and Ego3 are not found in the genomes
    of AM fungi. The red lines with the arrows indicate facilitation, whereas the
    blue lines with the dots indicate inhibition and the black arrows represent translocation
    or transformation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ure2
  - gtr1
  - gtr2
  - prp10
  - tip41
  - tor1
  - tor2
  - tco89
  - maf1
  - bit61
  - fhl1
  - MEP
  - SIT4
  - TOR1
  - TOR2
  - CTSL
  - NLN
  - NPEPPS
  - CAV1
  - PPP4C
  - SLC6A1
  - ABL2
  - RERE
  - RRAGC
  - SF3B1
  - NM
  - ERVW-4
  - TIPRL
  - RBBP8
  - RIMS1
  - RPTOR
  - MLST8
  - MAF1
  - FKBP1A
  - FKBP1AP1
  - FKBP1AP2
  - FKBP1AP3
  - FKBP1AP4
  - CRTC2
  - RICTOR
  - CRHR1
  - FHL1
  - CFH
  - CFHR1
  - HPLH1
  - RTS1
  - CDC55
  - TPD3
  - PPH3
  - PPH22
  - PPH21
  - URE2
  - GAT1
  - GPT2
  - GLN3
  - SSY1
  - GTR1
  - PPG1
  - GTR2
  - MEH1
  - SAP4
  - SAP155
  - SLM4
  - MSN2
  - GIS1
  - TIP41
  - TAP42
  - RRD2
  - RRD1
  - RIM15
  - KOG1
  - LST8
  - TCO89
  - SCH9
  - TSC11
  - AVO1
  - AVO2
  - BIT61
  - CRF1
  - YNR048W
---
