---
figid: PMC8568984__41598_2021_608_Fig3_HTML
figtitle: Cross-platform transcriptomic profiling of the response to recombinant human
  erythropoietin
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC8568984
filename: 41598_2021_608_Fig3_HTML.jpg
figlink: /pmc/articles/PMC8568984/figure/Fig3/
number: F3
caption: 'Biological network of the MGI RNA-seq dataset following Gene Ontology (biological
  process) gene set enrichment analysis in GSEA (v4.0.3) and visualisation in Cytoscape
  (3.8.0). Each circle (node) represents a gene set and two nodes are connected by
  lines (edges) indicating shared genes. The size of a node and width of an edge are
  proportional to the number of genes enriched in a gene set and the number of genes
  shared between gene sets, respectively. Gene sets that are similar were annotated
  and clustered to form a biological theme using the AutoAnnotate App in Cytoscape.
  The most significantly enriched gene set is used to label a gene set cluster, defined
  by NES. Red node: gene set enriched in EPO4. Purple node label: top gene sets with
  NES > 1.90. The enrichment map was created with pathway FDR < 0.1, nominal P < 0.05
  and Jaccard Overlap coefficient > 0.375 with combined constant k = 0.5.'
papertitle: Cross-platform transcriptomic profiling of the response to recombinant
  human erythropoietin.
reftext: Guan Wang, et al. Sci Rep. 2021;11:21705.
year: '2021'
doi: 10.1038/s41598-021-00608-9
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Computational biology and bioinformatics | Molecular biology | Systems biology
automl_pathway: 0.630743
figid_alias: PMC8568984__F3
figtype: Figure
redirect_from: /figures/PMC8568984__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8568984__41598_2021_608_Fig3_HTML.html
  '@type': Dataset
  description: 'Biological network of the MGI RNA-seq dataset following Gene Ontology
    (biological process) gene set enrichment analysis in GSEA (v4.0.3) and visualisation
    in Cytoscape (3.8.0). Each circle (node) represents a gene set and two nodes are
    connected by lines (edges) indicating shared genes. The size of a node and width
    of an edge are proportional to the number of genes enriched in a gene set and
    the number of genes shared between gene sets, respectively. Gene sets that are
    similar were annotated and clustered to form a biological theme using the AutoAnnotate
    App in Cytoscape. The most significantly enriched gene set is used to label a
    gene set cluster, defined by NES. Red node: gene set enriched in EPO4. Purple
    node label: top gene sets with NES > 1.90. The enrichment map was created with
    pathway FDR < 0.1, nominal P < 0.05 and Jaccard Overlap coefficient > 0.375 with
    combined constant k = 0.5.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - anon-70Dc
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - 'On'
  - Lon
  - schlank
  - Low
  - Pomp
  - pe
  - toy
  - mul
  - mu1
  - AP-1mu
  - GluClalpha
  - Mhc
  - zip
  - levy
  - COX6AL2
  - WARS1
  - LONP1
  - PARTICL
  - POMP
  - NT5C3A
  - TRIM37
  - GSTM1
  - HLA-C
  - SEMA6A
---
