---
figid: PMC8042281__fgene-12-639602-g003
figtitle: Histone modifications, dynamically added or removed around the DNA damage
  site, constitute an integrated signal for the DSBs repair pathway choice
organisms:
- Homo sapiens
- Mus musculus
- Saccharomyces cerevisiae
- Drosophila melanogaster
pmcid: PMC8042281
filename: fgene-12-639602-g003.jpg
figlink: pmc/articles/PMC8042281/figure/F3/
number: F3
caption: 'Histone modifications, dynamically added or removed around the DNA damage
  site, constitute an integrated signal for the DSBs repair pathway choice. (A) 53BP1
  is a master regulator of DNA double-strand break repair pathway choice toward NHEJ
  and its binding to chromatin is mainly regulated by specific histone modifications,
  comprising H2A ubiquitylation and H3 and H4 methylation. Indeed, specific HPTMs
  can generate a docking site for 53BP1, which binds the ubiquitinated H2A/H2A.X K13/15/119
  through its UDR motif and H4K20me2, H4K16me1, and H3K79me3 through its tandem Tudor
  domain. Therefore, despite H4K20me2 being the main histone mark involved in 53BP1
  recruitment, other marks may contribute to it. For example, the deacetylation of
  H4K16 by HDAC1/2 indirectly promotes the 53BP1 recruitment and the NHEJ over HR,
  because it makes possible the methylation of this lysine residue. Under normal conditions,
  the H4K20me2 mark is masked by various interacting proteins, including L3MBTL1 and
  KDM4A/JMJD2A. Upon DNA DSBs induction, the ATM phosphorylation cascade promotes
  the recruitment of the E3 ubiquitin ligase RNF8 to the DNA damage site, which, together
  with the E2 ubiquitin ligase UBC13, promotes the relocation of the E3 ubiquitin
  ligase RNF168 to the DNA break sites. LSD1 may favor the 53BP1 recruitment interacting
  with RNF168 and/or thanks to a possible epigenetic cross-talk between H3K4 demethylation
  and H2A ubiquitylation. RNF168 catalyzes in turn the H2A K13/15 mono-ubiquitination
  and the poly-ubiquitination of L3MBTL1 and KDM4A, leading to RNF8-dependent degradation
  of KDM4A and to VCP-mediated and ubiquitin-dependent removal of L3MBTL1 from chromatin,
  favoring the unmasking of H4K20me2 and the loading of 53BP1 around the damaged site.
  LSD1 can also disadvantage 53BP1 recruitment, negatively regulating the interaction
  between p53 and 53BP1 (not shown). After DSB induction by IR, the H3K36me2 markedly
  increases, improving the association of early NHEJ factors such as Ku70/Ku80 and
  NBS1 at the damage site. Once activated, ATM (not shown) may negatively regulate
  KDM2A chromatin-binding capacity, favoring its displacement, which favors an increase
  of H3K36 di-methylation mark near the DNA damage sites by SETMAR. H3K36me3 can favor
  the retention of the NHEJ-associated factor PHF1, which is recruited in a Ku70/Ku80
  dependent way and which supports an open chromatin for the efficient DNA repair.
  In contrast to H3K36me2, H3K36me3 regulates both NHEJ and HR, probably depending
  on the relative abundance of its reader factors and/or by the chromatin context.
  In this regard, pre-existing transcription-associated and cell cycle-regulated histone
  modifications together with those damage-induced regulate the DNA repair pathways.
  (B) The pre-existing H3K36me3 mark at actively transcribed gene bodies favors DNA
  repair by HR pathway choice: in human cells, both LEDGF (Lens epithelium-derived
  growth factor), via its PWWP reader domain, and MRG15, via its chromodomain, can
  bind H3K36me3, promoting the recruitment of CtIP and PALB2, two crucial factors
  for the DNA end-resection and strand invasion, respectively. Moreover, mutually
  exclusive histone modifications can contribute to tip the balance between HR and
  NHEJ in DSB repair pathway choice: a switch between ubiquitylation and acetylation
  on H2AK15 and H4K16 can favor HR over NHEJ. The human NuA4/TIP60 acetyltransferase
  complex is able to inhibit the 53BP1 recruitment both through this mechanism and
  by targeting the same H4K20me2 mark, by binding H4K20me with the MBT (malignant
  brain tumor) domain of MBTD1, its stable subunit. Moreover, the human NuA4/TIP60
  acetyltransferase complex promotes the repositioning of 53BP1 through other mechanisms,
  such as the acetylation of H2A/H2AX K13/15 that blocks the recruitment of RNF8/168
  and through the acetylation of H4K16 that impedes the methylation of this lysin
  residue, limiting the recruitment of 53BP1. KDM4A, besides masking H3K20me2 and
  limiting the 53BP1 recruitment thus inhibiting the NHEJ pathway, negatively regulates
  the DDR also removing H3K36me3 mark thus inhibiting the HR pathway. Conversely,
  KMT3A favors the HR through the methylation of H3K36. KMT1E favors the HR though
  the methylation of H3K9. Indeed, the H3K9me2 mark is involved in the recruitment
  of NuA4/TIP60 acetyltransferase complex, and H3K9me3 is involved in the recruitment
  of BARD1/BRCA1 Ub ligase activity on H2AK127 favoring the DNA end-resection, the
  repositioning of 53BP1 and thus the HR. Created using BioRender.'
papertitle: The Role of Histone Lysine Methylation in the Response of Mammalian Cells
  to Ionizing Radiation.
reftext: Elena Di Nisio, et al. Front Genet. 2021;12:639602.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.9637307
figid_alias: PMC8042281__F3
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Drosophila melanogaster
- Saccharomyces cerevisiae S288C
redirect_from: /figures/PMC8042281__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8042281__fgene-12-639602-g003.html
  '@type': Dataset
  description: 'Histone modifications, dynamically added or removed around the DNA
    damage site, constitute an integrated signal for the DSBs repair pathway choice.
    (A) 53BP1 is a master regulator of DNA double-strand break repair pathway choice
    toward NHEJ and its binding to chromatin is mainly regulated by specific histone
    modifications, comprising H2A ubiquitylation and H3 and H4 methylation. Indeed,
    specific HPTMs can generate a docking site for 53BP1, which binds the ubiquitinated
    H2A/H2A.X K13/15/119 through its UDR motif and H4K20me2, H4K16me1, and H3K79me3
    through its tandem Tudor domain. Therefore, despite H4K20me2 being the main histone
    mark involved in 53BP1 recruitment, other marks may contribute to it. For example,
    the deacetylation of H4K16 by HDAC1/2 indirectly promotes the 53BP1 recruitment
    and the NHEJ over HR, because it makes possible the methylation of this lysine
    residue. Under normal conditions, the H4K20me2 mark is masked by various interacting
    proteins, including L3MBTL1 and KDM4A/JMJD2A. Upon DNA DSBs induction, the ATM
    phosphorylation cascade promotes the recruitment of the E3 ubiquitin ligase RNF8
    to the DNA damage site, which, together with the E2 ubiquitin ligase UBC13, promotes
    the relocation of the E3 ubiquitin ligase RNF168 to the DNA break sites. LSD1
    may favor the 53BP1 recruitment interacting with RNF168 and/or thanks to a possible
    epigenetic cross-talk between H3K4 demethylation and H2A ubiquitylation. RNF168
    catalyzes in turn the H2A K13/15 mono-ubiquitination and the poly-ubiquitination
    of L3MBTL1 and KDM4A, leading to RNF8-dependent degradation of KDM4A and to VCP-mediated
    and ubiquitin-dependent removal of L3MBTL1 from chromatin, favoring the unmasking
    of H4K20me2 and the loading of 53BP1 around the damaged site. LSD1 can also disadvantage
    53BP1 recruitment, negatively regulating the interaction between p53 and 53BP1
    (not shown). After DSB induction by IR, the H3K36me2 markedly increases, improving
    the association of early NHEJ factors such as Ku70/Ku80 and NBS1 at the damage
    site. Once activated, ATM (not shown) may negatively regulate KDM2A chromatin-binding
    capacity, favoring its displacement, which favors an increase of H3K36 di-methylation
    mark near the DNA damage sites by SETMAR. H3K36me3 can favor the retention of
    the NHEJ-associated factor PHF1, which is recruited in a Ku70/Ku80 dependent way
    and which supports an open chromatin for the efficient DNA repair. In contrast
    to H3K36me2, H3K36me3 regulates both NHEJ and HR, probably depending on the relative
    abundance of its reader factors and/or by the chromatin context. In this regard,
    pre-existing transcription-associated and cell cycle-regulated histone modifications
    together with those damage-induced regulate the DNA repair pathways. (B) The pre-existing
    H3K36me3 mark at actively transcribed gene bodies favors DNA repair by HR pathway
    choice: in human cells, both LEDGF (Lens epithelium-derived growth factor), via
    its PWWP reader domain, and MRG15, via its chromodomain, can bind H3K36me3, promoting
    the recruitment of CtIP and PALB2, two crucial factors for the DNA end-resection
    and strand invasion, respectively. Moreover, mutually exclusive histone modifications
    can contribute to tip the balance between HR and NHEJ in DSB repair pathway choice:
    a switch between ubiquitylation and acetylation on H2AK15 and H4K16 can favor
    HR over NHEJ. The human NuA4/TIP60 acetyltransferase complex is able to inhibit
    the 53BP1 recruitment both through this mechanism and by targeting the same H4K20me2
    mark, by binding H4K20me with the MBT (malignant brain tumor) domain of MBTD1,
    its stable subunit. Moreover, the human NuA4/TIP60 acetyltransferase complex promotes
    the repositioning of 53BP1 through other mechanisms, such as the acetylation of
    H2A/H2AX K13/15 that blocks the recruitment of RNF8/168 and through the acetylation
    of H4K16 that impedes the methylation of this lysin residue, limiting the recruitment
    of 53BP1. KDM4A, besides masking H3K20me2 and limiting the 53BP1 recruitment thus
    inhibiting the NHEJ pathway, negatively regulates the DDR also removing H3K36me3
    mark thus inhibiting the HR pathway. Conversely, KMT3A favors the HR through the
    methylation of H3K36. KMT1E favors the HR though the methylation of H3K9. Indeed,
    the H3K9me2 mark is involved in the recruitment of NuA4/TIP60 acetyltransferase
    complex, and H3K9me3 is involved in the recruitment of BARD1/BRCA1 Ub ligase activity
    on H2AK127 favoring the DNA end-resection, the repositioning of 53BP1 and thus
    the HR. Created using BioRender.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Brca1
  - Kmt5b
  - Ube2n
  - Kdm1a
  - Kmt5c
  - Rnf8
  - Rnf168
  - Kmt5a
  - L3mbtl1
  - Nsd2
  - Prc1
  - Vcp
  - Kdm4a
  - Hdac2
  - Hdac1
  - Hdac1-ps
  - Targ1
  - Taar4
  - Emp3
  - Il4i1
  - Selenos
  - Me1
  - Tcf12
  - Me2
  - Tcf4
  - Tcf3
  - Rab8a
  - H2ax
  - Kdm2a
  - Kdm3a
  - Me3
  - H3
  - Xrcc6
  - Phf1
  - Xrcc5
  - Setmar
  - Kat5
  - Mbtd1
  - Palb2
  - Bard1
  - Morf4l1
  - Rad51
  - Xrcc2
  - ac
  - Krt20
  - Or4
  - Or4b13
  - Trp53bp1
  - Setd2
  - Suv39h1
  - Setdb1
  - BRCA1
  - KMT5B
  - UBE2N
  - KDM1A
  - KMT5C
  - RNF8
  - RNF168
  - KMT5A
  - PRTN3
  - L3MBTL1
  - NSD2
  - PRC1
  - BMI1
  - CBX2
  - CBX4
  - CBX8
  - PCGF2
  - PHC1
  - PHC2
  - PHC3
  - PHF1
  - RING1
  - RNF2
  - SCMH1
  - YY1
  - ZNF134
  - VCP
  - KDM4A
  - HDAC2
  - HDAC1
  - ME1
  - PCDHB16
  - ME2
  - CELSR1
  - PCDHB11
  - RAB8A
  - MAP2K1
  - H2AX
  - TUBA4A
  - H2AC18
  - H2AC20
  - TUBA3D
  - KDM2A
  - ME3
  - PCDHGA5
  - XRCC6
  - XRCC5
  - SETMAR
  - GBP1P1
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - KAT5
  - MBTD1
  - PALB2
  - BARD1
  - MORF4L1
  - RAD51
  - PSIP1
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - KRT20
  - TP53BP1
  - SLC9C1
  - SETD2
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - MARK1
  - SUV39H1
  - SETDB1
  - ben
  - Lsd-1
  - Su(var)3-3
  - mbt
  - l(3)mbt
  - feo
  - TER94
  - tud
  - Kdm4A
  - HDAC6
  - Tailor
  - Men
  - mel
  - l(1)16Fa
  - mei-W68
  - His2Av
  - Blm
  - Irbp
  - Ku80
  - Tip60
  - MRG15
  - Chro
  - Rad51D
  - spn-B
  - spn-A
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Nhe3
  - par-1
  - UBC13
  - RAD53
  - KDM2A  Me2  Me2
  - Ac
  - Me3
  - Me2
---
