---
figid: PMC8649922__fonc-11-777273-g001
figtitle: 'Microenvironmental Factors Modulating Tumor Lipid Metabolism: Paving the
  Way to Better Antitumoral Therapy'
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Candida dubliniensis
- gut metagenome
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Drosophila melanogaster
pmcid: PMC8649922
filename: fonc-11-777273-g001.jpg
figlink: /pmc/articles/PMC8649922/figure/f1/
number: F1
caption: A brief representation of altered lipid metabolism in cancer cells. Lipids
  are metabolized in cancer cells involving molecular process of uptake, de novo synthesis,
  degradation, storage and lipolysis. Aside from being structural components of membrane,
  lipids also function as energy sources and signaling factors coordinating various
  biological processes. Acetyl-CoA is the main substrate for lipid synthesis while
  the altered metabolism of glucose, glutamine and acetate could contribute to this
  metabolic pathway by providing precursors. FAs are then either break down through
  mitochondrial β-oxidation to produce energy, or utilized in membrane synthesis together
  with cholesterol. Excessive lipids are then usually stored in LDs as TG and CE,
  and could be mobilized for another source of lipids. LD is recognized as important
  cellular organelle in regulating autophagy and anti-lipotoxicity. SREBPs are pivotal
  transcription factors that respectively modulates the genes involved in the uptake
  and synthesis of fatty acid and cholesterol, and responds to the upstream signaling
  (i.e., PI3K/AKT/mTOR, AMPK pathway). Other pathway like PPAR and LXR signaling also
  play an active role in modulating lipid reprogramming.
papertitle: 'Microenvironmental Factors Modulating Tumor Lipid Metabolism: Paving
  the Way to Better Antitumoral Therapy.'
reftext: Limeng Cai, et al. Front Oncol. 2021;11:777273.
year: '2021'
doi: 10.3389/fonc.2021.777273
journal_title: Frontiers in Oncology
journal_nlm_ta: Front Oncol
publisher_name: Frontiers Media S.A.
keywords: tumor micoenvironment | metabolic reprogramming | microenvironment factor
  | cancer therapy | lipid metabolism
automl_pathway: 0.9665366
figid_alias: PMC8649922__F1
figtype: Figure
redirect_from: /figures/PMC8649922__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8649922__fonc-11-777273-g001.html
  '@type': Dataset
  description: A brief representation of altered lipid metabolism in cancer cells.
    Lipids are metabolized in cancer cells involving molecular process of uptake,
    de novo synthesis, degradation, storage and lipolysis. Aside from being structural
    components of membrane, lipids also function as energy sources and signaling factors
    coordinating various biological processes. Acetyl-CoA is the main substrate for
    lipid synthesis while the altered metabolism of glucose, glutamine and acetate
    could contribute to this metabolic pathway by providing precursors. FAs are then
    either break down through mitochondrial β-oxidation to produce energy, or utilized
    in membrane synthesis together with cholesterol. Excessive lipids are then usually
    stored in LDs as TG and CE, and could be mobilized for another source of lipids.
    LD is recognized as important cellular organelle in regulating autophagy and anti-lipotoxicity.
    SREBPs are pivotal transcription factors that respectively modulates the genes
    involved in the uptake and synthesis of fatty acid and cholesterol, and responds
    to the upstream signaling (i.e., PI3K/AKT/mTOR, AMPK pathway). Other pathway like
    PPAR and LXR signaling also play an active role in modulating lipid reprogramming.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ldlr
  - fa
  - Pik3r1
  - Cd36
  - Scarb1
  - Akt1
  - Stk11
  - Sirt1
  - Ppara
  - Nr1h3
  - Nr1h2
  - Srebf2
  - Srebf1
  - Hmgcr
  - Scd1
  - Fasn
  - Hps3
  - Gnat2
  - Acc
  - Acly
  - Tg
  - Acss2
  - Gnas
  - Mcph1
  - Slc1a5
  - Slc1a3
  - LDLR
  - CRABP1
  - CRABP2
  - FABP1
  - FABP12
  - FABP2
  - FABP3
  - FABP4
  - FABP5
  - FABP6
  - FABP7
  - FABP9
  - PMP2
  - RBP1
  - RBP2
  - RBP5
  - RBP7
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - CD36
  - MTOR
  - RPTOR
  - AKT1
  - AKT2
  - AKT3
  - STK11
  - SIRT1
  - PRKAA1
  - PRKAA2
  - PRKAB1
  - PRKAB2
  - PRKAG1
  - PRKAG2
  - PRKAG3
  - PPARA
  - PPARD
  - PPARG
  - NR1H2
  - NR1H3
  - SREBF2
  - SREBF1
  - ACSL1
  - ACSL3
  - ACSL4
  - ACSL5
  - ACSL6
  - CHPT1
  - DHDDS
  - HMGCR
  - SCD
  - FASN
  - HMGCS1
  - ACACA
  - BMS1
  - ACACB
  - ACLY
  - TG
  - ACSS2
  - GLS
  - ATP8A2
  - GNAS
  - SLC16A1
  - MCAT
  - MCPH1
  - SLC1A5
  - SLC2A1
  - Fabp2
  - Pik3cg
  - Prkaa2
  - Scd
  - Ccne1
  - Gls
  - modSP
  - LRP1
  - LpR2
  - mgl
  - fabp
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - crq
  - Crtc
  - Akt
  - Lkb1
  - SNF4Agamma
  - AMPKalpha
  - SREBP
  - Acsl
  - Cpt
  - CG7149
  - CPT2
  - de
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - FASN1
  - FASN2
  - Dsp1
  - Coa
  - CycE
  - cyc
  - ACC
  - tacc
  - acclinal-wing
  - ATPCL
  - ld
  - ATPsynbeta
  - Atpalpha
  - CG8028
  - Abca3
  - Eaat1
  - Glut1
  - Glut4EF
  - Glut3
---
