---
figid: PMC8136750__nihms-1692881-f0003
figtitle: Metabolic signaling to epigenetic transcriptional control in the failing
  heart
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC8136750
filename: nihms-1692881-f0003.jpg
figlink: /pmc/articles/PMC8136750/figure/F3/
number: F3
caption: 'Left: Mitochondrial metabolic flux of primary cardiac metabolites, glucose
  and fatty acids, directly alter NAD+/NADH ratios as well as the Acetyl-CoA pool
  that signal to the nucleus to impact the epigenetic environment. Additionally, glucose
  that does not enter oxidative metabolism in the mitochondria can signal through
  the HBP. Additional metabolic intermediates discussed in the text, βOHB, can also
  regulate activity of epigenetic regulating enzymes as can single carbon metabolism
  via folate and SAM. Middle: Metabolites and metabolic pathways regulate the activity
  and/or substrate availability of epigenetic modifiers of histones (e.g., HDAC, HAT,
  HDM, HMT) and DNA (e.g., DNMT and TET). Right: A) Histone modifications include
  inhibitory tri-methylation as well as activating acetylation. B) DNA modifications
  such as DNA methylation (5mC) at CGI in promoters are associated with gene silencing
  while CGI demethylation and gene body DNA hydroxymethylation (5hmC) modifications
  are associated with gene activation. C) Although metabolites are not known to be
  directly regulated by metabolic intermediates, a number of metabolic genes are regulated
  by miR and lncRNA expression. Please see references included in the main text for
  additional details. (Illustration credit: Ben Smith).Abbreviations: α-ketoglutarate
  (αKG), β-hydroxybutyrate (βOHB), CpG island (CGI), DNA methyltransferase (DNMT),
  fatty acid oxidation (FAO), hexosamine biosynthetic pathway (HBP), histones (H2A,
  H2B, H3, H4), histone acetyltransferase (HAT), histone deacetylase (HDAC), histone
  demethylase (HDM), histone methyltransferase (HMT), long noncoding RNA (lncRNA),
  microRNA (miR), oxidative phosphorylation (OXPHOS), pyruvate dehydrogenase (PDH)
  complex, S-adenosylmethionine (SAM), ten-eleven translocation (TET), and tricarboxylic
  acid (TCA) cycle.'
papertitle: Cardiac Energy Metabolism in Heart Failure.
reftext: Gary D. Lopaschuk, et al. Circ Res. ;128(10):1487-1513.
year: '2021'
doi: 10.1161/CIRCRESAHA.121.318241
journal_title: Circulation research
journal_nlm_ta: Circ Res
publisher_name: ''
keywords: mitochondria | glucose oxidation | fatty acid oxidation | ketones | insulin
  resistance | diabetic cardiomyopathy | lipotoxicity | branched chain amino acids
  | acetylation | NAD+ | NADH
automl_pathway: 0.9598975
figid_alias: PMC8136750__F3
figtype: Figure
redirect_from: /figures/PMC8136750__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8136750__nihms-1692881-f0003.html
  '@type': Dataset
  description: 'Left: Mitochondrial metabolic flux of primary cardiac metabolites,
    glucose and fatty acids, directly alter NAD+/NADH ratios as well as the Acetyl-CoA
    pool that signal to the nucleus to impact the epigenetic environment. Additionally,
    glucose that does not enter oxidative metabolism in the mitochondria can signal
    through the HBP. Additional metabolic intermediates discussed in the text, βOHB,
    can also regulate activity of epigenetic regulating enzymes as can single carbon
    metabolism via folate and SAM. Middle: Metabolites and metabolic pathways regulate
    the activity and/or substrate availability of epigenetic modifiers of histones
    (e.g., HDAC, HAT, HDM, HMT) and DNA (e.g., DNMT and TET). Right: A) Histone modifications
    include inhibitory tri-methylation as well as activating acetylation. B) DNA modifications
    such as DNA methylation (5mC) at CGI in promoters are associated with gene silencing
    while CGI demethylation and gene body DNA hydroxymethylation (5hmC) modifications
    are associated with gene activation. C) Although metabolites are not known to
    be directly regulated by metabolic intermediates, a number of metabolic genes
    are regulated by miR and lncRNA expression. Please see references included in
    the main text for additional details. (Illustration credit: Ben Smith).Abbreviations:
    α-ketoglutarate (αKG), β-hydroxybutyrate (βOHB), CpG island (CGI), DNA methyltransferase
    (DNMT), fatty acid oxidation (FAO), hexosamine biosynthetic pathway (HBP), histones
    (H2A, H2B, H3, H4), histone acetyltransferase (HAT), histone deacetylase (HDAC),
    histone demethylase (HDM), histone methyltransferase (HMT), long noncoding RNA
    (lncRNA), microRNA (miR), oxidative phosphorylation (OXPHOS), pyruvate dehydrogenase
    (PDH) complex, S-adenosylmethionine (SAM), ten-eleven translocation (TET), and
    tricarboxylic acid (TCA) cycle.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pdh
  - Pdha
  - Pdf
  - Pdhb
  - HDAC4
  - HDAC6
  - HDAC1
  - HDAC3
  - hat
  - Elp3
  - CG1894
  - chm
  - hdm
  - Su(var)3-3
  - Mt2
  - Nmt
  - sam
  - qkr54B
  - sls
  - Sam-S
  - His2Av
  - PDP1
  - DLAT
  - DLD
  - PDHA1
  - PDHA2
  - PDHB
  - PDHX
  - AZU1
  - HDLBP
  - SLBP
  - STAM2
  - HEBP1
  - HDAC9
  - HDAC2
  - HDAC8
  - HDAC5
  - HDAC7
  - HDAC10
  - SIRT1
  - SIRT2
  - SIRT3
  - SIRT4
  - SIRT5
  - SIRT6
  - SIRT7
  - HDAC11
  - OGT
  - TMPRSS11D
  - HNMT
  - DNMT1
  - DNMT3A
  - DNMT3B
  - DNMT3L
  - TUBA4A
  - H2AC18
  - H2AC20
  - TUBA3D
  - H2BC21
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
---
