---
figid: PMC8027424__41396_2020_805_Fig1_HTML
figtitle: Autotrophic and mixotrophic metabolism of an anammox bacterium revealed
  by in vivo 13C and 2H metabolic network mapping
organisms:
- Clostridium acetobutylicum
- Clostridium kluyveri
- Candidatus Brocadia fulgida
- Candidatus Jettenia caeni
- bacterium K
- Candidatus Anammoxoglobus propionicus
- Candidatus Brocadia sinica
- Arabidopsis thaliana
- Komagataella pastoris
- unidentified microorganism
organisms_ner:
- Arabidopsis thaliana
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC8027424
filename: 41396_2020_805_Fig1_HTML.jpg
figlink: pmc/articles/PMC8027424/figure/Fig1/
number: F1
caption: Black arrows represent reactions of the Wood–Ljungdahl pathway, gluconeogenesis,
  the TCA cycle, and the pentose phosphate pathway. Grey arrows indicate amino acid
  biosynthetic pathways. Pink metabolites represent reducing equivalents. Red metabolites
  indicate ATP equivalents. “?” indicates gene or pathway not identified in genome.
  A list of all reactions can be found in Supplementary Table . Confirmation of gene
  expression via proteomic analysis can be found in Supplementary Dataset .
papertitle: Autotrophic and mixotrophic metabolism of an anammox bacterium revealed
  by in vivo 13C and 2H metabolic network mapping.
reftext: Christopher E. Lawson, et al. ISME J. 2021 Mar;15(3):673-687.
year: '2021'
doi: .na.character
journal_title: .na.character
journal_nlm_ta: .na.character
publisher_name: .na.character
keywords: .na.character
automl_pathway: 0.919395
figid_alias: PMC8027424__F1
figtype: Figure
redirect_from: /figures/PMC8027424__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8027424__41396_2020_805_Fig1_HTML.html
  '@type': Dataset
  description: Black arrows represent reactions of the Wood–Ljungdahl pathway, gluconeogenesis,
    the TCA cycle, and the pentose phosphate pathway. Grey arrows indicate amino acid
    biosynthetic pathways. Pink metabolites represent reducing equivalents. Red metabolites
    indicate ATP equivalents. “?” indicates gene or pathway not identified in genome.
    A list of all reactions can be found in Supplementary Table . Confirmation of
    gene expression via proteomic analysis can be found in Supplementary Dataset .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PG1
  - PGI1
  - RSW10
  - RPE
  - FBP
  - ATA1
  - FDH
  - KCS10
  - TPI
  - PGK
  - CMPG2
  - CMPG1
  - PGM
  - FD
  - PEP
  - PAS2
  - MAX2
  - POR
  - ACS
  - MDH
  - FH
  - amon
  - G6P
  - gnd
  - Pgi
  - Rpi
  - PIG-Q
  - Rpe
  - fbp
  - Taldo
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
  - alphaTub84B
  - tai
  - Fdh
  - Aldh-III
  - Pfk
  - CG15812
  - Tpi
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - ATPsynbeta
  - Atpalpha
  - Pgk
  - Cys
  - mmy
  - Csp
  - Pgm1
  - fd
  - anon-16Fb
  - red
  - trc
  - Eno
  - pck
  - Pep
  - bgcn
  - pps
  - EcR
  - pyr
  - porin
  - por
  - AcCoAS
  - Acsl
  - cs
  - Mdh1
  - Mdh2
  - Men-b
  - Men
  - Acn
  - fh
  - CycE
  - cyc
  - Frd
  - arg
  - aos
  - Suc
  - DECR1
  - SDHD
  - BGN
  - GPI
  - PGPEP1
  - RPIA
  - GNPDA1
  - PIGQ
  - TKT
  - DDR2
  - FBP1
  - FOLR1
  - FOLR2
  - ECB2
  - FUBP1
  - RAB14
  - TALDO1
  - LRSAM1
  - TAAR1
  - ADH5
  - ALDH1L1
  - PFKL
  - PFKM
  - PFKP
  - TPI1
  - RASA1
  - RGS6
  - ATP8A2
  - AKTIP
  - VCAN
  - IK
  - DYRK3
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - PAEP
  - PREP
  - PTPN22
  - IRF6
  - INPP5K
  - ARFIP2
  - VDAC2
  - SLC27A5
  - PLA2G15
  - ACSS2
  - ACCS
  - CS
  - MDH2
  - NF2
  - ACIN1
  - GNPTAB
  - IZUMO1R
  - ABL2
  - RERE
  - 2 NADPH
  - CO
  - Ru5P
  - Phe
  - E4P-Trp
  - Tyr
  - NADH
  - DHAP
  - Formate
  - NADH99
  - ATP
  - CHO-THF
  - Ser
  - Cys
  - CH
  - THF
  - 5,10-CH,-THF
  - Fd
  - 5-CH3-THF
  - Val
  - Ala
  - Leu
  - Acetyl-CoA
  - Acetate
  - Asn
  - Thr
  - Asp
  - Lys
  - Oxaloacetate
  - Citrate
  - NADHY
  - TSA
  - Isocitrate
  - Succinate
  - Glu
  - Arg
  - Succinyl
---
