---
figid: PMC8901651__41598_2022_7810_Fig4_HTML
figtitle: Patients with low muscle mass have characteristic microbiome with low potential
  for amino acid synthesis in chronic liver disease
organisms:
- gut metagenome
- metagenome
- Homo sapiens
- Mus musculus
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8901651
filename: 41598_2022_7810_Fig4_HTML.jpg
figlink: /pmc/articles/PMC8901651/figure/Fig4/
number: F4
caption: Mapping of functional genes for amino acid synthesis. The genes predicted
  using PICRUSt2 are numbered with K numbers. Each K number related to amino acid
  synthesis refers to an enzyme. The ‘○’ in the pathway diagram shows the metabolite,
  and the ‘ → ’ indicates the metabolic enzyme. The amount of the gene for the enzyme
  in each patient's microbiome was compared between the two groups. The pathway with
  a q-value less than 0.1 is colored. Multiple blue or light blue arrows show that
  the N-SMI group had more enzymes associated with each amino acid synthesis compared
  to the L-SMI group. The pathways related to valine, leucine, and isoleucine were
  more abundant in the N-SMI group than the L-SMI group. Metabolic pathways are surrounded
  by blue squares. For better understanding, the important factors in amino acid synthesis,
  fructose-6-phosphate (†), pyruvate (‡), and oxaloacetic acid (§), are indicdated
  with a marked focus.
papertitle: Patients with low muscle mass have characteristic microbiome with low
  potential for amino acid synthesis in chronic liver disease.
reftext: Kenta Yamamoto, et al. Sci Rep. 2022;12:3674.
year: '2022'
doi: 10.1038/s41598-022-07810-3
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Liver cirrhosis | Translational research
automl_pathway: 0.9363096
figid_alias: PMC8901651__F4
figtype: Figure
redirect_from: /figures/PMC8901651__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8901651__41598_2022_7810_Fig4_HTML.html
  '@type': Dataset
  description: Mapping of functional genes for amino acid synthesis. The genes predicted
    using PICRUSt2 are numbered with K numbers. Each K number related to amino acid
    synthesis refers to an enzyme. The ‘○’ in the pathway diagram shows the metabolite,
    and the ‘ → ’ indicates the metabolic enzyme. The amount of the gene for the enzyme
    in each patient's microbiome was compared between the two groups. The pathway
    with a q-value less than 0.1 is colored. Multiple blue or light blue arrows show
    that the N-SMI group had more enzymes associated with each amino acid synthesis
    compared to the L-SMI group. The pathways related to valine, leucine, and isoleucine
    were more abundant in the N-SMI group than the L-SMI group. Metabolic pathways
    are surrounded by blue squares. For better understanding, the important factors
    in amino acid synthesis, fructose-6-phosphate (†), pyruvate (‡), and oxaloacetic
    acid (§), are indicdated with a marked focus.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PYM1
  - PLCL1
  - PRNP
  - NCOA6
  - PATE1
  - NHS
  - SPTA1
  - HPP1
  - Pym
  - Pfas
  - Prip
  - op
---
