---
figid: PMC8397972__nutrients-13-02521-g004
figtitle: Convergent Functional Genomics Analysis to Identify Biological Regulators
  Mediating Effects of Creatine Supplementation
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Xenopus laevis
pmcid: PMC8397972
filename: nutrients-13-02521-g004.jpg
figlink: /pmc/articles/PMC8397972/figure/nutrients-13-02521-f004/
number: F4
caption: 'Bioinformatics- and knowledge-based pathway reconstruction after Cr supplementation.
  This is a representation of pathways interactions based on the results of our enrichment
  analysis of differentially expressed genes after increasing cellular Cr concentration
  and the available experimental evidence. This functional network follows the ‘bio-logic’
  (integration of bottom-up and top-down directions) of the genotype–outcome interaction.
  MAPK activation can occur via osmosensing pathways that activate Ras/Raf (e.g.,
  S1P/SPHK1) and mechanosensing pathways that involve mechanical and energy optimization
  of the cytoskeleton (e.g., Four-and-a-Half Lim 2 is an important mechanosensor that
  triggers hypertrophy in response to strain and also docks key metabolic enzymes
  involved in the energy transduction process, such as M-CK, adenylate kinase, and
  phosphofructokinase). Several subunits of the protein complexes and the architecture
  of the cytoskeleton are not depicted for readability. We cautiously suggest two
  dose–response functional pathways for the regulation of the Cr uptake: a kinase-driven
  mechanism as a result of the initial Cr-enriched environment, which is more related
  to the anterograde trafficking via endolysosome-specific phosphoinositide compounds
  (1); and a ubiquitin-driven mechanism that controls the excessive Cr uptake, which
  is more related to the retrograde trafficking via clathrin-dependent and clathrin-independent
  processes (2). Interlinking protein filaments of the cytoskeleton are represented
  with lighter-colored lines in the background. See the sections of the manuscript
  for rationale, citations, and more abbreviations. Dashed arrows represent multiple
  steps. AMPK: AMP-activated protein kinase; CK: creatine kinase; GASP-1: growth and
  differentiation factor (GDF)-associated serum protein-1; GDP: guanosine diphosphate;
  GH: growth hormone; GLUT: glucose transporter; GRB2: growth factor receptor-bound
  protein 2; GTP: guanosine triphosphate; IGF-1: insulin-like growth factor-1; LST8:
  target of rapamycin complex subunit LST8; MRFs: myogenic regulatory factors; Nedd4-2:
  E3 ubiquitin–protein ligase NEDD4-like; OSR1: oxidative-stress-responsive kinase
  1; PDK1: phosphoinositide-dependent kinase-1; PFK: phosphofructokinase; PI(3,5)P2:
  phosphatidylinositol 3,5-bisphosphate; PIKfyve: 1-phosphatidylinositol 3-phosphate
  5-kinase; PI3K: phosphoinositide 3-kinase; PIP3: phosphatidylinositol (3,4,5)-trisphosphate;
  PK: pyruvate kinase; SGK: serum- and glucocorticoid-regulated kinase; SPAK: SPS1-related
  proline–alanine-rich kinase; RACK1: receptor for activated C kinase 1; RTK: receptor
  tyrosine kinases; TFs: transcription factors. Source: created by the authors (D.A.B.)
  with BioRender—https://biorender.com/ (accessed on 10 May 2021).'
papertitle: A Convergent Functional Genomics Analysis to Identify Biological Regulators
  Mediating Effects of Creatine Supplementation.
reftext: Diego A. Bonilla, et al. Nutrients. 2021 Aug;13(8):2521.
year: '2021'
doi: 10.3390/nu13082521
journal_title: Nutrients
journal_nlm_ta: Nutrients
publisher_name: MDPI
keywords: creatine kinase | systems biology | bioinformatics | MAP kinase signaling
  system | sodium-chloride-dependent neurotransmitter symporters | signal transduction
automl_pathway: 0.9375139
figid_alias: PMC8397972__F4
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Xenopus laevis
redirect_from: /figures/PMC8397972__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8397972__nutrients-13-02521-g004.html
  '@type': Dataset
  description: 'Bioinformatics- and knowledge-based pathway reconstruction after Cr
    supplementation. This is a representation of pathways interactions based on the
    results of our enrichment analysis of differentially expressed genes after increasing
    cellular Cr concentration and the available experimental evidence. This functional
    network follows the ‘bio-logic’ (integration of bottom-up and top-down directions)
    of the genotype–outcome interaction. MAPK activation can occur via osmosensing
    pathways that activate Ras/Raf (e.g., S1P/SPHK1) and mechanosensing pathways that
    involve mechanical and energy optimization of the cytoskeleton (e.g., Four-and-a-Half
    Lim 2 is an important mechanosensor that triggers hypertrophy in response to strain
    and also docks key metabolic enzymes involved in the energy transduction process,
    such as M-CK, adenylate kinase, and phosphofructokinase). Several subunits of
    the protein complexes and the architecture of the cytoskeleton are not depicted
    for readability. We cautiously suggest two dose–response functional pathways for
    the regulation of the Cr uptake: a kinase-driven mechanism as a result of the
    initial Cr-enriched environment, which is more related to the anterograde trafficking
    via endolysosome-specific phosphoinositide compounds (1); and a ubiquitin-driven
    mechanism that controls the excessive Cr uptake, which is more related to the
    retrograde trafficking via clathrin-dependent and clathrin-independent processes
    (2). Interlinking protein filaments of the cytoskeleton are represented with lighter-colored
    lines in the background. See the sections of the manuscript for rationale, citations,
    and more abbreviations. Dashed arrows represent multiple steps. AMPK: AMP-activated
    protein kinase; CK: creatine kinase; GASP-1: growth and differentiation factor
    (GDF)-associated serum protein-1; GDP: guanosine diphosphate; GH: growth hormone;
    GLUT: glucose transporter; GRB2: growth factor receptor-bound protein 2; GTP:
    guanosine triphosphate; IGF-1: insulin-like growth factor-1; LST8: target of rapamycin
    complex subunit LST8; MRFs: myogenic regulatory factors; Nedd4-2: E3 ubiquitin–protein
    ligase NEDD4-like; OSR1: oxidative-stress-responsive kinase 1; PDK1: phosphoinositide-dependent
    kinase-1; PFK: phosphofructokinase; PI(3,5)P2: phosphatidylinositol 3,5-bisphosphate;
    PIKfyve: 1-phosphatidylinositol 3-phosphate 5-kinase; PI3K: phosphoinositide 3-kinase;
    PIP3: phosphatidylinositol (3,4,5)-trisphosphate; PK: pyruvate kinase; SGK: serum-
    and glucocorticoid-regulated kinase; SPAK: SPS1-related proline–alanine-rich kinase;
    RACK1: receptor for activated C kinase 1; RTK: receptor tyrosine kinases; TFs:
    transcription factors. Source: created by the authors (D.A.B.) with BioRender—https://biorender.com/
    (accessed on 10 May 2021).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Igf1
  - Gh
  - ras
  - Hras
  - Kras
  - Rem1
  - Crk
  - Grap2
  - Syp
  - Mapk14
  - Ahsa1
  - Nfya
  - Sp1
  - Zhx2
  - Mdk
  - Grb2
  - Fos
  - Zmiz1
  - Myc
  - Nol3
  - Max
  - Pik3r1
  - Sema6a
  - S1pr1
  - Mbtps1
  - Ckm
  - Fhl2
  - Ezh2
  - Suz12
  - Ppp1r1b
  - Ppp1cc
  - Npy4r
  - Pdk1
  - Pdpk1
  - Akt1
  - Akt2
  - Mtor
  - Rptor
  - Eif4ebp1
  - Calr
  - Slc6a8
  - Pfdn4
  - Oog1
  - ci
  - Sgk1
  - Sfn
  - Nedd4l
  - Pikfyve
  - Gprasp1
  - Wfikkn2
  - Ptk2
  - Slc1a3
  - Serpina1c
  - pk
  - Nacc1
  - Jak3
  - Osr1
  - Oxsr1
  - Stk39
  - Txnip
  - Rack1
  - Sag
  - Mapk8
  - IGF1
  - MTG1
  - OPN1MW
  - OPN1MW2
  - OPN1MW3
  - KRAS
  - HRAS
  - NRAS
  - CRK
  - MAPK14
  - MAPK1
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - MAPK11
  - MAPK12
  - MAPK13
  - NFYA
  - SP1
  - PSG1
  - DAND5
  - ZHX2
  - ARAF
  - BRAF
  - RAF1
  - MAP2K7
  - MAP2K1
  - MAP2K2
  - MAPK3
  - MAPK8
  - MAPK9
  - MAPK10
  - GRB2
  - XYLT2
  - SOS1
  - SOS2
  - FOS
  - FOSB
  - FOSL1
  - FOSL2
  - ZMIZ1
  - MYC
  - MAX
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - SEMA6A
  - MBTPS1
  - CKM
  - FHL2
  - EZH2
  - SUZ12
  - PPP1R1B
  - PPA1
  - NPY4R
  - NPY4R2
  - CDKL1
  - CDKL2
  - CDKL3
  - CDKL4
  - CDKL5
  - CDK3
  - CDK4
  - CDK5
  - CDK6
  - CDK7
  - CDK8
  - CDK9
  - CDK10
  - CDK11A
  - CDK11B
  - CDK12
  - CDK13
  - CDK14
  - CDK15
  - CDK16
  - CDK17
  - CDK18
  - CDK19
  - CDK20
  - CDK1
  - CDK2
  - JAK1
  - JAK2
  - JAK3
  - TYK2
  - PDK1
  - PDPK1
  - AKT1
  - AKT2
  - AKT3
  - PTK2B
  - MTOR
  - MLST8
  - RPTOR
  - EIF4EBP1
  - CALCR
  - CALR
  - SLC6A8
  - C1QA
  - C1QB
  - C1QC
  - C1R
  - C1S
  - SGK1
  - YWHAQ
  - NEDD4L
  - PIKFYVE
  - GPRASP1
  - WFIKKN2
  - PTK2
  - SLC2A1
  - PI3
  - PFKL
  - PFKM
  - PFKP
  - SCN1A
  - NACC1
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - PRKAA1
  - PRKAA2
  - PRKAB1
  - PRKAB2
  - PRKAG1
  - PRKAG2
  - PRKAG3
  - OSR1
  - OXSR1
  - OSER1
  - STK39
  - TXNIP
  - RACK1
  - Pck1
  - Nfyb
  - Max-ps1
  - Pik3cg
  - Gls
  - Wfdc15b
  - Prkcg
  - Prkaa2
  - Oser1
  - igf1.L
  - igf1.S
  - mapk14.S
  - grap2.L
  - grap2.S
  - nfya.S
  - sp1.L
  - raf1.S
  - mapk1.L
  - mapk1.S
  - grb2.L
  - grb2.S
  - fos.S
  - myc.S
  - myc.L
  - max.S
  - max.L
  - pik3ca.L
  - pik3cg.L
  - pik3cg.S
  - pik3cb.L
  - pik3ca.S
  - pik3cb.S
  - ckm.L
  - ckm.S
  - ezh2.L
  - suz12
  - suz12.S
  - ppp1r1b.S
  - npy4r.L
  - npy4r.S
  - pdk1.L
  - prickle1.L
  - akt1.S
  - akt1.L
  - ptk2b.L
  - ptk2b.S
  - rptor.L
  - mlst8.L
  - calr.S
  - calr.L
  - sgk1.L
  - sgk1.S
  - nedd4l.S
  - nedd4l.L
  - mstn.1.L
  - mstn.1.S
  - ptk2.L
  - ptk2.S
  - pfkp.L
  - pfkm.S
  - pfkp.S
  - pfkm.L
  - scn1a.S
  - jak3.L
  - prkaa2.S
  - prkaa1.L
  - prkaa2.L
  - osr1.L
  - oxsr1.L
  - stk39.L
  - stk39.S
  - txnip.L
  - arrb2.L
  - arrb2.S
  - arrb1.S
  - arrb1.L
  - mapk8.L
  - mapk8.S
---
