---
figid: PMC8921697__fmicb-13-813480-g002
figtitle: Metatranscriptomic Analyses Unravel Dynamic Changes in the Microbial and
  Metabolic Transcriptional Profiles in Artisanal Austrian Hard-Cheeses During Ripening
organisms:
- Streptococcus thermophilus
- Brevibacterium linens
- Staphylococcus equorum
- Corynebacterium variabile
- Brevibacterium aurantiacum
- Micrococcales incertae sedis
- Corynebacteriales incertae sedis
- Corynebacterium variabile DSM 44702
- Saccharomyces cerevisiae
- Brassica oleracea
- Allium sativum
- Debaryomyces hansenii
- Geotrichum candidum
- Bos taurus
- metagenome
- Erebia epipsodea
- Homo sapiens
organisms_ner:
- Staphylococcus equorum
- Streptococcus thermophilus
- Brevibacterium aurantiacum
- Corynebacterium variabile
- Bos taurus
- Homo sapiens
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC8921697
filename: fmicb-13-813480-g002.jpg
figlink: /pmc/articles/PMC8921697/figure/fig2/
number: F2
caption: Metabolism of residual lactose, lactate and citrate. The different compounds
  that are generated in each metabolic pathway are showed and those harboring important
  organoleptic properties for cheese are colored in orange. The different metabolic
  pathways can be followed according to the corresponding arrows. Between every pair
  of compounds, the enzyme that catalyzes each reaction is listed (gene name and EC
  number). Below each enzyme are six squares showing the TPM values for each of the
  six different samples. The TPM color scale appears in the legend in the upper-right
  part of the figure. Above the TPM values, the ripening time is highlighted if significant
  differences occur in the transcription of that gene in that ripening time (p < 0.05).
  The horizontal bar-plot below the TPM values correspond to the percentage of transcripts
  assigned to the different taxa.
papertitle: Metatranscriptomic Analyses Unravel Dynamic Changes in the Microbial and
  Metabolic Transcriptional Profiles in Artisanal Austrian Hard-Cheeses During Ripening.
reftext: Narciso Martín Quijada, et al. Front Microbiol. 2022;13:813480.
year: '2022'
doi: 10.3389/fmicb.2022.813480
journal_title: Frontiers in Microbiology
journal_nlm_ta: Front Microbiol
publisher_name: Frontiers Media S.A.
keywords: metatranscriptomics | cheese ripening | microbial dynamics | metabolic pathways
  | differential gene transcription | organoleptic compounds
automl_pathway: 0.9433906
figid_alias: PMC8921697__F2
figtype: Figure
redirect_from: /figures/PMC8921697__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8921697__fmicb-13-813480-g002.html
  '@type': Dataset
  description: Metabolism of residual lactose, lactate and citrate. The different
    compounds that are generated in each metabolic pathway are showed and those harboring
    important organoleptic properties for cheese are colored in orange. The different
    metabolic pathways can be followed according to the corresponding arrows. Between
    every pair of compounds, the enzyme that catalyzes each reaction is listed (gene
    name and EC number). Below each enzyme are six squares showing the TPM values
    for each of the six different samples. The TPM color scale appears in the legend
    in the upper-right part of the figure. Above the TPM values, the ripening time
    is highlighted if significant differences occur in the transcription of that gene
    in that ripening time (p < 0.05). The horizontal bar-plot below the TPM values
    correspond to the percentage of transcripts assigned to the different taxa.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - galT
  - galE
  - galU
  - larA
  - pta
  - budA
  - adhE
  - galK
  - pgm
  - GALT
  - GALE
  - PER
  - GALK1
  - B3GLCT
  - VCAN
  - THADA
  - IDH1
  - IDH2
  - F11
  - PTCRA
  - PER1
  - lacZ
  - lacA
  - galF
  - dsbA
  - gltA
  - ackA
---
