---
figid: PMC8877110__metabolites-12-00107-g005
figtitle: Bioinformatics Modelling and Metabolic Engineering of the Branched Chain
  Amino Acid Pathway for Specific Production of Mycosubtilin Isoforms in Bacillus
  subtilis
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC8877110
filename: metabolites-12-00107-g005.jpg
figlink: /pmc/articles/PMC8877110/figure/metabolites-12-00107-f005/
number: F5
caption: 'New improved model of branched chain amino acid metabolic pathway developed
  for the overproduction of mycosubtilin anteiso-C17. Constraint was applied on the
  overproduction of anteiso-branched chain fatty acid (anteiso-C15 + anteiso-C17 =
  ↑) and change predictions were visualized. Modelling language and semantics were
  first described by Coutte et al. (2015) [], and improvements for this work are described
  in the paragraph 4.4 of the Materials and Methods section. This figure semantic
  description was extracted from Coutte et al. (2015) [] for better understanding
  of this figure. Reaction networks in our modeling language are represented as graphs
  similar to Petri nets. The concrete syntax of our reaction networks is based on
  XML, from which the graphs are computed. The XML representation is also the input
  for the prediction algorithm. These graphs contain two kinds of nodes: round nodes
  for representing its species and boxed nodes for representing its reactions. More
  precisely, any species S is represented by a round node  and any reaction with name
  r by a boxed node . Solid edges either link a reactant to its reaction  or a reaction
  to one of its products . There are three kinds of dashed edges, which start at the
  three kinds of modifiers. An accelerator edge links an accelerator to a reaction
  , an activator edge links an activator to a reaction , and an inhibitor edge links
  an inhibitor to a reaction . An input edge  points from the context to an inflow
  species S, while an outflow edge  points from an outflow species S to the context.
  For convenience, we introduced the last kind of edges  as a shortcut for a product
  that is degraded by a hidden reaction, i.e., as a shortcut for . Species nodes with
  three different colors were used, which indicate their biological roles. Yellow
  indicates metabolites (such as ) and blue indicates proteins (such as ). There is
  a third color for “artificial species” that serves to modulate regulation, such
  as the promoter of the ilv-leu operon . Reactions that are potential candidates
  for knockouts or overexpression will be annotated in orange. Dark orange indicates
  candidates that were selected by our knockout prediction, while light orange indicates
  candidates that were not. Genes knockout predictions are represented by ⇓ and gene
  overexpression by ↑.'
papertitle: Bioinformatics Modelling and Metabolic Engineering of the Branched Chain
  Amino Acid Pathway for Specific Production of Mycosubtilin Isoforms in Bacillus
  subtilis.
reftext: Jean-Sébastien Guez, et al. Metabolites. 2022 Feb;12(2):107.
year: '2022'
doi: 10.3390/metabo12020107
journal_title: Metabolites
journal_nlm_ta: Metabolites
publisher_name: MDPI
keywords: metabolic engineering | bioinformatic modelling | lipopeptides | mycosubtilin
  | antifungal | fatty acids | branched chain amino acids | Bacillus subtilis
automl_pathway: 0.8714457
figid_alias: PMC8877110__F5
figtype: Figure
redirect_from: /figures/PMC8877110__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8877110__metabolites-12-00107-g005.html
  '@type': Dataset
  description: 'New improved model of branched chain amino acid metabolic pathway
    developed for the overproduction of mycosubtilin anteiso-C17. Constraint was applied
    on the overproduction of anteiso-branched chain fatty acid (anteiso-C15 + anteiso-C17
    = ↑) and change predictions were visualized. Modelling language and semantics
    were first described by Coutte et al. (2015) [], and improvements for this work
    are described in the paragraph 4.4 of the Materials and Methods section. This
    figure semantic description was extracted from Coutte et al. (2015) [] for better
    understanding of this figure. Reaction networks in our modeling language are represented
    as graphs similar to Petri nets. The concrete syntax of our reaction networks
    is based on XML, from which the graphs are computed. The XML representation is
    also the input for the prediction algorithm. These graphs contain two kinds of
    nodes: round nodes for representing its species and boxed nodes for representing
    its reactions. More precisely, any species S is represented by a round node  and
    any reaction with name r by a boxed node . Solid edges either link a reactant
    to its reaction  or a reaction to one of its products . There are three kinds
    of dashed edges, which start at the three kinds of modifiers. An accelerator edge
    links an accelerator to a reaction , an activator edge links an activator to a
    reaction , and an inhibitor edge links an inhibitor to a reaction . An input edge  points
    from the context to an inflow species S, while an outflow edge  points from an
    outflow species S to the context. For convenience, we introduced the last kind
    of edges  as a shortcut for a product that is degraded by a hidden reaction, i.e.,
    as a shortcut for . Species nodes with three different colors were used, which
    indicate their biological roles. Yellow indicates metabolites (such as ) and blue
    indicates proteins (such as ). There is a third color for “artificial species”
    that serves to modulate regulation, such as the promoter of the ilv-leu operon
    . Reactions that are potential candidates for knockouts or overexpression will
    be annotated in orange. Dark orange indicates candidates that were selected by
    our knockout prediction, while light orange indicates candidates that were not.
    Genes knockout predictions are represented by ⇓ and gene overexpression by ↑.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ilvA
  - ilvD
  - leuA
  - ybgE
---
