---
figid: PMC8301375__biomedicines-09-00791-g001
figtitle: XBP1s in Transcriptional Regulation of Target Genes
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC8301375
filename: biomedicines-09-00791-g001.jpg
figlink: /pmc/articles/PMC8301375/figure/biomedicines-09-00791-f001/
number: F1
caption: 'Signaling pathways of UPR. Signaling pathways of UPR are mediated by three
  ER-resident proteins: IRE1, PERK, and ATF6. In the absence of ER stress, BiP binds
  to the ER luminal domain of the sensor proteins and inhibits their activation. However,
  ER stress induces the dissociation of BiP from the sensors, which activates UPR
  transducers. (A) IRE1 pathway: IRE1 has serine/threonine kinase and RNase domains
  in the cytoplasmic region. Upon ER stress, IRE1 is activated via oligomerization
  and autophosphorylation, leading to increased RNase activity. Activated IRE1 recognizes
  the stem-loop structure of Xbp1 mRNA and induces unconventional splicing by cleaving
  26 intronic nucleotides. The spliced Xbp1 mRNA is translated by frameshift into
  an active transcription factor, XBP1s. Then, XBP1s upregulates the expression of
  UPR target genes, including ER chaperones, ERAD components, and lipid biosynthetic
  enzymes. Furthermore, IRE1 recognizes various mRNAs besides Xbp1 mRNA as substrates
  under ER stress and induces the degradation of these mRNAs, a process known as regulated
  IRE-dependent decay (RIDD). RIDD is a mechanism that resolves ER stress by degrading
  mRNAs encoding ER-targeted proteins, thereby reducing protein loads into ER. (B)
  PERK pathway: PERK has a serine/threonine kinase domain in its cytoplasmic region.
  ER stress induces PERK activation via oligomerization and autophosphorylation in
  the kinase domain. Activated PERK then phosphorylates the serine 51 residue of eIF2α,
  resulting in alleviation of ER stress by attenuating translation. By contrast, phosphorylated
  eIF2α selectively promotes the translation of ATF4, which activates the transcription
  of CHOP and GADD34. After ER stress resolution, GADD34 interacts with PP1 to induce
  eIF2α dephosphorylation, restoring protein translation. However, if ER stress is
  not resolved, CHOP induces apoptosis. (C) ATF6 pathway: ATF6 has an N-terminal b-ZIP
  domain in the cytoplasmic region. ER stress induces ATF6 translocation from ER to
  the Golgi apparatus, where ATF6 is cleaved by S1P and S2P. This proteolytic cleavage
  produces the N-terminal region of ATF6, referred to as ATF6(N). ATF6(N) functions
  as an active transcription factor and upregulates target genes encoding ER chaperones,
  ERAD components, and XBP1. IRE1, inositol-requiring enzyme 1; PERK, protein kinase
  R-like ER kinase; ATF6, activating transcription factor 6; RNase, endoribonuclease;
  XBP1, X-box binding protein 1; ERAD, ER-associated protein degradation; eIF2α, eukaryotic
  translation initiation factor 2α; ATF4, activating transcription factor 4; CHOP,
  C/EBP homologous protein; GADD34, growth arrest and DNA damage-inducible protein
  34; PP1, protein phosphatase 1; S1P, site-1 protease.'
papertitle: Roles of XBP1s in Transcriptional Regulation of Target Genes.
reftext: Sung-Min Park, et al. Biomedicines. 2021 Jul;9(7):791.
year: '2021'
doi: 10.3390/biomedicines9070791
journal_title: Biomedicines
journal_nlm_ta: Biomedicines
publisher_name: MDPI
keywords: XBP1s | IRE1 | ATF6 | ER stress | unfolded protein response | UPR | RIDD
automl_pathway: 0.9086275
figid_alias: PMC8301375__F1
figtype: Figure
redirect_from: /figures/PMC8301375__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8301375__biomedicines-09-00791-g001.html
  '@type': Dataset
  description: 'Signaling pathways of UPR. Signaling pathways of UPR are mediated
    by three ER-resident proteins: IRE1, PERK, and ATF6. In the absence of ER stress,
    BiP binds to the ER luminal domain of the sensor proteins and inhibits their activation.
    However, ER stress induces the dissociation of BiP from the sensors, which activates
    UPR transducers. (A) IRE1 pathway: IRE1 has serine/threonine kinase and RNase
    domains in the cytoplasmic region. Upon ER stress, IRE1 is activated via oligomerization
    and autophosphorylation, leading to increased RNase activity. Activated IRE1 recognizes
    the stem-loop structure of Xbp1 mRNA and induces unconventional splicing by cleaving
    26 intronic nucleotides. The spliced Xbp1 mRNA is translated by frameshift into
    an active transcription factor, XBP1s. Then, XBP1s upregulates the expression
    of UPR target genes, including ER chaperones, ERAD components, and lipid biosynthetic
    enzymes. Furthermore, IRE1 recognizes various mRNAs besides Xbp1 mRNA as substrates
    under ER stress and induces the degradation of these mRNAs, a process known as
    regulated IRE-dependent decay (RIDD). RIDD is a mechanism that resolves ER stress
    by degrading mRNAs encoding ER-targeted proteins, thereby reducing protein loads
    into ER. (B) PERK pathway: PERK has a serine/threonine kinase domain in its cytoplasmic
    region. ER stress induces PERK activation via oligomerization and autophosphorylation
    in the kinase domain. Activated PERK then phosphorylates the serine 51 residue
    of eIF2α, resulting in alleviation of ER stress by attenuating translation. By
    contrast, phosphorylated eIF2α selectively promotes the translation of ATF4, which
    activates the transcription of CHOP and GADD34. After ER stress resolution, GADD34
    interacts with PP1 to induce eIF2α dephosphorylation, restoring protein translation.
    However, if ER stress is not resolved, CHOP induces apoptosis. (C) ATF6 pathway:
    ATF6 has an N-terminal b-ZIP domain in the cytoplasmic region. ER stress induces
    ATF6 translocation from ER to the Golgi apparatus, where ATF6 is cleaved by S1P
    and S2P. This proteolytic cleavage produces the N-terminal region of ATF6, referred
    to as ATF6(N). ATF6(N) functions as an active transcription factor and upregulates
    target genes encoding ER chaperones, ERAD components, and XBP1. IRE1, inositol-requiring
    enzyme 1; PERK, protein kinase R-like ER kinase; ATF6, activating transcription
    factor 6; RNase, endoribonuclease; XBP1, X-box binding protein 1; ERAD, ER-associated
    protein degradation; eIF2α, eukaryotic translation initiation factor 2α; ATF4,
    activating transcription factor 4; CHOP, C/EBP homologous protein; GADD34, growth
    arrest and DNA damage-inducible protein 34; PP1, protein phosphatase 1; S1P, site-1
    protease.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - bip
  - Hsc70-3
  - Ire1
  - crc
  - Xbp1
  - PEK
  - rl
  - eIF2alpha
  - eIF2beta
  - eIF2A
  - PPP1R15
  - Atf6
  - S2P
  - S1P
  - GDF10
  - HSPA5
  - ERN1
  - XBP1
  - XBP1P1
  - EIF2AK3
  - ELF2
  - EIF2S1
  - EIF2S3
  - EIF2S2
  - ATF4
  - EIF2A
  - PPP1R15A
  - DDIT3
  - ATF6
  - MBTPS2
  - MBTPS1
---
