---
figid: PMC8458546__41598_2021_97970_Fig3_HTML
figtitle: Proliferative diabetic retinopathy transcriptomes reveal angiogenesis, anti-angiogenic
  therapy escape mechanisms, fibrosis and lymphatic involvement
organisms:
- NA
pmcid: PMC8458546
filename: 41598_2021_97970_Fig3_HTML.jpg
figlink: /pmc/articles/PMC8458546/figure/Fig3/
number: F3
caption: VEGFA-ligand-dependent mechanisms are involved in proliferative diabetic
  retinopathy. (a) Network of DEGs affected by VEGFA (Ingenuity Pathway Analysis (IPA)),
  based on which VEGFA was predicted as an activated upstream regulator. (b) IPA canonical
  pathway VEGF family ligand receptor interactions. (c) IPA canonical pathway Enos
  signalling. (d) Heatmap of log2-transformed normalized read counts of DEGs in VEGF
  family ligand receptor interactions and Enos signalling IPA canonical pathways.
  Genes in VEGF family ligand receptor interactions, Enos signalling or both pathways
  are indicated in blue, red, and purple, respectively. Arrow indicates the PDR patient
  who had received no preoperative retinal laser photocoagulation (Patient 13). (*)
  Yellow arrows in (b) and (d) indicate that findings are inconsistent with the state
  of downstream molecule (IPA). The network and pathways in (a), (b) and (c) were
  generated through the use of IPA (QIAGEN Inc., https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis).
papertitle: Proliferative diabetic retinopathy transcriptomes reveal angiogenesis,
  anti-angiogenic therapy escape mechanisms, fibrosis and lymphatic involvement.
reftext: Ani Korhonen, et al. Sci Rep. 2021;11:18810.
year: '2021'
doi: 10.1038/s41598-021-97970-5
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Retinal diseases | RNA sequencing | Cellular signalling networks | Angiogenesis
  | Transcriptomics
automl_pathway: 0.7935728
figid_alias: PMC8458546__F3
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC8458546__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8458546__41598_2021_97970_Fig3_HTML.html
  '@type': Dataset
  description: VEGFA-ligand-dependent mechanisms are involved in proliferative diabetic
    retinopathy. (a) Network of DEGs affected by VEGFA (Ingenuity Pathway Analysis
    (IPA)), based on which VEGFA was predicted as an activated upstream regulator.
    (b) IPA canonical pathway VEGF family ligand receptor interactions. (c) IPA canonical
    pathway Enos signalling. (d) Heatmap of log2-transformed normalized read counts
    of DEGs in VEGF family ligand receptor interactions and Enos signalling IPA canonical
    pathways. Genes in VEGF family ligand receptor interactions, Enos signalling or
    both pathways are indicated in blue, red, and purple, respectively. Arrow indicates
    the PDR patient who had received no preoperative retinal laser photocoagulation
    (Patient 13). (*) Yellow arrows in (b) and (d) indicate that findings are inconsistent
    with the state of downstream molecule (IPA). The network and pathways in (a),
    (b) and (c) were generated through the use of IPA (QIAGEN Inc., https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ANGPT1
  - NRP1
  - NELL1
  - NR2F2
  - VEGFA
  - ANGPT2
  - ANGPT4
  - AREG
  - BDNF
  - CSF1
  - EFNA1
  - EFNA2
  - EFNA3
  - EFNA4
  - EFNA5
  - EGF
  - EREG
  - FGF1
  - FGF10
  - FGF16
  - FGF17
  - FGF18
  - FGF19
  - FGF2
  - FGF20
  - FGF21
  - FGF22
  - FGF23
  - FGF3
  - FGF4
  - FGF5
  - FGF6
  - FGF7
  - FGF8
  - FGF9
  - FLT3LG
  - HGF
  - IGF1
  - IGF2
  - INS
  - KITLG
  - NGF
  - NTF3
  - NTF4
  - PDGFA
  - PDGFB
  - PDGFC
  - PDGFD
  - PGF
  - TGFA
  - VEGFB
  - VEGFC
  - VEGFD
  - OMA1
  - NOSIP
  - NOS3
  - NANOS3
  - NOTCH1
  - CAV1
  - AQP1
  - AQP10
  - AQP11
  - AQP12A
  - AQP12B
  - AQP2
  - AQP3
  - AQP4
  - AQP5
  - AQP6
  - AQP7
  - AQP8
  - AQP9
  - MIP
  - DNM2
  - HSPA1A
  - HSPA4
  - CASP3
  - CASP8
  - NOS1
  - NOS2
  - NOSTRIN
  - ETS1
  - CALM2
  - CALM3
  - PICALM
  - SNAP91
  - CALM1
  - CRAT
  - SLC7A1
  - TRPV6
  - FLT1
  - ITGA1
  - MMP9
  - ITGAV
  - KDR
  - FGFR3
  - SGO1
  - PRRT2
  - PRKCA
  - PRKCB
  - PRKCD
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - PRKCQ
  - PRKCZ
  - PRKD3
  - FLT4
  - CNGA1
  - CNGA2
  - CNGA3
  - CNGA4
  - CNGB1
  - CNGB3
  - HCN1
  - HCN2
  - HCN3
  - HCN4
  - MTG1
  - NAP1L1
  - OPTN
  - PRKG1
  - PLA2G4A
  - KAT5
  - LONP1
  - PROK1
  - FZD4
  - NMUR1
  - LPAR3
  - LGR6
  - MRGPRX3
  - MRGPRX4
  - GPR151
  - OXER1
  - GPRC6A
  - MRGPRX1
  - VN1R17P
  - GPR166P
  - GRM1
  - GRM2
  - GRM3
  - GRM4
  - GRM5
  - GRM6
  - GRM7
  - GRM8
  - CASR
  - GABBR1
  - GABBR2
  - GPR156
  - GPR158
  - GPR179
  - GPRC5A
  - GPRC5B
  - GPRC5C
  - GPRC5D
  - BDKRB1
  - BDKRB2
  - ESR1
  - ESR2
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - SHC1
  - SHC2
  - SHC3
  - SHC4
  - GRB2
  - XYLT2
  - SOS1
  - SOS2
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - GNAS
  - GNAL
  - GNAQ
  - GNA11
  - GNA14
  - GNA15
  - GNA12
  - GNA13
  - RAF1
  - MAP2K7
  - MAP2K1
  - MAP2K2
  - EPHB2
  - MAPK1
  - MAPK3
  - KNG1
  - BCKDK
  - GAST
  - GALNS
  - PAGR1
  - ENO4
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - ATP8A2
  - AKT1
  - AKT2
  - AKT3
  - CECR
  - PRKAA1
  - PRKAA2
  - PRKAB1
  - PRKAB2
  - PRKAG1
  - PRKAG2
  - PRKAG3
  - GUCY1A1
  - LPAR6
  - RALB
  - GUCY1A2
  - ADCY4
  - ADCY1
  - LPAR4
  - PLCG2
  - CHRNE
  - ADCY7
  - RAP2A
  - MINDY4
  - RASD2
  - PLA2G2A
  - PVR
  - POLA1
---
