---
figid: PMC8484535__fpls-12-726762-g001
figtitle: 'Small RNAs: The Essential Regulators in Plant Thermotolerance'
organisms:
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Glycine max
- Solanum lycopersicum
- Triticum aestivum
- Cucumis sativus
- Magnoliopsida
- Helianthus annuus
- Medicago sativa
- eudicotyledons
- Musa acuminata
- Gossypium hirsutum
- Capsicum annuum
- Manihot esculenta
- Brassica rapa
- Liliopsida
- Carthamus tinctorius
- Brassica rapa subsp. pekinensis
- Panicum virgatum
- Phytophthora sojae
- Lolium arundinaceum
- Saccharina japonica
- Brassica rapa subsp. oleifera
- Solanum habrochaites
- Homo sapiens
- transposons
organisms_ner:
- Arabidopsis thaliana
- Solanum lycopersicum
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8484535
filename: fpls-12-726762-g001.jpg
figlink: /pmc/articles/PMC8484535/figure/fig1/
number: F1
caption: Biogenesis and modes of action of microRNAs (miRNA) and small interfering
  RNAs (siRNA) in plants. (A) Overview of miRNA pathway. MIR gene is transcribed by
  polymerase II (Pol II) to generate a long single-stranded primary miRNA (pri-miRNA)
  and form a stem-loop or hairpin secondary structure. The pri-miRNA is firstly cleaved
  into stem-loop precursor miRNA (pre-miRNA) and then processed into miRNA-miRNA*
  duplex. In this complex process, DCL1 forms a D-bodies with HYPONASTIC LEAVES1 (HYL1)
  and SE to ensure the accurate and efficient splicing of pre-mRNA. Other cofactors
  are involved for proper processing, such as cap-binding complex (CBC) and Dawdle
  (DDL). The miRNA-miRNA* duplex is then methylated by HUA ENHANCER1 (HEN1) to protect
  the 3' ends from uridylation. The guide strand (miRNA) is incorporated into Argonaute-1
  (AGO1) with the aid of TRANSPORIN 1 (TRN1) and then exported from the nucleus to
  the cytoplasm via HASTY (HST). In the cytoplasm, mature miRNA directs post-transcriptional
  gene silencing (PTGS) via transcript cleavage or translational repression. The green
  balls at the ends of miRNA-miRNA* duplex represent 2'-O-methy groups. (B) Overview
  of the heterochromatic siRNA (hc-siRNA) pathway. hc-siRNAs are derived from transposable
  and repetitive elements. The biogenesis of hc-siRNA requires Polymerase IV (Pol
  IV) to transcribe a single-stranded RNA (ssRNA) and synthesis into a double-stranded
  RNA (dsRNA) by RDR2. The dsRNA is then processed into 24-nt siRNA by DCL3. After
  the siRNA is incorporated into AGO4, Pol V transcribes transcripts that base pairing
  with siRNA and the siRNA-AGO4 complex is recruited. AGO4, RNA-DIRECTED DNA METHYLATION
  1 (RDM1) and DOMAINS REARRANGED METHYLTRANSFERASE 2 (DRM2) trigger DNA methylation
  at the transcriptional gene silencing (TGS) level. (C) Overview of the phased siRNA
  (phasiRNA) pathway. phasiRNAs originated from phasiRNA-generating (PHAS) or trans-acting
  siRNA (TAS) loci are transcribed by Pol II and their precursors are cleaved via
  the miRNA “one-hit” system (AGO1-miRNA complex) or the “two-hit” system (AGO7-miRNA
  complex). SUPPRESSOR OF GENE SILENCING3 (SGS3) stabilizes the 5'- or 3'-cleaved
  fragments to protect them from AGO-mediated slicing. Then the primary siRNAs are
  converted into dsRNAs by RDR6 and further processed into 21- or 22-nt siRNAs by
  DCL2 and DCL4 or 24-nt siRNAs by DCL3. For further processing, some of them will
  be recruited by the AGO1 complex and some of them may be incorporated into other
  uncertain AGOs to trigger the cleavage of target mRNA, translational repression
  or DNA methylation.
papertitle: 'Small RNAs: The Essential Regulators in Plant Thermotolerance.'
reftext: Zhi-Fang Zuo, et al. Front Plant Sci. 2021;12:726762.
year: '2021'
doi: 10.3389/fpls.2021.726762
journal_title: Frontiers in Plant Science
journal_nlm_ta: Front Plant Sci
publisher_name: Frontiers Media S.A.
keywords: microRNA | small interfering RNA | heat stress response | regulatory mechanism
  | thermotolerance
automl_pathway: 0.8326039
figid_alias: PMC8484535__F1
figtype: Figure
redirect_from: /figures/PMC8484535__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8484535__fpls-12-726762-g001.html
  '@type': Dataset
  description: Biogenesis and modes of action of microRNAs (miRNA) and small interfering
    RNAs (siRNA) in plants. (A) Overview of miRNA pathway. MIR gene is transcribed
    by polymerase II (Pol II) to generate a long single-stranded primary miRNA (pri-miRNA)
    and form a stem-loop or hairpin secondary structure. The pri-miRNA is firstly
    cleaved into stem-loop precursor miRNA (pre-miRNA) and then processed into miRNA-miRNA*
    duplex. In this complex process, DCL1 forms a D-bodies with HYPONASTIC LEAVES1
    (HYL1) and SE to ensure the accurate and efficient splicing of pre-mRNA. Other
    cofactors are involved for proper processing, such as cap-binding complex (CBC)
    and Dawdle (DDL). The miRNA-miRNA* duplex is then methylated by HUA ENHANCER1
    (HEN1) to protect the 3' ends from uridylation. The guide strand (miRNA) is incorporated
    into Argonaute-1 (AGO1) with the aid of TRANSPORIN 1 (TRN1) and then exported
    from the nucleus to the cytoplasm via HASTY (HST). In the cytoplasm, mature miRNA
    directs post-transcriptional gene silencing (PTGS) via transcript cleavage or
    translational repression. The green balls at the ends of miRNA-miRNA* duplex represent
    2'-O-methy groups. (B) Overview of the heterochromatic siRNA (hc-siRNA) pathway.
    hc-siRNAs are derived from transposable and repetitive elements. The biogenesis
    of hc-siRNA requires Polymerase IV (Pol IV) to transcribe a single-stranded RNA
    (ssRNA) and synthesis into a double-stranded RNA (dsRNA) by RDR2. The dsRNA is
    then processed into 24-nt siRNA by DCL3. After the siRNA is incorporated into
    AGO4, Pol V transcribes transcripts that base pairing with siRNA and the siRNA-AGO4
    complex is recruited. AGO4, RNA-DIRECTED DNA METHYLATION 1 (RDM1) and DOMAINS
    REARRANGED METHYLTRANSFERASE 2 (DRM2) trigger DNA methylation at the transcriptional
    gene silencing (TGS) level. (C) Overview of the phased siRNA (phasiRNA) pathway.
    phasiRNAs originated from phasiRNA-generating (PHAS) or trans-acting siRNA (TAS)
    loci are transcribed by Pol II and their precursors are cleaved via the miRNA
    “one-hit” system (AGO1-miRNA complex) or the “two-hit” system (AGO7-miRNA complex).
    SUPPRESSOR OF GENE SILENCING3 (SGS3) stabilizes the 5'- or 3'-cleaved fragments
    to protect them from AGO-mediated slicing. Then the primary siRNAs are converted
    into dsRNAs by RDR6 and further processed into 21- or 22-nt siRNAs by DCL2 and
    DCL4 or 24-nt siRNAs by DCL3. For further processing, some of them will be recruited
    by the AGO1 complex and some of them may be incorporated into other uncertain
    AGOs to trigger the cleavage of target mRNA, translational repression or DNA methylation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - POL
  - DCL1
  - AGO1
  - HST
  - TRN1
  - HEN1
  - RDR2
  - DCL3
  - AGO4
  - DCL2
  - DCL4
  - AGO1B
  - ERVW-4
  - CTAA1
  - LNPEP
  - SERPINB6
  - HACD1
  - CAP1
  - SORBS1
  - BRD4
  - CAP2
  - APP
  - AAAS
  - AAA1
  - MATK
  - CD302
  - PTPN5
  - FGF4
  - SULT2A1
  - TRN-GTT2-7
  - TRN-GTT2-1
  - NHLH1
  - HENMT1
  - ORM2
  - FBXW7
  - GOT1
  - GOT2
  - THAS
  - E(spl)mgamma-HLH
  - Polr2A
  - Polr2C
  - Polr2B
  - Polr2I
  - Polr2F
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - CAP
  - SMC3
  - Cpn
  - Capa
  - capt
  - bel
  - SERCA
  - Cadps
  - tRNA:Lys-TTT-1-1
  - tRNA:Lys-TTT-2-4
  - tRNA:Lys-TTT-2-1
  - tRNA:Lys-TTT-2-2
  - tRNA:Lys-TTT-2-5
  - tRNA:Lys-TTT-2-3
  - hyl
  - step
  - AGO2
  - Tnpo
  - Hen1
  - ago
  - Got2
---
