---
figid: PMC8895709__fcimb-12-826018-g002
figtitle: 'Gene Expression Regulation in Airway Pathogens: Importance for Otitis Media'
organisms:
- Streptococcus pneumoniae
- Haemophilus influenzae
- Moraxella catarrhalis
- Staphylococcus aureus
- Streptococcus pneumoniae D39
- Moraxella catarrhalis BBH18
- Mus musculus
- Rattus norvegicus
- Chinchilla chinchilla
- Influenza A virus
- Respiratory syncytial virus
- Homo sapiens
- NA
pmcid: PMC8895709
filename: fcimb-12-826018-g002.jpg
figlink: /pmc/articles/PMC8895709/figure/f2/
number: F2
caption: Gene expression regulatory system in S. pneumoniae (Spn), and nontypeable
  H. influenzae (NTHi), and M. catarrhalis (Mcat). (A) Pneumococcal cell-cell communication
  systems. Spn colonizing the upper airway mucosa (i.e., at nasopharynx and middle
  ear) form biofilm and interact with either neighbouring pneumococcal or incoming
  clone of Spn via quorum sensing or cell-cell communication in response to surrounding
  stimuli (upper panel). Pheromone peptide signalling pathways and their regulatory
  effect in gene expression during Spn cell-cell communication are shown in the lower
  panel. Pheromone peptides such as double glycine peptides [GG peptide (i.e., CSP,
  BriC, VP1)] [as shown in pathway (a)], RRNPP-dependent peptides (i.e., SHR and Phr)
  [shown in (b)], and lanthionine peptides (i.e., pneumolancidin PldA1-A4) (shown
  in (c)) are secreted by pneumococci and autoinduced in response to stimuli. Depending
  on the signal sequence (SP) of the peptides, they are exported via ABC transporter
  or general secretory (SEC) pathway. The precursor peptide is proteolytically processed
  into active peptide either during transportation by peptidase domain of ABC transporter
  or membrane-associated proteases, or after secretion via unknown extracellular protease.
  At a sufficient level of the signalling peptide pheromone, they interact with their
  cognate cell receptors in recipient cells (peptide responder bacteria) such as sensor
  kinase (SK) of two-component system (TCS) [shown in pathway (a) and (c)]. In canonical
  TCS, the sensor protein is a histidine kinase (i.e., ComAB, BlpAB and PptAB) that
  detects exogenous signals, and subsequently, sends a phosphoryl group (yellow sphere)
  to the cognate response regulator protein (RR). The phosphorylation of the regulator
  protein results in transcriptional regulation. Alternatively, some peptides such
  as bacteriocin CibAB and lanthibiotics (pneumolandin) can activate recipient cell
  directly and induce bactericidal effect. As shown in pathway (b), for RRNPP-dependent
  peptides such as SHP (i.e., SHP144, SHP939 and RgtS) and Phr (i.e., PhrA), they
  are transported into responder cell via oligopeptide permease system (i.e., AmiACDEF).
  Once inside the recipient cell, SHPs interacts with their cognate binding partner,
  Rgg regulators (i.e., Rgg144, Rgg939, abd RgtR), resulting in DNA binding of Rgg
  and activation of transcription. On the other hand, Phr peptides interact with Tpr
  regulators that are initially bound to DNA and inhibit the expression of the target
  gene. Binding of Phr to Tpr results in the releasing of Tpr-mediated inhibition
  hence activates gene expression. Lastly, lanthionine-containing peptide such as
  pneumolandin (i.e., PldA1-A4) is processed and translationally modified by intracellular
  modification enzyme such as LanM before transportation via ABC transporter (i.e.,
  LanT) and detected by SK of responder cells (i.e., LanA) [shown in pathway (c)].
  This results in either phosphorylation of the response regulator for downstream
  activation of gene expression, or directly causing bactericidal effect. ABC transporter,
  ATP-binding cassette transporter; Phr, phosphatase regulator; Rgg, regulator gene
  of glycosyltransferase; RRNPP, Rap, Rgg, NprC, PlcR and PrgX; SHP, short hydrophobic
  peptide. (B) Schematic representation of gene regulatory mechanism in NTHi and Mcat.
  Phase variation [shown in (a)] is caused by random mutations in a variable number
  of simple sequence repeats (SSR) within the open reading frame of DNA-methyl transferases
  (i.e., ModA and ModM] that alters the gene expression of Mod. Thus, any changes
  in the variable number of the SSRs as a result of DNA mutation could cause frame
  shifts in the ORF of modA and potentially lead to premature translation termination
  and generation of non-functional truncated ModA which is in the “OFF” mode (shown
  in (b)]. On the other hand, the functional Mod (switched “ON”) [shown in (c)] methylates
  genomic DNA at specific sites that governs transcription in both ways, either by
  inhibition or activation. The inhibition or activation of transcription depends
  on the methylated DNA sequence area that potentially bears the recognition sites
  for regulatory molecules and enzymes. Moreover, the phase-variable number of SSRs
  [as shown in (d)] can also be found in the transcriptional promoter which, as a
  result, switches between the “ON” or “OFF” expression status of the target gene
  (i.e., hia). In addition, transcriptional regulators [shown in (e)] can be activators
  (Act) (i.e., OxyR, Fur) and/or repressors (Rep) (i.e., Fur) of transcription. The
  regulators are triggered by various stimuli. (f) Another type of transcriptional
  regulation is a TCS (i.e., FirRS, mesSR, narX/narL) which consists of sensor kinase
  (SK) and response regulator proteins (RR). The activated regulatory proteins can
  cause transcription activation and/or repression as described in (e). Finally [indicated
  in (g)], the genes are transcribed into RNA, followed by translation as protein.
  Alterations in gene expression are manifested in a bacterial phenotype. Symbols
  used in panel (A, B) are defined at the bottom of this figure.
papertitle: 'Gene Expression Regulation in Airway Pathogens: Importance for Otitis
  Media.'
reftext: Martina Janoušková, et al. Front Cell Infect Microbiol. 2022;12:826018.
year: '2022'
doi: 10.3389/fcimb.2022.826018
journal_title: Frontiers in Cellular and Infection Microbiology
journal_nlm_ta: Front Cell Infect Microbiol
publisher_name: Frontiers Media S.A.
keywords: gene expression regulation | Haemophilus influenzae | Moraxella catarrhalis
  | otitis media | Streptococcus pneumoniae
automl_pathway: 0.573153
figid_alias: PMC8895709__F2
figtype: Figure
organisms_ner:
- Mus musculus
redirect_from: /figures/PMC8895709__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8895709__fcimb-12-826018-g002.html
  '@type': Dataset
  description: Gene expression regulatory system in S. pneumoniae (Spn), and nontypeable
    H. influenzae (NTHi), and M. catarrhalis (Mcat). (A) Pneumococcal cell-cell communication
    systems. Spn colonizing the upper airway mucosa (i.e., at nasopharynx and middle
    ear) form biofilm and interact with either neighbouring pneumococcal or incoming
    clone of Spn via quorum sensing or cell-cell communication in response to surrounding
    stimuli (upper panel). Pheromone peptide signalling pathways and their regulatory
    effect in gene expression during Spn cell-cell communication are shown in the
    lower panel. Pheromone peptides such as double glycine peptides [GG peptide (i.e.,
    CSP, BriC, VP1)] [as shown in pathway (a)], RRNPP-dependent peptides (i.e., SHR
    and Phr) [shown in (b)], and lanthionine peptides (i.e., pneumolancidin PldA1-A4)
    (shown in (c)) are secreted by pneumococci and autoinduced in response to stimuli.
    Depending on the signal sequence (SP) of the peptides, they are exported via ABC
    transporter or general secretory (SEC) pathway. The precursor peptide is proteolytically
    processed into active peptide either during transportation by peptidase domain
    of ABC transporter or membrane-associated proteases, or after secretion via unknown
    extracellular protease. At a sufficient level of the signalling peptide pheromone,
    they interact with their cognate cell receptors in recipient cells (peptide responder
    bacteria) such as sensor kinase (SK) of two-component system (TCS) [shown in pathway
    (a) and (c)]. In canonical TCS, the sensor protein is a histidine kinase (i.e.,
    ComAB, BlpAB and PptAB) that detects exogenous signals, and subsequently, sends
    a phosphoryl group (yellow sphere) to the cognate response regulator protein (RR).
    The phosphorylation of the regulator protein results in transcriptional regulation.
    Alternatively, some peptides such as bacteriocin CibAB and lanthibiotics (pneumolandin)
    can activate recipient cell directly and induce bactericidal effect. As shown
    in pathway (b), for RRNPP-dependent peptides such as SHP (i.e., SHP144, SHP939
    and RgtS) and Phr (i.e., PhrA), they are transported into responder cell via oligopeptide
    permease system (i.e., AmiACDEF). Once inside the recipient cell, SHPs interacts
    with their cognate binding partner, Rgg regulators (i.e., Rgg144, Rgg939, abd
    RgtR), resulting in DNA binding of Rgg and activation of transcription. On the
    other hand, Phr peptides interact with Tpr regulators that are initially bound
    to DNA and inhibit the expression of the target gene. Binding of Phr to Tpr results
    in the releasing of Tpr-mediated inhibition hence activates gene expression. Lastly,
    lanthionine-containing peptide such as pneumolandin (i.e., PldA1-A4) is processed
    and translationally modified by intracellular modification enzyme such as LanM
    before transportation via ABC transporter (i.e., LanT) and detected by SK of responder
    cells (i.e., LanA) [shown in pathway (c)]. This results in either phosphorylation
    of the response regulator for downstream activation of gene expression, or directly
    causing bactericidal effect. ABC transporter, ATP-binding cassette transporter;
    Phr, phosphatase regulator; Rgg, regulator gene of glycosyltransferase; RRNPP,
    Rap, Rgg, NprC, PlcR and PrgX; SHP, short hydrophobic peptide. (B) Schematic representation
    of gene regulatory mechanism in NTHi and Mcat. Phase variation [shown in (a)]
    is caused by random mutations in a variable number of simple sequence repeats
    (SSR) within the open reading frame of DNA-methyl transferases (i.e., ModA and
    ModM] that alters the gene expression of Mod. Thus, any changes in the variable
    number of the SSRs as a result of DNA mutation could cause frame shifts in the
    ORF of modA and potentially lead to premature translation termination and generation
    of non-functional truncated ModA which is in the “OFF” mode (shown in (b)]. On
    the other hand, the functional Mod (switched “ON”) [shown in (c)] methylates genomic
    DNA at specific sites that governs transcription in both ways, either by inhibition
    or activation. The inhibition or activation of transcription depends on the methylated
    DNA sequence area that potentially bears the recognition sites for regulatory
    molecules and enzymes. Moreover, the phase-variable number of SSRs [as shown in
    (d)] can also be found in the transcriptional promoter which, as a result, switches
    between the “ON” or “OFF” expression status of the target gene (i.e., hia). In
    addition, transcriptional regulators [shown in (e)] can be activators (Act) (i.e.,
    OxyR, Fur) and/or repressors (Rep) (i.e., Fur) of transcription. The regulators
    are triggered by various stimuli. (f) Another type of transcriptional regulation
    is a TCS (i.e., FirRS, mesSR, narX/narL) which consists of sensor kinase (SK)
    and response regulator proteins (RR). The activated regulatory proteins can cause
    transcription activation and/or repression as described in (e). Finally [indicated
    in (g)], the genes are transcribed into RNA, followed by translation as protein.
    Alterations in gene expression are manifested in a bacterial phenotype. Symbols
    used in panel (A, B) are defined at the bottom of this figure.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Spn
  - Ppp1r9b
  - Wdtc1
  - Mdga2
  - Nr0b2
  - Phmr
  - Eefsec
  - Fam167b
  - Tpr
  - Grid2
  - act
  - Fhl5
  - Acot7
  - Nectin1
  - Atp6ap2
---
