---
figid: PMC8862337__12870_2022_3455_Fig2_HTML
figtitle: Insights into glucosinolate accumulation and metabolic pathways in Isatis
  indigotica Fort.
organisms:
- NA
organisms_ner:
- Arabidopsis thaliana
pmcid: PMC8862337
filename: 12870_2022_3455_Fig2_HTML.jpg
figlink: /pmc/articles/PMC8862337/figure/Fig2/
number: F2
caption: 'The GSL metabolic pathway of I. indigotica. Numbers in brackets represent
  the numbers of genes homologous with Arabidopsis; Red, green and blue words or squares
  represent aliphatic, aromatic and indole GSL metabolism-related genes or products,
  respectively; Dashed lines indicate the predicted reaction or multiple-step reactions;
  BCAT: branched-chain amino acid aminotransferase; MAM: methylthioalkylmalate synthase;
  IPMI: isopropylmalate isomerase; IPMDH: isopropylmalate dehydrogenase; CYP: cytochrome
  P450 monooxygenase; GST: glutathione S-transferase; SUR: S-alkyl-thiohydroximate
  lyase; UGT: uridine 5′-diphospho-glucuronosyltransferase; ST: sulphotransferase;
  FMO: flavin-containing monooxygenase; AOP: alkenyl hydroxyalkyl producing; IGMT:
  indole glucosinolate O-methyltransferase; TGG: thioglucoside glucohydrolase; PEN2:
  penetration-resistance gene 2; BGLU: beta glucosidase; APS: adenosine 5′-phosphosulphate;
  APS: adenosine 5′-phosphosulphate; APK: APS kinase; GSH: glutathione; PAPS: 3′-phospho-adenosine-5′-phosphosulphate;
  PAP: 3′-phospho-adenosine 5′-phosphate'
papertitle: Insights into glucosinolate accumulation and metabolic pathways in Isatis
  indigotica Fort..
reftext: Tianyi Zhang, et al. BMC Plant Biol. 2022;22:78.
year: '2022'
doi: 10.1186/s12870-022-03455-6
journal_title: BMC Plant Biology
journal_nlm_ta: BMC Plant Biol
publisher_name: BioMed Central
keywords: ''
automl_pathway: 0.5967073
figid_alias: PMC8862337__F2
figtype: Figure
redirect_from: /figures/PMC8862337__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8862337__12870_2022_3455_Fig2_HTML.html
  '@type': Dataset
  description: 'The GSL metabolic pathway of I. indigotica. Numbers in brackets represent
    the numbers of genes homologous with Arabidopsis; Red, green and blue words or
    squares represent aliphatic, aromatic and indole GSL metabolism-related genes
    or products, respectively; Dashed lines indicate the predicted reaction or multiple-step
    reactions; BCAT: branched-chain amino acid aminotransferase; MAM: methylthioalkylmalate
    synthase; IPMI: isopropylmalate isomerase; IPMDH: isopropylmalate dehydrogenase;
    CYP: cytochrome P450 monooxygenase; GST: glutathione S-transferase; SUR: S-alkyl-thiohydroximate
    lyase; UGT: uridine 5′-diphospho-glucuronosyltransferase; ST: sulphotransferase;
    FMO: flavin-containing monooxygenase; AOP: alkenyl hydroxyalkyl producing; IGMT:
    indole glucosinolate O-methyltransferase; TGG: thioglucoside glucohydrolase; PEN2:
    penetration-resistance gene 2; BGLU: beta glucosidase; APS: adenosine 5′-phosphosulphate;
    APS: adenosine 5′-phosphosulphate; APK: APS kinase; GSH: glutathione; PAPS: 3′-phospho-adenosine-5′-phosphosulphate;
    PAP: 3′-phospho-adenosine 5′-phosphate'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BAK1
  - TGG2
  - BGLU34
  - BGLU28
  - DIN2
  - PEN2
  - BARS1
  - PYK10
---
