---
figid: PMC8895431__fgene-13-784397-g001
figtitle: Benchmark of Data Processing Methods and Machine Learning Models for Gut
  Microbiome-Based Diagnosis of Inflammatory Bowel Disease
organisms:
- gut metagenome
- metagenome
- human metagenome
- Homo sapiens
- Venustaconcha ellipsiformis
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8895431
filename: fgene-13-784397-g001.jpg
figlink: /pmc/articles/PMC8895431/figure/F1/
number: F1
caption: 'Leave-one-dataset-out cross-validation pipeline. The experiments comprised
  three different stages to go from raw sequence files to the performance metrics.
  1) Raw sequences were processed with Dada2 or Deblur and close-reference clustered
  into OTUs at 99% identity. The OTUs were classified to taxonomy at 99% confidence
  with a Naive Bayes classifier and used to infer functional profiles with PICRUSt2.
  2) Generating predictions for the 15 iterations of our LODO cross validation consisted
  of all possible combinations of the listed feature selection method, normalization
  or transformation methods, batch effect correction methods, and models. 3) The average
  confusion matrix proportions across each iteration was used to generate the overall
  confusion matrix. The F1 Score and MCC were calculated using the proportions from
  the average confusion matrix. The descriptions of acronyms and abbreviations are
  the following: Clusters of Orthologous Groups of proteins (COG), Kyoto Encyclopedia
  of Genes and Genomes (KEGG) orthologs (KO), Enzyme Commission (EC), Pfam protein
  domain (PFAM), TIGR protein family (TIGRFAM) and MetaCyc pathways (pathway), centered
  log-ratio (CLR), isometric log-ratio (ILR), arcsine square root transformation (ARS),
  variance stabilizing transformation (VST), log transformation (LOG), total sum scaling
  (TSS), no normalization (NOT), Bernoulli Naive Bayes (BNB), logistic regression
  (LR), linear support vector machine (Linear SVC), random forest (RF), K nearest
  neighbours (KNN), radial support vector machine (Radial SVC), eXtreme Gradient Boosting
  (XGBoost), convolutional neural network (CNN), multilayer perceptron (MLP).'
papertitle: Benchmark of Data Processing Methods and Machine Learning Models for Gut
  Microbiome-Based Diagnosis of Inflammatory Bowel Disease.
reftext: Ryszard Kubinski, et al. Front Genet. 2022;13:784397.
year: '2022'
doi: 10.3389/fgene.2022.784397
journal_title: Frontiers in Genetics
journal_nlm_ta: Front Genet
publisher_name: Frontiers Media S.A.
keywords: inflammatory bowel disease | machine learning | gut microbiome | batch effect
  reduction | data normalization | QIIME2 | PICRUSt2
automl_pathway: 0.6956912
figid_alias: PMC8895431__F1
figtype: Figure
redirect_from: /figures/PMC8895431__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8895431__fgene-13-784397-g001.html
  '@type': Dataset
  description: 'Leave-one-dataset-out cross-validation pipeline. The experiments comprised
    three different stages to go from raw sequence files to the performance metrics.
    1) Raw sequences were processed with Dada2 or Deblur and close-reference clustered
    into OTUs at 99% identity. The OTUs were classified to taxonomy at 99% confidence
    with a Naive Bayes classifier and used to infer functional profiles with PICRUSt2.
    2) Generating predictions for the 15 iterations of our LODO cross validation consisted
    of all possible combinations of the listed feature selection method, normalization
    or transformation methods, batch effect correction methods, and models. 3) The
    average confusion matrix proportions across each iteration was used to generate
    the overall confusion matrix. The F1 Score and MCC were calculated using the proportions
    from the average confusion matrix. The descriptions of acronyms and abbreviations
    are the following: Clusters of Orthologous Groups of proteins (COG), Kyoto Encyclopedia
    of Genes and Genomes (KEGG) orthologs (KO), Enzyme Commission (EC), Pfam protein
    domain (PFAM), TIGR protein family (TIGRFAM) and MetaCyc pathways (pathway), centered
    log-ratio (CLR), isometric log-ratio (ILR), arcsine square root transformation
    (ARS), variance stabilizing transformation (VST), log transformation (LOG), total
    sum scaling (TSS), no normalization (NOT), Bernoulli Naive Bayes (BNB), logistic
    regression (LR), linear support vector machine (Linear SVC), random forest (RF),
    K nearest neighbours (KNN), radial support vector machine (Radial SVC), eXtreme
    Gradient Boosting (XGBoost), convolutional neural network (CNN), multilayer perceptron
    (MLP).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TCF23
  - ALG13
  - OTULINL
  - OTULIN
  - VCPIP1
  - ZRANB1
  - OTUB1
  - OTUB2
  - OTUD1
  - YOD1
  - OTUD3
  - OTUD4
  - OTUD5
  - OTUD6A
  - OTUD6B
  - OTUD7A
  - OTUD7B
  - TNFAIP3
  - DCLK3
  - BCR
  - IGKV3D-20
  - TRAT1
  - RIEG2
  - DCPS
  - SLURP1
  - SET
  - MCC
  - MUC2
  - CSRP3
  - MARCKSL1
  - raw
  - out
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - otu
  - Calr
  - ali
  - alpha-Est7
  - gus
  - bnb
  - ec
  - ko
  - Ars
  - AlaRS
  - vst
  - rf
  - Set
  - I-2
  - Mccc1
  - Mlp60A
  - Mlp84B
  - cnn
---
