---
figid: PMC8600466__fmicb-12-740555-g007
figtitle: Genome-Wide Analyses of Proteome and Acetylome in Zymomonas mobilis Under
  N2-Fixing Condition
organisms:
- Zymomonas mobilis
- Escherichia coli
- Bacillus subtilis
- Salmonella enterica subsp. enterica serovar Typhimurium
- Klebsiella pneumoniae
- Vibrio cholerae
- Rhodobacter capsulatus
- phototrophic bacterium
- Azotobacter vinelandii
- Rhodopseudomonas palustris
- Saccharopolyspora erythraea
- Azospirillum brasilense
- Rhodospirillum rubrum
- Borreliella burgdorferi B31
- Pseudomonas jessenii
- Saccharomyces cerevisiae
- Glycine max
- Medicago truncatula
- NA
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC8600466
filename: fmicb-12-740555-g007.jpg
figlink: /pmc/articles/PMC8600466/figure/fig7/
number: F7
caption: Regulation and acetylation of proteins in metabolic pathways of central carbon
  metabolism and nitrogen fixation in Z. mobilis in the present study. On the left
  side (A) the Entner–Doudoroff (ED) pathway, glycolysis, ethanol fermentation pathway
  and TCA cycle are shown. On the right side, nifA-regulated N-fixing island (B) and
  nitrogen fixation (C) are shown. Both of the metabolic circuits are connected in
  the center (D) by the ammonia assimilation pathway (GS/GOGAT cycle). At the bottom
  (E) the regulation of nitrogen fixation is depicted, where nifA acts as a master
  regulator to control the expression of nitrogen-fixing genes, and the activities
  of GS and nifA are putatively regulated by PII regulatory proteins. Whereas, concentrations
  of the regulatory metabolites glutamine and 2-oxoglutarate (2-OG) putatively regulate
  the interactions of PII proteins by their uridylylation (PII-UMP). The gene IDs
  in red color represent the up-regulated proteins, in blue color represents the downregulated
  proteins, and in green color represents the proteins that were not expressed differentially.
  The differentially acetylated proteins are shown with a small star in red color.
  The phosphorylated proteins () are shown with a small triangle in blue color. All
  the pathways, reactions, and metabolites are consistent with the KEGG database record
  of Z. mobilis.
papertitle: Genome-Wide Analyses of Proteome and Acetylome in Zymomonas mobilis Under
  N2-Fixing Condition.
reftext: Ayesha Nisar, et al. Front Microbiol. 2021;12:740555.
year: '2021'
doi: 10.3389/fmicb.2021.740555
journal_title: Frontiers in Microbiology
journal_nlm_ta: Front Microbiol
publisher_name: Frontiers Media S.A.
keywords: Zymomonas mobilis | 4D label-free proteomics | nitrogen fixation | post-translational
  modification | lysine acetylome
automl_pathway: 0.8664973
figid_alias: PMC8600466__F7
figtype: Figure
redirect_from: /figures/PMC8600466__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8600466__fmicb-12-740555-g007.html
  '@type': Dataset
  description: Regulation and acetylation of proteins in metabolic pathways of central
    carbon metabolism and nitrogen fixation in Z. mobilis in the present study. On
    the left side (A) the Entner–Doudoroff (ED) pathway, glycolysis, ethanol fermentation
    pathway and TCA cycle are shown. On the right side, nifA-regulated N-fixing island
    (B) and nitrogen fixation (C) are shown. Both of the metabolic circuits are connected
    in the center (D) by the ammonia assimilation pathway (GS/GOGAT cycle). At the
    bottom (E) the regulation of nitrogen fixation is depicted, where nifA acts as
    a master regulator to control the expression of nitrogen-fixing genes, and the
    activities of GS and nifA are putatively regulated by PII regulatory proteins.
    Whereas, concentrations of the regulatory metabolites glutamine and 2-oxoglutarate
    (2-OG) putatively regulate the interactions of PII proteins by their uridylylation
    (PII-UMP). The gene IDs in red color represent the up-regulated proteins, in blue
    color represents the downregulated proteins, and in green color represents the
    proteins that were not expressed differentially. The differentially acetylated
    proteins are shown with a small star in red color. The phosphorylated proteins
    () are shown with a small triangle in blue color. All the pathways, reactions,
    and metabolites are consistent with the KEGG database record of Z. mobilis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CD5L
  - SFTPA1
  - SFTPA2
  - RESF1
  - ACAT1
  - MAT1A
  - BCR
  - GNAS
  - GNAL
  - SIRT2
  - SIRT1
  - COMP
  - CBLIF
  - ATP8A2
  - WDTC1
  - mo
  - ed
  - Agxt
  - sv
  - mat
  - Mad
  - Dr
  - g5
  - ml
  - mi
  - gs
  - ton
  - tondo
  - Fs(3)Ton
  - Sirt1
  - Sirt2
  - if
  - ad
  - we
  - pr
  - de
  - SmF
  - DebC
  - ATPsynbeta
  - Atpalpha
  - adp
---
