---
figid: PMC8616465__cancers-13-05678-g001
figtitle: Impact of Chromatin Dynamics and DNA Repair on Genomic Stability and Treatment
  Resistance in Pediatric High-Grade Gliomas
organisms:
- Saccharomyces cerevisiae
- Schizosaccharomyces pombe
- Homo sapiens
- Mus musculus
- Caenorhabditis elegans
- Gallus gallus
- transposons
- Panthera leo
organisms_ner:
- Homo sapiens
- Mus musculus
- Panthera leo
- Gallus gallus
- Drosophila melanogaster
pmcid: PMC8616465
filename: cancers-13-05678-g001.jpg
figlink: /pmc/articles/PMC8616465/figure/cancers-13-05678-f001/
number: F1
caption: Schematics of the major DSB repair pathways. In NHEJ, the broken ends are
  bound by the Ku heterodimer, leading to the recruitment and activation of DNA-dependent
  protein kinase catalytic subunit (DNA-PKcs) and subsequent juxtaposition of the
  ends, followed by an eventual processing step—for instance, in the case of “dirty”
  ends produced by IR []—to generate ligatable ends. Central to HR, SSA and MMEJ is
  a DNA end resection step [] that generates 3′-ended, single-stranded DNA (ssDNA)
  tails. In MMEJ, a pathway that is promoted by poly(ADP-ribose) Polymerase 1 (PARP1)
  acting in part to prevent Ku binding to DSBs [], end-resection exposes short regions
  of complementary sequences (microhomologies) that facilitate end-bridging before
  processing the non-homologous flaps by the XPF/ERCC1 endonuclease (arrows), filling-in
  DNA synthesis and ligation. In SSA, homologous sequences exposed by long ssDNA strands
  are annealed by the RAD52 recombinase [] to generate an intermediate with non-homologous
  flaps that are clipped by XPF/ERCC1, allowing filling-in DNA synthesis and ligation
  []. In DSBR, The RAD51 recombinase acts on long resected tails to assemble nucleoprotein
  filaments (i) that invade the homologous donor sequence to generate a displacement
  loop (D-loop) (ii) and prime repair DNA synthesis at the invading end, templated
  by the homologous sequence (iii) []. DSB repair can then proceed via classical DSBR
  (also called gene conversion) or synthesis-dependent strand annealing (SDSA). In
  DSBR, capture of the second end by the displaced strand of the D-loop is associated
  with extended DNA repair synthesis and the formation of a double Holliday junction
  (dHJ) (iv) whose resolution/dissolution can lead to non-crossovers or crossovers
  between the sister chromatids. On the other hand, SDSA involves limited DNA synthesis
  as well as the displacement of the invading strand from the donor sequence and its
  annealing to the second, unengaged resected end, thereby preventing dHJ formation
  and crossover products. Single-ended DSBs are formed when the replication fork collapses
  upon an encounter with, e.g., SSBs or trapped protein-DNA complexes. The repair
  of seDSBs can occur through HR or BIR mechanisms mediated by the RAD51 or RAD52
  recombinase. BIR is initiated by strand invasion to form a D-loop and progresses
  via D-loop migration. Of note, recent studies suggest that RAD51-dependent BIR can
  also take place to repair two-ended DSBs in mammals, as previously shown in yeast
  [,]. BIR mechanisms and HR (DSBR and SDSA) can also take place at telomeres. In
  cells with intact ATRX/DAXX, the model of Lovejoy et al. [] proposes that telomere
  cohesion promoted by ATRX fosters “in-register” HR/BIR using the sister telomere,
  while loss of ATRX promotes “out-of-register” or ectopic HR/BIR, resulting in a
  net increase in telomere length, as observed in ALT cells. The authors further propose
  that disruption of a DAXX-dependent function of ATRX promotes BIR over other pathways
  []. See text for details. For the sake of clarity, only a few of the DNA repair
  factors involved in DSB repair have been indicated. Created with BioRender.
papertitle: Impact of Chromatin Dynamics and DNA Repair on Genomic Stability and Treatment
  Resistance in Pediatric High-Grade Gliomas.
reftext: Lia Pinto, et al. Cancers (Basel). 2021 Nov;13(22):5678.
year: '2021'
doi: 10.3390/cancers13225678
journal_title: Cancers
journal_nlm_ta: Cancers (Basel)
publisher_name: MDPI
keywords: pediatric high-grade gliomas | chemoresistance | chromatin dynamics | DNA
  repair | genomic instability | variant H3.3 histone | ATRX | DAXX | (peri)centromere
  | telomere | telomerase | alternative lengthening of telomere (ALT) | synthetic
  lethality
automl_pathway: 0.9625764
figid_alias: PMC8616465__F1
figtype: Figure
redirect_from: /figures/PMC8616465__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8616465__cancers-13-05678-g001.html
  '@type': Dataset
  description: Schematics of the major DSB repair pathways. In NHEJ, the broken ends
    are bound by the Ku heterodimer, leading to the recruitment and activation of
    DNA-dependent protein kinase catalytic subunit (DNA-PKcs) and subsequent juxtaposition
    of the ends, followed by an eventual processing step—for instance, in the case
    of “dirty” ends produced by IR []—to generate ligatable ends. Central to HR, SSA
    and MMEJ is a DNA end resection step [] that generates 3′-ended, single-stranded
    DNA (ssDNA) tails. In MMEJ, a pathway that is promoted by poly(ADP-ribose) Polymerase
    1 (PARP1) acting in part to prevent Ku binding to DSBs [], end-resection exposes
    short regions of complementary sequences (microhomologies) that facilitate end-bridging
    before processing the non-homologous flaps by the XPF/ERCC1 endonuclease (arrows),
    filling-in DNA synthesis and ligation. In SSA, homologous sequences exposed by
    long ssDNA strands are annealed by the RAD52 recombinase [] to generate an intermediate
    with non-homologous flaps that are clipped by XPF/ERCC1, allowing filling-in DNA
    synthesis and ligation []. In DSBR, The RAD51 recombinase acts on long resected
    tails to assemble nucleoprotein filaments (i) that invade the homologous donor
    sequence to generate a displacement loop (D-loop) (ii) and prime repair DNA synthesis
    at the invading end, templated by the homologous sequence (iii) []. DSB repair
    can then proceed via classical DSBR (also called gene conversion) or synthesis-dependent
    strand annealing (SDSA). In DSBR, capture of the second end by the displaced strand
    of the D-loop is associated with extended DNA repair synthesis and the formation
    of a double Holliday junction (dHJ) (iv) whose resolution/dissolution can lead
    to non-crossovers or crossovers between the sister chromatids. On the other hand,
    SDSA involves limited DNA synthesis as well as the displacement of the invading
    strand from the donor sequence and its annealing to the second, unengaged resected
    end, thereby preventing dHJ formation and crossover products. Single-ended DSBs
    are formed when the replication fork collapses upon an encounter with, e.g., SSBs
    or trapped protein-DNA complexes. The repair of seDSBs can occur through HR or
    BIR mechanisms mediated by the RAD51 or RAD52 recombinase. BIR is initiated by
    strand invasion to form a D-loop and progresses via D-loop migration. Of note,
    recent studies suggest that RAD51-dependent BIR can also take place to repair
    two-ended DSBs in mammals, as previously shown in yeast [,]. BIR mechanisms and
    HR (DSBR and SDSA) can also take place at telomeres. In cells with intact ATRX/DAXX,
    the model of Lovejoy et al. [] proposes that telomere cohesion promoted by ATRX
    fosters “in-register” HR/BIR using the sister telomere, while loss of ATRX promotes
    “out-of-register” or ectopic HR/BIR, resulting in a net increase in telomere length,
    as observed in ALT cells. The authors further propose that disruption of a DAXX-dependent
    function of ATRX promotes BIR over other pathways []. See text for details. For
    the sake of clarity, only a few of the DNA repair factors involved in DSB repair
    have been indicated. Created with BioRender.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CALR
  - TRIM21
  - PRKDC
  - PARP1
  - COL11A2
  - PARP2
  - PARP3
  - PARP4
  - PARP6
  - PARP8
  - PARP9
  - PARP10
  - PARP11
  - PARP12
  - PARP14
  - PARP15
  - PARP16
  - TNKS
  - TNKS2
  - TIPARP
  - RAD52
  - RAD51
  - SSB
  - KCNJ11
  - AHR
  - GPT
  - Ro60
  - Prkdc
  - Parp1
  - Rad52
  - Rad51
  - Xrcc2
  - Ssb
  - Ahr
  - Gpt
  - AHR1A
  - dsb
  - ss
  - Parp
  - double
  - Rad51D
  - spn-B
  - spn-A
  - x2
  - mtSSB
  - alt
  - CG1640
  - akirin
---
