---
figid: PMC8262738__gkab405fig4
figtitle: 'Mergeomics 2.0: a web server for multi-omics data integration to elucidate
  disease networks and predict therapeutics'
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC8262738
filename: gkab405fig4.jpg
figlink: /pmc/articles/PMC8262738/figure/F4/
number: F4
caption: 'Meta-MSEA use case study overview. To showcase the function and output of
  the web server, we utilized multiple human psoriasis GWAS and EWAS data and ran
  the multiple omics data workflow (Case 3 in Figure , Meta-MSEA). Firstly, we uploaded
  the psoriasis GWAS data, mapped the SNPs to genes using a combined skin and blood
  eQTL file, and filtered for LD > 0.7 to remove redundant SNPs in LD. Next, we uploaded
  our psoriasis EWAS association datasets and mapped the CpG sites to genes based
  on a 5 kb distance. Finally, we uploaded KEGG pathways with a psoriasis control
  set. Pathway enrichment results are produced, and each pathway''s top genes, markers,
  and corresponding association values are displayed. Psoriasis-associated pathways
  are used as input into KDA as well as PharmOmics drug repositioning (using genes
  from significant pathways/modules). In the KDA, along with the Meta-MSEA input,
  we chose the blood GIANT network option and ran the KDA providing KD results and
  visualization (Figure ) and additionally utilized the network genes as an input
  into PharmOmics. Finally, two sets of drug repositioning results were produced using
  gene overlap-based drug repositioning in PharmOmics: one based on the genes of significant
  pathways from the Meta-MSEA results and the other based on the KDA subnetwork genes.'
papertitle: 'Mergeomics 2.0: a web server for multi-omics data integration to elucidate
  disease networks and predict therapeutics.'
reftext: Jessica Ding, et al. Nucleic Acids Res. 2021 Jul 2;49(W1):W375-W387.
year: '2021'
doi: 10.1093/nar/gkab405
journal_title: Nucleic Acids Research
journal_nlm_ta: Nucleic Acids Res
publisher_name: Oxford University Press
keywords: ''
automl_pathway: 0.524463
figid_alias: PMC8262738__F4
figtype: Figure
redirect_from: /figures/PMC8262738__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8262738__gkab405fig4.html
  '@type': Dataset
  description: 'Meta-MSEA use case study overview. To showcase the function and output
    of the web server, we utilized multiple human psoriasis GWAS and EWAS data and
    ran the multiple omics data workflow (Case 3 in Figure , Meta-MSEA). Firstly,
    we uploaded the psoriasis GWAS data, mapped the SNPs to genes using a combined
    skin and blood eQTL file, and filtered for LD > 0.7 to remove redundant SNPs in
    LD. Next, we uploaded our psoriasis EWAS association datasets and mapped the CpG
    sites to genes based on a 5 kb distance. Finally, we uploaded KEGG pathways with
    a psoriasis control set. Pathway enrichment results are produced, and each pathway''s
    top genes, markers, and corresponding association values are displayed. Psoriasis-associated
    pathways are used as input into KDA as well as PharmOmics drug repositioning (using
    genes from significant pathways/modules). In the KDA, along with the Meta-MSEA
    input, we chose the blood GIANT network option and ran the KDA providing KD results
    and visualization (Figure ) and additionally utilized the network genes as an
    input into PharmOmics. Finally, two sets of drug repositioning results were produced
    using gene overlap-based drug repositioning in PharmOmics: one based on the genes
    of significant pathways from the Meta-MSEA results and the other based on the
    KDA subnetwork genes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SNRNP70
  - A1BG
  - CTLA4
  - SNHG6
  - FYN
  - LCK
  - CCR7
  - IL2RG
  - SYNE1
  - THOP1
  - LINC00273
  - TOP1
  - TOP1MT
  - TOP3A
  - TOP3B
  - TOP2A
  - TOP2B
  - ABL1
  - TLR6
  - CCR2
  - GAPDH
  - GAPDHP44
---
