---
figid: PMC8946700__biomolecules-12-00417-g003
figtitle: 'DNA Methylation Malleability and Dysregulation in Cancer Progression: Understanding
  the Role of PARP1'
organisms:
- NA
pmcid: PMC8946700
filename: biomolecules-12-00417-g003.jpg
figlink: /pmc/articles/PMC8946700/figure/biomolecules-12-00417-f003/
number: F3
caption: Steps for dynamic modifications of Cytosine and TET-mediated oxidation. (A)
  The methylation of deoxycytosine (C) residues to 5-methylcytosine (5mC) are introduced
  by DNA methyltransferase (DNMT) enzymes and sequentially oxidized by ten–eleven
  translocation (TET) enzymes via 5-hydroxymethylcytosine (5mC), 5-formylcytosine
  (5fC) and 5-carboxylcytosine (5caC). SAM, S-adenosylmethionine; SAH, S-adenosylhomocysteine;
  α-KG, α-ketoglutarate. (B) 5fC and 5caC are identified and excised by thymine DNA
  glycosylase (TDG) to produce an abasic site. The base-excision-repair (BER) pathway
  implicates excision of the abasic site, replacement of the nucleotide using unmodified
  deoxycytidine triphosphate (dCTP) by a DNA polymerase (generating pyrophosphate,
  PPi) and ligation to repair the nick.
papertitle: 'DNA Methylation Malleability and Dysregulation in Cancer Progression:
  Understanding the Role of PARP1.'
reftext: Rakesh Srivastava, et al. Biomolecules. 2022 Mar;12(3):417.
year: '2022'
doi: 10.3390/biom12030417
journal_title: Biomolecules
journal_nlm_ta: Biomolecules
publisher_name: MDPI
keywords: DNA demethylases | DNA demethylases inhibitors | PARP1 | poly(ADP-ribose)
  | DNA methylation | tumor suppressor gene | oncogene | tumor progression | cancer
  cells
automl_pathway: 0.7475351
figid_alias: PMC8946700__F3
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC8946700__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8946700__biomolecules-12-00417-g003.html
  '@type': Dataset
  description: Steps for dynamic modifications of Cytosine and TET-mediated oxidation.
    (A) The methylation of deoxycytosine (C) residues to 5-methylcytosine (5mC) are
    introduced by DNA methyltransferase (DNMT) enzymes and sequentially oxidized by
    ten–eleven translocation (TET) enzymes via 5-hydroxymethylcytosine (5mC), 5-formylcytosine
    (5fC) and 5-carboxylcytosine (5caC). SAM, S-adenosylmethionine; SAH, S-adenosylhomocysteine;
    α-KG, α-ketoglutarate. (B) 5fC and 5caC are identified and excised by thymine
    DNA glycosylase (TDG) to produce an abasic site. The base-excision-repair (BER)
    pathway implicates excision of the abasic site, replacement of the nucleotide
    using unmodified deoxycytidine triphosphate (dCTP) by a DNA polymerase (generating
    pyrophosphate, PPi) and ligation to repair the nick.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ber
  - sam
  - qkr54B
  - sls
  - Sam-S
  - Mt2
  - Nmt
  - Tet
  - sch
  - sah
  - La
  - Tdg
  - VhaAC39-1
  - de
---
