---
figid: PMC8182918__12934_2021_1594_Fig2_HTML
figtitle: In-situ muconic acid extraction reveals sugar consumption bottleneck in
  a xylose-utilizing Saccharomyces cerevisiae strain
organisms:
- Saccharomyces cerevisiae
- Candida albicans
- Talaromyces atroroseus
- Blastobotrys adeninivorans
- Escherichia coli
- Klebsiella pneumoniae
- NA
- Homo sapiens
organisms_ner:
- Homo sapiens
- Arabidopsis thaliana
pmcid: PMC8182918
filename: 12934_2021_1594_Fig2_HTML.jpg
figlink: /pmc/articles/PMC8182918/figure/Fig2/
number: F2
caption: Pathways used to produce muconic acid from glucose and xylose. Glucose and
  xylose are converted after uptake into phosphoenolpyruvate (PEP) and erythrose-4-phosphate
  (E4P) by the glycolytic and pentose-phosphate pathways. The two compounds are combined
  to enter the shikimate pathway and produce 3-dihydroshikimate (DHS), which is then
  converted using the bacterial heterologous pathway for muconic acid biosynthesis
  (MApw) into protocatechuic acid (PCA), catechol and muconic acid. The MApw consists
  of a DHS dehydratase from Podospora anserina (aroZPan), PCA decarboxylase (PCAD)
  from K. pneumoniae (aroY-Ciso) and catechol dioxygenase (CDO) from C. albicans (HQD2Ca).
  Feedback inhibition of aromatic amino acids acting on Aro3 and Aro4, has been relieved
  by deletion of Aro3 and site-specific mutagenesis of Aro4 (ARO4K229L). Overexpression
  of PAD1 stimulates production of prenylated flavine mononucleotide (prFMN), a co-factor
  of the protocatechuic acid decarboxylase (PCAD) enzyme encoded by aroY-Ciso. To
  further enhance muconic acid production, the synthesis of ethanol was abolished
  by deletion of the pyruvase decarboxylase (PDC) genes (pdcΔ). Muconic acid is exported
  from the cells by one or more unknown carriers
papertitle: In-situ muconic acid extraction reveals sugar consumption bottleneck in
  a xylose-utilizing Saccharomyces cerevisiae strain.
reftext: Thomas Nicolaï, et al. Microb Cell Fact. 2021;20:114.
year: '2021'
doi: 10.1186/s12934-021-01594-3
journal_title: Microbial Cell Factories
journal_nlm_ta: Microb Cell Fact
publisher_name: BioMed Central
keywords: Yeast cell factory | Muconic acid | Xylose | PDC negative | In-situ product
  removal | Lignocellulosic biomass | Glucose | ISPR | Pyruvate decarboxylase negative
automl_pathway: 0.9053986
figid_alias: PMC8182918__F2
figtype: Figure
redirect_from: /figures/PMC8182918__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8182918__12934_2021_1594_Fig2_HTML.html
  '@type': Dataset
  description: Pathways used to produce muconic acid from glucose and xylose. Glucose
    and xylose are converted after uptake into phosphoenolpyruvate (PEP) and erythrose-4-phosphate
    (E4P) by the glycolytic and pentose-phosphate pathways. The two compounds are
    combined to enter the shikimate pathway and produce 3-dihydroshikimate (DHS),
    which is then converted using the bacterial heterologous pathway for muconic acid
    biosynthesis (MApw) into protocatechuic acid (PCA), catechol and muconic acid.
    The MApw consists of a DHS dehydratase from Podospora anserina (aroZPan), PCA
    decarboxylase (PCAD) from K. pneumoniae (aroY-Ciso) and catechol dioxygenase (CDO)
    from C. albicans (HQD2Ca). Feedback inhibition of aromatic amino acids acting
    on Aro3 and Aro4, has been relieved by deletion of Aro3 and site-specific mutagenesis
    of Aro4 (ARO4K229L). Overexpression of PAD1 stimulates production of prenylated
    flavine mononucleotide (prFMN), a co-factor of the protocatechuic acid decarboxylase
    (PCAD) enzyme encoded by aroY-Ciso. To further enhance muconic acid production,
    the synthesis of ethanol was abolished by deletion of the pyruvase decarboxylase
    (PDC) genes (pdcΔ). Muconic acid is exported from the cells by one or more unknown
    carriers
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - PREP
  - PTPN22
  - PDC
  - PNKD
  - ADA2
  - FLVCR1
  - PSMD14
  - PADI1
  - FMN1
  - PEP
  - PAS2
  - ARO3
  - PAN
  - PAD1
---
