---
figid: PMC8290834__MSB-17-e10099-g001
figtitle: Validation of the model’s phenotypic predictions
organisms:
- NA
pmcid: PMC8290834
filename: MSB-17-e10099-g001.jpg
figlink: /pmc/articles/PMC8290834/figure/msb202010099-fig-0006/
number: F6
caption: Growth phenotype observed on CSY medium supplemented with different carbohydrates
  (1% final concentration) compared with iJL208 predictions. Growth observations and
  predictions were normalized by growth on sucrose. Discrepancies between experimental
  data and predictions are highlighted (red rectangle).The metabolic network reconstruction
  allowed the identification of potential candidates carrying promiscuous reactions
  (pink) responsible for maltose and mannose catabolism. 3D structures of candidates
  (Mfl499, Mfl120, and Mfl254; red) are superimposed to available structures in the
  Protein Data Bank (light blue) for which the suspected enzymatic activity is annotated.
  FATCAT P‐values are shown.The 173 genes where the transcriptomic and proteomic expression
  data (Exp) are consistent (see Appendix Fig ) are compared with flux state predictions
  generated by parsimonious flux‐balance analysis (Flux). True positives (TP) and
  true negatives (TN) correspond to genes where both predictions and observations
  are consistent, while false positives (FP) and false negatives (TN) where they are
  inconsistent. Genes considered expressed or with an active flux are represented
  in dark blue, and silent in light blue.Revised gene essentiality data (see Fig )
  compared with essentiality predictions from iJL208. Essential and non‐essential
  protein‐coding genes are represented in green and red, respectively. TP, TN, FP,
  and TN are as in (C).The pentose phosphate pathway (PPP) of Mesoplasma florum along
  with ribose import. The transaldolase (TALA) reaction (pink) was missing in the
  network. Structural similarity between two 2‐deoxyribose‐phosphate aldolases (Mfl121,
  red in left box; Mfl639, red in right box) to a known transaldolase from the PDB
  (light blue) served as basis for the addition of this gene‐associated reaction in
  the model. The observed essentiality (Ess.) and expression (Exp.) of genes (see
  panels (C) and (D)) associated with each reaction of the PPP are indicated by green
  (essential or expressed) or red (non‐essential or non‐expressed) colored squares.
papertitle: Genome‐scale metabolic modeling reveals key features of a minimal gene
  set.
reftext: Jean‐Christophe Lachance, et al. Mol Syst Biol. 2021 Jul;17(7):e10099.
year: '2021'
doi: 10.15252/msb.202010099
journal_title: Molecular Systems Biology
journal_nlm_ta: Mol Syst Biol
publisher_name: John Wiley and Sons Inc.
keywords: genome design | genome‐scale models | Mesoplasma florum | minimal cells
  | synthetic biology | Metabolism | Microbiology, Virology & Host Pathogen Interaction
automl_pathway: 0.7240693
figid_alias: PMC8290834__F6
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC8290834__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8290834__MSB-17-e10099-g001.html
  '@type': Dataset
  description: Growth phenotype observed on CSY medium supplemented with different
    carbohydrates (1% final concentration) compared with iJL208 predictions. Growth
    observations and predictions were normalized by growth on sucrose. Discrepancies
    between experimental data and predictions are highlighted (red rectangle).The
    metabolic network reconstruction allowed the identification of potential candidates
    carrying promiscuous reactions (pink) responsible for maltose and mannose catabolism.
    3D structures of candidates (Mfl499, Mfl120, and Mfl254; red) are superimposed
    to available structures in the Protein Data Bank (light blue) for which the suspected
    enzymatic activity is annotated. FATCAT P‐values are shown.The 173 genes where
    the transcriptomic and proteomic expression data (Exp) are consistent (see Appendix Fig
    ) are compared with flux state predictions generated by parsimonious flux‐balance
    analysis (Flux). True positives (TP) and true negatives (TN) correspond to genes
    where both predictions and observations are consistent, while false positives
    (FP) and false negatives (TN) where they are inconsistent. Genes considered expressed
    or with an active flux are represented in dark blue, and silent in light blue.Revised
    gene essentiality data (see Fig ) compared with essentiality predictions from
    iJL208. Essential and non‐essential protein‐coding genes are represented in green
    and red, respectively. TP, TN, FP, and TN are as in (C).The pentose phosphate
    pathway (PPP) of Mesoplasma florum along with ribose import. The transaldolase
    (TALA) reaction (pink) was missing in the network. Structural similarity between
    two 2‐deoxyribose‐phosphate aldolases (Mfl121, red in left box; Mfl639, red in
    right box) to a known transaldolase from the PDB (light blue) served as basis
    for the addition of this gene‐associated reaction in the model. The observed essentiality
    (Ess.) and expression (Exp.) of genes (see panels (C) and (D)) associated with
    each reaction of the PPP are indicated by green (essential or expressed) or red
    (non‐essential or non‐expressed) colored squares.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - tn
  - exp
  - ex
  - Pep
  - bgcn
  - Hao
  - tre
  - Gr5a
  - Treh
  - Tre1
  - ATPsynbeta
  - Atpalpha
  - Pfk
  - CG15812
  - Ald1
  - Aldh-III
  - fru
  - Hex1
  - Hexo1
  - pyr
  - de
  - Pgi
  - dc
  - RpLP0
  - Rpi
  - alc
  - rib
  - Acf
---
