---
figid: PMC8781990__pathogens-11-00005-g005
figtitle: Amino Acid Sensing and Assimilation by the Fungal Pathogen Candida albicans
  in the Human Host
organisms:
- Candida albicans
- Homo sapiens
pmcid: PMC8781990
filename: pathogens-11-00005-g005.jpg
figlink: /pmc/articles/PMC8781990/figure/pathogens-11-00005-f005/
number: F5
caption: Nitrogen utilization in C. albicans. Most free amino acids, obtained from
  extracellular uptake (1), vacuolar release (2) or the degradation of small peptides
  by intracellular peptidases (3), are deaminated by specific aminotransferases (ATs)
  using α-ketoglutarate as the amino group acceptor, forming glutamate. Many ATs are
  found in the cytosol and there is an abundant supply of glutamate in the cytosol
  (Glucyto) (4). Glutamate can be imported into the mitochondria by transporters,
  such as Agc1 (5). Arginine and proline are converted to glutamate via the proline
  catabolic pathway (6). In the cytosolic portion of this pathway, arginine is converted
  to ornithine and urea by arginase (Car1). Ornithine is rapidly converted to glutamate
  semialdehyde (GSA) by ornithine transaminase (Car2), which is non-enzymatically
  converted to the cyclic Δ-1-pyrroline-5-carboxylate (P5C) and then reduced by P5C
  reductase (Pro3), generating proline (Procyto). Proline enters the mitochondria
  (Promito) via an unidentified transporter. Promito is catabolized to glutamate (Glumito)
  via the concerted activities of proline dehydrogenase (Put1) and P5C dehydrogenase
  (Put2). Glutamate produced in the mitochondria (Glumito) (8) is thought to exit
  the mitochondria and become part of Glucyto. Glucyto is used in the synthesis of
  proline (7); glutamate is first activated to produce glutamate-5-phosphate (G5P)
  by γ-glutamyl kinase (Pro1), followed by its conversion to GSA/P5C by γ-glutamyl
  phosphate reductase (Pro2) and is then reduced to proline by P5C reductase (Pro3).
  Cytosolic glutamate can be converted to α-ketoglutarate by the NAD+-dependent glutamate
  dehydrogenase (Gdh2), which is critical for maintaining the α-ketoglutarate pool
  in the cytosol (α-KGcyto). The reaction catalyzed by Gdh2 generates ammonia as a
  by-product (9). α-ketoglutarate can be transported in and out of mitochondria via
  putative oxodicarboxylate carriers (e.g., Odc1). The mitochondrial α-KGmito pool
  is linked to the TCA cycle (10). Urea, derived either from arginine or from extracellular
  uptake, can be converted to ammonia and CO2 via urea amidolyase (Dur1,2) (11). When
  grown in the presence of N-acetylglucosamine (GlcNac), ammonia is also produced
  when glucosamine-6-phosphate is converted to fructose-6-phosphate through glucosamine-6-phosphate
  isomerase (Nag1) (12). As the cytosolic pH is maintained near neutrality (pH~6.5),
  most ammonia is converted to its protonated form, ammonium (13). Free ammonia is
  membrane-permeable and can readily exit cells, where it contributes to the alkalization
  of the growth environment, a consequence of its conversion to ammonium (14). Ammonium
  in cells can be reassimilated to generate glutamate by the NADPH-dependent glutamate
  dehydrogenase (Gdh3), which uses α-ketoglutarate as a substrate (15); additionally,
  ammonium can be reassimilated via glutamine synthetase (Gln1), which catalyzes the
  conversion of glutamate to glutamine (16). Glutamate can also be generated from
  glutamine and α-ketoglutarate via the NADH-dependent glutamate synthase (Glt1) (17).
  In mitochondria, the NADH/FADH2 generated by the TCA cycle and proline catabolism
  can be oxidized via the electron transport chain (ETC) to generate ATP (18). Mitochondrial
  function and multiple enzymatic activities are repressed in cells grown in the presence
  of high glucose (≥0.2%, (19)). The localization of enzymes shown in green have been
  experimentally validated in C. albicans, whereas those shown in red are based on
  the localization of their corresponding orthologs in S. cerevisiae and the presence
  or absence of strong mitochondrial pre-sequences in the N-terminals of their respective
  protein sequences (Candida Genome Database (CGD, http://www.candidagenome.org (accessed
  on 18 December 2021)) analyzed using the MitoFate tool []. The following enzymes
  are present in the C. albicans genome—PRODH = PUT1 (C5_02600W), P5CDH = PUT2 (C5_04880C),
  GDH = GDH2 (C2_07900W), P5CR = PRO3 (C4_00240), OAT = CAR2 (C4_00160C), ARG = CAR1
  (C5_04490C), GK = PRO1 (CR_10580), GPR = PRO2 (C3_07220C). GLN1 (CR_05050W), GLT1
  (C1_06550W), DUR1,2 (C1_04660W), NADPH-dependent GDH = GDH3 (C4_06120W), AAT1 (C2_05250C)
  and AAT2 = AAT21 (CR_07620W).
papertitle: Amino Acid Sensing and Assimilation by the Fungal Pathogen Candida albicans
  in the Human Host.
reftext: Fitz Gerald S. Silao, et al. Pathogens. 2022 Jan;11(1):5.
year: '2022'
doi: 10.3390/pathogens11010005
journal_title: Pathogens
journal_nlm_ta: Pathogens
publisher_name: MDPI
keywords: Candida albicans | human fungal pathogen | nutrient sensing | amino acid
  metabolism | proline catabolism | mitochondria | SPS-sensor | nitrogen catabolite
  repression | glucose repression
automl_pathway: 0.9340368
figid_alias: PMC8781990__F5
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC8781990__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8781990__pathogens-11-00005-g005.html
  '@type': Dataset
  description: Nitrogen utilization in C. albicans. Most free amino acids, obtained
    from extracellular uptake (1), vacuolar release (2) or the degradation of small
    peptides by intracellular peptidases (3), are deaminated by specific aminotransferases
    (ATs) using α-ketoglutarate as the amino group acceptor, forming glutamate. Many
    ATs are found in the cytosol and there is an abundant supply of glutamate in the
    cytosol (Glucyto) (4). Glutamate can be imported into the mitochondria by transporters,
    such as Agc1 (5). Arginine and proline are converted to glutamate via the proline
    catabolic pathway (6). In the cytosolic portion of this pathway, arginine is converted
    to ornithine and urea by arginase (Car1). Ornithine is rapidly converted to glutamate
    semialdehyde (GSA) by ornithine transaminase (Car2), which is non-enzymatically
    converted to the cyclic Δ-1-pyrroline-5-carboxylate (P5C) and then reduced by
    P5C reductase (Pro3), generating proline (Procyto). Proline enters the mitochondria
    (Promito) via an unidentified transporter. Promito is catabolized to glutamate
    (Glumito) via the concerted activities of proline dehydrogenase (Put1) and P5C
    dehydrogenase (Put2). Glutamate produced in the mitochondria (Glumito) (8) is
    thought to exit the mitochondria and become part of Glucyto. Glucyto is used in
    the synthesis of proline (7); glutamate is first activated to produce glutamate-5-phosphate
    (G5P) by γ-glutamyl kinase (Pro1), followed by its conversion to GSA/P5C by γ-glutamyl
    phosphate reductase (Pro2) and is then reduced to proline by P5C reductase (Pro3).
    Cytosolic glutamate can be converted to α-ketoglutarate by the NAD+-dependent
    glutamate dehydrogenase (Gdh2), which is critical for maintaining the α-ketoglutarate
    pool in the cytosol (α-KGcyto). The reaction catalyzed by Gdh2 generates ammonia
    as a by-product (9). α-ketoglutarate can be transported in and out of mitochondria
    via putative oxodicarboxylate carriers (e.g., Odc1). The mitochondrial α-KGmito
    pool is linked to the TCA cycle (10). Urea, derived either from arginine or from
    extracellular uptake, can be converted to ammonia and CO2 via urea amidolyase
    (Dur1,2) (11). When grown in the presence of N-acetylglucosamine (GlcNac), ammonia
    is also produced when glucosamine-6-phosphate is converted to fructose-6-phosphate
    through glucosamine-6-phosphate isomerase (Nag1) (12). As the cytosolic pH is
    maintained near neutrality (pH~6.5), most ammonia is converted to its protonated
    form, ammonium (13). Free ammonia is membrane-permeable and can readily exit cells,
    where it contributes to the alkalization of the growth environment, a consequence
    of its conversion to ammonium (14). Ammonium in cells can be reassimilated to
    generate glutamate by the NADPH-dependent glutamate dehydrogenase (Gdh3), which
    uses α-ketoglutarate as a substrate (15); additionally, ammonium can be reassimilated
    via glutamine synthetase (Gln1), which catalyzes the conversion of glutamate to
    glutamine (16). Glutamate can also be generated from glutamine and α-ketoglutarate
    via the NADH-dependent glutamate synthase (Glt1) (17). In mitochondria, the NADH/FADH2
    generated by the TCA cycle and proline catabolism can be oxidized via the electron
    transport chain (ETC) to generate ATP (18). Mitochondrial function and multiple
    enzymatic activities are repressed in cells grown in the presence of high glucose
    (≥0.2%, (19)). The localization of enzymes shown in green have been experimentally
    validated in C. albicans, whereas those shown in red are based on the localization
    of their corresponding orthologs in S. cerevisiae and the presence or absence
    of strong mitochondrial pre-sequences in the N-terminals of their respective protein
    sequences (Candida Genome Database (CGD, http://www.candidagenome.org (accessed
    on 18 December 2021)) analyzed using the MitoFate tool []. The following enzymes
    are present in the C. albicans genome—PRODH = PUT1 (C5_02600W), P5CDH = PUT2 (C5_04880C),
    GDH = GDH2 (C2_07900W), P5CR = PRO3 (C4_00240), OAT = CAR2 (C4_00160C), ARG =
    CAR1 (C5_04490C), GK = PRO1 (CR_10580), GPR = PRO2 (C3_07220C). GLN1 (CR_05050W),
    GLT1 (C1_06550W), DUR1,2 (C1_04660W), NADPH-dependent GDH = GDH3 (C4_06120W),
    AAT1 (C2_05250C) and AAT2 = AAT21 (CR_07620W).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ATP8A2
  - OMA1
  - LMNA
  - ZMPSTE24
  - WDTC1
  - ACAN
  - SLC25A12
  - DECR1
  - ABL2
  - RERE
  - CA1
  - NR1I3
  - GNAS
  - GNAL
  - PYCR1
  - KDM5B
  - BRCA2
  - FANCD2
  - PSEN1
  - AAT1
  - GPT
  - CFAP91
  - GLUD2
  - GPD2
  - GDF15
  - MTG1
  - ODC1
  - SLC25A21
  - HT
  - GIN1
  - GIT1
  - LYPLA2P1
  - APP
---
