---
figid: PMC8242269__gr10
figtitle: Large-scale Proteomic and Phosphoproteomic Analyses of Maize Seedling Leaves
  During De-etiolation
organisms:
- Arabidopsis thaliana
- Nicotiana tabacum
- Oryza sativa
- Zea mays
- Magnoliopsida
- eudicotyledons
- Liliopsida
- Homo sapiens
- Rattus norvegicus
pmcid: PMC8242269
filename: gr10.jpg
figlink: /pmc/articles/PMC8242269/figure/f0050/
number: F10
caption: Phosphorylation of proteins involved inphotosynthetic carbon assimilation
  changes significantly during de-etiolationThis figure was modified from figures
  by Majeran et al.  and Wingler et al. . Proteins written in blue belong to the C4
  pathway, and proteins written in green belong to the Calvin cycle. The green and
  red circles separately indicate the quantified and phosphorylated proteins that
  function in photosynthetic carbon assimilation of the C4 plants. The number of circles
  for each component indicates the number of homologs identified in this study. A
  green circle with an upward or downward triangle indicates that the abundance of
  the protein was significantly upregulated or downregulated during de-etiolation;
  a red circle with an upward or downward triangle indicates that the NPL of the protein
  was significantly upregulated or downregulated during de-etiolation. CA, carbonic
  anhydrase; PEPC, phosphoenolpyruvate carboxylase; AspAT, aspartate aminotransferase;
  DIT, dicarboxylate transporter; NADP-MDH, malate dehydrogenase [NADP]; NADP-ME,
  NADP-dependent malic enzyme; MEP, envelope protein; PPT, phosphoenolpyruvate/phosphate
  translocator; PGK, phosphoglycerate kinase; TPI, triosephosphate isomerase; FBA,
  fructose-bisphosphate aldolase; FBP, fructose-1,6-bisphosphatase; TKL, transketolase;
  SBPase, seduheptulose bisphosphatase; RPI, ribose-5-phosphate isomerase; RPE, ribulose-phosphate
  3-epimerase; PRK, phosphoribulokinase; OAA, oxaloacetic acid; PEP, phosphoenolpyruvate;
  Asp, aspartate; RuBP, ribulose 1,5-bisphosphate; Ru5P, ribulose 5-P; R5P, ribose
  5-P; S7P, sedoheptulose 7-P; S1,7BP, sedoheptulose 1,7-bisphosphate; E4P, erythrose
  4-P; F6P, fructose 6-P; FBP, fructose bisphosphate; DHAP, dihydroxyacetone phosphate;
  GAP, glyceraldehyde 3-phosphate; 1,3BPG, 1,3 biphosphoglycerate; 3PGA, 3-phosphoglycerate.
papertitle: Large-scale Proteomic and Phosphoproteomic Analyses of Maize Seedling
  Leaves During De-etiolation.
reftext: Zhi-Fang Gao, et al. Genomics Proteomics Bioinformatics. 2020 Aug;18(4):397-414.
year: '2020'
doi: 10.1016/j.gpb.2020.12.004
journal_title: Genomics, Proteomics & Bioinformatics
journal_nlm_ta: Genomics Proteomics Bioinformatics
publisher_name: Elsevier
keywords: Maize seedling leaves | De-etiolation | Quantitative analysis | Proteome
  | Phosphoproteome
automl_pathway: 0.9155816
figid_alias: PMC8242269__F10
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
- Homo sapiens
- Rattus norvegicus
redirect_from: /figures/PMC8242269__F10
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8242269__gr10.html
  '@type': Dataset
  description: Phosphorylation of proteins involved inphotosynthetic carbon assimilation
    changes significantly during de-etiolationThis figure was modified from figures
    by Majeran et al.  and Wingler et al. . Proteins written in blue belong to the
    C4 pathway, and proteins written in green belong to the Calvin cycle. The green
    and red circles separately indicate the quantified and phosphorylated proteins
    that function in photosynthetic carbon assimilation of the C4 plants. The number
    of circles for each component indicates the number of homologs identified in this
    study. A green circle with an upward or downward triangle indicates that the abundance
    of the protein was significantly upregulated or downregulated during de-etiolation;
    a red circle with an upward or downward triangle indicates that the NPL of the
    protein was significantly upregulated or downregulated during de-etiolation. CA,
    carbonic anhydrase; PEPC, phosphoenolpyruvate carboxylase; AspAT, aspartate aminotransferase;
    DIT, dicarboxylate transporter; NADP-MDH, malate dehydrogenase [NADP]; NADP-ME,
    NADP-dependent malic enzyme; MEP, envelope protein; PPT, phosphoenolpyruvate/phosphate
    translocator; PGK, phosphoglycerate kinase; TPI, triosephosphate isomerase; FBA,
    fructose-bisphosphate aldolase; FBP, fructose-1,6-bisphosphatase; TKL, transketolase;
    SBPase, seduheptulose bisphosphatase; RPI, ribose-5-phosphate isomerase; RPE,
    ribulose-phosphate 3-epimerase; PRK, phosphoribulokinase; OAA, oxaloacetic acid;
    PEP, phosphoenolpyruvate; Asp, aspartate; RuBP, ribulose 1,5-bisphosphate; Ru5P,
    ribulose 5-P; R5P, ribose 5-P; S7P, sedoheptulose 7-P; S1,7BP, sedoheptulose 1,7-bisphosphate;
    E4P, erythrose 4-P; F6P, fructose 6-P; FBP, fructose bisphosphate; DHAP, dihydroxyacetone
    phosphate; GAP, glyceraldehyde 3-phosphate; 1,3BPG, 1,3 biphosphoglycerate; 3PGA,
    3-phosphoglycerate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DiT1
  - PGK
  - PEP
  - PAS2
  - PPDK
  - PRK
  - APE2
  - ASP2
  - TKL
  - RPE
  - RSW10
  - SBPASE
  - AIR
  - AIRN
  - PC
  - PAEP
  - PREP
  - PTPN22
  - PER1
  - PLK3
  - SHH
  - PDSS1
  - LMBR1
  - GAPDH
  - GAPDHP44
  - RASA1
  - RGS6
  - ME3
  - RN7SL263P
  - TXK
  - RPIA
  - FBXO3
  - Pc
  - Pgk1
  - Hspg2
  - Gapdh
  - Rasa1
  - Got1
  - Rpe
---
