---
figid: PMC8877502__microorganisms-10-00198-g002
figtitle: Proteomic Shifts Reflecting Oxidative Stress and Reduced Capacity for Protein
  Synthesis, and Alterations to Mitochondrial Membranes in Neurospora crassa Lacking
  VDAC
organisms:
- Neurospora crassa
- Homo sapiens
organisms_ner:
- Homo sapiens
pmcid: PMC8877502
filename: microorganisms-10-00198-g002.jpg
figlink: /pmc/articles/PMC8877502/figure/microorganisms-10-00198-f002/
number: F2
caption: 'Metabolic pathway that includes proteins present at non-wild-type levels
  in ΔPor-1. Pathway details were obtained from KEGG pathways (https://www.genome.jp/kegg/pathway.html,
  accessed 21 November 2021, []) and only the segments of pathways that include proteins
  in relatively higher or lower amounts are shown for clarity. (a) Shikimate pathway
  and aromatic amino acid biosynthesis (KEGG pathways shikimate ncr00400 and sulfur
  amino acids ncr00920, respectively). Intermediates are as follows: panel A: a: 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate;
  b: 3-dehydroxyquinate; c: 3-dehydroshikamate; d: shikimate 3-phosphate; e: 5-O-(1-carboxyvinyl)-3-phosphoshikamate;
  f: prephenate; g: 4-hydroxy-phenylpyruvate; h: phenylpyruvate; i: anthranilate;
  j: N-(5-phospho-β-D-ribosyl)-anthranilate; k: 1-(2-carboxyphenylamino)-1′-deoxy-D-ribulose-5-phosphate.
  (b) Amino acid biosynthesis (ncr00260, ncr00270) and sulfur metabolism (ncr00920)
  APS: adenosine 5′ phosphosulfate; PAPS: 3′ phosphoadenosine 5′ phosphosulfate: OAA:
  oxaloacetate; PEP, phosphoenolpyruvate (c) TCA cycle and pyruvate metabolism (ncr00020
  and ncr00620) AcCoA: acetyl coenzyme A; ssaThPP: succinate semialdehyde thiamine
  diphosphate; sdhla: s-succinyldihydrolipoamide-E.'
papertitle: Proteomic Shifts Reflecting Oxidative Stress and Reduced Capacity for
  Protein Synthesis, and Alterations to Mitochondrial Membranes in Neurospora crassa
  Lacking VDAC.
reftext: Sabbir R. Shuvo, et al. Microorganisms. 2022 Feb;10(2):198.
year: '2022'
doi: 10.3390/microorganisms10020198
journal_title: Microorganisms
journal_nlm_ta: Microorganisms
publisher_name: MDPI
keywords: VDAC | Neurospora crassa | mitochondria | cytosol | proteomics | unfolded
  protein response | oxidative stress | ergosterol | hyphae | shikimate | amino acid
  biosynthesis
automl_pathway: 0.9379033
figid_alias: PMC8877502__F2
figtype: Figure
redirect_from: /figures/PMC8877502__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8877502__microorganisms-10-00198-g002.html
  '@type': Dataset
  description: 'Metabolic pathway that includes proteins present at non-wild-type
    levels in ΔPor-1. Pathway details were obtained from KEGG pathways (https://www.genome.jp/kegg/pathway.html,
    accessed 21 November 2021, []) and only the segments of pathways that include
    proteins in relatively higher or lower amounts are shown for clarity. (a) Shikimate
    pathway and aromatic amino acid biosynthesis (KEGG pathways shikimate ncr00400
    and sulfur amino acids ncr00920, respectively). Intermediates are as follows:
    panel A: a: 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate; b: 3-dehydroxyquinate;
    c: 3-dehydroshikamate; d: shikimate 3-phosphate; e: 5-O-(1-carboxyvinyl)-3-phosphoshikamate;
    f: prephenate; g: 4-hydroxy-phenylpyruvate; h: phenylpyruvate; i: anthranilate;
    j: N-(5-phospho-β-D-ribosyl)-anthranilate; k: 1-(2-carboxyphenylamino)-1′-deoxy-D-ribulose-5-phosphate.
    (b) Amino acid biosynthesis (ncr00260, ncr00270) and sulfur metabolism (ncr00920)
    APS: adenosine 5′ phosphosulfate; PAPS: 3′ phosphoadenosine 5′ phosphosulfate:
    OAA: oxaloacetate; PEP, phosphoenolpyruvate (c) TCA cycle and pyruvate metabolism
    (ncr00020 and ncr00620) AcCoA: acetyl coenzyme A; ssaThPP: succinate semialdehyde
    thiamine diphosphate; sdhla: s-succinyldihydrolipoamide-E.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FUT3
  - PAEP
  - PREP
  - PTPN22
  - KLK3
  - SH2B2
---
