---
figid: PMC8384902__gr5
figtitle: ASCL1, NKX2-1, and PROX1 co-regulate subtype-specific genes in small-cell
  lung cancer
organisms:
- NA
pmcid: PMC8384902
filename: gr5.jpg
figlink: /pmc/articles/PMC8384902/figure/fig5/
number: F5
caption: Transcriptional targets of ASCL1, NKX2.1 and PROX1(A) The workflow used to
  identify transcriptional targets of the three TFs using ChIP-seq, HiChIP, and RNA-seq.(B)
  Venn showing the extensive overlap in TF-associated genes identified from the ChIP-seq
  and HiChIP analysis.(C) Venn showing the subset of ASCL1 ChIP-seq associated genes
  (from B) that are differentially expressed (DEG) in NCI-H2107 cells with siASCL1
  knock-down followed by RNA-seq.(D) Venn from (B) but with the DEG from NCI-H2107
  cells with siASCL1; siNKX2-1; siPROX1 knock-down followed by RNA-seq. Data is shown
  for TF activated genes only.(E) GO pathway analysis of genes activated by the 3
  TFs show an enrichment for terms associated with NOTCH signaling for ASCL1, and
  cell-cycle-related genes for NKX2-1. Genes in bold are also DEG with ASCL1 KD only;
  asterisked genes were bound by ASCL1 and NKX2-1 only; genes in green are those that
  were reproduced with knockdown of the three TFs in NCI-H889.See also  and  for the
  full lists of direct targets of ASCL1 and all 3TFs, and the pathway analysis.
papertitle: ASCL1, NKX2-1, and PROX1 co-regulate subtype-specific genes in small-cell
  lung cancer.
reftext: Karine Pozo, et al. iScience. 2021 Sep 24;24(9):102953.
year: '2021'
doi: 10.1016/j.isci.2021.102953
journal_title: iScience
journal_nlm_ta: iScience
publisher_name: Elsevier
keywords: molecular biology | bioinformatics | cancer systems biology
automl_pathway: 0.869236
figid_alias: PMC8384902__F5
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC8384902__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8384902__gr5.html
  '@type': Dataset
  description: Transcriptional targets of ASCL1, NKX2.1 and PROX1(A) The workflow
    used to identify transcriptional targets of the three TFs using ChIP-seq, HiChIP,
    and RNA-seq.(B) Venn showing the extensive overlap in TF-associated genes identified
    from the ChIP-seq and HiChIP analysis.(C) Venn showing the subset of ASCL1 ChIP-seq
    associated genes (from B) that are differentially expressed (DEG) in NCI-H2107
    cells with siASCL1 knock-down followed by RNA-seq.(D) Venn from (B) but with the
    DEG from NCI-H2107 cells with siASCL1; siNKX2-1; siPROX1 knock-down followed by
    RNA-seq. Data is shown for TF activated genes only.(E) GO pathway analysis of
    genes activated by the 3 TFs show an enrichment for terms associated with NOTCH
    signaling for ASCL1, and cell-cycle-related genes for NKX2-1. Genes in bold are
    also DEG with ASCL1 KD only; asterisked genes were bound by ASCL1 and NKX2-1 only;
    genes in green are those that were reproduced with knockdown of the three TFs
    in NCI-H889.See also  and  for the full lists of direct targets of ASCL1 and all
    3TFs, and the pathway analysis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SCLC1
  - ASCL1
  - PROX1
  - NKX2-1
  - DBH
  - DDC
  - INSM1
  - TGFB2
  - SNCAIP
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - TF
  - ELF3
  - HES1
  - GATD3
  - RBPJ
  - JAG2
  - YBX1
  - DLL1
  - DLL3
  - DLL4
  - LFNG
  - BCR
  - IGKV3D-20
  - BTBD3
  - GPRC5B
  - PLXNA2
  - TCF4
  - TCF7L2
  - DLX5
  - FOXA2
  - UGCG
  - YWHAZ
  - ID2
  - BTG2
  - SMG9
  - B3GNT5
  - CBFA2T2
  - MFSD2A
  - ZIC5
  - RAD1
  - ERCC4
  - RNASE6
  - RRAD
  - ROCK1
  - SYT4
  - SDK2
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - CCNA2
  - SET
  - TFDP1
  - CCNB2
  - CCNB1
  - VRK1
  - CDK1
  - PTTG1
  - CDC20
  - CENPE
  - NDC80
  - DBF4
  - HMMR
  - PPP1CC
  - KIF18A
---
