---
figid: PMC8367434__fmicb-12-698286-g003
figtitle: 'DiTing: A Pipeline to Infer and Compare Biogeochemical Pathways From Metagenomic
  and Metatranscriptomic Data'
organisms:
- Escherichia coli
- Salmonella enterica subsp. enterica serovar Typhimurium
- Escherichia coli K-12
- Pseudomonas putida
- Synechococcus elongatus PCC 7942  FACHB-805
- Escherichia coli B
- Hyphomicrobium sulfonivorans
- Rhodovulum sulfidophilum
- Streptomyces varsoviensis
- metagenome
- human metagenome
- metagenomes
- uncultured marine bacterium
- Arabidopsis thaliana
- PX clade
- Spinacia oleracea
- Homo sapiens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC8367434
filename: fmicb-12-698286-g003.jpg
figlink: /pmc/articles/PMC8367434/figure/F3/
number: F3
caption: Pie charts representing the relative abundance of carbon (A), nitrogen (B),
  sulfur (C), and DMSP (D) cycle-related pathways for five metagenomic samples from
  the Manus Basin. Normalized relative abundance was calculated through dividing the
  relative abundance of a pathway in an individual sample by the sum of this pathway’s
  relative abundance in all samples. The pie chart area reflects the relative abundance
  of the process according to the scale shown in pink. The dashed line in panel (D)
  means the data was not shown. (A) CBB, Calvin-Benson-Bassham cycle; rTCA, reductive
  citric acid cycle; WL, Wood-Ljungdahl pathway; 3HB, 3-hydroxypropionate bicycle.
  (B) ANRA, assimilatory nitrate reduction to ammonia; DNRA, Dissimilatory nitrate
  reduction to ammonia; Anammox, anaerobic ammonia oxidation. (C) ASR, assimilatory
  sulfate reduction; DSR, dissimilatory sulfate reduction. (D) DMSP, dimethylsulfoniopropionate;
  MMPA, methylmecaptopropionate; MeSH, methanethiol; DMSO, dimethyl sulfoxide; L-Met,
  L-methionine. This figure was the output from DiTing.
papertitle: 'DiTing: A Pipeline to Infer and Compare Biogeochemical Pathways From
  Metagenomic and Metatranscriptomic Data.'
reftext: Chun-Xu Xue, et al. Front Microbiol. 2021;12:698286.
year: '2021'
doi: 10.3389/fmicb.2021.698286
journal_title: Frontiers in Microbiology
journal_nlm_ta: Front Microbiol
publisher_name: Frontiers Media S.A.
keywords: biogeochemical cycle | metagenomics | pipleline | software | DiTing | metatranscriptomics
automl_pathway: 0.9288163
figid_alias: PMC8367434__F3
figtype: Figure
redirect_from: /figures/PMC8367434__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8367434__fmicb-12-698286-g003.html
  '@type': Dataset
  description: Pie charts representing the relative abundance of carbon (A), nitrogen
    (B), sulfur (C), and DMSP (D) cycle-related pathways for five metagenomic samples
    from the Manus Basin. Normalized relative abundance was calculated through dividing
    the relative abundance of a pathway in an individual sample by the sum of this
    pathway’s relative abundance in all samples. The pie chart area reflects the relative
    abundance of the process according to the scale shown in pink. The dashed line
    in panel (D) means the data was not shown. (A) CBB, Calvin-Benson-Bassham cycle;
    rTCA, reductive citric acid cycle; WL, Wood-Ljungdahl pathway; 3HB, 3-hydroxypropionate
    bicycle. (B) ANRA, assimilatory nitrate reduction to ammonia; DNRA, Dissimilatory
    nitrate reduction to ammonia; Anammox, anaerobic ammonia oxidation. (C) ASR, assimilatory
    sulfate reduction; DSR, dissimilatory sulfate reduction. (D) DMSP, dimethylsulfoniopropionate;
    MMPA, methylmecaptopropionate; MeSH, methanethiol; DMSO, dimethyl sulfoxide; L-Met,
    L-methionine. This figure was the output from DiTing.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ed
  - so
  - AstA-R1
  - dsr
  - hy
  - CycE
  - cyc
  - wl
  - VhaAC39-1
  - fw
  - mesh
  - form3
  - SP
  - modSP
  - Ms
---
