---
figid: PMC8537086__microorganisms-09-02148-g003
figtitle: Staphylococcus aureus Transcriptome Data and Metabolic Modelling Investigate
  the Interplay of Ser/Thr Kinase PknB, Its Phosphatase Stp, the glmR/yvcK Regulon
  and the cdaA Operon for Metabolic Adaptation
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC8537086
filename: microorganisms-09-02148-g003.jpg
figlink: /pmc/articles/PMC8537086/figure/microorganisms-09-02148-f003/
number: F3
caption: Different metabolic pathways are altered between the WT, ΔpknBΔstp, ΔpknB
  and Δstp. The three mutants are compared to the wild type. Shown is the central
  metabolism and how it is influenced by pknB. The corresponding textbook pathways
  are shown. Fluxes are indicated by arrows. All major (by flux strengths) EPMs that
  contribute to a specific textbook pathway in NewHG as calculated from the data are
  shown directly as labels for this textbook pathway. The numbering of the EPMs is
  given according to  (left column; followed by an abbreviated listing of the involved
  enzymes; next follows the input and output reaction catalysed by the EPM and on
  the right part the EPM flux activity compared in the wild type and mutants). The
  colour indicates an up- (green) or downregulated (red) EPM compared to the wild
  type. The pathways indicated include synthesis of peptidoglycan, nucleotide and
  aromatic amino acid and catabolism involving GOT.
papertitle: Staphylococcus aureus Transcriptome Data and Metabolic Modelling Investigate
  the Interplay of Ser/Thr Kinase PknB, Its Phosphatase Stp, the glmR/yvcK Regulon
  and the cdaA Operon for Metabolic Adaptation.
reftext: Chunguang Liang, et al. Microorganisms. 2021 Oct;9(10):2148.
year: '2021'
doi: 10.3390/microorganisms9102148
journal_title: Microorganisms
journal_nlm_ta: Microorganisms
publisher_name: MDPI
keywords: metabolism | flux balance analysis | phosphorylation | regulation | riboswitch
  | PknB | Stp | yvcK/glmR operon
automl_pathway: 0.8891802
figid_alias: PMC8537086__F3
figtype: Figure
redirect_from: /figures/PMC8537086__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8537086__microorganisms-09-02148-g003.html
  '@type': Dataset
  description: Different metabolic pathways are altered between the WT, ΔpknBΔstp,
    ΔpknB and Δstp. The three mutants are compared to the wild type. Shown is the
    central metabolism and how it is influenced by pknB. The corresponding textbook
    pathways are shown. Fluxes are indicated by arrows. All major (by flux strengths)
    EPMs that contribute to a specific textbook pathway in NewHG as calculated from
    the data are shown directly as labels for this textbook pathway. The numbering
    of the EPMs is given according to  (left column; followed by an abbreviated listing
    of the involved enzymes; next follows the input and output reaction catalysed
    by the EPM and on the right part the EPM flux activity compared in the wild type
    and mutants). The colour indicates an up- (green) or downregulated (red) EPM compared
    to the wild type. The pathways indicated include synthesis of peptidoglycan, nucleotide
    and aromatic amino acid and catabolism involving GOT.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Got2
  - pts
  - PheRS-m
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Pep
  - bgcn
  - GOT1
  - GOT2
  - PTS
  - RASA1
  - RGS6
  - PAEP
  - PREP
  - PTPN22
---
