---
figid: PMC8222170__nihms-1673829-f0008
figtitle: KEGG pathology and disease analysis
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- NA
pmcid: PMC8222170
filename: nihms-1673829-f0008.jpg
figlink: /pmc/articles/PMC8222170/figure/F7/
number: F7
caption: A) Venn diagram indicating the number of neuron and astrocyte pathology and
  disease modules altered by the KD, as well as modules that overlapped between neurons
  and astrocytes (n = 7, 7, 7, 6). B) The AD module was altered in neurons from KD
  mice. For each gene transcript displayed, individual mice fed a KD are represented
  by vertical color bars placed over the gene name to indicate relative expression
  compared to the mean of the chow cohort for the same gene. Blue reflects a decrease
  in relative expression, red represents increased relative expression, and grey indicates
  no change. The color scheme ranges from −1 to +1 indicating a log2 fold change.
  C) KD-induced changes in neuron oxidative phosphorylation gene expression helped
  implicate the AD pathway. The protocol we used did not capture mtDNA-encoded genes,
  so changes in mtDNA-encoded RNAs are unknown. In most samples, oxidative phosphorylation
  subunits showed some degree of increased expression.
papertitle: A Ketogenic Diet Differentially Affects Neuron And Astrocyte Transcription.
reftext: Scott J. Koppel, et al. J Neurochem. ;157(6):1930-1945.
year: '2021'
doi: 10.1111/jnc.15313
journal_title: Journal of neurochemistry
journal_nlm_ta: J Neurochem
publisher_name: ''
keywords: Ketogenic diet | Alzheimer’s disease | RNA-Seq | mitochondria | neurons
  | astrocytes
automl_pathway: 0.7137007
figid_alias: PMC8222170__F7
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
redirect_from: /figures/PMC8222170__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8222170__nihms-1673829-f0008.html
  '@type': Dataset
  description: A) Venn diagram indicating the number of neuron and astrocyte pathology
    and disease modules altered by the KD, as well as modules that overlapped between
    neurons and astrocytes (n = 7, 7, 7, 6). B) The AD module was altered in neurons
    from KD mice. For each gene transcript displayed, individual mice fed a KD are
    represented by vertical color bars placed over the gene name to indicate relative
    expression compared to the mean of the chow cohort for the same gene. Blue reflects
    a decrease in relative expression, red represents increased relative expression,
    and grey indicates no change. The color scheme ranges from −1 to +1 indicating
    a log2 fold change. C) KD-induced changes in neuron oxidative phosphorylation
    gene expression helped implicate the AD pathway. The protocol we used did not
    capture mtDNA-encoded genes, so changes in mtDNA-encoded RNAs are unknown. In
    most samples, oxidative phosphorylation subunits showed some degree of increased
    expression.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - cy
  - Hadha
  - Elavl2
  - hubb
  - Tiam2
  - Elavl4
  - Styk1
  - Cox4i1
  - Ctbp1
  - App
  - Ado
  - Kdm5d
  - Hya
  - ap
  - Comp
  - Kat8
  - Bad
  - Hm
  - Hopx
  - Mat1a
  - Mmp7
  - Cd36
  - Cpe
  - Ty
  - Igfals
  - Bax
  - Nkx3-1
  - Od
  - Rabep2
  - Om
  - HADHA
  - ELAVL2
  - TIAM2
  - ELAVL4
  - STYK1
  - COX8A
  - CPOX
  - COX5A
  - COX5B
  - COX6C
  - COX7C
  - .na.character
  - COX4I1
  - COX4I2
  - COX6A1
  - COX6A2
  - COX6B1
  - COX6B2
  - COX7A1
  - COX7A2
  - COX7B
  - COX7B2
  - COX8C
  - CTBP1
  - APP
  - ADO
  - COMP
  - KAT8
  - BAD
  - CLCN7
  - HOPX
  - ACAT1
  - MAT1A
  - GSTZ1
  - CD36
  - FAT1
  - ATM
  - IGFALS
  - SOD1
  - BAX
  - FOLR1
  - FOSL1
  - RABEP2
  - Th
  - Ccne1
  - Fat1
  - RGD1565355
---
