---
figid: PMC8541835__nihms-1748432-f0001
figtitle: 'RNases H1 and H2: guardians of the stability of the nuclear genome when
  supply of dNTPs is limiting for DNA synthesis'
organisms:
- NA
pmcid: PMC8541835
filename: nihms-1748432-f0001.jpg
figlink: /pmc/articles/PMC8541835/figure/F1/
number: F1
caption: Limiting supply of dNTPs leads to lethal R-loop-mediated replicative stress
  and DNA damage in the absence of RNases H1 and H2, and to catastrophic Top1-mediated
  DNA damage in the presence of a high load of unrepaired single genomic rNMPs. a
  Wild-type budding yeast cell. In unperturbed S-phase, the cytoplasmic Rnr1-containing-RNR
  complexes, which are hetero-tetramers comprised of two Rnr1 subunits (small, dark-green
  ovals) and one subunit each of Rnr2 (small, grey sphere) and Rnr4 (small, yellow
  sphere), provide dNTPs (blue dots) for the duplication and repair of nuclear DNA.
  The total cellular concentrations of rNTPs (pink dots) are several-fold higher than
  those of dNTPs. A replication fork is represented in the nucleus. Pol ε (labelled
  ε and colored in purple) continuously synthesizes the leading strand (one horizontal,
  blue arrow). Pol δ (labelled δ and colored in brown) discontinuously synthesizes
  Okazaki fragments, which form the lagging strand (three horizontal, blue arrows).
  Pol α, which initiates the synthesis of leading strand and Okazaki fragments, is
  omitted for clarity. Parental/template DNA strands for replication are represented
  by black horizontal lines. Pol ε and Pol δ incorporate rNMPs (small, pink dashes)
  in nascent DNA strands at low frequency. Pol ε naturally has ∼ 3-fold lower discrimination
  against the utilization of rNTPs than Pol δ. RNase H2 (labelled H2), which is a
  hetero-trimeric enzymatic complex (three spheres colored pale-yellow, light-green
  and red), incises at the 5′-end of the rNMP, thereby initiating error-free removal
  of the rNMP by the cellular RER pathway. The replicative helicase complex CMG (Cdc45-MCM-GINS;
  large, redcrimson oval) unwinds the double-stranded DNA duplex ahead of the replication
  fork. The RNA (pink line) extruding from the RNA polymerase (large, dark-blue oval)
  could hybridize with the transcribed strand (bottom, black line), thereby forming
  a tripartite R-loop structure, which is comprised of an RNA/DNA hybrid and an unpaired
  non-transcribed strand (top, black line). The RNA moiety of the RNA/DNA hybrid could
  be cleaved by RNase H1 (purple sphere labelled H1) or RNase H2. The colors of the
  nascent DNA strands and the nascent RNA match the colors of their corresponding
  building blocks; i.e. blue for deoxyribonucleoside-monophosphate and pink for rNMP,
  respectively. The directions of replication and transcription are represented by
  small, horizontal, black arrows. The nucleus and cytoplasm are not drawn to scale.
  b Budding yeast cell depleted of Rnr1 and lacking RNases H1 and H2. In S-phase in
  absence of Rnr1, Rnr3 is modestly expressed, due to mild activation of the Mec1-Rad53-Dun1-dependent-S-phase
  checkpoint. The cytoplasmic Rnr3-containing-RNR complexes, which are hetero-tetramers
  comprised of two Rnr3 subunits (brown ovals) and one subunit each of Rnr2 and Rnr4,
  provide the cell with ∼ 3-fold lower dNTP concentrations, as compared to Rnr1-containing-RNR
  complexes in panel a. This significantly slows down the replication fork and concomitantly
  increases (by ∼ 5-fold) the incorporation of rNMPs by replicative Pols in newly
  synthesized DNA. Pol ε would frequently hand over to Pol δ at the nascent leading
  strand (labelled ε/δ), because of acute replicative stress. DNA repair could also
  be compromised by low dNTP pools. As RNase H2 is absent, rNMPs accumulate in the
  newly synthesized DNA, particularly the leading strand, and Top1 (orange sphere)
  incises at the 3′-end of some embedded rNMPs. Top1-mediated cleavages would lead
  to DNA mutagenesis and/or genome instability. The absence of RNases H1 and H2 would
  lead to a persistent R-loop. This could block the advancement of the replication
  fork (red, lightning signal), thereby triggering irreversible fork collapse and
  breakage, and ultimately leading to cell lethality. Note that RNA-polymerase-associated-R-loops
  can be co-directional or head-on with respect to the direction of the replication
  fork, and head-on collisions are likely to be more deleterious than co-directional
  collisions. Other details are as in panel a. c Budding yeast cell that bears an
  rNTP-permissive form of Pol ε, lacks RNase H2, and is also depleted of Rnr1. In
  S-phase in absence of Rnr1, the rNTP-permissive form of Pol ε, which is encoded
  by pol2-M644G allele (labelled ε*), excessively incorporates rNMPs in nascent leading
  strand. Processing of rNMPs by Top1 in absence of RNase H2, in the nascent leading
  strand, would induce single-strand breaks or DSBs. Repair of these DNA lesions (e.g.
  of DSBs by Rad51/Rad52-dependent-homologous recombination) might be compromised,
  ultimately leading to severe growth defects. Pol ε* would frequently hand over to
  Pol δ at the nascent leading strand (labelled ε*/δ). Other details are as in panels
  a and b. d Budding yeast cell that bears an rNTP-permissive form of Pol δ, lacks
  RNase H2, and is also depleted of Rnr1. In S-phase in absence of Rnr1, the rNTP-permissive
  form of Pol δ, which is encoded by pol3-L612M allele (labelled δ* and colored in
  brown), incorporates high loads of rNMPs in both nascent leading and lagging strands.
  Pol ε would frequently hand over to Pol δ* at the nascent leading strand (labelled
  ε/δ*). Processing of rNMPs by Top1 in absence of RNase H2, in both nascent DNA strands,
  would induce single-strand breaks or DSBs. Other details are as in panels a–c
papertitle: 'RNases H1 and H2: guardians of the stability of the nuclear genome when
  supply of dNTPs is limiting for DNA synthesis.'
reftext: Susana M. Cerritelli, et al. Curr Genet. ;66(6):1073-1084.
year: '2020'
doi: 10.1007/s00294-020-01086-8
journal_title: Current genetics
journal_nlm_ta: Curr Genet
publisher_name: ''
keywords: RNase H | Topoisomerase 1 | Ribonucleotide reductase | Ribonucleotide |
  R-loop | DNA damage
automl_pathway: 0.5678178
figid_alias: PMC8541835__F1
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC8541835__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8541835__nihms-1748432-f0001.html
  '@type': Dataset
  description: Limiting supply of dNTPs leads to lethal R-loop-mediated replicative
    stress and DNA damage in the absence of RNases H1 and H2, and to catastrophic
    Top1-mediated DNA damage in the presence of a high load of unrepaired single genomic
    rNMPs. a Wild-type budding yeast cell. In unperturbed S-phase, the cytoplasmic
    Rnr1-containing-RNR complexes, which are hetero-tetramers comprised of two Rnr1
    subunits (small, dark-green ovals) and one subunit each of Rnr2 (small, grey sphere)
    and Rnr4 (small, yellow sphere), provide dNTPs (blue dots) for the duplication
    and repair of nuclear DNA. The total cellular concentrations of rNTPs (pink dots)
    are several-fold higher than those of dNTPs. A replication fork is represented
    in the nucleus. Pol ε (labelled ε and colored in purple) continuously synthesizes
    the leading strand (one horizontal, blue arrow). Pol δ (labelled δ and colored
    in brown) discontinuously synthesizes Okazaki fragments, which form the lagging
    strand (three horizontal, blue arrows). Pol α, which initiates the synthesis of
    leading strand and Okazaki fragments, is omitted for clarity. Parental/template
    DNA strands for replication are represented by black horizontal lines. Pol ε and
    Pol δ incorporate rNMPs (small, pink dashes) in nascent DNA strands at low frequency.
    Pol ε naturally has ∼ 3-fold lower discrimination against the utilization of rNTPs
    than Pol δ. RNase H2 (labelled H2), which is a hetero-trimeric enzymatic complex
    (three spheres colored pale-yellow, light-green and red), incises at the 5′-end
    of the rNMP, thereby initiating error-free removal of the rNMP by the cellular
    RER pathway. The replicative helicase complex CMG (Cdc45-MCM-GINS; large, redcrimson
    oval) unwinds the double-stranded DNA duplex ahead of the replication fork. The
    RNA (pink line) extruding from the RNA polymerase (large, dark-blue oval) could
    hybridize with the transcribed strand (bottom, black line), thereby forming a
    tripartite R-loop structure, which is comprised of an RNA/DNA hybrid and an unpaired
    non-transcribed strand (top, black line). The RNA moiety of the RNA/DNA hybrid
    could be cleaved by RNase H1 (purple sphere labelled H1) or RNase H2. The colors
    of the nascent DNA strands and the nascent RNA match the colors of their corresponding
    building blocks; i.e. blue for deoxyribonucleoside-monophosphate and pink for
    rNMP, respectively. The directions of replication and transcription are represented
    by small, horizontal, black arrows. The nucleus and cytoplasm are not drawn to
    scale. b Budding yeast cell depleted of Rnr1 and lacking RNases H1 and H2. In
    S-phase in absence of Rnr1, Rnr3 is modestly expressed, due to mild activation
    of the Mec1-Rad53-Dun1-dependent-S-phase checkpoint. The cytoplasmic Rnr3-containing-RNR
    complexes, which are hetero-tetramers comprised of two Rnr3 subunits (brown ovals)
    and one subunit each of Rnr2 and Rnr4, provide the cell with ∼ 3-fold lower dNTP
    concentrations, as compared to Rnr1-containing-RNR complexes in panel a. This
    significantly slows down the replication fork and concomitantly increases (by
    ∼ 5-fold) the incorporation of rNMPs by replicative Pols in newly synthesized
    DNA. Pol ε would frequently hand over to Pol δ at the nascent leading strand (labelled
    ε/δ), because of acute replicative stress. DNA repair could also be compromised
    by low dNTP pools. As RNase H2 is absent, rNMPs accumulate in the newly synthesized
    DNA, particularly the leading strand, and Top1 (orange sphere) incises at the
    3′-end of some embedded rNMPs. Top1-mediated cleavages would lead to DNA mutagenesis
    and/or genome instability. The absence of RNases H1 and H2 would lead to a persistent
    R-loop. This could block the advancement of the replication fork (red, lightning
    signal), thereby triggering irreversible fork collapse and breakage, and ultimately
    leading to cell lethality. Note that RNA-polymerase-associated-R-loops can be
    co-directional or head-on with respect to the direction of the replication fork,
    and head-on collisions are likely to be more deleterious than co-directional collisions.
    Other details are as in panel a. c Budding yeast cell that bears an rNTP-permissive
    form of Pol ε, lacks RNase H2, and is also depleted of Rnr1. In S-phase in absence
    of Rnr1, the rNTP-permissive form of Pol ε, which is encoded by pol2-M644G allele
    (labelled ε*), excessively incorporates rNMPs in nascent leading strand. Processing
    of rNMPs by Top1 in absence of RNase H2, in the nascent leading strand, would
    induce single-strand breaks or DSBs. Repair of these DNA lesions (e.g. of DSBs
    by Rad51/Rad52-dependent-homologous recombination) might be compromised, ultimately
    leading to severe growth defects. Pol ε* would frequently hand over to Pol δ at
    the nascent leading strand (labelled ε*/δ). Other details are as in panels a and
    b. d Budding yeast cell that bears an rNTP-permissive form of Pol δ, lacks RNase
    H2, and is also depleted of Rnr1. In S-phase in absence of Rnr1, the rNTP-permissive
    form of Pol δ, which is encoded by pol3-L612M allele (labelled δ* and colored
    in brown), incorporates high loads of rNMPs in both nascent leading and lagging
    strands. Pol ε would frequently hand over to Pol δ* at the nascent leading strand
    (labelled ε/δ*). Processing of rNMPs by Top1 in absence of RNase H2, in both nascent
    DNA strands, would induce single-strand breaks or DSBs. Other details are as in
    panels a–c
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NR2E3
  - TOP1
  - ERVW-4
  - RNR1
  - NR4A2
---
