---
figid: PMC8901598__fendo-13-828780-g002
figtitle: Metabolic and Epigenetic Regulation by Estrogen in Adipocytes
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Danio rerio
pmcid: PMC8901598
filename: fendo-13-828780-g002.jpg
figlink: /pmc/articles/PMC8901598/figure/f2/
number: F2
caption: Epigenetic effects of ERα/β in adipocytes. (A) ERα can bind to promoter regions
  with repressive H3K9me3 marks (I-VI), where it recruits the histone demethylase
  KDM4B (also known as JMJD2B), which specifically removes these marks (II). This
  enables the recruitment and activity of the methyl transferase MLL2, which trimethylates
  lysine 4 on histone 3, forming activating H3K4me3 marks, which promotes gene expression
  (III). This process may occur on the promoters of Pparg and C/ebp, promoting adipogenesis.
  Conversely, ERα can also bind to actively transcribed genes characterized by H3K27ac
  marks (IV), where it binds various coregulators, including NRIP1, that enables binding
  of histone deacetylases (HDACs), which remove the acetyl groups on H3K27 (V). Finally,
  the ERα/NRIP1/HDAC complex can further bind the PRC2/EZH2 polycomb complex, which
  adds methyl groups to form repressive H3K27me3 marks (VI). This process can also
  occur on the Pparg and C/ebp promoter/enhancers, inhibiting adipogenesis. Although
  the repressive pathway appears most predominant, further studies should investigate
  whether the activating pathway indeed plays a role in certain preadipocyte/mesenchymal
  stem cell subpopulations. (B) ERα and ERβ affects DNA methylation through several
  mechanisms. ERα promotes de novo methylation and gene silencing by binding to actively
  transcribed regions (I), where the ERα/HDAC/PRC2/EZH2 complex first converts activating
  H3K27ac marks to repressive H3K27me3 marks (see  IV-VI for details). The DNA methyl
  transferase DNMT3 recognizes the H3K27me3 marks, and stabilized by the ERα/HDAC/PRC2/EZH2
  complex it adds a methyl group to cytosine residues on the surrounding DNA, leading
  to stable gene silencing (II-III). Conversely, ERα can inhibit passive DNA methylation
  after cell division. This occurs by transcriptional inhibition of DNMT1, which copies
  the DNA methylation pattern of the old DNA strand onto the newly synthesized DNA
  (IV-V). Red methyl groups (bottom panel V) represent hypomethylated regions in response
  to ERα-mediated repression of DNMT1, leading to increased beiging. ERα and/or ERβ
  can also promote active demethylation by recruitment of TET2, AID/APOBEC/BER complexes,
  which alter methylated cytosines in numerous ways that ultimately restores unmodified
  cytosine (VI-I). Active demethylation likely remodels adipogenic super-enhancers,
  and has been found to inhibit adipogenesis and increase Glut4 expression. C, Cytosine;
  5mC, 5-methylcytosine; 5hmC, 5-hydroxymethylcytosine; 5fmC, 5-formylcytosine; 5caC,
  5-carboxylcytosine; 5hmU, 5-hydroxymethyluracyl. Figure created in BioRender.com.
papertitle: Metabolic and Epigenetic Regulation by Estrogen in Adipocytes.
reftext: Jan-Inge Bjune, et al. Front Endocrinol (Lausanne). 2022;13:828780.
year: '2022'
doi: 10.3389/fendo.2022.828780
journal_title: Frontiers in Endocrinology
journal_nlm_ta: Front Endocrinol (Lausanne)
publisher_name: Frontiers Media S.A.
keywords: sexual dimorphism | steroids | estrogen | adipocyte | epigenetic
automl_pathway: 0.9295655
figid_alias: PMC8901598__F2
figtype: Figure
redirect_from: /figures/PMC8901598__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8901598__fendo-13-828780-g002.html
  '@type': Dataset
  description: Epigenetic effects of ERα/β in adipocytes. (A) ERα can bind to promoter
    regions with repressive H3K9me3 marks (I-VI), where it recruits the histone demethylase
    KDM4B (also known as JMJD2B), which specifically removes these marks (II). This
    enables the recruitment and activity of the methyl transferase MLL2, which trimethylates
    lysine 4 on histone 3, forming activating H3K4me3 marks, which promotes gene expression
    (III). This process may occur on the promoters of Pparg and C/ebp, promoting adipogenesis.
    Conversely, ERα can also bind to actively transcribed genes characterized by H3K27ac
    marks (IV), where it binds various coregulators, including NRIP1, that enables
    binding of histone deacetylases (HDACs), which remove the acetyl groups on H3K27
    (V). Finally, the ERα/NRIP1/HDAC complex can further bind the PRC2/EZH2 polycomb
    complex, which adds methyl groups to form repressive H3K27me3 marks (VI). This
    process can also occur on the Pparg and C/ebp promoter/enhancers, inhibiting adipogenesis.
    Although the repressive pathway appears most predominant, further studies should
    investigate whether the activating pathway indeed plays a role in certain preadipocyte/mesenchymal
    stem cell subpopulations. (B) ERα and ERβ affects DNA methylation through several
    mechanisms. ERα promotes de novo methylation and gene silencing by binding to
    actively transcribed regions (I), where the ERα/HDAC/PRC2/EZH2 complex first converts
    activating H3K27ac marks to repressive H3K27me3 marks (see  IV-VI for details).
    The DNA methyl transferase DNMT3 recognizes the H3K27me3 marks, and stabilized
    by the ERα/HDAC/PRC2/EZH2 complex it adds a methyl group to cytosine residues
    on the surrounding DNA, leading to stable gene silencing (II-III). Conversely,
    ERα can inhibit passive DNA methylation after cell division. This occurs by transcriptional
    inhibition of DNMT1, which copies the DNA methylation pattern of the old DNA strand
    onto the newly synthesized DNA (IV-V). Red methyl groups (bottom panel V) represent
    hypomethylated regions in response to ERα-mediated repression of DNMT1, leading
    to increased beiging. ERα and/or ERβ can also promote active demethylation by
    recruitment of TET2, AID/APOBEC/BER complexes, which alter methylated cytosines
    in numerous ways that ultimately restores unmodified cytosine (VI-I). Active demethylation
    likely remodels adipogenic super-enhancers, and has been found to inhibit adipogenesis
    and increase Glut4 expression. C, Cytosine; 5mC, 5-methylcytosine; 5hmC, 5-hydroxymethylcytosine;
    5fmC, 5-formylcytosine; 5caC, 5-carboxylcytosine; 5hmU, 5-hydroxymethyluracyl.
    Figure created in BioRender.com.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PPARG
  - EBP
  - GLB1
  - SH3D19
  - TET2
  - TDG
  - SLC2A4
  - ERG
  - ESR1
  - ERAL1
  - KDM4B
  - HDAC9
  - HDAC1
  - HDAC2
  - HDAC3
  - HDAC8
  - HDAC4
  - HDAC5
  - HDAC7
  - HDAC6
  - HDAC10
  - SIRT1
  - SIRT2
  - SIRT3
  - SIRT4
  - SIRT5
  - SIRT6
  - SIRT7
  - HDAC11
  - ESR2
  - EZH2
  - ADRB3
  - DIO2
  - UCP1
  - KMT2D
  - KMT2B
  - EED
  - EZH1
  - SUZ12
  - DNMT1
  - Pparg
  - Ebp
  - Apobec3
  - Tet2
  - Tdg
  - Slc2a4
  - Erg
  - Esr1
  - Eral1
  - Kdm4b
  - Esr2
  - Opn3
  - Ezh2
  - Adrb3
  - Dio2
  - Ucp1
  - Kmt2b
  - Kmt2d
  - Dnmt1
  - Cebpg
  - pparg
  - ebp
  - tet2
  - tdg.1
  - erg
  - kcnh6a
  - esr1
  - kdm4b
  - dnmt3bb.2
  - ezh2
  - dio2
  - ucp1
  - kmt2d
  - dnmt1
---
