---
figid: PMC8762163__fpls-12-795274-g001
figtitle: 'Coordinated Epigenetic Regulation in Plants: A Potent Managerial Tool to
  Conquer Biotic Stress'
organisms:
- NA
organisms_ner:
- Arabidopsis thaliana
pmcid: PMC8762163
filename: fpls-12-795274-g001.jpg
figlink: /pmc/articles/PMC8762163/figure/fig1/
number: F1
caption: The epigenetic regulatory mechanisms act coordinately in reprogramming gene
  expression when plants encounter biotic stress. The known factors in Arabidopsis
  that regulate the biotic stress involve small RNAs, RdDM, methylation and demethylation
  of DNA, histone modification, and chromatin remodelers, which are present here.
  (A) Small RNAs participate in conical and non-canonical RdDM pathways to regulate
  DNA methylation on TEs and defense-related genes. The RDR2- and DCL3-dependent hc-siRNAs
  are key components in PolIV-RDR2-DCL3-AGO4-PolV-DRM2, the conical RdDM pathway,
  which establish and reinforce DNA methylation at TEs and regulate the nearby defense
  response genes. The siRNAs derived from mRNA precursor, TE non-coding RNAs (TE ncRNAs),
  inverted repeats, or dsRNAs produced by RDR6 are processed by DCL2/4 also participate
  in establishing the DNA methylation and regulate the expression of defense-related
  genes. (B) Deposition of DNA methylation and on the NLRs, PRRs, or defense response
  genes leads to a transcriptionally suppressed status. Other components acting with
  the chromatin remodeler SWP73A (dark red oval) are not clear (unknown component
  is shown in gray oval). RdDM components including PolIV and PolV, DCLs, RDRs, AGO4/6,
  and DRD1 are shown in a dark green oval. DNA methyltransferases DRM2, MET1, and
  CMT3 are shown in a light green oval. SWI2/SNF2-like ATPases Decreased DNA Methylation
  1 (DDM1), Swi2/Snf2-related 1 (SWR1), and Classy1 (CLSY1) are shown in an orange
  oval. The H3K9me2 reader EDM2 is shown in the light brown oval. Mutant plants that
  display a resistant phenotype to pathogen (such as Pst) relying on an SA-dependent
  response are surrounded with a red outline, whereas the blue outline indicates a
  more susceptible phenotype.
papertitle: 'Coordinated Epigenetic Regulation in Plants: A Potent Managerial Tool
  to Conquer Biotic Stress.'
reftext: Chien-Yu Huang, et al. Front Plant Sci. 2021;12:795274.
year: '2021'
doi: 10.3389/fpls.2021.795274
journal_title: Frontiers in Plant Science
journal_nlm_ta: Front Plant Sci
publisher_name: Frontiers Media S.A.
keywords: epigenetics | DNA methylation | histone modification | small RNAs | chromatin
  remodelers | plant defense | plant-microbes interaction
automl_pathway: 0.9404026
figid_alias: PMC8762163__F1
figtype: Figure
redirect_from: /figures/PMC8762163__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8762163__fpls-12-795274-g001.html
  '@type': Dataset
  description: The epigenetic regulatory mechanisms act coordinately in reprogramming
    gene expression when plants encounter biotic stress. The known factors in Arabidopsis
    that regulate the biotic stress involve small RNAs, RdDM, methylation and demethylation
    of DNA, histone modification, and chromatin remodelers, which are present here.
    (A) Small RNAs participate in conical and non-canonical RdDM pathways to regulate
    DNA methylation on TEs and defense-related genes. The RDR2- and DCL3-dependent
    hc-siRNAs are key components in PolIV-RDR2-DCL3-AGO4-PolV-DRM2, the conical RdDM
    pathway, which establish and reinforce DNA methylation at TEs and regulate the
    nearby defense response genes. The siRNAs derived from mRNA precursor, TE non-coding
    RNAs (TE ncRNAs), inverted repeats, or dsRNAs produced by RDR6 are processed by
    DCL2/4 also participate in establishing the DNA methylation and regulate the expression
    of defense-related genes. (B) Deposition of DNA methylation and on the NLRs, PRRs,
    or defense response genes leads to a transcriptionally suppressed status. Other
    components acting with the chromatin remodeler SWP73A (dark red oval) are not
    clear (unknown component is shown in gray oval). RdDM components including PolIV
    and PolV, DCLs, RDRs, AGO4/6, and DRD1 are shown in a dark green oval. DNA methyltransferases
    DRM2, MET1, and CMT3 are shown in a light green oval. SWI2/SNF2-like ATPases Decreased
    DNA Methylation 1 (DDM1), Swi2/Snf2-related 1 (SWR1), and Classy1 (CLSY1) are
    shown in an orange oval. The H3K9me2 reader EDM2 is shown in the light brown oval.
    Mutant plants that display a resistant phenotype to pathogen (such as Pst) relying
    on an SA-dependent response are surrounded with a red outline, whereas the blue
    outline indicates a more susceptible phenotype.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CHR1
  - RDR2
  - RPS2
  - rps2
  - SWI2
  - DCL3
  - DRD1
  - EDM2
  - RPS4
  - rps4
  - RRS1
  - SUVH4
  - AGO4
  - CMT3
  - MET1
  - DCL1
  - DCL2
  - DCL4
  - DRM2
  - AT2G33830
  - AGO6
  - CHR38
  - CNGC4
  - RDR6
  - TXR1
  - AT5G15400
  - DML1
---
