---
figid: PMC8112447__fcell-09-660069-g007
figtitle: Zebrafish Cancer Predisposition Models
organisms:
- Staphylococcus aureus
- Mycobacterium avium complex sp.
- Paenibacillus larvae
- Paenibacillus larvae subsp. larvae
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
- Danio rerio
- Lareunionomyces loeiensis
- Panthera pardus
pmcid: PMC8112447
filename: fcell-09-660069-g007.jpg
figlink: /pmc/articles/PMC8112447/figure/F7/
number: F7
caption: Molecular complexes formed and DNA repair pathways mediated by CPG proteins.
  (A) Mechanisms of ATR and ATM activation by different types of DNA damage. Stalled
  replication forks resulting from a blockage of replisome progression (indicated
  by the “STOP” diagram). The single-stranded DNA (ssDNA) region is recognized by
  Replication Protein A (RPA) forming the filaments which can then recruit ATR-ATRIP
  (ATR Interacting Protein) complexes. The RAD17-RFC complex (not shown) loads the
  9-1-1 complex (RAD9-RAD1-HUS1) onto the ssDNA-dsDNA junction. RAD9 S387 phosphorylation
  provides a binding site for DNA topoisomerase 2-binding protein 1 (TopBP1), which
  in turn binds and activates ATR kinase. ATR phosphorylates and activates Chk1 kinase
  mediating many of its functional effects and directly phosphorylates many other
  target proteins. DNA double-strand breaks results in a complex signaling cascade
  resulting in binding of the MRN complex (MRE11-RAD50-NBN) to the DNA ends. MRN provides
  a DNA damage sensor platform for recruitment and activation of ATM, which then phosphorylates
  multiple targets including Chk2 responsible for cell cycle checkpoint activation
  and p53 activation. Chk2 and Chk1 can also functionally interact (hashed lines with
  arrows). p53 activation leads to apoptosis, cell cycle arrest and gene expression
  changes. (B) FA – BRCA2 pathway. A major role of this pathway is to respond to inter-strand
  crosslinks (ICL), which can be especially problematic during DNA replication since
  ICLs prevent replication fork convergence. At such blocked replication forks, TRAIP
  ubiquitin ligase ubiquitinates the CMG (CDC45-MCM-GINS) Helicase leading to its
  unloading. The ICL itself is recognized by FANCM / FAAP24 complex. FANCM then recruits
  the core FA complex components and FAAPs (FA associated proteins). E2 ubiquitin-conjugating
  enzyme FANCT and E3 ubiquitin ligase FANCL then cooperate to perform mono-ubiquitination
  of FANCD2 and FANCI forming the ID2 complex. Monoubiquitinated ID2 complex gets
  recruited the ICL lesion. In the process of unhooking, DNA near the ICL gets processed
  by nucleases to enable specific DNA repair pathways. The strand with the ICL structure
  gets repaired by Nucleotide Excision Repair (NER) and the translesion synthesis
  by REV1/Polζ, which gets recruited by the FA core complex. ID2 recruits FANCP (SLX4),
  FANCQ (ERCC1) and the nuclease FAN1 to mediate the ICL incision. The ID2 also interacts
  with BRCA2 (FANCD1), which gets recruited to the parts of blocked replication forks
  that will have to go through DNA recombination. BRCA2 in complex with BRCA1, BARD1,
  FANCN, FANCJ, RAD51 and RAD51C as well as other factors leads to loading of RAD51
  into resected ssDNA regions, which can then participate in homologous recombination
  (HR). The figure was constructed based on the information in several recent reviews
  (; ; ; ).
papertitle: Zebrafish Cancer Predisposition Models.
reftext: Kim Kobar, et al. Front Cell Dev Biol. 2021;9:660069.
year: '2021'
doi: 10.3389/fcell.2021.660069
journal_title: Frontiers in Cell and Developmental Biology
journal_nlm_ta: Front Cell Dev Biol
publisher_name: Frontiers Media S.A.
keywords: zebrafish | cancer predisposition | genetic models | cancer | model organism
  | p53
automl_pathway: 0.9294756
figid_alias: PMC8112447__F7
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
- Panthera pardus
- Danio rerio
- Drosophila melanogaster
redirect_from: /figures/PMC8112447__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8112447__fcell-09-660069-g007.html
  '@type': Dataset
  description: Molecular complexes formed and DNA repair pathways mediated by CPG
    proteins. (A) Mechanisms of ATR and ATM activation by different types of DNA damage.
    Stalled replication forks resulting from a blockage of replisome progression (indicated
    by the “STOP” diagram). The single-stranded DNA (ssDNA) region is recognized by
    Replication Protein A (RPA) forming the filaments which can then recruit ATR-ATRIP
    (ATR Interacting Protein) complexes. The RAD17-RFC complex (not shown) loads the
    9-1-1 complex (RAD9-RAD1-HUS1) onto the ssDNA-dsDNA junction. RAD9 S387 phosphorylation
    provides a binding site for DNA topoisomerase 2-binding protein 1 (TopBP1), which
    in turn binds and activates ATR kinase. ATR phosphorylates and activates Chk1
    kinase mediating many of its functional effects and directly phosphorylates many
    other target proteins. DNA double-strand breaks results in a complex signaling
    cascade resulting in binding of the MRN complex (MRE11-RAD50-NBN) to the DNA ends.
    MRN provides a DNA damage sensor platform for recruitment and activation of ATM,
    which then phosphorylates multiple targets including Chk2 responsible for cell
    cycle checkpoint activation and p53 activation. Chk2 and Chk1 can also functionally
    interact (hashed lines with arrows). p53 activation leads to apoptosis, cell cycle
    arrest and gene expression changes. (B) FA – BRCA2 pathway. A major role of this
    pathway is to respond to inter-strand crosslinks (ICL), which can be especially
    problematic during DNA replication since ICLs prevent replication fork convergence.
    At such blocked replication forks, TRAIP ubiquitin ligase ubiquitinates the CMG
    (CDC45-MCM-GINS) Helicase leading to its unloading. The ICL itself is recognized
    by FANCM / FAAP24 complex. FANCM then recruits the core FA complex components
    and FAAPs (FA associated proteins). E2 ubiquitin-conjugating enzyme FANCT and
    E3 ubiquitin ligase FANCL then cooperate to perform mono-ubiquitination of FANCD2
    and FANCI forming the ID2 complex. Monoubiquitinated ID2 complex gets recruited
    the ICL lesion. In the process of unhooking, DNA near the ICL gets processed by
    nucleases to enable specific DNA repair pathways. The strand with the ICL structure
    gets repaired by Nucleotide Excision Repair (NER) and the translesion synthesis
    by REV1/Polζ, which gets recruited by the FA core complex. ID2 recruits FANCP
    (SLX4), FANCQ (ERCC1) and the nuclease FAN1 to mediate the ICL incision. The ID2
    also interacts with BRCA2 (FANCD1), which gets recruited to the parts of blocked
    replication forks that will have to go through DNA recombination. BRCA2 in complex
    with BRCA1, BARD1, FANCN, FANCJ, RAD51 and RAD51C as well as other factors leads
    to loading of RAD51 into resected ssDNA regions, which can then participate in
    homologous recombination (HR). The figure was constructed based on the information
    in several recent reviews (; ; ; ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FANCD2
  - FANCI
  - TOPBP1
  - WARS1
  - ATR
  - ANTXR1
  - MMAB
  - SERPINA2
  - TRAIP
  - FANCM
  - FAAP100
  - REV1
  - ERVW-4
  - CHEK1
  - HFM1
  - CASK
  - MAP6
  - BRCA2
  - NR1H2
  - ATM
  - CHEK2
  - MRE11
  - NBN
  - RAD50
  - WDCP
  - TP53
  - TP63
  - TP73
  - SLX4
  - FAN1
  - FSCN1
  - ERCC1
  - BRCA1
  - PALB2
  - RAD51
  - RAD51C
  - BRIP1
  - Fancd2
  - Fanci
  - Topbp1
  - Gabrg2
  - Atr
  - Mmab
  - Traip
  - Fancm
  - Faap100
  - Rev1
  - Slc7a1
  - Mela
  - Gm42368
  - Chek1
  - Map6
  - Brca2
  - Atm
  - Tpm1
  - Chek2
  - Trp53
  - Slx4
  - Fan1
  - Fscn1
  - Ercc1
  - Brca1
  - Rad51
  - Xrcc2
  - Rad51c
  - Brip1
  - fancd2
  - fanci
  - topbp1
  - atr
  - traip
  - fancm
  - faap100
  - rev1
  - brca2
  - atm
  - tp53
  - slx4
  - fan1
  - ercc1
  - palb2
  - rad51
  - rad51c
  - brip1
  - mus101
  - anon-70Dc
  - mei-41
  - put
  - RYa-R
  - nopo
  - Polr2A
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - grp
  - stop
  - tefu
  - lok
  - CycE
  - cyc
  - mrn
  - p53
  - betaTub60D
  - hth
  - en
  - Rad51D
  - spn-B
  - spn-A
  - spn-D
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
---
