---
figid: PMC8803663__gr9
figtitle: Glutamine-Fructose-6-Phosphate Transaminase 2 (GFPT2) Is Upregulated in
  Breast Epithelial–Mesenchymal Transition and Responds to Oxidative Stress
organisms:
- Homo sapiens
- Mus musculus
- Agriosomyces longus
organisms_ner:
- Mus musculus
pmcid: PMC8803663
filename: gr9.jpg
figlink: /pmc/articles/PMC8803663/figure/fig9/
number: F9
caption: Summary of the study. The hexosamine biosynthesis pathway (HBP) and transsulfuration
  pathway, two main pathways associated with GFPT2 in this study, were illustrated.
  HBP is closely associated with the central carbon metabolism. The underscored metabolites
  were measured in this study. We used 1,2-13C Glc, 1-13C Gln, and 5-13C Gln in the
  13C tracing experiment to trace the carbon flux into UDP-GlcNAc. GFPT2 is the rate-limiting
  enzyme in the HBP and was upregulated in mesenchymal cells, especially in partial
  EMT. It affected the HBP flux, EMT program (e.g., VIM), cell growth, and cell invasion,
  and it was overexpressed in claudin-low breast cancer represented by the D492HER2
  cell line. GFPT2 was regulated by oxidative stress (H2O2 and GSH) and signaling
  regulators (insulin and EGF, and GSK3-β). Knockdown of GFPT2 with siRNAs decreased
  the cystathionine level and SQOR RNA expression in the transsulfuration pathway.
papertitle: Glutamine-Fructose-6-Phosphate Transaminase 2 (GFPT2) Is Upregulated in
  Breast Epithelial–Mesenchymal Transition and Responds to Oxidative Stress.
reftext: Qiong Wang, et al. Mol Cell Proteomics. 2022 Feb;21(2):100185.
year: '2022'
doi: 10.1016/j.mcpro.2021.100185
journal_title: 'Molecular & Cellular Proteomics : MCP'
journal_nlm_ta: Mol Cell Proteomics
publisher_name: American Society for Biochemistry and Molecular Biology
keywords: proteomics | EMT | GFPT2 | oxidative stress | claudin-low breast cancer
  | AKAP12, A-kinase anchor protein 12 | AKR1B1, Aldo-keto reductase family 1, member
  B1 (aldose reductase), isoform CRA_a | AKR1C1, Aldo-keto reductase family 1, member
  C1 | ALDH1A3, Aldehyde dehydrogenase 1 family, member A3, isoform CRA_a | ANXA3,
  Annexin A3 | BP, Biological process | BRENCs, Breast endothelial cells | CADM3,
  Cell adhesion molecule 3 | CCLE, Cancer Cell Line Encyclopedia | CDH1, Cadherin-1
  | CDH2, Cadherin-2 | CID, Collision-induced dissociation | CTGF, Connective tissue
  growth factor | DCD, Dermicidin | DDA, Data-dependent acquisition | DSP, Desmoplakin
  | DTT, Dithiothreitol | EDTA, Ethylenediaminetetraacetic acid | EGF, Epidermal growth
  factor | EMT, Epithelial-mesenchymal transition | EPCAM, Epithelial cell adhesion
  molecule | ERBB2 (HER2), Erb-B2 receptor tyrosine kinase 2 | ERK/MAPK, Mitogen-activated
  protein kinase | FASP, Filter-aided sample preparation | FBS, Fetal bovine serum
  | FDR, False discovery rate | FGFBP1, Fibroblast growth factor-binding protein 1
  | FLNC, Filamin-C | GALE, UDP-glucose 4-epimerase | GALNT7, N-acetylgalactosaminyltransferase
  7 | GES, Gene expression studies | GFPT1, Glutamine-fructose-6-phosphate aminotransferase
  1 | GFPT2, Glutamine-fructose-6-phosphate transaminase 2 | GlcNAc-P, N-acetylglucosamine
  phosphate | GLUT1, Glucose transporter 1 | GLUT4, Glucose transporter type 4 | GO,
  Gene ontology | GSH, Reduced glutathione | GSK3-β, Glycogen synthase kinase 3 beta
  | GSSG, Oxidized glutathione | H2O2, Hydrogen peroxide | H2S, Hydrogen sulfide |
  HBP, Hexosamine biosynthesis pathway | HER2 (ERBB2), Human epidermal growth factor
  receptor 2 | HMS LINCS, Harvard Medical School Library of Integrated Network-based
  Cellular Signatures | HPDL, 4-hydroxyphenylpyruvate dioxygenase-like protein | HUVECs,
  Human umbilical vein endothelial cells | IAA, Iodoacetamide | iBAQ, Intensity-based
  absolute quantification | IGF, Insulin like growth factor | IGF1R, Insulin like
  growth factor 1 receptor | IL18, Interleukin-18 | IPA, Ingenuity Pathway Analysis
  | ITGA6, Integrin subunit alpha 6 | ITGB4, Integrin subunit alpha 4 | K5/6/8/14/19,
  Keratin 5/6/8/14/19 | KEGG, Kyoto Encyclopedia of Genes and Genomes | KRAS, KRAS
  proto-oncogene, GTPase | KRT1, Keratin 1 | LAD1, Ladinin-1 | LAMA3, Laminin subunit
  alpha 3 | LAMB3, Laminin subunit beta 3 | LFQ, Label-free quantification | LKB1,
  Serine/threonine kinase 11 (STK11) | MGST1, Microsomal glutathione S-transferase
  1 | MMS, Mesenchymal metabolic signature | MYL9, Myosin light chain 9 | NDRG1, N-Myc
  downstream regulated 1 | NF-κB, Nuclear factor kappa B | NSCLC, Non-small-cell lung
  cancer | NT5E, 5′-Nucleotidase Ecto | ODC, Ornithine decarboxylase | OGT, O-Linked
  N-Acetylglucosamine (GlcNAc) Transferase | PBS, Phosphate-buffered saline | PCSK1N,
  Proprotein convertase subtilisin/kexin type 1 Inhibitor | PFA, Paraformaldehyde
  | PGM2L1, Glucose 1,6-bisphosphate synthase | PGM3, Phosphoacetylglucosamine mutase
  | PKCα, Protein kinase C alpha | PKP2, Plakophilin-2 | POMC, Pro-opiomelanocortin
  | PPP, Pentose phosphate pathway | PRSS23, Serine protease 23 | PVDF, Polyvinylidene
  fluoride | PYGB, Glycogen phosphorylase, brain form | RCN3, Reticulocalbin 3 | RELA,
  RELA Proto-Oncogene, NF-κB subunit, transcription factor p65 | RT, Room temperature
  | RTK, Receptor tyrosine kinase | RT-qPCR, Quantitative reverse transcription PCR
  | S100A14, S100 calcium binding protein A14 | S100A2, S100 calcium binding protein
  A2 | SD, Standard deviation | SDS, Sodium dodecyl sulfate | SERPINB5, Serpin family
  B member 5 | SERPINE1, Serpin family E member 1 | SILAC, Stable isotope labeling
  by amino acids in cell culture | SIRT6, Sirtuin 6 | SLC2A4, Solute carrier family
  2 member 4 | SLP-2, Stomatin-like protein 2 | SQOR, Sulfide:quinone oxidoreductase
  | STRING, Search Tool for the Retrieval of Interacting Genes/Proteins | sXBP1, Spliced
  X-box binding protein 1 | TAGLN, Transgelin | TCA, Tricarboxylic acid cycle | TCGA,
  The Cancer Genome Atlas | TFA, Trifluoroacetic acid | TGF-β, Transforming growth
  factor beta | TNFα, Tumor necrosis factor alpha | TWIST, Twist family BHLH transcription
  factor 1 | UAP1, UDP-N-acetylhexosamine pyrophosphorylase | UDP, Uridine diphosphate
  | UDP-Glc, UDP-glucose | UDP-GlcA, UDP-glucuronate | UDP-GlcNAc, UDP-N-acetylglucosamine
  | UGDH, UDP-glucose 6-dehydrogenase | UTP, Uridine-5′-triphosphate | VIM, Vimentin
automl_pathway: 0.9335788
figid_alias: PMC8803663__F9
figtype: Figure
redirect_from: /figures/PMC8803663__F9
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8803663__gr9.html
  '@type': Dataset
  description: Summary of the study. The hexosamine biosynthesis pathway (HBP) and
    transsulfuration pathway, two main pathways associated with GFPT2 in this study,
    were illustrated. HBP is closely associated with the central carbon metabolism.
    The underscored metabolites were measured in this study. We used 1,2-13C Glc,
    1-13C Gln, and 5-13C Gln in the 13C tracing experiment to trace the carbon flux
    into UDP-GlcNAc. GFPT2 is the rate-limiting enzyme in the HBP and was upregulated
    in mesenchymal cells, especially in partial EMT. It affected the HBP flux, EMT
    program (e.g., VIM), cell growth, and cell invasion, and it was overexpressed
    in claudin-low breast cancer represented by the D492HER2 cell line. GFPT2 was
    regulated by oxidative stress (H2O2 and GSH) and signaling regulators (insulin
    and EGF, and GSK3-β). Knockdown of GFPT2 with siRNAs decreased the cystathionine
    level and SQOR RNA expression in the transsulfuration pathway.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Itk
  - Slc22a3
  - Gnat2
  - Gfpt2
  - Gfpt1
  - Hps3
  - Egf
  - H20
  - H2-S
---
