---
figid: PMC8947103__cells-11-01048-g003
figtitle: 'Transposable Elements: Major Players in Shaping Genomic and Evolutionary
  Patterns'
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Sorghum bicolor
- Magnoliopsida
- Solanum melongena
- Pinus sylvestris
- Alocasia macrorrhizos
- Zymoseptoria tritici
- Drosophila melanogaster
- Drosophila virilis
- Biston betularia
- Drosophila buzzatii
- Drosophila miranda
- Human endogenous retroviruses
- Human endogenous retrovirus
- Mediterranean ruda virus
- Homo sapiens
- Pan troglodytes
- Mus musculus
- Danio rerio
- transposons
- Eutheria
pmcid: PMC8947103
filename: cells-11-01048-g003.jpg
figlink: /pmc/articles/PMC8947103/figure/cells-11-01048-f003/
number: F3
caption: 'Regulation of TEs in germline and soma under standard and stress conditions.
  (a) Stepwise model of transcriptional silencing guided by Piwi-interacting RNAs
  (piRNAs). In the germline of Drosophila the silencing of TEs is guided by specific,
  small RNAs organized in piRNA clusters enriched in TE sequences. At the core of
  the pathway is the piRNA-induced silencing complex (pi-RISC) that consists of a
  single-stranded piRNA bound to a Piwi family protein. Piwi (light green) can guide
  the transcriptional silencing of TEs through direct assembly of the complex at heterochromatin
  target sites. Drosophila harbors three Piwi-like proteins: Piwi, Aubergine (Aub,
  orange), and Argonaute 3 (Ago3, dark green), which, guided by piRNAs, silence TEs
  post-transcriptionally (in addition to transcriptional silencing referred to above)
  through homology-dependent cleavage. Anti-sense TE sequences are exported from the
  nucleus and processed into smaller fragments by Aub before being loaded into Ago3.
  The resulting piRNA–Ago3 complexes cleave newly antisense piRNA precursors from
  clusters loaded into Aub to produce anti-sense piRNAs, resulting in a “ping-pong”
  amplification cycle. Additionally, Aub–piRNA complexes can bind TE transcripts and
  repress their translation directly. (b) In the soma the repression of TEs is mediated
  predominantly by the small interfering RNA (siRNA) pathway. siRNA precursors form
  short hairpin structures before being processed and incorporated into the RNA-induced
  silencing complex (RISC). Once a siRNA binds to its target TE mRNA, it induces the
  cleavage of such an mRNA by RISC. Other silencing mechanisms involve chromatin remodeling.
  Several inhibitory marks, such as DNA methylation and histone methylation and deacetylation
  play an important role in repressing the mobilization of TEs. These epigenetic modifications
  may be passed on by dividing cells from one generation to the next. (c) When individuals
  are exposed to drastic environmental changes, they may experience stress-induced
  (re)activation of TEs. Epigenetic mechanisms like DNA methylation, histone modifications,
  expression of non-coding RNAs (ncRNAs) seem particularly relevant for this phenomenon.
  Additionally, environmental stress, such as heat shock, can induce de-repression
  of TEs by inducing the disruption of RISC through the action of the inducible Hsp70
  chaperone, which targets the complex to the lysosome. The generalized reactivation
  of TEs can generate genome instability leading to higher risk of disease when occurring
  in somatic cells and to infertility when arising in germ cells. However, in the
  latter it brings about increased genetic variability also, which is key for an adaptive
  response to extreme environmental change.'
papertitle: 'Transposable Elements: Major Players in Shaping Genomic and Evolutionary
  Patterns.'
reftext: Nunzia Colonna Romano, et al. Cells. 2022 Mar;11(6):1048.
year: '2022'
doi: 10.3390/cells11061048
journal_title: Cells
journal_nlm_ta: Cells
publisher_name: MDPI
keywords: transposable elements | environmental stress | evolution | epigenetics
automl_pathway: 0.8810076
figid_alias: PMC8947103__F3
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Arabidopsis thaliana
- Homo sapiens
redirect_from: /figures/PMC8947103__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8947103__cells-11-01048-g003.html
  '@type': Dataset
  description: 'Regulation of TEs in germline and soma under standard and stress conditions.
    (a) Stepwise model of transcriptional silencing guided by Piwi-interacting RNAs
    (piRNAs). In the germline of Drosophila the silencing of TEs is guided by specific,
    small RNAs organized in piRNA clusters enriched in TE sequences. At the core of
    the pathway is the piRNA-induced silencing complex (pi-RISC) that consists of
    a single-stranded piRNA bound to a Piwi family protein. Piwi (light green) can
    guide the transcriptional silencing of TEs through direct assembly of the complex
    at heterochromatin target sites. Drosophila harbors three Piwi-like proteins:
    Piwi, Aubergine (Aub, orange), and Argonaute 3 (Ago3, dark green), which, guided
    by piRNAs, silence TEs post-transcriptionally (in addition to transcriptional
    silencing referred to above) through homology-dependent cleavage. Anti-sense TE
    sequences are exported from the nucleus and processed into smaller fragments by
    Aub before being loaded into Ago3. The resulting piRNA–Ago3 complexes cleave newly
    antisense piRNA precursors from clusters loaded into Aub to produce anti-sense
    piRNAs, resulting in a “ping-pong” amplification cycle. Additionally, Aub–piRNA
    complexes can bind TE transcripts and repress their translation directly. (b)
    In the soma the repression of TEs is mediated predominantly by the small interfering
    RNA (siRNA) pathway. siRNA precursors form short hairpin structures before being
    processed and incorporated into the RNA-induced silencing complex (RISC). Once
    a siRNA binds to its target TE mRNA, it induces the cleavage of such an mRNA by
    RISC. Other silencing mechanisms involve chromatin remodeling. Several inhibitory
    marks, such as DNA methylation and histone methylation and deacetylation play
    an important role in repressing the mobilization of TEs. These epigenetic modifications
    may be passed on by dividing cells from one generation to the next. (c) When individuals
    are exposed to drastic environmental changes, they may experience stress-induced
    (re)activation of TEs. Epigenetic mechanisms like DNA methylation, histone modifications,
    expression of non-coding RNAs (ncRNAs) seem particularly relevant for this phenomenon.
    Additionally, environmental stress, such as heat shock, can induce de-repression
    of TEs by inducing the disruption of RISC through the action of the inducible
    Hsp70 chaperone, which targets the complex to the lysosome. The generalized reactivation
    of TEs can generate genome instability leading to higher risk of disease when
    occurring in somatic cells and to infertility when arising in germ cells. However,
    in the latter it brings about increased genetic variability also, which is key
    for an adaptive response to extreme environmental change.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - aub
  - AGO3
  - te
  - piwi
  - Me
  - CycE
  - cyc
  - Dcr-2
  - Dcr-1
  - ago
  - AGO1
  - AGO2
  - Tes
  - Camta
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Hsc70-3
  - Hsc70-1
  - Hsc70-4
  - Hsp70Ab
  - Hsp70Ba
  - Hsp70Aa
  - Hsp70Bb
  - Hsp70Bc
  - Hsp70Bbb
  - Hsp83
  - Gp93
  - Ago3
  - Scpep1
  - Fbxw7
  - Hspa1b
  - Hsp90ab1
  - Hsp90aa1
  - Hsp84-2
  - Hsp84-3
  - Hsp86-ps2
  - Hsp86-ps1
  - tm
  - WRNEXO
  - TES
  - HSP70
  - PIWIL1
  - H1-0
  - H1-1
  - H1-2
  - H1-3
  - H1-4
  - H1-5
  - H1-6
  - H1-7
  - H1-8
  - H1-10
  - .na.character
  - H2AB1
  - H2AJ
  - H2AZ2
  - H2AX
  - MACROH2A1
  - MACROH2A2
  - H2AZ1
  - H2AC1
  - H2AC6
  - H2AC7
  - H2AC12
  - H2AC14
  - H2AC21
  - H2AC20
  - H2AC25
  - H2BC1
  - H2BC3
  - H2BC5
  - H2BC9
  - H2BC11
  - H2BC12
  - H2BC13
  - H2BC14
  - H2BC15
  - H2BC17
  - H2BC18
  - H2BC19P
  - H2BC20P
  - H2BC21
  - H2BC26
  - H2BC12L
  - H2BW2
  - H2BW1
  - H3-4
  - H3-5
  - H4C7
  - SCPEP1
  - DICER1
  - FBXW7
  - HSPA1A
  - HSPA4
  - HSP90AA1
  - HSP90B1
  - TRAP1
  - HSP90AB1
---
