---
figid: PMC8618238__ijms-22-12462-g006
figtitle: Transcriptome Analysis of Insulin Signaling-Associated Transcription Factors
  in C. elegans Reveal Their Genome-Wide Target Genes Specificity and Complexity
organisms:
- Saccharomyces cerevisiae
- Escherichia coli
- Drosophila melanogaster
- Caenorhabditis elegans
- Symphonia globulifera
- Escherichia coli HT115
- Homo sapiens
organisms_ner:
- Drosophila melanogaster
- Caenorhabditis elegans
- Homo sapiens
pmcid: PMC8618238
filename: ijms-22-12462-g006.jpg
figlink: /pmc/articles/PMC8618238/figure/ijms-22-12462-f006/
number: F6
caption: Gene networks of DAF-16 and SKN-1-regulated genes under rIIS conditions.
  (A) DAF-16-dependent pathway network. Most of the genes are activated in FoxO and
  calcium signaling, while ribosome biogenesis, pyrimidine metabolism, phagosome,
  and glycerophospholipids-related genes were found to be in a repressed condition.
  (B) SKN-1-dependent pathway network. Longevity regulating genes are activated while
  phagosome, pentose, and glucuronate interconversion genes remain repressed. Activated
  and repressed genes are shown by green and red colors, respectively.
papertitle: Transcriptome Analysis of Insulin Signaling-Associated Transcription Factors
  in C. elegans Reveal Their Genome-Wide Target Genes Specificity and Complexity.
reftext: Neha Kaushik, et al. Int J Mol Sci. 2021 Nov;22(22):12462.
year: '2021'
doi: 10.3390/ijms222212462
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: FOXO/DAF-16 | NRF-2/SKN-1 | HSF1/HSF-1 | insulin-IGF-1 signaling | daf-2
  | C. elegans | transcriptomics | RNAi
automl_pathway: 0.6392322
figid_alias: PMC8618238__F6
figtype: Figure
redirect_from: /figures/PMC8618238__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC8618238__ijms-22-12462-g006.html
  '@type': Dataset
  description: Gene networks of DAF-16 and SKN-1-regulated genes under rIIS conditions.
    (A) DAF-16-dependent pathway network. Most of the genes are activated in FoxO
    and calcium signaling, while ribosome biogenesis, pyrimidine metabolism, phagosome,
    and glycerophospholipids-related genes were found to be in a repressed condition.
    (B) SKN-1-dependent pathway network. Longevity regulating genes are activated
    while phagosome, pentose, and glucuronate interconversion genes remain repressed.
    Activated and repressed genes are shown by green and red colors, respectively.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - vac
  - vid
  - rdgA
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - ha
  - da
  - nAChRalpha2
  - Wnt5
  - Wnt4
  - wg
  - Wnt2
  - lab
  - wa
  - mirr
  - fs(1)de1
  - foxo
  - hap
  - Hsp70Ab
  - Hsp70Ba
  - Hsp70Aa
  - Hsp70Bb
  - Hsp70Bc
  - Hsp70Bbb
  - Act79B
  - Act42A
  - Act57B
  - longevity
  - mec-7
  - hsp-70
  - act-4
  - act-2
  - act-3
  - mec-12
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - FOXO1
  - FOXO3
  - FOXO4
  - FOXO6
  - BAG1
  - RARB
  - SAFB
  - RTN3
  - HSPA1A
  - HSPA4
  - CCL4
---
