---
figid: PMC9075957__elife-72989-fig3
figtitle: Interacting genes form a diet-responsive network affecting adipogenesis
organisms:
- Homo sapiens
- Mus musculus
- Eutheria
pmcid: PMC9075957
filename: elife-72989-fig3.jpg
figlink: /pmc/articles/PMC9075957/figure/fig3/
number: F3
caption: (A) There are nine significant imprinting-by-imprinting epistatic ASE/DE/phenotype
  sets in the F16 advanced intercross population (n = 1002). Interactions are shown
  as lines connecting ASE (yellow) and DE genes (purple). Chromosome number is shown
  around the plot. (B) The epistatic parent-of-origin effect network is comprised
  of key steps in a putative pathway regulating differentiation and survival of adipocytes.
  This pathway was constructed by incorporating previously published cellular functions.
  The pathway members are color coded in blue for ASE genes (Plcd1, Nnat, and Cdkn1c)
  and green for DE genes (F2r, Hexb, Hmgcr, Car3, Tnfrsf11a, and Srgn). The network
  breaks down into potentiation, transduction, and response. Nnat and Hexb potentiate
  signaling by managing availability and accumulation of calcium necessary for signal
  transduction. Once a signal is received, F2r and Plcd1 transduce it by activating
  second messengers to initiate a response. This response initiates an adipogenesis
  cellular program that affects expression of Cdkn1c, Hmgcr, Car3, Tnfrsf11a, and
  Srgn.
papertitle: Parent-of-origin effects propagate through networks to shape metabolic
  traits.
reftext: Juan F Macias-Velasco, et al. eLife. 2022;11:e72989.
year: '2022'
doi: 10.7554/eLife.72989
journal_title: eLife
journal_nlm_ta: eLife
publisher_name: eLife Sciences Publications, Ltd
keywords: parent-of-origin effect | epistasis | adipose | RNA sequencing | metabolism
  | complex traits | Mouse
automl_pathway: 0.8662667
figid_alias: PMC9075957__F3
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
redirect_from: /figures/PMC9075957__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9075957__elife-72989-fig3.html
  '@type': Dataset
  description: (A) There are nine significant imprinting-by-imprinting epistatic ASE/DE/phenotype
    sets in the F16 advanced intercross population (n = 1002). Interactions are shown
    as lines connecting ASE (yellow) and DE genes (purple). Chromosome number is shown
    around the plot. (B) The epistatic parent-of-origin effect network is comprised
    of key steps in a putative pathway regulating differentiation and survival of
    adipocytes. This pathway was constructed by incorporating previously published
    cellular functions. The pathway members are color coded in blue for ASE genes
    (Plcd1, Nnat, and Cdkn1c) and green for DE genes (F2r, Hexb, Hmgcr, Car3, Tnfrsf11a,
    and Srgn). The network breaks down into potentiation, transduction, and response.
    Nnat and Hexb potentiate signaling by managing availability and accumulation of
    calcium necessary for signal transduction. Once a signal is received, F2r and
    Plcd1 transduce it by activating second messengers to initiate a response. This
    response initiates an adipogenesis cellular program that affects expression of
    Cdkn1c, Hmgcr, Car3, Tnfrsf11a, and Srgn.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Srgn
  - Plcd1
  - Hexb
  - Tnfrsf11a
  - Nnat
  - Car3
  - Hmgcr
  - F2r
  - Cdkn1c
  - Creb1
  - Ebp
  - SRGN
  - PLCD1
  - HEXB
  - TNFRSF11A
  - NNAT
  - CA3
  - HMGCR
  - F2R
  - ATP2A1
  - ATP2A2
  - ATP2A3
  - CDKN1C
  - CREB1
  - CREB3
  - CREB5
  - CREB3L1
  - CREB3L2
  - CREB3L3
  - CREB3L4
  - CREM
  - ATF1
  - ATF2
  - EBP
  - GLB1
  - SH3D19
---
