---
figid: PMC9359488__gr2_lrg
figtitle: Host genetic diversity and genetic variations of SARS-CoV-2 in COVID-19
  pathogenesis and the effectiveness of vaccination
organisms:
- NA
pmcid: PMC9359488
filename: gr2_lrg.jpg
figlink: /pmc/articles/PMC9359488/figure/f0010/
number: F2
caption: The roles of immune-related genes in COVID-19. As illustrated, some HLA variants
  are associated with susceptibility to SARS-CoV-2 such as HLA-DRB1*0301, DRB1*14:04,
  DRB1*01:01, DRB1*04:01, DRB1*07:01, HLA-B*4601, B*0703, B*51:01, and B*40:01. Also,
  heterozygote HLA-E*0101/0103 can attenuate the NKG2C+ NK cell responses. The infection
  of a cell with SARS-CoV-2 could result in the secretion of type I IFNs which consequently
  interact with adjacent IFNR1/2-expressing cells to initiate the signaling pathway
  ending in anti-viral responses; however, the presence of rs74956615 in TYK2__endocing
  a tyrosine kinase-related to IFNR1 signaling__ induces TYK2 overexpression, and
  consequently, hurts the lung cells. Moreover, rs10735079 in TYK2 is associated with
  reduced OAS expression and attenuated anti-viral responses. Notably, some SNPs in
  OAS and MX1 could elevate the risk of COVID-19, as well. Similarly, mutations in
  IRF7 and IRF9__components of IFN signaling pathway__ boost SARS-CoV-2 susceptibility.
  rs12252 in IFITM3 also attenuates the ability of IFITM3 to block the entrance of
  viral products into the cells. CCR1 and CCR2-MCP-1 could enhance the infiltartion
  of monocytes and macrophages and induce inflammation; with this regard, the pressence
  of activating variants in CCR1 and CCR2 has been observed in critically ill COVID-19
  patients. Besides, the lower expression of CXCR6 could reduce the infiltration of
  CD8+ T cells in the lung, leading to more severe disease. The mutations in TLRs
  such as the deletion in TICAM2, which is related to TLR-4 initial signaling, could
  influence the risk of COVID-19. Similarly, loss-of-function of TLR-3 and TLR-7 impairs
  the IFNs responses. The overexpression of complement system components is another
  agent magnifying the severity of the disease. In patients carrying rs11385949, the
  levels of upper airway viral load and sC5b-9 were greater than others. Regarding
  the roles of DBP in the regulation of vitamin D levels, patients harboring GC rs2282679
  could experience the severe type of the disease. Furthermore, The NK cells isolated
  from severe cases of COVID-19 have shown higher expression of NKG2C and secretion
  of perforin. Moreover, it was shown that Cit-H3 has significant roles in NET formation
  and consequently higher inflammation and cardiovascular complications.
papertitle: Host genetic diversity and genetic variations of SARS-CoV-2 in COVID-19
  pathogenesis and the effectiveness of vaccination.
reftext: Mahda Delshad, et al. Int Immunopharmacol. 2022 Oct;111:109128-109128.
year: '2022'
doi: 10.1016/j.intimp.2022.109128
journal_title: International Immunopharmacology
journal_nlm_ta: Int Immunopharmacol
publisher_name: Elsevier B.V.
keywords: SARS-CoV-2 | COVID-19 | ACE2 | Genetic diversity | Mutation | Vaccine
automl_pathway: 0.9461434
figid_alias: PMC9359488__F2
figtype: Figure
organisms_ner:
- Homo sapiens
redirect_from: /figures/PMC9359488__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9359488__gr2_lrg.html
  '@type': Dataset
  description: The roles of immune-related genes in COVID-19. As illustrated, some
    HLA variants are associated with susceptibility to SARS-CoV-2 such as HLA-DRB1*0301,
    DRB1*14:04, DRB1*01:01, DRB1*04:01, DRB1*07:01, HLA-B*4601, B*0703, B*51:01, and
    B*40:01. Also, heterozygote HLA-E*0101/0103 can attenuate the NKG2C+ NK cell responses.
    The infection of a cell with SARS-CoV-2 could result in the secretion of type
    I IFNs which consequently interact with adjacent IFNR1/2-expressing cells to initiate
    the signaling pathway ending in anti-viral responses; however, the presence of
    rs74956615 in TYK2__endocing a tyrosine kinase-related to IFNR1 signaling__ induces
    TYK2 overexpression, and consequently, hurts the lung cells. Moreover, rs10735079
    in TYK2 is associated with reduced OAS expression and attenuated anti-viral responses.
    Notably, some SNPs in OAS and MX1 could elevate the risk of COVID-19, as well.
    Similarly, mutations in IRF7 and IRF9__components of IFN signaling pathway__ boost
    SARS-CoV-2 susceptibility. rs12252 in IFITM3 also attenuates the ability of IFITM3
    to block the entrance of viral products into the cells. CCR1 and CCR2-MCP-1 could
    enhance the infiltartion of monocytes and macrophages and induce inflammation;
    with this regard, the pressence of activating variants in CCR1 and CCR2 has been
    observed in critically ill COVID-19 patients. Besides, the lower expression of
    CXCR6 could reduce the infiltration of CD8+ T cells in the lung, leading to more
    severe disease. The mutations in TLRs such as the deletion in TICAM2, which is
    related to TLR-4 initial signaling, could influence the risk of COVID-19. Similarly,
    loss-of-function of TLR-3 and TLR-7 impairs the IFNs responses. The overexpression
    of complement system components is another agent magnifying the severity of the
    disease. In patients carrying rs11385949, the levels of upper airway viral load
    and sC5b-9 were greater than others. Regarding the roles of DBP in the regulation
    of vitamin D levels, patients harboring GC rs2282679 could experience the severe
    type of the disease. Furthermore, The NK cells isolated from severe cases of COVID-19
    have shown higher expression of NKG2C and secretion of perforin. Moreover, it
    was shown that Cit-H3 has significant roles in NET formation and consequently
    higher inflammation and cardiovascular complications.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CXCL16
  - TYK2
  - IRF9
  - STAT2
  - IFNA1
  - IFNAR2
  - IRF7
  - IRF3
  - IFNG
  - CD8A
  - CD8B
  - TLR3
  - IDUA
  - TLR7
  - KLRC2
  - DBP
  - GC
  - HSD17B4
  - CCL2
  - CCR2
  - MYD88
  - MXI1
  - TLR4
  - TICAM2
  - TMED7-TICAM2
  - ELK3
  - EPHB1
  - SLC6A2
  - ERBB2
  - NEU1
  - NEURL1
  - FGA
  - FGB
  - FGG
  - APP
  - SUCLA2
  - C6
  - CSH2
  - ERCC6
  - C9
  - FUT3
  - IFITM3
  - HLA-DRB1
---
