---
figid: PMC9368002__cells-11-02369-g001
figtitle: Rewiring of Glucose and Lipid Metabolism Induced by G Protein-Coupled Receptor
  17 Silencing Enables the Transition of Oligodendrocyte Progenitors to Myelinating
  Cells
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC9368002
filename: cells-11-02369-g001.jpg
figlink: /pmc/articles/PMC9368002/figure/cells-11-02369-f001/
number: F1
caption: Ingenuity pathway analysis tool (IPA, Qiagen) was used to analyse dataset
  and explore the diseases and biological processes which are predicted to be increasing
  or decreasing based on the pattern of differentially expressed genes in the dataset.
  From this analysis we extrapolated the scheme in the Figure that includes the genes
  related to lipid synthesis differentially expressed in our dataset (p-value = 9.67
  × 10−4). The full names of these genes, together with the relative fold change (log2
  ratio) are reported in . In red, upregulated genes; in green, downregulated genes.
papertitle: Rewiring of Glucose and Lipid Metabolism Induced by G Protein-Coupled
  Receptor 17 Silencing Enables the Transition of Oligodendrocyte Progenitors to Myelinating
  Cells.
reftext: Davide Marangon, et al. Cells. 2022 Aug;11(15):2369.
year: '2022'
doi: 10.3390/cells11152369
journal_title: Cells
journal_nlm_ta: Cells
publisher_name: MDPI
keywords: oligodendrocyte progenitor cell | oligodendrocyte | energy metabolism |
  glycolysis | lactate | myelin lipids | transcriptomics | metabolomics | lipidomics
  | myelination
automl_pathway: 0.8810291
figid_alias: PMC9368002__F1
figtype: Figure
redirect_from: /figures/PMC9368002__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9368002__cells-11-02369-g001.html
  '@type': Dataset
  description: Ingenuity pathway analysis tool (IPA, Qiagen) was used to analyse dataset
    and explore the diseases and biological processes which are predicted to be increasing
    or decreasing based on the pattern of differentially expressed genes in the dataset.
    From this analysis we extrapolated the scheme in the Figure that includes the
    genes related to lipid synthesis differentially expressed in our dataset (p-value
    = 9.67 × 10−4). The full names of these genes, together with the relative fold
    change (log2 ratio) are reported in . In red, upregulated genes; in green, downregulated
    genes.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - VEGFA
  - SREBF1
  - SPTSSB
  - SERINC2
  - WNT4
  - S1PR3
  - WNT5A
  - ACSL3
  - RGS2
  - RIEG2
  - PTGDS
  - ACSM1
  - PNPLA3
  - CD40
  - PHKA1
  - CXCL12
  - CXCL10
  - PGF
  - OPRL1
  - CYP7B1
  - NR1H3
  - NGFR
  - DAB2
  - MOG
  - ELOVL2
  - KLF5
  - GAD2
  - KDR
  - GGT1
  - GREM2
  - GRM5
  - APP
  - IGFBP2
---
