---
figid: PMC9613846__12284_2022_600_Fig3_HTML
figtitle: Transcriptome and Metabolome Analyses Reveals the Pathway and Metabolites
  of Grain Quality Under Phytochrome B in Rice (Oryza sativa L.)
organisms:
- NA
pmcid: PMC9613846
filename: 12284_2022_600_Fig3_HTML.jpg
figlink: /pmc/articles/PMC9613846/figure/Fig3/
number: F3
caption: 'Schematic model of gene regulatory networks for the grain size and chalkiness
  formation of osphyb. Pathway: Glycolysis, TCA cycle, starch synthesis, amino acid
  metabolism, fatty acid synthesis, plant hormones, oxidative phosphorylation, antioxidation,
  translation regulation. Genes encoding enzyme (s) are indicated in orange boxes,
  metabolites and other materials in white boxes. The expression patterns of the genes
  encoding corresponding enzyme (s) are indicated above or below the arrows. Each
  row (6 boxes) represents a DEG. Six boxes in each row represent 3 biological repeats
  of one gene in NIP (first three boxes) and osphyb (last three boxes), respectively.
  The enriched DEGs boxes for 40S, 50S, 60S, Ribosome biogenesis and RNA polymerase
  are rotated 90° clockwise. Log2 (fold changes) are represented by a colour scale
  from green (down-regulated) to red (up-regulated)'
papertitle: Transcriptome and Metabolome Analyses Reveals the Pathway and Metabolites
  of Grain Quality Under Phytochrome B in Rice (Oryza sativa L.).
reftext: Fei Li, et al. Rice (N Y). 2022;15:52.
year: '2022'
doi: 10.1186/s12284-022-00600-5
journal_title: Rice
journal_nlm_ta: Rice (N Y)
publisher_name: Springer US
keywords: OsPHYB | Chalkiness | Grain size | Transcriptome | Metabolome | Correlation
  analysis
automl_pathway: 0.9233475
figid_alias: PMC9613846__F3
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC9613846__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9613846__12284_2022_600_Fig3_HTML.html
  '@type': Dataset
  description: 'Schematic model of gene regulatory networks for the grain size and
    chalkiness formation of osphyb. Pathway: Glycolysis, TCA cycle, starch synthesis,
    amino acid metabolism, fatty acid synthesis, plant hormones, oxidative phosphorylation,
    antioxidation, translation regulation. Genes encoding enzyme (s) are indicated
    in orange boxes, metabolites and other materials in white boxes. The expression
    patterns of the genes encoding corresponding enzyme (s) are indicated above or
    below the arrows. Each row (6 boxes) represents a DEG. Six boxes in each row represent
    3 biological repeats of one gene in NIP (first three boxes) and osphyb (last three
    boxes), respectively. The enriched DEGs boxes for 40S, 50S, 60S, Ribosome biogenesis
    and RNA polymerase are rotated 90° clockwise. Log2 (fold changes) are represented
    by a colour scale from green (down-regulated) to red (up-regulated)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pl
  - pi
  - adp
  - gr
  - so
  - Sod2
  - Sod1
  - Arp1
  - Arfrp1
  - Arl1
  - Arf1
  - Arl2
  - PHGPx
  - SdhA
  - Loxl2
  - Loxl1
  - barr
  - ATPsynbeta
  - Atpalpha
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - Fancd2
  - trx
  - Trxr-1
  - Trx-2
  - CG11700
  - srp
  - Pp2C1
  - CG17598
  - CG7115
  - CG10376
  - Phlpp
  - Ppm1
  - CG17746
  - GstS1
  - GstE12
  - GstD1
  - GstD2
  - GstD3
  - GstD4
  - GstD5
  - GstD6
  - GstD7
  - GstD8
  - cona
  - l(1)sc
  - CycE
  - cyc
  - eRF1
  - Mdh1
  - Coa
  - bsk
  - CAP
  - SMC3
  - Cpn
  - Capa
  - capt
  - bel
  - SERCA
  - Cadps
  - eIF2alpha
  - eRF3
  - kra
  - pk
  - fab1
  - grn
  - row
  - min
  - mnb
  - .na.character
  - Hk
  - Eno
  - Acsf3
  - eIF3g1
  - eIF3b
  - eIF3c
  - eIF3a
  - eIF3h
  - grh
  - eIF1
  - eIF2beta
  - Pglym78
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - ww
  - anon-70Dc
  - tRNA:Lys-TTT-1-1
  - tRNA:Lys-TTT-2-4
  - tRNA:Lys-TTT-2-1
  - tRNA:Lys-TTT-2-2
  - tRNA:Lys-TTT-2-5
  - tRNA:Lys-TTT-2-3
  - su(sable)
  - Gapdh2
  - Gpdh1
  - Gapdh1
  - Max
  - mof
  - Skp2
  - cb
  - Cyt-c-d
  - Cyt-c-p
  - cype
  - tRNA:Val-TAC-1-2
  - tRNA:Val-TAC-1-1
  - Pep
  - bgcn
  - LanB2
  - anon-70Db
  - DECR1
  - WDTC1
  - SOD1
  - SOD2
  - SOD3
  - PITX2
  - NR2F2
  - ANGPTL1
  - ACTR1A
  - ARFRP1
  - APOBEC2
  - CDKN2A
  - GPX1
  - GPX2
  - GPX3
  - GPX4
  - GPX5
  - GPX6
  - GPX7
  - GPX8
  - SDHA
  - LOX
  - GCGR
  - GLRA2
  - ATP8A2
  - BRCA2
  - FANCD2
  - PSEN1
  - RMND5B
  - RMND5A
  - TXN
  - VAC14
  - EBF2
  - EBF1
  - SLCO6A1
  - GSTK1
  - GSTA1
  - GSTA2
  - GSTA3
  - GSTA4
  - GSTA5
  - GSTM1
  - GSTM2
  - GSTM3
  - GSTM4
  - GSTM5
  - GSTO1
  - GSTO2
  - GSTP1
  - GSTT1
  - GSTT2
  - GSTZ1
  - GSTT2B
  - ARHGEF1
  - ZFP36L1
  - ETF1
  - TFAP2C
  - ZFP36L2
  - MDH1
  - BAK1
  - BIN2
  - CALCR
  - SLC31A1
  - CCDC174
  - ELF4
  - EIF4G1
  - EIF4G2
  - EIF4G3
  - EIF4E
  - EIF4E2
  - EIF4E3
  - EIF4A1
  - EIF4A2
  - EIF4A3
  - CTAA1
  - LNPEP
  - SERPINB6
  - HACD1
  - CAP1
  - SORBS1
  - BRD4
  - CAP2
  - SPTBN1
  - GALM
  - ELDR
  - PIKFYVE
  - DLD
  - FAM3A
  - LDHD
  - ISCA1
  - ENO1
  - ENO2
  - ENO3
  - ENO4
  - ACSF3
  - HSD17B8
  - EIF3A
  - ELF1
  - EFNA2
  - ELOF1
  - EIF1
  - ELF2
  - EIF2S1
  - EIF2S3
  - EIF2S2
  - PGAM1
  - PGAM2
  - MPG
  - WARS1
  - APP
  - AAAS
  - AAA1
  - GAPDH
  - GAPDHP44
  - ERVK-9
  - ERVK-11
  - ERVK-19
  - ALDOA
  - ALDOB
  - ALDOC
  - MAX
  - CYCS
  - DDX21
  - PAEP
  - PREP
  - PTPN22
---
