---
figid: PMC9544697__NPH-235-1853-g006
figtitle: Thiamine metabolism genes in diatoms are not regulated by thiamine despite
  the presence of predicted riboswitches
organisms:
- Phaeodactylum tricornutum
- Homo sapiens
organisms_ner:
- Homo sapiens
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC9544697
filename: NPH-235-1853-g006.jpg
figlink: /pmc/articles/PMC9544697/figure/nph18296-fig-0002/
number: F2
caption: Proposed routes for thiamine biosynthesis in diatoms. (a) A tBlastn search
  using selected algal peptide sequences as queries (please refer to Supporting Information
  Table c) was performed against 19 diatom genomes to determine the presence (full
  circle P‐value > 10−20; half‐full circle P‐value > 10−3) or absence (empty circle)
  of different thiamine‐related genes. The presence of an associated predicted riboswitch
  in the 3′UTR of the gene is indicated with a hairpin symbol at the right of the
  circle. The genome abbreviations, accession numbers and references can be found
  in Table . (b) Potential thiamine biosynthetic, salvage and uptake routes in diatoms.
  The pathway steps with strong support across the diatom lineage are shown in green.
  AIR, 5‐aminoimidazole ribotide; dA, 5′‐deoxyadenosine; DXP, 1‐deoxy‐d‐xylulose 5‐phosphate;
  FAMP, N‐formyl‐4‐amino‐5‐aminomethyl‐2‐methylpyrimidine; GA3P, glyceraldehyde 3‐phosphate;
  HET‐P, hydroxyethyl‐thiazole phosphate; HMP‐P, hydroxymethyl‐pyrimidine phosphate;
  HMP‐PP, hydroxymethyl‐pyrimidine pyrophosphate; l‐Met, l‐methionine; NAD, nicotinamide
  adenine dinucleotide; PLP, pyridoxal 5′‐phosphate; SAM, S‐adenosyl methionine; TMP,
  thiamine monophosphate; TPP, thiamine pyrophosphate. &THI5/NMT1 candidates contain
  an NMT1 Pfam domain (PF09084). $THIG and THIS are encoded in the chloroplast in
  Phaeodactylum tricornutum, so the results can be biased in genomes that do not include
  chloroplast sequences. +THID, THIE and HMPK functions are performed by a single
  peptide in diatoms (TH1). *In some bacteria TenI accelerates a thiazole tautomerisation
  reaction, but it is not necessary to synthesise HET‐P (Hazra et al., ).
papertitle: Thiamine metabolism genes in diatoms are not regulated by thiamine despite
  the presence of predicted riboswitches.
reftext: Marcel Llavero‐Pasquina, et al. New Phytol. 2022 Sep;235(5):1853-1867.
year: '2022'
doi: 10.1111/nph.18296
journal_title: The New Phytologist
journal_nlm_ta: New Phytol
publisher_name: John Wiley and Sons Inc.
keywords: aptamer prediction | CRISPR/Cas9 | diatoms | Phaeodactylum tricornutum |
  thiamine biosynthesis | thiamine uptake | TPP riboswitch
automl_pathway: 0.9374431
figid_alias: PMC9544697__F2
figtype: Figure
redirect_from: /figures/PMC9544697__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9544697__NPH-235-1853-g006.html
  '@type': Dataset
  description: Proposed routes for thiamine biosynthesis in diatoms. (a) A tBlastn
    search using selected algal peptide sequences as queries (please refer to Supporting
    Information Table c) was performed against 19 diatom genomes to determine the
    presence (full circle P‐value > 10−20; half‐full circle P‐value > 10−3) or absence
    (empty circle) of different thiamine‐related genes. The presence of an associated
    predicted riboswitch in the 3′UTR of the gene is indicated with a hairpin symbol
    at the right of the circle. The genome abbreviations, accession numbers and references
    can be found in Table . (b) Potential thiamine biosynthetic, salvage and uptake
    routes in diatoms. The pathway steps with strong support across the diatom lineage
    are shown in green. AIR, 5‐aminoimidazole ribotide; dA, 5′‐deoxyadenosine; DXP,
    1‐deoxy‐d‐xylulose 5‐phosphate; FAMP, N‐formyl‐4‐amino‐5‐aminomethyl‐2‐methylpyrimidine;
    GA3P, glyceraldehyde 3‐phosphate; HET‐P, hydroxyethyl‐thiazole phosphate; HMP‐P,
    hydroxymethyl‐pyrimidine phosphate; HMP‐PP, hydroxymethyl‐pyrimidine pyrophosphate;
    l‐Met, l‐methionine; NAD, nicotinamide adenine dinucleotide; PLP, pyridoxal 5′‐phosphate;
    SAM, S‐adenosyl methionine; TMP, thiamine monophosphate; TPP, thiamine pyrophosphate.
    &THI5/NMT1 candidates contain an NMT1 Pfam domain (PF09084). $THIG and THIS are
    encoded in the chloroplast in Phaeodactylum tricornutum, so the results can be
    biased in genomes that do not include chloroplast sequences. +THID, THIE and HMPK
    functions are performed by a single peptide in diatoms (TH1). *In some bacteria
    TenI accelerates a thiazole tautomerisation reaction, but it is not necessary
    to synthesise HET‐P (Hazra et al., ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SMAD1
  - BSPH1
  - PLP1
  - PTHLH
  - PRDX5
  - PDXP
  - IMMT
  - NMT1
  - ACAA1
  - NELFCD
  - AIR
  - AIRN
  - SLC6A20
  - NFS1
  - SLC22A18
  - SAFB
  - TXNIP
  - EMP1
  - ACAT1
  - NR0B1
  - MPZ
  - PMP22
  - FOLR1
  - thiS
  - hmp
  - thiC
  - thiD
  - thiH
  - aer
  - iscS
  - thiG
  - thiE
  - thiF
  - thiI
  - dxs
  - thiM
  - thiL
  - deoA
---
