---
figid: PMC9303011__fonc-12-914594-g002
figtitle: 'Constraint-Based Reconstruction and Analyses of Metabolic Models: Open-Source
  Python Tools and Applications to Cancer'
organisms:
- NA
pmcid: PMC9303011
filename: fonc-12-914594-g002.jpg
figlink: /pmc/articles/PMC9303011/figure/f2/
number: F2
caption: Overview of Python software for major components of COBRA methods. Constraint-based
  metabolic modeling first requires loading a metabolic model into software that handles
  the various parts of the modeling framework (grey), such as metabolites, reactions,
  genes, stoichiometric matrix, and flux solutions. New metabolic models can be reconstructed
  from genome sequences and database, quality-checked by model testing software, made
  consistent using gap-filling tools, and visualized using web-based packages. Using
  the metabolic model, FBA (yellow) finds an optimal flux distribution that follows
  stoichiometry under steady state and can further be extended to dynamic systems.
  Since there are alternative optima (blue) to FBA, FVA and geometric FBA can be used
  to characterize the solution space. We can perturb (red) the system to predict the
  effect of knockouts and use such predictions to design an optimal system (‘strain’).
  To improve FBA predictions, we can add biophysical (green) constraints based on
  thermodynamics, proteins, and macromolecular expression. Metabolic modeling can
  be further enhanced by integration of multi-omics (purple) data, such as extracting
  reduced models based on omics data and adding regulatory constraints. Using omics
  data, metabolic modeling can become high-dimensional (brown), through single cell
  modeling and community modeling. Multiple metabolic models can be reduced into ensemble
  objects. In contrast to FBA, unbiased (pink) approaches do not require an objective
  function. These include methods for sampling flux distributions and pathway analyses.
  Names of software packages are in bold.
papertitle: 'Constraint-Based Reconstruction and Analyses of Metabolic Models: Open-Source
  Python Tools and Applications to Cancer.'
reftext: Rachel H. Ng, et al. Front Oncol. 2022;12:914594.
year: '2022'
doi: 10.3389/fonc.2022.914594
journal_title: Frontiers in Oncology
journal_nlm_ta: Front Oncol
publisher_name: Frontiers Media S.A.
keywords: cancer | metabolism | constraint-based modeling | genome-scale metabolic
  models | systems biology | omics | python | single-cell analysis
automl_pathway: 0.7528276
figid_alias: PMC9303011__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC9303011__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9303011__fonc-12-914594-g002.html
  '@type': Dataset
  description: Overview of Python software for major components of COBRA methods.
    Constraint-based metabolic modeling first requires loading a metabolic model into
    software that handles the various parts of the modeling framework (grey), such
    as metabolites, reactions, genes, stoichiometric matrix, and flux solutions. New
    metabolic models can be reconstructed from genome sequences and database, quality-checked
    by model testing software, made consistent using gap-filling tools, and visualized
    using web-based packages. Using the metabolic model, FBA (yellow) finds an optimal
    flux distribution that follows stoichiometry under steady state and can further
    be extended to dynamic systems. Since there are alternative optima (blue) to FBA,
    FVA and geometric FBA can be used to characterize the solution space. We can perturb
    (red) the system to predict the effect of knockouts and use such predictions to
    design an optimal system (‘strain’). To improve FBA predictions, we can add biophysical
    (green) constraints based on thermodynamics, proteins, and macromolecular expression.
    Metabolic modeling can be further enhanced by integration of multi-omics (purple)
    data, such as extracting reduced models based on omics data and adding regulatory
    constraints. Using omics data, metabolic modeling can become high-dimensional
    (brown), through single cell modeling and community modeling. Multiple metabolic
    models can be reduced into ensemble objects. In contrast to FBA, unbiased (pink)
    approaches do not require an objective function. These include methods for sampling
    flux distributions and pathway analyses. Names of software packages are in bold.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ald1
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Max
  - mof
  - ml
  - vib
  - COX6B
  - nAChRbeta1
  - nAChRalpha1
  - prom
  - Set
  - I-2
  - ct
  - double
  - ag
  - tf
  - min
  - mnb
  - .na.character
  - app
  - Appl
  - ApepP
  - APP-BP1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - FBXO3
  - RASA1
  - RGS6
  - MAX
  - F11
  - PTCRA
  - SET
  - ANGPTL6
  - HOPX
  - TF
  - CTSB
---
