---
figid: PMC9986505__gr1
figtitle: Application of the PHENotype SIMulator for rapid identification of potential
  candidates in effective COVID-19 drug repurposing
organisms:
- NA
pmcid: PMC9986505
filename: gr1.jpg
figlink: /pmc/articles/PMC9986505/figure/fig1/
number: F1
caption: 'Schematic representation of the PHENSIM Drug repurposing Strategy. Outline
  for our approach to acquire a cell-specific viral signature in silico and formulate
  repurposing hypotheses: (a) first, logFold Changes (logFCs) of Differentially Expressed
  Genes (DEGs) arising from transcriptomic genome wide expression analysis of infected
  vs. baseline uninfected cells are used to represent a virus in the meta-pathway;
  (b) then, we run the PHENSIM simulation by upregulating the viral node; (c) therefore,
  we collect all perturbation values computed by PHENSIM for pathway endpoints to
  define a cell-specific pathogen signature. (d) The same process is applied to expression
  data arising from whole transcriptome-wide expression analysis of treated vs. mock-treated
  cell lines, (e) to perform a PHENSIM simulation of the drug activity, (f) yielding
  a cell-specific drug signature. (g) Thus, this process is iterated for each drug
  we wish to test and collected in a database of drug signatures. (h) Finally, a Pearson
  correlation analysis between the pathogen and each drug signature is used to score
  repurposing candidates, yielding hypothesis for further laboratory tests. In panels
  (a) and (d), we report upregulated DEGs in red and downregulated ones in blue. In
  panels (c), (f), and (g), we report positively perturbated endpoints in green and
  negatively perturbated endpoints in blue. Finally, in panel (h), negative correlation
  (reported in green) predicts promising drug candidates that inhibit the pathogen
  signature and positive correlation (reported in red) suggests exacerbation of the
  viral signature when introducing the drug.'
papertitle: Application of the PHENotype SIMulator for rapid identification of potential
  candidates in effective COVID-19 drug repurposing.
reftext: Naomi I. Maria, et al. Heliyon. 2023 Mar 6 ;9(3):e14115.
year: '2023'
doi: 10.1016/j.heliyon.2023.e14115
journal_title: Heliyon
journal_nlm_ta: Heliyon
publisher_name: Elsevier
keywords: COVID-19 | Drug repurposing | Systems biology | Cellular simulation models
  | Cellular SARS-CoV-2 signatures | Cellular host-immune response | PHENSIM, PHENotype
  SIMulator | SARS-CoV-2, Severe acute respiratory syndrome coronavirus 2 | COVID-19,
  Coronavirus disease 2019 | DEGs, Differentially Expressed Genes | ACE2, Angiotensin-converting
  enzyme 2 | IFN, Interferon | NHBE, Normal human bronchial epithelial cells | MOI,
  Multiplicity of infection | ISGs, IFN-stimulated genes | MITHrIL, Mirna enrIched
  paTHway Impact anaLysis | TLR, Toll-like Receptor | Calu-3, Epithelial cell line
  | Caco-2, Human colon epithelial carcinoma cell line | DEPs, Differentially expressed
  proteins | HCQ-CQ, (Hydroxy)chloroquine | 2DG, 2-Deoxy-Glucose | MP, Methylprednisolone
automl_pathway: 0.9025821
figid_alias: PMC9986505__F1
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC9986505__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9986505__gr1.html
  '@type': Dataset
  description: 'Schematic representation of the PHENSIM Drug repurposing Strategy.
    Outline for our approach to acquire a cell-specific viral signature in silico
    and formulate repurposing hypotheses: (a) first, logFold Changes (logFCs) of Differentially
    Expressed Genes (DEGs) arising from transcriptomic genome wide expression analysis
    of infected vs. baseline uninfected cells are used to represent a virus in the
    meta-pathway; (b) then, we run the PHENSIM simulation by upregulating the viral
    node; (c) therefore, we collect all perturbation values computed by PHENSIM for
    pathway endpoints to define a cell-specific pathogen signature. (d) The same process
    is applied to expression data arising from whole transcriptome-wide expression
    analysis of treated vs. mock-treated cell lines, (e) to perform a PHENSIM simulation
    of the drug activity, (f) yielding a cell-specific drug signature. (g) Thus, this
    process is iterated for each drug we wish to test and collected in a database
    of drug signatures. (h) Finally, a Pearson correlation analysis between the pathogen
    and each drug signature is used to score repurposing candidates, yielding hypothesis
    for further laboratory tests. In panels (a) and (d), we report upregulated DEGs
    in red and downregulated ones in blue. In panels (c), (f), and (g), we report
    positively perturbated endpoints in green and negatively perturbated endpoints
    in blue. Finally, in panel (h), negative correlation (reported in green) predicts
    promising drug candidates that inhibit the pathogen signature and positive correlation
    (reported in red) suggests exacerbation of the viral signature when introducing
    the drug.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - lep
  - Dr
  - ali
  - alpha-Est7
  - gus
  - E(spl)malpha-BFM
  - px
  - ave
---
