---
figid: PMC9328443__TPJ-103-512-g003
figtitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks'
organisms:
- NA
pmcid: PMC9328443
filename: TPJ-103-512-g003.jpg
figlink: /pmc/articles/PMC9328443/figure/tpj14746-fig-0004/
number: F4
caption: Functional SNPs in the lignin biosynthesis pathway of P. trichocarpa in three
  representative genotypes.Distribution of SNPs in three representative genotypes
  which have a low (RGR = 0.04), average (RGR = 7.25), and high (RGR = 15.96) growth
  rate. Metabolite names are shown in blue, gene IDs in black and corresponding enzymes
  in orange. Activating SNPs are marked in green and inactivating SNPs in red below
  the gene that they feature in. The net fold‐change in flux (increase/decrease) through
  a reaction due to SNPs is given next to it, with a flux increase shown in green
  with an upward arrow and a flux decrease shown in red with a downward arrow. CCoAMT,
  caffeoyl‐CoA O‐methyltransferase; CCR, cinnamoyl‐CoA reductase; F5H, ferulate 5‐hydroxylase;
  HCT, shikimate hydroxycinnamoyltransferase; PAL, phenylalanine ammonia‐lyase.
papertitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks.'
reftext: Debolina Sarkar, et al. Plant J. 2020 Jul;103(2):512-531.
year: '2020'
doi: 10.1111/tpj.14746
journal_title: The Plant Journal
journal_nlm_ta: Plant J
publisher_name: John Wiley and Sons Inc.
keywords: flux balance analysis | SNPs | metabolic networks | Arabidopsis | poplar
  | plant metabolic modeling | complementary GWAS
automl_pathway: 0.7967154
figid_alias: PMC9328443__F4
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
redirect_from: /figures/PMC9328443__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9328443__TPJ-103-512-g003.html
  '@type': Dataset
  description: Functional SNPs in the lignin biosynthesis pathway of P. trichocarpa
    in three representative genotypes.Distribution of SNPs in three representative
    genotypes which have a low (RGR = 0.04), average (RGR = 7.25), and high (RGR = 15.96)
    growth rate. Metabolite names are shown in blue, gene IDs in black and corresponding
    enzymes in orange. Activating SNPs are marked in green and inactivating SNPs in
    red below the gene that they feature in. The net fold‐change in flux (increase/decrease)
    through a reaction due to SNPs is given next to it, with a flux increase shown
    in green with an upward arrow and a flux decrease shown in red with a downward
    arrow. CCoAMT, caffeoyl‐CoA O‐methyltransferase; CCR, cinnamoyl‐CoA reductase;
    F5H, ferulate 5‐hydroxylase; HCT, shikimate hydroxycinnamoyltransferase; PAL,
    phenylalanine ammonia‐lyase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - bai
  - pe
  - pal
  - Pal2
  - Snp
  - cal
  - Cam
  - tRNA:Gln-TTG-2-1
  - tRNA:Gln-TTG-1-1
  - tRNA:Gln-TTG-1-2
  - tRNA:Gln-TTG-1-3
  - cha
  - ChAT
  - Ubc7
  - kn
  - crl
  - coi
  - Curl
  - sam
  - qkr54B
  - sls
  - Sam-S
  - sah
  - ca
  - Cpr
  - Coa
  - fs(1)h
  - Lgr1
  - rgr
  - CAL
  - CA1
  - HCT
  - CCOAMT
  - AXR4
  - PAM
  - LRIT1
  - SHCBP1
  - FBLIM1
  - GOPC
  - TSHZ1
  - DDR1
  - CCNH
  - CDK7
  - TCFL5
  - COX1
  - DECR1
  - ACSM3
  - BRD2
  - CGA
  - FSHB
  - RGR
  - RGL4
---
