---
figid: PMC9136270__gr1
figtitle: Novel integrative computational framework for breast cancer radiogenomic
  biomarker discovery
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC9136270
filename: gr1.jpg
figlink: /pmc/articles/PMC9136270/figure/f0005/
number: F1
caption: 'The overall workflow of this study. A DL model (3DU-net) was built, trained,
  and validated to segment the tumor region from the raw three-dimensional DCE-MRI
  image. After the 3DU-net were well-trained, DL-based radiomic (DLR) features were
  extracted from the last hidden layer in the encoding phase of the model. Gradient-based
  saliency maps were generated to show the importance of each input pixel to the 3DU-net
  of making its segmentation decision. A: Single-radiogenomic stage. In this stage,
  we first focus on the paired data (top panel of A). Three-level gene expression
  features (197 breast cancer risk gene expressions, 182 KEGG pathway activities,
  and 6 well-established breast cancer gene signatures) are generated. Then, lasso
  models are built to predict each DLR feature and semi-auto radiomic (SAR) feature
  using these three-level gene expression features. After the predictive lasso models
  are well-trained and validated, we turn to the unpaired data (bottom panel of A).
  We generate the same three-level gene expression features using the unpaired data,
  then we apply the well-trained lasso models to get the predicted DLR and SAR features.
  In this way, we could generate the DLR and SAR features for the 1002 patients without
  medical images. Then, we performed survival analysis on the predicted DLR and SAR
  features. The significant ones are the identified prognostic radiogenomic biomarkers.
  Mediation analysis is then performed on these identified radiogenomic biomarkers
  to check the potential biological mechanisms of them. B: Multi-radiogenomic stage.
  In this stage, similar procedures of the single-radiogenomic stage are performed.
  We first focus on the paired data (top panel of B). We perform Bayesian tensor factorization
  (BTF) on the multi-genomic data tensor to extract 17 BTF features. We also run gene
  set enrichment analysis (GSEA) to identify the key biological pathway of each BTF
  feature. These key pathways could explain the key functions of the identified multi-genomic
  BTF features. Then we train lasso models to utilize these 17 BTF features for predicting
  the DLR and SAR features. After the lasso models are well-trained and well-validated,
  we turn to the unpaired data (bottom panel of B). We obtain the BTF features using
  the multi-genomic data, then we apply the well-trained lasso models in the previous
  step to get the predicted DLR and SAR features. In this way, we could get the DLR
  and SAR features for the 701 patients without medical images. Then, we perform survival
  analysis on the predicted DLR and SAR features. The significant ones are the identified
  radiogenomic biomarkers. Mediation analysis is then performed on each of these identified
  radiogenomic biomarkers to check the potential biological mechanisms of them.'
papertitle: A novel integrative computational framework for breast cancer radiogenomic
  biomarker discovery.
reftext: Qian Liu, et al. Comput Struct Biotechnol J. 2022;20:2484-2494.
year: '2022'
doi: 10.1016/j.csbj.2022.05.031
journal_title: Computational and Structural Biotechnology Journal
journal_nlm_ta: Comput Struct Biotechnol J
publisher_name: Research Network of Computational and Structural Biotechnology
keywords: Breast cancer | Radiogenomics | Tensor factorization | Unpaired image-genomics
  problem | Mediation analysis
automl_pathway: 0.821487
figid_alias: PMC9136270__F1
figtype: Figure
redirect_from: /figures/PMC9136270__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9136270__gr1.html
  '@type': Dataset
  description: 'The overall workflow of this study. A DL model (3DU-net) was built,
    trained, and validated to segment the tumor region from the raw three-dimensional
    DCE-MRI image. After the 3DU-net were well-trained, DL-based radiomic (DLR) features
    were extracted from the last hidden layer in the encoding phase of the model.
    Gradient-based saliency maps were generated to show the importance of each input
    pixel to the 3DU-net of making its segmentation decision. A: Single-radiogenomic
    stage. In this stage, we first focus on the paired data (top panel of A). Three-level
    gene expression features (197 breast cancer risk gene expressions, 182 KEGG pathway
    activities, and 6 well-established breast cancer gene signatures) are generated.
    Then, lasso models are built to predict each DLR feature and semi-auto radiomic
    (SAR) feature using these three-level gene expression features. After the predictive
    lasso models are well-trained and validated, we turn to the unpaired data (bottom
    panel of A). We generate the same three-level gene expression features using the
    unpaired data, then we apply the well-trained lasso models to get the predicted
    DLR and SAR features. In this way, we could generate the DLR and SAR features
    for the 1002 patients without medical images. Then, we performed survival analysis
    on the predicted DLR and SAR features. The significant ones are the identified
    prognostic radiogenomic biomarkers. Mediation analysis is then performed on these
    identified radiogenomic biomarkers to check the potential biological mechanisms
    of them. B: Multi-radiogenomic stage. In this stage, similar procedures of the
    single-radiogenomic stage are performed. We first focus on the paired data (top
    panel of B). We perform Bayesian tensor factorization (BTF) on the multi-genomic
    data tensor to extract 17 BTF features. We also run gene set enrichment analysis
    (GSEA) to identify the key biological pathway of each BTF feature. These key pathways
    could explain the key functions of the identified multi-genomic BTF features.
    Then we train lasso models to utilize these 17 BTF features for predicting the
    DLR and SAR features. After the lasso models are well-trained and well-validated,
    we turn to the unpaired data (bottom panel of B). We obtain the BTF features using
    the multi-genomic data, then we apply the well-trained lasso models in the previous
    step to get the predicted DLR and SAR features. In this way, we could get the
    DLR and SAR features for the 701 patients without medical images. Then, we perform
    survival analysis on the predicted DLR and SAR features. The significant ones
    are the identified radiogenomic biomarkers. Mediation analysis is then performed
    on each of these identified radiogenomic biomarkers to check the potential biological
    mechanisms of them.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SARDH
  - SRL
  - NR1I2
  - BCLAF1
  - SGSM3
  - BAX
---
