---
figid: PMC9066720__12864_2022_8581_Fig1_HTML
figtitle: Survival-related genes are diversified across cancers but generally enriched
  in cancer hallmark pathways
organisms:
- Iris germanica
- Ficus carica
- Homo sapiens
organisms_ner:
- Homo sapiens
- Drosophila melanogaster
pmcid: PMC9066720
filename: 12864_2022_8581_Fig1_HTML.jpg
figlink: /pmc/articles/PMC9066720/figure/Fig1/
number: F1
caption: 'The workflow for data pre-processing, model fitting and functional analysis.
  The flowchart illustrates the working process of the present paper. RNA-Seq and
  clinical survival data were retrieved from Broad GDAC firehose. mRNA expression
  data from Illumina Hiseq were used. The RSEM-derived TPM were log2 transformed and
  standardized. Genes with median absolute deviation (MAD) greater than zero were
  fused with clinical survival data. In the model fitting section, the derived data
  were directly applied to the log-rank test or were examined for the proportional
  hazards assumption before applying the Cox model. Both models were fitted individually
  for each gene in each cancer type. The result tables indicate the simplified information
  generating from the models. Multiple testing corrections were performed before subsequently
  analysed by pathway enrichment and clustering. Abbreviation: RSEM, RNA-Seq by expectation
  maximization. RMST, restricted mean survival time. TPM, transcript per million'
papertitle: Survival-related genes are diversified across cancers but generally enriched
  in cancer hallmark pathways.
reftext: Po-Wen Wang, et al. BMC Genomics. 2021;22(Suppl 5):918.
year: '2021'
doi: 10.1186/s12864-022-08581-x
journal_title: BMC Genomics
journal_nlm_ta: BMC Genomics
publisher_name: BioMed Central
keywords: Survival | Pan-cancer | Biomarker | Univariate analysis | Cox proportional
  hazards model | Log-rank test
automl_pathway: 0.5739747
figid_alias: PMC9066720__F1
figtype: Figure
redirect_from: /figures/PMC9066720__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9066720__12864_2022_8581_Fig1_HTML.html
  '@type': Dataset
  description: 'The workflow for data pre-processing, model fitting and functional
    analysis. The flowchart illustrates the working process of the present paper.
    RNA-Seq and clinical survival data were retrieved from Broad GDAC firehose. mRNA
    expression data from Illumina Hiseq were used. The RSEM-derived TPM were log2
    transformed and standardized. Genes with median absolute deviation (MAD) greater
    than zero were fused with clinical survival data. In the model fitting section,
    the derived data were directly applied to the log-rank test or were examined for
    the proportional hazards assumption before applying the Cox model. Both models
    were fitted individually for each gene in each cancer type. The result tables
    indicate the simplified information generating from the models. Multiple testing
    corrections were performed before subsequently analysed by pathway enrichment
    and clustering. Abbreviation: RSEM, RNA-Seq by expectation maximization. RMST,
    restricted mean survival time. TPM, transcript per million'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AMPD1
  - MXD1
  - RMST
  - ACACA
  - BMS1
  - ACACB
  - YES1
  - COX8A
  - CPOX
  - COX5A
  - COX5B
  - COX6C
  - COX7C
  - .na.character
  - COX4I1
  - COX4I2
  - COX6A1
  - COX6A2
  - COX6B1
  - COX6B2
  - COX7A1
  - COX7A2
  - COX7B
  - COX7B2
  - COX8C
  - Mad
  - CG2662
  - Mnt
  - ash2
  - 'On'
  - Low
  - ACC
  - tacc
  - acclinal-wing
  - br
  - shakB
  - COX4
---
