---
figid: PMC9023015__ETLS-6-153-g0001
figtitle: Simplified glucosinolate biosynthetic pathway illustrating the power and
  limitations of integrated transcriptomics and metabolomics for identifying specialized
  metabolic pathways
organisms:
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Oryza sativa
- Zea mays
- Solanum lycopersicum
- Solanum tuberosum
- Camellia sinensis
- Nicotiana benthamiana
- Salvia miltiorrhiza
- Phaseolus vulgaris
- Artemisia annua
- Capsicum annuum
- Papaver somniferum
- Narcissus pseudonarcissus
- Piper methysticum
- Curcuma wenyujin
- Sindora glabra
- Homo sapiens
organisms_ner:
- NA
pmcid: PMC9023015
filename: ETLS-6-153-g0001.jpg
figlink: /pmc/articles/PMC9023015/figure/ETLS-6-153F1/
number: F1
caption: The simplified pathway uses tryptophan as a precursor and generates two stable
  glucosinolate products using several genes and enzymes. Enzymes A and B are directly
  involved in the production of the first stable product. Enzyme C must be activated
  by Enzyme D (a kinase) before it can also contribute to the pathway. As a result,
  Genes A, B, and C are detected in the co-expression network, but Gene C* is not
  detected because the enzyme isn't generated directly from a transcript. Gene D encoding
  the kinase may also not be reliably detected in the co-expression network. Enzyme
  E is directly involved in a final functional modification of stable product 1 to
  form stable product 2, so it also may not reliably co-express at the same time as
  the other biosynthetic genes. Metabolic profiles reveal both stable glucosinolate
  products in addition to the precursor molecule.
papertitle: The ease and complexity of identifying and using specialized metabolites
  for crop engineering.
reftext: Anna Jo Muhich, et al. Emerg Top Life Sci. 2022 Apr 13;6(2):153-162.
year: '2022'
doi: 10.1042/ETLS20210248
journal_title: Emerging Topics in Life Sciences
journal_nlm_ta: Emerg Top Life Sci
publisher_name: Portland Press Ltd.
keywords: engineering | integration | plants | specialized metabolism | synthetic
  biology
automl_pathway: 0.7599345
figid_alias: PMC9023015__F1
figtype: Figure
redirect_from: /figures/PMC9023015__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9023015__ETLS-6-153-g0001.html
  '@type': Dataset
  description: The simplified pathway uses tryptophan as a precursor and generates
    two stable glucosinolate products using several genes and enzymes. Enzymes A and
    B are directly involved in the production of the first stable product. Enzyme
    C must be activated by Enzyme D (a kinase) before it can also contribute to the
    pathway. As a result, Genes A, B, and C are detected in the co-expression network,
    but Gene C* is not detected because the enzyme isn't generated directly from a
    transcript. Gene D encoding the kinase may also not be reliably detected in the
    co-expression network. Enzyme E is directly involved in a final functional modification
    of stable product 1 to form stable product 2, so it also may not reliably co-express
    at the same time as the other biosynthetic genes. Metabolic profiles reveal both
    stable glucosinolate products in addition to the precursor molecule.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - indol-3-ylmethyl
  - 1-hydroxy-indol-3-ylmethyl
  - Tryptophan
  - Glucose
---
