---
figid: PMC9803694__gr5
figtitle: Genetic structure and molecular mechanism underlying the stalk lodging traits
  in maize (Zea mays L.)
organisms:
- NA
pmcid: PMC9803694
filename: gr5.jpg
figlink: /pmc/articles/PMC9803694/figure/fig0025/
number: F5
caption: 'Proposed genetic work model for maize stalk lodging resistance by integrating
  current knowledges. (A) The lignin synthesis pathway which is regarded as the important
  pathway influencing stalk lodging in maize. PAL, phenylalanine ammonia-lyase; C4H,
  cinnamate 4-hudroxylase; 4CL, 4-coumarate: CoA ligase; CCR, cinnamoyl CoA reductase;
  CAD, cinnamyl alcohol dehydrogenase; HCT, p-hydroxycinnamoyl-CoA; C3H, p-coumarate
  3-hydroxylase: CCoADMT, caffeoyl-CoA O-methyltransferase. (B) The combined phytohormone
  metabolic pathways that work together to influence stalk lodging traits. GGPP, geranylgeranyl
  diphosphate; TPS, trehalose-6-phosphate synthase; P450, cytochrome p450; GA2ox,
  GA 2-oxidase; GA20ox, GA 20-oxidase; GA3ox, GA 3-oxidase; PIN, PIN-formed protein;
  JAZ, jasmonate ZIM-domain protein; MYC2, bHLHzip transcription factor MYC2; BZR1,
  brassinazole resistant 1; AN1, anther ear1; D3, dwarf plant 3; D8, dwarf plant8;
  D9, dwarf plant9; D14, strigolactone receptor D14; NA1, nana plant1; NA2, nana plant2.
  The enzymes involved in the lignin biosynthesis pathways in maize are shown in orange
  letters. The known proteins involved in hormone metabolic pathways are shown in
  red.'
papertitle: Genetic structure and molecular mechanism underlying the stalk lodging
  traits in maize (Zea mays L.).
reftext: Shuai Wang, et al. Comput Struct Biotechnol J. 2023;21:485-494.
year: '2023'
doi: 10.1016/j.csbj.2022.12.037
journal_title: Computational and Structural Biotechnology Journal
journal_nlm_ta: Comput Struct Biotechnol J
publisher_name: Research Network of Computational and Structural Biotechnology
keywords: Stalk lodging | Quantitative trait locus (QTL) | Quantitative trait nucleotide
  (QTN) | GWAS | Gene | Meta-analysis | Molecular breeding | Maize
automl_pathway: 0.9317207
figid_alias: PMC9803694__F5
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC9803694__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9803694__gr5.html
  '@type': Dataset
  description: 'Proposed genetic work model for maize stalk lodging resistance by
    integrating current knowledges. (A) The lignin synthesis pathway which is regarded
    as the important pathway influencing stalk lodging in maize. PAL, phenylalanine
    ammonia-lyase; C4H, cinnamate 4-hudroxylase; 4CL, 4-coumarate: CoA ligase; CCR,
    cinnamoyl CoA reductase; CAD, cinnamyl alcohol dehydrogenase; HCT, p-hydroxycinnamoyl-CoA;
    C3H, p-coumarate 3-hydroxylase: CCoADMT, caffeoyl-CoA O-methyltransferase. (B)
    The combined phytohormone metabolic pathways that work together to influence stalk
    lodging traits. GGPP, geranylgeranyl diphosphate; TPS, trehalose-6-phosphate synthase;
    P450, cytochrome p450; GA2ox, GA 2-oxidase; GA20ox, GA 20-oxidase; GA3ox, GA 3-oxidase;
    PIN, PIN-formed protein; JAZ, jasmonate ZIM-domain protein; MYC2, bHLHzip transcription
    factor MYC2; BZR1, brassinazole resistant 1; AN1, anther ear1; D3, dwarf plant
    3; D8, dwarf plant8; D9, dwarf plant9; D14, strigolactone receptor D14; NA1, nana
    plant1; NA2, nana plant2. The enzymes involved in the lignin biosynthesis pathways
    in maize are shown in orange letters. The known proteins involved in hormone metabolic
    pathways are shown in red.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HCT
  - C4H
  - ATCAD4
  - MYC2
  - AT3G03990
  - BZR1
  - BRI1
---
