---
figid: PMC9040349__12885_2022_9381_Fig14_HTML
figtitle: Innovative mouse models for the tumor suppressor activity of Protocadherin-10
  isoforms
organisms:
- Armoracia rusticana
- Lareunionomyces loeiensis
- Matthiola incana
- Homo sapiens
- Mus musculus
- Human papillomavirus
- Parthenolecanium pruinosum
organisms_ner:
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC9040349
filename: 12885_2022_9381_Fig14_HTML.jpg
figlink: /pmc/articles/PMC9040349/figure/Fig14/
number: F14
caption: 'IPA-generated ‘Cell signalling’ network of DE genes in Pcdh10-rescued PTD25_RS
  versus Pcdh10-lacking PTD25 cells. The analysis was performed by the Ingenuity Pathway
  Analysis software (IPA, Qiagen) (see also Table  and legends to Fig.  and Additional file 24b:
  Fig. S13). The network graph depicts how a selection of DE genes relates to one
  another and to some key molecules. The genes are shown in their respective subcellular
  compartment, with connecting lines indicating direct (solid lines) or indirect (broken
  lines) relationships. DE genes with a positive z-score (activated) are colored orange,
  and those with a negative z-score (inhibited) are colored blue. Underneath each
  of the DE genes is given: baseMean value (mean of normalized counts for all PTD25
  and PTD25_RS samples), log2FoldChange (condition PTD25_RS versus PTD25), padj (Benjamin-Hochberg
  adjusted Wald test p-value). Network shapes used here include rectangles with broken
  lines for growth factors, trapezia for transporters, vertical ovals for transmembrane
  receptors, vertical rectangles for G-protein coupled receptors, inverted triangles
  for kinases, triangles for phosphatases, vertical diamonds for non-kinase enzymes,
  circles for other molecules'
papertitle: Innovative mouse models for the tumor suppressor activity of Protocadherin-10
  isoforms.
reftext: Irene Kleinberger, et al. BMC Cancer. 2022;22:451.
year: '2022'
doi: 10.1186/s12885-022-09381-y
journal_title: BMC Cancer
journal_nlm_ta: BMC Cancer
publisher_name: BioMed Central
keywords: Protocadherin-10 isoforms | Tumor suppression | Conditional gene knockout
  | GFAP-Cre | Mouse medulloblastoma model | Mouse auricular tumors | Somatic stem
  cells | Tumor-derived cells | Allografts | RNA-Seq
automl_pathway: 0.8794333
figid_alias: PMC9040349__F14
figtype: Figure
redirect_from: /figures/PMC9040349__F14
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9040349__12885_2022_9381_Fig14_HTML.html
  '@type': Dataset
  description: 'IPA-generated ‘Cell signalling’ network of DE genes in Pcdh10-rescued
    PTD25_RS versus Pcdh10-lacking PTD25 cells. The analysis was performed by the
    Ingenuity Pathway Analysis software (IPA, Qiagen) (see also Table  and legends
    to Fig.  and Additional file 24b: Fig. S13). The network graph depicts how a selection
    of DE genes relates to one another and to some key molecules. The genes are shown
    in their respective subcellular compartment, with connecting lines indicating
    direct (solid lines) or indirect (broken lines) relationships. DE genes with a
    positive z-score (activated) are colored orange, and those with a negative z-score
    (inhibited) are colored blue. Underneath each of the DE genes is given: baseMean
    value (mean of normalized counts for all PTD25 and PTD25_RS samples), log2FoldChange
    (condition PTD25_RS versus PTD25), padj (Benjamin-Hochberg adjusted Wald test
    p-value). Network shapes used here include rectangles with broken lines for growth
    factors, trapezia for transporters, vertical ovals for transmembrane receptors,
    vertical rectangles for G-protein coupled receptors, inverted triangles for kinases,
    triangles for phosphatases, vertical diamonds for non-kinase enzymes, circles
    for other molecules'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Gas6
  - Sorbs2
  - Elmo1
  - Ehd2
  - Ehd3
  - Crmp1
  - ras
  - Hras
  - Kras
  - Rem1
  - Nnmt
  - Efna5
  - Akt1
  - Angpt4
  - Gfra4
  - Synm
  - Epn3
  - Lingo1
  - Prex1
  - Inpp4b
  - Trim72
  - Gfra2
  - Klhl13
  - Sh2b2
  - Slc22a17
  - Dgkd
  - Dgkq
  - Chmp4c
  - Sec14l2
  - Afap1l2
  - Msi1
  - Cnksr1
  - Prokr1
  - Prex2
  - Rbm38
  - GAS6
  - SORBS2
  - ELMO1
  - EHD2
  - CRMP1
  - KRAS
  - HRAS
  - NRAS
  - NNMT
  - BCR
  - IGKV3D-20
  - AKT1
  - RAC1
  - RAC2
  - RAC3
  - RHOG
  - ANGPT4
  - PAK1
  - PAK2
  - PAK3
  - PAK4
  - PAK5
  - PAK6
  - GFRA4
  - SYNM
  - EPN3
  - LINGO1
  - PREX1
  - INPP4B
  - TRIM72
  - GFRA2
  - AKT2
  - AKT3
  - KLHL13
  - SH2B2
  - SLC22A17
  - DGKD
  - CHMP4C
  - SEC14L2
  - AFAP1L2
  - MSI1
  - CNKSR1
  - PROKR1
  - PREX2
  - RBM38
  - gas6
  - elmo1
  - ehd2b
  - crmp1
  - rab1ab
  - efna5b
  - angpt4
  - gfra4a
  - synm
  - epn3b
  - lingo1b
  - prex1
  - inpp4b
  - gfra2a
  - gfra2b
  - klhl13
  - sh2b2
  - slc22a17
  - chmp4c
  - afap1l2
  - msi1a
  - msi1b
  - cnksr1
  - prex2
  - rbm38
---
