---
figid: PMC9534852__41598_2022_20572_Fig6_HTML
figtitle: AHR is a master regulator of diverse pathways in endogenous metabolism
organisms:
- Clostridioides difficile
- Gluconacetobacter-like organism
- Homo sapiens
- Mus musculus
- gut metagenome
organisms_ner:
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
pmcid: PMC9534852
filename: 41598_2022_20572_Fig6_HTML.jpg
figlink: /pmc/articles/PMC9534852/figure/Fig6/
number: F6
caption: 'Metabolic reconstructions combining transcriptomic data with serum metabolomics
  data reveal roles for AHR in cellular and subcellular processes. (A) Schematic describing
  integration of multi-omics data through metabolic reconstruction. Serum metabolomics
  from the knockout mice and transcriptomic data from two different experiments (Ahr−/−
  liver and AHR knockdown MCF7 cells) were combined using Recon3D to develop distinct
  genome scale metabolic reconstructions for the Ahr−/− and Ahr+/+ mice. These models
  were then compared to determine the number of differentially active reactions. (B)
  Functional pathway analysis of metabolic alterations in AHR wildtype versus knockout
  in mouse liver. Nitrogen handling was noted to be a prominent metabolic process
  involved in the enzymatic reaction fluxes that were increased in knockout relative
  to wildtype conditions. Two pathways consisting of reactions that were differentially
  active based on flux simulations are depicted. The top pathway is dependent on transfer
  of ortho- and pyro-phosphate. The lower pathway involves NAD/NADH cofactors and
  CoA. Reactions are provided in Supplemental Table . Metabolite abbreviations: ura:
  uracil, acald: acetaldehyde, uri: uridine, duri: deoxyuridine, ptr: putrescine,
  4aabutn: 4-acetamidobutanoate, Pi: orthophosphate, PPi: pyrophosphate, CoA: coenzyme
  A. (C) Functional pathway analysis of metabolic alterations in AHR wildtype versus
  knockout. Selected, representative reactions highlighting the organelle-specific
  pathways with predicted increased fluxes in AHR knockdown. Peroxisomal fatty acid
  oxidation with associated generation of hydrogen peroxide (and reduction by catalase)
  was increased for multiple fatty acids. Degradation of multiple amino acids, including
  tryptophan, converge on the increased production of multiple organic acids. Increased
  mitochondrial elimination pathways include glutaryl-CoA that is ultimately converted
  to 3-hydroxybutyrylcarnitine before being transported out of the cell. Most biosynthetic
  and degradation reactions deplete the NADPH and NADH cofactor pools, however, some
  of the mitochondrial processes recharge the FADH2 and Q102 pools. Reactions are
  provided in Supplemental Table . Metabolite abbreviations (black text): L-tyr: L
  tyrosine, L-ser: L-serine, L-met: L-methionine, L-cys: L-cystine, L-trp: L-tryptophan,
  L-lys: L-lysine, pmt-CoA: palmitoyl-Coenzyme A, ac-CoA: acetyl-Coenzyme A, orn:
  ornithine, Lkynr: L-kynurenine, glutCoA: glutaryl-Coenzyme A, hexdi-CoA: Hexadecanedioyl-Coenzyme
  A, 5mthf: 5-methyl tetrahydrofolate, 2oxoadp: 2-oxoadipate, 3bcrn: 3-hydroxybutyrylcarnitine,
  3hadpac: 3-hydroxyadipate, 4aabutn: 4-acetamidobutanoate, 3hexdcrn: 3-hydroxyhexadecanoylcarnitine,
  fadh2: flavin adenine dinucleotide (reduced), fad:, flavin adenine dinucleotide
  (oxidized), q10h2: Ubiquinol-10, q10: Ubiquinone-10, 2obut: 2- oxobutanoate, h2o2:
  hydrogen peroxide. Abbreviations for metabolites and reactions use nomenclature
  from the BiGG database (http://bigg.ucsd.edu/).'
papertitle: AHR is a master regulator of diverse pathways in endogenous metabolism.
reftext: Jeffry C. Granados, et al. Sci Rep. 2022;12:16625.
year: '2022'
doi: 10.1038/s41598-022-20572-2
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Biochemical networks | Transcription | Metabolomics | Biochemical reaction
  networks
automl_pathway: 0.9423557
figid_alias: PMC9534852__F6
figtype: Figure
redirect_from: /figures/PMC9534852__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9534852__41598_2022_20572_Fig6_HTML.html
  '@type': Dataset
  description: 'Metabolic reconstructions combining transcriptomic data with serum
    metabolomics data reveal roles for AHR in cellular and subcellular processes.
    (A) Schematic describing integration of multi-omics data through metabolic reconstruction.
    Serum metabolomics from the knockout mice and transcriptomic data from two different
    experiments (Ahr−/− liver and AHR knockdown MCF7 cells) were combined using Recon3D
    to develop distinct genome scale metabolic reconstructions for the Ahr−/− and
    Ahr+/+ mice. These models were then compared to determine the number of differentially
    active reactions. (B) Functional pathway analysis of metabolic alterations in
    AHR wildtype versus knockout in mouse liver. Nitrogen handling was noted to be
    a prominent metabolic process involved in the enzymatic reaction fluxes that were
    increased in knockout relative to wildtype conditions. Two pathways consisting
    of reactions that were differentially active based on flux simulations are depicted.
    The top pathway is dependent on transfer of ortho- and pyro-phosphate. The lower
    pathway involves NAD/NADH cofactors and CoA. Reactions are provided in Supplemental
    Table . Metabolite abbreviations: ura: uracil, acald: acetaldehyde, uri: uridine,
    duri: deoxyuridine, ptr: putrescine, 4aabutn: 4-acetamidobutanoate, Pi: orthophosphate,
    PPi: pyrophosphate, CoA: coenzyme A. (C) Functional pathway analysis of metabolic
    alterations in AHR wildtype versus knockout. Selected, representative reactions
    highlighting the organelle-specific pathways with predicted increased fluxes in
    AHR knockdown. Peroxisomal fatty acid oxidation with associated generation of
    hydrogen peroxide (and reduction by catalase) was increased for multiple fatty
    acids. Degradation of multiple amino acids, including tryptophan, converge on
    the increased production of multiple organic acids. Increased mitochondrial elimination
    pathways include glutaryl-CoA that is ultimately converted to 3-hydroxybutyrylcarnitine
    before being transported out of the cell. Most biosynthetic and degradation reactions
    deplete the NADPH and NADH cofactor pools, however, some of the mitochondrial
    processes recharge the FADH2 and Q102 pools. Reactions are provided in Supplemental
    Table . Metabolite abbreviations (black text): L-tyr: L tyrosine, L-ser: L-serine,
    L-met: L-methionine, L-cys: L-cystine, L-trp: L-tryptophan, L-lys: L-lysine, pmt-CoA:
    palmitoyl-Coenzyme A, ac-CoA: acetyl-Coenzyme A, orn: ornithine, Lkynr: L-kynurenine,
    glutCoA: glutaryl-Coenzyme A, hexdi-CoA: Hexadecanedioyl-Coenzyme A, 5mthf: 5-methyl
    tetrahydrofolate, 2oxoadp: 2-oxoadipate, 3bcrn: 3-hydroxybutyrylcarnitine, 3hadpac:
    3-hydroxyadipate, 4aabutn: 4-acetamidobutanoate, 3hexdcrn: 3-hydroxyhexadecanoylcarnitine,
    fadh2: flavin adenine dinucleotide (reduced), fad:, flavin adenine dinucleotide
    (oxidized), q10h2: Ubiquinol-10, q10: Ubiquinone-10, 2obut: 2- oxobutanoate, h2o2:
    hydrogen peroxide. Abbreviations for metabolites and reactions use nomenclature
    from the BiGG database (http://bigg.ucsd.edu/).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FBXO3
  - AGO2
  - BRCA2
  - FANCD2
  - PSEN1
  - URI1
  - CASR
  - CXADR
  - PRKAR1A
  - SPG7
  - NR1I3
  - TRIM13
  - CXADRP1
  - ETV3
  - NR4A1
  - MRC1
  - Fbxo3
  - Hps3
  - H2-Q10
  - Fancd2
  - Nr1i3
  - Casr
  - Cxadr
  - Rcvrn
  - Trim13
  - Arr3
  - Etv3
  - Nr4a1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Ald1
  - Coa
  - orn
  - Desat2
  - Fadd
  - Fad2
  - Desat1
  - uri
  - car
  - pi
  - Hmr
  - sesB
---
