---
figid: PMC9505912__plants-11-02408-g007
figtitle: Targeted Metabolomics Provide Chemotaxonomic Insights of Medicago ruthenica,
  with Coupled Transcriptomics Elucidating the Mechanism Underlying Floral Coloration
organisms:
- Medicago ruthenica
organisms_ner:
- Arabidopsis thaliana
pmcid: PMC9505912
filename: plants-11-02408-g007.jpg
figlink: /pmc/articles/PMC9505912/figure/plants-11-02408-f007/
number: F7
caption: Model representing the anthocyanin synthesis process in the M. ruthenica
  species. In this study, PFMs served as the control group. The referred model for
  a better understanding of the formation of the pale-yellow M. ruthenica flower was
  drawn using Adobe Illustrator 2019. ER represents the endoplasmic reticulum. The
  crucial candidate gene IDs are indicated at the side of related enzymes. Blue-colored
  grids indicate the downregulated genes, and red-colored grids indicate upregulated
  genes. The arrow refers to the steps of the anthocyanin biosynthesis pathway (ABP).
  CHS, CHI, F3H, and F3′H were upstream genes involved in the ABP. DFR, ANS, and UFGT
  were downstream genes involved in the ABP. Briefly, the low expression level of
  4 CHS, 2 CHI, 3 F3H, 2 F3′H, 4 DFR, 2 ANS, and 2 UFGT might disrupt the anthocyanin
  synthesis, leading to the formation of the pale-yellow petals seen in YFM. UFGT
  catalyzes the last step of the ABP, the UFGT enzyme glycolyzed anthocyanidin into
  anthocyanin by adding sugar moieties, thereby increasing the hydrophilicity and
  stability of anthocyanin.
papertitle: Targeted Metabolomics Provide Chemotaxonomic Insights of Medicago ruthenica,
  with Coupled Transcriptomics Elucidating the Mechanism Underlying Floral Coloration.
reftext: Lin Zhu, et al. Plants (Basel). 2022 Sep;11(18):2408.
year: '2022'
doi: 10.3390/plants11182408
journal_title: Plants
journal_nlm_ta: Plants (Basel)
publisher_name: MDPI
keywords: Medicago ruthenica | alfalfa | chemotaxonomy | flavonoids | anthocyanins
  | gene expression
automl_pathway: 0.9448398
figid_alias: PMC9505912__F7
figtype: Figure
redirect_from: /figures/PMC9505912__F7
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9505912__plants-11-02408-g007.html
  '@type': Dataset
  description: Model representing the anthocyanin synthesis process in the M. ruthenica
    species. In this study, PFMs served as the control group. The referred model for
    a better understanding of the formation of the pale-yellow M. ruthenica flower
    was drawn using Adobe Illustrator 2019. ER represents the endoplasmic reticulum.
    The crucial candidate gene IDs are indicated at the side of related enzymes. Blue-colored
    grids indicate the downregulated genes, and red-colored grids indicate upregulated
    genes. The arrow refers to the steps of the anthocyanin biosynthesis pathway (ABP).
    CHS, CHI, F3H, and F3′H were upstream genes involved in the ABP. DFR, ANS, and
    UFGT were downstream genes involved in the ABP. Briefly, the low expression level
    of 4 CHS, 2 CHI, 3 F3H, 2 F3′H, 4 DFR, 2 ANS, and 2 UFGT might disrupt the anthocyanin
    synthesis, leading to the formation of the pale-yellow petals seen in YFM. UFGT
    catalyzes the last step of the ABP, the UFGT enzyme glycolyzed anthocyanidin into
    anthocyanin by adding sugar moieties, thereby increasing the hydrophilicity and
    stability of anthocyanin.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TT4
  - CHI
  - TT5
  - F3H
  - DFR
  - LDOX
---
