---
figid: PMC9672095__41598_2022_23652_Fig4_HTML
figtitle: Sirt3 deficiency induced down regulation of insulin degrading enzyme in
  comorbid Alzheimer’s disease with metabolic syndrome
organisms:
- Mus musculus
- Rattus norvegicus
- Human alphaherpesvirus 3
- Catharanthus roseus
- Homo sapiens
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
- Drosophila melanogaster
pmcid: PMC9672095
filename: 41598_2022_23652_Fig4_HTML.jpg
figlink: /pmc/articles/PMC9672095/figure/Fig4/
number: F4
caption: 'Pathway analysis: The differentially expressed genes (DEGs, 25,000) from
  wild type, Sirt3-/-, APP/PS1 and APP/PS1/Sirt3-/- mouse brain samples were uploaded
  into the DAVID database (https://david.ncifcrf.gov/) for KEGG-GO pathway enrichment
  analysis in APP/PS1/Sirt3-/- (comorbid-AD) mice. Enrichment analysis of KEGG revealed
  that DEGs were primarily involved in metabolic, inflammation, Alzheimer disease (AD)
  and insulin signaling pathways. A representative AD cascade pathway was obtained
  from KEGG Pathway Maps. For the functional annotation of differentially expressed
  genes, the web-based DAVID Bioinformatics Resources v 6.8 tool was used (https://david.ncifcrf.gov).'
papertitle: Sirt3 deficiency induced down regulation of insulin degrading enzyme in
  comorbid Alzheimer’s disease with metabolic syndrome.
reftext: Alpna Tyagi, et al. Sci Rep. 2022;12:19808.
year: '2022'
doi: 10.1038/s41598-022-23652-5
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Neuroscience | Neurology | Pathogenesis
automl_pathway: 0.9076357
figid_alias: PMC9672095__F4
figtype: Figure
redirect_from: /figures/PMC9672095__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9672095__41598_2022_23652_Fig4_HTML.html
  '@type': Dataset
  description: 'Pathway analysis: The differentially expressed genes (DEGs, 25,000)
    from wild type, Sirt3-/-, APP/PS1 and APP/PS1/Sirt3-/- mouse brain samples were
    uploaded into the DAVID database (https://david.ncifcrf.gov/) for KEGG-GO pathway
    enrichment analysis in APP/PS1/Sirt3-/- (comorbid-AD) mice. Enrichment analysis
    of KEGG revealed that DEGs were primarily involved in metabolic, inflammation,
    Alzheimer disease (AD) and insulin signaling pathways. A representative AD cascade
    pathway was obtained from KEGG Pathway Maps. For the functional annotation of
    differentially expressed genes, the web-based DAVID Bioinformatics Resources v
    6.8 tool was used (https://david.ncifcrf.gov).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Dynll1
  - Ide
  - Snca
  - Ddr1
  - Mme
  - Tns2
  - Lpl
  - Lvs
  - Acp2
  - Cebpb
  - elap
  - Acp3
  - Lap3
  - Cenpj
  - Grin1
  - Decr1
  - Insr
  - Alpha-re
  - App
  - Bpi
  - Bace1
  - Pcsk1n
  - Tspan33
  - Eefsec
  - Fam167b
  - Pkm
  - Prickle3
  - Angptl1
  - dm
  - Hspg2
  - Csnk2a1
  - Csnk2a2
  - Bcar1
  - Cse1l
  - Zswim2
  - ap
  - Ace
  - Nos1
  - Apc
  - Ecscr
  - Nrg1
  - Crtap
  - cy
  - Mcu
  - tor
  - Rorc
  - Apo
  - Aopep
  - ATP6
  - Atp6-ps
  - Abo
  - Cox4i1
  - Il6
  - Tnf
  - Il1
  - Ppia-ps1
  - Cblif
  - mic
  - Altre
  - Ddit3
  - S100a8
  - Ad
  - Dnmbp
  - Pik3r1
  - Wapl
  - FGFR3
  - DYNLL1
  - IDE
  - SMARCA1
  - SNCA
  - DDR1
  - MME
  - CLIC1
  - CLIC2
  - CLIC3
  - CLIC4
  - CLIC5
  - CLIC6
  - CTSB
  - LPL
  - LCP1
  - LYPLA2P1
  - LAP
  - ACP2
  - TGFB1
  - PICALM
  - LAP3
  - CENPJ
  - PDON2
  - GRIN1
  - GRIN2A
  - GRIN2B
  - GRIN2C
  - GRIN2D
  - GRIN3A
  - GRIN3B
  - DAPK3
  - ZGPAT
  - AMACR
  - DECR1
  - INSR
  - IGF1R
  - INSRR
  - APP
  - BPI
  - GZMB
  - BACE1
  - PUM3
  - PCSK1N
  - TSPAN33
  - APH1A
  - PRICKLE3
  - PKM
  - CHUK
  - IKBKB
  - IKBKG
  - IKBKE
  - TBK1
  - ANGPTL1
  - HSPG2
  - PLCE1
  - PLCZ1
  - PLCB1
  - PLCB2
  - PLCB3
  - PLCB4
  - PLCD1
  - PLCD3
  - PLCD4
  - PLCG1
  - PLCG2
  - CSNK2A1
  - CSNK2A2
  - CSNK2B
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - PICK1
  - ZSWIM2
  - ACE
  - NOS1
  - NOS2
  - NOS3
  - APC
  - PROC
  - NRG1
  - ECSCR
  - CUX1
  - CYTIP
  - CRTAP
  - MCU
  - RORC
  - HSD17B10
  - GYPB
  - AOPEP
  - ABO
  - YWHAE
  - ZNF598
  - COX8A
  - CPOX
  - COX5A
  - COX5B
  - COX6C
  - COX7C
  - .na.character
  - COX4I1
  - COX4I2
  - COX6A1
  - COX6A2
  - COX6B1
  - COX6B2
  - COX7A1
  - COX7A2
  - COX7B
  - COX7B2
  - COX8C
  - IL6
  - TNF
  - IL1B
  - CYCS
  - CBLIF
  - ATP8A2
  - DDIT3
  - S100A8
  - DNMBP
  - TUBA1A
  - TUBA1B
  - TUBA1C
  - TUBA3C
  - TUBA3D
  - TUBA3E
  - TUBA4A
  - TUBA8
  - TUBAL3
  - NUBP1
  - TP63
  - COASY
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - MOB3C
  - IRF6
  - WAPL
  - Mageb16
  - Th
  - Sqstm1
  - Csnk2b
  - Hsd17b10
  - Abo3
  - Pik3cg
  - achi
  - emc
  - betaTub60D
  - hfw
  - Iswi
  - Nep1
  - Clic
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Ae2
  - lncRNA:noe
  - Ms
  - apt
  - Apart
  - lap
  - Lap-A
  - Nmdar2
  - Nmdar1
  - zip
  - uzip
  - Ance
  - InR
  - app
  - Appl
  - ApepP
  - APP-BP1
  - Bace
  - Pen
  - peng
  - peo
  - aph-1
  - Alp1
  - pim
  - ck
  - IKKepsilon
  - key
  - IKKbeta
  - wake
  - lolal
  - mir-ban
  - norpA
  - sl
  - Plc21C
  - CkIIbeta2
  - CkIIalpha
  - cas
  - caz
  - Cse1
  - cass
  - cash
  - ThrRS
  - Ppb
  - se
  - mex
  - mo
  - Kr
  - Acer
  - Nos
  - nos
  - Rep
  - rept
  - Apc2
  - Axn
  - Fs(3)Apc
  - sing
  - Np
  - casp
  - blw
  - Tor
  - Egfr
  - Pa
  - apo
  - Xbp1
  - abo
  - ABO-2R
  - ABO-X
  - ABO-YL
  - ABO-YS
  - COX4
  - egr
  - JIL-1
  - ato
  - cyc
  - CycE
  - Cyt-c-d
  - Cyt-c-p
  - cype
  - if
  - anon-92Ec
  - ATPsynbeta
  - Atpalpha
  - ad
  - tak
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - pit
  - wapl
  - NADPH
  - BPI
  - PEN
  - Se
  - ACE RADE
  - MPTP
  - ATP
  - ATO
  - NBP
  - ATO13-ATOM
  - DM
  - diabete
  - Aria APC C
  - FERKABIATE
  - Mahendral dysfunction
---
