---
figid: PMC9009446__fcimb-12-736397-g002
figtitle: The Microbiome in the Pathogenesis of COVID-19
organisms:
- Escherichia coli
- Pseudomonas aeruginosa
- Vibrio cholerae
- Enterococcus faecalis
- Lacticaseibacillus casei
- Acinetobacter baumannii
- Staphylococcus epidermidis
- Faecalibacterium prausnitzii
- Akkermansia muciniphila
- Enterococcus faecium
- Lacticaseibacillus rhamnosus
- Clostridium butyricum
- Bacteroides thetaiotaomicron
- Limosilactobacillus fermentum
- Burkholderia cepacia
- Eubacterium rectale
- Enterobacter sp.
- Streptococcus uberis
- Clostridium leptum
- Streptococcus sp.
- Prevotella melaninogenica
- Veillonella parvula
- Parvimonas micra
- Collinsella aerofaciens
- Metamycoplasma hominis
- Bacteroides ovatus
- Morganella morganii
- Thomasclavelia ramosa
- Enterococcus sp.
- Haemophilus parainfluenzae
- Hungatella hathewayi
- Lachnospiraceae bacterium
- Parabacteroides merdae
- Streptococcus infantis
- Bacteroides stercoris
- Lancefieldella parvula
- Phocaeicola dorei
- Phocaeicola massiliensis
- Neisseria subflava
- Fusobacterium periodonticum
- Metamycoplasma orale
- Escherichia coli O26
- Neisseria mucosa
- Eikenella corrodens
- Peptostreptococcus sp.
- Alistipes onderdonkii
- Collinsella tanakaei
- Aggregatibacter aphrophilus
- Capnocytophaga sputigena
- Corynebacterium accolens
- Coprobacillus sp.
- Gemella morbillorum
- Candida albicans
- Brassica oleracea
- Aspergillus niger
- Aspergillus flavus
- Aspergillus sp.
- Penicillium citrinum
- Candida auris
- Candida sp. uncertain placement
- Saccharomyces sp.
- Malassezia sp.
- Nakaseomyces sp.
- Severe acute respiratory syndrome coronavirus 2
- Human gammaherpesvirus 4
- Coronaviridae
- Severe acute respiratory syndrome-related coronavirus
- Middle East respiratory syndrome-related coronavirus
- H1N1 subtype
- Betacoronavirus
- Human orthopneumovirus
- Rhinovirus B
- Riboviria
- gut metagenome
- metagenome
- human gut metagenome
- human metagenome
- human oral metagenome
- oral viral metagenome
- Lactobacillus phage phiadh
- Streptococcus phage EJ-1
- Staphylococcus phage ROSA
- Mus musculus
- Rattus norvegicus
- Caenorhabditis elegans
- Mustela lutreola
- Manis javanica
- Viana
- Homo sapiens
- Anas platyrhynchos
pmcid: PMC9009446
filename: fcimb-12-736397-g002.jpg
figlink: /pmc/articles/PMC9009446/figure/f2/
number: F2
caption: The figure shows the different mechanisms by which the microbiome has been
  found to regulate inflammation. The diagram shows that the microbiome undergoes
  dysbiosis due to various factors. The abnormal microbiome causes inflammation via
  different pathways, which may include the kynurenine pathway (KP), the endocannabinoid
  system (eCB), barrier dysfunction (via gut peptides), and regulation of synthesis
  of different host metabolites like serotonin (5-hydroxytryptamine, 5-HT). Both serotonin
  and kynurenine are products of tryptophan catabolism. Serotonin is a crucial neurotransmitter
  helping in the communication across the gut–brain axis. Apart from this, the microbiome
  itself produces a host of metabolites like SCFAs, TMAO, and curli, which modulates
  the immune system and other cellular and metabolic processes leading to outcomes
  like inflammation and perturbation of homeostasis. These conclusions are based on
  evidence gathered from other disorders. These pathways are hypothesized to significantly
  contribute to the pathogenesis of COVID-19, which is characterized by severe inflammation
  and tissue injury.
papertitle: Role of the Microbiome in the Pathogenesis of COVID-19.
reftext: Rituparna De, et al. Front Cell Infect Microbiol. 2022;12:736397.
year: '2022'
doi: 10.3389/fcimb.2022.736397
journal_title: Frontiers in Cellular and Infection Microbiology
journal_nlm_ta: Front Cell Infect Microbiol
publisher_name: Frontiers Media S.A.
keywords: COVID-19 | microbiome | SARS-CoV2 | inflammation | ACE2 | serotonin | kynurenine
automl_pathway: 0.9418455
figid_alias: PMC9009446__F2
figtype: Figure
organisms_ner:
- Rattus norvegicus
- Mus musculus
- Homo sapiens
redirect_from: /figures/PMC9009446__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9009446__fcimb-12-736397-g002.html
  '@type': Dataset
  description: The figure shows the different mechanisms by which the microbiome has
    been found to regulate inflammation. The diagram shows that the microbiome undergoes
    dysbiosis due to various factors. The abnormal microbiome causes inflammation
    via different pathways, which may include the kynurenine pathway (KP), the endocannabinoid
    system (eCB), barrier dysfunction (via gut peptides), and regulation of synthesis
    of different host metabolites like serotonin (5-hydroxytryptamine, 5-HT). Both
    serotonin and kynurenine are products of tryptophan catabolism. Serotonin is a
    crucial neurotransmitter helping in the communication across the gut–brain axis.
    Apart from this, the microbiome itself produces a host of metabolites like SCFAs,
    TMAO, and curli, which modulates the immune system and other cellular and metabolic
    processes leading to outcomes like inflammation and perturbation of homeostasis.
    These conclusions are based on evidence gathered from other disorders. These pathways
    are hypothesized to significantly contribute to the pathogenesis of COVID-19,
    which is characterized by severe inflammation and tissue injury.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ccne1
  - Ffar2
  - Acsl1
  - Pla2g15
  - Il6
  - Tnf
  - Acss2
  - Gusb
  - FFAR2
  - PLA2G15
  - ACSS2
  - ACCS
  - IL6
  - TNF
  - IFNA1
---
