---
figid: PMC9592602__41467_2022_33144_Fig1_HTML
figtitle: Schematic of the mevalonate (MVA) pathway and overview of the computational
  pharmacogenomics workflow
organisms:
- Mus musculus
- Rattus norvegicus
- Homo sapiens
- Bos taurus
- Carnegiea gigantea
pmcid: PMC9592602
filename: 41467_2022_33144_Fig1_HTML.jpg
figlink: /pmc/articles/PMC9592602/figure/Fig1/
number: F1
caption: a In response to fluvastatin treatment (labeled with 1), MVA pathway end-product
  levels decrease, triggering an SREBP-mediated feedback response that activates MVA
  pathway-associated gene expression to restore cholesterol and other non-sterol end-products.
  Dipyridamole (DP) (labeled with 2) blocks the SREBP-mediated feedback response,
  thereby potentiating fluvastatin-induced cancer cell death. b An overview of the
  computational pharmacogenomics workflow, MVA-DNF, was used to identify the top 19
  compounds and visualized as a compound network. MVA-DNF combines drug structure,
  drug-induced gene perturbation datasets restricted to six MVA pathway-specific genes,
  and drug sensitivity. Permutation specificity testing was performed to select compounds
  that have a degree of specificity to the MVA pathway and dipyridamole. Statistical
  significance of compounds similar to dipyridamole was assessed by comparing to 999
  networks generated from random selection of six genes within the drug perturbation
  layer. A network representation of dipyridamole and the top 19 statistically-significant
  (p-value <0.05) compounds are shown. Each node represents a compound, and edges
  connect compounds based on a statistical significance cutoff of p-value <0.05. Blue
  nodes and orange edges represent the compounds connected to dipyridamole, and edge
  thickness represents the associated p-value between the compounds. c Radar plot
  of the top 19 compounds (p-value <0.05), where the contribution of each individual
  layer of the MVA-DNF (drug structure, perturbation, and sensitivity) is depicted.
  The radar plot was generated by comparing the score of DP and the top hit compound,
  across the affinity matrices of the perturbation, structure, and sensitivity layers.
  The percent contribution of each layer is shown from the center (0%) to the outer
  edges (100%).
papertitle: Computational pharmacogenomic screen identifies drugs that potentiate
  the anti-breast cancer activity of statins.
reftext: Jenna E. van Leeuwen, et al. Nat Commun. 2022;13:6323.
year: '2022'
doi: 10.1038/s41467-022-33144-9
journal_title: Nature Communications
journal_nlm_ta: Nat Commun
publisher_name: Nature Publishing Group UK
keywords: Breast cancer | Virtual drug screening | Pharmacogenomics | Cancer genomics
  | Cancer therapy
automl_pathway: 0.7899947
figid_alias: PMC9592602__F1
figtype: Figure
organisms_ner:
- Rattus norvegicus
redirect_from: /figures/PMC9592602__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9592602__41467_2022_33144_Fig1_HTML.html
  '@type': Dataset
  description: a In response to fluvastatin treatment (labeled with 1), MVA pathway
    end-product levels decrease, triggering an SREBP-mediated feedback response that
    activates MVA pathway-associated gene expression to restore cholesterol and other
    non-sterol end-products. Dipyridamole (DP) (labeled with 2) blocks the SREBP-mediated
    feedback response, thereby potentiating fluvastatin-induced cancer cell death.
    b An overview of the computational pharmacogenomics workflow, MVA-DNF, was used
    to identify the top 19 compounds and visualized as a compound network. MVA-DNF
    combines drug structure, drug-induced gene perturbation datasets restricted to
    six MVA pathway-specific genes, and drug sensitivity. Permutation specificity
    testing was performed to select compounds that have a degree of specificity to
    the MVA pathway and dipyridamole. Statistical significance of compounds similar
    to dipyridamole was assessed by comparing to 999 networks generated from random
    selection of six genes within the drug perturbation layer. A network representation
    of dipyridamole and the top 19 statistically-significant (p-value <0.05) compounds
    are shown. Each node represents a compound, and edges connect compounds based
    on a statistical significance cutoff of p-value <0.05. Blue nodes and orange edges
    represent the compounds connected to dipyridamole, and edge thickness represents
    the associated p-value between the compounds. c Radar plot of the top 19 compounds
    (p-value <0.05), where the contribution of each individual layer of the MVA-DNF
    (drug structure, perturbation, and sensitivity) is depicted. The radar plot was
    generated by comparing the score of DP and the top hit compound, across the affinity
    matrices of the perturbation, structure, and sensitivity layers. The percent contribution
    of each layer is shown from the center (0%) to the outer edges (100%).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Hmgcs1
  - Hmgcs2
  - Hmgcr
  - Line
---
