---
figid: PMC9503257__40168_2022_1346_Fig6_HTML
figtitle: Intestinal microbiome-mediated resistance against vibriosis for Cynoglossus
  semilaevis
organisms:
- NA
organisms_ner:
- Danio rerio
- Homo sapiens
pmcid: PMC9503257
filename: 40168_2022_1346_Fig6_HTML.jpg
figlink: /pmc/articles/PMC9503257/figure/Fig6/
number: F6
caption: 'Proposed regulatory networks involved in anti-infection associated metabolism
  pathway in C. semilaevis. A Steroid, steroid hormone, and bile acid (BA) biosynthesis
  metabolism. B Biosynthesis of unsaturated fatty acids (UFAs). C Immune signaling
  transduction pathways. The enriched pathway of the DEGs (|log2(FoldChange)| > 1
  and adjusted p < 0.05) that have significant associations (Spearman rank correlations
  ≥ 0.5 or ≤ −0.5, hypothesis test, p < 0.05) with microbes (relative abundance ≥
  0.1% and coverage > 10% samples) were selected. Differences in host gene expression
  and microbe abundances in two families are shown along the pathway: the red bold
  words were the KOs that were involved in the anti-infection associated metabolism
  pathways. The upward and downward red arrows indicate the DEGs were up-expressed
  and down-expressed in the resistant family, respectively. While the upward and downward
  purple arrows indicate the microbes were more abundant in the resistant (orange
  box) and susceptible families (green box), respectively. The red and blue boxes
  in the top right sub-figure represent the abundance (TPM) of genes in the resistant
  and susceptible families, respectively. While the orange and green boxes in the
  top right sub-figure represent the relative abundance of intestinal microbes in
  the resistant and susceptible families, respectively. “*” means the microbes were
  significantly differentiated (p < 0.05) in relative abundance between the resistant
  and susceptible families of C. semilaevis'
papertitle: Intestinal microbiome-mediated resistance against vibriosis for Cynoglossus
  semilaevis.
reftext: Qian Zhou, et al. Microbiome. 2022;10:153.
year: '2022'
doi: 10.1186/s40168-022-01346-4
journal_title: Microbiome
journal_nlm_ta: Microbiome
publisher_name: BioMed Central
keywords: ''
automl_pathway: 0.896384
figid_alias: PMC9503257__F6
figtype: Figure
redirect_from: /figures/PMC9503257__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9503257__40168_2022_1346_Fig6_HTML.html
  '@type': Dataset
  description: 'Proposed regulatory networks involved in anti-infection associated
    metabolism pathway in C. semilaevis. A Steroid, steroid hormone, and bile acid
    (BA) biosynthesis metabolism. B Biosynthesis of unsaturated fatty acids (UFAs).
    C Immune signaling transduction pathways. The enriched pathway of the DEGs (|log2(FoldChange)|
    > 1 and adjusted p < 0.05) that have significant associations (Spearman rank correlations
    ≥ 0.5 or ≤ −0.5, hypothesis test, p < 0.05) with microbes (relative abundance
    ≥ 0.1% and coverage > 10% samples) were selected. Differences in host gene expression
    and microbe abundances in two families are shown along the pathway: the red bold
    words were the KOs that were involved in the anti-infection associated metabolism
    pathways. The upward and downward red arrows indicate the DEGs were up-expressed
    and down-expressed in the resistant family, respectively. While the upward and
    downward purple arrows indicate the microbes were more abundant in the resistant
    (orange box) and susceptible families (green box), respectively. The red and blue
    boxes in the top right sub-figure represent the abundance (TPM) of genes in the
    resistant and susceptible families, respectively. While the orange and green boxes
    in the top right sub-figure represent the relative abundance of intestinal microbes
    in the resistant and susceptible families, respectively. “*” means the microbes
    were significantly differentiated (p < 0.05) in relative abundance between the
    resistant and susceptible families of C. semilaevis'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - cyp19a1a
  - hsd3b
  - casd1
  - kita
  - ngfra
  - tlr1
  - tlr2
  - rac1a
  - degs1
  - mhc1ufa
  - elovl5
  - acox1
  - il6
  - tnfa
  - tnfb
  - ticam1
  - azi2
  - rigi
  - traf3
  - tlr3
  - tank
  - tbk1
  - irf7
  - irf3
  - CYP2B6
  - CYP2B7P
  - HSD3B1
  - HSD3B2
  - CYP27A1
  - TCF23
  - SOAT1
  - CASD1
  - SLC10A6
  - CTH
  - SLC2A1
  - TLR1
  - TLR2
  - RAC1
  - RNASE1
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - AKT1
  - AKT2
  - AKT3
  - NFKB1
  - DEGS1
  - ELOVL5
  - ACOX1
  - POLR3K
  - PPIP5K1
  - ISYNA1
  - MAVS
  - IL6
  - TNF
  - IL12A
  - IL12B
  - IL18
  - TRIM69
  - TICAM1
  - CXCL8
  - NAP1L1
  - NAPSA
  - ACOT8
  - NCKAP1
  - TRMO
  - AZI2
  - NAA25
  - TAB3
  - RIGI
  - TRAF3
  - TLR3
  - TANK
  - IKBKE
  - TBK1
  - IRF7
  - IRF3
  - IFNA1
  - IFNB1
---
