---
figid: PMC9437888__MBT2-15-2324-g007
figtitle: 'Developing a CRISPR‐assisted base‐editing system for genome engineering
  of Pseudomonas chlororaphis '
organisms:
- NA
pmcid: PMC9437888
filename: MBT2-15-2324-g007.jpg
figlink: /pmc/articles/PMC9437888/figure/mbt214075-fig-0002/
number: F2
caption: Inactivation of the hmgA gene of P. chlororaphis GP72 by CRISPR‐assisted
  base editing.A. Overview of the homogentisate pathway in P. chlororaphis GP72. The
  three main metabolic blocks involved in biosynthesis and degradation of homogentisate
  are identified with different colour shadings. Grey arrowheads indicate further
  biochemical steps, both upstream and downstream. PEP, phosphoenolpyruvate; E4P,
  erythrose 4‐phosphate; DAHP, 3‐deoxy‐D‐arabinoheptulosonate 7‐phosphate; CHA, chorismate
  and TYR, L‐tyrosine.B. Phenotypes of hmgA inactivation mutants (indicated with an
  asterisk, *) and alignment of the targeted 20‐nt sequence in hmgA mutants obtained
  by base editing. The wild‐type hmgA locus is shown as the double‐stranded DNA sequence
  in the P. chlororaphis GP72 chromosome. The three bases indicated with grey boxes
  represent codons encoding amino acids, and the three bases highlighted in green
  identify the nearest PAM sequence (i.e. NGG). The C residue in the non‐template
  strand, shown in red, is a candidate site for engineering a STOP codon (i.e. CAG→TAG).
  Actual hmgA loci after base editing illustrate the sequencing results of selected
  hmgA* strains, and bases labelled in red in each line pinpoint the occurrence of
  a C → T mutation. Numbers on the left indicate the occurrence of different types
  of mutations relative to the number of individual clones selected for sequencing;
  red, mutations leading to the intended premature STOP codon; blue, mutations leading
  to non‐terminating codons; and black, no mutation. The light blue numbers below
  the sequence alignment represent the position of the N 20 region targeted in the
  hmgA gene. Inactivation efficiency (in red, bold face) was defined as the proportion
  of mutants with premature STOP codons relative to the number of all clones selected
  for sequencing. Base‐editing efficiency (in blue, bold face) was calculated as the
  proportion of strains with any type of mutation relative to the number of sequenced
  clones.
papertitle: Developing a CRISPR‐assisted base‐editing system for genome engineering
  of Pseudomonas chlororaphis .
reftext: Sheng‐Jie Yue, et al. Microb Biotechnol. 2022 Sep;15(9):2324-2336.
year: '2022'
doi: 10.1111/1751-7915.14075
journal_title: Microbial Biotechnology
journal_nlm_ta: Microb Biotechnol
publisher_name: John Wiley and Sons Inc.
keywords: ''
automl_pathway: 0.5183631
figid_alias: PMC9437888__F2
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
redirect_from: /figures/PMC9437888__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9437888__MBT2-15-2324-g007.html
  '@type': Dataset
  description: Inactivation of the hmgA gene of P. chlororaphis GP72 by CRISPR‐assisted
    base editing.A. Overview of the homogentisate pathway in P. chlororaphis GP72.
    The three main metabolic blocks involved in biosynthesis and degradation of homogentisate
    are identified with different colour shadings. Grey arrowheads indicate further
    biochemical steps, both upstream and downstream. PEP, phosphoenolpyruvate; E4P,
    erythrose 4‐phosphate; DAHP, 3‐deoxy‐D‐arabinoheptulosonate 7‐phosphate; CHA,
    chorismate and TYR, L‐tyrosine.B. Phenotypes of hmgA inactivation mutants (indicated
    with an asterisk, *) and alignment of the targeted 20‐nt sequence in hmgA mutants
    obtained by base editing. The wild‐type hmgA locus is shown as the double‐stranded
    DNA sequence in the P. chlororaphis GP72 chromosome. The three bases indicated
    with grey boxes represent codons encoding amino acids, and the three bases highlighted
    in green identify the nearest PAM sequence (i.e. NGG). The C residue in the non‐template
    strand, shown in red, is a candidate site for engineering a STOP codon (i.e. CAG→TAG).
    Actual hmgA loci after base editing illustrate the sequencing results of selected
    hmgA* strains, and bases labelled in red in each line pinpoint the occurrence
    of a C → T mutation. Numbers on the left indicate the occurrence of different
    types of mutations relative to the number of individual clones selected for sequencing;
    red, mutations leading to the intended premature STOP codon; blue, mutations leading
    to non‐terminating codons; and black, no mutation. The light blue numbers below
    the sequence alignment represent the position of the N 20 region targeted in the
    hmgA gene. Inactivation efficiency (in red, bold face) was defined as the proportion
    of mutants with premature STOP codons relative to the number of all clones selected
    for sequencing. Base‐editing efficiency (in blue, bold face) was calculated as
    the proportion of strains with any type of mutation relative to the number of
    sequenced clones.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pep
  - bgcn
  - cha
  - ChAT
  - Hpd
  - tt
  - ca
  - gt
  - Ccp84Ad
  - tRNA:Ala-CGC-1-2
  - tRNA:Ala-CGC-1-3
  - tRNA:Ala-CGC-1-1
  - Tg
  - cag
  - tRNA:Gln-CTG-4-1
  - tRNA:Gln-CTG-3-1
  - tRNA:Gln-CTG-1-1
  - tRNA:Gln-CTG-2-2
  - tRNA:Gln-CTG-2-4
  - tRNA:Gln-CTG-2-1
  - tRNA:Gln-CTG-2-5
  - tRNA:Gln-CTG-2-3
  - PAEP
  - PREP
  - PTPN22
  - TCFL5
  - HPD
  - S100A12
  - GUCY2C
  - GCG
  - TG
  - CXCR1
  - TGM2
---
