---
figid: PMC9246330__BST-50-1151-g0002
figtitle: Crosstalk between the circadian clock and heat stress-responsive pathways
organisms:
- NA
pmcid: PMC9246330
filename: BST-50-1151-g0002.jpg
figlink: /pmc/articles/PMC9246330/figure/BST-50-1151F2/
number: F2
caption: The diagram of circadian oscillation (top) was adapted from []. The colors
  of the clock components represent their expression level in response to elevated
  temperatures from Grinevich et al. [], and Blair et al. []. Increased expression
  is shown in orange and decreased expression in light blue. The heat stress-responsive
  pathway (bottom) was adapted from []. The diagram includes only HSFs with known
  variation in basal expression between times of day in Grinevich et al. []. Colors
  indicate whether HSFs are targeted by the clock proteins based on the ChIP-Seq data
  []. Direct targets of circadian components are shown in dark pink, and light purple
  means they are not the direct targets. For both sections, the shapes of the text
  box represent their expression peaks under normal temperature. Ovals, trapezoids,
  and parallelogram mean high expression in the morning, afternoon, and evening, respectively.
  Rectangles mean no difference in expression between dawn and dusk. Solid lines indicate
  that the regulation occurs via transcriptional activation or repression. Dashed
  lines indicate regulation via post-translational modification. Thin dotted lines
  represent the regulation through a chaperone function. Thick dotted lines indicate
  that the regulatory mechanisms are unknown. The figure was created with BioRender.com.
papertitle: The intersection between circadian and heat-responsive regulatory networks
  controls plant responses to increasing temperatures.
reftext: Kanjana Laosuntisuk, et al. Biochem Soc Trans. 2022 Jun 30;50(3):1151-1165.
year: '2022'
doi: 10.1042/BST20190572
journal_title: Biochemical Society Transactions
journal_nlm_ta: Biochem Soc Trans
publisher_name: Portland Press Ltd.
keywords: circadian clock | eukaryotic gene expression | gene regulatory networks
  | plant signal transduction | temperature sensing | temperature stress
automl_pathway: 0.9251276
figid_alias: PMC9246330__F2
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC9246330__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9246330__BST-50-1151-g0002.html
  '@type': Dataset
  description: The diagram of circadian oscillation (top) was adapted from []. The
    colors of the clock components represent their expression level in response to
    elevated temperatures from Grinevich et al. [], and Blair et al. []. Increased
    expression is shown in orange and decreased expression in light blue. The heat
    stress-responsive pathway (bottom) was adapted from []. The diagram includes only
    HSFs with known variation in basal expression between times of day in Grinevich
    et al. []. Colors indicate whether HSFs are targeted by the clock proteins based
    on the ChIP-Seq data []. Direct targets of circadian components are shown in dark
    pink, and light purple means they are not the direct targets. For both sections,
    the shapes of the text box represent their expression peaks under normal temperature.
    Ovals, trapezoids, and parallelogram mean high expression in the morning, afternoon,
    and evening, respectively. Rectangles mean no difference in expression between
    dawn and dusk. Solid lines indicate that the regulation occurs via transcriptional
    activation or repression. Dashed lines indicate regulation via post-translational
    modification. Thin dotted lines represent the regulation through a chaperone function.
    Thick dotted lines indicate that the regulatory mechanisms are unknown. The figure
    was created with BioRender.com.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CCA1
  - LHY
  - RVE8
  - LCL1
  - PRR5
  - PRR7
  - PRR9
  - AT-HSFA5
  - HSFA9
  - TOC1
  - ELF3
  - PCL1
  - ELF4
  - ZTL
  - HSP70
  - GI
  - DREB2A
  - HSFA2
  - HSFA3
---
