---
figid: PMC9238060__11658_2022_344_Fig2_HTML
figtitle: Epigenetic modifications in drug resistance and treatment of breast cancer
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9238060
filename: 11658_2022_344_Fig2_HTML.jpg
figlink: /pmc/articles/PMC9238060/figure/Fig2/
number: F2
caption: Epigenetic changes in routine and tumor cells in breast cancer. Epigenetic
  modifications such as DNA methylation and histone modification alter DNA accessibility
  and chromatin structure, thereby regulating patterns of gene expression. In normal
  cells, DNA methyltransferases (DNMTs) add a methyl group at position 5 of the pyrimidine
  ring of cytosine, which is known as DNA methylation. The Ten Eleven Translocation
  (TET) family of proteins catalyze the subsequent oxidations of 5-methylcytosine
  (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC), and 5-carboxylcytosine
  (5caC). TET proteins therefore provide an active pathway for DNA demethylation and
  consequently have relevance for regulation of gene expression. TET proteins can
  also mediate active demethylation via excision of 5fC and 5caC by thymine DNA glycosylase
  (TDG). Additionally, histone acetyltransferases (HATs) acetylate conserved lysine
  amino acids on histone proteins by transferring an acetyl group from acetyl-CoA
  to form ε-N-acetyllysine. Also, the methylation of histone H3 at lysine 27 (H3K27me1/2/3)
  is catalyzed by polycomb repressive complex 2 (PRC2)—which is a chromatin-modifying
  enzyme. Moreover, both gene transcriptional repression and normal organismal development
  can be maintained by this complex. Nonetheless, cancer cells have a different epigenetic
  profile. Decreased methylation was seen in the epigenetic profile of cancer cells.
  This decreased methylation can affect the activity of large numbers of genes. Since
  methylation has been linked with decreased gene activity, the implication of hypomethylation
  is to escalate the activity of the affected genes. In this condition, if genes involved
  are in cell growth, this event can result in cell division and, thus, cancer progression.
  In addition, increased activity of histone deacetylases (HDACs) has been found in
  different types of cancer cells such as breast cancer, and it is marked by a loss
  of histone acetyl markers. Histone methyltransferases (HMTs) are enzymes that add
  methyl groups to histones, while histone demethylases (HDMs) have the inverse function.
  In breast cancer cells, HMTs may be altered so that they place methyl groups at
  the wrong spot, thereby silencing tumor suppressor genes. All these events as well
  as mutation of SWI/SNF chromatin remodeling complex subunits can lead to abnormal
  chromatin structure, contributing to cancer development
papertitle: The role of epigenetic modifications in drug resistance and treatment
  of breast cancer.
reftext: Mohsen Karami Fath, et al. Cell Mol Biol Lett. 2022;27:52.
year: '2022'
doi: 10.1186/s11658-022-00344-6
journal_title: Cellular & Molecular Biology Letters
journal_nlm_ta: Cell Mol Biol Lett
publisher_name: BioMed Central
keywords: Breast cancer | Epigenetic modifications | microRNAs | Treatment | Chemoresistance
automl_pathway: 0.9608746
figid_alias: PMC9238060__F2
figtype: Figure
redirect_from: /figures/PMC9238060__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9238060__11658_2022_344_Fig2_HTML.html
  '@type': Dataset
  description: Epigenetic changes in routine and tumor cells in breast cancer. Epigenetic
    modifications such as DNA methylation and histone modification alter DNA accessibility
    and chromatin structure, thereby regulating patterns of gene expression. In normal
    cells, DNA methyltransferases (DNMTs) add a methyl group at position 5 of the
    pyrimidine ring of cytosine, which is known as DNA methylation. The Ten Eleven
    Translocation (TET) family of proteins catalyze the subsequent oxidations of 5-methylcytosine
    (5mC) to 5-hydroxymethylcytosine (5hmC), 5-formylcytosine (5fC), and 5-carboxylcytosine
    (5caC). TET proteins therefore provide an active pathway for DNA demethylation
    and consequently have relevance for regulation of gene expression. TET proteins
    can also mediate active demethylation via excision of 5fC and 5caC by thymine
    DNA glycosylase (TDG). Additionally, histone acetyltransferases (HATs) acetylate
    conserved lysine amino acids on histone proteins by transferring an acetyl group
    from acetyl-CoA to form ε-N-acetyllysine. Also, the methylation of histone H3
    at lysine 27 (H3K27me1/2/3) is catalyzed by polycomb repressive complex 2 (PRC2)—which
    is a chromatin-modifying enzyme. Moreover, both gene transcriptional repression
    and normal organismal development can be maintained by this complex. Nonetheless,
    cancer cells have a different epigenetic profile. Decreased methylation was seen
    in the epigenetic profile of cancer cells. This decreased methylation can affect
    the activity of large numbers of genes. Since methylation has been linked with
    decreased gene activity, the implication of hypomethylation is to escalate the
    activity of the affected genes. In this condition, if genes involved are in cell
    growth, this event can result in cell division and, thus, cancer progression.
    In addition, increased activity of histone deacetylases (HDACs) has been found
    in different types of cancer cells such as breast cancer, and it is marked by
    a loss of histone acetyl markers. Histone methyltransferases (HMTs) are enzymes
    that add methyl groups to histones, while histone demethylases (HDMs) have the
    inverse function. In breast cancer cells, HMTs may be altered so that they place
    methyl groups at the wrong spot, thereby silencing tumor suppressor genes. All
    these events as well as mutation of SWI/SNF chromatin remodeling complex subunits
    can lead to abnormal chromatin structure, contributing to cancer development
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Brca2
  - Tdg
  - de
  - Tet
  - Me
  - hdm
  - Su(var)3-3
  - His3.3B
  - His3.3A
  - hat
  - Elp3
  - CG1894
  - chm
  - snf
  - hfw
  - Iswi
  - ac
  - Coa
  - sam
  - qkr54B
  - sls
  - Sam-S
  - BRCA1
  - BRCA2
  - TDG
  - DNMT3A
  - DNMT1
  - TMPRSS11D
  - SMARCA1
  - ADCY1
  - ADCY2
  - ADCY3
  - ADCY4
  - ADCY5
  - ADCY6
  - ADCY7
  - ADCY8
  - ADCY9
  - ADCY10
  - EED
  - EZH1
  - EZH2
  - SUZ12
  - ME3
  - PCDHGA5
---
