---
figid: PMC9622871__41598_2022_21091_Fig8_HTML
figtitle: Drug transporters OAT1 and OAT3 have specific effects on multiple organs
  and gut microbiome as revealed by contextualized metabolic network reconstructions
organisms:
- Mus musculus
- Lactococcus lactis
- Bacillus subtilis
- Mycobacterium tuberculosis
- Avena sativa
- Escherichia coli str. K-12 substr. MG1655
- Escherichia coli
- human metagenome
- Drosophila melanogaster
- gut metagenome
- Danio rerio
- Homo sapiens
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
- Mus musculus
- Danio rerio
- Arabidopsis thaliana
pmcid: PMC9622871
filename: 41598_2022_21091_Fig8_HTML.jpg
figlink: /pmc/articles/PMC9622871/figure/Fig8/
number: F8
caption: 'Kidney-microbiome interactions revealed by analysis of reaction co-sets
  connects host (kidney) and gut microbial metabolic pathways via transport. Reaction
  co-sets provide a means of identifying functionally correlated pathways by delineation
  of groups of reactions that are linked together based upon flux correlations. The
  kidney-microbiome steady state flux space was interrogated in order to identify
  reaction co-sets involving the host as well as the microbiome, reflected as changes
  in the plasma metabolite concentrations. The illustrated central metabolic pathways
  centering around hexose and pentose sugar metabolism provide direct links between
  the host and microbiome. Links between glycolysis and the non-oxidative pentose
  phosphate pathway are directly coupled via inter-organismal (host-microbiome) transport
  of dihydroxyacetone phosphate and dihydroxyacetone. Other metabolites such as xanthine
  and L-alanine are present in some but not all of the co-sets for the four models
  (Oat1WT, Oat1KO, Oat3WT, Oat3KO), reflecting the condition/context dependence of
  the co-sets. Reactions with blue names are present in all four models (reactions
  with red names are present in one, two, or three models). Abbreviations are as follows,
  with those enzymes and transporters present in the microbiome as opposed to the
  host indicated as “x_mb (microbiome)”: ALATA_L l-alanine transaminase, PSERT phosphoserine
  transaminase, PGCD phosphoglycerate dehydrogenase, PSP_L phosphoserine phosphatase
  (l-serine), r0841 RU5P ER to cytosol transport, r0249 d-ribose-5-phosphate ketol-isomerase,
  r0840 R5P ER to cytosol transport, PPM phosphopentomutase, TKT1: Transketolase,
  RPEc ribulose 5-phosphate 3-epimerase, TKT2 transketolase, TRIOK triokinase, GLYALDDr
  d-Glyceraldehyde dehydrogenase, HPYRRy hydroxypyruvate reductase (NADPH), TPI triose-phosphate
  isomerase, XANt xanthine reversible transport, r0242 glycerone phosphate phosphohydrolase,
  DHAPtr dihydroxyacetone phosphate transport, XANt2_mb xanthine transport (microbiome),
  PPM_mb phosphopentomutase (microbiome), RPI_mb ribose phosphate isomerase (microbiome),
  RPE_mb ribulose 5-phosphate 3-epimerase (microbiome), TKT1_mb transketolase (microbiome),
  TKT2_mb transketolase (microbiome), TALA_mb transaldolase (microbiome), TPI_mb triose-phosphate
  isomerase (microbiome), GAPD_mb glyceraldehyde-3-phosphate dehydrogenase (microbiome),
  PGK_mb phosphoglycerate kinase (microbiome), ATPS4r_mb ATP synthase (microbiome),
  DHAt_mb dihydroxyacetone transport (microbiome), ALKP_mb alkaline phosphatase (microbiome),
  FBA_mb: Fructose-bisphosphate aldolase (microbiome), PFK_mb phosphofructokinase
  (microbiome), PYK_mb pyruvate kinase (microbiome), AKG alpha ketoglutarate, GLU-L
  l-glutamate, R1P ribose-1-phosphate, G3P glyceraldehyde 3-phosphate, XAN xanthine,
  DHAP dihydroxyacetone phosphate, DHA dihydroxyacetone, PYR pyruvate.'
papertitle: Drug transporters OAT1 and OAT3 have specific effects on multiple organs
  and gut microbiome as revealed by contextualized metabolic network reconstructions.
reftext: Neema Jamshidi, et al. Sci Rep. 2022;12:18308.
year: '2022'
doi: 10.1038/s41598-022-21091-w
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Biochemistry | Computational biology and bioinformatics | Kidney | Metabolism
  | Systems biology | Biochemical networks | Computer modelling | Genetic interaction
  | Systems analysis
automl_pathway: 0.957745
figid_alias: PMC9622871__F8
figtype: Figure
redirect_from: /figures/PMC9622871__F8
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9622871__41598_2022_21091_Fig8_HTML.html
  '@type': Dataset
  description: 'Kidney-microbiome interactions revealed by analysis of reaction co-sets
    connects host (kidney) and gut microbial metabolic pathways via transport. Reaction
    co-sets provide a means of identifying functionally correlated pathways by delineation
    of groups of reactions that are linked together based upon flux correlations.
    The kidney-microbiome steady state flux space was interrogated in order to identify
    reaction co-sets involving the host as well as the microbiome, reflected as changes
    in the plasma metabolite concentrations. The illustrated central metabolic pathways
    centering around hexose and pentose sugar metabolism provide direct links between
    the host and microbiome. Links between glycolysis and the non-oxidative pentose
    phosphate pathway are directly coupled via inter-organismal (host-microbiome)
    transport of dihydroxyacetone phosphate and dihydroxyacetone. Other metabolites
    such as xanthine and L-alanine are present in some but not all of the co-sets
    for the four models (Oat1WT, Oat1KO, Oat3WT, Oat3KO), reflecting the condition/context
    dependence of the co-sets. Reactions with blue names are present in all four models
    (reactions with red names are present in one, two, or three models). Abbreviations
    are as follows, with those enzymes and transporters present in the microbiome
    as opposed to the host indicated as “x_mb (microbiome)”: ALATA_L l-alanine transaminase,
    PSERT phosphoserine transaminase, PGCD phosphoglycerate dehydrogenase, PSP_L phosphoserine
    phosphatase (l-serine), r0841 RU5P ER to cytosol transport, r0249 d-ribose-5-phosphate
    ketol-isomerase, r0840 R5P ER to cytosol transport, PPM phosphopentomutase, TKT1:
    Transketolase, RPEc ribulose 5-phosphate 3-epimerase, TKT2 transketolase, TRIOK
    triokinase, GLYALDDr d-Glyceraldehyde dehydrogenase, HPYRRy hydroxypyruvate reductase
    (NADPH), TPI triose-phosphate isomerase, XANt xanthine reversible transport, r0242
    glycerone phosphate phosphohydrolase, DHAPtr dihydroxyacetone phosphate transport,
    XANt2_mb xanthine transport (microbiome), PPM_mb phosphopentomutase (microbiome),
    RPI_mb ribose phosphate isomerase (microbiome), RPE_mb ribulose 5-phosphate 3-epimerase
    (microbiome), TKT1_mb transketolase (microbiome), TKT2_mb transketolase (microbiome),
    TALA_mb transaldolase (microbiome), TPI_mb triose-phosphate isomerase (microbiome),
    GAPD_mb glyceraldehyde-3-phosphate dehydrogenase (microbiome), PGK_mb phosphoglycerate
    kinase (microbiome), ATPS4r_mb ATP synthase (microbiome), DHAt_mb dihydroxyacetone
    transport (microbiome), ALKP_mb alkaline phosphatase (microbiome), FBA_mb: Fructose-bisphosphate
    aldolase (microbiome), PFK_mb phosphofructokinase (microbiome), PYK_mb pyruvate
    kinase (microbiome), AKG alpha ketoglutarate, GLU-L l-glutamate, R1P ribose-1-phosphate,
    G3P glyceraldehyde 3-phosphate, XAN xanthine, DHAP dihydroxyacetone phosphate,
    DHA dihydroxyacetone, PYR pyruvate.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tpi
  - mb
  - E(spl)mbeta-HLH
  - Rpi
  - Rpe
  - Gapdh2
  - Gapdh1
  - Pfk
  - CG15812
  - PyK
  - pyr
  - TKTL1
  - TPI1
  - TKT
  - MB
  - RPIA
  - RPE
  - GAPDH
  - PFKL
  - PFKM
  - PFKP
  - MSMB
  - PSPN
  - PSPH
  - REG1A
  - STXBP3
  - RIDA
  - BPIFA2
  - GLUL
  - PHKA2
  - Tpi1
  - Mb
  - Rpia
  - Gapdh
  - Bpifa2
  - Pspn
  - Reg1
  - Reg2
  - Psph
  - Glul
  - dharma
  - rpe
  - gapdh
  - pfkmb
  - TPI
  - RSW10
  - PSP
  - DHA
---
