---
figid: PMC9692235__fcell-10-1010639-g006
figtitle: Transducin-like enhancer of split (TLE) family proteins in tumorigenesis
  and immune regulation
organisms:
- Mus musculus
- Rattus norvegicus
- Mouse mammary tumor virus
- Lareunionomyces loeiensis
- Drosophila melanogaster
- Danio rerio
- Xenopus laevis
- Homo sapiens
organisms_ner:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Drosophila melanogaster
- Danio rerio
- Xenopus laevis
pmcid: PMC9692235
filename: fcell-10-1010639-g006.jpg
figlink: /pmc/articles/PMC9692235/figure/F6/
number: F6
caption: TLE-regulated pathways in AML and T-ALL development. On the one hand, deletion
  of a portion of the long arm of chromosome 9, del (9q), or CpG island hypermethylation
  in the TLE1 promoter causes genetic or epigenetic silencing of TLE1 and TLE4 in
  AML cells. The loss of TLE1/TLE4 disrupts the TLE/HES1/HDAC complex, which otherwise
  inhibits AML growth by increasing p53/p21 and decreasing Bcl-2 expression (). Loss
  of HES1/TLEs may also promote AML growth by derepressing FLT3 expression () and
  enhancing Wnt signaling activation. On the other hand, AML oncogenic events promote
  cell proliferation and disrupt the terminal differentiation of myeloid cells, both
  of which are required for AML development. Notch activation promotes T-ALL development
  by several means. It directly upregulates the Notch target gene MYC to promote tumor
  cell survival. Notch activation inhibits E47, which leads to committed T lineage
  differentiation of T-ALL cells. In addition, Notch activates HES1 expression and
  suppresses the expression of PTEN and CYLD, thereby enhancing PI3K and NF-κB pathway
  activation and cell proliferation.
papertitle: Roles of transducin-like enhancer of split (TLE) family proteins in tumorigenesis
  and immune regulation.
reftext: Guiping Yu, et al. Front Cell Dev Biol. 2022;10:1010639.
year: '2022'
doi: 10.3389/fcell.2022.1010639
journal_title: Frontiers in Cell and Developmental Biology
journal_nlm_ta: Front Cell Dev Biol
publisher_name: Frontiers Media S.A.
keywords: TLE family proteins | transcriptional repressors | tumorigenesis | inflammation
  | cancer therapy
automl_pathway: 0.9576773
figid_alias: PMC9692235__F6
figtype: Figure
redirect_from: /figures/PMC9692235__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9692235__fcell-10-1010639-g006.html
  '@type': Dataset
  description: TLE-regulated pathways in AML and T-ALL development. On the one hand,
    deletion of a portion of the long arm of chromosome 9, del (9q), or CpG island
    hypermethylation in the TLE1 promoter causes genetic or epigenetic silencing of
    TLE1 and TLE4 in AML cells. The loss of TLE1/TLE4 disrupts the TLE/HES1/HDAC complex,
    which otherwise inhibits AML growth by increasing p53/p21 and decreasing Bcl-2
    expression (). Loss of HES1/TLEs may also promote AML growth by derepressing FLT3
    expression () and enhancing Wnt signaling activation. On the other hand, AML oncogenic
    events promote cell proliferation and disrupt the terminal differentiation of
    myeloid cells, both of which are required for AML development. Notch activation
    promotes T-ALL development by several means. It directly upregulates the Notch
    target gene MYC to promote tumor cell survival. Notch activation inhibits E47,
    which leads to committed T lineage differentiation of T-ALL cells. In addition,
    Notch activates HES1 expression and suppresses the expression of PTEN and CYLD,
    thereby enhancing PI3K and NF-κB pathway activation and cell proliferation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TCF3
  - NOTCH1
  - NOTCH2
  - NOTCH3
  - NOTCH4
  - MYC
  - TLE1
  - TLX1
  - HES1
  - GATD3
  - TLE4
  - PTEN
  - CYLD
  - NFKB1
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - HES4
  - BCL2
  - TP53
  - TP63
  - TP73
  - CDKN1A
  - TCEAL1
  - NSG1
  - H3P16
  - FLT3
  - TLE3
  - FOXC1
  - RUNX1
  - HDAC1
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - Tcf3
  - Myc
  - Nol3
  - Tle1
  - Tlx1
  - Hes1
  - Hesx1
  - Tle4
  - Pten
  - Cyld
  - Nfkb1
  - Pik3r1
  - Bcl2
  - Trp53
  - Cdkn1a
  - Nsg1
  - Tpt1
  - Tceal1
  - Flt3
  - Tle3
  - Foxc1
  - Runx1
  - Hdac1
  - Hdac1-ps
  - Notch1
  - Pik3cg
  - Tp53
  - Kras
  - Wnt2
  - Dif
  - dl
  - Rel
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Pi3K21B
  - Pi3K59F
  - Pi3K68D
  - Pi3K92E
  - del
  - tx
  - mirr
  - fs(1)de1
  - Debcl
  - p53
  - betaTub60D
  - hth
  - dap
  - Wnt5
  - Wnt4
  - wg
  - myca
  - mycbp2
  - tle2b
  - tlx1
  - ptenb
  - cylda
  - bcl2a
  - tp53
  - cdkn1a
  - flt3
  - hdac1
  - notch1.S
  - notch1.L
  - myc.S
  - myc.L
  - tle1.S
  - tlx1.L
  - hes1.S
  - hes1.L
  - tle4.S
  - nfkb1.L
  - nfkb1.S
  - pik3ca.L
  - pik3cg.L
  - pik3cg.S
  - pik3cb.L
  - pik3ca.S
  - pik3cb.S
  - zbtb7b.S
  - zbtb7b.L
  - runx1.L
  - runx1.S
  - hes4.L
  - bcl2.S
  - bcl2.L
  - tp53.L
  - tp53.S
  - cdkn1a.L
  - nsg1.L
  - nsg1.S
  - cdkn1a.S
  - flt3.L
  - flt3.S
  - tle2.L
  - foxc1.S
  - foxc1.L
  - hdac1.S
---
