---
figid: PMC9569789__ijms-23-11301-g002
figtitle: 'Impact of Non-Nodulating Diazotrophic Bacteria in Agriculture: Understanding
  the Molecular Mechanisms That Benefit Crops'
organisms:
- NA
pmcid: PMC9569789
filename: ijms-23-11301-g002.jpg
figlink: /pmc/articles/PMC9569789/figure/ijms-23-11301-f002/
number: F2
caption: 'Molecular regulatory models identified in plant–diazotrophic bacteria association.
  The main studies involve the recognition via 16S rRNA of the plant microbiome and
  the regulation of plant genes involved in nitrogen metabolism, phytohormone regulation,
  antioxidant metabolism, cell wall, and plant receptors involved in recognition of
  diazotrophic bacteria. Certain pathways lead to the promotion of growth and tolerance
  to biotic and abiotic stresses. Blue arrows: expression is up-regulated; Red arrows:
  expression is down-regulated; Gray arrow: flow of the pathway. ACC: 1-aminocyclopropane-1-carboxylic
  acid, AdoMet: adenosylmethionine, AFB2: Auxin signaling F-box 2, AFB4: Auxin Signaling
  F-Box 4, APX: ascorbate peroxidase, AUX/IAA31: Indole-3-Acetic Acid Inducible 31,
  CAT: Catalase, CES: cellulose synthase, CYP: cytochrome P450s, ERF: ethylene response
  elongation factor, ET: ethylene, ETR: Ethylene Receptors, FLS2: Flagellin-Sensing
  2, GH3: Gretchen Hagen 3, GS: glutamine synthetase, ILL/ILR: IAA-Leucine Resistant
  (IRL)-Like, LAC: Laccase, N: nitrogen, NBS-LLR: Nucleotide-Binding Sites and Leucine-Rich
  Repeats, NR: nitrate reductase, NiR: nitrite reductase, rRNA: ribosomal RNA, SAUR:
  Small auxin-up RNA, UGT: UDP glucosyltransferases, WAK: Wall-Associated Kinases.
  Created with BioRender.com accessed on 31 August 2022.'
papertitle: 'The Impact of Non-Nodulating Diazotrophic Bacteria in Agriculture: Understanding
  the Molecular Mechanisms That Benefit Crops.'
reftext: Flávia Thiebaut, et al. Int J Mol Sci. 2022 Oct;23(19):11301.
year: '2022'
doi: 10.3390/ijms231911301
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: bioinoculants | non-legume plants | non-nodular bacteria
automl_pathway: 0.8720077
figid_alias: PMC9569789__F2
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC9569789__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9569789__ijms-23-11301-g002.html
  '@type': Dataset
  description: 'Molecular regulatory models identified in plant–diazotrophic bacteria
    association. The main studies involve the recognition via 16S rRNA of the plant
    microbiome and the regulation of plant genes involved in nitrogen metabolism,
    phytohormone regulation, antioxidant metabolism, cell wall, and plant receptors
    involved in recognition of diazotrophic bacteria. Certain pathways lead to the
    promotion of growth and tolerance to biotic and abiotic stresses. Blue arrows:
    expression is up-regulated; Red arrows: expression is down-regulated; Gray arrow:
    flow of the pathway. ACC: 1-aminocyclopropane-1-carboxylic acid, AdoMet: adenosylmethionine,
    AFB2: Auxin signaling F-box 2, AFB4: Auxin Signaling F-Box 4, APX: ascorbate peroxidase,
    AUX/IAA31: Indole-3-Acetic Acid Inducible 31, CAT: Catalase, CES: cellulose synthase,
    CYP: cytochrome P450s, ERF: ethylene response elongation factor, ET: ethylene,
    ETR: Ethylene Receptors, FLS2: Flagellin-Sensing 2, GH3: Gretchen Hagen 3, GS:
    glutamine synthetase, ILL/ILR: IAA-Leucine Resistant (IRL)-Like, LAC: Laccase,
    N: nitrogen, NBS-LLR: Nucleotide-Binding Sites and Leucine-Rich Repeats, NR: nitrate
    reductase, NiR: nitrite reductase, rRNA: ribosomal RNA, SAUR: Small auxin-up RNA,
    UGT: UDP glucosyltransferases, WAK: Wall-Associated Kinases. Created with BioRender.com
    accessed on 31 August 2022.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CESA4
  - CESA2
  - CESA9
  - MIR408
  - IRX12
  - LAC13
  - FLS2
  - SPP
  - AFB4
  - IAA31
  - AFB2
  - CNGC4
  - APX1
  - APX2
  - CSD1
  - NIR1
---
