---
figid: PMC9316036__metabolites-12-00569-g001
figtitle: Early Evolutionary Selection of NAD Biosynthesis Pathway in Bacteria
organisms:
- NA
organisms_ner:
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC9316036
filename: metabolites-12-00569-g001.jpg
figlink: /pmc/articles/PMC9316036/figure/metabolites-12-00569-f001/
number: F1
caption: Schematic representation of reactions involved in the biosynthesis of NAD
  in different organisms. Metabolite names are written in bold blue letters, while
  the enzyme names are denoted in green colour. The abbreviation NCE (NAD-consuming
  enzymes) represents all enzymes catalysing signalling reactions consuming NAD. The
  bold black lines represent enzyme-catalysed reactions, whereas the dotted red lines
  denote non-enzymatic hydrolysis, mainly occurring at high temperatures. Bold yellow
  lines are used to denote reactions that require ATP. Whether the bacterial 5′-nucleotidase
  SurE accepts NMN/NAMN as substrate is unknown; a homologue to the mammalian NT5
  could not be identified in bacteria. NAD, Nam adenine dinucleotide; NMN, Nam mononucleotide;
  NR, Nam riboside; NA, nicotinic acid; NAR, NA riboside; NAMN, NA mononucleotide;
  NAAD, NA adenine dinucleotide; NadD, NA/NMN adenylyltransferase; Nampt, Nam phosphoribosyltransferase;
  NadR, NA/Nam riboside kinase; SurE, 5′-nucleotidase; PNP, purine nucleoside phosphorylase;
  PncA, nicotinamidase; PncB, NA phosphoribosyltransferase; NadE, NAD synthase.
papertitle: Early Evolutionary Selection of NAD Biosynthesis Pathway in Bacteria.
reftext: Suraj Sharma, et al. Metabolites. 2022 Jul;12(7):569.
year: '2022'
doi: 10.3390/metabo12070569
journal_title: Metabolites
journal_nlm_ta: Metabolites
publisher_name: MDPI
keywords: NAD biosynthesis | metabolic modelling | kinetic models | phylogenetic analysis
automl_pathway: 0.948046
figid_alias: PMC9316036__F1
figtype: Figure
redirect_from: /figures/PMC9316036__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9316036__metabolites-12-00569-g001.html
  '@type': Dataset
  description: Schematic representation of reactions involved in the biosynthesis
    of NAD in different organisms. Metabolite names are written in bold blue letters,
    while the enzyme names are denoted in green colour. The abbreviation NCE (NAD-consuming
    enzymes) represents all enzymes catalysing signalling reactions consuming NAD.
    The bold black lines represent enzyme-catalysed reactions, whereas the dotted
    red lines denote non-enzymatic hydrolysis, mainly occurring at high temperatures.
    Bold yellow lines are used to denote reactions that require ATP. Whether the bacterial
    5′-nucleotidase SurE accepts NMN/NAMN as substrate is unknown; a homologue to
    the mammalian NT5 could not be identified in bacteria. NAD, Nam adenine dinucleotide;
    NMN, Nam mononucleotide; NR, Nam riboside; NA, nicotinic acid; NAR, NA riboside;
    NAMN, NA mononucleotide; NAAD, NA adenine dinucleotide; NadD, NA/NMN adenylyltransferase;
    Nampt, Nam phosphoribosyltransferase; NadR, NA/Nam riboside kinase; SurE, 5′-nucleotidase;
    PNP, purine nucleoside phosphorylase; PncA, nicotinamidase; PncB, NA phosphoribosyltransferase;
    NadE, NAD synthase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pncB
  - umpG
  - nadD
  - nadR
  - pnp
  - pncA
  - nadE
---
