---
figid: PMC9047021__fcimb-12-804689-g001
figtitle: Impacts of MicroRNAs Induced by the Gut Microbiome on Regulating the Development
  of Colorectal Cancer
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9047021
filename: fcimb-12-804689-g001.jpg
figlink: /pmc/articles/PMC9047021/figure/f1/
number: F1
caption: miRNA processing pathway. Host-derived miRNAs are synthesized in the nucleus
  and processed and then function in the cytoplasm. miRNA genes are transcribed into
  pri-miRNA through RNA polymerase II or RNA polymerase III subsequently cleaved by
  the microprocessor complex Drosha-DGCR8. The resulting precursor hairpins, the pre-miRNAs,
  are exported from the nucleus to the cytoplasm by exportin-5–Ran-GTP. In the cytoplasm,
  pre-miRNAs are cleaved into mature length by the Dicer in complex with TRBP. Functional
  strands of mature miRNAs are assembled with AGO proteins and a glycinetryptophan
  repeat-containing protein of 182 kDa (GW182), and then miRISC mediating target mRNAs
  silencing are recruited, whereas passenger strands are degraded. miRNAs regulate
  gene expression through two mechanisms. First, miRNA, with a wide range of complementary
  base pairs with mRNA, will guide miRISC to degrade mRNA, resulting in the instability
  or suppression of translation. Second, if miRNA and mRNA have partially complementary
  sequences, the miRISC will inhibit mRNA translation through the AGO protein.
papertitle: Impacts of MicroRNAs Induced by the Gut Microbiome on Regulating the Development
  of Colorectal Cancer.
reftext: Juan Xing, et al. Front Cell Infect Microbiol. 2022;12:804689.
year: '2022'
doi: 10.3389/fcimb.2022.804689
journal_title: Frontiers in Cellular and Infection Microbiology
journal_nlm_ta: Front Cell Infect Microbiol
publisher_name: Frontiers Media S.A.
keywords: colorectal cancer | microRNAs | gut microbiome | bioactive dietary components
  | modulation
automl_pathway: 0.9459608
figid_alias: PMC9047021__F1
figtype: Figure
redirect_from: /figures/PMC9047021__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9047021__fcimb-12-804689-g001.html
  '@type': Dataset
  description: miRNA processing pathway. Host-derived miRNAs are synthesized in the
    nucleus and processed and then function in the cytoplasm. miRNA genes are transcribed
    into pri-miRNA through RNA polymerase II or RNA polymerase III subsequently cleaved
    by the microprocessor complex Drosha-DGCR8. The resulting precursor hairpins,
    the pre-miRNAs, are exported from the nucleus to the cytoplasm by exportin-5–Ran-GTP.
    In the cytoplasm, pre-miRNAs are cleaved into mature length by the Dicer in complex
    with TRBP. Functional strands of mature miRNAs are assembled with AGO proteins
    and a glycinetryptophan repeat-containing protein of 182 kDa (GW182), and then
    miRISC mediating target mRNAs silencing are recruited, whereas passenger strands
    are degraded. miRNAs regulate gene expression through two mechanisms. First, miRNA,
    with a wide range of complementary base pairs with mRNA, will guide miRISC to
    degrade mRNA, resulting in the instability or suppression of translation. Second,
    if miRNA and mRNA have partially complementary sequences, the miRISC will inhibit
    mRNA translation through the AGO protein.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ran
  - RanGAP
  - pasha
  - drosha
  - Polr2A
  - Prim1
  - PolA1
  - PolA2
  - sv
  - Prim2
  - svr
  - ago
  - AGO1
  - AGO2
  - loqs
  - Dcr-2
  - Dcr-1
  - tt
  - MTG1
  - RAN
  - DGCR8
  - DROSHA
  - ERVW-4
  - POLR2A
  - POLR2B
  - POLR2C
  - POLR2D
  - POLR2E
  - POLR2F
  - POLR2G
  - POLR2H
  - POLR2I
  - POLR2J
  - POLR2J2
  - POLR2K
  - POLR2L
  - FBXW7
  - TARBP2
  - TARBP2P1
  - NCOA6
  - DICER1
---
