---
figid: PMC9656133__cells-11-03396-g006a
figtitle: Integrated Multi-Omic Characterization of the Detachment Process of Adherent
  Vero Cells with Animal-Based and Animal-Origin-Free Enzymes
organisms:
- Mus musculus
- Rattus norvegicus
- Variola virus
- Yellow fever virus
- Lyssavirus rabies
- Severe acute respiratory syndrome coronavirus 2
- Agriosomyces longus
- Porcine epidemic diarrhea virus
- Venustaconcha ellipsiformis
- Enterovirus A71
- Saccharomyces cerevisiae
- Juglans regia
- Candida albicans
- Danio rerio
- Homo sapiens
- Canis lupus familiaris
- NA
- NA
organisms_ner:
- Mus musculus
- Homo sapiens
- Escherichia coli str. K-12 substr. MG1655
pmcid: PMC9656133
filename: cells-11-03396-g006a.jpg
figlink: .na.character
number: F6a
caption: Combined metabolome and transcriptome analysis. (a) PathView enrichment pathway
  diagram (positive ion mode). According to the enrichment results, the metabolic
  pathways enriched in both metabolomics and transcriptomics are shown as pathway
  maps. In the KEGG pathway diagram, circles represent metabolites; blue circles represent
  downregulated differential metabolites, and yellow circles represent upregulated
  differential metabolites. The boxes represent genes or proteins, where the green
  boxes represent genes whose expression levels are differentially downregulated and
  the red boxes represent genes whose expression levels are differentially upregulated.
  (b) PathView enrichment pathway diagram (negative ion mode).
papertitle: Integrated Multi-Omic Characterization of the Detachment Process of Adherent
  Vero Cells with Animal-Based and Animal-Origin-Free Enzymes.
reftext: Shouzhi Yu, et al. Cells. 2022 Nov;11(21):3396.
year: '2022'
doi: 10.3390/cells11213396
journal_title: Cells
journal_nlm_ta: Cells
publisher_name: MDPI
keywords: cell detachment | cell culture | transcriptomics | proteomics | metabolomics
automl_pathway: 0.9081026
figid_alias: PMC9656133__F6a
figtype: Figure
redirect_from: /figures/PMC9656133__F6a
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9656133__cells-11-03396-g006a.html
  '@type': Dataset
  description: Combined metabolome and transcriptome analysis. (a) PathView enrichment
    pathway diagram (positive ion mode). According to the enrichment results, the
    metabolic pathways enriched in both metabolomics and transcriptomics are shown
    as pathway maps. In the KEGG pathway diagram, circles represent metabolites; blue
    circles represent downregulated differential metabolites, and yellow circles represent
    upregulated differential metabolites. The boxes represent genes or proteins, where
    the green boxes represent genes whose expression levels are differentially downregulated
    and the red boxes represent genes whose expression levels are differentially upregulated.
    (b) PathView enrichment pathway diagram (negative ion mode).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Fam120a
  - Decr1
  - Pepb
  - Lap3
  - Pepd
  - rs
  - Prss2
  - FAM120A
  - DECR1
  - ANPEP
  - CNDP2
  - PEPB
  - LAP3
  - CXCR4
  - PEPD
  - fnr
  - pepN
  - pepA
  - pepB
  - pepD
  - trpA
  - L-Amino acid
  - Glutathione
  - chsulfide
  - 5-Oxoproline
  - Glutamyl
  - L-amino
  - NADPH
  - Arginine
  - Cysteine
  - methionine
  - Cysteinyl
  - glycine
  - NADP
  - Aminopropyl
  - cadaverine
  - Cyanoamino acid
  - GSH
  - Spermidine
  - Spermine
  - R-S-Glutathione
  - Taurine
  - Glycine
  - L-Glutamate
  - Glutamylcysteine
  - Glutamate
  - Glutathionyl
  - spermidine
  - Trypanothione
  - spermine
  - aminopropyl
  - R-S-Cysteine
  - glutathionyl
  - Homotrypanothione
  - glutamylcystine
  - QAcetyl-CoA
  - disulfide
  - Ascorbate
  - Mercapturic acid
  - Bis(glutathionyl)spermine
---
