---
figid: PMC9509380__41598_2022_20305_Fig1_HTML
figtitle: Lead-exposure associated miRNAs in humans and Alzheimer’s disease as potential
  biomarkers of the disease and disease processes
organisms:
- Mus musculus
- Rattus norvegicus
- Human alphaherpesvirus 1
- Human endogenous retrovirus K
- Homo sapiens
- Clostridium perfringens
organisms_ner:
- Drosophila melanogaster
pmcid: PMC9509380
filename: 41598_2022_20305_Fig1_HTML.jpg
figlink: /pmc/articles/PMC9509380/figure/Fig1/
number: F1
caption: Workflow of the study. (1) We explored heavy metal (lead and cadmium) associated
  miRNAs in blood from a general population derived from the NSHDS by using feature
  selection and linear mixed model. (2) The resulting miRNAs were validated for expression
  in human brain by data from Memories, and publicly available dataset with larger
  sample size for different brain regions was used as additional validation and checked
  for differential expression. Subsequently, we derived experimentally validated gene
  targets for these miRNAs, which were also validated for expression in human brain
  (data derived from the Memories project). Next, we selected gene targets that showed
  an association with the identified miRNAs in transcriptomics data of NSHDS by using
  linear mixed model, and selected AD related gene targets based on the transcriptomics
  data of Memories project by using feature selection. (3) The identified gene targets
  were subjected to pathway analysis to elucidate molecular mechanisms of heavy metal-associated
  miRNAs. Moreover, we performed network analysis to interpret complex miRNAs-TFs-targets
  regulation.
papertitle: Lead-exposure associated miRNAs in humans and Alzheimer’s disease as potential
  biomarkers of the disease and disease processes.
reftext: Qingfeng Wen, et al. Sci Rep. 2022;12:15966.
year: '2022'
doi: 10.1038/s41598-022-20305-5
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Alzheimer's disease | Environmental impact | Gene expression | Computational
  biology and bioinformatics | Biomarkers
automl_pathway: 0.8866157
figid_alias: PMC9509380__F1
figtype: Figure
redirect_from: /figures/PMC9509380__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9509380__41598_2022_20305_Fig1_HTML.html
  '@type': Dataset
  description: Workflow of the study. (1) We explored heavy metal (lead and cadmium)
    associated miRNAs in blood from a general population derived from the NSHDS by
    using feature selection and linear mixed model. (2) The resulting miRNAs were
    validated for expression in human brain by data from Memories, and publicly available
    dataset with larger sample size for different brain regions was used as additional
    validation and checked for differential expression. Subsequently, we derived experimentally
    validated gene targets for these miRNAs, which were also validated for expression
    in human brain (data derived from the Memories project). Next, we selected gene
    targets that showed an association with the identified miRNAs in transcriptomics
    data of NSHDS by using linear mixed model, and selected AD related gene targets
    based on the transcriptomics data of Memories project by using feature selection.
    (3) The identified gene targets were subjected to pathway analysis to elucidate
    molecular mechanisms of heavy metal-associated miRNAs. Moreover, we performed
    network analysis to interpret complex miRNAs-TFs-targets regulation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - fig
  - ad
  - FIG4
---
