---
figid: PMC9246714__41564_2022_1152_Fig1_HTML
figtitle: Enzymatic pathway for inositol lipid synthesis in BT
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC9246714
filename: 41564_2022_1152_Fig1_HTML.jpg
figlink: /pmc/articles/PMC9246714/figure/Fig1/
number: F1
caption: a, The de novo sphingolipid synthesis metabolic pathway in relation to inositol
  lipid synthesis, with BT enzymes investigated in this study (bolded black) and representative
  lipid structures, with chain length and branching reflecting predominant structures
  determined by fatty acid methyl ester analysis (see , Supplementary Figs.  and ,
  and Table ). Branching patterns of dihydroceramide-based lipids are predicted and
  not confirmed. Sphingolipid structures are named in purple and shown on a light
  blue background; glycerophospholipid structures are named in green on a grey background.
  Inositol lipids are outlined with a dashed line box. SPT, serine palmitoyltransferase.
  b, The genomic region of BT inositol and inositol lipid synthesis, with chromosomal
  numbering (shown in reverse orientation). Gene colour (blue or red) indicates membership
  in a predicted operon by BioCyc. Annotations are of the enzyme functions elucidated
  in this study (due to the lack of a lipid phenotype in its knockout strain, BT_1524
  remains hypothetical). Substrates and products for the sphingolipid portion of a
  are not listed, as these reactions (beyond SPT) have not been characterized in the
  Bacteroidetes.
papertitle: Characterization of inositol lipid metabolism in gut-associated Bacteroidetes.
reftext: Stacey L. Heaver, et al. Nat Microbiol. 2022;7(7):986-1000.
year: '2022'
doi: 10.1038/s41564-022-01152-6
journal_title: Nature Microbiology
journal_nlm_ta: Nat Microbiol
publisher_name: Nature Publishing Group UK
keywords: X-ray crystallography | Bacterial genes | Sphingolipids | Phospholipids
automl_pathway: 0.9400389
figid_alias: PMC9246714__F1
figtype: Figure
redirect_from: /figures/PMC9246714__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9246714__41564_2022_1152_Fig1_HTML.html
  '@type': Dataset
  description: a, The de novo sphingolipid synthesis metabolic pathway in relation
    to inositol lipid synthesis, with BT enzymes investigated in this study (bolded
    black) and representative lipid structures, with chain length and branching reflecting
    predominant structures determined by fatty acid methyl ester analysis (see , Supplementary
    Figs.  and , and Table ). Branching patterns of dihydroceramide-based lipids are
    predicted and not confirmed. Sphingolipid structures are named in purple and shown
    on a light blue background; glycerophospholipid structures are named in green
    on a grey background. Inositol lipids are outlined with a dashed line box. SPT,
    serine palmitoyltransferase. b, The genomic region of BT inositol and inositol
    lipid synthesis, with chromosomal numbering (shown in reverse orientation). Gene
    colour (blue or red) indicates membership in a predicted operon by BioCyc. Annotations
    are of the enzyme functions elucidated in this study (due to the lack of a lipid
    phenotype in its knockout strain, BT_1524 remains hypothetical). Substrates and
    products for the sphingolipid portion of a are not listed, as these reactions
    (beyond SPT) have not been characterized in the Bacteroidetes.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Spt
  - lace
  - en
  - Spt-I
  - srp
  - Dhc16F
  - Dhc36C
  - btv
  - Dhc62B
  - Dhc64C
  - Dhc93AB
  - Dhc98D
  - kl-5
  - Coa
  - pi
  - cmp
---
