---
figid: PMC9498451__foods-11-02876-g002
figtitle: 'Probiotic Oxalate-Degrading Bacteria: New Insight of Environmental Variables
  and Expression of the oxc and frc Genes on Oxalate Degradation Activity'
organisms:
- Lacticaseibacillus casei
- Lactobacillus acidophilus
- Bifidobacterium longum
- Levilactobacillus brevis
- Bifidobacterium adolescentis
- Bifidobacterium breve
- Bifidobacterium longum subsp. infantis
- Lactobacillus gasseri
- Bifidobacterium animalis subsp. lactis
- Lactobacillus sp.
- Bifidobacterium animalis
- Oxalobacter formigenes
- Oxalobacter formigenes DSM 4420
- gut metagenome
- human metagenome
- metagenomes
- enrichment cultures
- Saccharomyces cerevisiae
- Theobroma cacao
- Aspergillus niger
- Mus musculus
- Rattus norvegicus
- Oryctolagus cuniculus
- Ovis aries
- Homo sapiens
organisms_ner:
- NA
pmcid: PMC9498451
filename: foods-11-02876-g002.jpg
figlink: /pmc/articles/PMC9498451/figure/foods-11-02876-f002/
number: F2
caption: 'Oxalate synthesis pathway from glyoxal. In this mechanism, enzymes: alanine:
  glyoxylate and aminotransferase are active. (1) glyoxylase I, (2) glyoxylase II,
  (3) aldehyde dehydrogenase, (4) glycolate oxidase, (5) glyoxylate reductase, (6)
  lactate dehydrogenase, (7) D-amino acid oxidase and (8) alanine: glyoxylate aminotransferase.'
papertitle: 'Probiotic Oxalate-Degrading Bacteria: New Insight of Environmental Variables
  and Expression of the oxc and frc Genes on Oxalate Degradation Activity.'
reftext: Dina Karamad, et al. Foods. 2022 Sep;11(18):2876.
year: '2022'
doi: 10.3390/foods11182876
journal_title: Foods
journal_nlm_ta: Foods
publisher_name: MDPI
keywords: probiotic bacteria | oxalate-degrading | variables | in vivo | in vitro
  | oxaluria
automl_pathway: 0.9281839
figid_alias: PMC9498451__F2
figtype: Figure
redirect_from: /figures/PMC9498451__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9498451__foods-11-02876-g002.html
  '@type': Dataset
  description: 'Oxalate synthesis pathway from glyoxal. In this mechanism, enzymes:
    alanine: glyoxylate and aminotransferase are active. (1) glyoxylase I, (2) glyoxylase
    II, (3) aldehyde dehydrogenase, (4) glycolate oxidase, (5) glyoxylate reductase,
    (6) lactate dehydrogenase, (7) D-amino acid oxidase and (8) alanine: glyoxylate
    aminotransferase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GSH
  - Glyoxal
  - NADPH
  - NAD
  - Glyoxylate
  - Glycine
  - Alanine
  - Oxalate
---
