---
figid: PMC9120235__gr4
figtitle: 'OncoboxPD: human 51 672 molecular pathways database with tools for activity
  calculating and visualization'
organisms:
- NA
organisms_ner:
- Homo sapiens
pmcid: PMC9120235
filename: gr4.jpg
figlink: /pmc/articles/PMC9120235/figure/f0020/
number: F4
caption: 'TRAF molecular pathway visualization using OncoboxPD software. Nodes correspond
  to individual pathway components or to their complexes. Color of every node reflects
  logarithm of mean CNR for all node components, according to a scale given with green
  upregulated, and red downregulated nodes. Grey nodes have no gene components, and
  no CNR values. Nodes that are unaffected (with CNR ∼ 1) are shown white. Red and
  green arrows stand for inhibitory and activating interactions, respectively. Ovals
  denote gene products, octagons - metabolites, hexagons – complexes, and other nodes
  (text labels without recognized molecular components) are shown as rectangles. A
  bold black border indicates outlier values that are outside of scale limits. In
  this example, RNA sequencing gene expression profile of thyroid papillary cancer
  sample TC15  was compared against six healthy thyroid samples from ANTE collection
  . A) Static pathway graph. B) Projection of dynamic interactive pathway graph. The
  figure can be found following the path: result file in Folders(“example” with green
  icon)->sample TC15 in Sample table-> Pathway activation level tab->TRAF Pathway->
  static and dynamic graphs. (For interpretation of the references to color in this
  figure legend, the reader is referred to the web version of this article.)'
papertitle: 'OncoboxPD: human 51 672 molecular pathways database with tools for activity
  calculating and visualization.'
reftext: Marianna A. Zolotovskaia, et al. Comput Struct Biotechnol J. 2022;20:2280-2291.
year: '2022'
doi: 10.1016/j.csbj.2022.05.006
journal_title: Computational and Structural Biotechnology Journal
journal_nlm_ta: Comput Struct Biotechnol J
publisher_name: Research Network of Computational and Structural Biotechnology
keywords: Molecular pathway database | Interactomics | Protein–protein interactions
  | Metabolomics | Pathway activation level | Pathway visualization | ARR, activation/repressor
  role coefficient | BEL, Biological Expression Language | KEGG, Kyoto Encyclopedia
  of Genes and Genomes | PAL, pathway activation level | SPIKE, Signaling Pathways
  Integrated Knowledge Engine | SPOKE, Scalable Precision Medicine Open Knowledge
  Engine | VM, virtual machine
automl_pathway: 0.8219613
figid_alias: PMC9120235__F4
figtype: Figure
redirect_from: /figures/PMC9120235__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9120235__gr4.html
  '@type': Dataset
  description: 'TRAF molecular pathway visualization using OncoboxPD software. Nodes
    correspond to individual pathway components or to their complexes. Color of every
    node reflects logarithm of mean CNR for all node components, according to a scale
    given with green upregulated, and red downregulated nodes. Grey nodes have no
    gene components, and no CNR values. Nodes that are unaffected (with CNR ∼ 1) are
    shown white. Red and green arrows stand for inhibitory and activating interactions,
    respectively. Ovals denote gene products, octagons - metabolites, hexagons – complexes,
    and other nodes (text labels without recognized molecular components) are shown
    as rectangles. A bold black border indicates outlier values that are outside of
    scale limits. In this example, RNA sequencing gene expression profile of thyroid
    papillary cancer sample TC15  was compared against six healthy thyroid samples
    from ANTE collection . A) Static pathway graph. B) Projection of dynamic interactive
    pathway graph. The figure can be found following the path: result file in Folders(“example”
    with green icon)->sample TC15 in Sample table-> Pathway activation level tab->TRAF
    Pathway-> static and dynamic graphs. (For interpretation of the references to
    color in this figure legend, the reader is referred to the web version of this
    article.)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TNFRSF1B
  - TNFRSF1A
  - AGFG1
  - RIPK1
  - DCAF1
  - RPAIN
  - TNF
  - BIRC2
  - TNFSF13B
  - FADD
  - TNFRSF13B
  - CD40
  - MADD
  - ETFA
  - ETFB
  - ETFDH
  - TNFRSF11A
  - NGFR
  - TNFSF11
  - CD40LG
  - SRC
  - FGR
  - FYN
  - YES1
  - PIK3CA
  - PIK3CB
  - PIK3CD
  - PIK3CG
  - PIK3R3
  - PIK3R4
  - PIK3R5
  - PIK3R6
  - PIK3R1
  - PIK3R2
  - IL1B
  - CHUK
  - IKBKB
  - IKBKG
  - IKBKE
  - TBK1
  - TRADD
  - MAP3K1
  - MAP3K2
  - MAP3K3
  - MAP3K4
  - MAP3K5
  - MAP3K6
  - MAP3K7
  - MAP3K8
  - MAP3K9
  - MAP3K10
  - MAP3K11
  - MAP3K12
  - MAP3K13
  - MAP3K14
  - MAP3K15
  - MAP3K19
  - MAP3K20
  - MAP3K21
  - ARAF
  - BRAF
  - RAF1
  - TAOK1
  - TAOK2
  - TAOK3
  - NFKB1
  - RELA
  - RELB
  - REL
  - NFKB2
  - TAB1
  - TAB2
  - NR2C2
  - TNFSF13
  - ANP32B
  - TOLLIP
  - TLR1
  - TLR2
  - TLR3
  - TLR4
  - TLR5
  - TLR6
  - TLR7
  - TLR8
  - TLR9
  - TLR10
  - IRAK1
  - MYD88
  - TNFRSF17
  - MAP2K3
  - MAP2K6
  - MAPK8
  - MAPK9
  - MAPK10
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - CRK
  - AIMP2
  - GRAP2
  - AHSA1
  - POLDIP2
  - TIRAP
  - MAP2K4
  - MAP2K7
  - FOS
  - JUN
---
