---
figid: PMC9009446__fcimb-12-736397-g001
figtitle: The Microbiome in the Pathogenesis of COVID-19
organisms:
- Escherichia coli
- Pseudomonas aeruginosa
- Vibrio cholerae
- Enterococcus faecalis
- Lacticaseibacillus casei
- Acinetobacter baumannii
- Staphylococcus epidermidis
- Faecalibacterium prausnitzii
- Akkermansia muciniphila
- Enterococcus faecium
- Lacticaseibacillus rhamnosus
- Clostridium butyricum
- Bacteroides thetaiotaomicron
- Limosilactobacillus fermentum
- Burkholderia cepacia
- Eubacterium rectale
- Enterobacter sp.
- Streptococcus uberis
- Clostridium leptum
- Streptococcus sp.
- Prevotella melaninogenica
- Veillonella parvula
- Parvimonas micra
- Collinsella aerofaciens
- Metamycoplasma hominis
- Bacteroides ovatus
- Morganella morganii
- Thomasclavelia ramosa
- Enterococcus sp.
- Haemophilus parainfluenzae
- Hungatella hathewayi
- Lachnospiraceae bacterium
- Parabacteroides merdae
- Streptococcus infantis
- Bacteroides stercoris
- Lancefieldella parvula
- Phocaeicola dorei
- Phocaeicola massiliensis
- Neisseria subflava
- Fusobacterium periodonticum
- Metamycoplasma orale
- Escherichia coli O26
- Neisseria mucosa
- Eikenella corrodens
- Peptostreptococcus sp.
- Alistipes onderdonkii
- Collinsella tanakaei
- Aggregatibacter aphrophilus
- Capnocytophaga sputigena
- Corynebacterium accolens
- Coprobacillus sp.
- Gemella morbillorum
- Candida albicans
- Brassica oleracea
- Aspergillus niger
- Aspergillus flavus
- Aspergillus sp.
- Penicillium citrinum
- Candida auris
- Candida sp. uncertain placement
- Saccharomyces sp.
- Malassezia sp.
- Nakaseomyces sp.
- Severe acute respiratory syndrome coronavirus 2
- Human gammaherpesvirus 4
- Coronaviridae
- Severe acute respiratory syndrome-related coronavirus
- Middle East respiratory syndrome-related coronavirus
- H1N1 subtype
- Betacoronavirus
- Human orthopneumovirus
- Rhinovirus B
- Riboviria
- gut metagenome
- metagenome
- human gut metagenome
- human metagenome
- human oral metagenome
- oral viral metagenome
- Lactobacillus phage phiadh
- Streptococcus phage EJ-1
- Staphylococcus phage ROSA
- Mus musculus
- Rattus norvegicus
- Caenorhabditis elegans
- Mustela lutreola
- Manis javanica
- Viana
- Homo sapiens
- Anas platyrhynchos
pmcid: PMC9009446
filename: fcimb-12-736397-g001.jpg
figlink: /pmc/articles/PMC9009446/figure/f1/
number: F1
caption: The figure shows how the pathogenesis of SARS-CoV-2 affects the microbiome.
  The diagram shows that in people already predisposed toward the disease due to age,
  comorbidities, and environmental factors like pollution, diet, and microbiome, the
  pathogenesis is facilitated due to prior downregulation of the ACE2 receptor, which
  is the major receptor for SARS-CoV-2 entry into the lung epithelia. On viral entry,
  ACE2 is further downregulated and initiates a cascade of events like dysregulation
  of RAS, slows conversion of AngII to Ang 1-7, upregulation of the ACE-Ang-II-AT1R
  axis (indicated by *** in the figure) and downregulation of the ACE-2-Ang-(1-7)-Mas
  axis (indicated by * in the figure), leading to clinical symptoms like an increase
  in blood pressure, imbalance of fluids and electrolytes, oxidative stress, vascular
  permeability alterations, and eventually increase in inflammation leading to the
  cytokine storm accentuated by the activation of NF-κβ, MAPK pathway, and finally
  leading to lung damage. Downregulation of ACE2 also leads to the absence of the
  tryptophan transporter B°AT1 leading to the inhibition of tryptophan uptake, which
  consequently leads to the reduction of the mTOR pathway, impaired expression of
  antimicrobial peptides (AMPs), and increase in autophagy leading to diarrhea and
  dysbiosis of the microbiome. Microbiome dysbiosis in COVID-19 has been found to
  be characterized by the upsurge of certain taxa and decrease of others, as indicated
  in .
papertitle: Role of the Microbiome in the Pathogenesis of COVID-19.
reftext: Rituparna De, et al. Front Cell Infect Microbiol. 2022;12:736397.
year: '2022'
doi: 10.3389/fcimb.2022.736397
journal_title: Frontiers in Cellular and Infection Microbiology
journal_nlm_ta: Front Cell Infect Microbiol
publisher_name: Frontiers Media S.A.
keywords: COVID-19 | microbiome | SARS-CoV2 | inflammation | ACE2 | serotonin | kynurenine
automl_pathway: 0.8073313
figid_alias: PMC9009446__F1
figtype: Figure
organisms_ner:
- Mus musculus
- Homo sapiens
- Rattus norvegicus
redirect_from: /figures/PMC9009446__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9009446__fcimb-12-736397-g001.html
  '@type': Dataset
  description: The figure shows how the pathogenesis of SARS-CoV-2 affects the microbiome.
    The diagram shows that in people already predisposed toward the disease due to
    age, comorbidities, and environmental factors like pollution, diet, and microbiome,
    the pathogenesis is facilitated due to prior downregulation of the ACE2 receptor,
    which is the major receptor for SARS-CoV-2 entry into the lung epithelia. On viral
    entry, ACE2 is further downregulated and initiates a cascade of events like dysregulation
    of RAS, slows conversion of AngII to Ang 1-7, upregulation of the ACE-Ang-II-AT1R
    axis (indicated by *** in the figure) and downregulation of the ACE-2-Ang-(1-7)-Mas
    axis (indicated by * in the figure), leading to clinical symptoms like an increase
    in blood pressure, imbalance of fluids and electrolytes, oxidative stress, vascular
    permeability alterations, and eventually increase in inflammation leading to the
    cytokine storm accentuated by the activation of NF-κβ, MAPK pathway, and finally
    leading to lung damage. Downregulation of ACE2 also leads to the absence of the
    tryptophan transporter B°AT1 leading to the inhibition of tryptophan uptake, which
    consequently leads to the reduction of the mTOR pathway, impaired expression of
    antimicrobial peptides (AMPs), and increase in autophagy leading to diarrhea and
    dysbiosis of the microbiome. Microbiome dysbiosis in COVID-19 has been found to
    be characterized by the upsurge of certain taxa and decrease of others, as indicated
    in .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Ace2
  - ras
  - Hras
  - Kras
  - Rem1
  - Agt
  - Nfkb1
  - Mtor
  - ACE2
  - KRAS
  - HRAS
  - NRAS
  - ANGPT2
  - VPS51
  - MAPK1
  - MAPK3
  - MAPK11
  - MAPK12
  - MAPK13
  - MAPK14
  - MAPK8
  - MAPK9
  - MAPK10
  - NFKB1
  - LINC00689
  - ATXN1L
  - MTOR
  - ADSL
  - Th
---
