---
figid: PMC9321552__PCE-45-1891-g001
figtitle: Transcriptomic changes during the establishment of long‐term methyl jasmonate‐induced
  resistance in Norway spruce
organisms:
- Picea abies
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9321552
filename: PCE-45-1891-g001.jpg
figlink: /pmc/articles/PMC9321552/figure/pce14320-fig-0006/
number: F6
caption: 'Genes involved in the biosynthesis of salicylic acid (SA) are transiently
  upregulated in Norway spruce bark in response to methyl jasmonate (MeJA) treatment.
  Expression profiles of genes displaying a significantly (adjusted p < 0.001) altered
  expression pattern across time as a result of MeJA treatment and which were annotated
  as encoding for enzymes and transporters involved in the biosynthesis of SA (yellow
  boxes). In the plots, faint lines indicate the mean expression profiles, with 95%
  confidence intervals, of individual transcripts and the thicker lines depict the
  mean per protein category profile, for water (blue) and MeJA (red) treatments. Read
  counts were normalised using the variance stabilising transformation (vst) in DEseq.
  2. The pathway is based on what is known about the biosynthesis of SA in angiosperms.
  Enzymes in grey are involved in the biosynthesis pathway but no differentially expressed
  genes were annotated to encode for them. Due to space limitations not all enzymes
  have associated expression profiles, see Data Set  for the full list of differentially
  expressed biosynthesis genes. Compound abbreviations: DQ, 3‐dehydroquinate; DHS,
  3‐dehydroshikimate; Shikimate‐3‐P, shikimate 3‐phosphate; EPSP; 5‐enolpyruvoyl‐shikimate
  3‐phosphate; t‐CA, trans‐cinnamic acid. Enzyme/transporter abbreviations: ADT/PDT,
  arogenate dehydratase/prephenate dehydratase (annotated as being encoded for by
  the same genes); AO, aldehyde oxidase; BA2H, benzoic acid 2‐hydroxylase; CM1, chorismate
  mutase 1; CM2, chorismate mutase 2; CS, chorismate synthase; DHQ‐SDH, dehydroquinate‐shikimate
  dehydrogenase; EDS5, ENHANCED DISEASE SUSCEPTIBILITY 5; EPSPS, 5‐enolpyruvylshikimate‐3‐phosphate
  synthase; ICS, isochorismate synthase; IGS, isochorismoyl‐glutamate synthase; IPGL,
  IC‐9‐Glu pyruvoyl‐glutamate lyase; PAL, phenylalanine ammonia‐lyase; pCAT, plastidial
  cationic amino‐acid transporter; PPA, prephenate aminotransferase; PPY, phenylpyruvate
  aminotransferase; SK, shikimate kinase; TAT, tyrosine aminotransferase [Color figure
  can be viewed at wileyonlinelibrary.com]'
papertitle: Transcriptomic changes during the establishment of long‐term methyl jasmonate‐induced
  resistance in Norway spruce.
reftext: Samuel W. Wilkinson, et al. Plant Cell Environ. 2022 Jun;45(6):1891-1913.
year: '2022'
doi: 10.1111/pce.14320
journal_title: Plant, Cell & Environment
journal_nlm_ta: Plant Cell Environ
publisher_name: John Wiley and Sons Inc.
keywords: epigenetics | induced resistance | methyl jasmonate | phytohormones | Picea
  abies | transcriptome
automl_pathway: 0.7472411
figid_alias: PMC9321552__F6
figtype: Figure
redirect_from: /figures/PMC9321552__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9321552__PCE-45-1891-g001.html
  '@type': Dataset
  description: 'Genes involved in the biosynthesis of salicylic acid (SA) are transiently
    upregulated in Norway spruce bark in response to methyl jasmonate (MeJA) treatment.
    Expression profiles of genes displaying a significantly (adjusted p < 0.001) altered
    expression pattern across time as a result of MeJA treatment and which were annotated
    as encoding for enzymes and transporters involved in the biosynthesis of SA (yellow
    boxes). In the plots, faint lines indicate the mean expression profiles, with
    95% confidence intervals, of individual transcripts and the thicker lines depict
    the mean per protein category profile, for water (blue) and MeJA (red) treatments.
    Read counts were normalised using the variance stabilising transformation (vst)
    in DEseq. 2. The pathway is based on what is known about the biosynthesis of SA
    in angiosperms. Enzymes in grey are involved in the biosynthesis pathway but no
    differentially expressed genes were annotated to encode for them. Due to space
    limitations not all enzymes have associated expression profiles, see Data Set 
    for the full list of differentially expressed biosynthesis genes. Compound abbreviations:
    DQ, 3‐dehydroquinate; DHS, 3‐dehydroshikimate; Shikimate‐3‐P, shikimate 3‐phosphate;
    EPSP; 5‐enolpyruvoyl‐shikimate 3‐phosphate; t‐CA, trans‐cinnamic acid. Enzyme/transporter
    abbreviations: ADT/PDT, arogenate dehydratase/prephenate dehydratase (annotated
    as being encoded for by the same genes); AO, aldehyde oxidase; BA2H, benzoic acid
    2‐hydroxylase; CM1, chorismate mutase 1; CM2, chorismate mutase 2; CS, chorismate
    synthase; DHQ‐SDH, dehydroquinate‐shikimate dehydrogenase; EDS5, ENHANCED DISEASE
    SUSCEPTIBILITY 5; EPSPS, 5‐enolpyruvylshikimate‐3‐phosphate synthase; ICS, isochorismate
    synthase; IGS, isochorismoyl‐glutamate synthase; IPGL, IC‐9‐Glu pyruvoyl‐glutamate
    lyase; PAL, phenylalanine ammonia‐lyase; pCAT, plastidial cationic amino‐acid
    transporter; PPA, prephenate aminotransferase; PPY, phenylpyruvate aminotransferase;
    SK, shikimate kinase; TAT, tyrosine aminotransferase [Color figure can be viewed
    at wileyonlinelibrary.com]'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - vst
  - ics
  - Dhps
  - SK
  - cs
  - PpY-55A
  - pal
  - Pal2
  - Ppa
  - CUBN
  - DHPS
  - PIEZO1
  - CS
  - PPY
  - PAM
  - LRIT1
  - SHCBP1
---
