---
figid: PMC9273332__nihms-1820192-f0001
figtitle: Single-cell deletion analyses show control of pro-T cell developmental speed
  and pathway by Tcf7, Spi1, Gata3, Bcl11a, Erg, and Bcl11b
organisms:
- Ficus racemosa
- Mus musculus
- bacterium NOTE
- Matthiola incana
- Homo sapiens
- NA
organisms_ner:
- Homo sapiens
- Mus musculus
- Drosophila melanogaster
- Danio rerio
pmcid: PMC9273332
filename: nihms-1820192-f0001.jpg
figlink: /pmc/articles/PMC9273332/figure/F1/
number: F1
caption: 'Developmental framework: Pool-based, batch-controlled transcription factor
  perturbation scRNA-seq to define impacts of timed deletions of developmentally regulated
  transcription factors. (A) Identification of stages in T cell development in the
  thymus. All stages indicated are CD4− CD8− TCR− (DN). TSP, thymus-seeding precursor:
  earliest Flt3+ subset of Early T-cell Precursor (ETP) population; ETP: c-Kithi CD44+
  CD25− DN. DN2a: c-Kithi CD44+ CD25+. DN2b: c-Kitint CD44+ CD25+. DN3: c-Kitlo CD44−
  CD25+, the stage when TCR rearrangement (rearr.) produces complete TCR coding genes.
  Timing of Bcl11b activation and stages studied here are indicated. (B) Diagram of
  gene expression patterns of selected transcription factors in relation to measured
  alternative lineage potentials of the cells at each stage (). (C) Representative
  surface expression phenotypes analyzed by flow cytometry. Acute transcription factor
  deletions were induced in precursors by sgRNA transduction, then cells were cultured
  for 6 days with OP9-DLL1. Figure shows developmental staging based on surface expression
  of CD44 and CD25. Stages identified in (A) correspond to the indicated patterns
  of CD44 and CD25. ETP and TSP cells, but potentially also alternative lineage cells,
  are included in the “DN1” quadrant. (D) Pool-based dual gRNA cloning strategy used
  in pool-perturbation experiments in this study. Left: diagram of structure of the
  paired gRNA oligomers as initially synthesized by array-based oligo synthesis. Middle,
  each distinct oligomer was joined by Gibson assembly with the invariant Cap2 oligo
  and mU6 promoter segments in the orientation shown, to generate the paired dual
  gRNA insert pool. Right, incorporation of the paired dual gRNA inserts into the
  retroviral vector backbone containing the hU6 promoter and the Cap1 oligo sequence.
  Note that the assembly of these segments into the indicated site ends up linking
  the mU6 promoter with g1 and the Cap1 tag, downstream of the hU6 promoter driving
  g2 and the Cap2 oligo sequence. Retroviral backbones were packaged into the final
  retroviral vector library. (E) Internal construct- and batch- controlled experimental
  setup for single-cell pool-perturbation RNA-seq. Two independently packaged stocks
  of the pool-perturbation viral library were each infected into separate LSK cultures
  at MOI=1.0 and MOI=0.5, in parallel (plus an extra separate culture for one of the
  MOI=0.5 samples). These separate transductions of the whole pool were then cultured
  in parallel as indicated, for 5 days on OP9-DLL1 to initiate T cell development.
  The five cultures were then harvested, sorting Infected (inf+) Lin− (within the
  T-cell pathway) Kit+ (uncommitted) cells. Each culture was indexed with different
  hashtags before cells were pooled for scRNA-seq analysis as described in the text.
  The cells were processed by 10X Chromium to associate unique barcodes with each
  cell, and mRNA, gRNA, and hashtag libraries were made. The unique barcodes enabled
  mRNA sequences from one cell to be correlated with the gRNA and hashtag sequences
  from the same cell.'
papertitle: Single-cell deletion analyses show control of pro-T cell developmental
  speed and pathway by Tcf7, Spi1, Gata3, Bcl11a, Erg, and Bcl11b.
reftext: Wen Zhou, et al. Sci Immunol. ;7(71):eabm1920-eabm1920.
year: '2023'
doi: 10.1126/sciimmunol.abm1920
journal_title: Science immunology
journal_nlm_ta: Sci Immunol
publisher_name: ''
keywords: ''
automl_pathway: 0.786546
figid_alias: PMC9273332__F1
figtype: Figure
redirect_from: /figures/PMC9273332__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9273332__nihms-1820192-f0001.html
  '@type': Dataset
  description: 'Developmental framework: Pool-based, batch-controlled transcription
    factor perturbation scRNA-seq to define impacts of timed deletions of developmentally
    regulated transcription factors. (A) Identification of stages in T cell development
    in the thymus. All stages indicated are CD4− CD8− TCR− (DN). TSP, thymus-seeding
    precursor: earliest Flt3+ subset of Early T-cell Precursor (ETP) population; ETP:
    c-Kithi CD44+ CD25− DN. DN2a: c-Kithi CD44+ CD25+. DN2b: c-Kitint CD44+ CD25+.
    DN3: c-Kitlo CD44− CD25+, the stage when TCR rearrangement (rearr.) produces complete
    TCR coding genes. Timing of Bcl11b activation and stages studied here are indicated.
    (B) Diagram of gene expression patterns of selected transcription factors in relation
    to measured alternative lineage potentials of the cells at each stage (). (C)
    Representative surface expression phenotypes analyzed by flow cytometry. Acute
    transcription factor deletions were induced in precursors by sgRNA transduction,
    then cells were cultured for 6 days with OP9-DLL1. Figure shows developmental
    staging based on surface expression of CD44 and CD25. Stages identified in (A)
    correspond to the indicated patterns of CD44 and CD25. ETP and TSP cells, but
    potentially also alternative lineage cells, are included in the “DN1” quadrant.
    (D) Pool-based dual gRNA cloning strategy used in pool-perturbation experiments
    in this study. Left: diagram of structure of the paired gRNA oligomers as initially
    synthesized by array-based oligo synthesis. Middle, each distinct oligomer was
    joined by Gibson assembly with the invariant Cap2 oligo and mU6 promoter segments
    in the orientation shown, to generate the paired dual gRNA insert pool. Right,
    incorporation of the paired dual gRNA inserts into the retroviral vector backbone
    containing the hU6 promoter and the Cap1 oligo sequence. Note that the assembly
    of these segments into the indicated site ends up linking the mU6 promoter with
    g1 and the Cap1 tag, downstream of the hU6 promoter driving g2 and the Cap2 oligo
    sequence. Retroviral backbones were packaged into the final retroviral vector
    library. (E) Internal construct- and batch- controlled experimental setup for
    single-cell pool-perturbation RNA-seq. Two independently packaged stocks of the
    pool-perturbation viral library were each infected into separate LSK cultures
    at MOI=1.0 and MOI=0.5, in parallel (plus an extra separate culture for one of
    the MOI=0.5 samples). These separate transductions of the whole pool were then
    cultured in parallel as indicated, for 5 days on OP9-DLL1 to initiate T cell development.
    The five cultures were then harvested, sorting Infected (inf+) Lin− (within the
    T-cell pathway) Kit+ (uncommitted) cells. Each culture was indexed with different
    hashtags before cells were pooled for scRNA-seq analysis as described in the text.
    The cells were processed by 10X Chromium to associate unique barcodes with each
    cell, and mRNA, gRNA, and hashtag libraries were made. The unique barcodes enabled
    mRNA sequences from one cell to be correlated with the gRNA and hashtag sequences
    from the same cell.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LCK
  - MATK
  - SPI1
  - MEIS1
  - GPT
  - THBS1
  - HOXA9
  - BCL11A
  - CAP2
  - SERPINB8
  - TMPRSS4
  - WARS1
  - BBS1
  - ERG
  - TCF7
  - GATA3
  - BCL11B
  - CAP1
  - PRSS8
  - TRAF3
  - CD44
  - IGHD6-13
  - IL2RA
  - ISG20
  - KIT
  - CBLIF
  - TRBV20OR9-2
  - TRA
  - TRB
  - TRD
  - TRG
  - PER1
  - Lck
  - Spi1
  - Serpina3d-ps
  - Meis1
  - Gpt
  - Ctr9
  - Hoxa9
  - Bcl11a
  - Cap2
  - Serpinb8
  - Tmprss4
  - Gabrg2
  - Bbs1
  - Erg
  - Tcf7
  - Gata3
  - Bcl11b
  - Cap1
  - Traf3
  - Prss8
  - Cd44
  - Il2ra
  - Kit
  - Cblif
  - Trav6-3
  - Hspg2
  - Per1
  - lds
  - hth
  - l(1)16Fa
  - mei-W68
  - alt
  - CG1640
  - Ca-alpha1T
  - alphaTub84B
  - Tsp
  - LanB2
  - anon-70Db
  - cpb
  - Capa
  - anon-70Dc
  - sei
  - 'On'
  - cpa
  - Cont
  - blo
  - lin
  - drl
  - if
  - Tcr
  - per
  - spi1b
  - meis1b
  - hoxa9b
  - bcl11aa
  - cap2
  - bbs1
  - erg
  - kcnh6a
  - tcf7
  - gata3
  - grna
  - grnas
  - cap1
  - cd4-1
  - kita
  - Lin
  - infection
---
