---
figid: PMC9038741__41396_2021_1177_Fig5_HTML
figtitle: Metabolic reconstruction of Angelarchaeales-1 genome
organisms:
- Archaeoglobus fulgidus
- Nitrosomonas europaea
- Nitrosopumilus maritimus
- Thermoproteus tenax
- Nitrososphaera viennensis
- Candidatus Nitrosocaldus yellowstonensis
- metagenome
- metagenomes
- environmental samples
- uncultured bacterium
- Tharsalea heteronea
- Agriopis marginaria
- Coronaviridae
- Ficus racemosa
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9038741
filename: 41396_2021_1177_Fig5_HTML.jpg
figlink: /pmc/articles/PMC9038741/figure/Fig5/
number: F5
caption: For full reaction, gene list, and list of compound abbreviations see Supplementary
  Tables –. Green boxes indicate a reaction (and its reference number in Supplementary
  Table ) that could be linked to a gene with the predicted metabolic function. Black
  arrows with solid lines indicate a reaction that could be identified, associated
  smaller arrows with colored dots indicate consumed, and generated reaction substrates
  and products are indicated in the key at the bottom left. Black arrows with dotted
  lines indicate flow of metabolites to other pathways or reactions. Gray arrows with
  dotted lines and gray boxes indicate reactions that were searched for and could
  not be identified. Amino acids are in red text to highlight their locations throughout
  the figure. Metabolites in blue text indicate hubs for carbon derived from amino
  acid catabolism. For ease of viewing the reactions of glycolysis, the pentose phosphate
  pathway, and the TCA cycle have been highlighted with beige, red, and orange backgrounds.
  The upper panel is a blow up of the electron transport reactions showing predicted
  organizations of subunits in each complex. Reference numbers for each subunit can
  be found in the larger figure panel, and colors of subunits are the same as those
  used in that figure panel. Transparent HAO-QRED indicates a putative/proposed functionality.
  Black arrows with dotted lines indicate putative reactions. Protein subunits with
  solid color but dotted borders indicate a protein was found but functionality is
  unclear.
papertitle: Soils and sediments host Thermoplasmata archaea encoding novel copper
  membrane monooxygenases (CuMMOs).
reftext: Spencer Diamond, et al. ISME J. 2022 May;16(5):1348-1362.
year: '2022'
doi: 10.1038/s41396-021-01177-5
journal_title: The ISME Journal
journal_nlm_ta: ISME J
publisher_name: Nature Publishing Group UK
keywords: Environmental microbiology | Microbial ecology | Soil microbiology | Next-generation
  sequencing
automl_pathway: 0.9138264
figid_alias: PMC9038741__F5
figtype: Figure
redirect_from: /figures/PMC9038741__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9038741__41396_2021_1177_Fig5_HTML.html
  '@type': Dataset
  description: For full reaction, gene list, and list of compound abbreviations see
    Supplementary Tables –. Green boxes indicate a reaction (and its reference number
    in Supplementary Table ) that could be linked to a gene with the predicted metabolic
    function. Black arrows with solid lines indicate a reaction that could be identified,
    associated smaller arrows with colored dots indicate consumed, and generated reaction
    substrates and products are indicated in the key at the bottom left. Black arrows
    with dotted lines indicate flow of metabolites to other pathways or reactions.
    Gray arrows with dotted lines and gray boxes indicate reactions that were searched
    for and could not be identified. Amino acids are in red text to highlight their
    locations throughout the figure. Metabolites in blue text indicate hubs for carbon
    derived from amino acid catabolism. For ease of viewing the reactions of glycolysis,
    the pentose phosphate pathway, and the TCA cycle have been highlighted with beige,
    red, and orange backgrounds. The upper panel is a blow up of the electron transport
    reactions showing predicted organizations of subunits in each complex. Reference
    numbers for each subunit can be found in the larger figure panel, and colors of
    subunits are the same as those used in that figure panel. Transparent HAO-QRED
    indicates a putative/proposed functionality. Black arrows with dotted lines indicate
    putative reactions. Protein subunits with solid color but dotted borders indicate
    a protein was found but functionality is unclear.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - fd
  - anon-16Fb
  - ATPsynbeta
  - Atpalpha
  - adp
  - Amph
  - aa
  - EcR
  - mesh
  - db
  - Suc
  - Fum1
  - sqh
  - zip
  - tor
  - Tor
  - Egfr
  - Had1
  - gm
  - arg
  - aos
  - Res
  - cup
  - t-cup
  - mal
  - bs
  - Mrtf
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - Hao
  - G6P
  - fbp
  - Pep
  - bgcn
  - sti
  - ke
  - fal
  - Fancl
  - side
  - Sidpn
  - act
  - Act5C
  - .na.character
  - Act42A
  - Actbeta
  - Cys
  - mmy
  - Csp
  - pan
  - Panx
  - hh
  - gl
  - sno
  - hp
  - nej
  - Mthfs
  - E(spl)mgamma-HLH
  - hpo
  - hop
  - wo
  - ca
  - na
  - He
  - ARID4B
  - ATP8A2
  - WDTC1
  - APRT
  - MFAP1
  - FOXG1
  - FOXH1
  - FOXM1
  - FOXQ1
  - FOXS1
  - FOXA1
  - FOXA2
  - FOXA3
  - FOXB1
  - FOXB2
  - FOXC1
  - FOXC2
  - FOXD1
  - FOXD2
  - FOXD3
  - FOXD4
  - FOXE1
  - FOXE3
  - FOXF1
  - FOXF2
  - FOXI1
  - FOXI2
  - FOXI3
  - FOXJ1
  - FOXJ2
  - FOXJ3
  - FOXK1
  - FOXK2
  - FOXL1
  - FOXL2
  - FOXN1
  - FOXN2
  - FOXN3
  - FOXN4
  - FOXO1
  - FOXO3
  - FOXO4
  - FOXO6
  - FOXP1
  - FOXP2
  - FOXP3
  - FOXP4
  - FOXR1
  - FOXR2
  - FOXD4L1
  - FOXD4L3
  - FOXD4L4
  - FOXD4L5
  - FOXD4L6
  - RORC
  - HADH
  - ABL2
  - RERE
  - MAL
  - MRTFA
  - TIRAP
  - RASA1
  - RGS6
  - HAAO
  - FBP1
  - FOLR1
  - FOLR2
  - ECB2
  - FUBP1
  - RAB14
  - PAEP
  - PREP
  - PTPN22
  - CIT
  - DHDDS
  - KLF6
  - ZNHIT6
  - SERPINA3
  - ACTG1
  - ACTG2
  - FHL5
  - ACOT7
  - ACTBL2
  - POTEKP
  - POTEM
  - OMA1
  - ADA2
  - HP
  - RBP1
  - GFER
---
