---
figid: PMC9097893__fimmu-13-882521-g002
figtitle: NLRX1 Deficiency Alters the Gut Microbiome and Is Further Exacerbated by
  Adherence to a Gluten-Free Diet
organisms:
- Mycobacterium tuberculosis
- Klebsiella pneumoniae
- Clostridioides difficile
- Enterococcus faecalis
- Fusobacterium nucleatum
- Arthrospira platensis
- Lactobacillales incertae sedis
- unclassified Enterococcaceae
- Fagopyrum esculentum
- Chenopodium quinoa
- Amaranthus caudatus
- Oryza longistaminata
- Homo sapiens
- Mus musculus
- gut metagenome
- NA
pmcid: PMC9097893
filename: fimmu-13-882521-g002.jpg
figlink: /pmc/articles/PMC9097893/figure/f2/
number: F2
caption: Dysregulation of NF-κB Pathway Downstream of Impaired NLR Signaling May Potentiate
  CeD Pathogenesis. (A) CompBio AI algorithm predicted increased NF-κB/TNF signaling,
  NLR inflammasome signaling, and regulatory NLR signaling that is consistent with
  CeD pathology. (B) Schematic representative of IPA data output predicts that partial
  digestion of gluten is recognized as a DAMP by PRRs to signal NLRs in CeD. This
  results in downstream activation of canonical NF-κB and inflammasome formation,
  while inhibiting noncanonical NF-κB.
papertitle: NLRX1 Deficiency Alters the Gut Microbiome and Is Further Exacerbated
  by Adherence to a Gluten-Free Diet.
reftext: Holly A. Morrison, et al. Front Immunol. 2022;13:882521.
year: '2022'
doi: 10.3389/fimmu.2022.882521
journal_title: Frontiers in Immunology
journal_nlm_ta: Front Immunol
publisher_name: Frontiers Media S.A.
keywords: pattern recognition receptors (PRRs) | NOD-like receptors (NLRs) | gluten
  free diet | microbiome | dysbiosis | metabolites | probiotics
automl_pathway: 0.8250282
figid_alias: PMC9097893__F2
figtype: Figure
organisms_ner:
- Homo sapiens
- Mus musculus
redirect_from: /figures/PMC9097893__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9097893__fimmu-13-882521-g002.html
  '@type': Dataset
  description: Dysregulation of NF-κB Pathway Downstream of Impaired NLR Signaling
    May Potentiate CeD Pathogenesis. (A) CompBio AI algorithm predicted increased
    NF-κB/TNF signaling, NLR inflammasome signaling, and regulatory NLR signaling
    that is consistent with CeD pathology. (B) Schematic representative of IPA data
    output predicts that partial digestion of gluten is recognized as a DAMP by PRRs
    to signal NLRs in CeD. This results in downstream activation of canonical NF-κB
    and inflammasome formation, while inhibiting noncanonical NF-κB.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TNF
  - TH
  - IL10
  - IL18
  - NLRP1
  - STS
  - PYCARD
  - NLRP7
  - DAND5
  - IL15
  - NFKB1
  - CD40
  - ARHGEF7
  - ASCC1
  - H3P40
  - IL33
  - ST2
  - SULT2A1
  - IL1RL1
  - SDCBP2
  - GNAO1
  - GNAI1
  - GNAI2
  - GNAI3
  - GNAT1
  - GNAT2
  - GNAT3
  - GNAZ
  - BIRC3
  - TRAF2
  - TANK
  - TRAF3
  - NLRX1
  - MAP3K14
  - MAP4K4
  - CUX1
  - NFKB2
  - OAS3
  - PMEL
  - SART3
  - TPX2
  - SND1
  - ASCC2
  - RELB
  - CXCL12
  - Tnf
  - Igk
  - Th
  - Il10
  - Il18
  - Nlrp1a
  - Pycard
  - Dand5
  - Il15
  - Nfkb1
  - Cdc37
  - Lsp1
  - Pold2
  - Cd40
  - Dctn2
  - Il33
  - Il1rl1
  - Birc3
  - Birc2
  - Traf2
  - Traf3
  - Nlrx1
  - Map4k4
  - Map3k14
  - Tpx2
  - Relb
  - Cxcl12
---
