---
figid: PMC9622871__41598_2022_21091_Fig6_HTML
figtitle: Drug transporters OAT1 and OAT3 have specific effects on multiple organs
  and gut microbiome as revealed by contextualized metabolic network reconstructions
organisms:
- Mus musculus
- Lactococcus lactis
- Bacillus subtilis
- Mycobacterium tuberculosis
- Avena sativa
- Escherichia coli str. K-12 substr. MG1655
- Escherichia coli
- human metagenome
- Drosophila melanogaster
- gut metagenome
- Danio rerio
- Homo sapiens
- NA
organisms_ner:
- Drosophila melanogaster
- Mus musculus
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
- Danio rerio
- Arabidopsis thaliana
pmcid: PMC9622871
filename: 41598_2022_21091_Fig6_HTML.jpg
figlink: /pmc/articles/PMC9622871/figure/Fig6/
number: F6
caption: 'Highlight of reactions in Purine Synthesis (OAT1, Table ) and Bile Acid
  Synthesis (OAT3, Table ). Purine Synthesis: Reactions in the Purine Synthesis pathway
  that are not present in OAT1 WT models but are present in OAT1 KO are outlined.
  Fold change ratios of particular metabolites that were increased in OAT1 KO are
  presented next to their 3D bond-line molecular structures and include adenine, inosine,
  and IMP. Bile Acid Synthesis: A subset of the reactions that are present in the
  OAT3 knockout, but not the WT, are highlighted. Since the plasma concentration of
  choline is significantly higher in the OAT3 knockouts, the model pathways directly
  involving choline and choline-phospholipids were not increased (gray-shaded/opacified
  pathway at the base of the figure). Consistent with experimental data, however,
  pathways and metabolites with more complex, ring containing structures were identified
  in the OAT3 KO, including xoltetrol_m (3α,7α,12α,26-Tetrahydroxy-5β-cholestane)
  and thcholst_m (3α, 7α, 12α-Trihydroxy-5β-cholestan-26-al).'
papertitle: Drug transporters OAT1 and OAT3 have specific effects on multiple organs
  and gut microbiome as revealed by contextualized metabolic network reconstructions.
reftext: Neema Jamshidi, et al. Sci Rep. 2022;12:18308.
year: '2022'
doi: 10.1038/s41598-022-21091-w
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Biochemistry | Computational biology and bioinformatics | Kidney | Metabolism
  | Systems biology | Biochemical networks | Computer modelling | Genetic interaction
  | Systems analysis
automl_pathway: 0.9089526
figid_alias: PMC9622871__F6
figtype: Figure
redirect_from: /figures/PMC9622871__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9622871__41598_2022_21091_Fig6_HTML.html
  '@type': Dataset
  description: 'Highlight of reactions in Purine Synthesis (OAT1, Table ) and Bile
    Acid Synthesis (OAT3, Table ). Purine Synthesis: Reactions in the Purine Synthesis
    pathway that are not present in OAT1 WT models but are present in OAT1 KO are
    outlined. Fold change ratios of particular metabolites that were increased in
    OAT1 KO are presented next to their 3D bond-line molecular structures and include
    adenine, inosine, and IMP. Bile Acid Synthesis: A subset of the reactions that
    are present in the OAT3 knockout, but not the WT, are highlighted. Since the plasma
    concentration of choline is significantly higher in the OAT3 knockouts, the model
    pathways directly involving choline and choline-phospholipids were not increased
    (gray-shaded/opacified pathway at the base of the figure). Consistent with experimental
    data, however, pathways and metabolites with more complex, ring containing structures
    were identified in the OAT3 KO, including xoltetrol_m (3α,7α,12α,26-Tetrahydroxy-5β-cholestane)
    and thcholst_m (3α, 7α, 12α-Trihydroxy-5β-cholestan-26-al).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - uni
  - Hao
  - Ne
  - wisp
  - PAPLA1
  - Pli
  - Pdf
  - skd
  - hrg
  - ale
  - cas
  - caz
  - Cse1
  - cass
  - cash
  - Nach
  - eor
  - l(1)N30
  - homer
  - Adh
  - ned
  - Hmr
  - sesB
  - side
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - dati
  - pic
  - ado
  - ad
  - ACC
  - tacc
  - acclinal-wing
  - Had1
  - Bm
  - Slc7a1
  - Trpv6
  - Git1
  - Ppie
  - Gh
  - Asap1
  - Reg3b
  - Papola
  - Acp3
  - Papolb
  - Mrps30
  - Tusc2
  - Asap2
  - Pdap1
  - Cacna1e
  - Bcar1
  - Cse1l
  - Slc22a8
  - Decr1
  - Cat3
  - Slc7a3
  - Hc
  - Nr4a1
  - Brap
  - H20
  - Ado
  - Ad
  - Gadl1
  - Azin2
  - Acc
  - Tgfbi
  - Wdr5
  - atpH
  - casC
  - asd
  - insK
  - frmA
  - adhE
  - lptD
  - udp
  - ampC
  - ARHGAP9
  - CRAT
  - SLC7A1
  - TRPV6
  - PPIE
  - GH2
  - HAAO
  - REG3A
  - ASAP2
  - MRPS30
  - PAPOLA
  - PDAP1
  - TUSC2
  - ASAP1
  - CSE1L
  - CTNND1
  - BCAR1
  - TMX2-CTNND1
  - SLC22A8
  - DECR1
  - SLC7A3
  - AVP
  - ADH1A
  - ADH1B
  - ADH1C
  - ADH4
  - ADH5
  - ADH6
  - ADH7
  - ADHFE1
  - NR4A1
  - MRC1
  - IMPA1
  - BRAP
  - MATN3
  - MTG1
  - SLC25A3
  - ADO
  - AZIN2
  - GADL1
  - ACACA
  - BMS1
  - ACACB
  - HADH
  - pcxb
  - ppie
  - gh1
  - cse1l
  - sox32
  - slc22a6l
  - slc35b2
  - acaca
  - CAT1
  - AAT1
  - ALE1
  - CaS
  - CAT3
  - ADH1
  - PTAC12
  - AFO
  - BIG
  - Purine
  - Bile Acid
  - O
  - prane
  - Inosine
  - Choline Phosphate
  - Adenine
  - IATAD
---
