---
figid: PMC9348619__evac108f6
figtitle: Differentially Active and Conserved Neural Enhancers Define Two Forms of
  Adaptive Noncoding Evolution in Humans
organisms:
- Homo sapiens
- Pan troglodytes
- Mus musculus
- Drosophila melanogaster
- Caprine lentivirus
pmcid: PMC9348619
filename: evac108f6.jpg
figlink: /pmc/articles/PMC9348619/figure/evac108-F6/
number: F6
caption: Genes modulating canonical Wnt signaling associated with selection and DE
  CREs. CREs were associated with genes if they occur within 500 kb of TSSs. Genes
  associated with selection CREs have square outlines. Genes associated with CREs
  that are DE between orthologs have red (higher in human) or blue (higher in chimpanzee)
  fill. Genes associated with CREs that are DE between cell types are marked with
  horizontal (higher in NPC) or vertical (higher in neuron) stripes, and stripe colors
  indicate if the human (stripes) or chimpanzee (stripes) CRE was DE. The selection
  and DE profiles of CREs associated with paralogous genes are combined for each gene
  in the pathway but are separately defined in . Genes with circular outlines and
  white color are not associated with CREs in our assay but are included to inform
  about relevant interactions.
papertitle: Differentially Active and Conserved Neural Enhancers Define Two Forms
  of Adaptive Noncoding Evolution in Humans.
reftext: Jason Pizzollo, et al. Genome Biol Evol. 2022 Aug;14(8):evac108.
year: '2022'
doi: 10.1093/gbe/evac108
journal_title: Genome Biology and Evolution
journal_nlm_ta: Genome Biol Evol
publisher_name: Oxford University Press
keywords: Cis-regulation | human brain evolution | induced pluripotent stem cells
  | massively parallel enhancer assay
automl_pathway: 0.9212046
figid_alias: PMC9348619__F6
figtype: Figure
organisms_ner:
- Homo sapiens
- Pan troglodytes
- Mus musculus
- Drosophila melanogaster
redirect_from: /figures/PMC9348619__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9348619__evac108f6.html
  '@type': Dataset
  description: Genes modulating canonical Wnt signaling associated with selection
    and DE CREs. CREs were associated with genes if they occur within 500 kb of TSSs.
    Genes associated with selection CREs have square outlines. Genes associated with
    CREs that are DE between orthologs have red (higher in human) or blue (higher
    in chimpanzee) fill. Genes associated with CREs that are DE between cell types
    are marked with horizontal (higher in NPC) or vertical (higher in neuron) stripes,
    and stripe colors indicate if the human (stripes) or chimpanzee (stripes) CRE
    was DE. The selection and DE profiles of CREs associated with paralogous genes
    are combined for each gene in the pathway but are separately defined in . Genes
    with circular outlines and white color are not associated with CREs in our assay
    but are included to inform about relevant interactions.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CER1
  - C3CER1
  - WNT1
  - WNT2
  - WNT2B
  - WNT3
  - WNT3A
  - WNT4
  - WNT6
  - WNT7A
  - WNT7B
  - WNT8A
  - WNT8B
  - WNT9A
  - WNT9B
  - WNT10A
  - WNT10B
  - WNT11
  - WNT16
  - WNT5A
  - WNT5B
  - SFRP2
  - ADGRA2
  - DRAXIN
  - RPSA
  - LRP1
  - PTPRA
  - MVP
  - FZD1
  - FZD10
  - FZD2
  - FZD3
  - FZD4
  - FZD5
  - FZD6
  - FZD7
  - FZD8
  - FZD9
  - GPC3
  - GNAQ
  - DVL1
  - DVL1P1
  - DVL2
  - DVL3
  - TAFAZZIN
  - WWTR1
  - LATS1
  - LATS2
  - CSNK1E
  - STK4
  - DDX3X
  - GSKIP
  - CDH1
  - CDH10
  - CDH11
  - CDH12
  - CDH13
  - CDH15
  - CDH16
  - CDH17
  - CDH18
  - CDH19
  - CDH2
  - CDH20
  - CDH22
  - CDH24
  - CDH3
  - CDH4
  - CDH5
  - CDH6
  - CDH7
  - CDH8
  - CDH9
  - CTNNB1
  - PRKAR1A
  - PRKAR1B
  - PRKAR2A
  - PRKAR2B
  - PRKACA
  - PRKACB
  - PRKACG
  - GSK3B
  - PSEN1
  - APC
  - PROC
  - AXIN1
  - AXIN2
  - PPP2CB
  - SIAH1
  - TBL1X
  - SKP1
  - GPRC5B
  - FAM53B
  - RAPGEF1
  - SOX7
  - PSMA1
  - PSMA2
  - PSMA3
  - PSMA4
  - PSMA5
  - PSMA6
  - PSMA7
  - PSMA8
  - PSMB1
  - PSMB10
  - PSMB2
  - PSMB3
  - PSMB4
  - PSMB5
  - PSMB6
  - PSMB7
  - PSMB8
  - PSMB9
  - PSMC1
  - PSMC2
  - PSMC3
  - PSMC4
  - PSMC5
  - PSMC6
  - PSMD1
  - PSMD10
  - PSMD11
  - PSMD12
  - PSMD13
  - PSMD14
  - PSMD2
  - PSMD3
  - PSMD4
  - PSMD5
  - PSMD6
  - PSMD7
  - PSMD8
  - PSMD9
  - PSME1
  - PSME2
  - PSME3
  - PSMF1
  - SEM1
  - USP5
  - CST8
  - SOX4
  - KLF4
  - JADE1
  - JRK
  - CTBP2
  - TLE3
  - TLE4
  - CLCN7
  - HOPX
  - PYGO1
  - UBR5
  - BCL9L
  - HNF4A
  - NLK
  - GPI
  - MAP3K7
  - MYC
  - PPARD
  - NPC1
  - TLE1
  - Cer1
  - Acer1
  - Sfrp2
  - Adgra2
  - Draxin
  - Lrp1
  - Mvp
  - Txlng
  - Gpc3
  - Gnaq
  - Dvl1
  - Tafazzin
  - Wwtr1
  - Csnk1e
  - Stk4
  - Pak3
  - Ddx3x
  - Gskip
  - Ctnnb1
  - Gsk3b
  - Psen1
  - Apc
  - Axin1
  - Ppp2cb
  - Tbl1x
  - Skp1
  - Gprc5b
  - Fam53b
  - Rapgef1
  - Sox7
  - Cst8
  - Sox4
  - Klf4
  - Jade1
  - Jrk
  - Ctbp2
  - Tle3
  - Tle4
  - Hopx
  - Pygo1
  - Ubr5
  - Bcl9l
  - Nlk
  - Gpi1
  - Map3k7
  - Myc
  - Nol3
  - Ppard
  - Wnt5
  - Wnt4
  - wg
  - Wnt2
  - arr
  - fz
  - dsh
  - Taz
  - Dis3
  - wts
  - arm
  - Pka-C1
  - Pka-R2
  - Pka-C3
  - Pka-R1
  - Pka-C2
  - sgg
  - Axn
  - gskt
  - Apc2
  - Fs(3)Apc
  - SkpA
  - Clk
  - Fs(3)Hod
  - hyd
  - pan
  - nmo
  - mirr
  - de
---
