---
figid: PMC9328443__TPJ-103-512-g005
figtitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks'
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9328443
filename: TPJ-103-512-g005.jpg
figlink: /pmc/articles/PMC9328443/figure/tpj14746-fig-0002/
number: F2
caption: Functional SNPs in amino acid biosynthesis pathway of A. thaliana in three
  representative genotypes.Distribution of SNPs in three representative genotypes
  which have a low (RGR = 0.118), average (RGR = 0.475), and high (RGR = 2.02) growth
  rate. Metabolite names are shown in blue (with relative metabolite levels shown
  in cyan below them, wherever available), gene IDs in black and corresponding enzymes
  in orange. Activating SNPs are marked in green and inactivating SNPs in red, below
  the gene that they feature in (marked in black). The net fold‐change in flux (increase/decrease)
  through a reaction (with respect to the reference genotype) due to SNPs is given
  next to it, with a flux increase shown in green with an upward arrow and a flux
  decrease shown in red with a downward arrow. 2OG, 2‐oxoglutarate; DAHP, 3‐deoxy‐d‐arabino‐hept‐2‐ulosonate
  7‐phosphate; DAHP synthase, 3‐deoxy‐7‐phosphoheptulonate synthase; DHAD, dihydroxy‐acid
  dehydratase; E4P, erythrose 4 phosphate; KARI, ketol‐acid reductoisomerase; PEP,
  phosphoenolpyruvate; PRAI, phosphoribosylanthranilate isomerase; PRT, anthranilate
  phosphoribosyltransferase.
papertitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks.'
reftext: Debolina Sarkar, et al. Plant J. 2020 Jul;103(2):512-531.
year: '2020'
doi: 10.1111/tpj.14746
journal_title: The Plant Journal
journal_nlm_ta: Plant J
publisher_name: John Wiley and Sons Inc.
keywords: flux balance analysis | SNPs | metabolic networks | Arabidopsis | poplar
  | plant metabolic modeling | complementary GWAS
automl_pathway: 0.7854689
figid_alias: PMC9328443__F2
figtype: Figure
redirect_from: /figures/PMC9328443__F2
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9328443__TPJ-103-512-g005.html
  '@type': Dataset
  description: Functional SNPs in amino acid biosynthesis pathway of A. thaliana in
    three representative genotypes.Distribution of SNPs in three representative genotypes
    which have a low (RGR = 0.118), average (RGR = 0.475), and high (RGR = 2.02) growth
    rate. Metabolite names are shown in blue (with relative metabolite levels shown
    in cyan below them, wherever available), gene IDs in black and corresponding enzymes
    in orange. Activating SNPs are marked in green and inactivating SNPs in red, below
    the gene that they feature in (marked in black). The net fold‐change in flux (increase/decrease)
    through a reaction (with respect to the reference genotype) due to SNPs is given
    next to it, with a flux increase shown in green with an upward arrow and a flux
    decrease shown in red with a downward arrow. 2OG, 2‐oxoglutarate; DAHP, 3‐deoxy‐d‐arabino‐hept‐2‐ulosonate
    7‐phosphate; DAHP synthase, 3‐deoxy‐7‐phosphoheptulonate synthase; DHAD, dihydroxy‐acid
    dehydratase; E4P, erythrose 4 phosphate; KARI, ketol‐acid reductoisomerase; PEP,
    phosphoenolpyruvate; PRAI, phosphoribosylanthranilate isomerase; PRT, anthranilate
    phosphoribosyltransferase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - sup
  - san
  - ano
  - atl
  - atlas
  - ArgRS-m
  - ww
  - BomBc3
  - Con
  - pt
  - form3
  - Ag5r
  - SNAP25
  - HERPUD1
  - ATM
  - NAA50
  - ANO1
  - ANO10
  - ANO2
  - ANO3
  - ANO4
  - ANO5
  - ANO6
  - ANO7
  - ANO8
  - ANO9
  - GRHL2
  - NBN
  - INA
---
