---
figid: PMC9328443__TPJ-103-512-g004
figtitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks'
organisms:
- NA
pmcid: PMC9328443
filename: TPJ-103-512-g004.jpg
figlink: /pmc/articles/PMC9328443/figure/tpj14746-fig-0003/
number: F3
caption: Functional SNPs in the purine biosynthesis pathway of A. thaliana in three
  representative genotypes.Distribution of SNPs in three representative genotypes
  which have a low (RGR = 0.118), average (RGR = 0.475), and high (RGR = 2.02) growth
  rate. Metabolite names are shown in blue (with relative metabolite levels shown
  in cyan below them, wherever available), gene IDs in black and corresponding enzymes
  in orange. Activating SNPs are marked in green and inactivating SNPs in red, below
  the gene that they feature in. The net fold‐change in flux (increase/decrease) through
  a reaction due to SNPs is given next to it, with a flux increase shown in green
  with an upward arrow and a flux decrease shown in red with a downward arrow. AIR,
  5‐aminoimidazole ribonucleotide; AICAR, aminoimidazole carboxamide ribonucleotide;
  AICAR transformylase, phosphoribosyl aminoimidazole carboxamide formyltransferase;
  FAICAR, 5‐phosphoribosyl‐formamido‐carboxamide; FGAR, 5‐phosphoribosyl‐N‐formylglycinamide;
  GAR, 5‐phospho‐β‐d‐ribosyl‐glycinamide; GAR synthetase, phosphoribosylamine—glycine
  ligase; GAR TFase, phosphoribosylglycinamide formyltransferase; PRA, 5‐P‐β‐d‐ribosyl‐amine;
  THF, tetrahydrofolate.
papertitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks.'
reftext: Debolina Sarkar, et al. Plant J. 2020 Jul;103(2):512-531.
year: '2020'
doi: 10.1111/tpj.14746
journal_title: The Plant Journal
journal_nlm_ta: Plant J
publisher_name: John Wiley and Sons Inc.
keywords: flux balance analysis | SNPs | metabolic networks | Arabidopsis | poplar
  | plant metabolic modeling | complementary GWAS
automl_pathway: 0.701991
figid_alias: PMC9328443__F3
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
redirect_from: /figures/PMC9328443__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9328443__TPJ-103-512-g004.html
  '@type': Dataset
  description: Functional SNPs in the purine biosynthesis pathway of A. thaliana in
    three representative genotypes.Distribution of SNPs in three representative genotypes
    which have a low (RGR = 0.118), average (RGR = 0.475), and high (RGR = 2.02) growth
    rate. Metabolite names are shown in blue (with relative metabolite levels shown
    in cyan below them, wherever available), gene IDs in black and corresponding enzymes
    in orange. Activating SNPs are marked in green and inactivating SNPs in red, below
    the gene that they feature in. The net fold‐change in flux (increase/decrease)
    through a reaction due to SNPs is given next to it, with a flux increase shown
    in green with an upward arrow and a flux decrease shown in red with a downward
    arrow. AIR, 5‐aminoimidazole ribonucleotide; AICAR, aminoimidazole carboxamide
    ribonucleotide; AICAR transformylase, phosphoribosyl aminoimidazole carboxamide
    formyltransferase; FAICAR, 5‐phosphoribosyl‐formamido‐carboxamide; FGAR, 5‐phosphoribosyl‐N‐formylglycinamide;
    GAR, 5‐phospho‐β‐d‐ribosyl‐glycinamide; GAR synthetase, phosphoribosylamine—glycine
    ligase; GAR TFase, phosphoribosylglycinamide formyltransferase; PRA, 5‐P‐β‐d‐ribosyl‐amine;
    THF, tetrahydrofolate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - pra
  - prg
  - Snp
  - adp
  - pi
  - ATPsynbeta
  - Atpalpha
  - gar
  - air
  - Imp
  - CG11110
  - CG17029
  - CG17028
  - CG17027
  - CG9391
  - CG9389
  - rgr
  - PI
  - AT2G35040
  - AXR4
  - S100A6
  - WDTC1
  - ATP8A2
  - GAN
  - AIR
  - AIRN
  - ATIC
  - IMPA1
  - BRAP
  - RGR
  - RGL4
---
