---
figid: PMC9294428__bbac219f4
figtitle: Discovering gene regulatory networks of multiple phenotypic groups using
  dynamic Bayesian networks
organisms:
- Homo sapiens
- gut metagenome
organisms_ner:
- Homo sapiens
- Danio rerio
pmcid: PMC9294428
filename: bbac219f4.jpg
figlink: /pmc/articles/PMC9294428/figure/f4/
number: F4
caption: Subnetworks of DBN transition structures discovered for the GSE37182 dataset.
  Structures of time-varying DBN models without intra-edges were learned independently
  for normal and tumor subgroups. The blue edges denote the edges which are specific
  to the normal model; red edges are specific to the tumor model. Black edges are
  present in both models. The solid lines correspond to edges between genes which
  were found as interactors in the STRING database. Genes from the colorectal cancer
  pathway (KEGG) are highlighted in orange. (A) Most differently connected genes (FOSB,
  JUN, FOS, GADD45B) in DBN transition structures of cancer and normal DBN models
  that are either enriched in the colorectal cancer signaling pathway or were previously
  validated as biomarkers of cancer and their parents in the learned DBN models. (B)
  Most similarly connected genes, which were also found on the generic transcription
  pathway [] (highlighted in green).
papertitle: Discovering gene regulatory networks of multiple phenotypic groups using
  dynamic Bayesian networks.
reftext: Polina Suter, et al. Brief Bioinform. 2022 Jul;23(4):bbac219.
year: '2022'
doi: 10.1093/bib/bbac219
journal_title: Briefings in Bioinformatics
journal_nlm_ta: Brief Bioinform
publisher_name: Oxford University Press
keywords: dynamic Bayesian networks | time series | gene expression | Bayesian learning
  | classification | MCMC
automl_pathway: 0.9184073
figid_alias: PMC9294428__F4
figtype: Figure
redirect_from: /figures/PMC9294428__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9294428__bbac219f4.html
  '@type': Dataset
  description: Subnetworks of DBN transition structures discovered for the GSE37182
    dataset. Structures of time-varying DBN models without intra-edges were learned
    independently for normal and tumor subgroups. The blue edges denote the edges
    which are specific to the normal model; red edges are specific to the tumor model.
    Black edges are present in both models. The solid lines correspond to edges between
    genes which were found as interactors in the STRING database. Genes from the colorectal
    cancer pathway (KEGG) are highlighted in orange. (A) Most differently connected
    genes (FOSB, JUN, FOS, GADD45B) in DBN transition structures of cancer and normal
    DBN models that are either enriched in the colorectal cancer signaling pathway
    or were previously validated as biomarkers of cancer and their parents in the
    learned DBN models. (B) Most similarly connected genes, which were also found
    on the generic transcription pathway [] (highlighted in green).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CDK6
  - ATF2
  - GDNF
  - FOSB
  - H2AC6
  - RN7SK
  - FOXO1
  - FOXO3
  - FOXO4
  - FOXO6
  - HNF4A
  - HNF1A
  - NFKBIA
  - SMAD3
  - MXD3
  - JUN
  - JUNB
  - JUND
  - MXD4
  - TBP
  - FOS
  - FOSL1
  - FOSL2
  - RNU2-1
  - TFAP2A
  - H2BC15
  - H2BC26
  - VDR
  - CYP27B1
  - SMAD1
  - GARS1
  - RARA
  - SMAD2
  - E2F1
  - UBE2L3
  - PPARG
  - cdk6
  - atf2
  - fosb
  - rn7sk
  - hnf4a
  - nfkbiab
  - jun
  - mxd4
  - tbp
  - fosab
  - tfap2a
  - vdra
  - smad1
  - raraa
  - smad2
  - e2f1
  - pparg
---
