---
figid: PMC9682168__fmicb-13-1032515-g003
figtitle: Novel oligopeptide utilizing species Anaeropeptidivorans aminofermentans
  M3/9T, its role in anaerobic digestion and occurrence as deduced from large-scale
  fragment recruitment analyses
organisms:
- metagenome
- Methanobacterium congolense
- Methanothermobacter wolfeii
- Acetoanaerobium sticklandii
- Fermentimonas caenicola
- Herbinix hemicellulosilytica
- Anaerotignum lactatifermentans
- Petrimonas mucosa
- Proteiniphilum saccharofermentans
- methanogenic archaeon
- Anaerotignum propionicum
- Phalaris aquatica
- Cellulosilyticum lentocellum
- Clostridium colinum
- Anaerotignum neopropionicum
- metagenomes
- human gut metagenome
- soil metagenome
- Zea mays
- Saccharomyces cerevisiae
- uncultured bacterium EC5
- gut metagenome
- uncultured bacterium
- Gallus gallus
- Homo sapiens
- Equus caballus
- Anaerotignum propionicum DSM 1682
- NA
- NA
- NA
- NA
- NA
- NA
- NA
- NA
organisms_ner:
- Drosophila melanogaster
pmcid: PMC9682168
filename: fmicb-13-1032515-g003.jpg
figlink: /pmc/articles/PMC9682168/figure/fig3/
number: F3
caption: Metabolic reconstruction of relevant metabolic features and amino acid utilization
  in A. aminofermentans M3/9T. The metabolic reconstruction was performed with KEGG
  pathway maps represented in GenDB. The glycolysis and the pentose phosphate pathways
  are highlighted in pink and green, respectively. Amino acids proposed to be utilized
  by A. aminofermentans M3/9T are written in green and corresponding transport systems
  are displayed in blue ovals. Metabolic pathways shown in the figure are completely
  encoded in the genome.
papertitle: The novel oligopeptide utilizing species Anaeropeptidivorans aminofermentans
  M3/9T, its role in anaerobic digestion and occurrence as deduced from large-scale
  fragment recruitment analyses.
reftext: Irena Maus, et al. Front Microbiol. 2022;13:1032515.
year: '2022'
doi: 10.3389/fmicb.2022.1032515
journal_title: Frontiers in Microbiology
journal_nlm_ta: Front Microbiol
publisher_name: Frontiers Media S.A.
keywords: metagenome | plant biomass hydrolysis | renewable energy | Lachnospiraceae
  | omics analyses
automl_pathway: 0.9390081
figid_alias: PMC9682168__F3
figtype: Figure
redirect_from: /figures/PMC9682168__F3
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9682168__fmicb-13-1032515-g003.html
  '@type': Dataset
  description: Metabolic reconstruction of relevant metabolic features and amino acid
    utilization in A. aminofermentans M3/9T. The metabolic reconstruction was performed
    with KEGG pathway maps represented in GenDB. The glycolysis and the pentose phosphate
    pathways are highlighted in pink and green, respectively. Amino acids proposed
    to be utilized by A. aminofermentans M3/9T are written in green and corresponding
    transport systems are displayed in blue ovals. Metabolic pathways shown in the
    figure are completely encoded in the genome.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - na
  - ATPsynbeta
  - Atpalpha
  - pi
  - Gapdh2
  - CdGAPr
  - Gapdh1
  - RasGAP1
  - ArfGAP1
  - aa
  - fax
  - fd
  - anon-16Fb
  - glucuronate
  - Asparagine
  - Aspartate
  - 2-Oxoglytarate L-glutamate
  - 1-Butanol
  - ADP
  - Na
  - Ribulose
  - ATP
  - D-Ribose-5P
  - Butanoyl
  - Fructose-6P
  - Ribose
  - Glucose
  - Oxaloacetate Alanine Pyruvate
  - Glucose-6-P
  - Fructose-6-P
  - Acetyl-CoA
  - Acetate ATP
  - 2-Ketobutyrate
  - NH
  - Propanoyl-CoA
  - CO2
  - Glycine
  - Serine
  - Arginine
  - Threonine Ornithine
  - Propanoate
  - Cysteine
  - NADP
  - Acyl-CoA
  - Fd oxido
  - HCOOH
  - H2
  - Threonine
  - Acetaldehyde Glycine
  - NADH
  - Acetate
  - Acetyl-P CO2
  - Carboxybiotin Biotin
  - Methylmalonyl
  - oxaloacetate
  - 2-Amino-3
  - ketobutyrate
  - Methylene-THF
  - Folate
  - Glucose-6-P  Fructose-6-P
---
