---
figid: PMC9570313__ijms-23-12025-g010
figtitle: Transcriptional and Metabolic Characterization of Feeding Ramie Growth Enhanced
  by a Combined Application of Gibberellin and Ethrel
organisms:
- NA
pmcid: PMC9570313
filename: ijms-23-12025-g010.jpg
figlink: /pmc/articles/PMC9570313/figure/ijms-23-12025-f010/
number: F10
caption: A Schematic Diagram Developed from the Integration of Differentially Expressed
  Genes (DEGs) and Differentially Accumulated Metabolites (DAMs) in the Phenylpropanoid
  Pathway in Leaves. Three biological control leaf samples (CK-Y1, CK-Y2, and CK-Y3)
  and three biological treated leaf samples (C-Y1, C-Y2, and C-Y3) are coded by three
  squares or rectangles in the order of CK-Y1, CK-Y2, CK-Y3, C-Y1, C-Y2, and C-Y3.
  The six squares or rectangles were used to create heatmaps. The expression levels
  of genes and accumulation levels of metabolites in C-Y1, C-Y2, C-Y3, C-Y1, C-Y2,
  and C-Y3 are coded by colored heatmaps of the six squares or rectangles. PAL, 4CL,
  CCR, CYP98A, and CAD coded with a red color on each arrow mean that their gene expression
  levels are upregulated in the treated leaves. HCT coded by a light blue means that
  its gene expression is reduced. Six metabolite names, which are coded with a red
  color at end of arrows and included together in a heatmap contained in a dashed
  box, mean that their levels are increased in the treated leaves.
papertitle: Transcriptional and Metabolic Characterization of Feeding Ramie Growth
  Enhanced by a Combined Application of Gibberellin and Ethrel.
reftext: Hongdong Jie, et al. Int J Mol Sci. 2022 Oct;23(19):12025.
year: '2022'
doi: 10.3390/ijms231912025
journal_title: International Journal of Molecular Sciences
journal_nlm_ta: Int J Mol Sci
publisher_name: MDPI
keywords: GA3 | ETH | transcriptomics | metabolomics | secondary metabolites
automl_pathway: 0.910872
figid_alias: PMC9570313__F10
figtype: Figure
organisms_ner:
- Arabidopsis thaliana
redirect_from: /figures/PMC9570313__F10
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9570313__ijms-23-12025-g010.html
  '@type': Dataset
  description: A Schematic Diagram Developed from the Integration of Differentially
    Expressed Genes (DEGs) and Differentially Accumulated Metabolites (DAMs) in the
    Phenylpropanoid Pathway in Leaves. Three biological control leaf samples (CK-Y1,
    CK-Y2, and CK-Y3) and three biological treated leaf samples (C-Y1, C-Y2, and C-Y3)
    are coded by three squares or rectangles in the order of CK-Y1, CK-Y2, CK-Y3,
    C-Y1, C-Y2, and C-Y3. The six squares or rectangles were used to create heatmaps.
    The expression levels of genes and accumulation levels of metabolites in C-Y1,
    C-Y2, C-Y3, C-Y1, C-Y2, and C-Y3 are coded by colored heatmaps of the six squares
    or rectangles. PAL, 4CL, CCR, CYP98A, and CAD coded with a red color on each arrow
    mean that their gene expression levels are upregulated in the treated leaves.
    HCT coded by a light blue means that its gene expression is reduced. Six metabolite
    names, which are coded with a red color at end of arrows and included together
    in a heatmap contained in a dashed box, mean that their levels are increased in
    the treated leaves.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SHT
  - FdC1
  - HCT
  - ATCAD4
  - LysoPL2
  - MEE23
---
