---
figid: PMC9617413__12934_2022_1953_Fig4_HTML
figtitle: Transcriptomic and metabolomic analyses for providing insights into the
  influence of polylysine synthetase on the metabolism of Streptomyces albulus
organisms:
- NA
pmcid: PMC9617413
filename: 12934_2022_1953_Fig4_HTML.jpg
figlink: /pmc/articles/PMC9617413/figure/Fig4/
number: F4
caption: Effects of the high expression and inactivation of Pls on the expression
  levels of genes involved in the ε-PL metabolic pathway. The red lines, arrows and
  gene names represent significantly up-regulated gene expression catalyzing the step,
  while blue ones represent significantly down-regulated gene expression. The numbers
  following the gene names are the FCs in gene expression. fruk, phosphofructokinase;
  pdhA, pyruvate dehydrogenase (acetyl-transferring) E1 component subunit alpha; pdhB,
  pyruvate dehydrogenase E1 component subunit beta; aceF, pyruvate dehydrogenase complex,
  dihydrolipoamide acyltransferase (E2) component; aceA, isocitrate lyase; aceB, malate
  synthase A; sdhB, succinate dehydrogenase; ectA, L-2,4-diaminobutyrate acetyltransferase;
  ectB, diaminobutyrate-2-oxoglutarate transaminase; ectC, l-ectoine synthase; ectD,
  ectoine hydroxylase; dapA, 4-hydroxy-tetrahydrodipicolinate synthase; pls, polylysine
  synthetase; argD, acetylornithine aminotransferase; lysA, diaminopimelate decarboxylase;
  gpmB, phosphoglycerate mutase; ppdK, pyruvate phosphate dikinase
papertitle: Transcriptomic and metabolomic analyses for providing insights into the
  influence of polylysine synthetase on the metabolism of Streptomyces albulus.
reftext: Congcong Lian, et al. Microb Cell Fact. 2022;21:224.
year: '2022'
doi: 10.1186/s12934-022-01953-8
journal_title: Microbial Cell Factories
journal_nlm_ta: Microb Cell Fact
publisher_name: BioMed Central
keywords: ε-poly-l-lysine | Polylysine synthetase | Streptomyces albulus | Transcriptomics
  | Metabonomics
automl_pathway: 0.9345205
figid_alias: PMC9617413__F4
figtype: Figure
organisms_ner:
- Drosophila melanogaster
- Escherichia coli str. K-12 substr. MG1655
- Homo sapiens
redirect_from: /figures/PMC9617413__F4
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9617413__12934_2022_1953_Fig4_HTML.html
  '@type': Dataset
  description: Effects of the high expression and inactivation of Pls on the expression
    levels of genes involved in the ε-PL metabolic pathway. The red lines, arrows
    and gene names represent significantly up-regulated gene expression catalyzing
    the step, while blue ones represent significantly down-regulated gene expression.
    The numbers following the gene names are the FCs in gene expression. fruk, phosphofructokinase;
    pdhA, pyruvate dehydrogenase (acetyl-transferring) E1 component subunit alpha;
    pdhB, pyruvate dehydrogenase E1 component subunit beta; aceF, pyruvate dehydrogenase
    complex, dihydrolipoamide acyltransferase (E2) component; aceA, isocitrate lyase;
    aceB, malate synthase A; sdhB, succinate dehydrogenase; ectA, L-2,4-diaminobutyrate
    acetyltransferase; ectB, diaminobutyrate-2-oxoglutarate transaminase; ectC, l-ectoine
    synthase; ectD, ectoine hydroxylase; dapA, 4-hydroxy-tetrahydrodipicolinate synthase;
    pls, polylysine synthetase; argD, acetylornithine aminotransferase; lysA, diaminopimelate
    decarboxylase; gpmB, phosphoglycerate mutase; ppdK, pyruvate phosphate dikinase
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - nrv2
  - dap
  - AstA
  - pls
  - Pis
  - vs
  - del
  - tx
  - mirr
  - fs(1)de1
  - Pdha
  - Pdhb
  - SdhB
  - LKRSDH
  - SdhA
  - Sodh-2
  - SdhC
  - asd
  - lysA
  - argH
  - argD
  - fruK
  - aceF
  - aceB
  - sdhB
  - aceA
  - DAP
  - IAPP
  - DNPEP
  - CTSC
  - CDIPT
  - PDHA1
  - PDHB
  - SDHB
  - SARDH
  - SORD
  - SDS
---
