---
figid: PMC9213282__11103_2021_1219_Fig1_HTML
figtitle: Influence of endophytes on rice fitness under environmental stresses
organisms:
- Oryza sativa
- Pseudomonas aeruginosa
- Bacillus subtilis
- Klebsiella pneumoniae
- Pseudomonas putida
- Pseudomonas fluorescens
- Streptomyces sp.
- Staphylococcus epidermidis
- Streptomyces hygroscopicus
- Enterobacter cloacae
- Bacillus amyloliquefaciens
- Micrococcus luteus
- Bacillus sp. in firmicutes
- Pseudomonas sp.
- Xanthomonas oryzae pv. oryzae
- Pantoea agglomerans
- Xanthomonas oryzae
- Bacillus pumilus
- Klebsiella oxytoca
- Enterobacter sp.
- Burkholderia sp.
- Lysinibacillus sphaericus
- endophytic bacterium
- Gluconacetobacter diazotrophicus
- Achromobacter sp.
- Azospirillum lipoferum
- Paenibacillus sp.
- Burkholderia glumae
- Bacillus tequilensis
- Bacillus velezensis
- Sphingobium yanoikuyae
- Brevibacillus laterosporus
- Sphingomonas melonis
- Priestia aryabhattai
- Leclercia adecarboxylata
- Azospirillum sp. B510
- Micrococcus yunnanensis
- Herbaspirillum sp.
- Bacillus siamensis
- Chryseobacterium sp.
- Pseudomonas koreensis
- Pantoea dispersa
- Azoarcus sp. CIB
- Azoarcus sp.
- Bradyrhizobium sp. SUTN9-2
- Glutamicibacter sp.
- Burkholderia plantarii
- Burkholderia vietnamiensis
- Streptomyces mutabilis
- Streptomyces sp. GMKU 3100
- Streptomyces sp. GMKU 336
- Fictibacillus barbaricus
- Pantoea alhagi
- Kosakonia sp.
- Burkholderia sp. KJ006
- Paracidovorax oryzae
- Enterobacter soli
- Saccharomyces cerevisiae
- Arabidopsis thaliana
- Zea mays
- Glycine max
- Aspergillus fumigatus
- eudicotyledons
- Saccharum officinarum
- Catharanthus roseus
- Ipomoea batatas
- Fusarium oxysporum
- Pyricularia oryzae
- Melia azedarach
- Oryza sativa Japonica Group
- Pyricularia grisea
- Oryza sativa Indica Group
- Fusarium verticillioides
- Candida tropicalis
- Aspergillus sp.
- Rhizoctonia solani
- Jasminum officinale
- Fusarium fujikuroi
- Trichoderma harzianum
- Fusarium sp.
- Piper nigrum
- Zizania
- fungal endophyte
- Suaeda salsa
- Serendipita indica
- Alternaria sp.
- Rhizophagus intraradices
- Chaetomium sp.
- Avicennia marina
- Curvularia sp.
- Paecilomyces maximus
- Falciphora oryzae
- Cladosporium sphaerospermum
- Leersia oryzoides
- Arthrinium phaeospermum
- Didymella glomerata
- Meloidogyne javanica
- Meloidogyne graminicola
- Lissorhoptrus oryzophilus
- Homo sapiens
- NA
organisms_ner:
- Arabidopsis thaliana
- Homo sapiens
- Glycine max
- Oryza sativa Japonica Group
- Drosophila melanogaster
pmcid: PMC9213282
filename: 11103_2021_1219_Fig1_HTML.jpg
figlink: /pmc/articles/PMC9213282/figure/Fig1/
number: F1
caption: Molecular mechanisms regulating endophyte-mediated rice responses to salt
  stress. The salt stress signal is perceived by various signal transduction sensors
  transducing the signal along different pathways into the cell. These sensors include
  membrane-associated kinases, G-protein coupled receptors, glutamate receptor-like
  channels, calmodulin binding receptors, cyclic nucleotide-gated channels, and Ca2+
  channel opening (Jamla and Archak ). Endophytes target different components in this
  signalling system during salt stress in the rice cell. They modulate either phytohormone-
  such as auxin, ethylene and abscisic acid (ABA) (Liu et al. ; Khan et al. ; Sultana
  et al. ), mitogen-activated protein kinase (MAPK)-, or Ca2+-signalling (through
  calcium-dependent protein kinases (CDPK)) pathways (Jaemsaeng et al. ; Chauhan et
  al. ), which transduce the signal into the cell to modulate the expression of transcription
  factors (TFs) such as ethylene responsive factor (ERF), dehydration-responsive element-binding
  2 (DREB2), NAC (NAM, ATAF and CUC), auxin response factor (ARF), and myeloblastosis
  (MYB). These TFs in turn induce salt-responsive genes (for each case, only representative
  genes are shown) such as Salt Overly Sensitive 1 (SOS1) and Sodium/Hydrogen eXchanger
  1 (NHX1) for ion-homeostasis, Betaine Aldehyde Dehydrogenase (BADH) for osmotic
  adjustment, and CATALASE b (CATb) for antioxidation (Nautiyal et al. ; Ji et al.
  ). They can also target Ca2+-signalling pathway (through calmodulin, CaM1-1) leading
  to the SOS pathway-mediated ion-homeostasis in the rice cell by activating plasma
  membrane-bound SOS1 and vacuolar NHX1 (Jaemsaeng et al. ) proteins. Endophytes also
  induce the expression of various salt-responsive miRNAs which can regulate these
  TFs and structural genes like SOS1 and high-affinity K+transporter 1 (HKT1) (Kord
  et al. ), supporting the existence of crosstalk between signalling pathways, conferring
  enhanced salt tolerance in rice. Dashed arrows indicate possible mechanisms. Blue
  arrows along the pathway from MAPK through ERF indicate endophyte-mediated pathway
  downregulation
papertitle: The influence of endophytes on rice fitness under environmental stresses.
reftext: Showkat Ahmad Ganie, et al. Plant Mol Biol. 2022;109(4-5):447-467.
year: '2022'
doi: 10.1007/s11103-021-01219-8
journal_title: Plant Molecular Biology
journal_nlm_ta: Plant Mol Biol
publisher_name: Springer Netherlands
keywords: Abiotic/biotic stress | Endophyte | Microbiome | Oryza sativa | Rice | Salinity
  | Systemic resistance
automl_pathway: 0.9017878
figid_alias: PMC9213282__F1
figtype: Figure
redirect_from: /figures/PMC9213282__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9213282__11103_2021_1219_Fig1_HTML.html
  '@type': Dataset
  description: Molecular mechanisms regulating endophyte-mediated rice responses to
    salt stress. The salt stress signal is perceived by various signal transduction
    sensors transducing the signal along different pathways into the cell. These sensors
    include membrane-associated kinases, G-protein coupled receptors, glutamate receptor-like
    channels, calmodulin binding receptors, cyclic nucleotide-gated channels, and
    Ca2+ channel opening (Jamla and Archak ). Endophytes target different components
    in this signalling system during salt stress in the rice cell. They modulate either
    phytohormone- such as auxin, ethylene and abscisic acid (ABA) (Liu et al. ; Khan
    et al. ; Sultana et al. ), mitogen-activated protein kinase (MAPK)-, or Ca2+-signalling
    (through calcium-dependent protein kinases (CDPK)) pathways (Jaemsaeng et al.
    ; Chauhan et al. ), which transduce the signal into the cell to modulate the expression
    of transcription factors (TFs) such as ethylene responsive factor (ERF), dehydration-responsive
    element-binding 2 (DREB2), NAC (NAM, ATAF and CUC), auxin response factor (ARF),
    and myeloblastosis (MYB). These TFs in turn induce salt-responsive genes (for
    each case, only representative genes are shown) such as Salt Overly Sensitive
    1 (SOS1) and Sodium/Hydrogen eXchanger 1 (NHX1) for ion-homeostasis, Betaine Aldehyde
    Dehydrogenase (BADH) for osmotic adjustment, and CATALASE b (CATb) for antioxidation
    (Nautiyal et al. ; Ji et al. ). They can also target Ca2+-signalling pathway (through
    calmodulin, CaM1-1) leading to the SOS pathway-mediated ion-homeostasis in the
    rice cell by activating plasma membrane-bound SOS1 and vacuolar NHX1 (Jaemsaeng
    et al. ) proteins. Endophytes also induce the expression of various salt-responsive
    miRNAs which can regulate these TFs and structural genes like SOS1 and high-affinity
    K+transporter 1 (HKT1) (Kord et al. ), supporting the existence of crosstalk between
    signalling pathways, conferring enhanced salt tolerance in rice. Dashed arrows
    indicate possible mechanisms. Blue arrows along the pathway from MAPK through
    ERF indicate endophyte-mediated pathway downregulation
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - APK
  - DREB2B
  - DREB2A
  - MIR164a
  - MIR164b
  - MIR164c
  - ALDH10A9
  - bZIP
  - SOS2
  - ERF
  - ETF1
  - XK
  - NLRP1
  - CDKN2A
  - MYB
  - LOC732579
  - WRKY14
  - MIR164A
  - CDPK
  - ARF
  - MIR159F
  - LOC4326471
  - LOC4334553
  - LOC4336081
  - LOC4326208
  - LOC4339157
  - LOC4350088
  - LOC4324302
  - LOC4336407
  - LOC4343580
  - LOC4343988
  - LOC4346997
  - LOC4348906
  - LOC4351379
  - LOC4324384
  - salt
  - ap
  - Nplp1
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - nac
  - Nacalpha
  - Arl1
  - Arf1
  - Arl2
  - cato
  - Myb
  - ca
  - na
---
