---
figid: PMC9234166__fcvm-09-848045-g0001
figtitle: Tandem Mass Tagging Based Identification of Proteome Signatures for Reductive
  Stress Cardiomyopathy
organisms:
- Homo sapiens
- Mus musculus
- Cucumis sativus
organisms_ner:
- Mus musculus
- Homo sapiens
- Danio rerio
pmcid: PMC9234166
filename: fcvm-09-848045-g0001.jpg
figlink: /pmc/articles/PMC9234166/figure/F1/
number: F1
caption: Effect of chronic reductive stress on myocardial proteome. (A) Overall methodology
  adopted for Tandem Mass Tagged LC-MS/MS analysis. Heart tissues were harvested from
  non-transgenic (NTg) and caNrf2-Tg (N = 4 mice/group) and protein concentration
  was determined in homogenates with BCA kit (Bio-Rad, USA). After trypsin digestion,
  peptides were reconstituted in 0.5 M TEAB and processed for TMT tagging (Tandem
  Mass Tag kit, ThermoFisher Scientific). LC-MS/MS analysis was performed on a Q Exactive
  Plus mass spectrometer (Thermo Fisher Scientific) equipped with an EASY-Spray nanoESI
  source. MS and MS/MS data were searched against the amino acid sequence of the Uniprot
  mouse protein database using Thermo Proteome Discoverer v 2.4.0.305 (Thermo Fisher
  Scientific). The protein and peptide identification results were further analyzed
  with Scaffold Q + S v 4.11.1 (Proteome Software Inc.). (B) Comprehensive approach
  used to calculate the “redox score” in TG by comparing to basal redox factors in
  NTg. Here, we included the levels of small molecular antioxidants (i.e. GSH, Cysteine/Cystine
  ratio); antioxidant proteins (i.e. GST MU, NQO1, CAT, GPX 1, SOD1, GCLM, GCLC, SOD2);
  antioxidant enzymatic levels (i.e. TAC); antioxidant gene expression (i.e. GCLM,
  NQO1, GSR, GST alpha, GCLC, GCLM, GSR, NQO1, GST MU, GPX 1, CAT) as well as the
  reactive oxygen species levels (i.e. DHE). (C) Quantitative genotyping in caNrf2
  mice caNrf2 genotyping primer, endogenous primers for Nrf2, NQO1 and GCLC. (D) Principal
  Component Analysis (PCA) Plot generated using Total Identified Proteins (TIPs) showed
  segregation of NTg and CaNrf2 TG as distinct groups. (E) Venn diagram showing the
  number of Differentially Expressed Proteins (DEPs) based on different fold change
  (CaNrf2 TG vs. NTg) identified in TMT proteome software. (F) Global heat map generated
  using R studio for TIPs (1,105) proteins identified in TMT analysis. (G) Gene ontology
  pathway by PANTHER analysis. Protein Analysis THrough Evolutionary Relationships
  (PANTHER) analysis (http://pantherdb.org/) for biological function distributed the
  total identified proteins (TIPs) into different metabolic categories. About 25%
  of the proteome function grouped under metabolite interconversion category, with
  oxidoreductase enzyme family as the top upregulated one. The proteins identified
  under oxidoreductase group are shown in (H).
papertitle: Tandem Mass Tagging Based Identification of Proteome Signatures for Reductive
  Stress Cardiomyopathy.
reftext: Sini Sunny, et al. Front Cardiovasc Med. 2022;9:848045.
year: '2022'
doi: 10.3389/fcvm.2022.848045
journal_title: Frontiers in Cardiovascular Medicine
journal_nlm_ta: Front Cardiovasc Med
publisher_name: Frontiers Media S.A.
keywords: reductive stress | caNrf2 | myocardial proteome | Tandem Mass Tag proteomic
  analysis | speckle tracking echocardiography
automl_pathway: 0.5396605
figid_alias: PMC9234166__F1
figtype: Figure
redirect_from: /figures/PMC9234166__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9234166__fcvm-09-848045-g0001.html
  '@type': Dataset
  description: Effect of chronic reductive stress on myocardial proteome. (A) Overall
    methodology adopted for Tandem Mass Tagged LC-MS/MS analysis. Heart tissues were
    harvested from non-transgenic (NTg) and caNrf2-Tg (N = 4 mice/group) and protein
    concentration was determined in homogenates with BCA kit (Bio-Rad, USA). After
    trypsin digestion, peptides were reconstituted in 0.5 M TEAB and processed for
    TMT tagging (Tandem Mass Tag kit, ThermoFisher Scientific). LC-MS/MS analysis
    was performed on a Q Exactive Plus mass spectrometer (Thermo Fisher Scientific)
    equipped with an EASY-Spray nanoESI source. MS and MS/MS data were searched against
    the amino acid sequence of the Uniprot mouse protein database using Thermo Proteome
    Discoverer v 2.4.0.305 (Thermo Fisher Scientific). The protein and peptide identification
    results were further analyzed with Scaffold Q + S v 4.11.1 (Proteome Software
    Inc.). (B) Comprehensive approach used to calculate the “redox score” in TG by
    comparing to basal redox factors in NTg. Here, we included the levels of small
    molecular antioxidants (i.e. GSH, Cysteine/Cystine ratio); antioxidant proteins
    (i.e. GST MU, NQO1, CAT, GPX 1, SOD1, GCLM, GCLC, SOD2); antioxidant enzymatic
    levels (i.e. TAC); antioxidant gene expression (i.e. GCLM, NQO1, GSR, GST alpha,
    GCLC, GCLM, GSR, NQO1, GST MU, GPX 1, CAT) as well as the reactive oxygen species
    levels (i.e. DHE). (C) Quantitative genotyping in caNrf2 mice caNrf2 genotyping
    primer, endogenous primers for Nrf2, NQO1 and GCLC. (D) Principal Component Analysis
    (PCA) Plot generated using Total Identified Proteins (TIPs) showed segregation
    of NTg and CaNrf2 TG as distinct groups. (E) Venn diagram showing the number of
    Differentially Expressed Proteins (DEPs) based on different fold change (CaNrf2
    TG vs. NTg) identified in TMT proteome software. (F) Global heat map generated
    using R studio for TIPs (1,105) proteins identified in TMT analysis. (G) Gene
    ontology pathway by PANTHER analysis. Protein Analysis THrough Evolutionary Relationships
    (PANTHER) analysis (http://pantherdb.org/) for biological function distributed
    the total identified proteins (TIPs) into different metabolic categories. About
    25% of the proteome function grouped under metabolite interconversion category,
    with oxidoreductase enzyme family as the top upregulated one. The proteins identified
    under oxidoreductase group are shown in (H).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Tg
  - Stab1
  - Musz1
  - n-TGccc2
  - Cbx4
  - Pcsk2
  - Pkd2
  - Ms6hm3
  - n-TGacc1
  - n-TGccc3
  - n-TGccc1
  - Tpsg1
  - fc
  - Idh3a
  - Acad10
  - Uqcrb
  - Uqcrq
  - Etfdh
  - Sod2
  - Fdx1
  - Ndufb8
  - Nfe2l2
  - Sdha
  - Cbr4
  - Rtn4ip1
  - Prdx2
  - Cox7a1
  - Pdha1
  - Nqo1
  - Ndufs6
  - Cox6c
  - Gpx1
  - Hsdl2
  - Xdh
  - Sord
  - Gpx4
  - Gpx4-ps2
  - Ndufa4
  - Bdh1
  - Acads
  - Ldha
  - Akr1a1
  - TG
  - OPA1
  - MS
  - KRT6B
  - KRT17
  - PCSK2
  - PKD2
  - CBX4
  - TPSG1
  - IDH3A
  - ACAD10
  - UQCRB
  - ETFDH
  - SOD2
  - SOD2-OT1
  - FDX1
  - NDUFB8
  - GABPA
  - NFE2L2
  - SDHA
  - CBR4
  - RTN4IP1
  - PRDX2
  - COX7A1
  - PDHA1
  - NQO1
  - NDUFS6
  - COX6C
  - GPX1
  - HSDL2
  - XDH
  - SORD
  - GPX4
  - NDUFA4
  - NDUFA4P1
  - BDH1
  - ACADS
  - LDHA
  - AKR1A1
  - tg
  - msi1b
  - kita
  - ngfra
  - idh3a
  - uqcrb
  - etfdh
  - sod2
  - fdx1
  - ndufb8
  - nfe2l2a
  - sdha
  - cbr4
  - rtn4ip1
  - prdx2
  - cox7a1
  - pdha1a
  - pdha1b
  - nqo1
  - ndufs6
  - cox6c
  - gpx1a
  - hsdl2
  - xdh
  - aox5
  - ndufa4b
  - bdh1
  - acads
  - ldha
  - akr1a1a
  - akr1a1b
---
