---
figid: PMC9328443__TPJ-103-512-g002
figtitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks'
organisms:
- NA
pmcid: PMC9328443
filename: TPJ-103-512-g002.jpg
figlink: /pmc/articles/PMC9328443/figure/tpj14746-fig-0005/
number: F5
caption: Functional SNPs in the purine and pyrimidine biosynthesis pathway of P. trichocarpa
  in three representative genotypes.Distribution of SNPs in three representative genotypes
  which have low (RGR = 1.17), average (RGR = 7.25), and high (RGR = 42.08) growth
  rates. Metabolite names are shown in blue, gene IDs in black and corresponding enzymes
  in orange. Activating SNPs are marked in green and inactivating SNPs in red, below
  the gene that they feature in. The net fold‐change in flux (increase/decrease) through
  a reaction due to SNPs is given next to it, with a flux increase shown in green
  with an upward arrow and a flux decrease shown in red with a downward arrow. ATCase,
  aspartate carbamoyltransferase; dUTPase, dUTP diphosphatase; OMP‐DC, orotidine‐5′‐phosphate
  decarboxylase; PRPP synthetase, 5‐phosphoribosyl‐1‐pyrophosphate synthetase; UPRTase,
  uracil phosphoribosyltransferase.
papertitle: 'SNPeffect: identifying functional roles of SNPs using metabolic networks.'
reftext: Debolina Sarkar, et al. Plant J. 2020 Jul;103(2):512-531.
year: '2020'
doi: 10.1111/tpj.14746
journal_title: The Plant Journal
journal_nlm_ta: Plant J
publisher_name: John Wiley and Sons Inc.
keywords: flux balance analysis | SNPs | metabolic networks | Arabidopsis | poplar
  | plant metabolic modeling | complementary GWAS
automl_pathway: 0.7659517
figid_alias: PMC9328443__F5
figtype: Figure
organisms_ner:
- Drosophila melanogaster
redirect_from: /figures/PMC9328443__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9328443__TPJ-103-512-g002.html
  '@type': Dataset
  description: Functional SNPs in the purine and pyrimidine biosynthesis pathway of
    P. trichocarpa in three representative genotypes.Distribution of SNPs in three
    representative genotypes which have low (RGR = 1.17), average (RGR = 7.25), and
    high (RGR = 42.08) growth rates. Metabolite names are shown in blue, gene IDs
    in black and corresponding enzymes in orange. Activating SNPs are marked in green
    and inactivating SNPs in red, below the gene that they feature in. The net fold‐change
    in flux (increase/decrease) through a reaction due to SNPs is given next to it,
    with a flux increase shown in green with an upward arrow and a flux decrease shown
    in red with a downward arrow. ATCase, aspartate carbamoyltransferase; dUTPase,
    dUTP diphosphatase; OMP‐DC, orotidine‐5′‐phosphate decarboxylase; PRPP synthetase,
    5‐phosphoribosyl‐1‐pyrophosphate synthetase; UPRTase, uracil phosphoribosyltransferase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Snp
  - ATPsynbeta
  - Atpalpha
  - CkIIbeta
  - Andorra
  - Anp
  - Acam
  - Amph
  - r-l
  - adp
  - dUTPase
  - rgr
---
