---
figid: PMC9326605__epigenomes-06-00017-g001
figtitle: DNA Hypomethylation May Contribute to Metabolic Recovery of Frozen Wood
  Frog Brains
organisms:
- NA
organisms_ner:
- Drosophila melanogaster
- Homo sapiens
pmcid: PMC9326605
filename: epigenomes-06-00017-g001.jpg
figlink: /pmc/articles/PMC9326605/figure/epigenomes-06-00017-f001/
number: F1
caption: CpG islands exist ubiquitously throughout the genome, often upstream of gene
  promoter elements. Methylation of CpG islands is typically associated with repression
  of downstream gene transcription. Cytosine residues in these CpG islands are often
  methylated by DNMT3A/3L and DNMT3B/3L complexes, whereas DNMT1 functions mainly
  to maintain 5-mC of hemi-methylated DNA during replication. DNMTs require the universal
  methyl donor SAM. DNA demethylation requires the coordinated action of TET enzymes
  with TDG and the base excision repair pathway, generating 5-hydroxymethylcytosine,
  (5-hmC), 5-fluorocytosine (5-fC), and 5-carboxylcytosine (5-caC) intermediates during
  the process. TETs require oxygen and α-ketoglutarate (αKG) as reaction substrates.
  Created with Biorender.com.
papertitle: DNA Hypomethylation May Contribute to Metabolic Recovery of Frozen Wood
  Frog Brains.
reftext: Tighe Bloskie, et al. Epigenomes. 2022 Sep;6(3):17.
year: '2022'
doi: 10.3390/epigenomes6030017
journal_title: Epigenomes
journal_nlm_ta: Epigenomes
publisher_name: MDPI
keywords: DNA methylation | epigenetics | freeze tolerance | metabolic rate depression
  | Rana sylvatica
automl_pathway: 0.811548
figid_alias: PMC9326605__F1
figtype: Figure
redirect_from: /figures/PMC9326605__F1
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9326605__epigenomes-06-00017-g001.html
  '@type': Dataset
  description: CpG islands exist ubiquitously throughout the genome, often upstream
    of gene promoter elements. Methylation of CpG islands is typically associated
    with repression of downstream gene transcription. Cytosine residues in these CpG
    islands are often methylated by DNMT3A/3L and DNMT3B/3L complexes, whereas DNMT1
    functions mainly to maintain 5-mC of hemi-methylated DNA during replication. DNMTs
    require the universal methyl donor SAM. DNA demethylation requires the coordinated
    action of TET enzymes with TDG and the base excision repair pathway, generating
    5-hydroxymethylcytosine, (5-hmC), 5-fluorocytosine (5-fC), and 5-carboxylcytosine
    (5-caC) intermediates during the process. TETs require oxygen and α-ketoglutarate
    (αKG) as reaction substrates. Created with Biorender.com.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - mbd
  - MBD-R2
  - Mt2
  - Nmt
  - Me
  - sam
  - qkr54B
  - sls
  - Sam-S
  - sah
  - Tdg
  - ber
  - Tet
  - DNMT1
  - DNMT3A
  - DNMT3B
  - DNMT3L
  - ACSM3
  - TDG
  - TET1
  - TET2
  - TET3
---
