---
figid: PMC9118424__ao2c01816_0006
figtitle: Transcriptome and Metabolome Analysis of the Synthesis Pathways of Allelochemicals
  in Eupatorium adenophorum
organisms:
- Oryza sativa
- Zea mays
- Triticum aestivum
- Brassica napus
- Helianthus annuus
- Phaseolus vulgaris
- Oryza sativa Japonica Group
- Lolium perenne
- Lactuca sativa
- Fusarium solani
- Paeonia suffruticosa
- Verticillium longisporum
- Cynara cardunculus var. scolymus
- Datura stramonium
- Quercus suber
- Picrorhiza kurrooa
- Nothapodytes nimmoniana
- Euphorbia fischeriana
- Ageratina adenophora
- Quercus variabilis
- Olea europaea subsp. cuspidata
- Cyanotis arachnoidea
- Halimium halimifolium
- Osmunda japonica
- Tipuana tipu
organisms_ner:
- Oryza sativa Japonica Group
- Drosophila melanogaster
- Arabidopsis thaliana
- Homo sapiens
pmcid: PMC9118424
filename: ao2c01816_0006.jpg
figlink: /pmc/articles/PMC9118424/figure/fig6/
number: F6
caption: Analysis of DAMs and DEGs in E. adenophorum. (A–C) DAMs identified in the
  phenylpropanoid metabolic pathway, DAMs identified in the flavonoid metabolic pathway,
  and DAMs identified in the sesquiterpene metabolic pathway, respectively. (Red and
  blue circles indicate that the contents of metabolites in the stem metabolism group
  are up-regulated and down-regulated, respectively, compared to the content in the
  blade metabolism group (VIP > 1).) The circle size indicates the difference multiple
  (FC). (D) Protein interaction of differential genes in the metabolic pathways of
  allelochemicals. (E) Correlation network analysis among six differential sensing
  substances. Purple circles represent DAMs involved in phenylpropanoid metabolism,
  yellow circles represent DMAs involved in flavonoid metabolism, and green circles
  represent DMAs involved in sesquiterpene metabolism. The round and triangular borders
  represent the up- and down-regulation of metabolites, respectively. Red and blue
  lines represent positive (PPC > 0) and negative (PPC < 0) correlations of metabolites
  with genes. (F–H) Correlation network diagrams of phenylpropanoid metabolic pathway
  DAMs and DEGs, flavonoid metabolic pathway DAMs and DEGs, and sesquiterpene metabolic
  pathway DAMs and DEGs. Circles and pentagons represent genes and metabolites, respectively.
  Pink circles represent DEGs involved in phenylpropanoid metabolism, yellow circles
  represent DEGs involved in brass metabolism, and brown circles represent DEGs involved
  in sesquiterpene metabolism. Red and black borders indicate the up- and down-regulation
  of genes and metabolites, respectively. Red and black lines represent metabolites
  and genes that are positively correlated (PPC > 0) and negatively correlated (PPC
  < 0), respectively.
papertitle: Transcriptome and Metabolome Analysis of the Synthesis Pathways of Allelochemicals
  in Eupatorium adenophorum.
reftext: Wenwen Cheng, et al. ACS Omega. 2022 May 17;7(19):16803-16816.
year: '2022'
doi: 10.1021/acsomega.2c01816
journal_title: ACS Omega
journal_nlm_ta: ACS Omega
publisher_name: American Chemical Society
keywords: ''
automl_pathway: 0.6558199
figid_alias: PMC9118424__F6
figtype: Figure
redirect_from: /figures/PMC9118424__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9118424__ao2c01816_0006.html
  '@type': Dataset
  description: Analysis of DAMs and DEGs in E. adenophorum. (A–C) DAMs identified
    in the phenylpropanoid metabolic pathway, DAMs identified in the flavonoid metabolic
    pathway, and DAMs identified in the sesquiterpene metabolic pathway, respectively.
    (Red and blue circles indicate that the contents of metabolites in the stem metabolism
    group are up-regulated and down-regulated, respectively, compared to the content
    in the blade metabolism group (VIP > 1).) The circle size indicates the difference
    multiple (FC). (D) Protein interaction of differential genes in the metabolic
    pathways of allelochemicals. (E) Correlation network analysis among six differential
    sensing substances. Purple circles represent DAMs involved in phenylpropanoid
    metabolism, yellow circles represent DMAs involved in flavonoid metabolism, and
    green circles represent DMAs involved in sesquiterpene metabolism. The round and
    triangular borders represent the up- and down-regulation of metabolites, respectively.
    Red and blue lines represent positive (PPC > 0) and negative (PPC < 0) correlations
    of metabolites with genes. (F–H) Correlation network diagrams of phenylpropanoid
    metabolic pathway DAMs and DEGs, flavonoid metabolic pathway DAMs and DEGs, and
    sesquiterpene metabolic pathway DAMs and DEGs. Circles and pentagons represent
    genes and metabolites, respectively. Pink circles represent DEGs involved in phenylpropanoid
    metabolism, yellow circles represent DEGs involved in brass metabolism, and brown
    circles represent DEGs involved in sesquiterpene metabolism. Red and black borders
    indicate the up- and down-regulation of genes and metabolites, respectively. Red
    and black lines represent metabolites and genes that are positively correlated
    (PPC > 0) and negatively correlated (PPC < 0), respectively.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LOC4330407
  - LOC4341718
  - LOC4350636
  - LOC4345054
  - LOC4334588
  - Pal2
  - Rala
  - r-l
  - nes
  - alpha-Cat
  - Acat2
  - CG8112
  - Mvd
  - Hmgs
  - Pal1
  - sdt
  - CSN1b
  - chm
  - ppk
  - vvl
  - Tps1
  - cd
  - CAH1
  - Chi
  - chic
  - 4CL2
  - PAL2
  - TT4
  - CH1
  - F3H
  - HMGS
  - PAL1
  - MSH1
  - 4CL1
  - DFR
  - CYP72C1
  - CHI
  - TT5
  - TOMT
  - HPV18I2
  - RALA
  - RALB
  - RNASE13
  - LYST
  - SUCO
  - NES
  - FDPS
  - ACAT1
  - CES1
  - SOAT1
  - MVD
  - CHM
  - MPZ
---
