---
figid: PMC9061815__41422_2022_635_Fig5_HTML
figtitle: scATAC-seq analysis and heterogeneity of regulation between MGE and LGE
organisms:
- Homo sapiens
- Mus musculus
- Rattus norvegicus
- Macaca fascicularis
- Ficus racemosa
organisms_ner:
- Mus musculus
- Macaca fascicularis
- Homo sapiens
- Rattus norvegicus
- Drosophila melanogaster
pmcid: PMC9061815
filename: 41422_2022_635_Fig5_HTML.jpg
figlink: /pmc/articles/PMC9061815/figure/Fig5/
number: F5
caption: a Cell types in scATAC-seq visualized by UMAP after lift-over based on scRNA-seq
  data. b Accessible degrees of classic marker genes expressed in progenitors and
  different GE regions. c Differentially accessible chromatin peaks upstream of ZNF503
  (top) and LHX8 (bottom). d Gene regulatory network generated based on integration
  of scRNA-seq and scATAC-seq. e Hub genes of different GE regions deduced by protein–protein
  interactions. f Predicted regulatory pathway of MIR9-1HG. Upstream of MIR9-1HG were
  divided into three groups based on predicted binding site represented by different
  line colors. Dot colors share legends as in d. g Expression pattern of MIR9-1HG
  in human integrated data and its correlation with other genes in different cell
  clusters. False discovery rates were adjusted by BH method to generate “P. adjust”.
  h Open region of MIR9-1HG and predicted related TF binding sites. Specific peaks
  in MGE or progenitor clusters compared with LGE were marked (left) and predicted
  binding sites of upstream TFs were mapped to these regions (right).
papertitle: Evolutionarily conservative and non-conservative regulatory networks during
  primate interneuron development revealed by single-cell RNA and ATAC sequencing.
reftext: Ziqi Zhao, et al. Cell Res. 2022 May;32(5):425-436.
year: '2022'
doi: 10.1038/s41422-022-00635-9
journal_title: Cell Research
journal_nlm_ta: Cell Res
publisher_name: Springer Nature Singapore
keywords: Transdifferentiation | Genome-wide association studies
automl_pathway: 0.6344492
figid_alias: PMC9061815__F5
figtype: Figure
redirect_from: /figures/PMC9061815__F5
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9061815__41422_2022_635_Fig5_HTML.html
  '@type': Dataset
  description: a Cell types in scATAC-seq visualized by UMAP after lift-over based
    on scRNA-seq data. b Accessible degrees of classic marker genes expressed in progenitors
    and different GE regions. c Differentially accessible chromatin peaks upstream
    of ZNF503 (top) and LHX8 (bottom). d Gene regulatory network generated based on
    integration of scRNA-seq and scATAC-seq. e Hub genes of different GE regions deduced
    by protein–protein interactions. f Predicted regulatory pathway of MIR9-1HG. Upstream
    of MIR9-1HG were divided into three groups based on predicted binding site represented
    by different line colors. Dot colors share legends as in d. g Expression pattern
    of MIR9-1HG in human integrated data and its correlation with other genes in different
    cell clusters. False discovery rates were adjusted by BH method to generate “P.
    adjust”. h Open region of MIR9-1HG and predicted related TF binding sites. Specific
    peaks in MGE or progenitor clusters compared with LGE were marked (left) and predicted
    binding sites of upstream TFs were mapped to these regions (right).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Pls3
  - Pou3f2
  - Olig2
  - E2f1
  - pl
  - Tcf4
  - Tcf7l2
  - Dnmt1
  - Maf
  - Sox11
  - Arx
  - Uba2
  - Mef2c
  - Dlx2
  - Sp8
  - Muc2
  - Mmut
  - A1cf
  - Nfib
  - Nr2f2
  - Nr2f1
  - Lhx6
  - Mafb
  - Nkx2-1
  - Smc3
  - Sox2
  - Dlx1
  - Dlx5
  - Zic4
  - Sox4
  - Sst
  - Lmx1a
  - Tmsb10
  - Zfp704
  - Tmem123
  - Esrrg
  - Pbx1
  - Bcl11a
  - Actg1
  - Tshz2
  - Isl1
  - Zfhx4
  - Rhob
  - Meis2
  - Hes5
  - Rnd3
  - Jund
  - Pou3f3
  - Foxm1
  - Jun
  - soo
  - Col11a1
  - Kitl
  - us
  - Map6
  - Pipox
  - Son
  - Zfp219
  - Max
  - Nrf1
  - Nfe2l1
  - Nrxn3
  - cla
  - Ncoa5
  - Rhbg
  - Zfp503
  - Lhx8
  - St8sia4
  - Mir9-1
  - Ccdc88a
  - Mef2d
  - Igf2bp3
  - Creb5
  - Id4
  - Maz
  - Ascl1
  - PLS3
  - POU3F2
  - OLIG2
  - E2F1
  - TCF4
  - MAF
  - SOX11
  - ARX
  - MEF2C
  - DLX2
  - SP8
  - MMUT
  - NFIB
  - NR2F2
  - NR2F1
  - LHX6
  - MAFB
  - NKX2-1
  - SMC3
  - SOX2
  - DLX1
  - DLX5
  - ZIC4
  - SOX4
  - SST
  - TMSB10
  - ZNF704
  - TMEM123
  - ESRRG
  - PBX1
  - BCL11A
  - ACTG1
  - TSHZ2
  - ISL1
  - ZFHX4
  - RHOB
  - MEIS2
  - HES5
  - RND3
  - JUND
  - POU3F3
  - FOXM1
  - JUN
  - SON
  - ZNF219
  - MAX
  - TCF7L2
  - NRF1
  - NRXN3
  - RHBG
  - ZNF503
  - LHX8
  - ST8SIA4
  - CCDC88A
  - MEF2D
  - IGF2BP3
  - CREB5
  - ID4
  - MAZ
  - ASCL1
  - ZNF107
  - GUCY2D
  - UBE2L3
  - DNMT1
  - DNMT3A
  - DNMT3B
  - DNMT3L
  - MAFA
  - MAFF
  - MAFG
  - MAFK
  - MIR9-1HG
  - UBA2
  - PIK3CA
  - MCM2
  - MCM3
  - MCM4
  - MCM5
  - MCM6
  - MCM7
  - MCM8
  - MCM9
  - MCMDC2
  - NPM3
  - JUNB
  - MAP6
  - NFE2L1
  - SELPLG
  - ERVK-1
  - ASF1A
  - NCOA5
  - MIR9-1
  - PROS1
  - Plscr3
  - Mmd
  - Qsox1
  - Max-ps1
  - Pros1
  - pre
  - mge
  - Mt2
  - Nmt
  - maf-S
  - Maf1
  - arx
  - Mef2
  - CG5909
  - cl
  - ml
  - MAGE
  - exd
  - rho
  - Jra
  - ches
  - cerv
  - VhaAC39-1
  - COM
  - comt
  - compressed
  - lncRNA:flam
  - Con
  - ov
  - uni
  - cer
  - stop
  - sw
  - cu
  - jigr1
  - Me
  - ww
  - mof
  - ewg
  - Chc
  - cia
  - ci
  - pros
  - hv
  - maz
  - Pri
  - Arr2
  - tal-2A
  - tal-AA
  - tal-3A
  - tal-1A
---
