---
figid: PMC9374747__41598_2022_17964_Fig6_HTML
figtitle: In vivo transomic analyses of glucose-responsive metabolism in skeletal
  muscle reveal core differences between the healthy and obese states
organisms:
- Mus musculus
- Rattus norvegicus
- Cricetus cricetus
- Rattus rattus
- Rattus sp.
- Mus booduga
- Bos taurus
- Sus scrofa
- Felis catus
- Delphinus delphis
- Escherichia coli
- Escherichia coli SCD1
- Homo sapiens
- Ficus racemosa
- Celastrina ladon
organisms_ner:
- Mus musculus
- Sus scrofa
- Homo sapiens
- Rattus norvegicus
- Bos taurus
- Felis catus
- Rattus rattus
- Danio rerio
pmcid: PMC9374747
filename: 41598_2022_17964_Fig6_HTML.jpg
figlink: /pmc/articles/PMC9374747/figure/Fig6/
number: F6
caption: 'Condensed regulatory transomic networks for glucose-responsive metabolic
  reactions. (A,B) The condensed regulatory transomic network of the response to glucose
  in WT and ob/ob mice. The color of nodes (glucose-responsive molecules) and edges
  (interlayer regulatory connections) indicate the type of molecules and regulation
  as described in Fig. B. In the TF layer, the transcription factors encoded by glucose-responsive
  genes are italicized, and those showing glucose-responsive phosphorylation have
  the prefix “p.” The Enzyme layer contained only the metabolic enzymes that were
  regulated by glucose-responsive changes in gene expression, and not those regulated
  only by phosphorylation. Two types of metabolic pathway nodes were included: the
  pathway that exhibited significant associations with any glucose-responsive molecule
  (Fig. B); and the pathway whose percentage of regulated reactions was in the top
  10% by either glucose-responsive metabolites or by glucose-responsive genes encoding
  metabolic enzymes. Dashed boxes enclose the nodes for the lipid, carbohydrate, and
  amino acid classes. Glucose-responsive metabolites that exhibited significant associations
  with any metabolic pathway were included. Glucose-responsive transcription factors
  that regulated five or more metabolic reactions were included. The interlayer regulatory
  connections from the Metabolite to the Pathway layer included only those that regulate
  five or more metabolic reactions. The size of the nodes and width of the edges indicate
  the relative number of the regulated metabolic reactions. See Data File  for the
  unabbreviated names of the metabolic pathway nodes.'
papertitle: In vivo transomic analyses of glucose-responsive metabolism in skeletal
  muscle reveal core differences between the healthy and obese states.
reftext: Toshiya Kokaji, et al. Sci Rep. 2022;12:13719.
year: '2022'
doi: 10.1038/s41598-022-17964-9
journal_title: Scientific Reports
journal_nlm_ta: Sci Rep
publisher_name: Nature Publishing Group UK
keywords: Metabolism | Biochemical reaction networks | Computational biology and bioinformatics
  | Physiology
automl_pathway: 0.9335699
figid_alias: PMC9374747__F6
figtype: Figure
redirect_from: /figures/PMC9374747__F6
ndex: ''
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC9374747__41598_2022_17964_Fig6_HTML.html
  '@type': Dataset
  description: 'Condensed regulatory transomic networks for glucose-responsive metabolic
    reactions. (A,B) The condensed regulatory transomic network of the response to
    glucose in WT and ob/ob mice. The color of nodes (glucose-responsive molecules)
    and edges (interlayer regulatory connections) indicate the type of molecules and
    regulation as described in Fig. B. In the TF layer, the transcription factors
    encoded by glucose-responsive genes are italicized, and those showing glucose-responsive
    phosphorylation have the prefix “p.” The Enzyme layer contained only the metabolic
    enzymes that were regulated by glucose-responsive changes in gene expression,
    and not those regulated only by phosphorylation. Two types of metabolic pathway
    nodes were included: the pathway that exhibited significant associations with
    any glucose-responsive molecule (Fig. B); and the pathway whose percentage of
    regulated reactions was in the top 10% by either glucose-responsive metabolites
    or by glucose-responsive genes encoding metabolic enzymes. Dashed boxes enclose
    the nodes for the lipid, carbohydrate, and amino acid classes. Glucose-responsive
    metabolites that exhibited significant associations with any metabolic pathway
    were included. Glucose-responsive transcription factors that regulated five or
    more metabolic reactions were included. The interlayer regulatory connections
    from the Metabolite to the Pathway layer included only those that regulate five
    or more metabolic reactions. The size of the nodes and width of the edges indicate
    the relative number of the regulated metabolic reactions. See Data File  for the
    unabbreviated names of the metabolic pathway nodes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Abl2
  - Rere
  - Albfm1
  - Myog
  - Klf4
  - Kif4
  - Ppara
  - Etv4
  - Rfx1
  - Hmga1b
  - Hmga1
  - Hoxc6
  - Pbx3
  - Pknox1
  - Prep
  - Ptpn22
  - Fndc5
  - Rela
  - Gnat2
  - Gli2
  - Glis2
  - Zbtb16
  - Homez
  - fa
  - Egr1
  - Sp100
  - Rreb1
  - TF
  - MYOG
  - KLF4
  - PPARA
  - ETV4
  - RFX1
  - ACID
  - HOXC6
  - PBX3
  - PKNOX1
  - RELA
  - GLI2
  - GLIS2
  - ZBTB16
  - HOMEZ
  - EGR-1
  - EGR1
  - RREB1
  - ABL2
  - RERE
  - KIF4A
  - PAEP
  - PREP
  - PTPN22
  - ATP8A2
  - SP100
  - Tf
  - Kif4a
  - Ccne1
  - BCL2L2
  - myog
  - klf4
  - klf17
  - kif4
  - pparab
  - etv4
  - hmga1b
  - hoxc6a
  - rela
  - gli2a
  - zbtb16a
  - homeza
  - egr1
---
