<!DOCTYPE html>
<html lang="en"><head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="apple-touch-icon" sizes="180x180" href="/assets/img/apple-touch-icon.png">
    <link rel="android-chrome" sizes="192x192" href="/assets/img/android-chrome-192x192.png">
    <link rel="android-chrome" sizes="512x512" href="/assets/img/android-chrome-512x512.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicon-16x16.png">
    <link rel="manifest" href="/assets/img/site.webmanifest">
    <link rel="shortcut icon" href="/assets/img/favicon.ico">
    <link href="/assets/fontawesome.all.min.css" rel="stylesheet">
    <!-- Bootstrap -->
      <script src="https://code.jquery.com/jquery-3.5.1.js" type="text/javascript"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
    <!-- Custom style overrides-->
    <link rel="stylesheet" href="/assets/main.css">
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-KGQFS041KG"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-KGQFS041KG');
    </script> 
    <!-- Hotjar Tracking Code for Pathway Figure OCR -->
    <script>
    (function(h,o,t,j,a,r){
      h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
      h._hjSettings={hjid:3415747,hjsv:6};
      a=o.getElementsByTagName('head')[0];
      r=o.createElement('script');r.async=1;
      r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
      a.appendChild(r);
    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
    </script>
    
      <!-- Metadata -->
      <meta name="organisms" content="Arabidopsis thaliana, Zea mays, Triticum aestivum, Hordeum vulgare, Brachypodium distachyon, Triticum monococcum, and Homo sapiens" />
      <meta name="citation_id" content="PMC2856676__pone.0010065.g001">
      <meta name="citation_title" content="Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon as a Model for the Temperate Grasses">
      <meta name="citation_abstract" content="Arrows show promoting effects, T-bars show repressing effects. Environmental cues are shown as lower case letters in square boxes; v is extended cold (vernalization); ld is long days; sd is short days; am is ambient (non-vernalizing) temperature. Genes are shown in italics and proteins in non-italics in ovals. Proteins are clearly involved at all stages but we make this distinction only in cases where separate controls are known to exist for transcription and protein function or stability. # indicates inhibition of protein function. The pathways were compiled from the following reviews , , , , , , , , , , , , , , , ,  plus individual articles referenced in the text. Box A. In Arabidopsis genes assigned to pathways are shown in colour (photoperiod pathway in green, vernalization pathway in blue, autonomous pathway in light blue). Flowering pathway integrators are shown in red. Triple headed arrows indicate activation by red or blue light. Box B. Rice genes identified as homologues of those in Arabidopsis are shown in black. Ambiguous homology is indicated with a question mark. Genes with published roles in photoperiod regulated flowering are shown in green and orthologues of pathway integrators in red. Box C. In the temperate grasses box Brachypodium genes identified as homologues of those of Arabidopsis or rice are shown in black. The VRN2 and CO3 genes from barley and wheat are also included. Genes with published roles in photoperiod or vernalization regulated flowering in barley or wheat are shown in green or blue, respectively, and orthologues of pathway integrators in red. In B and C likely orthologues of Arabidopsis genes are shown in equivalent positions within the figure (e.g. Hd1 is the rice orthologue of CO and Hd3a and RFT1 are rice orthologues of FT).  and  gives information on gene names and synonyms. Square brackets in box B show three maize genes with known flowering time effects. Notes: *1 TOC1 represses PRR9, PRR7 and PRR5 . *2 SUF4 interacts with FRI and FRL1. LD blocks SUF4 action to reduce FLC expression . *3 Cerdan and Chory  identified PFT1 and proposed that it promotes flowering by a photoperiod independent pathway activating FT. The model here follows Wollenberg et al.  who propose an alternative in which PFT1 acts by repressing Pfr phytochrome signalling. *4 SWR1 is a protein complex including SWC6, ARP6 and PIE1. *5 CO expression from phloem-specific promoters complements the co mutation, but expression from meristem-specific promoters does not. CO triggers flowering from the phloem by the cell-autonomous activation of FT expression . *6 High temperatures (27°C vs 23°C) in short days induce flowering by a mechanism that requires FT and gibberellic acid and likely involves MAF1 (FLM). High temperature induction of flowering is independent of the photoperiod pathway but is repressed by FLC and by autonomous pathway mutations that increase FLC levels . Kumar and Wigge  showed that ambient temperature responses, including the activation of FT, involve the replacement of histone H2A.Z with H2A in nucleosomes. Mutation of ARP6 (part of the SWR1 complex) blocks this process. The late-flowering phenotype of dominant fwa mutants is caused by hypomethylation in the FWA locus leading to ectopic expression of a homeodomain leucine zipper (HD-ZIP) protein. The FWA protein interacts with FT protein and so ectopic expression of FWA delays flowering by impairing FT function . FWA is not expressed during the vegetative phase in wild type plants and normally has no role in flowering so homologues are not considered in this paper. *7 TWIN SISTER OF FT (TSF) is closely related to FT, is activated by CO, has a similar role in flowering and is repressed by SVP (; reviewed by , ). *8 Activation of AP1 and LFY requires two paralogous homeobox genes (PENNYWISE (PNY; aka BELLRINGER and REPLUMLESS) and POUND-FOOLISH (PNF)). Double null mutants express SOC1 and FUL but not AP1 or LFY. The double mutants can receive floral inductive signals but do not properly restructure the shoot apical meristem to make flowers , . *9 GI promotion of Hd1/CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *10 Mayfield et al.  found that 14-3-3 proteins physically interact with CO in Arabidopsis. *11 It is unclear if there is a direct connection between OsMADS50 and OsMADS14/15 as well as the connection via Edh1. *12 Ppd-H1 promotion of CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *13 Downregulation of VRT2 by vernalization and a role in repressing VRN1 is proposed by Kane et al. ,  but is not supported by Trevaskis et al.  so VRT2 is shown in two positions. *14 Shitsukawa et al.  found that WSOC1 was expressed preferentially in leaves. Expression was not affected by photoperiod or vernalization but was upregulated in seedlings by a GA treatment. *15 X indicates an unknown target gene . Key to figures of phylogenetic trees A PRANK alignment of protein sequences (shown in the supplementary datasets) was used to construct phylogenetic trees using the neighbor-joining algorithm (subsequently NJ trees) with bootstrap support from 1000 iterations (values of ≥50% shown). Gene identifiers show the species as follows: At for Arabidopsis thaliana, Bradi for Brachypodium distachyon, Os for Orzya sativa (rice), Hv for Hordeum vulgare (barley), Ta for Triticum aestivum (bread wheat), Tm for Triticum monococcum (einkorn wheat), Sb for Sorghum bicolour (sorghum) and Zm for Zea mays (maize). Barley and wheat proteins are highly homologous and usually only one is shown. Gene names follow the identifiers where relevant to the text and genes with known flowering time roles are shown in colour following . In several instances we identified pairs of genes in grasses (paralogues) in segmental duplications (SegDup) which likely result from a grass whole genome duplication that predates the divergence of the rice, maize, sorghum, Brachypodium, barley and wheat lineages. These duplication events are shown as solid circles. Similarly, solid diamonds showduplication events in Arabidopsis; these are independent from the duplication event in the grasses. Boxed T shows tandem or closely arranged paralogues resulting from duplications within individual species. Where the protein belongs to a large family we show small clades containing genes of interest selected from analyses of whole families. The clades were identified either by using an online interrogatory tree of the transcription factor family using known flowering-time genes as query sequences  or from the family trees shown in the supplementary material. Key to figures of segmental duplications In these figures the size and spacing of genes in the respective genomic sequences is shown to the physical scale indicated in each figure. The components of each half of the duplication are grouped under the bars at the top of the figure. For example,  shows part of rice chromosome 3 which is colinear with part of Brachypodium chromosome 1 and is related to part of rice chromosome 7 by segmental duplication. This section of rice 7 is colinear with a separate section of Brachypodium 1. Identifiers and names of flowering-time genes and their homologues are shown, with colours following . Gene identifiers are also shown at the top and bottom of each segment to allow orientation to the respective genome browsers. The segmental duplications are recognized by conserved genes (black boxes connected by black lines). Some figures have grey boxes which show genes that are not conserved across the duplication in all species. Within these regions, other genes are conserved between species but not across the duplication (red boxes connected by red lines). These are deletion/insertion events that occurred after the duplication arose but before the divergence of the respective species. Genes in yellow are not conserved across species and represent candidates for deletion/insertion events that occurred after species divergence. Hypothetical genes and sequences annotated as transposons are excluded. In most cases the segmental duplications extend further than shown as we only use those portions that are relevant to the relationships of the flowering-time genes discussed in the text.">
      <meta name="citation_year" content="2010">
      <meta name="citation_fulltext_html_url" content="https://pfocr.wikipathways.org/figures/PMC2856676__pone.0010065.g001">
      <meta name="citation_publisher" content="WikiPathways">
      <!-- Scripts for figure pages -->
      <script type="module" src="/assets/js/clipboardtext.js" defer="defer"></script>
      <script type="application/ld+json">
        {"@context":"https://schema.org/","@id":"https://pfocr.wikipathways.org/figures/PMC2856676__pone.0010065.g001.html","@type":"Dataset","description":"Arrows show promoting effects, T-bars show repressing effects. Environmental cues are shown as lower case letters in square boxes; v is extended cold (vernalization); ld is long days; sd is short days; am is ambient (non-vernalizing) temperature. Genes are shown in italics and proteins in non-italics in ovals. Proteins are clearly involved at all stages but we make this distinction only in cases where separate controls are known to exist for transcription and protein function or stability. # indicates inhibition of protein function. The pathways were compiled from the following reviews , , , , , , , , , , , , , , , ,  plus individual articles referenced in the text. Box A. In Arabidopsis genes assigned to pathways are shown in colour (photoperiod pathway in green, vernalization pathway in blue, autonomous pathway in light blue). Flowering pathway integrators are shown in red. Triple headed arrows indicate activation by red or blue light. Box B. Rice genes identified as homologues of those in Arabidopsis are shown in black. Ambiguous homology is indicated with a question mark. Genes with published roles in photoperiod regulated flowering are shown in green and orthologues of pathway integrators in red. Box C. In the temperate grasses box Brachypodium genes identified as homologues of those of Arabidopsis or rice are shown in black. The VRN2 and CO3 genes from barley and wheat are also included. Genes with published roles in photoperiod or vernalization regulated flowering in barley or wheat are shown in green or blue, respectively, and orthologues of pathway integrators in red. In B and C likely orthologues of Arabidopsis genes are shown in equivalent positions within the figure (e.g. Hd1 is the rice orthologue of CO and Hd3a and RFT1 are rice orthologues of FT).  and  gives information on gene names and synonyms. Square brackets in box B show three maize genes with known flowering time effects. Notes: *1 TOC1 represses PRR9, PRR7 and PRR5 . *2 SUF4 interacts with FRI and FRL1. LD blocks SUF4 action to reduce FLC expression . *3 Cerdan and Chory  identified PFT1 and proposed that it promotes flowering by a photoperiod independent pathway activating FT. The model here follows Wollenberg et al.  who propose an alternative in which PFT1 acts by repressing Pfr phytochrome signalling. *4 SWR1 is a protein complex including SWC6, ARP6 and PIE1. *5 CO expression from phloem-specific promoters complements the co mutation, but expression from meristem-specific promoters does not. CO triggers flowering from the phloem by the cell-autonomous activation of FT expression . *6 High temperatures (27°C vs 23°C) in short days induce flowering by a mechanism that requires FT and gibberellic acid and likely involves MAF1 (FLM). High temperature induction of flowering is independent of the photoperiod pathway but is repressed by FLC and by autonomous pathway mutations that increase FLC levels . Kumar and Wigge  showed that ambient temperature responses, including the activation of FT, involve the replacement of histone H2A.Z with H2A in nucleosomes. Mutation of ARP6 (part of the SWR1 complex) blocks this process. The late-flowering phenotype of dominant fwa mutants is caused by hypomethylation in the FWA locus leading to ectopic expression of a homeodomain leucine zipper (HD-ZIP) protein. The FWA protein interacts with FT protein and so ectopic expression of FWA delays flowering by impairing FT function . FWA is not expressed during the vegetative phase in wild type plants and normally has no role in flowering so homologues are not considered in this paper. *7 TWIN SISTER OF FT (TSF) is closely related to FT, is activated by CO, has a similar role in flowering and is repressed by SVP (; reviewed by , ). *8 Activation of AP1 and LFY requires two paralogous homeobox genes (PENNYWISE (PNY; aka BELLRINGER and REPLUMLESS) and POUND-FOOLISH (PNF)). Double null mutants express SOC1 and FUL but not AP1 or LFY. The double mutants can receive floral inductive signals but do not properly restructure the shoot apical meristem to make flowers , . *9 GI promotion of Hd1/CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *10 Mayfield et al.  found that 14-3-3 proteins physically interact with CO in Arabidopsis. *11 It is unclear if there is a direct connection between OsMADS50 and OsMADS14/15 as well as the connection via Edh1. *12 Ppd-H1 promotion of CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *13 Downregulation of VRT2 by vernalization and a role in repressing VRN1 is proposed by Kane et al. ,  but is not supported by Trevaskis et al.  so VRT2 is shown in two positions. *14 Shitsukawa et al.  found that WSOC1 was expressed preferentially in leaves. Expression was not affected by photoperiod or vernalization but was upregulated in seedlings by a GA treatment. *15 X indicates an unknown target gene . Key to figures of phylogenetic trees A PRANK alignment of protein sequences (shown in the supplementary datasets) was used to construct phylogenetic trees using the neighbor-joining algorithm (subsequently NJ trees) with bootstrap support from 1000 iterations (values of ≥50% shown). Gene identifiers show the species as follows: At for Arabidopsis thaliana, Bradi for Brachypodium distachyon, Os for Orzya sativa (rice), Hv for Hordeum vulgare (barley), Ta for Triticum aestivum (bread wheat), Tm for Triticum monococcum (einkorn wheat), Sb for Sorghum bicolour (sorghum) and Zm for Zea mays (maize). Barley and wheat proteins are highly homologous and usually only one is shown. Gene names follow the identifiers where relevant to the text and genes with known flowering time roles are shown in colour following . In several instances we identified pairs of genes in grasses (paralogues) in segmental duplications (SegDup) which likely result from a grass whole genome duplication that predates the divergence of the rice, maize, sorghum, Brachypodium, barley and wheat lineages. These duplication events are shown as solid circles. Similarly, solid diamonds showduplication events in Arabidopsis; these are independent from the duplication event in the grasses. Boxed T shows tandem or closely arranged paralogues resulting from duplications within individual species. Where the protein belongs to a large family we show small clades containing genes of interest selected from analyses of whole families. The clades were identified either by using an online interrogatory tree of the transcription factor family using known flowering-time genes as query sequences  or from the family trees shown in the supplementary material. Key to figures of segmental duplications In these figures the size and spacing of genes in the respective genomic sequences is shown to the physical scale indicated in each figure. The components of each half of the duplication are grouped under the bars at the top of the figure. For example,  shows part of rice chromosome 3 which is colinear with part of Brachypodium chromosome 1 and is related to part of rice chromosome 7 by segmental duplication. This section of rice 7 is colinear with a separate section of Brachypodium 1. Identifiers and names of flowering-time genes and their homologues are shown, with colours following . Gene identifiers are also shown at the top and bottom of each segment to allow orientation to the respective genome browsers. The segmental duplications are recognized by conserved genes (black boxes connected by black lines). Some figures have grey boxes which show genes that are not conserved across the duplication in all species. Within these regions, other genes are conserved between species but not across the duplication (red boxes connected by red lines). These are deletion/insertion events that occurred after the duplication arose but before the divergence of the respective species. Genes in yellow are not conserved across species and represent candidates for deletion/insertion events that occurred after species divergence. Hypothetical genes and sequences annotated as transposons are excluded. In most cases the segmental duplications extend further than shown as we only use those portions that are relevant to the relationships of the flowering-time genes discussed in the text.","license":"CC0","name":"CreativeWork","creator":{"@type":"Organization","name":"WikiPathways"},"keywords":["PRR7","PRR9","PRR5","HD2B","PPD1","LHY","CCA1","ELF4","ZTL","PRR3","GI","FVE","FPA","FLK","LD","FLD","FY","FCA","PCL1","CLF","FIE","MSI1","SWN","VEL1","TCP11","VRN5","TOC1","CRY1","VIN3","VRN1","VRN2","COP1","ELF3","CRY2","FKF1","LKP2","CDF1","FRI","FRL1","SMT2","FRL2","FES1","CDF2","COL9","CDF3","LTP2","SUF4","EFS","MIR172b","MIR172a","HD1","AGL41","PHYB","SPA1","SPA3","SPA4","PEP","PAS2","HUA2","FLC","TFL2","SMZ","RAP2.7","PHYA","MAF1","TEM1","RAV2","CCC1","EHD1","AGL19","AGL24","CIB1","SVP","AGL20","FT","FD","FT1","ELF9","SPL3","AGL8","AGL17","AP1","TFL1","LFY","C1orf210","HDAC2","DLL4","GLI1","TRNT1","NECTIN3","GNAO1","GNAI1","GNAI2","GNAI3","GNAT1","GNAT2","GNAT3","GNAZ","CRLF1","CRNKL1","CLQTL1","ACKR2","DEFA6","HDAC6","VIL1","CARD16","FMNL3","HSPBP1","PCLAF","HDAC1","PLEC","ID1","SIPA1","SFTPA1","CD5L","SFTPA2","PAEP","PREP","PTPN22","LARP1","TOE1","SIRT2","SIRT1","CD248","RASD2","NFYC","AGL","TSC1","TSC2","SRCAP","AKTIP","RFT1","SLC19A1","SLC52A1","FOS","FOSB","JUN","JUNB","JUND","FOSL1","FOSL2","APEX1","LOC542075","LOC541825","LOC541693","LOC542393","LOC541767","LOC542663","LOC542042","LOC542464","LOC543096","LOC543103","LOC543351","LOC543320","LOC542840","LOC780635","LOC543392","LOC780649"]}
      </script>
    
    <title>
      
        Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon as a Model for the Temperate Grasses
      
    </title>
  </head> 

  <body><header class="site-header" role="banner">
    <div class="wrapper">
      <!-- NAV BAR -->
      <nav class="navbar navbar-light">
        <a class="navbar-brand" href="/">
          <img src="/assets/img/pfocr_logo_500.png" height="45" class="d-inline-block align-top" alt="PFOCR logo"> Pathway Figure OCR
        </a>     
        <div>
        <!-- SEARCH  -->
        <div id="search-widget" style="float:left; margin:4px 20px 0px 0px;">
          <form id="searchForm" action="/search.html">
            <i class="fa fa-search" aria-hidden="true"></i>
            <input type="text" name="searchQuery" placeholder="&nbsp;e.g., ace2 aldosterone human">
          </form>          
        </div>
        <script>
          const searchForm = document.getElementById('searchForm');
          searchForm.addEventListener('submit', function(event) {
          event.preventDefault(); // Prevent the default form submission behavior
          const searchQuery = encodeURIComponent(searchForm.elements.searchQuery.value); // Get the search query and encode it
          const searchUrl = `/search.html?query=${searchQuery}`; // Construct the search results page URL with the search query as a parameter
          window.location.href = searchUrl; // Navigate to the search results page
          });
        </script>
        <!-- MENU -->         
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        </div>
        <div class="collapse navbar-collapse " id="navbarNavAltMarkup">
          <div class="nav justify-content-center">
            <a class="nav-item nav-link" href="/about.html">About</a>
            <a class="nav-item nav-link" href="/search.html">Search</a>
            <a class="nav-item nav-link" href="/#browse">Browse</a>
            <a class="nav-item nav-link" href="/#download">Download</a>
            <a class="nav-item nav-link" href="/#download">Analyze</a>
            <a class="nav-item nav-link" href="/help.html">Help</a>
          </div>
        </div>
      </nav>
    </div>
  </header>    
  
    <main class="page-content" aria-label="Content">
      <div class="wrapper">
          <a class="page-title" href="http://localhost:4000/figures/PMC2856676__pone.0010065.g001.html">
    <h1 id="title">Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon as a Model for the Temperate Grasses
      <a href="https://github.com/wikipathways/pfocr-database/edit/main/_figures/PMC2856676__pone.0010065.g001.md"
      target="_blank">
        <span title="Edit title and caption" style="color: #666; font-size: 0.8em;">
          <i class="fa fa-pencil"></i>
        </span>
      </a></h1>
  </a>
  <div id="caption" style="max-height:75px; overflow-y:hidden;margin-bottom: 20px;">
    Arrows show promoting effects, T-bars show repressing effects. Environmental cues are shown as lower case letters in square boxes; v is extended cold (vernalization); ld is long days; sd is short days; am is ambient (non-vernalizing) temperature. Genes are shown in italics and proteins in non-italics in ovals. Proteins are clearly involved at all stages but we make this distinction only in cases where separate controls are known to exist for transcription and protein function or stability. # indicates inhibition of protein function. The pathways were compiled from the following reviews , , , , , , , , , , , , , , , ,  plus individual articles referenced in the text. Box A. In Arabidopsis genes assigned to pathways are shown in colour (photoperiod pathway in green, vernalization pathway in blue, autonomous pathway in light blue). Flowering pathway integrators are shown in red. Triple headed arrows indicate activation by red or blue light. Box B. Rice genes identified as homologues of those in Arabidopsis are shown in black. Ambiguous homology is indicated with a question mark. Genes with published roles in photoperiod regulated flowering are shown in green and orthologues of pathway integrators in red. Box C. In the temperate grasses box Brachypodium genes identified as homologues of those of Arabidopsis or rice are shown in black. The VRN2 and CO3 genes from barley and wheat are also included. Genes with published roles in photoperiod or vernalization regulated flowering in barley or wheat are shown in green or blue, respectively, and orthologues of pathway integrators in red. In B and C likely orthologues of Arabidopsis genes are shown in equivalent positions within the figure (e.g. Hd1 is the rice orthologue of CO and Hd3a and RFT1 are rice orthologues of FT).  and  gives information on gene names and synonyms. Square brackets in box B show three maize genes with known flowering time effects. Notes: *1 TOC1 represses PRR9, PRR7 and PRR5 . *2 SUF4 interacts with FRI and FRL1. LD blocks SUF4 action to reduce FLC expression . *3 Cerdan and Chory  identified PFT1 and proposed that it promotes flowering by a photoperiod independent pathway activating FT. The model here follows Wollenberg et al.  who propose an alternative in which PFT1 acts by repressing Pfr phytochrome signalling. *4 SWR1 is a protein complex including SWC6, ARP6 and PIE1. *5 CO expression from phloem-specific promoters complements the co mutation, but expression from meristem-specific promoters does not. CO triggers flowering from the phloem by the cell-autonomous activation of FT expression . *6 High temperatures (27°C vs 23°C) in short days induce flowering by a mechanism that requires FT and gibberellic acid and likely involves MAF1 (FLM). High temperature induction of flowering is independent of the photoperiod pathway but is repressed by FLC and by autonomous pathway mutations that increase FLC levels . Kumar and Wigge  showed that ambient temperature responses, including the activation of FT, involve the replacement of histone H2A.Z with H2A in nucleosomes. Mutation of ARP6 (part of the SWR1 complex) blocks this process. The late-flowering phenotype of dominant fwa mutants is caused by hypomethylation in the FWA locus leading to ectopic expression of a homeodomain leucine zipper (HD-ZIP) protein. The FWA protein interacts with FT protein and so ectopic expression of FWA delays flowering by impairing FT function . FWA is not expressed during the vegetative phase in wild type plants and normally has no role in flowering so homologues are not considered in this paper. *7 TWIN SISTER OF FT (TSF) is closely related to FT, is activated by CO, has a similar role in flowering and is repressed by SVP (; reviewed by , ). *8 Activation of AP1 and LFY requires two paralogous homeobox genes (PENNYWISE (PNY; aka BELLRINGER and REPLUMLESS) and POUND-FOOLISH (PNF)). Double null mutants express SOC1 and FUL but not AP1 or LFY. The double mutants can receive floral inductive signals but do not properly restructure the shoot apical meristem to make flowers , . *9 GI promotion of Hd1/CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *10 Mayfield et al.  found that 14-3-3 proteins physically interact with CO in Arabidopsis. *11 It is unclear if there is a direct connection between OsMADS50 and OsMADS14/15 as well as the connection via Edh1. *12 Ppd-H1 promotion of CO is shown directly but could act through the FKF1/CDF route as in Arabidopsis. *13 Downregulation of VRT2 by vernalization and a role in repressing VRN1 is proposed by Kane et al. ,  but is not supported by Trevaskis et al.  so VRT2 is shown in two positions. *14 Shitsukawa et al.  found that WSOC1 was expressed preferentially in leaves. Expression was not affected by photoperiod or vernalization but was upregulated in seedlings by a GA treatment. *15 X indicates an unknown target gene . Key to figures of phylogenetic trees A PRANK alignment of protein sequences (shown in the supplementary datasets) was used to construct phylogenetic trees using the neighbor-joining algorithm (subsequently NJ trees) with bootstrap support from 1000 iterations (values of ≥50% shown). Gene identifiers show the species as follows: At for Arabidopsis thaliana, Bradi for Brachypodium distachyon, Os for Orzya sativa (rice), Hv for Hordeum vulgare (barley), Ta for Triticum aestivum (bread wheat), Tm for Triticum monococcum (einkorn wheat), Sb for Sorghum bicolour (sorghum) and Zm for Zea mays (maize). Barley and wheat proteins are highly homologous and usually only one is shown. Gene names follow the identifiers where relevant to the text and genes with known flowering time roles are shown in colour following . In several instances we identified pairs of genes in grasses (paralogues) in segmental duplications (SegDup) which likely result from a grass whole genome duplication that predates the divergence of the rice, maize, sorghum, Brachypodium, barley and wheat lineages. These duplication events are shown as solid circles. Similarly, solid diamonds showduplication events in Arabidopsis; these are independent from the duplication event in the grasses. Boxed T shows tandem or closely arranged paralogues resulting from duplications within individual species. Where the protein belongs to a large family we show small clades containing genes of interest selected from analyses of whole families. The clades were identified either by using an online interrogatory tree of the transcription factor family using known flowering-time genes as query sequences  or from the family trees shown in the supplementary material. Key to figures of segmental duplications In these figures the size and spacing of genes in the respective genomic sequences is shown to the physical scale indicated in each figure. The components of each half of the duplication are grouped under the bars at the top of the figure. For example,  shows part of rice chromosome 3 which is colinear with part of Brachypodium chromosome 1 and is related to part of rice chromosome 7 by segmental duplication. This section of rice 7 is colinear with a separate section of Brachypodium 1. Identifiers and names of flowering-time genes and their homologues are shown, with colours following . Gene identifiers are also shown at the top and bottom of each segment to allow orientation to the respective genome browsers. The segmental duplications are recognized by conserved genes (black boxes connected by black lines). Some figures have grey boxes which show genes that are not conserved across the duplication in all species. Within these regions, other genes are conserved between species but not across the duplication (red boxes connected by red lines). These are deletion/insertion events that occurred after the duplication arose but before the divergence of the respective species. Genes in yellow are not conserved across species and represent candidates for deletion/insertion events that occurred after species divergence. Hypothetical genes and sequences annotated as transposons are excluded. In most cases the segmental duplications extend further than shown as we only use those portions that are relevant to the relationships of the flowering-time genes discussed in the text.
  </div>
  <input type="button" class="badge badge-secondary" id="descMore" value="more text" style="float: right; margin-top: -26px;">
  <input type="button" class="badge badge-secondary" id="descLess" value="less text" style="float: right; margin-top: -26px;">

  <div id="pathway-figure">
    <img src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856676/bin/pone.0010065.g001.jpg" alt="Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon as a Model for the Temperate Grasses" 
    style="max-height:600px;"/>
  </div>
  <!-- DOWNLOAD MENU -->
  <div class="d-flex">
    <div class="dropdown">
      <button class="dropdown-toggle" style="margin-right: -15px;background-color: transparent;color: transparent;border: none;cursor: pointer;" 
      type="button" id="dropdownMenuButton" data-toggle="dropdown"
      aria-haspopup="true" aria-expanded="false">
      <span title="Download figure as..." style="color: #FF8120; background-color: #FFFFFF; font-size: 1.3em;">
        <i class="fa fa-download"></i>
      </span>
      </button>
      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
      <a class="dropdown-item" style="color: #000000;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856676/bin/pone.0010065.g001.jpg" target="_blank">Download JPG</a>
      <a class="dropdown-item" style="color: #000000;" href="" target="_blank">Download GPML</a>
      </div>
    </div>
    <!-- SHARE MENU -->
    <div class="dropdown">
      <button class="dropdown-toggle" style="margin-right: -15px;background-color: transparent;color: transparent;border: none;cursor: pointer;" 
    type="button" id="dropdownMenuButton" data-toggle="dropdown"
    aria-haspopup="true" aria-expanded="false">
      <span title="Share pathway" style="color: #FF8120; background-color: #FFFFFF; font-size: 1.3em;">
        <i class="fa fa-share"></i>
      </span>
      </button> 
      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton2">
      <button class="dropdown-item copy-btn" type="button" to-copy="http://localhost:4000/figures/PMC2856676__pone.0010065.g001.html">Copy permalink</button>
      <a class="dropdown-item" style="color: #000000;" href="https://www.ncbi.nlm.nih.gov//pmc/articles/PMC2856676/figure/pone-0010065-g001/" target="_blank">View figure at PMC</a>
      <a class="dropdown-item" style="color: #000000;" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856676" target="_blank">Read paper at PMC</a>
      
      
        <a class="dropdown-item" style="color: #000000;" href="https://www.ndexbio.org/viewer/networks/1ec8baeb-df15-11ea-99da-0ac135e8bacf" target="_blank">Open at NDEx</a>
      
      </div> 
    </div>
    <!-- This badge is updated by Jekyll via the value of "pmcid" in each pathway .md-->
    <div style="padding: 4px;">
    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856676"
    target="_blank">
    <img alt="PMC"
    src="https://img.shields.io/static/v1?label=PMC&message=PMC2856676&color=blue">
    </a>
    </div>
    <!-- This badge is updated by Jekyll via the value of "ndex" in each pathway .md-->
    
    
    <div style="padding: 4px;">
    <a href="https://www.ndexbio.org/viewer/networks/1ec8baeb-df15-11ea-99da-0ac135e8bacf"
    target="_blank">
    <img alt="NDEx"
    src="https://img.shields.io/static/v1?label=NDEx&message=1ec8baeb...&color=brightgreen">
    </a>
    </div>
    
    <!-- This badge is updated by Jekyll via the value of "wpid" in each pathway .md-->
    
  </div>
  <div>
    <h2 id="publication">Publication</h2>
    Comparative Genomics of Flowering Time Pathways Using Brachypodium distachyon as a Model for the Temperate Grasses. (2010)
      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856676"
      target="_blank">Janet A. Higgins, et al. PLoS One. 2010;5(4):e10065. </a> Figure: F1.
    <!-- 
      <h2 id="organism">Organisms
      <a href="https://github.com/wikipathways/pfocr-database/edit/main/_figures/PMC2856676__pone.0010065.g001.md"
      target="_blank">
        <span title="Edit organims" style="color: #666; font-size: 0.8em;">
          <i class="fa fa-pencil"></i>
        </span>
      </a>
    </h2>
    
      
      
      
        <span>&nbsp;Arabidopsis thaliana&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Zea mays&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Triticum aestivum&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Hordeum vulgare&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Brachypodium distachyon&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Triticum monococcum&nbsp;</span>
      
    
      
      
      
        <span>&nbsp;Homo sapiens&nbsp;</span>
      
     
  -->

<!-- TABLES --> 




<div style="display: flex; align-items: baseline;">   
  <h2 id="participants">Gene mentions </h2>   
  <!-- download button -->
  <div style="padding:5px">
    <a href="https://raw.githubusercontent.com/wikipathways/wikipathways.github.io/main/_data/PMC2856676__pone.0010065.g001-genes.tsv"
      target="_blank">
        <span title="Download as TSV" style="color: #FF8120; font-size: 1.2em;">
          <i class="fa fa-download"></i>
        </span>
      </a>
  </div>
  <!-- share menu options -->
  <div class="dropdown">
    <button class="dropdown-toggle" style="padding: 4px;background-color: transparent;color: transparent;border: none;cursor: pointer;" 
      type="button" id="dropdownMenuButtonDatanodes" data-toggle="dropdown"
      aria-haspopup="true" aria-expanded="false">
      <span title="Copy to PathVisio" style="color: #FF8120; background-color: #FFFFFF; font-size: 1.2em;"> 
        <i class="fa fa-share"></i>
      </span>
    </button>
    <div class="dropdown-menu" aria-labelledby="dropdownMenuButtonDatanodes">
      
      <button title="Copy genes from selected organism as DataNodes for PathVisio"
      class="dropdown-item pvg-btn" type="button">Copy DataNodes for PathVisio</button>

      
    </div>
  </div>
</div>
<select id="organismSelect" onchange="filterTable()" style="margin-bottom: 10px;">
  
   
</select>
<br>
<input type="button" class="badge badge-secondary" id="tableLess1" value="fewer rows" style="float: left; margin-bottom: 2px;">
<div class="table-container">
  <table id="participants-table"> 
    <thead>
      <th title="Organism of lexicon that matched">Organism</th>
      <th title="Category of organism">Group</th>
      <th title="Word in figure as detected by OCR">Word</th>
      <th title="Transformed word that matched lexicon">Match</th>
      <th title="Source of lexicon that matched">Source</th>
      <th title="National Center for Biotechnology Information's Gene database">NCBI Symbol</th>
      <th title="National Center for Biotechnology Information's Gene database">NCBI ID</th>
    </thead>
    <tbody>
      
    </tbody>
  </table>
  <div id="tableGrad" class="table-gradient-overlay"></div>
</div>
<input type="button" class="badge badge-secondary" id="tableMore" value="more rows" style="float: left; margin: -8px 4px 0px 0px;">
<input type="button" class="badge badge-secondary" id="tableLess" value="fewer rows" style="float: left; margin-top: -8px;">

<br />
    <h2 id="chemicals">Chemical mentions
      
      <a href="https://raw.githubusercontent.com/wikipathways/wikipathways.github.io/main/_data/PMC2856676__pone.0010065.g001-chemicals.tsv"
      target="_blank">
        <span title="Download as TSV" style="color: #FF8120; font-size: 0.8em;">
          <i class="fa fa-download"></i>
        </span>
      </a>
      <!-- <a href="https://github.com/wikipathways/pfocr-database/edit/main/_figures/PMC2856676__pone.0010065.g001.md"
      target="_blank">
        <span title="Edit chemicals" style="color: #666; font-size: 0.8em;">
          <i class="fa fa-pencil"></i>
        </span>
      </a> -->
    </h2>
      <input type="button" class="badge badge-secondary" id="tableLessChems1" value="fewer rows" style="float: left; margin-bottom: 2px;">
      
      
      <!--  -->
      <div class="table-container">
      <table id="chems-table"> 
        <thead>
          <th title="Word in figure as detected by OCR">Word</th>
          <th title="Database maintaining identifiers">Datasource</th>
          <th title="Identifier representing word as maintained by datasource">Identifier</th>
        </thead>
        <tbody>
          
        </tbody>
      </table>
      <div id="tableGradChems" class="table-gradient-overlay"></div>
      </div> 
      <!-- <div class="dropdown" style="float:right; margin-top: -13px;">
        <button class="badge badge-secondary dropdown-toggle" type="button" id="dropdownMenuButtonChems" data-toggle="dropdown"
          aria-haspopup="true" aria-expanded="false">
          options
        </button>
        <div class="dropdown-menu" aria-labelledby="dropdownMenuButtonChems">
        <a class="dropdown-item" style="color: #000000;" href="https://raw.githubusercontent.com/wikipathways/pfocr-database/main/download/PMC2856676__pone.0010065.g001-chemicals.tsv">Download TSV</a>
        </div>
      </div> -->
      <input type="button" class="badge badge-secondary" id="tableMoreChems" value="more rows" style="float: left; margin-top: -8px;">
      <input type="button" class="badge badge-secondary" id="tableLessChems" value="fewer rows" style="float: left; margin-top: -8px;">

      <br />
      <h2 id="diseases">Disease mentions
        
        <a href="https://raw.githubusercontent.com/wikipathways/wikipathways.github.io/main/_data/PMC2856676__pone.0010065.g001-diseases.tsv"
        target="_blank">
          <span title="Download as TSV" style="color: #FF8120; font-size: 0.8em;">
            <i class="fa fa-download"></i>
          </span>
        </a>
        <!-- <a href="https://github.com/wikipathways/pfocr-database/edit/main/_figures/PMC2856676__pone.0010065.g001.md"
        target="_blank">
          <span title="Edit diseases" style="color: #666; font-size: 0.8em;">
            <i class="fa fa-pencil"></i>
          </span>
        </a> -->
      </h2>
        <input type="button" class="badge badge-secondary" id="tableLessDis1" value="fewer rows" style="float: left; margin-bottom: 2px;">
        
        
        <!--  -->
        <div class="table-container">
        <table id="dis-table"> 
          <thead>
            <th title="Word in figure as detected by OCR">Word</th>
            <th title="Database maintaining identifiers">Datasource</th>
            <th title="Identifier representing word as maintained by datasource">Identifier</th>
          </thead>
          <tbody>
            
          </tbody>
        </table>
        <div id="tableGradDis" class="table-gradient-overlay"></div>
      </div>
        <!-- <div class="dropdown" style="float:right; margin-top: -13px;">
          <button class="badge badge-secondary dropdown-toggle" type="button" id="dropdownMenuButtonDis" data-toggle="dropdown"
            aria-haspopup="true" aria-expanded="false">
            options
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButtonDis">
          <a class="dropdown-item" style="color: #000000;" href="https://raw.githubusercontent.com/wikipathways/pfocr-database/main/download/PMC2856676__pone.0010065.g001-diseases.tsv">Download TSV</a>
          </div>
        </div> -->
        <input type="button" class="badge badge-secondary" id="tableMoreDis" value="more rows" style="float: left; margin-top: -8px;">
        <input type="button" class="badge badge-secondary" id="tableLessDis" value="fewer rows" style="float: left; margin-top: -8px;">
  
  </div>

<script> 
  var trs = $("#participants-table tr");
  var trsChem = $("#chems-table tr");
  var trsDis = $("#dis-table tr");
  var btnMoreDesc = $("#descMore");
  var btnLessDesc = $("#descLess");
  var btnMore = $("#tableMore");
  var btnLess = $("#tableLess");
  var btnLess1 = $("#tableLess1");
  const tblGrad = $("#tableGrad")[0];
  var btnMoreChems = $("#tableMoreChems");
  var btnLessChems = $("#tableLessChems");
  var btnLessChems1 = $("#tableLessChems1");
  const tblGradChems = $("#tableGradChems")[0];
  var btnMoreDis = $("#tableMoreDis");
  var btnLessDis = $("#tableLessDis");
  var btnLessDis1 = $("#tableLessDis1");
  const tblGradDis = $("#tableGradDis")[0];
  var trsLength = trs.length;
  var trsLenChems = trsChem.length;
  var trsLenDis = trsDis.length;
  var startIndex = 6;
  var currentIndex = startIndex;
  var currentIndexChems = startIndex;
  var currentIndexDis = startIndex;
  const orgSelect = document.getElementById("organismSelect");
  
  function UrlExists(url) {
    var http = new XMLHttpRequest();
    http.open('HEAD', url, false);
    http.send();
    if (http.status != 404)
      return true;
    else
      return false;
  }

  trs.hide();
  trs.slice(0, startIndex).show();
  checkButton(trsLength,$("#participants-table tr:visible").length, btnMore, btnLess, btnLess1, tblGrad);
  trsChem.hide();
  trsChem.slice(0, startIndex).show();
  checkButton(trsLenChems, $("#chems-table tr:visible").length, btnMoreChems, btnLessChems, btnLessChems1, tblGradChems);
  trsDis.hide();
  trsDis.slice(0, startIndex).show();
  checkButton(trsLenDis, $("#dis-table tr:visible").length, btnMoreDis, btnLessDis, btnLessDis1, tblGradDis);
  checkButtonDesc();

  btnMoreDesc.click(function (e) {
    e.preventDefault();
    document.getElementById("caption").style.maxHeight = ""; 
    checkButtonDesc();
  });

  btnLessDesc.click(function (e) {
    e.preventDefault();
    document.getElementById("caption").style.maxHeight = "75px"; 
    checkButtonDesc();
  });

  function checkButtonDesc() {
    var currentHeight = $("#caption").height();
    var scrollHeight = document.getElementById("caption").scrollHeight;

    if (scrollHeight <= 75){ // no need to expand/collapse
      btnMoreDesc.hide();
      btnLessDesc.hide();
    } else {
      if (currentHeight > 75) { // time to collapse
        btnMoreDesc.hide();
        btnLessDesc.show();
      } else {                  // time to expand
        btnMoreDesc.show();
        btnLessDesc.hide();
      }
    } 
  }

  btnMore.click(function (e) {
    e.preventDefault();
    $("#participants-table tr").slice(startIndex, trsLength).show();
    checkButton(trsLength, $("#participants-table tr:visible").length, btnMore, btnLess, btnLess1,tblGrad);
  });
  btnLess.click(function (e) {
    e.preventDefault();
    orgSelect.selectedIndex = 0; // reset organism select
    $("#participants-table tr").slice(startIndex, trsLength).hide();
    checkButton(trsLength, $("#participants-table tr:visible").length, btnMore, btnLess, btnLess1,tblGrad);
  });
  btnLess1.click(function (e) {
    e.preventDefault();
    orgSelect.selectedIndex = 0; // reset organism select
    filterTable();
    $("#participants-table tr").slice(startIndex, trsLength).hide();
    checkButton(trsLength, $("#participants-table tr:visible").length, btnMore, btnLess, btnLess1,tblGrad);
  });

  btnMoreChems.click(function (e) {
    e.preventDefault();
    filterTable();
    $("#chems-table tr").slice(startIndex, trsLenChems).show();
    checkButton(trsLenChems, $("#chems-table tr:visible").length, btnMoreChems, btnLessChems, btnLessChems1, tblGradChems);
  });
  btnLessChems.click(function (e) {
    e.preventDefault();
    $("#chems-table tr").slice(startIndex, trsLenChems).hide();
    checkButton(trsLenChems, $("#chems-table tr:visible").length, btnMoreChems, btnLessChems, btnLessChems1, tblGradChems);
  });
  btnLessChems1.click(function (e) {
    e.preventDefault();
    $("#chems-table tr").slice(startIndex, trsLenChems).hide();
    checkButton(trsLenChems, $("#chems-table tr:visible").length, btnMoreChems, btnLessChems, btnLessChems1, tblGradChems);
  });

  btnMoreDis.click(function (e) {
    e.preventDefault();
    $("#dis-table tr").slice(startIndex, trsLenDis).show();
    checkButton(trsLenDis, $("#dis-table tr:visible").length, btnMoreDis, btnLessDis, btnLessDis1, tblGradDis);
  });
  btnLessDis.click(function (e) {
    e.preventDefault();
    $("#dis-table tr").slice(startIndex, trsLenDis).hide();
    checkButton(trsLenDis, $("#dis-table tr:visible").length, btnMoreDis, btnLessDis, btnLessDis1, tblGradDis);
  });
  btnLessDis.click(function (e) {
    e.preventDefault();
    $("#dis-table tr").slice(startIndex, trsLenDis).hide();
    checkButton(trsLenDis, $("#dis-table tr:visible").length, btnMoreDis, btnLessDis, btnLessDis1, tblGradDis);
  });

  function checkButton(trsLength, currentLength, btnM, btnL,btnL1,tblG) {
    if (trsLength > startIndex){
      if (currentLength == trsLength ) {
      btnM.hide();
      btnL.show();
      btnL1.show();
      tblG.style.display = 'none';
      } else {
      btnM.show();
      btnL.hide();
      btnL1.hide();
      tblG.style.display = 'block';
      }
    } else {
      btnM.hide();
      btnL.hide();
      btnL1.hide();
      tblG.style.display = 'none';
    }
  }

  const copyBtn = document.querySelector('.copy-btn');
  const toCopy = document.querySelector('.copy-btn').getAttribute('to-copy');

  copyBtn.addEventListener('click', () => {
    navigator.clipboard.writeText(toCopy)
      .then(() => {
        toCopy.value = '';
      })
      .catch(err => {
        console.log('Something went wrong', err);
      })
  });

    // Function to filter the table
    function filterTable() {
      const selectedOrganism = orgSelect.value;
      // Loop through each table row (starting from 1 to skip the header)
      newRowCount = 0;
      for (let i = 1; i < trs.length; i++) {
        const cell = trs[i].getElementsByTagName("td")[0];
        const organism = cell.textContent || cell.innerText;
        // Show or hide the row based on the selected organism
        if (organism === selectedOrganism || selectedOrganism === "All") {
          trs[i].style.display = "";
          newRowCount++;
        } else {
          trs[i].style.display = "none";
        }
      }
      checkButton(newRowCount, newRowCount, btnMore, btnLess, btnLess1, tblGrad);
    }

    const pvgBtn = document.querySelector('.pvg-btn');

    pvgBtn.addEventListener('click', () => {
      var toPvg = getPVgenes();
      navigator.clipboard.writeText(toPvg)
    .then(() => {
      toPvg = '';
    })
    .catch(err => {
      console.log('Something went wrong', err);
    })
    });

    // Function to prep datanodes for PV
    function getPVgenes() {
      var pvGenes = '<?xml version="1.0" encoding="UTF-8"?>'+
      '<Pathway xmlns="http://pathvisio.org/GPML/2013a" Name="untitled" Data-Source="COPIED">';
      var GraphIdNum = 1000;
      var CenterX = 100;
      var CenterY = 100;

      // for all rows for a given organism
      var myOrganism = orgSelect.value;
      if (myOrganism === "All") {
        myOrganism = orgSelect.options[1].value;
      }
      for (let i = 1; i < trs.length; i++) {
        const cell = trs[i].getElementsByTagName("td")[0];
        const organism = cell.textContent || cell.innerText;
        // Show or hide the row based on the selected organism
        if (organism === myOrganism ) {
          var GraphId = "a"+GraphIdNum;
          pvGenes = pvGenes + '<DataNode TextLabel="'+ trs[i].getElementsByTagName("td")[5].innerText +
          '" GraphId="'+ GraphId +'" Type="GeneProduct">'+
          '<Comment>OCR lexicon match: '+trs[i].getElementsByTagName("td")[3].innerText +'</Comment>'+
          '<Graphics CenterX="' + CenterX + '" CenterY="' + CenterY + '" Width="90" Height="25" ZOrder="33000" FontSize="12" Valign="Middle"/>'+  
          '<Xref Database="Entrez Gene" ID="' + trs[i].getElementsByTagName("td")[6].innerText +
           '" /> </DataNode>';
           GraphIdNum++;
           CenterY = CenterY + 25;
           var rowCheck = GraphIdNum % 10;
           if (rowCheck == 0) {
             CenterX = CenterX + 100;
             CenterY = 100;
           }
        } 
      }
      pvGenes = pvGenes + '<InfoBox CenterX="0.0" CenterY="0.0" /></Pathway>';
      return (pvGenes);
    }

    const drugBtn = document.querySelector('.drugstone-btn');
    if (drugBtn) {
     drugBtn.addEventListener('click', () => {
      var nodeList = getDrugstone();
      if (nodeList === '') {
        alert('No human genes found to query Drugst.one');
        return;
      }
      var drugstoneUrl = 'https://drugst.one/standalone?nodes='+nodeList+'&autofillEdges=true&activateNetworkMenuButtonAdjacentDrugs=true&interactionDrugProtein=NeDRex&licensedDatasets=true';
      window.open(drugstoneUrl, '_blank');
     });
    }

    // Function to prep genes for Drugst.one query
     function getDrugstone() {
      var drugstoneGenes = '';
      var queryCount = 0;
      // for all human rows 
      for (let i = 1; i < trs.length; i++) {
        const cell = trs[i].getElementsByTagName("td")[0];
        const organism = cell.textContent || cell.innerText;
        // Show or hide the row based on the selected organism
        if (organism === "Homo sapiens" & queryCount < 100 ) {
          drugstoneGenes = drugstoneGenes + 'ncbigene:' + trs[i].getElementsByTagName("td")[6].innerText + ',';
          queryCount++;
        } 
      }
      return (drugstoneGenes);
    }

</script>

      </div>
    </main><footer class="site-footer h-card"> 
  <data class="u-url" href="/"></data>

  <div class="wrapper">
      <a href="https://github.com/wikipathways/pfocr-database/commits" target="_blank">
        <img alt="GitHub last commit"
          src="https://img.shields.io/github/last-commit/wikipathways/pfocr-database?label=updated">
      </a>
      <a href="https://github.com/wikipathways/pfocr-database/deployments" target="_blank">
        <img alt="GitHub deployment state" 
        src="https://img.shields.io/github/deployments/wikipathways/pfocr-database/github-pages">
      </a>
      <p style="margin-bottom: 5px;" />

    <div class="row"> 

      <div class="col">
        <p style="width:300px; margin-bottom: 5px;">Mining the published literature for pathway figures and making their contents accessible for science.</p> 
        <a href="https://creativecommons.org/share-your-work/public-domain/cc0/" target="_blank"><img height="25px" src="/assets/img/cc0.jpg"/></a>
      </div>
 
      <div class="col"><ul class="social-media-list"><li><a href="https://github.com/wikipathways" target="_blank"><span style="color: #666;"><i class="fa fa-github fa"></i></span>&nbsp;&nbsp;<span style="color: #828282;">github</span></a></li><li><a href="https://www.twitter.com/wikipathways" target="_blank"><span style="color: #666;"><i class="fa fa-twitter fa"></i></span>&nbsp;&nbsp;<span style="color: #828282;">twitter</span></a></li><li><a href="https://github.com/wikipathways/wikipathways-help/discussions" target="_blank"><span style="color: #666;"><i class="fa fa-question-circle fa"></i></span>&nbsp;&nbsp;<span style="color: #828282;">forum</span></a></li></ul>
</div>
  
      <div class="col text-nowrap">
        <ul class="contact-list">
          <li><a href="/about.html">About</a></li>
          <li><a href="/help.html">Help</a></li>
          <li><a href="/help.html#analyze">Tools</a></li>
        </ul>
      </div>

      <div class="col text-nowrap">
        <ul class="contact-list">
          <li><a href="/stats.html">Statistics</a></li>
          <li><a href="/cite.html">How to cite</a></li>
          <li><a href="https://github.com/wikipathways/wikipathways-help/issues" target="_blank">Report a bug</a></li>
        </ul>
      </div>

    </div>
  </div>
</footer>
</body>
</html>
